
        <html>
            <head>
                <link rel='stylesheet' href='https://cdn.jsdelivr.net/gh/kognise/water.css@1.2.1/dist/light.css' />
            </head>
            <body>
                <h1>strains used in this study (7)</h1>
    <br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27455-v3.xml">elife-27455-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.27455.031</object-id><label>Table 2.</label><caption><title>Strains used in this study.</title></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3">Background</th><th colspan="2">Plasmids</th></tr><tr><th>Strain</th><th>Source</th><th>Relevant genotype</th><th>Plasmid 1</th><th>Plasmid 2</th></tr></thead><tbody><tr><td>VS115</td><td>V. Sourjik</td><td><inline-formula><mml:math id="inf279"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>YZ <inline-formula><mml:math id="inf280"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>FliC</td><td>pSJAB106</td><td>pZR1</td></tr><tr><td>VS104</td><td><xref ref-type="bibr" rid="bib91">Sourjik and Berg, 2002a</xref></td><td><inline-formula><mml:math id="inf281"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>CheYZ</td><td>pSJAB12</td><td>pBAD33</td></tr><tr><td>TSS58</td><td><italic>this work</italic></td><td><inline-formula><mml:math id="inf282"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>RBYZ <inline-formula><mml:math id="inf283"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>FliC</td><td>pSJAB106</td><td>pZR1</td></tr><tr><td>VS149</td><td><xref ref-type="bibr" rid="bib93">Sourjik and Berg, 2004</xref></td><td><inline-formula><mml:math id="inf284"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>RBYZ</td><td>pVS12</td><td>pVS33</td></tr><tr><td>VS124</td><td><xref ref-type="bibr" rid="bib20">Clausznitzer et al., 2010</xref></td><td><inline-formula><mml:math id="inf285"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>CheBYZ</td><td>pSJAB12</td><td>pVS112</td></tr><tr><td>VS124</td><td/><td><inline-formula><mml:math id="inf286"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>CheBYZ</td><td>pSJAB12</td><td>pVS97</td></tr><tr><td>VS124</td><td/><td><inline-formula><mml:math id="inf287"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>CheBYZ</td><td>pSJAB12</td><td>pVS91</td></tr><tr><td>UU2567</td><td><xref ref-type="bibr" rid="bib43">Kitanovic et al., 2015</xref></td><td><inline-formula><mml:math id="inf288"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>CheRBYZ,<inline-formula><mml:math id="inf289"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>MCP<sup>&#8224;</sup></td><td>pSJAB106</td><td>pPA114 Tsr</td></tr><tr><td>TSS1964</td><td><italic>this work</italic></td><td><inline-formula><mml:math id="inf290"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>CheRBYZ,<inline-formula><mml:math id="inf291"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>MCP FliC*</td><td>pSJAB106</td><td>pPA114 Tsr</td></tr><tr><td>UU2614</td><td>J.S. Parkinson</td><td>CheB <inline-formula><mml:math id="inf292"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>(4-345)</td><td>pTrc99a</td><td>pVS91,97,112</td></tr></tbody></table><table-wrap-foot><fn><p>All strains are descendants of E. coli K-12 HCB33 (RP437). In all FRET experiments, strains carry two plasmids and therefore&#160;confer resistance to chloramphenicol and ampicillin.</p><p><sup>&#8224;</sup>all five chemoreceptor genes <italic>tar tsr tap trg aer</italic> deleted.</p></fn><fn><p>*expresses sticky FliC filament (<xref ref-type="bibr" rid="bib80">Scharf et al., 1998</xref>)</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-30205-v1.xml">elife-30205-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.30205.016</object-id><label>Table 1.</label><caption><title>Strains used in this study.</title></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Strain</th><th valign="top">Number</th><th valign="top">Description</th><th valign="top">Source</th></tr></thead><tbody><tr><td colspan="4" valign="top"><italic>Pseudomonas aeruginosa</italic> strains</td></tr><tr><td valign="top">UCBPP-PA14</td><td valign="top"/><td valign="top">Clinical isolate UCBPP-PA14.</td><td valign="top"><xref ref-type="bibr" rid="bib51">Rahme et al. (1995)</xref></td></tr><tr><td valign="top">PA14 &#8710;<italic>phz</italic></td><td valign="top">LD24</td><td valign="top">PA14 with deletions in <italic>phzA1-G1</italic> and <italic>phzA2-G2</italic> <break/>operons.</td><td valign="top"><xref ref-type="bibr" rid="bib18">Dietrich et al., 2006a</xref></td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN1</italic></td><td valign="top">LD1784</td><td valign="top">PA14 with deletion in PA14_44370.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN2</italic></td><td valign="top">LD1614</td><td valign="top">PA14 with deletion in PA14_44340.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN3</italic></td><td valign="top">LD1620</td><td valign="top">PA14 with deletion in PA14_40510.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN4</italic></td><td valign="top">LD2833</td><td valign="top">PA14 with deletion in PA14_10500.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN1</italic> <break/>&#8710;<italic>ccoN2</italic></td><td valign="top">LD1888</td><td valign="top">PA14 with deletions in PA14_44370 and PA14_44340. Made by mating pLD1610 into LD1784.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN1</italic> <break/>&#8710;<italic>ccoN4</italic></td><td valign="top">LD1951</td><td valign="top">PA14 with deletions in PA14_44370 and PA14_10500. Made by mating pLD1264 into LD1784.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN2</italic> <break/>&#8710;<italic>ccoN4</italic></td><td valign="top">LD1692</td><td valign="top">PA14 with deletions in PA14_44340 and PA14_10500. Made by mating pLD1264 into LD1614.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN3</italic> <break/>&#8710;<italic>ccoN4</italic></td><td valign="top">LD1649</td><td valign="top">PA14 with deletions in PA14_40510 and PA14_10500. Made by mating pLD1264 into LD1620.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN1</italic> <break/>&#8710;<italic>ccoN2</italic> &#8710;<italic>ccoN3</italic></td><td valign="top">LD1977</td><td valign="top">PA14 with deletions in PA14_443470, PA14_44340, and PA14_40510. Made by mating pLD1616 into LD1888.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN1</italic> <break/>&#8710;<italic>ccoN2</italic> &#8710;<italic>ccoN4</italic></td><td valign="top">LD1976</td><td valign="top">PA14 with deletions in PA14_443470, PA14_44340, and PA14_10500. Made by mating pLD1264 into LD1888.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN1</italic> <break/>&#8710;<italic>ccoN2</italic> &#8710;<italic>ccoN4</italic> <break/>&#8710;<italic>ccoN3</italic></td><td valign="top">LD2020</td><td valign="top">PA14 with deletions in PA14_443470, PA14_44340, PA14_10500, and PA14_40510. Made by mating pLD1264 into LD1977.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>cco1cco2</italic></td><td valign="top">LD1933</td><td valign="top">PA14 with both <italic>cco</italic> operons (PA14_44340-PA14_44400) deleted simultaneously.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>cox</italic> &#8710;<italic>cyo</italic> &#8710;<italic>cio</italic></td><td valign="top">LD2587</td><td valign="top">PA14 with deletions in PA14_01290&#8211;01320 (<italic>cox</italic>/<italic>aa</italic>3 operon), PA14_47150&#8211;47210 (<italic>cyo</italic>/<italic>bo</italic>3 operon), and PA14_13030&#8211;13040 (<italic>cio</italic> operon). Made by mating pLD1966, pLD1967, and pLD2044, in that order, to PA14.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>hcn</italic></td><td valign="top">LD2827</td><td valign="top">PA14 with deletion in <italic>hcnABC</italic> operon (PA14_36310&#8211;36330).</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>phz</italic> &#8710;<italic>hcn</italic></td><td valign="top">LD2828</td><td valign="top">PA14 with deletions in <italic>phzA1-G1, phzA2-G2,</italic> and <italic>hcnABC</italic> operons. Made by mating pLD2791 into LD24.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN4</italic> &#8710;<italic>hcn</italic></td><td valign="top">LD2829</td><td valign="top">PA14 with deletions in PA14_10500 and <italic>hcnABC</italic> operon. Made by mating pLD2791 into LD2833.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN1</italic> <break/>&#8710;<italic>ccoN2</italic> &#8710;<italic>hcn</italic></td><td valign="top">LD2830</td><td valign="top">PA14 with deletions in PA14_44370, PA14_44340, and <italic>hcnABC</italic> operon. Made by mating pLD2791 into LD1888.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN1</italic> <break/>&#8710;<italic>ccoN2</italic> &#8710;<italic>ccoN4</italic> &#8710;<italic>hcn</italic></td><td valign="top">LD2831</td><td valign="top">PA14 with deletions in PA14_44370, PA14_44340, PA14_10500 and <italic>hcnABC</italic> operon. Made by mating pLD2791 into LD1976.</td><td valign="top">this study</td></tr><tr><td colspan="4" valign="top"><italic>Pseudomonas aeruginosa</italic> strains</td></tr><tr><td valign="top">PA14 &#8710;<italic>cco1cco2</italic> <break/>&#8710;<italic>hcn</italic></td><td valign="top">LD2832</td><td valign="top">PA14 with deletions in <italic>cco1</italic>, <italic>cco2</italic>, and <italic>hcnABC</italic> operons. Made by mating pLD2791 into LD1933.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 <italic>gacA</italic>::<italic>Tn</italic></td><td valign="top">LD1560</td><td valign="top">MAR2xT7 transposon insertion into PA14_30650.</td><td valign="top"><xref ref-type="bibr" rid="bib41">Liberati et al. (2006)</xref></td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN4</italic>::<italic>ccoN4</italic></td><td valign="top">LD1867</td><td valign="top">PA14 &#8710;<italic>ccoN4</italic> strain with wild-type <italic>ccoN4</italic> complemented back into the site of deletion. Made by mating pLD1853 into LD2833.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN1</italic> <break/>&#8710;<italic>ccoN2</italic> <break/>&#8710;<italic>ccoN4</italic>::<italic>ccoN4</italic></td><td valign="top">LD2576</td><td valign="top">PA14 &#8710;<italic>ccoN1</italic> &#8710;<italic>ccoN2</italic> &#8710;<italic>ccoN4</italic> strain with wild-type <italic>ccoN4</italic> complemented back into the site of deletion. Made by mating pLD1853 into LD1976.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 MCS-gfp</td><td valign="top">LD2820</td><td valign="top">PA14 without a promoter driving <italic>gfp</italic> expression.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 P<italic>cco-1</italic>-gfp</td><td valign="top">LD2784</td><td valign="top">PA14 with promoter of <italic>cco1</italic> operon driving <italic>gfp</italic> expression.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 P<italic>cco-2</italic>-gfp</td><td valign="top">LD2786</td><td valign="top">PA14 with promoter of <italic>cco2</italic> operon driving <italic>gfp</italic> expression.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 P<italic>ccoN4</italic>-gfp</td><td valign="top">LD2788</td><td valign="top">PA14 with promoter of <italic>ccoN4Q4</italic> operon driving <italic>gfp</italic> expression.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14-yfp</td><td valign="top">LD2780</td><td valign="top">WT PA14 constitutively expressing <italic>eyfp</italic>.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN1</italic> <break/>&#8710;<italic>ccoN2</italic>-yfp</td><td valign="top">LD2013</td><td valign="top">PA14 &#8710;<italic>ccoN1</italic> &#8710;<italic>ccoN2</italic> constitutively expressing <italic>eyfp</italic>. Made by mating pAKN69 into LD1888.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN4</italic>-yfp</td><td valign="top">LD2834</td><td valign="top">PA14 &#8710;<italic>ccoN4</italic> constitutively expressing <italic>eyfp</italic>. Made by mating pAKN69 into LD2833.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>ccoN1</italic> <break/>&#8710;<italic>ccoN2</italic> &#8710;<italic>ccoN4</italic>-yfp</td><td valign="top">LD2136</td><td valign="top">PA14 &#8710;<italic>ccoN1</italic> &#8710;<italic>ccoN2</italic> &#8710;<italic>ccoN4</italic> constitutively expressing <italic>eyfp</italic>. Made by mating pAKN69 into LD1976.</td><td valign="top">this study</td></tr><tr><td valign="top">PA14 &#8710;<italic>cco1cco2</italic>-yfp</td><td valign="top">LD2012</td><td valign="top">PA14 &#8710;<italic>cco1cco2</italic> constitutively expressing <italic>eyfp</italic>. Made by mating pAKN69 into LD1933.</td><td valign="top">this study</td></tr><tr><td colspan="4" valign="top"><italic>Escherichia coli</italic> strains</td></tr><tr><td valign="top">UQ950</td><td valign="top">LD44</td><td valign="top"><italic>E. coli</italic> DH5 &#955;pir strain for cloning. F-&#8710;(argF- lac) 169&#966;80 dlacZ58(&#8710;M15) glnV44(AS) rfbD1 gyrA96(NaIR) recA1 endA1 spoT thi-1 hsdR17 deoR &#955;pir+</td><td valign="top">D. Lies, Caltech</td></tr><tr><td valign="top">BW29427</td><td valign="top">LD661</td><td valign="top">Donor strain for conjugation. thrB1004 pro thi rpsL hsdS lacZ &#8710;M15RP4-1360 &#8710;(araBAD)567 <break/>&#8710;dapA1314::[erm pir(wt)]</td><td valign="top">W. Metcalf, University of Illinois</td></tr><tr><td valign="top">&#946;2155</td><td valign="top">LD69</td><td valign="top">Helper strain. thrB1004 pro thi strA hsdsS lacZ&#8710;M15 (F&#8217;lacZ&#8710;M15 lacIq traD36 proA&#160;+&#160;proB&#160;+&#160;) &#8710;dapA::erm (Ermr)pir::RP4 [::kan (Kmr) from SM10]</td><td valign="top"><xref ref-type="bibr" rid="bib16">Dehio and Meyer (1997)</xref></td></tr><tr><td valign="top">S17-1</td><td valign="top">LD2901</td><td valign="top">StrR, TpR, F&#8722; RP4-2-Tc::Mu aphA::Tn7 recA &#955;pir lysogen</td><td valign="top"><xref ref-type="bibr" rid="bib61">Simon et al. (1983)</xref></td></tr><tr><td colspan="4" valign="top"><italic>Saccharomyces cerevisiae</italic> strains</td></tr><tr><td valign="top">InvSc1</td><td valign="top">LD676</td><td valign="top">MATa/MATalpha leu2/leu2 trp1-289/trp1-289 ura3&#8722;52/ura3-52 his3-&#8710;1/his3-&#8710;1</td><td valign="top">Invitrogen</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32057-v1.xml">elife-32057-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app1table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.32057.032</object-id><label>Appendix 1&#8212;table 1.</label><caption><title>Strains used in this study.</title></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Strain</th><th valign="top">Relevant Genotype/markers</th><th valign="top">Source</th></tr></thead><tbody><tr><td valign="top">SH1000</td><td valign="top">Functional <italic>rsbU</italic><sup>+</sup> derivative of 8325&#8211;4</td><td valign="top">(<xref ref-type="bibr" rid="bib21">Horsburgh et al., 2002b</xref>)</td></tr><tr><td>RN4220</td><td>Restriction deficient transformation recipient</td><td>(<xref ref-type="bibr" rid="bib30">Kreiswirth et al., 1983</xref>)</td></tr><tr><td valign="top">CYL316</td><td valign="top"><italic>S. aureus</italic> RN4220 pCL112&#916;19 (cm)</td><td valign="top">(<xref ref-type="bibr" rid="bib37">Lee et al., 1991</xref>)</td></tr><tr><td valign="top">JGL227</td><td valign="top"><italic>S. aureus</italic> SH1000 <italic>ezrA-gfp+</italic> (ery)</td><td valign="top">(<xref ref-type="bibr" rid="bib55">Steele et al., 2011</xref>)</td></tr><tr><td valign="top">SH4386</td><td valign="top"><italic>S. aureus</italic> SH1000 <italic>ezrA-eyfp</italic> (kan)</td><td valign="top">This study</td></tr><tr><td valign="top">SH4388</td><td valign="top"><italic>S. aureus</italic> SH1000 <italic>ezrA-eyfp &#916;ezrA</italic> (kan, tet)</td><td valign="top">This study</td></tr><tr><td valign="top">SH4603</td><td valign="top"><italic>S. aureus</italic> SH1000 <italic>ezrA-meyfp</italic> (kan)</td><td valign="top">This study</td></tr><tr><td valign="top">SH4604</td><td valign="top"><italic>S. aureus</italic> SH1000 <italic>ezrA-meyfp &#916;ezrA</italic> (kan, tet)</td><td valign="top">This study</td></tr><tr><td valign="top">SH4639</td><td valign="top"><italic>S. aureus</italic> SH1000 <italic>ezrA-gfp</italic> (kan)</td><td valign="top">This study</td></tr><tr><td valign="top">SH4640</td><td valign="top"><italic>S. aureus</italic> SH1000 <italic>ezrA-gfp &#916;ezrA</italic> (kan, tet)</td><td valign="top">This study</td></tr><tr><td valign="top">SH4641</td><td valign="top"><italic>S. aureus</italic> SH1000 <italic>ezrA-snap</italic> (kan)</td><td valign="top">This study</td></tr><tr><td valign="top">SH4642</td><td valign="top"><italic>S. aureus</italic> SH1000 <italic>ezrA-snap &#916;ezrA</italic> (kan, tet)</td><td valign="top">This study</td></tr><tr><td valign="top">SH4652</td><td valign="top"><italic>S. aureus</italic> SH1000 <italic>ezrA-eyfp &#916;ezrA</italic> pCQ11-FtsZ-SNAP (kan, tet, ery)</td><td valign="top">This study</td></tr><tr><td valign="top">SH4665</td><td valign="top"><italic>S. aureus</italic> SH1000 pCQ11-FtsZ-eYFP (ery)</td><td valign="top">This study</td></tr><tr><td valign="top">NE679</td><td valign="top"><italic>S. aureus</italic> JE2 with transposon insertion in <italic>pbp4</italic> (ery)</td><td valign="top">(<xref ref-type="bibr" rid="bib14">Fey et al., 2013</xref>)</td></tr><tr><td valign="top">SH4425</td><td valign="top"><italic>S. aureus</italic> SH1000 <italic>pbp4</italic> (ery)</td><td valign="top">This study</td></tr><tr><td valign="top">N315</td><td valign="top">Methicillin-resistant <italic>S. aureus</italic></td><td valign="top">(<xref ref-type="bibr" rid="bib31">Kuroda et al., 2001</xref>)</td></tr><tr><td valign="top">SU492</td><td valign="top"><italic>B. subtilis</italic> SU5 P<italic><sub>xyl</sub>-ftsZ-yfp</italic> (spec)</td><td valign="top">(<xref ref-type="bibr" rid="bib43">Monahan et al., 2014</xref>)</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-34878-v1.xml">elife-34878-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.34878.007</object-id><label>Table 1.</label><caption><title>Strains used in this study.</title></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Strain Abbreviation*</th><th valign="top">Ancestor (source)<sup>&#8224;</sup></th><th valign="top">Genotype</th><th valign="top"># UAG Codons<sup>&#8225;</sup></th><th valign="top">RF1 Status<sup>&#167;</sup></th><th valign="top">Ribosomal rescue gene deletion</th><th valign="top">ssrA tag Status<sup>#</sup></th><th valign="top">Investigated in</th></tr></thead><tbody><tr><td valign="top">GRO.DD.<italic>prfA+</italic></td><td valign="top">GRO.AA (this study)</td><td valign="top">&#916;<italic>mutS</italic>:<italic>zeo</italic>.&#916;(<italic>ybhB-bioAB</italic>):[&#955;cI857.&#916;(<italic>cro-ea59</italic>):<italic>tetR-bla</italic>]</td><td valign="top">0</td><td valign="top">+RF1</td><td valign="top">n/a</td><td valign="top">DD</td><td valign="top">GFP expression for mass spectrometry (<xref ref-type="fig" rid="fig2">Figure 2</xref>)</td></tr><tr><td valign="top">GRO.DD</td><td valign="top">GRO.AA (this study)</td><td valign="top">&#916;<italic>mutS</italic>:<italic>zeo</italic>.&#916;(<italic>ybhB-bioAB</italic>):[&#955;cI857.&#916;(<italic>cro-ea59</italic>):<italic>tetR-bla</italic>], &#916;<italic>prfA</italic>, &#916;<italic>tolC</italic></td><td valign="top">0</td><td valign="top">&#8710;RF1</td><td valign="top">n/a</td><td valign="top">DD</td><td valign="top">GFP expression for mass spectrometry (<xref ref-type="fig" rid="fig2">Figure 2</xref>)</td></tr><tr><td valign="top">ECNR2.AA</td><td valign="top"><italic>E. coli</italic> MG1655 (<xref ref-type="bibr" rid="bib69">Wang et al., 2009</xref>)</td><td valign="top">MG1655 &#916;<italic>mutS</italic>:<italic>zeo</italic>.&#916;(<italic>ybhB-bioAB</italic>):[&#955;cI857.&#916;(<italic>cro-ea59</italic>):<italic>tetR-bla</italic>]</td><td valign="top">321</td><td valign="top">+RF1</td><td valign="top">n/a</td><td valign="top">AA</td><td valign="top">Fitness, conjugation, and viral infection (<xref ref-type="fig" rid="fig3">Figures 3</xref> and <xref ref-type="fig" rid="fig4">4</xref>)</td></tr><tr><td valign="top">GRO.AA</td><td valign="top">ECNR2.AA (<xref ref-type="bibr" rid="bib39">Lajoie et al., 2013b</xref>)</td><td valign="top">&#916;<italic>mutS</italic>:<italic>zeo</italic>.&#916;(<italic>ybhB-bioAB</italic>):[&#955;cI857.&#916;(<italic>cro-ea59</italic>):<italic>tetR-bla</italic>], &#916;<italic>prfA</italic>, &#916;<italic>tolC</italic></td><td valign="top">0</td><td valign="top">&#8710;RF1</td><td valign="top">n/a</td><td valign="top">AA</td><td valign="top">Fitness, conjugation, and viral infection (<xref ref-type="fig" rid="fig3">Figures 3</xref> and <xref ref-type="fig" rid="fig4">4</xref>)</td></tr><tr><td valign="top">GRO.AA.&#8710;<italic>ssrA</italic></td><td valign="top">GRO.AA (this study)</td><td valign="top">&#916;<italic>mutS</italic>:<italic>zeo</italic>.&#916;(<italic>ybhB-bioAB</italic>):[&#955;cI857.&#916;(<italic>cro-ea59</italic>):<italic>tetR-bla</italic>], &#916;<italic>prfA</italic>, &#916;<italic>tolC</italic></td><td valign="top">0</td><td valign="top">&#8710;RF1</td><td valign="top">&#8710;<italic>ssrA</italic></td><td valign="top">AA</td><td valign="top">Fitness, conjugation, and viral infection (<xref ref-type="fig" rid="fig3">Figures 3</xref> and <xref ref-type="fig" rid="fig4">4</xref>)</td></tr><tr><td valign="top">GRO.AA.&#8710;<italic>arfA</italic></td><td valign="top">GRO.AA (this study)</td><td valign="top">&#916;<italic>mutS</italic>:<italic>zeo</italic>.&#916;(<italic>ybhB-bioAB</italic>):[&#955;cI857.&#916;(<italic>cro-ea59</italic>):<italic>tetR-bla</italic>], &#916;<italic>prfA</italic>, &#916;<italic>tolC</italic></td><td valign="top">0</td><td valign="top">&#8710;RF1</td><td valign="top">&#8710;<italic>arfA</italic></td><td valign="top">AA</td><td valign="top">Fitness, conjugation, and viral infection (<xref ref-type="fig" rid="fig3">Figures 3</xref> and <xref ref-type="fig" rid="fig4">4</xref>)</td></tr><tr><td valign="top">GRO.AA.&#8710;<italic>arfB</italic></td><td valign="top">GRO.AA (this study)</td><td valign="top">&#916;<italic>mutS</italic>:<italic>zeo</italic>.&#916;(<italic>ybhB-bioAB</italic>):[&#955;cI857.&#916;(<italic>cro-ea59</italic>):<italic>tetR-bla</italic>], &#916;<italic>prfA</italic>, &#916;<italic>tolC</italic></td><td valign="top">0</td><td valign="top">&#8710;RF1</td><td valign="top">&#8710;<italic>arfB</italic></td><td valign="top">AA</td><td valign="top">Fitness, conjugation, and viral infection (<xref ref-type="fig" rid="fig3">Figures 3</xref> and <xref ref-type="fig" rid="fig4">4</xref>)</td></tr><tr><td valign="top">GRO.AA.&#8710;<italic>ssrA.</italic>&#8710;<italic>arfB</italic></td><td valign="top">GRO.AA (this study)</td><td valign="top">&#916;<italic>mutS</italic>:<italic>zeo</italic>.&#916;(<italic>ybhB-bioAB</italic>):[&#955;cI857.&#916;(<italic>cro-ea59</italic>):<italic>tetR-bla</italic>], &#916;<italic>prfA</italic>, &#916;<italic>tolC</italic></td><td valign="top">0</td><td valign="top">&#8710;RF1</td><td valign="top">&#8710;<italic>ssrA,</italic> &#8710;<italic>arfB</italic></td><td valign="top">AA</td><td valign="top">Fitness, conjugation, and viral infection (<xref ref-type="fig" rid="fig3">Figures 3</xref> and <xref ref-type="fig" rid="fig4">4</xref>)</td></tr><tr><td valign="top">GRO.AA.&#8710;<italic>arfA.</italic> &#8710;<italic>arfB</italic></td><td valign="top">GRO.AA (this study)</td><td valign="top">&#916;<italic>mutS</italic>:<italic>zeo</italic>.&#916;(<italic>ybhB-bioAB</italic>):[&#955;cI857.&#916;(<italic>cro-ea59</italic>):<italic>tetR-bla</italic>], &#916;<italic>prfA</italic>, &#916;<italic>tolC</italic></td><td valign="top">0</td><td valign="top">&#8710;RF1</td><td valign="top">&#8710;<italic>arfA,</italic> &#8710;<italic>arfB</italic></td><td valign="top">AA</td><td valign="top">Fitness, conjugation, and viral infection (<xref ref-type="fig" rid="fig3">Figures 3</xref> and <xref ref-type="fig" rid="fig4">4</xref>)</td></tr></tbody></table><table-wrap-foot><fn><p>*All strains derived from ECNR2, as described in <xref ref-type="bibr" rid="bib69">Wang et al. (2009)</xref>.</p><p>&#8224;See <bold>Key Resources Table</bold> for additional information on strains and sources. The GenBank accession number for <italic>E. coli</italic> MG1655 is U00096, and the GenBank accession number for GRO.AA is CP006698.</p></fn><fn><p>&#8225; Out of a total of 321 in the original ECNR2 strain.</p><p>&#167;RF1 terminates translation at UAG and UAA. Deletion of RF1 eliminates recognition of UAG during translation; translational termination continues through RF2, which recognizes UAA and UGA.</p></fn><fn><p>#The <italic>ssrA</italic> gene encodes the tmRNA, which appends the <italic>ssrA</italic> degradation tag to stalled ribosomes. The wild-type sequence is AANDENYAL<bold><underline>AA</underline></bold>; mutation of the C-terminus to AANDENYAL<bold><underline>DD</underline></bold> slows degradation of peptides to enable detection by mass spectrometry.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-48508-v2.xml">elife-48508-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.48508.027</object-id><label>Table 1.</label><caption><title>Strains used in this study.</title><p>&#8216;Aggregation&#8217; was determined as &#8216;yes&#8217; if the majority of cells (&gt;~50%) were observed in clusters of more than five individual cells, and &#8216;no&#8217; otherwise. Survival level was estimated as follows: &#8216;low&#8217;: almost no survival (&lt;3% of cells); &#8216;medium&#8217;: survival of 3% to 50% of the cells; &#8216;high&#8217;:&gt;50% of all cells survived.</p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="bottom">Genus</th><th valign="bottom">Species</th><th valign="bottom">Strain</th><th valign="bottom">Gram +/-</th><th valign="bottom">Major habitat</th><th valign="bottom">Aggregation</th><th valign="bottom">Survival at 24 hr</th><th valign="top">Gifted from</th></tr></thead><tbody><tr><td valign="bottom"><italic>Pseudomonas</italic></td><td valign="bottom"><italic>syringae</italic></td><td valign="bottom">B728a</td><td valign="bottom">-</td><td valign="bottom">phyllosphere</td><td valign="bottom">No</td><td valign="bottom">low</td><td valign="top">S. Lindow</td></tr><tr><td valign="bottom"><italic>Pseudomonas</italic></td><td valign="bottom"><italic>syringae</italic></td><td valign="bottom">DC3000</td><td valign="bottom">-</td><td valign="bottom">phyllosphere</td><td valign="bottom">No</td><td valign="bottom">low</td><td valign="top">O. Bahar</td></tr><tr><td valign="bottom"><italic>Pseudomonas</italic></td><td valign="bottom"><italic>fluorescens</italic></td><td valign="bottom">A506</td><td valign="bottom">-</td><td valign="bottom">phyllosphere</td><td valign="bottom">Yes</td><td valign="bottom">medium</td><td valign="top">S. Lindow</td></tr><tr><td valign="bottom"><italic>Pseudomonas</italic></td><td valign="bottom"><italic>fluorescens</italic></td><td valign="bottom">NT133</td><td valign="bottom">-</td><td valign="bottom">rhizosphere</td><td valign="bottom">Yes</td><td valign="bottom">low</td><td valign="top">D. Minz</td></tr><tr><td valign="bottom"><italic>Pseudomonas</italic></td><td valign="bottom"><italic>putida</italic></td><td valign="bottom">KT2440</td><td valign="bottom">-</td><td valign="bottom">soil</td><td valign="bottom">Yes</td><td valign="bottom">medium</td><td valign="top">Purchased from ATCC</td></tr><tr><td valign="bottom"><italic>Pseudomonas</italic></td><td valign="bottom"><italic>putida</italic></td><td valign="bottom">KT2442</td><td valign="bottom">-</td><td valign="bottom">soil</td><td valign="bottom">Yes</td><td valign="bottom">medium</td><td valign="top">Y. Friedman</td></tr><tr><td valign="bottom"><italic>Pseudomonas</italic></td><td valign="bottom"><italic>putida</italic></td><td valign="bottom">IsoF101</td><td valign="bottom">-</td><td valign="bottom">soil</td><td valign="bottom">Yes</td><td valign="bottom">medium</td><td valign="top">L. Eberl</td></tr><tr><td valign="bottom"><italic>Pseudomonas</italic></td><td valign="bottom"><italic>citronellolis</italic></td><td valign="bottom">13674 (ATCC)</td><td valign="bottom">-</td><td valign="bottom">soil</td><td valign="bottom">Yes</td><td valign="bottom">low</td><td valign="top">Y. Friedman</td></tr><tr><td valign="bottom"><italic>Pseudomonas</italic></td><td valign="bottom"><italic>aurantiaca</italic></td><td valign="bottom">33663 (ATCC)</td><td valign="bottom">-</td><td valign="bottom">soil</td><td valign="bottom">Yes</td><td valign="bottom">medium</td><td valign="top">Y. Friedman</td></tr><tr><td valign="bottom"><italic>Pseudomonas</italic></td><td valign="bottom"><italic>veronii</italic></td><td valign="bottom">700474 (ATCC)</td><td valign="bottom">-</td><td valign="bottom">water</td><td valign="bottom">Yes</td><td valign="bottom">high</td><td valign="top">Y. Friedman</td></tr><tr><td valign="bottom"><italic>Pantoea</italic></td><td valign="bottom"><italic>agglomerans</italic></td><td valign="bottom">299 r</td><td valign="bottom">-</td><td valign="bottom">phyllosphere</td><td valign="bottom">No</td><td valign="bottom">low</td><td valign="top">S. Lindow</td></tr><tr><td valign="bottom"><italic>Pantoea</italic></td><td valign="bottom"><italic>agglomerans</italic></td><td valign="bottom">BRT98</td><td valign="bottom">-</td><td valign="bottom">soil</td><td valign="bottom">Yes</td><td valign="bottom">high</td><td valign="top">Z. Cardon</td></tr><tr><td valign="bottom"><italic>Escherichia</italic></td><td valign="bottom"><italic>coli</italic></td><td valign="bottom">K-12 MG1655</td><td valign="bottom">-</td><td valign="bottom">human gut</td><td valign="bottom">No</td><td valign="bottom">medium</td><td valign="top">Y. Helman</td></tr><tr><td valign="bottom"><italic>Xanthomoas</italic></td><td valign="bottom"><italic>campestris</italic></td><td valign="bottom">85&#8211;10</td><td valign="bottom">-</td><td valign="bottom">phyllosphere</td><td valign="bottom">No</td><td valign="bottom">low</td><td valign="top">G. Sessa</td></tr><tr><td valign="bottom"><italic>Burkholderia</italic></td><td valign="bottom"><italic>cenocepacia</italic></td><td valign="bottom">H111</td><td valign="bottom">-</td><td valign="bottom">human</td><td valign="bottom">Yes</td><td valign="bottom">low</td><td valign="top">Y. Helman</td></tr><tr><td valign="bottom"><italic>Acidovorax</italic></td><td valign="bottom"><italic>citrulli</italic></td><td valign="bottom">M6</td><td valign="bottom">-</td><td valign="bottom">phyllosphere</td><td valign="bottom">No</td><td valign="bottom">low</td><td valign="top">S. Burdman</td></tr><tr><td valign="bottom"><italic>Bacillus</italic></td><td valign="bottom"><italic>subtilis</italic></td><td valign="bottom">3610</td><td valign="bottom">+</td><td valign="bottom">soil</td><td valign="bottom">No</td><td valign="bottom">low</td><td valign="top">Y. Helman</td></tr><tr><td valign="bottom"><italic>Clavibacter</italic></td><td valign="bottom"><italic>michiganensis</italic></td><td valign="bottom"/><td valign="bottom">+</td><td valign="bottom">soil</td><td valign="bottom">Yes</td><td valign="bottom">low</td><td valign="top">S. Burdman</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-50749-v1.xml">elife-50749-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><label>Table 4.</label><caption><title>Strains used in this study.</title></caption><table frame="hsides" rules="groups"><tbody><tr><td valign="bottom">DDY2733*</td><td valign="bottom"><italic>MATa his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 ABP1-GFP::HIS3</italic></td></tr><tr><td valign="bottom">DDY2734*</td><td valign="bottom"><italic>MATa his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 SLA1-GFP::HIS3</italic></td></tr><tr><td valign="bottom">DDY2735*</td><td valign="bottom"><italic>MAT&#945; his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 PAN1-GFP::HIS3</italic></td></tr><tr><td valign="bottom">DDY2795*</td><td valign="bottom"><italic>MATa his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 SLA2-GFP::HIS3</italic></td></tr><tr><td valign="bottom">DDY2736*</td><td valign="bottom"><italic>MATa his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 LAS17-GFP::HIS3</italic></td></tr><tr><td valign="bottom">DDY3201*</td><td valign="bottom"><italic>MATa his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 VRP1-GFP::HIS3</italic></td></tr><tr><td valign="bottom">DDY2752*</td><td valign="bottom"><italic>MAT&#945; his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 ARC15-GFP::HIS3</italic></td></tr><tr><td valign="bottom">DDY2960*</td><td valign="bottom"><italic>MAT&#945; his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 SAC6-GFP::HIS3</italic></td></tr><tr><td valign="bottom">YSY4303</td><td valign="bottom"><italic>MAT&#945; his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 LAS17-sfGFP::HIS3</italic></td></tr><tr><td valign="bottom">YSY4305</td><td valign="bottom"><italic>MAT&#945; his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 MYO5-sfGFP::HIS3</italic></td></tr><tr><td valign="bottom">YSY4307</td><td valign="bottom"><italic>MAT&#945; his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 SLA1-sfGFP::HIS3</italic></td></tr><tr><td valign="bottom">YSY4315</td><td valign="bottom"><italic>MAT&#945; his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 CSE4-sfGFP::HIS3</italic></td></tr><tr><td valign="bottom">YSY4218</td><td valign="bottom"><italic>MAT&#945; his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 MYO3-GFP::HIS3</italic></td></tr><tr><td valign="bottom">YSY4299</td><td valign="bottom"><italic>MAT a his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 MYO3-GFP::HIS, ABP1-RFP::HygMX</italic></td></tr><tr><td valign="bottom">YSY4300</td><td valign="bottom"><italic>MAT &#945; his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 MYO5-GFP::HIS, ABP1-RFP::HygMX</italic></td></tr><tr><td valign="bottom">YSY4125</td><td valign="bottom"><italic>MAT a his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 GFP-LAS17::KanMX6, ABP1-RFP::HygMX</italic></td></tr><tr><td valign="bottom">YSY4126</td><td valign="bottom"><italic>MAT &#945; his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 GFP-LAS17::KanMX6, MYO5-RFP::HIS</italic></td></tr><tr><td valign="bottom">YSY4291</td><td valign="bottom"><italic>MAT &#945; his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 Myo5-GFP::KanMX6, MYO3-GFP::HIS</italic></td></tr><tr><td valign="bottom">YSY4120</td><td valign="bottom"><italic>MAT &#945; his3-&#8710;200 leu2-3, 112 lys2-801 ura3-52 GFP-LAS17::KanMX6</italic></td></tr><tr><td valign="bottom">YSY4257</td><td valign="bottom"><italic>h+ leu1-32 ura4-D18 his3-D1 ade6-M210 pan1-mEGFP-kanMX6 fim1mcherry-natRMX6</italic></td></tr><tr><td valign="bottom">YSY4266</td><td valign="bottom"><italic>h- or h+ leu1-32 ura4-D18 his3-D1 ade6 kanMX6- Pwsp1-mEGFP-wsp1 fim1mcherry-natRMX6</italic></td></tr><tr><td valign="bottom">YSY4269</td><td valign="bottom"><italic>h- or h+ leu1-32 ura4-D18 his3-D1 ade6 kanMX6- Pmyo1-mEGFP-myo1 fim1mcherry-natRMX6</italic></td></tr><tr><td valign="bottom">YSY4335</td><td valign="bottom"><italic>h+ leu1-32 ura4-D18 his3-D1 ade6-M210 fim1mcherry-natRMX6 wsp1-CA&#8710;-GFP-kanMX6</italic></td></tr><tr><td valign="bottom">YSY4345</td><td valign="bottom"><italic>h- or h+ leu1-32 ura4-D18 his3-D1 ade6-M210 myo1-ca&#8710;-GFP-kanMX6 fim1-mcherry-natRMX6</italic></td></tr><tr><td valign="bottom">YSY4346</td><td valign="bottom"><italic>h- leu1-32 ura4-D18 his3-D1 ade6-M210 fim1mcherry-natRMX6, hob1-GFP-kanMX6</italic></td></tr><tr><td valign="bottom">YSY4346</td><td valign="bottom"><italic>h+ leu1-32 ura4-D18 his3-D1 ade6-M210 hob1-GFP-kanMX6</italic></td></tr><tr><td valign="bottom">TP401**</td><td valign="bottom"><italic>h+ leu1-32 ura4-D18 his3-D1 ade6-M210 pan1-mEGFP-kanMX6</italic></td></tr><tr><td valign="bottom">TP398**</td><td valign="bottom"><italic>h+ leu1-32 ura4-D18 his3-D1 ade6-M210 end4-mEGFP-kanMX6</italic></td></tr><tr><td valign="bottom">TP203**</td><td valign="bottom"><italic>h- leu1-32 ura4-D18 his3-D1 ade6-M210 kanMX6- Pwsp1-mEGFP-wsp1</italic></td></tr><tr><td valign="bottom">TP186**</td><td valign="bottom"><italic>h- or h+ leu1-32 ura4-D18 his3-D1 ade6-M210 vrp1-EGFP-kanMX6</italic></td></tr><tr><td valign="bottom">TP198**</td><td valign="bottom"><italic>h- leu1-32 ura4-D18 his3-D1 ade6-M210 kanMX6-myo1-mEGFP</italic></td></tr><tr><td valign="bottom">TP226**</td><td valign="bottom"><italic>h- leu1-32 ura4-D18 his3-D1 ade6-M210 arpc5-mEGFP-kanMX6</italic></td></tr><tr><td valign="bottom">TP347**</td><td valign="bottom"><italic>h- leu1-32 ura4-D18 ade6-M210 fim1-mEGFP-kanMX</italic></td></tr><tr><td valign="bottom">*</td><td valign="bottom">from <xref ref-type="bibr" rid="bib33">Kaksonen et al. (2003)</xref> or <xref ref-type="bibr" rid="bib65">Sun et al. (2006)</xref></td></tr><tr><td valign="bottom">**</td><td valign="bottom">from <xref ref-type="bibr" rid="bib61">Sirotkin et al. (2005)</xref> or&#160;<xref ref-type="bibr" rid="bib69">Wu and Pollard&#160;(2005)</xref></td></tr><tr><td valign="bottom">otherwise</td><td valign="bottom"><italic>This study</italic></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-52482-v2.xml">elife-52482-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Strains used in this study.</title></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Strain</th><th valign="top">Genotype/description</th><th valign="top">Construction</th><th valign="top">Reference</th></tr></thead><tbody><tr><td valign="top">LSH100</td><td valign="top">Wild-type: mouse-adapted G27 derivative</td><td valign="top">-</td><td valign="top"><xref ref-type="bibr" rid="bib25">Lowenthal et al., 2009</xref></td></tr><tr><td valign="top">LSH141 (<italic>&#916;csd2</italic>)</td><td valign="top">LSH100 <italic>csd2::cat</italic></td><td valign="top">-</td><td valign="top"><xref ref-type="bibr" rid="bib41">Sycuro et al., 2010</xref></td></tr><tr><td valign="top">TSH17 (<italic>&#916;csd6</italic>)</td><td valign="top">LSH100 <italic>csd6::cat</italic></td><td valign="top">-</td><td valign="top"><xref ref-type="bibr" rid="bib43">Sycuro et al., 2013</xref></td></tr><tr><td valign="top">LSH108</td><td valign="top">LSH100 <italic>rdxA::aphA3sacB</italic></td><td valign="top">-</td><td valign="top"><xref ref-type="bibr" rid="bib41">Sycuro et al., 2010</xref></td></tr><tr><td valign="top">HMJ_Ec_pLC292-KU</td><td valign="top"><italic>E. coli</italic> TOP10 pLC292-KU</td><td valign="top">Transformation of TOP10 with pLC292-KU</td><td valign="top">This study</td></tr><tr><td valign="top">HJH1</td><td valign="top">LSH100 <italic>rdxA::amgKmurU</italic></td><td valign="top">Integration of pLC292-KU into LSH108</td><td valign="top">This study</td></tr><tr><td valign="top">IM4</td><td valign="top">LSH100 <italic>mcGee:mreB</italic></td><td valign="top">Integration of pIM04into LSH100</td><td valign="top">This study</td></tr><tr><td valign="top">JTH3</td><td valign="top">LSH100 <italic>ccmA:2X-FLAG:aphA3</italic></td><td valign="top">-</td><td valign="top"><xref ref-type="bibr" rid="bib2">Blair et al., 2018</xref></td></tr><tr><td valign="top">JTH5</td><td valign="top">LSH100 <italic>ccmA:2X-FLAG:aphA3 rdxA::amgKmurU</italic></td><td valign="top">Natural transformation of HJH1 with JTH3 genomic DNA</td><td valign="top">This study</td></tr><tr><td valign="top">KGH10</td><td valign="top">NSH57 <italic>ccmA::catsacB</italic></td><td valign="top">-</td><td valign="top"><xref ref-type="bibr" rid="bib41">Sycuro et al., 2010</xref></td></tr><tr><td valign="top">LSH117</td><td valign="top">LSH100 <italic>ccmA::catsacB</italic></td><td valign="top">Natural transformation of LSH100 with KGH10 genomic DNA</td><td valign="top">This study</td></tr><tr><td valign="top">SSH1</td><td valign="top">LSH100 <italic>ccmA<sup>I55A</sup></italic></td><td valign="top">Natural transformation with <italic>ccmA I55A</italic> PCR product</td><td valign="top">This study</td></tr><tr><td valign="top">SSH2</td><td valign="top">LSH100 <italic>ccmA<sup>L110S</sup></italic></td><td valign="top">Natural transformation with <italic>ccmA L110S</italic> PCR product</td><td valign="top">This study</td></tr><tr><td valign="top">LSH142 (<italic>&#916;ccmA</italic>)</td><td valign="top">LSH100 <italic>ccmA::cat</italic></td><td valign="top">-</td><td valign="top"><xref ref-type="bibr" rid="bib41">Sycuro et al., 2010</xref></td></tr><tr><td valign="top">JTH6</td><td valign="top">LSH100 <italic>rdxA::amgKmurU ccmA::cat</italic></td><td valign="top">Natural transformation of HJH1 with LSH142 genomic DNA</td><td valign="top">This study</td></tr></tbody></table></table-wrap>
            </body>
        </html>
    