
        <html>
            <head>
                <link rel='stylesheet' href='https://cdn.jsdelivr.net/gh/kognise/water.css@1.2.1/dist/light.css' />
            </head>
            <body>
                <h1>Tables with rowspan (693)</h1>
    <br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-00306-v1.xml">elife-00306-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.00306.008</object-id><label>Table 1.</label><caption><p>Early oxidation targets in yeast</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.00306.008">http://dx.doi.org/10.7554/eLife.00306.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene (Cys)</th><th rowspan="2">Protein</th><th rowspan="2">Loc.</th><th colspan="5">2% glucose (standard)</th><th colspan="7">0.5% glucose (CR)</th></tr><tr><th>D0</th><th>D1</th><th>D2</th><th>D3</th><th>D4</th><th>D0</th><th>D1</th><th>D2</th><th>D3</th><th>D4</th><th>D5</th><th>D7</th></tr></thead><tbody><tr><td>CCT4 (399)</td><td>T-complex protein 1 subunit delta</td><td>C</td><td align="char" char=".">12</td><td align="char" char=".">24</td><td align="char" char=".">58</td><td align="char" char=".">87</td><td align="char" char=".">86</td><td align="char" char=".">12</td><td align="char" char=".">14</td><td align="char" char=".">13</td><td align="char" char=".">11</td><td align="char" char=".">39</td><td align="char" char=".">72</td><td align="char" char=".">82</td></tr><tr><td>ARO2 (221)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>Chorismate synthase</td><td>C</td><td align="char" char=".">13</td><td align="char" char=".">13</td><td align="char" char=".">31</td><td align="char" char=".">71</td><td align="char" char=".">74</td><td/><td/><td/><td>nd</td><td/><td/><td/></tr><tr><td>CDC48 (115)</td><td>Cell division control protein 48</td><td>ER, C</td><td align="char" char=".">12</td><td align="char" char=".">29</td><td align="char" char=".">55</td><td align="char" char=".">75</td><td align="char" char=".">93</td><td align="char" char=".">13</td><td align="char" char=".">15</td><td align="char" char=".">14</td><td align="char" char=".">28</td><td align="char" char=".">62</td><td align="char" char=".">68</td><td align="char" char=".">90</td></tr><tr><td>CCT8 (336)</td><td>T-complex protein 1 subunit theta</td><td>C</td><td align="char" char=".">47</td><td align="char" char=".">40</td><td align="char" char=".">77</td><td align="char" char=".">76</td><td align="char" char=".">89</td><td/><td/><td/><td>nd</td><td/><td/><td/></tr><tr><td>TRR (142;145)</td><td>Thioredoxin reductase</td><td>C/M</td><td align="char" char=".">33</td><td align="char" char=".">33</td><td align="char" char=".">65</td><td align="char" char=".">77</td><td align="char" char=".">82</td><td align="char" char=".">34</td><td align="char" char=".">39</td><td align="char" char=".">42</td><td align="char" char=".">44</td><td align="char" char=".">73</td><td align="char" char=".">80</td><td align="char" char=".">86</td></tr><tr><td>UBC4 (108)</td><td>Ubiquitin-conjugating enzyme E2 4</td><td>N</td><td align="char" char=".">19</td><td align="char" char=".">23</td><td align="char" char=".">48</td><td align="char" char=".">70</td><td align="char" char=".">81</td><td align="char" char=".">20</td><td align="char" char=".">15</td><td align="char" char=".">22</td><td align="char" char=".">24</td><td align="char" char=".">49</td><td align="char" char=".">73</td><td align="char" char=".">88</td></tr><tr><td>YCR090C (124)</td><td>UPF0587 protein</td><td>C, N</td><td align="char" char=".">34</td><td align="char" char=".">39</td><td align="char" char=".">67</td><td align="char" char=".">76</td><td align="char" char=".">88</td><td/><td/><td/><td>nd</td><td/><td/><td/></tr><tr><td>LYS2 (614)</td><td>L-aminoadipate-semialdehyde DH</td><td>C</td><td align="char" char=".">15</td><td align="char" char=".">22</td><td align="char" char=".">49</td><td align="char" char=".">79</td><td align="char" char=".">86</td><td align="char" char=".">14</td><td align="char" char=".">11</td><td align="char" char=".">12</td><td align="char" char=".">48</td><td align="char" char=".">56</td><td align="char" char=".">74</td><td align="char" char=".">81</td></tr><tr><td>YDJ1 (185;188)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>Homologue of DnaJ</td><td>C</td><td align="char" char=".">46</td><td align="char" char=".">46</td><td align="char" char=".">82</td><td align="char" char=".">89</td><td align="char" char=".">75</td><td align="char" char=".">45</td><td align="char" char=".">44</td><td align="char" char=".">51</td><td align="char" char=".">70</td><td align="char" char=".">73</td><td align="char" char=".">83</td><td align="char" char=".">88</td></tr><tr><td>MES1 (353)</td><td>Methionyl-tRNA synthetase</td><td>C</td><td align="char" char=".">23</td><td align="char" char=".">34</td><td align="char" char=".">64</td><td align="char" char=".">86</td><td align="char" char=".">82</td><td align="char" char=".">20</td><td align="char" char=".">21</td><td align="char" char=".">31</td><td align="char" char=".">61</td><td align="char" char=".">78</td><td align="char" char=".">80</td><td align="char" char=".">98</td></tr><tr><td>OLA1 (43)</td><td>Uncharacterized GTP-binding protein</td><td>C</td><td align="char" char=".">17</td><td align="char" char=".">12</td><td align="char" char=".">46</td><td align="char" char=".">91</td><td align="char" char=".">83</td><td align="char" char=".">20</td><td align="char" char=".">10</td><td align="char" char=".">22</td><td align="char" char=".">29</td><td align="char" char=".">53</td><td align="char" char=".">78</td><td align="char" char=".">88</td></tr><tr><td>PAA1 (51;55)</td><td>Polyamine N-acetyltransferase 1</td><td>C</td><td align="char" char=".">26</td><td align="char" char=".">36</td><td align="char" char=".">53</td><td align="char" char=".">48</td><td align="char" char=".">61</td><td align="char" char=".">18</td><td align="char" char=".">15</td><td align="char" char=".">25</td><td align="char" char=".">52</td><td align="char" char=".">61</td><td align="char" char=".">71</td><td align="char" char=".">83</td></tr><tr><td>PRB1 (36)</td><td>Cerevisin</td><td>V</td><td align="char" char=".">18</td><td align="char" char=".">23</td><td align="char" char=".">57</td><td align="char" char=".">91</td><td align="char" char=".">94</td><td align="char" char=".">12</td><td align="char" char=".">14</td><td align="char" char=".">26</td><td align="char" char=".">23</td><td align="char" char=".">77</td><td align="char" char=".">80</td><td align="char" char=".">88</td></tr><tr><td>PUT2 (162)</td><td>&#948;-1-pyrroline-5-carboxylate DH</td><td>M</td><td align="char" char=".">12</td><td align="char" char=".">10</td><td align="char" char=".">35</td><td align="char" char=".">60</td><td align="char" char=".">66</td><td/><td/><td/><td>nd</td><td/><td/><td/></tr><tr><td>RPL10 (49)</td><td>60S protein L10</td><td>C</td><td align="char" char=".">15</td><td align="char" char=".">15</td><td align="char" char=".">31</td><td align="char" char=".">73</td><td align="char" char=".">89</td><td align="char" char=".">21</td><td align="char" char=".">17</td><td align="char" char=".">28</td><td align="char" char=".">23</td><td align="char" char=".">59</td><td align="char" char=".">80</td><td align="char" char=".">90</td></tr><tr><td>RPL42B (74)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>60S protein L42</td><td>C</td><td align="char" char=".">15</td><td align="char" char=".">22</td><td align="char" char=".">49</td><td align="char" char=".">86</td><td align="char" char=".">86</td><td align="char" char=".">15</td><td align="char" char=".">19</td><td align="char" char=".">25</td><td align="char" char=".">49</td><td align="char" char=".">54</td><td align="char" char=".">77</td><td align="char" char=".">93</td></tr><tr><td>RPS11B (58)</td><td>40S protein S11</td><td>C</td><td align="char" char=".">18</td><td align="char" char=".">17</td><td align="char" char=".">37</td><td align="char" char=".">77</td><td align="char" char=".">81</td><td align="char" char=".">14</td><td align="char" char=".">17</td><td align="char" char=".">27</td><td align="char" char=".">25</td><td align="char" char=".">36</td><td align="char" char=".">83</td><td align="char" char=".">86</td></tr><tr><td>RPS22B (72)</td><td>40S protein S22-B</td><td>C</td><td align="char" char=".">13</td><td align="char" char=".">9</td><td align="char" char=".">34</td><td align="char" char=".">62</td><td align="char" char=".">72</td><td align="char" char=".">32</td><td align="char" char=".">29</td><td align="char" char=".">36</td><td align="char" char=".">32</td><td align="char" char=".">41</td><td align="char" char=".">72</td><td align="char" char=".">79</td></tr><tr><td>SES1 (413;414)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>Seryl-tRNA synthetase</td><td>C</td><td align="char" char=".">22</td><td align="char" char=".">23</td><td align="char" char=".">55</td><td align="char" char=".">66</td><td align="char" char=".">79</td><td align="char" char=".">20</td><td align="char" char=".">19</td><td align="char" char=".">17</td><td align="char" char=".">43</td><td align="char" char=".">65</td><td align="char" char=".">71</td><td align="char" char=".">81</td></tr><tr><td>HEM1 (386)</td><td>5-aminolevulinate synthase</td><td>M</td><td align="char" char=".">13</td><td align="char" char=".">21</td><td align="char" char=".">58</td><td align="char" char=".">77</td><td align="char" char=".">88</td><td/><td/><td/><td>nd</td><td/><td/><td/></tr><tr><td>IDP1 (398)</td><td>Isocitrate dehydrogenase 1</td><td>M</td><td align="char" char=".">18</td><td align="char" char=".">19</td><td align="char" char=".">41</td><td align="char" char=".">61</td><td align="char" char=".">85</td><td align="char" char=".">16</td><td align="char" char=".">18</td><td align="char" char=".">32</td><td align="char" char=".">25</td><td align="char" char=".">49</td><td align="char" char=".">81</td><td align="char" char=".">80</td></tr><tr><td>KGD1 (983)</td><td>2-oxoglutarate dehydrogenase E1</td><td>M</td><td align="char" char=".">25</td><td align="char" char=".">21</td><td align="char" char=".">49</td><td align="char" char=".">57</td><td align="char" char=".">84</td><td align="char" char=".">18</td><td align="char" char=".">18</td><td align="char" char=".">19</td><td align="char" char=".">16</td><td align="char" char=".">57</td><td align="char" char=".">54</td><td align="char" char=".">70</td></tr><tr><td>FAS2 (917)</td><td>Fatty acid synthase subunit alpha</td><td>C, M</td><td align="char" char=".">22</td><td align="char" char=".">16</td><td align="char" char=".">30</td><td align="char" char=".">62</td><td align="char" char=".">86</td><td align="char" char=".">14</td><td align="char" char=".">8</td><td align="char" char=".">15</td><td align="char" char=".">59</td><td align="char" char=".">81</td><td align="char" char=".">87</td><td align="char" char=".">72</td></tr><tr><td>ERG13 (300)</td><td>Hydroxymethylglutaryl-CoA synthase</td><td>ER</td><td align="char" char=".">17</td><td align="char" char=".">38</td><td align="char" char=".">47</td><td align="char" char=".">80</td><td align="char" char=".">91</td><td align="char" char=".">16</td><td align="char" char=".">18</td><td align="char" char=".">30</td><td align="char" char=".">44</td><td align="char" char=".">55</td><td align="char" char=".">70</td><td align="char" char=".">84</td></tr><tr><td>FUS2 (371)</td><td>Nuclear fusion protein FUS2</td><td>N</td><td align="char" char=".">18</td><td align="char" char=".">33</td><td align="char" char=".">57</td><td align="char" char=".">68</td><td align="char" char=".">80</td><td align="char" char=".">9</td><td align="char" char=".">18</td><td align="char" char=".">12</td><td align="char" char=".">20</td><td align="char" char=".">31</td><td align="char" char=".">64</td><td align="char" char=".">91</td></tr><tr><td>LAP4 (202)</td><td>Vacuolar aminopeptidase 1</td><td>V</td><td align="char" char=".">19</td><td align="char" char=".">38</td><td align="char" char=".">45</td><td align="char" char=".">83</td><td align="char" char=".">86</td><td align="char" char=".">25</td><td align="char" char=".">35</td><td align="char" char=".">27</td><td align="char" char=".">22</td><td align="char" char=".">39</td><td align="char" char=".">61</td><td align="char" char=".">81</td></tr><tr><td>PYC2 (218)</td><td>Pyruvate carboxylase 2</td><td>C</td><td align="char" char=".">11</td><td align="char" char=".">28</td><td align="char" char=".">46</td><td align="char" char=".">81</td><td align="char" char=".">93</td><td align="char" char=".">13</td><td align="char" char=".">11</td><td align="char" char=".">22</td><td align="char" char=".">47</td><td align="char" char=".">66</td><td align="char" char=".">77</td><td align="char" char=".">81</td></tr><tr><td>TEF1 (409)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>Elongation factor 1-alpha</td><td>C</td><td align="char" char=".">13</td><td align="char" char=".">38</td><td align="char" char=".">40</td><td align="char" char=".">62</td><td align="char" char=".">70</td><td align="char" char=".">18</td><td align="char" char=".">30</td><td align="char" char=".">17</td><td align="char" char=".">26</td><td align="char" char=".">36</td><td align="char" char=".">57</td><td align="char" char=".">68</td></tr><tr><td>GapDH(150;154)<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>Glyceraldehyde-3-P Dehydrogenase</td><td>C</td><td align="char" char=".">26</td><td align="char" char=".">22</td><td align="char" char=".">28</td><td align="char" char=".">70</td><td align="char" char=".">81</td><td align="char" char=".">24</td><td align="char" char=".">22</td><td align="char" char=".">21</td><td align="char" char=".">26</td><td align="char" char=".">28</td><td align="char" char=".">72</td><td align="char" char=".">87</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>Peroxide sensitive (<xref ref-type="bibr" rid="bib3">Brandes et al. 2011</xref>).</p></fn><fn id="tblfn2"><label>&#8224;</label><p>Follows the general oxidation pattern.</p></fn><fn><p>All cluster D and E proteins thiols whose oxidation kinetics significantly preceded the general oxidation trend are listed. Thiol oxidation states, which are at least 2-fold higher as compared to day 0 or at least 1.5 fold higher as compared to day 0 and exceeding a total oxidation of 60% are shaded. Standard deviations can be found in <xref ref-type="supplementary-material" rid="SD1-data">Figure 1&#8212;Source data 1</xref>.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-00425-v1.xml">elife-00425-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.00425.005</object-id><label>Table 1.</label><caption><p>ROIs with corresponding Talairach coordinates (mean x, y, and z centre of mass and standard deviations) and sizes</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.00425.005">http://dx.doi.org/10.7554/eLife.00425.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">ROI name</th><th colspan="6">Tailarach coordinates</th><th colspan="2">ROI size</th></tr><tr><th>x</th><th>y</th><th>z</th><th>std x</th><th>std y</th><th>std z</th><th>mm<sup>3</sup></th><th>Nr voxels</th></tr></thead><tbody><tr><td>Parieto-frontal ROIs</td><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;L Superior parieto-occipital cortex (SPOC)</td><td align="char" char=".">&#8722;8</td><td align="char" char=".">&#8722;75</td><td align="char" char=".">29</td><td align="char" char=".">2.4</td><td align="char" char=".">2.9</td><td align="char" char=".">3.6</td><td align="char" char=".">1469</td><td align="char" char=".">54</td></tr><tr><td>&#8195;L Posterior intraparietal sulcus (pIPS)</td><td align="char" char=".">&#8722;22</td><td align="char" char=".">&#8722;68</td><td align="char" char=".">45</td><td align="char" char=".">3.4</td><td align="char" char=".">3.7</td><td align="char" char=".">4.1</td><td align="char" char=".">1640</td><td align="char" char=".">61</td></tr><tr><td>&#8195;L Middle intraparietal sulcus (midIPS)</td><td align="char" char=".">&#8722;32</td><td align="char" char=".">&#8722;56</td><td align="char" char=".">46</td><td align="char" char=".">4.4</td><td align="char" char=".">4</td><td align="char" char=".">3.9</td><td align="char" char=".">1943</td><td align="char" char=".">72</td></tr><tr><td>&#8195;L Posterior anterior intraparietal sulcus (post. aIPS)</td><td align="char" char=".">&#8722;42</td><td align="char" char=".">&#8722;49</td><td align="char" char=".">43</td><td align="char" char=".">3.6</td><td align="char" char=".">4.3</td><td align="char" char=".">3.9</td><td align="char" char=".">2290</td><td align="char" char=".">85</td></tr><tr><td>&#8195;L Anterior intraparietal sulcus (aIPS)</td><td align="char" char=".">&#8722;42</td><td align="char" char=".">&#8722;40</td><td align="char" char=".">42</td><td align="char" char=".">4.2</td><td align="char" char=".">4.3</td><td align="char" char=".">4.2</td><td align="char" char=".">2067</td><td align="char" char=".">77</td></tr><tr><td>&#8195;L Supramarginal gyrus (SMG)</td><td align="char" char=".">&#8722;56</td><td align="char" char=".">&#8722;35</td><td align="char" char=".">33</td><td align="char" char=".">3.7</td><td align="char" char=".">3.7</td><td align="char" char=".">4</td><td align="char" char=".">1479</td><td align="char" char=".">55</td></tr><tr><td>&#8195;L Motor cortex</td><td align="char" char=".">&#8722;38</td><td align="char" char=".">&#8722;29</td><td align="char" char=".">48</td><td align="char" char=".">4.3</td><td align="char" char=".">4.3</td><td align="char" char=".">4.3</td><td align="char" char=".">2407</td><td align="char" char=".">89</td></tr><tr><td>&#8195;L Dorsal premotor (PMd) cortex</td><td align="char" char=".">&#8722;26</td><td align="char" char=".">&#8722;14</td><td align="char" char=".">52</td><td align="char" char=".">4.2</td><td align="char" char=".">3.8</td><td align="char" char=".">4.1</td><td align="char" char=".">2135</td><td align="char" char=".">79</td></tr><tr><td>&#8195;L Ventral premotor (PMv) cortex</td><td align="char" char=".">&#8722;52</td><td align="char" char=".">3</td><td align="char" char=".">15</td><td align="char" char=".">3.3</td><td align="char" char=".">3.8</td><td align="char" char=".">3.1</td><td align="char" char=".">1460</td><td align="char" char=".">54</td></tr><tr><td>&#8195;L Somatosensory (SS) cortex</td><td align="char" char=".">&#8722;39</td><td align="char" char=".">&#8722;40</td><td align="char" char=".">48</td><td align="char" char=".">2.8</td><td align="char" char=".">2.8</td><td align="char" char=".">2.9</td><td align="char" char=".">1592</td><td align="char" char=".">59</td></tr><tr><td>Localizer-defined ROIs</td><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;L tool-anterior intraparietal sulcus (t-aIPS)</td><td align="char" char=".">&#8722;41</td><td align="char" char=".">&#8722;42</td><td align="char" char=".">46</td><td align="char" char=".">3.3</td><td align="char" char=".">3.4</td><td align="char" char=".">4.2</td><td align="char" char=".">1038</td><td align="char" char=".">38</td></tr><tr><td>&#8195;L Posterior middle temporal gyrus (pMTG)</td><td align="char" char=".">&#8722;53</td><td align="char" char=".">&#8722;57</td><td align="char" char=".">&#8722;3</td><td align="char" char=".">4.1</td><td align="char" char=".">3.1</td><td align="char" char=".">3.3</td><td align="char" char=".">621</td><td align="char" char=".">23</td></tr><tr><td>&#8195;L Extrastriate body area (EBA)</td><td align="char" char=".">&#8722;49</td><td align="char" char=".">&#8722;72</td><td align="char" char=".">1</td><td align="char" char=".">2.8</td><td align="char" char=".">3.6</td><td align="char" char=".">3.7</td><td align="char" char=".">851</td><td align="char" char=".">32</td></tr></tbody></table><table-wrap-foot><fn><p>Mean ROI sizes across subjects from ACPC data (in mm<sup>3</sup> and functional voxels). Std = standard deviation.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-00669-v1.xml">elife-00669-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.00669.014</object-id><label>Table 3.</label><caption><p>The values of the parameters of the Model 1 best fits shown in <xref ref-type="fig" rid="fig2">Figure 2</xref></p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.00669.014">http://dx.doi.org/10.7554/eLife.00669.014</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th align="center" colspan="7">Model 1</th></tr><tr><th>WT early night (panel F)</th><th>WT normal night (panel F)</th><th>WT late night<break/>(panel F)</th><th><italic>cca1/lhy</italic> early night (panel H)</th><th><italic>cca1/lhy</italic> normal night (panel H)</th><th>WT low light level (panel J)</th><th>WT normal light level (panel J)</th></tr></thead><tbody><tr><td><italic>&#961;</italic><sub>0</sub> (mg g<sup>&#8722;1</sup> FW) within 10% of the measured value</td><td align="char" char=".">8.5</td><td align="char" char=".">11.0</td><td align="char" char=".">11.7</td><td align="char" char=".">4.2</td><td align="char" char=".">5.1</td><td align="char" char=".">3.6</td><td align="char" char=".">6.1</td></tr><tr><td><italic>&#947;</italic> (0.7&#8211;3.0)</td><td align="char" char=".">1.8</td><td align="char" char=".">1.8</td><td align="char" char=".">1.9</td><td align="char" char=".">1.2</td><td align="char" char=".">1.2</td><td align="char" char=".">1.5</td><td align="char" char=".">1.3</td></tr><tr><td>(<italic>&#946;k</italic><sub><italic>ST</italic>1</sub>)<sup>&#8722;1</sup> (1.5&#8211;5.0) hr</td><td align="char" char=".">5.0</td><td align="char" char=".">5.0</td><td align="char" char=".">5.0</td><td align="char" char=".">1.6</td><td align="char" char=".">1.7</td><td align="char" char=".">2.1</td><td align="char" char=".">1.5</td></tr><tr><td><italic>t</italic><sub><italic>r</italic></sub> (9.0&#8211;12.0) hr</td><td align="char" char=".">9.0</td><td align="center">Any value in the specified range</td><td align="center">Any value in the specified range</td><td align="char" char=".">11.7</td><td align="char" char=".">11.7</td><td align="center">Any value in the specified range</td><td align="center">Any value in the specified range</td></tr><tr><td><italic>t</italic><sub>0</sub> (&#8722;5.0&#8211;5.0) hr for the <italic>cca1/lhy</italic> data, t<sub>0</sub> = 0 for WT</td><td align="char" char=".">0.0</td><td align="char" char=".">0.0</td><td align="char" char=".">0.0</td><td align="char" char=".">&#8722;4.2</td><td align="char" char=".">&#8722;2.5</td><td align="char" char=".">0.0</td><td align="char" char=".">0.0</td></tr></tbody></table><table-wrap-foot><fn><p>With each parameter the range used in the best fit search is indicated.</p></fn><fn><p>WT: wild-type plants.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-00669-v1.xml">elife-00669-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl4" position="float"><object-id pub-id-type="doi">10.7554/eLife.00669.015</object-id><label>Table 4.</label><caption><p>The values of the parameters of the Model 2 best fits shown in <xref ref-type="fig" rid="fig2">Figure 2</xref></p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.00669.015">http://dx.doi.org/10.7554/eLife.00669.015</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th align="center" colspan="7">Model 2</th></tr><tr><th>WT early night (panel G)</th><th>WT normal night (panel G)</th><th>WT late night (panel G)</th><th><italic>cca1/lhy</italic> early night (panel I)</th><th><italic>cca1/lhy</italic> normal night (panel I)</th><th>WT low light level (panel K)</th><th>WT normal light level (panel K)</th></tr></thead><tbody><tr><td><italic>&#961;</italic><sub>0</sub> (mg g<sup>&#8722;1</sup> FW) within 10% of the measured value</td><td align="char" char=".">8.3</td><td align="char" char=".">11.0</td><td align="char" char=".">12.1</td><td align="char" char=".">4.2</td><td align="char" char=".">5.1</td><td align="char" char=".">3.5</td><td align="char" char=".">5.8</td></tr><tr><td><italic>&#947;</italic> (0.7&#8211;3.0)</td><td align="char" char=".">1.1</td><td align="char" char=".">1.0</td><td align="char" char=".">1.1</td><td align="char" char=".">1.3</td><td align="char" char=".">1.2</td><td align="char" char=".">1.2</td><td align="char" char=".">0.9</td></tr><tr><td><italic>t</italic><sub><italic>r1</italic></sub> (9.0&#8211;12.0) hr</td><td align="char" char=".">11.0</td><td align="char" char=".">12.0</td><td align="char" char=".">12.0</td><td align="char" char=".">11.0</td><td align="char" char=".">11.4</td><td align="char" char=".">10.5</td><td align="char" char=".">10.2</td></tr><tr><td><italic>t</italic><sub><italic>r2</italic></sub> (20.0&#8211;23.0) hr</td><td align="char" char=".">20.1</td><td align="char" char=".">20.0</td><td align="char" char=".">20.0</td><td align="char" char=".">21.8</td><td align="char" char=".">21.4</td><td align="char" char=".">21.4</td><td align="char" char=".">22.5</td></tr><tr><td><italic>t</italic><sub>0</sub> (&#8722;5.0&#8211;5.0) hr for the <italic>cca1/lhy</italic> data, t<sub>0</sub> = 0 for WT</td><td align="char" char=".">0.0</td><td align="char" char=".">0.0</td><td align="char" char=".">0.0</td><td align="char" char=".">&#8722;2.4</td><td align="char" char=".">&#8722;1.3</td><td align="char" char=".">0.0</td><td align="char" char=".">0.0</td></tr></tbody></table><table-wrap-foot><fn><p>With each parameter the range used in the best fit search is indicated.</p></fn><fn><p>WT: wild-type plants.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-00669-v1.xml">elife-00669-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.00669.016</object-id><label>Table 5.</label><caption><p>The values of the parameters of the Model 1 best fits shown in <xref ref-type="fig" rid="fig3">Figure 3</xref></p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.00669.016">http://dx.doi.org/10.7554/eLife.00669.016</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th align="center" colspan="4">Model 1</th></tr><tr><th>Panel A</th><th>Panel B</th><th>Panel C</th><th>Panel D</th></tr></thead><tbody><tr><td><italic>&#961;</italic><sub>0</sub> (mg g<sup>&#8722;1</sup> FW) within 10% of the measured value</td><td align="char" char=".">5.4</td><td align="char" char=".">9.0</td><td align="char" char=".">7.7</td><td align="char" char=".">4.2</td></tr><tr><td><italic>&#947;</italic> (0.7&#8211;3.0)</td><td align="char" char=".">2.4</td><td align="char" char=".">1.4</td><td align="char" char=".">1.1</td><td align="char" char=".">1.0</td></tr><tr><td>(<italic>&#946;k</italic><sub><italic>ST</italic>1</sub>)<sup>&#8722;1</sup> (1.5&#8211;5.0) hr</td><td align="char" char=".">5.0</td><td align="char" char=".">2.6</td><td align="char" char=".">2.1</td><td align="char" char=".">2.0</td></tr><tr><td><italic>t</italic><sub><italic>r</italic></sub> (9.0&#8211;12.0) hr</td><td align="char" char=".">9.0</td><td align="char" char=".">9.0</td><td align="char" char=".">10.3</td><td align="char" char=".">12.0</td></tr><tr><td><italic>t</italic><sub>0</sub> (&#8722;5.0&#8212;5.0) hr</td><td align="char" char=".">4.3</td><td align="char" char=".">3.3</td><td align="char" char=".">2.4</td><td align="char" char=".">&#8722;0.5</td></tr></tbody></table><table-wrap-foot><fn><p>With each parameter the range used in the best fit search is indicated.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-00669-v1.xml">elife-00669-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl6" position="float"><object-id pub-id-type="doi">10.7554/eLife.00669.017</object-id><label>Table 6.</label><caption><p>The values of the parameters of the Model 2 best fits shown in <xref ref-type="fig" rid="fig3s2">Figure 3&#8212;figure supplement 2</xref></p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.00669.017">http://dx.doi.org/10.7554/eLife.00669.017</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th align="center" colspan="4">Model 2</th></tr><tr><th>Panel A</th><th>Panel B</th><th>Panel C</th><th>Panel D</th></tr></thead><tbody><tr><td><italic>&#961;</italic><sub>0</sub> (mg g<sup>&#8722;1</sup> FW) within 10% of the measured value</td><td align="char" char=".">5.5</td><td align="char" char=".">9.1</td><td align="char" char=".">7.7</td><td align="char" char=".">4.2</td></tr><tr><td><italic>&#947;</italic> (0.7&#8211;3.0)</td><td align="char" char=".">1.5</td><td align="char" char=".">1.3</td><td align="char" char=".">0.7</td><td align="char" char=".">0.7</td></tr><tr><td><italic>t</italic><sub><italic>r1</italic></sub> (9.0&#8211;12.0) hr</td><td align="char" char=".">12.0</td><td align="char" char=".">9.0</td><td align="char" char=".">9.0</td><td align="char" char=".">9.0</td></tr><tr><td><italic>t</italic><sub><italic>r2</italic></sub> (20.0&#8211;23.0) hr</td><td align="char" char=".">20.0</td><td align="char" char=".">20.9</td><td align="char" char=".">22.1</td><td align="char" char=".">22.1</td></tr><tr><td><italic>t</italic><sub>0</sub> (&#8722;5.0&#8211;5.0) hr</td><td align="char" char=".">5.0</td><td align="char" char=".">5.0</td><td align="char" char=".">2.4</td><td align="char" char=".">0.0</td></tr></tbody></table><table-wrap-foot><fn><p>With each parameter the range used in the best fit search is indicated.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-00747-v1.xml">elife-00747-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.00747.007</object-id><label>Table 1.</label><caption><p>Probability of treatment failure for combination therapy in patients</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.00747.007">http://dx.doi.org/10.7554/eLife.00747.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Patient</th><th rowspan="2">Primary tumor type</th><th rowspan="2">Number of metastases</th><th rowspan="2">Total tumor burden (number of cells)</th><th align="center" colspan="3">Probability of treatment failure</th></tr><tr><th>Monotherapy</th><th>Dual therapy: n<sub>12</sub> = 1</th><th>Dual therapy: n<sub>12</sub> = 0</th></tr></thead><tbody><tr><td>N1</td><td>Pancreas</td><td align="char" char=".">18</td><td>2.6 &#215; 10<sup>11</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.283</td></tr><tr><td>N2</td><td>Colon</td><td align="char" char=".">25</td><td>2.3 &#215; 10<sup>11</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.26</td></tr><tr><td>N3</td><td>Melanoma</td><td align="char" char=".">26</td><td>1.7 &#215; 10<sup>11</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.203</td></tr><tr><td>N4</td><td>Melanoma</td><td align="char" char=".">30</td><td>1.4 &#215; 10<sup>11</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.172</td></tr><tr><td>N5</td><td>Colon</td><td align="char" char=".">21</td><td>1.0 &#215; 10<sup>11</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.128</td></tr><tr><td>N6</td><td>Melanoma</td><td align="char" char=".">8</td><td>9.8 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.12</td></tr><tr><td>N7</td><td>Colon</td><td align="char" char=".">25</td><td>9.1 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.112</td></tr><tr><td>N8</td><td>Pancreas</td><td align="char" char=".">8</td><td>7.4 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.092</td></tr><tr><td>N9</td><td>Pancreas</td><td align="char" char=".">23</td><td>6.4 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.08</td></tr><tr><td>N10</td><td>Pancreas</td><td align="char" char=".">5</td><td>5.5 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.069</td></tr><tr><td>N11</td><td>Colon</td><td align="char" char=".">14</td><td>5.4 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.068</td></tr><tr><td>N12</td><td>Rectal</td><td align="char" char=".">23</td><td>4.8 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.061</td></tr><tr><td>N13</td><td>Melanoma</td><td align="char" char=".">9</td><td>4.1 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.052</td></tr><tr><td>N14</td><td>Pancreas</td><td align="char" char=".">13</td><td>4.1 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.051</td></tr><tr><td>N15</td><td>Pancreas</td><td align="char" char=".">8</td><td>3.3 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.042</td></tr><tr><td>N16</td><td>Melanoma</td><td align="char" char=".">7</td><td>2.2 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.028</td></tr><tr><td>N17</td><td>Melanoma</td><td align="char" char=".">10</td><td>2.1 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.027</td></tr><tr><td>N18</td><td>Colon</td><td align="char" char=".">4</td><td>2.0 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.026</td></tr><tr><td>N19</td><td>Melanoma</td><td align="char" char=".">9</td><td>1.8 &#215; 10<sup>10</sup></td><td align="char" char=".">1</td><td align="char" char=".">1</td><td align="char" char=".">0.023</td></tr><tr><td>N20</td><td>Colon</td><td align="char" char=".">3</td><td>1.6 &#215; 10<sup>9</sup></td><td align="char" char=".">1</td><td align="char" char=".">0.881</td><td align="char" char=".">0.002</td></tr><tr><td>N21</td><td>Melanoma</td><td align="char" char=".">21</td><td>1.3 &#215; 10<sup>9</sup></td><td align="char" char=".">1</td><td align="char" char=".">0.828</td><td align="char" char=".">0.002</td></tr><tr><td>N22</td><td>Pancreas</td><td align="char" char=".">1</td><td>8.5 &#215; 10<sup>8</sup></td><td align="char" char=".">1</td><td align="char" char=".">0.677</td><td align="char" char=".">0.001</td></tr></tbody></table><table-wrap-foot><fn><p>For monotherapy, we assume that 50 point mutations (<italic>n</italic> = 50) can in principle confer resistance to the drug. With dual therapy, we assume that 50 point mutations can in principle confer resistance to each drug individually (<italic>n</italic><sub>1</sub> = <italic>n</italic><sub>2</sub> = 50). Two scenarios are modeled: in the first, there is one mutation that can in principle confer resistance to both drugs (i.e., cross-resistance, <italic>n</italic><sub>12</sub> = 1). In the other case, there are no possible mutations that can confer resistance to both drugs (<italic>n</italic><sub>12</sub> = 0). Parameter values: birth rate, <italic>b</italic> = 0.14, death rate, <italic>d</italic> = 0.13, death rate for sensitive cells during treatment, <italic>d</italic>&#8242; = 0.17, point mutation rate <italic>u</italic> = 10<sup>&#8722;9</sup>.</p></fn><fn><p>Colon: colonic adenocarcinoma; Rectal: rectal adenocarcinoma; Pancreas: pancreatic ductal adenocarcinoma.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-00747-v1.xml">elife-00747-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.00747.011</object-id><label>Table 2.</label><caption><p>Simulation results for the probability of treatment failure when resistance is costly</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.00747.011">http://dx.doi.org/10.7554/eLife.00747.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2">Dual therapy:</th><th rowspan="2">Number of cells</th><th rowspan="2">Birth rate</th><th align="center" colspan="4">Probability of treatment failure</th></tr><tr><th>n<sub>1</sub> = n<sub>2</sub></th><th>n<sub>12</sub></th><th>c = 0%</th><th>c = 1%</th><th>c = 5%</th><th>c = 10%</th></tr></thead><tbody><tr><td align="char" char=".">50</td><td align="char" char=".">0</td><td align="char" char=".">10<sup>9</sup></td><td align="char" char=".">0.14</td><td align="char" char=".">0.0</td><td align="char" char=".">0.0</td><td align="char" char=".">0.0</td><td align="char" char=".">0.0</td></tr><tr><td align="char" char=".">50</td><td align="char" char=".">0</td><td align="char" char=".">10<sup>9</sup></td><td align="char" char=".">1</td><td align="char" char=".">0.01</td><td align="char" char=".">0.01</td><td align="char" char=".">0.01</td><td align="char" char=".">0.0</td></tr><tr><td align="char" char=".">50</td><td align="char" char=".">1</td><td align="char" char=".">10<sup>9</sup></td><td align="char" char=".">0.14</td><td align="char" char=".">0.74</td><td align="char" char=".">0.73</td><td align="char" char=".">0.72</td><td align="char" char=".">0.7</td></tr><tr><td align="char" char=".">50</td><td align="char" char=".">1</td><td align="char" char=".">10<sup>9</sup></td><td align="char" char=".">1</td><td align="char" char=".">0.74</td><td align="char" char=".">0.74</td><td align="char" char=".">0.72</td><td align="char" char=".">0.7</td></tr><tr><td align="char" char=".">50</td><td align="char" char=".">0</td><td align="char" char=".">10<sup>11</sup></td><td align="char" char=".">0.14</td><td align="char" char=".">0.12</td><td align="char" char=".">0.11</td><td align="char" char=".">0.08</td><td align="char" char=".">0.06</td></tr><tr><td align="char" char=".">50</td><td align="char" char=".">0</td><td align="char" char=".">10<sup>11</sup></td><td align="char" char=".">1</td><td align="char" char=".">0.53</td><td align="char" char=".">0.51</td><td align="char" char=".">0.42</td><td align="char" char=".">0.32</td></tr><tr><td align="char" char=".">50</td><td align="char" char=".">1</td><td align="char" char=".">10<sup>11</sup></td><td align="char" char=".">0.14</td><td align="char" char=".">1.0</td><td align="char" char=".">1.0</td><td align="char" char=".">1.0</td><td align="char" char=".">1.0</td></tr><tr><td align="char" char=".">50</td><td align="char" char=".">1</td><td align="char" char=".">10<sup>11</sup></td><td align="char" char=".">1</td><td align="char" char=".">1.0</td><td align="char" char=".">1.0</td><td align="char" char=".">1.0</td><td align="char" char=".">1.0</td></tr></tbody></table><table-wrap-foot><fn><p>Each resistance mutation reduces the net growth rate by a factor <italic>c</italic> via a decrease of the birth rate <italic>b</italic>. Parameter values are death rate, <italic>d</italic> = <italic>b</italic> &#8722; 0.01, death rate for sensitive cells during treatment, <italic>d</italic>&#8217; = <italic>b</italic> + 0.03, point mutation rate, <italic>u</italic> = 10<sup>&#8722;9</sup>. The simulation results are averages over 10<sup>6</sup> runs per parameter combination.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-00801-v1.xml">elife-00801-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.00801.008</object-id><label>Table 2.</label><caption><p>Cortical areas showing significantly less average activation for trained than untrained sequences. The opposite contrast did not result in any significant areas</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.00801.008">http://dx.doi.org/10.7554/eLife.00801.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Area</th><th rowspan="2">Area (cm<sup>2</sup>)</th><th rowspan="2"><italic>P</italic><sub>cluster</sub></th><th rowspan="2">Peak <italic>t</italic><sub>(15)</sub></th><th colspan="3">MNI</th></tr><tr><th>X</th><th>Y</th><th>Z</th></tr></thead><tbody><tr><td>Left hemisphere</td><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;PMd</td><td align="char" char=".">7.72</td><td align="char" char=".">&lt;0.002</td><td align="char" char=".">6.02</td><td align="char" char=".">&#8722;27</td><td align="char" char=".">&#8722;1</td><td align="char" char=".">48</td></tr><tr><td>&#8195;OPJ</td><td align="char" char=".">1.43</td><td align="char" char=".">&lt;0.002</td><td align="char" char=".">6.74</td><td align="char" char=".">&#8722;10</td><td align="char" char=".">&#8722;67</td><td align="char" char=".">59</td></tr><tr><td>&#8195;IPS</td><td align="char" char=".">0.16</td><td align="char" char=".">0.020</td><td align="char" char=".">7.55</td><td align="char" char=".">&#8722;42</td><td align="char" char=".">&#8722;43</td><td align="char" char=".">40</td></tr><tr><td>Right hemisphere</td><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;PMd</td><td align="char" char=".">0.14</td><td align="char" char=".">0.020</td><td align="char" char=".">4.25</td><td align="char" char=".">37</td><td align="char" char=".">16</td><td align="char" char=".">51</td></tr><tr><td>&#8195;IPS</td><td align="char" char=".">0.13</td><td align="char" char=".">0.028</td><td align="char" char=".">5.28</td><td align="char" char=".">19</td><td align="char" char=".">&#8722;49</td><td align="char" char=".">61</td></tr><tr><td>&#8195;IPS</td><td align="char" char=".">0.27</td><td align="char" char=".">&lt;0.002</td><td align="char" char=".">5.48</td><td align="char" char=".">29</td><td align="char" char=".">&#8722;52</td><td align="char" char=".">49</td></tr></tbody></table><table-wrap-foot><fn><p>Table shows the result of a surface-based random effects analysis (N = 16). The uncorrected threshold was p&lt;0.002, <italic>t</italic>(15) &gt; 3.39, and <italic>P</italic><sub>cluster</sub> is the p-value corrected for multiple comparisons over the whole cortical surface using the area of the cluster (<xref ref-type="bibr" rid="bib49">Worsley et al., 1996</xref>). The coordinates reflect the location of the cluster&#8217;s peak in MNI space.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-00857-v1.xml">elife-00857-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.00857.011</object-id><label>Table 2.</label><caption><p>Detailed list of the active BAF compounds</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.00857.011">http://dx.doi.org/10.7554/eLife.00857.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Compound</th><th rowspan="2">Molecular formula</th><th rowspan="2">Molecular weight<xref ref-type="table-fn" rid="tblfn5">*</xref></th><th rowspan="2">Sources/companies</th><th rowspan="2">Purity</th><th colspan="2">Rescuing percentage<xref ref-type="table-fn" rid="tblfn8">&#167;</xref> (%)</th><th rowspan="2">ZINC entry code<xref ref-type="table-fn" rid="tblfn9">&#182;</xref></th><th rowspan="2">SMILES string</th></tr><tr><th>PC12</th><th>Hela</th></tr></thead><tbody><tr><td>BAF1</td><td>C<sub>20</sub>H<sub>8</sub>Br<sub>4</sub>O<sub>5</sub></td><td align="char" char=".">647.9</td><td>Sigma-Aldrich</td><td>&#8764;99%</td><td align="char" char="plusmn">38 &#177; 11</td><td align="char" char="plusmn">44 &#177; 7</td><td>ZINC04261875</td><td>c1ccc2c(c1)C(=O)OC23c4ccc(c(c4Oc5c3ccc(c5Br)O)Br)O</td></tr><tr><td>BAF4</td><td>C<sub>24</sub>H<sub>16</sub>N<sub>2</sub>O<sub>6</sub></td><td align="char" char=".">428.4</td><td>Aldrich</td><td>&#8805;95%</td><td align="char" char="plusmn">85 &#177; 18</td><td align="char" char="plusmn">88 &#177; 22</td><td>ZINC13346907</td><td>c1cc(c(cc1O)O)c2cc3c(cc2N)oc-4cc(=O)c(cc4n3)c5ccc(cc5O)O</td></tr><tr><td>BAF8</td><td>C<sub>17</sub>H<sub>14</sub>N<sub>2</sub>O<sub>5</sub>S</td><td align="char" char=".">358.4</td><td>Sigma-Aldrich</td><td>&#8805;90%</td><td align="char" char="plusmn">26 &#177; 12</td><td align="char" char="plusmn">23 &#177; 11</td><td>ZINC12358966</td><td>Cc1ccc(c(c1)/N=N/c2c3ccccc3c(cc2O)S(=O)(=O)[O-])O</td></tr><tr><td>BAF11</td><td>C<sub>20</sub>H<sub>13</sub>N<sub>2</sub>O<sub>5</sub>S</td><td align="char" char=".">393.5</td><td>NCI plated 2007</td><td><xref ref-type="table-fn" rid="tblfn6">&#8224;</xref></td><td align="char" char="plusmn">51 &#177; 11</td><td align="char" char="plusmn">48 &#177; 12</td><td>ZINC04521479</td><td>c1ccc2c(c1)ccc(c2O)/N=N/c3c4ccccc4c(cc3O)S(=O)(=O)[O-]</td></tr><tr><td>BAF12</td><td>C<sub>13</sub>H<sub>8</sub>Br<sub>3</sub>NO</td><td align="char" char=".">433.9</td><td>NCI plated 2007</td><td><xref ref-type="table-fn" rid="tblfn6">&#8224;</xref></td><td align="char" char="plusmn">19 &#177; 6</td><td align="char" char="plusmn">38 &#177; 6</td><td>ZINC12428965</td><td>c1cc(ccc1/N=C/c2cc(cc(c2O)Br)Br)Br</td></tr><tr><td>BAF26</td><td>C<sub>17</sub>H<sub>10</sub>O<sub>4</sub></td><td align="char" char=".">278.3</td><td>Aldrich</td><td><xref ref-type="table-fn" rid="tblfn7">&#8225;</xref></td><td align="char" char="plusmn">60 &#177; 21</td><td align="char" char="plusmn">46 &#177; 23</td><td>ZINC05770717</td><td>c12c(cc(cc1)C(=O)C=O)Cc1c2ccc(c1)C(=O)C=O</td></tr><tr><td>BAF30</td><td>C<sub>14</sub>H<sub>8</sub>O<sub>5</sub></td><td align="char" char=".">256.2</td><td>Aldrich</td><td><xref ref-type="table-fn" rid="tblfn6">&#8225;</xref></td><td align="char" char="plusmn">37 &#177; 18</td><td align="char" char="plusmn">18 &#177; 13</td><td>ZINC03870461</td><td>c1cc2c(cc1O)C(=O)c3c(ccc(c3O)O)C2=O</td></tr><tr><td>BAF31</td><td>C<sub>19</sub>H<sub>21</sub>NO<sub>3</sub></td><td align="char" char=".">311.4</td><td>Sigma</td><td>&#8805;98%</td><td align="char" char="plusmn">92 &#177; 22</td><td align="char" char="plusmn">84 &#177; 12</td><td>ZINC03874841</td><td>CCCN1CCC2=C3C1CC4=C(C3=CC(=C2)O)C(=C(C=C4)O)O</td></tr></tbody></table><table-wrap-foot><fn><p>BAFs 1, 4, 8, 11, 12 are from the first round. BAFs 26, 30, 31 are from the second round.</p></fn><fn id="tblfn5"><label>*</label><p>Molecular weight (anhydrous basis) excluding the salt and water molecules.</p></fn><fn id="tblfn6"><label>&#8224;</label><p>With the standard of NCI free compound library.</p></fn><fn id="tblfn7"><label>&#8225;</label><p>Analytical data for Aldrich<sup>CPR</sup> products are not available.</p></fn><fn id="tblfn8"><label>&#167;</label><p>Rescue percentage is a scaled cell survival rate.</p></fn><fn id="tblfn9"><label>&#182;</label><p>Entry code for the ZINC database (<ext-link ext-link-type="uri" xlink:href="http://zinc.docking.org">http://zinc.docking.org</ext-link>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-00857-v1.xml">elife-00857-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl4" position="float"><object-id pub-id-type="doi">10.7554/eLife.00857.016</object-id><label>Table 4.</label><caption><p>Student&#8217;s <italic>t</italic>-test and p value analysis suggests that BAFs reduce the cytotoxicity of A&#946; fibers significantly</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.00857.016">http://dx.doi.org/10.7554/eLife.00857.016</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th rowspan="2">Average of cell viability (n = 4)</th><th rowspan="2">SD(&#963;)</th><th colspan="2">Comparison to A&#946; fiber alone</th></tr><tr><th><italic>t</italic> value</th><th>p value</th></tr></thead><tbody><tr><td colspan="5">HeLa cell line</td></tr><tr><td>&#8195;A&#946; fiber alone</td><td align="char" char=".">0.40</td><td align="char" char=".">0.05</td><td>/</td><td>/</td></tr><tr><td>&#8195;BAF1</td><td align="char" char=".">0.66</td><td align="char" char=".">0.04</td><td align="char" char=".">8.4</td><td>5E-05</td></tr><tr><td>&#8195;BAF4</td><td align="char" char=".">0.93</td><td align="char" char=".">0.13</td><td align="char" char=".">7.4</td><td>1E-4</td></tr><tr><td>&#8195;BAF8</td><td align="char" char=".">0.54</td><td align="char" char=".">0.06</td><td align="char" char=".">3.3</td><td>1E-2</td></tr><tr><td>&#8195;BAF11</td><td align="char" char=".">0.69</td><td align="char" char=".">0.07</td><td align="char" char=".">6.6</td><td>2E-04</td></tr><tr><td>&#8195;BAF12</td><td align="char" char=".">0.63</td><td align="char" char=".">0.04</td><td align="char" char=".">7.6</td><td>1E-04</td></tr><tr><td>&#8195;BAF26</td><td align="char" char=".">0.68</td><td align="char" char=".">0.14</td><td align="char" char=".">3.8</td><td>5E-3</td></tr><tr><td>&#8195;BAF30</td><td align="char" char=".">0.51</td><td align="char" char=".">0.08</td><td align="char" char=".">2.3</td><td>4E-2</td></tr><tr><td>&#8195;BAF31</td><td align="char" char=".">0.91</td><td align="char" char=".">0.07</td><td align="char" char=".">11.5</td><td>7E-06</td></tr><tr><td colspan="5">PC12 cell line</td></tr><tr><td>&#8195;A&#946; fiber alone</td><td align="char" char=".">0.37</td><td align="char" char=".">0.07</td><td>/</td><td>/</td></tr><tr><td>&#8195;BAF1</td><td align="char" char=".">0.61</td><td align="char" char=".">0.07</td><td align="char" char=".">4.9</td><td>1E-3</td></tr><tr><td>&#8195;BAF4</td><td align="char" char=".">0.90</td><td align="char" char=".">0.11</td><td align="char" char=".">8.0</td><td>7E-05</td></tr><tr><td>&#8195;BAF8</td><td align="char" char=".">0.53</td><td align="char" char=".">0.07</td><td align="char" char=".">3.2</td><td>1E-2</td></tr><tr><td>&#8195;BAF11</td><td align="char" char=".">0.69</td><td align="char" char=".">0.07</td><td align="char" char=".">6.5</td><td>2E-4</td></tr><tr><td>&#8195;BAF12</td><td align="char" char=".">0.49</td><td align="char" char=".">0.04</td><td align="char" char=".">2.9</td><td>2E-2</td></tr><tr><td>&#8195;BAF26</td><td align="char" char=".">0.74</td><td align="char" char=".">0.13</td><td align="char" char=".">5.0</td><td>1E-3</td></tr><tr><td>&#8195;BAF30</td><td align="char" char=".">0.60</td><td align="char" char=".">0.11</td><td align="char" char=".">3.5</td><td>8E-3</td></tr><tr><td>&#8195;BAF31</td><td align="char" char=".">0.95</td><td align="char" char=".">0.14</td><td align="char" char=".">7.4</td><td>1E-4</td></tr></tbody></table><table-wrap-foot><fn><p>The Student&#8217;s T-test and p-value are based on the comparison to A&#946; fiber alone.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-00857-v1.xml">elife-00857-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.00857.022</object-id><label>Table 5.</label><caption><p>Predicted binding energy and experimental measurement of the binding of two BAFs and orange G against both A&#946;<sub>16&#8211;21</sub> (KLVFFA) and full-length A&#946; fibers</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.00857.022">http://dx.doi.org/10.7554/eLife.00857.022</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2">Binding to KLVFFA fiber</th><th colspan="2">Binding to A&#946; fiber</th></tr><tr><th>Predicted binding energy (kcal/mol)</th><th>NMR Kd (&#181;M)</th><th>Predicted binding energy (kcal/mol)</th><th>NMR peak reduction (%)</th></tr></thead><tbody><tr><td>BAF1</td><td align="char" char=".">&#8722;8</td><td align="char" char=".">12</td><td align="char" char=".">&#8722;10</td><td align="char" char=".">8</td></tr><tr><td>BAF8</td><td align="char" char=".">&#8722;12</td><td align="char" char=".">24</td><td align="char" char=".">&#8722;12</td><td align="char" char=".">13</td></tr><tr><td>orange G</td><td align="char" char=".">&#8722;8</td><td align="char" char=".">43</td><td align="char" char=".">&#8722;9</td><td align="char" char=".">6</td></tr></tbody></table><table-wrap-foot><fn><p>The determination of the binding parameters with KLVFFA fiber is detailed in <xref ref-type="table" rid="tbl6">Table 6</xref>.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01071-v1.xml">elife-01071-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl7" position="float"><object-id pub-id-type="doi">10.7554/eLife.01071.021</object-id><label>Table 7.</label><caption><p>Quantification of centriole/centrosome numbers in <italic>dCPAP</italic> or <italic>dSTIL</italic> mutant larval brain cells expressing the indicated WT or mutant constructs</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01071.021">http://dx.doi.org/10.7554/eLife.01071.021</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Genotype</th><th rowspan="2">Number of brains</th><th rowspan="2">Total number of cells</th><th colspan="4">Cells with centrosome number (%)</th></tr><tr><th>0</th><th>1</th><th>2</th><th>3</th></tr></thead><tbody><tr><td>WT</td><td align="char" char=".">12</td><td align="char" char=".">944</td><td align="char" char=".">2.1</td><td align="char" char=".">2.6</td><td align="char" char=".">95.2</td><td align="char" char=".">0.0</td></tr><tr><td><italic>dCPAP</italic></td><td align="char" char=".">8</td><td align="char" char=".">661</td><td align="char" char=".">95.2</td><td align="char" char=".">4.2</td><td align="char" char=".">0.6</td><td align="char" char=".">0.0</td></tr><tr><td>dCPAP_WT-GFP</td><td align="char" char=".">9</td><td align="char" char=".">715</td><td align="char" char=".">2.8</td><td align="char" char=".">6.3</td><td align="char" char=".">90.5</td><td align="char" char=".">0.4</td></tr><tr><td>dCPAP_&#916;C-GFP</td><td align="char" char=".">13</td><td align="char" char=".">1147</td><td align="char" char=".">95.1</td><td align="char" char=".">3.8</td><td align="char" char=".">1.0</td><td align="char" char=".">0.0</td></tr><tr><td>dCPAP_MC1-GFP</td><td align="char" char=".">11</td><td align="char" char=".">968</td><td align="char" char=".">64.9</td><td align="char" char=".">30.1</td><td align="char" char=".">4.8</td><td align="char" char=".">0.3</td></tr><tr><td>dCPAP_MC2-GFP</td><td align="char" char=".">17</td><td align="char" char=".">1053</td><td align="char" char=".">43.5</td><td align="char" char=".">42.2</td><td align="char" char=".">14.1</td><td align="char" char=".">0.3</td></tr><tr><td>dCPAP_MC3-GFP</td><td align="char" char=".">16</td><td align="char" char=".">1870</td><td align="char" char=".">4.5</td><td align="char" char=".">13.3</td><td align="char" char=".">81.9</td><td align="char" char=".">0.3</td></tr><tr><td>dCPAP_MC1-3-GFP</td><td align="char" char=".">11</td><td align="char" char=".">888</td><td align="char" char=".">90.1</td><td align="char" char=".">8.0</td><td align="char" char=".">1.8</td><td align="char" char=".">0.1</td></tr><tr><td>dCPAP_E792V-GFP</td><td align="char" char=".">9</td><td align="char" char=".">1015</td><td align="char" char=".">13.7</td><td align="char" char=".">31.1</td><td align="char" char=".">54.8</td><td align="char" char=".">0.4</td></tr><tr><td><italic>dSTIL</italic><sup><italic>169</italic></sup></td><td align="char" char=".">9</td><td align="char" char=".">846</td><td align="char" char=".">98.1</td><td align="char" char=".">1.9</td><td align="char" char=".">0.0</td><td align="char" char=".">0.0</td></tr><tr><td><italic>dSTIL</italic><sup><italic>719</italic></sup></td><td align="char" char=".">9</td><td align="char" char=".">980</td><td align="char" char=".">99.2</td><td align="char" char=".">0.8</td><td align="char" char=".">0.0</td><td align="char" char=".">0.0</td></tr><tr><td>dSTIL_WT-GFP</td><td align="char" char=".">6</td><td align="char" char=".">424</td><td align="char" char=".">5.7</td><td align="char" char=".">9.0</td><td align="char" char=".">85.1</td><td align="char" char=".">0.2</td></tr><tr><td>dSTIL_&#916;N-GFP</td><td align="char" char=".">13</td><td align="char" char=".">884</td><td align="char" char=".">88.1</td><td align="char" char=".">9.8</td><td align="char" char=".">1.9</td><td align="char" char=".">0.1</td></tr><tr><td>dSTIL_P11A-GFP</td><td align="char" char=".">9</td><td align="char" char=".">1008</td><td align="char" char=".">11.0</td><td align="char" char=".">41.0</td><td align="char" char=".">48.0</td><td align="char" char=".">0.0</td></tr><tr><td>dSTIL_R12A-GFP</td><td align="char" char=".">9</td><td align="char" char=".">709</td><td align="char" char=".">7.0</td><td align="char" char=".">31.0</td><td align="char" char=".">62.0</td><td align="char" char=".">0.0</td></tr><tr><td>dSTIL_P11AR12A-GFP</td><td align="char" char=".">9</td><td align="char" char=".">727</td><td align="char" char=".">41.0</td><td align="char" char=".">43.0</td><td align="char" char=".">16.0</td><td align="char" char=".">0.0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01206-v1.xml">elife-01206-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.01206.015</object-id><label>Table 1.</label><caption><p>Comparison of two different intracellular EGTA concentrations on Ca<sup>2+</sup> signaling and synaptic transmission. The values indicated are all in units of s. Calcium imaging measurements were performed using Fluo-4 indicator.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01206.015">http://dx.doi.org/10.7554/eLife.01206.015</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2">Control</th><th colspan="2">+ &#969;-conotoxin GVIA</th></tr><tr><th>Ca<sup>2+</sup> imaging (20% rise)</th><th>paired recording (time to peak)</th><th>Ca<sup>2+</sup> imaging (20% rise)</th><th>paired recording (time to peak)</th></tr></thead><tbody><tr><td>0.5 mM EGTA</td><td align="char" char="plusmn">0.18 &#177; 0.16 (4 fish, 64 boutons)</td><td align="char" char="plusmn">2.88 &#177; 0.35 (n = 8)</td><td align="char" char="plusmn">1.98 &#177; 0.84 (6 fish, 68 boutons)</td><td align="char" char="plusmn">4.88 &#177; 0.35 (n = 8)</td></tr><tr><td>5 mM EGTA</td><td align="char" char="plusmn">0.71 &#177; 0.47 (3 fish, 38 boutons)</td><td align="char" char="plusmn">3.36 &#177; 0.50 (n = 15)</td><td align="char" char="plusmn">5.42 &#177; 1.95 (4 fish, 56 boutons)</td><td align="char" char="plusmn">10.54 &#177; 2.37 (n = 13)</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01229-v1.xml">elife-01229-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.01229.003</object-id><label>Table 1.</label><caption><p><italic>S.</italic> Typhi mutants constructed for this study</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01229.003">http://dx.doi.org/10.7554/eLife.01229.003</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><italic>S.</italic> Typhi strain</th><th rowspan="2">Genotype</th><th colspan="6">Minimum inhibitory concentrations (&#956;g/ml)</th></tr><tr><th>Nalidixic acid</th><th>Norfloxacin</th><th>Ofloxacin</th><th>Ciprofloxacin</th><th>Gatifloxacin</th><th>Levofloxacin</th></tr></thead><tbody><tr><td>Parent BRD948</td><td>&#916;<italic>aroA</italic>, &#916;<italic>aroC</italic>, &#916;<italic>htrA</italic></td><td align="char" char=".">1.5</td><td align="char" char=".">0.064</td><td align="char" char=".">0.047</td><td align="char" char=".">0.008</td><td align="char" char=".">0.008</td><td align="char" char=".">0.012</td></tr><tr><td>DPT001</td><td>SNP in &#916;<italic>aroC</italic> (codon 10)</td><td align="char" char=".">1.5</td><td align="char" char=".">0.064</td><td align="char" char=".">0.047</td><td align="char" char=".">0.008</td><td align="char" char=".">0.008</td><td align="char" char=".">0.012</td></tr><tr><td>S83Y</td><td>SNP in <italic>gyrA</italic> (codon 83)</td><td align="char" char=".">256</td><td align="char" char=".">0.5</td><td align="char" char=".">0.25</td><td align="char" char=".">0.125</td><td align="char" char=".">0.125</td><td align="char" char=".">0.125</td></tr><tr><td>S83F</td><td>SNP in <italic>gyrA</italic> (codon 83)</td><td align="char" char=".">256</td><td align="char" char=".">0.75</td><td align="char" char=".">0.38</td><td align="char" char=".">0.125</td><td align="char" char=".">0.125</td><td align="char" char=".">0.125</td></tr><tr><td>D87A</td><td>SNP in <italic>gyrA</italic> (codon 87)</td><td align="char" char=".">48</td><td align="char" char=".">0.75</td><td align="char" char=".">0.19</td><td align="char" char=".">0.094</td><td align="char" char=".">0.064</td><td align="char" char=".">0.064</td></tr><tr><td>D87N</td><td>SNP in <italic>gyrA</italic> (codon 87)</td><td align="char" char=".">48</td><td align="char" char=".">0.75</td><td align="char" char=".">0.25</td><td align="char" char=".">0.125</td><td align="char" char=".">0.125</td><td align="char" char=".">0.125</td></tr><tr><td>D87G</td><td>SNP in <italic>gyrA</italic> (codon 87)</td><td align="char" char=".">48</td><td align="char" char=".">0.75</td><td align="char" char=".">0.25</td><td align="char" char=".">0.125</td><td align="char" char=".">0.125</td><td align="char" char=".">0.25</td></tr><tr><td>S80I</td><td>SNP in <italic>parC</italic> (codon 80)</td><td align="char" char=".">3</td><td align="char" char=".">0.19</td><td align="char" char=".">0.047</td><td align="char" char=".">0.016</td><td align="char" char=".">0.016</td><td align="char" char=".">0.016</td></tr><tr><td>D87G-S80I</td><td>SNP in <italic>gyrA</italic> (codon 87) and SNP in <italic>parC</italic> (codon 80)</td><td align="char" char=".">256</td><td align="char" char=".">1</td><td align="char" char=".">0.25</td><td align="char" char=".">0.125</td><td align="char" char=".">0.094</td><td align="char" char=".">0.094</td></tr><tr><td>S83F-D87G</td><td>2 SNPs in <italic>gyrA</italic> (codons 83 and 87)</td><td align="char" char=".">256</td><td align="char" char=".">1</td><td align="char" char=".">0.38</td><td align="char" char=".">0.19</td><td align="char" char=".">0.25</td><td align="char" char=".">0.25</td></tr><tr><td>S83F-D87A</td><td>2 SNPs in <italic>gyrA</italic> (codons 83 and 87)</td><td align="char" char=".">192</td><td align="char" char=".">1.5</td><td align="char" char=".">0.38</td><td align="char" char=".">0.25</td><td align="char" char=".">0.38</td><td align="char" char=".">0.25</td></tr><tr><td>S83F-D87N</td><td>2 SNPs in <italic>gyrA</italic> (codons 83 and 87)</td><td align="char" char=".">64</td><td align="char" char=".">0.75</td><td align="char" char=".">0.38</td><td align="char" char=".">0.19</td><td align="char" char=".">0.19</td><td align="char" char=".">0.19</td></tr><tr><td>S83F-D87G-S80I</td><td>2 SNP in <italic>gyrA</italic> (codons 83 &amp; 87) and SNP in <italic>parC</italic> (codon 80)</td><td align="char" char=".">256</td><td align="char" char=".">24</td><td align="char" char=".">16</td><td align="char" char=".">8</td><td align="char" char=".">2</td><td align="char" char=".">3</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01322-v1.xml">elife-01322-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.01322.004</object-id><label>Table 1.</label><caption><p>Summary of the mutational spectrum observed in strains derived from directed evolution of resistance to ionizing radiation</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01322.004">http://dx.doi.org/10.7554/eLife.01322.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th/><th colspan="7">IR-1-20</th><th colspan="7">IR-2-20</th><th colspan="7">IR-3-20</th><th>IR-4-20</th><th colspan="5">IR-CB1000-20</th><th colspan="4">IR-CB1000-30</th></tr><tr><th rowspan="6">Transitions</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th></tr><tr><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th></tr><tr><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>2</th><th>2</th><th>2</th><th>2</th><th>2</th><th>2</th><th>2</th><th>3</th><th>3</th><th>3</th><th>3</th><th>3</th><th>3</th><th>3</th><th>4</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th></tr><tr><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th></tr><tr><th>0</th><th>1</th><th>1</th><th>1</th><th>1</th><th>2</th><th>2</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>1</th><th>1</th><th>0</th><th>0</th><th>0</th><th>0</th><th>1</th><th>1</th><th>1</th><th>0</th><th>2</th><th>2</th><th>2</th><th>2</th><th>3</th><th>0</th><th>0</th><th>0</th><th>0</th></tr><tr><th>0</th><th>2</th><th>3</th><th>4</th><th>5</th><th>4</th><th>5</th><th>0</th><th>3</th><th>4</th><th>7</th><th>9</th><th>4</th><th>6</th><th>0</th><th>2</th><th>6</th><th>8</th><th>4</th><th>7</th><th>9</th><th>0</th><th>2</th><th>4</th><th>5</th><th>7</th><th>0</th><th>1</th><th>4</th><th>5</th><th>8</th></tr></thead><tbody><tr><td>&#8195;C &#8594; T</td><td>19</td><td>20</td><td>6</td><td>10</td><td>15</td><td>6</td><td>9</td><td>12</td><td>13</td><td>11</td><td>12</td><td>11</td><td>9</td><td>9</td><td>14</td><td>14</td><td>13</td><td>14</td><td>13</td><td>12</td><td>12</td><td>14</td><td>59</td><td>64</td><td>69</td><td>69</td><td>63</td><td>60</td><td>62</td><td>63</td><td>62</td></tr><tr><td>&#8195;G &#8594; A</td><td>18</td><td>9</td><td>11</td><td>16</td><td>11</td><td>11</td><td>12</td><td>17</td><td>17</td><td>16</td><td>15</td><td>13</td><td>13</td><td>11</td><td>17</td><td>11</td><td>13</td><td>13</td><td>12</td><td>8</td><td>15</td><td>10</td><td>60</td><td>65</td><td>78</td><td>71</td><td>60</td><td>57</td><td>60</td><td>60</td><td>60</td></tr><tr><td>&#8195;T &#8594; C</td><td>7</td><td>7</td><td>6</td><td>4</td><td>6</td><td>13</td><td>4</td><td>11</td><td>8</td><td>16</td><td>10</td><td>9</td><td>9</td><td>11</td><td>11</td><td>4</td><td>13</td><td>11</td><td>7</td><td>6</td><td>9</td><td>7</td><td>51</td><td>51</td><td>40</td><td>44</td><td>47</td><td>47</td><td>47</td><td>47</td><td>47</td></tr><tr><td>&#8195;A &#8594; G</td><td>15</td><td>16</td><td>7</td><td>8</td><td>8</td><td>14</td><td>11</td><td>10</td><td>6</td><td>6</td><td>7</td><td>7</td><td>6</td><td>5</td><td>6</td><td>14</td><td>10</td><td>10</td><td>9</td><td>11</td><td>6</td><td>8</td><td>42</td><td>43</td><td>42</td><td>54</td><td>50</td><td>46</td><td>50</td><td>50</td><td>50</td></tr><tr><td>Transversions</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;G &#8594; T</td><td>4</td><td>4</td><td>1</td><td>1</td><td>1</td><td>3</td><td>5</td><td>3</td><td>4</td><td>2</td><td>3</td><td>5</td><td>4</td><td>2</td><td>5</td><td>5</td><td>4</td><td>4</td><td>4</td><td>6</td><td>4</td><td>2</td><td>10</td><td>9</td><td>7</td><td>8</td><td>6</td><td>5</td><td>6</td><td>6</td><td>6</td></tr><tr><td>&#8195;C &#8594; A</td><td>1</td><td>1</td><td>2</td><td>1</td><td>1</td><td>5</td><td>6</td><td>5</td><td>3</td><td>6</td><td>3</td><td>3</td><td>4</td><td>6</td><td>2</td><td>4</td><td>2</td><td>1</td><td>3</td><td>4</td><td>1</td><td>6</td><td>3</td><td>2</td><td>5</td><td>5</td><td>6</td><td>4</td><td>5</td><td>5</td><td>4</td></tr><tr><td>&#8195;A &#8594; C</td><td>1</td><td>1</td><td>2</td><td/><td/><td/><td/><td>1</td><td/><td/><td>1</td><td>1</td><td>2</td><td/><td>1</td><td/><td>1</td><td>2</td><td>2</td><td/><td>1</td><td>1</td><td>7</td><td>4</td><td>2</td><td>2</td><td>4</td><td>4</td><td>3</td><td>3</td><td>3</td></tr><tr><td>&#8195;T &#8594; G</td><td/><td/><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td/><td/><td>1</td><td/><td/><td/><td>1</td><td>1</td><td>1</td><td>1</td><td>2</td><td>1</td><td>2</td><td>2</td><td/><td>2</td><td>4</td><td/><td/><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td></tr><tr><td>&#8195;G &#8594; C</td><td>1</td><td>1</td><td>1</td><td>2</td><td>2</td><td/><td>1</td><td>1</td><td/><td/><td>1</td><td/><td/><td/><td/><td>1</td><td>1</td><td/><td/><td>1</td><td/><td>1</td><td>2</td><td>1</td><td>1</td><td>2</td><td>4</td><td>3</td><td>3</td><td>3</td><td>3</td></tr><tr><td>&#8195;C &#8594; G</td><td/><td>1</td><td/><td/><td/><td/><td>2</td><td>3</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td/><td/><td/><td/><td/><td>1</td><td/><td/><td>2</td><td>1</td><td>2</td><td>1</td><td>1</td><td>2</td><td>2</td><td>2</td><td>2</td><td>2</td></tr><tr><td>&#8195;A &#8594; T</td><td/><td/><td>1</td><td/><td>2</td><td/><td/><td>1</td><td>1</td><td>3</td><td>1</td><td>2</td><td>1</td><td>4</td><td>3</td><td>1</td><td>1</td><td>2</td><td>1</td><td>1</td><td>1</td><td/><td>2</td><td>2</td><td>1</td><td>2</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td></tr><tr><td>&#8195;T &#8594; A</td><td/><td>1</td><td/><td>1</td><td/><td>1</td><td/><td>1</td><td>1</td><td>1</td><td>1</td><td/><td/><td>1</td><td>1</td><td>1</td><td>3</td><td>1</td><td>1</td><td>1</td><td>2</td><td>1</td><td>3</td><td>3</td><td>2</td><td>3</td><td>3</td><td>3</td><td>3</td><td>3</td><td>2</td></tr><tr><td>Insertions</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;767bp IS1</td><td>2</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>Deletions</td><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;e14</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td></tr><tr><td>&#8195;93bp</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>Totals</td><td>70</td><td>63</td><td>40</td><td>46</td><td>49</td><td>56</td><td>53</td><td>66</td><td>55</td><td>64</td><td>56</td><td>53</td><td>50</td><td>51</td><td>63</td><td>57</td><td>63</td><td>61</td><td>55</td><td>53</td><td>54</td><td>53</td><td>243</td><td>251</td><td>249</td><td>265</td><td>250</td><td>236</td><td>246</td><td>247</td><td>244</td></tr></tbody></table><table-wrap-foot><fn><p>The mutational spectrum of the population is inferred based on the genetic alterations observed in the subset of strains sequenced.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01322-v1.xml">elife-01322-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.01322.005</object-id><label>Table 2.</label><caption><p>Summary of prominent mutational patterns observed in multiple evolved populations</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01322.005">http://dx.doi.org/10.7554/eLife.01322.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th/><th/><th/><th colspan="7">IR-1-20</th><th colspan="7">IR-2-20</th><th colspan="7">IR-3-20</th><th>IR-4-20</th><th>IR CB1000&#8722;+</th><th/><th/></tr><tr><th rowspan="6">Gene</th><th rowspan="6">Position</th><th rowspan="5">Ref.</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th>C</th><th rowspan="6"/><th rowspan="6">Change</th><th rowspan="5">Mutation</th></tr><tr><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th><th>B</th></tr><tr><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>1</th><th>2</th><th>2</th><th>2</th><th>2</th><th>2</th><th>2</th><th>2</th><th>3</th><th>3</th><th>3</th><th>3</th><th>3</th><th>3</th><th>3</th><th>4</th></tr><tr><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th></tr><tr><th>0</th><th>1</th><th>1</th><th>1</th><th>1</th><th>2</th><th>2</th><th>0</th><th>0</th><th>0</th><th>0</th><th>0</th><th>1</th><th>1</th><th>0</th><th>0</th><th>0</th><th>0</th><th>1</th><th>1</th><th>1</th><th>0</th></tr><tr><th>Allele</th><th>0</th><th>2</th><th>3</th><th>4</th><th>5</th><th>4</th><th>5</th><th>0</th><th>3</th><th>4</th><th>7</th><th>9</th><th>4</th><th>6</th><th>0</th><th>2</th><th>6</th><th>8</th><th>4</th><th>7</th><th>9</th><th>0</th><th>Type</th></tr></thead><tbody><tr><td><bold><italic>clpP</italic></bold></td><td><bold>456127</bold></td><td><bold>A</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td rowspan="3"><bold>+</bold></td><td><bold>Y75C</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>clpP/clpX</italic></bold></td><td><bold>456637</bold></td><td><bold>G</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>A</bold></td><td><bold>A</bold></td><td><bold>A</bold></td><td><bold>A</bold></td><td><bold>A</bold></td><td><bold>A</bold></td><td><bold>A</bold></td><td/><td><bold>-</bold></td><td><bold>I</bold></td></tr><tr><td><bold><italic>clpX</italic></bold></td><td><bold>457803</bold></td><td><bold>A</bold></td><td/><td/><td/><td><bold>G</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>Y384C</bold></td><td><bold>N</bold></td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td><bold><italic>gsiB</italic></bold></td><td><bold>868947</bold></td><td><bold>A</bold></td><td/><td/><td/><td/><td/><td/><td/><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td rowspan="3"/><td><bold>N104S</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>gsiB</italic></bold></td><td><bold>869499</bold></td><td><bold>T</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>C</bold></td><td><bold>C</bold></td><td><bold>C</bold></td><td><bold>C</bold></td><td><bold>C</bold></td><td><bold>C</bold></td><td><bold>C</bold></td><td/><td><bold>L288P</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>gsiB</italic></bold></td><td><bold>870075</bold></td><td><bold>T</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>C</bold></td><td><bold>V480A</bold></td><td><bold>N</bold></td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td><bold><italic>fnr</italic></bold></td><td><bold>1396995</bold></td><td><bold>A</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td/><td><bold>+</bold></td><td><bold>F185I</bold></td><td><bold>N</bold></td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td><bold><italic>rsxB</italic></bold></td><td><bold>1704735</bold></td><td><bold>A</bold></td><td/><td/><td/><td/><td/><td/><td/><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td rowspan="2"><bold>+</bold></td><td><bold>K121E</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>rsxD</italic></bold></td><td><bold>1707299</bold></td><td><bold>T</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>C</bold></td><td><bold>C</bold></td><td><bold>C</bold></td><td><bold>C</bold></td><td><bold>C</bold></td><td><bold>C</bold></td><td><bold>C</bold></td><td/><td><bold>V44A</bold></td><td><bold>N</bold></td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td><bold><italic>wcaM</italic></bold></td><td><bold>2113451</bold></td><td><bold>T</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>C</bold></td><td rowspan="3"><bold>+</bold></td><td><bold>N156S</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>wcaK</italic></bold></td><td><bold>2116031</bold></td><td><bold>T</bold></td><td/><td/><td/><td/><td/><td/><td/><td><bold>C</bold></td><td><bold>C</bold></td><td/><td><bold>C</bold></td><td><bold>C</bold></td><td><bold>C</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>Y132C</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>wcaC</italic></bold></td><td><bold>2129153</bold></td><td><bold>A</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td/><td><bold>S313S</bold></td><td><bold>S</bold></td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td><bold><italic>yfjK</italic></bold></td><td><bold>2759609</bold></td><td><bold>G</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>A</bold></td><td rowspan="4"/><td><bold>H651Y</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>yfjK</italic></bold></td><td><bold>2760683</bold></td><td><bold>T</bold></td><td/><td/><td/><td/><td/><td><bold>C</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>K293E</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>yfjK</italic></bold></td><td><bold>2760809</bold></td><td><bold>G</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>A</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>P251S</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>yfjK</italic></bold></td><td><bold>2761108</bold></td><td><bold>G</bold></td><td/><td/><td/><td/><td/><td/><td/><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>A151D</bold></td><td><bold>N</bold></td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td><bold><italic>recA</italic></bold></td><td><bold>2820924</bold></td><td><bold>C</bold></td><td/><td/><td/><td/><td/><td><bold>A</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td rowspan="3"/><td><bold>A289S</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>recA</italic></bold></td><td><bold>2820962</bold></td><td><bold>T</bold></td><td/><td/><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td/><td><bold>G</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>D276A</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>recA</italic></bold></td><td><bold>2820963</bold></td><td><bold>C</bold></td><td/><td/><td/><td/><td/><td/><td/><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td/><td/><td/><td/><td/><td/><td/><td><bold>T</bold></td><td><bold>D276N</bold></td><td><bold>N</bold></td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td><bold><italic>nanE</italic></bold></td><td><bold>3368674</bold></td><td><bold>C</bold></td><td/><td/><td/><td/><td/><td/><td/><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td rowspan="2"><bold>+</bold></td><td><bold>A128T</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>nanT</italic></bold></td><td><bold>3369380</bold></td><td><bold>A</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td/><td><bold>F405S</bold></td><td><bold>N</bold></td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td><bold><italic>dnaB</italic></bold></td><td><bold>4262560</bold></td><td><bold>T</bold></td><td/><td/><td/><td/><td/><td><bold>C</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td rowspan="6"><bold>+</bold></td><td><bold>L74S</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>dnaB</italic></bold></td><td><bold>4262578</bold></td><td><bold>C</bold></td><td/><td/><td/><td/><td/><td/><td/><td><bold>A</bold></td><td><bold>A</bold></td><td><bold>A</bold></td><td><bold>A</bold></td><td><bold>A</bold></td><td><bold>A</bold></td><td><bold>A</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>P80H</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>dnaB</italic></bold></td><td><bold>4262935</bold></td><td><bold>C</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>A</bold></td><td><bold>P199Q</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>priA</italic></bold></td><td><bold>4123174</bold></td><td><bold>C</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td><bold>T</bold></td><td/><td><bold>V553I</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>priC</italic></bold></td><td><bold>489549</bold></td><td><bold>A</bold></td><td><bold>G</bold></td><td><bold>G</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>L162P</bold></td><td><bold>N</bold></td></tr><tr><td><bold><italic>dnaT</italic></bold></td><td><bold>4599105</bold></td><td><bold>G</bold></td><td/><td/><td><bold>A</bold></td><td><bold>A</bold></td><td><bold>A</bold></td><td/><td><bold>A</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td><bold>R145C</bold></td><td><bold>N</bold></td></tr></tbody></table><table-wrap-foot><fn><p>Entries in red denote mutations that are present in CB2000.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01370-v1.xml">elife-01370-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.01370.010</object-id><label>Table 1.</label><caption><p>Eigenvalues (in %) obtained by PCA, and the angle between restrictive subspaces</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01370.010">http://dx.doi.org/10.7554/eLife.01370.010</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">PDB ID</th><th colspan="5">PIPER</th><th colspan="5">RosettaDock</th><th rowspan="2">Discrepancy (degrees)</th></tr><tr><th>&#955;<sub>1</sub></th><th>&#955;<sub>2</sub></th><th>&#955;<sub>3</sub></th><th>&#955;<sub>4</sub></th><th>&#955;<sub>5</sub></th><th>&#955;<sub>1</sub></th><th>&#955;<sub>2</sub></th><th>&#955;<sub>3</sub></th><th>&#955;<sub>4</sub></th><th>&#955;<sub>5</sub></th></tr></thead><tbody><tr><td>1AVX</td><td>59.4</td><td>32.8</td><td>6.2</td><td>1.2</td><td>0.3</td><td>67.4</td><td>15.5</td><td>13.4</td><td>3.4</td><td>0.3</td><td>5</td></tr><tr><td>1B6C</td><td>72.1</td><td>19.1</td><td>6.9</td><td>1.3</td><td>0.5</td><td>84.2</td><td>10.2</td><td>3.3</td><td>1.9</td><td>0.4</td><td>4</td></tr><tr><td>1 E6E</td><td>59.1</td><td>18.1</td><td>11.3</td><td>10.0</td><td>1.5</td><td>57.6</td><td>16.3</td><td>10.9</td><td>8.8</td><td>6.4</td><td>29</td></tr><tr><td>1EAW</td><td>44.3</td><td>31.7</td><td>22.2</td><td>1.0</td><td>0.9</td><td>57.9</td><td>33.0</td><td>4.6</td><td>4.0</td><td>0.4</td><td>25</td></tr><tr><td>1 E6J</td><td>78.7</td><td>13.5</td><td>7.1</td><td>0.3</td><td>0.3</td><td>47.6</td><td>31.5</td><td>18.7</td><td>1.2</td><td>1.0</td><td>16</td></tr><tr><td>1GLA</td><td>58.9</td><td>26.9</td><td>9.3</td><td>3.7</td><td>1.2</td><td>41.5</td><td>33.1</td><td>15.1</td><td>8.0</td><td>2.3</td><td>2</td></tr><tr><td>1IQD</td><td>74.7</td><td>13.5</td><td>7.8</td><td>3.6</td><td>0.4</td><td>58.0</td><td>26.7</td><td>12.7</td><td>1.9</td><td>0.7</td><td>13</td></tr><tr><td>1K74</td><td>47.8</td><td>28.0</td><td>19.0</td><td>3.6</td><td>1.5</td><td>61.0</td><td>22.0</td><td>10.5</td><td>5.2</td><td>1.2</td><td>19</td></tr><tr><td>1MAH</td><td>60.3</td><td>22.4</td><td>11.7</td><td>4.4</td><td>1.2</td><td>52.8</td><td>22.0</td><td>13.5</td><td>7.7</td><td>4.0</td><td>14</td></tr><tr><td>1N8O</td><td>56.1</td><td>23.4</td><td>13.3</td><td>6.4</td><td>0.9</td><td>66.9</td><td>22.3</td><td>10.3</td><td>0.3</td><td>0.2</td><td>20</td></tr><tr><td>1PPE</td><td>56.4</td><td>26.4</td><td>14.9</td><td>1.7</td><td>0.6</td><td>47.1</td><td>44.4</td><td>7.9</td><td>0.4</td><td>0.1</td><td>3</td></tr><tr><td>1PXV</td><td>68.3</td><td>17.0</td><td>9.6</td><td>4.3</td><td>0.8</td><td>32.1</td><td>27.2</td><td>23.6</td><td>14.5</td><td>2.7</td><td>8</td></tr><tr><td>1 R0R</td><td>55.0</td><td>26.8</td><td>15.3</td><td>2.7</td><td>0.2</td><td>69.3</td><td>20.2</td><td>6.7</td><td>3.0</td><td>0.9</td><td>13</td></tr><tr><td>2SNI</td><td>49.3</td><td>31.6</td><td>17.0</td><td>1.5</td><td>0.6</td><td>79.6</td><td>15.5</td><td>3.5</td><td>1.2</td><td>0.2</td><td>16</td></tr><tr><td>1KXQ</td><td>47.7</td><td>30.0</td><td>16.8</td><td>4.1</td><td>1.3</td><td>66.3</td><td>30.0</td><td>4.4</td><td>0.2</td><td>0.1</td><td>29</td></tr><tr><td>7CEI</td><td>44.7</td><td>28.5</td><td>20.9</td><td>4.6</td><td>1.3</td><td>47.7</td><td>27.9</td><td>18.9</td><td>3.6</td><td>1.9</td><td>19</td></tr><tr><td>2SIC</td><td>58.6</td><td>23.4</td><td>9.4</td><td>7.2</td><td>1.4</td><td>84.2</td><td>8.8</td><td>4.1</td><td>2.4</td><td>0.5</td><td>3</td></tr><tr><td>1AY7</td><td>56.9</td><td>20.3</td><td>15.0</td><td>5.4</td><td>2.4</td><td>42.1</td><td>32.5</td><td>12.6</td><td>9.2</td><td>3.7</td><td>27</td></tr><tr><td>1OPH</td><td>72.6</td><td>15.5</td><td>9.2</td><td>2.2</td><td>0.5</td><td>84.2</td><td>9.3</td><td>5.9</td><td>0.4</td><td>0.2</td><td>21</td></tr><tr><td>1UDI</td><td>64.6</td><td>18.6</td><td>12.6</td><td>2.4</td><td>1.8</td><td>51.2</td><td>27.1</td><td>14.2</td><td>6.4</td><td>1.1</td><td>33</td></tr><tr><td>1BUH</td><td>44.8</td><td>27.7</td><td>17.6</td><td>9.2</td><td>0.6</td><td>40.3</td><td>32.9</td><td>16.6</td><td>8.2</td><td>1.9</td><td>21</td></tr><tr><td>1FSK</td><td>45.0</td><td>28.0</td><td>22.1</td><td>3.5</td><td>1.4</td><td>42.6</td><td>29.7</td><td>19.9</td><td>5.8</td><td>1.9</td><td>21</td></tr><tr><td>1JPS</td><td>57.1</td><td>25.7</td><td>12.4</td><td>4.0</td><td>0.8</td><td>56.3</td><td>28.8</td><td>13.7</td><td>0.7</td><td>0.6</td><td>30</td></tr><tr><td>1DQJ</td><td>51.4</td><td>31.3</td><td>15.0</td><td>1.4</td><td>0.9</td><td>46.5</td><td>19.8</td><td>17.4</td><td>12.3</td><td>4.0</td><td>17</td></tr><tr><td>2B42</td><td>55.6</td><td>27.7</td><td>12.8</td><td>3.4</td><td>0.5</td><td>45.4</td><td>23.1</td><td>15.7</td><td>11.5</td><td>4.4</td><td>24</td></tr><tr><td>2FD6</td><td>65.1</td><td>18.1</td><td>9.9</td><td>4.6</td><td>2.2</td><td>36.4</td><td>23.9</td><td>21.0</td><td>13.7</td><td>5.0</td><td>20</td></tr><tr><td>2HQS</td><td>80.1</td><td>11.3</td><td>7.2</td><td>1.0</td><td>0.4</td><td>54.1</td><td>36.2</td><td>7.8</td><td>1.6</td><td>0.4</td><td>9</td></tr><tr><td>2I25</td><td>70.3</td><td>18.5</td><td>9.9</td><td>0.8</td><td>0.5</td><td>56.0</td><td>15.3</td><td>12.8</td><td>10.5</td><td>5.3</td><td>12</td></tr><tr><td>2MTA</td><td>45.8</td><td>26.1</td><td>20.5</td><td>5.2</td><td>2.5</td><td>45.5</td><td>32.2</td><td>12.2</td><td>7.4</td><td>2.7</td><td>30</td></tr><tr><td>1MLC</td><td>59.3</td><td>31.5</td><td>6.8</td><td>1.2</td><td>1.1</td><td>42.7</td><td>30.1</td><td>17.3</td><td>7.0</td><td>2.8</td><td>17</td></tr><tr><td>2HRK</td><td>57.7</td><td>31.0</td><td>9.7</td><td>0.9</td><td>0.7</td><td>61.2</td><td>16.4</td><td>10.5</td><td>8.6</td><td>3.3</td><td>30</td></tr><tr><td>1AHW</td><td>74.4</td><td>16.3</td><td>7.7</td><td>1.1</td><td>0.5</td><td>46.7</td><td>31.5</td><td>17.2</td><td>3.1</td><td>1.5</td><td>24</td></tr><tr><td>1Z5Y</td><td>66.1</td><td>18.0</td><td>8.9</td><td>5.4</td><td>1.5</td><td>57.1</td><td>32.3</td><td>9.7</td><td>0.6</td><td>0.4</td><td>29</td></tr><tr><td>2HLE</td><td>54.0</td><td>28.1</td><td>13.0</td><td>3.4</td><td>1.4</td><td>67.0</td><td>12.9</td><td>12.1</td><td>5.9</td><td>2.1</td><td>2</td></tr><tr><td>2NZ8</td><td>69.4</td><td>14.1</td><td>8.4</td><td>5.2</td><td>2.9</td><td>44.8</td><td>28.3</td><td>15.1</td><td>7.3</td><td>4.5</td><td>34</td></tr><tr><td>1BVN</td><td>61.4</td><td>20.3</td><td>14.3</td><td>3.6</td><td>0.4</td><td>37.2</td><td>28.5</td><td>17.8</td><td>10.0</td><td>6.5</td><td>32</td></tr><tr><td>1CGI</td><td>66.9</td><td>15.4</td><td>10.9</td><td>5.4</td><td>1.9</td><td>57.9</td><td>27.9</td><td>10.7</td><td>3.0</td><td>0.5</td><td>48</td></tr><tr><td>1GPW</td><td>53.8</td><td>20.9</td><td>14.2</td><td>6.4</td><td>4.7</td><td>46.5</td><td>26.5</td><td>17.9</td><td>5.0</td><td>4.0</td><td>27</td></tr><tr><td>2JEL</td><td>72.4</td><td>16.3</td><td>8.1</td><td>2.2</td><td>0.9</td><td>47.8</td><td>31.6</td><td>12.9</td><td>6.8</td><td>0.8</td><td>31</td></tr><tr><td>1NCA</td><td>76.7</td><td>18.8</td><td>3.0</td><td>1.2</td><td>0.4</td><td>55.9</td><td>24.3</td><td>15.6</td><td>3.2</td><td>1.0</td><td>27</td></tr><tr><td>2UUY</td><td>69.4</td><td>15.4</td><td>12.7</td><td>1.6</td><td>0.9</td><td>73.6</td><td>16.2</td><td>9.0</td><td>1.0</td><td>0.2</td><td>11</td></tr><tr><td>1KAC</td><td>48.9</td><td>34.6</td><td>13.3</td><td>2.0</td><td>1.2</td><td>53.8</td><td>19.5</td><td>15.6</td><td>6.8</td><td>4.2</td><td>30</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01457-v1.xml">elife-01457-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.01457.012</object-id><label>Table 3.</label><caption><p>The CDR3 sequences of bulk-sorted V&#945;11<sup>+</sup>V&#946;3<sup>+</sup> CD4<sup>+</sup> T cells from gp250 SC mice that overlap with the published data from normal mice (<xref ref-type="bibr" rid="bib20">Hedrick et al., 1988</xref>; <xref ref-type="bibr" rid="bib38">McHeyzer-Williams et al., 1999</xref>)</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01457.012">http://dx.doi.org/10.7554/eLife.01457.012</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Published CDR3&#945; (<xref ref-type="bibr" rid="bib20">Hedrick et al., 1988</xref>; <xref ref-type="bibr" rid="bib38">McHeyzer-Williams et al., 1999</xref>)</th><th colspan="2">Current study</th><th rowspan="2">Notes</th></tr><tr><th>B6.K</th><th>SC</th></tr><tr><th>3623</th><th>2906</th><th>Unique sequences</th></tr></thead><tbody><tr><td>CAAEASSGQKLVFG</td><td>+</td><td>+</td><td>AND TCR</td></tr><tr><td>CAAEPSSGQKLVFG</td><td>+</td><td>+</td><td>226 TCR</td></tr><tr><td>CAAEASGSWQLIFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAAEASNTNKVVFG</td><td>+</td><td>+</td><td>5c.c7 TCR</td></tr><tr><td>CAAEGSNTNKVVFG</td><td>+</td><td/><td/></tr><tr><td>CAAEASAGNKLTFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAAASSGSWQLIFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAAEAGSNAKLTFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAAEASNNNAPRFG</td><td>+</td><td/><td/></tr><tr><td>CAAEAASLGKLQFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAAEASSGSWQLIFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAAEASNYNVLYFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAAEASSSFSKLVFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAAEASNMGYKLTFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAAETGGYKVVFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAAEANYNQGKLIFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAAEAGSGTYQRFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAGLSGSFNKLTFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAAEGNTGNYKYVFG</td><td/><td>+</td><td/></tr><tr><td>CAAEEGNMGYKLTFG</td><td>+</td><td>+</td><td/></tr><tr><td>CAATSSGQKLVFG</td><td>+</td><td/><td/></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Published CDR3&#946; (<xref ref-type="bibr" rid="bib20">Hedrick et al., 1988</xref>; <xref ref-type="bibr" rid="bib38">McHeyzer-Williams et al., 1999</xref>)</th><th colspan="2">Current study</th><th rowspan="2">Notes</th></tr><tr><th>B6.K</th><th>SC</th></tr><tr><th>2821</th><th>1583</th><th>Unique sequences (mean)</th></tr></thead><tbody><tr><td>CASSLNSANSDY</td><td>+</td><td/><td>AND, 5c.c7, 226 TCRs</td></tr><tr><td>CASSLNNANSDY</td><td>+</td><td>+</td><td/></tr><tr><td>CASSLSTSQNTLYF</td><td>+</td><td/><td/></tr><tr><td>CASSLQGTNTEVFF</td><td>+</td><td>+</td><td/></tr><tr><td>CASRLGQNTLYF</td><td/><td>+</td><td/></tr><tr><td>CASSLGASAETLYF</td><td/><td>+</td><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01519-v1.xml">elife-01519-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.01519.006</object-id><label>Table 1.</label><caption><p>Crystallographic data collection and refinement statistics</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01519.006">http://dx.doi.org/10.7554/eLife.01519.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"/><th>PfDTD+D-Tyr3AA</th><th>PfDTD+D-Tyr3AA</th></tr><tr><th>Crystal I</th><th>Crystal II</th></tr><tr><th>(PDB id: <ext-link ext-link-type="uri" xlink:href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NBI">4NBI</ext-link>)</th><th>(PDB id: <ext-link ext-link-type="uri" xlink:href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NBJ">4NBJ</ext-link>)</th></tr></thead><tbody><tr><td>Data Collection</td><td/><td/></tr><tr><td>&#8195;Space group</td><td>C2</td><td>P2<sub>1</sub></td></tr><tr><td>&#8195;Cell dimensions:</td><td/><td/></tr><tr><td>&#8195;<italic>a</italic> (&#197;)</td><td align="char" char=".">82.13</td><td align="char" char=".">90.90</td></tr><tr><td>&#8195;<italic>b</italic> (&#197;)</td><td align="char" char=".">65.74</td><td align="char" char=".">79.91</td></tr><tr><td>&#8195;<italic>c</italic> (&#197;)</td><td align="char" char=".">56.92</td><td align="char" char=".">95.02</td></tr><tr><td>&#8195;&#946; (&#176;)</td><td align="char" char=".">93.30</td><td align="char" char=".">93.51</td></tr><tr><td>&#8195;Resolution range (&#197;)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>25.0&#8211;1.86 (1.93&#8211;1.86)</td><td>25.0&#8211;2.20 (2.28&#8211;2.20)</td></tr><tr><td>&#8195;Total Observations</td><td align="char" char=".">178996</td><td align="char" char=".">449245</td></tr><tr><td>&#8195;Unique reflections</td><td>25156 (2283)</td><td>69275 (6911)</td></tr><tr><td>&#8195;Completeness (%)</td><td>98.3 (89.3)</td><td>100 (99.9)</td></tr><tr><td>&#8195;R<sub>merge</sub> (%)</td><td>7.5 (28.3)</td><td>11.8 (59.3)</td></tr><tr><td>&#8195;&lt;I/(&#963;)I&gt;</td><td>29.4 (5.5)</td><td>17.7 (2.8)</td></tr><tr><td>&#8195;Redundancy</td><td>7.1 (6.4)</td><td>6.5 (5.9)</td></tr><tr><td>Data refinement</td><td/><td/></tr><tr><td>&#8195;Resolution (&#197;)</td><td align="char" char=".">1.86</td><td align="char" char=".">2.20</td></tr><tr><td>&#8195;No. of reflections</td><td align="char" char=".">23878</td><td align="char" char=".">65743</td></tr><tr><td>&#8195;R (%)</td><td align="char" char=".">16.77</td><td align="char" char=".">19.46</td></tr><tr><td>&#8195;R<sub>free</sub> (%)<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td align="char" char=".">19.25</td><td align="char" char=".">25.35</td></tr><tr><td>&#8195;Monomers/a.u.</td><td align="char" char=".">2</td><td align="char" char=".">8</td></tr><tr><td>&#8195;No. of residues</td><td align="char" char=".">323</td><td align="char" char=".">1289</td></tr><tr><td>&#8195;No. of atoms</td><td align="char" char=".">2917</td><td align="char" char=".">10727</td></tr><tr><td>&#8195;Protein</td><td align="char" char=".">2595</td><td align="char" char=".">10104</td></tr><tr><td>&#8195;Ligand</td><td align="char" char=".">72</td><td align="char" char=".">248</td></tr><tr><td>&#8195;Water</td><td align="char" char=".">250</td><td align="char" char=".">375</td></tr><tr><td>&#8195;R.m.s. deviation</td><td/><td/></tr><tr><td>&#8195;Bond lengths (&#197;)</td><td align="char" char=".">0.007</td><td align="char" char=".">0.010</td></tr><tr><td>&#8195;Bond angles (&#176;)</td><td align="char" char=".">1.093</td><td align="char" char=".">1.436</td></tr><tr><td>&#8195;Mean B value (&#197;<sup>2</sup>)</td><td align="char" char=".">30.33</td><td align="char" char=".">48.11</td></tr><tr><td>&#8195;Protein</td><td align="char" char=".">29.09</td><td align="char" char=".">47.69</td></tr><tr><td>&#8195;Ligand</td><td align="char" char=".">43.94</td><td align="char" char=".">53.30</td></tr><tr><td>&#8195;Water</td><td align="char" char=".">41.29</td><td align="char" char=".">57.16</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>Values in parentheses are for the highest resolution shell.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>Throughout the refinement, 5% of the total reflections were held aside for R<sub>free</sub>.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01569-v1.xml">elife-01569-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.01569.005</object-id><label>Table 1.</label><caption><p>Summary of all clones analysed in both embryo and L3 of the same individual</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01569.005">http://dx.doi.org/10.7554/eLife.01569.005</ext-link></p></caption><table frame="void" rules="none"><thead><tr><th rowspan="3"/><th colspan="20">Clone #</th></tr><tr><th colspan="2">1</th><th colspan="2">2</th><th colspan="2">3</th><th colspan="2">4</th><th colspan="2">5</th><th colspan="2">6</th><th colspan="2">7</th><th colspan="2">8</th><th colspan="2">9</th><th colspan="2">10</th></tr><tr><th>E</th><th>L</th><th>E</th><th>L</th><th>E</th><th>L</th><th>E</th><th>L</th><th>E</th><th>L</th><th>E</th><th>L</th><th>E</th><th>L</th><th>E</th><th>L</th><th>E</th><th>L</th><th>E</th><th>L</th></tr></thead><tbody><tr><td>&#8722;I</td><td>2</td><td>2 <bold>&#8722;I</bold></td><td>2</td><td>2 <bold>&#8722;I</bold></td><td>4</td><td>4 <bold>&#8722;I</bold></td><td>2</td><td>2 <bold>&#8722;I</bold></td><td>4</td><td>4 <bold>&#8722;I</bold></td><td>2</td><td>2 <bold>&#8722;I</bold></td><td>1</td><td>1 <bold>&#8722;I</bold></td><td>4</td><td>4 <bold>&#8722;I</bold></td><td>4</td><td>4 <bold>&#8722;I</bold></td><td>4</td><td>4 <bold>&#8722;I</bold></td></tr><tr><td rowspan="2">I</td><td rowspan="2">4</td><td>1 <bold>I&#8217;</bold></td><td rowspan="2">4</td><td>1 <bold>I&#8217;</bold></td><td rowspan="2">3</td><td>1 <bold>I&#8217;</bold></td><td rowspan="2">3</td><td>1 <bold>I&#8217;</bold></td><td rowspan="2">4</td><td>2 <bold>I&#8217;</bold> *</td><td rowspan="2">4</td><td>1 <bold>I&#8217;</bold></td><td rowspan="2">3</td><td>1 <bold>I&#8217;</bold></td><td rowspan="2">4</td><td>1 <bold>I&#8217;</bold></td><td rowspan="2">4</td><td>1 <bold>I&#8217;</bold></td><td rowspan="2">4</td><td>2 <bold>I&#8217;</bold> *</td></tr><tr><td>3 <bold>I</bold></td><td>3 <bold>I</bold></td><td>2 <bold>I</bold></td><td>2 <bold>I</bold></td><td>2 <bold>I</bold></td><td>3 <bold>I</bold></td><td>2 <bold>I</bold></td><td>3 <bold>I</bold></td><td>3 <bold>I</bold></td><td>2 <bold>I</bold></td></tr><tr><td>II</td><td>4</td><td>4 <bold>T1</bold></td><td>3</td><td>3 <bold>T1</bold></td><td>3</td><td>3 <bold>T1</bold></td><td>3</td><td>3 <bold>T1</bold></td><td>2</td><td>2 <bold>T1</bold></td><td>2</td><td>2 <bold>T1</bold></td><td>4</td><td>4 <bold>T1</bold></td><td>2</td><td>2 <bold>T1</bold></td><td>4</td><td>4 <bold>T1</bold></td><td>2</td><td>2 <bold>T1</bold></td></tr><tr><td>III</td><td>1</td><td>1 <bold>III</bold></td><td>2</td><td>2 <bold>III</bold></td><td>2</td><td>2 <bold>III</bold></td><td>3</td><td>3 <bold>III</bold></td><td>1</td><td>1 <bold>III</bold></td><td>4</td><td>4 <bold>III</bold></td><td>1</td><td>1 <bold>III</bold></td><td>2</td><td>2 <bold>III</bold></td><td>1</td><td>1 <bold>III</bold></td><td>2</td><td>2 <bold>III</bold></td></tr><tr><td rowspan="2">IV</td><td rowspan="2">4</td><td>3 <bold>IV&#8217;</bold></td><td rowspan="2">4</td><td>2 <bold>IV&#8217;</bold></td><td rowspan="2">3</td><td>1 <bold>IV&#8217;</bold></td><td rowspan="2">2</td><td rowspan="2">2 <bold>IV&#8217;</bold></td><td rowspan="2">4</td><td>3 <bold>IV&#8217;</bold></td><td rowspan="2">1</td><td rowspan="2">1 <bold>IV&#8217;</bold></td><td rowspan="2">4</td><td>2 <bold>IV&#8217;</bold></td><td rowspan="2">3</td><td>1 <bold>IV&#8217;</bold></td><td rowspan="2">3</td><td>1 <bold>IV&#8217;</bold></td><td rowspan="2">3</td><td>2 <bold>IV&#8217;</bold></td></tr><tr><td>1 <bold>IV</bold></td><td>2 <bold>IV</bold></td><td>2 <bold>IV</bold></td><td>1 <bold>IV</bold></td><td>2 <bold>IV</bold></td><td>2 <bold>IV</bold></td><td>2 <bold>IV</bold></td><td>1 <bold>IV</bold></td></tr><tr><td>V</td><td>2</td><td>2 <bold>T2</bold></td><td>2</td><td>2 <bold>T2</bold></td><td>3</td><td>3 <bold>T2</bold></td><td>2</td><td>2 <bold>T2</bold></td><td>2</td><td>2 <bold>T2</bold></td><td>3</td><td>3 <bold>T2</bold></td><td>3</td><td>3 <bold>T2</bold></td><td>2</td><td>2 <bold>T2</bold></td><td>2</td><td>2 <bold>T2</bold></td><td>2</td><td>2 <bold>T2</bold></td></tr><tr><td>VI</td><td>2</td><td>2 <bold>VI</bold></td><td>1</td><td>1 <bold>VI</bold></td><td>1</td><td>1 <bold>VI</bold></td><td>2</td><td>2 <bold>VI</bold></td><td>1</td><td>1 <bold>VI</bold></td><td>1</td><td>1 <bold>VI</bold></td><td>2</td><td>2 <bold>VI</bold></td><td>1</td><td>1 <bold>VI</bold></td><td>2</td><td>2 <bold>VI</bold></td><td>2</td><td>2 <bold>VI</bold></td></tr><tr><td>VII</td><td>5</td><td><bold>5 VII</bold></td><td>2</td><td>2 <bold>VII</bold></td><td>2</td><td>2 <bold>VII</bold></td><td>1</td><td>1 <bold>VII</bold></td><td>1</td><td>1 <bold>VII</bold></td><td>2</td><td>2 <bold>VII</bold></td><td>4</td><td>4 <bold>VII</bold></td><td>1</td><td>1 <bold>VII</bold></td><td>1</td><td>1 <bold>VII</bold></td><td>2</td><td>2 <bold>VII</bold></td></tr><tr><td>+VII</td><td>2</td><td>2 <bold>+VII</bold></td><td>1</td><td>1 <bold>+VII</bold></td><td>2</td><td>2 <bold>+VII</bold></td><td>2</td><td>2 <bold>+VII</bold></td><td>2</td><td>2 <bold>+VII</bold></td><td>4</td><td>4 <bold>+VII</bold></td><td>4</td><td>4 <bold>+VII</bold></td><td>4</td><td>4 <bold>+VII</bold></td><td>2</td><td>2 <bold>+VII</bold></td><td>5</td><td>5 <bold>+VII</bold></td></tr></tbody></table><table-wrap-foot><fn><p>The row headers at the left show the line of cells in the embryo from which each clone originates. The numbers of cells in each clone are shown in arabic numerals. The locations of these same cells (within the lines of cells) in the larva are highlighted in bold roman numerals. Cells in lines I and IV of the embryo contribute to two separate lines of cells in the larva; within the denticulate region, apart from lines I and IV, each embryonic cell contributes to only one line of cells in the larva. Cells in line &#8211;I in the embryo never make denticles in embryo or larva, but we do not know if they contribute to two lines of cells in the larva. However, since the P compartment is made of two lines of cells in the embryo and four lines of cells in the larva, one extra row has to come from somewhere and line &#8211;I is the obvious suspect. (*) Only one these two cells showed denticles in the larva.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01581-v1.xml">elife-01581-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.01581.004</object-id><label>Table 1.</label><caption><p>Stability of antisense ncRNAs</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01581.004">http://dx.doi.org/10.7554/eLife.01581.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Overlap size (bp)</th><th colspan="4">Overlap type</th><th colspan="2">Totals</th></tr><tr><th>ORF&#8211;ORF</th><th>ORF&#8211;XUT</th><th>ORF&#8211;CUT</th><th>ORF&#8211;SUT</th><th>ORF&#8211;ncRNA</th><th>ORF&#8211;unstable ncRNA</th></tr></thead><tbody><tr><td align="char" char=".">50</td><td align="char" char=".">700</td><td align="char" char=".">1066</td><td align="char" char=".">575</td><td align="char" char=".">522</td><td align="char" char=".">1596</td><td>1448 (91%)</td></tr><tr><td align="char" char=".">100</td><td align="char" char=".">449</td><td align="char" char=".">1008</td><td align="char" char=".">543</td><td align="char" char=".">475</td><td align="char" char=".">1507</td><td>1367 (91%)</td></tr><tr><td align="char" char=".">150</td><td align="char" char=".">216</td><td align="char" char=".">967</td><td align="char" char=".">508</td><td align="char" char=".">425</td><td align="char" char=".">1423</td><td>1306 (92%)</td></tr><tr><td align="char" char=".">200</td><td align="char" char=".">136</td><td align="char" char=".">931</td><td align="char" char=".">478</td><td align="char" char=".">403</td><td align="char" char=".">1358</td><td>1249 (92%)</td></tr><tr><td align="char" char=".">250</td><td align="char" char=".">96</td><td align="char" char=".">893</td><td align="char" char=".">434</td><td align="char" char=".">380</td><td align="char" char=".">1287</td><td>1181 (92%)</td></tr><tr><td align="char" char=".">300</td><td align="char" char=".">69</td><td align="char" char=".">812</td><td align="char" char=".">391</td><td align="char" char=".">363</td><td align="char" char=".">1189</td><td>1085 (91%)</td></tr><tr><td align="char" char=".">350</td><td align="char" char=".">62</td><td align="char" char=".">759</td><td align="char" char=".">335</td><td align="char" char=".">351</td><td align="char" char=".">1086</td><td>990 (91%)</td></tr><tr><td align="char" char=".">400</td><td align="char" char=".">54</td><td align="char" char=".">694</td><td align="char" char=".">292</td><td align="char" char=".">334</td><td align="char" char=".">992</td><td>899 (91%)</td></tr><tr><td align="char" char=".">450</td><td align="char" char=".">51</td><td align="char" char=".">637</td><td align="char" char=".">244</td><td align="char" char=".">322</td><td align="char" char=".">904</td><td>811 (90%)</td></tr><tr><td align="char" char=".">500</td><td align="char" char=".">48</td><td align="char" char=".">591</td><td align="char" char=".">204</td><td align="char" char=".">302</td><td align="char" char=".">828</td><td>741 (89%)</td></tr></tbody></table><table-wrap-foot><fn><p>Number of ORFs overlapping with ORFs and various classes of ncRNAs, with various minimum size cut-offs for the overlapping region.</p></fn><fn><p>Totals are given for ORFs overlapping with ncRNAs and with unstable ncRNAs, including a percentage of overlapping ncRNAs that are unstable.</p></fn><fn><p>ncRNA: non-protein coding RNA; ORF: optical reading frame; XUT: Xrn1-sensitive unstable transcript (degraded in cytoplasm); CUT: cryptic unstable transcript (degraded by nuclear exosome); SUT: stable unannotated transcript (not known to be degraded).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01604-v1.xml">elife-01604-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.01604.006</object-id><label>Table 1.</label><caption><p>Risk factors for HSV2 and HIV infection in 15&#8211;30 year old women in a population survey, Karonga District, Malawi 2007&#8211;8</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01604.006">http://dx.doi.org/10.7554/eLife.01604.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"/><th colspan="4">HSV2</th><th colspan="4">HIV</th></tr><tr><th rowspan="2">n/N</th><th rowspan="2">%</th><th rowspan="2">OR (95% CI)</th><th>Age adjusted</th><th rowspan="2">n/N</th><th rowspan="2">%</th><th rowspan="2">OR (95% CI)</th><th>Age adjusted</th></tr><tr><th>OR (95% CI)</th><th>OR (95% CI)</th></tr></thead><tbody><tr><td>All</td><td>870/3419</td><td>25.5</td><td/><td/><td>203/3651</td><td>5.6</td><td/><td/></tr><tr><td>Age</td><td colspan="8"/></tr><tr><td>&#8195;15&#8211;9</td><td>126/1240</td><td>10.2</td><td>1</td><td/><td>18/1314</td><td>1.4</td><td>1</td><td/></tr><tr><td>&#8195;20&#8211;24</td><td>276/1072</td><td>25.8</td><td>3.1 (2.4&#8211;3.9)</td><td/><td>64/1128</td><td>5.7</td><td>4.3 (2.6&#8211;7.4)</td><td/></tr><tr><td>&#8195;25&#8211;30</td><td>468/1107</td><td>42.3</td><td>6.5 (5.2&#8211;8.1)</td><td/><td>121/1209</td><td>10.0</td><td>8.0 (4.8&#8211;13.2)</td><td/></tr><tr><td>Age at menarche</td><td colspan="8"/></tr><tr><td>&#8195;&lt;14</td><td>245/815</td><td>30.1</td><td>1</td><td>1</td><td>51/861</td><td>5.9</td><td>1</td><td>1</td></tr><tr><td>&#8195;14</td><td>195/769</td><td>25.4</td><td>0.79 (0.63&#8211;0.99)</td><td>0.89 (0.71&#8211;1.1)</td><td>43/819</td><td>5.3</td><td>0.88 (0.60&#8211;1.3)</td><td>1.0 (0.66&#8211;1.5)</td></tr><tr><td>&#8195;15</td><td>225/960</td><td>23.4</td><td>0.71 (0.58&#8211;0.88)</td><td>0.71 (0.57&#8211;0.89)</td><td>41/1030</td><td>4.0</td><td>0.66 (0.43&#8211;1.0)</td><td>0.68 (0.44&#8211;1.0)</td></tr><tr><td>&#8195;&#8805;16</td><td>157/622</td><td>25.2</td><td>0.79 (0.62&#8211;0.99)</td><td>0.69 (0.54&#8211;0.89)</td><td>51/665</td><td>7.7</td><td>1.3 (0.88&#8211;2.0)</td><td>1.2 (0.81&#8211;1.8)</td></tr><tr><td>Age at first sex</td><td colspan="8"/></tr><tr><td>&#8195;Never</td><td>51/639</td><td>8.0</td><td>0.21 (0.15&#8211;0.29)</td><td>0.48 (0.33&#8211;0.69)</td><td>10/684</td><td>1.5</td><td>0.23 (0.12&#8211;0.44)</td><td>0.69 (0.32&#8211;1.3)</td></tr><tr><td>&#8195;&lt;16</td><td>296/1014</td><td>29.2</td><td>1</td><td>1</td><td>66/1076</td><td>6.1</td><td>1</td><td>1</td></tr><tr><td>&#8195;16&#8211;17</td><td>235/797</td><td>29.5</td><td>1.0 (0.83&#8211;1.2)</td><td>0.99 (0.80&#8211;1.2)</td><td>53/847</td><td>6.3</td><td>1.0 (0.70&#8211;1.4)</td><td>0.99 (0.68&#8211;1.5)</td></tr><tr><td>&#8195;18+</td><td>273/930</td><td>29.4</td><td>1.0 (0.83&#8211;1.2)</td><td>0.76 (0.62&#8211;0.93)</td><td>71/1002</td><td>7.1</td><td>1.2 (0.82&#8211;1.7)</td><td>0.90 (0.63&#8211;1.3)</td></tr><tr><td>Marital status</td><td colspan="8"/></tr><tr><td>&#8195;Never</td><td>76/826</td><td>9.2</td><td>1</td><td>1</td><td>19/885</td><td>2.2</td><td>1</td><td>1</td></tr><tr><td>&#8195;Current</td><td>681/2330</td><td>29.2</td><td>4.1 (3.2&#8211;5.2)</td><td>1.7 (1.3&#8211;2.3)</td><td>133/2481</td><td>5.4</td><td>2.6 (1.6&#8211;4.2)</td><td>0.78 (0.44&#8211;1.4)</td></tr><tr><td>&#8195;Previous</td><td>113/263</td><td>43.0</td><td>7.4 (5.3&#8211;10.4)</td><td>3.4 (2.4&#8211;5.0)</td><td>51/285</td><td>17.9</td><td>9.9 (5.8&#8211;17.2)</td><td>3.2 (1.7&#8211;5.9)</td></tr><tr><td>Lifetime no. of partners</td><td colspan="8"/></tr><tr><td>&#8195;0</td><td>51/639</td><td>8.0</td><td>0.33 (0.24&#8211;0.46)</td><td>0.76 (0.53&#8211;1.1)</td><td>10/684</td><td>1.5</td><td>0.49 (0.25&#8211;0.98)</td><td>1.4 (0.66&#8211;3.2)</td></tr><tr><td>&#8195;1</td><td>304/1475</td><td>20.6</td><td>1</td><td>1</td><td>46/1574</td><td>2.9</td><td>1</td><td>1</td></tr><tr><td>&#8195;2</td><td>320/900</td><td>35.6</td><td>2.1 (1.8&#8211;2.6)</td><td>2.0 (1.7&#8211;2.4)</td><td>82/951</td><td>5.6</td><td>3.1 (2.2&#8211;4.5)</td><td>2.9 (2.0&#8211;4.2)</td></tr><tr><td>&#8195;3</td><td>139/289</td><td>48.1</td><td>3.6 (2.7&#8211;4.6)</td><td>3.2 (2.4&#8211;4.2)</td><td>41/313</td><td>13.1</td><td>5.0 (3.2&#8211;7.8)</td><td>4.4 (2.8&#8211;6.8)</td></tr><tr><td>&#8195;&#8805;4</td><td>53/107</td><td>49.5</td><td>3.8 (2.5&#8211;5.6)</td><td>3.5 (2.3&#8211;5.3)</td><td>24/120</td><td>20.0</td><td>8.3 (4.9&#8211;14.2)</td><td>7.4 (4.3&#8211;12.8)</td></tr><tr><td>Distance from road</td><td colspan="8"/></tr><tr><td>&#8195;&lt;1 km</td><td>410/1589</td><td>25.8</td><td>1</td><td>1</td><td>100/1595</td><td>6.3</td><td>1</td><td>1</td></tr><tr><td>&#8195;&gt;1 km</td><td>460/1830</td><td>25.1</td><td>0.97 (0.83&#8211;1.1)</td><td>0.94 (0.80&#8211;1.1)</td><td>62/1835</td><td>3.4</td><td>0.52 (0.38&#8211;0.72)</td><td>0.50 (0.36&#8211;0.70)</td></tr><tr><td>Schooling</td><td colspan="8"/></tr><tr><td>&#8195;None/Primary 1&#8211;5</td><td>101/347</td><td>29.1</td><td>1</td><td/><td>16/378</td><td>4.2</td><td>1</td><td>1</td></tr><tr><td>&#8195;Primary 6&#8211;7</td><td>215/889</td><td>24.2</td><td>0.78 (0.59&#8211;1.0)</td><td>0.88 (0.66&#8211;1.2)</td><td>38/945</td><td>4.0</td><td>0.95 (0.52&#8211;1.7)</td><td>1.1 (0.59&#8211;2.0)</td></tr><tr><td>&#8195;Primary 8</td><td>271/1011</td><td>26.8</td><td>0.89 (0.68&#8211;1.2)</td><td>0.89 (0.67&#8211;1.2)</td><td>58/1069</td><td>5.4</td><td>1.3 (0.74&#8211;2.3)</td><td>1.3 (0.75&#8211;2.4)</td></tr><tr><td>&#8195;Secondary 1&#8211;3</td><td>218/860</td><td>25.4</td><td>0.83 (0.63&#8211;1.1)</td><td>0.88 (0.65&#8211;1.2)</td><td>70/916</td><td>7.6</td><td>1.9 (1.1&#8211;3.3)</td><td>2.0 (1.2&#8211;3.6)</td></tr><tr><td>&#8195;Secondary 4/Tertiary</td><td>36/196</td><td>18.4</td><td>0.55 (0.36&#8211;0.84)</td><td>0.42 (0.27&#8211;0.66)</td><td>18/215</td><td>8.4</td><td>2.1 (1.0&#8211;4.1)</td><td>1.8 (0.87&#8211;3.6)</td></tr><tr><td>Mother schooling</td><td colspan="8"/></tr><tr><td>&#8195;&lt;=Primary</td><td>716/2861</td><td>25.0</td><td>1</td><td>1</td><td>151/3021</td><td>5.0</td><td>1</td><td>1</td></tr><tr><td>&#8195;Secondary</td><td>80/352</td><td>22.7</td><td>0.88 (0.68&#8211;1.1)</td><td>1.0 (0.76&#8211;1.3)</td><td>30/382</td><td>7.9</td><td>1.6 (1.1&#8211;2.4)</td><td>1.8 (1.2&#8211;2.8)</td></tr><tr><td>Father schooling</td><td colspan="8"/></tr><tr><td>&#8195;&lt;=Primary</td><td>517/2046</td><td>25.3</td><td>1</td><td>1</td><td>94/2158</td><td>4.4</td><td>1</td><td>1</td></tr><tr><td>&#8195;Secondary</td><td>260/1095</td><td>23.7</td><td>0.92 (0.78&#8211;1.1)</td><td>0.96 (0.80&#8211;1.1)</td><td>81/1166</td><td>7.0</td><td>1.6 (1.2&#8211;2.2)</td><td>1.7 (1.3&#8211;2.3)</td></tr><tr><td>Housing quality</td><td colspan="8"/></tr><tr><td>&#8195;1 (Best)</td><td>148/662</td><td>22.4</td><td>1</td><td>1</td><td>71/732</td><td>9.7</td><td>1</td><td>1</td></tr><tr><td>&#8195;2</td><td>149/565</td><td>26.4</td><td>1.2 (0.96&#8211;1.6)</td><td>1.2 (0.89&#8211;1.5)</td><td>32/598</td><td>5.4</td><td>0.53 (0.34&#8211;0.81)</td><td>0.48 (0.31&#8211;0.75)</td></tr><tr><td>&#8195;3</td><td>317/1238</td><td>25.6</td><td>1.2 (0.96&#8211;1.5)</td><td>1.0 (0.96&#8211;1.3)</td><td>60/1296</td><td>4.6</td><td>0.45 (0.32&#8211;0.65)</td><td>0.38 (0.26&#8211;0.55)</td></tr><tr><td>&#8195;4 (Worst)</td><td>227/843</td><td>26.9</td><td>1.3 (1.0&#8211;1.6)</td><td>1.2 (0.96&#8211;1.6)</td><td>38/903</td><td>4.2</td><td>0.41 (0.27&#8211;0.61)</td><td>0.37 (0.25&#8211;0.56)</td></tr><tr><td>Occupation</td><td colspan="8"/></tr><tr><td>&#8195;Farmer</td><td>652/2229</td><td>29.3</td><td>1</td><td>1</td><td>143/2369</td><td>6.0</td><td>1</td><td>1</td></tr><tr><td>&#8195;In education</td><td>63/720</td><td>8.8</td><td>0.23 (0.18&#8211;0.31)</td><td>0.55 (0.40&#8211;0.76)</td><td>10/767</td><td>1.3</td><td>0.21 (0.11&#8211;0.39)</td><td>0.58 (0.28&#8211;1.2)</td></tr><tr><td>&#8195;Not working</td><td>23/77</td><td>29.9</td><td>1.0 (0.63&#8211;1.7)</td><td>1.1 (0.66&#8211;1.9)</td><td>8/88</td><td>9.1</td><td>1.6 (0.74&#8211;3.3)</td><td>1.6 (0.77&#8211;3.5)</td></tr><tr><td>&#8195;Other</td><td>102/272</td><td>37.5</td><td>1.5 (1.1&#8211;1.9)</td><td>1.3 (1.0&#8211;1.7)</td><td>38/293</td><td>13.0</td><td>2.3 (1.6&#8211;3.4)</td><td>2.1 (1.4&#8211;3.1)</td></tr><tr><td>Times of insufficient food in household in last year</td><td colspan="8"/></tr><tr><td>&#8195;No</td><td>674/2575</td><td>26.2</td><td>1</td><td>1</td><td>162/2748</td><td>5.9</td><td>1</td><td>1</td></tr><tr><td>&#8195;Yes</td><td>166/728</td><td>22.8</td><td>0.83 (0.69&#8211;1.0)</td><td>0.83 (0.67&#8211;1.0)</td><td>39/775</td><td>5.0</td><td>0.85 (0.59&#8211;1.2)</td><td>0.85 (0.59&#8211;1.2)</td></tr><tr><td>Times when can&#8217;t afford soap in last year</td><td colspan="8"/></tr><tr><td>&#8195;No</td><td>519/2021</td><td>25.7</td><td>1</td><td>1</td><td>136/2166</td><td>6.3</td><td>1</td><td>1</td></tr><tr><td>&#8195;Yes</td><td>321/1280</td><td>25.1</td><td>0.97 (0.82&#8211;1.1)</td><td>0.97 (0.82&#8211;1.1)</td><td>65/1355</td><td>4.8</td><td>0.75 (0.56&#8211;1.0)</td><td>0.75 (0.55&#8211;1.0)</td></tr></tbody></table><table-wrap-foot><fn><p>OR = odds ratio</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01604-v1.xml">elife-01604-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.01604.007</object-id><label>Table 2.</label><caption><p>Association between age at menarche and HSV2 and HIV, adjusted for other variables, among 15&#8211;30 year-old women in Karonga District Malawi</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01604.007">http://dx.doi.org/10.7554/eLife.01604.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th/><th/><th colspan="3">HSV2 (N = 3034)<xref ref-type="table-fn" rid="tblfn1">*</xref></th><th colspan="3">HIV (N = 3231)<xref ref-type="table-fn" rid="tblfn1">*</xref></th></tr><tr><th rowspan="2"/><th rowspan="2"/><th>Adjusted for age</th><th>Also adjusted for socio-economic variables</th><th>Also adjusted for behavioural variables</th><th>Adjusted for age</th><th>Also adjusted for socio-economic variables</th><th>Also adjusted for behavioural variables</th></tr><tr><th>OR (95% CI)</th><th>OR (95% CI)<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></th><th>OR (95% CI)<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></th><th>OR (95% CI)</th><th>OR (95% CI)<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></th><th>OR (95% CI)<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></th></tr></thead><tbody><tr><td>Age at menarche</td><td colspan="7"/></tr><tr><td/><td>&lt;14</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td><td>1</td></tr><tr><td/><td>14</td><td>0.91 (0.72&#8211;1.2)</td><td>0.92 (0.73&#8211;1.2)</td><td>0.92 (0.72&#8211;1.2)</td><td>0.97 (0.63&#8211;1.5)</td><td>0.94 (0.61&#8211;1.4)</td><td>0.96 (0.62&#8211;1.5)</td></tr><tr><td/><td>15</td><td>0.73 (0.58&#8211;0.91)</td><td>0.76 (0.60&#8211;0.95)</td><td>0.77 (0.61&#8211;0.98)</td><td>0.68 (0.44&#8211;1.0)</td><td>0.64 (0.41&#8211;0.98)</td><td>0.65 (0.42&#8211;1.0)</td></tr><tr><td/><td>&#8805;16</td><td>0.70 (0.54&#8211;0.89)</td><td>0.75 (0.58&#8211;0.97)</td><td>0.77 (0.59&#8211;1.0)</td><td>1.2 (0.79&#8211;1.8)</td><td>1.1 (0.69&#8211;1.6)</td><td>1.1 (0.70&#8211;1.7)</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>To allow the effects of these potential mediators to be explored fully, the analysis is restricted to individuals with no missing data for the factors included.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>age, schooling, occupation, lack of food.</p></fn><fn id="tblfn3"><label>&#8225;</label><p>age, schooling, occupation, lack of food, marital status, lifetime number of partners.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01694-v1.xml">elife-01694-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.01694.008</object-id><label>Table 1.</label><caption><p>Genes differentially induced by TM between control and Lrh-1-null mice and their regulation by ATF2 and LRH-1</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01694.008">http://dx.doi.org/10.7554/eLife.01694.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene</th><th colspan="2">Known target of</th></tr><tr><th>ATF2</th><th>LRH-1</th></tr></thead><tbody><tr><td>PLK3</td><td>&#10003;</td><td>&#10003;</td></tr><tr><td>GDF15</td><td/><td/></tr><tr><td>FGF21</td><td><xref ref-type="table-fn" rid="tblfn1">*</xref></td><td/></tr><tr><td>PMM1</td><td/><td/></tr><tr><td>CRELD2</td><td/><td>&#10003;</td></tr><tr><td>NFIL3</td><td/><td/></tr><tr><td>IGSF11</td><td/><td/></tr><tr><td>GSTM2</td><td/><td/></tr><tr><td>GOT1</td><td/><td/></tr><tr><td>DPP9</td><td>&#10003;</td><td>&#10003;</td></tr><tr><td>ST5</td><td/><td/></tr><tr><td>CCBL1</td><td>&#10003;</td><td>&#10003;</td></tr><tr><td>TMEM66</td><td>&#10003;</td><td/></tr><tr><td>KRTCAP2</td><td>&#10003;</td><td>&#10003;</td></tr><tr><td>LRRC59</td><td>&#10003;</td><td>&#10003;</td></tr><tr><td>CDK2AP2</td><td>&#10003;</td><td>&#10003;</td></tr><tr><td>ST3GAL1</td><td/><td/></tr><tr><td>TES</td><td/><td>&#10003;</td></tr><tr><td>CCDC134</td><td>&#10003;</td><td>&#10003;</td></tr><tr><td>UGT1L</td><td/><td/></tr><tr><td>ACOT2</td><td>&#10003;</td><td/></tr><tr><td>PLIN5</td><td/><td>&#10003;</td></tr><tr><td>LITAF</td><td>&#10003;</td><td/></tr><tr><td>RPS13</td><td>&#10003;</td><td/></tr><tr><td>LGALS3BP</td><td/><td/></tr><tr><td>CRYM</td><td/><td/></tr><tr><td>SUPT5H</td><td>&#10003;</td><td/></tr><tr><td>B3GAT3</td><td>&#10003;</td><td/></tr><tr><td>GRN</td><td>&#10003;</td><td/></tr><tr><td>ARRDC4</td><td/><td/></tr><tr><td>SLC30A7</td><td>&#10003;</td><td/></tr><tr><td>CRELD1</td><td>&#10003;</td><td/></tr><tr><td>NT5M</td><td/><td/></tr><tr><td>ARSG</td><td/><td/></tr><tr><td>SEP15</td><td/><td/></tr><tr><td>MKNK1</td><td/><td>&#10003;</td></tr><tr><td>DDX52</td><td/><td/></tr><tr><td>EXOSC5</td><td>&#10003;</td><td/></tr><tr><td>RABAC1</td><td>&#10003;</td><td/></tr><tr><td>RPS15</td><td>&#10003;</td><td>&#10003;</td></tr><tr><td>D830014E11RIK</td><td/><td/></tr><tr><td>AVPI1</td><td>&#10003;</td><td/></tr><tr><td>RPS9</td><td>&#10003;</td><td/></tr><tr><td>CRAT</td><td/><td>&#10003;</td></tr><tr><td>BHMT2</td><td/><td>&#10003;</td></tr><tr><td>B230217C12RIK</td><td/><td/></tr><tr><td>EIF3G</td><td>&#10003;</td><td>&#10003;</td></tr><tr><td>SLC25A28</td><td/><td>&#10003;</td></tr><tr><td>HR</td><td/><td/></tr><tr><td>CCL9</td><td/><td/></tr><tr><td>ANXA4</td><td>&#10003;</td><td>&#10003;</td></tr><tr><td>SMCO4</td><td/><td/></tr><tr><td>SMOX</td><td/><td/></tr><tr><td>ARL14EP</td><td>&#10003;</td><td/></tr><tr><td>SLC39A7</td><td/><td>&#10003;</td></tr><tr><td>ICA1</td><td/><td/></tr><tr><td>ENTPD5</td><td/><td>&#10003;</td></tr><tr><td>PIWIL2</td><td/><td/></tr><tr><td>ANG</td><td>&#10003;</td><td/></tr><tr><td>MCFD2</td><td>&#10003;</td><td>&#10003;</td></tr><tr><td>SOAT2</td><td/><td/></tr><tr><td>SLC41A3</td><td>&#10003;</td><td>&#10003;</td></tr><tr><td>MFGE8</td><td/><td/></tr><tr><td>CYP4A14</td><td/><td/></tr><tr><td>D12ERTD647E</td><td/><td/></tr></tbody></table><table-wrap-foot><fn><p>Microarray analysis was performed for control and <italic>Lrh-1</italic><sup><italic>LKO</italic></sup> mice treated with vehicle or 1 mg/kg tunicamycin for 24 hr (n = 3). Genes were screened for those induced at least 1.5 fold by TM in control mice with significantly different induction in <italic>Lrh-1</italic><sup><italic>LKO</italic></sup> mice by t-test (p&lt;0.05). This list was filtered for those with differential expression between genotypes with TM treatment by t-test (p&lt;0.05). Previously published genome-wide ATF2 and LRH-1 binding datasets were analyzed to identify genes with ATF2 or LRH-1 binding sites &#8722;500 to +275 bp from the TSS. Genes in our set were compared with these sets and genes that contain ATF2 or LRH-1 binding sites meeting the above criteria are marked.</p></fn><fn id="tblfn1"><label>*</label><p>No binding site for ENCODE data set but a known ATF2 direct target (<xref ref-type="bibr" rid="bib14">Hondares et al., 2011</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01879-v1.xml">elife-01879-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.01879.004</object-id><label>Table 1.</label><caption><p>Protein abundance, relative to lipids, in vacuoles or reconstituted proteoliposomes (RPL) fusion reactions</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01879.004">http://dx.doi.org/10.7554/eLife.01879.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Protein</th><th rowspan="2">Molar ratio of lipid:protein in RPL reactions<xref ref-type="table-fn" rid="tblfn1">*</xref></th><th colspan="2">Molar ratio of lipid:protein on vacuoles</th><th rowspan="2">Ratio (RPLs/vacuoles) of molar protein:lipid ratios in std. reactions<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></th></tr><tr><th>BJ3505</th><th>DKY6218</th></tr></thead><tbody><tr><td>Vam7p</td><td>2 &#215; 10<sup>3</sup></td><td>30 &#215; 10<sup>4</sup></td><td>6.5 &#215; 10<sup>4</sup></td><td>7 &#215; 10<sup>1</sup></td></tr><tr><td>Vam3p</td><td>2 &#215; 10<sup>3</sup></td><td>11 &#215; 10<sup>4</sup></td><td>22 &#215; 10<sup>4</sup></td><td>7 &#215; 10<sup>1</sup></td></tr><tr><td>Vti1p</td><td>2 &#215; 10<sup>3</sup></td><td>10 &#215; 10<sup>4</sup></td><td>13 &#215; 10<sup>4</sup></td><td>5 &#215; 10<sup>1</sup></td></tr><tr><td>Nyv1p</td><td>2 &#215; 10<sup>3</sup></td><td>4.3 &#215; 10<sup>4</sup></td><td>8.1 &#215; 10<sup>4</sup></td><td>3 &#215; 10<sup>1</sup></td></tr><tr><td>Ypt7p</td><td>4 &#215; 10<sup>3</sup></td><td>1.9 &#215; 10<sup>4</sup></td><td>1.8 &#215; 10<sup>4</sup></td><td>0.5 &#215; 10<sup>1</sup></td></tr><tr><td>Sec17p</td><td>7 &#215; 10<sup>3</sup></td><td>41 &#215; 10<sup>4</sup></td><td>13 &#215; 10<sup>4</sup></td><td>3 &#215; 10<sup>1</sup></td></tr><tr><td>Sec18p</td><td>1 &#215; 10<sup>3</sup></td><td>10 &#215; 10<sup>4</sup></td><td>13 &#215; 10<sup>4</sup></td><td>10 &#215; 10<sup>1</sup></td></tr><tr><td>Vps33p</td><td>6 &#215; 10<sup>3</sup></td><td>17 &#215; 10<sup>4</sup></td><td>31 &#215; 10<sup>4</sup></td><td>3 &#215; 10<sup>1</sup></td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>for SNAREs and Ypt7p, calculated, based on a 1:1000 lipid:protein ratio during reconstitution and an assumption of 50% outwardly-oriented SNAREs on proteoliposomes; for others, based on amounts of added proteins, and 0.74 mM lipids in standard proteoliposome reactions (see &#8216;Materials and methods&#8217;).</p></fn><fn id="tblfn2"><label>&#8224;</label><p>based on measured (see &#8216;Materials and methods&#8217;) values of 2.17 nmol lipid per &#181;g total vacuole protein for BJ3505 vacuoles and 1.00 nmol lipid per &#956;g total vacuole protein for DKY6218 vacuoles, and standard vacuole reactions containing 3 &#181;g protein of each vacuole in 30 &#181;l.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-01916-v1.xml">elife-01916-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl4" position="float"><object-id pub-id-type="doi">10.7554/eLife.01916.024</object-id><label>Table 4.</label><caption><p>Flagella waveform transition rates</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.01916.024">http://dx.doi.org/10.7554/eLife.01916.024</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Initial</th><th colspan="3">Final</th></tr><tr><th><italic>Normal</italic></th><th><italic>Semi-coiled</italic></th><th><italic>Curly-1</italic></th></tr></thead><tbody><tr><td><italic>Normal</italic></td><td/><td>0.28 &#177; 0.03 s<sup>&#8722;1</sup></td><td>0.08 &#177; 0.01 s<sup>&#8722;1</sup></td></tr><tr><td><italic>Semi-coiled</italic></td><td>1.6 &#177; 0.2 s<sup>&#8722;1</sup></td><td/><td>0.54 &#177; 0.08 s<sup>&#8722;1</sup></td></tr><tr><td><italic>Curly-1</italic></td><td>1.8 &#177; 0.2 s<sup>&#8722;1</sup></td><td>0.04 &#177; 0.02 s<sup>&#8722;1</sup></td><td/></tr></tbody></table><table-wrap-foot><fn><p>Transition rates between different flagellar waveforms: <italic>normal</italic> (CCW), <italic>semi-coiled</italic> and <italic>curly-1</italic> (both CW). Data from wild-type cells (<italic>N</italic> = 52 cells, 203 tumbles). Values are mean &#177; SEM.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02181-v1.xml">elife-02181-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.02181.009</object-id><label>Table 1.</label><caption><p>LC-MS analysis of co-immunoprecipitation assays from ovaries and embryos</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02181.009">http://dx.doi.org/10.7554/eLife.02181.009</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2">Drosophila</th><th rowspan="3"><italic>Human/yeast</italic></th><th colspan="4">Fandango-myc</th><th colspan="2">Prp19-myc</th></tr><tr><th>CG</th><th>gene</th><th colspan="2">ovaries</th><th colspan="2">embryos</th><th colspan="2">embryos</th></tr><tr><th/><th/><th>rep1</th><th>rep2</th><th>rep1</th><th>rep2</th><th>rep1</th><th>rep2</th></tr></thead><tbody><tr><td colspan="9">prp19 complex</td></tr><tr><td>&#8195;CG5519</td><td>prp19</td><td>PRP19/Prp19</td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+++</bold></td><td><bold>++</bold></td></tr><tr><td>&#8195;CG6905</td><td>cdc5-like</td><td>CDC5L/Cef1</td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+++</bold></td><td><bold>++</bold></td></tr><tr><td>&#8195;CG1796</td><td>Tango4</td><td>PLRG1/Prp46</td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td></tr><tr><td>&#8195;CG4980</td><td>-</td><td>BCAS2/Snt309</td><td><bold>&#8211;</bold></td><td><bold>+</bold></td><td><bold>-</bold></td><td><bold>-</bold></td><td><bold>+</bold></td><td><bold>+</bold></td></tr><tr><td>&#8195;CG12135</td><td>c12.1</td><td>CWC15/cwc15</td><td><bold>+</bold></td><td><bold>&#8211;</bold></td><td><bold>&#8211;</bold></td><td><bold>-</bold></td><td><bold>&#8211;</bold></td><td><bold>&#8211;</bold></td></tr><tr><td colspan="9">Prp19 related</td></tr><tr><td>&#8195;CG6197</td><td>Fandango</td><td>Xab2/Syf1</td><td><bold>+++</bold></td><td><bold>+++</bold></td><td><bold>+++</bold></td><td><bold>+++</bold></td><td><bold>+</bold></td><td><bold>+</bold></td></tr><tr><td>&#8195;CG31368</td><td>&#8211;</td><td>AQR/&#8211;</td><td><bold>+++</bold></td><td><bold>+++</bold></td><td><bold>+++</bold></td><td><bold>+++</bold></td><td><bold>+</bold></td><td><bold>+</bold></td></tr><tr><td>&#8195;CG4886</td><td>cyp33</td><td>PPIE/&#8211;</td><td><bold>++</bold></td><td><bold>++</bold></td><td><bold>+</bold></td><td><bold>++</bold></td><td><bold>+</bold></td><td><bold>+</bold></td></tr><tr><td>&#8195;CG9667</td><td>&#8211;</td><td>ISY1/ISY1</td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td></tr><tr><td>&#8195;CG8264</td><td>Bx42</td><td>SNW1/Prp45</td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td></tr><tr><td>&#8195;CG14641</td><td>&#8211;</td><td>RBM22/Cwc2</td><td><bold>&#8211;</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>+</bold></td><td><bold>&#8211;</bold></td></tr><tr><td>&#8195;CG3193</td><td>Crn</td><td>CRNKL1/Clf1</td><td><bold>&#8211;</bold></td><td><bold>+</bold></td><td><bold>&#8211;</bold></td><td><bold>&#8211;</bold></td><td><bold>+</bold></td><td><bold>&#8211;</bold></td></tr><tr><td>&#8195;CG13892</td><td>cypl</td><td>PPIL1/-</td><td><bold>&#8211;</bold></td><td><bold>&#8211;</bold></td><td><bold>&#8211;</bold></td><td><bold>+</bold></td><td><bold>&#8211;</bold></td><td><bold>&#8211;</bold></td></tr><tr><td>&#8195;CG1639</td><td>l(1)10Bb</td><td>BUD31/Bud31</td><td><bold>&#8211;</bold></td><td><bold>&#8211;</bold></td><td><bold>&#8211;</bold></td><td><bold>&#8211;</bold></td><td><bold>+</bold></td><td><bold>&#8211;</bold></td></tr></tbody></table><table-wrap-foot><fn><p>Co-immunoprecipitations were performed using total protein extracts from the different tissues expressing Myc-tagged Fandango or Myc-tagged-Prp19. Human and yeast homologues and the different sub-complexes are shown as described in (<xref ref-type="bibr" rid="bib19">Herold et al., 2009</xref>). (&#8722;), (+), (++), (+++) correspond to 0, 1&#8211;9, 10&#8211;19, and &gt;20 non-repeated peptides respectively. None of the proteins shown were detected in the negative controls (for detailed LC-MS analysis see <xref ref-type="supplementary-material" rid="SD1-data">Supplementary file 1</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02203-v2.xml">elife-02203-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.02203.023</object-id><label>Table 3.</label><caption><p>Light sources and filters used for wide field fluorescence imaging</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02203.023">http://dx.doi.org/10.7554/eLife.02203.023</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Fluorophore</th><th rowspan="2">Light source &amp; channel</th><th colspan="3">Filter cube</th></tr><tr><th>Excitation</th><th>Beamsplitter</th><th>Emission</th></tr></thead><tbody><tr><td>GFP</td><td>Colibri/LED470<xref ref-type="table-fn" rid="tblfn6">*</xref> Spectra-6/C<xref ref-type="table-fn" rid="tblfn9">&#167;</xref></td><td>FF01-475/28<xref ref-type="table-fn" rid="tblfn7">&#8224;</xref> LL01-488/1<xref ref-type="table-fn" rid="tblfn7">&#8224;</xref></td><td>T495LP<xref ref-type="table-fn" rid="tblfn8">&#8225;</xref></td><td>FF01-525/50<xref ref-type="table-fn" rid="tblfn7">&#8224;</xref></td></tr><tr><td>tdEos (red emission)</td><td>Colibri/LED555<xref ref-type="table-fn" rid="tblfn6">*</xref> Spectra-6/GY<xref ref-type="table-fn" rid="tblfn9">&#167;</xref></td><td>BP550/25<xref ref-type="table-fn" rid="tblfn6">*</xref> FF01-543/3<xref ref-type="table-fn" rid="tblfn7">&#8224;</xref></td><td>FT570<xref ref-type="table-fn" rid="tblfn6">*</xref> FF568-Di01<xref ref-type="table-fn" rid="tblfn7">&#8224;</xref></td><td>BP605/70<xref ref-type="table-fn" rid="tblfn6">*</xref> FF01-593/46<xref ref-type="table-fn" rid="tblfn7">&#8224;</xref></td></tr><tr><td>tdEos (photoconversion)</td><td>Colibri/LED405<xref ref-type="table-fn" rid="tblfn6">*</xref> Spectra-6/V<xref ref-type="table-fn" rid="tblfn9">&#167;</xref></td><td>FF01-405/10<xref ref-type="table-fn" rid="tblfn7">&#8224;</xref></td><td>59004BS<xref ref-type="table-fn" rid="tblfn8">&#8225;</xref></td><td>59004M<xref ref-type="table-fn" rid="tblfn8">&#8225;</xref></td></tr></tbody></table><table-wrap-foot><fn id="tblfn6"><p>Source: *Zeiss.</p></fn><fn id="tblfn7"><label>&#8224;</label><p>Semrock.</p></fn><fn id="tblfn8"><label>&#8225;</label><p>Chroma.</p></fn><fn id="tblfn9"><label>&#167;</label><p>Lumencor.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02206-v2.xml">elife-02206-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.02206.004</object-id><label>Table 1.</label><caption><p>Circadian periodicity analysis of the BxS RILs after photic or thermal entrainment</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02206.004">http://dx.doi.org/10.7554/eLife.02206.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th rowspan="2"><italic>Zeitgeber</italic></th><th rowspan="2">Mean<xref ref-type="table-fn" rid="tblfn1">*</xref></th><th rowspan="2">S. E</th><th colspan="2">95% confidence interval</th></tr><tr><th>Lower bound</th><th>Upper bound</th></tr></thead><tbody><tr><td rowspan="2">BxS</td><td>LD</td><td align="char" char=".">26.702</td><td align="char" char=".">0.035</td><td align="char" char=".">26.633</td><td align="char" char=".">26.771</td></tr><tr><td>TMP</td><td align="char" char=".">26.269</td><td align="char" char=".">0.034</td><td align="char" char=".">26.202</td><td align="char" char=".">26.335</td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th colspan="2">LD-TMP</th><th>BxS</th></tr></thead><tbody><tr><td rowspan="3">Pairwise comparisons</td><td>Mean difference (hr)</td><td>0.433</td></tr><tr><td>S. E</td><td>0.047</td></tr><tr><td>Significance</td><td>&lt;0.001</td></tr><tr><td rowspan="2">Correlations</td><td>Correlation coefficient</td><td>0.86</td></tr><tr><td>Significance</td><td>0.001</td></tr></tbody></table><table-wrap-foot><fn><p>LD stands for photic-<italic>zeitgeber</italic> and TMP for thermal-<italic>zeitgeber.</italic></p></fn><fn id="tblfn1"><label>*</label><p>denotes the modified population marginal mean for the 95% Confidence Interval.</p></fn><fn><p>S.E. denotes standard error.</p></fn><fn><p>LD-TMP denotes the pairwise comparisons such that TMP period is subtracted from LD period.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02206-v2.xml">elife-02206-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.02206.005</object-id><label>Table 2.</label><caption><p>Statistical analysis of <italic>CCR2</italic> circadian periodicity after the two <italic>zeitgeber</italic> protocols for BxS population</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02206.005">http://dx.doi.org/10.7554/eLife.02206.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th>Overall model</th><th colspan="2">Univariate BxS</th></tr><tr><th>Factor</th><th>F</th><th>P</th></tr></thead><tbody><tr><td>Genotype</td><td align="char" char=".">7.753</td><td align="char" char=".">&lt;0.001</td></tr><tr><td>Environment</td><td align="char" char=".">36.413</td><td align="char" char=".">&lt;0.001</td></tr><tr><td>Environment<xref ref-type="table-fn" rid="tblfn2">*</xref>Genotype</td><td align="char" char=".">2.183</td><td align="char" char=".">&lt;0.001</td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2"/><th colspan="2">Univariate LD</th><th rowspan="2">CV LD</th><th colspan="2">Univariate TMP</th><th rowspan="2">CV TMP</th></tr><tr><th>F</th><th>P</th><th>F</th><th>P</th></tr></thead><tbody><tr><td rowspan="2">BxS</td><td>RIL</td><td align="char" char=".">10.387</td><td align="char" char=".">&lt;0.001</td><td align="char" char="." rowspan="2">18.043</td><td align="char" char=".">7.382</td><td align="char" char=".">&lt;0.001</td><td align="char" char="." rowspan="2">16.498</td></tr><tr><td>TRANS (RIL)</td><td align="char" char=".">1.513</td><td align="char" char=".">&lt;0.001</td><td align="char" char=".">1.915</td><td align="char" char=".">&lt;0.001</td></tr></tbody></table><table-wrap-foot><fn><p>F = mean sum of squares\error sum of squares. P = significance value of the F-ratio.</p></fn><fn><p>Genotype denotes RIL, ENVIRONMENT denotes the different entrainments, TRANS denotes independent transformants of each genotype.</p></fn><fn id="tblfn2"><label>*</label><p>denotes the testing of an interaction between two factors, whereas B(A) denotes the testing main factor A in which a factor B is nested.</p></fn><fn><p>CV is the coefficient of genetic variation, LD stands for photic and TMP stands for thermal entrainment.</p></fn><fn><p>NS denotes nonsignificance.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02206-v2.xml">elife-02206-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.02206.006</object-id><label>Table 3.</label><caption><p>Localization of the main QTLs for the BxS population after photic vs thermal <italic>zeitgeber</italic> protocols</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02206.006">http://dx.doi.org/10.7554/eLife.02206.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Zeitgeber</th><th rowspan="2">H<sup>2</sup></th><th rowspan="2">Chromosome</th><th>Position</th><th rowspan="2">LOD score <sup>(*)</sup></th><th rowspan="2">% expl variance</th><th rowspan="2">F</th><th rowspan="2">p value</th><th>2a</th></tr><tr><th>(cM)</th><th>(h)</th></tr></thead><tbody><tr><td rowspan="3">LD</td><td rowspan="3">0.76</td><td>I</td><td align="char" char=".">63.7</td><td align="char" char=".">3.21</td><td align="char" char=".">14.9</td><td align="char" char=".">7.704</td><td align="char" char=".">0.007</td><td align="char" char=".">&#8722;0.614</td></tr><tr><td>II</td><td align="char" char=".">34.5</td><td align="char" char=".">5.72</td><td align="char" char=".">27.3</td><td align="char" char=".">20.784</td><td align="char" char=".">&lt;0.001</td><td align="char" char=".">1.003</td></tr><tr><td>IV</td><td align="char" char=".">69.9</td><td align="char" char=".">3.89</td><td align="char" char=".">17.1</td><td align="char" char=".">14.003</td><td align="char" char=".">&lt;0.001</td><td align="char" char=".">0.833</td></tr><tr><td rowspan="2">TMP</td><td rowspan="2">0.73</td><td>II</td><td align="char" char=".">34.5</td><td align="char" char=".">4.56</td><td align="char" char=".">25.5</td><td align="char" char=".">25.033</td><td align="char" char=".">&lt;0.001</td><td align="char" char=".">0.988</td></tr><tr><td>IV</td><td align="char" char=".">69.9</td><td align="char" char=".">3.26</td><td align="char" char=".">16.2</td><td align="char" char=".">6.501</td><td align="char" char=".">0.014</td><td align="char" char=".">0.508</td></tr></tbody></table><table-wrap-foot><fn><p><sup>H2</sup> denotes broad sense heritability.</p></fn><fn><p>(cM) denotes centiMorgan.</p></fn><fn><p>(*) LOD- score threshold was determined at 2.4.</p></fn><fn><p>% expl variance is the percent of explained variance.</p></fn><fn><p>F = mean sum of squares\error sum of squares.</p></fn><fn><p>P denotes the significance value of the F-ratio.</p></fn><fn><p>2a denotes the additive effect of Bay allele when the effect of the Sha allele on period is subtracted.</p></fn><fn><p>(h) denotes the effect in hours.</p></fn><fn><p>&#8722; denotes that the Sha allele displayed longer period than that of Bay allele.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02365-v2.xml">elife-02365-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.02365.011</object-id><label>Table 1.</label><caption><p>Specificity constants for the two activities of CYS-like enzymes (cysteine synthase, CYS and &#946;-cyanoalanine synthase, CAS)</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02365.011">http://dx.doi.org/10.7554/eLife.02365.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="3">CAS reaction</th><th colspan="3">CYS reaction</th><th>CAS/CYS</th></tr><tr><th>Activity (s<sup>&#8722;1</sup>)</th><th>K<sub>m</sub> (mM)</th><th>Specificity constant (mM<sup>&#8722;1</sup>. s<sup>&#8722;1</sup>)</th><th>Activity (s<sup>&#8722;1</sup>)</th><th>K<sub>m</sub> (mM)</th><th>Specificity constant (mM<sup>&#8722;1</sup>. s<sup>&#8722;1</sup>)</th><th>Ratio of specificity constants</th></tr></thead><tbody><tr><td><italic>Tetranychus urticae</italic> CAS</td><td>2.135<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>0.312</td><td>6.84<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>0.646<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>3.17</td><td>0.203<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td><bold>33.7</bold></td></tr><tr><td><italic>Arabidopsis thaliana</italic> CAS</td><td>2.66</td><td>0.14</td><td>19</td><td>2.0</td><td>8.03</td><td>0.250</td><td><bold>76</bold></td></tr><tr><td><italic>Glycine max</italic> CAS</td><td>38.9</td><td>0.81</td><td>48</td><td>1.82</td><td>8.87</td><td>0.205</td><td><bold>234</bold></td></tr><tr><td><italic>Glycine max</italic> CYS</td><td>0.21</td><td>0.30</td><td>0.7</td><td>57.5</td><td>3.6</td><td>15.97</td><td><bold>0.044</bold></td></tr><tr><td><italic>Corynebacterium glutamaticum</italic> CYS</td><td>n.d</td><td>n.d</td><td>&#8211;</td><td>435<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>7</td><td>62<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>&#8211;</td></tr><tr><td><italic>Lactobacillus casei</italic> CYS</td><td>n.d</td><td>n.d</td><td>&#8211;</td><td>89<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>0.6</td><td>148<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>&#8211;</td></tr></tbody></table><table-wrap-foot><fn><p>n.d.: not determined.</p></fn><fn id="tblfn1"><label>*</label><p>data as V<sub>max</sub> in &#181;mol.min<sup>&#8722;1</sup>mg<sup>&#8722;1</sup>.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>data as V<sub>max</sub>/K<sub>m</sub> in &#181;mol.min<sup>&#8722;1</sup>mg<sup>&#8722;1</sup>mM<sup>&#8722;1</sup>. CAS activity was measured with cysteine as substrate while CYS activity was measured with <italic>O</italic>-acetylserine as substrate. Data for the plants <italic>A. thaliana</italic> and <italic>G. max</italic> were obtained in <xref ref-type="bibr" rid="bib50">Yamaguchi et al. (2000)</xref> and <xref ref-type="bibr" rid="bib51">Yi et al. (2012)</xref>, respectively, while data for the bacteria <italic>C. glutamaticum</italic> and <italic>L. casei</italic> were retrieved from <xref ref-type="bibr" rid="bib46">Wada et al. (2004)</xref> and <xref ref-type="bibr" rid="bib4">Bogicevic et al. (2012)</xref>, respectively.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02384-v2.xml">elife-02384-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.02384.012</object-id><label>Table 2.</label><caption><p>Pol V Mut affinity to 12 nt oh HP DNA</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02384.012">http://dx.doi.org/10.7554/eLife.02384.012</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Pol V Mut</th><th align="center" colspan="2">K<sub>d</sub> (nM)</th></tr><tr><th>ATP</th><th>ATP&#947;S</th></tr></thead><tbody><tr><td>pol V Mut WT</td><td><italic>nb</italic></td><td align="char" char="plusmn">876 &#177; 52</td></tr><tr><td>pol V Mut E38K/K72R</td><td><italic>nb</italic></td><td align="char" char="plusmn">920 &#177; 112</td></tr><tr><td>polV Mut E38K/<bold>&#916;</bold>C17</td><td>312 &#177; 46</td><td align="char" char="plusmn">469 &#177; 27</td></tr></tbody></table><table-wrap-foot><fn><p>nb&#8211;binding not detected.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02525-v2.xml">elife-02525-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.02525.015</object-id><label>Table 1.</label><caption><p>The effect of EGL3 and its two mutated forms on root epidermal cell pattern formation</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02525.015">http://dx.doi.org/10.7554/eLife.02525.015</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Genotype</th><th colspan="2">Cells in the H position</th><th colspan="2">Cells in the N position</th></tr><tr><th>Hair cells (%)</th><th>Non-hair cells (%)</th><th>Hair cells (%)</th><th>Non-hair cells (%)</th></tr></thead><tbody><tr><td>Col-0</td><td align="char" char="plusmn">98.9 &#177; 3.3</td><td align="char" char="plusmn">1.1 &#177; 3.3</td><td align="char" char="plusmn">2.0 &#177; 4.5</td><td align="char" char="plusmn">98.0 &#177; 4.5</td></tr><tr><td><italic>PEGL3::EGL3-GFP</italic></td><td align="char" char="plusmn">95.8 &#177; 6.1</td><td align="char" char="plusmn">4.2 &#177; 6.1</td><td align="char" char="plusmn">1.9 &#177; 4.2</td><td align="char" char="plusmn">98.1 &#177; 4.2</td></tr><tr><td><italic>PEGL3::EGL3</italic><sup><italic>T399A</italic></sup><italic>-GFP</italic></td><td align="char" char="plusmn">93.1 &#177; 6.4</td><td align="char" char="plusmn">6.9 &#177; 6.4</td><td align="char" char="plusmn">2.8 &#177; 6.0</td><td align="char" char="plusmn">97.2 &#177; 6.0</td></tr><tr><td><italic>PEGL3::EGL3</italic><sup><italic>T209A/T213A</italic></sup><italic>-GFP</italic></td><td align="char" char="plusmn">85.5 &#177; 4.4<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td align="char" char="plusmn">14.5 &#177; 4.4<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td align="char" char="plusmn">1.0 &#177; 3.2</td><td align="char" char="plusmn">99.0 &#177; 3.2</td></tr></tbody></table><table-wrap-foot><fn><p>At least 10 different 5-day-old roots were examined for each strain. Values represent means &#177; SD. For statistical analysis, the F test was used to determine the variance, and the two-tailed <italic>t</italic> test with equal variance or unequal variance was used to determine the significance level of the difference among the transgenic plants.</p></fn><fn id="tblfn1"><label>*</label><p>p&lt;0.05.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02725-v2.xml">elife-02725-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.02725.003</object-id><label>Table 1.</label><caption><p>Standard diagnostic tests to assess MMR-deficiency</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02725.003">http://dx.doi.org/10.7554/eLife.02725.003</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Tumor</th><th rowspan="2">Histopathology</th><th rowspan="2">Grade</th><th rowspan="2">Stage</th><th colspan="2">Coverage</th><th align="center" colspan="3">IHC</th><th rowspan="2">MSI</th><th rowspan="2"><italic>MLH1</italic> hyper-methylation</th></tr><tr><th>Tumor</th><th>Germ-line</th><th>MLH1</th><th>MSH2</th><th>MSH6</th></tr></thead><tbody><tr><td>MMR&#8722; 1</td><td>Endometrioid</td><td align="char" char=".">3</td><td>IIIc</td><td align="char" char=".">87.1</td><td align="char" char=".">81.1</td><td>+</td><td>+</td><td>&#8722;(<xref ref-type="table-fn" rid="tblfn1">*</xref>)</td><td>+</td><td>&#8722;</td></tr><tr><td>MMR&#8722; 2</td><td>Serous/clear cell</td><td align="char" char=".">3</td><td>Ib</td><td align="char" char=".">24.8</td><td align="char" char=".">21.9</td><td>+</td><td>&#8722;</td><td>&#8722;</td><td>&#8722;</td><td>&#8722;</td></tr><tr><td>MMR&#8722; 3</td><td>Endometrioid</td><td align="char" char=".">2</td><td>Ib</td><td align="char" char=".">28.5</td><td align="char" char=".">30.0</td><td>&#8722;</td><td>+</td><td>+</td><td>+</td><td>+</td></tr><tr><td>MMR+ 1</td><td>Endometrioid</td><td align="char" char=".">3</td><td>I</td><td align="char" char=".">119.4</td><td align="char" char=".">73.1</td><td>+</td><td>+</td><td>+</td><td>&#8722;</td><td>+</td></tr><tr><td>MMR+ 2</td><td>Serous</td><td align="char" char=".">3</td><td>Ia</td><td align="char" char=".">79.2</td><td align="char" char=".">77.0</td><td>+</td><td>+</td><td>+</td><td>&#8722;</td><td>&#8722;</td></tr></tbody></table><table-wrap-foot><fn><p>Tumors and matched germ-line were whole-genome sequenced using either Complete Genomics or Illumina sequencing technology. For each tumor, microsatellite instability (MSI) using the extended Bethesda panel, standard immunohistochemistry of MMR proteins (MLH1, MSH2, and MSH6), and methylation status of the <italic>MLH1</italic> promoter are shown.</p></fn><fn id="tblfn1"><label>*</label><p>a weak positive nuclear staining in the minority of the tumor cells.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02725-v2.xml">elife-02725-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="anchor"><label>Table 1.</label><caption><p>Validation rate of somatic indels detected upon whole-genome sequencing. These additional validation experiments for indels have been included in the revised manuscript</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Tumor</th><th colspan="3">Somatic Indels</th></tr><tr><th>Confirmed</th><th>Not confirmed</th><th>Validation rate</th></tr></thead><tbody><tr><td>MMR- 1</td><td>187</td><td>20</td><td>90.3%</td></tr><tr><td>MMR- 2</td><td>103</td><td>21</td><td>83.1%</td></tr><tr><td>MMR- 3</td><td>54</td><td>6</td><td>90.0%</td></tr><tr><td>MMR+ 1</td><td>0</td><td>9</td><td>0.0%</td></tr><tr><td>MMR+ 2</td><td>2</td><td>7</td><td>22.2%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02798-v1.xml">elife-02798-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.02798.016</object-id><label>Table 1.</label><caption><p>Results of Kolmogrov-Smirnov tests for cumulative frequency distributions</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02798.016">http://dx.doi.org/10.7554/eLife.02798.016</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Groups compared</th><th>Non-deprived</th><th>Non-deprived</th><th>Non-deprived</th><th>Non-deprived</th><th>LTMD</th><th>LTMD</th><th>LTMD</th><th>Home cage</th><th>Home cage</th><th>noise + run</th></tr><tr><th>LTMD</th><th>Home cage</th><th>noise+run</th><th>grating+run</th><th>Home cage</th><th>noise+run</th><th>grating+run</th><th>noise+run</th><th>grating+run</th><th>grating+run</th></tr></thead><tbody><tr><td colspan="11">Broad-spiking cells</td></tr><tr><td>&#8195;Spontaneous firing</td><td>&lt;0.00001</td><td>&lt;0.0001</td><td>&lt;0.001</td><td>&lt;0.05</td><td>&lt;0.01</td><td>&lt;0.01</td><td>&lt;0.001</td><td>&lt;0.01</td><td>&lt;0.05</td><td>&lt;0.01</td></tr><tr><td>&#8195;Grating response: closed</td><td>&lt;0.00001</td><td>&lt;0.00001</td><td>&lt;0.01</td><td>n.s.</td><td>n.s.</td><td>&lt;0.0001</td><td>&lt;0.05</td><td>&lt;0.05</td><td>&lt;0.0001</td><td>&lt;0.01</td></tr><tr><td>&#8195;Grating response: open</td><td>&lt;0.01</td><td>&lt;0.05</td><td>&lt;0.05</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td></tr><tr><td>&#8195;Grating response: ODI</td><td>&lt;0.00001</td><td>&lt;0.00001</td><td>&lt;0.0001</td><td>n.s.</td><td>&lt;0.01</td><td>&lt;0.001</td><td>&lt;0.00001</td><td>&lt;0.01</td><td>&lt;0.00001</td><td>&lt;0.001</td></tr><tr><td>&#8195;|dO|</td><td>&lt;0.00001</td><td>&lt;0.00001</td><td>&lt;0.00001</td><td>&lt;0.01</td><td>n.s.</td><td>n.s.</td><td>&lt;0.001</td><td>n.s.</td><td>&lt;0.001</td><td>&lt;0.001</td></tr><tr><td>&#8195;OSI: closed</td><td>&lt;0.00001</td><td>&lt;0.00001</td><td>&lt;0.00001</td><td>&lt;0.00001</td><td>n.s.</td><td>&lt;0.05</td><td>&lt;0.001</td><td>&lt;0.001</td><td>&lt;0.001</td><td>&lt;0.01</td></tr><tr><td>&#8195;OSI: open</td><td>&lt;0.01</td><td>&lt;0.01</td><td>&lt;0.05</td><td>&lt;0.05</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td></tr><tr><td>&#8195;Orientation tuning width: closed</td><td>&lt;0.0001</td><td>&lt;0.05</td><td>&lt;0.05</td><td>&lt;0.05</td><td>n.s.</td><td>n.s.</td><td>&lt;0.05</td><td>n.s.</td><td>n.s.</td><td>n.s.</td></tr><tr><td>&#8195;Orientation tuning width: open</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td></tr><tr><td>&#8195;Preferred SF: closed</td><td>&lt;0.001</td><td>&lt;0.00001</td><td>&lt;0.001</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>&lt;0.05</td><td>&lt;0.05</td><td>0.001</td><td>&lt;0.05</td></tr><tr><td>&#8195;Preferred SF: open</td><td>&lt;0.00001</td><td>&lt;0.001</td><td>&lt;0.001</td><td>&lt;0.001</td><td>&lt;0.01</td><td>&lt;0.01</td><td>&lt;0.01</td><td>n.s.</td><td>n.s.</td><td>n.s.</td></tr><tr><td>&#8195;Noise response (F1): closed</td><td>&lt;0.00001</td><td>&lt;0.0001</td><td>&lt;0.01</td><td>&lt;0.001</td><td>&lt;0.05</td><td>&lt;0.00001</td><td>&lt;0.001</td><td>&lt;0.00001</td><td>&lt;0.05</td><td>&lt;0.0001</td></tr><tr><td>&#8195;Noise response (F1): open</td><td>&lt;0.0001</td><td>&lt;0.0001</td><td>&lt;0.0001</td><td>&lt;0.00001</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td></tr><tr><td>&#8195;Noise response: ODI</td><td>&lt;0.00001</td><td>&lt;0.00001</td><td>n.s.</td><td>&lt;0.0001</td><td>&lt;0.0001</td><td>&lt;0.00001</td><td>&lt;0.0001</td><td>&lt;0.00001</td><td>&lt;0.001</td><td>&lt;0.001</td></tr><tr><td colspan="11">Narrow-spiking cells</td></tr><tr><td>&#8195;Spontaneous firing</td><td>&lt;0.00001</td><td>&lt;0.00001</td><td>&lt;0.00001</td><td>&lt;0.00001</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td></tr><tr><td>&#8195;Grating response: closed</td><td>&lt;0.00001</td><td>&lt;0.0001</td><td>&lt;0.0001</td><td>&lt;0.001</td><td>n.s.</td><td>n.s.</td><td>&lt;0.01</td><td>n.s.</td><td>&lt;0.05</td><td>&lt;0.05</td></tr><tr><td>&#8195;Grating response: open</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td></tr><tr><td>&#8195;Grating response: ODI</td><td>&lt;0.00001</td><td>&lt;0.001</td><td>&lt;0.0001</td><td>&lt;0.05</td><td>&lt;0.01</td><td>n.s.</td><td>&lt;0.001</td><td>n.s.</td><td>n.s.</td><td>&lt;0.05</td></tr><tr><td>&#8195;OSI: closed</td><td>&lt;0.001</td><td>&lt;0.0001</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>&lt;0.01</td><td>&lt;0.01</td><td>n.s.</td></tr><tr><td>&#8195;OSI: open</td><td>&lt;0.01</td><td>&lt;0.01</td><td>&lt;0.01</td><td>&lt;0.01</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td></tr><tr><td>&#8195;Noise response (F1): closed</td><td>&lt;0.001</td><td>&lt;0.001</td><td>&lt;0.01</td><td>&lt;0.01</td><td>&lt;0.05</td><td>&lt;0.01</td><td>&lt;0.05</td><td>&lt;0.01</td><td>n.s.</td><td>n.s.</td></tr><tr><td>&#8195;Noise response (F1): open</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>n.s.</td></tr><tr><td>&#8195;Noise response: ODI</td><td>&lt;0.00001</td><td>&lt;0.00001</td><td>&lt;0.001</td><td>&lt;0.01</td><td>&lt;0.001</td><td>&lt;0.0001</td><td>&lt;0.0001</td><td>&lt;0.001</td><td>&lt;0.001</td><td>&lt;0.05</td></tr></tbody></table><table-wrap-foot><fn><p>Statistical significance levels are indicated after adjusting p-values using Bonferroni correction for multiple comparisons. n.s.:p&gt;0.05. closed: measures through deprived eye, open: measures through open eye. ODI: ocular dominance index. |dO|: absolute difference in preferred orientation between left and right eyes. OSI: orientation selectivity index.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02869-v2.xml">elife-02869-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.02869.018</object-id><label>Table 2.</label><caption><p>Clinical features at presentation of the CLL cohorts<xref ref-type="table-fn" rid="tblfn2">*</xref></p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02869.018">http://dx.doi.org/10.7554/eLife.02869.018</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="3">TEDG analysis</th><th colspan="3">SA analysis</th></tr><tr><th>N</th><th>Total</th><th>%</th><th>N</th><th>Total</th><th>%</th></tr></thead><tbody><tr><td><bold><italic>IGHV</italic> homology <underline>&gt;</underline>98%</bold></td><td>37</td><td>69</td><td>53.6</td><td>532</td><td>1380</td><td>38.6</td></tr><tr><td><bold>del13q14</bold></td><td>35</td><td>70</td><td>50.0</td><td>682</td><td>1403</td><td>48.6</td></tr><tr><td><bold>+12</bold></td><td>19</td><td>70</td><td>27.1</td><td>205</td><td>1403</td><td>14.6</td></tr><tr><td><bold>del11q22-q23</bold></td><td>8</td><td>70</td><td>11.4</td><td>136</td><td>1403</td><td>9.7</td></tr><tr><td><bold>del17p13</bold></td><td>20</td><td>70</td><td>28.6</td><td>114</td><td>1403</td><td>8.1</td></tr><tr><td><bold><italic>TP53</italic> mutation</bold></td><td>22</td><td>70</td><td>31.4</td><td>114</td><td>1401</td><td>8.1</td></tr><tr><td><bold><italic>NOTCH1</italic> mutation</bold></td><td>22</td><td>70</td><td>31.4</td><td>150</td><td>1403</td><td>10.7</td></tr><tr><td><bold><italic>SF3B1</italic> mutation</bold></td><td>17</td><td>70</td><td>24.3</td><td>100</td><td>1403</td><td>7.1</td></tr><tr><td><bold><italic>MYD88</italic> mutation</bold></td><td>6</td><td>70</td><td>8.6</td><td>52</td><td>1403</td><td>3.7</td></tr><tr><td><bold><italic>BIRC3</italic> mutation</bold></td><td>10</td><td>70</td><td>14.3</td><td>41</td><td>1403</td><td>2.9</td></tr><tr><td><bold><italic>BIRC3</italic> deletion</bold></td><td>6</td><td>70</td><td>8.6</td><td>79</td><td>1403</td><td>5.6</td></tr></tbody></table><table-wrap-foot><fn id="tblfn2"><label>*</label><p>TEDG analysis, Tumor Evolutionary Directed Graphs analysis; SA analysis, Statistical Association analysis; <italic>IGHV</italic>, immunoglobulin heavy variable gene; FISH, fluorescence in situ hybridization.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02917-v1.xml">elife-02917-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.02917.003</object-id><label>Table 1.</label><caption><p>Basic characteristics of patients with autism and ADHD</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02917.003">http://dx.doi.org/10.7554/eLife.02917.003</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2">Characteristic</th><th colspan="2">Autism</th><th colspan="2">ADHD</th></tr><tr><th>No.</th><th>%</th><th>No.</th><th>%</th></tr></thead><tbody><tr><td rowspan="2">Sex</td><td>Male</td><td>16808</td><td>69.6</td><td>33491</td><td>67.9</td></tr><tr><td>Female</td><td>7349</td><td>30.4</td><td>15857</td><td>32.1</td></tr><tr><td rowspan="4">Age (years)</td><td>&lt;10</td><td>5995</td><td>24.8</td><td>8129</td><td>16.5</td></tr><tr><td>10&#8211;19</td><td>9671</td><td>40.0</td><td>22854</td><td>46.3</td></tr><tr><td>20&#8211;29</td><td>4215</td><td>17.4</td><td>7727</td><td>15.7</td></tr><tr><td>30+</td><td>4276</td><td>17.7</td><td>10638</td><td>21.6</td></tr><tr><td rowspan="5">Time period</td><td>1987&#8211;1990</td><td>425</td><td>1.8</td><td>207</td><td>0.4</td></tr><tr><td>1991&#8211;1995</td><td>656</td><td>2.7</td><td>381</td><td>0.8</td></tr><tr><td>1996&#8211;2000</td><td>1199</td><td>5.0</td><td>1307</td><td>2.6</td></tr><tr><td>2001&#8211;2005</td><td>6457</td><td>26.7</td><td>9953</td><td>20.2</td></tr><tr><td>2006&#8211;2010</td><td>15420</td><td>63.8</td><td>37500</td><td>76.0</td></tr><tr><td>All</td><td/><td>24157</td><td>100.0</td><td>49348</td><td>100.0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02917-v1.xml">elife-02917-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.02917.004</object-id><label>Table 2.</label><caption><p>Risk of autism and ADHD in offspring when their parents were diagnosed with alcohol use disorder (AUD)</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02917.004">http://dx.doi.org/10.7554/eLife.02917.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">AUD in parent</th><th rowspan="2">No. of offspring at risk</th><th rowspan="2">Person-years of follow-up</th><th colspan="4">Autism</th><th colspan="4">ADHD</th></tr><tr><th>O</th><th>SIR</th><th colspan="2">95% CI</th><th>O</th><th>SIR</th><th colspan="2">95% CI</th></tr></thead><tbody><tr><td align="center" colspan="11">Risk in sons</td></tr><tr><td>Father</td><td align="char" char=".">235696</td><td align="char" char=".">4485707</td><td align="char" char=".">1793</td><td align="char" char="."><bold>1.39</bold></td><td align="char" char=".">1.33</td><td align="char" char=".">1.45</td><td align="char" char=".">5548</td><td align="char" char="."><bold>2.17</bold></td><td align="char" char=".">2.11</td><td align="char" char=".">2.22</td></tr><tr><td>Mother</td><td align="char" char=".">69214</td><td align="char" char=".">1344766</td><td align="char" char=".">567</td><td align="char" char="."><bold>1.55</bold></td><td align="char" char=".">1.43</td><td align="char" char=".">1.69</td><td align="char" char=".">1952</td><td align="char" char="."><bold>2.73</bold></td><td align="char" char=".">2.61</td><td align="char" char=".">2.85</td></tr><tr><td>Parents</td><td align="char" char=".">289763</td><td align="char" char=".">5540362</td><td align="char" char=".">2203</td><td align="char" char="."><bold>1.41</bold></td><td align="char" char=".">1.35</td><td align="char" char=".">1.47</td><td align="char" char=".">6800</td><td align="char" char="."><bold>2.20</bold></td><td align="char" char=".">2.15</td><td align="char" char=".">2.25</td></tr><tr><td align="center" colspan="11">Risk in daughters</td></tr><tr><td>Father</td><td align="char" char=".">225317</td><td align="char" char=".">4306194</td><td align="char" char=".">753</td><td align="char" char="."><bold>1.31</bold></td><td align="char" char=".">1.22</td><td align="char" char=".">1.4</td><td align="char" char=".">2609</td><td align="char" char="."><bold>2.12</bold></td><td align="char" char=".">2.04</td><td align="char" char=".">2.21</td></tr><tr><td>Mother</td><td align="char" char=".">65,479</td><td align="char" char=".">1275507</td><td align="char" char=".">245</td><td align="char" char="."><bold>1.50</bold></td><td align="char" char=".">1.32</td><td align="char" char=".">1.7</td><td align="char" char=".">970</td><td align="char" char="."><bold>2.77</bold></td><td align="char" char=".">2.60</td><td align="char" char=".">2.95</td></tr><tr><td>Parents</td><td align="char" char=".">276439</td><td align="char" char=".">5305285</td><td align="char" char=".">933</td><td align="char" char="."><bold>1.34</bold></td><td align="char" char=".">1.25</td><td align="char" char=".">1.43</td><td align="char" char=".">3247</td><td align="char" char="."><bold>2.18</bold></td><td align="char" char=".">2.10</td><td align="char" char=".">2.25</td></tr><tr><td align="center" colspan="11">Risk in offspring</td></tr><tr><td>Father</td><td align="char" char=".">461013</td><td align="char" char=".">8791901</td><td align="char" char=".">2546</td><td align="char" char="."><bold>1.36</bold></td><td align="char" char=".">1.31</td><td align="char" char=".">1.42</td><td align="char" char=".">8157</td><td align="char" char="."><bold>2.15</bold></td><td align="char" char=".">2.11</td><td align="char" char=".">2.20</td></tr><tr><td>Mother</td><td align="char" char=".">134693</td><td align="char" char=".">2620273</td><td align="char" char=".">812</td><td align="char" char="."><bold>1.54</bold></td><td align="char" char=".">1.43</td><td align="char" char=".">1.65</td><td align="char" char=".">2922</td><td align="char" char="."><bold>2.74</bold></td><td align="char" char=".">2.64</td><td align="char" char=".">2.84</td></tr><tr><td>Parents</td><td align="char" char=".">566202</td><td align="char" char=".">10845647</td><td align="char" char=".">3136</td><td align="char" char="."><bold>1.39</bold></td><td align="char" char=".">1.34</td><td align="char" char=".">1.44</td><td align="char" char=".">10,047</td><td align="char" char="."><bold>2.19</bold></td><td align="char" char=".">2.15</td><td align="char" char=".">2.23</td></tr><tr><td>PAF</td><td/><td/><td>3.6%</td><td/><td/><td/><td>11%</td><td/><td/><td/></tr></tbody></table><table-wrap-foot><fn><p>O, observed number of cases; SIR, standardized incidence ratio; CI, confidence interval.</p></fn><fn><p>Bold type, 95% CI does not include 1.00.</p></fn><fn><p>PAF, population attributable fraction.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02917-v1.xml">elife-02917-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.02917.005</object-id><label>Table 3.</label><caption><p>Risk of autism and ADHD in offspring when their parents were diagnosed with alcohol use disorder (AUD) before the birth of the offspring</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02917.005">http://dx.doi.org/10.7554/eLife.02917.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">AUD in parent</th><th rowspan="2">No. of offspring at risk</th><th rowspan="2">Person-years of follow-up</th><th colspan="4">Autism</th><th colspan="4">ADHD</th></tr><tr><th>O</th><th>SIR</th><th colspan="2">95% CI</th><th>O</th><th>SIR</th><th colspan="2">95% CI</th></tr></thead><tbody><tr><td align="center" colspan="11">Risk in sons</td></tr><tr><td>Father</td><td align="char" char=".">41832</td><td align="char" char=".">410327</td><td align="char" char=".">442</td><td align="char" char="."><bold>1.44</bold></td><td align="char" char=".">1.31</td><td align="char" char=".">1.59</td><td align="char" char=".">1555</td><td align="char" char="."><bold>2.60</bold></td><td align="char" char=".">2.48</td><td align="char" char=".">2.74</td></tr><tr><td>Mother</td><td align="char" char=".">8776</td><td align="char" char=".">80,689</td><td align="char" char=".">99</td><td align="char" char="."><bold>1.67</bold></td><td align="char" char=".">1.35</td><td align="char" char=".">2.03</td><td align="char" char=".">347</td><td align="char" char="."><bold>3.14</bold></td><td align="char" char=".">2.82</td><td align="char" char=".">3.49</td></tr><tr><td>Parents</td><td align="char" char=".">48704</td><td align="char" char=".">472596</td><td align="char" char=".">514</td><td align="char" char="."><bold>1.46</bold></td><td align="char" char=".">1.34</td><td align="char" char=".">1.59</td><td align="char" char=".">1778</td><td align="char" char="."><bold>2.61</bold></td><td align="char" char=".">2.49</td><td align="char" char=".">2.73</td></tr><tr><td align="center" colspan="11">Risk in daughters</td></tr><tr><td>Father</td><td align="char" char=".">39684</td><td align="char" char=".">390429</td><td align="char" char=".">145</td><td align="char" char="."><bold>1.40</bold></td><td align="char" char=".">1.18</td><td align="char" char=".">1.64</td><td align="char" char=".">565</td><td align="char" char="."><bold>3.05</bold></td><td align="char" char=".">2.80</td><td align="char" char=".">3.31</td></tr><tr><td>Mother</td><td align="char" char=".">8457</td><td align="char" char=".">77259</td><td align="char" char=".">36</td><td align="char" char="."><bold>1.79</bold></td><td align="char" char=".">1.25</td><td align="char" char=".">2.48</td><td align="char" char=".">119</td><td align="char" char="."><bold>3.44</bold></td><td align="char" char=".">2.85</td><td align="char" char=".">4.12</td></tr><tr><td>Parents</td><td align="char" char=".">46299</td><td align="char" char=".">450022</td><td align="char" char=".">175</td><td align="char" char="."><bold>1.47</bold></td><td align="char" char=".">1.26</td><td align="char" char=".">1.70</td><td align="char" char=".">634</td><td align="char" char="."><bold>3.00</bold></td><td align="char" char=".">2.77</td><td align="char" char=".">3.24</td></tr><tr><td align="center" colspan="11">Risk in offspring</td></tr><tr><td>Father</td><td align="char" char=".">81516</td><td align="char" char=".">800756</td><td align="char" char=".">587</td><td align="char" char="."><bold>1.43</bold></td><td align="char" char=".">1.32</td><td align="char" char=".">1.55</td><td align="char" char=".">2120</td><td align="char" char="."><bold>2.71</bold></td><td align="char" char=".">2.59</td><td align="char" char=".">2.83</td></tr><tr><td>Mother</td><td align="char" char=".">17233</td><td align="char" char=".">157948</td><td align="char" char=".">135</td><td align="char" char="."><bold>1.70</bold></td><td align="char" char=".">1.42</td><td align="char" char=".">2.01</td><td align="char" char=".">466</td><td align="char" char="."><bold>3.22</bold></td><td align="char" char=".">2.93</td><td align="char" char=".">3.52</td></tr><tr><td>Parents</td><td align="char" char=".">95003</td><td align="char" char=".">922618</td><td align="char" char=".">689</td><td align="char" char="."><bold>1.46</bold></td><td align="char" char=".">1.36</td><td align="char" char=".">1.58</td><td align="char" char=".">2412</td><td align="char" char="."><bold>2.70</bold></td><td align="char" char=".">2.59</td><td align="char" char=".">2.81</td></tr></tbody></table><table-wrap-foot><fn><p>O, observed number of cases; SIR, standardized incidence ratio; CI, confidence interval.</p></fn><fn><p>Bold type, 95% CI does not include 1.00.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02917-v1.xml">elife-02917-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl4" position="float"><object-id pub-id-type="doi">10.7554/eLife.02917.006</object-id><label>Table 4.</label><caption><p>Risk of autism and ADHD in adoptees when their biological or adoptive parents were diagnosed with alcohol use disorder (AUD)</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02917.006">http://dx.doi.org/10.7554/eLife.02917.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th rowspan="2">No. of offspring at risk</th><th rowspan="2">Person-years of follow-up</th><th colspan="4">Autism</th><th colspan="4">ADHD</th></tr><tr><th>O</th><th>SIR</th><th colspan="2">95% CI</th><th>O</th><th>SIR</th><th colspan="2">95% CI</th></tr></thead><tbody><tr><td colspan="11">Biological parents with AUD</td></tr><tr><td>&#8195;Father</td><td align="char" char=".">2175</td><td align="char" char=".">51746</td><td align="char" char=".">16</td><td align="char" char="."><bold>2.15</bold></td><td align="char" char=".">1.23</td><td align="char" char=".">3.50</td><td align="char" char=".">53</td><td align="char" char="."><bold>2.63</bold></td><td align="char" char=".">1.97</td><td align="char" char=".">3.44</td></tr><tr><td>&#8195;Mother</td><td align="char" char=".">1381</td><td align="char" char=".">32745</td><td align="char" char=".">10</td><td align="char" char=".">1.81</td><td align="char" char=".">0.86</td><td align="char" char=".">3.35</td><td align="char" char=".">18</td><td align="char" char=".">1.21</td><td align="char" char=".">0.72</td><td align="char" char=".">1.92</td></tr><tr><td>&#8195;Parents</td><td align="char" char=".">3251</td><td align="char" char=".">77310</td><td align="char" char=".">19</td><td align="char" char="."><bold>1.75</bold></td><td align="char" char=".">1.05</td><td align="char" char=".">2.74</td><td align="char" char=".">58</td><td align="char" char="."><bold>1.91</bold></td><td align="char" char=".">1.45</td><td align="char" char=".">2.47</td></tr><tr><td colspan="11">Adoptive parents with AUD</td></tr><tr><td>&#8195;Father</td><td align="char" char=".">457</td><td align="char" char=".">10,839</td><td align="char" char=".">1</td><td align="char" char=".">0.75</td><td align="char" char=".">0.00</td><td align="char" char=".">4.28</td><td align="char" char=".">7</td><td align="char" char=".">1.78</td><td align="char" char=".">0.70</td><td align="char" char=".">3.68</td></tr><tr><td>&#8195;Mother</td><td align="char" char=".">229</td><td align="char" char=".">5443</td><td align="char" char=".">0</td><td/><td/><td/><td align="char" char=".">4</td><td align="char" char=".">1.93</td><td align="char" char=".">0.50</td><td align="char" char=".">4.98</td></tr><tr><td>&#8195;Parents</td><td align="char" char=".">654</td><td align="char" char=".">15546</td><td align="char" char=".">1</td><td align="char" char=".">0.51</td><td align="char" char=".">0.00</td><td align="char" char=".">2.93</td><td align="char" char=".">9</td><td align="char" char=".">1.59</td><td align="char" char=".">0.72</td><td align="char" char=".">3.04</td></tr></tbody></table><table-wrap-foot><fn><p>O, observed number of cases; SIR, standardized incidence ratio; CI, confidence interval.</p></fn><fn><p>Bold type, 95% CI does not include 1.00.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02923-v2.xml">elife-02923-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.02923.005</object-id><label>Table 2.</label><caption><p>Number of Z line streaming sites within 100 &#956;m<sup>2</sup> grid in 2-month old wild-type and <italic>RyR1</italic><sup><italic>AG/+</italic></sup> soleus muscle</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02923.005">http://dx.doi.org/10.7554/eLife.02923.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Genotype</th><th align="center" rowspan="2">12-month old</th><th align="center" colspan="4">2-month old</th></tr><tr><th align="center">0.6% Diet</th><th align="center">5.2% Diet</th><th align="center">Enalapril</th><th align="center">Glibenclamide</th></tr></thead><tbody><tr><td align="center"><italic>Ryr1</italic><sup><italic>+/+</italic></sup></td><td align="center">0.48 &#177; 0.21</td><td align="center">0.04 &#177; 0.01</td><td align="center">0.03 &#177; 0.01</td><td align="center">0.05 &#177; 0.03</td><td align="center">0.09 &#177; 0.08</td></tr><tr><td align="center"><italic>Ryr1</italic><sup><italic>AG/+</italic></sup></td><td align="center">10.67 &#177; 0.82**</td><td align="center">3.75 &#177; 0.05**</td><td align="center">0.34 &#177; 0.16*</td><td align="center">1.13 &#177; 0.14**</td><td align="center">1.12 &#177; 0.02**</td></tr></tbody></table><table-wrap-foot><fn><p>Average values from five grids per mice and three mice per group. Asterisks indicate significant value *&lt;0.05, **&lt;0.005.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02923-v2.xml">elife-02923-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.02923.011</object-id><label>Table 3.</label><caption><p>Quantitative RT-PCR of muscles from 2-month old wild-type and heterozygous <italic>Ryr1</italic><sup><italic>AG/+</italic></sup> mice fed for 4 weeks on the indicated diets</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02923.011">http://dx.doi.org/10.7554/eLife.02923.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene</th><th align="center" colspan="2">0.6% K Diet</th><th align="center" colspan="2">5.2% K Diet</th></tr><tr><th align="center"><italic>Ryr1</italic><sup><italic>+/+</italic></sup></th><th align="center"><italic>Ryr1</italic><sup><italic>AG/+</italic></sup></th><th align="center"><italic>Ryr1</italic><sup><italic>+/+</italic></sup></th><th align="center"><italic>Ryr1</italic><sup><italic>AG/+</italic></sup></th></tr></thead><tbody><tr><td colspan="5"><italic>Abcc8</italic> (SUR1)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.07</td><td align="center">1.30 &#177; 0.21**</td><td align="center">1.05 &#177; 0.30</td><td align="center">1.51 &#177; 0.33*</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.04</td><td align="center">1.20 &#177; 0.17***</td><td align="center">1.09 &#177; 0.23</td><td align="center">1.31 &#177; 0.27*</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.12</td><td align="center">1.28 &#177; 0.22*</td><td align="center">0.98 &#177; 0.23</td><td align="center">1.61 &#177; 0.42*</td></tr><tr><td colspan="5"><italic>Abcc9</italic> (SUR2)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.19</td><td align="center">0.81 &#177; 0.24</td><td align="center">1.03 &#177; 0.20</td><td align="center">1.09 &#177; 0.18</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.23</td><td align="center">0.86 &#177; 0.21</td><td align="center">0.99 &#177; 0.13</td><td align="center">1.18 &#177; 0.20</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.18</td><td align="center">0.88 &#177; 0.19</td><td align="center">1.01 &#177; 0.19</td><td align="center">1.11 &#177; 0.11</td></tr><tr><td colspan="5"><italic>Atpa1</italic> (NKA&#945;1)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.15</td><td align="center">1.23 &#177; 0.21*</td><td align="center">0.94 &#177; 0.32</td><td align="center">0.18 &#177; 0.06***</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.21</td><td align="center">1.22 &#177; 0.13*</td><td align="center">0.75 &#177; 0.39</td><td align="center">0.23 &#177; 0.06***</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.26</td><td align="center">1.25 &#177; 0.17*</td><td align="center">0.93 &#177; 0.29</td><td align="center">0.24 &#177; 0.07***</td></tr><tr><td colspan="5"><italic>Clc1</italic> (CLC1)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.09</td><td align="center">1.05 &#177; 0.19</td><td align="center">0.90 &#177; 0.21</td><td align="center">0.84 &#177; 0.25</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.10</td><td align="center">1.10 &#177; 0.16</td><td align="center">0.93 &#177; 0.11</td><td align="center">0.82 &#177; 0.27</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.12</td><td align="center">1.03 &#177; 0.21</td><td align="center">0.89 &#177; 0.15</td><td align="center">0.88 &#177; 0.17</td></tr><tr><td colspan="5"><italic>Kcnj2</italic> (K<sub>IR</sub>2.1)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.09</td><td align="center">0.77 &#177; 0.12***</td><td align="center">1.75 &#177; 0.17***</td><td align="center">1.62 &#177; 0.13***</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.17</td><td align="center">0.68 &#177; 0.17***</td><td align="center">1.70 &#177; 0.26***</td><td align="center">1.55 &#177; 0.15***</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.18</td><td align="center">0.59 &#177; 0.13***</td><td align="center">1.67 &#177; 0.33**</td><td align="center">1.52 &#177; 0.18***</td></tr><tr><td colspan="5"><italic>Kcnj8</italic> (K<sub>ATP</sub>6.1)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.20</td><td align="center">0.56 &#177; 0.12***</td><td align="center">0.97 &#177; 0.20</td><td align="center">0.97 &#177; 0.23</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.13</td><td align="center">0.74 &#177; 0.19***</td><td align="center">0.87 &#177; 0.11</td><td align="center">0.91 &#177; 0.12</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.24</td><td align="center">0.69 &#177; 0.16**</td><td align="center">1.16 &#177; 0.20</td><td align="center">1.08 &#177; 0.17</td></tr><tr><td colspan="5"><italic>Kcnj11</italic> (K<sub>ATP</sub>6.2)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.15</td><td align="center">0.96 &#177; 0.17</td><td align="center">1.10 &#177; 0.13</td><td align="center">1.42 &#177; 0.14***</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.18</td><td align="center">1.01 &#177; 0.16</td><td align="center">1.05 &#177; 0.08</td><td align="center">1.34 &#177; 0.09***</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.17</td><td align="center">0.94 &#177; 0.09</td><td align="center">1.0 9&#177; 0.06</td><td align="center">1.38 &#177; 0.21***</td></tr><tr><td colspan="5"><italic>Prkaa1</italic> (AMPK)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.14</td><td align="center">1.23 &#177; 0.15**</td><td align="center">1.11 &#177; 0.10</td><td align="center">1.07 &#177; 0.14</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.04</td><td align="center">1.11 &#177; 0.06**</td><td align="center">1.08 &#177; 0.05</td><td align="center">1.07 &#177; 0.10</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.11</td><td align="center">1.20 &#177; 0.11***</td><td align="center">1.01 &#177; 0.11</td><td align="center">1.10 &#177; 0.06</td></tr></tbody></table><table-wrap-foot><fn><p>Values normalized to GAPDH before normalization to <italic>Ryr1</italic><sup><italic>+/+</italic></sup> control on 0.6% diet. Asterisks indicates significant value * = 0.05, ** = 0.001, *** = 0.005.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02923-v3.xml">elife-02923-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.02923.005</object-id><label>Table 2.</label><caption><p>Number of Z line streaming sites within 100 &#956;m<sup>2</sup> grid in 2-month old wild-type and <italic>RyR1</italic><sup><italic>AG/+</italic></sup> soleus muscle</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02923.005">http://dx.doi.org/10.7554/eLife.02923.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Genotype</th><th align="center" rowspan="2">12-month old</th><th align="center" colspan="4">2-month old</th></tr><tr><th align="center">0.6% Diet</th><th align="center">5.2% Diet</th><th align="center">Enalapril</th><th align="center">Glibenclamide</th></tr></thead><tbody><tr><td align="center"><italic>Ryr1</italic><sup><italic>+/+</italic></sup></td><td align="center">0.48 &#177; 0.21</td><td align="center">0.04 &#177; 0.01</td><td align="center">0.03 &#177; 0.01</td><td align="center">0.05 &#177; 0.03</td><td align="center">0.09 &#177; 0.08</td></tr><tr><td align="center"><italic>Ryr1</italic><sup><italic>AG/+</italic></sup></td><td align="center">10.67 &#177; 0.82**</td><td align="center">3.75 &#177; 0.05**</td><td align="center">0.34 &#177; 0.16*</td><td align="center">1.13 &#177; 0.14**</td><td align="center">1.12 &#177; 0.02**</td></tr></tbody></table><table-wrap-foot><fn><p>Average values from five grids per mice and three mice per group. Asterisks indicate significant value *&lt;0.05, **&lt;0.005.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02923-v3.xml">elife-02923-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.02923.011</object-id><label>Table 3.</label><caption><p>Quantitative RT-PCR of muscles from 2-month old wild-type and heterozygous <italic>Ryr1</italic><sup><italic>AG/+</italic></sup> mice fed for 4 weeks on the indicated diets</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.02923.011">http://dx.doi.org/10.7554/eLife.02923.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene</th><th align="center" colspan="2">0.6% K Diet</th><th align="center" colspan="2">5.2% K Diet</th></tr><tr><th align="center"><italic>Ryr1</italic><sup><italic>+/+</italic></sup></th><th align="center"><italic>Ryr1</italic><sup><italic>AG/+</italic></sup></th><th align="center"><italic>Ryr1</italic><sup><italic>+/+</italic></sup></th><th align="center"><italic>Ryr1</italic><sup><italic>AG/+</italic></sup></th></tr></thead><tbody><tr><td colspan="5"><italic>Abcc8</italic> (SUR1)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.07</td><td align="center">1.30 &#177; 0.21**</td><td align="center">1.05 &#177; 0.30</td><td align="center">1.51 &#177; 0.33*</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.04</td><td align="center">1.20 &#177; 0.17***</td><td align="center">1.09 &#177; 0.23</td><td align="center">1.31 &#177; 0.27*</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.12</td><td align="center">1.28 &#177; 0.22*</td><td align="center">0.98 &#177; 0.23</td><td align="center">1.61 &#177; 0.42*</td></tr><tr><td colspan="5"><italic>Abcc9</italic> (SUR2)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.19</td><td align="center">0.81 &#177; 0.24</td><td align="center">1.03 &#177; 0.20</td><td align="center">1.09 &#177; 0.18</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.23</td><td align="center">0.86 &#177; 0.21</td><td align="center">0.99 &#177; 0.13</td><td align="center">1.18 &#177; 0.20</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.18</td><td align="center">0.88 &#177; 0.19</td><td align="center">1.01 &#177; 0.19</td><td align="center">1.11 &#177; 0.11</td></tr><tr><td colspan="5"><italic>Atpa1</italic> (NKA&#945;1)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.15</td><td align="center">1.23 &#177; 0.21*</td><td align="center">0.94 &#177; 0.32</td><td align="center">0.18 &#177; 0.06***</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.21</td><td align="center">1.22 &#177; 0.13*</td><td align="center">0.75 &#177; 0.39</td><td align="center">0.23 &#177; 0.06***</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.26</td><td align="center">1.25 &#177; 0.17*</td><td align="center">0.93 &#177; 0.29</td><td align="center">0.24 &#177; 0.07***</td></tr><tr><td colspan="5"><italic>Clc1</italic> (CLC1)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.09</td><td align="center">1.05 &#177; 0.19</td><td align="center">0.90 &#177; 0.21</td><td align="center">0.84 &#177; 0.25</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.10</td><td align="center">1.10 &#177; 0.16</td><td align="center">0.93 &#177; 0.11</td><td align="center">0.82 &#177; 0.27</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.12</td><td align="center">1.03 &#177; 0.21</td><td align="center">0.89 &#177; 0.15</td><td align="center">0.88 &#177; 0.17</td></tr><tr><td colspan="5"><italic>Kcnj2</italic> (K<sub>IR</sub>2.1)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.09</td><td align="center">0.77 &#177; 0.12***</td><td align="center">1.75 &#177; 0.17***</td><td align="center">1.62 &#177; 0.13***</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.17</td><td align="center">0.68 &#177; 0.17***</td><td align="center">1.70 &#177; 0.26***</td><td align="center">1.55 &#177; 0.15***</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.18</td><td align="center">0.59 &#177; 0.13***</td><td align="center">1.67 &#177; 0.33**</td><td align="center">1.52 &#177; 0.18***</td></tr><tr><td colspan="5"><italic>Kcnj8</italic> (K<sub>ATP</sub>6.1)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.20</td><td align="center">0.56 &#177; 0.12***</td><td align="center">0.97 &#177; 0.20</td><td align="center">0.97 &#177; 0.23</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.13</td><td align="center">0.74 &#177; 0.19***</td><td align="center">0.87 &#177; 0.11</td><td align="center">0.91 &#177; 0.12</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.24</td><td align="center">0.69 &#177; 0.16**</td><td align="center">1.16 &#177; 0.20</td><td align="center">1.08 &#177; 0.17</td></tr><tr><td colspan="5"><italic>Kcnj11</italic> (K<sub>ATP</sub>6.2)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.15</td><td align="center">0.96 &#177; 0.17</td><td align="center">1.10 &#177; 0.13</td><td align="center">1.42 &#177; 0.14***</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.18</td><td align="center">1.01 &#177; 0.16</td><td align="center">1.05 &#177; 0.08</td><td align="center">1.34 &#177; 0.09***</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.17</td><td align="center">0.94 &#177; 0.09</td><td align="center">1.0 9&#177; 0.06</td><td align="center">1.38 &#177; 0.21***</td></tr><tr><td colspan="5"><italic>Prkaa1</italic> (AMPK)</td></tr><tr><td>&#8195;Vastus lateralis</td><td align="center">1.00 &#177; 0.14</td><td align="center">1.23 &#177; 0.15**</td><td align="center">1.11 &#177; 0.10</td><td align="center">1.07 &#177; 0.14</td></tr><tr><td>&#8195;Tibialis anterior</td><td align="center">1.00 &#177; 0.04</td><td align="center">1.11 &#177; 0.06**</td><td align="center">1.08 &#177; 0.05</td><td align="center">1.07 &#177; 0.10</td></tr><tr><td>&#8195;Adductor magnus</td><td align="center">1.00 &#177; 0.11</td><td align="center">1.20 &#177; 0.11***</td><td align="center">1.01 &#177; 0.11</td><td align="center">1.10 &#177; 0.06</td></tr></tbody></table><table-wrap-foot><fn><p>Values normalized to GAPDH before normalization to <italic>Ryr1</italic><sup><italic>+/+</italic></sup> control on 0.6% diet. Asterisks indicates significant value * = 0.05, ** = 0.001, *** = 0.005.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-02999-v1.xml">elife-02999-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tblu1" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Ref. No.</th><th rowspan="2">Sirt3 Localization</th><th rowspan="2">Evidence</th><th rowspan="2">Fig. No. in Ref.</th><th rowspan="2">Cell Type</th><th rowspan="2">Antibody</th><th colspan="4">Methods</th></tr><tr><th>IB</th><th>IF</th><th>GFP</th><th>Others</th></tr></thead><tbody><tr><td rowspan="3"><bold>1</bold></td><td>Mitochondria</td><td>Overexpressed Sirt3-FLAG was found in mitochondrial extraction.</td><td><bold>2b</bold></td><td>HEK293T</td><td>anti-FLAG</td><td>&#10004;</td><td/><td/><td/></tr><tr><td>Mitochondria</td><td>Endogenous Sirt3 was detected in mitochondrial fraction with rabbit sirt3 antiserum (anti-sirt3 C term 15 AAs).</td><td><bold>2c</bold></td><td>HEK293T</td><td>anti-Sirt3 C-term</td><td>&#10004;</td><td/><td/><td/></tr><tr><td>Mitochondria</td><td>Overexpressed Sirt3-GFP localized in mitochondria</td><td><bold>2a</bold></td><td>HeLa</td><td/><td/><td/><td>&#10004;</td><td/></tr><tr><td rowspan="2"><bold>2</bold></td><td>Mitochondria</td><td>Overexpressed Sirt3-EGFP was detected in mitochondria by immunoelectron microscopy.</td><td><bold>6</bold></td><td>Cervical carcinoma</td><td/><td/><td/><td/><td>Immunoelectron</td></tr><tr><td>Mitochondria</td><td>Overexpressed full length Sirt3-EGFP was present in mitochondria</td><td><bold>3, 4, 5</bold></td><td>COS7</td><td/><td/><td/><td>&#10004;</td><td/></tr><tr><td rowspan="4"><bold>3</bold></td><td>Mitochondria</td><td>Overexpressed Sirt3-GFP was detected in mitochondria.</td><td><bold>1a</bold></td><td>NHF</td><td/><td/><td/><td>&#10004;</td><td/></tr><tr><td>Mitochondria</td><td>There was no detectable signal of overexpressed Sirt3 with N-term-HA tag by immunofluorescence.</td><td><bold>&#8722;</bold></td><td>NHF</td><td>anti-HA,</td><td/><td>&#10004;</td><td/><td/></tr><tr><td>Mitochondria</td><td>Overexpressed of Sirt3-V5-His with C-term-tag was detected in mitochondria by immunofluorescence.</td><td><bold>3a, b</bold></td><td>NHF HeLa</td><td>anti-V5</td><td/><td>&#10004;</td><td/><td/></tr><tr><td>Mitochondria</td><td>Sirt3-V5-His was detected in mitochondrial extraction using anti-V5 antibody.</td><td><bold>3c</bold></td><td>HEK293T</td><td>anti-V5</td><td>&#10004;</td><td/><td/><td/></tr><tr><td><bold>4</bold></td><td>Mitochondria</td><td>Overexpressed mSirt3-FLAG was detected in mitochondria by immunofluorescence.</td><td><bold>2a</bold></td><td>NIH3T3 (mouse)</td><td>anti-FLAG</td><td/><td>&#10004;</td><td/><td/></tr><tr><td><bold>5</bold></td><td>Mitochondria</td><td>Co-expression of AceCS1/2 with Sirt3-FLAG down-regulated acetylation level on AceCS2 (in mitochondria), but not AceCS1 (in nucleus).</td><td><bold>1e, 3c</bold></td><td>COS7</td><td>anti-FLAG</td><td/><td/><td/><td>&#10004;</td></tr><tr><td rowspan="3"><bold>6</bold></td><td>Mitochondria</td><td>Endogenous Sirt3 was detected in mitochondrial extraction using anti-sirt3 C-term serum.</td><td><bold>1b-d</bold></td><td>HEK293</td><td>anti-sirt3 C-term</td><td>&#10004;</td><td/><td/><td/></tr><tr><td>Mitochondria</td><td>AceCS2-FLAG localized in mitochondria. Endogenous Sirt3 was co-precipitated from cell stably expressing AceCS2-FLAG using anti-FLAG antibody.</td><td><bold>1a, 3e</bold></td><td>HeLa HEK293</td><td>anti-FLAG anti-sirt3 C-term</td><td/><td/><td/><td>&#10004;</td></tr><tr><td>Mitochondria</td><td>Overexpression of Sirt3-FLAG down-regulated acetylation level on AceCS2.</td><td><bold>3d</bold></td><td>COS-1</td><td/><td/><td/><td/><td>&#10004;</td></tr><tr><td rowspan="6"><bold>7</bold></td><td>Mitochondria</td><td>Overexpressed Sirt3-myc was detected in mitochondria by immunofluorescence.</td><td><bold>1a</bold></td><td>COS7</td><td>anti-Myc tag</td><td/><td>&#10004;</td><td/><td/></tr><tr><td>Mitochondria</td><td>Overexpressed Sir3-Myc was detected in mitochondrial extraction using anti-Myc antibody.</td><td><bold>1b</bold></td><td>COS7</td><td>anti-Myc tag</td><td>&#10004;</td><td/><td/><td/></tr><tr><td><bold>Nucleus</bold></td><td>Co-expression of Sirt5-FLAG with Sirt3-Myc promoted translocation of Sirt3 into nucleus.</td><td><bold>2a</bold></td><td>COS7</td><td>anti-Myc tag</td><td/><td>&#10004;</td><td/><td/></tr><tr><td><bold>Nucleus</bold></td><td>Overexpressed Sirt3-Myc could be detected in total lysate but not in post-nuclear supernatant after co-expression with Sirt5-FLAG.</td><td><bold>2b, c</bold></td><td>COS7</td><td>anti-Myc tag</td><td>&#10004;</td><td/><td/><td/></tr><tr><td>MTS</td><td>Sirt3-Myc with mitochondria targeting signal (72-75) mutant to 4 Ala was found in in both nucleus and cytoplasm.</td><td><bold>3</bold></td><td>COS7</td><td>anti-Myc tag</td><td/><td>&#10004;</td><td/><td/></tr><tr><td>NTS</td><td>Sirt3-Myc with nuclear targeting signal (214-216) mutant to 3 Ala was detected in nucleus and cytoplasm when co-expressed with Sirt5.</td><td><bold>4b, c</bold></td><td>COS7</td><td>anti-Myc tag</td><td/><td>&#10004;</td><td/><td/></tr><tr><td><bold>8</bold></td><td>Mitochondria</td><td>Endogenous Sirt3 was detected in mitochondria isolation using anti-Sirt3 C-term antibody.</td><td><bold>1b-d</bold></td><td>mouse</td><td>anti-Sirt3 C-term</td><td>&#10004;</td><td/><td/><td/></tr><tr><td rowspan="3"><bold>9</bold></td><td>Mitochondria</td><td>Overexpressed Sirt3-FLAG or Sirt3-Myc was detected in mitochondria by immunofluorescence.</td><td><bold>1A</bold></td><td>HeLa HEK293T U2OS</td><td>anti-FLAG, anti-Myc</td><td/><td>&#10004;</td><td/><td/></tr><tr><td>Mitochondria</td><td>LMB treatment did not result in nuclear translocation of Sirt3-FLAG but did for HDAC7.</td><td><bold>1B, C</bold></td><td>HeLa U2OS</td><td>anti-FLAG, anti-Myc</td><td/><td>&#10004;</td><td/><td/></tr><tr><td>Mitochondria</td><td>Overexpressed POLG2-HA, a positive control for mitochondrial localized proteins, was detected in mitochondria isolation, by not for Sirt3-FLAG.</td><td><bold>2A</bold></td><td>HEK293</td><td>anti -Sirt3 SC49744</td><td>&#10004;</td><td/><td/><td/></tr><tr><td rowspan="10"><bold>10</bold></td><td><bold>Nucleus</bold></td><td>Full length Sirt3 was detected in all subcellular fractionation parts.</td><td><bold>5</bold></td><td>Cardiomyocytes</td><td>anti-sirt3 Ab56214,</td><td>&#10004;</td><td/><td/><td/></tr><tr><td><bold>Nucleus</bold></td><td>Endogenous Sirt3 was detected in nucleus by microscopy.</td><td><bold>6a-e</bold></td><td>Cardiomyocytes</td><td>anti-sirt3 SC49744</td><td/><td/><td/><td>&#10004;</td></tr><tr><td><bold>Nucleus</bold></td><td>Overexpressed Sirt3 was detected in nucleus by immunofluorescence.</td><td><bold>6f</bold> <bold>7a, b</bold></td><td>Cardiomyocytes</td><td>anti -Sirt3 SC49744</td><td/><td>&#10004;</td><td/><td/></tr><tr><td><bold>Nucleus</bold></td><td>Full length Sirt3 was detected in nuclear extraction using anti-Sirt3 antibody.</td><td><bold>7d, g</bold></td><td>Cardiomyocytes</td><td>anti-sirt3 SC49744</td><td>&#10004;</td><td/><td/><td/></tr><tr><td><bold>Nucleus</bold></td><td>Ku70, a nuclear protein, could be co-precipitated by anti-Sirt3 antibody.</td><td><bold>8a</bold></td><td>Cardiomyocytes</td><td>anti-Sirt3 AP6242a</td><td/><td/><td/><td>&#10004;</td></tr><tr><td><bold>Nucleus</bold></td><td>Ku70 was co-precipitated with FLAG antibody from Sirt3-FLAG expressing cells but not from cells expressing the FLAG tag alone.</td><td><bold>8b</bold></td><td>COS7</td><td>anti-FLAG,</td><td/><td/><td/><td>&#10004;</td></tr><tr><td><bold>Nucleus</bold></td><td>Sirt3 was pulled down with Ku70 using anti-Ku70 antibody.</td><td><bold>8c</bold></td><td>Cardiomyocytes</td><td>anti-sirt3 PAB11098</td><td/><td/><td/><td>&#10004;</td></tr><tr><td><bold>Nucleus</bold></td><td>Precipitation of recombinant Ku70 by recombinant Sirt3 verified their direct interaction <italic>in vitro.</italic></td><td><bold>8d</bold></td><td/><td/><td/><td/><td/><td>&#10004;</td></tr><tr><td><bold>Nucleus</bold></td><td>Acetylation level on Ku70 was reduced by overexpressed Sirt3 and enhanced by Sirt3 knockdown via RNAi.</td><td><bold>9d</bold><bold>10c, d</bold></td><td>HeLa</td><td/><td/><td/><td/><td>&#10004;</td></tr><tr><td><bold>Nucleus</bold></td><td>Sirt3 was capable of deacetylation on Ku70 to enhance Ku70/Bax bindig to prevent cell death.</td><td><bold>11, 12</bold></td><td>HeLa</td><td/><td/><td/><td/><td>&#10004;</td></tr><tr><td rowspan="5"><bold>11</bold></td><td><bold>Nucleus</bold></td><td>Endogenous full-length Sirt3 was detected in nucleus by immunofluorescence using anti-Sirt3 N-term antibody.</td><td><bold>1b</bold></td><td>HeLa &amp; HEK293T</td><td>anti-sirt3 N-term</td><td/><td>&#10004;</td><td/><td/></tr><tr><td><bold>Nucleus</bold></td><td>Co-transfection of Gal4-Sirt3 and luciferase reporter plasmid showed the nucleus localization of Gal4-Sirt3.</td><td><bold>3b</bold></td><td>HEK293T</td><td>anti-Gal-DBD</td><td>&#10004;</td><td/><td/><td/></tr><tr><td><bold>Nucleus</bold></td><td>Full length Sirt3 resided in the nucleus and expelled from nucleus upon cellular stress.</td><td><bold>5</bold></td><td>HeLa S3</td><td>anti-sirt3 N-term</td><td/><td>&#10004;</td><td/><td/></tr><tr><td><bold>Nucleus</bold></td><td>The synthesis of tetracycline-inducible Sirt3 harboring a C-term HA-tag is processed in the nucleus.</td><td><bold>6a</bold></td><td>HEK293T</td><td>anti-HA</td><td/><td>&#10004;</td><td/><td/></tr><tr><td><bold>Nucleus</bold></td><td>Overexpressed Sirt3-HA was detected in nuclear extraction.</td><td><bold>6b</bold></td><td>HEK293T</td><td>anti-HA</td><td>&#10004;</td><td/><td/><td/></tr><tr><td rowspan="3"><bold>12</bold></td><td><bold>Nucleus</bold></td><td>Stress-induced the degradation of full-length Sirt3 in nuclei.</td><td><bold>5b, c</bold></td><td>HeLa, U2OS, HEK293T</td><td>anti-HA anti-Sirt3 sc-49744</td><td>&#10004;</td><td/><td/><td/></tr><tr><td><bold>Nucleus</bold></td><td>SIRT3 localizes to defined chromatin regions and regulates gene expression.</td><td><bold>6, 7, 8</bold></td><td>U2OS</td><td/><td/><td/><td/><td>&#10004;</td></tr><tr><td><bold>Nucleus</bold></td><td>Interaction of endogenous Sirt3 and SKP2, a protein found outside mitochondria, was validated by immunoprecipitation.</td><td><bold>4</bold></td><td>HEK293T</td><td>anti-HA anti-FLAG anti-Sirt3 C73E3</td><td/><td/><td/><td>&#10004;</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03061-v1.xml">elife-03061-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.03061.008</object-id><label>Table 1.</label><caption><p>Results for individual mice</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.03061.008">http://dx.doi.org/10.7554/eLife.03061.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="3">Retrieval</th><th colspan="3">Encoding</th></tr><tr><th>Baseline</th><th>Peak</th><th>Trough</th><th>Baseline</th><th>Peak</th><th>Trough</th></tr></thead><tbody><tr><td rowspan="2">Mouse 1</td><td>0.55</td><td>0.49</td><td>0.71</td><td>0.58</td><td>0.48</td><td>0.43</td></tr><tr><td/><td>p=0.09</td><td><bold><italic>P</italic>=0.02</bold></td><td/><td>p=0.07</td><td><bold><italic>P</italic>=0.02</bold></td></tr><tr><td rowspan="2">Mouse 2</td><td>0.52</td><td>0.57</td><td>0.64</td><td>0.60</td><td>0.75</td><td>0.66</td></tr><tr><td/><td>p=0.09</td><td><bold><italic>P</italic>=0.02</bold></td><td/><td><bold><italic>P</italic>=0.01</bold></td><td>p=0.06</td></tr><tr><td rowspan="2">Mouse 3</td><td>0.50</td><td>0.53</td><td>0.67</td><td>0.62</td><td>0.73</td><td>0.64</td></tr><tr><td/><td>p=0.10</td><td><bold><italic>P</italic>=0.01</bold></td><td/><td><bold><italic>P</italic>=0.03</bold></td><td>p=0.11</td></tr><tr><td rowspan="2">Mouse 4</td><td>0.72</td><td>0.73</td><td>0.82</td><td>0.56</td><td>0.82</td><td>0.62</td></tr><tr><td/><td>p=0.12</td><td><bold><italic>P</italic>=0.04</bold></td><td/><td><bold><italic>P</italic>=0.0001</bold></td><td>p=0.08</td></tr></tbody></table><table-wrap-foot><fn><p>Probability of a correct response for four mice under six conditions: baseline (no stimulation), peak-triggered stimulation, and trough-triggered stimulation in both the retrieval and encoding segments. p-values computed from the p.d.f. of the binomial distribution with chance levels equal to the baseline performance for that mouse. Significant changes (p&lt;0.05) are highlighted in bold.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03185-v1.xml">elife-03185-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.03185.006</object-id><label>Table 1.</label><caption><p>Properties of USV categories emitted during social interactions.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.03185.006">http://dx.doi.org/10.7554/eLife.03185.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Call type</th><th rowspan="2">n</th><th colspan="3">Frequency (kHz)</th><th colspan="3">Band width (kHz)</th><th colspan="3">Duration (ms)</th></tr><tr><th>Mean</th><th>&#177; SEM</th><th>20th, 80th quantiles</th><th>Mean</th><th>&#177; SEM</th><th>20th, 80th quantiles</th><th>Mean</th><th>&#177; SEM</th><th>20th, 80th quantiles</th></tr></thead><tbody><tr><td>Trill</td><td>16,549</td><td>63.1</td><td>0.2</td><td>51.7, 76.9</td><td>10.3</td><td>0.1</td><td>5.1, 12.6</td><td>50.9</td><td>0.2</td><td>30.2, 67.1</td></tr><tr><td>Complex</td><td>13,662</td><td>54.8</td><td>0.1</td><td>48.1, 62.9</td><td>7.8</td><td>0.1</td><td>3.9, 8.7</td><td>39.0</td><td>0.2</td><td>23.8, 49.9</td></tr><tr><td>Flat</td><td>13,656</td><td>49.0</td><td>0.1</td><td>38.6, 56.0</td><td>5.2</td><td>0.1</td><td>2.7, 5.3</td><td>44.1</td><td>0.3</td><td>21.7, 56.4</td></tr><tr><td>Upward ramp</td><td>1280</td><td>51.6</td><td>0.4</td><td>45.7, 59.8</td><td>6.5</td><td>0.2</td><td>3.5, 8.4</td><td>26.8</td><td>0.3</td><td>15.9, 36.5</td></tr><tr><td>Downward ramp</td><td>258</td><td>48.0</td><td>1.0</td><td>31.7, 59.7</td><td>5.5</td><td>0.4</td><td>3.2, 6.8</td><td>19.9</td><td>0.8</td><td>11.6, 25.6</td></tr><tr><td>Inverted u</td><td>501</td><td>55.5</td><td>0.7</td><td>45.0, 67.4</td><td>5.7</td><td>0.2</td><td>3.2, 7.2</td><td>16.4</td><td>0.5</td><td>11.0, 18.4</td></tr><tr><td>Short</td><td>26</td><td>49.6</td><td>3.1</td><td>36.1, 65.1</td><td>4.1</td><td>0.3</td><td>3.1, 5.4</td><td>13.7</td><td>0.9</td><td>10.9, 15.6</td></tr><tr><td>Split</td><td>1072</td><td>45.2</td><td>0.5</td><td>34.0, 57.2</td><td>7.4</td><td>0.5</td><td>3.1, 7.3</td><td>68.1</td><td>1.4</td><td>38.9, 87.8</td></tr><tr><td>Fear call</td><td>299</td><td>28.0</td><td>0.5</td><td>22.9, 31.0</td><td>2.4</td><td>0.5</td><td>1.1, 2.3</td><td>206.5</td><td>16.6</td><td>30.2, 353.7</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03348-v2.xml">elife-03348-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.03348.019</object-id><label>Table 1.</label><caption><p>Average equilibrium force, extension change, folding energy, and folding energy barrier and position associated with C-terminal domain and linker domain transitions of the four different SNARE complexes</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.03348.019">http://dx.doi.org/10.7554/eLife.03348.019</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">SNARE complex</th><th colspan="5">C-terminal domain</th><th colspan="5">Linker domain</th></tr><tr><th>Force (pN)</th><th>Extension change (nm)</th><th>Folding energy (<italic>k</italic><sub><italic>B</italic></sub><italic>T</italic>)</th><th>Transition state energy<xref ref-type="tblfn" rid="tblfn1">*</xref> (<italic>k</italic><sub><italic>B</italic></sub><italic>T</italic>)</th><th>Transition state position<xref ref-type="tblfn" rid="tblfn2">&#8224;</xref> (a.a.)</th><th>Force (pN)</th><th>Extension change (nm)</th><th>Folding energy (<italic>k</italic><sub><italic>B</italic></sub><italic>T</italic>)</th><th>Transition state energy<xref ref-type="tblfn" rid="tblfn1">*</xref> (<italic>k</italic><sub><italic>B</italic></sub><italic>T</italic>)</th><th>Transition state position<xref ref-type="tblfn" rid="tblfn2">&#8224;</xref> (a.a.)</th></tr></thead><tbody><tr><td>Neuron</td><td>16.2 (0.9)</td><td>7.2 (1.2)</td><td>&#8722;27 (4.7)</td><td>&#8722;5.5 (1.5)</td><td>17 (3)</td><td>8 (1)</td><td>4.7 (0.5)</td><td>&#8722;9.7 (1.6)</td><td>5.5 (1.5)</td><td>31 (1)</td></tr><tr><td>GLUT4</td><td>18.5 (1.8)</td><td>6.0 (0.9)</td><td>&#8722;23 (4.1)</td><td>&#8722;0.8 (1.0)</td><td>11 (2)</td><td>8.6 (0.9)</td><td>5.6 (1.1)</td><td>&#8722;12 (2.7)</td><td>2 (1.0)</td><td>30 (1)</td></tr><tr><td>Endosome</td><td>11.9 (0.9)</td><td>6.9 (0.4)</td><td>&#8722;16 (1.5)</td><td>2.1 (1.4)</td><td>12 (2)</td><td>6.3 (1.2)</td><td>5.1 (1.8)</td><td>&#8722;6.1 (2.4)</td><td>4.9 (1.5)</td><td>32 (2)</td></tr><tr><td>Yeast</td><td>10.1 (1.4)</td><td>5.8 (0.8)</td><td>&#8722;13 (2.5)</td><td>3.2 (1.5)</td><td>13 (2)</td><td>6.0 (1.6)</td><td>5.1 (1.2)</td><td>&#8722;5.7 (2.0)</td><td>3.6 (2.0)</td><td>32 (2)</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>Here, negative energy indicates downhill protein folding (<xref ref-type="bibr" rid="bib55">Yang and Gruebele, 2003</xref>).</p></fn><fn id="tblfn2"><label>&#8224;</label><p>The number of the amino acids in the R SNARE C-terminal to the ionic layer (chosen as 0).</p></fn><fn><p>The equilibrium force and extension change were determined at an unfolding probability of 0.5 for the two-state processes. The standard deviations of the averages are shown in parenthesis. The equilibrium force distribution, the number of transitions, and the number of single molecules scored for C-terminal domain (CTD) transitions are shown in <xref ref-type="fig" rid="fig3">Figure 3</xref>. For parameters related to linker domain (LD) transitions, a total of 18, 35, 11, and 24 LD transitions in single neuronal, GLUT4, endosomal, and yeast SNARE complexes were scored, respectively.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03372-v1.xml">elife-03372-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.03372.006</object-id><label>Table 1.</label><caption><p>Yeast strains</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.03372.006">http://dx.doi.org/10.7554/eLife.03372.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Strain</th><th rowspan="2">Genotype</th><th>Figure(s)</th></tr><tr><th>Reference</th></tr></thead><tbody><tr><td rowspan="2">RPY753</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, P</italic><sub><italic>GAL1</italic></sub><italic>-ZZ-Tev-GFP-3xHA-GST-DYN1</italic><sub><italic>331kDa</italic></sub><italic>-gs-DHA, pac1&#196;::URA3, ndl1&#196;::cgLEU2</italic></td><td><xref ref-type="fig" rid="fig2">Figure 2</xref>, <xref ref-type="fig" rid="fig2s1 fig2s2">Figure 2&#8212;figure supplement 1,2</xref>, <xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1</xref></td></tr><tr><td><xref ref-type="bibr" rid="bib23">Huang et al., 2012</xref></td></tr><tr><td rowspan="2">RPY816</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, P</italic><sub><italic>GAL1</italic></sub><italic>-ZZ-Tev-PAC1, dyn1&#196;::cgLEU2, ndl1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig1 fig2 fig3 fig4 fig5">Figures 1&#8211;5</xref>, <xref ref-type="fig" rid="fig2s1 fig2s2">Figure 2&#8212;figure supplement 1,2</xref>, <xref ref-type="fig" rid="fig1s1">Figure 1&#8212;figure supplement 1</xref>, <xref ref-type="fig" rid="fig4s1">Figure 4&#8212;figure supplement 1</xref>, <xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1</xref></td></tr><tr><td>Julie Huang, Harvard Medical School</td></tr><tr><td rowspan="2">RPY842</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, P</italic><sub><italic>GAL1</italic></sub><italic>-ZZ-Tev-PAC1-g-1xFLAG-ga-SNAP-Kan</italic><sup><italic>R</italic></sup><italic>, dyn1&#196;::cgLEU2, ndl1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig3 fig5">Figures 3,5</xref>, <xref ref-type="fig" rid="fig3s1">Figure 3&#8212;figure supplement 1</xref>, <xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1</xref></td></tr><tr><td><xref ref-type="bibr" rid="bib23">Huang et al., 2012</xref></td></tr><tr><td rowspan="2">RPY844</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, PAC11-13xMYC-TRP1, P</italic><sub><italic>GAL1</italic></sub>-<italic>ZZ-Tev-GFP-3xHA-DYN1</italic><sub><italic>331kDa</italic></sub><italic>, pac1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig1 fig4">Figures 1,4</xref>, <xref ref-type="fig" rid="fig1s1">Figure 1&#8212;figure supplement 1</xref>, <xref ref-type="fig" rid="fig3s1">Figure 3&#8212;figure supplement 1</xref></td></tr><tr><td><xref ref-type="bibr" rid="bib23">Huang et al., 2012</xref></td></tr><tr><td rowspan="2">RPY1198</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, PAC11-13xMYC-TRP1, P</italic><sub><italic>GAL1</italic></sub>-<italic>ZZ-Tev-GFP-3xHA-DYN1</italic><sub><italic>331kDa</italic></sub><italic>-gs-DHA-Kan</italic><sup><italic>R</italic></sup><italic>, pac1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig5">Figure 5</xref>, <xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1</xref></td></tr><tr><td><xref ref-type="bibr" rid="bib23">Huang et al., 2012</xref></td></tr><tr><td rowspan="2">RPY1245</td><td rowspan="2">MATa, <italic>ura3-52, lys2-801, leu2-&#196;1, his3-&#196;200, trp1-&#196;63, SPC110-GFP::TRP1, HXT1-tdTomato::HIS3</italic></td><td><xref ref-type="fig" rid="fig2">Figure 2</xref></td></tr><tr><td>Jeff Moore, University of Colorado</td></tr><tr><td rowspan="2">RPY1248</td><td rowspan="2">MATa, <italic>ura3-52, lys2-801, leu2-&#196;1, his3-&#196;200, trp1-&#196;63, SPC110-GFP::TRP1, HXT1-tdTomato::HIS3, dyn1&#196;::URA3</italic></td><td><xref ref-type="fig" rid="fig2">Figure 2</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1302</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, PAC11-13xMYC-TRP1, P</italic><sub><italic>GAL1</italic></sub>-<italic>ZZ-Tev-DYN1</italic><sub><italic>331kDa</italic></sub><italic>, pac1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig1 fig3">Figures 1,3</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1400</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, PAC11-13xMYC-TRP1, P</italic><sub><italic>GAL1</italic></sub>-<italic>ZZ-Tev-GFP-3xHA-DYN1</italic><sub><italic>331kDa</italic></sub><italic>-L2441ybbR, pac1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig3">Figure 3</xref>, <xref ref-type="fig" rid="fig3s1">Figure 3&#8212;figure supplement 1</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1422</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-52, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, P</italic><sub><italic>GAL1</italic></sub>-<italic>ZZ-Tev-GFP-3xHA-DYN1</italic><sub><italic>314kDa</italic></sub><italic>-gs-DHA, pac1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig4 fig5">Figures 4,5</xref>, <xref ref-type="fig" rid="fig4s1">Figure 4&#8212;figure supplement 1</xref>, <xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1436</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-52, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, PAC11-13xMYC-TRP1, P</italic><sub><italic>GAL1</italic></sub>-<italic>ZZ-Tev- DYN1</italic><sub><italic>314kDa</italic></sub><italic>, pac1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig5">Figure 5</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1439</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, P</italic><sub><italic>GAL1</italic></sub><italic>-ZZ-Tev-GFP-3xHA-GST-DYN1</italic><sub><italic>314</italic></sub> <sub><italic>kDa</italic></sub><italic>-gs-DHA-Kan</italic><sup><italic>R</italic></sup><italic>, pac1&#196;:URA3, ndl1&#196;::cgLEU2</italic></td><td><xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1509</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, PAC11-13xMYC-TRP1, P</italic><sub><italic>GAL1</italic></sub>-<italic>ZZ-Tev-DYN1</italic><sub><italic>331kDa</italic></sub><italic>-gs-DHA-Kan</italic><sup><italic>R</italic></sup><italic>, pac1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1510</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, PAC11-13xMYC-TRP1, P</italic><sub><italic>GAL1</italic></sub>-<italic>ZZ-Tev-DYN1</italic><sub><italic>314kDa</italic></sub><italic>-gs-DHA-Kan</italic><sup><italic>R</italic></sup><italic>, pac1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1523</td><td rowspan="2">MATa, <italic>ura3-52, lys2-801, leu2-&#196;1, his3-&#196;200, trp1-&#196;3, SPC110-GFP::TRP1, HXT1-tdTomato::HIS3, pac1&#196;::URA3</italic></td><td><xref ref-type="fig" rid="fig2">Figure 2</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1524</td><td rowspan="2">MATa, <italic>ura3-52, lys2-801, leu2-&#196;1, his3-&#196;200, trp1-&#196;63, SPC110-GFP::TRP1, HXT1-tdTomato::HIS3, PAC1</italic><sup><italic>R378A</italic></sup></td><td><xref ref-type="fig" rid="fig2">Figure 2</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1525</td><td rowspan="2">MATa, <italic>ura3-52, lys2-801, leu2-&#196;1, his3-&#196;200, trp1-&#196;63, SPC110-GFP::TRP1, HXT1-tdTomato::HIS3, PAC1</italic><sup><italic>R275A,R301A,R378A,W419A,K437A</italic></sup></td><td><xref ref-type="fig" rid="fig2">Figure 2</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1543</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, P</italic><sub><italic>GAL1</italic></sub><italic>-ZZ-Tev-PAC1</italic><sup><italic>R275A</italic></sup><italic>, dyn1&#196;::cgLEU2, ndl1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig2s1">Figure 2&#8212;figure supplement 1</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1544</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, P</italic><sub><italic>GAL1</italic></sub><italic>-ZZ-Tev-PAC1</italic><sup><italic>R378A</italic></sup><italic>, dyn1&#196;::cgLEU2, ndl1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig2">Figure 2</xref>, <xref ref-type="fig" rid="fig2s1 fig2s2">Figure 2&#8212;figure supplement 1,2</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1545</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, P</italic><sub><italic>GAL1</italic></sub><italic>-ZZ-Tev-PAC1</italic><sup><italic>W419A</italic></sup><italic>, dyn1&#196;::cgLEU2, ndl1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig2s1">Figure 2&#8212;figure supplement 1</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1546</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, P</italic><sub><italic>GAL1</italic></sub><italic>-ZZ-Tev-PAC1</italic><sup><italic>K437A</italic></sup><italic>, dyn1&#196;::cgLEU2, ndl1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig2s1">Figure 2&#8212;figure supplement 1</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1547</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, P</italic><sub><italic>GAL1</italic></sub><italic>-ZZ-Tev-PAC1</italic> <sup><italic>R275A,R301A,R378A,W419A,K437A</italic></sup><italic>, dyn1&#196;::cgLEU2, ndl1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig2">Figure 2</xref>, <xref ref-type="fig" rid="fig2s1 fig2s2">Figure 2&#8212;figure supplement 1,2</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1548</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, P</italic><sub><italic>GAL1</italic></sub><italic>-ZZ-Tev-PAC1</italic><sup><italic>R301A</italic></sup><italic>, dyn1&#196;::cgLEU2, ndl1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig2s1">Figure 2&#8212;figure supplement 1</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1553</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, PAC11-13xMYC-TRP1, P</italic><sub><italic>GAL1</italic></sub>-<italic>ZZ-Tev-GFP-3xHA-DYN1</italic><sub><italic>331kDa</italic></sub><sup><italic>E1849Q</italic></sup><italic>, pac1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig4">Figure 4</xref>, <xref ref-type="fig" rid="fig4s1">Figure 4&#8212;figure supplement 1</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1554</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, PAC11-13xMYC-TRP1, P</italic><sub><italic>GAL1</italic></sub>-<italic>ZZ-Tev-GFP-3xHA-DYN1</italic><sub><italic>331kDa</italic></sub><sup><italic>E2819Q</italic></sup><italic>, pac1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig4">Figure 4</xref>, <xref ref-type="fig" rid="fig4s1">Figure 4&#8212;figure supplement 1</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1555</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-52, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, P</italic><sub><italic>GAL1</italic></sub>-<italic>ZZ-Tev-GFP-3xHA-DYN1</italic><sub><italic>314kDa</italic></sub><sup><italic>K3438E,R3445E,F3446D</italic></sup><italic>-gs-DHA, pac1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig4s1">Figure 4&#8212;figure supplement 1</xref>, <xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1557</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, PAC11-13xMYC-TRP1, P</italic><sub><italic>GAL1</italic></sub>-<italic>ZZ-Tev-GFP-3xHA-DYN1</italic><sub><italic>331kDa</italic></sub><sup><italic>K3438E,R3445E,F3446D</italic></sup><italic>-gs-DHA-Kan</italic><sup><italic>R</italic></sup><italic>, pac1&#196;::Hygro</italic><sup><italic>R</italic></sup></td><td><xref ref-type="fig" rid="fig4">Figure 4</xref>, <xref ref-type="fig" rid="fig4s1">Figure 4&#8212;figure supplement 1</xref></td></tr><tr><td>This work</td></tr><tr><td rowspan="2">RPY1623</td><td rowspan="2">MATa<italic>, his3-11,15, ura3-1, leu2-3,112, ade2-1, trp1-1, pep4&#196;::HIS5, prb1&#196;, P</italic><sub><italic>GAL1</italic></sub><italic>-ZZ-Tev-GFP-3xHA-GST- DYN1</italic><sub><italic>331kDa</italic></sub><sup><italic>R2857A,N2858A,K2859A,R2861A,S2862A</italic></sup><italic>-gs-DHA, pac1&#196;::URA3, ndl1&#196;::cgLEU2</italic></td><td><xref ref-type="fig" rid="fig1s2">Figure 1&#8212;figure supplement 2</xref></td></tr><tr><td>This work</td></tr></tbody></table><table-wrap-foot><fn><p><italic>DYN1, PAC11, PAC1, and NDL1</italic> encode the dynein heavy chain, dynein intermediate chain, Lis1 and Nudel orthologs, respectively. <italic>DHA</italic>, <italic>SNAP,</italic> and <italic>ybbR</italic> refer to the HaloTag (Promega), SNAP-tag (NEB), and ybbR tag (<xref ref-type="bibr" rid="bib66">Yin et al., 2005</xref>), respectively. <italic>TEV</italic> indicates a Tev protease cleavage site. <italic>P</italic><sub><italic>GAL1</italic></sub> denotes the galactose promoter, which was used for inducing strong expression of Lis1 and dynein motor domain constructs. Genes encoding proteases Pep4 and Prb1 were deleted as noted. Amino acid spacers are indicated by <italic>g</italic> (glycine), <italic>ga</italic> (glycine-alanine), and <italic>gs</italic> (glycine-serine).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03564-v2.xml">elife-03564-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.03564.010</object-id><label>Table 1.</label><caption><p>Statistical analysis (two-way ANOVA) of ABR threshold and P1 amplitude changes between control and neurotrophin overexpressing (<italic>Plp1/CreER</italic><sup><italic>T</italic></sup>) mice after acoustic trauma</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.03564.010">http://dx.doi.org/10.7554/eLife.03564.010</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2"/><th colspan="2"><italic>Ntf3</italic> overexpressor vs control</th><th colspan="2"><italic>Bdnf</italic> overexpressor vs control</th></tr><tr><th>p Value</th><th>F Statistic</th><th>p Value</th><th>F Statistic</th></tr></thead><tbody><tr><td rowspan="3">ABR threshold</td><td>AT + 1 day</td><td>0.0051</td><td>8.469</td><td>0.6801</td><td>0.1716</td></tr><tr><td>AT + 3 days</td><td>&lt;0.0001</td><td>19.00</td><td>0.6752</td><td>0.1774</td></tr><tr><td>AT + 10 days</td><td>&lt;0.0001</td><td>17.34</td><td>0.9862</td><td>0.0003</td></tr><tr><td rowspan="3">ABR P1 amplitude</td><td>AT + 1 day</td><td>0.0342</td><td>4.695</td><td>0.1899</td><td>1.7580</td></tr><tr><td>AT + 3 days</td><td>&lt;0.0001</td><td>18.08</td><td>0.5692</td><td>0.3276</td></tr><tr><td>AT + 10 days</td><td>&lt;0.0001</td><td>18.13</td><td>0.0421</td><td>4.3160</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03722-v2.xml">elife-03722-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.03722.023</object-id><label>Table 1.</label><caption><p>Permutation tests for null model 1a: shuffled coordinate labels</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.03722.023">http://dx.doi.org/10.7554/eLife.03722.023</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Measures of overlap</th><th colspan="2" rowspan="2">Image analysis</th><th rowspan="2">Observed overlap</th><th colspan="4">Shuffled overlap Values</th><th rowspan="2">Significance</th></tr><tr><th>Mean</th><th>std</th><th>min</th><th>max</th></tr></thead><tbody><tr><td rowspan="6">Range/Sensitivity <inline-formula><mml:math id="inf316"><mml:mrow><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>&#963;</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>NI</mml:mtext></mml:mrow></mml:msub><mml:mo>&#8901;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>s</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>PP</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula></td><td rowspan="3">N = 2</td><td>R = 32</td><td>0.999</td><td>0.859</td><td>0.9 &#215; 10<sup>&#8722;1</sup></td><td>0.704</td><td>0.983</td><td>&lt;0.04</td></tr><tr><td>R = 48</td><td>0.993</td><td>0.832</td><td>1.1 &#215; 10<sup>&#8722;1</sup></td><td>0.651</td><td>0.978</td><td>&lt;0.04</td></tr><tr><td>R = 64</td><td>0.987</td><td>0.809</td><td>1.1 &#215; 10<sup>&#8722;1</sup></td><td>0.614</td><td>0.974</td><td>&lt;0.04</td></tr><tr><td rowspan="3">N = 4</td><td>R = 32</td><td>0.998</td><td>0.825</td><td>1.1 &#215; 10<sup>&#8722;1</sup></td><td>0.638</td><td>0.969</td><td>&lt;0.04</td></tr><tr><td>R = 48</td><td>0.994</td><td>0.812</td><td>1.1 &#215; 10<sup>&#8722;1</sup></td><td>0.646</td><td>0.990</td><td>&lt;0.04</td></tr><tr><td>R = 64</td><td>0.991</td><td>0.794</td><td>1.1 &#215; 10<sup>&#8722;1</sup></td><td>0.617</td><td>0.985</td><td>&lt;0.04</td></tr><tr><td rowspan="6">Inverse Range/Threshold <inline-formula><mml:math id="inf317"><mml:mrow><mml:mo>&#9001;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>&#969;</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>NI</mml:mtext></mml:mrow></mml:msub><mml:mo>&#8901;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>&#969;</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>PP</mml:mtext></mml:mrow></mml:msub><mml:mo>&#9002;</mml:mo></mml:mrow></mml:math></inline-formula></td><td rowspan="3">N = 2</td><td>R = 32</td><td>0.971</td><td>0.709</td><td>1.5 &#215; 10<sup>&#8722;1</sup></td><td>0.508</td><td>0.924</td><td>&lt;0.04</td></tr><tr><td>R = 48</td><td>0.969</td><td>0.692</td><td>1.6 &#215; 10<sup>&#8722;1</sup></td><td>0.469</td><td>0.924</td><td>&lt;0.04</td></tr><tr><td>R = 64</td><td>0.953</td><td>0.685</td><td>1.7 &#215; 10<sup>&#8722;1</sup></td><td>0.450</td><td>0.913</td><td>&lt;0.04</td></tr><tr><td rowspan="3">N = 4</td><td>R = 32</td><td>0.967</td><td>0.679</td><td>1.7 &#215; 10<sup>&#8722;1</sup></td><td>0.447</td><td>0.908</td><td>&lt;0.04</td></tr><tr><td>R = 48</td><td>0.975</td><td>0.632</td><td>1.5 &#215; 10<sup>&#8722;1</sup></td><td>0.400</td><td>0.880</td><td>&lt;0.04</td></tr><tr><td>R = 64</td><td>0.977</td><td>0.648</td><td>1.6 &#215; 10<sup>&#8722;1</sup></td><td>0.411</td><td>0.894</td><td>&lt;0.04</td></tr><tr><td rowspan="6">Fractional Principal Components <inline-formula><mml:math id="inf318"><mml:mrow><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>f</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>NI</mml:mtext></mml:mrow></mml:msub><mml:mo>&#8901;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>f</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>PP</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula></td><td rowspan="3">N = 2</td><td>R = 32</td><td>0.994</td><td>0.382</td><td>1.5 &#215; 10<sup>&#8722;1</sup></td><td>0.160</td><td>0.657</td><td>&lt;0.04</td></tr><tr><td>R = 48</td><td>0.995</td><td>0.485</td><td>1.2 &#215; 10<sup>&#8722;1</sup></td><td>0.287</td><td>0.727</td><td>&lt;0.04</td></tr><tr><td>R = 64</td><td>0.991</td><td>0.487</td><td>0.7 &#215; 10<sup>&#8722;1</sup></td><td>0.372</td><td>0.632</td><td>&lt;0.04</td></tr><tr><td rowspan="3">N = 4</td><td>R = 32</td><td>0.995</td><td>0.459</td><td>1.4 &#215; 10<sup>&#8722;1</sup></td><td>0.238</td><td>0.732</td><td>&lt;0.04</td></tr><tr><td>R = 48</td><td>0.996</td><td>0.444</td><td>1.0 &#215; 10<sup>&#8722;1</sup></td><td>0.277</td><td>0.601</td><td>&lt;0.04</td></tr><tr><td>R = 64</td><td>0.996</td><td>0.450</td><td>1.1 &#215; 10<sup>&#8722;1</sup></td><td>0.279</td><td>0.614</td><td>&lt;0.04</td></tr><tr><td rowspan="6">Full Principal Components <inline-formula><mml:math id="inf319"><mml:mrow><mml:mo>&#9001;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>F</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>NI</mml:mtext></mml:mrow></mml:msub><mml:mo>&#8901;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>F</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>PP</mml:mtext></mml:mrow></mml:msub><mml:mo>&#9002;</mml:mo></mml:mrow></mml:math></inline-formula></td><td rowspan="3">N = 2</td><td>R = 32</td><td>0.917</td><td>0.316</td><td>1.3 &#215; 10<sup>&#8722;1</sup></td><td>0.123</td><td>0.578</td><td>&lt;0.04</td></tr><tr><td>R = 48</td><td>0.828</td><td>0.401</td><td>1.0 &#215; 10<sup>&#8722;1</sup></td><td>0.228</td><td>0.611</td><td>&lt;0.04</td></tr><tr><td>R = 64</td><td>0.911</td><td>0.363</td><td>0.7 &#215; 10<sup>&#8722;1</sup></td><td>0.282</td><td>0.532</td><td>&lt;0.04</td></tr><tr><td rowspan="3">N = 4</td><td>R = 32</td><td>0.882</td><td>0.376</td><td>1.2 &#215; 10<sup>&#8722;1</sup></td><td>0.180</td><td>0.618</td><td>&lt;0.04</td></tr><tr><td>R = 48</td><td>0.917</td><td>0.362</td><td>1.0 &#215; 10<sup>&#8722;1</sup></td><td>0.201</td><td>0.520</td><td>&lt;0.04</td></tr><tr><td>R = 64</td><td>0.919</td><td>0.357</td><td>1.0 &#215; 10<sup>&#8722;1</sup></td><td>0.196</td><td>0.522</td><td>&lt;0.04</td></tr></tbody></table><table-wrap-foot><fn><p>We separately permute the sets of coordinate labels <inline-formula><mml:math id="inf320"><mml:mrow><mml:mrow><mml:mo>{</mml:mo><mml:mrow><mml:msub><mml:mi>&#946;</mml:mi><mml:mrow><mml:mo>|</mml:mo><mml:mo>&#8722;</mml:mo></mml:mrow></mml:msub><mml:mo>,</mml:mo><mml:msub><mml:mi>&#946;</mml:mi><mml:mrow><mml:mo>\</mml:mo><mml:mo>/</mml:mo></mml:mrow></mml:msub><mml:mo>,</mml:mo><mml:mi>&#952;</mml:mi><mml:mo>,</mml:mo><mml:mi>&#945;</mml:mi></mml:mrow><mml:mo>}</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula>. We apply these permutations to the psychophysical data, therein examining all 23 non-identity permutations of the four labels. This shuffling significantly decreases the overlap between image analyses and psychophysical data. Results are significant across all six analyses considered in <xref ref-type="fig" rid="fig1 fig3">Figures 1 and 3</xref> (<italic>N</italic> = 2, 4 and <italic>R</italic> = 32, 48, 64). p-values, estimated as the fraction of permutations for which the shuffled overlap exceeds the true overlap, are less than 0.04 (the minimum value given 23 permutations) for each image analysis.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03722-v2.xml">elife-03722-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.03722.024</object-id><label>Table 2.</label><caption><p>Permutation Tests for null model 1b: shuffled coordinate labels</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.03722.024">http://dx.doi.org/10.7554/eLife.03722.024</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Measures of overlap</th><th colspan="2" rowspan="2">Image analysis</th><th rowspan="2">Observed overlap</th><th colspan="4">Shuffled overlap Values</th><th rowspan="2">Significance</th></tr><tr><th>Mean</th><th>std</th><th>min</th><th>max</th></tr></thead><tbody><tr><td rowspan="6">Range/Sensitivity <inline-formula><mml:math id="inf184"><mml:mrow><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>&#963;</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>NI</mml:mtext></mml:mrow></mml:msub><mml:mo>&#8901;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>s</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>PP</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula></td><td rowspan="3">N = 2</td><td>R = 32</td><td>0.999</td><td>0.806</td><td>6.8 &#215; 10<sup>&#8722;2</sup></td><td>0.659</td><td>0.999</td><td>0.0003</td></tr><tr><td>R = 48</td><td>0.993</td><td>0.775</td><td>7.7 &#215; 10<sup>&#8722;2</sup></td><td>0.610</td><td>0.993</td><td>&lt;0.0001</td></tr><tr><td>R = 64</td><td>0.987</td><td>0.762</td><td>8.0 &#215; 10<sup>&#8722;2</sup></td><td>0.579</td><td>0.987</td><td>&lt;0.0001</td></tr><tr><td rowspan="3">N = 4</td><td>R = 32</td><td>0.998</td><td>0.828</td><td>6.0 &#215; 10<sup>&#8722;2</sup></td><td>0.707</td><td>0.998</td><td>&lt;0.0001</td></tr><tr><td>R = 48</td><td>0.994</td><td>0.798</td><td>7.1 &#215; 10<sup>&#8722;2</sup></td><td>0.660</td><td>0.994</td><td>0.0002</td></tr><tr><td>R = 64</td><td>0.991</td><td>0.780</td><td>7.6 &#215; 10<sup>&#8722;2</sup></td><td>0.630</td><td>0.991</td><td>&lt;0.0001</td></tr><tr><td rowspan="6">Inverse Range/Threshold <inline-formula><mml:math id="inf185"><mml:mrow><mml:mo>&#9001;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>&#969;</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>NI</mml:mtext></mml:mrow></mml:msub><mml:mo>&#8901;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>&#969;</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>PP</mml:mtext></mml:mrow></mml:msub><mml:mo>&#9002;</mml:mo></mml:mrow></mml:math></inline-formula></td><td rowspan="3">N = 2</td><td>R = 32</td><td>0.971</td><td>0.693</td><td>8.1 &#215; 10<sup>&#8722;2</sup></td><td>0.499</td><td>0.972</td><td>0.0002</td></tr><tr><td>R = 48</td><td>0.969</td><td>0.682</td><td>8.4 &#215; 10<sup>&#8722;2</sup></td><td>0.476</td><td>0.969</td><td>0.0003</td></tr><tr><td>R = 64</td><td>0.953</td><td>0.671</td><td>8.5 &#215; 10<sup>&#8722;2</sup></td><td>0.446</td><td>0.954</td><td>0.0002</td></tr><tr><td rowspan="3">N = 4</td><td>R = 32</td><td>0.967</td><td>0.696</td><td>7.6 &#215; 10<sup>&#8722;2</sup></td><td>0.521</td><td>0.964</td><td>&lt;0.0001</td></tr><tr><td>R = 48</td><td>0.975</td><td>0.692</td><td>8.0 &#215; 10<sup>&#8722;2</sup></td><td>0.509</td><td>0.976</td><td>0.0002</td></tr><tr><td>R = 64</td><td>0.977</td><td>0.689</td><td>8.2 &#215; 10<sup>&#8722;2</sup></td><td>0.493</td><td>0.978</td><td>0.0003</td></tr><tr><td rowspan="6">Fractional Principal Components <inline-formula><mml:math id="inf186"><mml:mrow><mml:mo>&#9001;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>f</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>NI</mml:mtext></mml:mrow></mml:msub><mml:mo>&#8901;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>f</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>PP</mml:mtext></mml:mrow></mml:msub><mml:mo>&#9002;</mml:mo></mml:mrow></mml:math></inline-formula></td><td rowspan="3">N = 2</td><td>R = 32</td><td>0.994</td><td>0.592</td><td>1.2 &#215; 10<sup>&#8722;1</sup></td><td>0.271</td><td>0.995</td><td>0.0003</td></tr><tr><td>R = 48</td><td>0.995</td><td>0.604</td><td>1.3 &#215; 10<sup>&#8722;1</sup></td><td>0.281</td><td>0.995</td><td>0.0004</td></tr><tr><td>R = 64</td><td>0.991</td><td>0.591</td><td>1.2 &#215; 10<sup>&#8722;1</sup></td><td>0.278</td><td>0.991</td><td>0.0003</td></tr><tr><td rowspan="3">N = 4</td><td>R = 32</td><td>0.995</td><td>0.590</td><td>1.2 &#215; 10<sup>&#8722;1</sup></td><td>0.218</td><td>0.995</td><td>0.0001</td></tr><tr><td>R = 48</td><td>0.996</td><td>0.577</td><td>1.2 &#215; 10<sup>&#8722;1</sup></td><td>0.251</td><td>0.996</td><td>0.0002</td></tr><tr><td>R = 64</td><td>0.996</td><td>0.581</td><td>1.2 &#215; 10<sup>&#8722;1</sup></td><td>0.266</td><td>0.996</td><td>0.0004</td></tr><tr><td rowspan="6">Full Principal Components <inline-formula><mml:math id="inf187"><mml:mrow><mml:mo>&#9001;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>F</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>NI</mml:mtext></mml:mrow></mml:msub><mml:mo>&#8901;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>F</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>PP</mml:mtext></mml:mrow></mml:msub><mml:mo>&#9002;</mml:mo></mml:mrow></mml:math></inline-formula></td><td rowspan="3">N = 2</td><td>R = 32</td><td>0.917</td><td>0.391</td><td>1.2 &#215; 10<sup>&#8722;1</sup></td><td>0.100</td><td>0.927</td><td>0.0002</td></tr><tr><td>R = 48</td><td>0.828</td><td>0.391</td><td>1.2 &#215; 10<sup>&#8722;1</sup></td><td>0.086</td><td>0.856</td><td>0.0008</td></tr><tr><td>R = 64</td><td>0.911</td><td>0.396</td><td>1.2 &#215; 10<sup>&#8722;1</sup></td><td>0.120</td><td>0.953</td><td>0.0003</td></tr><tr><td rowspan="3">N = 4</td><td>R = 32</td><td>0.882</td><td>0.381</td><td>1.2 &#215; 10<sup>&#8722;1</sup></td><td>0.066</td><td>0.989</td><td>0.0003</td></tr><tr><td>R = 48</td><td>0.917</td><td>0.380</td><td>1.2 &#215; 10<sup>&#8722;1</sup></td><td>0.090</td><td>0.902</td><td>&lt;0.0001</td></tr><tr><td>R = 64</td><td>0.919</td><td>0.387</td><td>1.2 &#215; 10<sup>&#8722;1</sup></td><td>0.095</td><td>0.937</td><td>0.0004</td></tr></tbody></table><table-wrap-foot><fn><p>We separately permute all nine coordinate labels <inline-formula><mml:math id="inf188"><mml:mrow><mml:mo>{</mml:mo><mml:msub><mml:mi>&#946;</mml:mi><mml:mo>|</mml:mo></mml:msub><mml:mo>,</mml:mo><mml:msub><mml:mi>&#946;</mml:mi><mml:mo>&#8722;</mml:mo></mml:msub><mml:mo>,</mml:mo><mml:msub><mml:mi>&#946;</mml:mi><mml:mo>\</mml:mo></mml:msub><mml:mo>,</mml:mo></mml:mrow></mml:math></inline-formula> <inline-formula><mml:math id="inf189"><mml:mrow><mml:msub><mml:mi>&#946;</mml:mi><mml:mo>/</mml:mo></mml:msub><mml:mo>,</mml:mo><mml:msub><mml:mi>&#952;</mml:mi><mml:mi mathvariant="normal">&#8988;</mml:mi></mml:msub><mml:mo>,</mml:mo><mml:msub><mml:mi>&#952;</mml:mi><mml:mi mathvariant="normal">&#8989;</mml:mi></mml:msub><mml:mo>,</mml:mo><mml:msub><mml:mi>&#952;</mml:mi><mml:mi mathvariant="normal">&#8991;</mml:mi></mml:msub><mml:mo>,</mml:mo><mml:msub><mml:mi>&#952;</mml:mi><mml:mi mathvariant="normal">&#8990;</mml:mi></mml:msub><mml:mo>,</mml:mo><mml:mi>&#945;</mml:mi><mml:mo>}</mml:mo></mml:mrow></mml:math></inline-formula>. This shuffling, applied to the psychophysical data, significantly decreases the overlap between image analyses and psychophysical data. Results are significant across all six analyses considered in <xref ref-type="fig" rid="fig1 fig3">Figures 1 and 3</xref> (<italic>N</italic> = 2, 4 and <italic>R</italic> = 32, 48, 64). p-values, estimated as the fraction of permutations for which the shuffled overlap exceeds the true overlap, are less than 0.0005 for all image analyses.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03722-v2.xml">elife-03722-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.03722.025</object-id><label>Table 3.</label><caption><p>Permutation tests for null model 2: shuffled patch labels</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.03722.025">http://dx.doi.org/10.7554/eLife.03722.025</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Comparisons</th><th colspan="2" rowspan="2">Image analysis</th><th rowspan="2">Observed overlap</th><th colspan="4">Shuffled overlap Values</th><th rowspan="2">Significance</th></tr><tr><th>Mean</th><th>std</th><th>min</th><th>max</th></tr></thead><tbody><tr><td rowspan="6">Inverse Range/Threshold <inline-formula><mml:math id="inf190"><mml:mrow><mml:mo>&#9001;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>&#969;</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>NI</mml:mtext></mml:mrow></mml:msub><mml:mo>&#8901;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>&#969;</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>PP</mml:mtext></mml:mrow></mml:msub><mml:mo>&#9002;</mml:mo></mml:mrow></mml:math></inline-formula></td><td rowspan="3">N = 2</td><td>R = 32</td><td>0.971</td><td>0.924</td><td>0.70 &#215; 10<sup>&#8722;3</sup></td><td>0.921</td><td>0.926</td><td>&lt;0.0001</td></tr><tr><td>R = 48</td><td>0.969</td><td>0.921</td><td>1.1 &#215; 10<sup>&#8722;3</sup></td><td>0.917</td><td>0.925</td><td>&lt;0.0001</td></tr><tr><td>R = 64</td><td>0.953</td><td>0.912</td><td>1.3 &#215; 10<sup>&#8722;3</sup></td><td>0.908</td><td>0.917</td><td>&lt;0.0001</td></tr><tr><td rowspan="3">N = 4</td><td>R = 32</td><td>0.967</td><td>0.919</td><td>1.7 &#215; 10<sup>&#8722;3</sup></td><td>0.914</td><td>0.926</td><td>&lt;0.0001</td></tr><tr><td>R = 48</td><td>0.975</td><td>0.922</td><td>1.9 &#215; 10<sup>&#8722;3</sup></td><td>0.916</td><td>0.930</td><td>&lt;0.0001</td></tr><tr><td>R = 64</td><td>0.977</td><td>0.924</td><td>2.8 &#215; 10<sup>&#8722;3</sup></td><td>0.916</td><td>0.935</td><td>&lt;0.0001</td></tr><tr><td rowspan="6">Fractional Principal Components <inline-formula><mml:math id="inf191"><mml:mrow><mml:mo>&#9001;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>f</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>NI</mml:mtext></mml:mrow></mml:msub><mml:mo>&#8901;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>f</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>PP</mml:mtext></mml:mrow></mml:msub><mml:mo>&#9002;</mml:mo></mml:mrow></mml:math></inline-formula></td><td rowspan="3">N = 2</td><td>R = 32</td><td>0.994</td><td>0.806</td><td>9.1 &#215; 10<sup>&#8722;6</sup></td><td>0.806</td><td>0.806</td><td>&lt;0.0001</td></tr><tr><td>R = 48</td><td>0.995</td><td>0.806</td><td>8.3 &#215; 10<sup>&#8722;6</sup></td><td>0.806</td><td>0.806</td><td>&lt;0.0001</td></tr><tr><td>R = 64</td><td>0.991</td><td>0.806</td><td>3.7 &#215; 10<sup>&#8722;6</sup></td><td>0.806</td><td>0.806</td><td>&lt;0.0001</td></tr><tr><td rowspan="3">N = 4</td><td>R = 32</td><td>0.995</td><td>0.807</td><td>2.5 &#215; 10<sup>&#8722;4</sup></td><td>0.806</td><td>0.809</td><td>&lt;0.0001</td></tr><tr><td>R = 48</td><td>0.996</td><td>0.807</td><td>4.1 &#215; 10<sup>&#8722;4</sup></td><td>0.806</td><td>0.810</td><td>&lt;0.0001</td></tr><tr><td>R = 64</td><td>0.996</td><td>0.807</td><td>3.5 &#215; 10<sup>&#8722;4</sup></td><td>0.806</td><td>0.810</td><td>&lt;0.0001</td></tr><tr><td rowspan="6">Full Principal Components <inline-formula><mml:math id="inf192"><mml:mrow><mml:mo>&#9001;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>F</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>NI</mml:mtext></mml:mrow></mml:msub><mml:mo>&#8901;</mml:mo><mml:msub><mml:mrow><mml:mover accent="true"><mml:mi>F</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow><mml:mrow><mml:mtext>PP</mml:mtext></mml:mrow></mml:msub><mml:mo>&#9002;</mml:mo></mml:mrow></mml:math></inline-formula></td><td rowspan="3">N = 2</td><td>R = 32</td><td>0.917</td><td>0.448</td><td>5.8 &#215; 10<sup>&#8722;2</sup></td><td>0.406</td><td>0.596</td><td>&lt;0.0001</td></tr><tr><td>R = 48</td><td>0.828</td><td>0.502</td><td>5.9 &#215; 10<sup>&#8722;2</sup></td><td>0.408</td><td>0.675</td><td>&lt;0.0001</td></tr><tr><td>R = 64</td><td>0.911</td><td>0.458</td><td>4.8 &#215; 10<sup>&#8722;2</sup></td><td>0.407</td><td>0.591</td><td>&lt;0.0001</td></tr><tr><td rowspan="3">N = 4</td><td>R = 32</td><td>0.881</td><td>0.489</td><td>4.9 &#215; 10<sup>&#8722;2</sup></td><td>0.409</td><td>0.638</td><td>&lt;0.0001</td></tr><tr><td>R = 48</td><td>0.917</td><td>0.454</td><td>3.0 &#215; 10<sup>&#8722;2</sup></td><td>0.408</td><td>0.637</td><td>&lt;0.0001</td></tr><tr><td>R = 64</td><td>0.919</td><td>0.492</td><td>4.2 &#215; 10<sup>&#8722;2</sup></td><td>0.411</td><td>0.648</td><td>&lt;0.0001</td></tr></tbody></table><table-wrap-foot><fn><p>Within each image analyses, we separately permute image patch labels along individual coordinate axes. This shuffling does not alter the range of variation observed along individual coordinates; as a result, this test only applies to <inline-formula><mml:math id="inf193"><mml:mrow><mml:mrow><mml:mover accent="true"><mml:mi>&#969;</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow></mml:mrow></mml:math></inline-formula> ,<inline-formula><mml:math id="inf194"><mml:mrow><mml:mrow><mml:mover accent="true"><mml:mi>f</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow></mml:mrow></mml:math></inline-formula> and <inline-formula><mml:math id="inf195"><mml:mrow><mml:mrow><mml:mover accent="true"><mml:mi>F</mml:mi><mml:mo>&#8594;</mml:mo></mml:mover></mml:mrow></mml:mrow></mml:math></inline-formula>. We find that this shuffling significantly decreases the overlap between image analyses and psychophysical data. Results are significant across all six analyses considered in <xref ref-type="fig" rid="fig1 fig3">Figures 1 and 3</xref> (<italic>N</italic> = 2, 4 and <italic>R</italic> = 32, 48, 64). p-values, estimated as the fraction of permutations for which the shuffled overlap exceeds the true overlap, are less than 0.0001 for each image analysis.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03925-v1.xml">elife-03925-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.03925.004</object-id><label>Table 1.</label><caption><p>Baseline characteristics of 8390 children tested for malaria using blood film, by survey location</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.03925.004">http://dx.doi.org/10.7554/eLife.03925.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><italic>Characteristic</italic></th><th align="center" colspan="4">Location of survey</th><th align="center" rowspan="2">Overall (n=8390)</th></tr><tr><th align="center">Burkina Faso (n=2821)</th><th align="center">Mozambique (n=2266)</th><th align="center">Rwanda (n=2085)</th><th align="center">Senegal (n=1218)</th></tr></thead><tbody><tr><td>Communities surveyed, n</td><td align="center">537</td><td align="center">607</td><td align="center">490</td><td align="center">371</td><td align="center">2005</td></tr><tr><td colspan="6">(a) Children tested for malaria, n (%)</td></tr><tr><td>&#8195;<italic>Parasitemia</italic></td><td align="center">2821 (99)</td><td align="center">2266 (100)</td><td align="center">2085 (99)</td><td align="center">1218 (99)</td><td align="center">8390 (99)</td></tr><tr><td>&#8195;Positive result among children tested for parasitemia</td><td align="center">&#8195;1696 (60)</td><td align="center">&#8195;578 (25)</td><td align="center">&#8195;16 (0.8)</td><td align="center">&#8195;22 (1.8)</td><td align="center">&#8195;2312 (28)</td></tr><tr><td>&#8195;<italic>P&#402;-HRP-2</italic></td><td align="center">2821 (100)</td><td align="center">na</td><td align="center">2060 (99)</td><td align="center">1217 (99)</td><td align="center">6098 (99)<xref ref-type="table-fn" rid="tblfn1">*</xref></td></tr><tr><td>&#8195;Positive results among children tested for <italic>P&#402;-HRP-2</italic></td><td align="center">&#8195;2051 (73)</td><td align="center">na</td><td align="center">&#8195;35 (1.7)</td><td align="center">&#8195;27 (2.2)</td><td align="center">&#8195;2113 (35)</td></tr><tr><td colspan="6">(b) Type of immunization received, n (%)</td></tr><tr><td>&#8195;<italic>Bacille Calmette Guerin (BCG)</italic></td><td align="center">2799 (99)</td><td align="center">2004 (96)</td><td align="center">2050 (100)</td><td align="center">1160 (98)</td><td align="center">8013 (98)</td></tr><tr><td>&#8195;<italic>Diphtheria&#8211;tetanus&#8211;pertussis (DTP)</italic></td><td align="center">2721 (97)</td><td align="center">2107 (97)</td><td align="center">2055 (98)</td><td align="center">1161 (98)</td><td align="center">8044 (98)</td></tr><tr><td>&#8195;<italic>Measles</italic></td><td align="center">2225 (80)</td><td align="center">1668 (78)</td><td align="center">1727 (86)</td><td align="center">830 (73)</td><td align="center">6450 (80)</td></tr><tr><td>&#8195;<italic>Poliomyelitis</italic></td><td align="center">2810 (100)</td><td align="center">2156 (99)</td><td align="center">2061 (100)</td><td align="center">1204 (100)</td><td align="center">8231 (99)</td></tr><tr><td>&#8195;<italic>Vitamin A</italic></td><td align="center">87 (9.3)</td><td align="center">1554 (87)</td><td align="center">351 (72)</td><td align="center">190 (59)</td><td align="center">2182 (62)</td></tr><tr><td colspan="6">(c) Children's characteristics</td></tr><tr><td>&#8195;<italic>Age in months, median (IQR)</italic></td><td align="center">22 (13&#8211;32)</td><td align="center">20 (13&#8211;31)</td><td align="center">24 (14&#8211;36)</td><td align="center">19 (12&#8211;29)</td><td align="center">22 (13&#8211;32)</td></tr><tr><td>&#8195;<italic>Girls, n (%)</italic></td><td align="center">1347 (48)</td><td align="center">1151 (51)</td><td align="center">1020 (49)</td><td align="center">551 (45)</td><td align="center">4069 (48)</td></tr><tr><td>&#8195;<italic>Primigravidae, n (%)</italic></td><td align="center">443 (16)</td><td align="center">477 (21)</td><td align="center">432 (21)</td><td align="center">251 (21)</td><td align="center">1603 (19)</td></tr><tr><td>&#8195;<italic>Low birth weight, n (%)</italic></td><td align="center">867 (31)</td><td align="center">1003 (44)</td><td align="center">672 (32)</td><td align="center">536 (44)</td><td align="center">3078 (37)</td></tr><tr><td colspan="6">(d) Malaria-based interventions, n (%)</td></tr><tr><td>&#8195;<italic>Child's family owns bed net</italic></td><td align="center">2160 (77)</td><td align="center">1528 (67)</td><td align="center">1983 (95)</td><td align="center">1040 (85)</td><td align="center">6711 (80)</td></tr><tr><td>&#8195;Child received antimalarial during past week</td><td align="center">314 (11)</td><td align="center">117 (5.2)</td><td align="center">50 (2.4)</td><td align="center">24 (2.0)</td><td align="center">505 (6.0)</td></tr><tr><td>&#8195;Child's house had indoor insecticide spraying</td><td align="center">22 (0.8)</td><td align="center">541 (24)</td><td align="center">na</td><td align="center">148 (12)</td><td align="center">711 (8.5)<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td></tr><tr><td>&#8195;Mother took antimalarial during child's gestational period</td><td align="center">2616 (93)</td><td align="center">1109 (50)</td><td align="center">337 (16)</td><td align="center">1110 (91)</td><td align="center">5162 (62)</td></tr><tr><td colspan="6">(e) Genetic mechanisms of malaria protection, median (IQR)</td></tr><tr><td>&#8195;Mean predicted HbS allele frequency</td><td align="center">0.06 (0.05&#8211;0.06)</td><td align="center">0.03 (0.01&#8211;0.03)</td><td align="center">0.03 (0.03&#8211;0.03)</td><td align="center">0.07 (0.06&#8211;0.07)</td><td align="center">0.04 (0.03&#8211;0.06)</td></tr><tr><td>&#8195;Median predicted G6PDd allele frequency</td><td align="center">0.06 (0.05&#8211;0.09)</td><td align="center">0.15 (0.15&#8211;0.17)</td><td align="center">0.04 (0.04&#8211;0.05)</td><td align="center">0.10 (0.09&#8211;0.13)</td><td align="center">0.08 (0.05&#8211;0.14)</td></tr><tr><td colspan="6">(f) Climate of communities surveyed, median (IQR)</td></tr><tr><td>&#8195;<italic>Annual range of enhanced vegetation index (EVI)</italic></td><td align="center">0.29 (0.24&#8211;0.32)</td><td align="center">0.33 (0.22&#8211;0.40)</td><td align="center">0.25 (0.20&#8211;0.29)</td><td align="center">0.28 (0.18&#8211;0.34)</td><td align="center">0.28 (0.22&#8211;0.33)</td></tr><tr><td>&#8195;<italic>Annual mean of EVI for the year</italic></td><td align="center">0.22 (0.18&#8211;0.25)</td><td align="center">0.40 (0.32&#8211;0.47)</td><td align="center">0.39 (0.36&#8211;0.42)</td><td align="center">0.18 (0.16&#8211;0.20)</td><td align="center">0.29 (0.20&#8211;0.40)</td></tr><tr><td>&#8195;<italic>No. of days which EVI above annual mean</italic></td><td align="center">136 (120&#8211;160)</td><td align="center">88 (56&#8211;128)</td><td align="center">320 (168&#8211;384)</td><td align="center">208 (184&#8211;264)</td><td align="center">152 (120&#8211;200)</td></tr><tr><td>&#8195;<italic>No. of days for rainy season (corresponding to first and last day EVI above annual mean)</italic></td><td align="center">136 (120&#8211;152)</td><td align="center">128 (72&#8211;344)</td><td align="center">344 (296&#8211;352)</td><td align="center">216 (152&#8211;352)</td><td align="center">168 (120&#8211;344)</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p><italic>P</italic>&#402;-HRP-2 testing not conducted as part of DHS survey protocol for Mozambique.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>Information on insecticide spraying not collected as part of survey.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03925-v1.xml">elife-03925-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.03925.005</object-id><label>Table 2.</label><caption><p>Relative risk of malaria infection after standard vaccination and vitamin A supplementation among children 6&#8211;59 months of age</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.03925.005">http://dx.doi.org/10.7554/eLife.03925.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Type of immunization</th><th rowspan="2">No. of children vaccinated/total tested (%)</th><th colspan="2">No. of children with positive blood test (%)</th><th rowspan="2">Unadjusted RR</th><th align="center" colspan="2">Adjusted RR (95% CI)<xref ref-type="table-fn" rid="tblfn5">&#8225;</xref>,<xref ref-type="table-fn" rid="tblfn6">&#167;</xref></th></tr><tr><th>No vaccine</th><th>Vaccine</th><th align="center">Unweighted</th><th align="center">Weighted (IPW)</th></tr></thead><tbody><tr><td colspan="7">(a) <italic>Plasmodium</italic> species (parasitemia)<xref ref-type="table-fn" rid="tblfn3">*</xref></td></tr><tr><td>&#8195;Bacille Calmette Guerin (BCG)</td><td align="center">8013/8140 (98)</td><td align="center">41 (32)</td><td align="center">2227 (28)</td><td align="center">0.81</td><td align="center">1.25 (0.81&#8211;1.91)</td><td align="center">1.24 (0.76&#8211;2.05)</td></tr><tr><td>&#8195;Diphtheria&#8211;tetanus&#8211;pertussis (DTP)</td><td align="center">8044/8235 (98)</td><td align="center">83 (44)</td><td align="center">2202 (27)</td><td align="center">0.49</td><td align="center">0.88 (0.64&#8211;1.20)</td><td align="center">0.06 (0.01&#8211;0.47)</td></tr><tr><td>&#8195;Measles</td><td align="center">6450/8069 (80)</td><td align="center">473 (29)</td><td align="center">1784 (28)</td><td align="center">0.93</td><td align="center">1.11 (0.96&#8211;1.29)</td><td align="center">1.01 (0.20&#8211;5.19)</td></tr><tr><td>&#8195;Poliomyelitis</td><td align="center">8231/8272 (99)</td><td align="center">14 (34)</td><td align="center">2278 (28)</td><td align="center">0.74</td><td align="center">0.80 (0.37&#8211;1.73)</td><td align="center">0.74 (0.27&#8211;2.01)</td></tr><tr><td>&#8195;Vitamin A supplement</td><td align="center">2182/3523 (62)</td><td align="center">596 (44)</td><td align="center">438 (20)</td><td align="center">0.31</td><td align="center">0.46 (0.39&#8211;0.54)</td><td align="center">0.43 (0.36&#8211;0.52)</td></tr><tr><td colspan="7">(b) <italic>Plasmodium falciparum</italic> (antigenemia)</td></tr><tr><td>&#8195;Bacille Calmette Guerin (BCG)</td><td align="center">6006/6047 (99)</td><td align="center">9 (22)</td><td align="center">2102 (35)</td><td align="center">1.91</td><td align="center">4.06 (2.00&#8211;8.28)</td><td align="center">3.52 (1.66&#8211;7.48)</td></tr><tr><td>&#8195;Diphtheria&#8211;tetanus&#8211;pertussis (DTP)</td><td align="center">5933/6054 (98)</td><td align="center">59 (49)</td><td align="center">2049 (35)</td><td align="center">0.55</td><td align="center">1.34 (0.88&#8211;2.02)</td><td align="center">0.06 (0.01&#8211;0.38)</td></tr><tr><td>&#8195;Measles</td><td align="center">4776/5937 (80)</td><td align="center">410 (35)</td><td align="center">1679 (35)</td><td align="center">0.99</td><td align="center">1.15 (0.97&#8211;1.38)</td><td align="center">0.68 (0.15&#8211;3.12)</td></tr><tr><td>&#8195;Poliomyelitis</td><td align="center">6.072/6084 (99)</td><td align="center">5 (42)</td><td align="center">2111 (35)</td><td align="center">0.75</td><td align="center">1.39 (0.55&#8211;3.49)</td><td align="center">0.93 (0.37&#8211;2.35)</td></tr><tr><td>&#8195;Vitamin A supplement</td><td align="center">629/1749 (36)</td><td align="center">621 (56)</td><td align="center">75 (12)</td><td align="center">0.10</td><td align="center">0.23 (0.17&#8211;0.29)</td><td align="center">0.22 (0.16&#8211;0.29)</td></tr></tbody></table><table-wrap-foot><fn><p>HRP-2: histidine rich protein-2; RR: relative risk; CI: confidence interval; IPW: inverse probability weighted model.</p></fn><fn id="tblfn3"><label>*</label><p>Tested in four countries: Burkina Faso, Mozambique, Rwanda and Senegal.</p></fn><fn id="tblfn4"><label>&#8224;</label><p>Tested in three countries: Burkina Faso, Rwanda and Senegal.</p></fn><fn id="tblfn5"><label>&#8225;</label><p>Adjusted for the following factors: age, gender, wealth index score, mother's highest level of education, malaria treatment during previous week, ownership of bed net, proportion of household members under 5 years using bed net during previous night, indoor household insecticide spraying, mother's access to antenatal care during last pregnancy, mother's knowledge regarding vertical HIV transmission, malaria transmission season, and type of community setting (urban vs rural).</p></fn><fn id="tblfn6"><label>&#167;</label><p>Inverse probability weighting (IPW) based on propensity score model with following factors: age, gender, low birth weight, presence of radio or television, urban versus rural setting, breastfeeding status, wealth index score, mother's age, mother's highest education level, antenatal care during last pregnancy, and mother's tetanus status during last pregnancy.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03925-v1.xml">elife-03925-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.03925.007</object-id><label>Table 3.</label><caption><p>Modifiers of the association between vitamin A supplementation and malaria infection among children 6&#8211;59 months of age</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.03925.007">http://dx.doi.org/10.7554/eLife.03925.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="3">Characteristics at blood testing</th><th align="center" rowspan="3">Level in the model</th><th align="center" colspan="5"><italic>Plasmodium</italic> species (parasitemia)<xref ref-type="table-fn" rid="tblfn7">*</xref></th><th colspan="2" rowspan="2"><italic>Plasmodium falciparum</italic> (antigenemia)<xref ref-type="table-fn" rid="tblfn8">&#8224;</xref></th></tr><tr><th align="center" colspan="2">No. of children with positive blood film (%)</th><th align="center" rowspan="2">Unadjusted RR</th><th align="center" colspan="2">Adjusted model<sup>3</sup></th></tr><tr><th align="center">No vitamin A</th><th align="center">Vitamin A</th><th align="center">RR (95% CI)</th><th align="center">p Value (interaction term)</th><th>Adjusted RR<xref ref-type="table-fn" rid="tblfn9">&#8225;</xref> (95%CI)</th><th>p Value (interaction term)</th></tr></thead><tbody><tr><td colspan="9">(a) Individual level</td></tr><tr><td colspan="9">Children's characteristics</td></tr><tr><td rowspan="2">&#8195;Age at malaria screening</td><td>6&#8211;35 Months</td><td align="center">484 (43)</td><td align="center">344 (20)</td><td align="center">0.33</td><td align="center">0.46 (0.38&#8211;0.55)</td><td align="center">&lt;0.01<xref ref-type="table-fn" rid="tblfn11">#</xref></td><td align="center">0.26 (0.20&#8211;0.34)</td><td align="center">&lt;0.01<xref ref-type="table-fn" rid="tblfn11">#</xref></td></tr><tr><td>36&#8211;59 Months</td><td align="center">112 (54)</td><td align="center">94 (21)</td><td align="center">0.23</td><td align="center">0.34 (0.25&#8211;0.47)</td><td/><td align="center">0.09 (0.05&#8211;0.14)</td><td/></tr><tr><td rowspan="2">&#8195;Gender</td><td>Girl</td><td align="center">274 (44)</td><td align="center">226 (21)</td><td align="center">0.34</td><td align="center">0.54 (0.44&#8211;0.67)</td><td align="center">0.29</td><td align="center">0.23 (0.16&#8211;0.32)</td><td align="center">0.99</td></tr><tr><td>Boy</td><td align="center">322 (45)</td><td align="center">212 (19)</td><td align="center">0.29</td><td align="center">0.40 (0.32&#8211;0.49)</td><td/><td align="center">0.23 (0.17&#8211;0.30)</td><td/></tr><tr><td rowspan="2">&#8195;Pregnancy order of child</td><td>Primigravidae</td><td align="center">522 (47)</td><td align="center">366 (21)</td><td align="center">0.30</td><td align="center">0.45 (0.38&#8211;0.53)</td><td align="center">0.27</td><td align="center">0.20 (0.15&#8211;0.27)</td><td align="center">0.04</td></tr><tr><td>Multigravidae</td><td align="center">74 (31)</td><td align="center">72 (15)</td><td align="center">0.41</td><td align="center">0.51 (0.39&#8211;0.66)</td><td/><td align="center">0.33 (0.22&#8211;0.48)</td><td/></tr><tr><td rowspan="2">&#8195;Birth weight</td><td>2500 mg or greater</td><td align="center">332 (42)</td><td align="center">204 (15)</td><td align="center">0.24</td><td align="center">0.39 (0.32&#8211;0.48)</td><td align="center">0.01</td><td align="center">0.27 (0.20&#8211;0.36)</td><td align="center">0.02</td></tr><tr><td>Less than 2500 mg</td><td align="center">264 (48)</td><td align="center">234 (28)</td><td align="center">0.43</td><td align="center">0.53 (0.43&#8211;0.66)</td><td/><td align="center">0.13 (0.09&#8211;0.18)</td><td/></tr><tr><td rowspan="2">&#8195;Treatment for intestinal worms during past 6 months</td><td>Not received</td><td align="center">554 (46)</td><td align="center">212 (25)</td><td align="center">0.39</td><td align="center">0.50 (0.40&#8211;0.61)</td><td align="center">0.08</td><td align="center">0.38 (0.28&#8211;0.52)</td><td align="center">0.07</td></tr><tr><td>Received</td><td align="center">37 (30)</td><td align="center">221 (17)</td><td align="center">0.46</td><td align="center">0.66 (0.52&#8211;0.86)</td><td/><td align="center">0.15 (0.11&#8211;0.21)</td><td/></tr><tr><td colspan="9">Malaria-based interventions</td></tr><tr><td rowspan="2">&#8195;Malaria treatment during previous week</td><td>Not received</td><td align="center">550 (44)</td><td align="center">404 (19)</td><td align="center">0.31</td><td align="center">0.44 (0.37&#8211;0.52)</td><td align="center">0.24</td><td align="center">0.20 (0.16&#8211;0.27)</td><td align="center">0.02</td></tr><tr><td>Received</td><td align="center">46 (49)</td><td align="center">34 (33)</td><td align="center">0.51</td><td align="center">0.88 (0.62&#8211;1.26)</td><td/><td align="center">1.01 (0.60&#8211;1.68)</td><td/></tr><tr><td rowspan="2">&#8195;Mother took antimalarial during child's gestational period</td><td>No</td><td align="center">102 (28)</td><td align="center">226 (22)</td><td align="center">0.69</td><td align="center">0.78 (0.59&#8211;1.03)</td><td align="center">&lt;0.001</td><td align="center">0.38 (0.19&#8211;0.74)</td><td align="center">0.05</td></tr><tr><td>Yes</td><td align="center">491 (50)</td><td align="center">210 (19)</td><td align="center">0.23</td><td align="center">0.36 (0.30&#8211;0.45)</td><td/><td align="center">0.20 (0.16&#8211;0.27)</td><td/></tr><tr><td rowspan="2">&#8195;Family owns bed net</td><td>Does not own bed net</td><td align="center">141 (47)</td><td align="center">123 (22)</td><td align="center">0.32</td><td align="center">0.43 (0.34&#8211;0.56)</td><td align="center">0.42</td><td align="center">0.20 (0.15&#8211;0.26)</td><td align="center">0.70</td></tr><tr><td>Owns bed net</td><td align="center">455 (44)</td><td align="center">315 (41)</td><td align="center">0.31</td><td align="center">0.48 (0.40&#8211;0.58)</td><td/><td align="center">0.24 (0.18&#8211;0.32)</td><td/></tr><tr><td colspan="9">(B) Community level (primary sampling unit)</td></tr><tr><td rowspan="2">&#8195;Type of setting</td><td>Rural</td><td align="center">518 (50)</td><td align="center">370 (25)</td><td align="center">0.35</td><td align="center">0.46 (0.39&#8211;0.56)</td><td align="center">0.95</td><td align="center">0.22 (0.17&#8211;0.30)</td><td align="center">0.91</td></tr><tr><td>Urban</td><td align="center">78 (26)</td><td align="center">68 (9.4)</td><td align="center">0.29</td><td align="center">0.34 (0.22&#8211;0.51)</td><td/><td align="center">0.22 (0.12&#8211;0.40)</td><td/></tr><tr><td colspan="9">Genetic mechanisms of malaria protection</td></tr><tr><td rowspan="3">&#8195;Mean predicted HbS allele frequency<xref ref-type="table-fn" rid="tblfn10">&#167;</xref></td><td>Less than 2.5%</td><td align="center">14 (20)</td><td align="center">104 (14)</td><td align="center">0.78</td><td align="center">0.78 (0.32&#8211;1.91)</td><td align="center">&lt;0.01<xref ref-type="table-fn" rid="tblfn11">#</xref></td><td align="center">-</td><td align="center">0.26<xref ref-type="table-fn" rid="tblfn11">#</xref></td></tr><tr><td>2.5&#8211;4.9%</td><td align="center">181 (41)</td><td align="center">269 (25)</td><td align="center">1.49</td><td align="center">0.95 (0.64&#8211;1.42)</td><td/><td align="center">0.96 (0.24&#8211;3.91)</td><td/></tr><tr><td>5% or greater</td><td align="center">401 (48)</td><td align="center">65 (18)</td><td align="center">0.21</td><td align="center">0.31 (0.17&#8211;0.56)</td><td/><td align="center">0.17 (0.08&#8211;0.39)</td><td/></tr><tr><td rowspan="3">&#8195;Median predicted G6PDd allele frequency<xref ref-type="table-fn" rid="tblfn10">&#167;</xref></td><td>Less than 7.5%</td><td align="center">336 (47)</td><td align="center">47 (11)</td><td align="center">0.04</td><td align="center">5.42 (2.01&#8211;14.6)</td><td align="center">&lt;0.001<xref ref-type="table-fn" rid="tblfn11">#</xref></td><td align="center">1.43 (0.28&#8211;7.21)</td><td align="center">0.02<xref ref-type="table-fn" rid="tblfn11">#</xref></td></tr><tr><td>7.5&#8211;14.9%</td><td align="center">194 (43)</td><td align="center">128 (18)</td><td align="center">0.80</td><td align="center">0.89 (0.52&#8211;1.50)</td><td/><td align="center">0.41 (0.12&#8211;1.34)</td><td/></tr><tr><td>15% or greater</td><td align="center">66 (40)</td><td align="center">263 (25)</td><td align="center">0.65</td><td align="center">0.74 (0.46&#8211;1.19)</td><td/><td align="center">-</td><td/></tr><tr><td colspan="9">Climate of communities surveyed</td></tr><tr><td rowspan="2">&#8195;Season of malaria transmission</td><td>Dry season</td><td align="center">198 (42)</td><td align="center">220 (19)</td><td align="center">0.32</td><td align="center">0.40 (0.31&#8211;0.52)</td><td align="center">0.45</td><td align="center">0.06 (0.03&#8211;0.13)</td><td align="center">&lt;0.001</td></tr><tr><td>Wet season</td><td align="center">398 (46)</td><td align="center">218 (22)</td><td align="center">0.32</td><td align="center">0.53 (0.42&#8211;0.66)</td><td/><td align="center">0.39 (0.28&#8211;0.53)</td><td/></tr><tr><td rowspan="3">&#8195;Length of rainy season</td><td>Less than 120 days</td><td align="center">168 (49)</td><td align="center">151 (21)</td><td align="center">0.27</td><td align="center">0.43 (0.31&#8211;0.61)</td><td align="center">&lt;0.001<xref ref-type="table-fn" rid="tblfn11">#</xref></td><td align="center">0.28 (0.15&#8211;0.50)</td><td align="center">0.03<xref ref-type="table-fn" rid="tblfn11">#</xref></td></tr><tr><td>120&#8211;179 Days</td><td align="center">334 (56)</td><td align="center">105 (26)</td><td align="center">0.27</td><td align="center">0.45 (0.33&#8211;0.62)</td><td/><td align="center">0.22 (0.15&#8211;0.31)</td><td/></tr><tr><td>180 Days or more</td><td align="center">94 (23)</td><td align="center">182 (17)</td><td align="center">0.70</td><td align="center">0.77 (0.56&#8211;1.05)</td><td/><td align="center">0.21 (0.10&#8211;0.43)</td><td/></tr><tr><td rowspan="3">&#8195;Length of time enhanced vegetation index above annual mean</td><td>Less than 120 days</td><td align="center">105 (42)</td><td align="center">220 (21)</td><td align="center">0.38</td><td align="center">0.50 (0.37&#8211;0.69)</td><td align="center">0.18<xref ref-type="table-fn" rid="tblfn11">#</xref></td><td align="center">0.37 (0.09&#8211;1.60)</td><td align="center">0.84<xref ref-type="table-fn" rid="tblfn11">#</xref></td></tr><tr><td>120&#8211;179 Days</td><td align="center">486 (52)</td><td align="center">206 (27)</td><td align="center">0.34</td><td align="center">0.52 (0.42&#8211;0.66)</td><td/><td align="center">0.33 (0.24&#8211;0.45)</td><td/></tr><tr><td>180 Days or more</td><td align="center">5 (3.3)</td><td align="center">12 (3.2)</td><td align="center">0.97</td><td align="center">0.43 (0.12&#8211;1.62)</td><td/><td align="center">0.66 (0.16&#8211;2.85)</td><td/></tr><tr><td rowspan="3">&#8195;Range of enhanced vegetation index per year</td><td>Less than 0.20</td><td align="center">46 (25)</td><td align="center">41 (7.9)</td><td align="center">0.26</td><td align="center">0.38 (0.22&#8211;0.63)</td><td align="center">0.18<xref ref-type="table-fn" rid="tblfn11">#</xref></td><td align="center">0.24 (0.13&#8211;0.47)</td><td align="center">0.13<xref ref-type="table-fn" rid="tblfn11">#</xref></td></tr><tr><td>0.20&#8211;0.29</td><td align="center">247 (44)</td><td align="center">105 (17)</td><td align="center">0.25</td><td align="center">0.48 (0.36&#8211;0.64)</td><td/><td align="center">0.36 (0.25&#8211;0.53)</td><td/></tr><tr><td>0.30 or greater</td><td align="center">303 (51)</td><td align="center">292 (28)</td><td align="center">0.38</td><td align="center">0.48 (0.39&#8211;0.61)</td><td/><td align="center">0.19 (0.13&#8211;0.29)</td><td/></tr><tr><td rowspan="3">&#8195;Annual mean for enhanced vegetation index</td><td>Less than 0.20</td><td align="center">150 (37)</td><td align="center">21 (7.2)</td><td align="center">0.13</td><td align="center">0.17 (0.11&#8211;0.28)</td><td align="center">&lt;0.01<xref ref-type="table-fn" rid="tblfn11">#</xref></td><td align="center">0.16 (0.10&#8211;0.25)</td><td align="center">0.01<xref ref-type="table-fn" rid="tblfn11">#</xref></td></tr><tr><td>0.20&#8211;0.29</td><td align="center">309 (59)</td><td align="center">89 (24)</td><td align="center">0.22</td><td align="center">0.51 (0.37&#8211;0.70)</td><td/><td align="center">0.34 (0.23&#8211;0.51)</td><td/></tr><tr><td>0.30 or greater</td><td align="center">137 (33)</td><td align="center">328 (22)</td><td align="center">0.56</td><td align="center">0.61 (0.46&#8211;0.81)</td><td/><td align="center">0.56 (0.20&#8211;1.53)</td><td/></tr></tbody></table><table-wrap-foot><fn><p>HRP-2: histidine rich protein 2; RR: relative risk; CI: confidence interval; na: not applicable: ref: reference category; HbS: hemoglobin S; G6PD: glucose 6-phosphate dehydrogenase deficiency.</p></fn><fn id="tblfn7"><label>*</label><p>Tested in four countries: Burkina Faso, Mozambique, Rwanda and Senegal.</p></fn><fn id="tblfn8"><label>&#8224;</label><p>Tested in three countries: Burkina Faso, Rwanda and Senegal.</p></fn><fn id="tblfn9"><label>&#8225;</label><p>Adjusted for the following factors: age, gender, wealth index score, mother's highest level of education, malaria treatment during previous week, ownership of bed net, proportion of household members under 5 years using bed net during previous night, indoor household insecticide spraying, mother's access to antenatal care during last pregnancy, mother's knowledge regarding vertical HIV transmission, malaria transmission season, and type of community setting (urban vs rural).</p></fn><fn id="tblfn10"><label>&#167;</label><p>Geographical waypoints were not recorded for 40 communities. Subjects from these PSUs were excluded from analysis.</p></fn><fn id="tblfn11"><label>#</label><p>Covariate treated as continuous term when testing for effect modification in the model.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03925-v1.xml">elife-03925-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tblu1" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Type of Vaccine</th><th colspan="3"><italic>Adjusted RR (95%CI)</italic></th></tr><tr><th colspan="2"><italic>Plasmodium spp.</italic> Parasitemia (Microscopic Examination of Blood Film)</th><th rowspan="2"><italic>Plasmodium falciparum</italic> Antigenemia (Rapid Detection of HRP2) (i.e., excl. Mozambique)</th></tr><tr><th>All Children (i.e., incl. Mozambique)</th><th>Children from countries using both film &amp; RDT (i.e., excl. Mozambique)</th></tr></thead><tbody><tr><td><italic>BCG</italic></td><td>1.24 (0.76-2.05)</td><td>2.71 (1.36-5.39)</td><td>3.52 (1.66-7.48)</td></tr><tr><td><italic>DTP</italic></td><td>0.06 (0.01-0.47)</td><td>0.38 (0.01-10.1)</td><td>0.06 (0.01-0.38)</td></tr><tr><td><italic>Measles</italic></td><td>1.01 (0.20-5.19)</td><td>1.00 (0.18-5.45)</td><td>0.68 (0.15-3.12)</td></tr><tr><td><italic>Polio</italic></td><td>0.74 (0.27-2.01)</td><td>0.60 (0.22-1.62)</td><td>0.93 (0.37-2.35)</td></tr><tr><td><italic>Vitamin A</italic></td><td><bold>0.43 (0.36-0.52)</bold></td><td><bold>0.24 (0.18-0.34)</bold></td><td><bold>0.22 (0.16-0.29)</bold></td></tr><tr><td><italic>Countries included in analysis</italic></td><td><italic>Burkina Faso, Rwanda, Senegal &amp; Mozambique</italic></td><td colspan="2"><italic>Burkina Faso, Rwanda, Senegal</italic></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03925-v1.xml">elife-03925-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tblu2" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Type of Vaccine</th><th colspan="4"><italic>Adjusted RR (95%CI)</italic></th></tr><tr><th colspan="2"><italic>Plasmodium spp.</italic> Parasitemia (Microscopic Examination of Blood Film)</th><th colspan="2"><italic>Plasmodium falciparum</italic> Antigenemia (Rapid Detection of HRP2)</th></tr><tr><th>Not adjusting for mother&#8217;s HIV status</th><th>Adjusting for mother&#8217;s HIV status</th><th>Not adjusting for mother&#8217;s HIV status</th><th>Adjusting for mother&#8217;s HIV status</th></tr></thead><tbody><tr><td><italic>BCG</italic></td><td>2.71 (1.36-5.39)</td><td>2.20 (0.68-7.10)</td><td>3.23 (1.58-6.61)</td><td>2.17 (0.66-7.10)</td></tr><tr><td><italic>DTP</italic></td><td>0.38 (0.01-10.1)</td><td>0.93 (0.23-3.82)</td><td>0.07 (0.01-0.48)</td><td>1.34 (0.51-3.54)</td></tr><tr><td><italic>Measles</italic></td><td>1.00 (0.18-5.45)</td><td>0.66 (0.30-1.45)</td><td>0.73 (0.15-3.61)</td><td>0.86 (0.43-1.72)</td></tr><tr><td><italic>Polio</italic></td><td>0.60 (0.22-1.62)</td><td>1.75 (0.56-5.49)</td><td>0.81 (0.28-2.31)</td><td>3.27 (1.17-9.15)</td></tr><tr><td><italic>Vitamin A</italic></td><td>0.24 (0.18-0.34)</td><td>0.09 (0.04-0.19)</td><td>0.23 (0.17-0.31)</td><td>0.13 (0.06-0.27)</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-03925-v1.xml">elife-03925-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tblu3" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Type of Vaccination/Supplementation</th><th colspan="4">Country</th></tr><tr><th>Burkina Faso</th><th>Mozambique</th><th>Rwanda</th><th>Senegal</th></tr></thead><tbody><tr><td><italic>BCG</italic></td><td>Birth</td><td>Birth</td><td>Birth</td><td>Birth</td></tr><tr><td><italic>DTP</italic></td><td>8,12,16 weeks* (DTP)</td><td>6,10,14 weeks (DTPHep)</td><td>6,10,14 weeks (DTPHibHep)</td><td>6,10,14 weeks (DTPHibHep)</td></tr><tr><td><italic>Measles</italic></td><td>9 months</td><td>9 months</td><td>9 months</td><td>9 months</td></tr><tr><td><italic>Polio</italic></td><td>Birth,8,12,16 weeks*</td><td>6,10,14 weeks</td><td>Birth, 6,10,14 weeks</td><td>Birth, 6,10,14 weeks</td></tr><tr><td><italic>Vitamin A</italic></td><td>Every 6 months from ages 6-59 months <italic>Main Distribution Mechanism</italic>:Integrated Child Health campaign</td><td>Every 6 months from ages 6-59 months <italic>Main Distribution Mechanism</italic>:Integrated Child Health campaign</td><td>Every 6 months from ages 6-59 months <italic>Main Distribution Mechanism</italic>:Integrated Child Health campaign</td><td>Every 6 months from ages 6-59 months <italic>Main Distribution Mechanism</italic>:Integrated Child Health campaign</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04006-v2.xml">elife-04006-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.04006.004</object-id><label>Table 1.</label><caption><p>List of all n = 27 in vivo juxtacellularly filled PV interneurons and their identification based on immunocytochemistry and electron microscopy (EM)</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04006.004">http://dx.doi.org/10.7554/eLife.04006.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Cell ID#</th><th rowspan="2">Cell type</th><th rowspan="2">Parvalbumin</th><th rowspan="2">Somatostatin</th><th rowspan="2">mGluR1a</th><th rowspan="2">SATB1</th><th colspan="2">Bouton targets</th></tr><tr><th>AnkyrinG verification</th><th>EM verification</th></tr></thead><tbody><tr><td>123011c3</td><td>C-BC</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>Soma/proximal dendrite</td></tr><tr><td>cvi30</td><td>C-BC</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>Soma/proximal dendrite</td></tr><tr><td>cvi33</td><td>C-BC</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>cvi35</td><td>C-BC</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>cvi55</td><td>C-BC</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>Soma/proximal dendrite</td></tr><tr><td>cvi65</td><td>C-BC</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>cvi75</td><td>C-BC</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>cvi151</td><td>C-BC</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>cvi251</td><td>C-BC</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>cvi240b</td><td>H-BC</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>Soma/proximal dendrite</td></tr><tr><td>042911c5</td><td>H-BC</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>gs0920</td><td>H-BC</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>gs012913</td><td>O-Bistrat</td><td>+</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>gs022713</td><td>O-Bistrat</td><td>+</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>cvi017</td><td>O-Bistrat</td><td>n.t.</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>cvi255</td><td>O-Bistrat</td><td>&#8722;</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>cvi312</td><td>O-Bistrat</td><td>n.t.</td><td>+</td><td>&#8722;</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>cvi270</td><td>C-Bistrat</td><td>+</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>imi069</td><td>C-Bistrat</td><td>n.t.</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>cvi190</td><td>C-Bistrat</td><td>&#8722;</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td></tr><tr><td>090311c3</td><td>E-AAC</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>AIS</td></tr><tr><td>cvi059</td><td>E-AAC</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>AIS</td></tr><tr><td>cvi153</td><td>C-AAC</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>AIS</td></tr><tr><td>cvi258</td><td>C-AAC</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>AIS</td></tr><tr><td>cvi315</td><td>C-AAC</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>&#8722;</td><td>n.t.</td><td>n.t.</td></tr><tr><td>imi075</td><td>C-AAC</td><td>+</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>AIS</td><td>n.t</td></tr><tr><td>07082014cs6</td><td>C-AAC</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>n.t.</td><td>AIS</td><td>AIS</td></tr></tbody></table><table-wrap-foot><fn><p>Abbreviations: n.t.: not tested; AIS: axon initial segment.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04059-v3.xml">elife-04059-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.04059.006</object-id><label>Table 1.</label><caption><p>Descriptive statistics of the 16 electrophysiological parameters measured on substantia nigra pars compacta dopaminergic neurons across postnatal development</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04059.006">http://dx.doi.org/10.7554/eLife.04059.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Age</th><th colspan="3">ISI<sub>avg</sub> (s)</th><th colspan="3" rowspan="2">CV<sub>ISI</sub> (%)</th><th colspan="3" rowspan="2">&#964;<sub>m</sub> (ms)</th><th colspan="3" rowspan="2">R<sub>in</sub> (MOhm)</th><th colspan="3" rowspan="2">C<sub>m</sub> (pF)</th><th colspan="3" rowspan="2">Sag (mV)</th><th colspan="3" rowspan="2">Rebound (ms)</th><th colspan="3" rowspan="2">AP threshold (mV)</th></tr><tr><td>Mean</td><td>SD</td><td><italic>N</italic></td></tr></thead><tbody><tr><td>P2&#8211;P3</td><td>1.06</td><td>0.84</td><td><italic>9</italic></td><td>130.5</td><td>40.1</td><td><italic>9</italic></td><td>200.3</td><td>8.0</td><td><italic>5</italic></td><td>1642</td><td>420</td><td><italic>5</italic></td><td>129</td><td>34</td><td><italic>5</italic></td><td>28.2</td><td>4.0</td><td><italic>9</italic></td><td>465</td><td>134</td><td><italic>6</italic></td><td>&#8722;43.9</td><td>3.3</td><td><italic>9</italic></td></tr><tr><td>P5</td><td>1.36</td><td>1.60</td><td><italic>15</italic></td><td>57.6</td><td>22.6</td><td><italic>15</italic></td><td>86.9</td><td>13.7</td><td><italic>11</italic></td><td>642</td><td>163</td><td><italic>11</italic></td><td>146</td><td>52</td><td><italic>11</italic></td><td>29.4</td><td>5.7</td><td><italic>14</italic></td><td>339</td><td>188</td><td><italic>12</italic></td><td>&#8722;40.3</td><td>4.5</td><td><italic>15</italic></td></tr><tr><td>P6</td><td>0.78</td><td>0.56</td><td><italic>18</italic></td><td>51.5</td><td>43.0</td><td><italic>18</italic></td><td>70.3</td><td>23.9</td><td><italic>18</italic></td><td>612</td><td>170</td><td><italic>18</italic></td><td>118</td><td>32</td><td><italic>18</italic></td><td>30.2</td><td>7.2</td><td><italic>18</italic></td><td>316</td><td>127</td><td><italic>18</italic></td><td>&#8722;43.1</td><td>4.3</td><td><italic>19</italic></td></tr><tr><td>P7</td><td>0.97</td><td>0.93</td><td><italic>36</italic></td><td>38.4</td><td>28.0</td><td><italic>36</italic></td><td>89.3</td><td>24.4</td><td><italic>25</italic></td><td>650</td><td>174</td><td><italic>26</italic></td><td>143</td><td>38</td><td><italic>25</italic></td><td>31.2</td><td>5.2</td><td><italic>25</italic></td><td>249</td><td>89</td><td><italic>19</italic></td><td>&#8722;39.8</td><td>4.0</td><td><italic>35</italic></td></tr><tr><td>P8&#8211;P9</td><td>1.67</td><td>1.95</td><td><italic>18</italic></td><td>38.0</td><td>23.2</td><td><italic>18</italic></td><td>90.9</td><td>38.4</td><td><italic>21</italic></td><td>576</td><td>268</td><td><italic>21</italic></td><td>200</td><td>141</td><td><italic>21</italic></td><td>29.6</td><td>6.2</td><td><italic>17</italic></td><td>401</td><td>212</td><td><italic>15</italic></td><td>&#8722;42.8</td><td>4.3</td><td><italic>20</italic></td></tr><tr><td>P10&#8211;P11</td><td>0.68</td><td>0.41</td><td><italic>12</italic></td><td>21.1</td><td>8.6</td><td><italic>12</italic></td><td>80.6</td><td>28.6</td><td><italic>10</italic></td><td>327</td><td>210</td><td><italic>10</italic></td><td>281</td><td>109</td><td><italic>10</italic></td><td>33.3</td><td>4.2</td><td><italic>10</italic></td><td>330</td><td>165</td><td><italic>10</italic></td><td>&#8722;41.7</td><td>4.2</td><td><italic>11</italic></td></tr><tr><td>P12&#8211;P13</td><td>1.04</td><td>0.67</td><td><italic>15</italic></td><td>19.6</td><td>15.1</td><td><italic>15</italic></td><td>87.9</td><td>32.7</td><td><italic>17</italic></td><td>369</td><td>166</td><td><italic>17</italic></td><td>297</td><td>226</td><td><italic>17</italic></td><td>32.4</td><td>3.7</td><td><italic>16</italic></td><td>378</td><td>341</td><td><italic>16</italic></td><td>&#8722;45.4</td><td>3.7</td><td><italic>18</italic></td></tr><tr><td>P14</td><td>0.64</td><td>0.48</td><td><italic>16</italic></td><td>12.8</td><td>8.3</td><td><italic>16</italic></td><td>92.3</td><td>44.0</td><td><italic>14</italic></td><td>460</td><td>192</td><td><italic>14</italic></td><td>205</td><td>68</td><td><italic>14</italic></td><td>35.6</td><td>2.9</td><td><italic>10</italic></td><td>489</td><td>448</td><td><italic>9</italic></td><td>&#8722;44.2</td><td>2.3</td><td><italic>16</italic></td></tr><tr><td>P15&#8211;P16</td><td>0.88</td><td>0.38</td><td><italic>22</italic></td><td>13.3</td><td>6.6</td><td><italic>22</italic></td><td>77.8</td><td>28.5</td><td><italic>19</italic></td><td>325</td><td>117</td><td><italic>19</italic></td><td>245</td><td>67</td><td><italic>19</italic></td><td>27.9</td><td>6.5</td><td><italic>33</italic></td><td>340</td><td>207</td><td><italic>30</italic></td><td>&#8722;44.4</td><td>3.4</td><td><italic>30</italic></td></tr><tr><td>P17</td><td>0.82</td><td>0.40</td><td><italic>16</italic></td><td>9.9</td><td>5.6</td><td><italic>16</italic></td><td>83.5</td><td>25.7</td><td><italic>20</italic></td><td>323</td><td>109</td><td><italic>22</italic></td><td>283</td><td>160</td><td><italic>20</italic></td><td>28.1</td><td>7.4</td><td><italic>35</italic></td><td>375</td><td>270</td><td><italic>26</italic></td><td>&#8722;44.6</td><td>3.5</td><td><italic>28</italic></td></tr><tr><td>P18</td><td>0.63</td><td>0.30</td><td><italic>18</italic></td><td>7.6</td><td>4.0</td><td><italic>18</italic></td><td>89.0</td><td>24.2</td><td><italic>18</italic></td><td>340</td><td>118</td><td><italic>18</italic></td><td>280</td><td>86</td><td><italic>18</italic></td><td>29.5</td><td>4.2</td><td><italic>22</italic></td><td>433</td><td>222</td><td><italic>20</italic></td><td>&#8722;41.6</td><td>3.0</td><td><italic>21</italic></td></tr><tr><td>P19</td><td>0.71</td><td>0.32</td><td><italic>24</italic></td><td>9.6</td><td>6.1</td><td><italic>24</italic></td><td>89.9</td><td>28.1</td><td><italic>30</italic></td><td>352</td><td>103</td><td><italic>30</italic></td><td>270</td><td>106</td><td><italic>30</italic></td><td>28.6</td><td>6.2</td><td><italic>38</italic></td><td>456</td><td>278</td><td><italic>32</italic></td><td>&#8722;44.2</td><td>4.1</td><td><italic>33</italic></td></tr><tr><td>P20</td><td>0.77</td><td>0.42</td><td><italic>21</italic></td><td>9.3</td><td>4.8</td><td><italic>21</italic></td><td>88.5</td><td>41.8</td><td><italic>22</italic></td><td>375</td><td>116</td><td><italic>23</italic></td><td>231</td><td>66</td><td><italic>22</italic></td><td>28.5</td><td>8.3</td><td><italic>24</italic></td><td>572</td><td>493</td><td><italic>22</italic></td><td>&#8722;44.3</td><td>4.4</td><td><italic>24</italic></td></tr><tr><td>P21&#8211;P23</td><td>0.72</td><td>0.32</td><td><italic>19</italic></td><td>7.5</td><td>4.5</td><td><italic>19</italic></td><td>93.1</td><td>48.2</td><td><italic>21</italic></td><td>319</td><td>115</td><td><italic>23</italic></td><td>293</td><td>128</td><td><italic>21</italic></td><td>27.5</td><td>6.6</td><td><italic>23</italic></td><td>388</td><td>254</td><td><italic>22</italic></td><td>&#8722;43.9</td><td>4.0</td><td><italic>24</italic></td></tr><tr><td>P28&#8211;P29</td><td>0.92</td><td>0.54</td><td><italic>12</italic></td><td>12.1</td><td>4.7</td><td><italic>12</italic></td><td>148.3</td><td>42.6</td><td><italic>10</italic></td><td>472</td><td>88</td><td><italic>10</italic></td><td>316</td><td>83</td><td><italic>10</italic></td><td>34.1</td><td>4.5</td><td><italic>13</italic></td><td>736</td><td>375</td><td><italic>10</italic></td><td>&#8722;42.8</td><td>3.0</td><td><italic>12</italic></td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th>Age</th><th colspan="3">AP amplitude (mV)</th><th colspan="3">AP half-width (ms)</th><th colspan="3">AP rise slope (mV/ms)</th><th colspan="3">AP decay slope (mV/ms)</th><th colspan="3">AHP (mV)</th><th colspan="3">Gain start (Hz/100pA)</th><th colspan="3">Gain end (Hz/100pA)</th><th colspan="3">SFA index</th></tr></thead><tbody><tr><td>P2&#8211;P3</td><td>59.5</td><td>9.6</td><td><italic>9</italic></td><td>2.38</td><td>0.62</td><td><italic>9</italic></td><td>47.6</td><td>21.7</td><td><italic>9</italic></td><td>&#8722;19.6</td><td>5.2</td><td><italic>9</italic></td><td>14.1</td><td>4.9</td><td><italic>9</italic></td><td>32.1</td><td>5.9</td><td><italic>6</italic></td><td>20</td><td>7.8</td><td><italic>6</italic></td><td>1.76</td><td>0.53</td><td><italic>6</italic></td></tr><tr><td>P5</td><td>47.1</td><td>7.4</td><td><italic>15</italic></td><td>2.99</td><td>0.95</td><td><italic>15</italic></td><td>24.5</td><td>10.8</td><td><italic>15</italic></td><td>&#8722;16.9</td><td>4.6</td><td><italic>15</italic></td><td>20.1</td><td>4.5</td><td><italic>15</italic></td><td>17.7</td><td>5</td><td><italic>7</italic></td><td>8.8</td><td>2.1</td><td><italic>7</italic></td><td>2.07</td><td>0.71</td><td><italic>7</italic></td></tr><tr><td>P6</td><td>49.0</td><td>9.2</td><td><italic>19</italic></td><td>2.99</td><td>0.71</td><td><italic>19</italic></td><td>25.1</td><td>19.2</td><td><italic>19</italic></td><td>&#8722;18.0</td><td>6.6</td><td><italic>19</italic></td><td>21.0</td><td>3.7</td><td><italic>19</italic></td><td>22.7</td><td>6.5</td><td><italic>10</italic></td><td>11.7</td><td>3.7</td><td><italic>10</italic></td><td>1.97</td><td>0.27</td><td><italic>10</italic></td></tr><tr><td>P7</td><td>46.3</td><td>7.8</td><td><italic>35</italic></td><td>2.92</td><td>0.64</td><td><italic>35</italic></td><td>23.8</td><td>8.6</td><td><italic>35</italic></td><td>&#8722;17.6</td><td>4.1</td><td><italic>35</italic></td><td>22.6</td><td>3.7</td><td><italic>35</italic></td><td>17.5</td><td>4.7</td><td><italic>13</italic></td><td>8.83</td><td>2.4</td><td><italic>13</italic></td><td>2.08</td><td>0.71</td><td><italic>13</italic></td></tr><tr><td>P8&#8211;P9</td><td>54.4</td><td>7.4</td><td><italic>20</italic></td><td>2.74</td><td>0.82</td><td><italic>20</italic></td><td>32.8</td><td>14.8</td><td><italic>20</italic></td><td>&#8722;20.2</td><td>5.9</td><td><italic>20</italic></td><td>20.9</td><td>4.2</td><td><italic>20</italic></td><td>18.1</td><td>4.3</td><td><italic>13</italic></td><td>10.4</td><td>3.3</td><td><italic>13</italic></td><td>1.80</td><td>0.31</td><td><italic>13</italic></td></tr><tr><td>P10&#8211;P11</td><td>50.6</td><td>11.6</td><td><italic>11</italic></td><td>2.51</td><td>0.83</td><td><italic>11</italic></td><td>36.3</td><td>21.2</td><td><italic>11</italic></td><td>&#8722;21.4</td><td>7.5</td><td><italic>11</italic></td><td>20.7</td><td>4.9</td><td><italic>11</italic></td><td>11</td><td>4.5</td><td><italic>10</italic></td><td>11.5</td><td>5.3</td><td><italic>10</italic></td><td>1.01</td><td>0.30</td><td><italic>10</italic></td></tr><tr><td>P12&#8211;P13</td><td>62.1</td><td>6.3</td><td><italic>18</italic></td><td>1.86</td><td>0.27</td><td><italic>18</italic></td><td>54.2</td><td>14.5</td><td><italic>18</italic></td><td>&#8722;29.7</td><td>4.2</td><td><italic>18</italic></td><td>23.0</td><td>6.0</td><td><italic>18</italic></td><td>13.2</td><td>5</td><td><italic>15</italic></td><td>11.6</td><td>4.5</td><td><italic>15</italic></td><td>1.29</td><td>0.80</td><td><italic>15</italic></td></tr><tr><td>P14</td><td>61.0</td><td>7.1</td><td><italic>16</italic></td><td>1.91</td><td>0.54</td><td><italic>16</italic></td><td>54.9</td><td>18.4</td><td><italic>16</italic></td><td>&#8722;30.9</td><td>8.5</td><td><italic>16</italic></td><td>24.1</td><td>5.5</td><td><italic>16</italic></td><td>12.2</td><td>4</td><td><italic>10</italic></td><td>9.8</td><td>2.6</td><td><italic>10</italic></td><td>1.30</td><td>0.50</td><td><italic>10</italic></td></tr><tr><td>P15&#8211;P16</td><td>60.1</td><td>6.5</td><td><italic>30</italic></td><td>1.63</td><td>0.49</td><td><italic>30</italic></td><td>56.2</td><td>18.3</td><td><italic>30</italic></td><td>&#8722;39.7</td><td>11.5</td><td><italic>30</italic></td><td>26.1</td><td>4.0</td><td><italic>30</italic></td><td>9.28</td><td>3.2</td><td><italic>26</italic></td><td>9.28</td><td>4.2</td><td><italic>26</italic></td><td>1.11</td><td>0.44</td><td><italic>26</italic></td></tr><tr><td>P17</td><td>62.1</td><td>8.6</td><td><italic>28</italic></td><td>1.61</td><td>0.37</td><td><italic>28</italic></td><td>58.2</td><td>16.2</td><td><italic>28</italic></td><td>&#8722;38.8</td><td>8.8</td><td><italic>28</italic></td><td>28.1</td><td>5.0</td><td><italic>28</italic></td><td>7.13</td><td>2.1</td><td><italic>14</italic></td><td>8.28</td><td>2.9</td><td><italic>14</italic></td><td>0.92</td><td>0.31</td><td><italic>14</italic></td></tr><tr><td>P18</td><td>55.5</td><td>4.7</td><td><italic>21</italic></td><td>1.45</td><td>0.21</td><td><italic>21</italic></td><td>50.6</td><td>15.6</td><td><italic>21</italic></td><td>&#8722;41.1</td><td>7.3</td><td><italic>21</italic></td><td>28.4</td><td>3.2</td><td><italic>21</italic></td><td>6.97</td><td>2</td><td><italic>12</italic></td><td>7.59</td><td>2.9</td><td><italic>12</italic></td><td>0.99</td><td>0.35</td><td><italic>12</italic></td></tr><tr><td>P19</td><td>58.2</td><td>8.8</td><td><italic>33</italic></td><td>1.55</td><td>0.38</td><td><italic>33</italic></td><td>54.3</td><td>21.1</td><td><italic>33</italic></td><td>&#8722;41.4</td><td>11.4</td><td><italic>33</italic></td><td>27.4</td><td>4.6</td><td><italic>33</italic></td><td>8.32</td><td>2.6</td><td><italic>19</italic></td><td>9.3</td><td>2.6</td><td><italic>19</italic></td><td>0.93</td><td>0.31</td><td><italic>19</italic></td></tr><tr><td>P20</td><td>61.6</td><td>8.8</td><td><italic>24</italic></td><td>1.47</td><td>0.31</td><td><italic>24</italic></td><td>59.3</td><td>21.2</td><td><italic>24</italic></td><td>&#8722;42.6</td><td>8.8</td><td><italic>24</italic></td><td>28.0</td><td>5.0</td><td><italic>24</italic></td><td>8.15</td><td>2.4</td><td><italic>16</italic></td><td>9.21</td><td>3</td><td><italic>16</italic></td><td>0.95</td><td>0.33</td><td><italic>16</italic></td></tr><tr><td>P21&#8211;P23</td><td>62.6</td><td>8.2</td><td><italic>24</italic></td><td>1.50</td><td>0.37</td><td><italic>24</italic></td><td>61.5</td><td>21.0</td><td><italic>24</italic></td><td>&#8722;44.1</td><td>11.4</td><td><italic>24</italic></td><td>29.1</td><td>4.3</td><td><italic>24</italic></td><td>6.79</td><td>2.1</td><td><italic>24</italic></td><td>7</td><td>1.8</td><td><italic>24</italic></td><td>1.01</td><td>0.32</td><td><italic>24</italic></td></tr><tr><td>P28&#8211;P29</td><td>62.4</td><td>5.3</td><td><italic>12</italic></td><td>1.49</td><td>0.29</td><td><italic>12</italic></td><td>63.0</td><td>14.3</td><td><italic>12</italic></td><td>&#8722;43.2</td><td>8.7</td><td><italic>12</italic></td><td>28.6</td><td>6.2</td><td><italic>12</italic></td><td>6.99</td><td>2.3</td><td><italic>10</italic></td><td>6.82</td><td>1.4</td><td><italic>10</italic></td><td>1.07</td><td>0.43</td><td><italic>10</italic></td></tr></tbody></table><table-wrap-foot><fn><p>Abbreviations: ISI<sub>avg</sub>, averaged interspike interval; CV<sub>ISI</sub>, coefficient of variation of the interspike interval; &#964;<sub>m</sub>, membrane time constant; R<sub>in</sub>, input resistance; C<sub>m</sub>, membrane capacitance; Rebound, rebound delay; AP, action potential; AHP, after hyperpolarization; SFA, spike frequency adaptation.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04070-v3.xml">elife-04070-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.04070.005</object-id><label>Table 1.</label><caption><p>Classification of mutant phenotypes</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04070.005">http://dx.doi.org/10.7554/eLife.04070.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th rowspan="2">Mutagen used</th><th colspan="2">Observed rosette induction<xref ref-type="table-fn" rid="tblfn1">*</xref></th><th colspan="2">Other phenotypes</th></tr><tr><th>ACM</th><th>Live bacteria</th><th>Swimming<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></th><th>Chain morphology</th></tr></thead><tbody><tr><td>Wild type</td><td>N/A</td><td>86%</td><td>88%</td><td>Wild type</td><td>Primarily linear</td></tr><tr><td colspan="6">Mutant class A</td></tr><tr><td>&#8195;Rosetteless</td><td>EMS</td><td>0</td><td>0</td><td>Wild type</td><td>Primarily linear</td></tr><tr><td colspan="6">Mutant class B</td></tr><tr><td>&#8195;Insensate</td><td>X-rays</td><td>0</td><td>5</td><td>Wild type</td><td>Primarily linear</td></tr><tr><td colspan="6">Mutant class C</td></tr><tr><td>&#8195;Slacker</td><td>X-rays</td><td>20</td><td>42</td><td>Wild type</td><td>Primarily linear</td></tr><tr><td colspan="6">Mutant class D</td></tr><tr><td>&#8195;Uptight</td><td>X-rays</td><td>33</td><td>56</td><td>Wild type</td><td>Branched</td></tr><tr><td colspan="6">Mutant class E</td></tr><tr><td>&#8195;Jumble</td><td>EMS</td><td>0</td><td>0</td><td>Wild type</td><td>Branched</td></tr><tr><td>&#8195;Branched</td><td>X-rays</td><td>0</td><td>0</td><td>Wild type</td><td>Branched</td></tr><tr><td colspan="6">Mutant class F</td></tr><tr><td>&#8195;Seafoam</td><td>X-rays</td><td>0</td><td>0</td><td>Wild type</td><td>Large clusters</td></tr><tr><td>&#8195;Soapsuds</td><td>X-rays</td><td>0</td><td>0</td><td>Wild type</td><td>Large clusters</td></tr><tr><td colspan="6">Mutant class G</td></tr><tr><td>&#8195;Solo</td><td>X-rays</td><td>0</td><td>0</td><td>Slow, shaking</td><td>Primarily solitary</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>The percentage of cells in rosettes following induction.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>Swimming phenotypes of single cells.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04114-v2.xml">elife-04114-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.04114.018</object-id><label>Table 1.</label><caption><p>Cytoplasmic and nuclear subviral complexes in MDM</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04114.018">http://dx.doi.org/10.7554/eLife.04114.018</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="3">Cytoplasm</th><th colspan="3">Nucleus</th><th rowspan="2">Proportion of nuclear complexes [%]</th><th rowspan="2">Number of cells imaged</th></tr><tr><th>RTC/PIC (n)</th><th>CA-positive RTC/PIC (n)</th><th>CA-positive RTC/PIC [%]</th><th>PIC (n)</th><th>CA-positive PIC (n)</th><th>CA-positive PIC [%]</th></tr></thead><tbody><tr><td><bold>24 hr</bold></td><td>90</td><td>46</td><td><bold>51</bold></td><td>9</td><td>9</td><td><bold>100</bold></td><td>9</td><td>367</td></tr><tr><td><bold>48 hr</bold></td><td>45</td><td>13</td><td><bold>29</bold></td><td>61</td><td>59</td><td><bold>97</bold></td><td>58</td><td>200</td></tr><tr><td><bold>Total</bold></td><td>135</td><td>59</td><td><bold>44</bold></td><td>70</td><td>68</td><td><bold>97</bold></td><td>34</td><td>567</td></tr></tbody></table><table-wrap-foot><fn><p>The table summarizes data from primary MDM infected with NL4-3-R5(IN.eGFP) or NL4-3-4059(IN.eGFP) for 24 hr or 48 hr, respectively. Infected MDM were immunostained with antibodies against nuclear pore complexes and CA. Data were obtained as outlined in <xref ref-type="fig" rid="fig7">Figure 7</xref> in cells from eight different donors in five independent experiments (cells from three donors were infected for both 24 hr and 48 hr).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04251-v1.xml">elife-04251-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.04251.010</object-id><label>Table 1.</label><caption><p>Key descriptors of FG particle constitution</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04251.010">http://dx.doi.org/10.7554/eLife.04251.010</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th rowspan="2">Molecular weight<xref ref-type="table-fn" rid="tblfn1">*</xref></th><th rowspan="2">Number of residues<xref ref-type="table-fn" rid="tblfn1">*</xref></th><th rowspan="2">Number of FG motifs<xref ref-type="table-fn" rid="tblfn1">*</xref></th><th colspan="3">Estimated critical concentration<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></th><th rowspan="2">Estimated intra-particle concentration<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></th><th rowspan="2">NQ-content<xref ref-type="table-fn" rid="tblfn1">*</xref></th><th rowspan="2">Relative Thioflavin-T signal<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></th></tr><tr><th colspan="2">FG domain</th><th>FG motifs</th></tr></thead><tbody><tr><td><bold><italic>Homo sapiens</italic></bold> HsNup98</td><td>&#8776;49 kDa</td><td>500</td><td>39</td><td>&#8776;25 nM</td><td>&#8776;1 &#181;g/ml</td><td>&#8776;1 &#181;M</td><td>&#8776;175 mg/ml</td><td>11%</td><td>11%</td></tr><tr><td><bold><italic>Branchiostoma floridae</italic></bold> BfNup98</td><td>&#8776;46 kDa</td><td>479</td><td>40</td><td>&#8776;20 nM</td><td>&#8776;1 &#181;g/ml</td><td>&#8776;1 &#181;M</td><td>&#8776;200 mg/ml</td><td>8%</td><td>8%</td></tr><tr><td><bold><italic>Branchiostoma floridae</italic></bold> BfNup98 + GlcNAc</td><td>&#8776;55 kDa</td><td>479</td><td>40</td><td>&#8776;150 nM</td><td>&#8776;10 &#181;g/ml</td><td>&#8776;6 &#181;M</td><td>ND</td><td>8%</td><td>ND<xref ref-type="table-fn" rid="tblfn5">#</xref></td></tr><tr><td><bold><italic>Drosophila melongaster</italic></bold> DmNup98</td><td>&#8776;56 kDa</td><td>581</td><td>46</td><td>&#8776;200 nM</td><td>&#8776;10 &#181;g/ml</td><td>&#8776;9 &#181;M</td><td>&#8776;300 mg/ml</td><td>10%</td><td>3%</td></tr><tr><td><bold><italic>Caenorhabditis elegans</italic></bold> CeNup98</td><td>&#8776;48 kDa</td><td>494</td><td>36</td><td>&#8776;700 nM</td><td>&#8776;40 &#181;g/ml</td><td>&#8776;25 &#181;M</td><td>&#8776;300 mg/ml</td><td>18%</td><td>52%</td></tr><tr><td><bold><italic>Saccharomyces cerevisiae</italic></bold> ScNup100</td><td>&#8776;58 kDa</td><td>578</td><td>43</td><td>&#8776;175 nM</td><td>&#8776;10 &#181;g/ml</td><td>&#8776;7.5 &#181;M</td><td>&#8776;275 mg/ml</td><td>28%</td><td>100%</td></tr><tr><td><bold><italic>Saccharomyces cerevisiae</italic></bold> ScNup116</td><td>&#8776;65 kDa</td><td>737</td><td>47</td><td>&#8776;700 nM</td><td>&#8776;50 &#181;g/ml</td><td>&#8776;33 &#181;M</td><td>&#8776;350 mg/ml</td><td>26%</td><td>78%</td></tr><tr><td><bold><italic>Dictyostelium discoideum</italic></bold> DdNup220</td><td>&#8776;68 kDa</td><td>719</td><td>56</td><td>&#8776;125 nM</td><td>&#8776;10 &#181;g/ml</td><td>&#8776;7 &#181;M</td><td>&#8776;300 mg/ml</td><td>12%</td><td>4%</td></tr><tr><td><bold><italic>Arabidopsis thaliana</italic></bold> AtNup98B</td><td>&#8776;66 kDa</td><td>668</td><td>52</td><td>&#8776;25 nM</td><td>&#8776;1 &#181;g/ml</td><td>&#8776;1.5 &#181;M</td><td>&#8776;200 mg/ml</td><td>13%</td><td>1%</td></tr><tr><td><bold><italic>Tetrahymena thermophila</italic></bold> TtMacNup98A</td><td>&#8776;61 kDa</td><td>666</td><td>42</td><td>&#8776;25 nM</td><td>&#8776;1 &#181;g/ml</td><td>&#8776;1 &#181;M</td><td>&#8776;175 mg/ml</td><td>18%</td><td>2%</td></tr><tr><td><bold><italic>Trypanosoma brucei</italic></bold> TbNup158</td><td>&#8776;50 kDa</td><td>565</td><td>58</td><td>&#8776;300 nM</td><td>&#8776;15 &#181;g/ml</td><td>&#8776;17.5 &#181;M</td><td>&#8776;250 mg/ml</td><td>12%</td><td>2%</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>All values are given for full-length FG domains (including GLEBS domains). The molecular weight of glycosylated BfNup98 was estimated by SDS-PAGE.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>The critical concentrations for phase separation were estimated as described in <xref ref-type="fig" rid="fig1">Figure 1C</xref> and the methods section.</p></fn><fn id="tblfn3"><label>&#8225;</label><p>Intra-particle FG domain concentrations were estimated as described in <xref ref-type="fig" rid="fig1">Figure 1F</xref> and the methods section.</p></fn><fn id="tblfn4"><label>&#167;</label><p>Thioflavin-T signals are normalised to the ScNup100 Thioflavin-T signal; also see <xref ref-type="fig" rid="fig6">Figure 6</xref>.</p></fn><fn id="tblfn5"><label>#</label><p>Experiments with macroscopic hydrogels suggest that the Thioflavin-T signal of glycosylated BfNup98 FG particles is even lower than the observed signal for the non-glycosylated BfNup98 FG particles.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04390-v2.xml">elife-04390-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.04390.006</object-id><label>Table 1.</label><caption><p>Quantification of dendritic spine density in the hippocampus of <italic>Thy1-eGFP</italic> WT, <italic>Sema5</italic> and <italic>Plxna</italic> mice</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04390.006">http://dx.doi.org/10.7554/eLife.04390.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Genotype</th><th rowspan="2">Age</th><th colspan="4">Dentate GC dendrite</th><th colspan="4">CA1 primary dendrite</th><th colspan="4">CA1 secondary dendrite</th></tr><tr><th>N</th><th>p Value (vs wt)</th><th>Spine density (mean &#177; SEM)</th><th>Distance from soma</th><th>n</th><th>p Value (vs wt)</th><th>Spine density (mean &#177; SEM)</th><th>Distance from soma</th><th>n</th><th>p Value (vs wt)</th><th>Spine density (mean &#177; SEM)</th><th>Distance from soma</th></tr></thead><tbody><tr><td><bold><italic>WT</italic></bold></td><td>P30&#8211;33</td><td>4 animals (74 neurons)</td><td/><td>1.67 &#177; 0.01</td><td>50&#8211;100 &#956;m</td><td>3 animals (26 neurons)</td><td/><td>2.76 &#177; 0.07</td><td>50&#8211;100 &#956;m</td><td>3 animals (15 neurons)</td><td/><td>2.93 &#177; 0.02</td><td>50&#8211;100 &#956;m</td></tr><tr><td><bold><italic>Sema5A</italic><sup><italic>+/&#8722;</italic></sup><italic>; Sema5B</italic><sup><italic>+/&#8722;</italic></sup></bold></td><td>P30&#8211;33</td><td>3 animals (41 neurons)</td><td>0.8795</td><td>1.68 &#177; 0.05</td><td>50&#8211;100 &#956;m</td><td>3 animals (24 neurons)</td><td>0.6549</td><td>2.79 &#177; 0.019</td><td>50&#8211;100 &#956;m</td><td>3 animals (24 neurons)</td><td>0.6333</td><td>2.99 &#177; 0.09</td><td>50&#8211;100 &#956;m</td></tr><tr><td><bold><italic>Sema5A</italic><sup><italic>&#8722;/&#8722;</italic></sup><italic>; Sema5B</italic><sup><italic>+/&#8722;</italic></sup></bold></td><td>P30&#8211;33</td><td>4 animals (65 neurons)</td><td><bold>0.0012</bold></td><td>2.05 &#177; 0.05</td><td>50&#8211;100 &#956;m</td><td>3 animals (27 neurons)</td><td>0.5122</td><td>2.81 &#177; 0.03</td><td>50&#8211;100 &#956;m</td><td>3 animals (21 neurons)</td><td>0.618</td><td>2.99 &#177; 0.09</td><td>50&#8211;100 &#956;m</td></tr><tr><td><bold><italic>Sema5A</italic><sup><italic>+/&#8722;</italic></sup><italic>; Sema5B</italic><sup><italic>&#8722;/&#8722;</italic></sup></bold></td><td>P30&#8211;33</td><td>3 animals (46 neurons)</td><td>0.8844</td><td>1.66 &#177; 0.05</td><td>50&#8211;100 &#956;m</td><td>3 animals (26 neurons)</td><td>0.9797</td><td>2.76 &#177; 0.15</td><td>50&#8211;100 &#956;m</td><td>3 animals (21 neurons)</td><td>0.9661</td><td>2.94 &#177; 0.02</td><td>50&#8211;100 &#956;m</td></tr><tr><td><bold><italic>Sema5A</italic><sup><italic>&#8722;/&#8722;</italic></sup><italic>; Sema5B</italic><sup><italic>&#8722;/&#8722;</italic></sup></bold></td><td>P30&#8211;33</td><td>3 animals (30 neurons)</td><td><bold>0.0066</bold></td><td>2.06 &#177; 0.08</td><td>50&#8211;100 &#956;m</td><td>3 animals (27 neurons)</td><td>0.5389</td><td>2.81 &#177; 0.02</td><td>50&#8211;100 &#956;m</td><td>3 animals (23 neurons)</td><td>0.4006</td><td>2.98 &#177; 0.03</td><td>50&#8211;100 &#956;m</td></tr><tr><td><bold><italic>Plxna1</italic><sup><italic>&#8722;/&#8722;</italic></sup></bold></td><td>P30&#8211;33</td><td>3 animals (56 neurons)</td><td>0.154</td><td>1.79 &#177; 0.05</td><td>50&#8211;100 &#956;m</td><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td><bold><italic>Plxna2</italic><sup><italic>&#8722;/&#8722;</italic></sup></bold></td><td>P30&#8211;33</td><td>3 animals (54 neurons)</td><td><bold>&lt;0.0001</bold></td><td>2.24 &#177; 0.07</td><td>50&#8211;100 &#956;m</td><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td><bold><italic>Plxna2</italic><sup><italic>+/&#8722;</italic></sup></bold></td><td>P30&#8211;33</td><td>4 animals (74 neurons)</td><td>0.0359</td><td>1.86 &#177; 0.05</td><td>50&#8211;100 &#956;m</td><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td><bold><italic>Sema5A</italic><sup><italic>+/&#8722;</italic></sup><italic>; Plxna2</italic><sup><italic>+/&#8722;</italic></sup></bold></td><td>P30&#8211;33</td><td>4 animals (72 neurons)</td><td><bold>&lt;0.0001</bold></td><td>2.39 &#177; 0.05</td><td>50&#8211;100 &#956;m</td><td/><td/><td/><td/><td/><td/><td/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04490-v1.xml">elife-04490-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.04490.029</object-id><label>Appendix Table 1.</label><caption><p>Volatiles (percent IS plant<sup>&#8722;1</sup>, mean &#177; SEM) trapped in the headspace around single plants in experimental season one (June 8th&#8211;9th).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04490.029">http://dx.doi.org/10.7554/eLife.04490.029</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3" rowspan="2">&#8201;</th><th align="center" colspan="8">GLVs (percent IS plant<sup>-1</sup>)</th><th align="center" colspan="17">TPS10 products (percent IS plant<sup>-1</sup>)</th><th align="center" colspan="10">Non-target volatiles (percent IS plant<sup>-1</sup>)</th></tr><tr><th align="center" colspan="8">(<italic>Z</italic>)-Hexen-3-ol</th><th align="center" colspan="10">TAB</th><th align="center" colspan="7">TBF</th><th align="center" colspan="6">&#945;-Duprezianene</th><th align="center" colspan="4">Germacrene A</th></tr></thead><tbody><tr><td><bold>Genotype</bold></td><td><bold>Day n</bold></td><td><bold>Night n</bold></td><td colspan="3" style="background:silver"><bold>Day</bold></td><td colspan="5" style="background:#8C8C8C"><bold>Night</bold></td><td colspan="4" style="background:silver"><bold>Day</bold></td><td colspan="6" style="background:#8C8C8C"><bold>Night</bold></td><td colspan="4" style="background:silver"><bold>Day</bold></td><td colspan="3" style="background:#8C8C8C"><bold>Night</bold></td><td colspan="3" style="background:silver"><bold>Day</bold></td><td colspan="3" style="background:#8C8C8C"><bold>Night</bold></td><td colspan="3" style="background:silver"><bold>Day</bold></td><td style="background:#8C8C8C"><bold>Night</bold></td></tr><tr><td><bold>WT</bold></td><td>8</td><td>8</td><td style="background:silver">0.99%</td><td style="background:silver">&#177;</td><td style="background:silver">0.99%</td><td style="background:#8C8C8C">7.96%</td><td style="background:#8C8C8C">&#177;</td><td style="background:#8C8C8C">4.25%</td><td rowspan="2" style="background:#8C8C8C"><named-content content-type="author-callout-style1">}</named-content></td><td rowspan="2" style="background:#8C8C8C"><bold><named-content content-type="author-callout-style1">a</named-content></bold><sub><bold><named-content content-type="author-callout-style1">lox</named-content></bold></sub></td><td style="background:silver">0.37%</td><td style="background:silver">&#177;</td><td style="background:silver">0.29%</td><td style="background:silver"><bold><named-content content-type="author-callout-style1">a</named-content></bold></td><td colspan="3" style="background:#8C8C8C">&#8212;</td><td style="background:#8C8C8C">&#8201;</td><td rowspan="3" style="background:#8C8C8C"><named-content content-type="author-callout-style1">}</named-content></td><td rowspan="2" style="background:#8C8C8C"><bold><named-content content-type="author-callout-style1">a</named-content></bold><sub><bold><named-content content-type="author-callout-style1">TPS</named-content></bold></sub></td><td colspan="3" style="background:silver">&#8212;</td><td style="background:silver"><bold><named-content content-type="author-callout-style1">a</named-content></bold></td><td colspan="3" style="background:#8C8C8C">&#8212;</td><td style="background:silver">2.13%</td><td style="background:silver">&#177;</td><td style="background:silver">0.85%</td><td style="background:#8C8C8C">0.96%</td><td style="background:#8C8C8C">&#177;</td><td style="background:#8C8C8C">0.42%</td><td style="background:silver">1.68%</td><td style="background:silver">&#177;</td><td style="background:silver">0.98%</td><td style="background:#8C8C8C">&#8212;</td></tr><tr><td><bold><italic>TPS10</italic></bold></td><td>7</td><td>7</td><td style="background:silver">2.37%</td><td style="background:silver">&#177;</td><td style="background:silver">1.55%</td><td style="background:#8C8C8C">2.84%</td><td style="background:#8C8C8C">&#177;</td><td style="background:#8C8C8C">0.63%</td><td style="background:silver">18.97%</td><td style="background:silver">&#177;</td><td style="background:silver">6.03%</td><td style="background:silver"><bold><named-content content-type="author-callout-style2">b</named-content></bold></td><td style="background:#8C8C8C">0.94%</td><td style="background:#8C8C8C">&#177;</td><td style="background:#8C8C8C">0.53%</td><td rowspan="3" style="background:#8C8C8C"><named-content content-type="author-callout-style2">}</named-content></td><td style="background:silver">9.34%</td><td style="background:silver">&#177;</td><td style="background:silver">3.44%</td><td style="background:silver"><bold><named-content content-type="author-callout-style2">b</named-content></bold></td><td style="background:#8C8C8C">0.20%</td><td style="background:#8C8C8C">&#177;</td><td style="background:#8C8C8C">0.20%</td><td style="background:silver">9.47%</td><td style="background:silver">&#177;</td><td style="background:silver">5.26%</td><td style="background:#8C8C8C">0.94%</td><td style="background:#8C8C8C">&#177;</td><td style="background:#8C8C8C">0.26%</td><td style="background:silver">2.78%</td><td style="background:silver">&#177;</td><td style="background:silver">1.52%</td><td style="background:#8C8C8C">&#8212;</td></tr><tr><td><bold><italic>lox2/3</italic></bold></td><td>7</td><td>8</td><td style="background:silver">0.13%</td><td style="background:silver">&#177;</td><td style="background:silver">0.13%</td><td style="background:#8C8C8C">1.06%</td><td style="background:#8C8C8C">&#177;</td><td style="background:#8C8C8C">0.64%</td><td rowspan="2" style="background:#8C8C8C"><named-content content-type="author-callout-style2">}</named-content></td><td rowspan="2" style="background:#8C8C8C"><bold><named-content content-type="author-callout-style2">b</named-content></bold><sub><bold><named-content content-type="author-callout-style2">lox</named-content></bold></sub></td><td colspan="3" style="background:silver">&#8212;</td><td style="background:silver"><bold><named-content content-type="author-callout-style1">a</named-content></bold></td><td style="background:#8C8C8C">0.15%</td><td style="background:#8C8C8C">&#177;</td><td style="background:#8C8C8C">0.15%</td><td rowspan="2" style="background:#8C8C8C"><bold><named-content content-type="author-callout-style2">b</named-content></bold><sub><bold><named-content content-type="author-callout-style2">TPS</named-content></bold></sub></td><td colspan="3" style="background:silver">&#8212;</td><td style="background:silver"><bold><named-content content-type="author-callout-style1">a</named-content></bold></td><td colspan="3" style="background:#8C8C8C">&#8212;</td><td style="background:silver">2.75%</td><td style="background:silver">&#177;</td><td style="background:silver">1.12%</td><td style="background:#8C8C8C">0.60%</td><td style="background:#8C8C8C">&#177;</td><td style="background:#8C8C8C">0.19%</td><td style="background:silver">1.68%</td><td style="background:silver">&#177;</td><td style="background:silver">1.49%</td><td style="background:#8C8C8C">&#8212;</td></tr><tr><td><bold><italic>lox2/3</italic>x<italic>TPS10</italic></bold></td><td>7</td><td>7</td><td style="background:silver">0.07%</td><td style="background:silver">&#177;</td><td style="background:silver">0.07%</td><td style="background:#8C8C8C">1.24%</td><td style="background:#8C8C8C">&#177;</td><td style="background:#8C8C8C">0.84%</td><td style="background:silver">7.39%</td><td style="background:silver">&#177;</td><td style="background:silver">2.56%</td><td style="background:silver"><bold><named-content content-type="author-callout-style2">b</named-content></bold></td><td style="background:#8C8C8C">2.08%</td><td style="background:#8C8C8C">&#177;</td><td style="background:#8C8C8C">0.84%</td><td style="background:#8C8C8C">&#8201;</td><td style="background:silver">4.47%</td><td style="background:silver">&#177;</td><td style="background:silver">1.70%</td><td style="background:silver"><bold><named-content content-type="author-callout-style2">b</named-content></bold></td><td style="background:#8C8C8C">0.40%</td><td style="background:#8C8C8C">&#177;</td><td style="background:#8C8C8C">0.40%</td><td style="background:silver">3.02%</td><td style="background:silver">&#177;</td><td style="background:silver">1.42%</td><td style="background:#8C8C8C">0.73%</td><td style="background:#8C8C8C">&#177;</td><td style="background:#8C8C8C">0.31%</td><td style="background:silver">0.66%</td><td style="background:silver">&#177;</td><td style="background:silver">0.37%</td><td style="background:#8C8C8C">&#8212;</td></tr></tbody></table><table-wrap-foot><fn><p><named-content content-type="author-callout-style1">a</named-content>, <named-content content-type="author-callout-style2">b</named-content> Different letters indicate significant differences between lines for a vertical category (corrected <italic>P</italic>&lt;0.05) in Wilcoxon rank sum tests following significant Kruskal-Wallis tests across all genotypes (see Results text); <italic>P</italic>-values were corrected for multiple testing using the Holm-Bonferroni method. Community type was also tested and found not significant. Where there are no letters, there are no significant pairwise differences within a category.</p></fn><fn><p><named-content content-type="author-callout-style1">}</named-content> <named-content content-type="author-callout-style1">a</named-content><sub><named-content content-type="author-callout-style1">lox</named-content></sub>, <named-content content-type="author-callout-style2">}</named-content> <named-content content-type="author-callout-style2">b</named-content><sub><named-content content-type="author-callout-style2">lox</named-content></sub> / <named-content content-type="author-callout-style1">}</named-content> <named-content content-type="author-callout-style1">a</named-content><sub><named-content content-type="author-callout-style1">TPS</named-content></sub>, <named-content content-type="author-callout-style2">}</named-content> <named-content content-type="author-callout-style2">b</named-content><sub><named-content content-type="author-callout-style2">TPS</named-content></sub> Indicate significant differences (corrected P&lt;0.05) between lines with and without either the <italic>lox</italic> construct (WT and <italic>TPS</italic> v. <italic>lox</italic> and <italic>loxTPS</italic>) or the TPS construct (WT and <italic>lox</italic> v. <italic>TPS</italic> and <italic>loxTPS</italic>) in Wilcoxon rank sum tests for a category; P-values were corrected for multiple testing using the Holm-Bonferroni method. There are no significant pairwise differences in these categories.</p></fn><fn><p>IS, internal standard; GLVs, green leaf volatiles; TAB, (<italic>E</italic>)-&#945;-bergamotene; TBF, (<italic>E</italic>)-&#946;-farnesene; &#8212;, not detected.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04490-v2.xml">elife-04490-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="A2-tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.04490.029</object-id><label>Appendix 2&#8212;Table 1.</label><caption><p>Volatiles (percent IS plant<sup>&#8722;1</sup>, mean &#177; SEM) trapped in the headspace around single plants in experimental season one (June 8th&#8211;9th).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04490.029">http://dx.doi.org/10.7554/eLife.04490.029</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3" rowspan="2">&#8201;</th><th align="center" colspan="8">GLVs (percent IS plant<sup>-1</sup>)</th><th align="center" colspan="17">TPS10 products (percent IS plant<sup>-1</sup>)</th><th align="center" colspan="10">Non-target volatiles (percent IS plant<sup>-1</sup>)</th></tr><tr><th align="center" colspan="8">(<italic>Z</italic>)-Hexen-3-ol</th><th align="center" colspan="10">TAB</th><th align="center" colspan="7">TBF</th><th align="center" colspan="6">&#945;-Duprezianene</th><th align="center" colspan="4">Germacrene A</th></tr></thead><tbody><tr><td><bold>Genotype</bold></td><td><bold>Day n</bold></td><td><bold>Night n</bold></td><td colspan="3" style="background: #C0C0C0;"><bold>Day</bold></td><td colspan="5" style="background: #808080;"><bold>Night</bold></td><td colspan="4" style="background: #C0C0C0;"><bold>Day</bold></td><td colspan="6" style="background: #808080;"><bold>Night</bold></td><td colspan="4" style="background: #C0C0C0;"><bold>Day</bold></td><td colspan="3" style="background: #808080;"><bold>Night</bold></td><td colspan="3" style="background: #C0C0C0;"><bold>Day</bold></td><td colspan="3" style="background: #808080;"><bold>Night</bold></td><td colspan="3"><bold>Day</bold></td><td><bold>Night</bold></td></tr><tr><td><bold>WT</bold></td><td>8</td><td>8</td><td style="background: #C0C0C0;">0.99%</td><td style="background: #C0C0C0;">&#177;</td><td style="background: #C0C0C0;">0.99%</td><td style="background: #808080;">7.96%</td><td style="background: #808080;">&#177;</td><td style="background: #808080;">4.25%</td><td rowspan="2" style="background: #808080;" valign="middle"><named-content content-type="author-callout-style-a3">}</named-content></td><td rowspan="2" style="background: #808080;" valign="middle"><bold><named-content content-type="author-callout-style-a3">a</named-content></bold><sub><bold><named-content content-type="author-callout-style-a3">lox</named-content></bold></sub></td><td style="background: #C0C0C0;">0.37%</td><td style="background: #C0C0C0;">&#177;</td><td style="background: #C0C0C0;">0.29%</td><td style="background: #C0C0C0;"><bold><named-content content-type="author-callout-style-a3">a</named-content></bold></td><td colspan="3" style="background: #808080;">&#8212;</td><td style="background: #808080;">&#8201;</td><td rowspan="3" style="background: #808080;" valign="middle"><named-content content-type="author-callout-style-a3">}</named-content></td><td rowspan="2" style="background: #808080;" valign="bottom"><bold><named-content content-type="author-callout-style-a3">a</named-content></bold><sub><bold><named-content content-type="author-callout-style-a3">TPS</named-content></bold></sub></td><td colspan="3" style="background: #C0C0C0;">&#8212;</td><td style="background: #C0C0C0;"><bold><named-content content-type="author-callout-style-a3">a</named-content></bold></td><td colspan="3" style="background: #808080;">&#8212;</td><td style="background: #C0C0C0;">2.13%</td><td style="background: #C0C0C0;">&#177;</td><td style="background: #C0C0C0;">0.85%</td><td style="background: #808080;">0.96%</td><td style="background: #808080;">&#177;</td><td style="background: #808080;">0.42%</td><td>1.68%</td><td>&#177;</td><td>0.98%</td><td>&#8212;</td></tr><tr><td><bold><italic>TPS10</italic></bold></td><td>7</td><td>7</td><td style="background: #C0C0C0;">2.37%</td><td style="background: #C0C0C0;">&#177;</td><td style="background: #C0C0C0;">1.55%</td><td style="background: #808080;">2.84%</td><td style="background: #808080;">&#177;</td><td style="background: #808080;">0.63%</td><td style="background: #C0C0C0;">18.97%</td><td style="background: #C0C0C0;">&#177;</td><td style="background: #C0C0C0;">6.03%</td><td style="background: #C0C0C0;"><bold><named-content content-type="author-callout-style-a1">b</named-content></bold></td><td style="background: #808080;">0.94%</td><td style="background: #808080;">&#177;</td><td style="background: #808080;">0.53%</td><td rowspan="3" style="background: #808080;" valign="middle"><named-content content-type="author-callout-style-a1">}</named-content></td><td style="background: #C0C0C0;">9.34%</td><td style="background: #C0C0C0;">&#177;</td><td style="background: #C0C0C0;">3.44%</td><td style="background: #C0C0C0;"><bold><named-content content-type="author-callout-style-a1">b</named-content></bold></td><td style="background: #808080;">0.20%</td><td style="background: #808080;">&#177;</td><td style="background: #808080;">0.20%</td><td style="background: #C0C0C0;">9.47%</td><td style="background: #C0C0C0;">&#177;</td><td style="background: #C0C0C0;">5.26%</td><td style="background: #808080;">0.94%</td><td style="background: #808080;">&#177;</td><td style="background: #808080;">0.26%</td><td>2.78%</td><td>&#177;</td><td>1.52%</td><td>&#8212;</td></tr><tr><td><bold><italic>lox2/3</italic></bold></td><td>7</td><td>8</td><td style="background: #C0C0C0;">0.13%</td><td style="background: #C0C0C0;">&#177;</td><td style="background: #C0C0C0;">0.13%</td><td style="background: #808080;">1.06%</td><td style="background: #808080;">&#177;</td><td style="background: #808080;">0.64%</td><td rowspan="2" style="background: #808080;" valign="middle"><named-content content-type="author-callout-style-a1">}</named-content></td><td rowspan="2" style="background: #808080;" valign="middle"><bold><named-content content-type="author-callout-style-a1">b</named-content></bold><sub><bold><named-content content-type="author-callout-style-a1">lox</named-content></bold></sub></td><td colspan="3" style="background: #C0C0C0;">&#8212;</td><td style="background: #C0C0C0;"><bold><named-content content-type="author-callout-style-a3">a</named-content></bold></td><td style="background: #808080;">0.15%</td><td style="background: #808080;">&#177;</td><td style="background: #808080;">0.15%</td><td rowspan="2" style="background: #808080;"><bold><named-content content-type="author-callout-style-a1">b</named-content></bold><sub><bold><named-content content-type="author-callout-style-a1">TPS</named-content></bold></sub></td><td colspan="3" style="background: #C0C0C0;">&#8212;</td><td style="background: #C0C0C0;"><bold><named-content content-type="author-callout-style-a3">a</named-content></bold></td><td colspan="3" style="background: #808080;">&#8212;</td><td style="background: #C0C0C0;">2.75%</td><td style="background: #C0C0C0;">&#177;</td><td style="background: #C0C0C0;">1.12%</td><td style="background: #808080;">0.60%</td><td style="background: #808080;">&#177;</td><td style="background: #808080;">0.19%</td><td>1.68%</td><td>&#177;</td><td>1.49%</td><td>&#8212;</td></tr><tr><td><bold><italic>lox2/3</italic>x<italic>TPS10</italic></bold></td><td>7</td><td>7</td><td style="background: #C0C0C0;">0.07%</td><td style="background: #C0C0C0;">&#177;</td><td style="background: #C0C0C0;">0.07%</td><td style="background: #808080;">1.24%</td><td style="background: #808080;">&#177;</td><td style="background: #808080;">0.84%</td><td style="background: #C0C0C0;">7.39%</td><td style="background: #C0C0C0;">&#177;</td><td style="background: #C0C0C0;">2.56%</td><td style="background: #C0C0C0;"><bold><named-content content-type="author-callout-style-a1">b</named-content></bold></td><td style="background: #808080;">2.08%</td><td style="background: #808080;">&#177;</td><td style="background: #808080;">0.84%</td><td style="background: #808080;">&#8201;</td><td style="background: #C0C0C0;">4.47%</td><td style="background: #C0C0C0;">&#177;</td><td style="background: #C0C0C0;">1.70%</td><td style="background: #C0C0C0;"><bold><named-content content-type="author-callout-style-a1">b</named-content></bold></td><td style="background: #808080;">0.40%</td><td style="background: #808080;">&#177;</td><td style="background: #808080;">0.40%</td><td style="background: #C0C0C0;">3.02%</td><td style="background: #C0C0C0;">&#177;</td><td style="background: #C0C0C0;">1.42%</td><td style="background: #808080;">0.73%</td><td style="background: #808080;">&#177;</td><td style="background: #808080;">0.31%</td><td>0.66%</td><td>&#177;</td><td>0.37%</td><td>&#8212;</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p><named-content content-type="author-callout-style-a3">a</named-content>, <named-content content-type="author-callout-style-a1">b</named-content> Different letters indicate significant differences between lines for a vertical category (corrected <italic>P</italic>&lt;0.05) in Wilcoxon rank sum tests following significant Kruskal-Wallis tests across all genotypes (see Results text); <italic>P</italic>-values were corrected for multiple testing using the Holm-Bonferroni method. Community type was also tested and found not significant. Where there are no letters, there are no significant pairwise differences within a category.</p></fn><fn id="tblfn2"><p><named-content content-type="author-callout-style-a3">}</named-content><named-content content-type="author-callout-style-a3">a</named-content><sub><named-content content-type="author-callout-style-a3">lox</named-content></sub>, <named-content content-type="author-callout-style-a1">}</named-content><named-content content-type="author-callout-style-a1">b</named-content><sub><named-content content-type="author-callout-style-a1">lox</named-content></sub> / <named-content content-type="author-callout-style-a3">}</named-content><named-content content-type="author-callout-style-a3">a</named-content><sub><named-content content-type="author-callout-style-a3">TPS</named-content></sub>, <named-content content-type="author-callout-style-a1">}</named-content><named-content content-type="author-callout-style-a1">b</named-content><sub><named-content content-type="author-callout-style-a1">TPS</named-content></sub> Indicate significant differences (corrected P&lt;0.05) between lines with and without either the <italic>lox</italic> construct (WT and <italic>TPS</italic> v. <italic>lox</italic> and <italic>loxTPS</italic>) or the TPS construct (WT and <italic>lox</italic> v. <italic>TPS</italic> and <italic>loxTPS</italic>) in Wilcoxon rank sum tests for a category; P-values were corrected for multiple testing using the Holm-Bonferroni method. There are no significant pairwise differences in these categories.</p></fn><fn id="tblfn3"><p>IS, internal standard; GLVs, green leaf volatiles; TAB, (<italic>E</italic>)-&#945;-bergamotene; TBF, (<italic>E</italic>)-&#946;-farnesene; &#8212;, not detected.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04531-v2.xml">elife-04531-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.04531.010</object-id><label>Table 2.</label><caption><p>Composition and codon usage of E7 wt and mutant insert sequences</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04531.010">http://dx.doi.org/10.7554/eLife.04531.010</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Region</th><th rowspan="2">Sequence (Symbol)</th><th rowspan="2">G+C content</th><th rowspan="2">CpG Total<xref ref-type="table-fn" rid="tblfn4">*</xref></th><th rowspan="2">O/E ratio<xref ref-type="table-fn" rid="tblfn5">&#8224;</xref>,<xref ref-type="table-fn" rid="tblfn6">&#8225;</xref></th><th rowspan="2">UpA Total<xref ref-type="table-fn" rid="tblfn4">*</xref></th><th rowspan="2">O/E ratio<xref ref-type="table-fn" rid="tblfn5">&#8224;</xref>,<xref ref-type="table-fn" rid="tblfn6">&#8225;</xref></th><th colspan="3">Codon Usage</th></tr><tr><th>CAI<xref ref-type="table-fn" rid="tblfn7">&#182;</xref></th><th>ENc</th><th>CP Bias</th></tr></thead><tbody><tr><td>1</td><td>Native (WT)</td><td>47.6%</td><td>51 (&#8722;)</td><td>0.730</td><td>62 (&#8722;)</td><td>0.742</td><td>0.685</td><td>56.5</td><td>&#8722;0.043</td></tr><tr><td/><td>Permuted (P)</td><td>47.6%</td><td>51 (0)</td><td>0.730</td><td>2 (0)</td><td>0.742</td><td>0.694</td><td>55.8</td><td>&#8722;0.025</td></tr><tr><td/><td>CpG/UpAL (cu)</td><td>47.5%</td><td>0 (&#8722;51)</td><td><named-content content-type="author-callout-style2">0</named-content></td><td>19 (&#8722;43)</td><td><named-content content-type="author-callout-style2">0.227</named-content></td><td>0.686</td><td>43.5</td><td>0.087</td></tr><tr><td/><td>Max-U</td><td>50.1%</td><td>47 (&#8722;4)</td><td>0.610</td><td>43 (&#8722;19)</td><td>0.573</td><td>0.708</td><td>49.6</td><td><named-content content-type="author-callout-style1">0.328</named-content></td></tr><tr><td/><td>Min_E</td><td>47.5%</td><td>51 (0)</td><td>0.736</td><td>62 (0)</td><td>0.735</td><td>0.748</td><td>54.3</td><td><named-content content-type="author-callout-style2">&#8722;0.131</named-content></td></tr><tr><td/><td>Min_U</td><td>47.5%</td><td>69 (+18)</td><td>0.992</td><td>76 (+14)</td><td>0.939</td><td>0.709</td><td>58.3</td><td><named-content content-type="author-callout-style2">&#8722;0.134</named-content></td></tr><tr><td/><td>Min_H</td><td>49.8%</td><td>106 (+55)</td><td>1.400</td><td>79 (+17)</td><td>0.981</td><td>0.696</td><td>49.2</td><td><named-content content-type="author-callout-style2">&#8722;0.130</named-content></td></tr><tr><td>2</td><td>Native (WT)</td><td>47.1%</td><td>18 (&#8722;)</td><td>0.320</td><td>48 (&#8722;)</td><td>0.695</td><td>0.743</td><td>53.2</td><td>0.015</td></tr><tr><td/><td>Permuted (P)</td><td>47.6%</td><td>18 (0)</td><td>0.320</td><td>48 (0)</td><td>0.695</td><td>0.739</td><td>49.0</td><td>0.013</td></tr><tr><td/><td>CpG/UpAL (cu)</td><td>48.5%</td><td>0 (&#8722;18)</td><td><named-content content-type="author-callout-style2">0</named-content></td><td>48 (0)</td><td><named-content content-type="author-callout-style2">0.214</named-content></td><td>0.739</td><td>47.2</td><td>0.118</td></tr><tr><td/><td>Max-U</td><td>46.3%</td><td>24 (+6)</td><td>0.440</td><td>43 (&#8722;3)</td><td>0.601</td><td>0.750</td><td>46.1</td><td><named-content content-type="author-callout-style1">0.311</named-content></td></tr><tr><td/><td>Min-E</td><td>45.7%</td><td>18 (0)</td><td>0.343</td><td>48 (0)</td><td>0.657</td><td>0.785</td><td>53.3</td><td><named-content content-type="author-callout-style2">&#8722;0.091</named-content></td></tr><tr><td/><td>Min-U</td><td>47.4%</td><td>37 (+19)</td><td>0.649</td><td>50 (+2)</td><td>0.738</td><td>0.767</td><td>57.6</td><td><named-content content-type="author-callout-style2">&#8722;0.083</named-content></td></tr><tr><td/><td>Min-H</td><td>47.8%</td><td>68 (+50)</td><td>1.172</td><td>65 (+15)</td><td>0.970</td><td>0.715</td><td>49.7</td><td><named-content content-type="author-callout-style2">&#8722;0.085</named-content></td></tr></tbody></table><table-wrap-foot><fn id="tblfn4"><label>*</label><p>Total number of CpG and UpA dinucleotides in sequence. Changes in numbers between mutated and original WT sequence are indicated in parentheses.</p></fn><fn id="tblfn5"><label>&#8224;</label><p>Ratio of observed dinucleotide frequency (O) to that expected based on mononucleotide composition (E) that is, f(CpG)/f(C) &#215; f(G).</p></fn><fn id="tblfn6"><label>&#8225;</label><p>Values deliberately changed are shown in red (maximised) and blue (minimised).</p></fn><fn id="tblfn7"><label>&#182;</label><p>Calculated from <ext-link ext-link-type="uri" xlink:href="http://genomes.urv.es/CAIcal/">http://genomes.urv.es/CAIcal/</ext-link> (<xref ref-type="bibr" rid="bib24">Puigbo et al., 2008</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04539-v2.xml">elife-04539-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.04539.005</object-id><label>Table 1.</label><caption><p>Direct repeat recombinant frequencies</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04539.005">http://dx.doi.org/10.7554/eLife.04539.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Genotype</th><th rowspan="2"><italic>RTS1</italic> orientation</th><th rowspan="2">Position of direct repeat</th><th rowspan="2">Colonies analysed</th><th colspan="2">Ade<sup>+</sup> His<sup>+</sup> recombinant frequency (&#215; 10<sup>&#8722;4</sup>)<xref ref-type="table-fn" rid="tblfn1">*</xref></th><th colspan="2">Ade<sup>+</sup> His<sup>&#8722;</sup> recombinant frequency (&#215; 10<sup>&#8722;4</sup>)<xref ref-type="table-fn" rid="tblfn1">*</xref></th></tr><tr><th>Mean</th><th>p value<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></th><th>Mean</th><th>p value<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></th></tr></thead><tbody><tr><td>wild type</td><td>IO</td><td>Flanking <italic>RTS1</italic></td><td>77</td><td>1.36 (+/&#8722; 0.51)</td><td>&#8211;</td><td>3.48 (+/&#8722; 0.89)</td><td>&#8211;</td></tr><tr><td>wild type</td><td>AO</td><td>Flanking <italic>RTS1</italic></td><td>77</td><td>134.03 (+/&#8722; 38.50)</td><td>&#8211;</td><td>113.41 (+/&#8722; 49.72)</td><td>&#8211;</td></tr><tr><td><italic>rad52&#8710;</italic></td><td>IO</td><td>Flanking <italic>RTS1</italic></td><td>15</td><td>0.03 (+/&#8722; 0.04)</td><td>&lt;0.001<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>1.20 (+/&#8722; 0.21)</td><td>&lt;0.001<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td></tr><tr><td><italic>rad52&#8710;</italic></td><td>AO</td><td>Flanking <italic>RTS1</italic></td><td>15</td><td>0.13 (+/&#8722; 0.11)</td><td>&lt;0.001<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td><td>4.07 (+/&#8722; 1.16)</td><td>&lt;0.001<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td></tr><tr><td><italic>ori-1253&#8710;</italic></td><td>IO</td><td>Flanking <italic>RTS1</italic></td><td>15</td><td>2.33 (+/&#8722; 1.86)</td><td>0.060<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>4.80 (+/&#8722; 1.58)</td><td>0.007<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td></tr><tr><td><italic>ori-1253&#8710;</italic></td><td>AO</td><td>Flanking <italic>RTS1</italic></td><td>16</td><td>339.88 (+/&#8722; 83.61)</td><td>&lt;0.001<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td><td>221.02 (+/&#8722; 57.21)</td><td>&lt;0.001<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td></tr><tr><td>wild type</td><td>IO</td><td>Site A</td><td>18</td><td>1.44 (+/&#8722; 0.86)</td><td>0.698<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>4.61 (+/&#8722; 1.66)</td><td>0.011<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td></tr><tr><td>wild type</td><td>AO</td><td>Site A</td><td>21</td><td>128.66 (+/&#8722; 43.85)</td><td>0.746<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td><td>680.48 (+/&#8722; 305.44)</td><td>&lt;0.001<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td></tr><tr><td>wild type</td><td>IO</td><td>Site B</td><td>18</td><td>0.88 (+/&#8722; 0.51)</td><td>0.023<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>2.45 (+/&#8722; 1.88)</td><td>0.001<xref ref-type="table-fn" rid="tblfn5">#</xref></td></tr><tr><td>wild type</td><td>AO</td><td>Site B</td><td>15</td><td>5.12 (+/&#8722; 2.41)</td><td>&lt;0.001<xref ref-type="table-fn" rid="tblfn6">&#182;</xref></td><td>85.02 (+/&#8722; 33.72)</td><td>&lt;0.001<xref ref-type="table-fn" rid="tblfn6">&#182;</xref></td></tr><tr><td><italic>ori-1253&#8710;</italic></td><td>IO</td><td>Site B</td><td>15</td><td>1.58 (+/&#8722; 0.50)</td><td>&lt;0.001<xref ref-type="table-fn" rid="tblfn7">**</xref></td><td>4.86 (+/&#8722; 2.56)</td><td>0.006<xref ref-type="table-fn" rid="tblfn7">**</xref></td></tr><tr><td><italic>ori-1253&#8710;</italic></td><td>AO</td><td>Site B</td><td>15</td><td>86.03 (+/&#8722; 33.63)</td><td>&lt;0.001<xref ref-type="table-fn" rid="tblfn8">&#8224;&#8224;</xref></td><td>1094.46 (+/&#8722; 443.32)</td><td>&lt;0.001<xref ref-type="table-fn" rid="tblfn8">&#8224;&#8224;</xref></td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>The values in parentheses are the standard deviations about the mean.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>p values are derived from independent-sample <italic>t</italic>-tests comparing the mean values as indicated.</p></fn><fn id="tblfn3"><label>&#8225;</label><p>Compared to the equivalent mean recombinant frequency in wild type with <italic>RTS1-IO</italic> flanked by <italic>ade6</italic><sup><italic>-</italic></sup> direct repeats.</p></fn><fn id="tblfn4"><label>&#167;</label><p>Compared to the equivalent mean recombinant frequency in wild type with <italic>RTS1-AO</italic> flanked by <italic>ade6</italic><sup><italic>-</italic></sup> direct repeats.</p></fn><fn id="tblfn5"><label>#</label><p>Compared to the equivalent mean recombinant frequency in wild type with <italic>RTS1-IO</italic> and site A <italic>ade6</italic><sup><italic>-</italic></sup> direct repeats.</p></fn><fn id="tblfn6"><label>&#182;</label><p>Compared to the equivalent mean recombinant frequency in wild type with <italic>RTS1-IO</italic> and site A <italic>ade6</italic><sup><italic>-</italic></sup> direct repeats.</p></fn><fn id="tblfn7"><label>**</label><p>Compared to the equivalent mean recombinant frequency in wild type with <italic>RTS1-IO</italic> and site B <italic>ade6</italic><sup><italic>-</italic></sup> direct repeats.</p></fn><fn id="tblfn8"><label>&#8224;&#8224;</label><p>Compared to the equivalent mean recombinant frequency in wild type with <italic>RTS1-AO</italic> and site B <italic>ade6</italic><sup><italic>-</italic></sup> direct repeats.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04550-v2.xml">elife-04550-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.04550.003</object-id><label>Table 1.</label><caption><p>Virulence of the <italic>A. candida</italic> races on different plant host accessions</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04550.003">http://dx.doi.org/10.7554/eLife.04550.003</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><italic>A. candida</italic> race</th><th colspan="2"><italic>Arabidopsis thaliana</italic></th><th colspan="2"><italic>Brassica rapa</italic></th><th colspan="2"><italic>Brassica junceae</italic></th><th colspan="2"><italic>Brassica oleraceae</italic></th></tr><tr><th>+</th><th>&#8722;</th><th>+</th><th>&#8722;</th><th>+</th><th>&#8722;</th><th>+</th><th>&#8722;</th></tr></thead><tbody><tr><td>AcNc2</td><td>137</td><td>219</td><td>0</td><td>1</td><td>0</td><td>1</td><td>0</td><td>18</td></tr><tr><td>AcBoT</td><td>1<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>34</td><td>0</td><td>1</td><td>0</td><td>2</td><td>15</td><td>0</td></tr><tr><td>Ac2V</td><td>1<xref ref-type="table-fn" rid="tblfn1">*</xref>,<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>107</td><td>1<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>4<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>6<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>0<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>0</td><td>2</td></tr></tbody></table><table-wrap-foot><fn><p>+ Host-pathogen compatible interactions (number of susceptible accessions).</p></fn><fn><p>&#8722; Host-pathogen incompatible interactions (number of resistant accessions).</p></fn><fn id="tblfn1"><label>*</label><p><italic>A. thaliana</italic> Ws-<italic>eds1</italic> (<underline>e</underline>nhanced <underline>d</underline>isease <underline>s</underline>usceptibility) mutants were susceptible to all tested <italic>A. candida</italic> races.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>In the laboratory conditions, the cotyledons of the <italic>A. thaliana</italic> accession Ws-3 were found to be susceptible to the Ac2V (<xref ref-type="bibr" rid="bib11">Cooper et al., 2008</xref>).</p></fn><fn id="tblfn3"><label>&#8225;</label><p>Data from (<xref ref-type="bibr" rid="bib47">Rimmer et al., 2000</xref>) incorporated; in this study, one cultivar <italic>B. rapa</italic> (CrGC1-18, rapid-cycling accession) was infected by Ac2V race and four other tested cultivars (&#8216;Torch&#8217;, &#8216;Colt&#8217;, &#8216;Horizon&#8217;, &#8216;Reward&#8217;) were incompatible with Ac2V race. All analysed cultivars of <italic>B. juncea</italic> (CrGC4-1S, &#8216;Burgonde&#8217;, &#8216;Domo&#8217;, &#8216;Cutlass&#8217;) were susceptible to Ac2V.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04585-v2.xml">elife-04585-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.04585.011</object-id><label>Table 2.</label><caption><p>Summary of results from hypothesis-driven analyses conducted on 8 ms-IPI Experiment 1 and control Experiment 2 (IPI of 500 ms)</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04585.011">http://dx.doi.org/10.7554/eLife.04585.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2"/><th colspan="4">Expt 1 (IPI 8 ms)</th><th colspan="4">Expt 2 (IPI 500 ms)</th><th colspan="4">Expt 1 vs Expt 2</th></tr><tr><th>PMv-M1</th><th>AIP-PMv</th><th>pSPL-PMd</th><th>PMd-M1</th><th>PMv-M1</th><th>AIP-PMv</th><th>pSPL-PMd</th><th>PMd-M1</th><th>PMv-M1</th><th>AIP-PMv</th><th>pSPL-PMd</th><th>PMd-M1</th></tr></thead><tbody><tr><td rowspan="2">Functional connectivity (fc)</td><td>grasp</td><td>t(13) = &#8722;2.59; p = 0.023*</td><td colspan="3" rowspan="2"/><td>t(13) = 0.94; p = 0.36</td><td colspan="3" rowspan="2"/><td>F(1,26) = 4.64; p = 0.041*</td><td colspan="3" rowspan="2"/></tr><tr><td>rest</td><td>t(14) = &#8722;0.07; p = 0.94</td><td>t(14) = 0.07; p = 0.95</td><td>n.a.</td></tr><tr><td rowspan="2">partial correlation fc</td><td>grasp</td><td>t(13) = &#8722;3.72; p = 0.003*</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>t(13) = 1.00; p = 0.34</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>F(1,26) = 7.76; p = 0.011*</td><td>n.s.</td><td>n.s.</td><td>n.s.</td></tr><tr><td>rest</td><td>t(14) = &#8722;0.07; p = 0.95</td><td>t(14) = &#8722;2.50; p = 0.025*</td><td>t(14) = 2.22; p = 0.04*</td><td>t(14) = 2.84; p = 0.013*</td><td>t(14) = &#8722;0.39; p = 0.70</td><td>t(14) = 1.08; p = 0.30</td><td>t(14) = &#8722;1.24; p = 0.24</td><td>t(14) = 0.47; p = 0.65</td><td>n.a.</td><td>F(1,28) = 7.15; p = 0.012*</td><td>F(1,28) = 5.29; p = 0.029*</td><td>F(1,28) = 5.92; p = 0.08</td></tr><tr><td rowspan="2">Psycho-physiological interaction (PPI)</td><td>grasp</td><td>t(13) = &#8722;4.78; p = 0.0004*</td><td colspan="3" rowspan="2"/><td>t(13) = 0.98; p = 0.35</td><td colspan="3" rowspan="2"/><td>F(1,26) = 6.92; p = 0.014*</td><td colspan="3" rowspan="2"/></tr><tr><td>rest</td><td>t(14) = 0.08; p = 0.93</td><td>t(14) = 0.20; p = 0.85</td><td>n.a.</td></tr><tr><td rowspan="2">multiple regression PPI</td><td>grasp</td><td>t(13) = &#8722;2.53; p = 0.0064*</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>t(13) = 1.18; p = 0.26</td><td>n.s.</td><td>n.s.</td><td>n.s.</td><td>F(1,26) = 7.47; p = 0.011*</td><td>n.s.</td><td>n.s.</td><td>n.s.</td></tr><tr><td>rest</td><td>n.a.</td><td>t(14) = &#8722;2.55; p = 0.023*</td><td>t(14) = 1.78; p = 0.097</td><td>t(14) = 2.84; p = 0.013*</td><td>n.a.</td><td>t(14) = 0.41; p = 0.96</td><td>t(14) = &#8722;1.18; p = 0.26</td><td>t(14) = 0.01; p = 0.99</td><td>n.a.</td><td>F(1,28) = 5.74; p = 0.024*</td><td>F(1,28) = 3.66; p = 0.066</td><td>F(1,28) = 4.44; p = 0.044*</td></tr></tbody></table><table-wrap-foot><fn><p>Analyses were conducted on rest and task data. Moreover in order to show that specific effects relate to plasticity induction (8 ms IPI) several higher-level analyses contrasting Experiment 1 and 2 are presented. T-tests were conducted as two-tailed paired t-tests (within subjects). Mixed-model ANOVAs were conducted between experiments (across subjects). Detailed information on all analyses is provided in the &#8216;Materials and methods&#8217; section. Asterisks indicate significant results, p &lt; 0.05. Abbreviations: n.s. = non-significant.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04677-v1.xml">elife-04677-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.04677.018</object-id><label>Table 2.</label><caption><p>Decoding using PMd and M1 separately, all trials</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04677.018">http://dx.doi.org/10.7554/eLife.04677.018</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="3">Forced-choice decoder performance (leave-one-out)</th><th colspan="3">Decoder agreement</th></tr><tr><th>Combined</th><th>PMd</th><th>M1</th><th>PMd-combined</th><th>M1-combined</th><th>PMd-M1</th></tr></thead><tbody><tr><td>J1-T</td><td>0.99</td><td>0.99</td><td>0.90</td><td>0.92</td><td>0.85</td><td>0.80</td></tr><tr><td>J1-S</td><td>1.00</td><td>1.00</td><td>0.80</td><td>0.93</td><td>0.86</td><td>0.81</td></tr><tr><td>J2-T</td><td>0.98</td><td>0.97</td><td>0.80</td><td>0.96</td><td>0.87</td><td>0.86</td></tr><tr><td>J2-S</td><td>1.00</td><td>1.00</td><td>0.85</td><td>0.98</td><td>0.84</td><td>0.83</td></tr><tr><td>N1-S</td><td>0.90</td><td>0.84</td><td>0.95</td><td>0.93</td><td>0.72</td><td>0.67</td></tr></tbody></table><table-wrap-foot><fn><p>Statistics for decoding choice from PMd and M1 data separately. Decoder performance refers to the fraction of trials for which the final decoded point agreed with the target that the animal reached to. This was assessed on forced-choice trials with delays of at least 300 ms, using leave-one-out cross-validation. Decoder agreement refers to the mean fraction of time points per trial for which the decoded choice was the same using the two datasets indicated. For this statistic, all successful trials with delay periods of at least 300 ms were included. &#8216;Combined&#8217; refers to the decoder trained using both PMd and M1 data.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04677-v1.xml">elife-04677-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.04677.019</object-id><label>Table 3.</label><caption><p>Decoding using PMd and M1 separately, free choice trials</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04677.019">http://dx.doi.org/10.7554/eLife.04677.019</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="3">Free-choice decoder performance</th><th colspan="3">Decoder agreement</th></tr><tr><th>Combined</th><th>PMd</th><th>M1</th><th>PMd-combined</th><th>M1-combined</th><th>PMd-M1</th></tr></thead><tbody><tr><td>J1-T</td><td>0.94</td><td>0.86</td><td>0.77</td><td>0.90</td><td>0.83</td><td>0.77</td></tr><tr><td>J1-S</td><td>0.96</td><td>0.94</td><td>0.89</td><td>0.86</td><td>0.82</td><td>0.73</td></tr><tr><td>J2-T</td><td>0.97</td><td>0.96</td><td>0.82</td><td>0.96</td><td>0.82</td><td>0.80</td></tr><tr><td>J2-S</td><td>0.95</td><td>0.95</td><td>0.86</td><td>0.98</td><td>0.78</td><td>0.78</td></tr><tr><td>N1-S</td><td>0.88</td><td>0.82</td><td>0.79</td><td>0.91</td><td>0.72</td><td>0.67</td></tr></tbody></table><table-wrap-foot><fn><p>Same as <xref ref-type="table" rid="tbl2">Table 2</xref>, but only considering free-choice trials with delay periods of at least 300 ms. The decoders were trained using forced-choice trials.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-04970-v2.xml">elife-04970-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.04970.011</object-id><label>Table 1.</label><caption><p>Pause sites and their responses to H-NS and transcription factors</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.04970.011">http://dx.doi.org/10.7554/eLife.04970.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Pause position</th><th colspan="4">Pausing</th><th colspan="3">Termination</th><th colspan="2">Sequence</th></tr><tr><th>H-NS</th><th>RfaH</th><th>GreB</th><th>GreA</th><th>Rho</th><th>Rho + H-NS</th><th>Rho + NusG</th><th align="right">Pause&#8595;</th><th/></tr></thead><tbody><tr><td>C134</td><td>&#8593;</td><td><bold>?</bold></td><td><bold>?</bold></td><td><bold>?</bold></td><td><bold>?</bold></td><td><bold>?</bold></td><td><bold>?</bold></td><td align="right">CGCUGAUAACUCAAG<bold>C</bold></td><td><underline>U</underline>UUCUUCCUG</td></tr><tr><td>G162</td><td>&#8593;</td><td>&#8595;</td><td>&#8595;</td><td><bold>?</bold></td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td align="right">AAUUAAGGCUGAACU<bold>G</bold></td><td><underline>A</underline>AAUUUUAUU</td></tr><tr><td>U169</td><td>&#8593;</td><td>&#8211;</td><td>&#8595;</td><td>&#8595;</td><td>&#8211;</td><td>&#8211;</td><td>&#8593;</td><td align="right">GCUGAACUGAAAUUU<bold>U</bold></td><td><underline>A</underline>UUAAUUGCA</td></tr><tr><td>C213</td><td>&#8593;</td><td>&#8211;</td><td>&#8595;</td><td>&#8595;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td align="right">GCGUGACACCUGCAA<bold>C</bold></td><td><underline>A</underline>UCCUCCAUA</td></tr><tr><td>C220</td><td>&#8593;</td><td>&#8595;</td><td>&#8211;</td><td>&#8595;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td align="right">ACCUGCAACAUCCUC<bold>C</bold></td><td><underline>A</underline>UAUUUCCGC</td></tr><tr><td>U226</td><td>&#8593;</td><td>&#8595;</td><td>&#8211;</td><td>&#8595;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td align="right">AACAUCCUCCAUAUU<bold>U</bold></td><td><underline>C</underline>CGCUCAUUU</td></tr><tr><td>C346</td><td>&#8593;&#8593;&#8593;</td><td>&#8211;</td><td>&#8595;</td><td>&#8211;</td><td>&#8211;</td><td>&#8593;</td><td>&#8211;</td><td align="right">UAGCUGGAACUCUUU<bold>C</bold></td><td><underline>G</underline>GGUAAAGCC</td></tr><tr><td>C370</td><td>&#8211;</td><td>&#8595;</td><td>&#8211;</td><td>&#8595;</td><td>&#8593;</td><td>&#8211;</td><td>&#8211;</td><td align="right">CCGCUGGAUAUCCCA<bold>C</bold></td><td><underline>A</underline>GCAACGGGU</td></tr><tr><td>C393, G394</td><td>&#8593;</td><td>&#8595;</td><td>&#8595;</td><td>&#8595;</td><td>&#8211;</td><td>&#8593;</td><td>&#8593;</td><td align="right">GGUUGGGCAGCAACA<bold>C</bold></td><td><bold><underline>G</underline></bold><underline>U</underline>UUUGCUGA</td></tr><tr><td>U588, U589</td><td>&#8593;&#8593;&#8593;</td><td>&#8211;</td><td>&#8595;</td><td>&#8595;</td><td>&#8593;</td><td>&#8211;</td><td>&#8211;</td><td align="right">UCAAGGCAUACUCUU<bold>U</bold></td><td><bold><underline>U</underline></bold><underline>U</underline>CUAUUCCA</td></tr><tr><td>A593</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td align="right">GCAUACUCUUUUUCU<bold>A</bold></td><td><underline>U</underline>UCCACUUGA</td></tr><tr><td>G624</td><td>&#8593;</td><td>&#8595;</td><td>&#8595;</td><td>&#8595;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td align="right">UUCUUUCGCCAGCGC<bold>G</bold></td><td><underline>U</underline>UUUUGAAAG</td></tr><tr><td>G643</td><td>&#8593;</td><td>&#8211;</td><td>&#8595;</td><td>&#8595;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td align="right">UUGAAAGCCAAUUCC<bold>G</bold></td><td><underline>C</underline>GCCCCAUGA</td></tr><tr><td>A746, U747</td><td>&#8211;</td><td>&#8211;</td><td>&#8595;</td><td>&#8211;</td><td>&#8593;</td><td>&#8211;</td><td>&#8593;</td><td align="right">GCAAGGACCUUUUUU<bold>A</bold></td><td><bold><underline>U</underline></bold><underline>A</underline>AACAAAAA</td></tr><tr><td>G926</td><td>&#8593;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>?</td><td>?</td><td align="right">AAUAUGACCAUGCUC<bold>G</bold></td><td><underline>C</underline>AGUUAUUAA</td></tr><tr><td>U996</td><td>&#8593;</td><td>&#8211;</td><td>&#8595;</td><td>&#8595;</td><td>&#8211;</td><td>?</td><td>?</td><td align="right">CCAAUAAUUAAGUUA<bold>U</bold></td><td><underline>U</underline>GGGAUUUGU</td></tr><tr><td>U1011</td><td>&#8593;</td><td>&#8595;</td><td>&#8595;</td><td>&#8595;</td><td>&#8211;</td><td>?</td><td>?</td><td align="right">UUGGGAUUUGUCUGG<bold>U</bold></td><td><underline>G</underline>AAUUAUUUG</td></tr><tr><td>U1022, U1024</td><td>&#8593;</td><td>&#8595;</td><td>&#8595;</td><td>&#8595;</td><td>&#8211;</td><td>?</td><td>?</td><td align="right">GUCUGGUGAAUUAUU<bold>U</bold></td><td><underline>G</underline><bold>U</bold><underline>C</underline>GCUAUCU</td></tr><tr><td>U1079 (<italic>ops</italic>)</td><td>&#8211;</td><td>&#8211;</td><td>&#8595;</td><td>&#8595;</td><td>&#8593;</td><td>?</td><td>?</td><td align="right">CUAGUGGCGGUAGCG<bold>U</bold></td><td><underline>G</underline>CUUUUUUCA</td></tr></tbody></table><table-wrap-foot><fn><p>Pause positions are given as 3&#8242; RNA nucleotide identity and distance from the transcription start site as mapped by high-resolution PAGE (<xref ref-type="fig" rid="fig2s1">Figure 2&#8212;figure supplement 1</xref>). &#8593;, increased pause or termination. &#8595;, decreased pause or termination. In the sequences shown, pause 3&#8242; ends are bold (under arrow) and the position corresponding to the incoming NTP is underlined.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-05005-v1.xml">elife-05005-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.05005.014</object-id><label>Table 1.</label><caption><p>The 26 features considered in the models, highlighting the 14 robustly selected through stepwise regression (bold)</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.05005.014">http://dx.doi.org/10.7554/eLife.05005.014</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Feature</th><th rowspan="2">Abbreviation</th><th rowspan="2">Description</th><th colspan="4">Frequency chosen</th></tr><tr><th>8mer</th><th>7mer-m8</th><th>7mer-A1</th><th>6mer</th></tr></thead><tbody><tr><td colspan="7"><bold>miRNA</bold></td></tr><tr><td><bold>&#8195;3&#8242;-UTR target-site abundance</bold></td><td><bold>TA_3UTR</bold></td><td><bold>Number of sites in all annotated 3&#8242; UTRs (</bold><xref ref-type="bibr" rid="bib5"><bold>Arvey et al., 2010</bold></xref><bold>;</bold> <xref ref-type="bibr" rid="bib60"><bold>Garcia et al., 2011</bold></xref><bold>)</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td></tr><tr><td>&#8195;ORF target-site abundance</td><td>TA_ORF</td><td>Number of sites in all annotated ORFs (<xref ref-type="bibr" rid="bib60">Garcia et al., 2011</xref>)</td><td>9.4%</td><td>0.7%</td><td>68.1%</td><td>93.4%</td></tr><tr><td>&#8195;<bold>Predicted seed-pairing stability</bold></td><td><bold>SPS</bold></td><td><bold>Predicted thermodynamic stability of seed pairing (</bold><xref ref-type="bibr" rid="bib60"><bold>Garcia et al., 2011</bold></xref><bold>)</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td></tr><tr><td>&#8195;<bold>sRNA position 1</bold></td><td><bold>sRNA1</bold></td><td><bold>Identity of nucleotide at position 1 of the sRNA</bold></td><td><bold>68%</bold></td><td><bold>100%</bold></td><td><bold>99.7%</bold></td><td><bold>97.7%</bold></td></tr><tr><td>&#8195;<bold>sRNA position 8</bold></td><td><bold>sRNA8</bold></td><td><bold>Identity of nucleotide at position 8 of the sRNA</bold></td><td><bold>0%</bold></td><td><bold>0.8%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td></tr><tr><td colspan="7"><bold>Site</bold></td></tr><tr><td>&#8195;Site position 1</td><td>site1</td><td>Identity of nucleotide at position 1 of the site</td><td>N/A</td><td>57.1%</td><td>N/A</td><td>2%</td></tr><tr><td>&#8195;<bold>Site position 8</bold></td><td><bold>site8</bold></td><td><bold>Identity of nucleotide at position 8 of the site</bold></td><td><bold>0.8%</bold></td><td><bold>95.1%</bold></td><td><bold>99.4%</bold></td><td><bold>100%</bold></td></tr><tr><td>&#8195;Site position 9</td><td>site9</td><td>Identity of nucleotide at position 9 of the site (<xref ref-type="bibr" rid="bib147">Lewis et al., 2005</xref>; <xref ref-type="bibr" rid="bib213">Nielsen et al., 2007</xref>)</td><td>15.4%</td><td>7.1%</td><td>0.9%</td><td>93.7%</td></tr><tr><td>&#8195;Site position 10</td><td>site10</td><td>Identity of nucleotide at position 10 of the site (<xref ref-type="bibr" rid="bib213">Nielsen et al., 2007</xref>)</td><td>0.1%</td><td>100%</td><td>8.5%</td><td>26.3%</td></tr><tr><td>&#8195;<bold>Local AU content</bold></td><td><bold>local_AU</bold></td><td><bold>AU content near the site (</bold><xref ref-type="bibr" rid="bib78"><bold>Grimson et al., 2007</bold></xref><bold>;</bold> <xref ref-type="bibr" rid="bib213"><bold>Nielsen et al., 2007</bold></xref><bold>)</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td></tr><tr><td>&#8195;<bold>3&#8242; supplementary pairing</bold></td><td><bold>3P_score</bold></td><td><bold>Supplementary pairing at the miRNA 3&#8242; end (</bold><xref ref-type="bibr" rid="bib78"><bold>Grimson et al., 2007</bold></xref><bold>)</bold></td><td><bold>42.5%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td></tr><tr><td>&#8195;Distance from stop codon</td><td>dist_stop</td><td>log<sub>10</sub>(Distance of site from stop codon)</td><td>62.4%</td><td>10.8%</td><td>8.7%</td><td>25.7%</td></tr><tr><td>&#8195;<bold>Predicted structural accessibility</bold></td><td><bold>SA</bold></td><td><bold>log</bold><sub>10</sub><bold>(Probability that a 14 nt segment centered on the match to sRNA positions 7 and 8 is unpaired)</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td></tr><tr><td>&#8195;<bold>Minimum distance</bold></td><td><bold>min_dist</bold></td><td><bold>log</bold><sub>10</sub><bold>(Minimum distance of site from stop codon or polyadenylation site) (</bold><xref ref-type="bibr" rid="bib59"><bold>Gaidatzis et al., 2007</bold></xref><bold>;</bold> <xref ref-type="bibr" rid="bib78"><bold>Grimson et al., 2007</bold></xref><bold>;</bold> <xref ref-type="bibr" rid="bib188"><bold>Majoros and Ohler, 2007</bold></xref><bold>)</bold></td><td><bold>99.9%</bold></td><td><bold>100%</bold></td><td><bold>87.4%</bold></td><td><bold>100%</bold></td></tr><tr><td>&#8195;<bold>Probability of conserved targeting</bold></td><td><bold><italic>P</italic></bold><sub>CT</sub></td><td><bold>Probability of site conservation, controlling for dinucleotide evolution and site context (</bold><xref ref-type="bibr" rid="bib58"><bold>Friedman et al., 2009</bold></xref><bold>)</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>20.8%</bold></td></tr><tr><td colspan="7"><bold>mRNA</bold></td></tr><tr><td>&#8195;5&#8242;-UTR length</td><td>len_5UTR</td><td>log<sub>10</sub>(Length of the 5&#8242; UTR)</td><td>98.2%</td><td>8.2%</td><td>4.6%</td><td>17.2%</td></tr><tr><td>&#8195;<bold>ORF length</bold></td><td><bold>len_ORF</bold></td><td><bold>log</bold><sub>10</sub><bold>(Length of the ORF)</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td></tr><tr><td>&#8195;<bold>3</bold>&#8242;-<bold>UTR length</bold></td><td><bold>len_3UTR</bold></td><td><bold>log</bold><sub>10</sub><bold>(Length of the 3&#8242; UTR) (</bold><xref ref-type="bibr" rid="bib90"><bold>Hausser et al., 2009</bold></xref><bold>)</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td></tr><tr><td>&#8195;5&#8242;-UTR AU content</td><td>AU_5UTR</td><td>Fraction of AU nucleotides in the 5&#8242; UTR</td><td>13%</td><td>38.9%</td><td>91.1%</td><td>31.3%</td></tr><tr><td>&#8195;ORF AU content</td><td>AU_ORF</td><td>Fraction of AU nucleotides in the ORF</td><td>1.2%</td><td>72.4%</td><td>28.4%</td><td>35.8%</td></tr><tr><td>&#8195;3&#8242;-UTR AU content</td><td>AU_3UTR</td><td>Fraction of AU nucleotides in the 3&#8242; UTR (<xref ref-type="bibr" rid="bib219">Robins and Press, 2005</xref>; <xref ref-type="bibr" rid="bib90">Hausser et al., 2009</xref>)</td><td>5.4%</td><td>73.3%</td><td>65.3%</td><td>80.6%</td></tr><tr><td>&#8195;<bold>3</bold>&#8242;<bold>-UTR offset-6mer sites</bold></td><td><bold>off6m</bold></td><td><bold>Number of offset-6mer sites in the 3&#8242; UTR (</bold><xref ref-type="bibr" rid="bib58"><bold>Friedman et al., 2009</bold></xref><bold>)</bold></td><td><bold>65.9%</bold></td><td><bold>89.6%</bold></td><td><bold>99.8%</bold></td><td><bold>100%</bold></td></tr><tr><td>&#8195;<bold>ORF 8mer sites</bold></td><td><bold>ORF8m</bold></td><td><bold>Number of 8mer sites in the ORF (</bold><xref ref-type="bibr" rid="bib147"><bold>Lewis et al., 2005</bold></xref><bold>;</bold> <xref ref-type="bibr" rid="bib217"><bold>Reczko et al., 2012</bold></xref><bold>)</bold></td><td><bold>99.5%</bold></td><td><bold>99.1%</bold></td><td><bold>100%</bold></td><td><bold>100%</bold></td></tr><tr><td>&#8195;ORF 7mer-m8 sites</td><td>ORF7m8</td><td>Number of 7mer-m8 sites in the ORF (<xref ref-type="bibr" rid="bib217">Reczko et al., 2012</xref>)</td><td>4.7%</td><td>4.3%</td><td>85.3%</td><td>100%</td></tr><tr><td>&#8195;ORF 7mer-A1 sites</td><td>ORF7A1</td><td>Number of 7mer-A1 sites in the ORF (<xref ref-type="bibr" rid="bib217">Reczko et al., 2012</xref>)</td><td>68.4%</td><td>34.2%</td><td>97.8%</td><td>98.4%</td></tr><tr><td>&#8195;ORF 6mer sites</td><td>ORF6m</td><td>Number of 6mer sites in the ORF (<xref ref-type="bibr" rid="bib217">Reczko et al., 2012</xref>)</td><td>91%</td><td>13.3%</td><td>0.7%</td><td>36.7%</td></tr></tbody></table><table-wrap-foot><fn><p>The feature description does not include the scaling performed (<xref ref-type="table" rid="tbl3">Table 3</xref>) to generate more comparable regression coefficients.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-05005-v1.xml">elife-05005-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.05005.023</object-id><label>Table 3.</label><caption><p>Scaling parameters used to normalize data to the (0, 1) interval</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.05005.023">http://dx.doi.org/10.7554/eLife.05005.023</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Feature</th><th colspan="2">8mer</th><th colspan="2">7mer-m8</th><th colspan="2">7mer-A1</th><th colspan="2">6mer</th></tr><tr><th>5th %</th><th>95th %</th><th>5th %</th><th>95th %</th><th>5th %</th><th>95th %</th><th>5th %</th><th>95th %</th></tr></thead><tbody><tr><td>3P_score</td><td>1.000</td><td>3.500</td><td>1.000</td><td>3.500</td><td>1.000</td><td>3.500</td><td>1.000</td><td>3.500</td></tr><tr><td>SPS</td><td>&#8722;11.130</td><td>&#8722;5.520</td><td>&#8722;11.130</td><td>&#8722;5.490</td><td>&#8722;8.410</td><td>&#8722;3.330</td><td>&#8722;8.570</td><td>&#8722;3.330</td></tr><tr><td>TA_3UTR</td><td>3.113</td><td>3.865</td><td>3.067</td><td>3.887</td><td>3.145</td><td>3.887</td><td>3.113</td><td>3.887</td></tr><tr><td>Len_3UTR</td><td>2.392</td><td>3.637</td><td>2.409</td><td>3.615</td><td>2.413</td><td>3.630</td><td>2.405</td><td>3.620</td></tr><tr><td>Len_ORF</td><td>2.788</td><td>3.753</td><td>2.773</td><td>3.729</td><td>2.773</td><td>3.730</td><td>2.775</td><td>3.731</td></tr><tr><td>Min_dist</td><td>1.415</td><td>3.113</td><td>1.491</td><td>3.096</td><td>1.431</td><td>3.117</td><td>1.477</td><td>3.106</td></tr><tr><td>Local_AU</td><td>0.308</td><td>0.814</td><td>0.277</td><td>0.782</td><td>0.342</td><td>0.801</td><td>0.295</td><td>0.772</td></tr><tr><td>SA</td><td>&#8722;4.356</td><td>&#8722;0.661</td><td>&#8722;5.218</td><td>&#8722;0.725</td><td>&#8722;4.230</td><td>&#8722;0.588</td><td>&#8722;5.082</td><td>&#8722;0.666</td></tr><tr><td>P<sub>CT</sub></td><td>0.000</td><td>0.816</td><td>0.000</td><td>0.364</td><td>0.000</td><td>0.449</td><td>0.000</td><td>0.193</td></tr></tbody></table><table-wrap-foot><fn><p>Provided are the 5th and 95th percentile values for continuous features that were scaled, after the values of the feature were appropriately transformed as indicated (<xref ref-type="table" rid="tbl1">Table 1</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-05075-v1.xml">elife-05075-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.05075.002</object-id><label>Table 1.</label><caption><p>Frequency of inclusion of species in titles and abstracts of published papers</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.05075.002">http://dx.doi.org/10.7554/eLife.05075.002</ext-link></p><p><supplementary-material id="SD1-data"><object-id pub-id-type="doi">10.7554/eLife.05075.003</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Is the species name mentioned in the title, impact statement, abstract and digest of eLife papers (July&#8211;Sept 2014)?</title><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.05075.003">http://dx.doi.org/10.7554/eLife.05075.003</ext-link></p></caption><media mime-subtype="docx" mimetype="application" xlink:href="elife-05075-data1-v1.docx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Journal</th><th align="center" rowspan="2">Issues</th><th align="center" rowspan="2"># of articles<xref ref-type="table-fn" rid="tblfn1">*</xref></th><th align="center" colspan="3">Is the species included in the title?</th><th align="center" colspan="3">If not included in the title, is the species included in the abstract?</th><th align="center" rowspan="2">Species not included in title or abstract%</th></tr><tr><th align="center">Yes</th><th align="center">No</th><th align="center">% No</th><th align="center">Yes</th><th align="center">No</th><th align="center">% No</th></tr></thead><tbody><tr><td align="center">Nature Genetics</td><td align="center">Jan&#8211;Apr 2012</td><td align="center">52</td><td align="center">19</td><td align="center">33</td><td align="center">65</td><td align="center">24</td><td align="center">9</td><td align="center">27</td><td align="center">17</td></tr><tr><td/><td align="center">June&#8211;Sept 2014</td><td align="center">60</td><td align="center">20</td><td align="center">40</td><td align="center">67</td><td align="center">32</td><td align="center">8</td><td align="center">20</td><td align="center">13</td></tr><tr><td align="center">Cell</td><td align="center">Feb&#8211;May 2012</td><td align="center">96</td><td align="center">13</td><td align="center">83</td><td align="center">90</td><td align="center">46</td><td align="center">37</td><td align="center">45</td><td align="center">39</td></tr><tr><td/><td align="center">July&#8211;Sept 2014</td><td align="center">82</td><td align="center">18</td><td align="center">64</td><td align="center">78</td><td align="center">29</td><td align="center">35</td><td align="center">55</td><td align="center">43</td></tr><tr><td align="center">Science</td><td align="center">Mar&#8211;May 2012</td><td align="center">54</td><td align="center">22</td><td align="center">32</td><td align="center">59</td><td align="center">21</td><td align="center">11</td><td align="center">34</td><td align="center">20</td></tr><tr><td/><td align="center">July&#8211;Sept 2014</td><td align="center">59</td><td align="center">29</td><td align="center">30</td><td align="center">51</td><td align="center">20</td><td align="center">10</td><td align="center">33</td><td align="center">17</td></tr><tr><td align="center">PNAS</td><td align="center">Mar&#8211;May 2012</td><td align="center">252</td><td align="center">111</td><td align="center">141</td><td align="center">56</td><td align="center">72</td><td align="center">69</td><td align="center">49</td><td align="center">27</td></tr><tr><td/><td align="center">July 2014</td><td align="center">119</td><td align="center">49</td><td align="center">70</td><td align="center">59</td><td align="center">36</td><td align="center">34</td><td align="center">49</td><td align="center">29</td></tr><tr><td align="center">PLOS Genetics</td><td align="center">Apr&#8211;May 2012</td><td align="center">107</td><td align="center">56</td><td align="center">51</td><td align="center">48</td><td align="center">46</td><td align="center">5</td><td align="center">10</td><td align="center">5</td></tr><tr><td/><td align="center">Aug&#8211;Sept 2014</td><td align="center">82</td><td align="center">44</td><td align="center">38</td><td align="center">46</td><td align="center">31</td><td align="center">7</td><td align="center">18</td><td align="center">9</td></tr><tr><td align="center">Hum Mol Genetics</td><td align="center">Apr&#8211;June 2012</td><td align="center">67</td><td align="center">26</td><td align="center">41</td><td align="center">61</td><td align="center">35</td><td align="center">6</td><td align="center">15</td><td align="center">9</td></tr><tr><td align="center">eLife</td><td align="center">Mar&#8211;June 2014</td><td align="center">105</td><td align="center">23</td><td align="center">82</td><td align="center">78</td><td align="center">34</td><td align="center">44</td><td align="center">54</td><td align="center">42</td></tr><tr><td/><td align="center">July&#8211;Sept 2014</td><td align="center">93</td><td align="center">27</td><td align="center">66</td><td align="center">71</td><td align="center">32</td><td align="center">34</td><td align="center">52</td><td align="center">37</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>Only biological articles are included.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-05166-v1.xml">elife-05166-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.05166.004</object-id><label>Table 1.</label><caption><p>Characteristics of patients included in the analysis and distribution of lineages</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.05166.004">http://dx.doi.org/10.7554/eLife.05166.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="4">Lineage</th><th rowspan="2">Overall</th><th rowspan="2">p<xref ref-type="table-fn" rid="tblfn1">*</xref></th></tr><tr><th>1</th><th>2</th><th>3</th><th>4</th></tr></thead><tbody><tr><td>Overall</td><td>269 (16.0)</td><td>74 (4.4)</td><td>205 (12.2)</td><td>1139 (67.5)</td><td>1687</td><td/></tr><tr><td colspan="7">Age</td></tr><tr><td>&#8195;&lt;20</td><td>9 (12.3)</td><td>7 (9.6)</td><td>9 (12.3)</td><td>48 (65.7)</td><td>73</td><td/></tr><tr><td>&#8195;20&#8211;29</td><td>46 (10.3)</td><td>26 (5.8)</td><td>48 (10.7)</td><td>327 (73.2)</td><td>447</td><td/></tr><tr><td>&#8195;30&#8211;39</td><td>109 (18.4)</td><td>17 (2.9)</td><td>81 (13.7)</td><td>386 (65.1)</td><td>593</td><td/></tr><tr><td>&#8195;40&#8211;49</td><td>61 (19.8)</td><td>18 (5.8)</td><td>39 (12.7)</td><td>190 (61.7)</td><td>308</td><td/></tr><tr><td>&#8195;50+</td><td>44 (16.5)</td><td>6 (2.3)</td><td>28 (10.5)</td><td>188 (70.7)</td><td>266</td><td>0.001</td></tr><tr><td colspan="7">Sex</td></tr><tr><td>&#8195;Female</td><td>130 (14.6)</td><td>47 (5.3)</td><td>94 (10.6)</td><td>617 (69.5)</td><td>888</td><td/></tr><tr><td>&#8195;Male</td><td>139 (17.4)</td><td>27 (3.4)</td><td>111 (13.9)</td><td>522 (65.3)</td><td>799</td><td>0.02</td></tr><tr><td colspan="7">Year</td></tr><tr><td>&#8195;1995&#8211;1998</td><td>55 (15.5)</td><td>8 (2.3)</td><td>29 (8.2)</td><td>263 (74.1)</td><td>355</td><td/></tr><tr><td>&#8195;1999&#8211;2001</td><td>43 (11.5)</td><td>23 (6.1)</td><td>43 (11.5)</td><td>266 (70.9)</td><td>375</td><td/></tr><tr><td>&#8195;2002&#8211;2004</td><td>80 (19.4)</td><td>22 (5.3)</td><td>54 (13.1)</td><td>257 (62.2)</td><td>413</td><td/></tr><tr><td>&#8195;2005&#8211;2007</td><td>54 (17.4)</td><td>11 (3.5)</td><td>44 (14.2)</td><td>202 (65.0)</td><td>311</td><td/></tr><tr><td>&#8195;2008&#8211;2010</td><td>37 (15.9)</td><td>10 (4.3)</td><td>35 (15.0)</td><td>151 (64.8)</td><td>233</td><td>0.004</td></tr><tr><td colspan="7">TB type</td></tr><tr><td>&#8195;Smear+</td><td>212 (17.3)</td><td>52 (4.3)</td><td>156 (12.8)</td><td>804 (65.7)</td><td>1224</td><td/></tr><tr><td>&#8195;Smear&#8722;</td><td>46 (12.1)</td><td>19 (5.0)</td><td>38 (10.0)</td><td>276 (72.8)</td><td>379</td><td/></tr><tr><td>&#8195;Extrapulmonary</td><td>11 (13.1)</td><td>3 (3.6)</td><td>11 (13.1)</td><td>59 (70.2)</td><td>84</td><td>0.1</td></tr><tr><td colspan="7">HIV status</td></tr><tr><td>&#8195;Negative</td><td>47 (10.8)</td><td>23 (5.3)</td><td>57 (13.0)</td><td>310 (70.9)</td><td>437</td><td/></tr><tr><td>&#8195;Positive</td><td>148 (19.3)</td><td>28 (3.6)</td><td>107 (13.9)</td><td>486 (63.2)</td><td>769</td><td>0.001</td></tr><tr><td colspan="7">Previous TB</td></tr><tr><td>&#8195;No</td><td>251 (16.7)</td><td>66 (4.4)</td><td>171 (11.4)</td><td>1019 (67.6)</td><td>1507</td><td/></tr><tr><td>&#8195;Yes</td><td>18 (10.0)</td><td>8 (4.4)</td><td>34 (18.9)</td><td>120 (66.7)</td><td>180</td><td>0.007</td></tr><tr><td colspan="7">Isoniazid resistance</td></tr><tr><td>&#8195;Resistant</td><td>20 (17.2)</td><td>0 (0.0)</td><td>21 (18.1)</td><td>75 (64.7)</td><td>116</td><td/></tr><tr><td>&#8195;Sensitive</td><td>244 (15.9)</td><td>74 (4.8)</td><td>181 (11.8)</td><td>1033 (67.4)</td><td>1532</td><td>0.03</td></tr><tr><td colspan="7">Residence</td></tr><tr><td>&#8195;Karonga</td><td>198 (16.4)</td><td>53 (4.4)</td><td>148 (12.3)</td><td>806 (66.9)</td><td>1205</td><td/></tr><tr><td>&#8195;Malawi</td><td>48 (16.6)</td><td>13 (4.5)</td><td>32 (11.1)</td><td>196 (67.8)</td><td>289</td><td/></tr><tr><td>&#8195;Other country</td><td>11 (11.5)</td><td>7 (7.3)</td><td>17 (17.7)</td><td>61 (63.5)</td><td>96</td><td>0.4</td></tr><tr><td colspan="7">Birth place</td></tr><tr><td>&#8195;Karonga</td><td>174 (17.0)</td><td>46 (4.5)</td><td>135 (13.2)</td><td>667 (65.3)</td><td>1022</td><td/></tr><tr><td>&#8195;Malawi</td><td>55 (16.3)</td><td>14 (4.1)</td><td>31 (9.2)</td><td>238 (70.4)</td><td>338</td><td/></tr><tr><td>&#8195;Other country</td><td>34 (11.7)</td><td>14 (4.8)</td><td>37 (12.7)</td><td>206 (70.8)</td><td>291</td><td>0.2</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>From &#935;<sup>2</sup> comparison between lineages.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-05166-v1.xml">elife-05166-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.05166.012</object-id><label>Table 2.</label><caption><p>Characteristics associated with disease due to recent infection</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.05166.012">http://dx.doi.org/10.7554/eLife.05166.012</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Characteristic</th><th colspan="2">Linked/Total</th><th>Association with links (unadjusted)</th><th rowspan="2">p (lrtest)</th><th>Adjusted for age, sex, year, lineage</th><th>Adjusted for other variables included in model<xref ref-type="table-fn" rid="tblfn2">*</xref></th><th rowspan="2">p (lrtest)</th></tr><tr><th>n/N</th><th>%</th><th>OR (95% CI)</th><th>OR (95% CI)</th><th>OR (95% CI)</th></tr></thead><tbody><tr><td>Overall</td><td>409/1074</td><td>38.1</td><td/><td/><td/><td/><td/></tr><tr><td colspan="8">Lineage</td></tr><tr><td>&#8195;1</td><td>56/183</td><td>30.6</td><td>0.76 (0.53&#8211;1.1)</td><td/><td>0.81 (0.57&#8211;1.2)</td><td>0.81 (0.57&#8211;1.2)</td><td/></tr><tr><td>&#8195;2</td><td>34/52</td><td>65.4</td><td>3.2 (1.8&#8211;5.9)</td><td/><td>3.0 (1.6&#8211;5.4)</td><td>3.2 (1.7&#8211;5.8)</td><td/></tr><tr><td>&#8195;3</td><td>58/129</td><td>45.0</td><td>1.4 (0.96&#8211;2.1)</td><td/><td>1.5 (1.0&#8211;2.2)</td><td>1.5 (1.0&#8211;2.2)</td><td/></tr><tr><td>&#8195;4</td><td>261/710</td><td>36.8</td><td>1</td><td>&lt;0.001</td><td>1</td><td>1</td><td>&lt;0.001</td></tr><tr><td colspan="8">Age</td></tr><tr><td>&#8195;&lt;20</td><td>19/36</td><td>65.8</td><td>2.9 (1.4&#8211;6.0)</td><td/><td>2.5 (1.2&#8211;5.4)</td><td>2.6 (1.2&#8211;5.6)</td><td/></tr><tr><td>&#8195;20&#8211;29</td><td>113/276</td><td>45.8</td><td>1.8 (1.2&#8211;2.7)</td><td/><td>1.6 (1.1&#8211;2.5)</td><td>1.8 (1.2&#8211;2.8)</td><td/></tr><tr><td>&#8195;30&#8211;39</td><td>152/404</td><td>39.6</td><td>1.5 (1.0&#8211;2.3)</td><td/><td>1.5 (0.99&#8211;2.2)</td><td>1.6 (1.0&#8211;2.3)</td><td/></tr><tr><td>&#8195;40&#8211;49</td><td>81/201</td><td>44.2</td><td>1.7 (1.1&#8211;2.7)</td><td/><td>1.0 (1.0&#8211;2.6)</td><td>1.7 (1.1&#8211;2.6)</td><td/></tr><tr><td>&#8195;50+</td><td>44/157</td><td>33.5</td><td>1</td><td>0.007<xref ref-type="table-fn" rid="tblfn3">&#8224;</xref></td><td>1</td><td>1</td><td>0.03<xref ref-type="table-fn" rid="tblfn3">&#8224;</xref></td></tr><tr><td colspan="8">Sex</td></tr><tr><td>&#8195;Female</td><td>229/575</td><td>39.8</td><td>1</td><td/><td/><td/><td/></tr><tr><td>&#8195;Male</td><td>180/499</td><td>36.1</td><td>0.85 (0.67&#8211;1.1)</td><td>0.05</td><td>0.93 (0.72&#8211;1.2)</td><td>0.94 (0.72&#8211;1.2)</td><td>0.4</td></tr><tr><td colspan="8">Year</td></tr><tr><td>&#8195;1999&#8211;2001</td><td>141/311</td><td>45.3</td><td>1</td><td/><td>1</td><td>1</td><td>&lt;0.001<xref ref-type="table-fn" rid="tblfn3">&#8224;</xref></td></tr><tr><td>&#8195;2002&#8211;2004</td><td>117/322</td><td>36.3</td><td>0.69 (0.50&#8211;0.95)</td><td/><td>0.73 (0.52&#8211;1.0)</td><td>0.69 (0.50&#8211;0.97)</td><td/></tr><tr><td>&#8195;2005&#8211;2007</td><td>92/244</td><td>37.7</td><td>0.73 (0.52&#8211;1.0)</td><td/><td>0.78 (0.55&#8211;1.1)</td><td>0.70 (0.49&#8211;1.0)</td><td/></tr><tr><td>&#8195;2008&#8211;2010</td><td>59/197</td><td>30.0</td><td>0.52 (0.35&#8211;0.75)</td><td>0.001<xref ref-type="table-fn" rid="tblfn3">&#8224;</xref></td><td>0.53 (0.36&#8211;0.77)</td><td>0.48 (0.32&#8211;0.70)</td><td/></tr><tr><td colspan="8">TB type</td></tr><tr><td>&#8195;Smear-positive pulmonary</td><td>312/821</td><td>38.0</td><td>1</td><td/><td>1</td><td/><td/></tr><tr><td>&#8195;Smear-negative pulmonary</td><td>97/253</td><td>38.3</td><td>1.0 (0.76&#8211;1.4)</td><td>0.9</td><td>0.95 (0.71&#8211;1.3)</td><td/><td/></tr><tr><td colspan="8">HIV status</td></tr><tr><td>&#8195;HIV&#8722;</td><td>102/283</td><td>36.0</td><td>1</td><td/><td/><td/><td/></tr><tr><td>&#8195;HIV+ no ART</td><td>173/436</td><td>39.7</td><td>1.2 (0.85&#8211;1.6)</td><td/><td>1.1 (0.75&#8211;1.5)</td><td/><td/></tr><tr><td>&#8195;HIV+ on ART</td><td>27/77</td><td>35.1</td><td>0.96 (0.56&#8211;1.6)</td><td>0.5</td><td>1.0 (0.56&#8211;1.8)</td><td/><td/></tr><tr><td colspan="8">INH resistance</td></tr><tr><td>&#8195;No</td><td>375/979</td><td>38.3</td><td>1</td><td/><td>1</td><td/><td/></tr><tr><td>&#8195;Yes</td><td>28/64</td><td>43.8</td><td>1.3 (0.75&#8211;2.1)</td><td>0.4</td><td>1.4 (0.81&#8211;2.3)</td><td/><td/></tr><tr><td>&#8195;Unknown</td><td/><td/><td/><td/><td/><td/><td/></tr><tr><td colspan="8">Recent residence</td></tr><tr><td>&#8195;Karonga</td><td>328/816</td><td>40.2</td><td>1</td><td/><td/><td>1</td><td>0.005</td></tr><tr><td>&#8195;Other Malawi</td><td>56/176</td><td>31.8</td><td>0.69 (0.49&#8211;0.98)</td><td/><td>0.58 (0.41&#8211;0.84)</td><td>0.58 (0.40&#8211;0.84)</td><td/></tr><tr><td>&#8195;Other country</td><td>16/54</td><td>29.6</td><td>0.63 (0.34&#8211;1.1)</td><td>0.04</td><td>0.48 (0.26&#8211;0.91)</td><td>0.48 (0.26&#8211;0.91)</td><td/></tr><tr><td colspan="8">Birth place</td></tr><tr><td>&#8195;Karonga</td><td>267/659</td><td>40.5</td><td>1</td><td/><td>1</td><td/><td/></tr><tr><td>&#8195;Other Malawi</td><td>81/227</td><td>35.7</td><td>0.81 (0.60&#8211;1.1)</td><td/><td>0.79 (0.57&#8211;1.1)</td><td/><td/></tr><tr><td>&#8195;Other country</td><td>59/180</td><td>32.8</td><td>0.72 (0.51&#8211;1.0)</td><td>0.1</td><td>0.67 (0.47&#8211;0.97)</td><td/><td/></tr></tbody></table><table-wrap-foot><fn><p>In this analysis individuals are defined as linked (&#8216;backwards links&#8217;) using the cut-offs described in the text and if the closest link was with a patient within the previous 5 years. Extrapulmonary, recurrent cases, and cases before 1999 were excluded. Odds ratios (OR) calculated using logistic regression.</p></fn><fn id="tblfn2"><label>*</label><p>In this model a dummy variable was used for the 32 individuals with missing data on recent residence.</p></fn><fn id="tblfn3"><label>&#8224;</label><p>Test for trend.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-05166-v1.xml">elife-05166-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.05166.013</object-id><label>Table 3.</label><caption><p>Characteristics associated with transmissibility</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.05166.013">http://dx.doi.org/10.7554/eLife.05166.013</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Characteristic</th><th colspan="2">Any Linked/Total</th><th>Association with links</th><th rowspan="2">p</th><th>Adjusted for age, sex, year, lineage, smear status</th><th rowspan="2">p (lrtest)</th></tr><tr><th>n/N</th><th>%</th><th>OR (95% CI)</th><th>OR (95% CI)</th></tr></thead><tbody><tr><td>Overall</td><td>431/1346</td><td>32.0</td><td/><td/><td/><td/></tr><tr><td colspan="7">Lineage</td></tr><tr><td>&#8195;1</td><td>59/217</td><td>27.2</td><td>0.87 (0.63&#8211;1.2)</td><td/><td>0.94 (0.66&#8211;1.3)</td><td/></tr><tr><td>&#8195;2</td><td>27/61</td><td>44.3</td><td>1.7 (1.0&#8211;2.7)</td><td/><td>1.9 (1.1&#8211;3.2)</td><td/></tr><tr><td>&#8195;3</td><td>65/154</td><td>42.2</td><td>1.6 (1.2&#8211;2.3)</td><td/><td>1.9 (1.4&#8211;2.7)</td><td/></tr><tr><td>&#8195;4</td><td>280/914</td><td>30.6</td><td>1</td><td>0.006</td><td>1</td><td>&lt;0.001</td></tr><tr><td colspan="7">Age</td></tr><tr><td>&#8195;&lt;20</td><td>20/50</td><td>40.0</td><td>2.3 (1.2&#8211;4.4)</td><td/><td>1.9 (0.98&#8211;3.7)</td><td/></tr><tr><td>&#8195;20&#8211;29</td><td>134/349</td><td>38.4</td><td>2.3 (1.5&#8211;3.3)</td><td/><td>2.2 (1.5&#8211;3.3)</td><td/></tr><tr><td>&#8195;30&#8211;39</td><td>159/490</td><td>32.5</td><td>1.7 (1.2&#8211;2.5)</td><td/><td>2.0 (1.3&#8211;2.9)</td><td/></tr><tr><td>&#8195;40&#8211;49</td><td>71/238</td><td>29.8</td><td>1.6 (1.0&#8211;2.4)</td><td/><td>1.7 (1.1&#8211;2.7)</td><td/></tr><tr><td>&#8195;50+</td><td>47/219</td><td>21.5</td><td>1</td><td>&lt;0.001</td><td>1</td><td>0.002</td></tr><tr><td colspan="7">Sex</td></tr><tr><td>&#8195;Female</td><td>239/718</td><td>33.3</td><td>1</td><td/><td>1</td><td/></tr><tr><td>&#8195;Male</td><td>192/628</td><td>30.6</td><td>0.87 (0.69&#8211;1.1)</td><td>0.2</td><td>0.93 (0.73&#8211;1.2)</td><td>0.5</td></tr><tr><td colspan="7">Year</td></tr><tr><td>&#8195;1995&#8211;1998</td><td>159/314</td><td>50.6</td><td>1</td><td/><td>1</td><td/></tr><tr><td>&#8195;1999&#8211;2001</td><td>119/345</td><td>34.5</td><td>0.49 (0.36&#8211;0.66)</td><td/><td>0.42 (0.31&#8211;0.58)</td><td/></tr><tr><td>&#8195;2002&#8211;2004</td><td>95/389</td><td>24.4</td><td>0.30 (0.22&#8211;0.41)</td><td/><td>0.27 (0.19&#8211;0.37)</td><td/></tr><tr><td>&#8195;2005&#8211;2007</td><td>58/298</td><td>19.5</td><td>0.22 (0.16&#8211;0.32)</td><td>&lt;0.001</td><td>0.20 (0.14&#8211;0.29)</td><td>&lt;0.001</td></tr><tr><td colspan="7">TB type</td></tr><tr><td>&#8195;Smear pos pulm</td><td>338/1003</td><td>33.7</td><td>1</td><td/><td>1</td><td/></tr><tr><td>&#8195;Smear neg pulm</td><td>93/343</td><td>27.1</td><td>0.72 (0.55&#8211;0.94)</td><td>0.01</td><td>0.73 (0.55&#8211;0.96)</td><td>&lt;0.001</td></tr><tr><td colspan="7">HIV status</td></tr><tr><td>&#8195;HIV&#8722;</td><td>91/318</td><td>28.6</td><td>1</td><td/><td>1</td><td/></tr><tr><td>&#8195;HIV+ no ART</td><td>170/540</td><td>31.5</td><td>1.1 (0.83&#8211;1.5)</td><td/><td>1.1 (0.81&#8211;1.6)</td><td/></tr><tr><td>&#8195;HIV+ on ART</td><td>11/48</td><td>22.9</td><td>0.70 (0.35&#8211;1.4)</td><td>0.3</td><td>1.4 (0.62&#8211;3.1)</td><td>0.6</td></tr><tr><td colspan="7">Previous TB</td></tr><tr><td>&#8195;No</td><td>391/1200</td><td>32.6</td><td>1</td><td/><td>1</td><td/></tr><tr><td>&#8195;Yes</td><td>40/146</td><td>27.4</td><td>0.77 (0.53&#8211;1.1)</td><td>0.2</td><td>0.85 (0.58&#8211;1.3)</td><td>0.4</td></tr><tr><td colspan="7">INH resistance</td></tr><tr><td>&#8195;No</td><td>402/1237</td><td>32.5</td><td>1</td><td/><td>1</td><td/></tr><tr><td>&#8195;Yes</td><td>29/100</td><td>29.0</td><td>0.86 (0.55&#8211;1.3)</td><td>0.5</td><td>0.86 (0.54&#8211;1.4)</td><td>0.5</td></tr><tr><td colspan="7">Recent residence</td></tr><tr><td>&#8195;Karonga</td><td>284/942</td><td>30.2</td><td>1</td><td/><td>1</td><td/></tr><tr><td>&#8195;Other Malawi</td><td>80/234</td><td>34.2</td><td>1.2 (0.89&#8211;1.6)</td><td/><td>1.0 (0.74&#8211;1.4)</td><td/></tr><tr><td>&#8195;Other country</td><td>20/74</td><td>27.0</td><td>0.88(0.52&#8211;1.5)</td><td>0.4</td><td>0.57 (0.33&#8211;0.98)</td><td>0.09</td></tr><tr><td colspan="7">Birth place</td></tr><tr><td>&#8195;Karonga</td><td>276/811</td><td>34.0</td><td>1</td><td/><td>1</td><td/></tr><tr><td>&#8195;Other Malawi</td><td>80/272</td><td>29.4</td><td>0.83 (0.62&#8211;1.1)</td><td/><td>0.82 (0.60&#8211;1.1)</td><td/></tr><tr><td>&#8195;Other country</td><td>64/234</td><td>27.4</td><td>0.77 (0.56&#8211;1.1)</td><td>0.2</td><td>0.71 (0.51&#8211;0.99)</td><td>0.08</td></tr></tbody></table><table-wrap-foot><fn><p>The numbers of likely transmissions (&#8216;forward links&#8217;) were compared by individual characteristics using ordered logistic regression. Extrapulmonary cases and cases occurring after 2007 were excluded.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-05255-v1.xml">elife-05255-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.05255.013</object-id><label>Table 2.</label><caption><p>Correlation between methylation levels and environment-of-origin variables (<xref ref-type="bibr" rid="bib15">Hancock et al., 2011</xref>)</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.05255.013">http://dx.doi.org/10.7554/eLife.05255.013</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Environmental variable</th><th rowspan="2">Growing temp.</th><th colspan="3">CG</th><th colspan="3">CHG</th><th colspan="3">CHH</th></tr><tr><th>r</th><th>rho</th><th>p-value</th><th>r</th><th>rho</th><th>p-value</th><th>r</th><th>rho</th><th>p-value</th></tr></thead><tbody><tr><td rowspan="2">Latitude</td><td>10</td><td>0.69</td><td>0.52</td><td>7.8E-11</td><td>&#8722;0.24</td><td>&#8722;0.19</td><td>2.7E-02</td><td>0.10</td><td>0.14</td><td>1.1E-01</td></tr><tr><td>16</td><td>0.62</td><td>0.47</td><td>3.2E-07</td><td>&#8722;0.21</td><td>&#8722;0.20</td><td>4.2E-02</td><td>0.04</td><td>&#8722;0.11</td><td>2.5E-01</td></tr><tr><td rowspan="2">Longitude</td><td>10</td><td>0.59</td><td>0.54</td><td>1.2E-11</td><td>&#8722;0.14</td><td>&#8722;0.09</td><td>3.1E-01</td><td>0.23</td><td>0.28</td><td>7.5E-04</td></tr><tr><td>16</td><td>0.55</td><td>0.53</td><td>4.4E-09</td><td>&#8722;0.12</td><td>&#8722;0.03</td><td>7.4E-01</td><td>0.14</td><td>0.15</td><td>1.2E-01</td></tr><tr><td rowspan="2">Temperature seasonality</td><td>10</td><td>0.68</td><td>0.49</td><td>1.6E-09</td><td>&#8722;0.27</td><td>&#8722;0.24</td><td>4.8E-03</td><td>0.09</td><td>0.09</td><td>2.8E-01</td></tr><tr><td>16</td><td>0.62</td><td>0.42</td><td>1.1E-05</td><td>&#8722;0.23</td><td>&#8722;0.26</td><td>6.6E-03</td><td>0.04</td><td>&#8722;0.12</td><td>2.1E-01</td></tr><tr><td rowspan="2">Max. temp. (warmest month)</td><td>10</td><td>&#8722;0.14</td><td>0.06</td><td>4.6E-01</td><td>&#8722;0.07</td><td>&#8722;0.13</td><td>1.3E-01</td><td>0.14</td><td>0.20</td><td>2.0E-02</td></tr><tr><td>16</td><td>&#8722;0.03</td><td>0.10</td><td>2.9E-01</td><td>&#8722;0.10</td><td>&#8722;0.20</td><td>3.8E-02</td><td>0.05</td><td>0.03</td><td>7.3E-01</td></tr><tr><td rowspan="2">Min. temp. (coldest month)</td><td>10</td><td>&#8722;0.70</td><td>&#8722;0.56</td><td>9.1E-13</td><td>0.27</td><td>0.21</td><td>1.2E-02</td><td>&#8722;0.07</td><td>&#8722;0.06</td><td>4.7E-01</td></tr><tr><td>16</td><td>&#8722;0.63</td><td>&#8722;0.48</td><td>2.7E-07</td><td>0.24</td><td>0.24</td><td>1.4E-02</td><td>0.00</td><td>0.19</td><td>5.6E-02</td></tr><tr><td rowspan="2">Precipitation (wettest month)</td><td>10</td><td>0.45</td><td>0.52</td><td>1.2E-10</td><td>&#8722;0.25</td><td>&#8722;0.27</td><td>1.2E-03</td><td>&#8722;0.20</td><td>&#8722;0.12</td><td>1.7E-01</td></tr><tr><td>16</td><td>0.29</td><td>0.43</td><td>4.0E-06</td><td>&#8722;0.26</td><td>&#8722;0.24</td><td>1.2E-02</td><td>&#8722;0.22</td><td>&#8722;0.19</td><td>5.8E-02</td></tr><tr><td rowspan="2">Precipitation (driest month)</td><td>10</td><td>0.31</td><td>0.40</td><td>1.5E-06</td><td>&#8722;0.33</td><td>&#8722;0.29</td><td>6.5E-04</td><td>&#8722;0.24</td><td>&#8722;0.21</td><td>1.6E-02</td></tr><tr><td>16</td><td>0.21</td><td>0.32</td><td>7.4E-04</td><td>&#8722;0.26</td><td>&#8722;0.24</td><td>1.4E-02</td><td>&#8722;0.15</td><td>&#8722;0.18</td><td>6.0E-02</td></tr><tr><td rowspan="2">Precipitation seasonality</td><td>10</td><td>0.42</td><td>0.44</td><td>7.1E-08</td><td>&#8722;0.07</td><td>&#8722;0.16</td><td>5.4E-02</td><td>0.05</td><td>0.01</td><td>9.0E-01</td></tr><tr><td>16</td><td>0.36</td><td>0.37</td><td>1.2E-04</td><td>&#8722;0.13</td><td>&#8722;0.16</td><td>1.1E-01</td><td>0.01</td><td>&#8722;0.01</td><td>9.1E-01</td></tr><tr><td rowspan="2">PAR (spring)</td><td>10</td><td>0.04</td><td>0.22</td><td>8.9E-03</td><td>0.20</td><td>0.18</td><td>3.7E-02</td><td>0.24</td><td>0.23</td><td>7.3E-03</td></tr><tr><td>16</td><td>0.03</td><td>0.18</td><td>6.6E-02</td><td>0.27</td><td>0.21</td><td>3.5E-02</td><td>0.38</td><td>0.35</td><td>2.8E-04</td></tr><tr><td rowspan="2">Length of growing season</td><td>10</td><td>&#8722;0.59</td><td>&#8722;0.57</td><td>5.5E-13</td><td>0.24</td><td>0.23</td><td>7.3E-03</td><td>&#8722;0.16</td><td>&#8722;0.18</td><td>3.3E-02</td></tr><tr><td>16</td><td>&#8722;0.58</td><td>&#8722;0.54</td><td>4.0E-09</td><td>0.23</td><td>0.21</td><td>3.0E-02</td><td>&#8722;0.04</td><td>0.01</td><td>8.9E-01</td></tr><tr><td rowspan="2">No. consecutive cold days</td><td>10</td><td>0.60</td><td>0.53</td><td>4.0E-11</td><td>&#8722;0.19</td><td>&#8722;0.13</td><td>1.2E-01</td><td>0.17</td><td>0.28</td><td>1.1E-03</td></tr><tr><td>16</td><td>0.57</td><td>0.53</td><td>4.2E-09</td><td>&#8722;0.17</td><td>&#8722;0.09</td><td>3.7E-01</td><td>0.10</td><td>0.08</td><td>4.1E-01</td></tr><tr><td rowspan="2">No. consecutive frost-free days</td><td>10</td><td>&#8722;0.59</td><td>&#8722;0.49</td><td>1.2E-09</td><td>0.29</td><td>0.27</td><td>1.5E-03</td><td>0.02</td><td>0.03</td><td>7.1E-01</td></tr><tr><td>16</td><td>&#8722;0.51</td><td>&#8722;0.39</td><td>4.9E-05</td><td>0.30</td><td>0.30</td><td>1.6E-03</td><td>0.07</td><td>0.13</td><td>1.9E-01</td></tr><tr><td rowspan="2">Relative humidity (spring)</td><td>10</td><td>0.62</td><td>0.47</td><td>5.6E-09</td><td>&#8722;0.23</td><td>&#8722;0.18</td><td>3.9E-02</td><td>0.09</td><td>0.06</td><td>4.5E-01</td></tr><tr><td>16</td><td>0.53</td><td>0.37</td><td>1.2E-04</td><td>&#8722;0.20</td><td>&#8722;0.26</td><td>7.6E-03</td><td>0.04</td><td>&#8722;0.08</td><td>4.3E-01</td></tr><tr><td rowspan="2">Daylength (spring)</td><td>10</td><td>0.69</td><td>0.50</td><td>7.2E-10</td><td>&#8722;0.27</td><td>&#8722;0.21</td><td>1.4E-02</td><td>0.08</td><td>0.05</td><td>5.7E-01</td></tr><tr><td>16</td><td>0.63</td><td>0.41</td><td>1.5E-05</td><td>&#8722;0.23</td><td>&#8722;0.29</td><td>2.7E-03</td><td>0.04</td><td>&#8722;0.17</td><td>8.7E-02</td></tr><tr><td rowspan="2">Aridity</td><td>10</td><td>0.53</td><td>0.49</td><td>8.4E-10</td><td>&#8722;0.35</td><td>&#8722;0.31</td><td>1.9E-04</td><td>&#8722;0.18</td><td>&#8722;0.21</td><td>1.3E-02</td></tr><tr><td>16</td><td>0.43</td><td>0.42</td><td>8.4E-06</td><td>&#8722;0.28</td><td>&#8722;0.24</td><td>1.3E-02</td><td>&#8722;0.13</td><td>&#8722;0.20</td><td>3.8E-02</td></tr></tbody></table><table-wrap-foot><fn><p>r = Pearson's correlation, rho = Spearman's rank correlation, p-value = significance of rho.</p></fn><fn><p>PAR = photosynthetically active radiation.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-05423-v2.xml">elife-05423-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.05423.007</object-id><label>Table 2.</label><caption><p>Mapping of <italic>me69</italic> on chromosome I</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.05423.007">http://dx.doi.org/10.7554/eLife.05423.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">No. F2s<xref ref-type="table-fn" rid="tblfn4">*</xref></th><th>WBVar 00240 394<xref ref-type="table-fn" rid="tblfn5">&#8224;</xref></th><th>WBVar 00240 397</th><th>WBVar 00240 399</th><th>WBVar 00155 231</th><th>WBVar 00240 416</th><th>WBVar 00240 407</th><th>WBVar 00159 097</th><th>WBVar 00240 414</th><th>WBVar 00161 629</th></tr><tr><th>825026</th><th>5482531</th><th>6351803</th><th>8646304</th><th>10614690</th><th>11472093</th><th>12433167</th><th>13066381</th><th>14154889</th></tr></thead><tbody><tr><td>16</td><td>B/B</td><td/><td/><td/><td>B/B</td><td/><td/><td/><td>B/B</td></tr><tr><td>6</td><td>H/B</td><td/><td/><td/><td>B/B</td><td/><td/><td/><td>B/B</td></tr><tr><td>2</td><td>H/B</td><td>B/B</td><td>B/B</td><td>B/B</td><td>B/B</td><td>B/B</td><td>B/B</td><td>H/B</td><td>H/B</td></tr><tr><td>3</td><td>B/B</td><td>B/B</td><td>B/B</td><td>B/B</td><td>B/B</td><td>B/B</td><td>B/B</td><td>H/B</td><td>H/B</td></tr><tr><td>1</td><td>H/B</td><td>H/B</td><td>H/B</td><td>B/B</td><td>B/B</td><td>B/B</td><td>B/B</td><td>B/B</td><td>B/B</td></tr></tbody></table><table-wrap-foot><fn><p>F2 males from the cross described in <xref ref-type="table" rid="tbl1">Table 1</xref> were scored for SNPs across chromosome I. Animals were either homozygous Bristol (B/B) or heterozygous for the Hawaiian allele (H/B) at each SNP.</p></fn><fn id="tblfn4"><label>*</label><p>Number of F2 males showing each pattern.</p></fn><fn id="tblfn5"><label>&#8224;</label><p>SNP designation and genomic position on chromosome I. <xref ref-type="bibr" rid="bib98">Wicks et al. (2001)</xref>; <xref ref-type="bibr" rid="bib102">WormBase</xref>.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-05604-v1.xml">elife-05604-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl4" position="float"><object-id pub-id-type="doi">10.7554/eLife.05604.031</object-id><label>Table 4.</label><caption><p>PCA comparison of GSL profiles produced by GSL genotypes from synthetic laboratory population grown in the chamber and all three environments</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.05604.031">http://dx.doi.org/10.7554/eLife.05604.031</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Environment</th><th colspan="2">PCA1 = 48.5%</th><th colspan="2">PCA2 = 29%</th><th colspan="2">PCA3 = 16%</th><th colspan="2">PCA4 = 6%</th></tr><tr><th>R</th><th>p value</th><th>R</th><th>p value</th><th>R</th><th>p value</th><th>R</th><th>p value</th></tr></thead><tbody><tr><td>Chamber</td><td>1.00</td><td/><td>1.00</td><td/><td>1.00</td><td/><td>1.00</td><td/></tr><tr><td>UCD2012</td><td>0.97</td><td>&lt;0.001</td><td>0.97</td><td>&lt;0.001</td><td>0.82</td><td>&lt;0.001</td><td>0.96</td><td>&lt;0.001</td></tr><tr><td>UWY2011</td><td>0.91</td><td>&lt;0.001</td><td>0.95</td><td>&lt;0.001</td><td>0.74</td><td>&lt;0.001</td><td>0.97</td><td>&lt;0.001</td></tr><tr><td>UWY2012</td><td>0.90</td><td>&lt;0.001</td><td>0.91</td><td>&lt;0.001</td><td>0.85</td><td>&lt;0.001</td><td>0.86</td><td>&lt;0.001</td></tr></tbody></table><table-wrap-foot><fn><p>Glucosinolate analysis was conducted on the 17 genotypes within a Long-day growth chamber (16 hr light) set to match the median light regime for the three environmments. Principal component analysis was conducted on the mean glucosinolate accumulation for the aliphatic glucosinoles within the chamber environment. This creates a set of mathematical descriptors of the chemotype variation across the 17 genotypes. The first three eigenvectors were used to generate scores from the lsmeans of the glucosinolates across the 17 glucosinolate genotypes independently for the chamber and three different field environments values. These scores were then correlated to test if the GSL profiles were similar or not across the environments. The R of the correlation to the Chamber scores for the 17 genotypes for each of the three PCA vectors are shown in conjunction with the p value as determined by Pearson correlation. To the right of each PCA vector label is shown the fraction of total variance approximated by the given vector. In total, the four vectors describe &gt;99% of the GSL profile variance.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-05607-v2.xml">elife-05607-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.05607.011</object-id><label>Table 5.</label><caption><p>Correlation between human GWAS candidate gene expression in mouse liver and adipose tissue with hepatic TG level</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.05607.011">http://dx.doi.org/10.7554/eLife.05607.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2">Liver</th><th colspan="2">Adipose</th></tr><tr><th>r</th><th>p</th><th>r</th><th>p</th></tr></thead><tbody><tr><td><italic>Pnpla3</italic></td><td>0.07</td><td>0.423</td><td>&#8722;0.04</td><td>0.645</td></tr><tr><td><italic>Gckr</italic></td><td>0.19</td><td>0.044<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>0.14</td><td>0.156</td></tr><tr><td><italic>Ncan</italic></td><td>&#8722;0.10</td><td>0.316</td><td>0.37</td><td>6.1 &#215; 10-5<xref ref-type="table-fn" rid="tblfn1">*</xref></td></tr><tr><td><italic>Tm6sf2</italic></td><td>0.15</td><td>0.123</td><td>&#8722;0.23</td><td>0.012<xref ref-type="table-fn" rid="tblfn1">*</xref></td></tr><tr><td><italic>Lyplal1</italic></td><td>0.27</td><td>0.003<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>&#8722;0.12</td><td>0.228</td></tr><tr><td><italic>Trib1</italic></td><td>&#8722;0.10</td><td>0.313</td><td>0.24</td><td>0.012<xref ref-type="table-fn" rid="tblfn1">*</xref></td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>Denotes p &lt; 0.05.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-05660-v2.xml">elife-05660-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.05660.005</object-id><label>Table 1.</label><caption><p>Summary of data from electrophysiological recordings and locomotion analyses</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.05660.005">http://dx.doi.org/10.7554/eLife.05660.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2"/><th rowspan="2">Evoked EPSC Amp. (nA)</th><th colspan="2">Endogenous EPSC</th><th rowspan="2">Locomotion speed (&#181;m/s)</th></tr><tr><th>Frequency (Hz)</th><th>Amp. (pA)</th></tr></thead><tbody><tr><td>Wild type (N2)</td><td/><td>3.2 &#177; 0.2 (n = 23)</td><td>49.8 &#177; 2.5 (n = 30)</td><td>22.4 &#177; 1.0</td><td>140 &#177; 8 (n = 65)</td></tr><tr><td rowspan="30"><italic>unc-26 (s1710)</italic></td><td>&#8211;</td><td>0.9 &#177; 0.1 (n = 15)<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>12.0 &#177; 1.5 (n = 16)<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>20.7 &#177; 0.6</td><td>28 &#177; 3 (n = 65)<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td></tr><tr><td><italic>Si[Prab-3::unc-26::gfp]</italic></td><td>3.1 &#177; 0.2 (n = 16)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>50.8 &#177; 5.4 (n = 16)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>23.2 &#177; 1.4</td><td>139 &#177; 8 (n = 60)<xref ref-type="table-fn" rid="tblfn5">#</xref></td></tr><tr><td><italic>Si[Prab-3::unc-26&#8710;PRD::gfp]</italic></td><td>3.4 &#177; 0.3 (n = 16)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>47.4 &#177; 5.0 (n = 17)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>24.8 &#177; 1.3</td><td>141 &#177; 10 (n = 60)<xref ref-type="table-fn" rid="tblfn5">#</xref></td></tr><tr><td><italic>Ex[Psnb-1::mSYJ1&#8710;PRD]</italic></td><td>3.1 &#177; 0.3 (n = 7)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>49.6 &#177; 6.1 (n = 11)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>21.5 &#177; 0.6</td><td>143 &#177; 10 (n = 37)<xref ref-type="table-fn" rid="tblfn5">#</xref></td></tr><tr><td><italic>Ex[Prab-3::gfp::unc-26(C378S,D380N)]</italic></td><td>3.5 &#177; 0.2 (n = 10)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>53.1 &#177; 6.6 (n = 10)</td><td>23.8 &#177; 1.3</td><td>135 &#177; 9 (n = 60)<xref ref-type="table-fn" rid="tblfn5">#</xref></td></tr><tr><td><italic>Ex[Prab-3::gfp::unc-26&#8710;PRD(C378S,D380N)]</italic></td><td>3.2 &#177; 0.3 (n = 10)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>50.7 &#177; 5.7 (n = 10)</td><td>24.2 &#177; 0.9</td><td>130 &#177; 9 (n = 60)<xref ref-type="table-fn" rid="tblfn5">#</xref></td></tr><tr><td><italic>Ex[Prab-3::gfp::unc-26(D716A)]</italic></td><td>0.8 &#177; 0.1 (n = 14)</td><td>10.0 &#177; 1.8 (N = 15)</td><td>20.9 &#177; 0.8</td><td>28 &#177; 3 (n = 60)</td></tr><tr><td><italic>Ex[Prab-3::gfp::unc-26&#8710;Sac1]</italic></td><td>1.2 &#177; 0.2 (n = 10)</td><td>7.2 &#177; 1.9 (N = 10)</td><td>21.3 &#177; 1.2</td><td>31 &#177; 3 (n = 60)</td></tr><tr><td><italic>Ex[Prab-3::unc-26Sac1 + Prab-3::unc-26&#8710;Sac1]</italic></td><td>1.0 &#177; 0.2 (n = 10)</td><td>16.1 &#177; 1.8 (n = 10)</td><td>20.4 &#177; 0.5</td><td>32 &#177; 3 (n = 60)</td></tr><tr><td><italic>Ex[Prab-3::unc-26Sac1::IntN + Prab-3::IntC::unc-26&#8710;Sac1]</italic></td><td>3.2 &#177; 0.4 (n = 10)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>53.1 &#177; 7.1 (n = 10)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>24.8 &#177; 1.2</td><td>100 &#177; 7 (n = 60)<xref ref-type="table-fn" rid="tblfn5">#</xref></td></tr><tr><td><italic>Ex[Prab-3::unc26&#8710;Sac1::rab-3]</italic></td><td>1.0 &#177; 0.2 (n = 10)</td><td>13.2 &#177; 2.3 (n = 10)</td><td>20.4 &#177; 0.9</td><td/></tr><tr><td><italic>Ex[Prab-3::unc-26&#8710;Sac1::snb-1]</italic></td><td>1.4 &#177; 0.2 (n = 11)</td><td>17.9 &#177; 2.6 (n = 11)</td><td>21.1 &#177; 0.8</td><td/></tr><tr><td><italic>Ex[Prab-3::bem1PX::unc-26&#8710;Sac1]</italic></td><td>0.8 &#177; 0.1 (n = 7)</td><td>6.3 &#177; 0.6 (n = 7)</td><td>18.8 &#177; 1.2</td><td/></tr><tr><td><italic>Ex[Prab-3::plc&#8706;PH::unc-26&#8710;Sac1]</italic></td><td>1.0 &#177; 0.2 (n = 7)</td><td>8.5 &#177; 1.8 (n = 7)</td><td>18.7 &#177; 1.1</td><td/></tr><tr><td><italic>Ex[Prab-3::btkPH::unc-26&#8710;Sac1]</italic></td><td>1.2 &#177; 0.2 (n = 11)</td><td>11.4 &#177; 1.2 (n = 11)</td><td>18.4 &#177; 0.8</td><td/></tr><tr><td><italic>Ex[Prab-3::apa-2::unc-26&#8710;Sac1]</italic></td><td>1.1 &#177; 0.1 (n = 11)</td><td>12.8 &#177; 1.8 (n = 11)</td><td>19.4 &#177; 0.8</td><td/></tr><tr><td><italic>Ex[Prab-3::apb-1::unc-26&#8710;Sac1]</italic></td><td>0.9 &#177; 0.1 (n = 11)</td><td>13.2 &#177; 2.5 (n = 11)</td><td>18.2 &#177; 1.5</td><td/></tr><tr><td><italic>Ex[Prab-3::apm-2::unc-26&#8710;Sac1]</italic></td><td>1.1 &#177; 0.1 (n = 10)</td><td>15.3 &#177; 2.7 (n = 10)</td><td>21.0 &#177; 0.9</td><td/></tr><tr><td><italic>Ex[Prab-3::aps-2::unc-26&#8710;Sac1]</italic></td><td>1.3 &#177; 0.2 (n = 9)</td><td>12.7 &#177; 1.4 (n = 9)</td><td>20.1 &#177; 0.9</td><td/></tr><tr><td><italic>Ex[Prab-3::unc-57::unc-26&#8710;Sac1]</italic></td><td>2.5 &#177; 0.3 (n = 11)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>28.0 &#177; 4.1 (n = 11)<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td><td>23.0 &#177; 1.4</td><td/></tr><tr><td><italic>Ex[Prab-3::dyn-1::unc-26&#8710;Sac1]</italic></td><td>1.3 &#177; 0.3 (n = 10)</td><td>14.4 &#177; 2.9 (n = 10)</td><td>19.4 &#177; 0.9</td><td/></tr><tr><td><italic>Ex[Prab-3::itsn-1::unc-26&#8710;Sac1]</italic></td><td>0.9 &#177; 0.1 (n = 9)</td><td>13.9 &#177; 1.4 (n = 9)</td><td>21.5 &#177; 1.4</td><td/></tr><tr><td><italic>Ex[Prab-3::unc-57::unc-26&#8710;Sac1&#8710;PRD]</italic></td><td>3.0 &#177; 0.3 (n = 11)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>36.7 &#177; 5.4 (n = 11)<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td><td>21.7 &#177; 1.3</td><td/></tr><tr><td><italic>Ex[Prab-3::unc-57::unc-26&#8710;Sac1(D716A)]</italic></td><td>0.8 &#177; 0.2 (n = 9)</td><td>13.0 &#177; 2.3 (n = 9)</td><td>21.0 &#177; 0.7</td><td/></tr><tr><td><italic>Ex[Prab-3::unc-57BAR::unc-26&#8710;Sac1]</italic></td><td>2.9 &#177; 0.2 (n = 12)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>23.9 &#177; 2.5 (n = 12)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>21.3 &#177; 0.8</td><td/></tr><tr><td><italic>Ex[Prab-3::rEndoBAR::unc-26&#8710;Sac1]</italic></td><td>3.1 &#177; 0.4 (n = 13)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>28.5 &#177; 4.6 (n = 13)<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>24.1 &#177; 1.4</td><td/></tr><tr><td><italic>Ex[Prab-3::mAmphBAR::unc-26&#8710;Sac1]</italic></td><td>1.4 &#177; 0.2 (n = 10)</td><td>19.2 &#177; 2.3 (n = 10)</td><td>20.8 &#177; 0.8</td><td/></tr><tr><td><italic>Ex[Prab-3::mNadrin2BAR::unc-26&#8710;Sac1]</italic></td><td>1.7 &#177; 0.2 (n = 11)<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>16.1 &#177; 3.1 (n = 11)</td><td>23.8 &#177; 1.9</td><td/></tr><tr><td><italic>Ex[Prab-3::rEndoBAR&#8710;N::unc-26&#8710;Sac1]</italic></td><td>1.3 &#177; 0.2 (n = 12)</td><td>9.1 &#177; 0.9 (n = 12)</td><td>18.9 &#177; 0.4</td><td/></tr><tr><td><italic>Ex[Prab-3::rEndoBAR(K76E,K78E)::unc-26&#8710;Sac1]</italic></td><td>1.5 &#177; 0.2 (n = 10)</td><td>13.5 &#177; 1.3 (n = 10)</td><td>19.3 &#177; 0.7</td><td/></tr><tr><td rowspan="3">N2</td><td><italic>Prab-3::unc-26&#8710;PRD(D716A)</italic> overexpression</td><td>1.5 &#177; 0.3 (n = 9)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>24.6 &#177; 3.8 (n = 10)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>25.7 &#177; 0.9</td><td/></tr><tr><td><italic>Prab-3::unc-26&#8710;PRD</italic> overexpression</td><td>2.9 &#177; 0.3 (n = 9)</td><td>53.5 &#177; 4.6 (n = 9)</td><td>25.4 &#177; 1.3</td><td/></tr><tr><td><italic>Prab-3::unc-26&#8710;Sac1&#8710;PRD(D716A)</italic> overexpression</td><td>3.5 &#177; 0.3 (n = 10)</td><td>49.0 &#177; 7.5 (n = 10)</td><td>25.9 &#177; 1.9</td><td/></tr><tr><td rowspan="3"><italic>unc-57(e406); unc-26(s1710)</italic></td><td><italic>&#8211;</italic></td><td>0.8 &#177; 0.2 (n = 9)<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>8.6 &#177; 0.8 (n = 10)<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>21.9 &#177; 1.1</td><td>27 &#177; 3 (n = 60)<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td></tr><tr><td><italic>Si[Psnb-1::unc-57&#8710;SH3::mCherry]; Si[Prab-3::unc-26&#8710;PRD::gfp]</italic></td><td>3.2 &#177; 0.2 (n = 9)<xref ref-type="table-fn" rid="tblfn6">&#182;</xref></td><td>50.3 &#177; 4.1 (n = 9)<xref ref-type="table-fn" rid="tblfn6">&#182;</xref></td><td>23.1 &#177; 1.0</td><td>142 &#177; 9 (n = 62)<xref ref-type="table-fn" rid="tblfn6">&#182;</xref></td></tr><tr><td><italic>Si[Psnb-1::rEndoBAR::unc-26&#8710;Sac1&#8710;PRD]</italic></td><td>3.0 &#177; 0.3 (n = 10)<xref ref-type="table-fn" rid="tblfn6">&#182;</xref></td><td>50.9 &#177; 4.1 (n = 10)<xref ref-type="table-fn" rid="tblfn6">&#182;</xref></td><td>26.5 &#177; 1.1</td><td>109 &#177; 4 (n = 68)<xref ref-type="table-fn" rid="tblfn6">&#182;</xref></td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>p &lt; 0.001 when compared with N2.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>p &lt; 0.0001 when compared with N2.</p></fn><fn id="tblfn3"><label>&#8225;</label><p>p &lt; 0.05 when compared with <italic>unc-26</italic> mutant.</p></fn><fn id="tblfn4"><label>&#167;</label><p>p &lt; 0.001 when compared with <italic>unc-26</italic> mutant.</p></fn><fn id="tblfn5"><label>#</label><p>p &lt; 0.0001 when compared with <italic>unc-26</italic> mutant.</p></fn><fn id="tblfn6"><label>&#182;</label><p>p &lt; 0.0001 when compared with <italic>unc-57; unc-26</italic> double mutants.</p></fn><fn><p>Si: single-copy transgene (MosSci insertion).</p></fn><fn><p>Ex: extrachromosomal array.</p></fn><fn><p>&#8216;Amp.&#8217; indicates amplitude.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-05914-v1.xml">elife-05914-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.05914.011</object-id><label>Table 1.</label><caption><p>Analysis and stereological parameters of tibia proximal region in WT and gal-8 Tg mice</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.05914.011">http://dx.doi.org/10.7554/eLife.05914.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2">In vivo CT</th><th colspan="2">In vitro CT</th></tr><tr><th>WT (n = 7)</th><th>Tg (n = 7)</th><th>WT (n = 5)</th><th>Tg (n = 5)</th></tr></thead><tbody><tr><td>BV/TV</td><td>0.40 &#177; 0.02</td><td>0.25 &#177; 0.03**</td><td>0.12 &#177; 0.01</td><td>0.04 &#177; 0.01**</td></tr><tr><td>Tb.N (1/mm)</td><td>3.19 &#177; 0.02</td><td>1.86 &#177; 0.25**</td><td>3.86 &#177; 0.51</td><td>1.68 &#177; 0.34**</td></tr><tr><td>Tb.Sp (mm)</td><td>0.19 &#177; 0.01</td><td>0.48 &#177; 0.08**</td><td>0.24 &#177; 0.03</td><td>0.68 &#177; 0.18*</td></tr><tr><td>BMD (%)</td><td>100% &#177; 15%</td><td>52% &#177; 5%**</td><td>100% &#177; 8%</td><td>68% &#177; 6%**</td></tr></tbody></table><table-wrap-foot><fn><p>14-week-old WT and Tg mice (n = 7 each group) were scanned using a small animal in vivo &#956;CT scanner. Tibial bones were removed from 16-week-old WT and Tg mice (n = 5 each group) and scanned using an in vitro CT scanner. Analysis was performed on the proximal region of the tibia. The parameters calculated are Tb.N (trabecular number), Tb.Sp (trabecular separation), BV/TV (bone volume/tissue volume), and BMD (bone mineral density). Results shown are mean values &#177; SEM. BMD is given as relative to the average BMD of WT mice (**p &lt; 0.01 vs WT mice).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06303-v2.xml">elife-06303-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.06303.005</object-id><label>Table 1.</label><caption><p>Atypical cells: quantitation of denticle polarities in relation to neighbouring cells showing the effect of the Ds/Ft system</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06303.005">http://dx.doi.org/10.7554/eLife.06303.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="4">Wild type</th><th colspan="4"><italic>ds</italic><sup><italic>&#8722;</italic></sup> <italic>ft</italic><sup><italic>&#8722;</italic></sup></th></tr><tr><th rowspan="2">Anterior neighbour</th><th colspan="2">Denticle polarity of atypical Row 2 cells<xref ref-type="table-fn" rid="tblfn1">&#8224;</xref></th><th rowspan="2">Posterior neighbour</th><th rowspan="2">Anterior neighbour</th><th colspan="2">Denticle polarity of atypical Row 2 cells<xref ref-type="table-fn" rid="tblfn3">&#167;</xref></th><th rowspan="2">Posterior neighbour</th></tr><tr><th>Anteriorly</th><th>Posteriorly</th><th>Anteriorly</th><th>Posteriorly</th></tr></thead><tbody><tr><td>T1 cell</td><td>0</td><td><bold>52</bold></td><td>Row 3 cell</td><td>T1 cell</td><td>16</td><td>21</td><td>Row 3 cell</td></tr><tr><td>Row 2 cell</td><td>0</td><td><bold>35</bold></td><td>Row 3 cell</td><td>Row 2 cell</td><td>14</td><td>13</td><td>Row 3 cell</td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Anterior neighbour</th><th colspan="2">Denticle polarity of atypical Row 4 cells<xref ref-type="table-fn" rid="tblfn2">&#8225;</xref></th><th rowspan="2">Posterior neighbour</th><th rowspan="2">Anterior neighbour</th><th colspan="2">Denticle polarity of atypical Row 4 cells<xref ref-type="table-fn" rid="tblfn4">&#182;</xref></th><th rowspan="2">Posterior neighbour</th></tr><tr><th>Anteriorly</th><th>Posteriorly</th><th>Anteriorly</th><th>Posteriorly</th></tr></thead><tbody><tr><td>Row 3 cell</td><td><bold>110</bold></td><td>8*</td><td>T2 cell</td><td>Row 3 cell</td><td>54*</td><td>37</td><td>T2 cell</td></tr><tr><td>Row 3 cell</td><td>8**</td><td><bold>41</bold></td><td>Row 4 cell</td><td>Row 3 cell</td><td>24</td><td>20**</td><td>Row 4 cell</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>&#8224;</label><p>Denticles of 29 atypical cells.</p></fn><fn id="tblfn2"><label>&#8225;</label><p>Denticles of 27 atypical cells. 8 of 8* (and 6 of 8**) were predenticles located in ambiguous positions and their denticles were arbitrarily allocated to those classes favouring the null hypothesis. Fisher exact test p-value: &lt;2.2<sup>&#8722;16</sup>.</p></fn><fn id="tblfn3"><label>&#167;</label><p>Denticles of 23 atypical cells. Fisher exact test p-value: 0.6135.</p></fn><fn id="tblfn4"><label>&#182;</label><p>Denticles of 24 atypical cells. 1 of 54* (and 1 of 20**) were predenticles located in ambiguous positions and their denticles were arbitrarily allocated to those classes disfavouring the null hypothesis. Fisher exact test p-value: 0.7104.</p></fn><fn><p>Numbers in bold emphasise the main result of the table that is, the effect of neighbours on denticle polarities in the wildtype. This effect does not exist in the mutant larvae.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06322-v1.xml">elife-06322-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.06322.012</object-id><label>Table 1.</label><caption><p>Clinical characteristics and genetic causes of 134 index patients with PK</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06322.012">http://dx.doi.org/10.7554/eLife.06322.012</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="5">Genetic causes of 134 index patients with PK</th></tr><tr><th><italic>MVK</italic> (39<xref ref-type="table-fn" rid="tblfn1">*</xref>)</th><th><italic>PMVK</italic> (9)</th><th><italic>MVD</italic> (62<xref ref-type="table-fn" rid="tblfn1">*</xref>)</th><th><italic>FDPS</italic> (4)</th><th>Unknown (21)</th></tr></thead><tbody><tr><td colspan="6">Sex</td></tr><tr><td>&#8195;Male</td><td>27</td><td>9</td><td>38</td><td>1</td><td>17</td></tr><tr><td>&#8195;Female</td><td>12</td><td>0</td><td>24</td><td>3</td><td>4</td></tr><tr><td colspan="6">Number of lesions</td></tr><tr><td>&#8195;0&#8211;10</td><td>4</td><td>4</td><td>0</td><td>0</td><td>6</td></tr><tr><td>&#8195;10&#8211;100</td><td>20</td><td>5</td><td>13</td><td>0</td><td>5</td></tr><tr><td>&#8195;100&#8211;500</td><td>5</td><td>0</td><td>39</td><td>0</td><td>5</td></tr><tr><td>&#8195;&gt;500<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>10</td><td>0</td><td>10</td><td>4</td><td>5</td></tr><tr><td colspan="6">Diameter of lesions</td></tr><tr><td>&#8195;Minimum</td><td>2 mm</td><td>5 mm</td><td>1 mm</td><td>1 mm</td><td>1 mm</td></tr><tr><td>&#8195;Maximum</td><td>20 cm</td><td>5 cm</td><td>2 cm</td><td>1 cm</td><td>2 cm</td></tr><tr><td colspan="6">Age of onset</td></tr><tr><td>&#8195;At birth</td><td>0</td><td>0</td><td>0</td><td>0</td><td>3</td></tr><tr><td>&#8195;0&#8211;20</td><td>23</td><td>4</td><td>20</td><td>1</td><td>0</td></tr><tr><td>&#8195;20&#8211;40</td><td>12</td><td>3</td><td>28</td><td>1</td><td>9</td></tr><tr><td>&#8195;40&#8211;60</td><td>3</td><td>1</td><td>12</td><td>2</td><td>5</td></tr><tr><td>&#8195;&gt;60</td><td>1</td><td>1</td><td>2</td><td>0</td><td>4</td></tr><tr><td colspan="6">Variants of PK</td></tr><tr><td>&#8195;DSAP/DSP</td><td>26</td><td>0</td><td>56</td><td>4</td><td>10</td></tr><tr><td>&#8195;SFP</td><td>0</td><td>0</td><td>6</td><td>0</td><td>0</td></tr><tr><td>&#8195;PM</td><td>23</td><td>9</td><td>0</td><td>0</td><td>6</td></tr><tr><td>&#8195;HPM</td><td>13</td><td>3</td><td>1</td><td>0</td><td>0</td></tr><tr><td>&#8195;Giant plaque of PPt</td><td>19</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td>&#8195;Genital PK (localized)</td><td>0</td><td>4</td><td>0</td><td>0</td><td>0</td></tr><tr><td>&#8195;Porokeratoma</td><td>0</td><td>5</td><td>0</td><td>0</td><td>0</td></tr><tr><td>&#8195;LP</td><td>1</td><td>1</td><td>3</td><td>0</td><td>5</td></tr><tr><td colspan="6">Comorbidity</td></tr><tr><td>&#8195;Psoriasis vulgaris</td><td>4</td><td>0</td><td>2</td><td>0</td><td>0</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>One PK patient (proband of family-28, female), who has both the mutation c. 235G&gt;A (<italic>MVK</italic>) and the mutation c. 746T&gt;C (<italic>MVD</italic>), was included in both <italic>MVK</italic> and <italic>MVD</italic> groups.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>The number of lesions is more than 500.</p></fn><fn><p>DSAP, disseminated superficial actinic porokeratosis; DSP, disseminated superficial porokeratosis; HPM, hyperkeratotic porokeratosis, LP, linear porokeratosis, PK, porokeratosis; PM, porokeratosis of Mibelli, PPt, porokeratosis ptychotropica; SFP, solar facial porokeratosis.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06328-v2.xml">elife-06328-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.06328.012</object-id><label>Table 1.</label><caption><p>Acyl-chain composition of PC species with changed levels in LPCAT3-deficient mice</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06328.012">http://dx.doi.org/10.7554/eLife.06328.012</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">PC subclass</th><th rowspan="2">Acyl-chain composition</th><th colspan="2">LPCAT3+/+ peak area</th><th colspan="2">LPCAT3&#8722;/&#8722; peak area</th><th rowspan="2">KO/WT (%)</th><th rowspan="2">p value</th></tr><tr><th>Mean</th><th>SEM</th><th>Mean</th><th>SEM</th></tr></thead><tbody><tr><td rowspan="2">34:4</td><td>14:0&#8211;20:4</td><td>32,607</td><td>1253</td><td>11,609</td><td>835</td><td>36</td><td>&lt;0.0001</td></tr><tr><td>16:1&#8211;18:3</td><td>42,145</td><td>2574</td><td>41,340</td><td>4555</td><td>98</td><td>0.8816</td></tr><tr><td rowspan="3">36:4</td><td>16:0&#8211;20:4</td><td>1,037,887</td><td>21,426</td><td>917,642</td><td>8844</td><td>88</td><td>0.0008</td></tr><tr><td>18:1&#8211;18:3</td><td>122,575</td><td>8421</td><td>61,410</td><td>1726</td><td>50</td><td>0.0001</td></tr><tr><td>18:2&#8211;18:2</td><td>366,675</td><td>17,961</td><td>250,350</td><td>26,185</td><td>68</td><td>0.0064</td></tr><tr><td rowspan="3">36:5</td><td>16:0&#8211;20:5</td><td>147,054</td><td>8925</td><td>3872</td><td>1095</td><td>3</td><td>&lt;0.0001</td></tr><tr><td>16:1&#8211;20:4</td><td>104,768</td><td>7773</td><td>79,851</td><td>4883</td><td>76</td><td>0.0265</td></tr><tr><td>18:2&#8211;18:3</td><td>36,381</td><td>3669</td><td>16,762</td><td>1790</td><td>46</td><td>0.0013</td></tr><tr><td rowspan="4">38:4</td><td>16:0&#8211;22:4</td><td>81,093</td><td>6150</td><td>169,849</td><td>9408</td><td>209</td><td>&lt;0.0001</td></tr><tr><td>18:0&#8211;20:4</td><td>670,488</td><td>23,121</td><td>624,853</td><td>18,289</td><td>93</td><td>0.1602</td></tr><tr><td>18:1&#8211;20:3</td><td>230,517</td><td>10,302</td><td>108,788</td><td>8009</td><td>47</td><td>&lt;0.0001</td></tr><tr><td>18:2&#8211;20:2</td><td>29,789</td><td>1587</td><td>14,567</td><td>701</td><td>49</td><td>&lt;0.0001</td></tr><tr><td rowspan="5">40:4</td><td>16:0&#8211;24:4</td><td>10,392</td><td>1617</td><td>20,865</td><td>3072</td><td>201</td><td>0.0167</td></tr><tr><td>18:0&#8211;22:4</td><td>176,325</td><td>4788</td><td>293,832</td><td>6785</td><td>167</td><td>&lt;0.0001</td></tr><tr><td>18:1&#8211;22:3</td><td>22,450</td><td>2704</td><td>62,145</td><td>3805</td><td>277</td><td>&lt;0.0001</td></tr><tr><td>20:0&#8211;20:4</td><td>12,131</td><td>1265</td><td>6590</td><td>899</td><td>54</td><td>0.0073</td></tr><tr><td>20:1&#8211;20:3</td><td>32,191</td><td>2591</td><td>4991</td><td>1740</td><td>16</td><td>&lt;0.0001</td></tr><tr><td rowspan="4">40:5</td><td>16:0&#8211;24:5</td><td>1535</td><td>1258</td><td>5381</td><td>1162</td><td>350</td><td>0.0549</td></tr><tr><td>18:0&#8211;22:5</td><td>110,752</td><td>7361</td><td>163,723</td><td>14,186</td><td>148</td><td>0.0106</td></tr><tr><td>18:1&#8211;22:4</td><td>39,603</td><td>2862</td><td>117,362</td><td>5425</td><td>296</td><td>&lt;0.0001</td></tr><tr><td>20:1&#8211;20:4</td><td>58,465</td><td>3767</td><td>32,595</td><td>2075</td><td>56</td><td>0.0003</td></tr><tr><td rowspan="4">42:4</td><td>18:0&#8211;24:4</td><td>1907</td><td>403</td><td>12,412</td><td>1762</td><td>651</td><td>0.0004</td></tr><tr><td>18:1&#8211;24:3</td><td>1663</td><td>777</td><td>9584</td><td>639</td><td>576</td><td>&lt;0.0001</td></tr><tr><td>18:2&#8211;24:2</td><td>4497</td><td>950</td><td>1587</td><td>605</td><td>35</td><td>0.0736</td></tr><tr><td>22:0&#8211;20:4</td><td>11,180</td><td>930</td><td>5039</td><td>1154</td><td>45</td><td>0.0032</td></tr><tr><td rowspan="3">42:5</td><td>18:0&#8211;24:5</td><td>3092</td><td>629</td><td>14,907</td><td>882</td><td>482</td><td>&lt;0.0001</td></tr><tr><td>18:1&#8211;24:4</td><td>1258</td><td>405</td><td>11,476</td><td>858</td><td>912</td><td>&lt;0.0001</td></tr><tr><td>22:1&#8211;20:4</td><td>25,450</td><td>2790</td><td>11,728</td><td>1853</td><td>46</td><td>0.0035</td></tr></tbody></table><table-wrap-foot><fn><p>SRM analyses for discrimination of fatty acids were performed for the indicated PC species (n = 5). Anions of the fatty acids indicated at the second position (e.g., 20:4 for 14:0&#8211;20:4 PC) were used for selection at Q3. Sample concentration was adjusted based on tissue weight.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06328-v2.xml">elife-06328-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.06328.014</object-id><label>Table 2.</label><caption><p>Acyl-chain composition of PE species with changed levels in LPCAT3-deficient mice</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06328.014">http://dx.doi.org/10.7554/eLife.06328.014</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">PE subclass</th><th rowspan="2">Acyl-chain composition</th><th colspan="2">LPCAT3+/+ peak area</th><th colspan="2">LPCAT3&#8722;/&#8722; peak area</th><th rowspan="2">KO/WT (%)</th><th rowspan="2">p value</th></tr><tr><th>Mean</th><th>SEM</th><th>Mean</th><th>SEM</th></tr></thead><tbody><tr><td rowspan="3">38:4</td><td>16:0&#8211;22:4</td><td>373,458</td><td>20,239</td><td>1,073,902</td><td>50,565</td><td>288</td><td>&lt;0.0001</td></tr><tr><td>18:0&#8211;20:4</td><td>8,394,810</td><td>381,499</td><td>4,349,617</td><td>102,271</td><td>52</td><td>&lt;0.0001</td></tr><tr><td>18:1&#8211;20:3</td><td>1,503,041</td><td>38,884</td><td>689,131</td><td>14,846</td><td>46</td><td>0.0002</td></tr><tr><td rowspan="4">40:4</td><td>18:0&#8211;22:4</td><td>1,831,032</td><td>80,319</td><td>3,803,421</td><td>98,424</td><td>208</td><td>&lt;0.0001</td></tr><tr><td>18:1&#8211;22:3</td><td>124,638</td><td>4846</td><td>236,224</td><td>23,357</td><td>190</td><td>0.0016</td></tr><tr><td>20:0&#8211;20:4</td><td>187,358</td><td>8687</td><td>45,797</td><td>4286</td><td>24</td><td>&lt;0.0001</td></tr><tr><td>20:1&#8211;20:3</td><td>35,837</td><td>3019</td><td>11,721</td><td>2388</td><td>33</td><td>0.0002</td></tr><tr><td rowspan="3">40:5</td><td>18:0&#8211;22:5</td><td>523,416</td><td>4657</td><td>2,056,845</td><td>111,780</td><td>393</td><td>&lt;0.0001</td></tr><tr><td>18:1&#8211;22:4</td><td>731,947</td><td>43,088</td><td>1,934,984</td><td>35,746</td><td>264</td><td>&lt;0.0001</td></tr><tr><td>20:1&#8211;20:4</td><td>224,759</td><td>3572</td><td>64,708</td><td>3066</td><td>29</td><td>&lt;0.0001</td></tr></tbody></table><table-wrap-foot><fn><p>SRM analyses for discrimination of fatty acids were performed for the indicated PE species (n = 5). Anions of the fatty acids indicated at the second position (e.g., 22:4 for 16:0&#8211;22:4 PE) were used for selection at Q3. Sample concentration was adjusted based on tissue weight.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06356-v1.xml">elife-06356-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.06356.004</object-id><label>Table 1.</label><caption><p>Identification of all active brain regions in adults and infants following acute noxious stimulation at all stimulus intensities (applied force: adults 32&#8211;512 mN; infants 32&#8211;128 mN)</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06356.004">http://dx.doi.org/10.7554/eLife.06356.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="4"/><th colspan="7">Adults</th><th colspan="7">Infants</th></tr><tr><th rowspan="2"/><th rowspan="2"/><th rowspan="2">Anatomical area</th><th rowspan="2">Region</th><th rowspan="2">Peak Z within cluster</th><th colspan="3">MNI coords</th><th rowspan="2">Rank</th><th rowspan="2">Slope of regression (*E-03)</th><th rowspan="2">P val*</th><th rowspan="2">Peak Z within cluster</th><th colspan="3">Neonate template coords</th><th rowspan="2">Rank</th><th rowspan="2">Slope of regression (*E-03)</th><th rowspan="2">P val*</th></tr><tr><th>x</th><th>y</th><th>z</th><th>x</th><th>y</th><th>z</th></tr></thead><tbody><tr><td rowspan="32">Active regions in both adults and infants</td><td rowspan="11">Intensity encoding regions (in adults)</td><td style="background:#8C8C8C">Temporal gyrus</td><td style="background:#8C8C8C">Contra</td><td style="background:#8C8C8C">3.92</td><td style="background:#8C8C8C">64</td><td style="background:#8C8C8C">&#8722;34</td><td style="background:#8C8C8C">20</td><td style="background:#8C8C8C"><bold>1</bold></td><td style="background:#8C8C8C">1.01</td><td style="background:#8C8C8C">0.0002</td><td style="background:#8C8C8C">3.05</td><td style="background:#8C8C8C">32</td><td style="background:#8C8C8C">&#8722;32</td><td style="background:#8C8C8C">12</td><td style="background:#8C8C8C"><bold>1</bold></td><td style="background:#8C8C8C">2.46</td><td style="background:#8C8C8C">0.0083</td></tr><tr><td>Cingulate gyrus</td><td>Anterior</td><td>4.11</td><td>6</td><td>4</td><td>40</td><td><bold>2</bold></td><td>0.65</td><td>0.0005</td><td>2.58</td><td>&#8722;1</td><td>1</td><td>26</td><td><bold>11</bold></td><td>1.01</td><td>0.3971</td></tr><tr><td style="background:#8C8C8C">Opercular cortex</td><td style="background:#8C8C8C">Contra</td><td style="background:#8C8C8C">5.60</td><td style="background:#8C8C8C">40</td><td style="background:#8C8C8C">6</td><td style="background:#8C8C8C">10</td><td style="background:#8C8C8C"><bold>3</bold></td><td style="background:#8C8C8C">0.63</td><td style="background:#8C8C8C">0.0001</td><td style="background:#8C8C8C">3.38</td><td style="background:#8C8C8C">32</td><td style="background:#8C8C8C">&#8722;13</td><td style="background:#8C8C8C">19</td><td style="background:#8C8C8C"><bold>2</bold></td><td style="background:#8C8C8C">2.23</td><td style="background:#8C8C8C">0.0391</td></tr><tr><td>Insula</td><td>Contra</td><td>4.18</td><td>34</td><td>14</td><td>6</td><td><bold>4</bold></td><td>0.61</td><td>0.0001</td><td>3.04</td><td>19</td><td>&#8722;22</td><td>23</td><td><bold>3</bold></td><td>2.15</td><td>0.0207</td></tr><tr><td style="background:#8C8C8C">Supramarginal gyrus</td><td style="background:#8C8C8C">Contra</td><td style="background:#8C8C8C">4.33</td><td style="background:#8C8C8C">64</td><td style="background:#8C8C8C">&#8722;38</td><td style="background:#8C8C8C">20</td><td style="background:#8C8C8C"><bold>5</bold></td><td style="background:#8C8C8C">0.60</td><td style="background:#8C8C8C">0.0008</td><td style="background:#8C8C8C">3.29</td><td style="background:#8C8C8C">25</td><td style="background:#8C8C8C">&#8722;23</td><td style="background:#8C8C8C">39</td><td style="background:#8C8C8C"><bold>9</bold></td><td style="background:#8C8C8C">1.08</td><td style="background:#8C8C8C">0.1749</td></tr><tr><td>Postcentral gyrus</td><td>Contra</td><td>4.28</td><td>58</td><td>&#8722;18</td><td>22</td><td><bold>6</bold></td><td>0.60</td><td>0.0012</td><td>3.85</td><td>15</td><td>&#8722;22</td><td>52</td><td><bold>10</bold></td><td>1.01</td><td>0.2667</td></tr><tr><td style="background:#8C8C8C">Visual cortex</td><td style="background:#8C8C8C">Contra</td><td style="background:#8C8C8C">3.62</td><td style="background:#8C8C8C">44</td><td style="background:#8C8C8C">&#8722;62</td><td style="background:#8C8C8C">4</td><td style="background:#8C8C8C"><bold>7</bold></td><td style="background:#8C8C8C">0.59</td><td style="background:#8C8C8C">0.0004</td><td style="background:#8C8C8C">3.25</td><td style="background:#8C8C8C">21</td><td style="background:#8C8C8C">&#8722;52</td><td style="background:#8C8C8C">34</td><td style="background:#8C8C8C"><bold>6</bold></td><td style="background:#8C8C8C">1.41</td><td style="background:#8C8C8C">0.0814</td></tr><tr><td>Putamen</td><td>Contra</td><td>3.68</td><td>22</td><td>6</td><td>6</td><td><bold>8</bold></td><td>0.55</td><td>0.0001</td><td>3.30</td><td>17</td><td>&#8722;17</td><td>18</td><td><bold>8</bold></td><td>1.20</td><td>0.1656</td></tr><tr><td style="background:#8C8C8C">Thalamus</td><td style="background:#8C8C8C">Contra</td><td style="background:#8C8C8C">3.51</td><td style="background:#8C8C8C">14</td><td style="background:#8C8C8C">&#8722;14</td><td style="background:#8C8C8C">0</td><td style="background:#8C8C8C"><bold>9</bold></td><td style="background:#8C8C8C">0.50</td><td style="background:#8C8C8C">0.0010</td><td style="background:#8C8C8C">3.58</td><td style="background:#8C8C8C">6</td><td style="background:#8C8C8C">&#8722;16</td><td style="background:#8C8C8C">15</td><td style="background:#8C8C8C"><bold>4</bold></td><td style="background:#8C8C8C">1.91</td><td style="background:#8C8C8C">0.0592</td></tr><tr><td>Insula</td><td>Ipsi</td><td>4.67</td><td>&#8722;38</td><td>&#8722;18</td><td>14</td><td><bold>10</bold></td><td>0.49</td><td>0.0001</td><td>2.59</td><td>&#8722;26</td><td>&#8722;14</td><td>14</td><td><bold>5</bold></td><td>1.69</td><td>0.1015</td></tr><tr><td style="background:#8C8C8C">Supplementary motor area</td><td style="background:#8C8C8C">Contra</td><td style="background:#8C8C8C">3.91</td><td style="background:#8C8C8C">8</td><td style="background:#8C8C8C">4</td><td style="background:#8C8C8C">46</td><td style="background:#8C8C8C"><bold>11</bold></td><td style="background:#8C8C8C">0.39</td><td style="background:#8C8C8C">0.0008</td><td style="background:#8C8C8C">3.50</td><td style="background:#8C8C8C">6</td><td style="background:#8C8C8C">&#8722;18</td><td style="background:#8C8C8C">48</td><td style="background:#8C8C8C"><bold>7</bold></td><td style="background:#8C8C8C">1.23</td><td style="background:#8C8C8C">0.2315</td></tr><tr><td rowspan="21">Non intensity encoding regions (in adults)</td><td>Cerebellum</td><td>Ipsi</td><td>3.88</td><td>&#8722;20</td><td>&#8722;66</td><td>&#8722;44</td><td/><td>0.35</td><td>0.0029</td><td>3.53</td><td>&#8722;3</td><td>&#8722;46</td><td>&#8722;6</td><td/><td>3.57</td><td>0.0164</td></tr><tr><td style="background:#8C8C8C">Temporal gyrus</td><td style="background:#8C8C8C">Ipsi</td><td style="background:#8C8C8C">3.72</td><td style="background:#8C8C8C">&#8722;52</td><td style="background:#8C8C8C">&#8722;56</td><td style="background:#8C8C8C">10</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.18</td><td style="background:#8C8C8C">0.5487</td><td style="background:#8C8C8C">3.41</td><td style="background:#8C8C8C">&#8722;32</td><td style="background:#8C8C8C">&#8722;22</td><td style="background:#8C8C8C">14</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">2.90</td><td style="background:#8C8C8C">0.0196</td></tr><tr><td>Supramarginal gyrus</td><td>Ipsi</td><td>4.59</td><td>&#8722;64</td><td>&#8722;28</td><td>20</td><td/><td>0.51</td><td>0.0035</td><td>3.13</td><td>&#8722;31</td><td>&#8722;24</td><td>30</td><td/><td>2.79</td><td>0.0055</td></tr><tr><td style="background:#8C8C8C">Cerebellum</td><td style="background:#8C8C8C">Contra</td><td style="background:#8C8C8C">3.36</td><td style="background:#8C8C8C">20</td><td style="background:#8C8C8C">&#8722;70</td><td style="background:#8C8C8C">&#8722;50</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.31</td><td style="background:#8C8C8C">0.0246</td><td style="background:#8C8C8C">3.16</td><td style="background:#8C8C8C">2</td><td style="background:#8C8C8C">&#8722;44</td><td style="background:#8C8C8C">&#8722;6</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">2.72</td><td style="background:#8C8C8C">0.1634</td></tr><tr><td>Opercular cortex</td><td>Ipsi</td><td>5.23</td><td>&#8722;50</td><td>&#8722;28</td><td>26</td><td/><td>0.50</td><td>0.0018</td><td>2.69</td><td>&#8722;27</td><td>&#8722;12</td><td>13</td><td/><td>2.23</td><td>0.0710</td></tr><tr><td style="background:#8C8C8C">Postcentral gyrus</td><td style="background:#8C8C8C">Ipsi</td><td style="background:#8C8C8C">4.71</td><td style="background:#8C8C8C">&#8722;62</td><td style="background:#8C8C8C">&#8722;18</td><td style="background:#8C8C8C">24</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.44</td><td style="background:#8C8C8C">0.0375</td><td style="background:#8C8C8C">3.52</td><td style="background:#8C8C8C">&#8722;31</td><td style="background:#8C8C8C">&#8722;15</td><td style="background:#8C8C8C">41</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">2.12</td><td style="background:#8C8C8C">0.0845</td></tr><tr><td>Thalamus</td><td>Ipsi</td><td>3.52</td><td>&#8722;12</td><td>&#8722;14</td><td>10</td><td/><td>0.42</td><td>0.0018</td><td>3.48</td><td>&#8722;1</td><td>&#8722;20</td><td>13</td><td/><td>1.67</td><td>0.1009</td></tr><tr><td style="background:#8C8C8C">Angular gyrus</td><td style="background:#8C8C8C">Ipsi</td><td style="background:#8C8C8C">3.59</td><td style="background:#8C8C8C">&#8722;58</td><td style="background:#8C8C8C">&#8722;50</td><td style="background:#8C8C8C">18</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.53</td><td style="background:#8C8C8C">0.0107</td><td style="background:#8C8C8C">2.98</td><td style="background:#8C8C8C">&#8722;23</td><td style="background:#8C8C8C">&#8722;39</td><td style="background:#8C8C8C">33</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">1.56</td><td style="background:#8C8C8C">0.0528</td></tr><tr><td>Precentral gyrus</td><td>Ipsi</td><td>4.01</td><td>&#8722;58</td><td>0</td><td>10</td><td/><td>0.43</td><td>0.0578</td><td>3.46</td><td>&#8722;23</td><td>&#8722;17</td><td>48</td><td/><td>1.53</td><td>0.1247</td></tr><tr><td style="background:#8C8C8C">Frontal gyrus</td><td style="background:#8C8C8C">Contra</td><td style="background:#8C8C8C">3.88</td><td style="background:#8C8C8C">58</td><td style="background:#8C8C8C">12</td><td style="background:#8C8C8C">0</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.56</td><td style="background:#8C8C8C">0.0212</td><td style="background:#8C8C8C">3.11</td><td style="background:#8C8C8C">11</td><td style="background:#8C8C8C">&#8722;12</td><td style="background:#8C8C8C">48</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">1.42</td><td style="background:#8C8C8C">0.0646</td></tr><tr><td>Cingulate gyrus</td><td>Posterior</td><td>3.71</td><td>&#8722;14</td><td>&#8722;28</td><td>38</td><td/><td>0.08</td><td>0.2480</td><td>3.18</td><td>&#8722;9</td><td>&#8722;23</td><td>35</td><td/><td>1.42</td><td>0.1101</td></tr><tr><td style="background:#8C8C8C">Angular gyrus</td><td style="background:#8C8C8C">Contra</td><td style="background:#8C8C8C">3.71</td><td style="background:#8C8C8C">60</td><td style="background:#8C8C8C">&#8722;46</td><td style="background:#8C8C8C">18</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.54</td><td style="background:#8C8C8C">0.0080</td><td style="background:#8C8C8C">3.12</td><td style="background:#8C8C8C">22</td><td style="background:#8C8C8C">&#8722;51</td><td style="background:#8C8C8C">35</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">1.42</td><td style="background:#8C8C8C">0.0407</td></tr><tr><td>Precuneous cortex</td><td>Contra</td><td>3.60</td><td>16</td><td>&#8722;68</td><td>40</td><td/><td>0.38</td><td>0.0714</td><td>3.70</td><td>5</td><td>&#8722;30</td><td>52</td><td/><td>1.19</td><td>0.1623</td></tr><tr><td style="background:#8C8C8C">Visual cortex</td><td style="background:#8C8C8C">Ipsi</td><td style="background:#8C8C8C">3.82</td><td style="background:#8C8C8C">&#8722;52</td><td style="background:#8C8C8C">&#8722;70</td><td style="background:#8C8C8C">10</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">&#8722;0.09</td><td style="background:#8C8C8C">0.3758</td><td style="background:#8C8C8C">2.59</td><td style="background:#8C8C8C">&#8722;7</td><td style="background:#8C8C8C">&#8722;40</td><td style="background:#8C8C8C">11</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">1.17</td><td style="background:#8C8C8C">0.1657</td></tr><tr><td>Brainstem</td><td/><td>3.86</td><td>10</td><td>&#8722;26</td><td>&#8722;8</td><td/><td>0.33</td><td>0.1710</td><td>2.99</td><td>&#8722;3</td><td>&#8722;27</td><td>&#8722;10</td><td/><td>1.11</td><td>0.4350</td></tr><tr><td style="background:#8C8C8C">Parietal lobule</td><td style="background:#8C8C8C">Contra</td><td style="background:#8C8C8C">3.10</td><td style="background:#8C8C8C">20</td><td style="background:#8C8C8C">&#8722;44</td><td style="background:#8C8C8C">68</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.61</td><td style="background:#8C8C8C">0.1097</td><td style="background:#8C8C8C">3.10</td><td style="background:#8C8C8C">27</td><td style="background:#8C8C8C">&#8722;24</td><td style="background:#8C8C8C">46</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">1.09</td><td style="background:#8C8C8C">0.1271</td></tr><tr><td>Putamen</td><td>Ipsi</td><td>3.63</td><td>&#8722;16</td><td>10</td><td>&#8722;2</td><td/><td>0.45</td><td>0.0023</td><td>3.13</td><td>&#8722;14</td><td>&#8722;14</td><td>19</td><td/><td>0.92</td><td>0.2813</td></tr><tr><td style="background:#8C8C8C">Supplementary motor area</td><td style="background:#8C8C8C">Ipsi</td><td style="background:#8C8C8C">3.55</td><td style="background:#8C8C8C">&#8722;6</td><td style="background:#8C8C8C">4</td><td style="background:#8C8C8C">44</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.40</td><td style="background:#8C8C8C">0.0219</td><td style="background:#8C8C8C">3.16</td><td style="background:#8C8C8C">&#8722;4</td><td style="background:#8C8C8C">&#8722;10</td><td style="background:#8C8C8C">46</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.91</td><td style="background:#8C8C8C">0.3903</td></tr><tr><td>Precentral Gyrus</td><td>Contra</td><td>4.05</td><td>58</td><td>4</td><td>8</td><td/><td>0.44</td><td>0.0276</td><td>3.76</td><td>6</td><td>&#8722;20</td><td>53</td><td/><td>0.88</td><td>0.2672</td></tr><tr><td style="background:#8C8C8C">Frontal gyrus</td><td style="background:#8C8C8C">Ipsi</td><td style="background:#8C8C8C">3.57</td><td style="background:#8C8C8C">&#8722;8</td><td style="background:#8C8C8C">22</td><td style="background:#8C8C8C">32</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">&#8722;0.24</td><td style="background:#8C8C8C">0.1954</td><td style="background:#8C8C8C">2.79</td><td style="background:#8C8C8C">&#8722;13</td><td style="background:#8C8C8C">&#8722;9</td><td style="background:#8C8C8C">50</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.70</td><td style="background:#8C8C8C">0.4820</td></tr><tr><td>Pallidum</td><td>Contra</td><td>3.40</td><td>16</td><td>&#8722;4</td><td>&#8722;4</td><td/><td>0.49</td><td>0.0071</td><td>2.84</td><td>13</td><td>&#8722;13</td><td>13</td><td/><td>0.64</td><td>0.4863</td></tr><tr><td rowspan="4">Active regions in adults only</td><td rowspan="4"/><td style="background:#8C8C8C">Amygdala</td><td style="background:#8C8C8C">Contra</td><td style="background:#8C8C8C">3.49</td><td style="background:#8C8C8C">20</td><td style="background:#8C8C8C">&#8722;2</td><td style="background:#8C8C8C">&#8722;14</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.69</td><td style="background:#8C8C8C">0.0160</td><td colspan="7" style="background:#8C8C8C"/></tr><tr><td>Amygdala</td><td>Ipsi</td><td>4.28</td><td>&#8722;20</td><td>&#8722;2</td><td>&#8722;12</td><td/><td>0.43</td><td>0.0860</td><td colspan="7"/></tr><tr><td style="background:#8C8C8C">Orbitofrontal cortex</td><td style="background:#8C8C8C">Ipsi</td><td style="background:#8C8C8C">3.40</td><td style="background:#8C8C8C">&#8722;18</td><td style="background:#8C8C8C">4</td><td style="background:#8C8C8C">&#8722;16</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.42</td><td style="background:#8C8C8C">0.0157</td><td colspan="7" style="background:#8C8C8C">no activity</td></tr><tr><td>Orbitofrontal cortex</td><td>Contra</td><td>3.57</td><td>34</td><td>30</td><td>&#8722;2</td><td/><td>0.44</td><td>0.0460</td><td colspan="7"/></tr><tr><td rowspan="11">Active regions in infants only</td><td rowspan="11"/><td style="background:#8C8C8C">Precuneous cortex</td><td style="background:#8C8C8C">Ipsi</td><td colspan="7" style="background:#8C8C8C"/><td style="background:#8C8C8C">3.80</td><td style="background:#8C8C8C">&#8722;1</td><td style="background:#8C8C8C">&#8722;26</td><td style="background:#8C8C8C">52</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">1.26</td><td style="background:#8C8C8C">0.1699</td></tr><tr><td>Pallidum</td><td>Ipsi</td><td colspan="7"/><td>3.16</td><td>&#8722;8</td><td>&#8722;5</td><td>14</td><td/><td>0.59</td><td>0.4787</td></tr><tr><td style="background:#8C8C8C">Parietal lobule</td><td style="background:#8C8C8C">Ipsi</td><td colspan="7" style="background:#8C8C8C"/><td style="background:#8C8C8C">3.31</td><td style="background:#8C8C8C">&#8722;28</td><td style="background:#8C8C8C">&#8722;23</td><td style="background:#8C8C8C">33</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.99</td><td style="background:#8C8C8C">0.2711</td></tr><tr><td>Auditory cortex</td><td>Contra</td><td colspan="7"/><td>2.89</td><td>26</td><td>&#8722;14</td><td>18</td><td/><td>3.07</td><td>0.0119</td></tr><tr><td style="background:#8C8C8C">Auditory cortex</td><td style="background:#8C8C8C">Ipsi</td><td colspan="7" style="background:#8C8C8C"/><td style="background:#8C8C8C">3.34</td><td style="background:#8C8C8C">&#8722;17</td><td style="background:#8C8C8C">&#8722;29</td><td style="background:#8C8C8C">19</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">2.56</td><td style="background:#8C8C8C">0.0304</td></tr><tr><td>Caudate</td><td>Contra</td><td colspan="7">no activity</td><td>3.61</td><td>13</td><td>&#8722;17</td><td>22</td><td/><td>0.59</td><td>0.5822</td></tr><tr><td style="background:#8C8C8C">Caudate</td><td style="background:#8C8C8C">Ipsi</td><td colspan="7" style="background:#8C8C8C"/><td style="background:#8C8C8C">3.47</td><td style="background:#8C8C8C">&#8722;7</td><td style="background:#8C8C8C">&#8722;8</td><td style="background:#8C8C8C">18</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">1.05</td><td style="background:#8C8C8C">0.3415</td></tr><tr><td>Hippocampus</td><td>Contra</td><td colspan="7"/><td>2.61</td><td>21</td><td>&#8722;25</td><td>9</td><td/><td>1.84</td><td>0.1288</td></tr><tr><td style="background:#8C8C8C">Hippocampus</td><td style="background:#8C8C8C">Ipsi</td><td colspan="7" style="background:#8C8C8C"/><td style="background:#8C8C8C">2.77</td><td style="background:#8C8C8C">&#8722;15</td><td style="background:#8C8C8C">&#8722;31</td><td style="background:#8C8C8C">9</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">1.00</td><td style="background:#8C8C8C">0.3326</td></tr><tr><td>Parahippocampus</td><td>Contra</td><td colspan="7"/><td>3.02</td><td>11</td><td>&#8722;23</td><td>0</td><td/><td>1.53</td><td>0.3740</td></tr><tr><td style="background:#8C8C8C">Parahippocampus</td><td style="background:#8C8C8C">Ipsi</td><td colspan="7" style="background:#8C8C8C"/><td style="background:#8C8C8C">2.99</td><td style="background:#8C8C8C">&#8722;7</td><td style="background:#8C8C8C">&#8722;24</td><td style="background:#8C8C8C">&#8722;8</td><td style="background:#8C8C8C"/><td style="background:#8C8C8C">0.19</td><td style="background:#8C8C8C">0.9013</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06426-v1.xml">elife-06426-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.06426.005</object-id><label>Table 1.</label><caption><p>Univariate group-results for seed-to-voxel connectivity of PRC and PHC seeds with the EC</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06426.005">http://dx.doi.org/10.7554/eLife.06426.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th>Cluster</th><th>Cluster</th><th>Cluster</th><th>Peak</th><th colspan="3">Peak coordinate (template)</th><th rowspan="2">Side</th></tr><tr><th>P<sub>FWE-corr</sub></th><th>P<sub>FDR-corr</sub></th><th>Size</th><th>Z-score</th><th>x</th><th>y</th><th>z</th></tr></thead><tbody><tr><td colspan="9">Experiment 1</td></tr><tr><td rowspan="2">PRC seed</td><td>&lt;0.001</td><td>&lt;0.001</td><td>517</td><td>5.05</td><td>153</td><td>149</td><td>8</td><td>R</td></tr><tr><td>&lt;0.001</td><td>&lt;0.001</td><td>273</td><td>4.82</td><td>108</td><td>146</td><td>12</td><td>L</td></tr><tr><td rowspan="2">PHC seed</td><td>&lt;0.001</td><td>&lt;0.001</td><td>380</td><td>5.29</td><td>150</td><td>129</td><td>9</td><td>R</td></tr><tr><td>&lt;0.001</td><td>&lt;0.001</td><td>510</td><td>5.05</td><td>106</td><td>138</td><td>12</td><td>L</td></tr><tr><td rowspan="2">PRC &gt; PHC</td><td>0.038</td><td>0.022</td><td>42</td><td>4.31</td><td>151</td><td>150</td><td>8</td><td>R</td></tr><tr><td>0.001</td><td>0.001</td><td>91</td><td>4.04</td><td>101</td><td>146</td><td>9</td><td>L</td></tr><tr><td rowspan="2">PHC &gt; PRC</td><td>0.008</td><td>0.005</td><td>61</td><td>4.15</td><td>150</td><td>131</td><td>10</td><td>R</td></tr><tr><td>0.001</td><td>0.002</td><td>87</td><td>3.53</td><td>108</td><td>136</td><td>11</td><td>L</td></tr><tr><td colspan="9">Experiment 2</td></tr><tr><td rowspan="2">PRC seed</td><td>&lt;0.001</td><td>&lt;0.001</td><td>777</td><td>4.82</td><td>109</td><td>152</td><td>20</td><td>L</td></tr><tr><td>&lt;0.001</td><td>&lt;0.001</td><td>849</td><td>4.73</td><td>149</td><td>145</td><td>10</td><td>R</td></tr><tr><td rowspan="2">PHC seed</td><td>&lt;0.001</td><td>&lt;0.001</td><td>669</td><td>5.31</td><td>147</td><td>137</td><td>13</td><td>R</td></tr><tr><td>&lt;0.001</td><td>&lt;0.001</td><td>637</td><td>4.91</td><td>107</td><td>139</td><td>14</td><td>L</td></tr><tr><td>PRC &gt; PHC</td><td>&lt;0.001</td><td>&lt;0.001</td><td>167</td><td>4.70</td><td>105</td><td>153</td><td>15</td><td>L</td></tr><tr><td rowspan="2">PHC &gt; PRC</td><td>0.022</td><td>0.024</td><td>66</td><td>4.09</td><td>153</td><td>130</td><td>9</td><td>R</td></tr><tr><td>0.047</td><td>0.026</td><td>53</td><td>3.74</td><td>108</td><td>136</td><td>18</td><td>L</td></tr></tbody></table><table-wrap-foot><fn><p>Entorhinal subregions showing significant functional connectivity (one-sample t-test) or differential connectivity (paired t-test) with bilateral PRC or PHC seeds (Z &gt; 2.3, p<sub>cluster</sub> &lt;0.05, N<sub>Exp. 1</sub> = 15, N<sub>Exp. 2</sub> = 14). Single-subject beta maps were normalized on the group-specific T1-template and masked with a manually defined EC ROI. The EC covered 26 coronal slices on the template (y = 154: most anterior slice, y = 129: most posterior slice), with coronal slices being oriented orthogonal to the hippocampal long-axis. See also <xref ref-type="fig" rid="fig1">Figure 1</xref> and <xref ref-type="fig" rid="fig1s1">Figure 1&#8212;figure supplement 1</xref>.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06434-v1.xml">elife-06434-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tblu8" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th colspan="7">Figure 1B</th></tr><tr><th rowspan="2"/><th>Mean</th><th colspan="4">Assumed variance</th><th rowspan="2">Assumed N</th></tr><tr><th/><th>2%</th><th>15%</th><th>28%</th><th>40%</th></tr></thead><tbody><tr><td>Let-7b levels in LAPC4 CD44<sup>+</sup> cells</td><td>30%</td><td>0.6</td><td>4.5</td><td>8.4</td><td>12</td><td>3</td></tr><tr><td>miR-34 levels in LAPC4 CD44<sup>+</sup> cells</td><td>3%</td><td>0.06</td><td>0.45</td><td>0.84</td><td>1.2</td><td>3</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06487-v2.xml">elife-06487-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.06487.006</object-id><label>Table 1.</label><caption><p>Summary of the characteristics of cheetah-prey pursuits with prey nominally ranked in order of mass (top smallest to bottom largest)</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06487.006">http://dx.doi.org/10.7554/eLife.06487.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Prey</th><th colspan="8">Chase parameters</th></tr><tr><th>No. hunts</th><th>Success</th><th colspan="2">Total duration (s)</th><th colspan="2">No. of turns</th><th colspan="2">Turn rate (Hz)</th></tr><tr><th>N</th><th>%</th><th>Mean</th><th>SD</th><th>Mean</th><th>SD</th><th>Mean</th><th>SD</th></tr></thead><tbody><tr><td>Hare</td><td>2</td><td>100</td><td>10.8</td><td>&#8211;</td><td>5</td><td/><td>0.5</td><td>0.26</td></tr><tr><td>Steenbok</td><td>19</td><td>53</td><td>28.7</td><td>11.9</td><td>5.47</td><td>2.97</td><td>0.32</td><td>0.19</td></tr><tr><td>Duiker</td><td>1</td><td>100</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>Springbok</td><td>7</td><td>57</td><td>28.3</td><td>7</td><td>6.17</td><td>3.06</td><td>0.32</td><td>0.17</td></tr><tr><td>Ostrich</td><td>1</td><td>100</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>Wildebeest</td><td>2</td><td>50</td><td>35</td><td>3.2</td><td>6.5</td><td>0.71</td><td>0.22</td><td>0.09</td></tr><tr><td>Gemsbok</td><td>4</td><td>75</td><td>18.6</td><td>10.6</td><td>2.33</td><td>1.53</td><td>0.15</td><td>0.05</td></tr></tbody></table><table-wrap-foot><fn><p>Two species were pursued where no corresponding acceleration data were available.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06576-v1.xml">elife-06576-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.06576.006</object-id><label>Table 1.</label><caption><p>SS-induced changes in amplitude of low-frequency fluctuations (ALFF); SS group vs Saline group; p &lt; 0.001 corrected for multiple comparisons</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06576.006">http://dx.doi.org/10.7554/eLife.06576.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Brain region</th><th colspan="2">Left</th><th colspan="2">Right</th></tr><tr><th>Cluster size</th><th>t-value</th><th>Cluster size</th><th>t-value</th></tr></thead><tbody><tr><td colspan="5">ALFF increased</td></tr><tr><td>&#8195;ACx</td><td>167</td><td>3.812</td><td>178</td><td>3.746</td></tr><tr><td>&#8195;IC</td><td>72</td><td>3.383</td><td>68</td><td>3.473</td></tr><tr><td>&#8195;MGB</td><td>52</td><td>3.432</td><td>48</td><td>3.339</td></tr><tr><td>&#8195;SSCx</td><td>37</td><td>3.383</td><td>40</td><td>3.402</td></tr><tr><td>&#8195;VCx</td><td>32</td><td>3.342</td><td>39</td><td>3.312</td></tr><tr><td>&#8195;SC</td><td>40</td><td>4.123</td><td>43</td><td>4.094</td></tr><tr><td>&#8195;AMY</td><td>61</td><td>4.192</td><td>60</td><td>3.923</td></tr><tr><td>&#8195;RN/PnO</td><td>35</td><td>3.249</td><td>28</td><td>3.290</td></tr><tr><td>&#8195;PnO/PMr</td><td>12</td><td>3.498</td><td>10</td><td>3.313</td></tr><tr><td>&#8195;PFL</td><td>38</td><td>3.349</td><td>37</td><td>3.292</td></tr><tr><td>&#8195;CB4</td><td>38</td><td>3.349</td><td>37</td><td>3.292</td></tr><tr><td colspan="5">ALFF decreased</td></tr><tr><td>&#8195;HIP</td><td>108</td><td>&#8722;4.087</td><td>92</td><td>&#8722;4.002</td></tr><tr><td>&#8195;CPu</td><td>72</td><td>&#8722;3.772</td><td>71</td><td>&#8722;3.741</td></tr></tbody></table><table-wrap-foot><fn><p>Abbreviations: auditory cortex (ACx), inferior colliculus (IC), medial geniculate body (MGB), somatosensory cortex (SSCx), visual cortex (VCx), superior colliculi (SC), amygdala (AMY), gigantocellular reticular nucleus (RN), pontine reticular nucleus oral (PnO), paramedian raphe nucleus (PMr), parafloccular lobe of cerebellum (PFL), cerebellum lobule 4 (CB4), hippocampus (HIP), caudate-putamen (CPu), sodium salicylate (SS).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06576-v1.xml">elife-06576-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.06576.009</object-id><label>Table 2.</label><caption><p>SS-induced increases in functional connectivity (FC); p &lt; 0.001 corrected for multiple comparisons</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06576.009">http://dx.doi.org/10.7554/eLife.06576.009</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Seed region</th><th rowspan="2">Brain region</th><th colspan="2">Left</th><th colspan="2">Right</th></tr><tr><th>Cluster size</th><th>t-value</th><th>Cluster size</th><th>t-value</th></tr></thead><tbody><tr><td rowspan="6">ACx</td><td>MGB</td><td>62</td><td>3.900</td><td>70</td><td>3.859</td></tr><tr><td>IC</td><td>82</td><td>3.912</td><td>88</td><td>3.632</td></tr><tr><td>AMY</td><td>67</td><td>3.897</td><td>68</td><td>3.839</td></tr><tr><td>RN</td><td>52</td><td>3.983</td><td>48</td><td>3.902</td></tr><tr><td>PFL</td><td>39</td><td>3.992</td><td>35</td><td>3.954</td></tr><tr><td>CB4</td><td>51</td><td>4.066</td><td>53</td><td>4.070</td></tr><tr><td rowspan="2">MGB</td><td>ACx</td><td>195</td><td>4.074</td><td>210</td><td>4.084</td></tr><tr><td>HIP</td><td>162</td><td>4.032</td><td>178</td><td>4.109</td></tr><tr><td rowspan="2">IC</td><td>MGB</td><td>72</td><td>4.098</td><td>60</td><td>4.013</td></tr><tr><td>HIP</td><td>140</td><td>4.064</td><td>131</td><td>3.904</td></tr></tbody></table><table-wrap-foot><fn><p>Abbreviations: medial geniculate body (MGB), inferior colliculus (IC), amygdala (AMY), reticular nucleus (RN), parafloccular lobe of cerebellum (PFL), cerebellum lobule 4 (CB4), auditory cortex (ACx), hippocampus (HIP), sodium salicylate (SS).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06638-v1.xml">elife-06638-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.06638.007</object-id><label>Table 1.</label><caption><p>Protein intact mass MS analysis</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06638.007">http://dx.doi.org/10.7554/eLife.06638.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Protein</th><th rowspan="2">Vector</th><th colspan="3">Molecular mass (Da)</th></tr><tr><th>Calculated</th><th>Observed</th><th>&#916;</th></tr></thead><tbody><tr><td>SfbI-A40-TED</td><td>pOPIN-F<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>24,156.9</td><td>24,139.6</td><td>&#8722;17.3</td></tr><tr><td>SfbI-A40-TED:Cys109Ala</td><td>pOPIN-F<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>24,124.9</td><td>24,124.7</td><td>&#8722;0.2</td></tr><tr><td>SfbI-A346-TED</td><td>pDEST<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>24,835.8</td><td>24,818.2</td><td>&#8722;17.6</td></tr><tr><td>SfbI-A346-TED:Cys103Ala</td><td>pOPIN-E<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>25,626.6</td><td>25,626.3</td><td>&#8722;0.3</td></tr><tr><td>SfbI-A20-TED</td><td>pOPIN-E<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>25,996.1</td><td>25,978.5</td><td>&#8722;17.6</td></tr><tr><td>SfbI-A20-TED:Cys97Ala</td><td>pOPIN-E<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>25,964.1</td><td>25,963.6</td><td>&#8722;0.5</td></tr><tr><td>GfbA-TED</td><td>pDEST<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>24,382.4</td><td>24,364.7</td><td>&#8722;17.7</td></tr><tr><td>FbaB-TED</td><td>pOPIN-F<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>21,683.0</td><td>21,665.6</td><td>&#8722;17.4</td></tr><tr><td>FbaB-TED:Cys94Ala</td><td>pOPIN-E<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td><td>22,447.7</td><td>22,447.5</td><td>&#8722;0.2</td></tr><tr><td>Cpa-TED2</td><td>pOPIN-F<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>22,377.8</td><td>22,360.5</td><td>&#8722;17.3</td></tr><tr><td>CpTIE-TED</td><td>pOPIN-F<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>21,261.0</td><td>21,243.7</td><td>&#8722;17.3</td></tr><tr><td>CpTIE-TED:Cys138Ala</td><td>pOPIN-F<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>21,228.9</td><td>21,227.8</td><td>&#8722;1.1</td></tr><tr><td>CodTIE-TED</td><td>pOPIN-F<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>25,011.6</td><td>24,994.3</td><td>&#8722;17.3</td></tr><tr><td>CodTIE-TED:Cys157Ala</td><td>pOPIN-F<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>24,979.5</td><td>24,979.1</td><td>&#8722;0.4</td></tr><tr><td>PnTIE-TED</td><td>pDEST<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>25,779.0</td><td>25,761.7</td><td>&#8722;17.3</td></tr><tr><td>PnTIE-TED:Cys94Ala</td><td>pHisTEV<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>25,746.9</td><td>25,746.5</td><td>&#8722;0.4</td></tr><tr><td>BaTIE-TED</td><td>pHisTEV<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>28,541.1</td><td>28,523.7</td><td>&#8722;17.4</td></tr><tr><td>SaTIE-TED</td><td>pHisTEV<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>33,167.9</td><td>33,151.3</td><td>&#8722;16.6</td></tr><tr><td>CdTEP-TED</td><td>pHisTEV<xref ref-type="table-fn" rid="tblfn5">#</xref></td><td>46,363.7</td><td>46,345.7</td><td>&#8722;18.0</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>Non-native residues remaining after 3C cleavage: N-terminal GP.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>Non-native residues remaining after TEV cleavage: N-terminal GAM.</p></fn><fn id="tblfn3"><label>&#8225;</label><p>Non-native residues remaining: N-terminal M, C-terminal KHHHHHH.</p></fn><fn id="tblfn4"><label>&#167;</label><p>Non-native residues remaining: C-terminal KHHHHH only. N-terminal M removed.</p></fn><fn id="tblfn5"><label>#</label><p>Non-native residue remaining after TEV cleavage: N-terminal M.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06638-v1.xml">elife-06638-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.06638.010</object-id><label>Table 2.</label><caption><p>X-ray data collection and refinement statistics</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06638.010">http://dx.doi.org/10.7554/eLife.06638.010</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="4">(<bold>A</bold>) CpTIE-TED and CpTIE-TED:Cys138Ala</th></tr><tr><th rowspan="2"/><th>CpTIE-TED</th><th>CpTIE-TED:Cys138Ala</th><th rowspan="2">Iodide</th></tr><tr><th>Native</th><th>Native</th></tr></thead><tbody><tr><td colspan="4">Data collection</td></tr><tr><td>&#8195;Space group</td><td>P1</td><td>P2<sub>1</sub>2<sub>1</sub>2</td><td>P2</td></tr><tr><td colspan="4">&#8195;Cell dimensions</td></tr><tr><td>&#8195;&#8195;<italic>a</italic>, <italic>b</italic>, <italic>c</italic> (&#197;)</td><td>70.82, 74.36, 82.81</td><td>98.42, 110.72, 68.77</td><td>97.91, 110.63, 68.45</td></tr><tr><td>&#8195;&#8195;<italic>&#945;</italic>, <italic>&#946;</italic>, <italic>&#947;</italic> (&#176;)</td><td>107.32, 104.32, 98.63</td><td>90, 90, 90</td><td>90, 90, 90</td></tr><tr><td>&#8195;Resolution (&#197;)<xref ref-type="table-fn" rid="tblfn6">*</xref></td><td>44.88&#8211;2.62 (2.69&#8211;2.62)</td><td>44.97&#8211;1.60 (1.64&#8211;1.60)</td><td>58.21&#8211;2.83 (2.90&#8211;2.83)</td></tr><tr><td>&#8195;<italic>R</italic><sub>merge</sub></td><td>13.3 (76.4)</td><td>5.4 (74.4)</td><td>19.5 (101.8)</td></tr><tr><td>&#8195;<italic>I</italic>/&#963;<italic>I</italic></td><td>7.5 (1.8)</td><td>31.6 (4.3)</td><td>22.7 (4.4)</td></tr><tr><td colspan="4">&#8195;Completeness (%)</td></tr><tr><td>&#8195;&#8195;Overall</td><td>98.0 (97.3)</td><td>100 (99.9)</td><td>98.2 (97.3)</td></tr><tr><td>&#8195;&#8195;Anomalous</td><td/><td/><td>98.1 (96.2)</td></tr><tr><td colspan="4">&#8195;Redundancy</td></tr><tr><td>&#8195;&#8195;Overall</td><td>4.7 (4.9)</td><td>18.1 (18.1)</td><td>33.3 (33.0)</td></tr><tr><td>&#8195;&#8195;Anomalous</td><td/><td/><td>17.5 (16.8)</td></tr><tr><td>&#8195;CC(1/2) (%)</td><td>99.3 (58.9)</td><td>100 (90.9)</td><td>99.8 (86.0)</td></tr><tr><td colspan="4">Refinement</td></tr><tr><td>&#8195;Resolution (&#197;)</td><td>44.8&#8211;2.62 (2.69&#8211;2.62)</td><td>44.97&#8211;1.60 (1.64&#8211;1.60)</td><td/></tr><tr><td>&#8195;No. reflections</td><td>42,485 (3115)</td><td>94,622 (6864)</td><td/></tr><tr><td>&#8195;<italic>R</italic><sub>work</sub>/<italic>R</italic><sub>free</sub></td><td>19.6/22.6 (35.1/33.7)</td><td>17.7/21.0 (20.5/22.0)</td><td/></tr><tr><td colspan="4">&#8195;No. atoms</td></tr><tr><td>&#8195;Protein</td><td>8260</td><td>6310</td><td/></tr><tr><td>&#8195;&#8195;Ligand/ion/water</td><td>36</td><td>403</td><td/></tr><tr><td colspan="4">&#8195;B-factors</td></tr><tr><td>&#8195;&#8195;Protein</td><td>50.9</td><td>24.5</td><td/></tr><tr><td>&#8195;&#8195;Ligand/ion/water</td><td>37.8</td><td>28.5</td><td/></tr><tr><td colspan="4">&#8195;R.m.s deviations</td></tr><tr><td>&#8195;&#8195;Bond lengths (&#197;)</td><td>0.014</td><td>0.013</td><td/></tr><tr><td>&#8195;&#8195;Bond angles (&#176;)</td><td>1.60</td><td>1.57</td><td/></tr><tr><td>&#8195;MolProbity Score</td><td>1.11 (100th percentile)</td><td>1.13 (99th percentile)</td><td/></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th colspan="4">(<bold>B</bold>) SfbI-A40-TED and PnTIE-TED</th></tr><tr><th rowspan="2"/><th>SfbI-A40-TED</th><th>PnTIE-TED</th><th/></tr><tr><th>Native/Zinc</th><th>Native</th><th>Iodide</th></tr></thead><tbody><tr><td colspan="4">Data collection</td></tr><tr><td>&#8195;Space group</td><td>I4<sub>1</sub></td><td>P4<sub>1</sub>2<sub>1</sub>2</td><td>P4<sub>1</sub>2<sub>1</sub>2</td></tr><tr><td colspan="4">&#8195;Cell dimensions</td></tr><tr><td>&#8195;&#8195;<italic>a</italic>, <italic>b</italic>, <italic>c</italic> (&#197;)</td><td>165.12, 165.12, 42.52</td><td>59.86, 59.86, 121.70</td><td>59.37, 59.37, 122.4</td></tr><tr><td>&#8195;&#8195;<italic>&#945;</italic>, <italic>&#946;</italic>, <italic>&#947;</italic> (&#176;)</td><td>90, 90, 90</td><td>90, 90, 90</td><td>90, 90, 90</td></tr><tr><td>&#8195;Resolution (&#197;)<xref ref-type="table-fn" rid="tblfn7">&#8224;</xref></td><td>52.22&#8211;1.35 (1.39&#8211;1.35)</td><td>42.67&#8211;1.30 (1.32&#8211;1.30)</td><td>31.22&#8211;2.80 (2.95&#8211;2.80)</td></tr><tr><td>&#8195;<italic>R</italic><sub>merge</sub></td><td>5.3 (75.4)</td><td>5.4 (30.4)</td><td>13.3 (35.7)</td></tr><tr><td>&#8195;<italic>I</italic>/&#963;<italic>I</italic></td><td>18.7 (2.7)</td><td>33.8 (7.4)</td><td>24.9 (13.0)</td></tr><tr><td colspan="4">&#8195;Completeness (%)</td></tr><tr><td>&#8195;&#8195;Overall</td><td>99.9 (99.1)</td><td>93.6 (62.7)</td><td>100 (100)</td></tr><tr><td>&#8195;&#8195;Anomalous</td><td>99.6 (97.5)</td><td/><td>100 (100)</td></tr><tr><td colspan="4">&#8195;Redundancy</td></tr><tr><td>&#8195;&#8195;Overall</td><td>9.2 (9.1)</td><td>23.9 (11.9)</td><td>32.8 (31.9)</td></tr><tr><td>&#8195;&#8195;Anomalous</td><td>4.6 (4.6)</td><td/><td>18.1 (16.6)</td></tr><tr><td>&#8195;CC(1/2) (%)</td><td>99.9 (85.2)</td><td>100 (96.2)</td><td>99.6 (99.2)</td></tr><tr><td colspan="4">Refinement</td></tr><tr><td>&#8195;Resolution (&#197;)</td><td>52.22&#8211;1.35 (1.39&#8211;1.35)</td><td>53.71&#8211;1.30 (1.33&#8211;1.30)</td><td/></tr><tr><td>&#8195;No. reflections</td><td>119869 (8679)</td><td>48,978 (2374)</td><td/></tr><tr><td>&#8195;<italic>R</italic><sub>work</sub>/<italic>R</italic><sub>free</sub></td><td>13.1/15.3 (22.6/22.9)</td><td>11.9/15.1 (12.6/15.9)</td><td/></tr><tr><td colspan="4">&#8195;No. atoms</td></tr><tr><td>&#8195;&#8195;Protein</td><td>3389</td><td>1848</td><td/></tr><tr><td>&#8195;&#8195;Ligand/ion/water</td><td>470</td><td>252</td><td/></tr><tr><td colspan="4">&#8195;B-factors</td></tr><tr><td>&#8195;&#8195;Protein</td><td>23.0</td><td>12.5</td><td/></tr><tr><td>&#8195;&#8195;Ligand/ion/water</td><td>39.4</td><td>26.9</td><td/></tr><tr><td colspan="4">&#8195;R.m.s deviations</td></tr><tr><td>&#8195;&#8195;Bond lengths (&#197;)</td><td>0.012</td><td>0.012</td><td/></tr><tr><td>&#8195;&#8195;Bond angles (&#176;)</td><td>1.53</td><td>1.50</td><td/></tr><tr><td>&#8195;MolProbity Score</td><td>1.06 (99th percentile)</td><td>1.11 (98th percentile)</td><td/></tr></tbody></table><table-wrap-foot><fn id="tblfn6"><label>*</label><p>The highest resolution shell is shown in parenthesis.</p></fn><fn id="tblfn7"><label>&#8224;</label><p>The highest resolution shell is shown in parenthesis.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06738-v1.xml">elife-06738-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.06738.003</object-id><label>Table 1.</label><caption><title>Selection of regions associated with the posterior-medial (PM) and the anterior-temporal (AT) system (<xref ref-type="bibr" rid="bib34">Libby et al., 2012</xref>).</title><p>The coordinates of the PM system reflect peak voxel coordinates of a seed-based connectivity contrast of right parahippocampal cortex &gt; right perirhinal cortex connectivity reported by Libby et al. (<xref ref-type="bibr" rid="bib34">Libby et al., 2012</xref>). The coordinates of the AT system reflect peak voxel coordinates of a seed-based connectivity contrast of right perirhinal cortex &gt; right parahippocampal cortex connectivity. Coordinates are in MNI space.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06738.003">http://dx.doi.org/10.7554/eLife.06738.003</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="3">Left hemisphere</th><th colspan="3">Left hemisphere</th></tr><tr><th>x</th><th>y</th><th>z</th><th>x</th><th>y</th><th>z</th></tr></thead><tbody><tr><td colspan="7">PM System</td></tr><tr><td>&#8195;Medial posterior occipital cortex (BA 18)</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>14</td><td>&#8722;72</td><td>8</td></tr><tr><td>&#8195;Occipital pole (BA 17)</td><td>&#8722;16</td><td>&#8722;96</td><td>22</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Parahippocampal cortex</td><td>&#8722;12</td><td>&#8722;42</td><td>&#8722;8</td><td>22</td><td>&#8722;32</td><td>&#8722;8</td></tr><tr><td>&#8195;Posterior cingulate cortex (BA 29)</td><td>&#8722;4</td><td>&#8722;46</td><td>4</td><td>10</td><td>&#8722;44</td><td>10</td></tr><tr><td>&#8195;Posterior hippocampus</td><td>&#8722;20</td><td>&#8722;30</td><td>&#8722;2</td><td>18</td><td>&#8722;36</td><td>0</td></tr><tr><td>&#8195;Posterior thalamus</td><td>&#8722;20</td><td>&#8722;34</td><td>0</td><td>22</td><td>&#8722;30</td><td>6</td></tr><tr><td>&#8195;Retrosplenial cortex (BA 30)</td><td>&#8722;16</td><td>&#8722;52</td><td>&#8722;4</td><td>22</td><td>&#8722;46</td><td>0</td></tr><tr><td colspan="7">AT System</td></tr><tr><td>&#8195;Dorsolateral prefrontal cortex (BA 9)</td><td>&#8722;24</td><td>60</td><td>24</td><td>18</td><td>58</td><td>24</td></tr><tr><td>&#8195;Dorsomedial prefrontal cortex (BA 8)</td><td>&#8722;2</td><td>&#8722;60</td><td>34</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Frontal polar cortex (BA 10)</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>40</td><td>60</td><td>&#8722;2</td></tr><tr><td>&#8195;Lateral precentral gyrus (BA 6)</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>54</td><td>4</td><td>10</td></tr><tr><td>&#8195;Medial prefrontal cortex (BA 8)</td><td>&#8722;2</td><td>&#8722;60</td><td>34</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Orbitofrontal cortex (BA 11/47)</td><td>&#8722;6</td><td>16</td><td>&#8722;22</td><td>8</td><td>22</td><td>&#8722;20</td></tr><tr><td>&#8195;Postcentral gyrus (BA 4)</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>62</td><td>&#8722;10</td><td>16</td></tr><tr><td>&#8195;Posterior superior temporal gyrus (BA 22)</td><td>&#8722;62</td><td>&#8722;34</td><td>14</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Rostrolateral prefrontal cortex (BA 10)</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>38</td><td>60</td><td>&#8722;12</td></tr><tr><td>&#8195;Temporal polar cortex (BA 38)</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>34</td><td>22</td><td>&#8722;36</td></tr><tr><td>&#8195;Ventrolateral prefrontal cortex (BA 44/45)</td><td>&#8722;56</td><td>6</td><td>18</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06808-v2.xml">elife-06808-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.06808.007</object-id><label>Table 1.</label><caption><p>Parameter values for the models shown in <xref ref-type="fig" rid="fig3 fig6 fig7">Figures 3, 6, 7</xref></p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06808.007">http://dx.doi.org/10.7554/eLife.06808.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th>Model wild-type</th><th>Model <italic>shi2-1</italic></th><th>Model <italic>SHI ox-5</italic></th><th>Model <italic>CKX2oe</italic></th><th>Model <italic>IPT1oe</italic></th></tr><tr><th>(<xref ref-type="fig" rid="fig3">Figure 3F</xref>)</th><th>(<xref ref-type="fig" rid="fig6">Figure 6A</xref>)</th><th>(<xref ref-type="fig" rid="fig6">Figure 6B</xref>)</th><th>(<xref ref-type="fig" rid="fig7">Figure 7A</xref>)</th><th>(<xref ref-type="fig" rid="fig7">Figure 7B</xref>)</th></tr></thead><tbody><tr><td><italic>H</italic><sub><italic>apex</italic></sub></td><td><italic>&#181;</italic> = 80, &#963; = 20</td><td><italic>&#181;</italic> = 48, &#963; = 12</td><td><italic>&#181;</italic> = 240, &#963; = 60</td><td/><td/></tr><tr><td><italic>H</italic></td><td><italic>&#181;</italic> = 20, &#963; = 4.5</td><td><italic>&#181;</italic> = 12, &#963; = 2.7</td><td><italic>&#181;</italic> = 60, &#963; = 13.5</td><td/><td/></tr><tr><td><italic>T</italic></td><td><italic>&#181;</italic> = 3.0, &#963; = 0.8</td><td/><td/><td><italic>&#181;</italic> = 0.45, &#963; = 0.2</td><td><italic>&#181;</italic> = 5.4, &#963; = 0.8</td></tr><tr><td><italic>&#957;</italic></td><td>0.01</td><td/><td/><td/><td/></tr><tr><td><italic>K</italic><sub><italic>A</italic></sub></td><td>0.05</td><td/><td/><td/><td/></tr><tr><td><italic>K</italic><sub><italic>B</italic></sub></td><td>0.05</td><td/><td/><td/><td/></tr></tbody></table><table-wrap-foot><fn><p>Parameter <italic>&#181;</italic> represents the mean and &#963; the variance of the normally distributed stochastic variables <italic>H</italic> and <italic>T</italic>. Parameter values left blank are identical to the wild-type model.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-06808-v2.xml">elife-06808-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.06808.008</object-id><label>Table 2.</label><caption><p>Refitted parameter values for the models shown in <xref ref-type="fig" rid="fig9">Figure 9</xref></p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.06808.008">http://dx.doi.org/10.7554/eLife.06808.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th>Model wild-type</th><th>Model 10 &#181;M DDG</th><th>Model 25 &#181;M DDG</th></tr><tr><th>(<xref ref-type="fig" rid="fig9">Figure 9A</xref>)</th><th>(<xref ref-type="fig" rid="fig9">Figure 9B</xref>)</th><th>(<xref ref-type="fig" rid="fig9">Figure 9C</xref>)</th></tr></thead><tbody><tr><td><italic>H</italic><sub><italic>apex</italic></sub></td><td><italic>&#956;</italic> = 30, &#963; = 10</td><td/><td/></tr><tr><td><italic>H</italic></td><td><italic>&#956;</italic> = 5, &#963; = 1.5</td><td/><td/></tr><tr><td><italic>T</italic></td><td><italic>&#956;</italic> = 0.7, &#963; = 0.2</td><td/><td/></tr><tr><td><italic>&#957;</italic></td><td>0.002</td><td/><td/></tr><tr><td><italic>K</italic><sub><italic>A</italic></sub></td><td>0.025</td><td>0.045</td><td>0.07</td></tr><tr><td><italic>K</italic><sub><italic>B</italic></sub></td><td>0.025</td><td>0.045</td><td>0.07</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07166-v2.xml">elife-07166-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.07166.010</object-id><label>Table 1.</label><caption><p>Electrophysiological parameters of SPs and LPs listed in the order they appear in the main text</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07166.010">http://dx.doi.org/10.7554/eLife.07166.010</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Parameter</th><th colspan="2">Patch clamp</th><th>Suction electrode</th></tr><tr><th>SPs</th><th>LPs</th><th>SPs</th></tr></thead><tbody><tr><td>V<sub>dark</sub> (mV)</td><td>&#8722;43.2 &#177; 0.7 (n = 30)</td><td>&#8722;45.9 &#177; 1.1 (n = 10)</td><td>&#8211;</td></tr><tr><td>IR<sub>membrane</sub> (M&#937;)</td><td>518 &#177; 41 (n = 8)</td><td>442 &#177; 68 (n = 9)</td><td>&#8211;</td></tr><tr><td>&#964;<sub>membrane</sub> (ms)</td><td>31.9 &#177; 4.9 (n = 8)[]<sub>a</sub></td><td>12.9 &#177; 1.3 (n = 9) [***]<sub>a</sub></td><td>&#8211;</td></tr><tr><td>Max response (mv)</td><td>30</td><td>32</td><td>&#8211;</td></tr><tr><td><italic>i</italic><sub><italic>1/2</italic></sub> (520 nm ph&#183;&#181;m<sup>&#8722;2</sup>); <italic>control</italic></td><td>149 &#177; 25 (n = 10) []<sub>b</sub></td><td>1.9 &#215; 10<sup>5</sup> &#177; 1.1 &#215; 10<sup>5</sup> (n = 7) []<sub>c</sub></td><td>&#8211;</td></tr><tr><td><italic>i</italic><sub><italic>1/2</italic></sub> (520 nm ph&#183;&#181;m<sup>&#8722;2</sup>); <italic>regenerated</italic></td><td>63 &#177; 11 (n = 12) [**]<sub>b</sub></td><td>2385 &#177; 513 (n = 7) [*]<sub>c</sub></td><td>&#8211;</td></tr><tr><td>Sensitivity 520/590; <italic>control</italic></td><td>4.4 &#177; 0.9 (n = 5) []<sub>d</sub></td><td>1.1 &#177; 0.04 (n = 6) []<sub>e</sub></td><td>&#8211;</td></tr><tr><td>Sensitivity 520/590; <italic>regenerated</italic></td><td>5.6 &#177; 1.0 (n = 5) [n.s.]<sub>d</sub></td><td>1.8 &#177; 0.1 (n = 7) [**]<sub>e</sub></td><td>&#8211;</td></tr><tr><td>Integration time, dim flash (s); <italic>control</italic></td><td>0.32 &#177; 0.05 (n = 13) []<sub>f</sub></td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>Integration time, dim flash (s); <italic>regenerated</italic></td><td>0.81 &#177; 0.14 (n = 9) [***]<sub>f</sub></td><td>&#8211;</td><td>1.45 &#177; 0.10 (n = 10)</td></tr><tr><td>TTP at <italic>i</italic><sub><italic>1/2</italic></sub> (s); <italic>regenerated</italic></td><td>0.29 &#177; 0.04 (n = 11) []<sub>g</sub></td><td>0.11 &#177; 0.007 (n = 10) [***]<sub>g</sub></td><td>&#8211;</td></tr><tr><td>&#964;<sub>rec</sub> at <italic>i</italic><sub><italic>1/2</italic></sub> (s); <italic>regenerated</italic></td><td>1.05 &#177; 0.27 (n = 11) []<sub>h</sub></td><td>0.12 &#177; 0.02 (n = 10) [***]<sub>h</sub></td><td>&#8211;</td></tr><tr><td><italic>i</italic><sub><italic>1/2</italic></sub> (&#955;<sub>max</sub> ph&#183;&#181;m<sup>&#8722;2</sup>); <italic>regenerated</italic></td><td>63 &#177; 11 (n = 12) []<sub>i</sub></td><td>777 &#177; 167 (n = 7) [***]<sub>i</sub></td><td>&#8211;</td></tr><tr><td>Dim-flash sensitivity (mV&#183;ph<sup>&#8722;1</sup>&#183;&#181;m<sup>2</sup>); <italic>regenerated</italic></td><td>0.61 &#177; 0.17 (n = 8)</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>Dim-flash sensitivity (%&#183;ph<sup>&#8722;1</sup>&#183;&#181;m<sup>2</sup>); <italic>regenerated</italic></td><td>3.0 &#177; 0.6 (n = 8)</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td><italic>a</italic> (pA); <italic>regenerated</italic></td><td>&#8211;</td><td>&#8211;</td><td>0.41 &#177; 0.04 (n = 10)</td></tr><tr><td><italic>a</italic><sub><italic>%</italic></sub> (%&#183;R<sup>* &#8722;1</sup>); <italic>regenerated</italic></td><td>2.6 &#177; 0.5 (n = 8)</td><td>&#8211;</td><td>2.6 &#177; 0.3 (n = 10)</td></tr><tr><td>SNR; <italic>regenerated</italic></td><td>&#8211;</td><td>&#8211;</td><td>1.5 &#177; 0.1 (n = 10)</td></tr><tr><td>I<sub>dark</sub> (pA); <italic>regenerated</italic></td><td>13 &#177; 3 (n = 4)</td><td>&#8211;</td><td>16 &#177; 1 (n = 10)</td></tr><tr><td>Collecting area (&#181;m<sup>2</sup>&#183;R<sup>*</sup>&#183;ph<sup>&#8722;1</sup>); <italic>regenerated</italic></td><td>&#8211;</td><td>&#8211;</td><td>0.83 &#177; 0.17 (n = 10)</td></tr><tr><td>Amplification constant (s<sup>&#8722;2</sup>); <italic>regenerated</italic></td><td>&#8211;</td><td>&#8211;</td><td>0.59 &#177; 0.09 (n = 10)</td></tr></tbody></table><table-wrap-foot><fn><p>Values are given as &#8216;mean &#177; SEM (sample size) [statistical significance]<sub>identifier letter</sub>&#8217;; n.s.: not significant; *p &lt; 0.05; **p &lt; 0.01; ***p &lt; 0.001; V<sub>dark</sub>: dark membrane potential; IR<sub>membrane</sub>: input resistance; &#964;<sub>membrane</sub>: membrane time constant; i<sub>1/2</sub>: half-maximal response flash strength; TTP: time-to-peak; &#964;<sub>rec</sub>: decay time constant; <italic>a</italic>: absolute single photon response; <italic>a</italic><sub><italic>%</italic></sub>: fractional single photon response; SNR: signal-to-noise ratio; I<sub>dark</sub>: dark current; SPs: short photoreceptors; LPs: long photoreceptors.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07432-v2.xml">elife-07432-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.07432.012</object-id><label>Table 2.</label><caption><p>Localization of the cross-links identified in Mot1:TBP:DNA:NC2 complex</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07432.012">http://dx.doi.org/10.7554/eLife.07432.012</ext-link></p><p><supplementary-material id="SD1-data"><object-id pub-id-type="doi">10.7554/eLife.07432.013</object-id><label>Table 2&#8212;source data 1.</label><caption><p>Full list of the detected cross-links.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07432.013">http://dx.doi.org/10.7554/eLife.07432.013</ext-link></p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-07432-data1-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Experiment</th><th rowspan="2">Crystal structure</th><th colspan="2">Within Mot1<sup>CTD</sup></th><th rowspan="2">Latch-crystal structure</th><th rowspan="2">Latch&#8211; Mot1<sup>CTD</sup></th><th rowspan="2">Crystal structure&#8211; Mot1<sup>CTD</sup></th><th rowspan="2">Total</th><th rowspan="2">Decoy<xref ref-type="table-fn" rid="tblfn6">&#167;</xref></th><th rowspan="2">Estimated FDR [%]</th></tr><tr><th>Intralobe<xref ref-type="table-fn" rid="tblfn3">*</xref></th><th>Interlobe<xref ref-type="table-fn" rid="tblfn4"><sup>&#8224;</sup></xref></th></tr></thead><tbody><tr><td>ADP&#183;BeF<sub>x</sub></td><td>46 (42)</td><td>21 (17)</td><td>15 (12)</td><td>8 (8)</td><td>5 (5)</td><td>37 (31) + 1 (1)<xref ref-type="table-fn" rid="tblfn5">&#8225;</xref></td><td>133 (116)</td><td>2</td><td>0.8</td></tr><tr><td>ATP&#947;S</td><td>51 (44)</td><td>17 (14)</td><td>10 (9)</td><td>11 (11)</td><td>3 (3)</td><td>37 (28)</td><td>129 (109)</td><td>1</td><td>1.5</td></tr><tr><td>ADP</td><td>40 (36)</td><td>14 (11)</td><td>7 (7)</td><td>11 (10)</td><td>4 (3)</td><td>21 (15)</td><td>97 (82)</td><td>2</td><td>2.0</td></tr></tbody></table><table-wrap-foot><fn id="tblfn3"><label>*</label><p>Within RecA1 or RecA2 subdomain.</p></fn><fn id="tblfn4"><label>&#8224;</label><p>Between RecA1 and RecA2 subdomain.</p></fn><fn id="tblfn5"><label>&#8225;</label><p>Between Mot1<sup>CTD</sup> and the linker joining N- and C-terminal domains (could not be mapped).</p></fn><fn id="tblfn6"><label>&#167;</label><p>Detected from a reverse database, estimating false-discovery rate.</p></fn><fn><p>Numbers refer to the total number of cross links, including cross-linked sites which were detected more than once (i.e., from miss-cleaved peptides). Numbers in brackets refer to non-redundant linkages only.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07432-v2.xml">elife-07432-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.07432.017</object-id><label>Table 3.</label><caption><p>Distances of the cross-links detected between RecA1 and RecA2 subdomains of Mot1<sup>CTD</sup> mapped on different structural models</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07432.017">http://dx.doi.org/10.7554/eLife.07432.017</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2">Detected linkages</th><th colspan="3">Euclidean C<sub>&#945;</sub>&#8211;C<sub>&#945;</sub> distance [&#197;]</th><th colspan="3">Total number of detected cross links</th></tr><tr><th>Residue 1</th><th>Residue 2</th><th><italic>Sso</italic>Rad54-like (&#8216;open&#8217;)</th><th><italic>Dr</italic>Rad54 (&#8216;closed&#8217;)</th><th><italic>Sc</italic>Chd1 (&#8216;semi-closed&#8217;)</th><th>ADP&#183;BeF<sub>x</sub></th><th>ATP&#947;S</th><th>ADP</th></tr></thead><tbody><tr><td rowspan="7">A</td><td>796</td><td>1013</td><td>16</td><td>22</td><td>20</td><td>1</td><td>1</td><td>1</td></tr><tr><td>796</td><td>1200</td><td>25</td><td>17</td><td>16</td><td>1</td><td>1</td><td>0</td></tr><tr><td>842</td><td>1055</td><td>58</td><td>52</td><td>52</td><td>0</td><td>1</td><td>0</td></tr><tr><td>864</td><td>1039</td><td>39</td><td>49</td><td>50</td><td>1</td><td>1</td><td>1</td></tr><tr><td>1003</td><td>1013</td><td>12</td><td>11</td><td>10</td><td>1</td><td>0</td><td>1</td></tr><tr><td>1003</td><td>1200</td><td>15</td><td>18</td><td>16</td><td>2</td><td>1</td><td>1</td></tr><tr><td>1008</td><td>1200</td><td>12</td><td>14</td><td>12</td><td>2</td><td>0</td><td>1</td></tr><tr><td rowspan="2">B</td><td>864</td><td>1200</td><td>34</td><td>24</td><td>24</td><td>1</td><td>1</td><td>1</td></tr><tr><td>919</td><td>1086</td><td>25</td><td>42</td><td>47</td><td>1</td><td>1</td><td>1</td></tr><tr><td rowspan="4">C</td><td>865</td><td>1200</td><td>31</td><td>20</td><td>21</td><td>1</td><td>1</td><td>0</td></tr><tr><td>919</td><td>1051</td><td>63</td><td>22</td><td>43</td><td>1</td><td>0</td><td>0</td></tr><tr><td>919</td><td>1055</td><td>57</td><td>19</td><td>36</td><td>2</td><td>2</td><td>0</td></tr><tr><td>919</td><td>1060</td><td>51</td><td>25</td><td>36</td><td>1</td><td>0</td><td>0</td></tr></tbody></table><table-wrap-foot><fn><p>Part A shows the cross links, which do not distinguish between the conformations. Part B and C list cross-links, for which the mapped distances were significantly different depending on the model used (i.e., &lt;30 &#197; for one model and &gt;30 &#197; for another). Eight of these cross-links (shown also in <xref ref-type="fig" rid="fig6">Figure 6C</xref>) were detected only in the presence of ADP&#183;BeF<sub>x</sub> and ATP&#947;S and are listed in part C.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07433-v2.xml">elife-07433-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.07433.017</object-id><label>Table 2.</label><caption><p>Amino acid changes between PSI-LHCI structures</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07433.017">http://dx.doi.org/10.7554/eLife.07433.017</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Subunit</th><th rowspan="2">Number of amino acids</th><th colspan="2">Modeled amino acids</th><th rowspan="2">Number of changes</th></tr><tr><th>4Y28</th><th>2WSC</th></tr></thead><tbody><tr><td>PsaA</td><td>758</td><td>741</td><td>729</td><td>1</td></tr><tr><td>PsaB</td><td>734</td><td>732</td><td>732</td><td>1</td></tr><tr><td>PsaC</td><td>81</td><td>80</td><td>81</td><td>1</td></tr><tr><td>PsaD*</td><td>156</td><td>140</td><td>135</td><td>5</td></tr><tr><td>PsaE*</td><td>92</td><td>68</td><td>65</td><td>9</td></tr><tr><td>PsaF*</td><td>154</td><td>150</td><td>154</td><td>18</td></tr><tr><td>PsaG*</td><td>98</td><td>95</td><td>95</td><td>15</td></tr><tr><td>PsaH*</td><td>95</td><td>84</td><td>69</td><td>11</td></tr><tr><td>PsaJ*</td><td>42</td><td>41</td><td>42</td><td>5</td></tr><tr><td>PsaK*</td><td>134</td><td>79</td><td>85</td><td>10</td></tr><tr><td>PsaL*</td><td>168</td><td>160</td><td>161</td><td>21</td></tr><tr><td>Lhca1*</td><td>204</td><td>193</td><td>165</td><td>12</td></tr><tr><td>Lhca2</td><td>256</td><td>206</td><td>176</td><td>3</td></tr><tr><td>Lhca3</td><td>242</td><td>210</td><td>162</td><td>16</td></tr><tr><td>Lhca4</td><td>252</td><td>197</td><td>166</td><td>3</td></tr></tbody></table><table-wrap-foot><fn><p>The number of modeled amino acids in each subunit is shown and compared to the most recent PSI-LHCI structure (2WSC). Insertions, deletions and extensions are counted as a single change. Since the genome sequence of <italic>Pisum Sativum</italic> is not completely known (genes with no DNA data are marked with an *) we relied on high-throughput mRNA sequence data for verification (<xref ref-type="bibr" rid="bib22">Franssen et al., 2011</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07460-v1.xml">elife-07460-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.07460.002</object-id><label>Table 1.</label><caption><p>Number of variants and derived puzzles used in Fraxinus version 1</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07460.002">http://dx.doi.org/10.7554/eLife.07460.002</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Fungal sample</th><th colspan="2">SNP</th><th colspan="2">INDEL</th></tr><tr><th>Variants</th><th>Puzzles</th><th>Variants</th><th>Puzzles</th></tr></thead><tbody><tr><td>Ashwellthorpe1</td><td>250</td><td>2937</td><td>53</td><td>521</td></tr><tr><td>Ashwellthorpe2</td><td>353</td><td>1121</td><td>51</td><td>170</td></tr><tr><td>Upton broad and marshes1</td><td>397</td><td>4964</td><td>56</td><td>374</td></tr><tr><td>Total</td><td>1000</td><td>9022</td><td>160</td><td>1065</td></tr></tbody></table><table-wrap-foot><fn><p>SNP, single nucleotide polymorphisms; INDEL, insertion&#8211;deletion polymorphism.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07600-v2.xml">elife-07600-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.07600.017</object-id><label>Table 1.</label><caption><p>Benchmark of MCM on RNAs with crystal structures</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07600.017">http://dx.doi.org/10.7554/eLife.07600.017</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">RNA</th><th rowspan="2">Length</th><th colspan="2">M<sup>2</sup>/Rosetta (no MOHCA, control)</th><th colspan="2">MCM</th></tr><tr><th>RMSD to crystal (&#197;) (accuracy)</th><th>p-value<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></th><th>RMSD to crystal (&#197;) (accuracy)</th><th>p-value<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></th></tr></thead><tbody><tr><td><italic>Tetrahymena</italic> ribozyme P4&#8211;P6 domain</td><td>158</td><td>38.3</td><td>&gt;0.9</td><td>8.6</td><td>&lt;1.0 &#215; 10<sup>&#8722;16</sup></td></tr><tr><td>V. cholerae cyclic-di-GMP riboswitch aptamer, ligand-bound<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>89</td><td>11.3</td><td>2.6 &#215; 10<sup>&#8722;3</sup></td><td>7.6</td><td>6.3 &#215; 10<sup>&#8722;7</sup></td></tr><tr><td>F. nucleatum double glycine riboswitch ligand-binding domain, ligand-bound<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>159</td><td>30.5</td><td>&gt;0.9</td><td>7.9</td><td>&lt;1.0 &#215; 10<sup>&#8722;16</sup></td></tr><tr><td>S. thermophilum adenosylcobalamin riboswitch aptamer, ligand-bound<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>168</td><td>17.1<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>5.3 &#215; 10<sup>&#8722;7</sup></td><td>11.9</td><td>4.0 &#215; 10<sup>&#8722;15</sup></td></tr><tr><td>Class I ligase</td><td>127</td><td>26.3</td><td>&gt;0.9</td><td>14.5</td><td>6.8 &#215; 10<sup>&#8722;5</sup></td></tr><tr><td>Class I ligase, core domain<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>87</td><td>14.0</td><td>0.13</td><td>11.1</td><td>3.1 &#215; 10<sup>&#8722;3</sup></td></tr><tr><td>D. iridis lariat-capping ribozyme</td><td>188</td><td>9.6<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>&lt;1.0 &#215; 10<sup>&#8722;16</sup></td><td>8.2</td><td>&lt;1.0 &#215; 10<sup>&#8722;16</sup></td></tr><tr><td><italic>D. iridis</italic> lariat-capping ribozyme, MCM refined regions<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>69</td><td>17.0<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>n.a.<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td><td>11.2</td><td>n.a.<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>MCM modeling was performed with MOHCA-seq constraints from data sets collected on the ligand-bound state; ligands were not included during Rosetta modeling.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>M<sup>2</sup>/Rosetta statistics are reported for RNA-puzzle submission rank 1 models, which included subdomains built by homology modeling.</p></fn><fn id="tblfn3"><label>&#8225;</label><p>Calculated over core domain residues or refined regions after alignment using MAMMOTH; (<xref ref-type="bibr" rid="bib54">Ortiz et al., 2002</xref>) see &#8216;Materials and methods&#8217;.</p></fn><fn id="tblfn4"><label>&#167;</label><p>p-value computed using analytical formula for secondary-structure-constrained 3D modeling in ref. (<xref ref-type="bibr" rid="bib32">Hajdin et al., 2010</xref>); it is not applicable to peripheral domains. Values above 0.9 are not well-determined and are presented as &gt; 0.9.</p></fn><fn><p>MCM: multidimensional chemical mapping; RMSD: root-mean-squared-deviation.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07616-v1.xml">elife-07616-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.07616.003</object-id><label>Table 1.</label><caption><p>Description of environments defined by principal components analysis (PCA) of physico-chemical parameters</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07616.003">http://dx.doi.org/10.7554/eLife.07616.003</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="4">Akonolinga</th><th colspan="4">Bankim</th></tr><tr><th>PC1</th><th>PC2</th><th>PC3</th><th>PC4</th><th>PC1</th><th>PC2</th><th>PC3</th><th>PC4</th></tr></thead><tbody><tr><td>Variance explained</td><td>0.47</td><td>0.31</td><td>0.14</td><td>0.09</td><td>0.59</td><td>0.22</td><td>0.13</td><td>0.07</td></tr><tr><td colspan="9">Loadings</td></tr><tr><td>&#8195;<italic>pH</italic></td><td>&#8722;0.33</td><td>0.7</td><td>&#8722;0.47</td><td>0.41</td><td>&#8722;0.51</td><td>0.51</td><td>&#8722;0.43</td><td>0.55</td></tr><tr><td>&#8195;Dissolved oxygen</td><td>&#8722;0.6</td><td>0.31</td><td>0.31</td><td>&#8722;0.67</td><td>&#8722;0.57</td><td>0.33</td><td>0.11</td><td>&#8722;0.75</td></tr><tr><td>&#8195;Water flow</td><td>&#8722;0.59</td><td>&#8722;0.32</td><td>0.46</td><td>0.59</td><td>&#8722;0.51</td><td>&#8722;0.29</td><td>0.72</td><td>0.36</td></tr><tr><td>&#8195;Temperature</td><td>0.43</td><td>0.55</td><td>0.69</td><td>0.19</td><td>0.4</td><td>0.74</td><td>0.53</td><td>0.1</td></tr></tbody></table><table-wrap-foot><fn><p>Separate PCA was performed for Akonolinga and Bankim, and only the most potentially relevant parameters were included.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07616-v1.xml">elife-07616-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.07616.011</object-id><label>Appendix table 1.</label><caption><p>Results of multivariate analyses for Akonolinga (12 months of sampling) and Bankim (4 months of sampling)</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07616.011">http://dx.doi.org/10.7554/eLife.07616.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Variable</th><th colspan="3">Akonolinga (n = 183)</th><th colspan="3">Bankim (n = 61)</th></tr><tr><th>Effect</th><th>Std. error</th><th>p-value</th><th>Effect</th><th>Std. error</th><th>p-value</th></tr></thead><tbody><tr><td>Model AIC</td><td>400.7</td><td>&#8211;</td><td>&#8211;</td><td>182.7</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>Variance of random effect</td><td>0.20</td><td>&#8211;</td><td>&#8211;</td><td>1.77</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>(Intercept)</td><td>&#8722;12.56</td><td>4.40</td><td>&lt;0.001</td><td>&#8722;7.40</td><td>1.97</td><td>&lt;0.001</td></tr><tr><td colspan="7">Seasonality</td></tr><tr><td>&#8195;Sine(2pi*Month/12)</td><td>0.34</td><td>0.14</td><td>0.02</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Sine(2pi*Month/4)</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Cos(2pi*Month/12)</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Cos(2pi*Month/4)</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td colspan="7">Physico-chemical parameters</td></tr><tr><td>&#8195;Temperature</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;<italic>pH</italic></td><td>8.63</td><td>2.44</td><td>&lt;0.001</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Dissolved oxygen</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Conductivity</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Iron</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Water flow (Lentic)</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8722;2.10</td><td>0.47</td><td>&lt;0.001</td></tr><tr><td>&#8195;Water flow (Lotic)</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8722;3.18</td><td>0.69</td><td>&lt;0.001</td></tr><tr><td colspan="7">Physico-chemical parameters (PCA)</td></tr><tr><td>&#8195;PC1</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;PC2</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;PC3</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td colspan="7">Community</td></tr><tr><td>&#8195;Abundance</td><td>&#8722;0.64</td><td>0.17</td><td>&lt;0.001</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Shannon</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>4.16</td><td>0.97</td><td>&lt;0.001</td></tr><tr><td colspan="7">Orders (%)</td></tr><tr><td>&#8195;Fish</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8722;1.62</td><td>0.35</td><td>&lt;0.001</td></tr><tr><td>&#8195;Anura</td><td>&#8722;0.34</td><td>0.14</td><td>0.02</td><td>&#8722;0.84</td><td>0.32</td><td>0.01</td></tr><tr><td>&#8195;Gastropoda</td><td>&#8722;0.64</td><td>0.16</td><td>&lt;0.001</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Decapoda (presence)</td><td>&#8722;1.37</td><td>0.37</td><td>&lt;0.001</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Odonata</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Ephemeroptera</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8722;0.94</td><td>0.21</td><td>&lt;0.001</td></tr><tr><td>&#8195;Hemiptera</td><td>&#8722;0.47</td><td>0.20</td><td>0.02</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Tricoptera</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Oligochaeta (presence)</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Hirudinea (presence)</td><td>0.59</td><td>0.23</td><td>0.01</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Coleoptera</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>&#8195;Diptera</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>1.08</td><td>0.36</td><td>&lt;0.001</td></tr><tr><td>&#8195;Hydracarine</td><td>&#8211;</td><td>&#8211;</td><td>&#8211;</td><td>&#8722;1.58</td><td>0.49</td><td>&lt;0.001</td></tr></tbody></table><table-wrap-foot><fn><p>The models used are Binomial regressions with random effect site, selected with forward&#8211;backwards procedure (see section 1 for details).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07646-v2.xml">elife-07646-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.07646.007</object-id><label>Table 2.</label><caption><p>Ago2-target affinities</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07646.007">http://dx.doi.org/10.7554/eLife.07646.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">t1 nucleotide</th><th colspan="2"><italic>K</italic><sub>D</sub> (nM)</th></tr><tr><th>WT Ago2</th><th>A481T-Ago2</th></tr></thead><tbody><tr><td>A</td><td>0.75 &#177; 0.04</td><td>1.5 &#177; 0.13</td></tr><tr><td>G</td><td>1.9 &#177; 0.09</td><td>1.8 &#177; 0.11</td></tr><tr><td>U</td><td>1.9 &#177; 0.10</td><td>&#8211;</td></tr><tr><td>C</td><td>1.8 &#177; 0.12</td><td>&#8211;</td></tr><tr><td>DAP</td><td>0.45 &#177; 0.03</td><td>&#8211;</td></tr><tr><td>I</td><td>1.7 &#177; 0.09</td><td>&#8211;</td></tr><tr><td>m6A</td><td>1.8 &#177; 0.12</td><td>&#8211;</td></tr></tbody></table><table-wrap-foot><fn><p>Dissociation constants for wild-type (WT) and mutant Ago2 binding short target RNAs with different t1 nucleotides.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07750-v2.xml">elife-07750-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.07750.005</object-id><label>Table 1.</label><caption><p>Average Y&#697; telomere length in <italic>sir</italic>&#916; cells containing TLC1, tlc1&#916;48, or TLC1(Ku)<sub>3</sub></p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07750.005">http://dx.doi.org/10.7554/eLife.07750.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><italic>SIR</italic> Genotype</th><th colspan="3"><italic>TLC1</italic> Genotype</th></tr><tr><th><italic>TLC1</italic></th><th><italic>tlc1</italic>&#916;<italic>48</italic></th><th><italic>TLC1(Ku)</italic><sub><italic>3</italic></sub></th></tr></thead><tbody><tr><td><italic>SIR</italic></td><td>0</td><td>&#8722;86 &#177; 23<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>Dysregulated<xref ref-type="table-fn" rid="tblfn1">*</xref></td></tr><tr><td><italic>sir4</italic>&#916;</td><td>&#8722;53 &#177; 13<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>&#8722;71 &#177; 26<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>&#8722;148 &#177; 36<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td></tr><tr><td><italic>sir2</italic>&#916;<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td><td>&#8722;41 &#177; 16</td><td>&#8722;50 &#177; 74</td><td>&#8722;71 &#177; 26</td></tr><tr><td><italic>sir3</italic>&#916;<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td><td>&#8722;51 &#177; 20</td><td>&#8722;84 &#177; 14</td><td>&#8722;123 &#177; 2</td></tr></tbody></table><table-wrap-foot><fn><p>The weighted-average mobility of the Y&#697; telomeric restriction fragments was calculated as described in the &#8216;Materials and methods&#8217;. The numbers shown are averages of multiple biological-replicate samples &#177; standard deviation.</p></fn><fn id="tblfn1"><label>*</label><p>Y&#697; telomere length was not quantified in this condition because signal from Y&#697; telomere restriction fragments overlapped with that from the non-telomeric control fragment.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>n = 6.</p></fn><fn id="tblfn3"><label>&#8225;</label><p>n = 4.</p></fn><fn id="tblfn4"><label>&#167;</label><p>n = 2.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07902-v2.xml">elife-07902-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.07902.005</object-id><label>Table 1.</label><caption><p>Descriptive statistics for the three behavioural conditions</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07902.005">http://dx.doi.org/10.7554/eLife.07902.005</ext-link></p><p><supplementary-material id="SD1-data"><object-id pub-id-type="doi">10.7554/eLife.07902.006</object-id><label>Table 1&#8212;source data 1.</label><caption><p>Raw behavioural data from the oddity task.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07902.006">http://dx.doi.org/10.7554/eLife.07902.006</ext-link></p></caption><media mime-subtype="csv" mimetype="text" xlink:href="elife-07902-data1-v2.csv"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2">Accuracy</th><th colspan="2">RT</th></tr><tr><th><italic>Mean</italic></th><th><italic>SD</italic></th><th><italic>Mean</italic></th><th><italic>SD</italic></th></tr></thead><tbody><tr><td>Face</td><td>0.87</td><td>0.07</td><td>2932.40</td><td>479.68</td></tr><tr><td>Scene</td><td>0.85</td><td>0.06</td><td>3147.86</td><td>557.49</td></tr><tr><td>Size</td><td>0.84</td><td>0.12</td><td>2464.10</td><td>668.31</td></tr></tbody></table><table-wrap-foot><fn><p>Mean and standard deviation (SD) are reported for both accuracy (proportion correct) and reaction time (RT). Raw behavioural data for the three oddity categories are available in <xref ref-type="supplementary-material" rid="SD1-data">Table 1&#8212;source data 1</xref>.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07902-v2.xml">elife-07902-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.07902.007</object-id><label>Table 2.</label><caption><p>Descriptive statistics for the fornix (left) and inferior longitudinal fasciculus (ILF, right)</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07902.007">http://dx.doi.org/10.7554/eLife.07902.007</ext-link></p><p><supplementary-material id="SD2-data"><object-id pub-id-type="doi">10.7554/eLife.07902.008</object-id><label>Table 2&#8212;source data 1.</label><caption><p>Raw values for the DTI metrics.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07902.008">http://dx.doi.org/10.7554/eLife.07902.008</ext-link></p></caption><media mime-subtype="csv" mimetype="text" xlink:href="elife-07902-data2-v2.csv"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2">Fornix</th><th colspan="2">ILF</th></tr><tr><th><italic>Mean</italic></th><th><italic>SD</italic></th><th><italic>Mean</italic></th><th><italic>SD</italic></th></tr></thead><tbody><tr><td>FA</td><td>0.373</td><td>0.031</td><td>0.429</td><td>0.023</td></tr><tr><td>MD</td><td>1.058</td><td>0.054</td><td>0.758</td><td>0.017</td></tr></tbody></table><table-wrap-foot><fn><p>Mean and standard deviations are reported for fractional anisotropy (FA) and mean diffusivity (MD &#215; 10<sup>&#8722;3</sup> mm<sup>2</sup> s<sup>&#8722;1</sup>). Individual FA and MD values for each participant (and tract) are available in <xref ref-type="supplementary-material" rid="SD2-data">Table 2&#8212;source data 1</xref>.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07902-v2.xml">elife-07902-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.07902.009</object-id><label>Table 3.</label><caption><p>Summary table for the DTI-behaviour correlations</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07902.009">http://dx.doi.org/10.7554/eLife.07902.009</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"/><th colspan="4">Fornix</th><th colspan="4">ILF</th></tr><tr><th colspan="2">MD</th><th colspan="2">FA</th><th colspan="2">MD</th><th colspan="2">FA</th></tr><tr><th><italic>r</italic></th><th><italic>p</italic></th><th><italic>r</italic></th><th><italic>p</italic></th><th><italic>r</italic></th><th><italic>p</italic></th><th><italic>r</italic></th><th><italic>p</italic></th></tr></thead><tbody><tr><td>Face</td><td>&#8722;0.14</td><td>0.23</td><td>0.15</td><td>0.22</td><td>&#8722;0.57</td><td>0.00</td><td>0.30</td><td>0.06</td></tr><tr><td>Scene</td><td>&#8722;0.46</td><td>0.01</td><td>0.36</td><td>0.03</td><td>0.12</td><td>0.27</td><td>0.04</td><td>0.43</td></tr><tr><td>Size</td><td>&#8722;0.29</td><td>0.06</td><td>0.18</td><td>0.18</td><td>&#8722;0.26</td><td>0.09</td><td>0.09</td><td>0.32</td></tr></tbody></table><table-wrap-foot><fn><p>Correlation coefficients (and one-tailed p values) are reported for each metric (fractional anisotropy [FA], mean diffusivity [MD]) of fornix and inferior longitudinal fasciculus (ILF) microstructure and each task condition (face, scene, size).</p></fn><fn><p>DTI, diffusion tensor imaging.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07903-v2.xml">elife-07903-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.07903.011</object-id><label>Table 1.</label><caption><p>Reaction times (mean <bold>&#177;</bold> SEM) for the test phase.</p> <p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07903.011">http://dx.doi.org/10.7554/eLife.07903.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2"/><th colspan="3">HR</th><th colspan="3">LR</th></tr><tr><th>Direct</th><th>Inference 1</th><th>Inference 2</th><th>Direct</th><th>Inference 1</th><th>Inference 2</th></tr></thead><tbody><tr><td rowspan="2">Sleep</td><td>Mean</td><td>939.83 &#177; 19.16</td><td>1144.29 &#177; 21.51</td><td>1167.24 &#177; 22.35</td><td>1207.46 &#177; 21.36</td><td>1434.43 &#177; 22.69</td><td>1318.69 &#177; 22.61</td></tr><tr><td>Median</td><td>752.07 &#177; 18.59</td><td>1006.20 &#177; 21.39</td><td>1024.83 &#177; 21.86</td><td>1066.83 &#177; 20.12</td><td>1385.43 &#177; 23.82</td><td>1132.13 &#177; 22.21</td></tr><tr><td rowspan="2">Wake</td><td>Mean</td><td>1164.14 &#177; 19.18</td><td>1353 &#177; 27.81</td><td>1405.74 &#177; 21.91</td><td>1220.55 &#177; 20.87</td><td>1305.55 &#177; 21.74</td><td>1290.21 &#177; 26.50</td></tr><tr><td>Median</td><td>994.17 &#177; 20.83</td><td>1226.83 &#177; 29.18</td><td>1261.27 &#177; 22.99</td><td>1067.90 &#177; 21.86</td><td>1197.50 &#177; 23.13</td><td>1231.40 &#177; 27.78</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07903-v2.xml">elife-07903-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl4" position="float"><object-id pub-id-type="doi">10.7554/eLife.07903.014</object-id><label>Table 4.</label><caption><p>Reaction times (mean <bold>&#177;</bold> SEM) for the certain answers of the test phase.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07903.014">http://dx.doi.org/10.7554/eLife.07903.014</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2"/><th align="center" colspan="3"><p>HR</p> </th><th align="center" colspan="3"><p>LR</p></th></tr><tr><th align="center"><p>Direct</p></th><th>Inference 1</th><th align="center"><p>Inference 2</p></th><th>Direct</th><th>Inference 1</th><th align="center"><p>Inference 2</p></th></tr></thead><tbody><tr><td align="center" rowspan="2"><p>Sleep</p></td><td>Mean</td><td align="center"><p>727.98 &#177; 18.52</p></td><td>876.96 &#177; 19.29</td><td>882.20 &#177; 20.68</td><td align="center"><p>609.91 &#177; 21.69</p></td><td>952.90 &#177; 21.21</td><td>721.28 &#177; 23.30</td></tr><tr><td>Median</td><td align="center"><p>665.43 &#177; 17.20</p></td><td>792.3 &#177; 18.77</td><td>770.67 &#177; 19.71</td><td align="center"><p>553.5 &#177; 21.06</p></td><td>861.167 &#177; 21.05</td><td>682.93 &#177; 22.71</td></tr><tr><td align="center" rowspan="2"><p>Wake</p></td><td>Mean</td><td align="center"><p>844.88 &#177; 18.21</p></td><td>1064.95 &#177; 24.45</td><td>1095.13 &#177; 27.81</td><td align="center"><p>686.73 &#177; 24.63</p></td><td>881.59 &#177; 21.59</td><td>767.54 &#177; 22.88</td></tr><tr><td>Median</td><td align="center"><p>743.23 &#177; 18.11</p></td><td>1086.60 &#177; 28.67</td><td>1068.27 &#177; 27.28</td><td align="center"><p>637.33 &#177; 23.90</p></td><td>712.17 &#177; 20.62</td><td>651.80 &#177; 22.05</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07916-v2.xml">elife-07916-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.07916.008</object-id><label>Table 1.</label><caption><p>Donors included in the study.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07916.008">http://dx.doi.org/10.7554/eLife.07916.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">All donors</th><th>cHCV</th><th>SVR</th><th>HD</th></tr><tr><th>n=29</th><th>n=37</th><th>n=25</th></tr></thead><tbody><tr><td>Male, n (%)</td><td>16 (55)</td><td>21 (57)</td><td>12 (48)</td></tr><tr><td>Age, years, median (IQR1-3)</td><td>48 (42-55)</td><td>48 (44-58)</td><td>38 (31-46)</td></tr><tr><td>IgG anti-CMV positive, n (%)</td><td>14 (48)</td><td>24 (51)</td><td>11 (44)</td></tr><tr><td>Cirrhosis, n (%)</td><td>5 (17)</td><td>8 (22)</td><td>na</td></tr><tr><td>Treatment experienced, n (%)</td><td>18 (62)</td><td>37 (100)</td><td>na</td></tr><tr><td>Treatment (n per type: 0/1/2/3)</td><td>11/12/6/0</td><td>0/26/10/1</td><td>na</td></tr><tr><td>Delay post-treatment, years, median (IQR1-3)</td><td>3.8 (3.4-4.2)</td><td>1.7 (0.9-3.2)</td><td>na</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07919-v2.xml">elife-07919-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.07919.009</object-id><label>Table 1.</label><caption><p>Synapse properties of pair recordings</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07919.009">http://dx.doi.org/10.7554/eLife.07919.009</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="3">Amplitude (pA)</th><th rowspan="2">Success rate</th><th rowspan="2">Neurolucida analysis</th><th rowspan="2">Distance from soma (&#181;m)</th></tr><tr><th>mean</th><th>sd</th><th>max</th></tr></thead><tbody><tr><td>IPSC</td><td/><td/><td/><td/><td/><td/></tr><tr><td>CS4</td><td>&#8722;5.7</td><td>5.1</td><td>&#8722;19.5</td><td>0.6</td><td/><td/></tr><tr><td>CS8</td><td>&#8722;8.6</td><td>4.0</td><td>&#8722;17.6</td><td>0.5</td><td>yes</td><td>48.8</td></tr><tr><td>CS20</td><td>&#8722;10.9</td><td>5.3</td><td>&#8722;27.4</td><td>0.9</td><td>yes</td><td>51.8</td></tr><tr><td>CS21<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>&#8722;7.6</td><td>3.2</td><td>&#8722;17.8</td><td>0.6</td><td/><td/></tr><tr><td>CS22<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>&#8722;8.0</td><td>3.4</td><td>&#8722;14.5</td><td>0.6</td><td/><td/></tr><tr><td>CS28</td><td>&#8722;76.9</td><td>20.9</td><td>&#8722;107.3</td><td>1.0</td><td>yes</td><td>48.8</td></tr><tr><td>CS36<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>&#8722;6.5</td><td>2.6</td><td>&#8722;12.5</td><td>0.4</td><td/><td/></tr><tr><td>CS41</td><td>&#8722;8.6</td><td>4.3</td><td>&#8722;20.8</td><td>0.7</td><td>yes</td><td>41.3</td></tr><tr><td>CS44</td><td>&#8722;6.2</td><td>2.1</td><td>&#8722;12.6</td><td>0.5</td><td>yes</td><td>66.6</td></tr><tr><td>CS45<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>&#8722;7.1</td><td>4.1</td><td>&#8722;21.2</td><td>0.7</td><td>yes</td><td>53.2</td></tr><tr><td>CS55</td><td>&#8722;91.3</td><td>11.2</td><td>&#8722;111.0</td><td>1.0</td><td>yes</td><td>35.8</td></tr><tr><td>CS56</td><td>&#8722;17.3</td><td>3.0</td><td>&#8722;24.9</td><td>1.0</td><td>yes</td><td>20.6</td></tr><tr><td>CS61</td><td>&#8722;9.6</td><td>4.6</td><td>&#8722;22.2</td><td>0.8</td><td/><td/></tr><tr><td>CS62</td><td>&#8722;36.4</td><td>14.0</td><td>&#8722;69.5</td><td>1.0</td><td/><td/></tr><tr><td>EPSC</td><td/><td/><td/><td/><td/><td/></tr><tr><td>CS10</td><td>67.5</td><td>22.2</td><td>109.3</td><td>1.0</td><td>yes</td><td>26.5</td></tr><tr><td>CS21<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>18.6</td><td>8.7</td><td>44.5</td><td>0.9</td><td/><td/></tr><tr><td>CS22<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>70.9</td><td>38.3</td><td>201.6</td><td>1.0</td><td/><td/></tr><tr><td>CS23</td><td>45.3</td><td>14.2</td><td>83.4</td><td>1.0</td><td>yes</td><td>51.2</td></tr><tr><td>CS36<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>4.4</td><td>1.0</td><td>6.5</td><td>0.5</td><td/><td/></tr><tr><td>CS45<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>43.1</td><td>19.0</td><td>86.1</td><td>1.0</td><td>yes</td><td>53.2</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>Reciprocal connection between FS and pyramidal cell was observed.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-07919-v2.xml">elife-07919-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.07919.023</object-id><label>Table 5.</label><caption><p>IPSC properties of pair CS56 and CS55</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.07919.023">http://dx.doi.org/10.7554/eLife.07919.023</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"/><th colspan="2">CS56</th><th colspan="2">CS55</th></tr><tr><th>Electric</th><th>peak</th><th>Electric</th><th>peak</th></tr><tr><th>Charge (fC)</th><th>(pA)</th><th>Charge (fC)</th><th>(pA)</th></tr></thead><tbody><tr><td>Average</td><td>193.1</td><td>&#8722;17.3</td><td>895.2</td><td>&#8722;91.3</td></tr><tr><td>SD</td><td>56.2</td><td>3.0</td><td>96.2</td><td>11.2</td></tr><tr><td>Max</td><td>326.1</td><td>&#8722;24.9</td><td>1057.8</td><td>&#8722;111.0</td></tr><tr><td>Min</td><td>89.9</td><td>&#8722;11.8</td><td>766.0</td><td>&#8722;74.0</td></tr><tr><td>n</td><td>60</td><td>60</td><td>10</td><td>10</td></tr><tr><td>Average Trace</td><td>217.5</td><td>&#8722;14.2</td><td>994.6</td><td>&#8722;89.4</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-08105-v1.xml">elife-08105-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.08105.011</object-id><label>Table 2.</label><caption><p>Replay statistics</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.08105.011">http://dx.doi.org/10.7554/eLife.08105.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="5">Correct trials</th></tr><tr><th colspan="2">Candidate-start</th><th colspan="3">Candidate-stem</th></tr></thead><tbody><tr><td>Number</td><td colspan="2">1893</td><td/><td colspan="2">810</td></tr><tr><td>Replay type</td><td>Path-start</td><td>Episodic-start</td><td>Path-stem</td><td>Episodic-center</td><td>Episodic-junction</td></tr><tr><td>Number</td><td>345</td><td>71</td><td>190</td><td>17</td><td>17</td></tr><tr><td>Percent confirmed</td><td>18.2%</td><td>20.6%</td><td>23.5%</td><td>21.5%</td><td>15.3%</td></tr><tr><td>Percent SWR coincident</td><td>100%</td><td>100%</td><td>99.5%</td><td>98.7%</td><td>100%</td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="5">Erroneous trials</th></tr><tr><th colspan="2">Candidate-start</th><th colspan="3">Candidate-stem</th></tr></thead><tbody><tr><td>Number</td><td>59</td><td/><td colspan="3">58</td></tr><tr><td>Replay type</td><td>Path-start</td><td>Episodic-start</td><td>Path-stem</td><td>Episodic-center</td><td>Episodic-junction</td></tr><tr><td>Number</td><td>9</td><td>1</td><td>8</td><td>0</td><td>2</td></tr><tr><td>Percent confirmed</td><td>15.3%</td><td>12.5%</td><td>13.8%</td><td>0%</td><td>25.0%</td></tr><tr><td>Percent SWR coincident</td><td>100%</td><td>100%</td><td>100%</td><td>&#8211;</td><td>100%</td></tr></tbody></table><table-wrap-foot><fn><p>SWR, sharp wave/ripple.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-08401-v2.xml">elife-08401-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.08401.005</object-id><label>Table 1.</label><caption><p>The list of identified mutp53-interaction protein candidates</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.08401.005">http://dx.doi.org/10.7554/eLife.08401.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene names</th><th colspan="2">Average counts</th></tr><tr><th>Normal thymus</th><th>Thymic lymphoma</th></tr></thead><tbody><tr><td>Ctsb</td><td>0</td><td>19</td></tr><tr><td>Tfrc</td><td>0</td><td>15.5</td></tr><tr><td>Bag2</td><td>0</td><td>15</td></tr><tr><td>Stip1</td><td>0</td><td>14.5</td></tr><tr><td>Hyou1</td><td>0</td><td>14.5</td></tr><tr><td>Cad</td><td>0</td><td>14.5</td></tr><tr><td>Rps19</td><td>0</td><td>14.5</td></tr><tr><td>Pfn1</td><td>0</td><td>14</td></tr><tr><td>Cand1</td><td>0</td><td>11.5</td></tr><tr><td>Hspa2</td><td>0</td><td>11</td></tr><tr><td>Lcp1</td><td>0</td><td>11</td></tr><tr><td>Sar1a</td><td>0</td><td>10.5</td></tr><tr><td>Fam49b</td><td>0</td><td>10.5</td></tr><tr><td>Khsrp</td><td>0</td><td>10</td></tr><tr><td>Ifi47</td><td>0</td><td>9.5</td></tr><tr><td>Cse1l</td><td>0</td><td>9.5</td></tr><tr><td>Ipo5</td><td>0</td><td>9.5</td></tr><tr><td>Hsp90b1</td><td>0</td><td>9</td></tr><tr><td>Hspbp1</td><td>0</td><td>8.5</td></tr><tr><td>Rfc5</td><td>0</td><td>8.5</td></tr><tr><td>Tkt</td><td>0</td><td>8.5</td></tr><tr><td>myosin</td><td>0</td><td>8</td></tr><tr><td>Hadhb</td><td>0</td><td>8</td></tr><tr><td>Hsp70</td><td>1</td><td>66</td></tr><tr><td>Phgdh</td><td>1</td><td>20</td></tr><tr><td>Myh9</td><td>5</td><td>81.5</td></tr><tr><td>Hspd1</td><td>2</td><td>32.5</td></tr><tr><td>Rpl9-ps4</td><td>1</td><td>16</td></tr><tr><td>Ubr5</td><td>1</td><td>13.5</td></tr><tr><td>Dars</td><td>1</td><td>13.5</td></tr><tr><td>Iqgap1</td><td>1</td><td>12</td></tr><tr><td>Slc25a3</td><td>1</td><td>11.5</td></tr><tr><td>Rars</td><td>1</td><td>11.5</td></tr><tr><td>Ruvbl2</td><td>1</td><td>11</td></tr><tr><td>Ddb1</td><td>1</td><td>10</td></tr><tr><td>Hsph1</td><td>4</td><td>38.5</td></tr><tr><td>Dnajb4</td><td>1</td><td>9</td></tr><tr><td>Aldoa</td><td>1</td><td>8.5</td></tr><tr><td>Pcna</td><td>2</td><td>16.5</td></tr><tr><td>Eprs</td><td>1</td><td>8</td></tr><tr><td>Hsp90</td><td>2</td><td>15.5</td></tr><tr><td>Gm9755</td><td>2</td><td>12.5</td></tr><tr><td>Dnaja1</td><td>3</td><td>18</td></tr><tr><td>Atp5b</td><td>3</td><td>18</td></tr><tr><td>Cltc</td><td>7</td><td>41</td></tr><tr><td>Gm5506</td><td>5</td><td>26</td></tr><tr><td>Dnaja2</td><td>3</td><td>15</td></tr><tr><td>Bag5</td><td>7</td><td>31.5</td></tr><tr><td>Rps7</td><td>5</td><td>22.5</td></tr><tr><td>Ywhae</td><td>2</td><td>9</td></tr><tr><td>Eef2</td><td>10</td><td>38.5</td></tr><tr><td>Adsl</td><td>2</td><td>7.5</td></tr><tr><td>Hsp90ab1</td><td>20</td><td>74.5</td></tr><tr><td>Gnb2l1</td><td>6</td><td>22</td></tr><tr><td>Copg</td><td>2</td><td>7</td></tr><tr><td>Rpl23</td><td>2</td><td>7</td></tr><tr><td>Psmc6</td><td>2</td><td>7</td></tr><tr><td>Pcbp2</td><td>2</td><td>7</td></tr><tr><td>Pcbp1</td><td>3</td><td>10</td></tr><tr><td>Pabpc4</td><td>5</td><td>15.5</td></tr><tr><td>Hspa8</td><td>77</td><td>237.5</td></tr><tr><td>Fcgr4</td><td>13</td><td>38.5</td></tr><tr><td>Mcm7</td><td>4</td><td>11.5</td></tr><tr><td>Hadha</td><td>3</td><td>8.5</td></tr><tr><td>Kpnb1</td><td>5</td><td>14</td></tr><tr><td>Atp5a1</td><td>9</td><td>25</td></tr><tr><td>Pontin</td><td>4</td><td>11</td></tr><tr><td>Bat3</td><td>3</td><td>8</td></tr><tr><td>Pdia6</td><td>3</td><td>7.5</td></tr><tr><td>Dnajc7</td><td>38</td><td>86</td></tr><tr><td>Rps15a</td><td>6</td><td>13.5</td></tr><tr><td>Aldh2</td><td>4</td><td>9</td></tr><tr><td>Trim28</td><td>5</td><td>11</td></tr><tr><td>Eef1a1</td><td>16</td><td>35</td></tr><tr><td>St13</td><td>6</td><td>13</td></tr><tr><td>Cct8</td><td>36</td><td>54</td></tr><tr><td>Psmd11</td><td>6</td><td>9</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-08490-v3.xml">elife-08490-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.08490.021</object-id><label>Table 1.</label><caption><p>Accuracy of host prediction based on distance (d) between tetranucleotide frequencies of viral and microbial genomes</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.08490.021">http://dx.doi.org/10.7554/eLife.08490.021</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th rowspan="2">Predicted</th><th colspan="2">Host order</th><th colspan="2">Host family</th><th colspan="2">Host genus</th></tr><tr><th>Correct</th><th>Ratio (%)</th><th>Correct</th><th>Ratio (%)</th><th>Correct</th><th>Ratio (%)</th></tr></thead><tbody><tr><td colspan="8">All reference sequences</td></tr><tr><td>&#8195;d &lt; 4 &#215; 10<sup>&#8722;04</sup></td><td>98</td><td>97</td><td style="background:silver">98.98</td><td>97</td><td style="background:silver">98.98</td><td>97</td><td style="background:silver">98.98</td></tr><tr><td>&#8195;4 &#215; 10<sup>&#8722;04</sup> &#8804; d &lt; 1 &#215; 10<sup>&#8722;03</sup></td><td>10,173</td><td>9361</td><td style="background:silver">92.02</td><td>8971</td><td style="background:silver">88.18</td><td>5261</td><td>51.72</td></tr><tr><td>&#8195;1 &#215; 10<sup>&#8722;03</sup> &#8804; d</td><td>2508</td><td>1872</td><td>74.64</td><td>1757</td><td>70.06</td><td>917</td><td>36.56</td></tr><tr><td colspan="8">Host species excluded</td></tr><tr><td>&#8195;d &lt; 4 &#215; 10<sup>&#8722;04</sup></td><td>21</td><td>20</td><td style="background:silver">95.24</td><td>20</td><td style="background:silver">95.24</td><td>20</td><td style="background:silver">95.24</td></tr><tr><td>&#8195;4 &#215; 10<sup>&#8722;04</sup> &#8804; d &lt; 1 &#215; 10<sup>&#8722;03</sup></td><td>10,003</td><td>9067</td><td style="background:silver">90.64</td><td>8372</td><td style="background:silver">83.69</td><td>2992</td><td>29.91</td></tr><tr><td>&#8195;1 &#215; 10<sup>&#8722;03</sup> &#8804; d</td><td>2755</td><td>1981</td><td>71.91</td><td>1840</td><td>66.79</td><td>818</td><td>29.69</td></tr><tr><td colspan="8">Host genus excluded</td></tr><tr><td>&#8195;d &lt; 4 &#215; 10<sup>&#8722;04</sup></td><td>1</td><td>0</td><td>0.00</td><td>0</td><td>0.00</td><td>0</td><td>0.00</td></tr><tr><td>&#8195;4 &#215; 10<sup>&#8722;04</sup> &#8804; d &lt; 1 &#215; 10<sup>&#8722;03</sup></td><td>9085</td><td>7303</td><td style="background:silver">80.39</td><td>6181</td><td>68.04</td><td>0</td><td>0.00</td></tr><tr><td>&#8195;1 &#215; 10<sup>&#8722;03</sup> &#8804; d</td><td>3693</td><td>1768</td><td>47.87</td><td>1388</td><td>37.58</td><td>0</td><td>0.00</td></tr></tbody></table><table-wrap-foot><fn><p>For each viral genome, the order, family, and genus of its host were predicted from the taxonomy of the closest microbial genome (based on the mean absolute difference between tetranucleotide frequency vectors) and compared to the order, family, and genus of the actual host (i.e., the taxonomy of the genome with which the virus was identified). These predictions were computed with (i) all microbial genomes, (ii) excluding specifically all genomes from the host species, and (iii) excluding all genomes from the host genus. Cases with over 75% of prediction accuracy are highlighted in gray.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-08586-v1.xml">elife-08586-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.08586.011</object-id><label>Table 1.</label><caption><p>Probability distribution fit parameters from <xref ref-type="fig" rid="fig3">Figure 3</xref></p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.08586.011">http://dx.doi.org/10.7554/eLife.08586.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Sample</th><th colspan="4">Query</th><th colspan="3">Spc42-reference</th></tr><tr><th>x</th><th>y</th><th>FWHM<sub>x</sub></th><th>FWHM<sub>y</sub></th><th>x</th><th>FWHM<sub>x</sub></th><th>FWHM<sub>y</sub></th></tr></thead><tbody><tr><td rowspan="2">YFP-Spc42</td><td>&#8722;96.4 (2.2)</td><td>&#8722;15.6 (0.6)</td><td>141.1 (2.7)</td><td rowspan="2">108.1 (1.4)</td><td>&#8722;104.9 (1.3)</td><td>140.2 (3.0)</td><td rowspan="2">104.6 (1.2)</td></tr><tr><td>80.4 (2.8)</td><td>&#8722;15.6 (0.6)</td><td>148.4 (5.0)</td><td>104.9 (1.3)</td><td>136.4 (5.5)</td></tr><tr><td rowspan="2">Kar1-YFP</td><td rowspan="2">&#8722;12.4 (2.8)</td><td rowspan="2">44.2 (0.6)</td><td rowspan="2">178.8 (3.3)</td><td rowspan="2">118.8 (1.1)</td><td>&#8722;106.0 (1.6)</td><td>136.8 (4.6)</td><td rowspan="2">112.9 (1.4)</td></tr><tr><td>106.0 (1.6)</td><td>129.8 (8.0)</td></tr><tr><td rowspan="2">Mps3-YFP</td><td rowspan="2">10.9 (2.3)</td><td rowspan="2">&#8722;58.5 (0.5)</td><td rowspan="2">318.1 (17.8)</td><td rowspan="2">158.1 (1.2)</td><td>&#8722;109.2 (1.3)</td><td>158.0 (3.7)</td><td rowspan="2">114.7 (1.3)</td></tr><tr><td>109.2 (1.3)</td><td>133.7 (6.5)</td></tr><tr><td rowspan="2">GFP-Sfi1</td><td>&#8722;100.3 (7.5)</td><td>12.1 (0.8)</td><td>153.6 (3.8)</td><td rowspan="2">110.3 (2.2)</td><td>&#8722;108.4 (1.5)</td><td>139.5 (3.2)</td><td rowspan="2">131.3 (1.3)</td></tr><tr><td>106.4 (8.3)</td><td>12.1 (0.8)</td><td>194.9 (14.5)</td><td>108.4 (1.5)</td><td>131.4 (6.1)</td></tr><tr><td rowspan="2">Sfi1-GFP</td><td rowspan="2">&#8722;25.9 (3.2)</td><td rowspan="2">29.4 (1.5)</td><td rowspan="2">214.3 (6.6)</td><td rowspan="2">153.9 (5.1)</td><td>&#8722;110.8 (2.0)</td><td>147.0 (3.4)</td><td rowspan="2">123.8 (1.3)</td></tr><tr><td>110.8 (2.0)</td><td>140.1 (8.7)</td></tr><tr><td rowspan="2">YFP-Cdc31</td><td>&#8722;152.4 (15.4)</td><td>50.4 (2.3)</td><td>247.9 (29.4)</td><td rowspan="2">180.7 (7.6)</td><td>&#8722;118.9 (1.4)</td><td>195.7 (3.7)</td><td rowspan="2">158.2 (1.1)</td></tr><tr><td>49.2 (6.9)</td><td>50.4 (2.3)</td><td>143.8 (14.1)</td><td>118.9 (1.4)</td><td>156.7 (4.8)</td></tr></tbody></table><table-wrap-foot><fn><p>Notes: All values are in nm. In all cases &#8216;x&#8217; refers to the mother-satellite axis and &#8216;y&#8217; refers to the pole axis. The zero x and y axis value is defined as midway between the C-terminal Spc42 reference peaks in our realignment scheme. The full-width half-maximum (FWHM) is 2.35 times the standard deviation of the Gaussian fit, and this can be converted into 95% integral values by multiplying each by 1.7. In all cases the mother spindle pole body (SPB) peak is shown on the first line and the satellite peak, if applicable, is shown on the second line. FWHM<sub>y</sub> in cells containing two foci was determined by the averages over the two peaks. Errors are in parentheses and are standard deviations from Monte Carlo random fits as described in &#8216;Materials and methods&#8217;.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-08586-v1.xml">elife-08586-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.08586.016</object-id><label>Table 2.</label><caption><p>Probability distribution fit parameters from <xref ref-type="fig" rid="fig8">Figure 8</xref></p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.08586.016">http://dx.doi.org/10.7554/eLife.08586.016</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Sample</th><th colspan="4">Query</th><th colspan="2">Spc110-reference</th></tr><tr><th>x</th><th>y</th><th>FWHM<sub>x</sub></th><th>FWHM<sub>y</sub></th><th>FWHM<sub>x</sub></th><th>FWHM<sub>y</sub></th></tr></thead><tbody><tr><td rowspan="2">Spc42-mT</td><td>&#8722;60.3 (0.9)</td><td rowspan="2">0.0</td><td>73.0 (1.1)</td><td rowspan="2">111.2 (1.5)</td><td rowspan="2">109.4 (1.2)</td><td rowspan="2">144.6 (1.4)</td></tr><tr><td>60.3 (1.2)</td><td>71.2 (1.8)</td></tr><tr><td rowspan="2">Spc29-mT</td><td>&#8722;60.0 (0.9)</td><td rowspan="2">&#8722;25.8 (1.0)</td><td>70.9 (1.1)</td><td rowspan="2">111.4 (0.9)</td><td rowspan="2">101.5 (1.2)</td><td rowspan="2">157.7 (1.4)</td></tr><tr><td>49.5 (1.3)</td><td>69.7 (1.8)</td></tr><tr><td rowspan="2">Nud1-mT</td><td>&#8722;65.3 (1.2)</td><td rowspan="2">&#8722;11.6 (1.7)</td><td>78.0 (2.1)</td><td rowspan="2">112.3 (3.8)</td><td rowspan="2">101.8 (1.7)</td><td rowspan="2">148.2 (1.0)</td></tr><tr><td>61.6 (1.8)</td><td>73.5 (2.6)</td></tr><tr><td rowspan="2">Cnm67-mT</td><td>&#8722;60.1 (1.1)</td><td rowspan="2">&#8722;19.0 (1.6)</td><td>79.9 (1.9)</td><td rowspan="2">91.0 (2.8)</td><td rowspan="2">97.0 (1.3)</td><td rowspan="2">140.1 (1.6)</td></tr><tr><td>74.4 (2.6)</td><td>61.6 (4.1)</td></tr><tr><td rowspan="2">Ndc1-mT</td><td>&#8722;62.7 (2.3)</td><td rowspan="2">&#8722;34.2 (1.1)</td><td>97.0 (5.6)</td><td rowspan="2">130.9 (1.3)</td><td rowspan="2">106.9 (1.3)</td><td rowspan="2">137.4 (1.4)</td></tr><tr><td>56.0 (3.3)</td><td>56.2 (5.4)</td></tr><tr><td rowspan="2">Nbp1-mT</td><td>&#8722;66.5 (1.2)</td><td rowspan="2">&#8722;48.8 (1.0)</td><td>77.7 (4.3)</td><td rowspan="2">106.8 (1.1)</td><td rowspan="2">102.1 (1.0)</td><td rowspan="2">119.1 (1.0)</td></tr><tr><td>3.0 (1.8)</td><td>50.6 (2.3)</td></tr><tr><td rowspan="2">Mps2-mT</td><td>&#8722;63.1 (1.9)</td><td rowspan="2">&#8722;37.2 (1.4)</td><td>60.0 (6.4)</td><td rowspan="2">96.4 (2.1)</td><td rowspan="2">146.6 (3.3)</td><td rowspan="2">132.8 (2.0)</td></tr><tr><td>42.8 (3.0)</td><td>47.2 (6.2)</td></tr><tr><td rowspan="2">Bbp1-mT</td><td>&#8722;53.8 (1.5)</td><td rowspan="2">&#8722;35.8 (1.5)</td><td>87.5 (3.2)</td><td rowspan="2">108.2 (2.2)</td><td rowspan="2">113.0 (2.3)</td><td rowspan="2">137.6 (2.6)</td></tr><tr><td>62.9 (2.2)</td><td>65.3 (2.8)</td></tr></tbody></table><table-wrap-foot><fn><p>Notes: All values are in nm. In all cases &#8216;x&#8217; refers to the mother-satellite axis and &#8216;y&#8217; refers to the pole axis. The origin is at the center point between the C-terminal Spc42 peaks as in <xref ref-type="fig" rid="fig3">Figure 3</xref>. The Spc110 distributions are all centered at &#8722;44.2 (1.3) in x and &#8722;61.1 (0.8) in y. For Nbp1, Ndc1, and Mps2 a third Gaussian centered on the half-bridge with a broad width was added to allow for proper Gaussian fitting of the visible peaks. As in <xref ref-type="table" rid="tbl1">Table 1</xref>, the FWHM is 2.35 times the standard deviation of the Gaussian fit, and this can be converted into 95% integral values by multiplying each by 1.7. In all cases the mother SPB peak is shown on the first line and the satellite peak, if applicable, is shown on the second line. FWHM<sub>y</sub> in cells containing two foci was determined by the averages over the two peaks. Errors are in parentheses and are standard deviations from Monte Carlo random fits as described in &#8216;Materials and methods&#8217;.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-08648-v1.xml">elife-08648-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.08648.013</object-id><label>Table 1.</label><caption><p>GenTAC human data</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.08648.013">http://dx.doi.org/10.7554/eLife.08648.013</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"/><th colspan="2">Aortic dissection</th><th colspan="2">Aortic surgery</th></tr><tr><th>Marfan</th><th>Other</th><th>Marfan</th><th>Other</th></tr><tr><th>n = 531</th><th>n = 1819</th><th>n = 531</th><th>n = 1819</th></tr></thead><tbody><tr><td>Odds in CCB</td><td>5.1%</td><td>0.57%</td><td>28.1%</td><td>10.70%</td></tr><tr><td>Odds in non-CCB</td><td>0.41%</td><td>0.12%</td><td>5.1%</td><td>4.4%</td></tr><tr><td>Odds ratio</td><td>12.5</td><td>4.7</td><td>5.5</td><td>2.4</td></tr><tr><td>&#8195;p-value</td><td>0.032</td><td>NS</td><td>&lt;0.001</td><td>&lt;0.01</td></tr><tr><td>Odds ratio (BP)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>12.7</td><td>5.6</td><td>5.4</td><td>2.2</td></tr><tr><td>&#8195;p-value</td><td>0.06</td><td>NS</td><td>&lt;0.001</td><td>0.016</td></tr><tr><td>Odds ratio (Aortic Size)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>11.2</td><td>4.1</td><td>5.0</td><td>2.2</td></tr><tr><td>&#8195;p-value</td><td>0.08</td><td>NS</td><td>&lt;0.01</td><td>0.017</td></tr><tr><td>Odds ratio (&#946;-blocker)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>15.9</td><td>3.7</td><td>5.7</td><td>2.0</td></tr><tr><td>&#8195;p-value</td><td>0.045</td><td>NS</td><td>&lt;0.01</td><td>0.026</td></tr></tbody></table><table-wrap-foot><fn><p>Odds of &#8216;aortic dissection&#8217; and &#8216;aortic surgery&#8217; in patients with Marfan syndrome (&#8216;Marfan&#8217;) and other forms of inherited thoracic aortic aneurysm (&#8216;Other&#8217;). Odds (written as %) = number of people who incurred an event (i.e., dissection or surgery) divided by the number who did not. Odds of aortic dissection or surgery were calculated separately for patients who had used CCBs (&#8216;Odds in CCB&#8217;) compared to those who had not (i.e., &#8216;Odds in non-CCB&#8217;). Odds Ratio = Odds of aortic dissection or surgery in patients who had taken CCBs divided by the odds in patients who had not taken CCBs.</p></fn><fn id="tblfn1"><label>*</label><p>The Odds Ratio was then adjusted for blood pressure (&#8216;BP&#8217;), aortic size (&#8216;Aortic Size&#8217;), and &#946;-blocker use (&#8216;&#946;-blocker&#8217;) at enrollment, with corresponding p-values.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-08648-v2.xml">elife-08648-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.08648.013</object-id><label>Table 1.</label><caption><p>GenTAC human data</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.08648.013">http://dx.doi.org/10.7554/eLife.08648.013</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"/><th colspan="2">Aortic dissection</th><th colspan="2">Aortic surgery</th></tr><tr><th>Marfan</th><th>Other</th><th>Marfan</th><th>Other</th></tr><tr><th>n = 531</th><th>n = 1819</th><th>n = 531</th><th>n = 1819</th></tr></thead><tbody><tr><td>Odds in CCB</td><td>5.1%</td><td>0.57%</td><td>28.1%</td><td>10.70%</td></tr><tr><td>Odds in non-CCB</td><td>0.41%</td><td>0.12%</td><td>5.1%</td><td>4.4%</td></tr><tr><td>Odds ratio</td><td>12.5</td><td>4.7</td><td>5.5</td><td>2.4</td></tr><tr><td>&#8195;p-value</td><td>0.032</td><td>NS</td><td>&lt;0.001</td><td>&lt;0.01</td></tr><tr><td>Odds ratio (BP)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>12.7</td><td>5.6</td><td>5.4</td><td>2.2</td></tr><tr><td>&#8195;p-value</td><td>0.06</td><td>NS</td><td>&lt;0.001</td><td>0.016</td></tr><tr><td>Odds ratio (Aortic Size)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>11.2</td><td>4.1</td><td>5.0</td><td>2.2</td></tr><tr><td>&#8195;p-value</td><td>0.08</td><td>NS</td><td>&lt;0.01</td><td>0.017</td></tr><tr><td>Odds ratio (&#946;-blocker)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>15.9</td><td>3.7</td><td>5.7</td><td>2.0</td></tr><tr><td>&#8195;p-value</td><td>0.045</td><td>NS</td><td>&lt;0.01</td><td>0.026</td></tr></tbody></table><table-wrap-foot><fn><p>Odds of &#8216;aortic dissection&#8217; and &#8216;aortic surgery&#8217; in patients with Marfan syndrome (&#8216;Marfan&#8217;) and other forms of inherited thoracic aortic aneurysm (&#8216;Other&#8217;). Odds (written as %) = number of people who incurred an event (i.e., dissection or surgery) divided by the number who did not. Odds of aortic dissection or surgery were calculated separately for patients who had used CCBs (&#8216;Odds in CCB&#8217;) compared to those who had not (i.e., &#8216;Odds in non-CCB&#8217;). Odds Ratio = Odds of aortic dissection or surgery in patients who had taken CCBs divided by the odds in patients who had not taken CCBs.</p></fn><fn id="tblfn1"><label>*</label><p>The Odds Ratio was then adjusted for blood pressure (&#8216;BP&#8217;), aortic size (&#8216;Aortic Size&#8217;), and &#946;-blocker use (&#8216;&#946;-blocker&#8217;) at enrollment, with corresponding p-values.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-08709-v1.xml">elife-08709-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.08709.010</object-id><label>Table 2.</label><caption><p>Intact masses for proteins expressed and purified in this study</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.08709.010">http://dx.doi.org/10.7554/eLife.08709.010</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Protein</th><th rowspan="2">Vector</th><th colspan="3">Molecular Mass (Da)</th></tr><tr><th>Calculated</th><th>Observed</th><th>&#916;</th></tr></thead><tbody><tr><td>Pikp-HMA</td><td>pOPINS3C<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>7805.23</td><td>7804.97</td><td>&#8722;0.26</td></tr><tr><td>AVR-PikD</td><td>pOPINS3C<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>10,835.31</td><td>10,832.95</td><td>&#8722;2.36<xref ref-type="table-fn" rid="tblfn4">&#167;</xref></td></tr><tr><td>AVR-PikD</td><td>pOPINA<xref ref-type="table-fn" rid="tblfn2">&#8224;</xref></td><td>10,812.33</td><td>10,809.99</td><td>&#8722;2.34</td></tr><tr><td>AVR-PikD</td><td>pOPINE<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>11,786.33</td><td>11,784.16</td><td>&#8722;2.17</td></tr><tr><td>AVR-PikE</td><td>pOPINS3C<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>10,812.27</td><td>10,809.91</td><td>&#8722;2.36</td></tr><tr><td>AVR-PikE</td><td>pOPINE<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>11,763.29</td><td>11,760.96</td><td>&#8722;2.33</td></tr><tr><td>AVR-PikA</td><td>pOPINS3C<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>10,844.27</td><td>10,841.80</td><td>&#8722;2.47</td></tr><tr><td>AVR-PikA</td><td>pOPINE<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>11,795.29</td><td>11,793.01</td><td>&#8722;2.28</td></tr><tr><td>AVR-PikC</td><td>pOPINS3C<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>10,856.28</td><td>10,853.72</td><td>&#8722;2.56</td></tr><tr><td>AVR-PikC</td><td>pOPINE<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>11,807.30</td><td>11,804.97</td><td>&#8722;2.33</td></tr><tr><td>AVR-PikD<sup>His46Glu</sup></td><td>pOPINE<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>11,778.30</td><td>11,776.07</td><td>&#8722;2.23</td></tr><tr><td>AVR-PikD<sup>Ile49Glu</sup></td><td>pOPINE<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>11,802.28</td><td>11,800.04</td><td>&#8722;2.24</td></tr><tr><td>AVR-PikD<sup>Arg64Ala</sup></td><td>pOPINE<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>11,701.22</td><td>11,698.94</td><td>&#8722;2.28</td></tr><tr><td>AVR-PikD<sup>Asp66Arg</sup></td><td>pOPINE<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>11,827.43</td><td>11,825.31</td><td>&#8722;2.12</td></tr><tr><td>AVR-PikD<sup>Ala67Asp</sup></td><td>pOPINE<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>11,830.34</td><td>11,828.20</td><td>&#8722;2.14</td></tr><tr><td>AVR-PikD<sup>Pro47Ala, Gly48Asp</sup></td><td>pOPINE<xref ref-type="table-fn" rid="tblfn3">&#8225;</xref></td><td>11,818.32</td><td>11,816.20</td><td>&#8722;2.12</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>Non-native residues remaining after 3C cleavage: N-terminal Gly&#8211;Pro.</p></fn><fn id="tblfn2"><label>&#8224;</label><p>Non-native residues remaining: N-terminal Met.</p></fn><fn id="tblfn3"><label>&#8225;</label><p>Non-native residues remaining after 3C cleavage: N-terminal Gly&#8211;Pro; C-terminal Lys-His-His-His-His-His-His.</p></fn><fn id="tblfn4"><label>&#167;</label><p>The measured mass of each AVR-Pik protein should be 2.0156 Da (2 &#215; 1.0078) less than its calculated mass due to formation of the di-sulphide bond.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-08716-v2.xml">elife-08716-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.08716.019</object-id><label>Table 2.</label><caption><p>DNA constructs used in this study</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.08716.019">http://dx.doi.org/10.7554/eLife.08716.019</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Substrate</th><th rowspan="2">Oligonucleotide components</th><th colspan="4">Junction sequence</th></tr><tr><th>&#8722;2</th><th>&#8722;1</th><th>1</th><th>IC</th></tr></thead><tbody><tr><td>Substrate 1</td><td>1a, 1b, 1c</td><td>A</td><td>G</td><td>A</td><td>T</td></tr><tr><td>Substrate 1-FAM</td><td>1a, 1b-3&#8242;-FAM, 1c</td><td>A</td><td>G</td><td>A</td><td>T</td></tr><tr><td>SacI substrate</td><td>SacI-a, SacI-b, 1c</td><td>A</td><td>G</td><td>C</td><td>T</td></tr><tr><td>Substrate 2</td><td>2a, 2b, 1c</td><td>A</td><td>G</td><td>G</td><td>T</td></tr><tr><td>Substrate 3</td><td>2a, 3b, 1c</td><td>A</td><td>G</td><td>G</td><td>A</td></tr><tr><td>3&#8242;-phos substrate</td><td>2a, 3b, 1c-3&#8242;P</td><td>A</td><td>G</td><td>G</td><td>A</td></tr><tr><td>Substrate 4</td><td>2a, 4b, 1c</td><td>A</td><td>G</td><td>G</td><td>C</td></tr><tr><td>Substrate 5</td><td>2a, 5b, 1c</td><td>A</td><td>G</td><td>G</td><td>G</td></tr><tr><td>Substrate 6</td><td>6a, 6b, 1c</td><td>A</td><td>G</td><td>C</td><td>A</td></tr><tr><td>Substrate 7</td><td>7a, 7b, 1c</td><td>A</td><td>G</td><td>T</td><td>A</td></tr><tr><td>Substrate 8</td><td>1a, 8b, 1c</td><td>A</td><td>G</td><td>A</td><td>A</td></tr><tr><td>Substrate 9</td><td>9a, 3b, 9c</td><td>A</td><td>T</td><td>G</td><td>A</td></tr><tr><td>Substrate 10</td><td>10a, 3b, 10c</td><td>A</td><td>A</td><td>G</td><td>A</td></tr><tr><td>Substrate 11</td><td>11a, 3b, 11c</td><td>A</td><td>C</td><td>G</td><td>A</td></tr><tr><td>Substrate 12</td><td>12a, 3b, 12c</td><td>G</td><td>A</td><td>G</td><td>A</td></tr><tr><td>Substrate 13</td><td>13a, 3b, 13c</td><td>C</td><td>A</td><td>G</td><td>A</td></tr><tr><td>Substrate 14</td><td>14a, 3b, 14c</td><td>T</td><td>A</td><td>G</td><td>A</td></tr><tr><td>Substrate 15</td><td>2a, 4b, 1c</td><td>A</td><td>G</td><td>G</td><td>C</td></tr><tr><td>Substrate 16</td><td>11a, 4b, 11c</td><td>A</td><td>C</td><td>G</td><td>C</td></tr><tr><td>Substrate 17</td><td>10a, 4b, 10c</td><td>A</td><td>A</td><td>G</td><td>C</td></tr><tr><td>Substrate 18</td><td>9a, 4b, 9c</td><td>A</td><td>T</td><td>G</td><td>C</td></tr><tr><td>Substrate 19</td><td>19a, 19b, 19c</td><td>C</td><td>G</td><td>G</td><td>A</td></tr><tr><td>Gap10</td><td>19a, 19b, 19c, 20d-10</td><td>C</td><td>G</td><td>G</td><td>A</td></tr><tr><td>Gap5</td><td>19a, 19b, 19c, 20d-5</td><td>C</td><td>G</td><td>G</td><td>A</td></tr><tr><td>Nick</td><td>19a, 19b, 19c, 20d</td><td>C</td><td>G</td><td>G</td><td>A</td></tr><tr><td>Y-junction</td><td>19a, 19b, 20c-d</td><td>C</td><td>G</td><td>G</td><td>A</td></tr><tr><td>Spacer 3-1 substrate</td><td>Sp3-1a, Sp3-1b, Sp3-1c</td><td>G</td><td>A</td><td>G</td><td>A</td></tr><tr><td>Spacer 3-2 substrate</td><td>Sp3-2a, Sp3-2b, Sp3-2c</td><td>A</td><td>C</td><td>G</td><td>C</td></tr></tbody></table><table-wrap-foot><fn><p>The sequence of the central portion of the junction (positions &#8722;2, &#8722;1, 1 and the incoming nucleotide (IC)) for each substrate is shown. The oligonucleotides used to assemble the complete substrate are indicated.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-08843-v1.xml">elife-08843-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tblu1" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Protein</th><th rowspan="2">Molar ratio of lipid: protein in RPL reactions*</th><th colspan="2">Molar ratio of lipid: protein on vacuoles</th><th rowspan="2">Ratio (RPLs/vacuoles) of molar protein: lipid ratios in std. reactions&#8224;</th></tr><tr><th>BJ3505</th><th>DKY6218</th></tr></thead><tbody><tr><td>Vam7p</td><td>2 &#215; 10<sup>3</sup></td><td>30 &#215; 10<sup>4</sup></td><td>6.5 &#215; 10<sup>4</sup></td><td>7 &#215; 10<sup>1</sup></td></tr><tr><td>Vam3p</td><td>2 &#215; 10<sup>3</sup></td><td>11 &#215; 10<sup>4</sup></td><td>22 &#215; 10<sup>4</sup></td><td>7 &#215; 10<sup>1</sup></td></tr><tr><td>Vti1p</td><td>2 &#215; 10<sup>3</sup></td><td>10 &#215; 10<sup>4</sup></td><td>13 &#215; 10<sup>4</sup></td><td>5 &#215; 10<sup>1</sup></td></tr><tr><td>Nyv1p</td><td>2 &#215; 10<sup>3</sup></td><td>4.3 &#215; 10<sup>4</sup></td><td>8.1 &#215; 10<sup>4</sup></td><td>3 &#215; 10<sup>1</sup></td></tr><tr><td>Ypt7p</td><td>4 &#215; 10<sup>3</sup></td><td>1.9 &#215; 10<sup>4</sup></td><td>1.8 &#215; 10<sup>4</sup></td><td>0.5 &#215; 10<sup>1</sup></td></tr><tr><td>Sec17p</td><td>7 &#215; 10<sup>3</sup></td><td>41 &#215; 10<sup>4</sup></td><td>13 &#215; 10<sup>4</sup></td><td>3 &#215; 10<sup>1</sup></td></tr><tr><td>Sec18p</td><td>1 &#215; 10<sup>3</sup></td><td>10 &#215; 10<sup>4</sup></td><td>13 &#215; 10<sup>4</sup></td><td>10 &#215; 10<sup>1</sup></td></tr><tr><td>Vps33p</td><td>6 &#215; 10<sup>3</sup></td><td>17 &#215; 10<sup>4</sup></td><td>31 &#215; 10<sup>4</sup></td><td>3 &#215; 10<sup>1</sup></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-08843-v1.xml">elife-08843-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tblu2" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Protein</th><th rowspan="2">Molar ratio of lipid: protein in RPL reactions*</th><th colspan="2">Molar ratio of lipid: protein on vacuoles</th><th rowspan="2">Ratio (RPLs/vacuoles) of molar protein: lipid ratios in std. reactions&#8224;</th></tr><tr><th>BJ3505</th><th>DKY6218</th></tr></thead><tbody><tr><td>Vam7p</td><td>2 &#215; 10<sup>3</sup></td><td>30 &#215; 10<sup>5</sup></td><td>6.5 &#215; 10<sup>5</sup></td><td>7 &#215; 10<sup>1</sup></td></tr><tr><td>Vam3p</td><td>2 &#215; 10<sup>3</sup></td><td>11 &#215; 10<sup>5</sup></td><td>22 &#215; 10<sup>5</sup></td><td>7 &#215; 10<sup>1</sup></td></tr><tr><td>Vti1p</td><td>2 &#215; 10<sup>3</sup></td><td>10 &#215; 10<sup>5</sup></td><td>13 &#215; 10<sup>5</sup></td><td>5 &#215; 10<sup>1</sup></td></tr><tr><td>Nyv1p</td><td>2 &#215; 10<sup>3</sup></td><td>4.3 &#215; 10<sup>5</sup></td><td>8.1 &#215; 10<sup>5</sup></td><td>3 &#215; 10<sup>1</sup></td></tr><tr><td>Ypt7p</td><td>4 &#215; 10<sup>3</sup></td><td>1.9 &#215; 10<sup>5</sup></td><td>1.8 &#215; 10<sup>5</sup></td><td>0.5 &#215; 10<sup>1</sup></td></tr><tr><td>Sec17p</td><td>7 &#215; 10<sup>3</sup></td><td>41 &#215; 10<sup>5</sup></td><td>13 &#215; 10<sup>5</sup></td><td>3 &#215; 10<sup>1</sup></td></tr><tr><td>Sec18p</td><td>1 &#215; 10<sup>3</sup></td><td>10 &#215; 10<sup>5</sup></td><td>13 &#215; 10<sup>5</sup></td><td>10 &#215; 10<sup>1</sup></td></tr><tr><td>Vps33p</td><td>6 &#215; 10<sup>3</sup></td><td>17 &#215; 10<sup>5</sup></td><td>31 &#215; 10<sup>5</sup></td><td>3 &#215; 10<sup>1</sup></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-08935-v2.xml">elife-08935-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.08935.004</object-id><label>Table 1.</label><caption><p>Number of cleaned fascicles for the three pathways of interest in patients and control participants</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.08935.004">http://dx.doi.org/10.7554/eLife.08935.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Subject</th><th colspan="2">LGN &#8596; hMT+</th><th colspan="2">Crossing hMT+</th><th colspan="2">SC &#8596; hMT+</th></tr><tr><th>Ipsi-lesional</th><th>Contra-lesional</th><th>Left &#8594; right</th><th>Right &#8594; left</th><th>Ipsi-lesional</th><th>Contra-lesional</th></tr></thead><tbody><tr><td colspan="7">Blindsight positive patients</td></tr><tr><td>PB1</td><td>75</td><td>115</td><td><italic>9</italic></td><td><italic>no</italic></td><td>12</td><td>12</td></tr><tr><td>PB2</td><td>19</td><td>196</td><td><italic>6</italic></td><td><italic>no</italic></td><td><italic>no</italic></td><td>18</td></tr><tr><td>PB3</td><td>50</td><td>67</td><td>24</td><td>24</td><td>17</td><td>17</td></tr><tr><td>PB4</td><td>19</td><td>315</td><td><italic>no</italic></td><td><italic>no</italic></td><td><italic>no</italic></td><td>15</td></tr><tr><td>PB5</td><td>93</td><td>83</td><td>7</td><td>19</td><td>14</td><td>14</td></tr><tr><td>PB6</td><td>12</td><td>64</td><td>13</td><td>15</td><td><italic>no</italic></td><td>17</td></tr><tr><td>PB7</td><td>397</td><td>17</td><td><italic>no</italic></td><td><italic>no</italic></td><td>16</td><td>8</td></tr><tr><td>PB8</td><td>87</td><td>37</td><td>12</td><td>16</td><td>20</td><td>17</td></tr><tr><td>PB9</td><td>635</td><td>53</td><td><italic>no</italic></td><td><italic>no</italic></td><td>16</td><td><italic>no</italic></td></tr><tr><td>PB10</td><td>32</td><td>29</td><td><italic>no</italic></td><td><italic>no</italic></td><td><italic>no</italic></td><td><italic>no</italic></td></tr><tr><td>PB11</td><td>291</td><td>47</td><td>9</td><td>18</td><td>12</td><td><italic>no</italic></td></tr><tr><td>PB12</td><td>194</td><td>17</td><td>17</td><td>13</td><td>15</td><td><italic>no</italic></td></tr><tr><td colspan="7">Blindsight negative patients</td></tr><tr><td>PN1</td><td>157</td><td>19</td><td><italic>no</italic></td><td><italic>no</italic></td><td>15</td><td>8</td></tr><tr><td>PN2</td><td>17</td><td>226</td><td>13</td><td>15</td><td>7</td><td>21</td></tr><tr><td>PN3</td><td>351</td><td>89</td><td><italic>no</italic></td><td><italic>no</italic></td><td>19</td><td>16</td></tr><tr><td>PN4</td><td><italic>no</italic></td><td>101</td><td><italic>no</italic></td><td><italic>no</italic></td><td><italic>no</italic></td><td>19</td></tr><tr><td>PN5</td><td>15</td><td>122</td><td><italic>no</italic></td><td><italic>no</italic></td><td><italic>no</italic></td><td>13</td></tr><tr><td colspan="7">Controls</td></tr><tr><td>C1</td><td>308</td><td>339</td><td>19</td><td>14</td><td>14</td><td>18</td></tr><tr><td>C2</td><td>619</td><td>269</td><td><italic>no</italic></td><td><italic>no</italic></td><td>39</td><td>17</td></tr><tr><td>C3</td><td>57</td><td>59</td><td>8</td><td>16</td><td>18</td><td><italic>no</italic></td></tr><tr><td>C4</td><td>176</td><td>114</td><td>8</td><td>6</td><td>18</td><td>16</td></tr><tr><td>C5</td><td>84</td><td>30</td><td>5</td><td>8</td><td>17</td><td>16</td></tr><tr><td>C6</td><td>57</td><td>19</td><td><italic>no</italic></td><td><italic>no</italic></td><td>15</td><td>16</td></tr><tr><td>C7</td><td>78</td><td>46</td><td><italic>no</italic></td><td><italic>no</italic></td><td>9</td><td><italic>no</italic></td></tr><tr><td>C8</td><td>498</td><td>182</td><td>19</td><td>14</td><td>35</td><td>14</td></tr><tr><td>C9</td><td>653</td><td>62</td><td>57</td><td>52</td><td>31</td><td>9</td></tr></tbody></table><table-wrap-foot><fn><p>(1) Ipsilateral pathway between LGN and hMT+ (2) Pathway between hMT+ bilaterally via the corpus callosum (3) Ipsilateral pathway between SC and hMT+. Results are shown separately for the intact and damaged &#8216;ipsi-lesion&#8217; hemispheres (right and left for control participants). &#8216;no&#8217; = zero fascicles survived the cleaning process.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-09083-v2.xml">elife-09083-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl7" position="float"><object-id pub-id-type="doi">10.7554/eLife.09083.029</object-id><label>Table 7.</label><caption><p>Top 10 highly recurring CDR3 sequences (peptide and V(D)J recombination) detected in listed splenic B-1a samples from age-matched SPF and GF mice.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.09083.029">http://dx.doi.org/10.7554/eLife.09083.029</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">sB-1a samples (4 months)</th><th colspan="3">Top 10 IgH CDR3 sequences</th></tr><tr><th/><th>Peptide</th><th>V(D)J</th></tr></thead><tbody><tr><td rowspan="10">germ-free #1</td><td>1</td><td>MRYGSSYWYFDV</td><td>V11-2 D1-1 J1</td></tr><tr><td>2</td><td>ARGAY</td><td>V1-80 J2</td></tr><tr><td>3</td><td>ARNPDGYYTYYYAMDY</td><td>V2-2 D2-9 J4</td></tr><tr><td>4</td><td>ARDPFYYYGSSYWYFDV</td><td>V5-16 D1-1J1</td></tr><tr><td>5</td><td>MRYSNYWYFDV</td><td>V11-2 D2-6 J1</td></tr><tr><td>6</td><td>AITRAY</td><td>V1-55 J3</td></tr><tr><td>7</td><td>ARRYYGSSYWYFDV</td><td>V1-55 D1-1 J1</td></tr><tr><td>8</td><td>ARSDYYGSSSLSY</td><td>V1-26 D1-1 J2</td></tr><tr><td>9</td><td>ASGGNYFDY</td><td>V1-75 J2</td></tr><tr><td>10</td><td>ARSLYN</td><td>V1-9 J2</td></tr><tr><td rowspan="10">germ-free #2</td><td>1</td><td>ARNYGSSYDY</td><td>V1-53 D1-1 J2</td></tr><tr><td>2</td><td>TRPSYYGSDY</td><td>V14-4 D1-1 J2</td></tr><tr><td>3</td><td>TRESYDGYYVWYAMDY</td><td>V5-9-1 D2-9 J4</td></tr><tr><td>4</td><td>ARGDY</td><td>V14-3 J2</td></tr><tr><td>5</td><td>ASNWAY</td><td>V1-53 D4-1 J2</td></tr><tr><td>6</td><td>MRYSNYWYFDV</td><td>V11-2 D2-6 J1</td></tr><tr><td>7</td><td>AKGDYYGSSYYFDY</td><td>V1-9 D1-1 J2</td></tr><tr><td>8</td><td>VRHGPRAFDY</td><td>V10-1 D3-2 J2</td></tr><tr><td>9</td><td>ARLNGDY</td><td>V1-69 J2</td></tr><tr><td>10</td><td>MRYGNYWYFDV</td><td>V11-2 D2-8 J1</td></tr><tr><td rowspan="10">specific pathogen free #1 (from Caltech)</td><td>1</td><td>ASYSNSDV</td><td>V3-6 D2-6 J1</td></tr><tr><td>2</td><td>ARVSYSRAMDY</td><td>V14-3 D2-6 J4</td></tr><tr><td>3</td><td>ARSGNYGAMDY</td><td>V1-7 D2-8 J4</td></tr><tr><td>4</td><td>ASRLRSTFAY</td><td>V2-6-8 D1-1 J3</td></tr><tr><td>5</td><td>ARVTTVHAMDY</td><td>V1-55 D1-1 J4</td></tr><tr><td>6</td><td>ARNYGSSYWYFDV</td><td>V1-53 D1-1 J1</td></tr><tr><td>7</td><td>ARTPNWEARDY</td><td>V1-55 D4-1 J4</td></tr><tr><td>8</td><td>ARRYYGSSYWYFDV</td><td>V1-55 D1-1 J1</td></tr><tr><td>9</td><td>ARPLLYRYYFDY</td><td>V1-75 D2-6 J2</td></tr><tr><td>10</td><td>ARNYGSSYDWYFDV</td><td>V1-9 D1-1 J1</td></tr><tr><td rowspan="10">specific pathogen free #2 (from Caltech)</td><td>1</td><td>ARGGIYYDYDEVYYYAMDY</td><td>V1-55 D2-4 J4</td></tr><tr><td>2</td><td>MRYSNYWYFDV</td><td>V11-2 D2-6 J1</td></tr><tr><td>3</td><td>ARDYYGSSWYFDV</td><td>V1-26 D1-1 J1</td></tr><tr><td>4</td><td>MRYGNYWYFDV</td><td>V11-2 D2-8 J1</td></tr><tr><td>5</td><td>MRYGSSYWYFDV</td><td>V11-2 D1-1 J1</td></tr><tr><td>6</td><td>ARYYDGYYGYYAMDY</td><td>V1-26 D2-4 J4</td></tr><tr><td>7</td><td>ALITTWYFDV</td><td>V1-78 D1-2 J1</td></tr><tr><td>8</td><td>ARHYYGSSWGY</td><td>V1-53 D1-1 J2</td></tr><tr><td>9</td><td>ARSFSPYYFDY</td><td>V1-26 J2</td></tr><tr><td>10</td><td>ARSHGYYPFDY</td><td>V1-54 D2-9 J2</td></tr><tr><td rowspan="10">specific pathogen free #1 (from Stanford)</td><td>1</td><td>ARSADYGGYFDV</td><td>V1-64 D2-4 J1</td></tr><tr><td>2</td><td>ARGAY</td><td>V1-80 J2</td></tr><tr><td>3</td><td>ARSYYDYPWFAY</td><td>V1-76 D2-4 J3</td></tr><tr><td>4</td><td>ARRWLLNAMDY</td><td>V1-9 D2-9 J4</td></tr><tr><td>5</td><td>ARPYYYGSSPWFAY</td><td>V1-69 D1-1 J3</td></tr><tr><td>6</td><td>ARNDYPYWYFDV</td><td>V1-4 D2-4 J1</td></tr><tr><td>7</td><td>ARSGDY</td><td>V1-64 J2</td></tr><tr><td>8</td><td>ARVIGDY</td><td>V1-53 D2-14 J4</td></tr><tr><td>9</td><td>ARANY</td><td>V1-55 J3</td></tr><tr><td>10</td><td>AVNWDYAMDY</td><td>V1-84 D4-1 J4</td></tr><tr><td rowspan="10">specific pathogen free #2 (from Stanford)</td><td>1</td><td>ARGNY</td><td>V1-80 J2</td></tr><tr><td>2</td><td>ARWVYYGSSSYWYFDV</td><td>V1-54 D1-1 J1</td></tr><tr><td>3</td><td>ARSSNYAMDY</td><td>V1-78 D2-11 J4</td></tr><tr><td>4</td><td>ARYYYGSNYAMDY</td><td>V7-3 D1-1 J4</td></tr><tr><td>5</td><td>ARGAY</td><td>V1-80 J2</td></tr><tr><td>6</td><td>ARRYYGSSYWYFDV</td><td>V1-55 D1-1 J1</td></tr><tr><td>7</td><td>ARSPYYSNYEGYFDV</td><td>V1-72 D2-6 J1</td></tr><tr><td>8</td><td>ARKNYGSSYWYFDV</td><td>V1-55 D1-1 J1</td></tr><tr><td>9</td><td>ARLEIYYGNYGRVFDV</td><td>V1-80 D2-8 J2</td></tr><tr><td>10</td><td>ARRDYYGSSYVLAY</td><td>V1-9 D1-1 J3</td></tr></tbody></table><table-wrap-foot><fn id="tblfn6"><p>Table lists the top 10 highly recurring CDR3 sequences (peptide and V(D)J recombination) shown in each of CDR3 tree-map plot (<xref ref-type="fig" rid="fig9">Figure 9A</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-09083-v3.xml">elife-09083-v3.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl7" position="float"><object-id pub-id-type="doi">10.7554/eLife.09083.029</object-id><label>Table 7.</label><caption><p>Top 10 highly recurring CDR3 sequences (peptide and V(D)J recombination) detected in listed splenic B-1a samples from age-matched SPF and GF mice.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.09083.029">http://dx.doi.org/10.7554/eLife.09083.029</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">sB-1a samples (4 months)</th><th colspan="3">Top 10 IgH CDR3 sequences</th></tr><tr><th valign="top"/><th valign="top">Peptide</th><th valign="top">V(D)J</th></tr></thead><tbody><tr><td rowspan="10" valign="top">germ-free #1</td><td valign="top">1</td><td valign="top">MRYGSSYWYFDV</td><td valign="top">V11-2 D1-1 J1</td></tr><tr><td valign="top">2</td><td valign="top">ARGAY</td><td valign="top">V1-80 J2</td></tr><tr><td valign="top">3</td><td valign="top">ARNPDGYYTYYYAMDY</td><td valign="top">V2-2 D2-9 J4</td></tr><tr><td valign="top">4</td><td valign="top">ARDPFYYYGSSYWYFDV</td><td valign="top">V5-16 D1-1J1</td></tr><tr><td valign="top">5</td><td valign="top">MRYSNYWYFDV</td><td valign="top">V11-2 D2-6 J1</td></tr><tr><td valign="top">6</td><td valign="top">AITRAY</td><td valign="top">V1-55 J3</td></tr><tr><td valign="top">7</td><td valign="top">ARRYYGSSYWYFDV</td><td valign="top">V1-55 D1-1 J1</td></tr><tr><td valign="top">8</td><td valign="top">ARSDYYGSSSLSY</td><td valign="top">V1-26 D1-1 J2</td></tr><tr><td valign="top">9</td><td valign="top">ASGGNYFDY</td><td valign="top">V1-75 J2</td></tr><tr><td valign="top">10</td><td valign="top">ARSLYN</td><td valign="top">V1-9 J2</td></tr><tr><td rowspan="10" valign="top">germ-free #2</td><td valign="top">1</td><td valign="top">ARNYGSSYDY</td><td valign="top">V1-53 D1-1 J2</td></tr><tr><td valign="top">2</td><td valign="top">TRPSYYGSDY</td><td valign="top">V14-4 D1-1 J2</td></tr><tr><td valign="top">3</td><td valign="top">TRESYDGYYVWYAMDY</td><td valign="top">V5-9-1 D2-9 J4</td></tr><tr><td valign="top">4</td><td valign="top">ARGDY</td><td valign="top">V14-3 J2</td></tr><tr><td valign="top">5</td><td valign="top">ASNWAY</td><td valign="top">V1-53 D4-1 J2</td></tr><tr><td valign="top">6</td><td valign="top">MRYSNYWYFDV</td><td valign="top">V11-2 D2-6 J1</td></tr><tr><td valign="top">7</td><td valign="top">AKGDYYGSSYYFDY</td><td valign="top">V1-9 D1-1 J2</td></tr><tr><td valign="top">8</td><td valign="top">VRHGPRAFDY</td><td valign="top">V10-1 D3-2 J2</td></tr><tr><td valign="top">9</td><td valign="top">ARLNGDY</td><td valign="top">V1-69 J2</td></tr><tr><td valign="top">10</td><td valign="top">MRYGNYWYFDV</td><td valign="top">V11-2 D2-8 J1</td></tr><tr><td rowspan="10" valign="top">specific pathogen free #1 (from Caltech)</td><td valign="top">1</td><td valign="top">ASYSNSDV</td><td valign="top">V3-6 D2-6 J1</td></tr><tr><td valign="top">2</td><td valign="top">ARVSYSRAMDY</td><td valign="top">V14-3 D2-6 J4</td></tr><tr><td valign="top">3</td><td valign="top">ARSGNYGAMDY</td><td valign="top">V1-7 D2-8 J4</td></tr><tr><td valign="top">4</td><td valign="top">ASRLRSTFAY</td><td valign="top">V2-6-8 D1-1 J3</td></tr><tr><td valign="top">5</td><td valign="top">ARVTTVHAMDY</td><td valign="top">V1-55 D1-1 J4</td></tr><tr><td valign="top">6</td><td valign="top">ARNYGSSYWYFDV</td><td valign="top">V1-53 D1-1 J1</td></tr><tr><td valign="top">7</td><td valign="top">ARTPNWEARDY</td><td valign="top">V1-55 D4-1 J4</td></tr><tr><td valign="top">8</td><td valign="top">ARRYYGSSYWYFDV</td><td valign="top">V1-55 D1-1 J1</td></tr><tr><td valign="top">9</td><td valign="top">ARPLLYRYYFDY</td><td valign="top">V1-75 D2-6 J2</td></tr><tr><td valign="top">10</td><td valign="top">ARNYGSSYDWYFDV</td><td valign="top">V1-9 D1-1 J1</td></tr><tr><td rowspan="10" valign="top">specific pathogen free #2 (from Caltech)</td><td valign="top">1</td><td valign="top">ARGGIYYDYDEVYYYAMDY</td><td valign="top">V1-55 D2-4 J4</td></tr><tr><td valign="top">2</td><td valign="top">MRYSNYWYFDV</td><td valign="top">V11-2 D2-6 J1</td></tr><tr><td valign="top">3</td><td valign="top">ARDYYGSSWYFDV</td><td valign="top">V1-26 D1-1 J1</td></tr><tr><td valign="top">4</td><td valign="top">MRYGNYWYFDV</td><td valign="top">V11-2 D2-8 J1</td></tr><tr><td valign="top">5</td><td valign="top">MRYGSSYWYFDV</td><td valign="top">V11-2 D1-1 J1</td></tr><tr><td valign="top">6</td><td valign="top">ARYYDGYYGYYAMDY</td><td valign="top">V1-26 D2-4 J4</td></tr><tr><td valign="top">7</td><td valign="top">ALITTWYFDV</td><td valign="top">V1-78 D1-2 J1</td></tr><tr><td valign="top">8</td><td valign="top">ARHYYGSSWGY</td><td valign="top">V1-53 D1-1 J2</td></tr><tr><td valign="top">9</td><td valign="top">ARSFSPYYFDY</td><td valign="top">V1-26 J2</td></tr><tr><td valign="top">10</td><td valign="top">ARSHGYYPFDY</td><td valign="top">V1-54 D2-9 J2</td></tr><tr><td rowspan="10" valign="top">specific pathogen free #1 (from Stanford)</td><td valign="top">1</td><td valign="top">ARSADYGGYFDV</td><td valign="top">V1-64 D2-4 J1</td></tr><tr><td valign="top">2</td><td valign="top">ARGAY</td><td valign="top">V1-80 J2</td></tr><tr><td valign="top">3</td><td valign="top">ARSYYDYPWFAY</td><td valign="top">V1-76 D2-4 J3</td></tr><tr><td valign="top">4</td><td valign="top">ARRWLLNAMDY</td><td valign="top">V1-9 D2-9 J4</td></tr><tr><td valign="top">5</td><td valign="top">ARPYYYGSSPWFAY</td><td valign="top">V1-69 D1-1 J3</td></tr><tr><td valign="top">6</td><td valign="top">ARNDYPYWYFDV</td><td valign="top">V1-4 D2-4 J1</td></tr><tr><td valign="top">7</td><td valign="top">ARSGDY</td><td valign="top">V1-64 J2</td></tr><tr><td valign="top">8</td><td valign="top">ARVIGDY</td><td valign="top">V1-53 D2-14 J4</td></tr><tr><td valign="top">9</td><td valign="top">ARANY</td><td valign="top">V1-55 J3</td></tr><tr><td valign="top">10</td><td valign="top">AVNWDYAMDY</td><td valign="top">V1-84 D4-1 J4</td></tr><tr><td rowspan="10" valign="top">specific pathogen free #2 (from Stanford)</td><td valign="top">1</td><td valign="top">ARGNY</td><td valign="top">V1-80 J2</td></tr><tr><td valign="top">2</td><td valign="top">ARWVYYGSSSYWYFDV</td><td valign="top">V1-54 D1-1 J1</td></tr><tr><td valign="top">3</td><td valign="top">ARSSNYAMDY</td><td valign="top">V1-78 D2-11 J4</td></tr><tr><td valign="top">4</td><td valign="top">ARYYYGSNYAMDY</td><td valign="top">V7-3 D1-1 J4</td></tr><tr><td valign="top">5</td><td valign="top">ARGAY</td><td valign="top">V1-80 J2</td></tr><tr><td valign="top">6</td><td valign="top">ARRYYGSSYWYFDV</td><td valign="top">V1-55 D1-1 J1</td></tr><tr><td valign="top">7</td><td valign="top">ARSPYYSNYEGYFDV</td><td valign="top">V1-72 D2-6 J1</td></tr><tr><td valign="top">8</td><td valign="top">ARKNYGSSYWYFDV</td><td valign="top">V1-55 D1-1 J1</td></tr><tr><td valign="top">9</td><td valign="top">ARLEIYYGNYGRVFDV</td><td valign="top">V1-80 D2-8 J2</td></tr><tr><td valign="top">10</td><td valign="top">ARRDYYGSSYVLAY</td><td valign="top">V1-9 D1-1 J3</td></tr></tbody></table><table-wrap-foot><fn id="tblfn6"><p>Table lists the top 10 highly recurring CDR3 sequences (peptide and V(D)J recombination) shown in each of CDR3 tree-map plot (<xref ref-type="fig" rid="fig9">Figure 9A</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-09186-v2.xml">elife-09186-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.09186.004</object-id><label>Table 1.</label><caption><p>Overview of modeling publications on the 2013-present EVD epidemic.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.09186.004">http://dx.doi.org/10.7554/eLife.09186.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Ref.</th><th rowspan="2">Date of latest EVD data</th><th rowspan="2">Date published</th><th rowspan="2">EVD data was pre-existing and public</th><th colspan="6">Uncertainties addressed</th></tr><tr><th>R</th><th>Interventions</th><th>Forecast</th><th>Spread</th><th>Phylogenetics</th><th>Clinical trials</th></tr></thead><tbody><tr><td><xref ref-type="bibr" rid="bib6">Baize et al., 2014</xref></td> <td>3/20/14</td><td>4/16/14</td><td>No</td><td/><td/><td/><td/><td>*</td><td/></tr><tr><td><xref ref-type="bibr" rid="bib23">Dudas and Rambaut, 2014</xref></td> <td>3/20/14</td><td>5/2/14</td><td>Yes</td><td/><td/><td/><td/><td>*</td><td/></tr><tr><td><xref ref-type="bibr" rid="bib2">Alizon et al., 2014</xref></td> <td>6/18/14</td><td>12/13/14</td><td>Yes</td><td>*</td><td/><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib29">Gire et al., 2014</xref></td> <td>6/18/14</td><td>8/28/14</td><td>No</td><td/><td/><td/><td>*</td><td>*</td><td/></tr><tr><td><xref ref-type="bibr" rid="bib61">Stadler et al., 2014</xref></td> <td>6/18/14</td><td>10/6/14</td><td>Yes</td><td>*</td><td/><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib67">Volz and Pond, 2014</xref></td> <td>6/18/14</td><td>10/24/14</td><td>Yes</td><td>*</td><td/><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib48">Pandey et al., 2014</xref></td> <td>8/7/14</td><td>10/30/14</td><td>Yes</td><td>*</td><td>*</td><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib30">Gomes et al., 2014</xref></td> <td>8/9/14</td><td>9/2/14</td><td>Yes</td><td>*</td><td/><td>*</td><td>*</td><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib66">Valdez et al., 2015</xref></td> <td>8/15/14</td><td>7/20/15</td><td>Yes</td><td>*</td><td>*</td><td>*</td><td>*</td><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib42">Merler et al., 2015</xref></td> <td>8/16/14</td><td>1/7/15</td><td>Yes</td><td>*</td><td>*</td><td>*</td><td>*</td><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib52">Rainisch et al., 2015</xref></td> <td>8/16/14</td><td>2/18/15</td><td>Yes</td><td/><td/><td/><td>*</td><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib4">Althaus, 2014</xref></td> <td>8/20/14</td><td>9/2/14</td><td>Yes</td><td>*</td><td/><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib27">Fisman et al., 2014</xref></td> <td>8/22/14</td><td>9/8/14</td><td>Yes</td><td>*</td><td/><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib16">Nishiura and Chowell, 2014</xref></td> <td>8/26/14</td><td>9/11/14</td><td>Yes</td><td>*</td><td/><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib50">Poletto et al., 2014</xref></td> <td>8/27/14</td><td>10/23/14</td><td>Yes</td><td/><td>*</td><td/><td>*</td><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib41">Meltzer et al., 2014</xref></td> <td>8/28/14</td><td>9/26/14</td><td>Yes</td><td>*</td><td>*</td><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib1">Agusto et al., 2015</xref></td> <td>8/29/14</td><td>4/23/15</td><td>Yes</td><td>*</td><td>*</td><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib5">Althaus, 2015</xref></td> <td>8/31/14</td><td>4/19/15</td><td>Yes</td><td>*</td><td/><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib57">Scarpino et al., 2014</xref></td> <td>8/31/14</td><td>12/15/14</td><td>Yes</td><td>*</td><td/><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib70">Weitz and Dushoff, 2015</xref></td> <td>8/31/14</td><td>3/4/15</td><td>Yes</td><td>*</td><td>*</td><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib22">Drake et al., 2015</xref></td> <td>9/2/14</td><td>10/30/14</td><td>Yes</td><td>*</td><td>*</td><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib64">Towers et al., 2014</xref></td> <td>9/8/14</td><td>9/18/14</td><td>Yes</td><td>*</td><td/><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib9">Bellan et al., 2014</xref></td> <td>9/14/14</td><td>10/14/14</td><td>Yes</td><td/><td/><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib15">Chowell et al., 2015</xref></td> <td>9/14/14</td><td>1/19/15</td><td>Yes</td><td/><td>*</td><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib19">Cooper et al., 2015</xref></td> <td>9/14/14</td><td>4/14/15</td><td>Yes</td><td/><td/><td/><td/><td/><td>*</td></tr><tr><td><xref ref-type="bibr" rid="bib53">Read et al., 2015</xref></td> <td>9/14/14</td><td>11/12/14</td><td>Yes</td><td/><td>*</td><td/><td>*</td><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib72">WHO Ebola Response Team, 2014</xref></td> <td>9/14/14</td><td>9/23/14</td><td>No</td><td>*</td><td/><td>*</td><td>*</td><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib26">Faye et al., 2015</xref></td> <td>9/16/14</td><td>1/23/15</td><td>No</td><td>*</td><td>*</td><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib10">Bogoch et al., 2015</xref></td> <td>9/21/14</td><td>10/21/14</td><td>Yes</td><td/><td>*</td><td/><td>*</td><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib77">Yamin et al., 2014</xref></td> <td>9/22/14</td><td>10/28/14</td><td>No</td><td>*</td><td>*</td><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib39">Lewnard et al., 2014</xref></td> <td>9/23/14</td><td>10/24/14</td><td>Yes</td><td>*</td><td>*</td><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib69">Webb et al., 2015</xref></td> <td>9/23/14</td><td>1/30/15</td><td>Yes</td><td>*</td><td>*</td><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib58">Shaman et al., 2014</xref></td> <td>9/28/14</td><td>10/27/14</td><td>Yes</td><td>*</td><td/><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib16">Chowell et al., 2014</xref></td> <td>10/1/14</td><td>11/20/14</td><td>Yes</td><td>*</td><td/><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib24">Fasina et al., 2014</xref></td> <td>10/1/14</td><td>10/9/14</td><td>Yes</td><td/><td>*</td><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib34">Khan et al., 2015</xref></td> <td>10/1/14</td><td>2/24/15</td><td>Yes</td><td>*</td><td/><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib55">Rivers et al., 2014</xref></td> <td>10/5/14</td><td>10/16/14</td><td>Yes</td><td>*</td><td>*</td><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib76">Xia et al., 2015</xref></td> <td>10/7/14</td><td>9/8/15</td><td>Yes</td><td>*</td><td>*</td><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib40">Majumder et al., 2014</xref></td> <td>10/11/14</td><td>4/28/15</td><td>Yes</td><td>*</td><td>*</td><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib36">Kiskowski, 2014</xref></td> <td>10/15/14</td><td>11/13/14</td><td>Yes</td><td>*</td><td/><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib27">Fisman and Tuite, 2014</xref></td> <td>10/18/14</td><td>11/21/14</td><td>Yes</td><td>*</td><td>*</td><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib5">Althaus et al., 2015</xref></td> <td>10/20/14</td><td>1/15/15</td><td>Yes</td><td>*</td><td>*</td><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib60">Simon-Loriere et al., 2015</xref></td> <td>10/25/14</td><td>6/24/15</td><td>No</td><td/><td/><td/><td/><td>*</td><td/></tr><tr><td><xref ref-type="bibr" rid="bib52">Rainisch et al., 2015</xref></td> <td>10/31/14</td><td>6/16/15</td><td>Yes</td><td/><td/><td/><td>*</td><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib25">Fast et al., 2015</xref></td> <td>11/1/14</td><td>5/15/15</td><td>Yes</td><td/><td>*</td><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib37">Kucharski et al., 2015</xref></td> <td>11/1/14</td><td>2/18/15</td><td>Yes</td><td>*</td><td>*</td><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib62">Tong et al., 2015</xref></td> <td>11/11/14</td><td>5/13/15</td><td>No</td><td/><td/><td/><td>*</td><td>*</td><td/></tr><tr><td><xref ref-type="bibr" rid="bib32">Hoenen et al., 2015</xref></td> <td>11/21/14</td><td>3/26/15</td><td>No</td><td/><td/><td/><td/><td>*</td><td/></tr><tr><td><xref ref-type="bibr" rid="bib20">Cope et al., 2014</xref></td> <td>12/3/14</td><td>12/10/14</td><td>Yes</td><td/><td>*</td><td/><td>*</td><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib71">White et al., 2014</xref></td> <td>12/3/14</td><td>1/30/15</td><td>Yes</td><td>*</td><td>*</td><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib73">WHO Ebola Response Team, 2015</xref></td> <td>12/14/14</td><td>12/24/14</td><td>No</td><td>*</td><td/><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib16">Chowell et al., 2014</xref></td> <td>12/17/14</td><td>1/21/15</td><td>Yes</td><td>*</td><td/><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib59">Siettos et al., 2014</xref></td> <td>12/21/14</td><td>3/9/15</td><td>Yes</td><td>*</td><td/><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib49">Park et al., 2015</xref></td> <td>12/26/14</td><td>6/18/15</td><td>No</td><td/><td/><td/><td/><td>*</td><td/></tr><tr><td><xref ref-type="bibr" rid="bib44">Nadhem and Nejib, 2015</xref></td> <td>12/30/14</td><td>6/14/15</td><td>Yes</td><td/><td/><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib12">Camacho et al., 2015</xref></td> <td>1/18/15</td><td>2/10/15</td><td>Yes</td><td>*</td><td/><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib13">Carroll et al., 2015</xref></td> <td>1/31/15</td><td>6/17/15</td><td>No</td><td/><td/><td/><td>*</td><td>*</td><td/></tr><tr><td><xref ref-type="bibr" rid="bib8">Bellan et al., 2015</xref></td> <td>2/9/15</td><td>4/15/15</td><td>Yes</td><td/><td/><td>*</td><td/><td/><td>*</td></tr><tr><td><xref ref-type="bibr" rid="bib7">Barbarossa et al., 2015</xref></td> <td>2/13/15</td><td>7/21/15</td><td>Yes</td><td>*</td><td>*</td><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib38">Kugelman et al., 2015</xref></td> <td>2/14/15</td><td>6/12/15</td><td>No</td><td/><td/><td/><td/><td>*</td><td/></tr><tr><td><xref ref-type="bibr" rid="bib18">Cleaton et al., 2015</xref></td> <td>2/28/15</td><td>9/3/15</td><td>Yes</td><td>*</td><td/><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib68">Wang and Zhong, 2015</xref></td> <td>3/18/15</td><td>3/24/15</td><td>Yes</td><td>*</td><td/><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib63">Toth et al., 2015</xref></td> <td>3/31/15</td><td>7/14/15</td><td>Yes</td><td/><td>*</td><td/><td>*</td><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib21">Dong et al., 2015</xref></td> <td>4/3/15</td><td>9/5/15</td><td>Yes</td><td/><td>*</td><td>*</td><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib11">Browne et al., 2015</xref></td> <td>4/12/15</td><td>5/14/15</td><td>Yes</td><td>*</td><td>*</td><td/><td/><td/><td/></tr><tr><td><xref ref-type="bibr" rid="bib79">Zinszer et al., 2015</xref></td> <td>5/13/15</td><td>9/1/15</td><td>Yes</td><td/><td/><td/><td>*</td><td/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-09215-v1.xml">elife-09215-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.09215.016</object-id><label>Table 1.</label><caption><p>Electrophysiology results from sensory cortex single-unit recordings. See also <xref ref-type="supplementary-material" rid="SD3-data">Table 1&#8212;source data 1</xref>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.09215.016">http://dx.doi.org/10.7554/eLife.09215.016</ext-link></p><p><supplementary-material id="SD3-data"><object-id pub-id-type="doi">10.7554/eLife.09215.017</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Firing rates before, during, and after repeated 20 s stimulation periods for each of the 11 neurons recorded in somatosensory cortex.</title><p>Exact p values comparing the somatosensory cortex firing rate before and during stimulation are provided. The 20 s stimulation period was divided into four consecutive 5 s blocks to evaluate the change in firing rate over time.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.09215.017">http://dx.doi.org/10.7554/eLife.09215.017</ext-link></p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-09215-table1-data1-v1.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Stimulation frequency</th><th rowspan="2">Effect on sensory cortex firing rate</th><th colspan="4">Percentage of neurons with significant change in firing rate (n = 11)</th></tr><tr><th>0&#8211;5 s after stim. onset</th><th>5&#8211;10 s after stim. onset</th><th>10&#8211;15 s after stim. onset</th><th>15&#8211;20 s after stim. onset</th></tr></thead><tbody><tr><td rowspan="2">10 Hz</td><td>Increase</td><td>0%</td><td>0%</td><td>0%</td><td>0%</td></tr><tr><td>Decrease</td><td>0%</td><td>91%</td><td>82%</td><td>9%</td></tr><tr><td rowspan="2">40 Hz</td><td>Increase</td><td>100%</td><td>91%</td><td>36%</td><td>55%</td></tr><tr><td>Decrease</td><td>0%</td><td>0%</td><td>0%</td><td>0%</td></tr><tr><td rowspan="2">100 Hz</td><td>Increase</td><td>100%</td><td>82%</td><td>82%</td><td>82%</td></tr><tr><td>Decrease</td><td>0%</td><td>0%</td><td>0%</td><td>0%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-09248-v2.xml">elife-09248-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.09248.006</object-id><label>Table 1.</label><caption><p>Transmembrane protein benchmark</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.09248.006">http://dx.doi.org/10.7554/eLife.09248.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">PDB</th><th rowspan="2">Name</th><th rowspan="2">Seq/len</th><th colspan="2">Full protein</th><th colspan="2">Converged</th><th colspan="2">Aligned</th></tr><tr><th>rmsd</th><th>Length</th><th>rmsd</th><th>Length</th><th>rmsd</th><th>Length</th></tr></thead><tbody><tr><td>4HE8_H (3.3)</td><td>NADH-quinone oxidoreductase subunit 8</td><td>17.3</td><td>4.9</td><td>269</td><td>2.1</td><td>183</td><td>2.2</td><td>234</td></tr><tr><td>1SOR_A (N/A)</td><td>Aquaporin-0</td><td>26.2</td><td>2.7</td><td>221</td><td>2.1</td><td>188</td><td>2.0</td><td>200</td></tr><tr><td>4Q2E_A (3.4)</td><td>Phosphatidate cytidylyltransferase</td><td>18.6</td><td>5.4</td><td>262</td><td>3.5</td><td>176</td><td>2.8</td><td>178</td></tr><tr><td>4HTT_A (6.8)</td><td>Sec-independent protein translocase protein</td><td>14.6</td><td>3.9</td><td>225</td><td>1.8</td><td>124</td><td>2.4</td><td>181</td></tr><tr><td>4P6V_E (3.5)</td><td>Na(+)-translocating NADH-quinone reductase subunit D</td><td>14.3</td><td>5.0</td><td>194</td><td>1.4</td><td>49</td><td>2.8</td><td>155</td></tr><tr><td>4J72_A (3.3)</td><td>Phospho-N-acetylmuramoyl-pentapeptide-transferase</td><td>19.9</td><td>6.6</td><td>323</td><td>3.1</td><td>251</td><td>2.4</td><td>237</td></tr><tr><td>3V5U_A (1.9)</td><td>Sodium/Calcium exchanger</td><td>10.2</td><td>3.9</td><td>297</td><td>3.7</td><td>284</td><td>2.3</td><td>245</td></tr><tr><td>4PGS_A (2.5)</td><td>Uncharacterized protein YetJ</td><td>15.4</td><td>3.5</td><td>207</td><td>2.7</td><td>175</td><td>2.2</td><td>183</td></tr><tr><td>4QTN_A (2.8)</td><td>Vitamin B3 transporter PnuC</td><td>9.0</td><td>4.2</td><td>202</td><td>3.0</td><td>155</td><td>2.8</td><td>178</td></tr><tr><td>4OD4_A (3.3)</td><td>4-hydroxybenzoate octaprenyltransferase</td><td>22.8</td><td>3.9</td><td>275</td><td>3.4</td><td>242</td><td>2.8</td><td>231</td></tr><tr><td>4O6M_A (1.9)</td><td>CDP-alcohol phosphotransferase</td><td>13.3</td><td>4.1</td><td>188</td><td>4.0</td><td>165</td><td>2.3</td><td>159</td></tr><tr><td>4WD8_A (2.3)</td><td>Bestrophin domain protein</td><td>5.94</td><td>N/A</td><td>268</td><td colspan="4">Not converged</td></tr><tr><td>4F35_A (3.2)</td><td>Transporter, NadC family</td><td>14.5</td><td>N/A</td><td>434</td><td colspan="4">Not converged</td></tr></tbody></table><table-wrap-foot><fn><p>Column 1, PDB code (resolution of the crystal structure); column 2, protein name; column 3, sequences per length, after filtering to reduce the redundancy to 90%; column 4, RMSD of predicted structure to native structure; column 5, length of native structure modeled; column 6, RMSD over converged and constrained region; column 7, length of converged and constrained region; column 8, RMSD over TM-align structural alignment; column 9, length of structurally aligned region.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-09376-v1.xml">elife-09376-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.09376.006</object-id><label>Table 1.</label><caption><p>Effective diffusion coefficients (K; &#181;m<sup>2</sup>/s) and anomalous diffusion exponents (&#945;) for macromolecules in each condition.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.09376.006">http://dx.doi.org/10.7554/eLife.09376.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Condition</th><th colspan="2"> <italic>POA1</italic> Locus</th><th colspan="2">pLacO Plasmid</th><th colspan="2"> <italic>URA3</italic> Locus</th><th colspan="2"> <italic>GFA1</italic> mRNP</th><th colspan="2"> <italic>FBA1</italic> mRNP</th></tr><tr><th>K</th><th>&#945;</th><th>K</th><th>&#945;</th><th>K</th><th>&#945;</th><th>K</th><th>&#945;</th><th>K</th><th>&#945;</th></tr></thead><tbody><tr><td>(+) Glucose</td><td>0.0057</td><td>0.69</td><td>0.0067</td><td>0.78</td><td>0.0076</td><td>0.65</td><td>0.0420</td><td>0.83</td><td>0.0501</td><td>0.85</td></tr><tr><td>(-) Glucose</td><td>0.0023</td><td>0.64</td><td>0.0021</td><td>0.80</td><td>0.0022</td><td>0.73</td><td>0.0131</td><td>0.77</td><td>0.0139</td><td>0.77</td></tr><tr><td>(-) Glucose pH 7.4</td><td>0.0015</td><td>0.65</td><td>--</td><td>--</td><td>--</td><td>--</td><td>0.0120</td><td>0.75</td><td>--</td><td>--</td></tr><tr><td>DMSO</td><td>0.0059</td><td>0.56</td><td>0.0046</td><td>0.70</td><td>0.0060</td><td>0.61</td><td>0.0491</td><td>0.83</td><td>0.0541</td><td>0.83</td></tr><tr><td>Nocodazole</td><td>0.0040</td><td>0.48</td><td>0.0025</td><td>0.57</td><td>0.0046</td><td>0.51</td><td>0.0364</td><td>0.85</td><td>0.0397</td><td>0.85</td></tr><tr><td>Latrunculin A</td><td>0.0038</td><td>0.50</td><td>0.0024</td><td>0.63</td><td>0.0038</td><td>0.55</td><td>0.0476</td><td>0.82</td><td>0.0550</td><td>0.81</td></tr><tr><td>Nocodazole + LatA</td><td>0.0028</td><td>0.48</td><td>0.0014</td><td>0.49</td><td>0.0026</td><td>0.52</td><td>0.0303</td><td>0.81</td><td>0.0367</td><td>0.82</td></tr><tr><td>2 mM K<sup>+</sup>Sorbate</td><td>0.0056</td><td>0.73</td><td>0.0051</td><td>0.80</td><td>0.0059</td><td>0.68</td><td>0.0402</td><td>0.82</td><td>0.0296</td><td>0.80</td></tr><tr><td>4 mM K<sup>+</sup>Sorbate</td><td>0.0050</td><td>0.75</td><td>0.0044</td><td>0.72</td><td>0.0048</td><td>0.71</td><td>0.0406</td><td>0.78</td><td>0.0242</td><td>0.79</td></tr><tr><td>6 mM K<sup>+</sup>Sorbate</td><td>0.0039</td><td>0.70</td><td>0.0018</td><td>0.66</td><td>0.0027</td><td>0.66</td><td>0.0378</td><td>0.76</td><td>0.0270</td><td>0.78</td></tr><tr><td>8 mM K<sup>+</sup>Sorbate</td><td>0.0023</td><td>0.64</td><td>0.0012</td><td>0.61</td><td>0.0014</td><td>0.61</td><td>0.0340</td><td>0.76</td><td>0.0164</td><td>0.77</td></tr><tr><td>0.4 M NaCl</td><td>0.0030</td><td>0.69</td><td>0.0026</td><td>0.75</td><td>--</td><td>--</td><td>0.0129</td><td>0.83</td><td>0.0146</td><td>0.85</td></tr><tr><td>0.6 M NaCl</td><td>0.0012</td><td>0.60</td><td>0.0011</td><td>0.58</td><td>--</td><td>--</td><td>0.0047</td><td>0.83</td><td>0.0057</td><td>0.84</td></tr><tr><td>0.8 M NaCl</td><td>0.0009</td><td>0.63</td><td>0.0011</td><td>0.63</td><td>--</td><td>--</td><td>0.0013</td><td>0.67</td><td>0.0016</td><td>0.77</td></tr><tr><td>Quiescence</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>--</td><td>0.0004</td><td>0.29</td><td>0.0015</td><td>0.68</td></tr><tr><td>0.02% Azide (Wash)</td><td>0.0037</td><td>0.74</td><td>--</td><td>--</td><td>--</td><td>--</td><td>0.0293</td><td>0.82</td><td>--</td><td>--</td></tr><tr><td>0.02% Azide (Spike)</td><td>0.0012</td><td>0.67</td><td>--</td><td>--</td><td>--</td><td>--</td><td>0.0155</td><td>0.81</td><td>--</td><td>--</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-09376-v2.xml">elife-09376-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.09376.006</object-id><label>Table 1.</label><caption><p>Effective diffusion coefficients (K; &#181;m<sup>2</sup>/s) and anomalous diffusion exponents (&#945;) for macromolecules in each condition.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.09376.006">http://dx.doi.org/10.7554/eLife.09376.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Condition</th><th colspan="2" valign="bottom"> <italic>POA1</italic> Locus</th><th colspan="2" valign="bottom">pLacO Plasmid</th><th colspan="2" valign="bottom"> <italic>URA3</italic> Locus</th><th colspan="2" valign="bottom"> <italic>GFA1</italic> mRNP</th><th colspan="2" valign="bottom"> <italic>FBA1</italic> mRNP</th></tr><tr><th valign="bottom">K</th><th valign="bottom">&#945;</th><th valign="bottom">K</th><th valign="bottom">&#945;</th><th valign="bottom">K</th><th valign="bottom">&#945;</th><th valign="bottom">K</th><th valign="bottom">&#945;</th><th valign="bottom">K</th><th valign="bottom">&#945;</th></tr></thead><tbody><tr><td valign="bottom">(+) Glucose</td><td valign="bottom">0.0057</td><td valign="bottom">0.69</td><td valign="bottom">0.0067</td><td valign="bottom">0.78</td><td valign="bottom">0.0076</td><td valign="bottom">0.65</td><td valign="bottom">0.0420</td><td valign="bottom">0.83</td><td valign="bottom">0.0501</td><td valign="bottom">0.85</td></tr><tr><td valign="bottom">(-) Glucose</td><td valign="bottom">0.0023</td><td valign="bottom">0.64</td><td valign="bottom">0.0021</td><td valign="bottom">0.80</td><td valign="bottom">0.0022</td><td valign="bottom">0.73</td><td valign="bottom">0.0131</td><td valign="bottom">0.77</td><td valign="bottom">0.0139</td><td valign="bottom">0.77</td></tr><tr><td valign="bottom">(-) Glucose pH 7.4</td><td valign="bottom">0.0015</td><td valign="bottom">0.65</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">0.0120</td><td valign="bottom">0.75</td><td valign="bottom">--</td><td valign="bottom">--</td></tr><tr><td valign="bottom">DMSO</td><td valign="bottom">0.0059</td><td valign="bottom">0.56</td><td valign="bottom">0.0046</td><td valign="bottom">0.70</td><td valign="bottom">0.0060</td><td valign="bottom">0.61</td><td valign="bottom">0.0491</td><td valign="bottom">0.83</td><td valign="bottom">0.0541</td><td valign="bottom">0.83</td></tr><tr><td valign="bottom">Nocodazole</td><td valign="bottom">0.0040</td><td valign="bottom">0.48</td><td valign="bottom">0.0025</td><td valign="bottom">0.57</td><td valign="bottom">0.0046</td><td valign="bottom">0.51</td><td valign="bottom">0.0364</td><td valign="bottom">0.85</td><td valign="bottom">0.0397</td><td valign="bottom">0.85</td></tr><tr><td valign="bottom">Latrunculin A</td><td valign="bottom">0.0038</td><td valign="bottom">0.50</td><td valign="bottom">0.0024</td><td valign="bottom">0.63</td><td valign="bottom">0.0038</td><td valign="bottom">0.55</td><td valign="bottom">0.0476</td><td valign="bottom">0.82</td><td valign="bottom">0.0550</td><td valign="bottom">0.81</td></tr><tr><td valign="bottom">Nocodazole + LatA</td><td valign="bottom">0.0028</td><td valign="bottom">0.48</td><td valign="bottom">0.0014</td><td valign="bottom">0.49</td><td valign="bottom">0.0026</td><td valign="bottom">0.52</td><td valign="bottom">0.0303</td><td valign="bottom">0.81</td><td valign="bottom">0.0367</td><td valign="bottom">0.82</td></tr><tr><td valign="bottom">2 mM K<sup>+</sup>Sorbate</td><td valign="bottom">0.0056</td><td valign="bottom">0.73</td><td valign="bottom">0.0051</td><td valign="bottom">0.80</td><td valign="bottom">0.0059</td><td valign="bottom">0.68</td><td valign="bottom">0.0402</td><td valign="bottom">0.82</td><td valign="bottom">0.0296</td><td valign="bottom">0.80</td></tr><tr><td valign="bottom">4 mM K<sup>+</sup>Sorbate</td><td valign="bottom">0.0050</td><td valign="bottom">0.75</td><td valign="bottom">0.0044</td><td valign="bottom">0.72</td><td valign="bottom">0.0048</td><td valign="bottom">0.71</td><td valign="bottom">0.0406</td><td valign="bottom">0.78</td><td valign="bottom">0.0242</td><td valign="bottom">0.79</td></tr><tr><td valign="bottom">6 mM K<sup>+</sup>Sorbate</td><td valign="bottom">0.0039</td><td valign="bottom">0.70</td><td valign="bottom">0.0018</td><td valign="bottom">0.66</td><td valign="bottom">0.0027</td><td valign="bottom">0.66</td><td valign="bottom">0.0378</td><td valign="bottom">0.76</td><td valign="bottom">0.0270</td><td valign="bottom">0.78</td></tr><tr><td valign="bottom">8 mM K<sup>+</sup>Sorbate</td><td valign="bottom">0.0023</td><td valign="bottom">0.64</td><td valign="bottom">0.0012</td><td valign="bottom">0.61</td><td valign="bottom">0.0014</td><td valign="bottom">0.61</td><td valign="bottom">0.0340</td><td valign="bottom">0.76</td><td valign="bottom">0.0164</td><td valign="bottom">0.77</td></tr><tr><td valign="bottom">0.4 M NaCl</td><td valign="bottom">0.0030</td><td valign="bottom">0.69</td><td valign="bottom">0.0026</td><td valign="bottom">0.75</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">0.0129</td><td valign="bottom">0.83</td><td valign="bottom">0.0146</td><td valign="bottom">0.85</td></tr><tr><td valign="bottom">0.6 M NaCl</td><td valign="bottom">0.0012</td><td valign="bottom">0.60</td><td valign="bottom">0.0011</td><td valign="bottom">0.58</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">0.0047</td><td valign="bottom">0.83</td><td valign="bottom">0.0057</td><td valign="bottom">0.84</td></tr><tr><td valign="bottom">0.8 M NaCl</td><td valign="bottom">0.0009</td><td valign="bottom">0.63</td><td valign="bottom">0.0011</td><td valign="bottom">0.63</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">0.0013</td><td valign="bottom">0.67</td><td valign="bottom">0.0016</td><td valign="bottom">0.77</td></tr><tr><td valign="bottom">Quiescence</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">0.0004</td><td valign="bottom">0.29</td><td valign="bottom">0.0015</td><td valign="bottom">0.68</td></tr><tr><td valign="bottom">0.02% Azide (Wash)</td><td valign="bottom">0.0037</td><td valign="bottom">0.74</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">0.0293</td><td valign="bottom">0.82</td><td valign="bottom">--</td><td valign="bottom">--</td></tr><tr><td valign="bottom">0.02% Azide (Spike)</td><td valign="bottom">0.0012</td><td valign="bottom">0.67</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">--</td><td valign="bottom">0.0155</td><td valign="bottom">0.81</td><td valign="bottom">--</td><td valign="bottom">--</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-09418-v2.xml">elife-09418-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.09418.017</object-id><label>Table 2.</label><caption><p>The onset times in time-direction maps. The first main column shows the onset time calculated from trials aligned on target onset and the second main column shows the onset time calculated from trials aligned on saccade onset. Within each main column, the first minor column shows the time when the neural activity was significantly different from background activity (-20&#160;to&#160;0 before target onset). The second minor column shows the time when the neural activity represented the choice. In no-choice trials, this corresponds to the time when the activity of neurons with a preferred direction within &#177; 30&#176; of the target was significantly different from the activity of neurons where no target was presented (the neurons with preferred direction within 240&#8211;300&#176;). For choice trial, it corresponds to the time when the activity for the chosen target was significant different form the activity for the non-chosen target (in both cases the neurons with preferred direction within &#177; 30&#176; of their respective target). A permutation test with multiple comparison correction was used to calculate the onset times.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.09418.017">http://dx.doi.org/10.7554/eLife.09418.017</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2">Time from target onset</th><th colspan="2">Time from saccade onset</th></tr><tr><th>Activity vs background</th><th>Chosen vs non-chosen</th><th>Activity vs background</th><th>Chosen vs non-chosen</th></tr></thead><tbody><tr><td>No-choice</td><td>44&#160;ms</td><td/><td>-141&#160;ms</td><td/></tr><tr><td>Choice</td><td>40&#160;ms</td><td>105&#160;ms</td><td>-185&#160;ms</td><td>-70&#160;ms</td></tr><tr><td>Choice (dV&gt;=0.4)</td><td>44&#160;ms</td><td>92&#160;ms</td><td>-129&#160;ms</td><td>-68&#160;ms</td></tr><tr><td>Choice (dV&lt;0.4)</td><td>41&#160;ms</td><td>139&#160;ms</td><td>-169&#160;ms</td><td>-42&#160;ms</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-09419-v2.xml">elife-09419-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.09419.006</object-id><label>Table 1.</label><caption><p>Baseline demographic characteristics of the 201 patients who underwent CXCL13 analyses.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.09419.006">http://dx.doi.org/10.7554/eLife.09419.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Characteristics</th><th colspan="3">Total</th><th colspan="3">Highly polluted region (HPR)</th><th colspan="3">Control region (CR)</th><th rowspan="2">p values <break/>(HPR vs CR)</th></tr><tr><th>Case, n</th><th>CXCL13 high, <break/>n (%)</th><th>p values</th><th>Case, n</th><th>CXCL13 high, <break/>n (%)</th><th>p values</th><th>Case, n</th><th>CXCL13 high, <break/>n (%)</th><th>p values</th></tr></thead><tbody><tr><td>Total</td><td>201</td><td>134 (66.7)</td><td/><td>70</td><td>63 (90)</td><td/><td>131</td><td>71 (54.2)</td><td/><td>0.0000003</td></tr><tr><td>G: Male</td><td>134</td><td>88 (65.7)</td><td>0.67</td><td>47</td><td>41 (87.2)</td><td>0.27</td><td>87</td><td>47 (54)</td><td>0.95</td><td>0.0001</td></tr><tr><td>&#160;Female</td><td>67</td><td>46 (68.7)</td><td/><td>23</td><td>22 (95.7)</td><td/><td>44</td><td>24 (54.5)</td><td/><td>0.0006</td></tr><tr><td>A: &lt;65 y</td><td>140</td><td>98 (70)</td><td>0.21</td><td>56</td><td>51 (91.1)</td><td>0.55</td><td>84</td><td>47 (56)</td><td>0.7</td><td>0.000009</td></tr><tr><td>&#160;&#8805;65 y</td><td>56</td><td>34 (60.7)</td><td/><td>14</td><td>12 (85.7)</td><td/><td>42</td><td>22 (52.4)</td><td/><td>0.03</td></tr><tr><td>Unknown</td><td>5</td><td>2 (40)</td><td/><td/><td/><td/><td>5</td><td>2 (40)</td><td/><td/></tr><tr><td>S: Smoker</td><td>107</td><td>75 (70.1)</td><td>0.18</td><td>36</td><td>31 (86.1)</td><td>0.17</td><td>71</td><td>44 (62)</td><td>0.04</td><td>0.01</td></tr><tr><td>&#160;Non-smoker</td><td>87</td><td>53 (60.9)</td><td/><td>27</td><td>26 (96.3)</td><td/><td>60</td><td>27 (45)</td><td/><td>0.000006</td></tr><tr><td>&#160;Unknown</td><td>7</td><td>6 (85.7)</td><td/><td>7</td><td>6 (85.7)</td><td/><td/><td/><td/><td/></tr><tr><td>H: Adenocarcinoma (AD)</td><td>131</td><td>91 (69.5)</td><td>0.45</td><td>48</td><td>44 (91.7)</td><td>0.37</td><td>83</td><td>47 (56.6)</td><td>0.84</td><td>0.00003</td></tr><tr><td>Squamous cell carcinoma&#160;(SCC)</td><td>61</td><td>39 (63.9)</td><td/><td>19</td><td>16 (84.2)</td><td/><td>42</td><td>23 (54.8)</td><td/><td>0.03</td></tr><tr><td>Others</td><td>9</td><td>4 (44.4)</td><td/><td>3</td><td>3 (100)</td><td/><td>6</td><td>1 (16.7)</td><td/><td/></tr><tr><td>Tumor node metastasis&#160;(TNM): I</td><td>88</td><td>51 (58)</td><td>0.007</td><td>29</td><td>23 (79.3)</td><td>0.02</td><td>59</td><td>28 (47.5)</td><td>0.05</td><td>0.004</td></tr><tr><td>&#160;II</td><td>27</td><td>17 (63)</td><td/><td>9</td><td>8 (88.9)</td><td/><td>18</td><td>9 (50)</td><td/><td>0.005</td></tr><tr><td>&#160;III</td><td>58</td><td>43 (74.1)</td><td/><td>17</td><td>17 (100)</td><td/><td>41</td><td>26 (63.4)</td><td/><td>0.004</td></tr><tr><td>&#160;IV</td><td>22</td><td>19 (86.4)</td><td/><td>11</td><td>11 (100)</td><td/><td>11</td><td>8 (72.7)</td><td/><td>0.06</td></tr><tr><td>&#160;Unknown</td><td>6</td><td>4 (66.7)</td><td/><td>4</td><td>4 (100)</td><td/><td>2</td><td>0 (0)</td><td/><td/></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>G, gender; A, age; S, smoke; H, histology.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-09651-v2.xml">elife-09651-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.09651.008</object-id><label>Table 2.</label><caption><p>R<sup>2</sup> values quantifying goodness of fit for ideal observer (<italic>&#963;<sub>d</sub></italic> = 0) and three alternative decisional noise magnitudes (<italic>&#963;<sub>d</sub></italic> &gt;0) which cause increasing degrees of Performance without Awareness. Decisional noise greater than 0 &#8211; i.e., increased level of Performance without Awareness &#8211; causes a drop in goodness of fit between model and human data. See Methods and <xref ref-type="app" rid="app4">Appendix 4</xref> for more details.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.09651.008">http://dx.doi.org/10.7554/eLife.09651.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Expt</th><th rowspan="2">Subject</th><th colspan="4">Decisional noise <italic>&#963;<sub>d</sub></italic></th> <th/></tr></thead><tbody><tr><td>0 (Ideal observer)</td><td>0.1</td><td>0.2</td><td>0.3</td><td/><td/><td/></tr><tr><td rowspan="9">1</td><td>1</td><td>0.465</td><td>0.459</td><td>0.456</td><td>0.447</td><td/></tr><tr><td>2</td><td>0.580</td><td>0.578</td><td>0.565</td><td>0.544</td><td/></tr><tr><td>3</td><td>0.470</td><td>0.464</td><td>0.448</td><td>0.428</td><td/></tr><tr><td>4</td><td>0.396</td><td>0.392</td><td>0.381</td><td>0.363</td><td/></tr><tr><td>5</td><td>0.649</td><td>0.655</td><td>0.645</td><td>0.628</td><td/></tr><tr><td>6</td><td>0.480</td><td>0.473</td><td>0.458</td><td>0.434</td><td/></tr><tr><td>7</td><td>0.453</td><td>0.452</td><td>0.444</td><td>0.427</td><td/></tr><tr><td>8</td><td>0.602</td><td>0.595</td><td>0.583</td><td>0.563</td><td/></tr><tr><td>9</td><td>0.503</td><td>0.509</td><td>0.512</td><td>0.504</td><td/></tr><tr><td rowspan="3">2</td><td>1</td><td>0.624</td><td>0.624</td><td>0.622</td><td>0.612</td><td/></tr><tr><td>2</td><td>0.783</td><td>0.780</td><td>0.775</td><td>0.766</td><td/></tr><tr><td>3</td><td>0.777</td><td>0.778</td><td>0.767</td><td>0.753</td><td/></tr><tr><td colspan="2">Mean R<sup>2</sup> (&#963;)</td><td>0.565 (0.126)</td><td>0.563<break/>(0.128)</td><td>0.555<break/>(0.129)</td><td>0.539 (0.131)</td><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-09651-v3.xml">elife-09651-v3.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.09651.008</object-id><label>Table 2.</label><caption><p>R<sup>2</sup> values quantifying goodness of fit for ideal observer (<italic>&#963;<sub>d</sub></italic> = 0) and three alternative decisional noise magnitudes (<italic>&#963;<sub>d</sub></italic> &gt;0) which cause increasing degrees of Performance without Awareness. Decisional noise greater than 0 &#8211; i.e., increased level of Performance without Awareness &#8211; causes a drop in goodness of fit between model and human data. See Methods and <xref ref-type="app" rid="app4">Appendix 4</xref> for more details.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.09651.008">http://dx.doi.org/10.7554/eLife.09651.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Expt</th><th rowspan="2">Subject</th><th colspan="4">Decisional noise <italic>&#963;<sub>d</sub></italic></th><th/></tr></thead><tbody><tr><td>0 (Ideal observer)</td><td>0.1</td><td>0.2</td><td>0.3</td><td/><td/><td/></tr><tr><td rowspan="9">1</td><td>1</td><td>0.465</td><td>0.459</td><td>0.456</td><td>0.447</td><td/></tr><tr><td>2</td><td>0.580</td><td>0.578</td><td>0.565</td><td>0.544</td><td/></tr><tr><td>3</td><td>0.470</td><td>0.464</td><td>0.448</td><td>0.428</td><td/></tr><tr><td>4</td><td>0.396</td><td>0.392</td><td>0.381</td><td>0.363</td><td/></tr><tr><td>5</td><td>0.649</td><td>0.655</td><td>0.645</td><td>0.628</td><td/></tr><tr><td>6</td><td>0.480</td><td>0.473</td><td>0.458</td><td>0.434</td><td/></tr><tr><td>7</td><td>0.453</td><td>0.452</td><td>0.444</td><td>0.427</td><td/></tr><tr><td>8</td><td>0.602</td><td>0.595</td><td>0.583</td><td>0.563</td><td/></tr><tr><td>9</td><td>0.503</td><td>0.509</td><td>0.512</td><td>0.504</td><td/></tr><tr><td rowspan="3">2</td><td>1</td><td>0.624</td><td>0.624</td><td>0.622</td><td>0.612</td><td/></tr><tr><td>2</td><td>0.783</td><td>0.780</td><td>0.775</td><td>0.766</td><td/></tr><tr><td>3</td><td>0.777</td><td>0.778</td><td>0.767</td><td>0.753</td><td/></tr><tr><td colspan="2">Mean R<sup>2</sup> (&#963;)</td><td>0.565 (0.126)</td><td>0.563 <break/>(0.128)</td><td>0.555 <break/>(0.129)</td><td>0.539 (0.131)</td><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-09656-v1.xml">elife-09656-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.09656.007</object-id><label>Table 1.</label><caption><p>Urine output (A), fluid intake (B), Urine osmolality (C) and urine sodium concentration (D) in rats injected into the SON and PVN with either, control, scrambled shRNA or Caprin-2 shRNA, in euhydrated (water: W1&#8211;3) and salt-loading (SL 1&#8211;7) conditions</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.09656.007">http://dx.doi.org/10.7554/eLife.09656.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="3">Scrambled shRNA</th><th colspan="3">Caprin 2 shRNA</th></tr><tr><th>Mean</th><th>SEM</th><th>N</th><th>Mean</th><th>SEM</th><th>N</th></tr></thead><tbody><tr><td colspan="7">A. Urine output (ml/100 g b.m.)</td></tr><tr><td>W1</td><td>4.661</td><td>0.344</td><td>9</td><td>3.954</td><td>0.222</td><td>5</td></tr><tr><td>W2</td><td>4.784</td><td>0.339</td><td>9</td><td>4.048</td><td>0.423</td><td>5</td></tr><tr><td>W3</td><td>4.803</td><td>0.413</td><td>9</td><td>4.230</td><td>0.461</td><td>5</td></tr><tr><td>SL1</td><td>11.649</td><td>1.590</td><td>9</td><td>10.600</td><td>1.155</td><td>5</td></tr><tr><td>SL2</td><td>14.482</td><td>2.005</td><td>9</td><td>9.488</td><td>1.062</td><td>5</td></tr><tr><td>SL3</td><td>17.790</td><td>2.294</td><td>9</td><td>10.814</td><td>1.776</td><td>5</td></tr><tr><td>SL4</td><td>20.884</td><td>3.086</td><td>9</td><td>11.860</td><td>1.485</td><td>5</td></tr><tr><td>SL5</td><td>22.046</td><td>3.564</td><td>9</td><td>11.982</td><td>2.223</td><td>5</td></tr><tr><td>SL6</td><td>22.473</td><td>3.367</td><td>9</td><td>13.598</td><td>2.666</td><td>5</td></tr><tr><td>SL7</td><td>22.833</td><td>3.828</td><td>9</td><td>12.194</td><td>2.135</td><td>5</td></tr><tr><td colspan="7">B. Fluid intake (ml/100 g b.m.)</td></tr><tr><td>W1</td><td>9.353</td><td>0.668</td><td>9</td><td>9.674</td><td>0.421</td><td>5</td></tr><tr><td>W2</td><td>9.752</td><td>0.607</td><td>9</td><td>8.832</td><td>0.251</td><td>5</td></tr><tr><td>W3</td><td>9.684</td><td>0.542</td><td>9</td><td>9.203</td><td>0.425</td><td>5</td></tr><tr><td>SL1</td><td>14.127</td><td>1.665</td><td>9</td><td>13.806</td><td>1.224</td><td>5</td></tr><tr><td>SL2</td><td>18.270</td><td>2.145</td><td>9</td><td>14.483</td><td>1.156</td><td>5</td></tr><tr><td>SL3</td><td>22.002</td><td>2.325</td><td>9</td><td>15.707</td><td>1.460</td><td>5</td></tr><tr><td>SL4</td><td>25.351</td><td>3.211</td><td>9</td><td>16.680</td><td>1.886</td><td>5</td></tr><tr><td>SL5</td><td>26.842</td><td>3.580</td><td>9</td><td>16.818</td><td>2.437</td><td>5</td></tr><tr><td>SL6</td><td>27.166</td><td>3.912</td><td>9</td><td>18.209</td><td>3.124</td><td>5</td></tr><tr><td>SL7</td><td>28.952</td><td>3.805</td><td>9</td><td>16.554</td><td>2.108</td><td>5</td></tr><tr><td colspan="7">C. Urine osmolality (mOsmol/kg)</td></tr><tr><td>W1</td><td>1943.333</td><td>154.937</td><td>9</td><td>1994.000</td><td>175.516</td><td>5</td></tr><tr><td>W2</td><td>1986.667</td><td>190.343</td><td>9</td><td>2090.000</td><td>65.651</td><td>5</td></tr><tr><td>W3</td><td>2105.556</td><td>204.702</td><td>9</td><td>1860.000</td><td>176.437</td><td>5</td></tr><tr><td>SL1</td><td>1532.222</td><td>108.189</td><td>9</td><td>1362.000</td><td>87.772</td><td>5</td></tr><tr><td>SL2</td><td>1390.000</td><td>85.261</td><td>9</td><td>1690.000</td><td>120.831</td><td>5</td></tr><tr><td>SL3</td><td>1280.000</td><td>80.035</td><td>9</td><td>1606.000</td><td>178.342</td><td>5</td></tr><tr><td>SL4</td><td>1181.111</td><td>83.773</td><td>9</td><td>1520.000</td><td>149.767</td><td>5</td></tr><tr><td>SL5</td><td>1188.889</td><td>115.548</td><td>9</td><td>1620.000</td><td>195.090</td><td>5</td></tr><tr><td>SL6</td><td>1121.111</td><td>75.064</td><td>9</td><td>1438.000</td><td>152.918</td><td>5</td></tr><tr><td>SL7</td><td>1093.333</td><td>79.861</td><td>9</td><td>1474.000</td><td>156.831</td><td>5</td></tr><tr><td colspan="7">D. Urine sodium (mM)</td></tr><tr><td>W1-3</td><td>295.929</td><td>15.179</td><td>25</td><td>321.306</td><td>13.196</td><td>15</td></tr><tr><td>SL1-3</td><td>726.221</td><td>7.102</td><td>27</td><td>746.221</td><td>14.048</td><td>15</td></tr><tr><td>SL2-4</td><td>721.850</td><td>6.852</td><td>27</td><td>761.959</td><td>11.233</td><td>15</td></tr><tr><td>SL3-5</td><td>713.653</td><td>7.925</td><td>27</td><td>759.992</td><td>13.389</td><td>15</td></tr><tr><td>SL4-6</td><td>703.270</td><td>7.888</td><td>27</td><td>758.025</td><td>14.093</td><td>15</td></tr><tr><td>SL5-7</td><td>701.085</td><td>8.319</td><td>27</td><td>763.926</td><td>15.838</td><td>15</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-10034-v2.xml">elife-10034-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.10034.005</object-id><label>Table 1.</label><caption><p>Data collection and refinement statistics</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.10034.005">http://dx.doi.org/10.7554/eLife.10034.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="5"><sup><italic>Sc</italic></sup>XPO1-RanGppNHp-Yrb1p bound to NES of:</th></tr><tr><th>Selenomethione-hRio2</th><th>CPEB4</th><th>Selenomethione-hRio2-R</th><th>CPEB4-R</th><th>PKI-Flip3</th></tr></thead><tbody><tr><td colspan="6">Data collection</td></tr><tr><td>&#8195;Space group</td><td>P4<sub>3</sub>2<sub>1</sub>2</td><td/><td/><td/><td/></tr><tr><td>&#8195;Cell dimensions</td><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<italic>a</italic>, <italic>b</italic>, <italic>c</italic> (&#197;)</td><td>106.48, 106.48, 303.73</td><td>105.96, 105.96, 304.00</td><td>106.69, 106.69, 304.50</td><td>106.48, 106.48, 303.73</td><td>105.96, 105.96, 304.00</td></tr><tr><td>&#8195;&#8195;<italic>a</italic>, <italic>b</italic>, <italic>g</italic> (&#176;)</td><td>90, 90, 90</td><td>90, 90, 90</td><td>90, 90, 90</td><td>90, 90, 90</td><td>90, 90, 90</td></tr><tr><td>&#8195;Resolution (&#197;)</td><td>50.00&#8211;2.28 (2.32&#8211;2.28)<xref ref-type="table-fn" rid="tblfn1">*</xref></td><td>50.00&#8211;2.10 (2.14&#8211;2.10)</td><td>50.00&#8211;2.28 (2.32&#8211;2.28)</td><td>50.00&#8211;2.94 (3.00&#8211;2.94)</td><td>50.00&#8211;2.55 (2.59&#8211;2.55)</td></tr><tr><td>&#8195;<italic>R</italic><sub>pim</sub></td><td>2.9 (37.7)</td><td>3.5 (43.4)</td><td>3.5 (38.6)</td><td>4.9 (40.6)</td><td>4.1 (46.5)</td></tr><tr><td>&#8195;<italic>I</italic>/s<italic>I</italic></td><td>24.3 (2.17)</td><td>19.5 (1.70)</td><td>22.5 (2.72)</td><td>13.3 (1.87)</td><td>19.0 (1.92)</td></tr><tr><td>&#8195;Completeness (%)</td><td>98.6 (99.8)</td><td>99.5(100)</td><td>98.0 (99.2)</td><td>94.6 (96.0)</td><td>99.6 (100)</td></tr><tr><td>&#8195;Redundancy</td><td>7.0 (5.9)</td><td>6.0 (6.1)</td><td>7.0 (7.0)</td><td>6.2 (5.7)</td><td>5.5 (5.5)</td></tr><tr><td colspan="6">Refinement</td></tr><tr><td>&#8195;Resolution (&#197;)</td><td>45.7&#8211;2.28 (2.32&#8211;2.28)</td><td>40.2&#8211;2.09 (2.12&#8211;2.09)</td><td>37.7&#8211;2.28 (2.31&#8211;2.28)</td><td>47.5&#8211;2.94 (3.02&#8211;2.94)</td><td>47.5&#8211;2.54 (2.60&#8211;2.54)</td></tr><tr><td>&#8195;No. reflections</td><td>77,245 (2833)</td><td>98,659 (1793)</td><td>79,492 (3267)</td><td>34,265 (2013)</td><td>56862 (3361)</td></tr><tr><td>&#8195;<italic>R</italic><sub>work</sub>/<italic>R</italic><sub>free</sub></td><td>17.8 (25.8)/21.9 (27.3)</td><td>17.0 (23.8)/20.8 (27.0)</td><td>16.8 (24.7)/21.2 (27.6)</td><td>18.1 (25.2)/24.0 (31.3)</td><td>18.6 (25.0)/22.6 (30.6)</td></tr><tr><td>&#8195;No. atoms</td><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;Protein</td><td>10,859</td><td>11,114</td><td>10,823</td><td>10,708</td><td>10797</td></tr><tr><td>&#8195;&#8195;Ligand/ion</td><td>60</td><td>76</td><td>59</td><td>51</td><td>51</td></tr><tr><td>&#8195;&#8195;Water</td><td>271</td><td>660</td><td>358</td><td>8</td><td>253</td></tr><tr><td>&#8195;&#8195;NES Peptide/&#934;</td><td>111/46</td><td>122/43</td><td>130/46</td><td>112/43</td><td>105/43</td></tr><tr><td>&#8195;B-factors</td><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;Protein</td><td>42.0</td><td>39.3</td><td>43.9</td><td>53.8</td><td>46.5</td></tr><tr><td>&#8195;&#8195;Ligand/ion</td><td>44.3</td><td>51.7</td><td>46.9</td><td>41.8</td><td>41.6</td></tr><tr><td>&#8195;&#8195;Water</td><td>33.4</td><td>34.8</td><td>35.4</td><td>23.3</td><td>35.3</td></tr><tr><td>&#8195;&#8195;NES peptide/&#934;</td><td>80.5/77.3</td><td>77.6/70.4</td><td>67.5/61.7</td><td>81.2/80.5</td><td>98.6/96.0</td></tr><tr><td>&#8195;R.m.s deviations</td><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;Bond lengths (&#197;)</td><td>0.003</td><td>0.003</td><td>0.006</td><td>0.003</td><td>0.004</td></tr><tr><td>&#8195;&#8195;Bond angles (&#176;)</td><td>0.617</td><td>0.689</td><td>0.835</td><td>0.578</td><td>0.673</td></tr><tr><td>PDB code</td><td>5DHF</td><td>5DIF</td><td>5DI9</td><td>5DHA</td><td>5DH9</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><label>*</label><p>Highest resolution shell is shown in parenthesis.</p></fn><fn><p>One crystal was used for each structure.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-10095-v1.xml">elife-10095-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tblu1" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">RNA</th><th rowspan="2">Length</th><th colspan="2">M<sup>2</sup>/Rosetta (no MOHCA, control)</th><th colspan="2">MCM</th></tr><tr><th>RMSD to crystal (&#197;) (accuracy)</th><th>p-value&#167;</th><th>RMSD to crystal (&#197;) (accuracy)</th><th>p-value&#167;</th></tr></thead><tbody><tr><td><italic>Tetrahymena</italic> ribozyme P4&#8211;P6 domain</td><td>158</td><td>38.3</td><td>&gt;0.9</td><td>8.6</td><td><named-content content-type="author-callout-style2">&lt;1.0 &#215; 10<sup>&#8722;16</sup></named-content></td></tr><tr><td>V. cholerae cyclic-di-GMP riboswitch aptamer, ligand-bound*</td><td>89</td><td>11.3</td><td><named-content content-type="author-callout-style2">2.6 &#215; 10<sup>&#8722;3</sup></named-content></td><td>7.6</td><td><named-content content-type="author-callout-style2">6.3 &#215; 10<sup>&#8722;7</sup></named-content></td></tr><tr><td>F. nucleatum double glycine riboswitch ligand-binding domain, ligand-bound*</td><td>159</td><td>30.5</td><td>&gt;0.9</td><td>7.9</td><td><named-content content-type="author-callout-style2">&lt;1.0 &#215; 10<sup>&#8722;16</sup></named-content></td></tr><tr><td>S. thermophilum adenosylcobalamin riboswitch aptamer, ligand-bound*</td><td>168</td><td>17.1&#8224;</td><td><named-content content-type="author-callout-style2">5.3 &#215; 10<sup>&#8722;7</sup></named-content></td><td>11.9</td><td><named-content content-type="author-callout-style2">4.0 &#215; 10<sup>&#8722;15</sup></named-content></td></tr><tr><td>Class I ligase</td><td>127</td><td>26.3</td><td>&gt;0.9</td><td>14.5</td><td><named-content content-type="author-callout-style2">6.8 &#215; 10<sup>&#8722;5</sup></named-content></td></tr><tr><td>Class I ligase, core domain&#8225;</td><td>87</td><td>14.0</td><td><named-content content-type="author-callout-style2">0.13</named-content></td><td>11.1</td><td><named-content content-type="author-callout-style2">3.1 &#215; 10<sup>&#8722;3</sup></named-content></td></tr><tr><td>D. iridis lariat-capping ribozyme</td><td>188</td><td>9.6&#8224;</td><td><named-content content-type="author-callout-style2">&lt;1.0 &#215; 10<sup>&#8722;16</sup></named-content></td><td>8.2</td><td>&lt;1.0 &#215; 10<sup>&#8722;16</sup></td></tr><tr><td><italic>D. iridis</italic> lariat-capping ribozyme, MCM refined regions&#8225;</td><td>69</td><td>17.0&#8224;</td><td>n.a.&#167;</td><td>11.2</td><td>n.a.&#167;</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-10095-v1.xml">elife-10095-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tblu2" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">RNA</th><th rowspan="2">Length</th><th colspan="2">M<sup>2</sup>/Rosetta</th><th colspan="2">MCM</th></tr><tr><th>RMSD to crystal (&#197;) (accuracy)</th><th>p-value&#167;</th><th>RMSD to crystal (&#197;) (accuracy)</th><th>p-value&#167;</th></tr></thead><tbody><tr><td><italic>Tetrahymena</italic> ribozyme P4&#8211;P6 domain</td><td>158</td><td>38.3</td><td>1.0</td><td>8.6</td><td>5.1 &#215; 10<sup>&#8722;12</sup></td></tr><tr><td><italic>V. cholerae</italic> cyclic-di-GMP riboswitch aptamer, ligand-bound*</td><td>89</td><td>11.3</td><td>0.28</td><td>7.6</td><td>4.3 &#215; 10<sup>&#8722;3</sup></td></tr><tr><td><italic>F. nucleatum</italic> double glycine riboswitch ligand-binding domain, ligand-bound*</td><td>159</td><td>30.5</td><td>1.0</td><td>7.9</td><td>2.1 &#215; 10<sup>&#8722;13</sup></td></tr><tr><td><italic>S. thermophilum</italic> adenosylcobalamin riboswitch aptamer, ligand-bound*</td><td>168</td><td>17.1&#8224;</td><td>3.9 &#215; 10<sup>&#8722;3</sup></td><td>11.9</td><td>1.5 &#215; 10<sup>&#8722;8</sup></td></tr><tr><td>Class I ligase</td><td>127</td><td>26.3</td><td>1.0</td><td>14.5</td><td>5.6 &#215; 10<sup>&#8722;2</sup></td></tr><tr><td>Class I ligase, core domain&#8225;</td><td>87</td><td>14.0</td><td>0.86</td><td>11.1</td><td>0.30</td></tr><tr><td><italic>D. iridis</italic> lariat-capping ribozyme</td><td>188</td><td>9.6&#8224;</td><td>1.2 &#215; 10<sup>&#8722;15</sup></td><td>8.2</td><td>&lt;1.0 &#215; 10<sup>&#8722;16</sup></td></tr><tr><td><italic>D. iridis</italic> lariat-capping ribozyme, MCM refined regions&#8225;</td><td>69</td><td>17.0&#8224;</td><td>n.a.&#167;</td><td>11.2</td><td>n.a.&#167;</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-10565-v2.xml">elife-10565-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.10565.018</object-id><label>Table 3.</label><caption><p>Analysis of transposon integration sequences in human genomes induced by PGBD5</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.10565.018">http://dx.doi.org/10.7554/eLife.10565.018</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2">Intact transposon</th><th colspan="2">Mutant transposon</th></tr><tr><th>TTAA ITR</th><th>Non-ITR</th><th>TTAA ITR</th><th>Non-ITR</th></tr></thead><tbody><tr><td colspan="5">Transposase</td></tr><tr><td>&#8195;GFP-PGBD5</td><td>82% (65)<sup>&#8224;</sup></td><td>18% (14)</td><td>11% (4)<sup>&#8225;</sup></td><td>89% (33)</td></tr><tr><td>&#8195;GFP Control</td><td>17% (2)</td><td>83% (10)</td><td>40% (27)</td><td>60% (40)</td></tr></tbody></table><table-wrap-foot><fn><p>Cells expressing GFP-PGBD5 and intact transposons exhibit significantly higher frequency of genomic integration as compared to either GFP control, or GFP-PGBD5 with mutant transposons, with 82% (65 out of 79) of sequences demonstrating DNA transposition of ITR transposons into TTAA sites (&#8224;p = 1.8 &#215; 10-5). Mutation of the transposon ITR significantly reduces ITR-mediated integration, with only 11% (4 out of 37) of sequences (&#8225;p = 0.0016). Numbers in parentheses denote absolute numbers of identified insertion sites.</p></fn><fn><p>GFP, green fluorescent protein; ITR, inverted terminal repeat.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-10860-v1.xml">elife-10860-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tblu1" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">1</th><th>Sense</th><th>5&#8217;-GCUUGAAUCUCCCAAGAUATT-3&#8217;</th></tr><tr><th>Antisense</th><th>5&#8217;-UAUCUU GGGAGAUUCAAGCTT-3&#8217;</th></tr></thead><tbody><tr><td rowspan="2">2</td><td>Sense</td><td>5&#8217;-CAAAGUAUCUUGGAUCAAATT-3&#8217;</td></tr><tr><td>Antisense</td><td>5&#8217;-UUUGAUCCAAGAUACUUUGCC-3&#8217;</td></tr><tr><td rowspan="2">3</td><td>Sense</td><td>5&#8217;-GAGGAGUU UCGGGAGUUCAACAAAU-3&#8217;</td></tr><tr><td>Antisense</td><td>5&#8217;-AUUUGUUGAACUCCCGAA ACUCCUC-3&#8217;</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-11290-v3.xml">elife-11290-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.11290.008</object-id><label>Table 1.</label><caption><p>Optomotry-measured contrast sensitivity improvements after MD</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.11290.008">http://dx.doi.org/10.7554/eLife.11290.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="4"><bold>Contrast sensitivity</bold></th> </tr><tr><th colspan="4">Day 0</th></tr></thead><tbody><tr><td>Spatial frequency (cyc/deg)</td><td>Control</td><td>GM6001</td><td>Control+MD</td><td>GM6001+MD</td></tr><tr><td>0.031</td><td>3.6 &#177; 0.04</td><td>3.5 &#177; 0.02</td><td>3.5 &#177; 0.01</td><td>3.5 &#177; 0.01</td></tr><tr><td>0.064</td><td>12.7 &#177; 0.52</td><td>11.8 &#177; 0.16</td><td>11.6 &#177; 0.23</td><td>12.0 &#177; 0.15</td></tr><tr><td>0.092</td><td>11.7 &#177; 0.51</td><td>10.9 &#177; 0.18</td><td>10.7 &#177; 0.22</td><td>11.3 &#177; 0.17</td></tr><tr><td>0.103</td><td>10.3 &#177; 0.54</td><td>10.2 &#177; 0.10</td><td>10.0 &#177; 0.16</td><td>10.6 &#177; 0.12</td></tr><tr><td>0.192</td><td>6.9 &#177; 0.07</td><td>6.9 &#177; 0.12</td><td>7.0 &#177; 0.11</td><td>6.8 &#177; 0.05</td></tr><tr><td>0.272</td><td>3.5 &#177; 0.04</td><td>3.5 &#177; 0.02</td><td>3.5 &#177; 0.01</td><td>3.5 &#177; 0.01</td></tr><tr><td/><td colspan="4">Day 7</td></tr><tr><td>0.031</td><td>3.6 &#177; 0.04</td><td>3.5 &#177; 0.02</td><td>5.1 &#177; 0.17</td><td>3.6 &#177; 0.02</td></tr><tr><td>0.064</td><td>12.7 &#177; 0.47</td><td>11.8 &#177; 0.18</td><td>26.9 &#177; 2.35</td><td>12.6 &#177; 0.29</td></tr><tr><td>0.092</td><td>11.7 &#177; 0.44</td><td>11.0 &#177; 0.15</td><td>23.3 &#177; 2.17</td><td>12.0 &#177; 0.22</td></tr><tr><td>0.103</td><td>10.5 &#177; 0.49</td><td>10.3 &#177; 0.10</td><td>21.8 &#177; 1.83</td><td>11.3 &#177; 0.17</td></tr><tr><td>0.192</td><td>7.0 &#177; 0.08</td><td>7.0 &#177; 0.10</td><td>13.5 &#177; 1.08</td><td>7.5 &#177; 0.21</td></tr><tr><td>0.272</td><td>3.5 &#177; 0.04</td><td>3.5 &#177; 0.01</td><td>4.9 &#177; 0.17</td><td>3.8 &#177; 0.25</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-11290-v3.xml">elife-11290-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.11290.014</object-id><label>Table 2.</label><caption><p>Optomotry-measured contrast sensitivity improvements after MD</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.11290.014">http://dx.doi.org/10.7554/eLife.11290.014</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="4">Contrast sensitivity</th></tr><tr><th colspan="4">Day 0</th></tr></thead><tbody><tr><td>Spatial frequency (cyc/deg)</td><td>PT+vehicle</td><td>PT+GM6001</td><td>PT+MD<break/>vehicle</td><td>PT+MD<break/>GM6001</td></tr><tr><td>0.031</td><td>3.5 &#177; 0.02</td><td>3.5 &#177; 0.04</td><td>3.5 &#177; 0.02</td><td>3.5 &#177; 0.02</td></tr><tr><td>0.064</td><td>11.4 &#177; 0.35</td><td>11.4 &#177; 0.32</td><td>11.8 &#177; 0.57</td><td>11.8 &#177; 0.15</td></tr><tr><td>0.092</td><td>10.5 &#177; 0.36</td><td>10.5 &#177; 0.34</td><td>10.7 &#177; 0.39</td><td>11.0 &#177; 0.16</td></tr><tr><td>0.103</td><td>9.8 &#177; 0.26</td><td>9.9 &#177; 0.28</td><td>10.0 &#177; 0.37</td><td>10.0 &#177; 0.32</td></tr><tr><td>0.192</td><td>6.9 &#177; 0.12</td><td>6.7 &#177; 0.09</td><td>6.7 &#177; 0.13</td><td>6.5 &#177; 0.26</td></tr><tr><td>0.272</td><td>3.5 &#177; 0.03</td><td>3.5 &#177; 0.04</td><td>3.4 &#177; 0.01</td><td>3.5 &#177; 0.02</td></tr><tr><td/><td colspan="4">Day 7</td></tr><tr><td>0.031</td><td>3.5 &#177; 0.02</td><td>3.5 &#177; 0.04</td><td>3.5 &#177; 0.01</td><td>5.0 &#177; 0.11</td></tr><tr><td>0.064</td><td>11.4 &#177; 0.39</td><td>11.5 &#177; 0.32</td><td>12.1 &#177; 0.42</td><td>25.2 &#177; 1.53</td></tr><tr><td>0.092</td><td>10.5 &#177; 0.36</td><td>10.5 &#177; 0.34</td><td>11.4 &#177; 0.28</td><td>21.9. &#177; 1.14</td></tr><tr><td>0.103</td><td>9.9 &#177; 0.24</td><td>9.9 &#177; 0.28</td><td>10.6 &#177; 0.26</td><td>19.7 &#177; 1.25</td></tr><tr><td>0.192</td><td>6.9 &#177; 0.11</td><td>6.8 &#177; 0.07</td><td>7.0 &#177; 0.02</td><td>12.8 &#177; 0.91</td></tr><tr><td>0.272</td><td>3.5 &#177; 0.02</td><td>3.5 &#177; 0.04</td><td>3.5 &#177; 0.02</td><td>4.9 &#177; 0.12</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-11466-v4.xml">elife-11466-v4.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tblu1" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th rowspan="2">Analysis of CRM1 <break/>binders (all species)</th><th colspan="3">Analysis of nucleocytoplasmic partitioning</th></tr><tr><th>Replicate 1</th><th>Replicate 2</th><th>Replicate 3</th></tr></thead><tbody><tr><td>Sample preparation</td><td>SDS-PAGE, digestion with <break/>trypsin</td><td>SDS-PAGE, digestion with <break/>trypsin</td><td>SDS-PAGE, digestion with <break/>trypsin</td><td>Digestion with Lys-C and trypsin, <break/>reverse phase HPLC at pH 10</td></tr><tr><td>LC-MS instrumentation</td><td>Dionex Ultimate <break/>3000 HPLC <break/>Q-Exactive HF</td><td>EASY nLC-1000 <break/>Q-Exactive</td><td>Dionex Ultimate <break/>3000 HPLC <break/>Q-Exactive HF</td><td>Dionex Ultimate <break/>3000 HPLC <break/>Orbitrap Fusion</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-11479-v2.xml">elife-11479-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.11479.005</object-id><label>Table 1.</label><caption><p>Dissociation constants <italic>K</italic><sub>D</sub> of mant-nucleotides for mGBP2 WT and C586S mutant determined by fluorescence titrations and GTPase activity parameters obtained by protein concentration-dependent hydrolysis.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.11479.005">http://dx.doi.org/10.7554/eLife.11479.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"/><th colspan="4">Nucleotide binding</th><th colspan="3" rowspan="2">GTP-hydrolysis</th></tr><tr><th>mant-GTP&#947;S</th><th>mant-GDP</th><th colspan="2">mant-GMP</th></tr><tr><th><italic>K</italic><sub>D</sub> (&#181;M)</th><th><italic>K</italic><sub>D</sub> (&#181;M)</th><th colspan="2"><italic>K</italic><sub>D</sub> (&#181;M)</th><th><italic>K<sub>max</sub></italic> (min<sup>-1</sup>)</th><th>Dimer <italic>K</italic><sub>D</sub> (&#181;M)</th><th>GMP (%)</th></tr></thead><tbody><tr><td>WT</td><td>0.45</td><td>0.54</td><td colspan="2">14.4</td><td>102</td><td>0.029</td><td>74</td></tr><tr><td>C586S</td><td>0.50</td><td>0.45</td><td colspan="2">15.5</td><td>133</td><td>0.026</td><td>72</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>The&#160;% GMP indicates the relative amount of the two products, GDP and GMP</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-11571-v2.xml">elife-11571-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.11571.009</object-id><label>Table 1.</label><caption><p>List of the streotaxic coordinates for the functional networks nodes in rat brain.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.11571.009">http://dx.doi.org/10.7554/eLife.11571.009</ext-link></p></caption> <table frame="hsides" rules="groups"><thead> <tr> <th rowspan="2"/><th rowspan="2">Abbreviation</th> <th colspan="3">Paxinos atlas (mm)</th></tr> <tr><th>ML</th> <th>DV</th> <th>AP</th></tr> </thead><tbody> <tr><td>IC1</td> <td/> <td/> <td/> <td/></tr> <tr><td>Left Hippocampus</td> <td>Hp_L</td> <td>-2.6</td> <td>3.2</td> <td>-3.2</td></tr> <tr><td>Right Hippocampus</td> <td>Hp_R</td> <td>2.6</td> <td>3.2</td> <td>-3.2</td></tr> <tr><td>Medial Prefrontal Cotex</td> <td>MedF</td> <td>0.0</td> <td>3.8</td> <td>4</td></tr> <tr><td>IC5</td> <td/> <td/> <td/> <td/></tr> <tr><td>Retrosplenial Cortex</td> <td>Rsp</td> <td>0.0</td> <td>1.8</td> <td>-6.4</td></tr> <tr><td>IC8</td> <td/> <td/> <td/> <td/></tr> <tr><td>Left motor cortex</td> <td>Mot_L</td> <td>-2.2</td> <td>2.2</td> <td>2.2</td></tr> <tr><td>Right motor cortex</td> <td>Mot_R</td> <td>2.2</td> <td>2.2</td> <td>2.2</td></tr> <tr><td>IC9</td> <td/> <td/> <td/> <td/></tr> <tr><td>Left somatosensory cortex</td> <td>SS_L</td> <td>-5.0</td> <td>3.8</td> <td>0.6</td></tr> <tr><td>Right somatosensory cortex</td> <td>SS_R</td> <td>5.0</td> <td>3.8</td> <td>0.6</td></tr> </tbody></table><table-wrap-foot><fn id="tblfn1"><p>ML: medial-lateral; AP: anterior-posterior; DV: dorsal-ventral; IC: independent component.</p></fn></table-wrap-foot></table-wrap> <br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-11614-v2.xml">elife-11614-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.11614.006</object-id><label>Table 2.</label><caption><p>Cardiovascular abnormalities in E14.5 embryos. Table shows the numbers of different cardiac defects found in embryos from the indicated duplication strains and in Wt littermate controls.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.11614.006">http://dx.doi.org/10.7554/eLife.11614.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3" rowspan="2">Types of defects</th><th colspan="2">Dp1Tyb</th><th colspan="2">Dp9Tyb</th><th colspan="2">Dp2Tyb</th><th colspan="2">Dp3Tyb</th><th colspan="2">Ts1Rhr</th><th colspan="2">Dp4Tyb</th><th colspan="2">Dp5Tyb</th><th colspan="2">Dp6Tyb</th></tr><tr><th>Wt</th><th>Dp <break/>1Tyb</th><th>Wt</th><th>Dp <break/>9Tyb</th><th>Wt</th><th>Dp <break/>2Tyb</th><th>Wt</th><th>Dp <break/>3Tyb</th><th>Wt</th><th>Ts <break/>1Rhr</th><th>Wt</th><th>Dp <break/>4Tyb</th><th>Wt</th><th>Dp <break/>5Tyb</th><th>Wt</th><th>Dp <break/>6Tyb</th></tr></thead><tbody><tr><td rowspan="6">Single defects</td><td rowspan="3">Septal defects</td><td>ASD</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td>pVSD</td><td>3</td><td>9</td><td>0</td><td>1</td><td>2</td><td>5</td><td>2</td><td>0</td><td>4</td><td>1</td><td>0</td><td>4</td><td>1</td><td>1</td><td>4</td><td>1</td></tr><tr><td>mVSD</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>2</td><td>0</td><td>4</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2</td><td>1</td></tr><tr><td rowspan="2">OFT defects</td><td>OA</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td>DORV</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td colspan="2">AVSD</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td rowspan="4">Multiple defects</td><td colspan="2">OFT + VSD</td><td>1</td><td>2</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2</td><td>0</td><td>0</td><td>0</td></tr><tr><td colspan="2">pVSD+ mVSD</td><td>1</td><td>3</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>0</td><td>2</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td colspan="2">VSD + AVSD</td><td>1</td><td>3</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>6</td><td>1</td><td>2</td><td>0</td><td>0</td><td>0</td><td>2</td><td>0</td><td>0</td></tr><tr><td colspan="2">OFT + VSD + AVSD</td><td>0</td><td>5</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td colspan="3">Total number of CHD</td><td>7</td><td>24</td><td>1</td><td>1</td><td>2</td><td>9</td><td>3</td><td>11</td><td>5</td><td>5</td><td>0</td><td>4</td><td>3</td><td>3</td><td>6</td><td>2</td></tr><tr><td colspan="3">Embryos analyzed</td><td>26</td><td>39</td><td>22</td><td>17</td><td>16</td><td>26</td><td>26</td><td>25</td><td>20</td><td>21</td><td>15</td><td>19</td><td>11</td><td>20</td><td>28</td><td>19</td></tr><tr><td colspan="3">% of CHD</td><td>26.9</td><td>61.5</td><td>4.5</td><td>5.9</td><td>12.5</td><td>34.6</td><td>12</td><td>44</td><td>25</td><td>24</td><td>0</td><td>21</td><td>27</td><td>15</td><td>21</td><td>11</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-11614-v3.xml">elife-11614-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Cardiovascular abnormalities in E14.5 embryos.</title><p>Table shows the numbers of different cardiac defects found in embryos from the indicated duplication strains and in Wt littermate controls.</p></caption><table frame="hsides" rules="groups"><thead><tr><th/><th/><th/><th/><th/><th/><th/><th/><th/><th/><th/><th/><th/><th/><th/><th/><th/><th/><th/></tr><tr><th colspan="3" rowspan="2">Types of defects</th><th colspan="2">Dp1Tyb</th><th colspan="2">Dp9Tyb</th><th colspan="2">Dp2Tyb</th><th colspan="2">Dp3Tyb</th><th colspan="2">Ts1Rhr</th><th colspan="2">Dp4Tyb</th><th colspan="2">Dp5Tyb</th><th colspan="2">Dp6Tyb</th></tr><tr><th>Wt</th><th>Dp1Tyb</th><th>Wt</th><th>Dp9Tyb</th><th>Wt</th><th>Dp2Tyb</th><th>Wt</th><th>Dp3Tyb</th><th>Wt</th><th>Ts1Rhr</th><th>Wt</th><th>Dp4Tyb</th><th>Wt</th><th>Dp5Tyb</th><th>Wt</th><th>Dp6Tyb</th></tr></thead><tbody><tr><td rowspan="6">Single defects</td><td rowspan="3">Septal defects</td><td>ASD</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td>pVSD</td><td>3</td><td>9</td><td>0</td><td>1</td><td>2</td><td>5</td><td>2</td><td>0</td><td>4</td><td>1</td><td>0</td><td>4</td><td>1</td><td>1</td><td>4</td><td>1</td></tr><tr><td>mVSD</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>2</td><td>0</td><td>4</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2</td><td>1</td></tr><tr><td rowspan="2">OFT defects</td><td>OA</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td>DORV</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td colspan="2">AVSD</td><td>0</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td rowspan="4">Multiple defects</td><td colspan="2">OFT + VSD</td><td>1</td><td>2</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2</td><td>0</td><td>0</td><td>0</td></tr><tr><td colspan="2">pVSD&#160;+ mVSD</td><td>1</td><td>3</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>0</td><td>2</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td colspan="2">VSD + AVSD</td><td>1</td><td>4</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>6</td><td>1</td><td>2</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td></tr><tr><td colspan="2">OFT + VSD + AVSD</td><td>0</td><td>5</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td></tr><tr><td colspan="3">Total number of CHD</td><td>7</td><td>24</td><td>1</td><td>1</td><td>2</td><td>9</td><td>3</td><td>11</td><td>5</td><td>5</td><td>0</td><td>4</td><td>3</td><td>3</td><td>6</td><td>2</td></tr><tr><td colspan="3">Embryos analyzed</td><td>26</td><td>39</td><td>22</td><td>17</td><td>16</td><td>26</td><td>26</td><td>25</td><td>20</td><td>21</td><td>15</td><td>19</td><td>11</td><td>20</td><td>28</td><td>19</td></tr><tr><td colspan="3">% of CHD</td><td>26.9</td><td>61.5</td><td>4.5</td><td>5.9</td><td>12.5</td><td>34.6</td><td>11.5</td><td>44.0</td><td>25.0</td><td>23.8</td><td>0</td><td>21.0</td><td>27.2</td><td>15.0</td><td>21.4</td><td>10.5</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12089-v2.xml">elife-12089-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.12089.006</object-id><label>Table 1.</label><caption><p>Characteristics of disease associated variants at sixteen SLE risk loci.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12089.006">http://dx.doi.org/10.7554/eLife.12089.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Risk locus</th><th rowspan="2">Signal</th><th rowspan="2">Peak SNP</th><th rowspan="2">Minor allele</th><th rowspan="2">Odds ratio (Minor allele)</th><th rowspan="2">Allele Freq. (Cases)</th><th rowspan="2">Allele Freq. (Controls)</th><th rowspan="2">SLE association P-value</th><th rowspan="2">SLE associated <break/>Annotated variants</th><th colspan="3">Variants in LD with peak SNP (D' &gt;0.8)</th></tr><tr><th>Total variants</th><th>Total potentially functional variants</th><th>Total coding variants</th></tr></thead><tbody><tr><td><italic>STAT4</italic></td><td>1</td><td>rs12612769</td><td>C</td><td>1.7</td><td>0.29</td><td>0.19</td><td>5E-10</td><td>52</td><td>49</td><td>9</td><td>0</td></tr><tr><td rowspan="3"><italic>HLA-D</italic></td><td>1</td><td>rs9271593 (<italic>XL9</italic>)</td><td>C</td><td>1.7</td><td>0.55</td><td>0.42</td><td>7E-10</td><td>835</td><td>530</td><td>398</td><td>0</td></tr><tr><td>2</td><td>rs9274678 (<italic>DQB1</italic>)</td><td>G</td><td>2.1</td><td>0.24</td><td>0.13</td><td>6E-09</td><td>736</td><td>216</td><td>69</td><td>0</td></tr><tr><td>3</td><td>rs36101847 (<italic>DRB1</italic>)</td><td>T</td><td>0.5</td><td>0.13</td><td>0.23</td><td>8E-09</td><td>760</td><td>296</td><td>126</td><td>0</td></tr><tr><td><italic>ITGAM-ITGAX</italic></td><td>1</td><td>rs41476751</td><td>C</td><td>1.9</td><td>0.25</td><td>0.15</td><td>8E-09</td><td>153</td><td>121</td><td>62</td><td>3</td></tr><tr><td><italic>IRF5_TNPO3</italic></td><td>1</td><td>rs34350562</td><td>G</td><td>1.8</td><td>0.23</td><td>0.14</td><td>3E-09</td><td>245</td><td>189</td><td>124</td><td>0</td></tr><tr><td><italic>UBE2L3</italic></td><td>1</td><td>rs181366</td><td>T</td><td>1.5</td><td>0.27</td><td>0.20</td><td>2E-07</td><td>82</td><td>79</td><td>55</td><td>1</td></tr><tr><td><italic>BANK1</italic></td><td>1</td><td>rs4699260</td><td>T</td><td>0.7</td><td>0.20</td><td>0.28</td><td>9E-06</td><td>267</td><td>143</td><td>29</td><td>2</td></tr><tr><td><italic>TNIP1</italic></td><td>1</td><td>rs62382335</td><td>A</td><td>1.4</td><td>0.14</td><td>0.10</td><td>6E-05</td><td>46</td><td>22</td><td>16</td><td>0</td></tr><tr><td><italic>TNFAIP3</italic></td><td>1</td><td>rs57087937</td><td>T</td><td>1.9</td><td>0.10</td><td>0.06</td><td>2E-06</td><td>69</td><td>63</td><td>40</td><td>1</td></tr><tr><td><italic>CCL22-CX3CL1</italic></td><td>1</td><td>rs223889</td><td>T</td><td>1.5</td><td>0.34</td><td>0.27</td><td>5E-07</td><td>32</td><td>25</td><td>20</td><td>0</td></tr><tr><td><italic>RAVER1-ZGLP1</italic></td><td>1</td><td>rs35186095</td><td>T</td><td>1.3</td><td>0.21</td><td>0.17</td><td>2E-04</td><td>43</td><td>24</td><td>19</td><td>0</td></tr><tr><td><italic>ICA1</italic></td><td>1</td><td>rs74787882</td><td>A</td><td>0.7</td><td>0.06</td><td>0.09</td><td>2E-03</td><td>34</td><td>10</td><td>6</td><td>0</td></tr><tr><td rowspan="2"><italic>TNFSF4</italic></td><td>1</td><td>rs1819717</td><td>G</td><td>0.7</td><td>0.29</td><td>0.36</td><td>2E-05</td><td rowspan="2">73</td><td>30</td><td>14</td><td>0</td></tr><tr><td>2</td><td>rs4916313</td><td>C</td><td>1.3</td><td>0.39</td><td>0.32</td><td>2E-04</td><td>30</td><td>21</td><td>0</td></tr><tr><td><italic>BLK</italic></td><td>1</td><td>rs7822109</td><td>C</td><td>0.8</td><td>0.46</td><td>0.52</td><td>9E-05</td><td>97</td><td>61</td><td>38</td><td>0</td></tr><tr><td rowspan="2"><italic>XKR6</italic></td><td>1</td><td>rs4840545</td><td>A</td><td>2.0</td><td>0.13</td><td>0.07</td><td>1E-07</td><td rowspan="2">335</td><td>51</td><td>23</td><td>0</td></tr><tr><td>2</td><td>rs7000132</td><td>C</td><td>0.9</td><td>0.42</td><td>0.46</td><td>5E-04</td><td>178</td><td>118</td><td>0</td></tr><tr><td rowspan="2"><italic>NMNAT2-SMG7</italic></td><td>1</td><td>rs41272536</td><td>G</td><td>2.9</td><td>0.11</td><td>0.05</td><td>2E-08</td><td rowspan="2">33</td><td>8</td><td>8</td><td>0</td></tr><tr><td>2</td><td>rs111487113</td><td>A</td><td>0.6</td><td>0.13</td><td>0.18</td><td>5E-04</td><td>17</td><td>5</td><td>0</td></tr><tr><td><italic>ETS1</italic></td><td>1</td><td>rs34516251</td><td>A</td><td>0.8</td><td>0.18</td><td>0.21</td><td>7E-03</td><td>18</td><td>10</td><td>6</td><td>0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12115-v1.xml">elife-12115-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.12115.005</object-id><label>Table 1.</label><caption><p>Summary of foot feathering QTL.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12115.005">http://dx.doi.org/10.7554/eLife.12115.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Trait</th><th rowspan="2">LG</th><th rowspan="2">Loc (cM)</th><th rowspan="2">Scaffold</th><th rowspan="2">Position</th><th rowspan="2">LOD</th><th rowspan="2">PVE</th><th colspan="3">Mean &#177; S.D.</th></tr><tr><th>SS</th><th>SF</th><th>FF</th></tr></thead><tbody><tr><td>Proportion tarsus feathered</td><td>11</td><td>117</td><td>79</td><td>9,205,286</td><td>7.69</td><td>28.4</td><td>0.46 &#177; 0.04</td><td>0.58 &#177; 0.03**</td><td>0.80 &#177; 0.04***</td></tr><tr><td>Number of toe feathers</td><td>11</td><td>124</td><td>79</td><td>12,325,977</td><td>6.73</td><td>21.3</td><td>43.6 &#177; 8.9</td><td>67.3 &#177; 6.5*</td><td>105.3 &#177; 8.1***</td></tr><tr><td>Number of toe feathers</td><td>20</td><td>15</td><td>95</td><td>1,451,127</td><td>5.36</td><td>16.5</td><td>45.3 &#177; 8.1</td><td>78.2 &#177; 6.5**</td><td>100.3 &#177; 9.9***</td></tr><tr><td>Toe feather length (mm)</td><td>20</td><td>0</td><td>70</td><td>136,746</td><td>20.89</td><td>52.2</td><td>5.2 &#177; 2.3</td><td>19.9 &#177; 2.0***</td><td>37.3 &#177; 2.5***</td></tr><tr><td>Toe feather length (mm)</td><td>11</td><td>124</td><td>79</td><td>12,325,977</td><td>8.51</td><td>15.8</td><td>11.4 &#177; 3.3</td><td>18.7 &#177; 2.4*</td><td>28.5 &#177; 3.0**</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>LG, linkage group; Loc, genetic location of peak LOD score in centimorgans; S, allele from scaled parent; PVE, percent variance explained; F, allele from feathered parent; LOD, log10&#160;odds ratio. (Welch two sample t-test of means compared to SS homozygote; *p&#8804; 0.05, **p&#8804;0.005, ***p&#8804;0.0005.)</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12115-v1.xml">elife-12115-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.12115.006</object-id><label>Table 2.</label><caption><p>Summary of QTL for numbers of feathers on individual toes.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12115.006">http://dx.doi.org/10.7554/eLife.12115.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Digit</th><th rowspan="2">LG</th><th rowspan="2">Loc (cM)</th><th rowspan="2">Scaffold</th><th rowspan="2">Position</th><th rowspan="2">LOD</th><th rowspan="2">PVE</th><th colspan="3">Mean &#177; S.D.</th></tr><tr><th>SS</th><th>SF</th><th>FF</th></tr></thead><tbody><tr><td>Digit 2, left foot</td><td>11</td><td>148</td><td>79</td><td>11,624,701</td><td>5.20</td><td>20.24</td><td>3.75 &#177; 1.13</td><td>6.95 &#177; 1.12*</td><td>12.19 &#177; 1.16***</td></tr><tr><td>Digit 3, right foot</td><td>20</td><td>0</td><td>70</td><td>136,746</td><td>7.71</td><td>24.1</td><td>9.06 &#177; 1.70</td><td>17.35 &#177; 1.48**</td><td>19.86 &#177; 1.83***</td></tr><tr><td>Digit 3, right foot</td><td>11</td><td>148</td><td>79</td><td>11,624,701</td><td>6.47</td><td>19.5</td><td>9.98 &#177; 1.60</td><td>15.60 &#177; 1.62*</td><td>21.02 &#177; 1.65***</td></tr><tr><td>Digit 3, left foot</td><td>20</td><td>0</td><td>70</td><td>136,746</td><td>10.79</td><td>29.84</td><td>8.66 &#177; 1.59</td><td>17.06 &#177; 1.41**</td><td>21.09 &#177; 1.86***</td></tr><tr><td>Digit 3, left foot</td><td>11</td><td>148</td><td>79</td><td>11,624,701</td><td>9.48</td><td>25.52</td><td>9.37 &#177; 1.52</td><td>16.26 &#177; 1.52**</td><td>21.30 &#177; 1.57***</td></tr><tr><td>Digit 4, right foot</td><td>20</td><td>32</td><td>95</td><td>2,464,788</td><td>7.33</td><td>21.36</td><td>2.44 &#177; 0.90</td><td>7.24 &#177; 0.75**</td><td>10.14 &#177; 1.10***</td></tr><tr><td>Digit 4, right foot</td><td>11</td><td>118</td><td>79</td><td>5,475,474</td><td>6.78</td><td>19.5</td><td>3.17 &#177; 0.89</td><td>6.91 &#177; 0.92*</td><td>9.95 &#177; 1.00***</td></tr><tr><td>Digit 4, left foot</td><td>20</td><td>30</td><td>95</td><td>2,464,788</td><td>9.37</td><td>26.83</td><td>2.44 &#177; 0.79</td><td>6.63 &#177; 0.66**</td><td>9.65 &#177; 0.96***</td></tr><tr><td>Digit 4, left foot</td><td>11</td><td>148</td><td>79</td><td>11,624,701</td><td>7.58</td><td>20.83</td><td>3.75 &#177; 1.14</td><td>6.95 &#177; 1.13*</td><td>12.19 &#177; 1.17***</td></tr></tbody></table><table-wrap-foot><fn id="tblfn2"><p>LG, linkage group; Loc, genetic location of peak LOD score in centimorgans; S, allele from scaled parent; PVE, percent variance explained; F, allele from feathered parent; LOD, log10&#160;odds ratio. (Welch two sample t-test of means compared to SS homozygote; *p&#8804;0.05, **p&#8804;0.005, ***p&#8804;0.0005.)</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12192-v2.xml">elife-12192-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.12192.012</object-id><label>Table 2.</label><caption><p>Pattern of changes of mind for each subject. Total trials performed with percentage of trials for different types of changes of mind. The average additional points earned is the difference in the points earned on change of minds trials compared to those that would have been earned had the subject not changed their mind, divided by the total number of change of mind trials.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12192.012">http://dx.doi.org/10.7554/eLife.12192.012</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Subject</th><th rowspan="2">Total trials</th><th colspan="4">%Trials</th><th rowspan="2">Average additional points earned per trial with a change</th></tr><tr><th>&#916;confidence only</th><th>&#916;decision only</th><th>&#916;confidence &amp; &#916;decision</th><th>All changes</th></tr></thead><tbody><tr><td>1</td><td>9022</td><td>4.97</td><td>2.57</td><td>1.30</td><td>8.83</td><td>1.12</td></tr><tr><td>2</td><td>9023</td><td>2.78</td><td>1.31</td><td>0.40</td><td>4.49</td><td>0.90</td></tr><tr><td>3</td><td>9018</td><td>1.38</td><td>0.49</td><td>0.17</td><td>2.03</td><td>0.72</td></tr><tr><td>4</td><td>5000</td><td>4.26</td><td>2.40</td><td>1.62</td><td>8.28</td><td>0.90</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12215-v1.xml">elife-12215-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.12215.010</object-id><label>Table 1.</label><caption><p>Maximum likelihood parameters of the model (see Materials&#160;and&#160;methods for details) with the best BIC score (see Table 2).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12215.010">http://dx.doi.org/10.7554/eLife.12215.010</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Participant</th><th rowspan="2">Perception noise, &#963;<sub>p</sub></th><th rowspan="2">Prior bias, &#916;</th><th colspan="2">Decision noise</th></tr><tr><th>Stimulus-dependent, <italic>&#946;</italic></th><th>Stimulus-independent, &#954;</th></tr></thead><tbody><tr><td>1</td><td>0.5</td><td>0.58&#176;</td><td>1.4</td><td>0.044</td></tr><tr><td>2</td><td>0.5</td><td>0.61&#176;</td><td>1.9</td><td>0.12</td></tr><tr><td>3</td><td>0.3</td><td>0.54&#176;</td><td>1.5</td><td>0.10</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12215-v1.xml">elife-12215-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.12215.011</object-id><label>Table 2.</label><caption><p>Model comparison results using Bayesian information criterion (BIC, lower is better). Each row is a different model using a different combination of included (+) and excluded (&#8211;) parameters (columns, see Materials&#160;and&#160;methods for details). Last column shows BIC score relative to the BIC of the best model (number 4).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12215.011">http://dx.doi.org/10.7554/eLife.12215.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Model</th><th rowspan="2">Perception noise, &#963;<sub>p</sub></th><th colspan="2">Prior bias</th><th colspan="2">Decision noise</th><th rowspan="2">BIC</th></tr><tr><th>Scale, &#945;</th><th>Offset, &#916;</th><th>Stimulus-dependent, <italic>&#946;</italic></th><th>Stimulus-independent, &#954;</th></tr></thead><tbody><tr><td>1</td><td>+</td><td>&#8211;</td><td>&#8211;</td><td>+</td><td>&#8211;</td><td>160</td></tr><tr><td>2</td><td>+</td><td>&#8211;</td><td>&#8211;</td><td>+</td><td>+</td><td>139</td></tr><tr><td>3</td><td>+</td><td>&#8211;</td><td>+</td><td>+</td><td>&#8211;</td><td>58</td></tr><tr><td>4</td><td>+</td><td>&#8211;</td><td>+</td><td>+</td><td>+</td><td>0</td></tr><tr><td>5</td><td>+</td><td>+</td><td>&#8211;</td><td>+</td><td>&#8211;</td><td>105</td></tr><tr><td>6</td><td>+</td><td>+</td><td>&#8211;</td><td>+</td><td>+</td><td>102</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12215-v2.xml">elife-12215-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.12215.010</object-id><label>Table 1.</label><caption><p>Maximum likelihood parameters of the model (see Materials&#160;and&#160;methods for details) with the best BIC score (see Table 2).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12215.010">http://dx.doi.org/10.7554/eLife.12215.010</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Participant</th><th rowspan="2">Perception noise, &#963;<sub>p</sub></th><th rowspan="2">Prior bias, &#916;</th><th colspan="2">Decision noise</th></tr><tr><th>Stimulus-dependent, <italic>&#946;</italic></th><th>Stimulus-independent, &#954;</th></tr></thead><tbody><tr><td>1</td><td>0.5</td><td>0.58&#176;</td><td>1.4</td><td>0.044</td></tr><tr><td>2</td><td>0.5</td><td>0.61&#176;</td><td>1.9</td><td>0.12</td></tr><tr><td>3</td><td>0.3</td><td>0.54&#176;</td><td>1.5</td><td>0.10</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12215-v2.xml">elife-12215-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.12215.011</object-id><label>Table 2.</label><caption><p>Model comparison results using Bayesian information criterion (BIC, lower is better). Each row is a different model using a different combination of included (+) and excluded (&#8211;) parameters (columns, see Materials&#160;and&#160;methods for details). Last column shows BIC score relative to the BIC of the best model (number 4).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12215.011">http://dx.doi.org/10.7554/eLife.12215.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Model</th><th rowspan="2">Perception noise, &#963;<sub>p</sub></th><th colspan="2">Prior bias</th><th colspan="2">Decision noise</th><th rowspan="2">BIC</th></tr><tr><th>Scale, &#945;</th><th>Offset, &#916;</th><th>Stimulus-dependent, <italic>&#946;</italic></th><th>Stimulus-independent, &#954;</th></tr></thead><tbody><tr><td>1</td><td>+</td><td>&#8211;</td><td>&#8211;</td><td>+</td><td>&#8211;</td><td>160</td></tr><tr><td>2</td><td>+</td><td>&#8211;</td><td>&#8211;</td><td>+</td><td>+</td><td>139</td></tr><tr><td>3</td><td>+</td><td>&#8211;</td><td>+</td><td>+</td><td>&#8211;</td><td>58</td></tr><tr><td>4</td><td>+</td><td>&#8211;</td><td>+</td><td>+</td><td>+</td><td>0</td></tr><tr><td>5</td><td>+</td><td>+</td><td>&#8211;</td><td>+</td><td>&#8211;</td><td>105</td></tr><tr><td>6</td><td>+</td><td>+</td><td>&#8211;</td><td>+</td><td>+</td><td>102</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12215-v3.xml">elife-12215-v3.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.12215.010</object-id><label>Table 1.</label><caption><p>Maximum likelihood parameters of the model (see Materials&#160;and&#160;methods for details) with the best BIC score (see Table 2).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12215.010">http://dx.doi.org/10.7554/eLife.12215.010</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Participant</th><th rowspan="2">Perception noise, &#963;<sub>p</sub></th><th rowspan="2">Prior bias, &#916;</th><th colspan="2">Decision noise</th></tr><tr><th>Stimulus-dependent, &#946;</th><th>Stimulus-independent, &#954;</th></tr></thead><tbody><tr><td>1</td><td>0.5</td><td>0.58&#176;</td><td>1.4</td><td>0.044</td></tr><tr><td>2</td><td>0.5</td><td>0.61&#176;</td><td>1.9</td><td>0.12</td></tr><tr><td>3</td><td>0.3</td><td>0.54&#176;</td><td>1.5</td><td>0.10</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12215-v3.xml">elife-12215-v3.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.12215.011</object-id><label>Table 2.</label><caption><p>Model comparison results using Bayesian information criterion (BIC, lower is better). Each row is a different model using a different combination of included (+) and excluded (&#8211;) parameters (columns, see Materials&#160;and&#160;methods for details). Last column shows BIC score relative to the BIC of the best model (number 4).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12215.011">http://dx.doi.org/10.7554/eLife.12215.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Model</th><th rowspan="2">Perception noise, &#963;<sub>p</sub></th><th colspan="2">Prior bias</th><th colspan="2">Decision noise</th><th rowspan="2">BIC</th></tr><tr><th>Scale, &#945;</th><th>Offset, &#916;</th><th>Stimulus-dependent, &#946;</th><th>Stimulus-independent, &#954;</th></tr></thead><tbody><tr><td>1</td><td>+</td><td>&#8211;</td><td>&#8211;</td><td>+</td><td>&#8211;</td><td>160</td></tr><tr><td>2</td><td>+</td><td>&#8211;</td><td>&#8211;</td><td>+</td><td>+</td><td>139</td></tr><tr><td>3</td><td>+</td><td>&#8211;</td><td>+</td><td>+</td><td>&#8211;</td><td>58</td></tr><tr><td>4</td><td>+</td><td>&#8211;</td><td>+</td><td>+</td><td>+</td><td>0</td></tr><tr><td>5</td><td>+</td><td>+</td><td>&#8211;</td><td>+</td><td>&#8211;</td><td>105</td></tr><tr><td>6</td><td>+</td><td>+</td><td>&#8211;</td><td>+</td><td>+</td><td>102</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12572-v2.xml">elife-12572-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.12572.014</object-id><label>Table 1.</label><caption><p>Maximum likelihood fits of transition rates in wild type <italic>C. elegans</italic>. Each cohort was fitted separately; values are expressed as mean &#177; sem (<italic>n</italic> = 5 cohorts). Data from wild type cohorts were obtained on the same days as the experimental cohorts for which they served as controls (<xref ref-type="table" rid="tbl3">Tables 3</xref> and <xref ref-type="table" rid="tbl4">4</xref>), but experimental cohorts in this study were separated by weeks to months. All transition rates were constrained to be &#8805;0. Transition rates that were calculated using the synaptic constraints (<xref ref-type="disp-formula" rid="equ36">Equation 35</xref>) are shaded orange; other constrained values are shaded grey. Mean dwell times and state probabilities were calculated from the transition rates. Column A shows fits using the standard model, which has 8 rate constants with two synaptic constraints, resulting in 6 free parameters that determine the 6 synaptic weights (<xref ref-type="fig" rid="fig2">Figure 2C,D</xref>; Materials and methods <xref ref-type="disp-formula" rid="equ32 equ33 equ34 equ35 equ36">Equations 31&#8211;35</xref>). Column B shows fits to a model that has only one pause state (X); this model was derived from the standard model by imposing two more constraints: <inline-formula><mml:math id="inf87"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>F</mml:mi><mml:mi>Y</mml:mi></mml:mrow></mml:msub><mml:mo>=</mml:mo><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>R</mml:mi><mml:mi>Y</mml:mi></mml:mrow></mml:msub><mml:mo>&#8773;</mml:mo><mml:mn>0</mml:mn></mml:mrow></mml:math></inline-formula>, yielding 4 free parameters. To allow comparison of models A and B by the likelihood ratio test, which requires that model B be a special case of model A, <inline-formula><mml:math id="inf88"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>R</mml:mi><mml:mi>Y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> and <inline-formula><mml:math id="inf89"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>F</mml:mi><mml:mi>Y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> were set slightly &gt;0 (10<sup>-10</sup> s<sup>-1</sup>), thereby avoiding infinite values for <inline-formula><mml:math id="inf90"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>Y</mml:mi><mml:mi>F</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> and <inline-formula><mml:math id="inf91"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>Y</mml:mi><mml:mi>R</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> when applying the synaptic constraints, while maintaining a vanishingly small probability of being in state Y (<inline-formula><mml:math id="inf92"><mml:msub><mml:mi>p</mml:mi><mml:mtext>Y</mml:mtext></mml:msub><mml:mo>&lt;</mml:mo></mml:math></inline-formula>&#160;10<sup>-18</sup>). The log<sub>e</sub> likelihood (summed over the 5 cohorts) for model B was 1854 less than for model A, with 30 degrees of freedom for model A (6 per cohort &#215; 5 cohorts) and 20 degrees of freedom for model B (4 per cohort &#215; 5 cohorts). Applying the likelihood ratio test, the difference was highly significant (<italic>p</italic>&lt;10<sup>-100</sup>; <italic>p</italic>=Chi-squared(2<italic>L, df</italic>), where <italic>L&#160;</italic> =&#160;1854 and <italic>df&#160;</italic> =&#160;30&#8211;20 =10. Model C is the most general 3-state (F, R, P) model, which allows all six transitions between the three states. The fitted transition rates for model C were nearly identical to model B. Likelihood values are relative to model A.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12572.014">http://dx.doi.org/10.7554/eLife.12572.014</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2"/><th align="center"><p>A</p></th><th align="center"><p>B</p></th><th align="center"><p>C</p></th></tr><tr><th align="center"><p>2 pause states6 free parameters</p></th><th align="center"><p>1 pause state4 free parameters</p></th><th align="center"><p>1 pause state6 free parameters</p></th></tr><tr><th align="center"><p>&#916; log<sub>e</sub> likelihood</p></th><th align="center"><p>0</p></th><th align="center"><p>-1854</p></th><th align="center"><p>-1836</p></th></tr><tr><th align="center"><p>Degrees of freedom</p></th><th align="center"><p>30</p></th><th align="center"><p>20</p></th><th align="center"><p>30</p></th></tr><tr><th align="center"/><th align="center"><p>mean &#177; sem (<italic>n&#160;</italic> =&#160;5)</p></th><th align="center"><p>mean &#177; sem (<italic>n&#160;</italic> =&#160;5)</p></th><th align="center"><p>mean &#177; sem (<italic>n&#160;</italic> =&#160;5)</p></th></tr></thead><tbody><tr><td align="center"><p><inline-formula><mml:math id="inf93"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mtext>XR</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> (s<sup>-1</sup>)</p></td><td align="center"><p>1.201 &#177; 0.099</p></td><td align="center"><p>1.019 &#177; 0.085</p></td><td align="center"><p>1.008 &#177; 0.090</p></td></tr><tr><td align="center"><p><inline-formula><mml:math id="inf95"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mtext>XF</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> (s<sup>-1</sup>)</p></td><td align="center"><p>1.115 &#177; 0.087</p></td><td align="center"><p>1.915 &#177; 0.152</p></td><td align="center"><p>1.914 &#177; 0.152</p></td></tr><tr><td align="center"><p><inline-formula><mml:math id="inf97"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mtext>RX</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> (s<sup>-1</sup>)</p></td><td align="center"><p>0.025 &#177; 0.008</p></td><td align="center"><p>0.507 &#177; 0.013</p></td><td align="center"><p>0.507 &#177; 0.013</p></td></tr><tr><td align="center"><p><inline-formula><mml:math id="inf99"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mtext>RY</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> (s<sup>-1</sup>)</p></td><td align="center"><p>0.490 &#177; 0.015</p></td><td align="center" style="background: #C0C0C0;"><p>10<sup>-10</sup></p></td><td align="center"/></tr><tr><td align="center"><p><inline-formula><mml:math id="inf101"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mtext>FX</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> (s<sup>-1</sup>)</p></td><td align="center"><p>0.182 &#177; 0.007</p></td><td align="center"><p>0.198 &#177; 0.009</p></td><td align="center"><p>0.196 &#177; 0.008</p></td></tr><tr><td align="center"><p><inline-formula><mml:math id="inf103"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mtext>FY</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> (s<sup>-1</sup>)</p></td><td align="center"><p>0.007 &#177; 0.002</p></td><td align="center" style="background: #C0C0C0;"><p>10<sup>-10</sup></p></td><td align="center"/></tr><tr><td align="center"><p><inline-formula><mml:math id="inf105"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mtext>YR</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> (s<sup>-1</sup>)</p></td><td align="center" style="background: #FFCC99;"><p>0.411 &#177; 0.019</p></td><td align="center" style="background: #FFCC99;"><p>&gt;10<sup>9</sup></p> </td><td align="center"/></tr><tr><td align="center"><p><inline-formula><mml:math id="inf107"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mtext>YF</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> (s<sup>-1</sup>)</p></td><td align="center" style="background: #FFCC99;"><p>4.575 &#177; 0.533</p></td><td align="center" style="background: #FFCC99;"><p>&gt;10<sup>9</sup></p> </td><td align="center"/></tr><tr><td align="center"><p><inline-formula><mml:math id="inf109"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mtext>FR</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> (s<sup>-1</sup>)</p></td><td align="center"/><td align="center"/><td align="center"><p>0.001 &#177; 0.001</p></td></tr><tr><td align="center"><p><inline-formula><mml:math id="inf111"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mtext>RF</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:math></inline-formula> (s<sup>-1</sup>)</p></td><td align="center"/><td align="center"/><td align="center"><p>0.000 &#177; 0.000</p></td></tr><tr><td align="center"><p><inline-formula><mml:math id="inf113"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mtext>F</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula> (s)</p></td><td align="center"><p>5.329 &#177; 0.245</p></td><td align="center"><p>5.096 &#177; 0.235</p></td><td align="center"><p>5.135 &#177; 0.227</p></td></tr><tr><td align="center"><p><inline-formula><mml:math id="inf115"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mtext>R</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula> (s)</p></td><td align="center"><p>1.945 &#177; 0.043</p></td><td align="center"><p>1.975 &#177; 0.049</p></td><td align="center"><p>1.976 &#177; 0.049</p></td></tr><tr><td align="center"><p><inline-formula><mml:math id="inf117"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mtext>X</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula> (s)</p></td><td align="center"><p>0.441 &#177; 0.032</p></td><td align="center"><p>0.349 &#177; 0.026</p></td><td align="center"><p>0.351 &#177; 0.027</p></td></tr><tr><td align="center"><p><inline-formula><mml:math id="inf119"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mtext>Y</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula> (s)</p></td><td align="center"><p>0.208 &#177; 0.019</p></td><td align="center"><p>&lt;10<sup>-9</sup></p></td><td align="center"/></tr><tr><td align="center"><p><inline-formula><mml:math id="inf121"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mtext>F</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula></p></td><td align="center"><p>0.762 &#177; 0.015</p></td><td align="center"><p>0.7641 &#177; 0.015</p></td><td align="center"><p>0.764 &#177; 0.014</p></td></tr><tr><td align="center"><p><inline-formula><mml:math id="inf123"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mtext>R</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula></p></td><td align="center"><p>0.158 &#177; 0.007</p></td><td align="center"><p>0.158 &#177; 0.007</p></td><td align="center"><p>0.155 &#177; 0.007</p></td></tr><tr><td align="center"><p><inline-formula><mml:math id="inf125"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mtext>X</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula></p></td><td align="center"><p>0.063 &#177; 0.006</p></td><td align="center"><p>0.081 &#177; 0.008</p></td><td align="center"><p>0.080 &#177; 0.008</p></td></tr><tr><td align="center"><p><inline-formula><mml:math id="inf127"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mtext>Y</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula></p></td><td align="center"><p>0.017 &#177; 0.002</p></td><td align="center"><p>&lt;10<sup>-18</sup></p></td><td align="center"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12572-v2.xml">elife-12572-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl6" position="float"><object-id pub-id-type="doi">10.7554/eLife.12572.024</object-id><label>Table 6.</label><caption><p>Effects of mutations on model parameters. Significance was determined using the likelihood ratio test as described in <xref ref-type="table" rid="tbl4">Table 4</xref>. The sign of the change (&#916;) caused by the mutation is shown as &#8220;+&#8221; if the value moved away from 0, &#8220;&#8211;&#8221; if the value moved towards 0. All <italic>p</italic>-values are shown without correction for multiple comparisons. Blue denotes significance at p&lt;0.05. Red denotes significance at p&lt;0.05 after Bonferroni correction for 27 comparisons.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12572.024">http://dx.doi.org/10.7554/eLife.12572.024</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"/><th colspan="12">HYP</th><th colspan="8">DEP</th></tr><tr><th colspan="4">HYP A: <italic>eat-4(ad572)</italic></th><th colspan="4">HYP B: <italic>eat-4(ky5)</italic></th><th colspan="4">HYP C: <italic>glr-1(n2461)</italic></th><th colspan="4">DEP A: <italic>glr-1::glr-1(A/T)</italic></th><th colspan="4">DEP B: <italic>nmr-1::glr-1(A/T)</italic></th></tr><tr><th>Control</th><th>Mutant</th><th><inline-formula><mml:math id="inf258"><mml:mo>&#8710;</mml:mo></mml:math></inline-formula></th><th><italic>p</italic> &lt;</th><th>Control</th><th>Mutant</th><th>&#8710;</th><th>p&lt;</th><th>Control</th><th>Mutant</th><th>&#8710;</th><th>p&lt;</th><th>Control</th><th>Mutant</th><th>&#8710;</th><th>p&lt;</th><th>Control</th><th>Mutant</th><th>&#8710;</th><th>p&lt;</th></tr></thead><tbody><tr><td><inline-formula><mml:math id="inf260"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mtext>F</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula> (s)</td><td>4.771</td><td>9.564</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-87</sup></styled-content></td><td>4.956</td><td>8.643</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-64</sup></styled-content></td><td>5.181</td><td>7.871</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-33</sup></styled-content></td><td>4.771</td><td>0.940</td><td>&#8211;</td><td style="color: #FF0000;"><styled-content>10<sup>-99</sup></styled-content></td><td>5.181</td><td>0.742</td><td>&#8211;</td><td style="color: #FF0000;"><styled-content>10<sup>-99</sup></styled-content></td></tr><tr><td><inline-formula><mml:math id="inf262"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mtext>R</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula> (s)</td><td>2.043</td><td>2.821</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-7</sup></styled-content></td><td>1.910</td><td>2.769</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-12</sup></styled-content></td><td>2.018</td><td>3.004</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-16</sup></styled-content></td><td>2.045</td><td>0.875</td><td>&#8211;</td><td style="background: #FFFFFF; color: #FF0000;"><styled-content>10<sup>-99</sup></styled-content></td><td>2.018</td><td>0.709</td><td>&#8211;</td><td style="color: #FF0000;"><styled-content>10<sup>-9</sup></styled-content> <sup>9</sup></td></tr><tr><td><inline-formula><mml:math id="inf264"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mtext>X</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula> (s)</td><td>0.481</td><td>1.040</td><td>+</td><td style="color: #33CCCC;"><styled-content>0.005</styled-content></td><td>0.529</td><td>0.844</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-43</sup></styled-content></td><td>0.459</td><td>0.727</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-39</sup></styled-content></td><td>0.482</td><td>0.328</td><td>&#8211;</td><td style="color: #FF0000;"><styled-content>10<sup>-49</sup></styled-content></td><td>0.460</td><td>0.235</td><td>&#8211;</td><td style="color: #FF0000;"><styled-content>10<sup>-99</sup></styled-content></td></tr><tr><td><inline-formula><mml:math id="inf266"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mtext>Y</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula> (s)</td><td>0.247</td><td>0.382</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-5</sup></styled-content></td><td>0.238</td><td>0.290</td><td>+</td><td style="color: #33CCCC;"><styled-content>0.005</styled-content></td><td>0.221</td><td>0.164</td><td>&#8211;</td><td style="color: #FF0000;"><styled-content>10<sup>-5</sup></styled-content></td><td>0.247</td><td>0.097</td><td>&#8211;</td><td style="color: #FF0000;"><styled-content>10<sup>-93</sup></styled-content></td><td>0.221</td><td>0.079</td><td>&#8211;</td><td style="color: #FF0000;"><styled-content>10<sup>-99</sup></styled-content></td></tr><tr><td><inline-formula><mml:math id="inf268"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mtext>P</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula> (s)</td><td>0.428</td><td>0.982</td><td>+</td><td><styled-content>10<sup>-99</sup></styled-content></td><td>0.466</td><td>0.793</td><td>+</td><td><styled-content>10<sup>-52</sup></styled-content></td><td>0.409</td><td>0.677</td><td>+</td><td><styled-content>10<sup>-43</sup></styled-content></td><td>0.428</td><td>0.286</td><td>&#8211;</td><td><styled-content>10<sup>-61</sup></styled-content></td><td>0.409</td><td>0.204</td><td>&#8211;</td><td><styled-content>10<sup>-99</sup></styled-content></td></tr><tr><td><inline-formula><mml:math id="inf270"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mtext>F</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula></td><td>0.729</td><td>0.839</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-27</sup></styled-content></td><td>0.734</td><td>0.832</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-26</sup></styled-content></td><td>0.755</td><td>0.785</td><td>+</td><td style="color: #33CCCC;"><styled-content>0.003</styled-content></td><td>0.728</td><td>0.410</td><td>&#8211;</td><td style="color: #FF0000;"><styled-content>10<sup>-99</sup></styled-content></td><td>0.755</td><td>0.407</td><td>&#8211;</td><td style="color: #FF0000;"><styled-content>10<sup>-99</sup></styled-content></td></tr><tr><td><inline-formula><mml:math id="inf272"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mtext>R</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula></td><td>0.177</td><td>0.062</td><td>&#8211;</td><td style="color: #FF0000;"><styled-content>10<sup>-37</sup></styled-content></td><td>0.167</td><td>0.079</td><td>&#8211;</td><td style="color: #FF0000;"><styled-content>10<sup>-29</sup></styled-content></td><td>0.161</td><td>0.135</td><td>&#8211;</td><td style="color: #FF0000;"><styled-content>10<sup>-4</sup></styled-content></td><td>0.177</td><td>0.389</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-99</sup></styled-content></td><td>0.161</td><td>0.404</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-99</sup></styled-content></td></tr><tr><td><inline-formula><mml:math id="inf274"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mtext>X</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula></td><td>0.073</td><td>0.090</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-5</sup></styled-content></td><td>0.077</td><td>0.081</td><td>+</td><td>0.3</td><td>0.067</td><td>0.073</td><td>+</td><td style="color: #33CCCC;"><styled-content>0.03</styled-content></td><td>0.073</td><td>0.164</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-99</sup></styled-content></td><td>0.067</td><td>0.151</td><td>+</td><td style="color: #FF0000;"><styled-content>10<sup>-99</sup></styled-content></td></tr><tr><td><inline-formula><mml:math id="inf276"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mtext>Y</mml:mtext></mml:msub></mml:mrow></mml:math></inline-formula></td><td>0.022</td><td>0.009</td><td>&#8211;</td><td><styled-content>10<sup>-13</sup></styled-content></td><td>0.021</td><td>0.008</td><td>&#8211;</td><td><styled-content>10<sup>-25</sup></styled-content></td><td>0.018</td><td>0.007</td><td>&#8211;</td><td><styled-content>10<sup>-27</sup></styled-content></td><td>0.022</td><td>0.037</td><td>+</td><td><styled-content>10<sup>-21</sup></styled-content></td><td>0.018</td><td>0.037</td><td>+</td><td><styled-content>10<sup>-41</sup></styled-content></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12572-v3.xml">elife-12572-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.12572.014</object-id><label>Table 1.</label><caption><title>Maximum likelihood fits of transition rates in wild type <italic>C. elegans</italic>.</title><p>Each cohort was fitted separately; values are expressed as mean &#177; sem (<italic>n</italic> = 5 cohorts). Data from wild type cohorts were obtained on the same days as the experimental cohorts for which they served as controls (<xref ref-type="table" rid="table3">Tables 3</xref> and <xref ref-type="table" rid="table4">4</xref>), but experimental cohorts in this study were separated by weeks to months. All transition rates were constrained to be &#8805;0. Transition rates that were calculated using the synaptic constraints (<xref ref-type="disp-formula" rid="equ36">Equation 35</xref>) are shaded orange; other constrained values are shaded grey. Mean dwell times and state probabilities were calculated from the transition rates. Column A shows fits using the standard model, which has 8 rate constants with two synaptic constraints, resulting in 6 free parameters that determine the 6 synaptic weights (<xref ref-type="fig" rid="fig2">Figure 2C,D</xref>; Materials and methods <xref ref-type="disp-formula" rid="equ32 equ33 equ34 equ35 equ36">Equations 31&#8211;35</xref>). Column B shows fits to a model that has only one pause state (X); this model was derived from the standard model by imposing two more constraints: <inline-formula><mml:math id="inf87"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>F</mml:mi><mml:mi>Y</mml:mi></mml:mrow></mml:msub><mml:mo>=</mml:mo><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>R</mml:mi><mml:mi>Y</mml:mi></mml:mrow></mml:msub><mml:mo>&#8773;</mml:mo><mml:mn>0</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula>, yielding 4 free parameters. To allow comparison of models A and B by the likelihood ratio test, which requires tht model B be a special case of model A, <inline-formula><mml:math id="inf88"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>R</mml:mi><mml:mi>Y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> and <inline-formula><mml:math id="inf89"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>F</mml:mi><mml:mi>Y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> were set slightly &gt;0 (10<sup>-10</sup> s<sup>-1</sup>), thereby avoiding infinite values for <inline-formula><mml:math id="inf90"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>Y</mml:mi><mml:mi>F</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> and <inline-formula><mml:math id="inf91"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>Y</mml:mi><mml:mi>R</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> when applying the synaptic constraints, while maintaining a vanishingly small probability of being in state Y (<inline-formula><mml:math id="inf92"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mrow><mml:mi>Y</mml:mi></mml:mrow></mml:msub><mml:mo>&lt;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula>&#160;10<sup>-18</sup>). The log<sub>e</sub> likelihood (summed over the 5 cohorts) for model B was 1854 less than for model A, with 30 degrees of freedom for model A (6 per cohort &#215; 5 cohorts) and 20 degrees of freedom for model B (4 per cohort &#215; 5 cohorts). Applying the likelihood ratio test, the difference was highly significant (<italic>p</italic>&lt;10<sup>-100</sup>; <italic>p</italic>=Chi-squared(2<italic>L, df</italic>), where <italic>L&#160;</italic> =&#160;1854 and <italic>df&#160;</italic> =&#160;30&#8211;20 =10. Model C is the most general 3-state (F, R, P) model, which allows all six transitions between the three states. The fitted transition rates for model C were nearly identical to model B. Likelihood values are relative to model A.</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2"/><th align="center">A</th><th align="center">B</th><th align="center">C</th></tr><tr><th align="center">2 pause states6 free parameters</th><th align="center">1 pause state4 free parameters</th><th align="center">1 pause state6 free parameters</th></tr><tr><th align="center">&#916; log<sub>e</sub> likelihood</th><th align="center">0</th><th align="center">-1854</th><th align="center">-1836</th></tr><tr><th align="center">Degrees of freedom</th><th align="center">30</th><th align="center">20</th><th align="center">30</th></tr><tr><th align="center"/><th align="center">mean &#177; sem (<italic>n&#160;</italic> =&#160;5)</th><th align="center">mean &#177; sem (<italic>n&#160;</italic> =&#160;5)</th><th align="center">mean &#177; sem (<italic>n&#160;</italic> =&#160;5)</th></tr></thead><tbody><tr><td align="center"><inline-formula><mml:math id="inf93"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>X</mml:mi><mml:mi>R</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s<sup>-1</sup>)</td><td align="center">1.201 &#177; 0.099</td><td align="center">1.019 &#177; 0.085</td><td align="center">1.008 &#177; 0.090</td></tr><tr><td align="center"><inline-formula><mml:math id="inf94"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>X</mml:mi><mml:mi>F</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s<sup>-1</sup>)</td><td align="center">1.115 &#177; 0.087</td><td align="center">1.915 &#177; 0.152</td><td align="center">1.914 &#177; 0.152</td></tr><tr><td align="center"><inline-formula><mml:math id="inf95"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>R</mml:mi><mml:mi>X</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s<sup>-1</sup>)</td><td align="center">0.025 &#177; 0.008</td><td align="center">0.507 &#177; 0.013</td><td align="center">0.507 &#177; 0.013</td></tr><tr><td align="center"><inline-formula><mml:math id="inf96"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>R</mml:mi><mml:mi>Y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s<sup>-1</sup>)</td><td align="center">0.490 &#177; 0.015</td><td align="center" style="author-callout-style-b8">10<sup>-10</sup></td><td align="center"/></tr><tr><td align="center"><inline-formula><mml:math id="inf97"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>F</mml:mi><mml:mi>X</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s<sup>-1</sup>)</td><td align="center">0.182 &#177; 0.007</td><td align="center">0.198 &#177; 0.009</td><td align="center">0.196 &#177; 0.008</td></tr><tr><td align="center"><inline-formula><mml:math id="inf98"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>F</mml:mi><mml:mi>Y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s<sup>-1</sup>)</td><td align="center">0.007 &#177; 0.002</td><td align="center" style="author-callout-style-b8">10<sup>-10</sup></td><td align="center"/></tr><tr><td align="center"><inline-formula><mml:math id="inf99"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>Y</mml:mi><mml:mi>R</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s<sup>-1</sup>)</td><td align="center" style="author-callout-style-b3">0.411 &#177; 0.019</td><td align="center" style="author-callout-style-b3">&gt;10<sup>9</sup></td><td align="center"/></tr><tr><td align="center"><inline-formula><mml:math id="inf100"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>Y</mml:mi><mml:mi>F</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s<sup>-1</sup>)</td><td align="center" style="author-callout-style-b3">4.575 &#177; 0.533</td><td align="center" style="author-callout-style-b3">&gt;10<sup>9</sup></td><td align="center"/></tr><tr><td align="center"><inline-formula><mml:math id="inf101"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>F</mml:mi><mml:mi>R</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s<sup>-1</sup>)</td><td align="center"/><td align="center"/><td align="center">0.001 &#177; 0.001</td></tr><tr><td align="center"><inline-formula><mml:math id="inf102"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>a</mml:mi><mml:mrow><mml:mi>R</mml:mi><mml:mi>F</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s<sup>-1</sup>)</td><td align="center"/><td align="center"/><td align="center">0.000 &#177; 0.000</td></tr><tr><td align="center"><inline-formula><mml:math id="inf103"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mrow><mml:mi>F</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s)</td><td align="center">5.329 &#177; 0.245</td><td align="center">5.096 &#177; 0.235</td><td align="center">5.135 &#177; 0.227</td></tr><tr><td align="center"><inline-formula><mml:math id="inf104"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mrow><mml:mi>R</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s)</td><td align="center">1.945 &#177; 0.043</td><td align="center">1.975 &#177; 0.049</td><td align="center">1.976 &#177; 0.049</td></tr><tr><td align="center"><inline-formula><mml:math id="inf105"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mrow><mml:mi>X</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s)</td><td align="center">0.441 &#177; 0.032</td><td align="center">0.349 &#177; 0.026</td><td align="center">0.351 &#177; 0.027</td></tr><tr><td align="center"><inline-formula><mml:math id="inf106"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mrow><mml:mi>Y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s)</td><td align="center">0.208 &#177; 0.019</td><td align="center">&lt;10<sup>-9</sup></td><td align="center"/></tr><tr><td align="center"><inline-formula><mml:math id="inf107"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mrow><mml:mi>F</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center">0.762 &#177; 0.015</td><td align="center">0.7641 &#177; 0.015</td><td align="center">0.764 &#177; 0.014</td></tr><tr><td align="center"><inline-formula><mml:math id="inf108"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mrow><mml:mi>R</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center">0.158 &#177; 0.007</td><td align="center">0.158 &#177; 0.007</td><td align="center">0.155 &#177; 0.007</td></tr><tr><td align="center"><inline-formula><mml:math id="inf109"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mrow><mml:mi>X</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center">0.063 &#177; 0.006</td><td align="center">0.081 &#177; 0.008</td><td align="center">0.080 &#177; 0.008</td></tr><tr><td align="center"><inline-formula><mml:math id="inf110"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mrow><mml:mi>Y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center">0.017 &#177; 0.002</td><td align="center">&lt;10<sup>-18</sup></td><td align="center"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12572-v3.xml">elife-12572-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table6" position="float"><object-id pub-id-type="doi">10.7554/eLife.12572.024</object-id><label>Table 6.</label><caption><title>Effects of mutations on model parameters.</title><p>Significance was determined using the likelihood ratio test as described in <xref ref-type="table" rid="table4">Table 4</xref>. The sign of the change (&#916;) caused by the mutation is shown as &#8220;+&#8221; if the value moved away from 0, &#8220;&#8211;&#8221; if the value moved towards 0. All <italic>p</italic>-values are shown without correction for multiple comparisons. Blue denotes significance at p&lt;0.05. Red denotes significance at p&lt;0.05 after Bonferroni correction for 27 comparisons.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"/><th colspan="12">HYP</th><th colspan="8">DEP</th></tr><tr><th colspan="4">HYP A: <italic>eat-4(ad572)</italic></th><th colspan="4">HYP B: <italic>eat-4(ky5)</italic></th><th colspan="4">HYP C: <italic>glr-1(n2461)</italic></th><th colspan="4">DEP A: <italic>glr-1::glr-1(A/T)</italic></th><th colspan="4">DEP B: <italic>nmr-1::glr-1(A/T)</italic></th></tr><tr><th>Control</th><th>Mutant</th><th><inline-formula><mml:math id="inf227"><mml:mo>&#8710;</mml:mo></mml:math></inline-formula></th><th><italic>p</italic> &lt;</th><th>Control</th><th>Mutant</th><th>&#8710;</th><th>p&lt;</th><th>Control</th><th>Mutant</th><th>&#8710;</th><th>p&lt;</th><th>Control</th><th>Mutant</th><th>&#8710;</th><th>p&lt;</th><th>Control</th><th>Mutant</th><th>&#8710;</th><th>p&lt;</th></tr></thead><tbody><tr><td><inline-formula><mml:math id="inf229"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mrow><mml:mi>F</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s)</td><td>4.771</td><td>9.564</td><td>+</td><td><named-content content-type="author-callout-style-a3">10<sup>-87</sup></named-content></td><td>4.956</td><td>8.643</td><td>+</td><td><named-content content-type="author-callout-style-a3">10<sup>-64</sup></named-content></td><td>5.181</td><td>7.871</td><td>+</td><td><named-content content-type="author-callout-style-a3">10<sup>-33</sup></named-content></td><td>4.771</td><td>0.940</td><td>&#8211;</td><td><named-content content-type="author-callout-style-a3">10<sup>-99</sup></named-content></td><td>5.181</td><td>0.742</td><td>&#8211;</td><td><named-content content-type="author-callout-style-a3"><named-content content-type="author-callout-style1">10<sup>-99</sup></named-content></named-content> </td></tr><tr><td><inline-formula><mml:math id="inf230"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mrow><mml:mi>R</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s)</td><td>2.043</td><td>2.821</td><td>+</td><td><named-content content-type="author-callout-style1">10<sup>-7</sup></named-content></td><td>1.910</td><td>2.769</td><td>+</td><td><named-content content-type="author-callout-style-a3">10<sup>-12</sup></named-content></td><td>2.018</td><td>3.004</td><td>+</td><td><named-content content-type="author-callout-style-a3">10<sup>-16</sup></named-content></td><td>2.045</td><td>0.875</td><td>&#8211;</td><td><named-content content-type="author-callout-style-a3">10<sup>-99</sup></named-content></td><td>2.018</td><td>0.709</td><td>&#8211;</td><td><named-content content-type="author-callout-style1">10<sup>-9</sup></named-content> <sup>9</sup></td></tr><tr><td><inline-formula><mml:math id="inf231"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mrow><mml:mi>X</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s)</td><td>0.481</td><td>1.040</td><td>+</td><td><named-content content-type="author-callout-style-a1">0.005</named-content></td><td>0.529</td><td>0.844</td><td>+</td><td><named-content content-type="author-callout-style1">10<sup>-43</sup></named-content></td><td>0.459</td><td>0.727</td><td>+</td><td><named-content content-type="author-callout-style1">10<sup>-39</sup></named-content></td><td>0.482</td><td>0.328</td><td>&#8211;</td><td><named-content content-type="author-callout-style1">10<sup>-49</sup></named-content></td><td>0.460</td><td>0.235</td><td>&#8211;</td><td><named-content content-type="author-callout-style1">10<sup>-99</sup></named-content></td></tr><tr><td><inline-formula><mml:math id="inf232"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mrow><mml:mi>Y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s)</td><td>0.247</td><td>0.382</td><td>+</td><td><named-content content-type="author-callout-style1">10<sup>-5</sup></named-content></td><td>0.238</td><td>0.290</td><td>+</td><td><named-content content-type="author-callout-style-a1">0.005</named-content></td><td>0.221</td><td>0.164</td><td>&#8211;</td><td><named-content content-type="author-callout-style1">10<sup>-5</sup></named-content></td><td>0.247</td><td>0.097</td><td>&#8211;</td><td><named-content content-type="author-callout-style1">10<sup>-93</sup></named-content></td><td>0.221</td><td>0.079</td><td>&#8211;</td><td><named-content content-type="author-callout-style1">10<sup>-99</sup></named-content></td></tr><tr><td><inline-formula><mml:math id="inf233"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>d</mml:mi><mml:mrow><mml:mi>p</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula> (s)</td><td>0.428</td><td>0.982</td><td>+</td><td><named-content content-type="author-callout-style-a3">10<sup>-99</sup></named-content></td><td>0.466</td><td>0.793</td><td>+</td><td><named-content content-type="author-callout-style-a3">10<sup>-52</sup></named-content></td><td>0.409</td><td>0.677</td><td>+</td><td><named-content content-type="author-callout-style-a3">10<sup>-43</sup></named-content></td><td>0.428</td><td>0.286</td><td>&#8211;</td><td><named-content content-type="author-callout-style-a3">10<sup>-61</sup></named-content></td><td>0.409</td><td>0.204</td><td>&#8211;</td><td><named-content content-type="author-callout-style-a3">10<sup>-99</sup></named-content></td></tr><tr><td><inline-formula><mml:math id="inf234"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mrow><mml:mi>F</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td>0.729</td><td>0.839</td><td>+</td><td><named-content content-type="author-callout-style1">10<sup>-27</sup></named-content></td><td>0.734</td><td>0.832</td><td>+</td><td><named-content content-type="author-callout-style1">10<sup>-26</sup></named-content></td><td>0.755</td><td>0.785</td><td>+</td><td><named-content content-type="author-callout-style-a1">0.003</named-content></td><td>0.728</td><td>0.410</td><td>&#8211;</td><td><named-content content-type="author-callout-style1">10<sup>-99</sup></named-content></td><td>0.755</td><td>0.407</td><td>&#8211;</td><td><named-content content-type="author-callout-style1">10<sup>-99</sup></named-content></td></tr><tr><td><inline-formula><mml:math id="inf235"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mrow><mml:mi>R</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td>0.177</td><td>0.062</td><td>&#8211;</td><td><named-content content-type="author-callout-style1">10<sup>-37</sup></named-content></td><td>0.167</td><td>0.079</td><td>&#8211;</td><td><named-content content-type="author-callout-style1">10<sup>-29</sup></named-content></td><td>0.161</td><td>0.135</td><td>&#8211;</td><td><named-content content-type="author-callout-style1">10<sup>-4</sup></named-content></td><td>0.177</td><td>0.389</td><td>+</td><td><named-content content-type="author-callout-style1">10<sup>-99</sup></named-content></td><td>0.161</td><td>0.404</td><td>+</td><td><named-content content-type="author-callout-style1">10<sup>-99</sup></named-content></td></tr><tr><td><inline-formula><mml:math id="inf236"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mrow><mml:mi>X</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td>0.073</td><td>0.090</td><td>+</td><td><named-content content-type="author-callout-style-a3">10<sup>-5</sup></named-content></td><td>0.077</td><td>0.081</td><td>+</td><td>0.3</td><td>0.067</td><td>0.073</td><td>+</td><td><named-content content-type="author-callout-style-a1">0.03</named-content></td><td>0.073</td><td>0.164</td><td>+</td><td><named-content content-type="author-callout-style-a3">10<sup>-99</sup></named-content></td><td>0.067</td><td>0.151</td><td>+</td><td><named-content content-type="author-callout-style-a3"><named-content content-type="author-callout-style1">10<sup>-99</sup></named-content></named-content> </td></tr><tr><td><inline-formula><mml:math id="inf237"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>p</mml:mi><mml:mrow><mml:mi>Y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td>0.022</td><td>0.009</td><td>&#8211;</td><td><named-content content-type="author-callout-style-a3">10<sup>-13</sup></named-content></td><td>0.021</td><td>0.008</td><td>&#8211;</td><td><named-content content-type="author-callout-style-a3">10<sup>-25</sup></named-content></td><td>0.018</td><td>0.007</td><td>&#8211;</td><td><named-content content-type="author-callout-style-a3">10<sup>-27</sup></named-content></td><td>0.022</td><td>0.037</td><td>+</td><td><named-content content-type="author-callout-style-a3">10<sup>-21</sup></named-content></td><td>0.018</td><td>0.037</td><td>+</td><td><named-content content-type="author-callout-style-a3">10<sup>-41</sup></named-content></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12613-v1.xml">elife-12613-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="A1-tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.12613.020</object-id><label>Table A1.2.</label><caption><p>The fraction of discordant true and false SNPs between COCALL and GATK population-based variant calling. COCALL had a greater proportion of true SNPs &#8211; confirmed by visual inspection of mapping reads &#8211; both in terms of the number of variant sites, and across the population of samples.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12613.020">http://dx.doi.org/10.7554/eLife.12613.020</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">SNP caller</th><th colspan="2">Sites</th><th colspan="2">Mutations</th></tr><tr><th>True</th><th>False</th><th>True</th><th>False</th></tr></thead><tbody><tr><td rowspan="2">COCALL</td><td>106</td><td>33</td><td>991</td><td>118</td></tr><tr><td>76.3%</td><td>23.7%</td><td>89.4%</td><td>10.6%</td></tr><tr><td rowspan="2">GATK population caller</td><td>33</td><td>106</td><td>118</td><td>991</td></tr><tr><td>23.7%</td><td>76.3%</td><td>10.6%</td><td>89.4%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12613-v1.xml">elife-12613-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="A1-tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.12613.021</object-id><label>Table A1.3.</label><caption><p>The rate of SNP calling for different combinations of read libraries of samples BPK282/0cl4, BPK087/0cl11 and BPK275/0cl18. BPK282/0cl4 was the reference genome strain and so should possess only heterozygous SNPs, and so its variant discovery rate was substantially lower. COCALL identified more rare variant SNPs in mixed libraries.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12613.021">http://dx.doi.org/10.7554/eLife.12613.021</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3">Read library combinations</th><th rowspan="2">Unique to COCALL</th><th rowspan="2">Shared</th><th rowspan="2">Unique to GATK population caller</th></tr><tr><th>BPK282/0cl4</th><th>BPK275/0cl18</th><th>BPK087/0cl11</th></tr></thead><tbody><tr><td/><td>89%</td><td>11%</td><td>14</td><td>140</td><td>6</td></tr><tr><td/><td>80%</td><td>20%</td><td>7</td><td>182</td><td>7</td></tr><tr><td/><td>70%</td><td>30%</td><td>6</td><td>191</td><td>7</td></tr><tr><td/><td>60%</td><td>40%</td><td>9</td><td>193</td><td>7</td></tr><tr><td/><td>50%</td><td>50%</td><td>7</td><td>194</td><td>7</td></tr><tr><td/><td>40%</td><td>60%</td><td>5</td><td>196</td><td>6</td></tr><tr><td/><td>30%</td><td>70%</td><td>7</td><td>191</td><td>6</td></tr><tr><td/><td>20%</td><td>80%</td><td>7</td><td>178</td><td>8</td></tr><tr><td/><td>11%</td><td>89%</td><td>9</td><td>145</td><td>6</td></tr><tr><td>89%</td><td/><td>11%</td><td>10</td><td>50</td><td>5</td></tr><tr><td>80%</td><td/><td>20%</td><td>11</td><td>112</td><td>5</td></tr><tr><td>70%</td><td/><td>30%</td><td>5</td><td>129</td><td>4</td></tr><tr><td>60%</td><td/><td>40%</td><td>7</td><td>131</td><td>5</td></tr><tr><td>50%</td><td/><td>50%</td><td>7</td><td>131</td><td>6</td></tr><tr><td>40%</td><td/><td>60%</td><td>6</td><td>132</td><td>6</td></tr><tr><td>30%</td><td/><td>70%</td><td>5</td><td>133</td><td>6</td></tr><tr><td>20%</td><td/><td>80%</td><td>4</td><td>130</td><td>6</td></tr><tr><td>11%</td><td/><td>89%</td><td>4</td><td>127</td><td>6</td></tr><tr><td>89%</td><td>11%</td><td/><td>15</td><td>64</td><td>3</td></tr><tr><td>80%</td><td>20%</td><td/><td>14</td><td>107</td><td>4</td></tr><tr><td>70%</td><td>30%</td><td/><td>4</td><td>129</td><td>3</td></tr><tr><td>60%</td><td>40%</td><td/><td>4</td><td>132</td><td>3</td></tr><tr><td>50%</td><td>50%</td><td/><td>3</td><td>133</td><td>4</td></tr><tr><td>40%</td><td>60%</td><td/><td>3</td><td>132</td><td>5</td></tr><tr><td>30%</td><td>70%</td><td/><td>4</td><td>131</td><td>4</td></tr><tr><td>20%</td><td>80%</td><td/><td>3</td><td>130</td><td>5</td></tr><tr><td>11%</td><td>89%</td><td/><td>3</td><td>126</td><td>6</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12613-v1.xml">elife-12613-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="A1-tbl4" position="float"><object-id pub-id-type="doi">10.7554/eLife.12613.022</object-id><label>Table A1.4.</label><caption><p>The probabilities of observing alleles at a specific frequency in the core population and ISC1. We assessed whether or not the SNPs we discovered in our sample of ISC <italic>L. donovani</italic> were likely to be representative of the variation existing in the wider population of parasites circulating in the ISC. We first assessed the chance of discovering alleles at frequency f as 1-(1-f)^2n in the core population (n=191) and ISC1 (n=12) sample sets (<xref ref-type="bibr" rid="bib31">Gutenkunst et al., 2009</xref>). As&#160;a second approach, we evaluated the chance of sampling at least one derived allele in n sampled chromosomes (as above) assuming these come from a population of size N=1000 following a hypergeometric distribution (<xref ref-type="bibr" rid="bib74">Tennessen et al., 2012</xref>). Both approaches confirm we have good power (&gt;99% probability) of sampling alleles with population frequencies as low as 1.2% in the core population, but are likely to miss many variants in ISC1.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12613.022">http://dx.doi.org/10.7554/eLife.12613.022</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Allele Frequency</th><th colspan="2" rowspan="2">P(derived allele)=1-(1-f)<sup>2n</sup> <break/><break/></th><th colspan="2"><inline-formula><mml:math id="inf2"><mml:mfenced><mml:mtable><mml:mtr><mml:mtd><mml:mi>M</mml:mi></mml:mtd></mml:mtr><mml:mtr><mml:mtd><mml:mn>0</mml:mn></mml:mtd></mml:mtr></mml:mtable></mml:mfenced><mml:mo>&#160;</mml:mo><mml:mfenced><mml:mtable><mml:mtr><mml:mtd><mml:mi>N</mml:mi><mml:mo>-</mml:mo><mml:mi>M</mml:mi></mml:mtd></mml:mtr><mml:mtr><mml:mtd><mml:mi>n</mml:mi></mml:mtd></mml:mtr></mml:mtable></mml:mfenced><mml:mo>&#160;</mml:mo><mml:mo>/</mml:mo><mml:mfenced><mml:mtable><mml:mtr><mml:mtd><mml:mi>N</mml:mi></mml:mtd></mml:mtr><mml:mtr><mml:mtd><mml:mi>n</mml:mi></mml:mtd></mml:mtr></mml:mtable></mml:mfenced><mml:mspace linebreak="newline"/></mml:math></inline-formula> <break/></th></tr><tr><th colspan="2">P(derived allele)=1 &#8211;</th></tr><tr><th>Core population</th><th>ISC1</th><th>Core population</th><th>ISC1</th></tr></thead><tbody><tr><td>0.001</td><td>0.318</td><td>0.024</td><td>0.382</td><td>0.024</td></tr><tr><td>0.002</td><td>0.535</td><td>0.047</td><td>0.618</td><td>0.047</td></tr><tr><td>0.003</td><td>0.683</td><td>0.07</td><td>0.91</td><td>0.07</td></tr><tr><td>0.004</td><td>0.784</td><td>0.092</td><td>0.945</td><td>0.092</td></tr><tr><td>0.005</td><td>0.853</td><td>0.113</td><td>0.966</td><td>0.115</td></tr><tr><td>0.006</td><td>0.9</td><td>0.134</td><td>0.979</td><td>0.136</td></tr><tr><td>0.007</td><td>0.932</td><td>0.155</td><td>0.987</td><td>0.157</td></tr><tr><td>0.008</td><td>0.953</td><td>0.175</td><td>0.992</td><td>0.177</td></tr><tr><td>0.009</td><td>0.968</td><td>0.195</td><td>0.997</td><td>0.197</td></tr><tr><td>0.01</td><td>0.978</td><td>0.214</td><td>0.999</td><td>0.217</td></tr><tr><td>0.012</td><td>0.99</td><td>0.252</td><td>0.999</td><td>0.254</td></tr><tr><td>0.015</td><td>0.997</td><td>0.304</td><td>0.999</td><td>0.307</td></tr><tr><td>0.021</td><td>0.999</td><td>0.399</td><td>0.999</td><td>0.403</td></tr><tr><td>0.024</td><td>0.999</td><td>0.442</td><td>1</td><td>0.446</td></tr><tr><td>0.029</td><td>0.999</td><td>0.507</td><td>1</td><td>0.511</td></tr><tr><td>0.038</td><td>0.999</td><td>0.605</td><td>1</td><td>0.61</td></tr><tr><td>0.039</td><td>0.999</td><td>0.615</td><td>1</td><td>0.619</td></tr><tr><td>0.04</td><td>1</td><td>0.625</td><td>1</td><td>0.629</td></tr><tr><td>0.049</td><td>1</td><td>0.701</td><td>1</td><td>0.705</td></tr><tr><td>0.065</td><td>1</td><td>0.801</td><td>1</td><td>0.805</td></tr><tr><td>0.092</td><td>1</td><td>0.901</td><td>1</td><td>0.904</td></tr><tr><td>0.173</td><td>1</td><td>0.99</td><td>1</td><td>0.99</td></tr><tr><td>0.272</td><td>1</td><td>1</td><td>1</td><td>1</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-12613-v1.xml">elife-12613-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="A1-tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.12613.023</object-id><label>Table A1.5.</label><caption><p>The percentage of SNP pairs for all pairs on the same chromosome for which both SNPs occur within a block, for different block sizes (kb) for Cortex, GATK, Samtools Mpileup and Pileup (Supplementary Table 6). Singletons may have clustered for Samtools Pileup, but not for Cortex, GATK or Samtools Mpileup. The mean distance was calculated the average distance between SNPs of the given type on the same chromosome. Non-rare SNPs were those found in three or more samples.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.12613.023">http://dx.doi.org/10.7554/eLife.12613.023</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><p>SNP Caller</p></th><th rowspan="2"><p>SNP type</p></th><th colspan="9"><p>Block size (kb)</p></th></tr><tr><th><p>0.01</p></th><th><p>0.1</p></th><th><p>0.5</p></th><th><p>1</p></th><th><p>5</p></th><th><p>10</p></th><th><p>50</p></th><th><p>100</p></th><th><p>500</p></th></tr></thead><tbody><tr><td rowspan="4"><p>Cortex</p></td><td><p>Non-rare</p></td><td><p>0.0</p></td><td><p>0.3</p></td><td><p>0.9</p></td><td><p>1.1</p></td><td><p>1.7</p></td><td><p>3.4</p></td><td><p>9.6</p></td><td><p>17.0</p></td><td><p>61.5</p></td></tr><tr><td><p>Doubletons</p></td><td><p>1.5</p></td><td><p>1.5</p></td><td><p>1.5</p></td><td><p>1.5</p></td><td><p>3.1</p></td><td><p>3.1</p></td><td><p>10.8</p></td><td><p>13.8</p></td><td><p>64.6</p></td></tr><tr><td><p>Singletons</p></td><td><p>0.0</p></td><td><p>0.0</p></td><td><p>0.0</p></td><td><p>0.0</p></td><td><p>3.8</p></td><td><p>3.8</p></td><td><p>23.1</p></td><td><p>30.8</p></td><td><p>80.8</p></td></tr><tr><td><p>All</p></td><td><p>0.0</p></td><td><p>0.3</p></td><td><p>0.9</p></td><td><p>1.1</p></td><td><p>1.7</p></td><td><p>3.3</p></td><td><p>9.6</p></td><td><p>17.0</p></td><td><p>61.5</p></td></tr><tr><td rowspan="4"><p>GATK</p></td><td><p>Non-rare</p></td><td><p>0.0</p></td><td><p>0.1</p></td><td><p>0.2</p></td><td><p>0.3</p></td><td><p>1.1</p></td><td><p>2.1</p></td><td><p>9.3</p></td><td><p>17.8</p></td><td><p>64.4</p></td></tr><tr><td><p>Doubletons</p></td><td><p>1.3</p></td><td><p>1.3</p></td><td><p>1.3</p></td><td><p>1.3</p></td><td><p>1.3</p></td><td><p>1.3</p></td><td><p>5.3</p></td><td><p>13.3</p></td><td><p>61.3</p></td></tr><tr><td><p>Singletons</p></td><td><p>0.0</p></td><td><p>0.0</p></td><td><p>0.0</p></td><td><p>0.0</p></td><td><p>1.1</p></td><td><p>3.2</p></td><td><p>11.8</p></td><td><p>18.3</p></td><td><p>77.4</p></td></tr><tr><td><p>All</p></td><td><p>0.0</p></td><td><p>0.1</p></td><td><p>0.2</p></td><td><p>0.3</p></td><td><p>1.1</p></td><td><p>2.1</p></td><td><p>9.3</p></td><td><p>17.8</p></td><td><p>64.4</p></td></tr><tr><td rowspan="4"><p>Samtools Mpileup</p></td><td><p>Non-rare</p></td><td><p>0.0</p></td><td><p>0.1</p></td><td><p>0.3</p></td><td><p>0.4</p></td><td><p>1.2</p></td><td><p>2.2</p></td><td><p>9.5</p></td><td><p>17.9</p></td><td><p>64.4</p></td></tr><tr><td><p>Doubletons</p></td><td><p>0.0</p></td><td><p>0.0</p></td><td><p>0.0</p></td><td><p>0.0</p></td><td><p>6.7</p></td><td><p>6.7</p></td><td><p>6.7</p></td><td><p>20.0</p></td><td><p>60.0</p></td></tr><tr><td><p>Singletons</p></td><td><p>0.0</p></td><td><p>0.0</p></td><td><p>0.0</p></td><td><p>0.0</p></td><td><p>9.1</p></td><td><p>9.1</p></td><td><p>9.1</p></td><td><p>9.1</p></td><td><p>81.8</p></td></tr><tr><td><p>All</p></td><td><p>0.0</p></td><td><p>0.1</p></td><td><p>0.3</p></td><td><p>0.4</p></td><td><p>1.2</p></td><td><p>2.2</p></td><td><p>9.5</p></td><td><p>17.9</p></td><td><p>64.4</p></td></tr><tr><td rowspan="4"><p>Samtools Pileup</p></td><td><p>Non-rare</p></td><td><p>0.0</p></td><td><p>0.1</p></td><td><p>0.2</p></td><td><p>0.3</p></td><td><p>1.1</p></td><td><p>2.1</p></td><td><p>9.3</p></td><td><p>17.8</p></td><td><p>64.4</p></td></tr><tr><td><p>Doubletons</p></td><td><p>2.6</p></td><td><p>3.3</p></td><td><p>3.3</p></td><td><p>3.3</p></td><td><p>5.3</p></td><td><p>6.6</p></td><td><p>11.8</p></td><td><p>21.1</p></td><td><p>63.2</p></td></tr><tr><td><p>Singletons</p></td><td><p>0.8</p></td><td><p>4.4</p></td><td><p>5.2</p></td><td><p>5.2</p></td><td><p>5.7</p></td><td><p>6.3</p></td><td><p>13.4</p></td><td><p>24.9</p></td><td><p>71.6</p></td></tr><tr><td><p>All</p></td><td><p>0.0</p></td><td><p>0.1</p></td><td><p>0.2</p></td><td><p>0.3</p></td><td><p>1.1</p></td><td><p>2.1</p></td><td><p>9.3</p></td><td><p>17.8</p></td><td><p>64.5</p></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13073-v1.xml">elife-13073-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.13073.008</object-id><label>Table 1.</label><caption><p>Peak current amplitude of Na<sub>v</sub>1.1-F383S and Na<sub>v</sub>1.1-F383S-S1328P expressed in human NgN2 neurons.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13073.008">http://dx.doi.org/10.7554/eLife.13073.008</ext-link></p><p><supplementary-material id="SD2-data"><object-id pub-id-type="doi">10.7554/eLife.13073.009</object-id><label>Table 1&#8212;source data 1.</label><caption><title>The peak I<sub>Na</sub> amplitude for each neuron described in <xref ref-type="table" rid="tbl3">Table 1</xref>.</title><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13073.009">http://dx.doi.org/10.7554/eLife.13073.009</ext-link></p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-13073-table1-data1-v1.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Mean <break/>(95% CI)</th><th align="left" colspan="2" valign="top"><p>Na<sub>v</sub>1.1-F383S</p></th><th align="left" colspan="2" valign="top"><p>Na<sub>v</sub>1.1-F383S-S1328P</p></th></tr><tr><th valign="top">2 &#181;g/ml Dox <break/>(10 cells)</th><th valign="top">0.1 &#181;g/ml Dox <break/>(8 cells)</th><th valign="top">2 &#181;g/ml Dox <break/>(9 cells)</th><th valign="top">0.1 &#181;g/ml Dox <break/>(7 cells)</th></tr></thead><tbody><tr><td valign="top">I<sub>max</sub> (pA/pF)</td><td valign="top">152.4 <break/>(114.4, 190.3)</td><td valign="top">141.7 <break/>(83.04, 200.4)</td><td valign="top">109.6 <break/>(44.94, 174.2)</td><td valign="top">77.03 <break/>(17.77, 136.3)</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>I<sub>max</sub> is the maximal inward current detected in a neuron and normalized to the whole cell capacitance. By two-way ANOVA using Na<sub>v</sub>1.1 variants as factor 1 and Dox concentration as factor 2, F (1, 30) = 5.135 and p = 0.0308 for factor 1.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13073-v1.xml">elife-13073-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.13073.025</object-id><label>Table&#160;3.</label><caption><p>Cell-type specific comparisons of I<sub>Na</sub> activation between control and Dravet iPSC-derived neurons.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13073.025">http://dx.doi.org/10.7554/eLife.13073.025</ext-link></p><p><supplementary-material id="SD6-data"><object-id pub-id-type="doi">10.7554/eLife.13073.026</object-id><label>Table 3&#8212;source data 1.</label><caption><title>Parameters characterizing I<sub>Na</sub> activation for each neuron described in <xref ref-type="table" rid="tbl1">Table 3</xref>.</title><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13073.026">http://dx.doi.org/10.7554/eLife.13073.026</ext-link></p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-13073-table3-data1-v1.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Mean <break/>(95% CI)</th><th colspan="2" valign="top">Excitatory neurons</th><th colspan="2" valign="top">Inhibitory neurons</th></tr><tr><th valign="top">Control <break/>(3 subjects, <break/>45 cells)</th><th valign="top">Dravet <break/>(2 subjects, <break/>45 cells)</th><th valign="top">Control <break/>(4 subjects, <break/>28 cells)</th><th valign="top">Dravet <break/>(2 subjects, <break/>20 cells)</th></tr></thead><tbody><tr><td valign="top">I<sub>max</sub> (pA/pF)</td><td valign="top">63.62 <break/>(55.01, 72.23)</td><td valign="top">62.15 <sup>p1</sup> <break/>(52.19, 72.11)</td><td valign="top">100.5 <break/>(74.78, 126.2)</td><td valign="top">64.27 <sup>p4</sup> <break/>(46.07, 82.47)</td></tr><tr><td valign="top">V<sub>R</sub> (mV)</td><td valign="top">37.34 <break/>(34.51, 40.16)</td><td valign="top">34.23 <sup>p2</sup> <break/>(31.43, 37.02)</td><td valign="top">27.34 <break/>(24.32, 30.36)</td><td valign="top">28.28 <sup>p5</sup> <break/>(23.74, 32.83)</td></tr><tr><td valign="top">V<sub>1/2</sub> (mV)</td><td valign="top">&#8722;29.89 <break/>(&#8722;31.58,&#8722;28.21)</td><td valign="top">&#8722;28.81 <sup>p3</sup> <break/>(&#8722;30.31, &#8722;27.31)</td><td valign="top">&#8722;27.63 <break/>(&#8722; 30.06, &#8722;25.19)</td><td valign="top">&#8722;25.31 <sup>p6</sup> <break/>(&#8722;28.98, &#8722;21.64)</td></tr></tbody></table><table-wrap-foot><fn id="tblfn3"><p>I<sub>max</sub> is the maximal inward current detected in a neuron and normalized to the whole cell capacitance. V<sub>R</sub> is the reversal potential for sodium currents. V<sub>1/2</sub> describes the membrane potential at which 50% of the sodium channels are activated. V<sub>R</sub> and V<sub>1/2</sub> were determined for each neuron by curve fitting in GraphPad Prism using the equation I&#160;=&#160;g*(V-V<sub>R</sub>)/(1+exp(-0.03937*z*(V-V<sub>1/2</sub>))) (see Materials and methods). 95% CI is the 95% confidence interval. p1 = 0.8225, p2 = 0.1291, p3 = 0.3505, in corresponding t-tests between Control and Dravet excitatory neurons. p4 = 0.034, p5 = 0.7259, p6 = 0.2885, in corresponding t-tests between Control and Dravet inhibitory neurons.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13073-v1.xml">elife-13073-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl4" position="float"><object-id pub-id-type="doi">10.7554/eLife.13073.027</object-id><label>Table&#160;4.</label><caption><p>Cell-type specific comparisons of I<sub>Na</sub> steady-state inactivation between control and Dravet iPSC-derived neurons.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13073.027">http://dx.doi.org/10.7554/eLife.13073.027</ext-link></p><p><supplementary-material id="SD7-data"><object-id pub-id-type="doi">10.7554/eLife.13073.028</object-id><label>Table 4&#8212;source data 1.</label><caption><title>V<sub>1/2</sub><sup>&#8217;</sup> values for each neuron described in <xref ref-type="table" rid="tbl2">Table 4</xref>.</title><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13073.028">http://dx.doi.org/10.7554/eLife.13073.028</ext-link></p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-13073-table4-data1-v1.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Mean <break/>(95% CI)</th><th colspan="2" valign="top">Excitatory neurons</th><th colspan="2" valign="top">Inhibitory neurons</th></tr><tr><th valign="top">Control <break/>(3 subjects, <break/>37 cells)</th><th valign="top">Dravet <break/>(2 subjects, <break/>40 cells)</th><th valign="top">Control <break/>(4 subjects, <break/>25 cells)</th><th valign="top">Dravet <break/>(2 subjects, <break/>19 cells)</th></tr></thead><tbody><tr><td valign="top">V<sub>1/2</sub><sup>&#8217;</sup> (mV)</td><td valign="top">&#8722;44.87 <break/>(&#8722;46.13, &#8722;43.61)</td><td valign="top">&#8722;45.27 <sup>p1</sup> <break/>(&#8722;46.47, -44.07)</td><td valign="top">&#8722;45.32 <break/>(&#8722;47.84, &#8722;42.79)</td><td valign="top">&#8722;43.31 <sup>p2</sup> <break/>(&#8722;46.08, &#8722;40.55)</td></tr></tbody></table><table-wrap-foot><fn id="tblfn4"><p>V<sub>1/2</sub><sup>&#8217;</sup> describes the membrane potential at which 50% of the sodium channels are inactivated. V<sub>1/2</sub><sup>&#8217;</sup> was determined per neuron by curve fitting in GraphPad Prism using the equation I&#160;=&#160;1/(1+exp((V- V<sub>1/2</sub>&#8217;)/a&#8217;)) (see Materials and methods). 95% CI is the 95% confidence interval. p1 = 0.6551 by t-test between control and Dravet excitatory neurons. p2 = 0.3035 by t-test between control and Dravet inhibitory neurons.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13135-v1.xml">elife-13135-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.13135.004</object-id><label>Table 2.</label><caption><p>Summary of PASC analysis.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13135.004">http://dx.doi.org/10.7554/eLife.13135.004</ext-link></p><p><supplementary-material id="SD1-data"><object-id pub-id-type="doi">10.7554/eLife.13135.005</object-id><label>Table 2&#8212;source data 1.</label><caption><title>PASC analysis.</title><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13135.005">http://dx.doi.org/10.7554/eLife.13135.005</ext-link></p></caption><media mime-subtype="docx" mimetype="application" xlink:href="elife-13135-table2-data1-v1.docx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Sample ID</th><th rowspan="2" valign="top">Country of origin</th><th colspan="2" valign="top">PASC: S segment</th><th colspan="2" valign="top">PASC: L segment</th></tr><tr><th valign="top">Sequence identity (%)</th><th valign="top">Closest virus</th><th valign="top">Sequence identity (%)</th><th valign="top">Closest virus</th></tr></thead><tbody><tr><td valign="top">C0617</td><td valign="top">Cambodia</td><td valign="top">88.80</td><td valign="top">W&#275;nzh&#333;u virus</td><td valign="top">86.27</td><td valign="top">W&#275;nzh&#333;u virus</td></tr><tr><td valign="top">C0649</td><td valign="top">Cambodia</td><td valign="top">88.51</td><td valign="top">W&#275;nzh&#333;u virus</td><td valign="top">85.98</td><td valign="top">W&#275;nzh&#333;u virus</td></tr><tr><td valign="top">R4937</td><td valign="top">Thailand</td><td valign="top">70.63</td><td valign="top">W&#275;nzh&#333;u virus</td><td valign="top">62.71</td><td valign="top">W&#275;nzh&#333;u virus</td></tr><tr><td valign="top">R5074</td><td valign="top">Thailand</td><td valign="top">70.29</td><td valign="top">W&#275;nzh&#333;u virus</td><td valign="top">63.08</td><td valign="top">W&#275;nzh&#333;u virus</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13135-v2.xml">elife-13135-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.13135.004</object-id><label>Table 2.</label><caption><p>Summary of PASC analysis.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13135.004">http://dx.doi.org/10.7554/eLife.13135.004</ext-link></p><p><supplementary-material id="SD1-data"><object-id pub-id-type="doi">10.7554/eLife.13135.005</object-id><label>Table 2&#8212;source data 1.</label><caption><title>PASC analysis.</title><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13135.005">http://dx.doi.org/10.7554/eLife.13135.005</ext-link></p></caption><media mime-subtype="docx" mimetype="application" xlink:href="elife-13135-table2-data1-v2.docx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Sample ID</th><th rowspan="2" valign="top">Country of origin</th><th colspan="2" valign="top">PASC: S segment</th><th colspan="2" valign="top">PASC: L segment</th></tr><tr><th valign="top">Sequence identity (%)</th><th valign="top">Closest virus</th><th valign="top">Sequence identity (%)</th><th valign="top">Closest virus</th></tr></thead><tbody><tr><td valign="top">C0617</td><td valign="top">Cambodia</td><td valign="top">88.80</td><td valign="top">W&#275;nzh&#333;u virus</td><td valign="top">86.27</td><td valign="top">W&#275;nzh&#333;u virus</td></tr><tr><td valign="top">C0649</td><td valign="top">Cambodia</td><td valign="top">88.51</td><td valign="top">W&#275;nzh&#333;u virus</td><td valign="top">85.98</td><td valign="top">W&#275;nzh&#333;u virus</td></tr><tr><td valign="top">R4937</td><td valign="top">Thailand</td><td valign="top">70.63</td><td valign="top">W&#275;nzh&#333;u virus</td><td valign="top">62.71</td><td valign="top">W&#275;nzh&#333;u virus</td></tr><tr><td valign="top">R5074</td><td valign="top">Thailand</td><td valign="top">70.29</td><td valign="top">W&#275;nzh&#333;u virus</td><td valign="top">63.08</td><td valign="top">W&#275;nzh&#333;u virus</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13152-v2.xml">elife-13152-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.13152.018</object-id><label>Table 3.</label><caption><p>Summary of the best set 1 candidate models by interface energy.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13152.018">http://dx.doi.org/10.7554/eLife.13152.018</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Ligand</th><th rowspan="2">Set</th><th colspan="4">Best model by interface energy</th></tr><tr><th>Interface energy score (kcal/mol)</th><th valign="bottom">Rosetta total energy (kcal/mol)</th><th>H-bond and Salt bridges Interactions</th><th>Distances between corresponding atom pairs (&#197;)</th></tr></thead><tbody><tr><td rowspan="8">PS</td><td rowspan="3">1A</td><td rowspan="3">-6.64</td><td rowspan="3">-1464.65</td><td>Head - R29</td><td>2.90, 2.77</td></tr><tr><td>P - R66</td><td>2.67</td></tr><tr><td>Tail - R66</td><td>3.45, 2.93</td></tr><tr><td rowspan="2">1B</td><td rowspan="2">-3.06</td><td rowspan="2">-1461.49</td><td>Head - R29</td><td>2.84, 2.90, 3.32</td></tr><tr><td>P - R66</td><td>2.89, 3.32</td></tr><tr><td rowspan="3">1C</td><td rowspan="3">-3.15</td><td rowspan="3">-1461.72</td><td>Head - R29</td><td>2.87, 3.43</td></tr><tr><td>Head - E36</td><td>2.95</td></tr><tr><td>P - R66</td><td>2.86</td></tr><tr><td rowspan="9">PE</td><td rowspan="4">1A</td><td rowspan="4">-2.90</td><td rowspan="4">-1465.81</td><td>Head - E63</td><td>2.92</td></tr><tr><td>P - R66</td><td>3.19</td></tr><tr><td>Tail - R66</td><td>2.76, 3.24</td></tr><tr><td>Tail - R29</td><td>2.92</td></tr><tr><td rowspan="2">1B</td><td rowspan="2">-2.92</td><td rowspan="2">-1464.62</td><td>Head - E63</td><td>2.95</td></tr><tr><td>P - R66</td><td>2.88, 3.43</td></tr><tr><td rowspan="2">1C</td><td rowspan="2">-2.61</td><td rowspan="2">-1466.02</td><td>Head - E63</td><td>2.98</td></tr><tr><td>P - R66</td><td>2.84, 3.61</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>Ligand in <bold>set 1A</bold> has the longest tail construct (kept up to the &#945; carbon). Ligand in <bold>set 1B</bold> only has head group and phosphate. Ligand in <bold>set 1C</bold> has one carbon after the ester linkage. <bold>Head</bold> is the ligand head group. <bold>P</bold> is the ligand phosphate moiety. <bold>Tail</bold> includes the ligand glycerol moiety and two fatty acid chains.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13292-v1.xml">elife-13292-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.13292.015</object-id><label>Table 1.</label><caption><p>Proteins Co-Purified by PCH1 AP-MS in WT and <italic>phyB-9.</italic> Proteins co-purified with PCH1 were identified from affinity purification coupled with mass spectrometry (AP-MS) analyses using 12L:12D grown, 10-day-old PCH1ox3 plants (in either WT or <italic>phyB-9</italic> mutant backgrounds) harvested at ZT12.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13292.015">http://dx.doi.org/10.7554/eLife.13292.015</ext-link></p><p><supplementary-material id="SD3-data"><object-id pub-id-type="doi">10.7554/eLife.13292.016</object-id><label>Table 1&#8212;source data 1.</label><caption><title>The full list of proteins identified by AP-MS, listing unique peptides and the percent coverage.&#160;</title><p>The full list is generated and exported by Scaffold (Proteome Software Inc., Portland, Oregon; v.4.4.3) showing all co-purified proteins from all replicates of PCH1ox3 AP-MS and the GFP Control. The file contains reports on exclusive unique peptide counts and percent coverage for each co-purified proteins, with their names, accession numbers and molecular weight.</p> <p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13292.016">http://dx.doi.org/10.7554/eLife.13292.016</ext-link></p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-13292-data1-v1.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">AGI number</th><th rowspan="3">Protein name</th><th rowspan="3">ELF3 AP-MS<sup>b</sup></th><th colspan="5">Exclusive unique peptide count/Percent coverage<sup>a</sup></th></tr><tr><th colspan="3">PCH1ox3 in WT</th><th colspan="2">PCH1ox3 in <italic>phyB-9</italic></th> </tr><tr><th>rep1</th><th>rep2</th><th>rep3</th><th>rep1</th><th>rep2</th></tr></thead><tbody><tr><td>At2g16365</td><td>PCH1<sup>c</sup></td><td>Y</td><td>30/73%</td><td>41/85%</td><td>37/79%</td><td>40/85%</td><td>32/77%</td></tr><tr><td>At2g18790</td><td>phyB</td><td>Y</td><td>46/69%</td><td>47/65%</td><td>41/60%</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td>At5g35840</td><td>phyC</td><td>Y</td><td>31/44%</td><td>23/28%</td><td>25/29%</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td>At4g16250</td><td>phyD</td><td>Y</td><td>22/47%</td><td>19/34%</td><td>20/38%</td><td>22/38%</td><td>6/11%</td></tr><tr><td>At4g18130</td><td>phyE</td><td>Y</td><td>41/55%</td><td>40/52%</td><td>45/60%</td><td>49/60%</td><td>31/41%</td></tr><tr><td>At1g09570</td><td>phyA</td><td>Y</td><td>31/46%</td><td>36/49%</td><td>35/45%</td><td>36/48%</td><td>29/39%</td></tr><tr><td>At2g37678</td><td>FHY1</td><td>N</td><td>2/22%</td><td>2/21%</td><td>4/28%</td><td>4/28%</td><td>2/21%</td></tr><tr><td>At3g42170</td><td>DAYSLEEPER</td><td>Y</td><td>5/13%</td><td>&#8212;</td><td>4/11%</td><td>3/8%</td><td>2/6%</td></tr><tr><td>At1g09340</td><td>CRB</td><td>Y</td><td>&#8212;<sup>d</sup></td><td>&#8212;<sup>d</sup></td><td>4/17%</td><td>6/24%</td><td>3/13%</td></tr><tr><td>At5g43630</td><td>TZP</td><td>Y</td><td>9/15%</td><td>6/12%</td><td>12/23%</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td>At2g32950</td><td>COP1</td><td>Y</td><td>7/15%</td><td>8/16%</td><td>8/18%</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td>At2g46340</td><td>SPA1</td><td>Y</td><td>8/14%</td><td>5/7%</td><td>8/12%</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td>At2g25930</td><td>ELF3</td><td>Y</td><td>6/12%</td><td>11/25%</td><td>12/26%</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td>At2g40080</td><td>ELF4</td><td>Y</td><td>&#8212;<sup>d</sup></td><td>4/60%</td><td>3/42%</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td>At3g46640</td><td>LUX</td><td>Y</td><td>2/6%</td><td>&#8212;<sup>d</sup></td><td>4/15%</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td>At3g03940</td><td>MLK2</td><td>Y</td><td>&#8212;<sup>d</sup></td><td>2/6%</td><td>2/6%</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td>At1g15750</td><td>TPL</td><td>N</td><td>&#8212;<sup>d</sup></td><td>3/3%</td><td>4/5%</td><td>2/2%</td><td>&#8212;</td></tr><tr><td>At1g47128</td><td>RD21a</td><td>Y</td><td>&#8212;<sup>d</sup></td><td>3/8%</td><td>2/4%</td><td>&#8212;<sup>d</sup></td><td>&#8212;<sup>d</sup></td></tr><tr><td colspan="8">Also see <xref ref-type="table" rid="tbl1">Table 1&#8212;source data 1</xref></td> </tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p><sup>a</sup>&#160;All listed proteins match 99% protein threshold, minimum number peptides of 2 and peptide threshold as 95%. Proteins not matching the criteria were marked with "&#8212;".&#160;</p></fn><fn id="tblfn2"><p><sup>b</sup>&#160;ELF3 AP-MS (<xref ref-type="bibr" rid="bib24">Huang et al., 2015</xref>) was used for comparison.&#160;</p></fn><fn id="tblfn3"><p><sup>c</sup>&#160;Percent coverage for PCH1 is calculated using protein encoded by&#160;<italic>At2g16365.2</italic>.&#160;</p></fn><fn id="tblfn4"><p><sup>d</sup>&#160;Only one exclusive unique peptide was detected.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13292-v1.xml">elife-13292-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.13292.017</object-id><label>Table 2.</label><caption><p>Proteins Co-Purified by PCH1 AP-MS in <italic>elf4-2</italic> and <italic>elf3-2</italic>, compared to WT. Proteins co-purified with PCH1 were identified from affinity purification coupled with mass spectrometry (AP-MS) analyses using 12L:12D grown, 10-day-old PCH1ox3 plants in either <italic>elf4-2</italic> or <italic>elf3-2</italic> mutant backgrounds harvested at ZT12.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13292.017">http://dx.doi.org/10.7554/eLife.13292.017</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">AGI number</th><th rowspan="3">Protein name</th><th rowspan="3">ELF3 AP-MS<sup>b</sup></th><th colspan="7">Exclusive unique peptide count/Percent coverage<sup>a</sup></th></tr><tr><th colspan="3">PCH1ox3 in WT<sup>c</sup></th><th colspan="2">PCH1ox3 in <italic>elf4-2</italic></th> <th colspan="2">PCH1ox3 in <italic>elf3-2</italic></th> </tr><tr><th>rep1</th><th>rep2</th><th>rep3</th><th>rep1</th><th>rep2</th><th>rep1</th><th>rep2</th></tr></thead><tbody><tr><td>At2g16365</td><td>PCH1<sup>d</sup></td><td>Y</td><td>30/73%</td><td>41/85%</td><td>37/79%</td><td>29/77%</td><td>34/78%</td><td>42/82%</td><td>36/82%</td></tr><tr><td>At2g18790</td><td>phyB</td><td>Y</td><td>46/69%</td><td>47/65%</td><td>41/60%</td><td>47/70%</td><td>31/46%</td><td>42/63%</td><td>40/56%</td></tr><tr><td>At5g35840</td><td>phyC</td><td>Y</td><td>31/44%</td><td>23/28%</td><td>25/29%</td><td>30/43%</td><td>13/16%</td><td>20/28%</td><td>15/18%</td></tr><tr><td>At4g16250</td><td>phyD</td><td>Y</td><td>22/47%</td><td>19/34%</td><td>20/38%</td><td>20/42%</td><td>12/25%</td><td>20/37%</td><td>16/30%</td></tr><tr><td>At4g18130</td><td>phyE</td><td>Y</td><td>41/55%</td><td>40/52%</td><td>45/60%</td><td>42/57%</td><td>37/50%</td><td>43/55%</td><td>40/53%</td></tr><tr><td>At1g09570</td><td>phyA</td><td>Y</td><td>31/46%</td><td>36/49%</td><td>35/45%</td><td>32/47%</td><td>24/34%</td><td>34/49%</td><td>27/38%</td></tr><tr><td>At2g37678</td><td>FHY1</td><td>N</td><td>2/22%</td><td>2/21%</td><td>4/28%</td><td>&#8212;<sup>e</sup></td><td>3/21%</td><td>&#8212;<sup>e</sup></td><td>3/21%</td></tr><tr><td>At3g42170</td><td>DAYSLEEPER</td><td>Y</td><td>5/13%</td><td>&#8212;</td><td>4/11%</td><td>3/7%</td><td>5/11%</td><td>2/5%</td><td>&#8212;<sup>e</sup></td></tr><tr><td>At1g09340</td><td>CRB</td><td>Y</td><td>&#8212;<sup>e</sup></td><td>&#8212;<sup>e</sup></td><td>4/17%</td><td>2/9%</td><td>5/22%</td><td>3/13%</td><td>3/13%</td></tr><tr><td>At5g43630</td><td>TZP</td><td>Y</td><td>9/15%</td><td>6/12%</td><td>12/23%</td><td>4/7%</td><td>&#8212;<sup>e</sup></td><td>12/21%</td><td>2/3%</td></tr><tr><td>At2g32950</td><td>COP1</td><td>Y</td><td>7/15%</td><td>8/16%</td><td>8/18%</td><td>3/7%</td><td>&#8212;<sup>e</sup></td><td>12/25%</td><td>7/11%</td></tr><tr><td>At2g46340</td><td>SPA1</td><td>Y</td><td>8/14%</td><td>5/7%</td><td>8/12%</td><td>5/10%</td><td>2/4%</td><td>17/26%</td><td>7/11%</td></tr><tr><td>At2g25930</td><td>ELF3</td><td>Y</td><td>6/12%</td><td>11/25%</td><td>12/26%</td><td>4/9%</td><td>3/6%</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td>At2g40080</td><td>ELF4</td><td>Y</td><td>&#8212;<sup>e</sup></td><td>4/60%</td><td>3/42%</td><td>&#8212;</td><td>&#8212;</td><td>&#8212;</td><td>&#8212;<sup>e</sup></td></tr><tr><td>At3g46640</td><td>LUX</td><td>Y</td><td>2/6%</td><td>&#8212;<sup>e</sup></td><td>4/15%</td><td>&#8212;<sup>e</sup></td><td>&#8212;</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td>At3g03940</td><td>MLK2</td><td>Y</td><td>&#8212;<sup>e</sup></td><td>2/6%</td><td>2/6%</td><td>&#8212;</td><td>&#8212;</td><td>2/4%</td><td>&#8212;</td></tr><tr><td>At1g15750</td><td>TPL</td><td>N</td><td>&#8212;<sup>e</sup></td><td>3/3%</td><td>4/5%</td><td>2/3%</td><td>3/3%</td><td>4/5%</td><td>4/5%</td></tr><tr><td>At1g47128</td><td>RD21a</td><td>Y</td><td>&#8212;<sup>e</sup></td><td>3/8%</td><td>2/4%</td><td>&#8212;<sup>e</sup></td><td>2/8%</td><td>3/8%</td><td>&#8212;<sup>e</sup></td></tr><tr><td colspan="10">Also see <xref ref-type="table" rid="tbl1">Table 1&#8212;source data 1</xref></td> </tr></tbody></table><table-wrap-foot><fn id="tblfn5"><p><sup>a</sup> All listed proteins match 99% protein threshold, minimum number peptides of 2 and peptide threshold as 95%. Proteins not matching the criteria were marked with "&#8212;".</p></fn><fn id="tblfn6"><p><sup>b</sup> ELF3 AP-MS (<xref ref-type="bibr" rid="bib24">Huang et al., 2015</xref>) was used for comparison.</p></fn><fn id="tblfn7"><p><sup>c</sup> PCH1ox3 in WT is as shown in <xref ref-type="table" rid="tbl1">Table 1</xref>, for comparison with PCH1ox3 in <italic>elf4-2</italic> and <italic>elf3-2</italic>.</p></fn><fn id="tblfn8"><p><sup>d</sup> percent coverage for PCH1 is calculated using protein encoded by <italic>At2g16365.2</italic></p> </fn><fn id="tblfn9"><p><sup>e</sup> only one exclusive unique peptide was detected.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13292-v2.xml">elife-13292-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.13292.015</object-id><label>Table 1.</label><caption><p>Proteins Co-Purified by PCH1 AP-MS in WT and <italic>phyB-9.</italic> Proteins co-purified with PCH1 were identified from affinity purification coupled with mass spectrometry (AP-MS) analyses using 12L:12D grown, 10-day-old PCH1ox3 plants (in either WT or <italic>phyB-9</italic> mutant backgrounds) harvested at ZT12.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13292.015">http://dx.doi.org/10.7554/eLife.13292.015</ext-link></p><p><supplementary-material id="SD3-data"><object-id pub-id-type="doi">10.7554/eLife.13292.016</object-id><label>Table 1&#8212;source data 1.</label><caption><title>The full list of proteins identified by AP-MS, listing unique peptides and the percent coverage.&#160;</title><p>The full list is generated and exported by Scaffold (Proteome Software Inc., Portland, Oregon; v.4.4.3) showing all co-purified proteins from all replicates of PCH1ox3 AP-MS and the GFP Control. The file contains reports on exclusive unique peptide counts and percent coverage for each co-purified proteins, with their names, accession numbers and molecular weight.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13292.016">http://dx.doi.org/10.7554/eLife.13292.016</ext-link></p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-13292-table1-data1-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top">AGI number</th><th rowspan="3" valign="top">Protein name</th><th rowspan="3" valign="top">ELF3 AP-MS<sup>b</sup></th><th colspan="5" valign="top">Exclusive unique peptide count/Percent coverage<sup>a</sup></th></tr><tr><th colspan="3" valign="top">PCH1ox3 in WT</th><th colspan="2" valign="top">PCH1ox3 in <italic>phyB-9</italic></th></tr><tr><th valign="top">rep1</th><th valign="top">rep2</th><th valign="top">rep3</th><th valign="top">rep1</th><th valign="top">rep2</th></tr></thead><tbody><tr><td valign="top">At2g16365</td><td valign="top">PCH1<sup>c</sup></td><td valign="top">Y</td><td valign="top">30/73%</td><td valign="top">41/85%</td><td valign="top">37/79%</td><td valign="top">40/85%</td><td valign="top">32/77%</td></tr><tr><td valign="top">At2g18790</td><td valign="top">phyB</td><td valign="top">Y</td><td valign="top">46/69%</td><td valign="top">47/65%</td><td valign="top">41/60%</td><td valign="top">&#8212;</td><td valign="top">&#8212;</td></tr><tr><td valign="top">At5g35840</td><td valign="top">phyC</td><td valign="top">Y</td><td valign="top">31/44%</td><td valign="top">23/28%</td><td valign="top">25/29%</td><td valign="top">&#8212;</td><td valign="top">&#8212;</td></tr><tr><td valign="top">At4g16250</td><td valign="top">phyD</td><td valign="top">Y</td><td valign="top">22/47%</td><td valign="top">19/34%</td><td valign="top">20/38%</td><td valign="top">22/38%</td><td valign="top">6/11%</td></tr><tr><td valign="top">At4g18130</td><td valign="top">phyE</td><td valign="top">Y</td><td valign="top">41/55%</td><td valign="top">40/52%</td><td valign="top">45/60%</td><td valign="top">49/60%</td><td valign="top">31/41%</td></tr><tr><td valign="top">At1g09570</td><td valign="top">phyA</td><td valign="top">Y</td><td valign="top">31/46%</td><td valign="top">36/49%</td><td valign="top">35/45%</td><td valign="top">36/48%</td><td valign="top">29/39%</td></tr><tr><td valign="top">At2g37678</td><td valign="top">FHY1</td><td valign="top">N</td><td valign="top">2/22%</td><td valign="top">2/21%</td><td valign="top">4/28%</td><td valign="top">4/28%</td><td valign="top">2/21%</td></tr><tr><td valign="top">At3g42170</td><td valign="top">DAYSLEEPER</td><td valign="top">Y</td><td valign="top">5/13%</td><td valign="top">&#8212;</td><td valign="top">4/11%</td><td valign="top">3/8%</td><td valign="top">2/6%</td></tr><tr><td valign="top">At1g09340</td><td valign="top">CRB</td><td valign="top">Y</td><td valign="top">&#8212;<sup>d</sup></td><td valign="top">&#8212;<sup>d</sup></td><td valign="top">4/17%</td><td valign="top">6/24%</td><td valign="top">3/13%</td></tr><tr><td valign="top">At5g43630</td><td valign="top">TZP</td><td valign="top">Y</td><td valign="top">9/15%</td><td valign="top">6/12%</td><td valign="top">12/23%</td><td valign="top">&#8212;</td><td valign="top">&#8212;</td></tr><tr><td valign="top">At2g32950</td><td valign="top">COP1</td><td valign="top">Y</td><td valign="top">7/15%</td><td valign="top">8/16%</td><td valign="top">8/18%</td><td valign="top">&#8212;</td><td valign="top">&#8212;</td></tr><tr><td valign="top">At2g46340</td><td valign="top">SPA1</td><td valign="top">Y</td><td valign="top">8/14%</td><td valign="top">5/7%</td><td valign="top">8/12%</td><td valign="top">&#8212;</td><td valign="top">&#8212;</td></tr><tr><td valign="top">At2g25930</td><td valign="top">ELF3</td><td valign="top">Y</td><td valign="top">6/12%</td><td valign="top">11/25%</td><td valign="top">12/26%</td><td valign="top">&#8212;</td><td valign="top">&#8212;</td></tr><tr><td valign="top">At2g40080</td><td valign="top">ELF4</td><td valign="top">Y</td><td valign="top">&#8212;<sup>d</sup></td><td valign="top">4/60%</td><td valign="top">3/42%</td><td valign="top">&#8212;</td><td valign="top">&#8212;</td></tr><tr><td valign="top">At3g46640</td><td valign="top">LUX</td><td valign="top">Y</td><td valign="top">2/6%</td><td valign="top">&#8212;<sup>d</sup></td><td valign="top">4/15%</td><td valign="top">&#8212;</td><td valign="top">&#8212;</td></tr><tr><td valign="top">At3g03940</td><td valign="top">MLK2</td><td valign="top">Y</td><td valign="top">&#8212;<sup>d</sup></td><td valign="top">2/6%</td><td valign="top">2/6%</td><td valign="top">&#8212;</td><td valign="top">&#8212;</td></tr><tr><td valign="top">At1g15750</td><td valign="top">TPL</td><td valign="top">N</td><td valign="top">&#8212;<sup>d</sup></td><td valign="top">3/3%</td><td valign="top">4/5%</td><td valign="top">2/2%</td><td valign="top">&#8212;</td></tr><tr><td valign="top">At1g47128</td><td valign="top">RD21a</td><td valign="top">Y</td><td valign="top">&#8212;<sup>d</sup></td><td valign="top">3/8%</td><td valign="top">2/4%</td><td valign="top">&#8212;<sup>d</sup></td><td valign="top">&#8212;<sup>d</sup></td></tr><tr><td colspan="8" valign="top">Also see <xref ref-type="table" rid="tbl1">Table 1&#8212;source data 1</xref></td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p><sup>a</sup>&#160;All listed proteins match 99% protein threshold, minimum number peptides of 2 and peptide threshold as 95%. Proteins not matching the criteria were marked with "&#8212;".&#160;</p></fn><fn id="tblfn2"><p><sup>b</sup>&#160;ELF3 AP-MS (<xref ref-type="bibr" rid="bib24">Huang et al., 2015</xref>) was used for comparison.&#160;</p></fn><fn id="tblfn3"><p><sup>c</sup>&#160;Percent coverage for PCH1 is calculated using protein encoded by&#160;<italic>At2g16365.2</italic>.&#160;</p></fn><fn id="tblfn4"><p><sup>d</sup>&#160;Only one exclusive unique peptide was detected.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13292-v2.xml">elife-13292-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.13292.017</object-id><label>Table 2.</label><caption><p>Proteins Co-Purified by PCH1 AP-MS in <italic>elf4-2</italic> and <italic>elf3-2</italic>, compared to WT. Proteins co-purified with PCH1 were identified from affinity purification coupled with mass spectrometry (AP-MS) analyses using 12L:12D grown, 10-day-old PCH1ox3 plants in either <italic>elf4-2</italic> or <italic>elf3-2</italic> mutant backgrounds harvested at ZT12.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13292.017">http://dx.doi.org/10.7554/eLife.13292.017</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">AGI number</th><th rowspan="3">Protein name</th><th rowspan="3">ELF3 AP-MS<sup>b</sup></th><th colspan="7" valign="top">Exclusive unique peptide count/Percent coverage<sup>a</sup></th></tr><tr><th colspan="3" valign="top">PCH1ox3 in WT<sup>c</sup></th><th colspan="2" valign="top">PCH1ox3 in <italic>elf4-2</italic></th><th colspan="2" valign="top">PCH1ox3 in <italic>elf3-2</italic></th></tr><tr><th>rep1</th><th>rep2</th><th>rep3</th><th valign="top">rep1</th><th valign="top">rep2</th><th valign="top">rep1</th><th valign="top">rep2</th></tr></thead><tbody><tr><td valign="top">At2g16365</td><td valign="top">PCH1<sup>d</sup></td><td valign="top">Y</td><td valign="top">30/73%</td><td>41/85%</td><td>37/79%</td><td valign="bottom">29/77%</td><td valign="bottom">34/78%</td><td valign="bottom">42/82%</td><td valign="bottom">36/82%</td></tr><tr><td valign="top">At2g18790</td><td valign="top">phyB</td><td valign="top">Y</td><td>46/69%</td><td>47/65%</td><td>41/60%</td><td valign="bottom">47/70%</td><td valign="bottom">31/46%</td><td valign="bottom">42/63%</td><td valign="bottom">40/56%</td></tr><tr><td valign="top">At5g35840</td><td valign="top">phyC</td><td valign="top">Y</td><td>31/44%</td><td>23/28%</td><td>25/29%</td><td valign="bottom">30/43%</td><td valign="bottom">13/16%</td><td valign="bottom">20/28%</td><td valign="bottom">15/18%</td></tr><tr><td valign="top">At4g16250</td><td valign="top">phyD</td><td valign="top">Y</td><td>22/47%</td><td>19/34%</td><td>20/38%</td><td valign="bottom">20/42%</td><td valign="bottom">12/25%</td><td valign="bottom">20/37%</td><td valign="bottom">16/30%</td></tr><tr><td valign="top">At4g18130</td><td valign="top">phyE</td><td valign="top">Y</td><td>41/55%</td><td>40/52%</td><td>45/60%</td><td valign="bottom">42/57%</td><td valign="bottom">37/50%</td><td valign="bottom">43/55%</td><td valign="bottom">40/53%</td></tr><tr><td valign="top">At1g09570</td><td valign="top">phyA</td><td valign="top">Y</td><td>31/46%</td><td>36/49%</td><td>35/45%</td><td valign="bottom">32/47%</td><td valign="bottom">24/34%</td><td valign="bottom">34/49%</td><td valign="bottom">27/38%</td></tr><tr><td valign="top">At2g37678</td><td valign="top">FHY1</td><td valign="top">N</td><td>2/22%</td><td>2/21%</td><td>4/28%</td><td valign="bottom">&#8212;<sup>e</sup></td><td valign="bottom">3/21%</td><td valign="bottom">&#8212;<sup>e</sup></td><td valign="bottom">3/21%</td></tr><tr><td valign="top">At3g42170</td><td valign="top">DAYSLEEPER</td><td valign="top">Y</td><td>5/13%</td><td>&#8212;</td><td>4/11%</td><td valign="bottom">3/7%</td><td valign="bottom">5/11%</td><td valign="bottom">2/5%</td><td valign="bottom">&#8212;<sup>e</sup></td></tr><tr><td valign="top">At1g09340</td><td valign="top">CRB</td><td valign="top">Y</td><td>&#8212;<sup>e</sup></td><td>&#8212;<sup>e</sup></td><td>4/17%</td><td valign="bottom">2/9%</td><td valign="bottom">5/22%</td><td valign="bottom">3/13%</td><td valign="bottom">3/13%</td></tr><tr><td valign="top">At5g43630</td><td valign="top">TZP</td><td valign="top">Y</td><td>9/15%</td><td>6/12%</td><td>12/23%</td><td valign="bottom">4/7%</td><td valign="bottom">&#8212;<sup>e</sup></td><td valign="bottom">12/21%</td><td valign="bottom">2/3%</td></tr><tr><td valign="top">At2g32950</td><td valign="top">COP1</td><td valign="top">Y</td><td>7/15%</td><td>8/16%</td><td>8/18%</td><td valign="bottom">3/7%</td><td valign="bottom">&#8212;<sup>e</sup></td><td valign="bottom">12/25%</td><td valign="bottom">7/11%</td></tr><tr><td valign="top">At2g46340</td><td valign="top">SPA1</td><td valign="top">Y</td><td>8/14%</td><td>5/7%</td><td>8/12%</td><td valign="bottom">5/10%</td><td valign="bottom">2/4%</td><td valign="bottom">17/26%</td><td valign="bottom">7/11%</td></tr><tr><td valign="top">At2g25930</td><td valign="top">ELF3</td><td valign="top">Y</td><td>6/12%</td><td>11/25%</td><td>12/26%</td><td valign="bottom">4/9%</td><td valign="bottom">3/6%</td><td valign="bottom">&#8212;</td><td valign="bottom">&#8212;</td></tr><tr><td valign="top">At2g40080</td><td valign="top">ELF4</td><td valign="top">Y</td><td>&#8212;<sup>e</sup></td><td>4/60%</td><td>3/42%</td><td valign="bottom">&#8212;</td><td valign="bottom">&#8212;</td><td valign="bottom">&#8212;</td><td valign="bottom">&#8212;<sup>e</sup></td></tr><tr><td valign="top">At3g46640</td><td valign="top">LUX</td><td valign="top">Y</td><td>2/6%</td><td>&#8212;<sup>e</sup></td><td>4/15%</td><td valign="bottom">&#8212;<sup>e</sup></td><td valign="bottom">&#8212;</td><td valign="bottom">&#8212;</td><td valign="bottom">&#8212;</td></tr><tr><td valign="top">At3g03940</td><td valign="top">MLK2</td><td valign="top">Y</td><td>&#8212;<sup>e</sup></td><td>2/6%</td><td>2/6%</td><td valign="bottom">&#8212;</td><td valign="bottom">&#8212;</td><td valign="bottom">2/4%</td><td valign="bottom">&#8212;</td></tr><tr><td valign="top">At1g15750</td><td valign="top">TPL</td><td valign="top">N</td><td>&#8212;<sup>e</sup></td><td>3/3%</td><td>4/5%</td><td valign="bottom">2/3%</td><td valign="bottom">3/3%</td><td valign="bottom">4/5%</td><td valign="bottom">4/5%</td></tr><tr><td valign="top">At1g47128</td><td valign="top">RD21a</td><td valign="top">Y</td><td>&#8212;<sup>e</sup></td><td>3/8%</td><td>2/4%</td><td valign="bottom">&#8212;<sup>e</sup></td><td valign="bottom">2/8%</td><td valign="bottom">3/8%</td><td valign="bottom">&#8212;<sup>e</sup></td></tr><tr><td colspan="10" valign="top">Also see <xref ref-type="table" rid="tbl1">Table 1&#8212;source data 1</xref></td></tr></tbody></table><table-wrap-foot><fn id="tblfn5"><p><sup>a</sup> All listed proteins match 99% protein threshold, minimum number peptides of 2 and peptide threshold as 95%. Proteins not matching the criteria were marked with "&#8212;".</p></fn><fn id="tblfn6"><p><sup>b</sup> ELF3 AP-MS (<xref ref-type="bibr" rid="bib24">Huang et al., 2015</xref>) was used for comparison.</p></fn><fn id="tblfn7"><p><sup>c</sup> PCH1ox3 in WT is as shown in <xref ref-type="table" rid="tbl1">Table 1</xref>, for comparison with PCH1ox3 in <italic>elf4-2</italic> and <italic>elf3-2</italic>.</p></fn><fn id="tblfn8"><p><sup>d</sup> percent coverage for PCH1 is calculated using protein encoded by <italic>At2g16365.2</italic></p></fn><fn id="tblfn9"><p><sup>e</sup> only one exclusive unique peptide was detected.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13571-v3.xml">elife-13571-v3.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.13571.008</object-id><label>Table 1.</label><caption><p>Amino acid sequences of the synthetic multivalent R-motif containing peptides.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13571.008">http://dx.doi.org/10.7554/eLife.13571.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Peptide <break/>Name<sup>#</sup></th><th rowspan="2">Peptide amino acid sequence</th></tr></thead><tbody><tr><td>rpL5</td><td><sup>21</sup>RRRREGKTDY<sub>10</sub>YARKRLV<sup>37</sup></td></tr><tr><td>GNL2</td><td><sup>682</sup>RRRAVRQQRP<sub>10</sub>KKVGVRYYET<sub>20</sub>HNVKNRNR<sup>709</sup></td></tr><tr><td>SURF6</td><td><sup>299</sup>RRAQRQRRWE<sub>10</sub>KRTAGVVEKM<sub>20</sub>QQRQDRRR<sup>326</sup></td></tr><tr><td>rpL23a</td><td><sup>47</sup>RRPKTLRLRR<sub>10</sub>QPKYPRKSAP<sub>20</sub>RR<sup>68</sup></td></tr><tr><td>rpL5-RA</td><td><sup>21</sup>RRRREGKTDY<sub>10</sub>YAAKALV<sup>37</sup></td></tr><tr><td>rpL5-2xLinker</td><td>RRRREGKTDY<sub>10</sub>YAEGKTDYYA<sub>20</sub>RKRLV</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p><sup>#</sup>The peptides are referred to by the same name as the protein they originate from and the residue numbers of their N- and C-termini are indicated.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13571-v3.xml">elife-13571-v3.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.13571.013</object-id><label>Table 2.</label><caption><p>Binding affinities for interactions between N122 (NPM1 residues 1&#8211;122, displaying only A1) and N130 (displaying both A1 and A2) and the multivalent R-motif-containing peptides, determined using isothermal titration calorimetry (ITC), at concentrations below the critical phase separation threshold.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13571.013">http://dx.doi.org/10.7554/eLife.13571.013</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Peptide</th><th colspan="2"><p>N122</p></th><th colspan="2"><p>N130</p></th></tr><tr><th>N (sites)</th><th>K<sub>D</sub> (&#956;M)</th><th>N (sites)</th><th>K<sub>D</sub> (&#956;M)</th></tr></thead><tbody><tr><td>rpL5</td><td>0.63 &#177; 0.09</td><td>19.0 &#177; 2.7</td><td>2.18 &#177; 0.19</td><td>20.5 &#177; 5.0</td></tr><tr><td>GNL2</td><td>0.66 &#177; 0.04</td><td>10.2 &#177; 1.3</td><td>n.d.<sup>*</sup></td><td>n.d.<sup>*</sup></td></tr><tr><td>SURF6</td><td>0.55 &#177; 0.06</td><td>9.2 &#177; 0.5</td><td>1.4 &#177; 0.02</td><td>19.0 &#177; 0.8</td></tr><tr><td>rpL23a</td><td>n.d.<sup>#</sup></td><td>n.d.<sup>#</sup></td><td>n.d.<sup>*</sup></td><td>n.d.<sup>*</sup></td></tr><tr><td>rpL5-RA</td><td>1.0 &#177; 0.06</td><td>50.9 &#177; 3.18</td><td>0.84 &#177; 0.07</td><td>140.1 &#177; 9.29</td></tr></tbody></table><table-wrap-foot><fn id="tblfn2"><p><sup>*</sup> Not determined due to multiple, unresolved binding events</p></fn><fn id="tblfn3"><p><sup>#</sup> Heat change was too weak for accurate data analysis</p></fn><fn id="tblfn4"><p>Average values from a minimum of three independent experiments are reported &#177; SD</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13732-v2.xml">elife-13732-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.13732.004</object-id><label>Table 1.</label><caption><p>Behavioral performance.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13732.004">http://dx.doi.org/10.7554/eLife.13732.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Task</th><th colspan="2" valign="top">Placebo (n = 18)</th><th colspan="2" valign="top">Baclofen (n = 14)</th><th rowspan="2" valign="top">P value of <break/>group &#215; session interaction</th></tr><tr><th valign="top">Pre</th><th valign="top">Post</th><th valign="top">Pre</th><th valign="top">Post</th></tr></thead><tbody><tr><td valign="top">MMSE</td><td valign="top">29.89 &#177; 0.11</td><td valign="top">29.94 &#177; 0.06</td><td valign="top">29.93 &#177; 0.07</td><td valign="top">30.00 &#177; 0</td><td valign="top">0.86</td></tr><tr><td valign="top">Digit span (forward)</td><td valign="top">7.22 &#177; 0.22</td><td valign="top">7.72 &#177; 0.14</td><td valign="top">7.14 &#177; 0.31</td><td valign="top">7.43 &#177; 0.20</td><td valign="top">0.55</td></tr><tr><td valign="top">Digit span (backward)</td><td valign="top">6.00 &#177; 0.20</td><td valign="top">6.00 &#177; 0.29</td><td valign="top">5.57 &#177; 0.31</td><td valign="top">5.71 &#177; 0.27</td><td valign="top">0.69</td></tr><tr><td valign="top">Trail making test B (s)</td><td valign="top">48.32 &#177; 2.70</td><td valign="top">39.05 &#177; 2.01</td><td valign="top">57.62 &#177; 5.42</td><td valign="top">47.11 &#177; 5.67</td><td valign="top">0.85</td></tr><tr><td valign="top">Stanford sleepiness scale</td><td valign="top">2.44 &#177; 0.17</td><td valign="top">2.22 &#177; 0.21</td><td valign="top">1.93 &#177; 0.20</td><td valign="top">2.50 &#177; 0.33</td><td valign="top">0.041*</td></tr><tr><td valign="top">Sniffin&#8217; Sticks (odor detection threshold)</td><td valign="top">7.08 &#177; 0.84</td><td valign="top">9.65 &#177; 1.06</td><td valign="top">7.82 &#177; 0.95</td><td valign="top">8.57 &#177; 1.01</td><td valign="top">0.22</td></tr><tr><td valign="top">UPSIT (odor identification)</td><td valign="top">36.28 &#177; 0.61</td><td valign="top">36.00 &#177; 0.56</td><td valign="top">34.57 &#177; 0.49</td><td valign="top">33.79 &#177; 0.63</td><td valign="top">0.55</td></tr><tr><td valign="top">&#945;- vs. &#946;-pinene triangle test (fine odor discrimination)</td><td valign="top">0.66 &#177; 0.05</td><td valign="top">0.72 &#177; 0.05</td><td valign="top">0.72 &#177; 0.05</td><td valign="top">0.73 &#177; 0.07</td><td valign="top">0.45</td></tr><tr><td valign="top">Odor intensity ratings</td><td valign="top">4.00 &#177; 0.32</td><td valign="top">4.13 &#177; 0.31</td><td valign="top">3.05 &#177; 0.18</td><td valign="top">2.91 &#177; 0.28</td><td valign="top">0.39</td></tr><tr><td valign="top">Odor pleasantness ratings</td><td valign="top">5.43 &#177; 0.16</td><td valign="top">5.63 &#177; 0.16</td><td valign="top">5.64 &#177; 0.13</td><td valign="top">5.63 &#177; 0.16</td><td valign="top">0.12</td></tr><tr><td valign="top">Odor category descriptor ratings (within &#8211; across)</td><td valign="top">7.47 &#177; 0.44</td><td valign="top">7.49 &#177; 0.36</td><td valign="top">7.44 &#177; 0.46</td><td valign="top">7.78 &#177; 0.34</td><td valign="top">0.59</td></tr><tr><td valign="top">Odor pairwise similarity ratings (within &#8211; across)</td><td valign="top">4.16 &#177; 0.60</td><td valign="top">5.14 &#177; 0.55</td><td valign="top">3.93 &#177; 0.32</td><td valign="top">4.53 &#177; 0.44</td><td valign="top">0.59</td></tr><tr><td valign="top">Odor categorization catch trial accuracy</td><td valign="top">0.87 &#177; 0.04</td><td valign="top">0.89 &#177; 0.03</td><td valign="top">0.81 &#177; 0.04</td><td valign="top">0.81 &#177; 0.04</td><td valign="top">0.80</td></tr><tr><td valign="top">Odor categorization catch trial RT (s)</td><td valign="top">3.29 &#177; 0.23</td><td valign="top">2.85 &#177; 0.15</td><td valign="top">3.89 &#177; 0.38</td><td valign="top">3.48 &#177; 0.34</td><td valign="top">0.93</td></tr><tr><td valign="top">Visual categorization catch trial accuracy</td><td valign="top">0.97 &#177; 0.01 (n = 14)</td><td valign="top">0.99 &#177; 0.004</td><td valign="top">0.97 &#177; 0.01 (n = 11)</td><td valign="top">0.96 &#177; 0.01</td><td valign="top">0.21</td></tr><tr><td valign="top">Visual categorization catch trial RT (s)</td><td valign="top">0.42 &#177; 0.02</td><td valign="top">0.40 &#177; 0.03</td><td valign="top">0.44 &#177; 0.04</td><td valign="top">0.52 &#177; 0.06</td><td valign="top">0.25</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>Data are shown for cognitive and olfactory tests, as well as for behavioral performance in fMRI experiments from placebo and baclofen groups in pre- and post-drug sessions. Scores are presented as mean &#177; s.e.m. P values reported are for the interaction effects between group and session, based on a 2-way ANOVA, with one between-group &#8216;drug&#8217; factor (placebo/baclofen) and one within-subject &#8216;session&#8217; factor (pre/post). *P &lt; 0.05.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-13732-v3.xml">elife-13732-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.13732.004</object-id><label>Table 1.</label><caption><p>Behavioral performance.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.13732.004">http://dx.doi.org/10.7554/eLife.13732.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Task</th><th colspan="2" valign="top">Placebo (n = 18)</th><th colspan="2" valign="top">Baclofen (n = 14)</th><th rowspan="2" valign="top">P value of <break/>group &#215; session interaction</th></tr><tr><th valign="top">Pre</th><th valign="top">Post</th><th valign="top">Pre</th><th valign="top">Post</th></tr></thead><tbody><tr><td valign="top">MMSE</td><td valign="top">29.89 &#177; 0.11</td><td valign="top">29.94 &#177; 0.06</td><td valign="top">29.93 &#177; 0.07</td><td valign="top">30.00 &#177; 0</td><td valign="top">0.86</td></tr><tr><td valign="top">Digit span (forward)</td><td valign="top">7.22 &#177; 0.22</td><td valign="top">7.72 &#177; 0.14</td><td valign="top">7.14 &#177; 0.31</td><td valign="top">7.43 &#177; 0.20</td><td valign="top">0.55</td></tr><tr><td valign="top">Digit span (backward)</td><td valign="top">6.00 &#177; 0.20</td><td valign="top">6.00 &#177; 0.29</td><td valign="top">5.57 &#177; 0.31</td><td valign="top">5.71 &#177; 0.27</td><td valign="top">0.69</td></tr><tr><td valign="top">Trail making test B (s)</td><td valign="top">48.32 &#177; 2.70</td><td valign="top">39.05 &#177; 2.01</td><td valign="top">57.62 &#177; 5.42</td><td valign="top">47.11 &#177; 5.67</td><td valign="top">0.85</td></tr><tr><td valign="top">Stanford sleepiness scale</td><td valign="top">2.44 &#177; 0.17</td><td valign="top">2.22 &#177; 0.21</td><td valign="top">1.93 &#177; 0.20</td><td valign="top">2.50 &#177; 0.33</td><td valign="top">0.041*</td></tr><tr><td valign="top">Sniffin&#8217; Sticks (odor detection threshold)</td><td valign="top">7.08 &#177; 0.84</td><td valign="top">9.65 &#177; 1.06</td><td valign="top">7.82 &#177; 0.95</td><td valign="top">8.57 &#177; 1.01</td><td valign="top">0.22</td></tr><tr><td valign="top">UPSIT (odor identification)</td><td valign="top">36.28 &#177; 0.61</td><td valign="top">36.00 &#177; 0.56</td><td valign="top">34.57 &#177; 0.49</td><td valign="top">33.79 &#177; 0.63</td><td valign="top">0.55</td></tr><tr><td valign="top">&#945;- vs. &#946;-pinene triangle test (fine odor discrimination)</td><td valign="top">0.66 &#177; 0.05</td><td valign="top">0.72 &#177; 0.05</td><td valign="top">0.72 &#177; 0.05</td><td valign="top">0.73 &#177; 0.07</td><td valign="top">0.45</td></tr><tr><td valign="top">Odor intensity ratings</td><td valign="top">4.00 &#177; 0.32</td><td valign="top">4.13 &#177; 0.31</td><td valign="top">3.05 &#177; 0.18</td><td valign="top">2.91 &#177; 0.28</td><td valign="top">0.39</td></tr><tr><td valign="top">Odor pleasantness ratings</td><td valign="top">5.43 &#177; 0.16</td><td valign="top">5.63 &#177; 0.16</td><td valign="top">5.64 &#177; 0.13</td><td valign="top">5.63 &#177; 0.16</td><td valign="top">0.12</td></tr><tr><td valign="top">Odor category descriptor ratings (within &#8211; across)</td><td valign="top">7.47 &#177; 0.44</td><td valign="top">7.49 &#177; 0.36</td><td valign="top">7.44 &#177; 0.46</td><td valign="top">7.78 &#177; 0.34</td><td valign="top">0.59</td></tr><tr><td valign="top">Odor pairwise similarity ratings (within &#8211; across)</td><td valign="top">4.16 &#177; 0.60</td><td valign="top">5.14 &#177; 0.55</td><td valign="top">3.93 &#177; 0.32</td><td valign="top">4.53 &#177; 0.44</td><td valign="top">0.59</td></tr><tr><td valign="top">Odor categorization catch trial accuracy</td><td valign="top">0.87 &#177; 0.04</td><td valign="top">0.89 &#177; 0.03</td><td valign="top">0.81 &#177; 0.04</td><td valign="top">0.81 &#177; 0.04</td><td valign="top">0.80</td></tr><tr><td valign="top">Odor categorization catch trial RT (s)</td><td valign="top">3.29 &#177; 0.23</td><td valign="top">2.85 &#177; 0.15</td><td valign="top">3.89 &#177; 0.38</td><td valign="top">3.48 &#177; 0.34</td><td valign="top">0.93</td></tr><tr><td valign="top">Visual categorization catch trial accuracy</td><td valign="top">0.97 &#177; 0.01 (n = 14)</td><td valign="top">0.99 &#177; 0.004</td><td valign="top">0.97 &#177; 0.01 (n = 11)</td><td valign="top">0.96 &#177; 0.01</td><td valign="top">0.21</td></tr><tr><td valign="top">Visual categorization catch trial RT (s)</td><td valign="top">0.42 &#177; 0.02</td><td valign="top">0.40 &#177; 0.03</td><td valign="top">0.44 &#177; 0.04</td><td valign="top">0.52 &#177; 0.06</td><td valign="top">0.25</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>Data are shown for cognitive and olfactory tests, as well as for behavioral performance in fMRI experiments from placebo and baclofen groups in pre- and post-drug sessions. Scores are presented as mean &#177; s.e.m. P values reported are for the interaction effects between group and session, based on a 2-way ANOVA, with one between-group &#8216;drug&#8217; factor (placebo/baclofen) and one within-subject &#8216;session&#8217; factor (pre/post). *P &lt; 0.05.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-14003-v1.xml">elife-14003-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.14003.004</object-id><label>Table 1.</label><caption><p>Comparison of antimicrobial susceptibility by <italic>Salmonella</italic> Typhi lineage.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.14003.004">http://dx.doi.org/10.7554/eLife.14003.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">E test</th><th colspan="3">Non-H58 (N=13)</th><th colspan="3">H58 (N=65)</th><th>p value*</th></tr><tr><th>MIC50</th><th>MIC90</th><th>GM (range)</th><th>MIC50</th><th>MIC90</th><th>GM (range)</th><th/></tr></thead><tbody><tr><td>Amoxicillin</td><td>0.5</td><td>1</td><td>0.77 (0.38&#8211;38)</td><td>0.75</td><td>&gt;256</td><td>1.43 (0.38&#8211;&gt;256)</td><td>0.0412</td></tr><tr><td>Chloramphenicol</td><td>3</td><td>4</td><td>2.7 (1.5&#8211;8)</td><td>4</td><td>12</td><td>5.7 (2&#8211;&gt;256)</td><td>0.0147</td></tr><tr><td>Ceftriaxone</td><td>0.06</td><td>0.06</td><td>0.06 (0.05&#8211;0.13)</td><td>0.09</td><td>0.19</td><td>0.11 (0.03&#8211;0.64)</td><td>0.0004</td></tr><tr><td>Gatifloxacin</td><td>0.13</td><td>0.25</td><td>0.06 (0.01&#8211;2)</td><td>0.13</td><td>2</td><td>0.21 (0.01&#8211;3)</td><td>0.1197</td></tr><tr><td>Nalidixic acid</td><td>&gt;256</td><td>&gt;256</td><td>21.6 (1&#8211;&gt;256)</td><td>&gt;256</td><td>&gt;256</td><td>346.8 (1&#8211;&gt;256)</td><td>0.0004</td></tr><tr><td>Ofloxacin</td><td>0.25</td><td>0.75</td><td>0.24 (0.03&#8211;&gt;32)</td><td>0.5</td><td>&gt;3232</td><td>1.09 (0.03&#8211;&gt;32)</td><td>0.0240</td></tr><tr><td>Trimethoprim sulphate</td><td>0.02</td><td>0.05</td><td>0.03 (0.02&#8211;0.05)</td><td>0.05</td><td>0.32</td><td>0.09 (0.01&#8211;&gt;32)</td><td>0.0016</td></tr><tr><td>Ciprofloxacin</td><td>0.13</td><td>0.75</td><td>0.11 (0.01&#8211;&gt;32)</td><td>0.38</td><td>&gt;32</td><td>0.80 (0.02&#8211;&gt;32)</td><td>0.0051</td></tr><tr><td>Ciprofloxacin susceptibility group</td><td colspan="3"/><td colspan="3"/><td>0.0008<sup>#</sup></td></tr><tr><td>- Susceptible</td><td colspan="3">6 (46.2%)</td><td colspan="3">4 (6.2%)</td><td/></tr><tr><td>- Intermediate</td><td colspan="3">4 (30.8%)</td><td colspan="3">48 (73.8%)</td><td/></tr><tr><td>- Resistant</td><td colspan="3">3 (23.1%)</td><td colspan="3">13 (20.0%)</td><td/></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>*Comparisons between <italic>Salmonella</italic> Typhi lineage for MICs and ciprofloxacin susceptibility groups were based on the Wilcoxon rank sum test and Fisher&#8217;s exact test. respectively.</p></fn><fn id="tblfn2"><p>MIC: minimum inhibitory concentration, measured in &#181;g/ml</p></fn><fn id="tblfn3"><p>#p value for comparison of susceptible vs. intermediate/resistant combined between groups by Fisher&#8217;s exact test is 0.001.</p></fn><fn id="tblfn4"><p>GM: geometric mean, the upper range of the values was determined by multiplying the MIC by 2 if the result was &gt;X (for example, &gt;256 = 256*2 = 512).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-14022-v3.xml">elife-14022-v3.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.14022.008</object-id><label>Table 1.</label><caption><p>From an initial number of possible networks (Step 1), RDNets progressively identifies reaction-diffusion networks that can form a pattern (Step 6).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.14022.008">http://dx.doi.org/10.7554/eLife.14022.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Steps</th><th colspan="2" valign="top"><p>3 nodes</p></th><th colspan="2" valign="top"><p>4 nodes</p></th></tr><tr><th valign="top"># networks</th><th valign="top"># topologies</th><th valign="top"># networks</th><th valign="top"># topologies</th></tr></thead><tbody><tr><td valign="top">1. Minimal systems</td><td valign="top">84</td><td valign="top">5376</td><td valign="top">11440</td><td valign="top">1464320</td></tr><tr><td valign="top">2. Strongly connected</td><td valign="top">48</td><td valign="top">3072</td><td valign="top">2284</td><td valign="top">292352</td></tr><tr><td valign="top">3. Non-symmetrical</td><td valign="top">25</td><td valign="top">1600</td><td valign="top">597</td><td valign="top">76416</td></tr><tr><td valign="top">4. Stable</td><td valign="top">24</td><td valign="top">556</td><td valign="top">324</td><td valign="top">8640</td></tr><tr><td valign="top">5-6. Reaction-diffusion</td><td valign="top">21</td><td valign="top">84</td><td valign="top">64</td><td valign="top">512</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-14119-v2.xml">elife-14119-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.14119.003</object-id><label>Table 1.</label><caption><p>Properties of employed FG domain constructs. See <xref ref-type="supplementary-material" rid="SD1-data">Table 1&#8212;source data 1</xref> for the full amino acid sequences of these constructs.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.14119.003">http://dx.doi.org/10.7554/eLife.14119.003</ext-link></p><p><supplementary-material id="SD1-data"><object-id pub-id-type="doi">10.7554/eLife.14119.004</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Amino acid sequence of employed FG domain constructs.</title><p>FG domains are shown in black letters, His tags in blue letters, and remaining parts (i.e., TEV cleavage sites, Cys tags and spacers) in grey letters. FxFG motifs are marked in yellow, GLFG motifs in green, other FG motifs in purple. Nup98-glyco features O-GlcNAc on ~30 of the S and T residues.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.14119.004">http://dx.doi.org/10.7554/eLife.14119.004</ext-link></p></caption><media mime-subtype="zip" mimetype="application" xlink:href="elife-14119-table1-data1-v2.zip"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">FG domain</th><th rowspan="2">amino acids <sup>a</sup></th><th rowspan="2">sequence</th><th colspan="3">FG motifs</th><th rowspan="2">FG motifs/amino acids</th></tr><tr><th>FxFG</th><th>GLFG</th><th>Other</th></tr></thead><tbody><tr><td>Nsp1</td><td>615</td><td>irregular, natural</td><td>19</td><td>0</td><td>14</td><td>0.054</td></tr><tr><td>Nup98-glyco</td><td>496</td><td>irregular, natural</td><td>3</td><td>8</td><td>28</td><td>0.079</td></tr><tr><td>reg-FSFG</td><td>315</td><td>regular, artificial</td><td>16</td><td>0</td><td>0</td><td>0.051</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p><sup>a</sup> Excluding the His tags but including all other auxiliary amino acids (TEV cleavage sites, Cys tags and spacers).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-14170-v2.xml">elife-14170-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.14170.046</object-id><label>Table 5.</label><caption><p>pFRG/RTN slices respond to CO<sub>2</sub> if the EP3R is present. The average mean frequency of all cells in the network and the average mean frequency of NK1R-positive cells during the control period or during exposure to hypercapnia are shown (pCO<sub>2</sub> = 55 mmHg, pH = 7.5). N.S.: not significant. N: number of slices, n: number of cells. Data are presented as mean &#177; SD.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.14170.046">http://dx.doi.org/10.7554/eLife.14170.046</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"/><th colspan="6">Mean frequency (mHz)</th></tr><tr><th colspan="3">Network</th><th colspan="3">NK1R+ cells</th></tr><tr><th>Control</th><th>Hypercapnia</th><th/><th>Control</th><th>Hypercapnia</th><th/></tr></thead><tbody><tr><td>pFRG/RTN -&#160;WT <break/>(N=7, n=343)</td><td>21.6 &#177; 3.2</td><td>54.1 &#177; 2.7*</td><td>p&lt;0.05</td><td>14.4 &#177; 0.9</td><td>38.5 &#177; 5.5*</td><td>p&lt;0.05</td></tr><tr><td>pFRG/RTN -&#160;<italic>Ptger3</italic><sup>&#8722;/&#8722;</sup> <break/>(N=5, n=448)</td><td>25.0 &#177; 7.9</td><td>26.0 &#177; 1.9</td><td>N.S.</td><td>11.4 &#177; 5.8</td><td>11.6 &#177; 3.8</td><td>N.S.</td></tr><tr><td>preB&#246;tC -&#160;WT <break/>(N=5, n=1737)</td><td>16.4 &#177; 2.5</td><td>16.5 &#177; 1.3</td><td>N.S.</td><td>16.6 &#177; 4.6</td><td>15.7 &#177; 5.3</td><td>N.S.</td></tr><tr><td>preB&#246;tC -&#160;<italic>Ptger3</italic><sup>&#8722;/&#8722;</sup> <break/>(N=4, n=822)</td><td>21.1 &#177; 8.6</td><td>17.3 &#177; 3.8</td><td>N.S.</td><td>22.7 &#177; 5.9</td><td>17.8 &#177; 7.7</td><td>N.S.</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-14170-v2.xml">elife-14170-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl6" position="float"><object-id pub-id-type="doi">10.7554/eLife.14170.057</object-id><label>Table 6.</label><caption><p>PGE<sub>2</sub> increases the frequency of pFRG/RTN neurons and decreases the frequency of preB&#246;tC neurons.The mean frequencies of NK1R-positive cells during the control period or during exposure to 10 nM PGE<sub>2</sub> are shown. N.S.: not significant. N: number of slices, n: number of cells. Data are presented as mean &#177; SD.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.14170.057">http://dx.doi.org/10.7554/eLife.14170.057</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="3">Mean frequency (mHz)</th></tr><tr><th>Control</th><th>PGE<sub>2</sub></th><th/></tr></thead><tbody><tr><td>pFRG/RTN -&#160;WT <break/>(N=5, n=343)</td><td>13.7 &#177; 1.1</td><td>21.5 &#177; 2.9*</td><td>p&lt;0.05</td></tr><tr><td>pFRG/RTN -&#160;<italic>Ptger3</italic>&#8722;/&#8722; <break/>(N=4, n=448)</td><td>12.1 &#177; 2.0</td><td>8.5 &#177; 2.9</td><td>N.S.</td></tr><tr><td>preB&#246;tC -&#160;WT <break/>(N=7, n=1737)</td><td>20.3 &#177; 2.2</td><td>8.7 &#177; 1.4*</td><td>p&lt;0.05</td></tr><tr><td>preB&#246;tC -&#160;<italic>Ptger3</italic>&#8722;/&#8722; <break/>(N=5, n=822)</td><td>22.8 &#177; 2.3</td><td>16.4 &#177; 1.1</td><td>N.S.</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-14170-v2.xml">elife-14170-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl7" position="float"><object-id pub-id-type="doi">10.7554/eLife.14170.063</object-id><label>Table 7.</label><caption><p>Silencing of <italic>Ptger3</italic>-expressing cells inhibits the response to hypercapnia.Mean frequencies of the pFRG/RTN network during the control period and during exposure to hypercapnia with and without Halo57 stimulation are shown. N.S.: not significant. N: number of slices. Data are presented as mean &#177; SD.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.14170.063">http://dx.doi.org/10.7554/eLife.14170.063</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">N=41</th><th colspan="3">Mean frequency (mHz)</th></tr><tr><th>Control</th><th>Hypercapnia</th><th/></tr></thead><tbody><tr><td>Control</td><td>22.9 &#177; 9.0*</td><td>34.0 &#177; 4.3*</td><td>p&lt;0.05</td></tr><tr><td>Halo57 stimulation</td><td>9.0 &#177; 1.7</td><td>10.3 &#177; 1.1</td><td>N.S.</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-14734-v2.xml">elife-14734-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.14734.008</object-id><label>Table 3.</label><caption><p>Repartition of &#916;133p53 isoforms in the Tayside breast cancer cohort (2x2 table). (A) &#916;133p53&#945; X &#916;133p53&#946;. (B) &#916;133p53&#945; X &#916;133p53&#947;. (C) &#916;133p53&#946; X &#916;133p53&#947; (&#8722;) not detected, (+) amplified.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.14734.008">http://dx.doi.org/10.7554/eLife.14734.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">A</th><th valign="top"/><th colspan="2" valign="top"/><th valign="top"/></tr><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top"/><th colspan="3" valign="top">&#916;133p53&#945;</th></tr><tr><th valign="top">&#8722;</th><th valign="top">+</th><th valign="top">Total</th></tr></thead><tbody><tr><td valign="top"><bold>&#916;133p53&#946;</bold></td><td valign="top">&#8722; <break/>+</td><td valign="top">174 <break/>2</td><td valign="top">76 <break/>21</td><td valign="top">250 <break/>23</td></tr><tr><td valign="top"><bold>Total</bold></td><td valign="top"/><td valign="top">176</td><td valign="top">97</td><td valign="top">273</td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th valign="top">B</th><th valign="top"/><th colspan="2" valign="top"/><th valign="top"/></tr><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top"/><th colspan="3" valign="top">&#916;133p53&#945;</th></tr><tr><th valign="top">&#8722;</th><th valign="top">+</th><th valign="top">Total</th></tr></thead><tbody><tr><td valign="top"><bold>&#916;133p53&#947;</bold> </td><td valign="top">&#8722; <break/>+</td><td valign="top">138 <break/>3</td><td valign="top">27 <break/>53</td><td valign="top">165 <break/>56</td></tr><tr><td valign="top"><bold>Total</bold></td><td valign="top"/><td valign="top">141</td><td valign="top">80</td><td valign="top">221</td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th valign="top">C</th><th valign="top"/><th colspan="2" valign="top"/><th valign="top"/></tr><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top"/><th colspan="3" valign="top">&#916;133p53&#947;</th></tr><tr><th valign="top">&#8722;</th><th valign="top">+</th><th valign="top">Total</th></tr></thead><tbody><tr><td valign="top"><bold>&#916;133p53&#946;</bold></td><td valign="top">&#8722; <break/>+</td><td valign="top">156 <break/>9</td><td valign="top">42 <break/>14</td><td valign="top">198 <break/>23</td></tr><tr><td valign="top"><bold>Total</bold></td><td valign="top"/><td valign="top">165</td><td valign="top">56</td><td valign="top">221</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-14734-v2.xml">elife-14734-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl4" position="float"><object-id pub-id-type="doi">10.7554/eLife.14734.009</object-id><label>Table 4.</label><caption><p>Univariate analysis of &#916;133p53 isoforms expression in relation to clinical pathological markers. Tumor grade, cancer type (ductal or others), tumor size (&gt;or &lt; 20&#160;mm) were analyzed by Fisher&#8217;s t-test. Association with the number of invaded lymph nodes were analyzed by Mann-Whitney method.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.14734.009">http://dx.doi.org/10.7554/eLife.14734.009</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top"/><th colspan="3" valign="top">grade</th><th colspan="3" valign="top">type</th><th colspan="3" valign="top">size</th><th rowspan="2" valign="top">nb invaded <break/>Lymph nodes</th></tr><tr><th valign="top">1-2</th><th valign="top">3</th><th valign="top"><italic>p</italic> value</th><th valign="top">ductal</th><th valign="top">others</th><th valign="top"><italic>p</italic> value</th><th valign="top">&lt;20&#160;mm</th><th valign="top">&gt;20&#160;mm</th><th valign="top"><italic>p</italic> value</th></tr></thead><tbody><tr><td valign="top"><bold>&#8710;133p53&#945;</bold></td><td valign="top">&#8722; <break/>+</td><td valign="top">76 <break/>34</td><td valign="top">98 <break/>61</td><td valign="top">p&lt;0.21</td><td valign="top">140 <break/>80</td><td valign="top">36 <break/>17</td><td valign="top">p&lt;0.56</td><td valign="top">62 <break/>29</td><td valign="top">113 <break/>67</td><td valign="top">p&lt;0.39</td><td valign="top">p&lt;0.97</td></tr><tr><td valign="top"><bold>&#8710;133p53&#946;</bold></td><td valign="top">&#8722; <break/>+</td><td valign="top">102 <break/>8</td><td valign="top">144 <break/>15</td><td valign="top">p&lt;0.54</td><td valign="top">201 <break/>19</td><td valign="top">49 <break/>4</td><td valign="top">p&lt;0.8</td><td valign="top">81 <break/>10</td><td valign="top">167 <break/>13</td><td valign="top">p&lt;0.30</td><td valign="top">p&lt;0.60</td></tr><tr><td valign="top"><bold>&#8710;133p53&#947;</bold></td><td valign="top">&#8722; <break/>+</td><td valign="top">75 <break/>21</td><td valign="top">87 <break/>35</td><td valign="top">p&lt;0.25</td><td valign="top">133 <break/>46</td><td valign="top">32 <break/>10</td><td valign="top">p&lt;0.8</td><td valign="top">59 <break/>19</td><td valign="top">105 <break/>37</td><td valign="top">p&lt;0.79</td><td valign="top">p&lt;0.78</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-14734-v3.xml">elife-14734-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.14734.008</object-id><label>Table 3.</label><caption><p>Repartition of &#916;133p53 isoforms in the Tayside breast cancer cohort (2x2 table). (A) &#916;133p53&#945; X &#916;133p53&#946;. (B) &#916;133p53&#945; X &#916;133p53&#947;. (C) &#916;133p53&#946; X &#916;133p53&#947; (&#8722;) not detected, (+) amplified.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.14734.008">http://dx.doi.org/10.7554/eLife.14734.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">A<break/></th><th valign="top"/><th colspan="2" valign="top"><break/></th><th valign="top"><break/></th></tr><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top"/><th colspan="3" valign="top">&#916;133p53&#945;</th></tr><tr><th valign="top">&#8722;</th><th valign="top">+</th><th valign="top">Total</th></tr></thead><tbody><tr><td valign="top"><bold>&#916;133p53&#946;</bold></td><td valign="top">&#8722; <break/>+</td><td valign="top">174 <break/>2</td><td valign="top">76 <break/>21</td><td valign="top">250 <break/>23</td></tr><tr><td valign="top"><bold>Total</bold></td><td valign="top"><break/></td><td valign="top">176</td><td valign="top">97</td><td valign="top">273</td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th valign="top">B<break/></th><th valign="top"/><th colspan="2" valign="top"><break/></th><th valign="top"><break/></th></tr><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top"/><th colspan="3" valign="top">&#916;133p53&#945;</th></tr><tr><th valign="top">&#8722;</th><th valign="top">+</th><th valign="top">Total</th></tr></thead><tbody><tr><td valign="top"><bold>&#916;133p53&#947;</bold> </td><td valign="top">&#8722; <break/>+</td><td valign="top">138 <break/>3</td><td valign="top">27 <break/>53</td><td valign="top">165 <break/>56</td></tr><tr><td valign="top"><bold>Total</bold></td><td valign="top"><break/></td><td valign="top">141</td><td valign="top">80</td><td valign="top">221</td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th valign="top">C<break/></th><th valign="top"/><th colspan="2" valign="top"><break/></th><th valign="top"><break/></th></tr><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top"/><th colspan="3" valign="top">&#916;133p53&#947;</th></tr><tr><th valign="top">&#8722;</th><th valign="top">+</th><th valign="top">Total</th></tr></thead><tbody><tr><td valign="top"><bold>&#916;133p53&#946;</bold></td><td valign="top">&#8722; <break/>+</td><td valign="top">156 <break/>9</td><td valign="top">42 <break/>14</td><td valign="top">198 <break/>23</td></tr><tr><td valign="top"><bold>Total</bold></td><td valign="top"><break/></td><td valign="top">165</td><td valign="top">56</td><td valign="top">221</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-14734-v3.xml">elife-14734-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl4" position="float"><object-id pub-id-type="doi">10.7554/eLife.14734.009</object-id><label>Table 4.</label><caption><p>Univariate analysis of &#916;133p53 isoforms expression in relation to clinical pathological markers. Tumor grade, cancer type (ductal or others), tumor size (&gt;or &lt; 20&#160;mm) were analyzed by Fisher&#8217;s t-test. Association with the number of invaded lymph nodes were analyzed by Mann-Whitney method.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.14734.009">http://dx.doi.org/10.7554/eLife.14734.009</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top"/><th colspan="3" valign="top">grade</th><th colspan="3" valign="top">type</th><th colspan="3" valign="top">size</th><th rowspan="2" valign="top">nb invaded <break/>Lymph nodes</th></tr><tr><th valign="top">1-2</th><th valign="top">3</th><th valign="top"><italic>p</italic> value</th><th valign="top">ductal</th><th valign="top">others</th><th valign="top"><italic>p</italic> value</th><th valign="top">&lt;20&#160;mm</th><th valign="top">&gt;20&#160;mm</th><th valign="top"><italic>p</italic> value</th></tr></thead><tbody><tr><td valign="top"><bold>&#8710;133p53&#945;</bold></td><td valign="top">&#8722; <break/>+<break/></td><td valign="top">76 <break/>34</td><td valign="top">98 <break/>61</td><td valign="top">p&lt;0.21</td><td valign="top">140 <break/>80</td><td valign="top">36 <break/>17</td><td valign="top">p&lt;0.56</td><td valign="top">62 <break/>29</td><td valign="top">113 <break/>67</td><td valign="top">p&lt;0.39</td><td valign="top">p&lt;0.97</td></tr><tr><td valign="top"><bold>&#8710;133p53&#946;</bold></td><td valign="top">&#8722; <break/>+<break/></td><td valign="top">102 <break/>8</td><td valign="top">144 <break/>15</td><td valign="top">p&lt;0.54</td><td valign="top">201 <break/>19</td><td valign="top">49 <break/>4</td><td valign="top">p&lt;0.8</td><td valign="top">81 <break/>10</td><td valign="top">167 <break/>13</td><td valign="top">p&lt;0.30</td><td valign="top">p&lt;0.60</td></tr><tr><td valign="top"><bold>&#8710;133p53&#947;</bold></td><td valign="top">&#8722; <break/>+<break/></td><td valign="top">75 <break/>21</td><td valign="top">87 <break/>35</td><td valign="top">p&lt;0.25</td><td valign="top">133 <break/>46</td><td valign="top">32 <break/>10</td><td valign="top">p&lt;0.8</td><td valign="top">59 <break/>19</td><td valign="top">105 <break/>37</td><td valign="top">p&lt;0.79</td><td valign="top">p&lt;0.78</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15085-v1.xml">elife-15085-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.15085.003</object-id><label>Table 2.</label><caption><p>WHO-classified G6PD deficiency alleles observed in this study.&#160;Locus refers to GRCh37, dbSNP137 and Ensembl build 84. WHO refers to grade of G6PD deficiency based on the&#160;WHO classification scheme. Allele frequency is calculated in population controls. GM Gambia; ML, Mali; BF, Burkina Faso; GH-N, Ghana (Noguchi); GH-K, Ghana (Kumasi); NG Nigeria; CM, Cameroon; KY, Kenya; TZ, Tanzania; MW, Malawi; VN, Vietnam; PNG, Papua New Guinea.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15085.003">http://dx.doi.org/10.7554/eLife.15085.003</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><p>SNP</p></th><th rowspan="2"><p>Locus</p></th><th rowspan="2"><p>Base change</p></th><th rowspan="2"><p>WHO</p></th><th colspan="12" valign="top"><p>Allele frequency</p> </th></tr><tr><th><p>GM</p></th><th><p>ML</p></th><th><p>BF</p></th><th><p>GH-N</p></th><th><p>GH-K</p></th><th><p>NG</p></th><th><p>CM</p></th><th><p>KY</p></th><th><p>TZ</p></th><th><p>MW</p></th><th><p>VN</p></th><th><p>PNG</p></th></tr></thead><tbody><tr><td valign="bottom"><p>CM973154</p></td><td valign="bottom"><p>153760261</p></td><td valign="bottom"><p>A/C</p></td><td valign="bottom"><p>I</p></td><td valign="bottom"><p>0.0004</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs72554665</p></td><td valign="bottom"><p>153760484</p></td><td valign="bottom"><p>C/A</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0004</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0055</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>CM920290</p></td><td valign="bottom"><p>153760605</p></td><td valign="bottom"><p>G/A</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0002</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0014</p></td><td valign="bottom"><p>0.0028</p></td></tr><tr><td valign="bottom"><p>rs137852342</p></td><td valign="bottom"><p>153761184</p></td><td valign="bottom"><p>G/A</p></td><td valign="bottom"><p>III</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0020</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0014</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs76723693</p></td><td valign="bottom"><p>153761240</p></td><td valign="bottom"><p>968T&gt;C</p></td><td valign="bottom"><p>III</p></td><td valign="bottom"><p>0.0695</p></td><td valign="bottom"><p>0.0082</p></td><td valign="bottom"><p>0.0009</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0003</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs137852327</p></td><td valign="bottom"><p>153761337</p></td><td valign="bottom"><p>C/T</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0006</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0096</p></td><td valign="bottom"><p>0.0141</p></td></tr><tr><td valign="bottom"><p>CM014189</p></td><td valign="bottom"><p>153761820</p></td><td valign="bottom"><p>T/A</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0004</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0008</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0004</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs137852328</p></td><td valign="bottom"><p>153762340</p></td><td valign="bottom"><p>680G&gt;T</p></td><td valign="bottom"><p>III</p></td><td valign="bottom"><p>0.0002</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs137852330</p></td><td valign="bottom"><p>153762605</p></td><td valign="bottom"><p>G/A</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0004</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0003</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs5030868</p></td><td valign="bottom"><p>153762634</p></td><td valign="bottom"><p>563G&gt;A</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0003</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0085</p></td></tr><tr><td valign="bottom"><p>rs5030872</p></td><td valign="bottom"><p>153762655</p></td><td valign="bottom"><p>542A&gt;T</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0145</p></td><td valign="bottom"><p>0.0041</p></td><td valign="bottom"><p>0.0009</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0003</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>CM970547</p></td><td valign="bottom"><p>153763462</p></td><td valign="bottom"><p>G/A</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0008</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs78365220</p></td><td valign="bottom"><p>153763485</p></td><td valign="bottom"><p>A/G</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0002</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0366</p></td></tr><tr><td valign="bottom"><p>rs1050829</p></td><td valign="bottom"><p>153763492</p></td><td valign="bottom"><p>376T&gt;C</p></td><td valign="bottom"><p>IV</p></td><td valign="bottom"><p>0.3206</p></td><td valign="bottom"><p>0.4132</p></td><td valign="bottom"><p>0.4186</p></td><td valign="bottom"><p>0.4476</p></td><td valign="bottom"><p>0.4267</p></td><td valign="bottom"><p>0.5574</p></td><td valign="bottom"><p>0.3429</p></td><td valign="bottom"><p>0.4060</p></td><td valign="bottom"><p>0.3787</p></td><td valign="bottom"><p>0.3920</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td><p>rs1050828</p></td><td><p>153764217</p></td><td><p>202C&gt;T</p></td><td><p>III</p></td><td valign="bottom"><p>0.0273</p></td><td valign="bottom"><p>0.1616</p></td><td valign="bottom"><p>0.1447</p></td><td valign="bottom"><p>0.1713</p></td><td valign="bottom"><p>0.1847</p></td><td valign="bottom"><p>0.3167</p></td><td valign="bottom"><p>0.1075</p></td><td valign="bottom"><p>0.1930</p></td><td valign="bottom"><p>0.2071</p></td><td valign="bottom"><p>0.1988</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15085-v1.xml">elife-15085-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.15085.012</object-id><label>Table 5.</label><caption><p>Calculation of the G6PD deficiency (G6PDd) score.&#160;The G6PDd score is calculated on the basis of the haplotypes at the 15 WHO-classified SNPs. For each haplotype, the loss of normal G6PD function is determined according to the WHO severity class (0, I, II, III or IV) of the most severe mutation carried. See Materials and methods for more details. The G6PDd score for each individual is then the average loss of function across the two haplotypes, where males are treated as homozygous females. For example, the assigned loss of normal G6PD function for a haplotype where the most severe mutation is the class III mutation G6PD+202T is 65, so an individual with this haplotype and another with no deficiency mutations will average a G6PDd score of 32.5.&#160;Individuals were further categorized on the basis of their G6PDd score as normal G6PDd (G6PDd score&#160;=&#160;0); Low G6PDd (0 &lt; G6PDd score&#160;&lt;&#160;25, blue boxes); Medium G6PDd (25 &#8804; G6PDd score&#160;&lt;&#160;50, yellow boxes); High G6PDd (50 &#8804; G6PDd score&#160;&lt;&#160;85, orange boxes); or Very High (G6PDd&#160;&#8805;&#160;85, red box). Scores not observed in our study are shown in grey boxes.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15085.012">http://dx.doi.org/10.7554/eLife.15085.012</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3" rowspan="3"><p>G6PDd score</p> </th><th colspan="5"><p>Haplotype 1</p></th></tr><tr><th colspan="5"><p>Severity class (loss of normal G6PD function)</p></th></tr><tr><th><p>0 (0)</p></th><th><p>IV (20)</p></th><th><p>III (65)</p></th><th><p>II (95)</p></th><th><p>I (100)</p></th></tr></thead><tbody><tr><td rowspan="5"><p><bold>Haplotype 2</bold></p> </td><td rowspan="5"><p><bold>Severity class</bold></p> <p><bold>(Loss of normal G6PD function)</bold></p> </td><td><p><bold>0&#160;(0)</bold></p> </td><td><p>0</p></td><td style="author-callout-style-b1"><p>10</p></td><td style="author-callout-style-b4"><p>32.5</p></td><td style="author-callout-style-b4"><p>47.5</p></td><td style="author-callout-style-b3"><p>50</p></td></tr><tr><td><p><bold>IV&#160;(20)</bold></p> </td><td style="author-callout-style-b1"><p>10</p></td><td style="author-callout-style-b1"><p>20</p></td><td style="author-callout-style-b4"><p>42.5</p></td><td style="author-callout-style-b3"><p>57.5</p></td><td style="author-callout-style-b8"><p>60</p></td></tr><tr><td><p><bold>III&#160;(65)</bold></p> </td><td style="author-callout-style-b4"><p>32.5</p></td><td style="author-callout-style-b4"><p>42.5</p></td><td style="author-callout-style-b3"><p>65</p></td><td style="author-callout-style-b3"><p>80</p></td><td style="author-callout-style-b8"><p>82.5</p></td></tr><tr><td><p><bold>II&#160;(95)</bold></p> </td><td style="author-callout-style-b4"><p>47.5</p></td><td style="author-callout-style-b3"><p>57.5</p></td><td style="author-callout-style-b3"><p>80</p></td><td style="author-callout-style-b6"><p>95</p></td><td style="author-callout-style-b8"><p>97.5</p></td></tr><tr><td><p><bold>I&#160;(100&#65279;</bold></p> </td><td style="author-callout-style-b3"><p>50</p></td><td style="author-callout-style-b8"><p>60</p></td><td style="author-callout-style-b8"><p>82.5</p></td><td style="author-callout-style-b8"><p>97.7</p></td><td style="author-callout-style-b8"><p>100</p></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15085-v1.xml">elife-15085-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl6" position="float"><object-id pub-id-type="doi">10.7554/eLife.15085.013</object-id><label>Table 6.</label><caption><p>G6PD&#160;deficiency score categorical model association test results.&#160;Counts, odds ratios (ORs), 95% confidence intervals (95% CI) and <italic>P</italic>-values are presented for the effect of each category of G6PD deficiency (G6PDd) score, compared to individuals with G6PDd score&#160;=&#160;0, for all&#160;severe malaria cases and for the two subtypes cerebral malaria and severe malarial anaemia. <italic>P</italic>-value shown in bold is for overall model fit. P-value for the overall model fit. Results are shown for all individuals combined at each site and across all sites. Results are adjusted for the&#160;sickle-cell trait, ethnicity and sex. n.c., not calculated.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15085.013">http://dx.doi.org/10.7554/eLife.15085.013</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><p>Sample</p></th><th rowspan="2"><p>G6PDd score category</p></th><th><p>Controls</p></th><th colspan="3"><p>Severe malaria</p> </th><th colspan="3"><p>Cerebral malaria</p> </th><th colspan="3"><p>Severe malarial anaemia</p> </th></tr><tr><th><p>N&#160;(%)</p></th><th><p>N&#160;(%)</p></th><th><p>OR (95% CI)</p></th><th><p><italic>P</italic></p> </th><th><p>N&#160;(%)</p></th><th><p>OR (95% CI)</p></th><th><p><italic>P</italic></p> </th><th><p>N&#160;(%)</p></th><th><p>OR (95% CI)</p></th><th><p><italic>P</italic></p> </th></tr></thead><tbody><tr><td><p>Females</p></td><td><p>0</p></td><td valign="bottom"><p>3,761 (46.4)</p> </td><td valign="bottom"><p>2,299 (42.6)</p> </td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p>672 (42.3)</p> </td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p>400 (39.8)</p> </td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p>-&#8211;</p></td></tr><tr><td><p>All</p></td><td><p>(0&#8211;25)</p></td><td valign="bottom"><p>2,132 (26.3)</p> </td><td valign="bottom"><p>1,626 (30.1)</p> </td><td valign="bottom"><p>0.98 (0.9&#8211;1.08)</p> </td><td valign="bottom"><p>0.72</p></td><td valign="bottom"><p>495 (31.2)</p> </td><td valign="bottom"><p>1.02 (0.89&#8211;1.17)</p> </td><td valign="bottom"><p>0.81</p></td><td valign="bottom"><p>306 (30.4)</p> </td><td valign="bottom"><p>1.08 (0.9&#8211;1.29)</p> </td><td valign="bottom"><p>0.41</p></td></tr><tr><td/><td><p>(25&#8211;50)</p> </td><td valign="bottom"><p>2,001 (24.7)</p> </td><td valign="bottom"><p>1,310 (24.3)</p> </td><td valign="bottom"><p>0.88 (0.8&#8211;0.97)</p> </td><td valign="bottom"><p>8.59E-03</p></td><td valign="bottom"><p>377 (23.7)</p> </td><td valign="bottom"><p>0.83 (0.71&#8211;0.96)</p> </td><td valign="bottom"><p>1.04E-02</p></td><td valign="bottom"><p>252 (25.1)</p> </td><td valign="bottom"><p>1 (0.83&#8211;1.21)</p> </td><td valign="bottom"><p>0.99</p></td></tr><tr><td><p>All</p></td><td><p>(50&#8211;85)</p> </td><td valign="bottom"><p>210 (2.6)</p> </td><td valign="bottom"><p>156 (2.9)</p> </td><td valign="bottom"><p>1.02 (0.81&#8211;1.28)</p> </td><td valign="bottom"><p>0.85</p></td><td valign="bottom"><p>44 (2.8)</p> </td><td valign="bottom"><p>0.93 (0.66&#8211;1.31)</p> </td><td valign="bottom"><p>0.67</p></td><td valign="bottom"><p>45 (4.5)</p> </td><td valign="bottom"><p>1.71 (1.16&#8211;2.53)</p> </td><td valign="bottom"><p>6.86E-03</p></td></tr><tr><td/><td><p>&#8805;85</p> </td><td valign="bottom"><p>2 (0)</p> </td><td valign="bottom"><p>8 (0.1)</p> </td><td valign="bottom"><p>7.3 (1.43&#8211;37.24)</p> </td><td valign="bottom"><p>0.02</p></td><td valign="bottom"><p>0 (0)</p> </td><td valign="bottom"><p>n.c</p></td><td valign="bottom"><p>n.c.</p></td><td valign="bottom"><p>2 (0.2)</p> </td><td valign="bottom"><p>9.72 (1.01&#8211;93.12)</p> </td><td valign="bottom"><p>0.05</p></td></tr><tr><td/><td><p><bold>Overall</bold></p> </td><td valign="bottom"><p><bold>8,106</bold></p> </td><td valign="bottom"><p><bold>5,399</bold></p> </td><td valign="bottom"/><td valign="bottom"><p><bold>3.68E-03</bold></p> </td><td valign="bottom"><p><bold>1,588</bold></p> </td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p><bold>0.06</bold></p> </td><td valign="bottom"><p><bold>1,005</bold></p> </td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p><bold>0.03</bold></p> </td></tr><tr><td><p>Males</p></td><td><p>0</p></td><td valign="bottom"><p>5,960 (67.9)</p> </td><td valign="bottom"><p>4,216 (65.1)</p> </td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p>1,212 (68.4)</p> </td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p>707 (60)</p> </td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p>&#8211;</p></td></tr><tr><td><p>All</p></td><td><p>(0&#8211;25)</p></td><td valign="bottom"><p>1,572 (17.9)</p> </td><td valign="bottom"><p>1,303 (20.1)</p> </td><td valign="bottom"><p>1.08 (0.99&#8211;1.19)</p> </td><td valign="bottom"><p>0.08</p></td><td valign="bottom"><p>342 (19.3)</p> </td><td valign="bottom"><p>1.01 (0.88&#8211;1.16)</p> </td><td valign="bottom"><p>0.86</p></td><td valign="bottom"><p>227 (19.3)</p> </td><td valign="bottom"><p>1.05 (0.87&#8211;1.25)</p> </td><td valign="bottom"><p>0.63</p></td></tr><tr><td/><td><p>(25&#8211;50)</p> </td><td valign="bottom"><p>0 (0)</p> </td><td valign="bottom"><p>0 (0)</p> </td><td valign="bottom"><p>n.c</p></td><td valign="bottom"><p>n.c.</p></td><td valign="bottom"><p>0 (0)</p> </td><td valign="bottom"><p>n.c</p></td><td valign="bottom"><p>n.c.</p></td><td valign="bottom"><p>0 (0)</p> </td><td valign="bottom"><p>n.c</p></td><td valign="bottom"><p>n.c.</p></td></tr><tr><td><p>All</p></td><td><p>(50&#8211;85)</p> </td><td valign="bottom"><p>1,196 (13.6)</p> </td><td valign="bottom"><p>923 (14.3)</p> </td><td valign="bottom"><p>1.05 (0.95&#8211;1.16)</p> </td><td valign="bottom"><p>0.35</p></td><td valign="bottom"><p>214 (12.1)</p> </td><td valign="bottom"><p>0.79 (0.67&#8211;0.94)</p> </td><td valign="bottom"><p>6.31E-03</p></td><td valign="bottom"><p>236 (20)</p> </td><td valign="bottom"><p>1.44 (1.2&#8211;1.74)</p> </td><td valign="bottom"><p>1.17E-04</p></td></tr><tr><td/><td><p>&#8805;85</p> </td><td valign="bottom"><p>55 (0.6)</p> </td><td valign="bottom"><p>30 (0.5)</p> </td><td valign="bottom"><p>0.65 (0.4&#8211;1.04)</p> </td><td valign="bottom"><p>0.07</p></td><td valign="bottom"><p>3 (0.2)</p> </td><td valign="bottom"><p>0.26 (0.08&#8211;0.85)</p> </td><td valign="bottom"><p>0.03</p></td><td valign="bottom"><p>9 (0.8)</p> </td><td valign="bottom"><p>1.24 (0.58&#8211;2.67)</p> </td><td valign="bottom"><p>0.58</p></td></tr><tr><td/><td><p><bold>Overall</bold></p> </td><td valign="bottom"><p><bold>8,783</bold></p> </td><td valign="bottom"><p><bold>6,472</bold></p> </td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p><bold>0.08</bold></p> </td><td valign="bottom"><p><bold>1,771</bold></p> </td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p><bold>1.49E-03</bold></p> </td><td valign="bottom"><p><bold>1,179</bold></p> </td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p><bold>1.97E-03</bold></p> </td></tr><tr><td><p>All</p></td><td><p>0</p></td><td><p>9,721 (57.6)</p> </td><td><p>6,515 (54.9)</p> </td><td><p>&#8211;</p></td><td><p>&#8211;</p></td><td><p>1,884 (56.1)</p> </td><td><p>&#8211;</p></td><td><p>&#8211;</p></td><td><p>1,107 (50.7)</p> </td><td><p>&#8211;</p></td><td><p>&#8211;</p></td></tr><tr><td><p>All</p></td><td><p>(0&#8211;25)</p></td><td><p>3,704 (21.9)</p> </td><td><p>2,929 (24.7)</p> </td><td><p>1.05 (0.99&#8211;1.12)</p> </td><td><p>0.12</p></td><td><p>837 (24.9)</p> </td><td><p>1.03 (0.93&#8211;1.13)</p> </td><td><p>0.56</p></td><td><p>533 (24.4)</p> </td><td><p>1.05 (0.93&#8211;1.19)</p> </td><td><p>0.45</p></td></tr><tr><td/><td><p>(25&#8211;50)</p> </td><td><p>2,001 (11.8)</p> </td><td><p>1,310 (11)</p> </td><td><p>0.94 (0.86&#8211;1.03)</p> </td><td><p>0.16</p></td><td><p>377 (11.2)</p> </td><td><p>0.86 (0.75&#8211;0.99)</p> </td><td><p>0.04</p></td><td><p>252 (11.5)</p> </td><td><p>0.95 (0.8&#8211;1.14)</p> </td><td><p>0.6</p></td></tr><tr><td/><td><p>(50&#8211;85)</p> </td><td><p>1,406 (8.3)</p> </td><td><p>1,079 (9.1)</p> </td><td><p>1.02 (0.93&#8211;1.12)</p> </td><td><p>0.71</p></td><td><p>258 (7.7)</p> </td><td><p>0.8 (0.69&#8211;0.93)</p> </td><td><p>2.95E-03</p></td><td><p>281 (12.9)</p> </td><td><p>1.49 (1.26&#8211;1.76)</p> </td><td><p>2.22E-06</p></td></tr><tr><td/><td><p>&#8805;85</p> </td><td><p>57 (0.3)</p> </td><td><p>38 (0.3)</p> </td><td><p>0.87 (0.55&#8211;1.36)</p> </td><td><p>0.54</p></td><td><p>3 (0.1)</p> </td><td><p>0.28 (0.09&#8211;0.86)</p> </td><td><p>0.03</p></td><td><p>11 (0.5)</p> </td><td><p>1.44 (0.66&#8211;3.17)</p> </td><td><p>0.36</p></td></tr><tr><td><p><bold>All</bold></p> </td><td><p><bold>Overall</bold></p> </td><td><p><bold>16,889</bold></p> </td><td><p><bold>11,871</bold></p> </td><td><p>&#8211;</p></td><td><p><bold>0.14</bold></p> </td><td><p><bold>3,359</bold></p> </td><td><p>&#8211;</p></td><td><p><bold>2.29E-04</bold></p> </td><td><p><bold>2184</bold></p> </td><td><p>&#8211;</p></td><td><p><bold>1.02E-04</bold></p> </td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15085-v1.xml">elife-15085-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl7" position="float"><object-id pub-id-type="doi">10.7554/eLife.15085.016</object-id><label>Table 7.</label><caption><p>Fitness arrays suggested by observed OR estimates of effect at G6PD+202 on cerebral malaria and severe malarial anaemia. <italic>A<sub>1</sub></italic> represents the wild-type allele and <italic>A<sub>2</sub></italic> the derived allele. In females, odds ratios (ORs) for association with cerebral malaria in derived homozygotes and heterozygotes compared to wild-type homozygotes are 0.87 and&#160;~1, respectively, suggesting a symmetrical heterozygous advantage. In males, the OR for association with cerebral malaria in derived, compared to wild-type, hemizygotes is 0.82, suggesting a moderate fitness difference between male genotypes. For severe malarial anaemia, in females, the ORs in derived homozygotes and heterozygotes, compared to wild-type heterozygotes, are 1.84 and 1, respectively, suggesting selection against derived homozygotes; in males, the OR in derived&#160;hemizygotes, compared to wild-type, is 1.48, suggesting selection against derived male hemizygotes. The case fatality rate is estimated to be 20% (<italic>m</italic>&#160;=&#160;0.2) for cerebral and 10% (<italic>m</italic>&#160;=&#160;0.1) for severe malarial anaemia.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15085.016">http://dx.doi.org/10.7554/eLife.15085.016</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2" valign="top"><p>Genotype</p></th><th colspan="3" valign="top"><p>Females</p></th><th valign="top"/><th colspan="2" valign="top"><p>Males</p></th></tr><tr><th valign="top"><p><italic>A<sub>1</sub>A<sub>1</sub></italic></p> </th><th valign="top"><p><italic>A<sub>1</sub>A<sub>2</sub></italic></p> </th><th valign="top"><p><italic>A<sub>2</sub>A<sub>2</sub></italic></p> </th><th valign="top"/><th valign="top"><p><italic>A<sub>1</sub></italic></p> </th><th valign="top"><p><italic>A<sub>2</sub></italic></p> </th></tr><tr><th colspan="2" valign="top"/><th valign="top"><p><bold><italic>w</italic><sub>11</sub></bold></p> </th><th valign="top"><p><bold><italic>w</italic> <sub>12</sub></bold></p> </th><th valign="top"><p><bold><italic>w</italic> <sub>22</sub></bold></p> </th><th valign="top"/><th valign="top"><p><bold><italic>w</italic> <sub>1</sub></bold></p> </th><th valign="top"><p><bold><italic>w</italic> <sub>2</sub></bold></p> </th></tr></thead><tbody><tr><td colspan="4" valign="top"><p><bold>Cerebral malaria selection</bold></p> </td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"/><td valign="top"><p>Observed OR*</p></td><td valign="top"><p>1</p></td><td valign="top"><p>0.87 (0.75&#8211;1)</p> </td><td valign="top"><p>1.02 (0.7&#8211;1.48)</p> </td><td valign="top"/><td valign="top"><p>1</p></td><td valign="top"><p>0.82 (0.69&#8211;0.97)</p> </td></tr><tr><td valign="top"/><td valign="top"><p>Fitness Array<sup>&#8224;</sup></p></td><td valign="top"><p>1&#8211;<italic>s<sub>f</sub><sup>CM</sup></italic></p> </td><td valign="top"><p>1</p></td><td valign="top"><p>1&#8211;<italic>s<sub>f</sub><sup>CM</sup></italic></p> </td><td valign="top"/><td valign="top"><p>1&#8211;<italic>s<sub>m</sub><sup>CM</sup></italic></p> </td><td valign="top"><p>1</p></td></tr><tr><td valign="top"/><td valign="top"><p>Fitness (<bold><italic>w</italic>)</bold></p> </td><td valign="top"><p>0.974</p></td><td valign="top"><p>1</p></td><td valign="top"><p>0.974</p></td><td valign="top"/><td valign="top"><p>0.964</p></td><td valign="top"><p>1</p></td></tr><tr><td colspan="4" valign="top"><p><bold>Severe malarial anaemia selection</bold></p> </td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"/><td valign="top"><p>Observed OR*</p></td><td valign="top"><p>1</p></td><td valign="top"><p>0.97 (0.8&#8211;1.17)</p> </td><td valign="top"><p>1.84 (1.21&#8211;2.79)</p> </td><td valign="top"/><td valign="top"><p>1</p></td><td valign="top"><p>1.48 (1.22&#8211;1.8)</p> </td></tr><tr><td valign="top"/><td valign="top"><p>Fitness Array<sup>&#8225;</sup></p></td><td valign="top"><p>1</p></td><td valign="top"><p>1</p></td><td valign="top"><p>1&#8211;<italic>s<sub>f</sub><sup>SMA</sup></italic></p> </td><td valign="top"/><td valign="top"><p>1</p></td><td valign="top"><p>1&#8211;<italic>s<sub>m</sub><sup>SMA</sup></italic></p> </td></tr><tr><td valign="top"/><td valign="top"><p>Fitness (<bold><italic>w</italic>)</bold></p> </td><td valign="top"><p>1</p></td><td valign="top"><p>1</p></td><td valign="top"><p>0.954</p></td><td valign="top"/><td valign="top"><p>1</p></td><td valign="top"><p>0.968</p></td></tr></tbody></table><table-wrap-foot><fn id="tblfn9"><p>*Odds ratios are estimated from a genotypic model and are adjusted for sickle-cell trait and ethnicity.</p></fn><fn id="tblfn10"><p><sup>&#8224;</sup><italic>s<sub>f</sub><sup>CM</sup></italic> = <italic>m</italic>(1&#8211;OR)&#160;=&#160;0.2 (1 0.87)&#160;=&#160;0.26. <italic>s<sub>m</sub><sup>CM</sup></italic>&#160;=&#160;0.2(1&#8211;0.82)&#160;=&#160;0.036.</p></fn><fn id="tblfn11"><p><sup>&#8225;</sup><italic>s<sub>f</sub><sup>SMA</sup></italic> = <italic>m</italic> (1&#8211;1/OR)&#160;=&#160;0.1 (1&#8211;1/1.84)&#160;=&#160;0.046; <italic>s<sub>m</sub><sup>SMA</sup></italic> = m(1&#8211;1/OR)&#160;=&#160;0.1 (1 1/1.48)&#160;=&#160;0.032.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15085-v2.xml">elife-15085-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.15085.003</object-id><label>Table 2.</label><caption><p>WHO-classified G6PD deficiency alleles observed in this study.&#160;Locus refers to GRCh37, dbSNP137 and Ensembl build 84. WHO refers to grade of G6PD deficiency based on the&#160;WHO classification scheme. Allele frequency is calculated in population controls. GM Gambia; ML, Mali; BF, Burkina Faso; GH-N, Ghana (Noguchi); GH-K, Ghana (Kumasi); NG Nigeria; CM, Cameroon; KY, Kenya; TZ, Tanzania; MW, Malawi; VN, Vietnam; PNG, Papua New Guinea.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15085.003">http://dx.doi.org/10.7554/eLife.15085.003</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><p>SNP</p></th><th rowspan="2"><p>Locus</p></th><th rowspan="2"><p>Base change</p></th><th rowspan="2"><p>WHO</p></th><th colspan="12" valign="top"><p>Allele frequency</p> </th></tr><tr><th><p>GM</p></th><th><p>ML</p></th><th><p>BF</p></th><th><p>GH-N</p></th><th><p>GH-K</p></th><th><p>NG</p></th><th><p>CM</p></th><th><p>KY</p></th><th><p>TZ</p></th><th><p>MW</p></th><th><p>VN</p></th><th><p>PNG</p></th></tr></thead><tbody><tr><td valign="bottom"><p>CM973154</p></td><td valign="bottom"><p>153760261</p></td><td valign="bottom"><p>A/C</p></td><td valign="bottom"><p>I</p></td><td valign="bottom"><p>0.0004</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs72554665</p></td><td valign="bottom"><p>153760484</p></td><td valign="bottom"><p>C/A</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0004</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0055</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>CM920290</p></td><td valign="bottom"><p>153760605</p></td><td valign="bottom"><p>G/A</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0002</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0014</p></td><td valign="bottom"><p>0.0028</p></td></tr><tr><td valign="bottom"><p>rs137852342</p></td><td valign="bottom"><p>153761184</p></td><td valign="bottom"><p>G/A</p></td><td valign="bottom"><p>III</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0020</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0014</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs76723693</p></td><td valign="bottom"><p>153761240</p></td><td valign="bottom"><p>968T&gt;C</p></td><td valign="bottom"><p>III</p></td><td valign="bottom"><p>0.0695</p></td><td valign="bottom"><p>0.0082</p></td><td valign="bottom"><p>0.0009</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0003</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs137852327</p></td><td valign="bottom"><p>153761337</p></td><td valign="bottom"><p>C/T</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0006</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0096</p></td><td valign="bottom"><p>0.0141</p></td></tr><tr><td valign="bottom"><p>CM014189</p></td><td valign="bottom"><p>153761820</p></td><td valign="bottom"><p>T/A</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0004</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0008</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0004</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs137852328</p></td><td valign="bottom"><p>153762340</p></td><td valign="bottom"><p>680G&gt;T</p></td><td valign="bottom"><p>III</p></td><td valign="bottom"><p>0.0002</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs137852330</p></td><td valign="bottom"><p>153762605</p></td><td valign="bottom"><p>G/A</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0004</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0003</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs5030868</p></td><td valign="bottom"><p>153762634</p></td><td valign="bottom"><p>563G&gt;A</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0003</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0085</p></td></tr><tr><td valign="bottom"><p>rs5030872</p></td><td valign="bottom"><p>153762655</p></td><td valign="bottom"><p>542A&gt;T</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0145</p></td><td valign="bottom"><p>0.0041</p></td><td valign="bottom"><p>0.0009</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0003</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>CM970547</p></td><td valign="bottom"><p>153763462</p></td><td valign="bottom"><p>G/A</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0008</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td valign="bottom"><p>rs78365220</p></td><td valign="bottom"><p>153763485</p></td><td valign="bottom"><p>A/G</p></td><td valign="bottom"><p>II</p></td><td valign="bottom"><p>0.0002</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0366</p></td></tr><tr><td valign="bottom"><p>rs1050829</p></td><td valign="bottom"><p>153763492</p></td><td valign="bottom"><p>376T&gt;C</p></td><td valign="bottom"><p>IV</p></td><td valign="bottom"><p>0.3206</p></td><td valign="bottom"><p>0.4132</p></td><td valign="bottom"><p>0.4186</p></td><td valign="bottom"><p>0.4476</p></td><td valign="bottom"><p>0.4267</p></td><td valign="bottom"><p>0.5574</p></td><td valign="bottom"><p>0.3429</p></td><td valign="bottom"><p>0.4060</p></td><td valign="bottom"><p>0.3787</p></td><td valign="bottom"><p>0.3920</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr><tr><td><p>rs1050828</p></td><td><p>153764217</p></td><td><p>202C&gt;T</p></td><td><p>III</p></td><td valign="bottom"><p>0.0273</p></td><td valign="bottom"><p>0.1616</p></td><td valign="bottom"><p>0.1447</p></td><td valign="bottom"><p>0.1713</p></td><td valign="bottom"><p>0.1847</p></td><td valign="bottom"><p>0.3167</p></td><td valign="bottom"><p>0.1075</p></td><td valign="bottom"><p>0.1930</p></td><td valign="bottom"><p>0.2071</p></td><td valign="bottom"><p>0.1988</p></td><td valign="bottom"><p>0.0000</p></td><td valign="bottom"><p>0.0000</p></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15085-v2.xml">elife-15085-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.15085.012</object-id><label>Table 5.</label><caption><p>Calculation of the G6PD deficiency (G6PDd) score.&#160;The G6PDd score is calculated on the basis of the haplotypes at the 15 WHO-classified SNPs. For each haplotype, the loss of normal G6PD function is determined according to the WHO severity class (0, I, II, III or IV) of the most severe mutation carried. See Materials and methods for more details. The G6PDd score for each individual is then the average loss of function across the two haplotypes, where males are treated as homozygous females. For example, the assigned loss of normal G6PD function for a haplotype where the most severe mutation is the class III mutation G6PD+202T is 65, so an individual with this haplotype and another with no deficiency mutations will average a G6PDd score of 32.5.&#160;Individuals were further categorized on the basis of their G6PDd score as normal G6PDd (G6PDd score&#160;=&#160;0); Low G6PDd (0 &lt; G6PDd score&#160;&lt;&#160;25, blue boxes); Medium G6PDd (25 &#8804; G6PDd score&#160;&lt;&#160;50, yellow boxes); High G6PDd (50 &#8804; G6PDd score&#160;&lt;&#160;85, orange boxes); or Very High (G6PDd&#160;&#8805;&#160;85, red box). Scores not observed in our study are shown in grey boxes.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15085.012">http://dx.doi.org/10.7554/eLife.15085.012</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3" rowspan="3"><p>G6PDd score</p> </th><th colspan="5"><p>Haplotype 1</p></th></tr><tr><th colspan="5"><p>Severity class (loss of normal G6PD function)</p></th></tr><tr><th><p>0 (0)</p></th><th><p>IV (20)</p></th><th><p>III (65)</p></th><th><p>II (95)</p></th><th><p>I (100)</p></th></tr></thead><tbody><tr><td rowspan="5"><p><bold>Haplotype 2</bold></p> </td><td rowspan="5"><p><bold>Severity class</bold></p> <p><bold>(Loss of normal G6PD function)</bold></p> </td><td><p><bold>0&#160;(0)</bold></p> </td><td><p>0</p></td><td style="author-callout-style-b1"><p>10</p></td><td style="author-callout-style-b4"><p>32.5</p></td><td style="author-callout-style-b4"><p>47.5</p></td><td style="author-callout-style-b3"><p>50</p></td></tr><tr><td><p><bold>IV&#160;(20)</bold></p> </td><td style="author-callout-style-b1"><p>10</p></td><td style="author-callout-style-b1"><p>20</p></td><td style="author-callout-style-b4"><p>42.5</p></td><td style="author-callout-style-b3"><p>57.5</p></td><td style="author-callout-style-b8"><p>60</p></td></tr><tr><td><p><bold>III&#160;(65)</bold></p> </td><td style="author-callout-style-b4"><p>32.5</p></td><td style="author-callout-style-b4"><p>42.5</p></td><td style="author-callout-style-b3"><p>65</p></td><td style="author-callout-style-b3"><p>80</p></td><td style="author-callout-style-b8"><p>82.5</p></td></tr><tr><td><p><bold>II&#160;(95)</bold></p> </td><td style="author-callout-style-b4"><p>47.5</p></td><td style="author-callout-style-b3"><p>57.5</p></td><td style="author-callout-style-b3"><p>80</p></td><td style="author-callout-style-b6"><p>95</p></td><td style="author-callout-style-b8"><p>97.5</p></td></tr><tr><td><p><bold>I&#160;(100&#65279;</bold></p> </td><td style="author-callout-style-b3"><p>50</p></td><td style="author-callout-style-b8"><p>60</p></td><td style="author-callout-style-b8"><p>82.5</p></td><td style="author-callout-style-b8"><p>97.7</p></td><td style="author-callout-style-b8"><p>100</p></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15085-v2.xml">elife-15085-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl6" position="float"><object-id pub-id-type="doi">10.7554/eLife.15085.013</object-id><label>Table 6.</label><caption><p>G6PD&#160;deficiency score categorical model association test results.&#160;Counts, odds ratios (ORs), 95% confidence intervals (95% CI) and <italic>P</italic>-values are presented for the effect of each category of G6PD deficiency (G6PDd) score, compared to individuals with G6PDd score&#160;=&#160;0, for all&#160;severe malaria cases and for the two subtypes cerebral malaria and severe malarial anaemia. <italic>P</italic>-value shown in bold is for overall model fit. P-value for the overall model fit. Results are shown for all individuals combined at each site and across all sites. Results are adjusted for the&#160;sickle-cell trait, ethnicity and sex. n.c., not calculated.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15085.013">http://dx.doi.org/10.7554/eLife.15085.013</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><p>Sample</p></th><th rowspan="2"><p>G6PDd score category</p></th><th><p>Controls</p></th><th colspan="3"><p>Severe malaria</p> </th><th colspan="3"><p>Cerebral malaria</p> </th><th colspan="3"><p>Severe malarial anaemia</p> </th></tr><tr><th><p>N&#160;(%)</p></th><th><p>N&#160;(%)</p></th><th><p>OR (95% CI)</p></th><th><p><italic>P</italic></p> </th><th><p>N&#160;(%)</p></th><th><p>OR (95% CI)</p></th><th><p><italic>P</italic></p> </th><th><p>N&#160;(%)</p></th><th><p>OR (95% CI)</p></th><th><p><italic>P</italic></p> </th></tr></thead><tbody><tr><td><p>Females</p></td><td><p>0</p></td><td valign="bottom"><p>3,761 (46.4)</p> </td><td valign="bottom"><p>2,299 (42.6)</p> </td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p>672 (42.3)</p> </td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p>400 (39.8)</p> </td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p>-&#8211;</p></td></tr><tr><td><p>All</p></td><td><p>(0&#8211;25)</p></td><td valign="bottom"><p>2,132 (26.3)</p> </td><td valign="bottom"><p>1,626 (30.1)</p> </td><td valign="bottom"><p>0.98 (0.9&#8211;1.08)</p> </td><td valign="bottom"><p>0.72</p></td><td valign="bottom"><p>495 (31.2)</p> </td><td valign="bottom"><p>1.02 (0.89&#8211;1.17)</p> </td><td valign="bottom"><p>0.81</p></td><td valign="bottom"><p>306 (30.4)</p> </td><td valign="bottom"><p>1.08 (0.9&#8211;1.29)</p> </td><td valign="bottom"><p>0.41</p></td></tr><tr><td/><td><p>(25&#8211;50)</p> </td><td valign="bottom"><p>2,001 (24.7)</p> </td><td valign="bottom"><p>1,310 (24.3)</p> </td><td valign="bottom"><p>0.88 (0.8&#8211;0.97)</p> </td><td valign="bottom"><p>8.59E-03</p></td><td valign="bottom"><p>377 (23.7)</p> </td><td valign="bottom"><p>0.83 (0.71&#8211;0.96)</p> </td><td valign="bottom"><p>1.04E-02</p></td><td valign="bottom"><p>252 (25.1)</p> </td><td valign="bottom"><p>1 (0.83&#8211;1.21)</p> </td><td valign="bottom"><p>0.99</p></td></tr><tr><td><p>All</p></td><td><p>(50&#8211;85)</p> </td><td valign="bottom"><p>210 (2.6)</p> </td><td valign="bottom"><p>156 (2.9)</p> </td><td valign="bottom"><p>1.02 (0.81&#8211;1.28)</p> </td><td valign="bottom"><p>0.85</p></td><td valign="bottom"><p>44 (2.8)</p> </td><td valign="bottom"><p>0.93 (0.66&#8211;1.31)</p> </td><td valign="bottom"><p>0.67</p></td><td valign="bottom"><p>45 (4.5)</p> </td><td valign="bottom"><p>1.71 (1.16&#8211;2.53)</p> </td><td valign="bottom"><p>6.86E-03</p></td></tr><tr><td/><td><p>&#8805;85</p> </td><td valign="bottom"><p>2 (0)</p> </td><td valign="bottom"><p>8 (0.1)</p> </td><td valign="bottom"><p>7.3 (1.43&#8211;37.24)</p> </td><td valign="bottom"><p>0.02</p></td><td valign="bottom"><p>0 (0)</p> </td><td valign="bottom"><p>n.c</p></td><td valign="bottom"><p>n.c.</p></td><td valign="bottom"><p>2 (0.2)</p> </td><td valign="bottom"><p>9.72 (1.01&#8211;93.12)</p> </td><td valign="bottom"><p>0.05</p></td></tr><tr><td/><td><p><bold>Overall</bold></p> </td><td valign="bottom"><p><bold>8,106</bold></p> </td><td valign="bottom"><p><bold>5,399</bold></p> </td><td valign="bottom"/><td valign="bottom"><p><bold>3.68E-03</bold></p> </td><td valign="bottom"><p><bold>1,588</bold></p> </td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p><bold>0.06</bold></p> </td><td valign="bottom"><p><bold>1,005</bold></p> </td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p><bold>0.03</bold></p> </td></tr><tr><td><p>Males</p></td><td><p>0</p></td><td valign="bottom"><p>5,960 (67.9)</p> </td><td valign="bottom"><p>4,216 (65.1)</p> </td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p>1,212 (68.4)</p> </td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p>707 (60)</p> </td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p>&#8211;</p></td></tr><tr><td><p>All</p></td><td><p>(0&#8211;25)</p></td><td valign="bottom"><p>1,572 (17.9)</p> </td><td valign="bottom"><p>1,303 (20.1)</p> </td><td valign="bottom"><p>1.08 (0.99&#8211;1.19)</p> </td><td valign="bottom"><p>0.08</p></td><td valign="bottom"><p>342 (19.3)</p> </td><td valign="bottom"><p>1.01 (0.88&#8211;1.16)</p> </td><td valign="bottom"><p>0.86</p></td><td valign="bottom"><p>227 (19.3)</p> </td><td valign="bottom"><p>1.05 (0.87&#8211;1.25)</p> </td><td valign="bottom"><p>0.63</p></td></tr><tr><td/><td><p>(25&#8211;50)</p> </td><td valign="bottom"><p>0 (0)</p> </td><td valign="bottom"><p>0 (0)</p> </td><td valign="bottom"><p>n.c</p></td><td valign="bottom"><p>n.c.</p></td><td valign="bottom"><p>0 (0)</p> </td><td valign="bottom"><p>n.c</p></td><td valign="bottom"><p>n.c.</p></td><td valign="bottom"><p>0 (0)</p> </td><td valign="bottom"><p>n.c</p></td><td valign="bottom"><p>n.c.</p></td></tr><tr><td><p>All</p></td><td><p>(50&#8211;85)</p> </td><td valign="bottom"><p>1,196 (13.6)</p> </td><td valign="bottom"><p>923 (14.3)</p> </td><td valign="bottom"><p>1.05 (0.95&#8211;1.16)</p> </td><td valign="bottom"><p>0.35</p></td><td valign="bottom"><p>214 (12.1)</p> </td><td valign="bottom"><p>0.79 (0.67&#8211;0.94)</p> </td><td valign="bottom"><p>6.31E-03</p></td><td valign="bottom"><p>236 (20)</p> </td><td valign="bottom"><p>1.44 (1.2&#8211;1.74)</p> </td><td valign="bottom"><p>1.17E-04</p></td></tr><tr><td/><td><p>&#8805;85</p> </td><td valign="bottom"><p>55 (0.6)</p> </td><td valign="bottom"><p>30 (0.5)</p> </td><td valign="bottom"><p>0.65 (0.4&#8211;1.04)</p> </td><td valign="bottom"><p>0.07</p></td><td valign="bottom"><p>3 (0.2)</p> </td><td valign="bottom"><p>0.26 (0.08&#8211;0.85)</p> </td><td valign="bottom"><p>0.03</p></td><td valign="bottom"><p>9 (0.8)</p> </td><td valign="bottom"><p>1.24 (0.58&#8211;2.67)</p> </td><td valign="bottom"><p>0.58</p></td></tr><tr><td/><td><p><bold>Overall</bold></p> </td><td valign="bottom"><p><bold>8,783</bold></p> </td><td valign="bottom"><p><bold>6,472</bold></p> </td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p><bold>0.08</bold></p> </td><td valign="bottom"><p><bold>1,771</bold></p> </td><td valign="bottom"><p>-&#8211;</p></td><td valign="bottom"><p><bold>1.49E-03</bold></p> </td><td valign="bottom"><p><bold>1,179</bold></p> </td><td valign="bottom"><p>&#8211;</p></td><td valign="bottom"><p><bold>1.97E-03</bold></p> </td></tr><tr><td><p>All</p></td><td><p>0</p></td><td><p>9,721 (57.6)</p> </td><td><p>6,515 (54.9)</p> </td><td><p>&#8211;</p></td><td><p>&#8211;</p></td><td><p>1,884 (56.1)</p> </td><td><p>&#8211;</p></td><td><p>&#8211;</p></td><td><p>1,107 (50.7)</p> </td><td><p>&#8211;</p></td><td><p>&#8211;</p></td></tr><tr><td><p>All</p></td><td><p>(0&#8211;25)</p></td><td><p>3,704 (21.9)</p> </td><td><p>2,929 (24.7)</p> </td><td><p>1.05 (0.99&#8211;1.12)</p> </td><td><p>0.12</p></td><td><p>837 (24.9)</p> </td><td><p>1.03 (0.93&#8211;1.13)</p> </td><td><p>0.56</p></td><td><p>533 (24.4)</p> </td><td><p>1.05 (0.93&#8211;1.19)</p> </td><td><p>0.45</p></td></tr><tr><td/><td><p>(25&#8211;50)</p> </td><td><p>2,001 (11.8)</p> </td><td><p>1,310 (11)</p> </td><td><p>0.94 (0.86&#8211;1.03)</p> </td><td><p>0.16</p></td><td><p>377 (11.2)</p> </td><td><p>0.86 (0.75&#8211;0.99)</p> </td><td><p>0.04</p></td><td><p>252 (11.5)</p> </td><td><p>0.95 (0.8&#8211;1.14)</p> </td><td><p>0.6</p></td></tr><tr><td/><td><p>(50&#8211;85)</p> </td><td><p>1,406 (8.3)</p> </td><td><p>1,079 (9.1)</p> </td><td><p>1.02 (0.93&#8211;1.12)</p> </td><td><p>0.71</p></td><td><p>258 (7.7)</p> </td><td><p>0.8 (0.69&#8211;0.93)</p> </td><td><p>2.95E-03</p></td><td><p>281 (12.9)</p> </td><td><p>1.49 (1.26&#8211;1.76)</p> </td><td><p>2.22E-06</p></td></tr><tr><td/><td><p>&#8805;85</p> </td><td><p>57 (0.3)</p> </td><td><p>38 (0.3)</p> </td><td><p>0.87 (0.55&#8211;1.36)</p> </td><td><p>0.54</p></td><td><p>3 (0.1)</p> </td><td><p>0.28 (0.09&#8211;0.86)</p> </td><td><p>0.03</p></td><td><p>11 (0.5)</p> </td><td><p>1.44 (0.66&#8211;3.17)</p> </td><td><p>0.36</p></td></tr><tr><td><p><bold>All</bold></p> </td><td><p><bold>Overall</bold></p> </td><td><p><bold>16,889</bold></p> </td><td><p><bold>11,871</bold></p> </td><td><p>&#8211;</p></td><td><p><bold>0.14</bold></p> </td><td><p><bold>3,359</bold></p> </td><td><p>&#8211;</p></td><td><p><bold>2.29E-04</bold></p> </td><td><p><bold>2184</bold></p> </td><td><p>&#8211;</p></td><td><p><bold>1.02E-04</bold></p> </td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15085-v2.xml">elife-15085-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl7" position="float"><object-id pub-id-type="doi">10.7554/eLife.15085.016</object-id><label>Table 7.</label><caption><p>Fitness arrays suggested by observed OR estimates of effect at G6PD+202 on cerebral malaria and severe malarial anaemia. <italic>A<sub>1</sub></italic> represents the wild-type allele and <italic>A<sub>2</sub></italic> the derived allele. In females, odds ratios (ORs) for association with cerebral malaria in derived homozygotes and heterozygotes compared to wild-type homozygotes are 0.87 and&#160;~1, respectively, suggesting a symmetrical heterozygous advantage. In males, the OR for association with cerebral malaria in derived, compared to wild-type, hemizygotes is 0.82, suggesting a moderate fitness difference between male genotypes. For severe malarial anaemia, in females, the ORs in derived homozygotes and heterozygotes, compared to wild-type heterozygotes, are 1.84 and 1, respectively, suggesting selection against derived homozygotes; in males, the OR in derived&#160;hemizygotes, compared to wild-type, is 1.48, suggesting selection against derived male hemizygotes. The case fatality rate is estimated to be 20% (<italic>m</italic>&#160;=&#160;0.2) for cerebral and 10% (<italic>m</italic>&#160;=&#160;0.1) for severe malarial anaemia.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15085.016">http://dx.doi.org/10.7554/eLife.15085.016</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2" valign="top"><p>Genotype</p></th><th colspan="3" valign="top"><p>Females</p></th><th valign="top"/><th colspan="2" valign="top"><p>Males</p></th></tr><tr><th valign="top"><p><italic>A<sub>1</sub>A<sub>1</sub></italic></p> </th><th valign="top"><p><italic>A<sub>1</sub>A<sub>2</sub></italic></p> </th><th valign="top"><p><italic>A<sub>2</sub>A<sub>2</sub></italic></p> </th><th valign="top"/><th valign="top"><p><italic>A<sub>1</sub></italic></p> </th><th valign="top"><p><italic>A<sub>2</sub></italic></p> </th></tr><tr><th colspan="2" valign="top"/><th valign="top"><p><bold><italic>w</italic><sub>11</sub></bold></p> </th><th valign="top"><p><bold><italic>w</italic> <sub>12</sub></bold></p> </th><th valign="top"><p><bold><italic>w</italic> <sub>22</sub></bold></p> </th><th valign="top"/><th valign="top"><p><bold><italic>w</italic> <sub>1</sub></bold></p> </th><th valign="top"><p><bold><italic>w</italic> <sub>2</sub></bold></p> </th></tr></thead><tbody><tr><td colspan="4" valign="top"><p><bold>Cerebral malaria selection</bold></p> </td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"/><td valign="top"><p>Observed OR*</p></td><td valign="top"><p>1</p></td><td valign="top"><p>0.87 (0.75&#8211;1)</p> </td><td valign="top"><p>1.02 (0.7&#8211;1.48)</p> </td><td valign="top"/><td valign="top"><p>1</p></td><td valign="top"><p>0.82 (0.69&#8211;0.97)</p> </td></tr><tr><td valign="top"/><td valign="top"><p>Fitness Array<sup>&#8224;</sup></p></td><td valign="top"><p>1&#8211;<italic>s<sub>f</sub><sup>CM</sup></italic></p> </td><td valign="top"><p>1</p></td><td valign="top"><p>1&#8211;<italic>s<sub>f</sub><sup>CM</sup></italic></p> </td><td valign="top"/><td valign="top"><p>1&#8211;<italic>s<sub>m</sub><sup>CM</sup></italic></p> </td><td valign="top"><p>1</p></td></tr><tr><td valign="top"/><td valign="top"><p>Fitness (<bold><italic>w</italic>)</bold></p> </td><td valign="top"><p>0.974</p></td><td valign="top"><p>1</p></td><td valign="top"><p>0.974</p></td><td valign="top"/><td valign="top"><p>0.964</p></td><td valign="top"><p>1</p></td></tr><tr><td colspan="4" valign="top"><p><bold>Severe malarial anaemia selection</bold></p> </td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"/><td valign="top"><p>Observed OR*</p></td><td valign="top"><p>1</p></td><td valign="top"><p>0.97 (0.8&#8211;1.17)</p> </td><td valign="top"><p>1.84 (1.21&#8211;2.79)</p> </td><td valign="top"/><td valign="top"><p>1</p></td><td valign="top"><p>1.48 (1.22&#8211;1.8)</p> </td></tr><tr><td valign="top"/><td valign="top"><p>Fitness Array<sup>&#8225;</sup></p></td><td valign="top"><p>1</p></td><td valign="top"><p>1</p></td><td valign="top"><p>1&#8211;<italic>s<sub>f</sub><sup>SMA</sup></italic></p> </td><td valign="top"/><td valign="top"><p>1</p></td><td valign="top"><p>1&#8211;<italic>s<sub>m</sub><sup>SMA</sup></italic></p> </td></tr><tr><td valign="top"/><td valign="top"><p>Fitness (<bold><italic>w</italic>)</bold></p> </td><td valign="top"><p>1</p></td><td valign="top"><p>1</p></td><td valign="top"><p>0.954</p></td><td valign="top"/><td valign="top"><p>1</p></td><td valign="top"><p>0.968</p></td></tr></tbody></table><table-wrap-foot><fn id="tblfn9"><p>*Odds ratios are estimated from a genotypic model and are adjusted for sickle-cell trait and ethnicity.</p></fn><fn id="tblfn10"><p><sup>&#8224;</sup><italic>s<sub>f</sub><sup>CM</sup></italic> = <italic>m</italic>(1&#8211;OR)&#160;=&#160;0.2 (1 0.87)&#160;=&#160;0.26. <italic>s<sub>m</sub><sup>CM</sup></italic>&#160;=&#160;0.2(1&#8211;0.82)&#160;=&#160;0.036.</p></fn><fn id="tblfn11"><p><sup>&#8225;</sup><italic>s<sub>f</sub><sup>SMA</sup></italic> = <italic>m</italic> (1&#8211;1/OR)&#160;=&#160;0.1 (1&#8211;1/1.84)&#160;=&#160;0.046; <italic>s<sub>m</sub><sup>SMA</sup></italic> = m(1&#8211;1/OR)&#160;=&#160;0.1 (1 1/1.48)&#160;=&#160;0.032.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15155-v1.xml">elife-15155-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.15155.024</object-id><label>Table 1.</label><caption><p>Fitting parameters for the gap-derived strand memory experiment.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15155.024">http://dx.doi.org/10.7554/eLife.15155.024</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="3"><italic>%gap</italic> [95% Confidence Interval]</th><th rowspan="2"/><th colspan="3"><italic>%A&#8594;G repair</italic> [95% Confidence Interval]</th></tr><tr><th><italic>t<sub>1/2</sub></italic> (min)</th><th><italic>%gap</italic> at 0 min</th><th>R<sup>2</sup></th><th><italic>t<sub>1/2</sub></italic> (min)</th><th><italic>%repair</italic> at 0 min</th><th>R<sup>2</sup></th></tr></thead><tbody><tr><td>&#916;MutS/MutL</td><td>0.60 [0.51&#8722;0.73]</td><td>100 [96.7&#8722;103.3]</td><td>0.99</td><td/><td>1.87 [1.52&#8722;2.45]</td><td>74.8 [68.6&#8722;80.7]</td><td>0.97</td></tr><tr><td>&#916;MutS</td><td>0.68 [0.58&#8722;0.81]</td><td>100 [96.4&#8722;103.5]</td><td>0.99</td><td/><td>2.00 [1.76&#8722;2.31]</td><td>82.8 [79.0&#8722;86.6]</td><td>0.99</td></tr><tr><td>&#916;MutL</td><td>0.75 [0.66&#8722;0.88]</td><td>100 [96.5&#8722;103.5]</td><td>0.99</td><td/><td>39.6 [12.3&#8722; +&#8734;]</td><td>72.8 [63.6&#8722;81.9]</td><td>0.80</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>Parameters for the one-phase exponential decay fitting of the data described in <xref ref-type="fig" rid="fig7">Figure 7B and D</xref> are presented. <italic>%gap</italic>: percentage of remaining gaps (100 - <italic>%closed</italic>), <italic>t<sub>1/2</sub></italic>: half-life, <italic>R<sup>2</sup></italic>: coefficient of determination. <italic>n</italic> = 3. Curve fitting was carried out using the GraphPad Prism 6 software (GraphPad Software, CA, USA)</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15155-v1.xml">elife-15155-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.15155.029</object-id><label>Table 2.</label><caption><p>Fitting parameters for the gap-derived strand memory experiment with MutS&#945; mutants</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15155.029">http://dx.doi.org/10.7554/eLife.15155.029</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="3"><italic>%gap&#160;</italic>[95% Confidence Interval]</th><th colspan="3"><italic>%A&#8594;G repair</italic> [95% Confidence Interval]</th></tr><tr><th><italic>t<sub>1/2</sub></italic> (min)</th><th><italic>%gap</italic> at 0 min</th><th>R<sup>2</sup></th><th><italic>t<sub>1/2</sub></italic> (min)</th><th><italic>%repair</italic> at 0 min</th><th>R<sup>2</sup></th></tr></thead><tbody><tr><td>+buffer</td><td>1.44 [0.51&#8722;0.73]</td><td>100 [95.2&#8722;104.7]</td><td>0.99</td><td>2.94 [2.32&#8722;3.98]</td><td>75.7 [69.2&#8722;82.2]</td><td>0.97</td></tr><tr><td>+xMutS&#945;<sup>WT</sup></td><td>1.35 [0.58&#8722;0.81]</td><td>100 [95.1&#8722;104.9]</td><td>0.99</td><td>25.5 [17.6&#8722;46.1]</td><td>84.8 [80.5&#8722;89.2]</td><td>0.98</td></tr><tr><td>+xMutS&#945;<sup>PIP</sup></td><td>1.48 [1.18&#8722;2.00]</td><td>100 [95.4&#8722;104.6]</td><td>0.99</td><td>12.5 [10.9&#8722;14.7]</td><td>81.9 [78.5&#8722;85.4]</td><td>0.99</td></tr><tr><td>+xMutS&#945;<sup>&#916;N</sup></td><td>1.43 [1.14&#8722;1.93]</td><td>100 [95.6&#8722;104.4]</td><td>0.99</td><td>12.3 [11.3&#8722;13.6]</td><td>81.9 [79.8&#8722;84.0]</td><td>1.00</td></tr></tbody></table><table-wrap-foot><fn id="tblfn2"><p>Parameters for the one-phase exponential decay fitting of the data described in <xref ref-type="fig" rid="fig9">Figure 9B and D</xref> are presented. <italic>%gap</italic>: percentage of remaining gaps (100 - <italic>%closed</italic>), <italic>t<sub>1/2</sub></italic>: half-life, <italic>R<sup>2</sup></italic>: coefficient of determination. <italic>n</italic> = 3.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15691-v1.xml">elife-15691-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.15691.003</object-id><label>Table 1.</label><caption><p>Clinical and tumor characteristics of patients treated with cTACE and TILA-TACE in the nonrandomized study.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15691.003">http://dx.doi.org/10.7554/eLife.15691.003</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Variables</th><th colspan="2">Patients</th></tr><tr><th valign="top">TILA-TACE</th><th valign="top">cTACE</th></tr></thead><tbody><tr><td>Patient number</td><td>30</td><td>27</td></tr><tr><td>Median age, years</td><td>57 (Range 32&#8211;81)</td><td>54 (Range 37&#8211;81)</td></tr><tr><td>Gender (M/F)</td><td>27/3 (90.0%/10.0%)</td><td>27/0 (100%/0%)</td></tr><tr><td>Aetiology</td><td/><td/></tr><tr><td>&#8195;HBV</td><td>24 (80.0%)</td><td>25&#160;(92.6%)</td></tr><tr><td>&#8195;HCV</td><td>0 (0%)</td><td>0 (0%)</td></tr><tr><td>&#8195;Non B-non C</td><td>6 (20.0%)</td><td>2&#160;(7.4%)</td></tr><tr><td>Cirrhosis (radiology)</td><td>30 (100%)</td><td>27&#160;(100%)</td></tr><tr><td>Bilirubin, &#956;M</td><td>16.9 &#177; 9.4</td><td>22.5 &#177; 11.6</td></tr><tr><td>Albumin, g/L</td><td>39.0 &#177; 6.9</td><td>37.4 &#177; 5.3</td></tr><tr><td>AST, U/L</td><td>74.9 &#177; 102.3</td><td>83.5 &#177; 54.1</td></tr><tr><td>ALT, U/L</td><td>54.1 &#177; 80.4</td><td>67.3 &#177; 43.5</td></tr><tr><td>AFP, &gt;400&#160;ng/mL</td><td>9 (30.0%)</td><td>15&#160;(55.6%)</td></tr><tr><td>Child-Pugh class, A/B</td><td>27/3 (90.0%/10.0%)</td><td>25/2&#160;(92.6%/7.4%)</td></tr><tr><td>The size of largest tumor (cm)</td><td>9.2 (range 5.0&#8211;13.6)</td><td>10.3 (range 5.0&#8211;14.6)</td></tr><tr><td>&#8195;Tumor &gt;10 cm</td><td>14 (46.7%)</td><td>15&#160;(55.6%)</td></tr><tr><td>&#8195;Tumor 5~10&#160;cm</td><td>16 (53.3%)</td><td>12&#160;(44.4%)</td></tr><tr><td>Multifocal tumors in 1 lobe</td><td>8 (26.7%)</td><td>12&#160;(44.4%)</td></tr><tr><td>Multifocal tumors in 2 lobes</td><td>8 (26.7%)</td><td>12&#160;(44.4%)</td></tr><tr><td>BCLC stage</td><td/><td/></tr><tr><td>&#8195;B</td><td>19 (63.3%)</td><td>18&#160;(66.7%)</td></tr><tr><td>&#8195;C</td><td>11 (36.7%)</td><td>9&#160;(33.3%)</td></tr><tr><td>Macrovascular invasion</td><td>5 (16.7%)</td><td>4&#160;(14.8%)</td></tr><tr><td>&#8195;The right branch of portal vein</td><td>4 (13.3%)</td><td>2&#160;(7.4%)</td></tr><tr><td>&#8195;Hepatic vein</td><td>&#8211;</td><td>1&#160;(3.7%)</td></tr><tr><td>&#8195;The right branch of portal + hepatic vein</td><td>1 (3.3%)</td><td>1&#160;(3.7%)</td></tr><tr><td>Extra-hepatic metastasis</td><td>8 (26.7%)</td><td>8&#160;(29.6%)</td></tr><tr><td>&#8195;Lung</td><td>1 (3.3%)</td><td>6&#160;(22.2%)</td></tr><tr><td>&#8195;Lung + bone</td><td>1 (3.3%)</td><td>0 (0%)</td></tr><tr><td>&#8195;Soft tissue</td><td>&#8211;</td><td>0 (0%)</td></tr><tr><td>&#8195;Lymph nodes</td><td>5 (12.2%)</td><td>0 (0%)</td></tr><tr><td>&#8195;Bone</td><td>1 (2.0%)</td><td>1&#160;(3.7%)</td></tr><tr><td>&#8195;Bone+lymph node</td><td>&#8211;</td><td>1&#160;(3.7%)</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>HBV, hepatitis B virius;</p></fn><fn id="tblfn2"><p>HCV, hepatitis C virius;</p></fn><fn id="tblfn3"><p>AST, Aspartate transaminase;</p></fn><fn id="tblfn4"><p>ALT, Alanine aminotransferase;</p></fn><fn id="tblfn5"><p>AFP, alpha-feto-protein.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15691-v1.xml">elife-15691-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.15691.009</object-id><label>Table 3.</label><caption><p>Clinical and tumor characteristics of patients in RCT.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15691.009">http://dx.doi.org/10.7554/eLife.15691.009</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Variables</th><th colspan="2">Patients</th></tr><tr><th valign="top">TILA-TACE</th><th valign="top">cTACE</th></tr></thead><tbody><tr><td valign="top">Patient number</td><td valign="top">10</td><td valign="top">10</td></tr><tr><td valign="top">Median age, years</td><td valign="top">58 (Range 40&#8211;86)</td><td valign="top">53 (43&#8211;81)</td></tr><tr><td valign="top">Gender (M/F)</td><td valign="top">9 /1 (90.0%/10.0%)</td><td valign="top">7 /3 (70.0%/ 30.0%)</td></tr><tr><td valign="top">Aetiology</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">&#8195;HBV</td><td valign="top">9 (90.0%)</td><td valign="top">8 (80.0%)</td></tr><tr><td valign="top">&#8195;HCV</td><td valign="top">0 (0%)</td><td valign="top">0 (0%)</td></tr><tr><td valign="top">&#8195;Non B-non C</td><td valign="top">1 (10.0%)</td><td valign="top">2 (20.0%)</td></tr><tr><td valign="top">Cirrhosis (radiology)</td><td valign="top">10 (100%)</td><td valign="top">10 (100%)</td></tr><tr><td valign="top">Bilirubin, &#956;M</td><td valign="top">17.2 &#177; 10.1</td><td valign="top">16.7 &#177; 7.6</td></tr><tr><td valign="top">Albumin, g/L</td><td valign="top">38.7 &#177; 3.1</td><td valign="top">38.1 &#177; 5.2</td></tr><tr><td valign="top">AST, U/L</td><td valign="top">64.8 &#177; 44.8</td><td valign="top">52.8 &#177; 19.2</td></tr><tr><td valign="top">ALT, U/L</td><td valign="top">60.6 &#177; 48.0</td><td valign="top">41.0 &#177; 29.9</td></tr><tr><td valign="top">AFP, &gt;400&#160;ng/mL</td><td valign="top">3 (30.0%)</td><td valign="top">4 (40.0%)</td></tr><tr><td valign="top">Child-Pugh class, A/B</td><td valign="top">10/0 (100%/0%)</td><td valign="top">10/0 (100%/0%)</td></tr><tr><td valign="top">The size of largest tumor (cm)</td><td valign="top">7.9 (range 5.0&#8211;13.5)</td><td valign="top">7.5 (range 5.0&#8211;13.0)</td></tr><tr><td valign="top">&#8195;Tumor &gt;10 cm</td><td valign="top">3 (30.0%)</td><td valign="top">3&#160;(30.0%)</td></tr><tr><td valign="top">&#8195;Tumor 5~10&#160;cm</td><td valign="top">7 (70.0%)</td><td valign="top">7&#160;(70.0%)</td></tr><tr><td valign="top">Multifocal tumors in 1 lobe</td><td valign="top">3 (30.0%)</td><td valign="top">3 (30.0%)</td></tr><tr><td valign="top">Multifocal tumors in 2 lobes</td><td valign="top">4 (40.0%)</td><td valign="top">2 (20.0%)</td></tr><tr><td valign="top">BCLC stage</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">&#8195;B</td><td valign="top">7 (70.0%)</td><td valign="top">8 (70.0%)</td></tr><tr><td valign="top">&#8195;C</td><td valign="top">3 (30.0%)</td><td valign="top">2 (20.0%)</td></tr><tr><td valign="top">Macrovascular invasion</td><td valign="top">2 (20.0%)</td><td valign="top">1&#160;(10.0%)</td></tr><tr><td valign="top">&#8195;The right branch of portal vein</td><td valign="top">1&#160;(10.0%)</td><td valign="top">1&#160;(10.0%)</td></tr><tr><td valign="top">&#8195;The left branch of portal vein</td><td valign="top">1&#160;(10.0%)</td><td valign="top">0 (0%)</td></tr><tr><td valign="top">Extra-hepatic metastasis</td><td valign="top">1&#160;(10.0%)</td><td valign="top">2&#160;(20.0%)</td></tr><tr><td valign="top">&#8195;Brain</td><td valign="top">0 (0%)</td><td valign="top">1&#160;(10.0%)</td></tr><tr><td valign="top">&#8195;Lymph nodes</td><td valign="top">1&#160;(10.0%)</td><td valign="top">1 (10.0%)</td></tr></tbody></table><table-wrap-foot><fn id="tblfn10"><p>HBV, hepatitis B virius;</p></fn><fn id="tblfn11"><p>HCV, hepatitis C virius;</p></fn><fn id="tblfn12"><p>AST, Aspartate transaminase;</p></fn><fn id="tblfn13"><p>ALT, Alanine aminotransferase;</p></fn><fn id="tblfn14"><p>AFP, alpha-feto-protein.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15691-v2.xml">elife-15691-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Clinical and tumor characteristics of patients treated with cTACE and TILA-TACE in the nonrandomized study.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Variables</th><th colspan="2">Patients</th></tr><tr><th valign="top">TILA-TACE</th><th valign="top">cTACE</th></tr></thead><tbody><tr><td>Patient number</td><td>30</td><td>27</td></tr><tr><td>Median age, years</td><td>57 (Range 32&#8211;81)</td><td>54 (Range 37&#8211;81)</td></tr><tr><td>Gender (M/F)</td><td>27/3 (90.0%/10.0%)</td><td>27/0 (100%/0%)</td></tr><tr><td>Aetiology</td><td/><td/></tr><tr><td>&#8195;HBV</td><td>24 (80.0%)</td><td>25&#160;(92.6%)</td></tr><tr><td>&#8195;HCV</td><td>0 (0%)</td><td>0 (0%)</td></tr><tr><td>&#8195;Non B-non C</td><td>6 (20.0%)</td><td>2&#160;(7.4%)</td></tr><tr><td>Cirrhosis (radiology)</td><td>30 (100%)</td><td>27&#160;(100%)</td></tr><tr><td>Bilirubin, &#956;M</td><td>16.9 &#177; 9.4</td><td>22.5 &#177; 11.6</td></tr><tr><td>Albumin, g/L</td><td>39.0 &#177; 6.9</td><td>37.4 &#177; 5.3</td></tr><tr><td>AST, U/L</td><td>74.9 &#177; 102.3</td><td>83.5 &#177; 54.1</td></tr><tr><td>ALT, U/L</td><td>54.1 &#177; 80.4</td><td>67.3 &#177; 43.5</td></tr><tr><td>AFP, &gt;400&#160;ng/mL</td><td>9 (30.0%)</td><td>15&#160;(55.6%)</td></tr><tr><td>Child-Pugh class, A/B</td><td>27/3 (90.0%/10.0%)</td><td>25/2&#160;(92.6%/7.4%)</td></tr><tr><td>The size of largest tumor (cm)</td><td>9.2 (range 5.0&#8211;13.6)</td><td>10.3 (range 5.0&#8211;14.6)</td></tr><tr><td>&#8195;Tumor &gt;10 cm</td><td>14 (46.7%)</td><td>15&#160;(55.6%)</td></tr><tr><td>&#8195;Tumor 5~10&#160;cm</td><td>16 (53.3%)</td><td>12&#160;(44.4%)</td></tr><tr><td>Multifocal tumors in 1 lobe</td><td>8 (26.7%)</td><td>12&#160;(44.4%)</td></tr><tr><td>Multifocal tumors in 2 lobes</td><td>8 (26.7%)</td><td>12&#160;(44.4%)</td></tr><tr><td>BCLC stage</td><td/><td/></tr><tr><td>&#8195;B</td><td>19 (63.3%)</td><td>18&#160;(66.7%)</td></tr><tr><td>&#8195;C</td><td>11 (36.7%)</td><td>9&#160;(33.3%)</td></tr><tr><td>Macrovascular invasion</td><td>5 (16.7%)</td><td>4&#160;(14.8%)</td></tr><tr><td>&#8195;The right branch of portal vein</td><td>4 (13.3%)</td><td>2&#160;(7.4%)</td></tr><tr><td>&#8195;Hepatic vein</td><td>&#8211;</td><td>1&#160;(3.7%)</td></tr><tr><td>&#8195;The right branch of portal + hepatic vein</td><td>1 (3.3%)</td><td>1&#160;(3.7%)</td></tr><tr><td>Extra-hepatic metastasis</td><td>8 (26.7%)</td><td>8&#160;(29.6%)</td></tr><tr><td>&#8195;Lung</td><td>1 (3.3%)</td><td>6&#160;(22.2%)</td></tr><tr><td>&#8195;Lung + bone</td><td>1 (3.3%)</td><td>0 (0%)</td></tr><tr><td>&#8195;Soft tissue</td><td>&#8211;</td><td>0 (0%)</td></tr><tr><td>&#8195;Lymph nodes</td><td>5 (12.2%)</td><td>0 (0%)</td></tr><tr><td>&#8195;Bone</td><td>1 (2.0%)</td><td>1&#160;(3.7%)</td></tr><tr><td>&#8195;Bone+lymph node</td><td>&#8211;</td><td>1&#160;(3.7%)</td></tr></tbody></table><table-wrap-foot><fn><p>HBV, hepatitis B virius;</p><p>HCV, hepatitis C virius;</p></fn><fn><p>AST, Aspartate transaminase;</p><p>ALT, Alanine aminotransferase;</p></fn><fn><p>AFP, alpha-feto-protein.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15691-v2.xml">elife-15691-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><label>Table 3.</label><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Variables</th><th colspan="2">Patients</th></tr><tr><th valign="top">TILA-TACE</th><th valign="top">cTACE</th></tr></thead><tbody><tr><td valign="top">Patient number</td><td valign="top">10</td><td valign="top">10</td></tr><tr><td valign="top">Median age, years</td><td valign="top">58 (Range 40&#8211;86)</td><td valign="top">53 (43&#8211;81)</td></tr><tr><td valign="top">Gender (M/F)</td><td valign="top">9 /1 (90.0%/10.0%)</td><td valign="top">7 /3 (70.0%/ 30.0%)</td></tr><tr><td valign="top">Aetiology</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">&#8195;HBV</td><td valign="top">9 (90.0%)</td><td valign="top">8 (80.0%)</td></tr><tr><td valign="top">&#8195;HCV</td><td valign="top">0 (0%)</td><td valign="top">0 (0%)</td></tr><tr><td valign="top">&#8195;Non B-non C</td><td valign="top">1 (10.0%)</td><td valign="top">2 (20.0%)</td></tr><tr><td valign="top">Cirrhosis (radiology)</td><td valign="top">10 (100%)</td><td valign="top">10 (100%)</td></tr><tr><td valign="top">Bilirubin, &#956;M</td><td valign="top">17.2 &#177; 10.1</td><td valign="top">16.7 &#177; 7.6</td></tr><tr><td valign="top">Albumin, g/L</td><td valign="top">38.7 &#177; 3.1</td><td valign="top">38.1 &#177; 5.2</td></tr><tr><td valign="top">AST, U/L</td><td valign="top">64.8 &#177; 44.8</td><td valign="top">52.8 &#177; 19.2</td></tr><tr><td valign="top">ALT, U/L</td><td valign="top">60.6 &#177; 48.0</td><td valign="top">41.0 &#177; 29.9</td></tr><tr><td valign="top">AFP, &gt;400&#160;ng/mL</td><td valign="top">3 (30.0%)</td><td valign="top">4 (40.0%)</td></tr><tr><td valign="top">Child-Pugh class, A/B</td><td valign="top">10/0 (100%/0%)</td><td valign="top">10/0 (100%/0%)</td></tr><tr><td valign="top">The size of largest tumor (cm)</td><td valign="top">7.9 (range 5.0&#8211;13.5)</td><td valign="top">7.5 (range 5.0&#8211;13.0)</td></tr><tr><td valign="top">&#8195;Tumor &gt;10 cm</td><td valign="top">3 (30.0%)</td><td valign="top">3&#160;(30.0%)</td></tr><tr><td valign="top">&#8195;Tumor 5~10&#160;cm</td><td valign="top">7 (70.0%)</td><td valign="top">7&#160;(70.0%)</td></tr><tr><td valign="top">Multifocal tumors in 1 lobe</td><td valign="top">3 (30.0%)</td><td valign="top">3 (30.0%)</td></tr><tr><td valign="top">Multifocal tumors in 2 lobes</td><td valign="top">4 (40.0%)</td><td valign="top">2 (20.0%)</td></tr><tr><td valign="top">BCLC stage</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">&#8195;B</td><td valign="top">7 (70.0%)</td><td valign="top">8 (70.0%)</td></tr><tr><td valign="top">&#8195;C</td><td valign="top">3 (30.0%)</td><td valign="top">2 (20.0%)</td></tr><tr><td valign="top">Macrovascular invasion</td><td valign="top">2 (20.0%)</td><td valign="top">1&#160;(10.0%)</td></tr><tr><td valign="top">&#8195;The right branch of portal vein</td><td valign="top">1&#160;(10.0%)</td><td valign="top">1&#160;(10.0%)</td></tr><tr><td valign="top">&#8195;The left branch of portal vein</td><td valign="top">1&#160;(10.0%)</td><td valign="top">0 (0%)</td></tr><tr><td valign="top">Extra-hepatic metastasis</td><td valign="top">1&#160;(10.0%)</td><td valign="top">2&#160;(20.0%)</td></tr><tr><td valign="top">&#8195;Brain</td><td valign="top">0 (0%)</td><td valign="top">1&#160;(10.0%)</td></tr><tr><td valign="top">&#8195;Lymph nodes</td><td valign="top">1&#160;(10.0%)</td><td valign="top">1 (10.0%)</td></tr></tbody></table><table-wrap-foot><fn><p>HBV, hepatitis B virius;</p><p>HCV, hepatitis C virius;</p></fn><fn><p>AST, Aspartate transaminase;</p><p>ALT, Alanine aminotransferase;</p></fn><fn><p>AFP, alpha-feto-protein.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15691-v3.xml">elife-15691-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Clinical and tumor characteristics of patients treated with cTACE and TILA-TACE in the nonrandomized study.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Variables</th><th colspan="2">Patients</th></tr><tr><th valign="top">TILA-TACE</th><th valign="top">cTACE</th></tr></thead><tbody><tr><td>Patient number</td><td>30</td><td>27</td></tr><tr><td>Median age, years</td><td>57 (Range 32&#8211;81)</td><td>54 (Range 37&#8211;81)</td></tr><tr><td>Gender (M/F)</td><td>27/3 (90.0%/10.0%)</td><td>27/0 (100%/0%)</td></tr><tr><td>Aetiology</td><td/><td/></tr><tr><td>&#8195;HBV</td><td>24 (80.0%)</td><td>25&#160;(92.6%)</td></tr><tr><td>&#8195;HCV</td><td>0 (0%)</td><td>0 (0%)</td></tr><tr><td>&#8195;Non B-non C</td><td>6 (20.0%)</td><td>2&#160;(7.4%)</td></tr><tr><td>Cirrhosis (radiology)</td><td>30 (100%)</td><td>27&#160;(100%)</td></tr><tr><td>Bilirubin, &#956;M</td><td>16.9 &#177; 9.4</td><td>22.5 &#177; 11.6</td></tr><tr><td>Albumin, g/L</td><td>39.0 &#177; 6.9</td><td>37.4 &#177; 5.3</td></tr><tr><td>AST, U/L</td><td>74.9 &#177; 102.3</td><td>83.5 &#177; 54.1</td></tr><tr><td>ALT, U/L</td><td>54.1 &#177; 80.4</td><td>67.3 &#177; 43.5</td></tr><tr><td>AFP, &gt;400&#160;ng/mL</td><td>9 (30.0%)</td><td>15&#160;(55.6%)</td></tr><tr><td>Child-Pugh class, A/B</td><td>27/3 (90.0%/10.0%)</td><td>25/2&#160;(92.6%/7.4%)</td></tr><tr><td>The size of largest tumor (cm)</td><td>9.2 (range 5.0&#8211;13.6)</td><td>10.3 (range 5.0&#8211;14.6)</td></tr><tr><td>&#8195;Tumor &gt;10 cm</td><td>14 (46.7%)</td><td>15&#160;(55.6%)</td></tr><tr><td>&#8195;Tumor 5~10&#160;cm</td><td>16 (53.3%)</td><td>12&#160;(44.4%)</td></tr><tr><td>Multifocal tumors in 1 lobe</td><td>8 (26.7%)</td><td>12&#160;(44.4%)</td></tr><tr><td>Multifocal tumors in 2 lobes</td><td>8 (26.7%)</td><td>12&#160;(44.4%)</td></tr><tr><td>BCLC stage</td><td/><td/></tr><tr><td>&#8195;B</td><td>19 (63.3%)</td><td>18&#160;(66.7%)</td></tr><tr><td>&#8195;C</td><td>11 (36.7%)</td><td>9&#160;(33.3%)</td></tr><tr><td>Macrovascular invasion</td><td>5 (16.7%)</td><td>4&#160;(14.8%)</td></tr><tr><td>&#8195;The right branch of portal vein</td><td>4 (13.3%)</td><td>2&#160;(7.4%)</td></tr><tr><td>&#8195;Hepatic vein</td><td>&#8211;</td><td>1&#160;(3.7%)</td></tr><tr><td>&#8195;The right branch of portal + hepatic vein</td><td>1 (3.3%)</td><td>1&#160;(3.7%)</td></tr><tr><td>Extra-hepatic metastasis</td><td>8 (26.7%)</td><td>8&#160;(29.6%)</td></tr><tr><td>&#8195;Lung</td><td>1 (3.3%)</td><td>6&#160;(22.2%)</td></tr><tr><td>&#8195;Lung + bone</td><td>1 (3.3%)</td><td>0 (0%)</td></tr><tr><td>&#8195;Soft tissue</td><td>&#8211;</td><td>0 (0%)</td></tr><tr><td>&#8195;Lymph nodes</td><td>5 (12.2%)</td><td>0 (0%)</td></tr><tr><td>&#8195;Bone</td><td>1 (2.0%)</td><td>1&#160;(3.7%)</td></tr><tr><td>&#8195;Bone+lymph node</td><td>&#8211;</td><td>1&#160;(3.7%)</td></tr></tbody></table><table-wrap-foot><fn><p>HBV, hepatitis B virius;</p><p>HCV, hepatitis C virius;</p></fn><fn><p>AST, Aspartate transaminase;</p><p>ALT, Alanine aminotransferase;</p></fn><fn><p>AFP, alpha-feto-protein.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15691-v3.xml">elife-15691-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><label>Table 3.</label><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Variables</th><th colspan="2">Patients</th></tr><tr><th valign="top">TILA-TACE</th><th valign="top">cTACE</th></tr></thead><tbody><tr><td valign="top">Patient number</td><td valign="top">10</td><td valign="top">10</td></tr><tr><td valign="top">Median age, years</td><td valign="top">58 (Range 40&#8211;86)</td><td valign="top">53 (43&#8211;81)</td></tr><tr><td valign="top">Gender (M/F)</td><td valign="top">9 /1 (90.0%/10.0%)</td><td valign="top">7 /3 (70.0%/ 30.0%)</td></tr><tr><td valign="top">Aetiology</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">&#8195;HBV</td><td valign="top">9 (90.0%)</td><td valign="top">8 (80.0%)</td></tr><tr><td valign="top">&#8195;HCV</td><td valign="top">0 (0%)</td><td valign="top">0 (0%)</td></tr><tr><td valign="top">&#8195;Non B-non C</td><td valign="top">1 (10.0%)</td><td valign="top">2 (20.0%)</td></tr><tr><td valign="top">Cirrhosis (radiology)</td><td valign="top">10 (100%)</td><td valign="top">10 (100%)</td></tr><tr><td valign="top">Bilirubin, &#956;M</td><td valign="top">17.2 &#177; 10.1</td><td valign="top">16.7 &#177; 7.6</td></tr><tr><td valign="top">Albumin, g/L</td><td valign="top">38.7 &#177; 3.1</td><td valign="top">38.1 &#177; 5.2</td></tr><tr><td valign="top">AST, U/L</td><td valign="top">64.8 &#177; 44.8</td><td valign="top">52.8 &#177; 19.2</td></tr><tr><td valign="top">ALT, U/L</td><td valign="top">60.6 &#177; 48.0</td><td valign="top">41.0 &#177; 29.9</td></tr><tr><td valign="top">AFP, &gt;400&#160;ng/mL</td><td valign="top">3 (30.0%)</td><td valign="top">4 (40.0%)</td></tr><tr><td valign="top">Child-Pugh class, A/B</td><td valign="top">10/0 (100%/0%)</td><td valign="top">10/0 (100%/0%)</td></tr><tr><td valign="top">The size of largest tumor (cm)</td><td valign="top">7.9 (range 5.0&#8211;13.5)</td><td valign="top">7.5 (range 5.0&#8211;13.0)</td></tr><tr><td valign="top">&#8195;Tumor &gt;10 cm</td><td valign="top">3 (30.0%)</td><td valign="top">3&#160;(30.0%)</td></tr><tr><td valign="top">&#8195;Tumor 5~10&#160;cm</td><td valign="top">7 (70.0%)</td><td valign="top">7&#160;(70.0%)</td></tr><tr><td valign="top">Multifocal tumors in 1 lobe</td><td valign="top">3 (30.0%)</td><td valign="top">3 (30.0%)</td></tr><tr><td valign="top">Multifocal tumors in 2 lobes</td><td valign="top">4 (40.0%)</td><td valign="top">2 (20.0%)</td></tr><tr><td valign="top">BCLC stage</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">&#8195;B</td><td valign="top">7 (70.0%)</td><td valign="top">8 (70.0%)</td></tr><tr><td valign="top">&#8195;C</td><td valign="top">3 (30.0%)</td><td valign="top">2 (20.0%)</td></tr><tr><td valign="top">Macrovascular invasion</td><td valign="top">2 (20.0%)</td><td valign="top">1&#160;(10.0%)</td></tr><tr><td valign="top">&#8195;The right branch of portal vein</td><td valign="top">1&#160;(10.0%)</td><td valign="top">1&#160;(10.0%)</td></tr><tr><td valign="top">&#8195;The left branch of portal vein</td><td valign="top">1&#160;(10.0%)</td><td valign="top">0 (0%)</td></tr><tr><td valign="top">Extra-hepatic metastasis</td><td valign="top">1&#160;(10.0%)</td><td valign="top">2&#160;(20.0%)</td></tr><tr><td valign="top">&#8195;Brain</td><td valign="top">0 (0%)</td><td valign="top">1&#160;(10.0%)</td></tr><tr><td valign="top">&#8195;Lymph nodes</td><td valign="top">1&#160;(10.0%)</td><td valign="top">1 (10.0%)</td></tr></tbody></table><table-wrap-foot><fn><p>HBV, hepatitis B virius;</p><p>HCV, hepatitis C virius;</p></fn><fn><p>AST, Aspartate transaminase;</p><p>ALT, Alanine aminotransferase;</p></fn><fn><p>AFP, alpha-feto-protein.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15784-v2.xml">elife-15784-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.15784.017</object-id><label>Table 1.</label><caption><p>The distribution of the postsynaptic targets.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15784.017">http://dx.doi.org/10.7554/eLife.15784.017</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Postsynaptic target</th><th colspan="2" valign="bottom">Completely traced axons</th><th colspan="2" valign="bottom">Partially traced axons</th><th colspan="2" valign="bottom">All</th></tr><tr><th valign="bottom">count</th><th valign="bottom">%</th><th valign="bottom">count</th><th valign="bottom">%</th><th valign="bottom">count</th><th valign="bottom">%</th></tr></thead><tbody><tr><td valign="bottom">Excitatory somata</td><td valign="bottom">60*</td><td valign="bottom">15.0</td><td valign="bottom">46<sup>&#8224;</sup></td><td valign="bottom">10.5</td><td valign="bottom">106</td><td valign="bottom">12.6</td></tr><tr><td valign="bottom">Excitatory dendritic shafts</td><td valign="bottom">184</td><td valign="bottom">45.9</td><td valign="bottom">225</td><td valign="bottom">51.4</td><td valign="bottom">409</td><td valign="bottom">48.7</td></tr><tr><td valign="bottom">Excitatory dendritic spines</td><td valign="bottom">119</td><td valign="bottom">29.7</td><td valign="bottom">134</td><td valign="bottom">30.6</td><td valign="bottom">253</td><td valign="bottom">30.2</td></tr><tr><td valign="bottom">Excitatory axons</td><td valign="bottom">7</td><td valign="bottom">1.7</td><td valign="bottom">8</td><td valign="bottom">1.8</td><td valign="bottom">15</td><td valign="bottom">1.8</td></tr><tr><td valign="bottom"><italic>All excitatory</italic></td><td valign="bottom"><italic>370</italic></td><td valign="bottom"><italic>92.3</italic></td><td valign="bottom"><italic>413</italic></td><td valign="bottom"><italic>94.3</italic></td><td valign="bottom"><italic>783</italic></td><td valign="bottom"><italic>93.3</italic></td></tr><tr><td valign="bottom">Inhibitory dendritic shafts</td><td valign="bottom">17</td><td valign="bottom">4.2</td><td valign="bottom">20</td><td valign="bottom">4.6</td><td valign="bottom">37</td><td valign="bottom">4.4</td></tr><tr><td valign="bottom">Inhibitory dendritic spines</td><td valign="bottom">0</td><td valign="bottom">0</td><td valign="bottom">0</td><td valign="bottom">0</td><td valign="bottom">0</td><td valign="bottom">0</td></tr><tr><td valign="bottom">Inhibitory axons</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">1</td><td valign="bottom">0.2</td><td valign="bottom">1</td><td valign="bottom">0.1</td></tr><tr><td valign="bottom"><italic>All inhibitory</italic></td><td valign="bottom"><italic>17</italic></td><td valign="bottom"><italic>4.2</italic></td><td valign="bottom"><italic>21</italic></td><td valign="bottom"><italic>4.8</italic></td><td valign="bottom"><italic>38</italic></td><td valign="bottom"><italic>4.5</italic></td></tr><tr><td valign="bottom">Uncategorized dendritic spines</td><td valign="bottom">14</td><td valign="bottom">3.5</td><td valign="bottom">4</td><td valign="bottom">0.9</td><td valign="bottom">18</td><td valign="bottom">2.1</td></tr><tr><td valign="bottom"><bold>Total</bold></td><td valign="bottom"><bold>401</bold></td><td valign="bottom"/><td valign="bottom"><bold>438</bold></td><td valign="bottom"/><td valign="bottom"><bold>839</bold></td><td valign="bottom"/></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>* Including 10 synapses onto somatic spines;</p></fn><fn id="tblfn2"><p><sup>&#8224;</sup>&#160;Including 8 synapses on somatic spines&#65279;</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15828-v2.xml">elife-15828-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.15828.004</object-id><label>Table 1.</label><caption><p>In vitro properties of GCD-0810.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.15828.004">http://dx.doi.org/10.7554/eLife.15828.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Compound</th><th colspan="2">ER binding<sup>&#8727;</sup></th><th colspan="2">Transcription<sup>&#8224;</sup></th><th colspan="2">Cell viability<sup>&#8225;</sup></th><th colspan="2">ER&#945; degradation<sup>&#167;</sup></th></tr><tr><th>ER&#945;</th><th>ER&#946;</th><th colspan="2">3X ERE~LUC</th><th colspan="2">CellTiter-Glo</th><th colspan="2">In-Cell Western</th></tr><tr><th colspan="2">K<sub>i</sub> [nM]</th><th>IC<sub>50</sub> <break/>[nM]</th><th>E<sub>max</sub> <break/>[% E2]</th><th>IC<sub>50</sub> <break/>[nM]</th><th>E<sub>max</sub> <break/>[% E2]</th><th>EC<sub>50</sub> <break/>[nM]</th><th>E<sub>max</sub> <break/>[% Veh.]</th></tr></thead><tbody><tr><td>GDC-0810</td><td>3.8 &#177; 1.6</td><td>3.7 &#177; 4.0</td><td>1.3 &#177; 0.8</td><td>6.1 &#177; 2.8</td><td>2.5 &#177; 2.1</td><td>24.6 &#177; 3.3</td><td>0.65 &#177; 0.50</td><td>15.3 &#177; 3.4</td></tr><tr><td>4-OH Tam</td><td>2.2 &#177; 1.3</td><td>3.6 &#177; 1.7</td><td>6.7 &#177; 3.6</td><td>4.7 &#177; 2.9</td><td>0.53 &#177; 0.25</td><td>48.0 &#177; 4.7</td><td>0.14 &#177; 0.04</td><td>51.9 &#177; 2.7<sup>#</sup></td></tr><tr><td>Fulvestrant</td><td>13.1 &#177; 10.8</td><td>13.2 &#177; 7.6</td><td>0.3 &#177; 0.2</td><td>4.1 &#177; 2.6</td><td>0.56 &#177; 0.70</td><td>25.4 &#177; 3.7</td><td>0.39 &#177; 0.18</td><td>6.4 &#177; 2.0</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p><sup>&#8727;</sup>&#160;Binding affinities (Ki) of GDC-0810, 4-hydroxytamoxifen (4-OHT), and fulvestrant for ER&#945; and ER&#946;. Shown are the mean and standard deviation of 3&#8211;4 experiments run in duplicate.</p></fn><fn id="tblfn2"><p><sup>&#8224;</sup> ER&#945; antagonist reporter assay. Results are the mean and standard deviation of 3 experiments.</p></fn><fn id="tblfn3"><p><sup>&#8225;</sup>&#160;Relative cell viability after 5 d incubation with compound. Shown are the mean and standard deviation of more than 50 assays run in triplicate.</p></fn><fn id="tblfn4"><p><sup>&#167;</sup>&#160;Relative ER&#945; immunofluorescence activity in MCF7 In-Cell Western.</p></fn><fn id="tblfn5"><p><sup>#</sup>&#160;The apparent reduction in ER&#945; immunoreactivity is not reproduced in western blots.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-15828-v3.xml">elife-15828-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.15828.004</object-id><label>Table 1.</label><caption><title>In vitro properties of GCD-0810.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Compound</th><th colspan="2">ER binding<sup>&#8727;</sup></th><th colspan="2">Transcription<sup>&#8224;</sup></th><th colspan="2">Cell viability<sup>&#8225;</sup></th><th colspan="2">ER&#945; degradation<sup>&#167;</sup></th></tr><tr><th>ER&#945;</th><th>ER&#946;</th><th colspan="2">3X ERE~LUC</th><th colspan="2">CellTiter-Glo</th><th colspan="2">In-Cell Western</th></tr><tr><th colspan="2">K<sub>i</sub> [nM]</th><th>IC<sub>50</sub> <break/>[nM]</th><th>E<sub>max</sub> <break/>[% E2]</th><th>IC<sub>50</sub> <break/>[nM]</th><th>E<sub>max</sub> <break/>[% E2]</th><th>EC<sub>50</sub> <break/>[nM]</th><th>E<sub>max</sub> <break/>[% Veh.]</th></tr></thead><tbody><tr><td>GDC-0810</td><td>3.8 &#177; 1.6</td><td>3.7 &#177; 4.0</td><td>1.3 &#177; 0.8</td><td>6.1 &#177; 2.8</td><td>2.5 &#177; 2.1</td><td>24.6 &#177; 3.3</td><td>0.65 &#177; 0.50</td><td>15.3 &#177; 3.4</td></tr><tr><td>4-OH Tam</td><td>2.2 &#177; 1.3</td><td>3.6 &#177; 1.7</td><td>6.7 &#177; 3.6</td><td>4.7 &#177; 2.9</td><td>0.53 &#177; 0.25</td><td>48.0 &#177; 4.7</td><td>0.14 &#177; 0.04</td><td>51.9 &#177; 2.7<sup>#</sup></td></tr><tr><td>Fulvestrant</td><td>13.1 &#177; 10.8</td><td>13.2 &#177; 7.6</td><td>0.3 &#177; 0.2</td><td>4.1 &#177; 2.6</td><td>0.56 &#177; 0.70</td><td>25.4 &#177; 3.7</td><td>0.39 &#177; 0.18</td><td>6.4 &#177; 2.0</td></tr></tbody></table><table-wrap-foot><fn><p><sup>&#8727;</sup>&#160;Binding affinities (Ki) of GDC-0810, 4-hydroxytamoxifen (4-OHT), and fulvestrant for ER&#945; and ER&#946;. Shown are the mean and standard deviation of 3&#8211;4 experiments run in duplicate.</p><p><sup>&#8224;</sup> ER&#945; antagonist reporter assay. Results are the mean and standard deviation of 3 experiments.</p></fn><fn><p><sup>&#8225;</sup>&#160;Relative cell viability after 5 d incubation with compound. Shown are the mean and standard deviation of more than 50 assays run in triplicate.</p><p><sup>&#167;</sup>&#160;Relative ER&#945; immunofluorescence activity in MCF7 In-Cell Western.</p></fn><fn><p><sup>#</sup>&#160;The apparent reduction in ER&#945; immunoreactivity is not reproduced in western blots.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-16090-v2.xml">elife-16090-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.16090.006</object-id><label>Table 1.</label><caption><p>Summary of data collated in the three meta-analyses. The number of data points is subdivided according to the insecticides or LLIN tested and the predominant mosquito species in each population tested. Studies which did not determine species in the <italic>Anopheles gambiae</italic> complex are shown separately. All Published Data can be downloaded from Dryad Digital Repository whilst a list of the studies included their geographical range are given in the Material and methods.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.16090.006">http://dx.doi.org/10.7554/eLife.16090.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2">Meta-analysis description</th><th rowspan="2">Details</th><th rowspan="2">No. Studies</th><th colspan="5">Number data points</th></tr><tr><th><italic>Anopheles gambiae</italic> s.s.</th><th><italic>Anopheles arabiensis</italic></th><th><italic>Anopheles gambiae</italic> s.l.</th><th><italic>Anopheles funestus</italic></th><th><bold>Total</bold></th></tr></thead><tbody><tr><td rowspan="4" valign="top"><italic><bold>M1</bold></italic></td><td rowspan="4" valign="top"><italic>Bioassay and experimental hut trial mortality</italic></td><td valign="top">Deltamethrin</td><td>5</td><td>2</td><td>1</td><td>10</td><td>0</td><td>13</td></tr><tr><td valign="top">Permethrin</td><td>8</td><td>2</td><td>1</td><td>3</td><td>0</td><td>6</td></tr><tr><td valign="top">Other</td><td>1</td><td>0</td><td>0</td><td>1</td><td>1</td><td>2</td></tr><tr><td valign="top"><bold>Total</bold></td><td>13</td><td>4</td><td>2</td><td>14</td><td>1</td><td><bold>21</bold></td></tr><tr><td rowspan="4" valign="top"><italic><bold>M2</bold></italic></td><td rowspan="4" valign="top"><italic>Impact of PBO in pyrethroid bioassays</italic></td><td valign="top">Deltamethrin</td><td>16</td><td>15</td><td>5</td><td>29</td><td>8</td><td>57</td></tr><tr><td valign="top">Permethrin</td><td>20</td><td>22</td><td>7</td><td>30</td><td>9</td><td>68</td></tr><tr><td valign="top">Other</td><td>4</td><td>2</td><td>0</td><td>4</td><td>6</td><td>12</td></tr><tr><td valign="top"><bold>Total</bold></td><td>24</td><td>39</td><td>12</td><td>63</td><td>23</td><td><bold>137</bold></td></tr><tr><td rowspan="3" valign="top"><italic><bold>M3</bold></italic></td><td rowspan="3" valign="top"><italic>Experimental hut trials of standard and PBO LLINS</italic></td><td valign="top">Olyset</td><td>6</td><td>6</td><td>0</td><td>10</td><td>0</td><td>16</td></tr><tr><td valign="top">PermaNet</td><td>6</td><td>18</td><td>4</td><td>6</td><td>0</td><td>28</td></tr><tr><td valign="top"><bold>Total</bold></td><td>12</td><td>24</td><td>4</td><td>16</td><td>0</td><td><bold>44</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-16371-v2.xml">elife-16371-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.16371.010</object-id><label>Table 3.</label><caption><p>Model results for GLMMs testing the occurrence of tool use. <italic>p</italic>-values are presented only for the first model as the two other models were not significant (see <xref ref-type="table" rid="tbl2">Table 2</xref>). All numeric predictor variables were standardized to mean&#160;=&#160;0 and SD&#160;=&#160;1. For sex, &#8216;female&#8217; is the reference level.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.16371.010">http://dx.doi.org/10.7554/eLife.16371.010</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="4" valign="top">1 week</th><th colspan="4" valign="top">7 weeks</th><th colspan="4" valign="top">13 weeks</th></tr><tr><th valign="top"><italic>&#946;</italic></th><th valign="top"><italic>&#177; se</italic></th><th valign="top"><italic>z</italic></th><th valign="top"><italic>p</italic></th><th valign="top"><italic>&#946;</italic></th><th valign="top"><italic>&#177; se</italic></th><th valign="top"><italic>z</italic></th><th valign="top"><italic>p</italic></th><th valign="top"><italic>&#946;</italic></th><th valign="top"><italic>&#177; se</italic></th><th valign="top"><italic>z</italic></th><th valign="top"><italic>p</italic></th></tr></thead><tbody><tr><td valign="top">Intercept</td><td valign="top">&#8722;3.55</td><td valign="top">0.42</td><td valign="top">&#8722;8.47</td><td valign="top">0.0000</td><td valign="top">&#8722;3.39</td><td valign="top">0.37</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8722;3.40</td><td valign="top">0.37</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td></tr><tr><td valign="top">Ripe fruit feeding</td><td valign="top">&#8722;0.24</td><td valign="top">0.26</td><td valign="top">&#8722;0.93</td><td valign="top">0.3525</td><td valign="top">&#8722;0.33</td><td valign="top">0.24</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8722;0.35</td><td valign="top">0.24</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td></tr><tr><td valign="top">Travel time</td><td valign="top">0.67</td><td valign="top">0.30</td><td valign="top">2.25</td><td valign="top">0.0242</td><td valign="top">0.30</td><td valign="top">0.27</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">0.32</td><td valign="top">0.26</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td></tr><tr><td valign="top">Sex (male)</td><td valign="top">&#8722;0.07</td><td valign="top">0.58</td><td valign="top">&#8722;0.12</td><td valign="top">0.9062</td><td valign="top">0.04</td><td valign="top">0.54</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">0.04</td><td valign="top">0.55</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td></tr><tr><td valign="top">Age</td><td valign="top">&#8722;0.56</td><td valign="top">0.33</td><td valign="top">&#8722;1.72</td><td valign="top">0.0855</td><td valign="top">&#8722;0.56</td><td valign="top">0.29</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8722;0.55</td><td valign="top">0.29</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td></tr><tr><td valign="top">Auto-correlation</td><td valign="top">0.80</td><td valign="top">0.18</td><td valign="top">4.44</td><td valign="top">0.0000</td><td valign="top">0.52</td><td valign="top">0.16</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">0.50</td><td valign="top">0.16</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-16443-v2.xml">elife-16443-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.16443.005</object-id><label>Table 1.</label><caption><p>Neuronal database. N is the number of recorded neurons that passed inclusion criteria. The recording span represents the stable recording period (in seconds) for each neuron. The isolation score ranges from 0 to 1. The recording span and isolation score were averaged for each structure and state. Values are means &#177; standard deviation (SD). For each neuronal assembly, statistics were calculated and are presented for both monkeys.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.16443.005">http://dx.doi.org/10.7554/eLife.16443.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Neuronal assembly</th><th colspan="3">before MPTP</th><th colspan="3">after MPTP</th></tr><tr><th>N</th><th>Recording <break/>span (s)</th><th>Isolation score</th><th>N</th><th>Recording <break/>span (s)</th><th>Isolation <break/>score</th></tr></thead><tbody><tr><td>striatum (MSN)</td><td>150</td><td>1096.8 &#177; 564.8</td><td>0.85 &#177; 0.10</td><td>128</td><td>950.6 &#177; 616.1</td><td>0.85 &#177; 0.12</td></tr><tr><td>striatum (TAN)</td><td>116</td><td>1568.8 &#177; 907.6</td><td>0.89 &#177; 0.08</td><td>81</td><td>1217.8 &#177; 708.8</td><td>0.89 &#177; 0.09</td></tr><tr><td>STN</td><td>103</td><td>1080.0 &#177; 411.1</td><td>0.77 &#177; 0.12</td><td>111</td><td>780.0 &#177; 317.7</td><td>0.80 &#177; 0.12</td></tr><tr><td>GPe</td><td>182</td><td>1760.4 &#177; 978.2</td><td>0.91 &#177; 0.08</td><td>105</td><td>877.7 &#177; 332.7</td><td>0.91 &#177; 0.09</td></tr><tr><td>GPi</td><td>119</td><td>1437.0 &#177; 607.9</td><td>0.89 &#177; 0.08</td><td>none</td><td/><td/></tr><tr><td>SNr</td><td>110</td><td>1205.2 &#177; 366.4</td><td>0.90 &#177; 0.08</td><td>121</td><td>874.7 &#177; 445.4</td><td>0.90 &#177; 0.09</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-16616-v2.xml">elife-16616-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.16616.004</object-id><label>Table 1.</label><caption><p>Atoms coordinating the binding site ions in the crystal structures and from the MD simulations. O is the backbone carbonyl oxygen atom. OG and OG1 are the hydroxyl oxygen atoms in serine and threonine. OD1 and OD2 are the carboxyl oxygen atoms in asparate. OE1 and OE2 are the carboxyl oxygen atoms in glutamate. OH2 is the water oxygen.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.16616.004">http://dx.doi.org/10.7554/eLife.16616.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><bold>E<sub>1</sub></bold></th><th colspan="4" valign="top"><bold>Site I</bold></th><th colspan="4" valign="top"><bold>Site II</bold></th><th colspan="4" valign="top"><bold>Site III</bold></th></tr><tr><th colspan="2" valign="top"><bold>x-ray</bold></th><th colspan="2" valign="top"><bold>MD</bold></th><th colspan="2" valign="top"><bold>x-ray</bold></th><th colspan="2" valign="top"><bold>MD</bold></th><th colspan="2" valign="top"><bold>x-ray</bold></th><th colspan="2" valign="top"><bold>MD</bold></th></tr></thead><tbody><tr><td valign="top"/><td valign="top">A323</td><td valign="top">O</td><td valign="top">T772</td><td valign="top">OG1</td><td valign="top">V322</td><td valign="top">O</td><td valign="top">E779</td><td valign="top">OE1</td><td valign="top">Y771</td><td valign="top">O</td><td valign="top">Y771</td><td valign="top">O</td></tr><tr><td valign="top"/><td valign="top">E779</td><td valign="top">OE1</td><td valign="top">T772</td><td valign="top">O</td><td valign="top">V325</td><td valign="top">O</td><td valign="top">D804</td><td valign="top">OD1</td><td valign="top">T774</td><td valign="top">O</td><td valign="top">T774</td><td valign="top">O</td></tr><tr><td valign="top"/><td valign="top">D808</td><td valign="top">OD1</td><td valign="top">N776</td><td valign="top">OD1</td><td valign="top">E327</td><td valign="top">OE2</td><td valign="top">D808</td><td valign="top">OD1</td><td valign="top">Q923</td><td valign="top">OE1</td><td valign="top">Q923</td><td valign="top">OE1</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top">D808</td><td valign="top">OD1</td><td valign="top">D804</td><td valign="top">OD1</td><td valign="top">Water</td><td valign="top">OH2</td><td valign="top"/><td valign="top"/><td valign="top">D926</td><td valign="top">OD1</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top">D808</td><td valign="top">OD2</td><td valign="top">Water</td><td valign="top">OH2</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">E<sub>2</sub></th><th colspan="4" valign="top">Site I</th><th colspan="4" valign="top">Site II</th><th valign="top"/><th valign="top"/><th valign="top"/><th valign="top"/></tr><tr><th colspan="2" valign="top">x-ray</th><th colspan="2" valign="top">MD</th><th colspan="2" valign="top">x-ray</th><th colspan="2" valign="top">MD</th><th valign="top"/><th valign="top"/><th valign="top"/><th valign="top"/></tr></thead><tbody><tr><td valign="top"/><td valign="top">T772</td><td valign="top">O</td><td valign="top">S775</td><td valign="top">OG</td><td valign="top">V322</td><td valign="top">O</td><td valign="top">A323</td><td valign="top">O</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"/><td valign="top">S775</td><td valign="top">OG</td><td valign="top">N776</td><td valign="top">OD1</td><td valign="top">V325</td><td valign="top">O</td><td valign="top">V325</td><td valign="top">O</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"/><td valign="top">N776</td><td valign="top">OD1</td><td valign="top">D804</td><td valign="top">OD2</td><td valign="top">E779</td><td valign="top">OE2</td><td valign="top">E779</td><td valign="top">OE1</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"/><td valign="top">D804</td><td valign="top">OD2</td><td valign="top"/><td valign="top"/><td valign="top">D804</td><td valign="top">OD2</td><td valign="top">D804</td><td valign="top">OD1</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-16620-v1.xml">elife-16620-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.16620.005</object-id><label>Table 1.</label><caption><p>Karyotypes are stable in select BubR1 mutant overexpressing MEFs. Karyotype analysis of passage 5 MEFs of indicated genotype. <italic>n</italic> &#8805; 3 lines, 50 cells per line. Data are mean &#177; s.d. WT, wild-type. FL, full-length. (See associated <xref ref-type="supplementary-material" rid="SD1-data">Table 1&#8212; source data 1</xref>).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.16620.005">http://dx.doi.org/10.7554/eLife.16620.005</ext-link></p><p><supplementary-material id="SD1-data"><object-id pub-id-type="doi">10.7554/eLife.16620.006</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Source file for MEF aneuploidy rate data.</title><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.16620.006">http://dx.doi.org/10.7554/eLife.16620.006</ext-link></p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-16620-table1-data1-v1.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Mitotic MEF <break/>genotype (n)</th><th rowspan="2" valign="top">Mitotic figures</th><th rowspan="2" valign="top">Aneuploid <break/>figures&#160;% (s.d)</th><th colspan="9">Karyotype with indicated chromosome number</th></tr><tr><th valign="bottom">36</th><th valign="bottom">37</th><th valign="bottom">38</th><th valign="bottom">39</th><th valign="bottom">40</th><th valign="bottom">41</th><th valign="bottom">42</th><th valign="bottom">43</th><th valign="bottom">44</th></tr></thead><tbody><tr><td>WT (<italic>5</italic>)&#160;</td><td>250</td><td>9 (6)</td><td>1</td><td>1</td><td>0</td><td>5</td><td>227</td><td>9</td><td>7</td><td>0</td><td>0</td></tr><tr><td>FL-<italic>Bub1b</italic> (3)</td><td>150</td><td>10 (3)</td><td>0</td><td>0</td><td>1</td><td>2</td><td>135</td><td>7</td><td>5</td><td>0</td><td>0</td></tr><tr><td><italic>Bub1b</italic><sup>&#916;I</sup> (3)</td><td>150</td><td>8 (3)</td><td>0</td><td>0</td><td>1</td><td>2</td><td>138</td><td>9</td><td>0</td><td>0</td><td>0</td></tr><tr><td><italic>Bub1b</italic><sup>&#916;N</sup> (5)</td><td>250</td><td>18 (6)*</td><td>0</td><td>0</td><td>2</td><td>15</td><td>204</td><td>23</td><td>6</td><td>0</td><td>0</td></tr><tr><td><italic>Bub1b</italic><sup>N</sup> (3)</td><td>150</td><td>6 (2)</td><td>0</td><td>0</td><td>1</td><td>3</td><td>141</td><td>4</td><td>1</td><td>0</td><td>0</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>*p&lt;0.05.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-16950-v2.xml">elife-16950-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.16950.008</object-id><label>Table 1.</label><caption><p>Prediction output and performance of HeLa organellar maps. The table shows the combined organellar prediction output from six replicate maps from HeLa cells. Prediction performance is judged by the proportion of correctly assigned organellar marker proteins. Please also refer to <xref ref-type="supplementary-material" rid="SD1-data">Supplementary file 1</xref> (compact format) and <xref ref-type="supplementary-material" rid="SD4-data">4</xref> (complete database), which contain detailed information for all 8710 proteins covered in this study, including nuclear and cytosolic predictions. <xref ref-type="supplementary-material" rid="SD2-data">Supplementary file 2</xref> shows the performance of each individual map.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.16950.008">http://dx.doi.org/10.7554/eLife.16950.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Compartment</th><th rowspan="2" valign="top">Number of marker proteins</th><th colspan="2" valign="top">Correctly predicted <break/>markers</th><th rowspan="2" valign="top">All proteins predicted in this compartment</th></tr><tr><th valign="top">Number</th><th valign="top">%</th></tr></thead><tbody><tr><td valign="top">Endosome</td><td valign="top">85</td><td valign="top">75</td><td valign="top">88.2%</td><td valign="top">304</td></tr><tr><td valign="top">ER</td><td valign="top">127</td><td valign="top">127</td><td valign="top">100.0%</td><td valign="top">530</td></tr><tr><td valign="top">ER, high curvature</td><td valign="top">11</td><td valign="top">11</td><td valign="top">100.0%</td><td valign="top">45</td></tr><tr><td valign="top">ERGIC/cisGolgi</td><td valign="top">26</td><td valign="top">25</td><td valign="top">96.2%</td><td valign="top">73</td></tr><tr><td valign="top">Golgi</td><td valign="top">33</td><td valign="top">29</td><td valign="top">87.9%</td><td valign="top">190</td></tr><tr><td valign="top">Lysosome</td><td valign="top">43</td><td valign="top">41</td><td valign="top">95.3%</td><td valign="top">88</td></tr><tr><td valign="top">Mitochondrion</td><td valign="top">242</td><td valign="top">239</td><td valign="top">98.8%</td><td valign="top">658</td></tr><tr><td valign="top">Peroxisome</td><td valign="top">21</td><td valign="top">15</td><td valign="top">71.4%</td><td valign="top">25</td></tr><tr><td valign="top">Plasma membrane</td><td valign="top">127</td><td valign="top">123</td><td valign="top">96.9%</td><td valign="top">510</td></tr><tr><td valign="top">All organellar proteins</td><td valign="top">715</td><td valign="top">685</td><td valign="top">95.8%</td><td valign="top">2423</td></tr><tr><td valign="top">Average per organelle</td><td valign="top"/><td valign="top"/><td valign="top">92.7%</td><td valign="top"/></tr><tr><td valign="top">Large Protein Complexes</td><td valign="top">361</td><td valign="top">353</td><td valign="top">97.8%</td><td valign="top">2739</td></tr><tr><td valign="top"><bold>Total</bold></td><td valign="top"><bold>1076</bold></td><td valign="top"><bold>1038</bold></td><td valign="top"><bold>96.5%</bold></td><td valign="top"><bold>5162</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-16950-v3.xml">elife-16950-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.16950.008</object-id><label>Table 1.</label><caption><p>Prediction output and performance of HeLa organellar maps. The table shows the combined organellar prediction output from six replicate maps from HeLa cells. Prediction performance is judged by the proportion of correctly assigned organellar marker proteins. Please also refer to <xref ref-type="supplementary-material" rid="SD1-data">Supplementary file 1</xref> (compact format) and <xref ref-type="supplementary-material" rid="SD4-data">4</xref> (complete database), which contain detailed information for all 8710 proteins covered in this study, including nuclear and cytosolic predictions. <xref ref-type="supplementary-material" rid="SD2-data">Supplementary file 2</xref> shows the performance of each individual map.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.16950.008">http://dx.doi.org/10.7554/eLife.16950.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Compartment</th><th rowspan="2" valign="top">Number of marker proteins</th><th colspan="2" valign="top">Correctly predicted <break/>markers</th><th rowspan="2" valign="top">All proteins predicted in this compartment</th></tr><tr><th valign="top">Number</th><th valign="top">%</th></tr></thead><tbody><tr><td valign="top">Endosome</td><td valign="top">85</td><td valign="top">75</td><td valign="top">88.2%</td><td valign="top">304</td></tr><tr><td valign="top">ER</td><td valign="top">127</td><td valign="top">127</td><td valign="top">100.0%</td><td valign="top">530</td></tr><tr><td valign="top">ER, high curvature</td><td valign="top">11</td><td valign="top">11</td><td valign="top">100.0%</td><td valign="top">45</td></tr><tr><td valign="top">ERGIC/cisGolgi</td><td valign="top">26</td><td valign="top">25</td><td valign="top">96.2%</td><td valign="top">73</td></tr><tr><td valign="top">Golgi</td><td valign="top">33</td><td valign="top">29</td><td valign="top">87.9%</td><td valign="top">190</td></tr><tr><td valign="top">Lysosome</td><td valign="top">43</td><td valign="top">41</td><td valign="top">95.3%</td><td valign="top">88</td></tr><tr><td valign="top">Mitochondrion</td><td valign="top">242</td><td valign="top">239</td><td valign="top">98.8%</td><td valign="top">658</td></tr><tr><td valign="top">Peroxisome</td><td valign="top">21</td><td valign="top">15</td><td valign="top">71.4%</td><td valign="top">25</td></tr><tr><td valign="top">Plasma membrane</td><td valign="top">127</td><td valign="top">123</td><td valign="top">96.9%</td><td valign="top">510</td></tr><tr><td valign="top">All organellar proteins</td><td valign="top">715</td><td valign="top">685</td><td valign="top">95.8%</td><td valign="top">2423</td></tr><tr><td valign="top">Average per organelle</td><td valign="top"/><td valign="top"/><td valign="top">92.7%</td><td valign="top"/></tr><tr><td valign="top">Large Protein Complexes</td><td valign="top">361</td><td valign="top">353</td><td valign="top">97.8%</td><td valign="top">2739</td></tr><tr><td valign="top"><bold>Total</bold></td><td valign="top"><bold>1076</bold></td><td valign="top"><bold>1038</bold></td><td valign="top"><bold>96.5%</bold></td><td valign="top"><bold>5162</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-16962-v1.xml">elife-16962-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl4" position="float"><object-id pub-id-type="doi">10.7554/eLife.16962.034</object-id><label>Table 4.</label><caption><p>Input to left and right dorsal and medial muscle bands from motor neuron pairs at their neuromuscular junctions.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.16962.034">http://dx.doi.org/10.7554/eLife.16962.034</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"><p><bold>Motor neuron pair</bold></p> </th><th colspan="4" valign="bottom"><p><bold>Left muscle band</bold></p> </th><th colspan="4" valign="bottom"><p><bold>Right muscle band</bold></p> </th><th colspan="2" valign="bottom"><p><bold>Ratios</bold></p> </th></tr><tr><th align="center" colspan="2" valign="bottom"><p><bold>Dorsal</bold></p> </th><th align="center" colspan="2" valign="bottom"><p><bold>Medial</bold></p> </th><th align="center" colspan="2" valign="bottom"><p><bold>Dorsal</bold></p> </th><th align="center" colspan="2" valign="bottom"><p><bold>Medial</bold></p> </th><th colspan="2" valign="bottom"><p><bold>Left: Right</bold></p> </th></tr><tr><th valign="bottom"><p><bold>No. syn</bold></p> </th><th valign="bottom"><p><bold>No. sec</bold></p> </th><th valign="bottom"><p><bold>No. syn</bold></p> </th><th valign="bottom"><p><bold>No. sec</bold></p> </th><th valign="bottom"><p><bold>No. syn</bold></p> </th><th valign="bottom"><p><bold>No. sec</bold></p> </th><th valign="bottom"><p><bold>No. syn</bold></p> </th><th valign="bottom"><p><bold>No. sec</bold></p> </th><th valign="bottom"><p><bold>No. syn</bold></p> </th><th valign="bottom"><p><bold>No. sec</bold></p> </th></tr></thead><tbody><tr><td valign="bottom"><p><bold>MN1</bold></p> </td><td valign="bottom"><p>192</p></td><td valign="bottom"><p>969</p></td><td valign="bottom"><p>47</p></td><td valign="bottom"><p>145</p></td><td valign="bottom"><p>230</p></td><td valign="bottom"><p>1181</p></td><td valign="bottom"><p>130</p></td><td valign="bottom"><p>558</p></td><td valign="bottom"><p>40: 60</p></td><td valign="bottom"><p>39: 61</p></td></tr><tr><td valign="bottom"><p><bold>MN2</bold></p> </td><td valign="bottom"><p>224</p></td><td valign="bottom"><p>1583</p></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"><p>258</p></td><td valign="bottom"><p>1636</p></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"><p>46: 54</p></td><td valign="bottom"><p>49: 51</p></td></tr><tr><td valign="bottom"><p><bold>MN3</bold></p> </td><td valign="bottom"><p>42</p></td><td valign="bottom"><p>156</p></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"><p>28</p></td><td valign="bottom"><p>101</p></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"><p>60: 40</p></td><td valign="bottom"><p>61: 39</p></td></tr><tr><td valign="bottom"><p><bold>MN4</bold></p> </td><td><p>45</p></td><td valign="bottom"><p>189</p></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"><p>30</p></td><td valign="bottom"><p>116</p></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"><p>60: 40</p></td><td valign="bottom"><p>62: 38</p></td></tr><tr><td valign="bottom"><p><bold>MN5</bold></p> </td><td valign="bottom"><p>21</p></td><td valign="bottom"><p>128</p></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"><p>15</p></td><td valign="bottom"><p>55</p></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"><p>58: 42</p></td><td valign="bottom"><p>70: 30</p></td></tr></tbody></table><table-wrap-foot><fn id="tblfn5"><p>Number of synapses (No. syn) and number of synaptic profiles (No. sec) provided for each motor neuron and left:right ratios expressed as percentages of neuromuscular junction input from left and right partners for each motor neuron pair.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-17061-v1.xml">elife-17061-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.17061.015</object-id><label>Table 1.</label><caption><p>FGT1 interacts with chromatin remodeling proteins&#160;<italic>in vivo</italic>. FGT1-interacting proteins identified by native co-immunoprecipitation followed by mass spectrometry (nHPLC-MS/MS) from 5 d-old <italic>35S::FGT1-YFP</italic> seedlings subjected to ACC or NHS 28&#160;h before sampling. Col-0 and <italic>35S::YFP</italic> were used as controls. The&#160;data represent the&#160;number of unique peptides found in&#160;the indicated experiments.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.17061.015">http://dx.doi.org/10.7554/eLife.17061.015</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Background</th><th rowspan="2">Treatment</th><th rowspan="2">Exp</th><th colspan="9">Number of peptides</th></tr><tr><th>FGT1</th><th>CHR11/ CHR17</th><th>Chr11</th><th>Chr17</th><th>BRM</th><th>SWI3a</th><th>SWI3b</th><th>SWI3d</th><th>SWP73b</th></tr></thead><tbody><tr><td rowspan="6">35S::FGT1-YFP</td><td rowspan="3">ACC</td><td>1</td><td>58</td><td>4</td><td>1</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>2</td><td>56</td><td>12</td><td>3</td><td>2</td><td>2</td><td>3</td><td>1</td><td>1</td><td>3</td></tr><tr><td>3</td><td>43</td><td>11</td><td>-</td><td>-</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td rowspan="3">NHS</td><td>1</td><td>33</td><td>2</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>1</td></tr><tr><td>2</td><td>52</td><td>11</td><td>4</td><td>2</td><td>1</td><td>4</td><td>-</td><td>-</td><td>3</td></tr><tr><td>3</td><td>51</td><td>4</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td rowspan="2">Col-0</td><td>ACC</td><td>1-3</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>NHS</td><td>1-3</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>35S::YFP</td><td>NHS</td><td>1-3</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-17219-v2.xml">elife-17219-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.17219.010</object-id><label>Table 1.</label><caption><p>Structure refinement of macromolecular assemblies from cryo-EM maps using Rosetta.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.17219.010">http://dx.doi.org/10.7554/eLife.17219.010</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top">EMD ID</th><th rowspan="2" valign="top">PDB ID</th><th rowspan="2" valign="top">Reported resolution [&#197;]</th><th rowspan="2" valign="top">Symmetry</th><th rowspan="2" valign="top">Number of amino acids*</th><th colspan="4" valign="top">MolProbity<sup>&#8224;</sup></th><th rowspan="2" valign="top">EMRinger score<sup>&#8224;</sup></th><th rowspan="2" valign="top">iFSC<sup>&#8225;</sup></th></tr><tr><th valign="top">Score</th><th valign="top">Clash&#160;score</th><th valign="top">Rotamer outliers [%]</th><th valign="top">Ramachandran favored [%]</th></tr></thead><tbody><tr><td valign="top">TRPV1</td><td valign="top">5778</td><td valign="top">3j5p</td><td valign="top">3.4</td><td valign="top">C4</td><td valign="top">489 (1956)</td><td valign="top">3.81 / 1.45</td><td valign="top">86.35 / 1.96</td><td valign="top">28.78 / 0.00</td><td valign="top">95.65 / 91.93</td><td valign="top">0.65 / 2.34</td><td valign="top">0.612 / 0.607</td></tr><tr><td valign="top">Frh</td><td valign="top">2513</td><td valign="top">4ci0</td><td valign="top">3.4</td><td valign="top">T</td><td valign="top">893 (10,716)<sup>&#167;</sup></td><td valign="top">3.98 / 1.59</td><td valign="top">120.42 / 3.22</td><td valign="top">39.11 / 0.27</td><td valign="top">96.51 / 92.18</td><td valign="top">1.06 / 2.17</td><td valign="top">0.743 / 0.708</td></tr><tr><td valign="top">Mitoribosome</td><td valign="top">2762</td><td valign="top">3j7y</td><td valign="top">3.4</td><td valign="top">N/A</td><td valign="top">7469<sup>&#182;</sup></td><td valign="top">2.71 / 1.50</td><td valign="top">8.38 / 3.51</td><td valign="top">8.49 / 0.08</td><td valign="top">89.86 / 94.86</td><td valign="top">2.09 / 2.40</td><td valign="top">0.692 / 0.676</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>*Number of protein residues in the asymmetric unit and (the total residues) modeled.</p></fn><fn id="tblfn2"><p><sup>&#8224;</sup>Scores from deposited (left) versus (/) Rosetta refined (right) model.</p></fn><fn id="tblfn3"><p><sup>&#8225;</sup>Integrated Fourier Shell Correlation (iFSC) from 10&#8211;3.4&#197; resolution shells.</p></fn><fn id="tblfn4"><p><sup>&#167;</sup>In addition to protein residues, nine residues of ligand per asymmetric unit&#8211;including a [NiFe] cluster, two metal ions (Fe and Zn), and four [4Fe4S] clusters, and an FAD&#8211;were included in the refinement.</p></fn><fn id="tblfn5"><p><sup>&#182;</sup>In addition to protein residues, 1529 base pairs of RNA molecule were included in the refinement.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-17219-v2.xml">elife-17219-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.17219.017</object-id><label>Table 2.</label><caption><p>Comparison of structure refinement results between Rosetta and phenix.real_space_refine*.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.17219.017">http://dx.doi.org/10.7554/eLife.17219.017</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top">RSCC*<sup>,&#8224;,&#8225;</sup> <break/>validation map</th><th rowspan="2" valign="top">iFSC*<sup>,&#8224;,&#167;</sup> <break/>validation map</th><th rowspan="2" valign="top">EMRinger Score*<sup>,&#8224;</sup> <break/>validation map</th><th colspan="4" valign="top">MolProbity<sup>&#8224;</sup></th><th rowspan="2" valign="top">Number of residues with better RSCC<sup>&#8224;,&#182;</sup></th></tr><tr><th valign="top">Score</th><th valign="top">Clash&#160;score</th><th valign="top">Rotamer outliers [%]</th><th valign="top">Ramachandran favored [%]</th></tr></thead><tbody><tr><td valign="top">TRPV1</td><td valign="top">0.785 / 0.790</td><td valign="top">0.546 / 0.566</td><td valign="top">1.84 / 1.90</td><td valign="top">1.59 / 1.48</td><td valign="top">4.30 / 2.14</td><td valign="top">0.00 / 0.00</td><td valign="top">94.41 / 91.72</td><td valign="top">86 / 250</td></tr><tr><td valign="top">Frh</td><td valign="top">0.835 / 0.835</td><td valign="top">0.504 / 0.517</td><td valign="top">1.36 / 1.27</td><td valign="top">1.68 / 1.62</td><td valign="top">7.99 / 3.66</td><td valign="top">0.68 / 0.13</td><td valign="top">96.31 / 92.67</td><td valign="top">677 / 1328</td></tr><tr><td valign="top">Mitoribosome</td><td valign="top">0.832 / 0.832</td><td valign="top">0.476 / 0.478</td><td valign="top">2.05 / 1.98</td><td valign="top">1.88 / 1.62</td><td valign="top">6.17 / 4.08</td><td valign="top">0.38 / 0.00</td><td valign="top">90.19 / 93.49</td><td valign="top">415 / 564</td></tr></tbody></table><table-wrap-foot><fn id="tblfn6"><p>*To avoid over-fitting, refinement using both methods was carried out using the half-map approach, in which the models were subject to refinement using the training maps. The results showing here were evaluated using the validation-maps. The input model information is the same as reported at <xref ref-type="table" rid="tbl1">Table 1</xref>.</p></fn><fn id="tblfn7"><p><sup>&#8224;</sup>Numbers (scores) from <italic>phenix.real_space_refine</italic> (left) versus (/) Rosetta refined (right) model.</p></fn><fn id="tblfn8"><p><sup>&#8225;</sup>Real-space correlation coefficients were evaluated using UCSF Chimera.</p></fn><fn id="tblfn9"><p><sup>&#167;</sup>Integrated Fourier shell correlation (iFSC) from 10&#8211;3.4&#197; resolution shells.</p></fn><fn id="tblfn10"><p><sup>&#182;</sup>We calculate per-residue real-space correlation coefficient and report the number of residues which show the value of &#916;RSCC greater than 0.05.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-17243-v2.xml">elife-17243-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.17243.003</object-id><label>Table 1.</label><caption><p>Patients' demographic, clinical and experimental details.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.17243.003">http://dx.doi.org/10.7554/eLife.17243.003</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top">Patient code*</th><th rowspan="3" valign="top">Sex</th><th rowspan="3" valign="top">Age</th><th rowspan="3" valign="top">Seizure onset zone(s)</th><th rowspan="3" valign="top">Voluntary blinks blocks</th><th rowspan="3" valign="top">Black <break/>/gray gap control</th><th rowspan="3" valign="top">Gradual / abrupt gap control</th><th rowspan="3" valign="top">Total analyzed electrodes</th><th rowspan="3" valign="top">Total visually responsive electrodes</th><th colspan="7" valign="top">Number of visually responsive electrodes in each ROI</th></tr><tr><th colspan="5" valign="top">Retinotopic</th><th colspan="2" valign="top">High-level</th></tr><tr><th valign="top">V1</th><th valign="top">V2</th><th valign="top">V3</th><th valign="top">V4</th><th valign="top">VO</th><th valign="top">FC</th><th valign="top">N-FC</th></tr></thead><tbody><tr><td valign="top">P20</td><td valign="top">F</td><td valign="top">30</td><td valign="top">RH: Supramarginal Gyrus</td><td valign="top">&#10003;</td><td valign="top"/><td valign="top"/><td valign="top">103</td><td valign="top">6</td><td valign="top">0</td><td valign="top">0</td><td valign="top">2</td><td valign="top">0</td><td valign="top">0</td><td valign="top">1</td><td valign="top">2</td></tr><tr><td valign="top">P25</td><td valign="top">M</td><td valign="top">45</td><td valign="top">RH: Inferior Frontal Gyrus, Precentral S.</td><td valign="top">&#10003;</td><td valign="top"/><td valign="top"/><td valign="top">117</td><td valign="top">8</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">2</td><td valign="top">6</td></tr><tr><td valign="top">P32</td><td valign="top">M</td><td valign="top">23</td><td valign="top">RH: Superior Temporal Gyrus, Hippocampus</td><td valign="top">&#10003;</td><td valign="top"/><td valign="top"/><td valign="top">181</td><td valign="top">10</td><td valign="top">2</td><td valign="top">2</td><td valign="top">0</td><td valign="top">1</td><td valign="top">0</td><td valign="top">1</td><td valign="top">2</td></tr><tr><td valign="top">P33</td><td valign="top">F</td><td valign="top">52</td><td valign="top">LH: Hippocampus, Middle Entorhinal Cortex</td><td valign="top">&#10003;</td><td valign="top"/><td valign="top"/><td valign="top">83</td><td valign="top">8</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">4</td><td valign="top">1</td></tr><tr><td valign="top">P36</td><td valign="top">M</td><td valign="top">24</td><td valign="top">RH: Parahippocampal Gyrus, Temporal Pole</td><td valign="top">**</td><td valign="top">&#10003;</td><td valign="top"/><td valign="top">58</td><td valign="top">8</td><td valign="top">0</td><td valign="top">0</td><td valign="top">1</td><td valign="top">0</td><td valign="top">1</td><td valign="top">1</td><td valign="top">4</td></tr><tr><td valign="top">P39</td><td valign="top">M</td><td valign="top">25</td><td valign="top">RH: Hippocampus, Amygdala</td><td valign="top">&#10003;</td><td valign="top">&#10003;</td><td valign="top"/><td valign="top">128</td><td valign="top">18</td><td valign="top">3</td><td valign="top">2</td><td valign="top">2</td><td valign="top">1</td><td valign="top">2</td><td valign="top">3</td><td valign="top">4</td></tr><tr><td valign="top">P44</td><td valign="top">M</td><td valign="top">30</td><td valign="top">RH: Anterior Temporal Lobe</td><td valign="top">&#10003;</td><td valign="top">&#10003;</td><td valign="top"/><td valign="top">118</td><td valign="top">10</td><td valign="top">1</td><td valign="top">1</td><td valign="top">1</td><td valign="top">0</td><td valign="top">1</td><td valign="top">1</td><td valign="top">1</td></tr><tr><td valign="top">P46</td><td valign="top">M</td><td valign="top">45</td><td valign="top">RH: Hippocampus, Parahippocampal Gyrus</td><td valign="top">&#10003;</td><td valign="top">&#10003;</td><td valign="top">&#10003;</td><td valign="top">58</td><td valign="top">15</td><td valign="top">3</td><td valign="top">2</td><td valign="top">2</td><td valign="top">1</td><td valign="top">2</td><td valign="top">1</td><td valign="top">4</td></tr><tr><td valign="top">P47</td><td valign="top">F</td><td valign="top">34</td><td valign="top">LH: Anterior Temporal Lobe</td><td valign="top">&#10003;</td><td valign="top">&#10003;</td><td valign="top">&#10003;</td><td valign="top">142</td><td valign="top">7</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">1</td><td valign="top">1</td><td valign="top">3</td><td valign="top">2</td></tr><tr><td valign="top">P50</td><td valign="top">M</td><td valign="top">27</td><td valign="top">LH: Amygdala, Hippocampus, Parahippocampal Gyrus, Anterior Fusiform Gyrus, Post Central Gyrus</td><td valign="top">&#10003;</td><td valign="top">&#10003;</td><td valign="top">&#10003;</td><td valign="top">108</td><td valign="top">3</td><td valign="top">2</td><td valign="top">1</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top">P54</td><td valign="top">M</td><td valign="top">21</td><td valign="top">RH: Medial Temporal, Middle Occipital Gyrus, Parieto-Occipital-Sulcus, Middle Temporal Gyrus, LH: Hippocampus</td><td valign="top">&#10003;</td><td valign="top">&#10003;</td><td valign="top">&#10003;</td><td valign="top">160</td><td valign="top">23</td><td valign="top">2</td><td valign="top">3</td><td valign="top">4</td><td valign="top">2</td><td valign="top">0</td><td valign="top">2</td><td valign="top">0</td></tr><tr><td valign="top">P57</td><td valign="top">M</td><td valign="top">29</td><td valign="top">RH: Amygdala</td><td valign="top"/><td valign="top">&#10003;</td><td valign="top">&#10003;</td><td valign="top">110</td><td valign="top">6</td><td valign="top">1</td><td valign="top">0</td><td valign="top">1</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">2</td></tr><tr><td valign="top">P59</td><td valign="top">M</td><td valign="top">50</td><td valign="top">RH: Parieto-Occipital Sulcus</td><td valign="top"/><td valign="top">&#10003;</td><td valign="top">&#10003;</td><td valign="top">94</td><td valign="top">13</td><td valign="top">3</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">1</td><td valign="top">0</td><td valign="top">2</td></tr><tr><td valign="top">P62</td><td valign="top">F</td><td valign="top">44</td><td valign="top">LH: Hippocampus, Anterior Cingulate Gyrus, Amygdala, Parahippocampal Gyrus</td><td valign="top"/><td valign="top">&#10003;</td><td valign="top">&#10003;</td><td valign="top">125</td><td valign="top">8</td><td valign="top">0</td><td valign="top">1</td><td valign="top">2</td><td valign="top">1</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>LH/RH &#8211; left/right hemisphere, VO &#8211; ventral-occipital, FC &#8211; face-selective electrodes, N-FC non-face selective high-level electrodes. * Patients' identities were coded by order of admission to surgery. Since not every admitted patient performed the experiment, the codes are not consecutive. ** Failed to follow the instruction to voluntary blink due to language barrier.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-17328-v1.xml">elife-17328-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.17328.004</object-id><label>Table 1.</label><caption><p>Summary of task conditions.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.17328.004">http://dx.doi.org/10.7554/eLife.17328.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Task</th><th colspan="5">CS (% outcome)</th><th rowspan="2">Reward trials (%)</th><th rowspan="2">Free reward (%)</th></tr><tr><th>Outcome</th><th>Odor A <break/>(Reward CS)</th><th>Odor B <break/>(Nothing CS)</th><th>Odor C <break/>(Air puff CS)</th><th>Odor D <break/>(Reward and air puff CS)</th></tr></thead><tbody><tr><td rowspan="2">Mixed prediction task</td><td>Water</td><td>25</td><td>0</td><td>0</td><td>25</td><td rowspan="2">13</td><td rowspan="2">2</td></tr><tr><td>Air puff</td><td>0</td><td>0</td><td>75</td><td>75</td></tr><tr><td rowspan="2">Low reward probability task</td><td>Water</td><td>20</td><td>0</td><td>0</td><td rowspan="2"/><td rowspan="2">7</td><td rowspan="2">6</td></tr><tr><td>Air puff</td><td>0</td><td>0</td><td>90</td></tr><tr><td rowspan="2">High reward probability task</td><td>Water</td><td>90</td><td>0</td><td>0</td><td rowspan="2"/><td rowspan="2">30</td><td rowspan="2">6</td></tr><tr><td>Air puff</td><td>0</td><td>0</td><td>90</td></tr><tr><td rowspan="2">High reward probability task 2</td><td>Water</td><td>90</td><td>0</td><td>0</td><td rowspan="2"/><td rowspan="2">30</td><td rowspan="2">7</td></tr><tr><td>Air puff</td><td>0</td><td>0</td><td>80</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-17556-v1.xml">elife-17556-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.17556.008</object-id><label>Table 1.</label><caption><p>Thermodynamic parameters describing the&#160;binding of Sir3 protein to nucleosomes.&#160;Data from more than 2 replicate titration experiments were pooled and the BLI data were fit with Hill equation (see Materials&#160;and&#160;methods). Uncertainties show 68% confidence intervals around fit parameters (&#177;1 SD) reported by fitting algorithm.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.17556.008">http://dx.doi.org/10.7554/eLife.17556.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left" colspan="2" rowspan="2"><p>Binding experiments</p> </th><th align="left" colspan="2"><p>BLI</p></th><th align="left"><p>EMSA<sup>*</sup></p></th></tr><tr><th><p><italic>Apparent K<sub>D</sub></italic> (&#181;M)</p></th><th><p><italic>Hill coefficient</italic></p> </th><th><p><italic>Apparent K<sub>D</sub></italic> (&#181;M)</p></th></tr></thead><tbody><tr><td rowspan="5"><p>MonoN</p></td><td><p>Sir3</p></td><td><p>1.4&#160;&#177;&#160;0.06</p> </td><td><p>1.3&#160;&#177;&#160;0.1</p> </td><td><p>1.7&#160;&#177;&#160;0.20</p> </td></tr><tr><td><p>Sir3+Sir4CC</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>1.4&#160;&#177;&#160;0.10</p> </td></tr><tr><td><p>Sir3&#8710;wH</p></td><td><p>1.2&#160;&#177;&#160;0.10</p> </td><td><p>0.93&#160;&#177;&#160;0.07</p> </td><td><p>1.0&#160;&#177;&#160;0.10</p> </td></tr><tr><td><p>Sir3&#8710;wH+Sir4CC</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>0.9&#160;&#177;&#160;0.05</p> </td></tr><tr><td><p>Sir3BAH</p></td><td><p>1.4&#160;&#177;&#160;0.10</p> </td><td><p>1.5&#160;&#177;&#160;0.2</p> </td><td><p>2.1&#160;&#177;&#160;0.20</p> </td></tr><tr><td rowspan="5"><p>DiN</p></td><td><p>Sir3</p></td><td><p>0.12&#160;&#177;&#160;0.01</p> </td><td><p>1.9&#160;&#177;&#160;0.2</p> </td><td><p>0.17&#160;&#177;&#160;0.10</p> </td></tr><tr><td><p>Sir3+Sir4CC</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>0.08&#160;&#177;&#160;0.01</p> </td></tr><tr><td><p>Sir3&#8710;wH</p></td><td><p>1.1&#160;&#177;&#160;0.05</p> </td><td><p>1.2&#160;&#177;&#160;0.1</p> </td><td><p>0.62&#160;&#177;&#160;0.10</p> </td></tr><tr><td><p>Sir3&#8710;wH+Sir4CC</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>0.12&#160;&#177;&#160;0.01</p> </td></tr><tr><td><p>Sir3BAH</p></td><td><p>1.6&#160;&#177;&#160;0.10</p> </td><td><p>1.4&#160;&#177;&#160;0.1</p> </td><td><p>1.40&#160;&#177;&#160;0.20</p> </td></tr><tr><td rowspan="6"><p>Sir3</p></td><td><p>acMonoN</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>4.0&#160;&#177;&#160;0.20</p> </td></tr><tr><td><p>meMonoN</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>5.2&#160;&#177;&#160;0.20</p> </td></tr><tr><td><p>acDiN</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>0.7&#160;&#177;&#160;0.05</p> </td></tr><tr><td><p>meDiN</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>0.8&#160;&#177;&#160;0.05</p> </td></tr><tr><td><p>ac/meMonoN</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>&gt;11<sup>&#8224;</sup></p></td></tr><tr><td><p>ac/meDiN</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>&gt;3<sup>&#8224;</sup></p></td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>*Hill coefficients obtained from EMSA appeared unreliable due to assay artifacts, such as non-specific binding to DNA, and are not reported.</p></fn><fn id="tblfn2"><p><bold><sup>&#8224;</sup> </bold>Nonspecific binding could not be measured accurately due to low affinity.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-17556-v2.xml">elife-17556-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.17556.008</object-id><label>Table 1.</label><caption><p>Thermodynamic parameters describing the&#160;binding of Sir3 protein to nucleosomes.&#160;Data from more than 2 replicate titration experiments were pooled and the BLI data were fit with Hill equation (see Materials&#160;and&#160;methods). Uncertainties show 68% confidence intervals around fit parameters (&#177;1 SD) reported by fitting algorithm.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.17556.008">http://dx.doi.org/10.7554/eLife.17556.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left" colspan="2" rowspan="2"><p>Binding experiments</p> </th><th align="left" colspan="2"><p>BLI</p></th><th align="left"><p>EMSA<sup>*</sup></p></th></tr><tr><th><p><italic>Apparent K<sub>D</sub></italic> (&#181;M)</p></th><th><p><italic>Hill coefficient</italic></p> </th><th><p><italic>Apparent K<sub>D</sub></italic> (&#181;M)</p></th></tr></thead><tbody><tr><td rowspan="5"><p>MonoN</p></td><td><p>Sir3</p></td><td><p>1.4&#160;&#177;&#160;0.06</p> </td><td><p>1.3&#160;&#177;&#160;0.1</p> </td><td><p>1.7&#160;&#177;&#160;0.20</p> </td></tr><tr><td><p>Sir3+Sir4CC</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>1.4&#160;&#177;&#160;0.10</p> </td></tr><tr><td><p>Sir3&#8710;wH</p></td><td><p>1.2&#160;&#177;&#160;0.10</p> </td><td><p>0.93&#160;&#177;&#160;0.07</p> </td><td><p>1.0&#160;&#177;&#160;0.10</p> </td></tr><tr><td><p>Sir3&#8710;wH+Sir4CC</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>0.9&#160;&#177;&#160;0.05</p> </td></tr><tr><td><p>Sir3BAH</p></td><td><p>1.4&#160;&#177;&#160;0.10</p> </td><td><p>1.5&#160;&#177;&#160;0.2</p> </td><td><p>2.1&#160;&#177;&#160;0.20</p> </td></tr><tr><td rowspan="5"><p>DiN</p></td><td><p>Sir3</p></td><td><p>0.12&#160;&#177;&#160;0.01</p> </td><td><p>1.9&#160;&#177;&#160;0.2</p> </td><td><p>0.17&#160;&#177;&#160;0.10</p> </td></tr><tr><td><p>Sir3+Sir4CC</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>0.08&#160;&#177;&#160;0.01</p> </td></tr><tr><td><p>Sir3&#8710;wH</p></td><td><p>1.1&#160;&#177;&#160;0.05</p> </td><td><p>1.2&#160;&#177;&#160;0.1</p> </td><td><p>0.62&#160;&#177;&#160;0.10</p> </td></tr><tr><td><p>Sir3&#8710;wH+Sir4CC</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>0.12&#160;&#177;&#160;0.01</p> </td></tr><tr><td><p>Sir3BAH</p></td><td><p>1.6&#160;&#177;&#160;0.10</p> </td><td><p>1.4&#160;&#177;&#160;0.1</p> </td><td><p>1.40&#160;&#177;&#160;0.20</p> </td></tr><tr><td rowspan="6"><p>Sir3</p></td><td><p>acMonoN</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>4.0&#160;&#177;&#160;0.20</p> </td></tr><tr><td><p>meMonoN</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>5.2&#160;&#177;&#160;0.20</p> </td></tr><tr><td><p>acDiN</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>0.7&#160;&#177;&#160;0.05</p> </td></tr><tr><td><p>meDiN</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>0.8&#160;&#177;&#160;0.05</p> </td></tr><tr><td><p>ac/meMonoN</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>&gt;11<sup>&#8224;</sup></p></td></tr><tr><td><p>ac/meDiN</p></td><td><p>N/A</p></td><td><p>N/A</p></td><td><p>&gt;3<sup>&#8224;</sup></p></td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>*Hill coefficients obtained from EMSA appeared unreliable due to assay artifacts, such as non-specific binding to DNA, and are not reported.</p></fn><fn id="tblfn2"><p><bold><sup>&#8224;</sup> </bold>Nonspecific binding could not be measured accurately due to low affinity.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-17666-v2.xml">elife-17666-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.17666.018</object-id><label>Table 3.</label><caption><p>Changes in marker gene expression in the placodes after injection of various constructs.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.17666.018">http://dx.doi.org/10.7554/eLife.17666.018</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top"/><th valign="top">Injection</th><th valign="top"><italic>Six1</italic>-MO*</th><th valign="top"><italic>Eya1</italic>-MO*</th><th valign="top">Six1-EnR</th><th valign="top"><italic>Eya1-</italic>mmMO</th><th valign="top">Six1-GR<sup>&#167;</sup></th><th valign="top">Eya1-GR<sup>&#167;</sup></th></tr><tr><th rowspan="2" valign="top"/><th rowspan="2">Phenotype</th><th valign="top">%</th><th valign="top">%</th><th valign="top">%</th><th valign="top">%</th><th valign="top">%</th><th valign="top">%</th></tr><tr><th valign="top">(n)</th><th valign="top">(n)</th><th valign="top">(n)</th><th valign="top">(n)</th><th valign="top">(n)</th><th valign="top">(n)</th></tr></thead><tbody><tr><td rowspan="4">Atoh1</td><td rowspan="2">Reduced</td><td valign="top">77**</td><td valign="top">90<sup>&#8225;</sup></td><td valign="top">94</td><td valign="bottom">10</td><td valign="top">26</td><td valign="top">42</td></tr><tr><td valign="top">(26)</td><td valign="top">(20)</td><td valign="top">(18)</td><td valign="bottom">(21)</td><td valign="top">(19)</td><td valign="top">(12)</td></tr><tr><td rowspan="2">Increased</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="bottom">0</td><td valign="top">35</td><td valign="top">42</td></tr><tr><td valign="top">(26)</td><td valign="top">(20)</td><td valign="top">(18)</td><td valign="bottom">(21)</td><td valign="top">(17)</td><td valign="top">(12)</td></tr><tr><td rowspan="4">Gfi1a</td><td rowspan="2">Reduced</td><td valign="top">82<sup>&#8224;</sup></td><td valign="top">67<sup>&#8224;</sup></td><td valign="top">69</td><td valign="bottom">31</td><td valign="top">57</td><td valign="top">36</td></tr><tr><td valign="top">(27)</td><td valign="top">(17)</td><td valign="top">(16)</td><td valign="bottom">(26)</td><td valign="top">(14)</td><td valign="top">(14)</td></tr><tr><td rowspan="2">Increased</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="bottom">0</td><td valign="top">7</td><td valign="top">43</td></tr><tr><td valign="top">(27)</td><td valign="top">(17)</td><td valign="top">(16)</td><td valign="bottom">(26)</td><td valign="top">(14)</td><td valign="top">(14)</td></tr><tr><td rowspan="4">Hes8</td><td rowspan="2">Reduced</td><td valign="top">74<sup>&#8225;</sup></td><td valign="top">83<sup>&#8225;</sup></td><td valign="top">70</td><td valign="bottom">17</td><td valign="top">60</td><td valign="top">57</td></tr><tr><td valign="top">(35)</td><td valign="top">(35)</td><td valign="top">(46)</td><td valign="bottom">(24)</td><td valign="top">(40)</td><td valign="top">(56)</td></tr><tr><td rowspan="2">Increased</td><td valign="top">0</td><td valign="top">0</td><td valign="top">24</td><td valign="bottom">0</td><td valign="top">15</td><td valign="top">29</td></tr><tr><td valign="top">(35)</td><td valign="top">(35)</td><td valign="top">(46)</td><td valign="bottom">(24)</td><td valign="top">(40)</td><td valign="top">(56)</td></tr><tr><td rowspan="4">Hes9</td><td rowspan="2">Reduced</td><td valign="top">73<sup>&#8225;</sup></td><td valign="top">76<sup>&#8225;</sup></td><td valign="top">84</td><td valign="bottom">11</td><td valign="top">75</td><td valign="top">29</td></tr><tr><td valign="top">(45)</td><td valign="top">(33)</td><td valign="top">(38)</td><td valign="bottom">(27)</td><td valign="top">(12)</td><td valign="top">(29)</td></tr><tr><td rowspan="2">Increased</td><td valign="top">0</td><td valign="top">0</td><td valign="top">8</td><td valign="bottom">0</td><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top">(45)</td><td valign="top">(33)</td><td valign="top">(38)</td><td valign="bottom">(27)</td><td valign="top">(12)</td><td valign="top">(29)</td></tr><tr><td rowspan="4">Isl2</td><td rowspan="2">Reduced</td><td valign="top">66<sup>&#8224;</sup></td><td valign="top">100<sup>&#8225;</sup></td><td valign="top">nd</td><td valign="bottom">27</td><td valign="top">50</td><td valign="top">24</td></tr><tr><td valign="top">(38)</td><td valign="top">(17)</td><td valign="top">nd</td><td valign="bottom">(22)</td><td valign="top">(18)</td><td valign="top">(17)</td></tr><tr><td rowspan="2">Increased</td><td valign="top">6</td><td valign="top">0</td><td valign="top">nd</td><td valign="bottom">0</td><td valign="top">31</td><td valign="top">41</td></tr><tr><td valign="top">(38)</td><td valign="top">(17)</td><td valign="top">nd</td><td valign="bottom">(22)</td><td valign="top">(16)</td><td valign="top">(17)</td></tr><tr><td rowspan="4">Ngn1</td><td rowspan="2">Reduced</td><td valign="top">65<sup>&#8225;</sup></td><td valign="top">49<sup>&#8224;</sup></td><td valign="top">84</td><td valign="bottom">17</td><td valign="top">17</td><td valign="top">36</td></tr><tr><td valign="top">(51)</td><td valign="top">(43)</td><td valign="top">(31)</td><td valign="bottom">(24)</td><td valign="top">(30)</td><td valign="top">(59)</td></tr><tr><td rowspan="2">Increased</td><td valign="top">0</td><td valign="top">16</td><td valign="top">6</td><td valign="bottom">4</td><td valign="top">23</td><td valign="top">41</td></tr><tr><td valign="top">(51)</td><td valign="top">(43)</td><td valign="top">(31)</td><td valign="bottom">(24)</td><td valign="top">(30)</td><td valign="top">(59)</td></tr><tr><td rowspan="4">Pou4f1.2</td><td rowspan="2">Reduced</td><td valign="top">67<sup>&#8225;</sup></td><td valign="top">63<sup>&#8224;</sup></td><td valign="top">71</td><td valign="bottom">16</td><td valign="top">47</td><td valign="top">81</td></tr><tr><td valign="top">(48)</td><td valign="top">(30)</td><td valign="top">(35)</td><td valign="bottom">(19)</td><td valign="top">(15)</td><td valign="top">(37)</td></tr><tr><td rowspan="2">Increased</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="bottom">0</td><td valign="top">13</td><td valign="top">0</td></tr><tr><td valign="top">(48)</td><td valign="top">(30)</td><td valign="top">(35)</td><td valign="bottom">(19)</td><td valign="top">(15)</td><td valign="top">(37)</td></tr><tr><td rowspan="4">Sox2</td><td rowspan="2">Reduced</td><td valign="top">74<sup>&#8225;</sup></td><td valign="top">78<sup>&#8225;</sup></td><td valign="top">87</td><td valign="bottom">6</td><td valign="top">90</td><td valign="top">48</td></tr><tr><td valign="top">(19)</td><td valign="top">(18)</td><td valign="top">(30)</td><td valign="bottom">(16)</td><td valign="top">(21)</td><td valign="top">(33)</td></tr><tr><td rowspan="2">Increased</td><td valign="top">0</td><td valign="top">0</td><td valign="top">23<sup>#</sup></td><td valign="bottom">0</td><td valign="top">0</td><td valign="top">12</td></tr><tr><td valign="top">(19)</td><td valign="top">(18)</td><td valign="top">(30)</td><td valign="bottom">(16)</td><td valign="top">(21)</td><td valign="top">(33)</td></tr><tr><td rowspan="4">Sox3</td><td rowspan="2">Reduced</td><td valign="top">68<sup>&#8225;</sup></td><td valign="top">54<sup>&#8224;</sup></td><td valign="top">39</td><td valign="bottom">9</td><td valign="top">49</td><td valign="top">40</td></tr><tr><td valign="top">(25)</td><td valign="top">(26)</td><td valign="top">(31)</td><td valign="bottom">(22)</td><td valign="top">(25)</td><td valign="top">(23)</td></tr><tr><td rowspan="2">Increased</td><td valign="top">0</td><td valign="top">0</td><td valign="top">71<sup>#</sup></td><td valign="bottom">0</td><td valign="top">16</td><td valign="top">17</td></tr><tr><td valign="top">(25)</td><td valign="top">(26)</td><td valign="top">(31)</td><td valign="bottom">(22)</td><td valign="top">(25)</td><td valign="top">(23)</td></tr><tr><td rowspan="4">Tlx1</td><td rowspan="2">Reduced</td><td valign="top">84<sup>&#8224;</sup></td><td valign="top">91<sup>&#8225;</sup></td><td valign="top">100</td><td valign="bottom">33</td><td valign="top">40</td><td valign="top">7</td></tr><tr><td valign="top">(31)</td><td valign="top">(32)</td><td valign="top">(13)</td><td valign="bottom">(15)</td><td valign="top">(10)</td><td valign="top">(15)</td></tr><tr><td rowspan="2">Increased</td><td valign="top">6</td><td valign="top">0</td><td valign="top">0</td><td valign="bottom">0</td><td valign="top">40</td><td valign="top">73</td></tr><tr><td valign="top">(31)</td><td valign="top">(32)</td><td valign="top">(13)</td><td valign="bottom">(15)</td><td valign="top">(10)</td><td valign="top">(15)</td></tr></tbody></table><table-wrap-foot><fn id="tblfn13"><p>* Significant differences (Fisher&#8217;s exact test);</p></fn><fn id="tblfn14"><p><sup>&#8224;</sup> p&lt;0.05,</p></fn><fn id="tblfn15"><p><sup>&#8225;</sup> p&lt;0.001) to <italic>Eya1-</italic>mmMO injections are indicated.</p></fn><fn id="tblfn16"><p><sup>&#167;</sup> Dexamethasone treatment from stages 16&#8211;18 on.</p></fn><fn id="tblfn17"><p><sup>#</sup> Expression ectopic in epidermis.</p></fn><fn id="tblfn18"><p>n: Number of embryos analysed at both neural plate (stage 14&#8211;16) and tail bud (stage 21&#8211;26) stage.</p></fn><fn id="tblfn19"><p>nd: Not determined.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-17686-v1.xml">elife-17686-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.17686.007</object-id><label>Table 3.</label><caption><p>Synaptic targets of GABA(+); UNC-47(+) neurons. Coloured box indicates that GABAergic output neuron synapses onto this target cell (&lt;5 synapses [grey]; 5-20 synapses [orange]; &gt;20 synapses [pink]) and genes in the boxes represent expression of ionotropic GABA<sub>A</sub>-type receptor subunits in the target cell (<xref ref-type="fig" rid="fig4">Figure&#160;4</xref>). For a complete list of GABA receptor (+) neurons, see <xref ref-type="table" rid="tbl4">Table 4</xref>. Blue shading indicates that the target neurons is GABA(+). Wiring data is from <ext-link ext-link-type="uri" xlink:href="http://www.wormwiring.org">www.wormwiring.org</ext-link> and includes all synapses observed.&#160;*&#160;=&#160;RMDL/R&#160;only.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.17686.007">http://dx.doi.org/10.7554/eLife.17686.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr valign="top"><th colspan="2" rowspan="2">Targets</th><th colspan="8">GABAergic output neurons</th></tr><tr valign="top"><th>ALA</th><th>RIB</th><th>RIS</th><th>AVL</th><th>DVB</th><th>RME</th><th>SMD</th><th>DD/VD</th></tr></thead><tbody><tr valign="top"><td>muscle</td><td/><td/><td/><td/><td style="author-callout-style-b3"><italic>exp-1, lgc-37, gab-1</italic></td><td style="author-callout-style-b3"><italic>exp-1, lgc-37, gab-1</italic></td><td style="author-callout-style-b7"><italic>unc-49, lgc-37, gab-1</italic></td><td style="author-callout-style-b7"><italic>unc-49, lgc-37, gab-1</italic></td><td style="author-callout-style-b7"><italic>unc-49, lgc-37, gab-1</italic></td></tr><tr valign="top"><td rowspan="10">sensory neuron(9 classes)</td><td>ASG</td><td/><td/><td/><td/><td/><td/><td style="author-callout-style-b8"/><td/></tr><tr valign="top"><td>ASJ</td><td style="author-callout-style-b8"/><td/><td/><td/><td/><td/><td style="author-callout-style-b8"/><td/></tr><tr valign="top"><td>BAG</td><td/><td style="author-callout-style-b3"/><td/><td/><td/><td/><td/><td/></tr><tr valign="top"><td>CEPD</td><td/><td/><td style="author-callout-style-b3"/><td/><td/><td/><td/><td/></tr><tr valign="top"><td>CEPV</td><td/><td style="author-callout-style-b3"/><td style="author-callout-style-b3"/><td/><td/><td/><td style="author-callout-style-b8"/><td/></tr><tr valign="top"><td>IL1</td><td/><td/><td/><td/><td/><td style="author-callout-style-b3"/><td/><td/></tr><tr valign="top"><td>IL2</td><td/><td/><td/><td/><td/><td/><td style="author-callout-style-b8"><italic>lgc-38</italic></td><td/></tr><tr valign="top"><td>PHC</td><td/><td/><td/><td/><td style="author-callout-style-b8"/><td/><td/><td/></tr><tr valign="top"><td>OLL</td><td/><td style="author-callout-style-b8"><italic>lgc-38</italic></td><td style="author-callout-style-b3"><italic>lgc-38</italic></td><td/><td/><td style="author-callout-style-b8"><italic>lgc-38</italic></td><td/><td/></tr><tr valign="top"><td>URY</td><td/><td style="author-callout-style-b8"/><td style="author-callout-style-b3"/><td/><td/><td/><td/><td/></tr><tr valign="top"><td rowspan="21">inter-neuron (22 classes)</td><td>AIB</td><td/><td style="author-callout-style-b3"><italic>lgc-38</italic></td><td/><td/><td/><td/><td style="author-callout-style-b8"><italic>lgc-38</italic></td><td/></tr><tr valign="top"><td>AIN</td><td/><td style="author-callout-style-b8"><italic>lgc-38</italic></td><td/><td/><td/><td/><td/><td/></tr><tr valign="top"><td>AIZ</td><td/><td style="author-callout-style-b8"><italic>lgc-37, lgc-38, gab-1</italic></td><td/><td/><td/><td/><td style="author-callout-style-b8"><italic>lgc-37, lgc-38, gab-1</italic></td><td/></tr><tr valign="top"><td>ALN</td><td/><td/><td/><td/><td/><td/><td style="author-callout-style-b8"/><td/></tr><tr valign="top"><td>AUA</td><td/><td style="author-callout-style-b3"/><td/><td/><td/><td/><td/><td/></tr><tr valign="top"><td>AVA</td><td/><td style="author-callout-style-b3"><italic>lgc-37, gab-1</italic></td><td/><td/><td/><td/><td style="author-callout-style-b8"><italic>lgc-37, gab-1</italic></td><td/></tr><tr valign="top"><td>AVB</td><td/><td style="author-callout-style-b8"/><td/><td/><td/><td/><td/><td/></tr><tr valign="top"><td>AVD</td><td style="author-callout-style-b8"><italic>lgc-35</italic></td><td style="author-callout-style-b8"><italic>lgc-35</italic></td><td/><td/><td/><td/><td/><td/></tr><tr valign="top"><td>AVE</td><td style="author-callout-style-b3"><italic>gab-1</italic></td><td style="author-callout-style-b7"><italic>gab-1</italic></td><td style="author-callout-style-b7"><italic>gab-1</italic></td><td style="author-callout-style-b8"><italic>gab-1</italic></td><td/><td/><td style="author-callout-style-b8"><italic>gab-1</italic></td><td/></tr><tr valign="top"><td>AVK</td><td/><td/><td style="author-callout-style-b3"/><td style="author-callout-style-b8"/><td/><td/><td/><td/></tr><tr valign="top"><td>DVC</td><td/><td style="author-callout-style-b8"/><td/><td style="author-callout-style-b8"/><td style="author-callout-style-b8"/><td/><td/><td/></tr><tr valign="top"><td>PVP</td><td style="author-callout-style-b8"/><td/><td/><td style="author-callout-style-b8"/><td style="author-callout-style-b8"/><td/><td/><td/></tr><tr valign="top"><td>PVR</td><td/><td/><td/><td/><td/><td/><td style="author-callout-style-b8"><italic>lgc-37, gab-1</italic></td><td/></tr><tr valign="top"><td>PVW</td><td/><td/><td/><td style="author-callout-style-b8"><italic>lgc-37, gab-1</italic></td><td/><td/><td/><td/></tr><tr valign="top"><td>RIA</td><td style="author-callout-style-b8"/><td style="author-callout-style-b3"/><td/><td/><td/><td/><td style="author-callout-style-b7"/><td/></tr><tr valign="top"><td style="author-callout-style-b1">RIB</td><td/><td/><td style="author-callout-style-b7"/><td/><td/><td style="author-callout-style-b8"/><td style="author-callout-style-b8"/><td/></tr><tr valign="top"><td>RIG</td><td/><td style="author-callout-style-b3"/><td/><td/><td/><td/><td/><td/></tr><tr valign="top"><td>RIH</td><td/><td style="author-callout-style-b8"/><td/><td/><td/><td style="author-callout-style-b8"/><td/><td/></tr><tr valign="top"><td>RIP</td><td/><td/><td/><td/><td/><td style="author-callout-style-b8"/><td/><td/></tr><tr valign="top"><td style="author-callout-style-b1">RIS</td><td/><td style="author-callout-style-b8"/><td/><td style="author-callout-style-b8"/><td/><td style="author-callout-style-b8"/><td style="author-callout-style-b8"/><td/></tr><tr valign="top"><td>SAA</td><td/><td/><td/><td/><td/><td/><td style="author-callout-style-b8"/><td/></tr><tr valign="top"><td rowspan="17">Motor neuron (16 classes)</td><td style="author-callout-style-b1">AVL</td><td/><td/><td style="author-callout-style-b3"/><td/><td style="author-callout-style-b3"/><td/><td/><td/></tr><tr valign="top"><td>DA</td><td/><td/><td/><td/><td style="author-callout-style-b3"><italic>lgc-37, gab-1, lgc-35</italic> </td><td/><td/><td/></tr><tr valign="top"><td style="author-callout-style-b1">DD</td><td/><td/><td/><td style="author-callout-style-b3"/><td style="author-callout-style-b3"/><td/><td/><td/></tr><tr valign="top"><td style="author-callout-style-b1">DVB</td><td/><td/><td/><td style="author-callout-style-b8"/><td/><td/><td/><td/></tr><tr valign="top"><td>HSN</td><td/><td/><td/><td style="author-callout-style-b8"><italic>lgc-37, gab-1</italic></td><td/><td/><td/><td/></tr><tr valign="top"><td>PDA</td><td/><td/><td/><td/><td style="author-callout-style-b8"><italic>exp-1, lgc-35</italic></td><td/><td/><td/></tr><tr valign="top"><td>RIM</td><td/><td style="author-callout-style-b8"/><td style="author-callout-style-b7"/><td style="author-callout-style-b8"/><td/><td/><td style="author-callout-style-b8"/><td/></tr><tr valign="top"><td>RIV</td><td/><td/><td/><td/><td/><td/><td style="author-callout-style-b3"><italic>lgc-38</italic></td><td/></tr><tr valign="top"><td style="author-callout-style-b1">RME</td><td/><td style="author-callout-style-b8"/><td style="author-callout-style-b8"/><td/><td/><td style="author-callout-style-b3"/><td style="author-callout-style-b7"/><td/></tr><tr valign="top"><td>RMD*&#160;</td><td style="author-callout-style-b8"><italic>gab-1, lgc-37</italic></td><td style="author-callout-style-b8"><italic>gab-1, lgc-37</italic></td><td style="author-callout-style-b7"><italic>gab-1, lgc-37</italic></td><td/><td/><td style="author-callout-style-b8"><italic>gab-1, lgc-37</italic></td><td style="author-callout-style-b3"><italic>gab-1, lgc-37</italic></td><td/></tr><tr valign="top"><td>RMH</td><td/><td/><td/><td/><td/><td style="author-callout-style-b8"/><td/><td/></tr><tr valign="top"><td>SAB</td><td/><td/><td/><td style="author-callout-style-b7"><italic>exp-1, lgc-37, gab-1, lgc-36</italic></td><td/><td/><td/><td/></tr><tr valign="top"><td>SIA</td><td/><td/><td/><td/><td/><td/><td style="author-callout-style-b8"><italic>lgc-37, gab-1</italic></td><td/></tr><tr valign="top"><td>SIB</td><td/><td/><td/><td/><td/><td/><td style="author-callout-style-b8"/><td/></tr><tr valign="top"><td style="author-callout-style-b1">SMD</td><td/><td style="author-callout-style-b3"><italic>lgc-37, lgc-38, gab-1</italic></td><td style="author-callout-style-b3"><italic>lgc-37, lgc-38, gab-1</italic></td><td/><td/><td style="author-callout-style-b8"><italic>lgc-37, lgc-38, gab-1</italic></td><td style="author-callout-style-b7"><italic>lgc-37, lgc-38, gab-1</italic></td><td/></tr><tr valign="top"><td>VC</td><td/><td/><td/><td style="author-callout-style-b8"><italic>lgc-37, gab-1</italic></td><td/><td/><td/><td/></tr><tr valign="top"><td style="author-callout-style-b1">VD</td><td/><td/><td/><td style="author-callout-style-b8"/><td style="author-callout-style-b8"/><td/><td/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-17686-v1.xml">elife-17686-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.17686.014</object-id><label>Table 5.</label><caption><p>Genetic characterization of <italic>tab-1</italic> function in the RME neurons. Expression in RME can not always be unambiguously assigned to the dorsal, ventral, left or right RME neuron, hence only the total number of RME neurons was counted. <italic>otEx6804-6806</italic> are <italic>tab-1-</italic>rescuing arrays (see Material and methods).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.17686.014">http://dx.doi.org/10.7554/eLife.17686.014</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr valign="top"><th rowspan="2">marker</th><th rowspan="2">Genotype</th><th colspan="6">Expression observed in:</th></tr><tr valign="top"><th>4 RMEs</th><th>3 RMEs</th><th>2 RMEs</th><th>1 RME</th><th>0 RME</th><th>n</th></tr></thead><tbody><tr valign="top"><td rowspan="2"><italic>unc-47</italic><sup><italic>prom</italic></sup><italic>::gfp&#160;</italic>(<italic>otIs509</italic>)</td><td>wild type</td><td>100%</td><td>0%</td><td>0%</td><td>0%</td><td>0%</td><td>31</td></tr><tr valign="top"><td><italic>ot796</italic></td><td>0%</td><td>26.7%</td><td>70%</td><td>3.3%</td><td>0%</td><td>30</td></tr><tr valign="top"><td rowspan="2"><italic>unc-47</italic><sup><italic>prom</italic></sup><italic>::gfp&#160;</italic>(<italic>oxIs12</italic>)</td><td>wild type</td><td>100%</td><td>0%</td><td>0%</td><td>0%</td><td>0%</td><td colspan="2">30</td></tr><tr valign="top"><td><italic>gk753</italic></td><td>16.7%</td><td>30%</td><td>53.3%</td><td>3.3%</td><td>0%</td><td colspan="2">30</td></tr><tr valign="top"><td rowspan="8"><italic>unc-25</italic><sup><italic>prom</italic></sup><italic>::gfp&#160;</italic>(<italic>otIs549</italic>)</td><td>wild type</td><td>100%</td><td>0%</td><td>0%</td><td>0%</td><td>0%</td><td colspan="2">30</td></tr><tr valign="top"><td><italic>ot796/+</italic> </td><td>100%</td><td>0%</td><td>0%</td><td>0%</td><td>0%</td><td colspan="2">22</td></tr><tr valign="top"><td><italic>ot796</italic></td><td>0%</td><td>23.3%</td><td>76.7%</td><td>0%</td><td>0%</td><td colspan="2">30</td></tr><tr valign="top"><td><italic>gk753</italic></td><td>0%</td><td>20%</td><td>80%</td><td>0%</td><td>0%</td><td colspan="2">30</td></tr><tr valign="top"><td><italic>ot796 otIs549/+</italic> </td><td>6.7%</td><td>36.7%</td><td>56.6%</td><td>0%</td><td>0%</td><td colspan="2">30</td></tr><tr valign="top"><td><italic>ot796/ok2198</italic></td><td>0%</td><td>42.8%</td><td>57.2%</td><td>0%</td><td>0%</td><td colspan="2">28</td></tr><tr valign="top"><td><italic>ot796/u271</italic></td><td>3.3%</td><td>36.7%</td><td>60%</td><td>0%</td><td>0%</td><td colspan="2">30</td></tr><tr valign="top"><td><italic>ot796/gk753</italic></td><td>3.3%</td><td>36.7%</td><td>60%</td><td>0%</td><td>0%</td><td colspan="2">30</td></tr><tr valign="top"><td rowspan="2"><p><italic>unc-46</italic><sup><italic>prom</italic></sup><italic>::gfp</italic>(<italic>otIs575</italic>)</p></td><td>wild type</td><td>93.3%</td><td>6.7%</td><td>0%</td><td>0%</td><td>0%</td><td colspan="2">30</td></tr><tr valign="top"><td><italic>gk753</italic></td><td>13.3%</td><td>36.7%</td><td>46.7%</td><td>3.3%</td><td>0%</td><td colspan="2">30</td></tr><tr valign="top"><td rowspan="2">anti-GABA staining</td><td>wild type</td><td>70%</td><td>16.7%</td><td>10%</td><td>0%</td><td>3.3%</td><td colspan="2">30</td></tr><tr valign="top"><td><italic>ot796</italic></td><td>10%</td><td>20%</td><td>63.4%</td><td>3.3%</td><td>3.3%</td><td colspan="2">30</td></tr><tr valign="top"><td rowspan="6"><p><italic>tab-1(gk753); otIs549; Ex[tab-1(+),ttx-3::gfp]</italic></p> </td><td>without <italic>otEx6804</italic></td><td>13.3%</td><td>16.7%</td><td>70%</td><td>0%</td><td>0%</td><td colspan="2">30</td></tr><tr valign="top"><td>with <italic>otEx6804</italic></td><td>80%</td><td>8%</td><td>12%</td><td>0%</td><td>0%</td><td colspan="2">25</td></tr><tr valign="top"><td>without <italic>otEx6805</italic></td><td>5%</td><td>25%</td><td>65%</td><td>0%</td><td>5%</td><td colspan="2">20</td></tr><tr valign="top"><td>with <italic>otEx6805</italic></td><td>41.9%</td><td>9.7%</td><td>48.4%</td><td>0%</td><td>0%</td><td colspan="2">31</td></tr><tr valign="top"><td>without <italic>otEx6806</italic></td><td>5.6%</td><td>16.7%</td><td>77.8%</td><td>0%</td><td>0%</td><td colspan="2">18</td></tr><tr valign="top"><td>with <italic>otEx6806</italic></td><td>66.7%</td><td>10%</td><td>23.3%</td><td>0%</td><td>0%</td><td colspan="2">30</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18096-v2.xml">elife-18096-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl4" position="float"><object-id pub-id-type="doi">10.7554/eLife.18096.025</object-id><label>Table 4.</label><caption><p>TEC assembly ratios and reaction buffers.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18096.025">http://dx.doi.org/10.7554/eLife.18096.025</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="4">Concentrations during assembly, &#181;M</th><th rowspan="2" valign="top">&#160;Assembly buffer</th><th colspan="2" valign="top">Reaction buffer*</th><th valign="top"/></tr><tr><th valign="top">RNA</th><th valign="top">tDNA</th><th valign="top">ntDNA</th><th valign="top">RNAP</th><th valign="top">TEC</th><th valign="top">additive</th><th valign="top"/></tr></thead><tbody><tr><td>Nucleotide addition (gel)</td><td>1</td><td>1.4</td><td>2</td><td>1.5</td><td>TB10</td><td>TB10</td><td>TB10</td><td><xref ref-type="fig" rid="fig2">Figure 2B</xref></td></tr><tr><td>RNA cleavage (gel)</td><td>1</td><td>1.4</td><td>2</td><td>1.5</td><td>TB0</td><td>TB0</td><td>TB2</td><td><xref ref-type="fig" rid="fig3">Figure 3B</xref></td></tr><tr><td>Forward translocation (nucleotide addition)</td><td>1.4</td><td>1</td><td>2</td><td>1.5</td><td>TB10</td><td>TB10</td><td>TB10</td><td><xref ref-type="fig" rid="fig2">Figure 2B</xref></td></tr><tr><td>Backward translocation (pyrophosphorolysis)</td><td>1.4</td><td>1</td><td>2</td><td>1.5</td><td>TB10</td><td>TB10</td><td>TB10</td><td><xref ref-type="fig" rid="fig2">Figure 2C</xref></td></tr><tr><td>Backtracking (RNA cleavage)</td><td>1.4</td><td>1</td><td>2</td><td>1.5</td><td>TB0</td><td>TB0</td><td>TB2</td><td><xref ref-type="fig" rid="fig3">Figure 3B</xref></td></tr><tr><td>NusG binding</td><td>1.4</td><td>1</td><td>2</td><td>1</td><td>TB1</td><td>TB1</td><td>TB1</td><td><xref ref-type="fig" rid="fig6">Figure 6C</xref></td></tr><tr><td>Equilibrium 6-MI assays</td><td>1.4</td><td>1</td><td>2</td><td>1.5</td><td>TB10</td><td colspan="2">TB10</td><td><xref ref-type="fig" rid="fig2">Figures 2D</xref>,<xref ref-type="fig" rid="fig6">6</xref>&#8211;<xref ref-type="fig" rid="fig7">7</xref></td></tr><tr><td>Dinucleotide release (RNA cleavage)</td><td>1</td><td>1.4</td><td>2</td><td>1.5</td><td>TB0</td><td>TB0</td><td>TB2</td><td><xref ref-type="fig" rid="fig3">Figure 3</xref></td></tr><tr><td>8-MP crosslinking</td><td>1.2</td><td>1</td><td>1</td><td>1.5</td><td>TB10</td><td colspan="2">TB10</td><td><xref ref-type="fig" rid="fig4">Figures 4</xref>,<xref ref-type="fig" rid="fig7">7A</xref></td></tr><tr><td>6-TG crosslinking</td><td>1</td><td>1</td><td>2</td><td>1.5</td><td>TB10</td><td colspan="2">TB10</td><td><xref ref-type="fig" rid="fig5">Figure 5</xref></td></tr></tbody></table><table-wrap-foot><fn id="tblfn6"><p>* In time resolved assays the equal volumes of the TEC and the additive solutions were mixed to initiate the reaction.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18096-v2.xml">elife-18096-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.18096.026</object-id><label>Table 5.</label><caption><p>The number of repeats for each experiment.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18096.026">http://dx.doi.org/10.7554/eLife.18096.026</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Figure</th><th rowspan="2">Data</th><th colspan="7" valign="top">Number of experiments</th></tr><tr><th colspan="2" valign="top">with independently assembled TECs</th><th colspan="3" valign="top">including the experiments with the same TEC preparation</th><th colspan="2" valign="top">with independently assembled TECs <break/>not in the figures</th></tr></thead><tbody><tr><td valign="top"/><td valign="top"/><td valign="top">control</td><td valign="top">+NusG</td><td valign="top">control</td><td colspan="2" valign="top">+NusG</td><td valign="top">control</td><td valign="top">+NusG</td></tr><tr><td valign="top">2B</td><td valign="top">WT catalysis <break/>WT translocation</td><td valign="top">4 <break/>3</td><td valign="top">3 <break/>2</td><td valign="top">8 <break/>&gt;12</td><td colspan="2" valign="top">6 <break/>&gt;8</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">2C</td><td valign="top">WT pyrophosphorolysis</td><td valign="top">2</td><td valign="top">2</td><td valign="top">&gt;8</td><td colspan="2" valign="top">&gt;8</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">2D</td><td valign="top">WT TGT binding</td><td valign="top">2</td><td valign="top">2</td><td valign="top"/><td colspan="2" valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">3BC</td><td valign="top">WT RNA cleavage <break/>WT 6-MI <break/>WT 2-AP <break/>&#916;RL 2-AP <break/>&#916;LL 2-AP <break/>&#916;GL 2-AP <break/>WT mm1 2-AP <break/>WT mm1-2 2-AP <break/>WT 3&#8217;mm 2-AP</td><td valign="top">2 <break/>2 <break/>2 <break/>2 <break/>2 <break/>2 <break/>2 <break/>2 <break/>2</td><td valign="top">2 <break/>2 <break/>2 <break/>2 <break/>2 <break/>2 <break/>2 <break/>2 <break/>2</td><td valign="top">3 <break/>&gt;8 <break/>&gt;8 <break/>&gt;8 <break/>&gt;8 <break/>&gt;8 <break/>&gt;8 <break/>&gt;8 <break/>&gt;8</td><td colspan="2" valign="top">4 <break/>&gt;8 <break/>&gt;8 <break/>&gt;8 <break/>&gt;8 <break/>&gt;8 <break/>&gt;8 <break/>&gt;8 <break/>&gt;8</td><td valign="top">6</td><td valign="top">2</td></tr><tr><td>3S3</td><td valign="top">WT 6-MI <break/>WT 2-AP <break/>WT mm1-2 AA 2-AP <break/>WT mm1-2 GG 2-AP</td><td valign="top">2 <break/>1 <break/>1 <break/>2</td><td valign="top">2 <break/>1 <break/>1 <break/>2</td><td valign="top">5 <break/>3 <break/>7 <break/>&gt;8</td><td colspan="2" valign="top">5 <break/>3 <break/>8 <break/>&gt;8</td><td valign="top">1</td><td valign="top">1</td></tr><tr><td valign="top"/><td colspan="8" valign="top">The experiments reported in the figures were performed with the same batch of GreA. The older and the newer experiments cannot be directly combined with the reported experiments due to the variations in the specific activity of the GreA preparations. However, the relative effect of NusG on the reactions involving backtracking can be estimated from all available data. In the WT TEC NusG inhibits reactions that involve backtracking: <break/>2.61 &#177; 0.28 fold (n&#160;=&#160;5, 2 &#181;M GreA); <break/>2.62 &#177; 0.22 fold (n&#160;=&#160;6, 8 &#181;M GreA)</td></tr><tr><td valign="top">4AB</td><td valign="top">WT 8-MP <break/>&#916;RL 8-MP <break/>&#916;LL 8-MP <break/>WT TEC18+TGT 8-MP</td><td valign="top">7 <break/>3 <break/>3 <break/>2</td><td valign="top">3 <break/>2 <break/>2 <break/></td><td valign="top"/><td colspan="2" valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">5AB <break/>5S2</td><td valign="top">WT 6-TG <break/>&#916;RL 6-TG <break/>&#916;LL 6-TG <break/>WT TEC18+TGT 6-TG</td><td valign="top">7 <break/>2 <break/>3 <break/>2</td><td valign="top">3 <break/>2 <break/>2 <break/></td><td valign="top"/><td colspan="2" valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td rowspan="2" valign="top">6AB,C(<italic>Left</italic>)</td><td valign="top">WT 6-MI <break/>&#916;RL 6-MI <break/>&#916;LL 6-MI</td><td valign="top">2 <break/>2 <break/>2</td><td valign="top">2 <break/>2 <break/>2</td><td valign="top">&gt;5 <break/>5* <break/>5*</td><td colspan="2" valign="top">&gt;3 <break/>3* <break/>&gt;3</td><td valign="top">&gt;7</td><td valign="top">&gt;7</td></tr><tr><td colspan="8" valign="top">* Except TEC19. <break/>SD of the fluorescence measurements with the same TEC preparation = 2.3 &#177; 1.3% (n = 122). SD of the fluorescence measurements with the independently assembled TECs = 16% (n = 8, WT TEC17 measured with different batches of the fluorescent oligonucleotides). Accordingly, the primary data from all fluorescent experiments cannot be directly combined with a figure but some NusG effects can be estimated with the highest accuracy and precision from all available data. NusG effects on the fluorescence intensity of the WT TECs: <break/>(TEC17NusG&#8211;TEC16NusG)/(TEC17-TEC16) = 1.45 &#177; 0.09 (n&#160;=&#160;9) <break/>(TEC18NusG&#8211;TEC16NusG)/(TEC18-TEC16) = 0.99 &#177; 0.02 (n&#160;=&#160;7) <break/>(TEC19NusG&#8211;TEC16NusG)/(TEC19-TEC16) = 0.50 &#177; 0.35 (n=3)</td></tr><tr><td valign="top">6C (<italic>Right</italic>)</td><td valign="top">WT+NusG 6-MI</td><td colspan="2" valign="top">2</td><td colspan="2" valign="top">&gt;6</td><td colspan="3" valign="top"/></tr><tr><td rowspan="3" valign="top">7A</td><td valign="top">TEC16-19 6-MI</td><td colspan="2" valign="top">2</td><td colspan="2" valign="top">9 (except TEC19)</td><td colspan="3" rowspan="3" valign="top"/></tr><tr><td valign="top">TEC16-19 6-MI</td><td colspan="2" valign="top">2</td><td colspan="2" valign="top">4 (except TEC19)</td></tr><tr><td valign="top">TEC16-18 8-MP</td><td colspan="2" valign="top">2</td><td colspan="2" valign="top">2</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18432-v2.xml">elife-18432-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.18432.025</object-id><label>Table 2.</label><caption><p>Interactions<sup>*</sup> of sterols at the three high-affinity cholesterol-binding sites.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18432.025">http://dx.doi.org/10.7554/eLife.18432.025</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"><p>Cholesterol/Cholesterol analogue</p></th><th colspan="6" valign="top"><p><bold>High-affinity cholesterol interaction sites</bold></p> </th></tr><tr><th colspan="2" valign="top"><p><bold>IC1</bold></p> </th><th colspan="2" valign="top"><p><bold>IC2</bold></p> </th><th colspan="2" valign="top"><p><bold>EC1</bold></p> </th></tr><tr><th valign="top"><p>vdW interaction energy (kJ/mol)</p></th><th valign="top"><p>No. of contacts</p></th><th valign="top"><p>vdW interaction energy (kJ/mol)</p></th><th valign="top"><p>No. of contacts</p></th><th valign="top"><p>vdW interaction energy (kJ/mol)</p></th><th valign="top"><p>No. of contacts</p></th></tr></thead><tbody><tr><td valign="top"><p>Cholesterol<sup>&#8224;</sup></p></td><td valign="top"><p>&#8722;138.04&#160;&#177;&#160;0.20</p> </td><td valign="top"><p>141.02&#160;&#177;&#160;0.22</p> </td><td valign="top"><p>&#8722;95.06&#160;&#177;&#160;0.12</p> </td><td valign="top"><p>90.65&#160;&#177;&#160;0.16</p> </td><td valign="top"><p>&#8722;129.51 &#177; 0.29</p></td><td valign="top"><p>104.38&#160;&#177;&#160;0.28</p> </td></tr><tr><td valign="top"><p>CHS</p></td><td valign="top"><p>&#8722;29.63&#160;&#177;&#160;0.14</p> </td><td valign="top"><p>28.78&#160;&#177;&#160;0.16</p> </td><td valign="top"><p>&#8722;98.75&#160;&#177;&#160;0.11</p> </td><td valign="top"><p>96.30&#160;&#177;&#160;0.16</p> </td><td valign="top"><p>-</p></td><td valign="top"><p>-</p></td></tr><tr><td valign="top"><p>27-OH-Chol</p></td><td valign="top"><p>&#8722;32.17&#160;&#177;&#160;0.30</p> </td><td valign="top"><p>34.95&#160;&#177;&#160;0.33</p> </td><td valign="top"><p>&#8722;22.69&#160;&#177;&#160;0.23</p> </td><td valign="top"><p>28.41&#160;&#177;&#160;0.28</p> </td><td valign="top"><p>&#8722;132.85&#160;&#177;&#160;0.27</p> </td><td valign="top"><p>120.20&#160;&#177;&#160;0.30</p> </td></tr><tr><td valign="top"><p>4&#946;-OH-Chol</p></td><td valign="top"><p>-</p></td><td valign="top"><p>-</p></td><td valign="top"><p>-</p></td><td valign="top"><p>-</p></td><td valign="top"><p>&#8722;41.80&#160;&#177;&#160;0.48</p> </td><td valign="top"><p>33.41&#160;&#177;&#160;0.42</p> </td></tr></tbody></table><table-wrap-foot><fn id="tblfn5"><p><sup>*</sup> Shown are the total van der Waals (vdW) interaction energy and the number of contacts between cholesterol and &#946;<sub>2</sub>AR, when cholesterol is in the IC1, IC2, or EC1 binding site (and similarly for the cholesterol analogues).</p></fn><fn id="tblfn6"><p><sup>&#8224;</sup> Calculations are based on systems having&#160;&#8805;10 mol% cholesterol. Shown here are the average values over different trajectories.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.076</object-id><label>Appendix 1&#8212;table 5.</label><caption><p>Structural connection parameters for Pyramidal cells, based on <xref ref-type="bibr" rid="bib13">Bezaire and Soltesz (2013)</xref>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.076">http://dx.doi.org/10.7554/eLife.18566.076</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Other type</th><th colspan="4">Other cell to pyr</th><th colspan="4">Pyr to other cell</th></tr><tr><th>#</th><th>Syn.s</th><th>#</th><th>Post</th><th>#</th><th>Syn.s</th><th>#</th><th>Post</th></tr><tr><th>Conn.s</th><th>/Conn.</th><th>#</th><th>Loc.</th><th>Conn.s</th><th>/Conn.</th><th>#</th><th>Loc.</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right">6</td><td align="right">6</td><td align="right">36</td><td align="left">axon</td><td align="right">1</td><td align="right">3</td><td align="right">2</td><td align="left">apical dendrite</td></tr><tr><td align="left">Bis</td><td align="right">10</td><td align="right">10</td><td align="right">100</td><td align="left">any dendrite</td><td align="right">3</td><td align="right">3</td><td align="right">7</td><td align="left">apical dendrite</td></tr><tr><td align="left">CCK+B</td><td align="right">13</td><td align="right">8</td><td align="right">104</td><td align="left">any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td align="left"/></tr><tr><td align="left">Ivy</td><td align="right">42</td><td align="right">10</td><td align="right">420</td><td align="left">any dendrite</td><td align="right">0</td><td align="right">3</td><td align="right">0</td><td align="left">apical dendrite</td></tr><tr><td align="left">NGF</td><td align="right">14</td><td align="right">10</td><td align="right">140</td><td align="left">apical dendrite</td><td align="right"/><td align="right"/><td align="right"/><td align="left"/></tr><tr><td align="left">O-LM</td><td align="right">8</td><td align="right">10</td><td align="right">80</td><td align="left">apical dendrite</td><td align="right">13</td><td align="right">3</td><td align="right">37</td><td align="left">basal dendrite</td></tr><tr><td align="left">Pyr</td><td align="right">197</td><td align="right">1</td><td align="right">197</td><td align="left">apical dendrite</td><td align="right">197</td><td align="right">1</td><td align="right">197</td><td align="left">apical dendrite</td></tr><tr><td align="left">PV+B</td><td align="right">17</td><td align="right">11</td><td align="right">187</td><td align="left">soma</td><td align="right">8</td><td align="right">3</td><td align="right">22</td><td align="left">apical dendrite</td></tr><tr><td align="left">SC-A</td><td align="right"/><td align="right"/><td align="right"/><td align="left"/><td align="right">0</td><td align="right">3</td><td align="right">0</td><td align="left">apical dendrite</td></tr><tr><td align="left">CA3</td><td align="right">5985</td><td align="right">2</td><td align="right">11970</td><td align="left">any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td align="left"/></tr><tr><td align="left">ECIII</td><td align="right">1299</td><td align="right">2</td><td align="right">2598</td><td align="left">any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td align="left"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl8" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.079</object-id><label>Appendix 1&#8212;table 8.</label><caption><p>Model synaptic parameters for Pyramidal cells in the control network.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.079">http://dx.doi.org/10.7554/eLife.18566.079</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="4">Other cell to pyr</th><th align="center" colspan="4">Pyr to other cell</th></tr><tr><th align="center"><inline-formula><mml:math id="inf243"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf244"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf245"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">r</mml:mi><mml:mi mathvariant="bold-sans-serif">i</mml:mi><mml:mi mathvariant="bold-sans-serif">s</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf246"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf247"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf248"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf249"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">r</mml:mi><mml:mi mathvariant="bold-sans-serif">i</mml:mi><mml:mi mathvariant="bold-sans-serif">s</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf250"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right">&#8722;60.0</td><td align="right">1.150e-03</td><td align="right">0.28</td><td align="right">8.40</td><td align="right">0.0</td><td align="right">4.000e-05</td><td align="right">0.30</td><td align="right">0.60</td></tr><tr><td align="left">Bis</td><td align="right">&#8722;60.0</td><td align="right">5.100e-04</td><td align="right">0.11</td><td align="right">9.70</td><td align="right">0.0</td><td align="right">1.900e-03</td><td align="right">0.11</td><td align="right">0.25</td></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;60.0</td><td align="right">5.200e-04</td><td align="right">0.20</td><td align="right">4.20</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">Ivy</td><td align="right">&#8722;60.0</td><td align="right">4.100e-05</td><td align="right">1.10</td><td align="right">11.00</td><td align="right">0.0</td><td align="right">4.050e-04</td><td align="right">0.30</td><td align="right">0.60</td></tr><tr><td align="left">NGF</td><td align="right">&#8722;60.0</td><td align="right">6.500e-05</td><td align="right">9.00</td><td align="right">39.00</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">O-LM</td><td align="right">&#8722;60.0</td><td align="right">3.000e-04</td><td align="right">0.13</td><td align="right">11.00</td><td align="right">0.0</td><td align="right">2.000e-04</td><td align="right">0.30</td><td align="right">0.60</td></tr><tr><td align="left">Pyr</td><td align="right">0.0</td><td align="right">7.000e-02</td><td align="right">0.10</td><td align="right">1.50</td><td align="right">0.0</td><td align="right">7.000e-02</td><td align="right">0.10</td><td align="right">1.50</td></tr><tr><td align="left">PV+B</td><td align="right">&#8722;60.0</td><td align="right">2.000e-04</td><td align="right">0.30</td><td align="right">6.20</td><td align="right">0.0</td><td align="right">7.000e-04</td><td align="right">0.07</td><td align="right">0.20</td></tr><tr><td align="left">SC-A</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">0.0</td><td align="right">4.050e-04</td><td align="right">0.30</td><td align="right">0.60</td></tr><tr><td align="left">CA3</td><td align="right">0.0</td><td align="right">2.000e-04</td><td align="right">0.50</td><td align="right">3.00</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">ECIII</td><td align="right">0.0</td><td align="right">2.000e-04</td><td align="right">0.50</td><td align="right">3.00</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl9" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.080</object-id><label>Appendix 1&#8212;table 9.</label><caption><p>Model synaptic properties under voltage clamp at &#8722;50 mV with physiological reversal potentials</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.080">http://dx.doi.org/10.7554/eLife.18566.080</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="5">Other cell to pyr</th><th align="center" colspan="5">Pyr to other cell</th></tr><tr><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf251"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf252"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf253"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf254"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf255"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf256"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">36.45</td><td align="right">0.85</td><td align="right">11.57</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">1.85</td><td align="right">0.78</td><td align="right">2.53</td></tr><tr><td align="left">Bis</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">13.47</td><td align="right">2.17</td><td align="right">15.20</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">64.48</td><td align="right">0.28</td><td align="right">1.42</td></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">24.86</td><td align="right">0.52</td><td align="right">6.03</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">Ivy</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">1.63</td><td align="right">3.63</td><td align="right">15.35</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">40.70</td><td align="right">0.58</td><td align="right">1.28</td></tr><tr><td align="left">NGF</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">1.10</td><td align="right">65.58</td><td align="right">0.00</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">O-LM</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">0.54</td><td align="right">3.70</td><td align="right">14.10</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">17.47</td><td align="right">0.60</td><td align="right">1.53</td></tr><tr><td align="left">Pyr</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">22.13</td><td align="right">2.22</td><td align="right">9.65</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">22.13</td><td align="right">2.22</td><td align="right">9.65</td></tr><tr><td align="left">PV+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">20.56</td><td align="right">0.50</td><td align="right">6.70</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">14.75</td><td align="right">0.25</td><td align="right">1.77</td></tr><tr><td align="left">SC-A</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">17.42</td><td align="right">0.68</td><td align="right">3.05</td></tr><tr><td align="left">CA3</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">7.15</td><td align="right">1.83</td><td align="right">7.08</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">ECIII</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">1.41</td><td align="right">3.25</td><td align="right">13.63</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl12" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.085</object-id><label>Appendix 1&#8212;table 12.</label><caption><p>Structural connection parameters for Axo-axonic cells, based on <xref ref-type="bibr" rid="bib13">Bezaire and Soltesz (2013)</xref>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.085">http://dx.doi.org/10.7554/eLife.18566.085</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="3">Other type</th><th align="center" colspan="4">Other cell to axo</th><th align="center" colspan="4">Axo to other cell</th></tr><tr><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th></tr><tr><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th></tr></thead><tbody><tr><td align="left">Bis</td><td align="right">16</td><td align="right">10</td><td align="right">160</td><td>any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">CCK+B</td><td align="right">12</td><td align="right">8</td><td align="right">96</td><td>any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">Ivy</td><td align="right">24</td><td align="right">10</td><td align="right">240</td><td>any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">O-LM</td><td align="right">8</td><td align="right">10</td><td align="right">80</td><td>apical dendrite</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">Pyr</td><td align="right">162</td><td align="right">3</td><td align="right">486</td><td>apical dendrite</td><td align="right">1271</td><td align="right">6</td><td align="right">7628</td><td align="right">axon</td></tr><tr><td align="left">PV+B</td><td align="right">39</td><td align="right">1</td><td align="right">39</td><td>soma</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">SC-A</td><td align="right">1</td><td align="right">6</td><td align="right">6</td><td>any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">CA3</td><td align="right">4170</td><td align="right">2</td><td align="right">8340</td><td>any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">ECIII</td><td align="right">485</td><td align="right">2</td><td align="right">970</td><td>any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl14" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.087</object-id><label>Appendix 1&#8212;table 14.</label><caption><p>Model synaptic parameters for Axo-axonic cells in the control network.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.087">http://dx.doi.org/10.7554/eLife.18566.087</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="4">Other cell to axo</th><th align="center" colspan="4">Axo to other cell</th></tr><tr><th align="center"><inline-formula><mml:math id="inf260"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf261"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf262"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">r</mml:mi><mml:mi mathvariant="bold-sans-serif">i</mml:mi><mml:mi mathvariant="bold-sans-serif">s</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf263"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf264"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf265"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf266"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">r</mml:mi><mml:mi mathvariant="bold-sans-serif">i</mml:mi><mml:mi mathvariant="bold-sans-serif">s</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf267"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Bis</td><td align="right">&#8722;60.0</td><td align="right">6.000e-04</td><td align="right">0.29</td><td align="right">2.67</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;60.0</td><td align="right">7.000e-04</td><td align="right">0.43</td><td align="right">4.49</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">Ivy</td><td align="right">&#8722;60.0</td><td align="right">5.700e-05</td><td align="right">2.90</td><td align="right">3.10</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">O-LM</td><td align="right">&#8722;60.0</td><td align="right">1.200e-04</td><td align="right">0.73</td><td align="right">10.00</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">Pyr</td><td align="right">0.0</td><td align="right">4.000e-05</td><td align="right">0.30</td><td align="right">0.60</td><td align="right">&#8722;60.0</td><td align="right">1.150e-03</td><td align="right">0.28</td><td align="right">8.40</td></tr><tr><td align="left">PV+B</td><td align="right">&#8722;60.0</td><td align="right">1.200e-04</td><td align="right">0.29</td><td align="right">2.67</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">SC-A</td><td align="right">&#8722;60.0</td><td align="right">6.000e-04</td><td align="right">0.42</td><td align="right">4.99</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">CA3</td><td align="right">0.0</td><td align="right">1.200e-04</td><td align="right">2.00</td><td align="right">6.30</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">ECIII</td><td align="right">0.0</td><td align="right">1.200e-04</td><td align="right">2.00</td><td align="right">6.30</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl15" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.088</object-id><label>Appendix 1&#8212;table 15.</label><caption><p>Model synaptic properties under voltage clamp at &#8722;50 mV with physiological reversal potentials</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.088">http://dx.doi.org/10.7554/eLife.18566.088</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="5">Other cell to axo</th><th align="center" colspan="5">Axo to other cell</th></tr><tr><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf268"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf269"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf270"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf271"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf272"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf273"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Bis</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">36.77</td><td align="right">0.70</td><td align="right">3.70</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">47.29</td><td align="right">0.75</td><td align="right">5.27</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">Ivy</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">4.34</td><td align="right">2.13</td><td align="right">6.57</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">O-LM</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">4.76</td><td align="right">2.55</td><td align="right">12.03</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">Pyr</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">1.85</td><td align="right">0.78</td><td align="right">2.53</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">36.45</td><td align="right">0.85</td><td align="right">11.57</td></tr><tr><td align="left">PV+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">1.08</td><td align="right">0.45</td><td align="right">3.13</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">SC-A</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">24.00</td><td align="right">1.00</td><td align="right">6.13</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">CA3</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">10.85</td><td align="right">2.30</td><td align="right">8.80</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">ECIII</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">8.74</td><td align="right">3.08</td><td align="right">9.20</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl18" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.093</object-id><label>Appendix 1&#8212;table 18.</label><caption><p>Structural connection parameters for Bistratified cells, based on <xref ref-type="bibr" rid="bib13">Bezaire and Soltesz (2013)</xref>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.093">http://dx.doi.org/10.7554/eLife.18566.093</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="3">Other type</th><th align="center" colspan="4">Other cell to bis</th><th align="center" colspan="4">Bis to other cell</th></tr><tr><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th></tr><tr><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th></tr></thead><tbody><tr><td>Axo</td><td align="right"/><td align="right"/><td align="right"/><td/><td align="right">11</td><td align="right">10</td><td align="right">106</td><td>any dendrite</td></tr><tr><td>Bis</td><td align="right">16</td><td align="right">10</td><td align="right">160</td><td>any dendrite</td><td align="right">16</td><td align="right">10</td><td align="right">160</td><td>any dendrite</td></tr><tr><td>CCK+B</td><td align="right">12</td><td align="right">8</td><td align="right">96</td><td>any dendrite</td><td align="right">26</td><td align="right">10</td><td align="right">260</td><td>any dendrite</td></tr><tr><td>Ivy</td><td align="right">24</td><td align="right">10</td><td align="right">240</td><td>any dendrite</td><td align="right">12</td><td align="right">10</td><td align="right">119</td><td>any dendrite</td></tr><tr><td>O-LM</td><td align="right">8</td><td align="right">10</td><td align="right">80</td><td>apical dendrite</td><td align="right">29</td><td align="right">10</td><td align="right">289</td><td>any dendrite</td></tr><tr><td>Pyr</td><td align="right">366</td><td align="right">3</td><td align="right">1098</td><td>apical dendrite</td><td align="right">1410</td><td align="right">10</td><td align="right">14095</td><td>any dendrite</td></tr><tr><td>PV+B</td><td align="right">39</td><td align="right">1</td><td align="right">39</td><td>soma</td><td align="right">40</td><td align="right">10</td><td align="right">400</td><td>any dendrite</td></tr><tr><td>SC-A</td><td align="right">1</td><td align="right">6</td><td align="right">6</td><td>any dendrite</td><td align="right">3</td><td align="right">10</td><td align="right">30</td><td>any dendrite</td></tr><tr><td>CA3</td><td align="right">5782</td><td align="right">2</td><td align="right">11564</td><td>any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td/></tr><tr><td>ECIII</td><td align="right">432</td><td align="right">2</td><td align="right">864</td><td>any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl21" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.096</object-id><label>Appendix 1&#8212;table 21.</label><caption><p>Model synaptic parameters for Bistratified cells in the control network.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.096">http://dx.doi.org/10.7554/eLife.18566.096</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="4">Other cell to bis</th><th align="center" colspan="4">Bis to other cell</th></tr><tr><th align="center"><inline-formula><mml:math id="inf280"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf281"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf282"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">r</mml:mi><mml:mi mathvariant="bold-sans-serif">i</mml:mi><mml:mi mathvariant="bold-sans-serif">s</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf283"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf284"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf285"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf286"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">r</mml:mi><mml:mi mathvariant="bold-sans-serif">i</mml:mi><mml:mi mathvariant="bold-sans-serif">s</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf287"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;60.0</td><td align="right">6.000e-04</td><td align="right">0.29</td><td align="right">2.67</td></tr><tr><td align="left">Bis</td><td align="right">&#8722;60.0</td><td align="right">5.100e-04</td><td align="right">0.29</td><td align="right">2.67</td><td align="right">&#8722;60.0</td><td align="right">5.100e-04</td><td align="right">0.29</td><td align="right">2.67</td></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;60.0</td><td align="right">7.000e-04</td><td align="right">0.43</td><td align="right">4.49</td><td align="right">&#8722;60.0</td><td align="right">8.000e-04</td><td align="right">0.29</td><td align="right">2.67</td></tr><tr><td align="left">Ivy</td><td align="right">&#8722;60.0</td><td align="right">7.700e-05</td><td align="right">2.90</td><td align="right">3.10</td><td align="right">&#8722;60.0</td><td align="right">5.000e-04</td><td align="right">0.29</td><td align="right">2.67</td></tr><tr><td align="left">O-LM</td><td align="right">&#8722;60.0</td><td align="right">1.100e-04</td><td align="right">0.60</td><td align="right">15.00</td><td align="right">&#8722;60.0</td><td align="right">2.000e-05</td><td align="right">1.00</td><td align="right">8.00</td></tr><tr><td align="left">Pyr</td><td align="right">0.0</td><td align="right">1.900e-03</td><td align="right">0.11</td><td align="right">0.25</td><td align="right">&#8722;60.0</td><td align="right">5.100e-04</td><td align="right">0.11</td><td align="right">9.70</td></tr><tr><td align="left">PV+B</td><td align="right">&#8722;60.0</td><td align="right">2.900e-03</td><td align="right">0.18</td><td align="right">0.45</td><td align="right">&#8722;60.0</td><td align="right">9.000e-03</td><td align="right">0.29</td><td align="right">2.67</td></tr><tr><td align="left">SC-A</td><td align="right">&#8722;60.0</td><td align="right">6.000e-04</td><td align="right">0.42</td><td align="right">4.99</td><td align="right">&#8722;60.0</td><td align="right">8.000e-04</td><td align="right">0.29</td><td align="right">2.67</td></tr><tr><td align="left">CA3</td><td align="right">0.0</td><td align="right">1.500e-04</td><td align="right">2.00</td><td align="right">6.30</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">ECIII</td><td align="right">0.0</td><td align="right">1.500e-04</td><td align="right">2.00</td><td align="right">6.30</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl22" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.097</object-id><label>Appendix 1&#8212;table 22.</label><caption><p>Model synaptic properties under voltage clamp at &#8722;50 mV with physiological reversal potentials</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.097">http://dx.doi.org/10.7554/eLife.18566.097</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="5">Other cell to bis</th><th align="center" colspan="5">Bis to other cell</th></tr><tr><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf288"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf289"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf290"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf291"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf292"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf293"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">36.77</td><td align="right">0.70</td><td align="right">3.70</td></tr><tr><td align="left">Bis</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">34.34</td><td align="right">0.70</td><td align="right">3.72</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">34.34</td><td align="right">0.70</td><td align="right">3.72</td></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">48.13</td><td align="right">0.78</td><td align="right">5.35</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">48.55</td><td align="right">0.73</td><td align="right">4.15</td></tr><tr><td align="left">Ivy</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">6.39</td><td align="right">2.15</td><td align="right">6.63</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">43.40</td><td align="right">0.60</td><td align="right">3.17</td></tr><tr><td align="left">O-LM</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">6.31</td><td align="right">2.70</td><td align="right">17.05</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">1.86</td><td align="right">1.78</td><td align="right">8.13</td></tr><tr><td align="left">Pyr</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">64.48</td><td align="right">0.28</td><td align="right">1.42</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">13.47</td><td align="right">2.17</td><td align="right">15.20</td></tr><tr><td align="left">PV+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">24.45</td><td align="right">0.17</td><td align="right">0.73</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">429.34</td><td align="right">0.57</td><td align="right">4.13</td></tr><tr><td align="left">SC-A</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">26.43</td><td align="right">1.02</td><td align="right">6.20</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">50.35</td><td align="right">0.70</td><td align="right">4.10</td></tr><tr><td align="left">CA3</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">13.81</td><td align="right">2.38</td><td align="right">8.82</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">ECIII</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">12.04</td><td align="right">3.05</td><td align="right">9.30</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl25" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.102</object-id><label>Appendix 1&#8212;table 25.</label><caption><p>Structural connection parameters for CCK+ Basket cells, based on <xref ref-type="bibr" rid="bib13">Bezaire and Soltesz (2013)</xref>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.102">http://dx.doi.org/10.7554/eLife.18566.102</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="3">Other type</th><th align="center" colspan="4">Other cell to CCK+B</th><th align="center" colspan="4">CCK+B to other cell</th></tr><tr><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th></tr><tr><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right"/><td align="right"/><td align="right"/><td align="left"/><td align="right">5</td><td align="right">8</td><td align="right">39</td><td align="left">any dendrite</td></tr><tr><td align="left">Bis</td><td align="right">16</td><td align="right">10</td><td align="right">160</td><td align="left">any dendrite</td><td align="right">7</td><td align="right">8</td><td align="right">58</td><td align="left">any dendrite</td></tr><tr><td align="left">CCK+B</td><td align="right">35</td><td align="right">8</td><td align="right">280</td><td align="left">any dendrite</td><td align="right">35</td><td align="right">8</td><td align="right">280</td><td align="left">any dendrite</td></tr><tr><td align="left">Ivy</td><td align="right">96</td><td align="right">10</td><td align="right">960</td><td align="left">any dendrite</td><td align="right">20</td><td align="right">8</td><td align="right">156</td><td align="left">any dendrite</td></tr><tr><td align="left">O-LM</td><td align="right">40</td><td align="right">10</td><td align="right">400</td><td align="left">apical dendrite</td><td align="right">9</td><td align="right">8</td><td align="right">72</td><td align="left">any dendrite</td></tr><tr><td align="left">Pyr</td><td align="right"/><td align="right"/><td align="right"/><td align="left"/><td align="right">1125</td><td align="right">8</td><td align="right">8998</td><td align="left">any dendrite</td></tr><tr><td align="left">PV+B</td><td align="right">38</td><td align="right">1</td><td align="right">38</td><td align="left">soma</td><td align="right">18</td><td align="right">8</td><td align="right">147</td><td align="left">any dendrite</td></tr><tr><td align="left">SC-A</td><td align="right">6</td><td align="right">6</td><td align="right">36</td><td align="left">any dendrite</td><td align="right">3</td><td align="right">8</td><td align="right">24</td><td align="left">any dendrite</td></tr><tr><td align="left">CA3</td><td align="right">2000</td><td align="right">2</td><td align="right">4000</td><td align="left">any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td align="left"/></tr><tr><td align="left">ECIII</td><td align="right">559</td><td align="right">2</td><td align="right">1118</td><td align="left">any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td align="left"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl27" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.104</object-id><label>Appendix 1&#8212;table 27.</label><caption><p>Model synaptic parameters for CCK+ Basket cells in the control network.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.104">http://dx.doi.org/10.7554/eLife.18566.104</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="4">Other cell to CCK+B</th><th align="center" colspan="4">CCK+B to other cell</th></tr><tr><th align="center"><inline-formula><mml:math id="inf297"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf298"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf299"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">r</mml:mi><mml:mi mathvariant="bold-sans-serif">i</mml:mi><mml:mi mathvariant="bold-sans-serif">s</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf300"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf301"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf302"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf303"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">r</mml:mi><mml:mi mathvariant="bold-sans-serif">i</mml:mi><mml:mi mathvariant="bold-sans-serif">s</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf304"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;60.0</td><td align="right">7.000e-04</td><td align="right">0.43</td><td align="right">4.49</td></tr><tr><td align="left">Bis</td><td align="right">&#8722;60.0</td><td align="right">8.000e-04</td><td align="right">0.29</td><td align="right">2.67</td><td align="right">&#8722;60.0</td><td align="right">7.000e-04</td><td align="right">0.43</td><td align="right">4.49</td></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;60.0</td><td align="right">4.500e-04</td><td align="right">0.43</td><td align="right">4.49</td><td align="right">&#8722;60.0</td><td align="right">4.500e-04</td><td align="right">0.43</td><td align="right">4.49</td></tr><tr><td align="left">Ivy</td><td align="right">&#8722;60.0</td><td align="right">3.700e-05</td><td align="right">2.90</td><td align="right">3.10</td><td align="right">&#8722;60.0</td><td align="right">3.000e-04</td><td align="right">0.43</td><td align="right">4.49</td></tr><tr><td align="left">O-LM</td><td align="right">&#8722;60.0</td><td align="right">1.200e-03</td><td align="right">0.73</td><td align="right">20.20</td><td align="right">&#8722;60.0</td><td align="right">7.000e-04</td><td align="right">1.00</td><td align="right">8.00</td></tr><tr><td align="left">Pyr</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;60.0</td><td align="right">5.200e-04</td><td align="right">0.20</td><td align="right">4.20</td></tr><tr><td align="left">PV+B</td><td align="right">&#8722;60.0</td><td align="right">1.200e-03</td><td align="right">0.29</td><td align="right">2.67</td><td align="right">&#8722;60.0</td><td align="right">9.000e-03</td><td align="right">0.43</td><td align="right">4.49</td></tr><tr><td align="left">SC-A</td><td align="right">&#8722;60.0</td><td align="right">8.500e-04</td><td align="right">0.42</td><td align="right">4.99</td><td align="right">&#8722;60.0</td><td align="right">7.000e-04</td><td align="right">0.43</td><td align="right">4.49</td></tr><tr><td align="left">CA3</td><td align="right">0.0</td><td align="right">6.500e-04</td><td align="right">2.00</td><td align="right">6.30</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">ECIII</td><td align="right">0.0</td><td align="right">6.500e-04</td><td align="right">2.00</td><td align="right">6.30</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl28" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.105</object-id><label>Appendix 1&#8212;table 28.</label><caption><p>Model synaptic properties under voltage clamp at &#8722;50 mV with physiological reversal potentials</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.105">http://dx.doi.org/10.7554/eLife.18566.105</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="5">Other cell to CCK+B</th><th align="center" colspan="5">CCK+B to other cell</th></tr><tr><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf305"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf306"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf307"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf308"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf309"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf310"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">47.29</td><td align="right">0.75</td><td align="right">5.27</td></tr><tr><td align="left">Bis</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">48.55</td><td align="right">0.73</td><td align="right">4.15</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">48.13</td><td align="right">0.78</td><td align="right">5.35</td></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">32.19</td><td align="right">0.73</td><td align="right">5.30</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">32.19</td><td align="right">0.73</td><td align="right">5.30</td></tr><tr><td align="left">Ivy</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">3.00</td><td align="right">2.25</td><td align="right">6.95</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">22.34</td><td align="right">0.80</td><td align="right">5.05</td></tr><tr><td align="left">O-LM</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">40.32</td><td align="right">3.10</td><td align="right">28.42</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">54.98</td><td align="right">1.35</td><td align="right">9.05</td></tr><tr><td align="left">Pyr</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">24.86</td><td align="right">0.52</td><td align="right">6.03</td></tr><tr><td align="left">PV+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">11.31</td><td align="right">0.42</td><td align="right">3.08</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">523.11</td><td align="right">0.68</td><td align="right">5.70</td></tr><tr><td align="left">SC-A</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">33.81</td><td align="right">1.05</td><td align="right">6.90</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">49.55</td><td align="right">0.70</td><td align="right">5.38</td></tr><tr><td align="left">CA3</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">55.24</td><td align="right">2.53</td><td align="right">9.35</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">ECIII</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">43.27</td><td align="right">3.40</td><td align="right">10.87</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl31" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.110</object-id><label>Appendix 1&#8212;table 31.</label><caption><p>Structural connection parameters for Ivy cells, based on <xref ref-type="bibr" rid="bib13">Bezaire and Soltesz (2013)</xref>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.110">http://dx.doi.org/10.7554/eLife.18566.110</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="3">OtherType</th><th align="center" colspan="4">Other cell to ivy</th><th align="center" colspan="4">Ivy to other cell</th></tr><tr><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th></tr><tr><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th></tr></thead><tbody><tr><td>Axo</td><td align="right"/><td align="right"/><td align="right"/><td/><td align="right">4</td><td align="right">10</td><td align="right">40</td><td>any dendrite</td></tr><tr><td>Bis</td><td align="right">3</td><td align="right">10</td><td align="right">30</td><td>any dendrite</td><td align="right">6</td><td align="right">10</td><td align="right">60</td><td>any dendrite</td></tr><tr><td>CCK+B</td><td align="right">8</td><td align="right">8</td><td align="right">64</td><td>any dendrite</td><td align="right">39</td><td align="right">10</td><td align="right">392</td><td>any dendrite</td></tr><tr><td>Ivy</td><td align="right">24</td><td align="right">10</td><td align="right">240</td><td>any dendrite</td><td align="right">24</td><td align="right">10</td><td align="right">240</td><td>any dendrite</td></tr><tr><td>NGF</td><td align="right"/><td align="right"/><td align="right"/><td/><td align="right">11</td><td align="right">10</td><td align="right">113</td><td>any dendrite</td></tr><tr><td>O-LM</td><td align="right"/><td align="right"/><td align="right"/><td/><td align="right">25</td><td align="right">10</td><td align="right">253</td><td>any dendrite</td></tr><tr><td>Pyr</td><td align="right">9</td><td align="right">3</td><td align="right">27</td><td>apical dendrite</td><td align="right">1485</td><td align="right">10</td><td align="right">14850</td><td>any dendrite</td></tr><tr><td>PV+B</td><td align="right">8</td><td align="right">1</td><td align="right">8</td><td>soma</td><td align="right">15</td><td align="right">10</td><td align="right">150</td><td>any dendrite</td></tr><tr><td>SC-A</td><td align="right">2</td><td align="right">6</td><td align="right">12</td><td>any dendrite</td><td align="right">5</td><td align="right">10</td><td align="right">46</td><td>any dendrite</td></tr><tr><td>CA3</td><td align="right">1923</td><td align="right">2</td><td align="right">3846</td><td>any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.059</object-id><label>Table 5.</label><caption><p>Preferred theta firing phases for each model cell type.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.059">http://dx.doi.org/10.7554/eLife.18566.059</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Cell type</th><th align="center" rowspan="2">Firing rate (Hz)</th><th align="center" colspan="2">Modulation</th><th align="center" rowspan="2">Phase (0<inline-formula><mml:math id="inf19"><mml:msup><mml:mi/><mml:mi>o</mml:mi></mml:msup></mml:math></inline-formula>=trough)</th></tr><tr><th align="center">Level</th><th align="center">p</th></tr></thead><tbody><tr><td align="left">Axo.</td><td align="right">8.9</td><td align="right">0.07</td><td align="right">4.58e &#8722;&#160;130</td><td align="right">163.4</td></tr><tr><td align="left">Bis.</td><td align="right">18.0</td><td align="right">0.76</td><td align="right">0.00e&#160;+&#160;00</td><td align="right">340.0</td></tr><tr><td align="left">CCK+ B.</td><td align="right">54.4</td><td align="right">0.10</td><td align="right">0.00e&#160;+&#160;00</td><td align="right">202.8</td></tr><tr><td align="left">Ivy</td><td align="right">43.3</td><td align="right">0.33</td><td align="right">0.00e&#160;+&#160;00</td><td align="right">142.1</td></tr><tr><td align="left">NGF.</td><td align="right">55.1</td><td align="right">0.07</td><td align="right">1.46e &#8722;&#160;32</td><td align="right">176.3</td></tr><tr><td align="left">O-LM</td><td align="right">17.4</td><td align="right">0.76</td><td align="right">0.00e&#160;+&#160;00</td><td align="right">334.7</td></tr><tr><td align="left">Pyr.</td><td align="right">6.0</td><td align="right">0.74</td><td align="right">0.00e&#160;+&#160;00</td><td align="right">339.7</td></tr><tr><td align="left">PV+ B.</td><td align="right">0.9</td><td align="right">0.46</td><td align="right">0.00e&#160;+&#160;00</td><td align="right">356.8</td></tr><tr><td align="left">S.C.-A.</td><td align="right">5.2</td><td align="right">0.03</td><td align="right">1.13e &#8722;&#160;07</td><td align="right">197.9</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl34" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.113</object-id><label>Appendix 1&#8212;table 34.</label><caption><p>Model synaptic parameters for Ivy cells in the control network.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.113">http://dx.doi.org/10.7554/eLife.18566.113</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="4">Other cell to ivy</th><th align="center" colspan="4">Ivy to other cell</th></tr><tr><th align="center"><inline-formula><mml:math id="inf317"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf318"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf319"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">r</mml:mi><mml:mi mathvariant="bold-sans-serif">i</mml:mi><mml:mi mathvariant="bold-sans-serif">s</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf320"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf321"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf322"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf323"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">r</mml:mi><mml:mi mathvariant="bold-sans-serif">i</mml:mi><mml:mi mathvariant="bold-sans-serif">s</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf324"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;60.0</td><td align="right">5.700e-05</td><td align="right">2.90</td><td align="right">3.10</td></tr><tr><td align="left">Bis</td><td align="right">&#8722;60.0</td><td align="right">5.000e-04</td><td align="right">0.29</td><td align="right">2.67</td><td align="right">&#8722;60.0</td><td align="right">7.700e-05</td><td align="right">2.90</td><td align="right">3.10</td></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;60.0</td><td align="right">3.000e-04</td><td align="right">0.43</td><td align="right">4.49</td><td align="right">&#8722;60.0</td><td align="right">3.700e-05</td><td align="right">2.90</td><td align="right">3.10</td></tr><tr><td align="left">Ivy</td><td align="right">&#8722;60.0</td><td align="right">5.700e-05</td><td align="right">2.90</td><td align="right">3.10</td><td align="right">&#8722;60.0</td><td align="right">5.700e-05</td><td align="right">2.90</td><td align="right">3.10</td></tr><tr><td align="left">NGF</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;60.0</td><td align="right">5.700e-05</td><td align="right">2.90</td><td align="right">3.10</td></tr><tr><td align="left">O-LM</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;60.0</td><td align="right">5.700e-05</td><td align="right">2.90</td><td align="right">3.10</td></tr><tr><td align="left">Pyr</td><td align="right">0.0</td><td align="right">4.050e-04</td><td align="right">0.30</td><td align="right">0.60</td><td align="right">&#8722;60.0</td><td align="right">4.100e-05</td><td align="right">1.10</td><td align="right">11.00</td></tr><tr><td align="left">PV+B</td><td align="right">&#8722;60.0</td><td align="right">1.600e-04</td><td align="right">0.29</td><td align="right">2.67</td><td align="right">&#8722;60.0</td><td align="right">7.000e-04</td><td align="right">2.90</td><td align="right">3.10</td></tr><tr><td align="left">SC-A</td><td align="right">&#8722;60.0</td><td align="right">8.500e-04</td><td align="right">0.42</td><td align="right">4.99</td><td align="right">&#8722;60.0</td><td align="right">3.700e-05</td><td align="right">2.90</td><td align="right">3.10</td></tr><tr><td align="left">CA3</td><td align="right">0.0</td><td align="right">3.000e-04</td><td align="right">2.00</td><td align="right">6.30</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl35" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.114</object-id><label>Appendix 1&#8212;table 35.</label><caption><p>Model synaptic properties under voltage clamp at &#8722;50 mV with physiological reversal potentials</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.114">http://dx.doi.org/10.7554/eLife.18566.114</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="5">Other cell to ivy</th><th align="center" colspan="5">Ivy to other cell</th></tr><tr><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf325"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf326"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf327"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf328"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf329"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf330"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:msub><mml:mi mathvariant="bold-italic">&#964;</mml:mi><mml:mrow><mml:mi mathvariant="bold-sans-serif">d</mml:mi><mml:mi mathvariant="bold-sans-serif">e</mml:mi><mml:mi mathvariant="bold-sans-serif">c</mml:mi><mml:mi mathvariant="bold-sans-serif">a</mml:mi><mml:mi mathvariant="bold-sans-serif">y</mml:mi></mml:mrow></mml:msub></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">4.34</td><td align="right">2.13</td><td align="right">6.57</td></tr><tr><td align="left">Bis</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">43.40</td><td align="right">0.60</td><td align="right">3.17</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">6.39</td><td align="right">2.15</td><td align="right">6.63</td></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">22.34</td><td align="right">0.80</td><td align="right">5.05</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">3.00</td><td align="right">2.25</td><td align="right">6.95</td></tr><tr><td align="left">Ivy</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">5.48</td><td align="right">1.88</td><td align="right">6.42</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">5.48</td><td align="right">1.88</td><td align="right">6.42</td></tr><tr><td align="left">NGF</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">5.48</td><td align="right">1.88</td><td align="right">6.42</td></tr><tr><td align="left">O-LM</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">5.32</td><td align="right">2.10</td><td align="right">6.33</td></tr><tr><td align="left">Pyr</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">40.70</td><td align="right">0.58</td><td align="right">1.28</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">1.63</td><td align="right">3.63</td><td align="right">15.35</td></tr><tr><td align="left">PV+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">1.44</td><td align="right">0.55</td><td align="right">3.13</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">51.35</td><td align="right">2.05</td><td align="right">6.75</td></tr><tr><td align="left">SC-A</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">46.62</td><td align="right">0.85</td><td align="right">5.58</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">3.09</td><td align="right">2.22</td><td align="right">6.88</td></tr><tr><td align="left">CA3</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">29.42</td><td align="right">2.05</td><td align="right">8.60</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl38" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.119</object-id><label>Appendix 1&#8212;table 38.</label><caption><p>Structural connection parameters for Neurogliaform cells, based on <xref ref-type="bibr" rid="bib13">Bezaire and Soltesz (2013)</xref>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.119">http://dx.doi.org/10.7554/eLife.18566.119</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="3">Other type</th><th align="center" colspan="4">Other cell to NGF</th><th align="center" colspan="4">NGF to other cell</th></tr><tr><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th></tr><tr><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th></tr></thead><tbody><tr><td>Ivy</td><td align="right">28</td><td align="right">10</td><td align="right">280</td><td align="left">any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td/></tr><tr><td>NGF</td><td align="right">17</td><td align="right">10</td><td align="right">170</td><td align="left">apical dendrite</td><td align="right">17</td><td align="right">10</td><td align="right">170</td><td>apical dendrite</td></tr><tr><td>O-LM</td><td align="right">13</td><td align="right">10</td><td align="right">130</td><td align="left">apical dendrite</td><td align="right"/><td align="right"/><td align="right"/><td/></tr><tr><td>Pyr</td><td align="right"/><td align="right"/><td align="right"/><td align="left"/><td align="right">1218</td><td align="right">10</td><td align="right">12181</td><td>apical dendrite</td></tr><tr><td>ECIII</td><td align="right">523</td><td align="right">2</td><td align="right">1046</td><td align="left">any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl41" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.122</object-id><label>Appendix 1&#8212;table 41.</label><caption><p>Model synaptic parameters for Neurogliaform cells in the control network.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.122">http://dx.doi.org/10.7554/eLife.18566.122</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="4">Other cell to NGF</th><th align="center" colspan="4">NGF to other cell</th></tr><tr><th align="center"><inline-formula><mml:math id="inf337"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf338"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf339"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">rise</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf340"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf341"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf342"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf343"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">rise</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf344"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Ivy</td><td align="right">&#8722;60.0</td><td align="right">5.700e-05</td><td align="right">2.90</td><td align="right">3.10</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">NGF</td><td align="right">&#8722;60.0</td><td align="right">1.600e-04</td><td align="right">3.10</td><td align="right">42.00</td><td align="right">&#8722;60.0</td><td align="right">1.600e-04</td><td align="right">3.10</td><td align="right">42.00</td></tr><tr><td align="left">O-LM</td><td align="right">&#8722;60.0</td><td align="right">9.800e-05</td><td align="right">1.30</td><td align="right">10.20</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">Pyr</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;60.0</td><td align="right">6.500e-05</td><td align="right">9.00</td><td align="right">39.00</td></tr><tr><td align="left">ECIII</td><td align="right">0.0</td><td align="right">3.500e-03</td><td align="right">2.00</td><td align="right">6.30</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl42" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.123</object-id><label>Appendix 1&#8212;table 42.</label><caption><p>Model synaptic properties under voltage clamp at &#8722;50 mV with physiological reversal potentials</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.123">http://dx.doi.org/10.7554/eLife.18566.123</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="5">Other cell to NGF</th><th align="center" colspan="5">NGF to other cell</th></tr><tr><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf345"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf346"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf347"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf348"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf349"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf350"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Ivy</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">5.48</td><td align="right">1.88</td><td align="right">6.42</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">NGF</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">17.52</td><td align="right">5.67</td><td align="right">14.32</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">17.52</td><td align="right">5.67</td><td align="right">14.32</td></tr><tr><td align="left">O-LM</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">9.14</td><td align="right">1.98</td><td align="right">11.63</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">Pyr</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">1.10</td><td align="right">65.58</td><td align="right">0.00</td></tr><tr><td align="left">ECIII</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">324.35</td><td align="right">2.13</td><td align="right">8.80</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl6" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.060</object-id><label>Table 6.</label><caption><p>Firing rates and theta phase preferences for various cell types in various conditions. Theta phase is relative to the LFP recorded in the pyramidal layer, where 0<sup>o</sup> and 360<sup>o</sup> are at the trough of the oscillation. non: non-theta/non-SWR state. SWR: sharp wave/ripple. u+k and x: urethane + supplemental doses of ketamine and xylazine.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.060">http://dx.doi.org/10.7554/eLife.18566.060</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2"><bold>Cell</bold> type</th><th align="center" colspan="3">Firing rate (Hz)</th><th align="center" rowspan="2">Theta phase (<inline-formula><mml:math id="inf20"><mml:msup><mml:mi/><mml:mi>o</mml:mi></mml:msup></mml:math></inline-formula>)</th><th align="center" rowspan="2">State of animal</th><th align="center" rowspan="2">Animal</th><th align="center" rowspan="2">Ref.</th></tr><tr><th align="center">Theta</th><th align="center">Non</th><th align="center">SWR</th></tr></thead><tbody><tr><td>ADI</td><td align="right"><inline-formula><mml:math id="inf21"><mml:mn>8.60</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf22"><mml:mn>0.06</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf23"><mml:mn>0.25</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf24"><mml:mn>156</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib76">Klausberger et al., 2005</xref>)</td></tr><tr><td>Axo-axonic</td><td align="right"><inline-formula><mml:math id="inf25"><mml:mn>17.10</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf26"><mml:mn>3.50</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf27"><mml:mn>2.95</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf28"><mml:mn>185</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib75">Klausberger et al., 2003</xref>)</td></tr><tr><td>Axo-axonic</td><td align="right"><inline-formula><mml:math id="inf29"><mml:mn>27</mml:mn></mml:math></inline-formula></td><td align="right"/><td align="right"><inline-formula><mml:math id="inf30"><mml:mn>27</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf31"><mml:mn>251</mml:mn></mml:math></inline-formula></td><td>awake, head restraint</td><td>mouse</td><td>(<xref ref-type="bibr" rid="bib150">Varga et al., 2014</xref>)</td></tr><tr><td>Bistratified</td><td align="right"><inline-formula><mml:math id="inf32"><mml:mn>5.90</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf33"><mml:mn>0.90</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf34"><mml:mn>42.80</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf35"><mml:mn>1</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib77">Klausberger et al., 2004</xref>)</td></tr><tr><td>Bistratified</td><td align="right"><inline-formula><mml:math id="inf36"><mml:mn>34</mml:mn></mml:math></inline-formula></td><td align="right"/><td align="right"><inline-formula><mml:math id="inf37"><mml:mn>36</mml:mn></mml:math></inline-formula></td><td align="right">0</td><td>awake, head restraint</td><td>mouse</td><td>(<xref ref-type="bibr" rid="bib150">Varga et al., 2014</xref>)</td></tr><tr><td>Bistratified</td><td align="right"><inline-formula><mml:math id="inf38"><mml:mn>30.42</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf39"><mml:mn>27.65</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf40"><mml:mn>35.82</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf41"><mml:mn>2</mml:mn></mml:math></inline-formula></td><td>awake</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib71">Katona et al., 2014</xref>)</td></tr><tr><td>CCK+ Basket</td><td align="right"><inline-formula><mml:math id="inf42"><mml:mn>9.40</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf43"><mml:mn>1.60</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf44"><mml:mn>2.70</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf45"><mml:mn>174</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib76">Klausberger et al., 2005</xref>)</td></tr><tr><td>Ivy</td><td align="right"><inline-formula><mml:math id="inf46"><mml:mn>0.70</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf47"><mml:mn>1.70</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf48"><mml:mn>0.80</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf49"><mml:mn>31</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib46">Fuentealba et al., 2008</xref>)</td></tr><tr><td>Ivy</td><td align="right"><inline-formula><mml:math id="inf50"><mml:mn>2.80</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf51"><mml:mn>2.10</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf52"><mml:mn>5.20</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf53"><mml:mn>46</mml:mn></mml:math></inline-formula></td><td>awake, free</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib84">Lapray et al., 2012</xref>)</td></tr><tr><td>Ivy</td><td align="right"><inline-formula><mml:math id="inf54"><mml:mn>2.40</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf55"><mml:mn>3.00</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf56"><mml:mn>6.70</mml:mn></mml:math></inline-formula></td><td align="right"/><td>awake, free</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib46">Fuentealba et al., 2008</xref>)</td></tr><tr><td>NGF</td><td align="right"><inline-formula><mml:math id="inf57"><mml:mn>6.00</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf58"><mml:mn>2.65</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf59"><mml:mn>2.30</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf60"><mml:mn>196</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib47">Fuentealba et al., 2010</xref>)</td></tr><tr><td>O-LM</td><td align="right"><inline-formula><mml:math id="inf61"><mml:mn>4.90</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf62"><mml:mn>2.30</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf63"><mml:mn>0.23</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf64"><mml:mn>19</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib75">Klausberger et al., 2003</xref>)</td></tr><tr><td>O-LM</td><td align="right"><inline-formula><mml:math id="inf65"><mml:mn>29.80</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf66"><mml:mn>10.40</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf67"><mml:mn>25.40</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf68"><mml:mn>346</mml:mn></mml:math></inline-formula></td><td>awake, head restraint</td><td>mouse</td><td>(<xref ref-type="bibr" rid="bib149">Varga et al., 2012</xref>)</td></tr><tr><td>O-LM</td><td align="right"><inline-formula><mml:math id="inf69"><mml:mn>17.30</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf70"><mml:mn>11.88</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf71"><mml:mn>18.95</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf72"><mml:mn>342</mml:mn></mml:math></inline-formula></td><td>awake</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib71">Katona et al., 2014</xref>)</td></tr><tr><td>PPA</td><td align="right"><inline-formula><mml:math id="inf73"><mml:mn>5.75</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf74"><mml:mn>1.95</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf75"><mml:mn>1.50</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf76"><mml:mn>100</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib76">Klausberger et al., 2005</xref>)</td></tr><tr><td>PV+ Basket</td><td align="right"><inline-formula><mml:math id="inf77"><mml:mn>7.30</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf78"><mml:mn>2.74</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf79"><mml:mn>32.68</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf80"><mml:mn>271</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib75">Klausberger et al., 2003</xref>)</td></tr><tr><td>PV+ Basket</td><td align="right"/><td align="right"/><td align="right"/><td align="right"><inline-formula><mml:math id="inf81"><mml:mn>234</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib76">Klausberger et al., 2005</xref>)</td></tr><tr><td>PV+ Basket</td><td align="right"><inline-formula><mml:math id="inf82"><mml:mn>21.00</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf83"><mml:mn>6.50</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf84"><mml:mn>122.00</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf85"><mml:mn>289</mml:mn></mml:math></inline-formula></td><td>awake, free</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib84">Lapray et al., 2012</xref>)</td></tr><tr><td>PV+ Basket</td><td align="right"><inline-formula><mml:math id="inf86"><mml:mn>25.00</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf87"><mml:mn>8.20</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf88"><mml:mn>75.00</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf89"><mml:mn>307</mml:mn></mml:math></inline-formula></td><td>awake, head restraint</td><td>mouse</td><td>(<xref ref-type="bibr" rid="bib149">Varga et al., 2012</xref>)</td></tr><tr><td>PV+ Basket</td><td align="right"><inline-formula><mml:math id="inf90"><mml:mn>28</mml:mn></mml:math></inline-formula></td><td align="right"/><td align="right"><inline-formula><mml:math id="inf91"><mml:mn>77</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf92"><mml:mn>310</mml:mn></mml:math></inline-formula></td><td>awake, head restraint</td><td>mouse</td><td>(<xref ref-type="bibr" rid="bib150">Varga et al., 2014</xref>)</td></tr><tr><td>Pyramidal</td><td align="right"/><td align="right"/><td align="right"/><td align="right"><inline-formula><mml:math id="inf93"><mml:mn>20</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib75">Klausberger et al., 2003</xref>)</td></tr><tr><td>Trilaminar</td><td align="right"><inline-formula><mml:math id="inf94"><mml:mn>0.20</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf95"><mml:mn>0.10</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf96"><mml:mn>69.00</mml:mn></mml:math></inline-formula></td><td align="right">trough</td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib43">Ferraguti et al., 2005</xref>)</td></tr><tr><td>Double Proj.</td><td align="right"><inline-formula><mml:math id="inf97"><mml:mn>0.90</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf98"><mml:mn>0.55</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf99"><mml:mn>26.93</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf100"><mml:mn>77</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib69">Jinno et al., 2007</xref>)</td></tr><tr><td>Oriens Retro.</td><td align="right"><inline-formula><mml:math id="inf101"><mml:mn>0.53</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf102"><mml:mn>0.37</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf103"><mml:mn>53.37</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf104"><mml:mn>28</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib69">Jinno et al., 2007</xref>)</td></tr><tr><td>Radiatum Retro.</td><td align="right"><inline-formula><mml:math id="inf105"><mml:mn>5.15</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf106"><mml:mn>1.90</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf107"><mml:mn>0.70</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf108"><mml:mn>298</mml:mn></mml:math></inline-formula></td><td>anesth: u+k and x</td><td>rat</td><td>(<xref ref-type="bibr" rid="bib69">Jinno et al., 2007</xref>)</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl45" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.128</object-id><label>Appendix 1&#8212;table 45.</label><caption><p>Structural connection parameters for O-LM cells, based on <xref ref-type="bibr" rid="bib13">Bezaire and Soltesz (2013)</xref>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.128">http://dx.doi.org/10.7554/eLife.18566.128</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="3">Other type</th><th align="center" colspan="4">Other cell to O-LM</th><th align="center" colspan="4">O-LM to other cell</th></tr><tr><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th></tr><tr><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th></tr></thead><tbody><tr><td>Axo</td><td align="right"/><td align="right"/><td align="right"/><td/><td align="right">7</td><td align="right">10</td><td align="right">71</td><td>apical dendrite</td></tr><tr><td>Bis</td><td align="right">39</td><td align="right">10</td><td align="right">390</td><td>any dendrite</td><td align="right">11</td><td align="right">10</td><td align="right">107</td><td>apical dendrite</td></tr><tr><td>CCK+B</td><td align="right">20</td><td align="right">8</td><td align="right">160</td><td>any dendrite</td><td align="right">88</td><td align="right">10</td><td align="right">878</td><td>apical dendrite</td></tr><tr><td>Ivy</td><td align="right">136</td><td align="right">10</td><td align="right">1360</td><td>any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td/></tr><tr><td>NGF</td><td align="right"/><td align="right"/><td align="right"/><td/><td align="right">28</td><td align="right">10</td><td align="right">283</td><td>apical dendrite</td></tr><tr><td>O-LM</td><td align="right">6</td><td align="right">10</td><td align="right">60</td><td>basal dendrite</td><td align="right">6</td><td align="right">10</td><td align="right">60</td><td>basal dendrite</td></tr><tr><td>Pyr</td><td align="right">2379</td><td align="right">3</td><td align="right">7137</td><td>basal dendrite</td><td align="right">1520</td><td align="right">10</td><td align="right">15195</td><td>apical dendrite</td></tr><tr><td>PV+B</td><td align="right"/><td align="right"/><td align="right"/><td/><td align="right">27</td><td align="right">10</td><td align="right">269</td><td>apical dendrite</td></tr><tr><td>SC-A</td><td align="right"/><td align="right"/><td align="right"/><td/><td align="right">10</td><td align="right">10</td><td align="right">97</td><td>apical dendrite</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl47" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.130</object-id><label>Appendix 1&#8212;table 47.</label><caption><p>Model synaptic parameters for O-LM cells in the control network.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.130">http://dx.doi.org/10.7554/eLife.18566.130</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="4">Other cell to O-LM</th><th align="center" colspan="4">O-LM to other cell</th></tr><tr><th align="center"><inline-formula><mml:math id="inf354"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf355"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf356"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">rise</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf357"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf358"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf359"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf360"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">rise</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf361"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;60.0</td><td align="right">1.200e-04</td><td align="right">0.73</td><td align="right">10.00</td></tr><tr><td align="left">Bis</td><td align="right">&#8722;60.0</td><td align="right">2.000e-05</td><td align="right">1.00</td><td align="right">8.00</td><td align="right">&#8722;60.0</td><td align="right">1.100e-04</td><td align="right">0.60</td><td align="right">15.00</td></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;60.0</td><td align="right">7.000e-04</td><td align="right">1.00</td><td align="right">8.00</td><td align="right">&#8722;60.0</td><td align="right">1.200e-03</td><td align="right">0.73</td><td align="right">20.20</td></tr><tr><td align="left">Ivy</td><td align="right">&#8722;60.0</td><td align="right">5.700e-05</td><td align="right">2.90</td><td align="right">3.10</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">NGF</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;60.0</td><td align="right">9.800e-05</td><td align="right">1.30</td><td align="right">10.20</td></tr><tr><td align="left">O-LM</td><td align="right">&#8722;60.0</td><td align="right">1.200e-03</td><td align="right">0.25</td><td align="right">7.50</td><td align="right">&#8722;60.0</td><td align="right">1.200e-03</td><td align="right">0.25</td><td align="right">7.50</td></tr><tr><td align="left">Pyr</td><td align="right">0.0</td><td align="right">2.000e-04</td><td align="right">0.30</td><td align="right">0.60</td><td align="right">&#8722;60.0</td><td align="right">3.000e-04</td><td align="right">0.13</td><td align="right">11.00</td></tr><tr><td align="left">PV+B</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;60.0</td><td align="right">1.100e-03</td><td align="right">0.25</td><td align="right">7.50</td></tr><tr><td align="left">SC-A</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;60.0</td><td align="right">1.500e-04</td><td align="right">0.07</td><td align="right">29.00</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl48" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.131</object-id><label>Appendix 1&#8212;table 48.</label><caption><p>Model synaptic properties under voltage clamp at &#8722;50 mV with physiological reversal potentials</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.131">http://dx.doi.org/10.7554/eLife.18566.131</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="5">Other cell to O-LM</th><th align="center" colspan="5">O-LM to other cell</th></tr><tr><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf362"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf363"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf364"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf365"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf366"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf367"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula>(ms)</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">4.76</td><td align="right">2.55</td><td align="right">12.03</td></tr><tr><td align="left">Bis</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">1.86</td><td align="right">1.78</td><td align="right">8.13</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">6.31</td><td align="right">2.70</td><td align="right">17.05</td></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">54.98</td><td align="right">1.35</td><td align="right">9.05</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">40.32</td><td align="right">3.10</td><td align="right">28.42</td></tr><tr><td align="left">Ivy</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">5.32</td><td align="right">2.10</td><td align="right">6.33</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">NGF</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">9.14</td><td align="right">1.98</td><td align="right">11.63</td></tr><tr><td align="left">O-LM</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">78.69</td><td align="right">1.05</td><td align="right">9.30</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">78.69</td><td align="right">1.05</td><td align="right">9.30</td></tr><tr><td align="left">Pyr</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">17.47</td><td align="right">0.60</td><td align="right">1.53</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">0.54</td><td align="right">3.70</td><td align="right">14.10</td></tr><tr><td align="left">PV+B</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">35.53</td><td align="right">1.65</td><td align="right">10.18</td></tr><tr><td align="left">SC-A</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">7.91</td><td align="right">3.90</td><td align="right">29.83</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl51" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.136</object-id><label>Appendix 1&#8212;table 51.</label><caption><p>Structural connection parameters for PV+ Basket cells, based on <xref ref-type="bibr" rid="bib13">Bezaire and Soltesz (2013)</xref>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.136">http://dx.doi.org/10.7554/eLife.18566.136</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="3">Other type</th><th align="center" colspan="4">Other cell to PV+B</th><th align="center" colspan="4">PV+B to other cell</th></tr><tr><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th></tr><tr><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th></tr></thead><tbody><tr><td>Axo</td><td align="right"/><td align="right"/><td align="right"/><td/><td align="right">10</td><td align="right">1</td><td align="right">10</td><td>soma</td></tr><tr><td>Bis</td><td align="right">16</td><td align="right">10</td><td align="right">160</td><td>any dendrite</td><td align="right">16</td><td align="right">1</td><td align="right">15</td><td>soma</td></tr><tr><td>CCK+B</td><td align="right">12</td><td align="right">8</td><td align="right">96</td><td>any dendrite</td><td align="right">25</td><td align="right">1</td><td align="right">24</td><td>soma</td></tr><tr><td>Ivy</td><td align="right">24</td><td align="right">10</td><td align="right">240</td><td>any dendrite</td><td align="right">13</td><td align="right">1</td><td align="right">12</td><td>soma</td></tr><tr><td>O-LM</td><td align="right">8</td><td align="right">10</td><td align="right">80</td><td>apical dendrite</td><td align="right"/><td align="right"/><td align="right"/><td/></tr><tr><td>Pyr</td><td align="right">424</td><td align="right">3</td><td align="right">1272</td><td>apical dendrite</td><td align="right">958</td><td align="right">11</td><td align="right">10533</td><td>soma</td></tr><tr><td>PV+B</td><td align="right">39</td><td align="right">1</td><td align="right">39</td><td>soma</td><td align="right">39</td><td align="right">1</td><td align="right">39</td><td>soma</td></tr><tr><td>SC-A</td><td align="right"/><td align="right"/><td align="right"/><td/><td align="right">2</td><td align="right">1</td><td align="right">1</td><td>soma</td></tr><tr><td>CA3</td><td align="right">6047</td><td align="right">2</td><td align="right">12094</td><td>any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl54" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.139</object-id><label>Appendix 1&#8212;table 54.</label><caption><p>Model synaptic parameters for PV+ Basket cells in the control network.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.139">http://dx.doi.org/10.7554/eLife.18566.139</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="4">Other cell to PV+B</th><th align="center" colspan="4">PV+B to other cell</th></tr><tr><th align="center"><inline-formula><mml:math id="inf374"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf375"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula>(nS)</th><th align="center"><inline-formula><mml:math id="inf376"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">rise</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf377"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf378"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center"><inline-formula><mml:math id="inf379"><mml:msub><mml:mi mathvariant="normal">G</mml:mi><mml:mi mathvariant="normal">max</mml:mi></mml:msub></mml:math></inline-formula> (nS)</th><th align="center"><inline-formula><mml:math id="inf380"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">rise</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf381"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;60.0</td><td align="right">1.200e-04</td><td align="right">0.29</td><td align="right">2.67</td></tr><tr><td align="left">Bis</td><td align="right">&#8722;60.0</td><td align="right">9.000e-03</td><td align="right">0.29</td><td align="right">2.67</td><td align="right">&#8722;60.0</td><td align="right">2.900e-03</td><td align="right">0.18</td><td align="right">0.45</td></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;60.0</td><td align="right">9.000e-03</td><td align="right">0.43</td><td align="right">4.49</td><td align="right">&#8722;60.0</td><td align="right">1.200e-03</td><td align="right">0.29</td><td align="right">2.67</td></tr><tr><td align="left">Ivy</td><td align="right">&#8722;60.0</td><td align="right">7.000e-04</td><td align="right">2.90</td><td align="right">3.10</td><td align="right">&#8722;60.0</td><td align="right">1.600e-04</td><td align="right">0.29</td><td align="right">2.67</td></tr><tr><td align="left">O-LM</td><td align="right">&#8722;60.0</td><td align="right">1.100e-03</td><td align="right">0.25</td><td align="right">7.50</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">Pyr</td><td align="right">0.0</td><td align="right">7.000e-04</td><td align="right">0.07</td><td align="right">0.20</td><td align="right">&#8722;60.0</td><td align="right">2.000e-04</td><td align="right">0.30</td><td align="right">6.20</td></tr><tr><td align="left">PV+B</td><td align="right">&#8722;60.0</td><td align="right">1.600e-03</td><td align="right">0.08</td><td align="right">4.80</td><td align="right">&#8722;60.0</td><td align="right">1.600e-03</td><td align="right">0.08</td><td align="right">4.80</td></tr><tr><td align="left">SC-A</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;60.0</td><td align="right">6.000e-04</td><td align="right">0.29</td><td align="right">2.67</td></tr><tr><td align="left">CA3</td><td align="right">0.0</td><td align="right">2.200e-04</td><td align="right">2.00</td><td align="right">6.30</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl55" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.140</object-id><label>Appendix 1&#8212;table 55.</label><caption><p>Model synaptic properties under voltage clamp at &#8722;50 mV with physiological reversal potentials</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.140">http://dx.doi.org/10.7554/eLife.18566.140</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="5">Other cell to PV+B</th><th align="center" colspan="5">PV+B to other cell</th></tr><tr><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf382"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf383"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf384"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf385"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf386"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf387"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">1.08</td><td align="right">0.45</td><td align="right">3.13</td></tr><tr><td align="left">Bis</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">429.34</td><td align="right">0.57</td><td align="right">4.13</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">24.45</td><td align="right">0.17</td><td align="right">0.73</td></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">523.11</td><td align="right">0.68</td><td align="right">5.70</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">11.31</td><td align="right">0.42</td><td align="right">3.08</td></tr><tr><td align="left">Ivy</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">51.35</td><td align="right">2.05</td><td align="right">6.75</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">1.44</td><td align="right">0.55</td><td align="right">3.13</td></tr><tr><td align="left">O-LM</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">35.53</td><td align="right">1.65</td><td align="right">10.18</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">Pyr</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">14.75</td><td align="right">0.25</td><td align="right">1.77</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">20.56</td><td align="right">0.50</td><td align="right">6.70</td></tr><tr><td align="left">PV+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">13.94</td><td align="right">0.23</td><td align="right">5.25</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">13.94</td><td align="right">0.23</td><td align="right">5.25</td></tr><tr><td align="left">SC-A</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">5.71</td><td align="right">0.42</td><td align="right">3.08</td></tr><tr><td align="left">CA3</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">19.71</td><td align="right">2.38</td><td align="right">8.78</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl58" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.145</object-id><label>Appendix 1&#8212;table 58.</label><caption><p>Structural connection parameters for Schaffer Collateral-Associated cells, based on <xref ref-type="bibr" rid="bib13">Bezaire and Soltesz (2013)</xref>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.145">http://dx.doi.org/10.7554/eLife.18566.145</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="3">Other type</th><th align="center" colspan="4">Other cell to SC-A</th><th align="center" colspan="4">SC-A to other cell</th></tr><tr><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th><th align="center">#</th><th align="center">Syn.s</th><th align="center">#</th><th align="center">Post</th></tr><tr><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th><th align="center">Conn.s</th><th align="center">/Conn.</th><th align="center">#</th><th align="center">Loc.</th></tr></thead><tbody><tr><td>Axo</td><td align="right"/><td align="right"/><td align="right"/><td/><td align="right">4</td><td align="right">6</td><td align="right">22</td><td>any dendrite</td></tr><tr><td>Bis</td><td align="right">17</td><td align="right">10</td><td align="right">170</td><td>any dendrite</td><td align="right">6</td><td align="right">6</td><td align="right">33</td><td>any dendrite</td></tr><tr><td>CCK+B</td><td align="right">27</td><td align="right">8</td><td align="right">216</td><td>any dendrite</td><td align="right">54</td><td align="right">6</td><td align="right">324</td><td>any dendrite</td></tr><tr><td>Ivy</td><td align="right">102</td><td align="right">10</td><td align="right">1020</td><td>any dendrite</td><td align="right">44</td><td align="right">6</td><td align="right">264</td><td>any dendrite</td></tr><tr><td>O-LM</td><td align="right">40</td><td align="right">10</td><td align="right">400</td><td>apical dendrite</td><td align="right"/><td align="right"/><td align="right"/><td/></tr><tr><td>Pyr</td><td align="right">105</td><td align="right">3</td><td align="right">315</td><td>apical dendrite</td><td align="right"/><td align="right"/><td align="right"/><td/></tr><tr><td>PV+B</td><td align="right">24</td><td align="right">1</td><td align="right">24</td><td>soma</td><td align="right"/><td align="right"/><td align="right"/><td/></tr><tr><td>CA3</td><td align="right">1940</td><td align="right">2</td><td align="right">3880</td><td>any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td/></tr><tr><td>ECIII</td><td align="right">573</td><td align="right">2</td><td align="right">1146</td><td>any dendrite</td><td align="right"/><td align="right"/><td align="right"/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl62" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.149</object-id><label>Appendix 1&#8212;table 62.</label><caption><p>Model synaptic properties under voltage clamp at &#8722;50 mV with physiological reversal potentials</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.149">http://dx.doi.org/10.7554/eLife.18566.149</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Type</th><th align="center" colspan="5">Other cell to SC-A</th><th align="center" colspan="5">SC-A to other cell</th></tr><tr><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf402"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf403"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf404"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th><th align="center">Hold (mV)</th><th align="center"><inline-formula><mml:math id="inf405"><mml:msub><mml:mi mathvariant="normal">E</mml:mi><mml:mi mathvariant="normal">rev</mml:mi></mml:msub></mml:math></inline-formula> (mV)</th><th align="center">Amp. (pA)</th><th align="center"><inline-formula><mml:math id="inf406"><mml:msub><mml:mi mathvariant="normal">t</mml:mi><mml:mrow><mml:mn mathvariant="normal">10</mml:mn><mml:mo mathvariant="bold">-</mml:mo><mml:mn mathvariant="normal">90</mml:mn></mml:mrow></mml:msub></mml:math></inline-formula> (ms)</th><th align="center"><inline-formula><mml:math id="inf407"><mml:msub><mml:mi mathvariant="bold">&#120533;</mml:mi><mml:mi mathvariant="normal">decay</mml:mi></mml:msub></mml:math></inline-formula> (ms)</th></tr></thead><tbody><tr><td align="left">Axo</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">24.00</td><td align="right">1.00</td><td align="right">6.13</td></tr><tr><td align="left">Bis</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">50.35</td><td align="right">0.70</td><td align="right">4.10</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">26.43</td><td align="right">1.02</td><td align="right">6.20</td></tr><tr><td align="left">CCK+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">49.55</td><td align="right">0.70</td><td align="right">5.38</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">33.81</td><td align="right">1.05</td><td align="right">6.90</td></tr><tr><td align="left">Ivy</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">3.09</td><td align="right">2.22</td><td align="right">6.88</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">46.62</td><td align="right">0.85</td><td align="right">5.58</td></tr><tr><td align="left">O-LM</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">7.91</td><td align="right">3.90</td><td align="right">29.83</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">Pyr</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">17.42</td><td align="right">0.68</td><td align="right">3.05</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">PV+B</td><td align="right">&#8722;50.0</td><td align="right">&#8722;60.0</td><td align="right">5.71</td><td align="right">0.42</td><td align="right">3.08</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">CA3</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">27.10</td><td align="right">2.35</td><td align="right">9.13</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td align="left">ECIII</td><td align="right">&#8722;50.0</td><td align="right">0.0</td><td align="right">31.82</td><td align="right">3.38</td><td align="right">10.47</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl63" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.151</object-id><label>Appendix 1&#8212;table 63.</label><caption><p>Measured dendritic lengths and somatic diameters for ivy and neurogliaform cells from the hippocampal CA1 area in Wistar rats, with calculation of somatic surface area included. Cells were characterized in our lab and their function has been reported in <xref ref-type="bibr" rid="bib82">Krook-Magnuson et al. (2011)</xref>. Source Data available in <xref ref-type="table" rid="Atbl1">Appendix 1&#8212;table 1</xref> - Source Data.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.151">http://dx.doi.org/10.7554/eLife.18566.151</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Cell type</th><th align="center" rowspan="2">Cell name</th><th align="center"/><th align="center" colspan="4">Dendritic length (<inline-formula><mml:math id="inf408"><mml:mi mathsize="125%">&#956;</mml:mi></mml:math></inline-formula>M))</th><th align="center" rowspan="2">Somatic dia-meter (<inline-formula><mml:math id="inf409"><mml:mi mathsize="125%">&#956;</mml:mi></mml:math></inline-formula>m)</th><th align="center" rowspan="2">Calculated synap-tic Area (<inline-formula><mml:math id="inf410"><mml:mrow><mml:mi mathsize="125%">&#956;</mml:mi><mml:mo mathsize="80%" mathvariant="bold">&#8290;</mml:mo><mml:msup><mml:mi mathsize="125%">m</mml:mi><mml:mn mathsize="125%" mathvariant="normal">2</mml:mn></mml:msup></mml:mrow></mml:math></inline-formula>)</th></tr><tr><th align="center"># Sections</th><th align="center">SO</th><th align="center">SP</th><th align="center">SR</th><th align="center">SLM</th></tr></thead><tbody><tr><td>Ivy</td><td>0217&#8211;1 DAB 3_2_10 left slice</td><td align="right">1</td><td align="right">129.2</td><td align="right">64.5</td><td align="right">1200.6</td><td align="right">0</td><td align="right">38.9</td><td align="right">1188.5</td></tr><tr><td>Ivy</td><td>9 n23-7 DAB 12_16_09 left+middle slice</td><td align="right">2</td><td align="right">0</td><td align="right">0</td><td align="right">2703.2</td><td align="right">300.3</td><td align="right">45.2</td><td align="right">1604.6</td></tr><tr><td>Ivy</td><td>9 n23-6 DAB 06_10 left slice</td><td align="right">1</td><td align="right">75.4</td><td align="right">133.8</td><td align="right">2115.4</td><td align="right">0</td><td align="right">36.6</td><td align="right">1052.1</td></tr><tr><td>Ivy</td><td>9 n16-3 DAB 12_29_09 left slice</td><td align="right">1</td><td align="right">0</td><td align="right">0</td><td align="right">1015.2</td><td align="right">0</td><td align="right">52.5</td><td align="right">2164.8</td></tr><tr><td>Ivy</td><td>Average</td><td align="right"/><td align="right">51.15</td><td align="right">49.575</td><td align="right">1758.6</td><td align="right">75.075</td><td align="right"/><td align="right">1502.5</td></tr><tr><td>Neurogliaform</td><td>9n 12&#8211;5 DAB 1_06_09</td><td align="right">1</td><td align="right">0</td><td align="right">0</td><td align="right">2097.7</td><td align="right">525</td><td align="right">34.4</td><td align="right">929.4</td></tr><tr><td>Neurogliaform</td><td>91021 DAB 3_18_10 second,third,fourth from left slice</td><td align="right">3</td><td align="right">0</td><td align="right">0</td><td align="right">1230.7</td><td align="right">780.1</td><td align="right">28</td><td align="right">615.8</td></tr><tr><td>Neurogliaform</td><td>9d 8&#8211;3 DAB 1_15_10 left and right slice</td><td align="right">2</td><td align="right">0</td><td align="right">0</td><td align="right">2328.2</td><td align="right">1382.4</td><td align="right">32.2</td><td align="right">814.3</td></tr><tr><td>Neurogliaform</td><td>Average</td><td align="right"/><td align="right">0</td><td align="right">0</td><td align="right">1885.5</td><td align="right">895.8</td><td align="right"/><td align="right">786.5</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl64" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.152</object-id><label>Appendix 1&#8212;table 64.</label><caption><p>Estimated or observed somatic area and dendritic length. Experimental observations of the dendritic length of broad interneuron classes were used as the basis for these estimations. The relative lengths for PV+ basket cells and axo-axonic cells were further differentiated based on experimental observations in region CA3 <xref ref-type="bibr" rid="bib113">Papp et al. (2013)</xref>. The observations published in <xref ref-type="bibr" rid="bib109">M&#225;ty&#225;s et al. (2004)</xref> for CCK+ basket cells were also applied to the CCK+ Schaffer Collateral-Associated cells, based on the discussion in <xref ref-type="bibr" rid="bib109">M&#225;ty&#225;s et al. (2004)</xref>. The data for ivy and neurogliaform cells were based on measurements from filled cells from slices. Due to the compact nature of their morphology, especially the neurogliaform cells, the dendritic lengths within the slices were assumed to comprise most or all of the dendritic extents of those cells. See section below for raw data. The O-LM cell morphological measurements were taken from <xref ref-type="bibr" rid="bib14">Blasco-Ib&#225;&#241;ez and Freund (1995)</xref>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.152">http://dx.doi.org/10.7554/eLife.18566.152</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2">Interneuron</th><th align="center" rowspan="2">Soma area (100 <inline-formula><mml:math id="inf411"><mml:mrow><mml:mi>&#956;</mml:mi><mml:mo mathvariant="bold">&#8290;</mml:mo><mml:msup><mml:mi>m</mml:mi><mml:mn mathvariant="normal">2</mml:mn></mml:msup></mml:mrow></mml:math></inline-formula>)</th><th align="center" colspan="5">Dendritic length (<inline-formula><mml:math id="inf412"><mml:mrow><mml:mi>&#956;</mml:mi><mml:mo mathvariant="bold">&#8290;</mml:mo><mml:msup><mml:mi>m</mml:mi><mml:mn mathvariant="normal">2</mml:mn></mml:msup></mml:mrow></mml:math></inline-formula>)</th><th align="center" rowspan="2">Reference</th></tr><tr><th align="center">Total</th><th align="center">SO</th><th align="center">SP</th><th align="center">SR</th><th align="center">SLM</th></tr></thead><tbody><tr><td>Ivy</td><td align="right">1502</td><td align="right">1934.4</td><td align="right">51.15</td><td align="right">49.575</td><td align="right">1758.6</td><td align="right">75.075</td><td>See below</td></tr><tr><td>Neurogliaform</td><td align="right">786</td><td align="right">2781.4</td><td align="right">0</td><td align="right">0</td><td align="right">1885.5</td><td align="right">895.8</td><td>See below</td></tr><tr><td>PV+ basket</td><td align="right">3428</td><td align="right">4359</td><td align="right">1493</td><td align="right">697</td><td align="right">1877</td><td align="right">292</td><td>(<xref ref-type="bibr" rid="bib113">Papp et al., 2013</xref>)</td></tr><tr><td>Bistratified</td><td align="right">1006</td><td align="right">4347.75</td><td align="right">1074.57</td><td align="right">248.28</td><td align="right">2369.24</td><td align="right">655.66</td><td>(<xref ref-type="bibr" rid="bib52">Guly&#225;s et al., 1999</xref>)</td></tr><tr><td>Axo-axonic</td><td align="right">2329</td><td align="right">2825</td><td align="right">570</td><td align="right">659</td><td align="right">1259</td><td align="right">337</td><td>(<xref ref-type="bibr" rid="bib113">Papp et al., 2013</xref>)</td></tr><tr><td>CCK+ basket</td><td align="right">966</td><td align="right">6338.31</td><td align="right">1213.92</td><td align="right">310.61</td><td align="right">3522.6</td><td align="right">1291.18</td><td>(<xref ref-type="bibr" rid="bib109">M&#225;ty&#225;s et al., 2004</xref>)</td></tr><tr><td>SCA</td><td align="right">966</td><td align="right">6338.31</td><td align="right">1213.92</td><td align="right">310.61</td><td align="right">3522.6</td><td align="right">1291.18</td><td>(<xref ref-type="bibr" rid="bib109">M&#225;ty&#225;s et al., 2004</xref>)</td></tr><tr><td>O-LM</td><td align="right">3007.78</td><td align="right">4165.68</td><td align="right">4165.68</td><td align="right">0</td><td align="right">0</td><td align="right">0</td><td>(<xref ref-type="bibr" rid="bib14">Blasco-Ib&#225;&#241;ez and Freund, 1995</xref>)</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl67" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.155</object-id><label>Appendix 1&#8212;table 67.</label><caption><p>Ion channels included in the model. GHK: based on Goldman-Hodgkin-Katz equation; Q-O: quasi-ohmic; Hyperpol.-act: Hyperpolarization-activated; Nucleo.-gated: Nucleotide-gated; voltage-act.: voltage activated; voltage-dep.: voltage dependent; Calcium-act.: Calcium-activated; Pyr.: pyramidal; NGF: neurogliaform; dist.: distal; prox.: proximal.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.155">http://dx.doi.org/10.7554/eLife.18566.155</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th>Ion</th><th/><th>Model</th><th/><th/><th/><th/><th/><th/><th/><th/><th/></tr><tr><th align="center">Channel</th><th align="center">Description</th><th align="center">Type</th><th align="center" rowspan="-4">Pyramidal</th><th rowspan="-4">Axo-axonic</th><th rowspan="-4">Bistratified</th><th rowspan="-4">CCK+ Basket</th><th rowspan="-4">Ivy</th><th rowspan="-4">Neurogliaform</th><th rowspan="-4">O-LM</th><th rowspan="-4">PV+ Basket</th><th rowspan="-4">S.C.-Assoc.</th></tr></thead><tbody><tr><td>Ca<inline-formula><mml:math id="inf416"><mml:msub><mml:mi/><mml:mrow><mml:mi id="XM15" mathbackground="white" mathsize="125%">v</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mi id="XM16" mathbackground="white" mathsize="125%">L</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td>L-type Calcium</td><td>GHK</td><td/><td><inline-formula><mml:math id="inf417"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf418"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf419"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf420"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf421"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td><inline-formula><mml:math id="inf422"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf423"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td></tr><tr><td>Ca<inline-formula><mml:math id="inf424"><mml:msub><mml:mi/><mml:mrow><mml:mi id="XM17" mathbackground="white" mathsize="125%">v</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mi id="XM18" mathbackground="white" mathsize="125%">N</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td>N-type Calcium</td><td>Q-O</td><td/><td><inline-formula><mml:math id="inf425"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf426"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf427"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf428"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf429"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td><inline-formula><mml:math id="inf430"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf431"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td></tr><tr><td>HCN</td><td>Hyperpol.-act, Cyclic Nucleo.-gated</td><td>Q-O</td><td/><td/><td/><td><inline-formula><mml:math id="inf432"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/><td/><td><inline-formula><mml:math id="inf433"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td></tr><tr><td>HCN<inline-formula><mml:math id="inf434"><mml:msub><mml:mi/><mml:mrow><mml:mi mathbackground="white" mathsize="125%">O</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">L</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">M</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td>Hyperpol.-act, Cyclic Nucleo.-gated for O-LM cells</td><td>Q-O</td><td/><td/><td/><td/><td/><td/><td><inline-formula><mml:math id="inf435"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/></tr><tr><td>HCN<inline-formula><mml:math id="inf436"><mml:msub><mml:mi/><mml:mi mathbackground="white" mathsize="125%">p</mml:mi></mml:msub></mml:math></inline-formula></td><td>Hyperpol.-act, Cyclic Nucleo.-gated for Pyr. cells</td><td>Q-O</td><td><inline-formula><mml:math id="inf437"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>K<inline-formula><mml:math id="inf438"><mml:msub><mml:mi/><mml:mrow><mml:mrow id="XM19"><mml:mi mathbackground="white" mathsize="125%">C</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">a</mml:mi></mml:mrow><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mi id="XM20" mathbackground="white" mathsize="125%">S</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td>Small (SK) Calcium-activated potassium</td><td>Q-O</td><td/><td><inline-formula><mml:math id="inf439"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf440"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf441"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf442"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf443"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td><inline-formula><mml:math id="inf444"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf445"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td></tr><tr><td>K<inline-formula><mml:math id="inf446"><mml:msub><mml:mi/><mml:mrow><mml:mrow id="XM21"><mml:mi mathbackground="white" mathsize="125%">d</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">r</mml:mi></mml:mrow><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mrow id="XM22"><mml:mi mathbackground="white" mathsize="125%">f</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">a</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">s</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">t</mml:mi></mml:mrow></mml:mrow></mml:msub></mml:math></inline-formula></td><td>Fast delayed rectifier potassium</td><td>Q-O</td><td/><td><inline-formula><mml:math id="inf447"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf448"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf449"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td><inline-formula><mml:math id="inf450"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf451"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf452"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td></tr><tr><td>K<inline-formula><mml:math id="inf453"><mml:msub><mml:mi/><mml:mrow><mml:mrow id="XM23"><mml:mi mathbackground="white" mathsize="125%">d</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">r</mml:mi></mml:mrow><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mrow id="XM24"><mml:mi mathbackground="white" mathsize="125%">f</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">a</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">s</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">t</mml:mi></mml:mrow><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mrow id="XM25"><mml:mi mathbackground="white" mathsize="125%">n</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">g</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">f</mml:mi></mml:mrow></mml:mrow></mml:msub></mml:math></inline-formula></td><td>Fast delayed rectifier potassium for NGF-family cells</td><td>Q-O</td><td/><td/><td/><td/><td><inline-formula><mml:math id="inf454"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf455"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/></tr><tr><td>K<inline-formula><mml:math id="inf456"><mml:msub><mml:mi/><mml:mrow><mml:mrow id="XM26"><mml:mi mathbackground="white" mathsize="125%">d</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">r</mml:mi></mml:mrow><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mi id="XM27" mathbackground="white" mathsize="125%">p</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td>Delayed rectifier potassium for Pyr. cells</td><td>Q-O</td><td><inline-formula><mml:math id="inf457"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>K<inline-formula><mml:math id="inf458"><mml:msub><mml:mi/><mml:mrow><mml:mi id="XM28" mathbackground="white" mathsize="125%">v</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mi id="XM29" mathbackground="white" mathsize="125%">A</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td>A-type voltage-act. potassium</td><td>Q-O</td><td/><td><inline-formula><mml:math id="inf459"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf460"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf461"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/><td><inline-formula><mml:math id="inf462"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf463"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td></tr><tr><td>K<inline-formula><mml:math id="inf464"><mml:msub><mml:mi/><mml:mrow><mml:mi id="XM30" mathbackground="white" mathsize="125%">v</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mi id="XM31" mathbackground="white" mathsize="125%">A</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mrow id="XM32"><mml:mi mathbackground="white" mathsize="125%">d</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">i</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">s</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">t</mml:mi></mml:mrow><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mi id="XM33" mathbackground="white" mathsize="125%">p</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td>A-type voltage-act. potassium for dist. Pyr. dendrites</td><td>Q-O</td><td><inline-formula><mml:math id="inf465"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>K<inline-formula><mml:math id="inf466"><mml:msub><mml:mi/><mml:mrow><mml:mi id="XM34" mathbackground="white" mathsize="125%">v</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mi id="XM35" mathbackground="white" mathsize="125%">A</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mrow id="XM36"><mml:mi mathbackground="white" mathsize="125%">n</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">g</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">f</mml:mi></mml:mrow></mml:mrow></mml:msub></mml:math></inline-formula></td><td>A-type voltage-act. potassium for NGF-family cells</td><td>Q-O</td><td/><td/><td/><td/><td><inline-formula><mml:math id="inf467"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf468"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/></tr><tr><td>K<inline-formula><mml:math id="inf469"><mml:msub><mml:mi/><mml:mrow><mml:mi id="XM37" mathbackground="white" mathsize="125%">v</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mi id="XM38" mathbackground="white" mathsize="125%">A</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mrow id="XM39"><mml:mi mathbackground="white" mathsize="125%">o</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">l</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">m</mml:mi></mml:mrow></mml:mrow></mml:msub></mml:math></inline-formula></td><td>A-type voltage-act. potassium for O-LM cells</td><td>Q-O</td><td/><td/><td/><td/><td/><td/><td><inline-formula><mml:math id="inf470"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/></tr><tr><td>K<inline-formula><mml:math id="inf471"><mml:msub><mml:mi/><mml:mrow><mml:mi id="XM40" mathbackground="white" mathsize="125%">v</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mi id="XM41" mathbackground="white" mathsize="125%">A</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mrow id="XM42"><mml:mi mathbackground="white" mathsize="125%">p</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">r</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">o</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">x</mml:mi></mml:mrow><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mi id="XM43" mathbackground="white" mathsize="125%">p</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td>A-type voltage-act. potassium for prox. Pyr. dendrites</td><td>Q-O</td><td><inline-formula><mml:math id="inf472"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>K<inline-formula><mml:math id="inf473"><mml:msub><mml:mi/><mml:mrow><mml:mi id="XM44" mathbackground="white" mathsize="125%">v</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mrow id="XM45"><mml:mi mathbackground="white" mathsize="125%">C</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">a</mml:mi></mml:mrow><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mi id="XM46" mathbackground="white" mathsize="125%">B</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td>Big (BK) Calcium-act., voltage-dep. potassium</td><td>Q-O</td><td/><td><inline-formula><mml:math id="inf474"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf475"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf476"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf477"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf478"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td><inline-formula><mml:math id="inf479"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf480"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td></tr><tr><td>K<inline-formula><mml:math id="inf481"><mml:msub><mml:mi/><mml:mrow><mml:mi id="XM47" mathbackground="white" mathsize="125%">v</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mrow id="XM48"><mml:mi mathbackground="white" mathsize="125%">G</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">r</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">o</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">u</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">p</mml:mi></mml:mrow></mml:mrow></mml:msub></mml:math></inline-formula></td><td>Multiple slower voltage-dep. potassium</td><td>Q-O</td><td/><td/><td/><td><inline-formula><mml:math id="inf482"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/><td/><td><inline-formula><mml:math id="inf483"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td></tr><tr><td>leak</td><td>Leak</td><td>Q-O</td><td/><td><inline-formula><mml:math id="inf484"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf485"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf486"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf487"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf488"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf489"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf490"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf491"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td></tr><tr><td>Na<inline-formula><mml:math id="inf492"><mml:msub><mml:mi/><mml:mi mathbackground="white" mathsize="125%">v</mml:mi></mml:msub></mml:math></inline-formula></td><td>Voltage-dep. sodium</td><td>Q-O</td><td/><td><inline-formula><mml:math id="inf493"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/><td><inline-formula><mml:math id="inf494"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td><inline-formula><mml:math id="inf495"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td/></tr><tr><td>Na<inline-formula><mml:math id="inf496"><mml:msub><mml:mi/><mml:mrow><mml:mi id="XM49" mathbackground="white" mathsize="125%">v</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mrow id="XM50"><mml:mi mathbackground="white" mathsize="125%">b</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">i</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">s</mml:mi></mml:mrow></mml:mrow></mml:msub></mml:math></inline-formula></td><td>Voltage-dep. sodium for bistratified cells</td><td>Q-O</td><td/><td/><td><inline-formula><mml:math id="inf497"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/><td/><td/><td/></tr><tr><td>Na<inline-formula><mml:math id="inf498"><mml:msub><mml:mi/><mml:mrow><mml:mi id="XM51" mathbackground="white" mathsize="125%">v</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mrow id="XM52"><mml:mi mathbackground="white" mathsize="125%">c</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">c</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">k</mml:mi></mml:mrow></mml:mrow></mml:msub></mml:math></inline-formula></td><td>Voltage-dep. sodium for CCK+ cells</td><td>Q-O</td><td/><td/><td/><td><inline-formula><mml:math id="inf499"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/><td/><td><inline-formula><mml:math id="inf500"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td></tr><tr><td>Na<inline-formula><mml:math id="inf501"><mml:msub><mml:mi/><mml:mrow><mml:mi id="XM53" mathbackground="white" mathsize="125%">v</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mrow id="XM54"><mml:mi mathbackground="white" mathsize="125%">n</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">g</mml:mi><mml:mo mathbackground="white" mathsize="80%">&#8290;</mml:mo><mml:mi mathbackground="white" mathsize="125%">f</mml:mi></mml:mrow></mml:mrow></mml:msub></mml:math></inline-formula></td><td>Voltage-dep. sodium for NGF-family cells</td><td>Q-O</td><td/><td/><td/><td/><td><inline-formula><mml:math id="inf502"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf503"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/></tr><tr><td>Na<inline-formula><mml:math id="inf504"><mml:msub><mml:mi/><mml:mrow><mml:mi id="XM55" mathbackground="white" mathsize="125%">v</mml:mi><mml:mo mathbackground="white" mathsize="125%">,</mml:mo><mml:mi id="XM56" mathbackground="white" mathsize="125%">p</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td>Voltage-dep. sodium for Pyr. cells</td><td>Q-O</td><td><inline-formula><mml:math id="inf505"><mml:mo mathbackground="#FFFFFF">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>pas</td><td>Leak</td><td>Q-O</td><td><inline-formula><mml:math id="inf506"><mml:mo mathbackground="#ECECEC">&#8729;</mml:mo></mml:math></inline-formula></td><td/><td/><td/><td/><td/><td/><td/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18566-v2.xml">elife-18566-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="Atbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.18566.069</object-id><label>Appendix 1&#8212;table 2.</label><caption><p>AxoClamp raw data files.&#160;Sch. Coll.-Assoc.: Schaffer Collateral-Associated; Super: superficial. Current sweep injection levels are reported as&#160;minimum (most hyperpolarized) : step size : maximum (depolarized) level in units of pA.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18566.069">http://dx.doi.org/10.7554/eLife.18566.069</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Cell type</th><th rowspan="2">Lab</th><th rowspan="2">Cell name</th><th colspan="2">Current inj.</th><th rowspan="2">Original use and methods reference</th></tr><tr><th>Species</th><th>Levels (pA)</th></tr></thead><tbody><tr><td>Axo-axonic</td><td>Soltesz</td><td>CA203LF57</td><td>mouse</td><td>&#8722;200:50:+500</td><td>unpublished</td></tr><tr><td>Axo-axonic</td><td>Soltesz</td><td>CA204LF59</td><td>mouse</td><td>&#8722;200:50:+300</td><td>unpublished</td></tr><tr><td>Axo-axonic</td><td>Soltesz</td><td>CA204RF59</td><td>mouse</td><td>&#8722;200:50:+400</td><td>unpublished</td></tr><tr><td>Bistratified</td><td>Soltesz</td><td>PV16IM</td><td>mouse</td><td>&#8722;300:50:+400</td><td>unpublished</td></tr><tr><td>Bistratified</td><td>Soltesz</td><td>PV74</td><td>mouse</td><td>&#8722;300:50:+350</td><td>unpublished</td></tr><tr><td>Bistratified</td><td>Soltesz</td><td>PV27IM</td><td>mouse</td><td>&#8722;300:50:+450</td><td>unpublished</td></tr><tr><td>PV+ Basket</td><td>Soltesz</td><td>PV34</td><td>mouse</td><td>&#8722;300:50:+500</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>PV+ Basket</td><td>Soltesz</td><td>PV36</td><td>mouse</td><td>&#8722;300:50:+800</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>PV+ Basket</td><td>Soltesz</td><td>PV37</td><td>mouse</td><td>&#8722;300:50:+500</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>PV+ Basket</td><td>Soltesz</td><td>PV38</td><td>mouse</td><td>&#8722;300:50:+300</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>PV+ Basket</td><td>Soltesz</td><td>PV72</td><td>mouse</td><td>&#8722;300:50:+400</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>PV+ Basket</td><td>Soltesz</td><td>PV80</td><td>mouse</td><td>&#8722;300:50:+450</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Deep Pyramidal</td><td>Soltesz</td><td>D1_25abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Deep Pyramidal</td><td>Soltesz</td><td>D1_45abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Deep Pyramidal</td><td>Soltesz</td><td>D2_06abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Deep Pyramidal</td><td>Soltesz</td><td>D2_49abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Deep Pyramidal</td><td>Soltesz</td><td>D3_55abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Deep Pyramidal</td><td>Soltesz</td><td>D4_11abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Deep Pyramidal</td><td>Soltesz</td><td>D5_15abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Deep Pyramidal</td><td>Soltesz</td><td>D6_19abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Deep Pyramidal</td><td>Soltesz</td><td>D7</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Super. Pyramidal</td><td>Soltesz</td><td>S1_04abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Super. Pyramidal</td><td>Soltesz</td><td>S1_47abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Super. Pyramidal</td><td>Soltesz</td><td>S2_08abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Super. Pyramidal</td><td>Soltesz</td><td>S2_31abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Super. Pyramidal</td><td>Soltesz</td><td>S2_51abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Super. Pyramidal</td><td>Soltesz</td><td>S3_13abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Super. Pyramidal</td><td>Soltesz</td><td>S4</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Super. Pyramidal</td><td>Soltesz</td><td>S5_21abf</td><td>mouse</td><td>&#8722;400:50:+550</td><td><xref ref-type="bibr" rid="bib89">Lee et al. (2014)</xref></td></tr><tr><td>Ivy</td><td>Soltesz</td><td>0422&#8211;1 (File 5)</td><td>rat</td><td>&#8722;100:20:+890</td><td><xref ref-type="bibr" rid="bib82">Krook-Magnuson et al. (2011)</xref></td></tr><tr><td>Ivy</td><td>Soltesz</td><td>0428&#8211;1 (File 4)</td><td>rat</td><td>&#8722;100:20:+300</td><td><xref ref-type="bibr" rid="bib82">Krook-Magnuson et al. (2011)</xref></td></tr><tr><td>Neurogliaform</td><td>Soltesz</td><td>09o21 (File 4)</td><td>rat</td><td>&#8722;100:20:+120</td><td><xref ref-type="bibr" rid="bib82">Krook-Magnuson et al. (2011)</xref></td></tr><tr><td>Neurogliaform</td><td>Soltesz</td><td>09o27 (File 7)</td><td>rat</td><td>&#8722;100:20:+490</td><td><xref ref-type="bibr" rid="bib82">Krook-Magnuson et al. (2011)</xref></td></tr><tr><td>CCK+ Basket</td><td>Soltesz</td><td>sh108_BC</td><td>rat</td><td>&#8722;100:20:+80</td><td><xref ref-type="bibr" rid="bib88">Lee et al. (2010)</xref></td></tr><tr><td>Sch. Coll.-Assoc.</td><td>Soltesz</td><td>sh114_SCA</td><td>rat</td><td>&#8722;100:20:+60</td><td><xref ref-type="bibr" rid="bib88">Lee et al. (2010)</xref></td></tr><tr><td>Sch. Coll.-Assoc.</td><td>Soltesz</td><td>sh153_SCA</td><td>rat</td><td>&#8722;100:20:+60</td><td><xref ref-type="bibr" rid="bib88">Lee et al. (2010)</xref></td></tr><tr><td>O-LM</td><td>Maccaferri</td><td>1May2012_P3</td><td>mouse</td><td>&#8722;100:30:+250</td><td><xref ref-type="bibr" rid="bib118">Quattrocolo and Maccaferri (2013)</xref></td></tr><tr><td>O-LM</td><td>Maccaferri</td><td>20Sept2011_P2</td><td>mouse</td><td>&#8722;100:30:+250</td><td><xref ref-type="bibr" rid="bib118">Quattrocolo and Maccaferri (2013)</xref></td></tr><tr><td>O-LM</td><td>Maccaferri</td><td>24October2012_C2</td><td>mouse</td><td>&#8722;100:30:+250</td><td><xref ref-type="bibr" rid="bib118">Quattrocolo and Maccaferri (2013)</xref></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18659-v1.xml">elife-18659-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.18659.007</object-id><label>Table 1.</label><caption><p>Measured and calculated values (&#181;m) for the FWHM of the PSF in laser scanning confocal mode and wide-field epi-fluorescence mode for a conventional 5x/0.15 N.A. lens and the 4x/0.47 N.A. Mesolens used with oil immersion. We have assumed emission &#955;=550 nm for the calculations.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18659.007">http://dx.doi.org/10.7554/eLife.18659.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="2" valign="top">5x/0.15 N.A.</th><th colspan="2" valign="top">Mesolens (4x/0.47 N.A.)</th></tr><tr><th valign="top">Theoretical</th><th valign="top">Measured</th><th valign="top">Theoretical</th><th valign="top">Measured</th></tr></thead><tbody><tr><td valign="top">XY FWHM, epi</td><td valign="top">1.9</td><td valign="top">1.9</td><td valign="top">0.6</td><td valign="top">0.7</td></tr><tr><td valign="top">Z FWHM, epi</td><td valign="top">41</td><td valign="top">51</td><td valign="top">6.3</td><td valign="top">7</td></tr><tr><td valign="top">XY FWHM, confocal</td><td valign="top">1.8</td><td valign="top">1.7</td><td valign="top">0.6</td><td valign="top">0.8</td></tr><tr><td valign="top">Z FWHM, confocal</td><td valign="top">25</td><td valign="top">38</td><td valign="top">3.7</td><td valign="top">8</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-18972-v3.xml">elife-18972-v3.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.18972.006</object-id><label>Table 1.</label><caption><p>Determination of the Fe/EncFtn<sub>sH</sub> protein ratio by ICP-MS. EncFtn<sub>sH</sub> was purified as a SeMet derivative from <italic>E. coli</italic> B834(DE3) cells grown in SeMet medium with 1 mM Fe(NH<sub>4</sub>)<sub>2</sub>(SO<sub>4</sub>)<sub>2</sub>. Fractions from SEC were collected, acidified and analysed by ICP-MS. EncFtn<sub>sH</sub> concentration was calculated based on the presence of two SeMet per mature monomer. Samples where the element was undetectable are labelled with n.d. These data were collected from EncFtn<sub>sH</sub> fractions from a single gel-filtration run.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.18972.006">http://dx.doi.org/10.7554/eLife.18972.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Peak</th><th rowspan="2">EncFtn<sub>sH</sub> <break/>retention volume (ml)</th><th colspan="4">Element concentration (&#181;M)</th><th rowspan="2">Derived EncFtn<sub>sH</sub>concentration (&#181;M)</th><th rowspan="2">Derived Fe/ <break/>EncFtn<sub>sH</sub> monomer</th></tr><tr><th>Ca</th><th>Fe</th><th>Zn</th><th>Se</th></tr></thead><tbody><tr><td rowspan="8">Decamer</td><td>66.5</td><td>n.d.</td><td>6.7</td><td>n.d.</td><td>24.6</td><td><p>12.3</p></td><td><p>0.5</p></td></tr><tr><td>68.3</td><td>n.d.</td><td>28.4</td><td>n.d</td><td>124.5</td><td><p>62.3</p></td><td><p>0.5</p></td></tr><tr><td>70.1</td><td>2.9</td><td>93.7</td><td>2.4</td><td>301.7</td><td><p>150.9</p></td><td><p>0.6</p></td></tr><tr><td>71.9</td><td>6.9</td><td>120.6</td><td>3.7</td><td>379.8</td><td><p>189.9</p></td><td><p>0.6</p></td></tr><tr><td>73.7</td><td>1.9</td><td>64.4</td><td>0.8</td><td>240.6</td><td><p>120.3</p></td><td><p>0.5</p></td></tr><tr><td>75.5</td><td>0.9</td><td>21.1</td><td>n.d.</td><td>101.7</td><td><p>50.8</p></td><td><p>0.4</p></td></tr><tr><td>77.3</td><td>n.d.</td><td>6.2</td><td>n.d.</td><td>42.6</td><td><p>21.3</p></td><td><p>0.3</p></td></tr><tr><td>79.1</td><td>0.1</td><td>2.4</td><td>n.d.</td><td>26.5</td><td><p>13.3</p></td><td><p>0.2</p></td></tr><tr><td/><td>80.9</td><td>1.0</td><td>1.5</td><td>n.d.</td><td>22.3</td><td><p>11.2</p></td><td><p>0.1</p></td></tr><tr><td/><td>82.7</td><td>n.d.</td><td>0.2</td><td>n.d.</td><td>29.2</td><td><p>14.6</p></td><td><p>n.d</p></td></tr><tr><td rowspan="5">Monomer</td><td>84.5</td><td>n.d.</td><td>0.1</td><td>n.d.</td><td>34.9</td><td><p>17.5</p></td><td><p>n.d</p></td></tr><tr><td>86.3</td><td>n.d.</td><td>n.d</td><td>n.d.</td><td>28.9</td><td><p>14.4</p></td><td><p>n.d</p></td></tr><tr><td>88.1</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td>17.4</td><td><p>8.7</p></td><td><p>n.d.</p></td></tr><tr><td>89.9</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td>5.5</td><td><p>2.8</p></td><td><p>n.d.</p></td></tr><tr><td>91.7</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td>0.1</td><td><p>0.07</p></td><td><p>0.2</p></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-19089-v2.xml">elife-19089-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.19089.005</object-id><label>Table 1.</label><caption><p>Patient details and movement-related modulated in beta and gamma bands.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.19089.005">http://dx.doi.org/10.7554/eLife.19089.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"><p>Patient ID</p></th><th rowspan="3" valign="top"><p>Age (yrs)</p></th><th rowspan="3" valign="top"><p>Gender</p></th><th rowspan="3" valign="top"><p>PD duration (yrs)</p></th><th rowspan="3" valign="top"><p>Main symptoms</p> </th><th rowspan="3" valign="top"><p>Daily dose (mg)</p></th><th colspan="2" rowspan="2" valign="top"><p>UPDRS part III (Pre-op)</p></th><th colspan="12" valign="top"><p>Movement related power change (%) and electrode localisation</p></th><th rowspan="3"><p>Stimulation effect</p> </th></tr><tr><th colspan="6" valign="top"><p>Left STN</p></th><th colspan="6" valign="top"><p>Right STN</p></th></tr><tr><th valign="top"><p>OFF</p></th><th valign="top"><p>ON</p></th><th valign="top"><p>Bipoloar channel</p> </th><th valign="top"><p>Beta</p><p>ERD</p></th><th valign="top"><p>Gamma ERS</p></th><th valign="top"><p>Localisation</p></th><th valign="top"><p>Most %beta</p> </th><th valign="top"><p>Stim setting</p> </th><th valign="top"><p>Bipoloar channel</p> </th><th valign="top"><p>Beta</p><p>ERD</p></th><th valign="top"><p>Gamma ERS</p></th><th valign="top"><p>Localisation</p></th><th valign="top"><p>Most % beta</p> </th><th valign="top"><p>Stim setting</p> </th></tr></thead><tbody><tr><td valign="top"><p>1 QS</p></td><td valign="top"><p>49</p></td><td valign="top"><p>M</p></td><td valign="top"><p>13</p></td><td valign="top"><p>Stiffness, bradykinesia, bilateral tremor, freezing</p></td><td valign="top"><p>Levodopa 800</p><p>Apomorphine (6.5 mg/hour)</p><p>Rotigotine 8</p></td><td valign="top"><p>38</p></td><td valign="top"><p>13</p></td><td valign="top"><p>L1L2</p></td><td valign="top"><p>&#8722;4.52</p> <p>(*)</p></td><td valign="top"><p>17.99</p><p>(*)</p></td><td valign="top"><p>L1,L2 border</p><p>MED</p></td><td valign="top"><p>L1</p></td><td valign="top"><p>Case: + L1: -</p></td><td valign="top"><p>R1R2</p></td><td valign="top"><p>1.15</p></td><td valign="top"><p>33.2</p><p>(*)</p></td><td valign="top"><p>R0:inside/border/MED; R1: border/MED</p></td><td valign="top"><p>R0</p></td><td valign="top"><p>Case: + R1: -</p></td><td valign="top"><p>UPDRS OFF Med, Stim ON/OFF: 13/38</p></td></tr><tr><td valign="top"><p>2 Ox</p></td><td valign="top"><p>69</p></td><td valign="top"><p>M</p></td><td valign="top"><p>11</p></td><td valign="top"><p>Rigidity, bradykinesia, freezing</p></td><td valign="top"><p>Ropinirole 8</p><p>Pramipexole 0.75</p><p>Levodopa 900</p></td><td valign="top"><p>38</p></td><td valign="top"><p>18</p></td><td valign="top"/><td valign="top"><p>--</p></td><td valign="top"><p>--</p></td><td valign="top"><p>Electrode was not in target and therefore not recorded</p></td><td valign="top"/><td valign="top"/><td valign="top"><p>R0R1</p></td><td valign="top"><p>&#8722;4.79</p> </td><td valign="top"><p>0.92</p></td><td valign="top"><p>R0,R1 inside only</p></td><td valign="top"><p>R1</p></td><td valign="top"><p>None</p></td><td valign="top"><p>Stimulation was discontinued shortly after surgery due to unsatisfactory clinical effect</p></td></tr><tr><td valign="top"><p>3 King</p></td><td valign="top"><p>65</p></td><td valign="top"><p>F</p></td><td valign="top"><p>17</p></td><td valign="top"><p>Rigidity, tremor</p></td><td valign="top"><p>Amantadine 400 Levodopa 600</p></td><td valign="top"><p>55</p></td><td valign="top"><p>49</p></td><td valign="top"><p>L0L1</p></td><td valign="top"><p>&#8722;5.12 (*)</p></td><td valign="top"><p>7.44</p><p>(*)</p></td><td valign="top"><p>All inside</p></td><td valign="top"><p>L1</p></td><td valign="top"><p>Case: +</p><p>L0: -</p></td><td valign="top"><p>R0R1</p></td><td valign="top"><p>&#8722;29.03</p> <p>(*)</p></td><td valign="top"><p>3.29</p><p>(*)</p></td><td valign="top"><p>All inside</p></td><td valign="top"><p>R1</p></td><td valign="top"><p>Case: +</p><p>R1: -</p></td><td valign="top"><p>Not evaluated</p></td></tr><tr><td valign="top"><p>4 QS</p></td><td valign="top"><p>56</p></td><td valign="top"><p>M</p></td><td valign="top"><p>10</p></td><td valign="top"><p>Bradykinesia, rigidity, tremor limping gait</p></td><td valign="top"><p>Levodopa 1000 Rasagiline 1</p><p>Citalopram 20</p></td><td valign="top"><p>40</p></td><td valign="top"><p>12</p></td><td valign="top"><p>L1L2</p></td><td valign="top"><p>&#8722;2.48</p> <p>(*)</p></td><td valign="top"><p>37.73</p><p>(*)</p></td><td valign="top"><p>L1,L2,L3 inside; L2 dorsolat</p></td><td valign="top"><p>L2</p></td><td valign="top"><p>Case: + L1: -</p></td><td valign="top"><p>R0R1</p></td><td valign="top"><p>&#8722;18.92 (*)</p></td><td valign="top"><p>2.98</p><p>(*)</p></td><td valign="top"><p>R0 inside; R1 border/dorsolat</p></td><td valign="top"><p>R1</p></td><td valign="top"><p>Case: + R0, R1 (alternating): -</p></td><td valign="top"><p>UPDRS OFF Med, Stim ON/OFF: 29/40</p></td></tr><tr><td valign="top"><p>5 QS</p></td><td valign="top"><p>60</p></td><td valign="top"><p>F</p></td><td valign="top"><p>11</p></td><td valign="top"><p>Tremor@Left; poor coordination, bended gait</p></td><td valign="top"><p>Levodopa 600 Pramipexole 0.75</p></td><td valign="top"><p>53</p></td><td valign="top"><p>16</p></td><td valign="top"><p>L1L2</p></td><td valign="top"><p>&#8722;4.94</p> <p>(*)</p></td><td valign="top"><p>6.68</p><p>(*)</p></td><td valign="top"><p>All inside; L1 dorsolat</p></td><td valign="top"><p>L1</p></td><td valign="top"><p>Case: + L1: -</p></td><td valign="top"><p>R2R3</p></td><td valign="top"><p>&#8722;0.049</p> </td><td valign="top"><p>2.14</p></td><td valign="top"><p>R1 inside; R2 border</p></td><td valign="top"><p>R2</p></td><td valign="top"><p>Case: + R1: -</p></td><td valign="top"><p>Not evaluated</p></td></tr><tr><td valign="top"><p>6 Kings</p></td><td valign="top"><p>65</p></td><td valign="top"><p>M</p></td><td valign="top"><p>5</p></td><td valign="top"><p>Rigidity, bradykinesia, motor fluctuation, tremor</p></td><td valign="top"><p>Levodopa 400</p><p>Entacapone 800 Rotigotine 8</p></td><td valign="top"><p>41</p></td><td valign="top"><p>29</p></td><td valign="top"><p>L1L2</p></td><td valign="top"><p>3.33</p></td><td valign="top"><p>1.81</p></td><td valign="top"><p>All inside</p></td><td valign="top"><p>None</p></td><td valign="top"><p>Case +</p><p>L2: -</p></td><td valign="top"><p>R1R2</p></td><td valign="top"><p>&#8722;7.37</p> <p>(*)</p></td><td valign="top"><p>0.66</p></td><td valign="top"><p>All inside</p></td><td valign="top"><p>R2</p></td><td valign="top"><p>Case +</p><p>R1: -</p></td><td valign="top"><p>Not evaluated</p></td></tr><tr><td valign="top"><p>7 QS</p></td><td valign="top"><p>56</p></td><td valign="top"><p>M</p></td><td valign="top"><p>10</p></td><td valign="top"><p>tremor@all four limbs</p></td><td valign="top"><p>Levodopa 600 Rotigotine 8 Selegiline 10</p></td><td valign="top"><p>52</p></td><td valign="top"><p>19</p></td><td valign="top"><p>L0L1</p></td><td valign="top"><p>&#8722;10.68</p> <p>(*)</p></td><td valign="top"><p>8.42</p><p>(*)</p></td><td valign="top"><p>L2, L3 in superior STN</p></td><td valign="top"><p>L0</p></td><td valign="top"><p>Case: + L1: -</p></td><td valign="top"><p>R0R1</p></td><td valign="top"><p>&#8722;22.76</p> <p>(*)</p></td><td valign="top"><p>14.02</p><p>(*)</p></td><td valign="top"><p>R0, R1 in STN, R2 lateral border of superior STN</p></td><td valign="top"><p>R1</p></td><td valign="top"><p>Case +</p><p>R1: -</p></td><td valign="top"><p>Relocation after recording due to side effects on speech</p></td></tr><tr><td valign="top"><p>8 Kings</p></td><td valign="top"><p>73</p></td><td valign="top"><p>M</p></td><td valign="top"><p>14</p></td><td valign="top"><p>Bradykinesia, tremor</p></td><td valign="top"><p>Rotigotine 16 Selegeline 10</p><p>Levodopa 700</p></td><td valign="top"><p>35</p></td><td valign="top"><p>15</p></td><td valign="top"><p>L0L1</p></td><td valign="top"><p>0.157</p></td><td valign="top"><p>&#8722;0.186</p> </td><td valign="top"><p>All inside</p></td><td valign="top"><p>None</p></td><td valign="top"><p>Case: +</p><p>L1: -</p></td><td valign="top"><p>R1R2</p></td><td valign="top"><p>&#8722;4.93 (*)</p></td><td valign="top"><p>5.57</p><p>(*)</p></td><td valign="top"><p>All inside</p></td><td valign="top"><p>R1</p></td><td valign="top"><p>Case: +</p><p>R1: -</p></td><td valign="top"><p>Not evaluated</p></td></tr><tr><td valign="top"><p>9 Ox</p></td><td valign="top"><p>63</p></td><td valign="top"><p>F</p></td><td valign="top"><p>14</p></td><td valign="top"><p>Rigidity, bradykinesia</p></td><td valign="top"><p>Ropinirole 23</p><p>Levodopa 150</p></td><td valign="top"><p>35</p></td><td valign="top"><p>24</p></td><td valign="top"/><td valign="top"><p>3.197</p></td><td valign="top"><p>&#8722;1.14</p> </td><td valign="top"><p>None inside</p></td><td valign="top"><p>None</p></td><td valign="top"><p>None</p></td><td valign="top"/><td valign="top"><p>&#8722;2.59</p> </td><td valign="top"><p>7.20</p></td><td valign="top"><p>None inside</p></td><td valign="top"><p>None</p></td><td valign="top"><p>None</p></td><td valign="top"><p>Post-op imaging show mis-location, and electrodes were relocated to GPi</p></td></tr><tr><td valign="top"><p>10 QS</p></td><td valign="top"><p>66</p></td><td valign="top"><p>F</p></td><td valign="top"><p>16</p></td><td valign="top"><p>Shuffle, poor balance, NO tremor</p></td><td valign="top"><p>Levodopa 600</p><p>Amantadine 200</p><p>Ropinirole 24</p><p>Rasagiline 1</p></td><td valign="top"><p>32</p></td><td valign="top"><p>13</p></td><td valign="top"><p>L0L2(L1 no signal)</p></td><td valign="top"><p>4.33</p></td><td valign="top"><p>2.95</p></td><td valign="top"><p>L0,L1 inside</p></td><td valign="top"><p>L0</p></td><td valign="top"><p>Case: +</p><p>L1: -</p></td><td valign="top"><p>R0R1</p><p>(bipolar reduced modulation)</p></td><td valign="top"><p>&#8722;1.37</p> </td><td valign="top"><p>7.41</p><p>(*)</p></td><td valign="top"><p>R1,R2 inside</p></td><td valign="top"><p>R1</p></td><td valign="top"><p>Case: +</p><p>R1: -</p></td><td valign="top"><p>UPDRS OFF Med, Stim ON/OFF: 26/32</p></td></tr><tr><td valign="top"><p>11 QS</p></td><td valign="top"><p>52</p></td><td valign="top"><p>M</p></td><td valign="top"><p>7</p></td><td valign="top"><p>Freezing, falls, postural instability, tremor@right side</p></td><td valign="top"><p>Levodopa 1300</p><p>Citalopram 20 Trihexyphenidyl 6</p></td><td valign="top"><p>58</p></td><td valign="top"><p>13</p></td><td valign="top"><p>L1L2</p><p>(bipolar reduced modulation)</p></td><td valign="top"><p>38.77</p></td><td valign="top"><p>13.22</p></td><td valign="top"/><td valign="top"><p>L2</p></td><td valign="top"><p>Case: +</p><p>L1: -</p></td><td valign="top"><p>R1R2</p><p>(bipolar reduced modulation)</p></td><td valign="top"><p>&#8722;1.05</p> </td><td valign="top"><p>1.11</p></td><td valign="top"/><td valign="top"><p>R1</p></td><td valign="top"><p>Case: +</p><p>R1: -</p></td><td valign="top"><p>Relocation after recording</p></td></tr><tr><td valign="top"><p>Mean</p></td><td valign="top"><p>61.3</p></td><td valign="top"/><td valign="top"><p>11.3</p></td><td valign="top"/><td valign="top"/><td valign="top"><p>43.4</p></td><td valign="top"><p>20.1</p></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>(*) Indicate significant movement-related modulation in the power of the activity of the specific frequency band; Ox, Kings, QS indicate the three neurosurgical centres where the data were recorded: Ox = John Radcliffe Hospital, University of Oxford; Kings = Department of Neurosurgery, Kings College Hospital, Kings College London; QS = Sobell Department of Motor Neuroscience and Movement Disorders, UCL Institute of Neurology.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-19130-v2.xml">elife-19130-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.19130.013</object-id><label>Table 2.</label><caption><p>Amino acid levels showing consistent positive correlation with longevity traits.</p><p>For the mammalian fibroblast dataset, the number of longevity traits (out of Maximum Lifespan; Female Time to Maturity; Maximum Lifespan Residual; and Female Time to Maturity Residual) with significant positive correlation with the amino acid levels at two different cut-offs (p value.robust&#160;&lt;&#160;0.01 and p value.robust&#160;&lt;&#160;0.05) are shown. For the primate and bird fibroblast dataset, the regression was performed using primate data only, bird data only, and the pooled data of both. The regression slope p value&#160;&lt;&#160;0.05 are in bold.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.19130.013">http://dx.doi.org/10.7554/eLife.19130.013</ext-link></p><p><supplementary-material id="SD3-data"><object-id pub-id-type="doi">10.7554/eLife.19130.014</object-id><label>Table 2&#8212;source data 1.</label><caption><title>Phylogenetic regression of metabolite levels against longevity traits.</title><p>Regression against (<bold>A</bold>) Adult Weight; (<bold>B</bold>) Maximum Lifespan; (<bold>C</bold>) Female Time to Maturity; (<bold>D</bold>) Maximum Lifespan Residual; and (<bold>E</bold>) Female Time to Maturity Residual. &#8216;coef.all&#8217;, &#8216;p value.all&#8217;, and &#8216;q value.all&#8217; refer to the regression slope, p value, and FDR-adjusted q value using all the species. &#8216;p value.robust&#8217; and &#8216;q value.robust&#8217; refer to the statistics after removing the potential outlier point. &#8216;p value.max&#8217; and &#8216;q value.max&#8217; refer to the maximal (least significant) regression p value and q value when each one of the species was left out, one at a time. Only genes with p value.robust&#160;&lt;&#160;0.01 and p value.max&#160;&lt;&#160;0.05 are shown. (<bold>F</bold>) Top hits identified by two or more longevity traits. The p value.robust against each of the four longevity traits (ML, FTM, MLres, and FTMres) as well as adult weight (AW) are shown. These metabolites were the input for pathway enrichment analysis. Pathway enrichment analysis of metabolites showing (<bold>G</bold>) positive and (<bold>H</bold>) negative correlation with longevity traits. Enrichment was performed based on hypergeometric statistics. (<bold>I</bold>) Top hits identified by two or more longevity traits, using cut-off of p value.robust&#160;&lt;&#160;0.05. The p value.robust against each of the four longevity traits (ML, FTM, MLres, and FTMres) as well as adult weight (AW) are shown.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.19130.014">http://dx.doi.org/10.7554/eLife.19130.014</ext-link></p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-19130-table2-data1-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"><p>Amino acid</p></th><th colspan="2" valign="top"><p>Mammalian fibroblasts</p> </th><th colspan="6" valign="top"><p>Primate and bird fibroblasts</p> </th></tr><tr><th colspan="2" valign="top"><p>No. of longevity traits (out of four) with significant correlation</p></th><th colspan="3" valign="top"><p>Regression slope p value with species maximum lifespan</p> </th><th colspan="3" valign="top"><p>Regression slope p value with species maximum lifespan residual</p> </th></tr><tr><th valign="top"><p>p value.robust&#160;&lt;&#160;0.01</p> </th><th valign="top"><p>p value.robust&#160;&lt;&#160;0.05</p> </th><th valign="top"><p>Primates only</p></th><th valign="top"><p>Birds only</p></th><th valign="top"><p>Primates and birds</p></th><th valign="top"><p>Primates only</p></th><th valign="top"><p>Birds only</p></th><th valign="top"><p>Primates and birds</p></th></tr></thead><tbody><tr><td valign="top"><p>arginine</p></td><td valign="top"><p>3</p></td><td valign="top"><p>4</p></td><td valign="bottom"><p><bold>3.4&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="bottom"><p>8.6&#160;&#215;&#160;10<sup>&#8722;2</sup></p></td><td valign="bottom"><p><bold>3.1&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="top"><p>3.8&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p><bold>1.1&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="top"><p><bold>2.1&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td></tr><tr><td valign="top"><p>glutamate</p></td><td valign="top"><p>2</p></td><td valign="top"><p>4</p></td><td valign="bottom"><p>6.5&#160;&#215;&#160;10<sup>&#8722;2</sup></p></td><td valign="bottom"><p><bold>1.8&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="bottom"><p><bold>1.1&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="top"><p><bold>4.6&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="top"><p>2.8&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p>1.3&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td></tr><tr><td valign="top"><p>histidine</p></td><td valign="top"><p>0</p></td><td valign="top"><p>4</p></td><td valign="bottom"><p>9.4&#160;&#215;&#160;10<sup>&#8722;2</sup></p></td><td valign="bottom"><p>6.0&#160;&#215;&#160;10<sup>&#8722;2</sup></p></td><td valign="bottom"><p><bold>4.3&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="top"><p>2.3&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p>1.4&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p>1.7&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td></tr><tr><td valign="top"><p>leucine</p></td><td valign="top"><p>2</p></td><td valign="top"><p>4</p></td><td valign="bottom"><p><bold>2.9&#160;&#215;&#160;10<sup>&#8722;3</sup></bold></p> </td><td valign="bottom"><p>6.0&#160;&#215;&#160;10<sup>&#8722;2</sup></p></td><td valign="bottom"><p><bold>4.8&#160;&#215;&#160;10<sup>&#8722;3</sup></bold></p> </td><td valign="top"><p><bold>1.4&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="top"><p>5.9&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p>2.3&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td></tr><tr><td valign="top"><p>lysine</p></td><td valign="top"><p>3</p></td><td valign="top"><p>3</p></td><td valign="bottom"><p><bold>9.8&#160;&#215;&#160;10<sup>&#8722;3</sup></bold></p> </td><td valign="bottom"><p>8.2&#160;&#215;&#160;10<sup>&#8722;2</sup></p></td><td valign="bottom"><p><bold>1.4&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="top"><p>9.1&#160;&#215;&#160;10<sup>&#8722;2</sup></p></td><td valign="top"><p>2.9&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p>2.5&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td></tr><tr><td valign="top"><p>methionine</p></td><td valign="top"><p>1</p></td><td valign="top"><p>3</p></td><td valign="bottom"><p>3.2&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="bottom"><p><bold>1.4&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="bottom"><p><bold>2.7&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="top"><p>3.0&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p><bold>3.0&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="top"><p><bold>4.9&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td></tr><tr><td valign="top"><p>phenylalanine</p></td><td valign="top"><p>1</p></td><td valign="top"><p>4</p></td><td valign="bottom"><p><bold>9.8&#160;&#215;&#160;10<sup>&#8722;3</sup></bold></p> </td><td valign="bottom"><p><bold>1.2&#160;&#215;&#160;10<sup>&#8722;3</sup></bold></p> </td><td valign="bottom"><p><bold>2.1&#160;&#215;&#160;10<sup>&#8722;4</sup></bold></p> </td><td valign="top"><p>8.2&#160;&#215;&#160;10<sup>&#8722;2</sup></p></td><td valign="top"><p>1.3&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p>1.2&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td></tr><tr><td valign="top"><p>proline</p></td><td valign="top"><p>1</p></td><td valign="top"><p>4</p></td><td valign="bottom"><p><bold>4.4&#160;&#215;&#160;10<sup>&#8722;3</sup></bold></p> </td><td valign="bottom"><p><bold>3.9&#160;&#215;&#160;10<sup>&#8722;4</sup></bold></p> </td><td valign="bottom"><p><bold>3.6&#160;&#215;&#160;10<sup>&#8722;5</sup></bold></p> </td><td valign="top"><p><bold>3.5&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="top"><p>1.2&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p>5.4&#160;&#215;&#160;10<sup>&#8722;2</sup></p></td></tr><tr><td valign="top"><p>tryptophan</p></td><td valign="top"><p>2</p></td><td valign="top"><p>4</p></td><td valign="bottom"><p><bold>9.2&#160;&#215;&#160;10<sup>&#8722;3</sup></bold></p> </td><td valign="bottom"><p><bold>7.8&#160;&#215;&#160;10<sup>&#8722;4</sup></bold></p> </td><td valign="bottom"><p><bold>1.2&#160;&#215;&#160;10<sup>&#8722;4</sup></bold></p> </td><td valign="top"><p><bold>2.6&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="top"><p>2.5&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p>1.5&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td></tr><tr><td valign="top"><p>tyrosine</p></td><td valign="top"><p>1</p></td><td valign="top"><p>3</p></td><td valign="bottom"><p>3.2&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="bottom"><p><bold>8.8&#160;&#215;&#160;10<sup>&#8722;3</sup></bold></p> </td><td valign="bottom"><p><bold>1.8&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="top"><p>4.3&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p>1.7&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p>2.9&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td></tr><tr><td valign="top"><p>valine</p></td><td valign="top"><p>0</p></td><td valign="top"><p>3</p></td><td valign="bottom"><p><bold>1.2&#160;&#215;&#160;10<sup>&#8722;2</sup></bold></p> </td><td valign="bottom"><p><bold>5.4&#160;&#215;&#160;10<sup>&#8722;3</sup></bold></p> </td><td valign="bottom"><p><bold>1.0&#160;&#215;&#160;10<sup>&#8722;3</sup></bold></p> </td><td valign="top"><p>2.0&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p>2.8&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td><td valign="top"><p>3.2&#160;&#215;&#160;10<sup>&#8722;1</sup></p></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-19469-v2.xml">elife-19469-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.19469.004</object-id><label>Table 1.</label><caption><p>Susceptibility of <italic>E</italic>. <italic>coli</italic> strains determined by the disk diffusion assay.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.19469.004">http://dx.doi.org/10.7554/eLife.19469.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><p>Antibiotic</p></th><th rowspan="2" valign="top"><p>Load (&#181;g)</p></th><th rowspan="2" valign="top"><p>BW25113</p></th><th colspan="5"><p>Diameter of inhibition zones (mm) for the indicated strains*</p></th></tr><tr><th valign="bottom"><p>BW25113&#916;4</p></th><th valign="bottom"><p>M1IPTG 50 &#181;M</p> </th><th valign="bottom"><p>M1<sub>cured</sub></p></th><th valign="bottom"><p>M1<sub>cured</sub>pJEH12(<italic>ycbB</italic>)</p></th><th valign="bottom"><p>M1<sub>cured</sub>pJEH12(<italic>ycbB</italic>)IPTG 50 &#181;M</p> </th></tr></thead><tbody><tr><td valign="bottom"><p>Amoxicillin</p></td><td><p>25</p></td><td><p>23</p></td><td><p>24</p></td><td><p>13</p></td><td><p>28</p></td><td><p>28</p></td><td><p>16</p></td></tr><tr><td valign="bottom"><p>Ampicillin</p> </td><td><p>10</p></td><td><p>21</p></td><td><p>21</p></td><td><p>ND<sup>&#167;</sup></p></td><td><p>27</p></td><td><p>26</p></td><td><p>ND</p></td></tr><tr><td valign="bottom"><p>Amox+Clav<sup>&#8224;</sup></p></td><td><p>20+10</p> </td><td><p>20</p></td><td><p>23</p></td><td><p>13</p></td><td><p>27</p></td><td><p>30</p></td><td><p>11</p></td></tr><tr><td valign="bottom"><p>Piperacillin</p></td><td><p>75</p></td><td><p>29</p></td><td><p>28</p></td><td><p>ND</p></td><td><p>36</p></td><td><p>36</p></td><td><p>ND</p></td></tr><tr><td valign="bottom"><p>Pip+Tazo<sup>&#8225;</sup></p></td><td><p>75+10</p> </td><td><p>29</p></td><td><p>30</p></td><td><p>ND</p></td><td><p>37</p></td><td><p>37</p></td><td><p>ND</p></td></tr><tr><td valign="bottom"><p>Ticarcillin</p></td><td><p>75</p></td><td><p>26</p></td><td><p>27</p></td><td><p>ND</p></td><td><p>37</p></td><td><p>31</p></td><td><p>ND</p></td></tr><tr><td valign="bottom"><p>Mecillinam</p></td><td><p>10</p></td><td><p>17</p></td><td><p>22</p></td><td><p>ND</p></td><td><p>ND</p></td><td><p>ND</p></td><td><p>ND</p></td></tr><tr><td valign="bottom"><p>Aztreonam</p></td><td><p>30</p></td><td><p>33</p></td><td><p>36</p></td><td><p>ND</p></td><td><p>49</p></td><td><p>47</p></td><td><p>ND</p></td></tr><tr><td valign="bottom"><p>Cefalotin</p></td><td><p>30</p></td><td><p>16</p></td><td><p>18</p></td><td><p>ND</p></td><td><p>23</p></td><td><p>23</p></td><td><p>ND</p></td></tr><tr><td valign="bottom"><p>Cefoxitin</p></td><td><p>30</p></td><td><p>20</p></td><td><p>25</p></td><td><p>29</p></td><td><p>30</p></td><td><p>30</p></td><td><p>30</p></td></tr><tr><td valign="bottom"><p>Cefotetan</p></td><td><p>30</p></td><td><p>30</p></td><td><p>31</p></td><td><p>25</p></td><td><p>41</p></td><td><p>41</p></td><td><p>27</p></td></tr><tr><td valign="bottom"><p>Ceftazidime</p></td><td><p>30</p></td><td><p>29</p></td><td><p>30</p></td><td><p>ND</p></td><td><p>39</p></td><td><p>37</p></td><td><p>9</p></td></tr><tr><td valign="bottom"><p>Cefotaxime</p></td><td><p>30</p></td><td><p>33</p></td><td><p>36</p></td><td><p>ND</p></td><td><p>44</p></td><td><p>44</p></td><td><p>9</p></td></tr><tr><td valign="bottom"><p>Cefixime</p></td><td><p>10</p></td><td><p>28</p></td><td><p>30</p></td><td><p>ND</p></td><td><p>37</p></td><td><p>38</p></td><td><p>ND</p></td></tr><tr><td valign="bottom"><p>Cefpirome</p></td><td><p>30</p></td><td><p>31</p></td><td><p>33</p></td><td><p>ND</p></td><td><p>41</p></td><td><p>41</p></td><td><p>9</p></td></tr><tr><td valign="bottom"><p>Cefoperazone</p></td><td><p>30</p></td><td><p>27</p></td><td><p>28</p></td><td><p>ND</p></td><td><p>41</p></td><td><p>40</p></td><td><p>ND</p></td></tr><tr><td valign="bottom"><p>Moxalactam</p></td><td><p>30</p></td><td><p>31</p></td><td><p>32</p></td><td><p>15</p></td><td><p>43</p></td><td><p>40</p></td><td><p>17</p></td></tr><tr><td valign="bottom"><p>Ceftriaxone</p> </td><td><p>30</p></td><td><p>33</p></td><td><p>32</p></td><td><p>15</p></td><td><p>42</p></td><td><p>42</p></td><td><p>18</p></td></tr><tr><td valign="bottom"><p><bold>Carbapenems</bold></p> </td><td/><td/><td/><td/><td/><td/><td/></tr><tr><td valign="bottom"><p>&#8195;Doripenem</p></td><td><p>10</p></td><td><p>31</p></td><td><p>34</p></td><td><p>32</p></td><td><p>38</p></td><td><p>37</p></td><td><p>38</p></td></tr><tr><td valign="bottom"><p>&#8195;Meropenem</p></td><td><p>10</p></td><td><p>30</p></td><td><p>34</p></td><td><p>35</p></td><td><p>40</p></td><td><p>41</p></td><td><p>34</p></td></tr><tr><td valign="bottom"><p>&#8195;Imipenem</p></td><td><p>10</p></td><td><p>26</p></td><td><p>30</p></td><td><p>35</p></td><td><p>25</p></td><td><p>27</p></td><td><p>28</p></td></tr><tr><td><p>&#8195;Ertapenem</p></td><td><p>10</p></td><td><p>30</p></td><td><p>35</p></td><td><p>37</p></td><td><p>49</p></td><td><p>47</p></td><td><p>37</p></td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>*BW25113&#916;4 is a derivative of <italic>E. coli</italic> BW25113 that does not harbor the <italic>ynhG</italic>, <italic>ybiS</italic>, <italic>erfK</italic>, and <italic>ycfS</italic> genes encoding YcbB paralogues. M1 is a &#946;-lactam-resistant mutant of BW25113&#916;4 harboring pJEH11-1(<italic>ycbB</italic>). M1<sub>cured</sub> is a derivative of M1 resulting from the spontaneous loss of pJEH11-1(<italic>ycbB</italic>). Plasmid pJEH12(<italic>ycbB</italic>) was obtained by replacing the origin of replication (ColE1) and resistance marker (kanamycin) of pJEH11-1(<italic>ycbB</italic>) by the p15A replication origin and tetracycline resistance marker of plasmid pACY184. The L,D-transpeptidase gene <italic>ycbB</italic> of pJEH11-1 and pJEH12 are expressed under the control of the IPTG-inducible <italic>trc</italic> promoter and regulated by the LacI Arg<sup>127</sup>Leu repressor.</p></fn><fn id="tblfn2"><p><sup>&#8224;</sup>Combination of amoxicillin (20 &#181;g) and clavulanate (10 &#181;g).</p></fn><fn id="tblfn3"><p><sup>&#8225;</sup>Combination of piperacillin (75 &#181;g) and tazobactam (10 &#181;g).</p></fn><fn id="tblfn4"><p><sup>&#167;</sup>ND, not detected as the strains grew at the contact of the disk.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-19469-v2.xml">elife-19469-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.19469.016</object-id><label>Table 5.</label><caption><p>Minimal inhibitory concentration of &#946;-lactams against <italic>E</italic>. <italic>coli</italic> strains harboring various plasmids<sup>*</sup>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.19469.016">http://dx.doi.org/10.7554/eLife.19469.016</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom"><p>&#946;-lactam</p></th><th rowspan="2" valign="bottom"><p>Inducer<sup>&#8224;</sup></p></th><th colspan="8" valign="bottom"><p><bold>Strains</bold></p> </th></tr><tr><th><p>BW25113</p></th><th><p>BW25113pJEH12(<italic>ycbB</italic>)</p></th><th><p>M1cured</p></th><th><p>M1curedpJEH12(<italic>ycbB</italic>)</p></th><th><p>BW25113pKT8(<italic>relA</italic>')</p></th><th><p>BW25113pKT8(<italic>relA</italic>')pKT2(<italic>ycbB</italic>)</p></th><th><p>BW25113&#916;relApKT8(<italic>relA</italic>')</p></th><th><p><bold>BW25113&#916;relApKT8(<italic>relA</italic>')pKT2(<italic>ycbB</italic>)</bold></p> </th></tr></thead><tbody><tr><td rowspan="4" valign="bottom"><p>Ampicillin</p> </td><td valign="bottom"><p>None</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>4</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>4</p></td></tr><tr><td valign="bottom"><p>IPTG</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>128</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>4</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>4</p></td></tr><tr><td valign="bottom"><p>Ara</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>4</p></td></tr><tr><td valign="bottom"><p>IPTG+Ara</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>128</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>64</p></td><td valign="bottom"><p>8</p></td><td valign="bottom"><p>128</p></td></tr><tr><td rowspan="4" valign="bottom"><p>Ceftriaxone</p> </td><td valign="bottom"><p>None</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td></tr><tr><td valign="bottom"><p>IPTG</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>32</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td></tr><tr><td valign="bottom"><p>Ara</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,1</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td></tr><tr><td valign="bottom"><p>IPTG+Ara</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>32</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>32</p></td><td valign="bottom"><p>0,05</p></td><td valign="bottom"><p>32</p></td></tr></tbody></table><table-wrap-foot><fn id="tblfn12"><p><sup>*</sup>Minimal inhibitory concentrations were determined by the agar dilution method with an inoculum of 10<sup>5</sup> colony forming units per spot in brain heart infusion agar after 24 hr of incubation at 37&#176;C. The same results were obtained with 10<sup>4</sup> and 10<sup>6</sup> colony forming units, indicating that YcbB in combination with elevated (p)ppGpp rendered the bulk of the population resistant.</p></fn><fn id="tblfn13"><p><sup>&#8224;</sup>Induction was performed with 50 mM IPTG, 1% arabinose (Ara), and a combination of both inducers (IPTG+Ara). IPTG induces expression of the <italic>ycbB</italic> gene of plasmids pJEH12(<italic>ycbB</italic>) and pKT2(<italic>ycbB</italic>). Arabinose induces expression of <italic>relA&#8217;</italic> encoding the first 455 residues of RelA.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-19568-v1.xml">elife-19568-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.19568.019</object-id><label>Table 2.</label><caption><p>Dimensional parameters measured and derived from the Laetoli Site S tracks and stature and body mass estimates for S1 and S2.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.19568.019">http://dx.doi.org/10.7554/eLife.19568.019</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><p><bold>Footprint</bold></p> </th><th rowspan="2"><p><bold>Side</bold></p> </th><th rowspan="2"><p><bold>Length (mm)</bold></p> </th><th rowspan="2"><p><bold>Max width (mm)</bold></p> </th><th rowspan="2"><p><bold>Foot index (%)</bold></p> </th><th rowspan="2"><p><bold>Heel width (mm)</bold></p> </th><th rowspan="2"><p><bold>Angle of gait (degrees)</bold></p> </th><th colspan="3"><p><bold>Estimated stature (cm)</bold></p> </th><th colspan="2"><p><bold>Estimated body mass (kg)</bold></p> </th></tr><tr><th><p><bold><italic>H. sapiens</italic><sup>&#167;</sup></bold></p> </th><th><p><bold><italic>H. sapiens</italic><sup>&#176;</sup></bold></p> </th><th><p><bold><italic>Au. afarensis<sup>&#8225;</sup></italic></bold></p> </th><th><p><bold><italic>H. sapiens</italic><sup>&#176;</sup></bold></p> </th><th><p><bold><italic>Au. afarensis<sup>&#8225;</sup></italic></bold></p> </th></tr></thead><tbody><tr><td><p>TP2/S1-1</p></td><td><p>right</p></td><td><p>271</p></td><td><p>101</p></td><td><p>37.2</p></td><td><p>83</p></td><td><p>6</p></td><td><p>194&#8211;170</p> </td><td><p>175.4</p></td><td><p>167&#8211;175</p> </td><td><p>53.8</p></td><td><p>42.9&#8211;50.0</p> </td></tr><tr><td><p>TP2/S1-2</p></td><td><p>left</p></td><td><p>271</p></td><td><p>99</p></td><td><p>36.6</p></td><td><p>81</p></td><td><p>4</p></td><td><p>193&#8211;169</p> </td><td><p>175.1</p></td><td><p>167&#8211;175</p> </td><td><p>53.1</p></td><td><p>42.8&#8211;49.8</p> </td></tr><tr><td><p>M9/S1-1</p></td><td><p>left</p></td><td><p>250</p></td><td><p>102</p></td><td><p>40.6</p></td><td><p>73</p></td><td><p>2</p></td><td><p>179&#8211;156</p> </td><td><p>167.5</p></td><td><p>154&#8211;161</p> </td><td><p>51.6</p></td><td><p>39.6&#8211;46.0</p> </td></tr><tr><td><p>M9/S1-2</p></td><td><p>right</p></td><td><p>264</p></td><td><p>105</p></td><td><p>39.7</p></td><td><p>80</p></td><td><p>3</p></td><td><p>189&#8211;165</p> </td><td><p>172.8</p></td><td><p>163&#8211;171</p> </td><td><p>54.2</p></td><td><p>41.8&#8211;48.7</p> </td></tr><tr><td><p>M9/S1-3</p></td><td><p>left</p></td><td><p>268</p></td><td><p>111</p></td><td><p>41.2</p></td><td><p>91</p></td><td><p>4</p></td><td><p>192&#8211;168</p> </td><td><p>174.3</p></td><td><p>166&#8211;173</p> </td><td><p>56.3</p></td><td><p>42.5&#8211;49.4</p> </td></tr><tr><td><p>M9/S1-4</p></td><td><p>right</p></td><td><p>245</p></td><td><p>101</p></td><td><p>41.2</p></td><td><p>71</p></td><td><p>4</p></td><td><p>175&#8211;153</p> </td><td><p>165.6</p></td><td><p>151&#8211;158</p> </td><td><p>50.9</p></td><td><p>38.8&#8211;45.1</p> </td></tr><tr><td><p>L8/S1-1</p></td><td><p>right</p></td><td><p>245</p></td><td><p>104</p></td><td><p>42.4</p></td><td><p>78</p></td><td><p>8</p></td><td><p>175&#8211;153</p> </td><td><p>165.6</p></td><td><p>151&#8211;158</p> </td><td><p>51.7</p></td><td><p>38.8&#8211;45.1</p> </td></tr><tr><td><p>L8/S1-2</p></td><td><p>left</p></td><td valign="bottom"><p>265</p></td><td><p>106</p></td><td><p>40.0</p></td><td><p>82</p></td><td><p>11</p></td><td><p>189&#8211;166</p> </td><td><p>173.1</p></td><td><p>164&#8211;171</p> </td><td><p>54.5</p></td><td><p>41.9&#8211;48.8</p> </td></tr><tr><td><p>L8/S1-3</p></td><td><p>right</p></td><td><p>260</p></td><td><p>103</p></td><td><p>39.6</p></td><td><p>77</p></td><td><p>3</p></td><td><p>186&#8211;163</p> </td><td><p>171.3</p></td><td><p>161&#8211;168</p> </td><td><p>53.1</p></td><td><p>41.2&#8211;47.9</p> </td></tr><tr><td><p>L8/S1-4</p></td><td><p>left</p></td><td><p>274</p></td><td><p>106</p></td><td><p>38.6</p></td><td><p>81</p></td><td><p>10</p></td><td><p>196&#8211;171</p> </td><td><p>176.5</p></td><td><p>169&#8211;177</p> </td><td><p>55.6</p></td><td><p>43.4&#8211;50.5</p> </td></tr><tr><td><p>L8/S1-5</p></td><td><p>right</p></td><td><p>-</p></td><td><p>-</p></td><td><p>-</p></td><td><p>-</p></td><td><p>-</p></td><td><p>-</p></td><td><p>-</p></td><td><p>-</p></td><td><p>-</p></td><td><p>-</p></td></tr><tr><td><p>L8/S1-6</p></td><td><p>left</p></td><td><p>-</p></td><td><p>-</p></td><td><p>-</p></td><td><p>86</p></td><td><p>3</p></td><td><p>-</p></td><td><p>-</p></td><td><p>-</p></td><td><p>-</p></td><td><p>-</p></td></tr><tr><td><p>L8/S1-7</p></td><td><p>right</p></td><td><p>258</p></td><td><p>110</p></td><td><p>42.7</p></td><td><p>90</p></td><td><p>8</p></td><td><p>184&#8211;161</p> </td><td><p>170.3</p></td><td><p>159&#8211;166</p> </td><td><p>54.8</p></td><td><p>40.7&#8211;47.4</p> </td></tr><tr><td><p><bold>Average S1</bold></p> </td><td><p><bold>-</bold></p> </td><td><p><bold>261</bold></p> </td><td><p><bold>104</bold></p> </td><td><p><bold>40.0</bold></p> </td><td><p><bold>81</bold></p> </td><td><p><bold>6</bold></p> </td><td><p><bold>184&#8211;163</bold></p> </td><td><p><bold>171.6</bold></p> </td><td><p><bold>161&#8211;168</bold></p> </td><td><p><bold>53.6</bold></p> </td><td><p><bold>41.3&#8211;48.1</bold></p> </td></tr><tr><td><p>TP2/S2-1</p></td><td><p>right</p></td><td><p>231</p></td><td valign="bottom"><p>120<sup>*</sup></p></td><td valign="bottom"><p>51.9<sup>*</sup></p></td><td valign="bottom"><p>86</p></td><td valign="bottom"><p>-</p></td><td valign="bottom"><p>165&#8211;144</p> </td><td><p>160</p></td><td><p>142&#8211;149</p> </td><td><p>46.7</p></td><td><p>36.5&#8211;42.4</p> </td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th colspan="3"><p><bold>Step length</bold></p> </th><th colspan="3"><p><bold>Stride length</bold></p> </th></tr><tr><th><p><bold>Footprints</bold></p> </th><th><p><bold>Side</bold></p> </th><th><p><bold>Step length (mm)</bold></p> </th><th><p><bold>Footprints</bold></p> </th><th><p><bold>Side</bold></p> </th><th><p><bold>Stride length (mm)</bold></p> </th></tr></thead><tbody><tr><td><p>TP2/S1-1 &#8594; 2</p></td><td><p>right &#8594; left</p></td><td><p>553</p></td><td><p>M9/S1-1 &#8594; 3</p></td><td><p>left</p></td><td><p>1044</p></td></tr><tr><td><p>M9/S1-1 &#8594; 2</p></td><td><p>left &#8594; right</p></td><td><p>548</p></td><td><p>M9/S1-2 &#8594; 4</p></td><td><p>right</p></td><td><p>1069</p></td></tr><tr><td><p>M9/S1-2 &#8594; 3</p></td><td><p>right &#8594; left</p></td><td><p>505</p></td><td><p>L8/S1-1 &#8594; 3</p></td><td><p>right</p></td><td><p>1140</p></td></tr><tr><td><p>M9/S1-3 &#8594; 4</p></td><td><p>left &#8594; right</p></td><td><p>571</p></td><td><p>L8/S1-2 &#8594; 4</p></td><td><p>left</p></td><td><p>1159</p></td></tr><tr><td><p>L8/S1-1 &#8594; 2</p></td><td><p>right &#8594; left</p></td><td><p>552</p></td><td><p>L8/S1-4 &#8594; 6</p></td><td><p>left</p></td><td><p>1284</p></td></tr><tr><td><p>L8/S1-2 &#8594; 3</p></td><td><p>left &#8594; right</p></td><td><p>587</p></td><td><p>Average right</p></td><td/><td><p>1105</p></td></tr><tr><td><p>L8/S1-3 &#8594; 4</p></td><td><p>right &#8594; left</p></td><td><p>573</p></td><td><p>Average left</p></td><td/><td><p>1162</p></td></tr><tr><td><p>L8/S1-6 &#8594; 7</p></td><td><p>left &#8594; right</p></td><td><p>660</p></td><td><p>Average</p></td><td/><td><p>1139</p></td></tr><tr><td><p>Average right &#8594; left</p></td><td/><td><p>545</p></td></tr><tr><td><p>Average left &#8594; right</p></td><td/><td><p>591</p></td></tr><tr><td><p>Average</p></td><td/><td><p>568</p></td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>*Values overestimated because of the enlarged morphology of the only preserved track of S2. <sup>&#167;</sup>Estimation based on the relationship between foot length and stature in <italic>Homo sapiens</italic> (<xref ref-type="bibr" rid="bib58">Tuttle, 1987</xref>). <sup>&#176;</sup>Estimation based on the relationship between footprint length and stature/body mass in <italic>H. sapiens</italic> (<xref ref-type="bibr" rid="bib11">Dingwall et al., 2013</xref>). <sup>&#8225;</sup>Estimation based on the relationship between foot length and stature/body mass in <italic>Au. afarensis</italic> (<xref ref-type="bibr" rid="bib11">Dingwall et al., 2013</xref>). See Materials and methods for details.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-19720-v1.xml">elife-19720-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.19720.005</object-id><label>Table 1.</label><caption><p>FLIM analysis of the PS1 NT-CT proximity in phosphorylation-inhibited PS1 mutants.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.19720.005">http://dx.doi.org/10.7554/eLife.19720.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><p>Construct</p></th><th colspan="2" valign="top"><p><sup>*</sup>Relative FRET efficiency (%)</p></th><th colspan="2" valign="top"><p>p value</p></th></tr><tr><th valign="top"><p>DMSO</p></th><th valign="top"><p>A23187</p></th><th valign="top"><p><sup>&#8225;</sup>vs WT (in DMSO)</p></th><th valign="top"><p><sup>&#167;</sup>DMSO vs A23187</p></th></tr></thead><tbody><tr><td valign="top"><p>PS1 Wild type (WT)</p></td><td valign="top"><p>100&#160;&#177;&#160;8.9</p> <p>(n&#160;=&#160;18)</p></td><td valign="top"><p>134.7&#160;&#177;&#160;7.5</p> <p>(n&#160;=&#160;25)</p></td><td valign="top"><p>-</p></td><td valign="top"><p><sup>&#8224;</sup>p&lt;0.05</p> </td></tr><tr><td valign="top"><p>PS1 S28A</p></td><td valign="top"><p>106.9&#160;&#177;&#160;7.0</p> <p>(n&#160;=&#160;19)</p></td><td valign="top"><p>143.4&#160;&#177;&#160;8.6</p> <p>(n&#160;=&#160;19)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p><sup>&#8224;</sup>p&lt;0.05</p> </td></tr><tr><td valign="top"><p>PS1 T74A</p></td><td valign="top"><p>96.5&#160;&#177;&#160;13.9</p> <p>(n&#160;=&#160;14)</p></td><td valign="top"><p>98.5&#160;&#177;&#160;8.3</p> <p>(n&#160;=&#160;22)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p>n.s.</p></td></tr><tr><td valign="top"><p>PS1 S310A</p></td><td valign="top"><p>88.8&#160;&#177;&#160;10.2</p> <p>(n&#160;=&#160;17)</p></td><td valign="top"><p>95.1&#160;&#177;&#160;8.6</p> <p>(n&#160;=&#160;24)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p>n.s.</p></td></tr><tr><td valign="top"><p>PS1 S313A</p></td><td valign="top"><p>92.7&#160;&#177;&#160;8.4</p> <p>(n&#160;=&#160;15)</p></td><td valign="top"><p>108.0&#160;&#177;&#160;8.8</p> <p>(n&#160;=&#160;21)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p>n.s.</p></td></tr><tr><td valign="top"><p>PS1 S310A/S313A</p></td><td valign="top"><p>84.2&#160;&#177;&#160;6.7</p> <p>(n&#160;=&#160;26)</p></td><td valign="top"><p>87.8&#160;&#177;&#160;9.0</p> <p>(n&#160;=&#160;17)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p>n.s.</p></td></tr><tr><td valign="top"><p>PS1 S319A/T320A</p></td><td valign="top"><p>106.2&#160;&#177;&#160;8.1</p> <p>(n&#160;=&#160;20)</p></td><td valign="top"><p>131.8&#160;&#177;&#160;8.1</p> <p>(n&#160;=&#160;20)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p><sup>&#8224;</sup>p&lt;0.05</p> </td></tr><tr><td valign="top"><p>PS1 S324A</p></td><td valign="top"><p>93.8&#160;&#177;&#160;6.2</p> <p>(n&#160;=&#160;19)</p></td><td valign="top"><p>129.9&#160;&#177;&#160;8.4</p> <p>(n&#160;=&#160;15)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p><sup>&#8224;</sup>p&lt;0.05</p> </td></tr><tr><td valign="top"><p>PS1 S337A</p></td><td valign="top"><p>99.9&#160;&#177;&#160;8.8</p> <p>(n&#160;=&#160;13)</p></td><td valign="top"><p>132.4&#160;&#177;&#160;10.9</p> <p>(n&#160;=&#160;16)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p><sup>&#8224;</sup>p&lt;0.05</p> </td></tr><tr><td valign="top"><p>PS1 S346A</p></td><td valign="top"><p>100.1&#160;&#177;&#160;8.9</p> <p>(n&#160;=&#160;18)</p></td><td valign="top"><p>135.0&#160;&#177;&#160;5.2</p> <p>(n&#160;=&#160;19)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p><sup>&#8224;</sup>p&lt;0.05</p> </td></tr><tr><td valign="top"><p>PS1 S353A</p></td><td valign="top"><p>89.6&#160;&#177;&#160;7.4</p> <p>(n&#160;=&#160;18)</p></td><td valign="top"><p>129.8&#160;&#177;&#160;8.3</p> <p>(n&#160;=&#160;23)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p><sup>&#8224;</sup>p&lt;0.05</p> </td></tr><tr><td valign="top"><p>PS1 T354A</p></td><td valign="top"><p>74.4&#160;&#177;&#160;5.6</p> <p>(n&#160;=&#160;14)</p></td><td valign="top"><p>113.4&#160;&#177;&#160;7.7</p> <p>(n&#160;=&#160;23)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p><sup>&#8224;</sup>p&lt;0.05</p> </td></tr><tr><td valign="top"><p>PS1 S357A</p></td><td valign="top"><p>91.2&#160;&#177;&#160;7.9</p> <p>(n&#160;=&#160;18)</p></td><td valign="top"><p>119.8&#160;&#177;&#160;9.2</p> <p>(n&#160;=&#160;23)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p><sup>&#8224;</sup>p&lt;0.05</p> </td></tr><tr><td valign="top"><p>PS1 S353A/S357A</p></td><td valign="top"><p>98.9&#160;&#177;&#160;9.1</p> <p>(n&#160;=&#160;14)</p></td><td valign="top"><p>125.9&#160;&#177;&#160;7.1</p> <p>(n&#160;=&#160;13)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p><sup>&#8224;</sup>p&lt;0.05</p> </td></tr><tr><td valign="top"><p>PS1 S365A</p></td><td valign="top"><p>102.3&#160;&#177;&#160;6.4</p> <p>(n&#160;=&#160;19)</p></td><td valign="top"><p>87.3&#160;&#177;&#160;9.9</p> <p>(n&#160;=&#160;21)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p>n.s.</p></td></tr><tr><td valign="top"><p>PS1 S366A</p></td><td valign="top"><p>102.8&#160;&#177;&#160;6.7</p> <p>(n&#160;=&#160;16)</p></td><td valign="top"><p>101.6&#160;&#177;&#160;12.6</p> <p>(n&#160;=&#160;16)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p>n.s.</p></td></tr><tr><td valign="top"><p>PS1 S367A</p></td><td valign="top"><p>99.7&#160;&#177;&#160;6.1</p> <p>(n&#160;=&#160;10)</p></td><td valign="top"><p>99.1&#160;&#177;&#160;11.4</p> <p>(n&#160;=&#160;15)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p>n.s.</p></td></tr><tr><td valign="top"><p>PS1 S365A/S367A</p></td><td valign="top"><p>104.1&#160;&#177;&#160;6.7</p> <p>(n&#160;=&#160;19)</p></td><td valign="top"><p>102.8&#160;&#177;&#160;8.4</p> <p>(n&#160;=&#160;19)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p>n.s.</p></td></tr><tr><td valign="top"><p>PS1 S366A/S367A</p></td><td valign="top"><p>102.7&#160;&#177;&#160;11.2</p> <p>(n&#160;=&#160;10)</p></td><td valign="top"><p>108.2&#160;&#177;&#160;9.9</p> <p>(n&#160;=&#160;19)</p></td><td valign="top"><p>n.s.</p></td><td valign="top"><p>n.s.</p></td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>*The FRET efficiency in DMSO-treated cells expressing WT PS1 is set as 100%, and relative FRET efficiency in phosphorylation-inhibited mutants of PS1 is shown. Mean &#177; SEM, Student&#8217;s t-test, n: cell number, <sup>&#8224;</sup>: p&lt;0.05, n.s.: not significant.</p></fn><fn id="tblfn2"><p><sup>&#8225;</sup>p-value is shown for the comparison between WT PS1 and phosphorylation-inhibited mutants of PS1 in DMSO-treated conditions, or <sup>&#167;</sup>for the comparison between DMSO-treated and A23187 (5 &#181;M for 15 min)-treated cells expressing the same PS1 construct.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-20147-v2.xml">elife-20147-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="A1-tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.20147.019</object-id><label>Appendix 1&#8212;table 1.</label><caption><p>Parameters used for simulations. For the serial, two state and multistate models, the plasticity parameter listed is the transition probability between adjacent states. For the pooled resource model, the plasticity parameters are the minimum and maximum transition probability for the constituent two-state synapses. For the cascade and non-uniform multistate models, the plasticity parameter is the ratio of adjacent transition probabilities.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.20147.019">http://dx.doi.org/10.7554/eLife.20147.019</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Model</th><th rowspan="2" valign="top"># states</th><th colspan="3" valign="top">Plasticity parameter</th><th colspan="3" valign="top"><inline-formula><mml:math id="inf20"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msup><mml:mi>f</mml:mi><mml:mrow><mml:mtext>dep</mml:mtext></mml:mrow></mml:msup></mml:mrow></mml:mstyle></mml:math></inline-formula></th><th rowspan="2" valign="top"><inline-formula><mml:math id="inf21"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mi>r</mml:mi><mml:msub><mml:mi>t</mml:mi><mml:mrow><mml:mtext>pre</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula></th></tr><tr><th valign="top">Pot</th><th valign="top">WT dep</th><th valign="top">DKO dep</th><th valign="top">Base</th><th valign="top">Inc</th><th valign="top">Dec</th></tr></thead><tbody><tr><td valign="top">Serial</td><td valign="top">10</td><td valign="top">0.12</td><td valign="top">0.14</td><td valign="top">0.2</td><td valign="top">0.5</td><td valign="top">0.89</td><td valign="top">0.11</td><td valign="top">100</td></tr><tr><td valign="top">Two-state</td><td valign="top">2</td><td valign="top">0.1</td><td valign="top">0.1</td><td valign="top">0.2</td><td valign="top">0.5</td><td valign="top">0.6</td><td valign="top">0.4</td><td valign="top">5</td></tr><tr><td valign="top">Multistate</td><td valign="top">10</td><td valign="top">0.3</td><td valign="top">0.3</td><td valign="top">0.4</td><td valign="top">0.5</td><td valign="top">0.8</td><td valign="top">0.2</td><td valign="top">5</td></tr><tr><td valign="top">Pooled res.</td><td valign="top">7</td><td valign="top">0.008</td><td valign="top">[0.0006, 0.6]</td><td valign="top">[0.001, 1]</td><td valign="top">0.5</td><td valign="top">0.9</td><td valign="top">0.1</td><td valign="top">20</td></tr><tr><td valign="top">Cascade</td><td valign="top">14</td><td valign="top">0.386</td><td valign="top">0.398</td><td valign="top">0.466</td><td valign="top">0.478</td><td valign="top">0.63</td><td valign="top">0.002</td><td valign="top">200</td></tr><tr><td valign="top">Non-uni.</td><td valign="top">12</td><td valign="top">0.4</td><td valign="top">0.4</td><td valign="top">0.53</td><td valign="top">0.5</td><td valign="top">0.7</td><td valign="top">0.1</td><td valign="top">500</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-20147-v2.xml">elife-20147-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="A1-tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.20147.024</object-id><label>Appendix 1&#8212;table 2.</label><caption><p>Parameters used for simulations. For the serial, two state and multistate models, the plasticity parameter listed is the transition probability between adjacent states. For the pooled resource model, the plasticity parameters are the minimum and maximum transition probability for the constituent two-state synapses. For the cascade and non-uniform multistate models, the plasticity parameter is the ratio of adjacent transition probabilities.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.20147.024">http://dx.doi.org/10.7554/eLife.20147.024</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Model</th><th rowspan="2" valign="top"># states</th><th colspan="2" valign="top">Plasticity parameter</th><th colspan="3" valign="top"><inline-formula><mml:math id="inf151"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msup><mml:mi>f</mml:mi><mml:mrow><mml:mtext>dep</mml:mtext></mml:mrow></mml:msup></mml:mrow></mml:mstyle></mml:math></inline-formula></th><th rowspan="2" valign="top"><inline-formula><mml:math id="inf152"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mi>r</mml:mi><mml:msub><mml:mi>t</mml:mi><mml:mrow><mml:mtext>pre</mml:mtext></mml:mrow></mml:msub></mml:mrow></mml:mstyle></mml:math></inline-formula></th></tr><tr><th valign="top">Pot</th><th valign="top">Dep</th><th valign="top">Base</th><th valign="top">Inc</th><th valign="top">CF stim</th></tr></thead><tbody><tr><td valign="top">Serial</td><td valign="top">10</td><td valign="top">0.12</td><td valign="top">0.14</td><td valign="top">0.5</td><td valign="top">0.89</td><td valign="top">0.9879</td><td valign="top">100</td></tr><tr><td valign="top">Two-state</td><td valign="top">2</td><td valign="top">0.1</td><td valign="top">0.4</td><td valign="top">0.5</td><td valign="top">0.7</td><td valign="top">0.91</td><td valign="top">5</td></tr><tr><td valign="top">Multistate</td><td valign="top">10</td><td valign="top">0.3</td><td valign="top">0.3</td><td valign="top">0.5</td><td valign="top">0.8</td><td valign="top">0.96</td><td valign="top">5</td></tr><tr><td valign="top">Pooled res.</td><td valign="top">7</td><td valign="top">0.08</td><td valign="top">[0.006, 0.6]</td><td valign="top">0.5</td><td valign="top">0.9</td><td valign="top">0.99</td><td valign="top">20</td></tr><tr><td valign="top">Cascade</td><td valign="top">14</td><td valign="top">0.386</td><td valign="top">0.398</td><td valign="top">0.522</td><td valign="top">0.63</td><td valign="top">0.99</td><td valign="top">200</td></tr><tr><td valign="top">Non-uni.</td><td valign="top">12</td><td valign="top">0.4</td><td valign="top">0.4</td><td valign="top">0.5</td><td valign="top">0.7</td><td valign="top">0.99</td><td valign="top">500</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-20198-v2.xml">elife-20198-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.20198.005</object-id><label>Table 1.</label><caption><p>Activity of <italic>T. brucei</italic> AdoMetDC and AdoMetDC/prozyme complexes.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.20198.005">http://dx.doi.org/10.7554/eLife.20198.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="bottom"><p><bold><italic>Tb</italic>AdoMetDC proteins</bold></p> </th><th rowspan="3" valign="bottom"><p><bold>Prozyme proteins</bold></p> </th><th colspan="4" valign="top"><p><bold><italic>k<sub>cat</sub></italic>/K<sub>m</sub> (s<sup>&#8722;1</sup>M<sup>&#8722;1</sup>)</bold></p> </th></tr><tr><th colspan="2" valign="top"><p><bold><italic>Tb</italic>AdoMetDC monomer</bold></p> </th><th colspan="2" valign="top"><p><bold><italic>Tb</italic>AdoMetDC/prozyme heterodimer</bold></p> </th></tr><tr><th><p>(+) Put</p></th><th valign="top"><p>(&#8722;) Put</p></th><th><p>(+) Put</p></th><th valign="top"><p>(&#8722;) Put</p></th></tr></thead><tbody><tr><td><p>Wild-type</p></td><td><p>Wild-type</p></td><td><p>9.7&#160;&#177;&#160;3.5</p> </td><td><p>0.48&#160;&#177;&#160;0.08</p> </td><td><p>3.2&#160;&#177;&#160;0.6&#215;10<sup>3</sup></p> </td><td><p>2.6&#160;&#177;&#160;0.3&#215;10<sup>3</sup></p> </td></tr><tr><td><p>&#916;16</p></td><td><p>Wild-type</p></td><td><p>16&#177;3<sup>*</sup></p></td><td><p>16&#177;11<sup>*</sup></p></td><td><p>28&#177;5<sup>*</sup></p></td><td><p>18&#177;3<sup>*</sup></p></td></tr><tr><td><p>&#916;26</p></td><td><p>Wild-type</p></td><td><p>0.32&#160;&#177;&#160;0.07</p> </td><td><p>0.11&#160;&#177;&#160;0.02</p> </td><td><p>7.6&#160;&#177;&#160;1.7</p> </td><td><p>2.2&#160;&#177;&#160;0.3</p> </td></tr><tr><td><p>H172A</p></td><td><p>Wild-type</p></td><td><p>ND</p></td><td><p>ND</p></td><td><p>2.1&#160;&#177;&#160;0.4&#215;10<sup>3</sup></p> </td><td><p>93&#160;&#177;&#160;42</p> </td></tr><tr><td><p>W137A/M146A</p></td><td><p>Wild-type</p></td><td><p>ND</p></td><td><p>ND</p></td><td><p>87&#160;&#177;&#160;8</p> </td><td><p>5.0&#160;&#177;&#160;0.9</p> </td></tr><tr><td><p>Wild-type</p></td><td><p>M148'A/Y152'A</p></td><td><p>as above</p></td><td><p>as above</p></td><td><p>0.66&#160;&#177;&#160;0.23&#215;10<sup>3</sup></p> </td><td><p>16&#160;&#177;&#160;4</p> </td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p><sup>*</sup> data taken from (<xref ref-type="bibr" rid="bib48">Velez et al., 2013</xref>). Data were collected in the presence of 4 mM putrescine (Put), except for the &#916;16 mutant (5 mM putrescine<sup>*</sup>). In all cases, the heterodimer with the wild-type counterpart formed with sufficient affinity that the subunits could be copurified as a stable complex. Error represents the standard deviation for the fit of triplicate data points. ND, not determined.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-20600-v1.xml">elife-20600-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.20600.008</object-id><label>Table 2.</label><caption><p>&#945;SMA stress fiber localization and expression levels in assorted na&#239;ve pancreatic fibroblasts (stellate cells) cultured overnight within different D-ECMs.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.20600.008">http://dx.doi.org/10.7554/eLife.20600.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"><p>(#X na&#239;ve cell) / (#Y D-ECM)</p></th><th colspan="6"><p>&#945;SMA</p> </th></tr><tr><th colspan="3"><p>Stress fiber localization</p></th><th colspan="3"><p>Expression</p> </th></tr><tr><th><p>25% percentile</p></th><th><p>Median</p></th><th><p>75% percentile</p></th><th><p>25% percentile</p></th><th><p>Median</p></th><th><p>75% percentile</p></th></tr></thead><tbody><tr><td><p><bold>(2)/(1)</bold></p> </td><td><p>0.35</p></td><td><p><bold>1.00</bold></p> </td><td><p>1.26</p></td><td><p>0.65</p></td><td><p><bold>1.00</bold></p> </td><td><p>1.19</p></td></tr><tr><td><p><bold>(2)/(2)</bold></p> </td><td><p>0.90</p></td><td><p><bold>1.15</bold></p> </td><td><p>1.38</p></td><td><p>0.87</p></td><td><p><bold>1.10</bold></p> </td><td><p>1.30</p></td></tr><tr><td><p><bold>(3)/(2)</bold></p> </td><td><p>0.92</p></td><td><p><bold>1.08</bold></p> </td><td><p>1.36</p></td><td><p>0.86</p></td><td><p><bold>1.20</bold></p> </td><td><p>1.72</p></td></tr><tr><td><p><bold>(4)/(1)</bold></p> </td><td><p>0.08</p></td><td><p><bold>0.85</bold></p> </td><td><p>1.03</p></td><td><p>0.79</p></td><td><p><bold>2.62</bold></p> </td><td><p>3.04</p></td></tr><tr><td><p><bold>(4)/(5)</bold></p> </td><td><p>0.06</p></td><td><p><bold>0.83</bold></p> </td><td><p>1.36</p></td><td><p>0.29</p></td><td><p><bold>1.15</bold></p> </td><td><p>2.40</p></td></tr><tr><td><p><bold>(2)/(5)</bold></p> </td><td><p>0.52</p></td><td><p><bold>1.24</bold></p> </td><td><p>1.33</p></td><td><p>0.60</p></td><td><p><bold>1.44</bold></p> </td><td><p>2.31</p></td></tr><tr><td><p><bold>(2)/(1 +TGF&#946;i)</bold></p> </td><td><p>0.00</p></td><td><p><bold>0.00</bold></p> </td><td><p>0.04</p></td><td><p>0.04</p></td><td><p><bold>0.07</bold></p> </td><td><p>0.10</p></td></tr><tr><td><p><bold>(2)/(5 +TGF&#946;i)</bold></p> </td><td><p>0.00</p></td><td><p><bold>0.06</bold></p> </td><td><p>1.20</p></td><td><p>0.01</p></td><td><p><bold>0.14</bold></p> </td><td><p>0.76</p></td></tr><tr><td><p><bold>(3)/(1 +TGF&#946;i)</bold></p> </td><td><p>0.01</p></td><td><p><bold>0.06</bold></p> </td><td><p>0.23</p></td><td><p>0.01</p></td><td><p><bold>0.15</bold></p> </td><td><p>0.28</p></td></tr><tr><td><p><bold>(4)/(5 +TGF&#946;i)</bold></p> </td><td><p>0.11</p></td><td><p><bold>0.22</bold></p> </td><td><p>0.71</p></td><td><p>0.25</p></td><td><p><bold>0.74</bold></p> </td><td><p>1.66</p></td></tr></tbody></table><table-wrap-foot><fn id="tblfn9"><p>Values obtained from <bold>na&#239;ve</bold> cells (e.g., inactive stellate cells) isolated from patient number &#8216;<bold>#2</bold>&#8217; cultured overnight within <bold>D-ECMs</bold> made from CAFs isolated from patient <bold>#1</bold> were used for normalization and assigned an arbitrary unit of 1.00. Assorted, na&#239;ve cells (patient numbers indicated) were cultured within <bold>D-ECMs</bold> derived from the indicated CAFs, while experimental conditions included assorted D-ECMs treated with SB-431542 (<bold>D+TGF&#946;i</bold>) during ECM production. Note that quantitative immunofluorescent obtained values of &#945;SMA and F-actin were used to calculate stress fiber localization and expression of <bold>&#945;SMA</bold>. P values, listed below, were all calculated using the Mann Whitney test, compared to the normalized (2)/(1) experimental condition, to account for non-paired, two-tailed and non-Gaussian distributions of the data.</p></fn><fn id="tblfn10"><p>(2)/(1) vs. (2)/(2); <bold>p=0.0836</bold> stress fiber localization; <bold>p=0.3825</bold> expression</p></fn><fn id="tblfn11"><p>(2)/(1) vs. (3)/(2); p=0.1680 stress fiber localization; <bold>p=0.1736</bold> expression</p></fn><fn id="tblfn12"><p>(2)/(1) vs. (4)/(1); <bold>p=0.4266</bold> stress fiber localization; <bold>p=0.0755</bold> expression</p></fn><fn id="tblfn13"><p>(2)/(1) vs. (4)/(5); <bold>p=0.8927</bold> stress fiber localization; <bold>p=0.7509</bold> expression</p></fn><fn id="tblfn14"><p>(2)/(1) vs. (2)/(5); p=0.1885 stress fiber localization; p=0.4192 expression</p></fn><fn id="tblfn15"><p>(2)/(1) vs. (2)/(1<bold>+</bold>TGF&#946;i); <bold>p&lt;0.0001</bold> stress fiber localization; p&lt;0.0001 expression</p></fn><fn id="tblfn16"><p>(2)/(1) vs. (2)/(5+TGF&#946;i); <bold>p=0.0909</bold> stress fiber localization; p=0.0040 expression</p></fn><fn id="tblfn17"><p>(2)/(1) vs. (3)/(1+TGF&#946;i); <bold>p=0.0018</bold> stress fiber localization; p&lt;0.0001 expression</p></fn><fn id="tblfn18"><p>(2)/(1) vs. (4)/(5+TGF&#946;i); <bold>p=0.1018</bold> stress fiber localization; p=0.5181 expression</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-20707-v2.xml">elife-20707-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.20707.014</object-id><label>Table 3.</label><caption><p>Time course of pathway enrichment relating to affected and unaffected developmental and functional phenotypes.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.20707.014">http://dx.doi.org/10.7554/eLife.20707.014</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom"><p>Phenotype<sup>&#8224;</sup></p></th><th colspan="6" valign="bottom"><p>Development stage*</p></th></tr><tr><th valign="bottom"><p>3 dpf/E1</p></th><th valign="bottom"><p>6 dpf/E2</p></th><th valign="bottom"><p>10 dpf/E3</p></th><th valign="bottom"><p>11 dpf/E4</p></th><th valign="bottom"><p>0 Dph/E5</p></th><th valign="bottom"><p>three Dph/E6</p></th></tr></thead><tbody><tr><td valign="bottom"><p>Cardiovascular</p></td><td valign="bottom"><p>0 (0/8)</p> </td><td valign="bottom"><p>22.4 (11/49)</p> </td><td valign="bottom"><p>5.7 (4/70)</p> </td><td valign="bottom"><p>7.0 (4/57)</p> </td><td valign="bottom"><p>4.7 (2/43)</p> </td><td valign="bottom"><p>2.1 (1/48)</p> </td></tr><tr><td valign="bottom"><p>Craniofacial</p></td><td valign="bottom"><p>0 (0/8)</p> </td><td valign="bottom"><p>12.2 (6/49)</p> </td><td valign="bottom"><p>10 (7/70)</p> </td><td valign="bottom"><p>5.3 (3/57)</p> </td><td valign="bottom"><p>7.0 (3/43)</p> </td><td valign="bottom"><p>2.1 (1/48)</p> </td></tr><tr><td valign="bottom"><p>Liver</p></td><td valign="bottom"><p>12.5 (1/8)</p> </td><td valign="bottom"><p>0 (0/49)</p> </td><td valign="bottom"><p>5.7 (4/70)</p> </td><td valign="bottom"><p>8.8 (5/57)</p> </td><td valign="bottom"><p>0 (0/43)</p> </td><td valign="bottom"><p>0 (0/48)</p> </td></tr><tr><td valign="bottom"><p>Eye</p></td><td valign="bottom"><p>0 (0/8)</p> </td><td valign="bottom"><p>4.1 (2/49)</p> </td><td valign="bottom"><p>20 (14/70)</p> </td><td valign="bottom"><p>48.6 (17/35)</p> </td><td valign="bottom"><p>51.2 (22/43)</p> </td><td valign="bottom"><p>50.0 (24/48)</p> </td></tr><tr><td valign="bottom"><p>Osmoregulation</p></td><td valign="bottom"><p>--</p></td><td valign="bottom"><p>43.3 (13/30)</p> </td><td valign="bottom"><p>29.3 (12/41)</p> </td><td valign="bottom"><p>15.0 (3/20)</p> </td><td valign="bottom"><p>16 (4/25)</p> </td><td valign="bottom"><p>--</p></td></tr><tr><td valign="bottom"><p>Cholesterol</p></td><td valign="bottom"><p>0/30</p></td><td valign="bottom"><p>0 (0/27)</p> </td><td valign="bottom"><p>27.1 (13/48)</p> </td><td valign="bottom"><p>31.3 (10/32)</p> </td><td valign="bottom"><p>25.5 (12/47)</p> </td><td valign="bottom"><p>--</p></td></tr><tr><td valign="bottom"><p>Lipid</p></td><td valign="bottom"><p>0/30</p></td><td valign="bottom"><p>40.7 (11/27)</p> </td><td valign="bottom"><p>35.4 (17/48)</p> </td><td valign="bottom"><p>50.0 (16/32)</p> </td><td valign="bottom"><p>48.9 (23/47)</p> </td><td valign="bottom"><p>--</p></td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>*Percentage of total enriched pathways (absolute values).</p></fn><fn id="tblfn2"><p><sup>&#8224;</sup>Numbers of affected pathways representing Cardiovascular, Craniofacial, Liver and Eye were extracted from the combined Development category in IPA results; numbers of pathways representing osmoregulation/ion transport were extracted from the Molecular Transport category; numbers of pathways affecting Cholesterol/sterol metabolism and other non-cholesterol lipids (Lipid) were extracted from the Lipid Metabolism category.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-20713-v1.xml">elife-20713-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.20713.007</object-id><label>Table 1.</label><caption><p>Cell counts for <italic>dsx</italic>-intersected neurons in male and female adult CNS. Male and female <italic>dsx/elav</italic>, <italic>dsx/vGlut</italic> and <italic>dsx/Gad1</italic> cell counts are listed in black. Subsets of neurons that co-express Fru<sup>M</sup> in males are listed in italics.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.20713.007">http://dx.doi.org/10.7554/eLife.20713.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><p><italic>dsx</italic> neuronal clusters</p></th><th colspan="2" valign="top"><p><italic>dsx/elav</italic></p> </th><th colspan="2" valign="top"><p><italic>dsx/vGlut</italic></p> </th><th colspan="2" valign="top"><p><italic>dsx/Gad1</italic></p> </th></tr><tr><th valign="top"><p>Male</p></th><th valign="top"><p>Female</p></th><th valign="top"><p>Male</p></th><th valign="top"><p>Female</p></th><th valign="top"><p>Male</p></th><th valign="top"><p>Female</p></th></tr></thead><tbody><tr><td valign="top"><p>Brain</p></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><p>&#8195;pC1*</p></td><td valign="top"><p>52.8&#160;&#177;&#160;4.1 (12)</p> </td><td valign="top"><p>8.3&#160;&#177;&#160;1.6 (12)</p> </td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td></tr><tr><td valign="top"><p>&#8195;pC2*</p></td><td valign="top"><p>78.3&#160;&#177;&#160;4.8 (12)</p> </td><td valign="top"><p>14.2&#160;&#177;&#160;1.5 (12)</p> </td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>1.0&#160;&#177;&#160;0(12) <italic>0.9 &#177; 0.3 (10)</italic></p> </td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td></tr><tr><td valign="top"><p>&#8195;pC3*</p></td><td valign="top"><p>13.8&#160;&#177;&#160;0.9 (12)</p> </td><td valign="top"><p>8.0&#160;&#177;&#160;1.0 (12)</p> </td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>3.5&#160;&#177;&#160;0.5 (12) <italic>0.5 &#177; 0.5 (10)</italic></p> </td><td valign="top"><p>3.0&#160;&#177;&#160;0 (12)</p> </td></tr><tr><td valign="top"><p>&#8195;aDN*</p></td><td valign="top"><p>2.0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>2.0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>1.9&#160;&#177;&#160;0.3 (12)<italic>0 &#177; 0 (10)</italic></p> </td><td valign="top"><p>2.0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td></tr><tr><td valign="top"><p>&#8195;SN*</p></td><td valign="top"><p>1.0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>n.a.</p></td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>n.a.</p></td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>n.a.</p></td></tr><tr><td valign="top"><p>Ventral Nerve Cord</p> </td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><p>&#8195;TN1*</p></td><td valign="top"><p>23.0&#160;&#177;&#160;1.5 (12)</p> </td><td valign="top"><p>n.a.</p></td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>n.a.</p></td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>n.a.</p></td></tr><tr><td valign="top"><p>&#8195;TN2*</p></td><td valign="top"><p>7.9&#160;&#177;&#160;0.3 (12)</p> </td><td valign="top"><p>n.a.</p></td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>n.a.</p></td><td valign="top"><p>0&#160;&#177;&#160;0 (12)</p> </td><td valign="top"><p>n.a.</p></td></tr><tr><td valign="top"><p>&#8195;Abg<sup>&#8224;</sup></p></td><td valign="top"><p>275.0&#160;&#177;&#160;21.7 (10)</p> </td><td valign="top"><p>314.8&#160;&#177;&#160;18.9 (10)</p> </td><td valign="top"><p>79.8&#160;&#177;&#160;2.3 (10)<italic>7.4 &#177; 3.1 (10)</italic></p> </td><td valign="top"><p>101.8&#160;&#177;&#160;6.7 (10)</p> </td><td valign="top"><p>151.2&#160;&#177;&#160;3.8 (10)<italic>30.0 &#177; 4.8 (10)</italic></p> </td><td valign="top"><p>213.1&#160;&#177;&#160;2.1 (10)</p> </td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>*Neuronal cluster away from CNS midline. Count represents one cluster per hemisegment of the CNS.</p></fn><fn id="tblfn2"><p><sup>&#8224;</sup>Neuronal cluster spans the CNS midline. Count given is for the entire Abg. Counts represent mean &#177; S.D. n&#8217;s listed in parentheses.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-20718-v4.xml">elife-20718-v4.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.20718.004</object-id><label>Table 1.</label><caption><p>Comparison of PSICOV predictions for Tat component contacts with those of other co-evolutionary methods. The programs PSICOV (<xref ref-type="bibr" rid="bib44">Jones et al., 2012</xref>), CCMPRED (<xref ref-type="bibr" rid="bib71">Seemayer et al., 2014</xref>), and FreeContact (mfDCA) (<xref ref-type="bibr" rid="bib37">Hopf et al., 2012</xref>;&#160;<xref ref-type="bibr" rid="bib46">Kaj&#225;n et al., 2014</xref>) use methodologically distinct algorithms to calculate sequence co-evolution. Contacts predicted by all three of the co-evolutionary methods are colored green. Meta-PSICOV (<xref ref-type="bibr" rid="bib45">Jones et al., 2015</xref>) aggregates information from the other three prediction methods as well as other sequence information in Stage 1, then weights by vicinity to other contacts in Stage 2. The table is divided to include the co-evolutionary contacts for TatA<sub>All</sub>, TatA, TatB, TatA with TatA<sub>basal</sub>, TatA with TatB<sub>basal,</sub> and TatC-TatC contacts with separations greater than 20 &#197;. For the TatC-TatC contacts the three high-probabilty contacts identified by the three distinct methods suggest plausible contacts between the TM2-3 linker and residues at the C-terminal end of TM3 (green).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.20718.004">http://dx.doi.org/10.7554/eLife.20718.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="4"><p>PSICOV</p></th><th colspan="4"><p>FreeContact (mfDCA)</p></th><th colspan="4"><p>CCMPRED</p></th><th colspan="4"><p>Meta-PSICOV Stage 1</p></th><th colspan="4"><p>Meta-PSICOV Stage 2</p></th></tr><tr><th><p>TMH</p></th><th><p>TatC</p></th><th><p>Score</p></th><th><p>Rank</p></th><th><p>TMH</p></th><th><p>TatC</p></th><th><p>Score</p></th><th><p>Rank</p></th><th><p>TMH</p></th><th><p>TatC</p></th><th><p>Score</p></th><th><p>Rank</p></th><th><p>TMH</p></th><th><p>TatC</p></th><th><p>Score</p></th><th><p>Rank</p></th><th><p>TMH</p></th><th><p>TatC</p></th><th><p>Score</p></th><th><p>Rank</p></th></tr></thead><tbody><tr><td rowspan="17"><p>TatA<sub>All</sub></p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.71</p></td><td style="author-callout-style-b2"><p>51</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>1.46</p></td><td style="author-callout-style-b2"><p>108</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.30</p></td><td style="author-callout-style-b2"><p>28</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.75</p></td><td style="author-callout-style-b2"><p>22</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.46</p></td><td style="author-callout-style-b2"><p>112</p></td></tr><tr><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.56</p></td><td style="author-callout-style-b2"><p>88</p></td><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>201</p></td><td style="author-callout-style-b2"><p>1.00</p></td><td style="author-callout-style-b2"><p>305</p></td><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.27</p></td><td style="author-callout-style-b2"><p>59</p></td><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.71</p></td><td style="author-callout-style-b2"><p>30</p></td><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>201</p></td><td style="author-callout-style-b2"><p>0.40</p></td><td style="author-callout-style-b2"><p>145</p></td></tr><tr><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>201</p></td><td style="author-callout-style-b2"><p>0.5</p></td><td style="author-callout-style-b2"><p>109</p></td><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.95</p></td><td style="author-callout-style-b2"><p>335</p></td><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>201</p></td><td style="author-callout-style-b2"><p>0.23</p></td><td style="author-callout-style-b2"><p>117</p></td><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>201</p></td><td style="author-callout-style-b2"><p>0.64</p></td><td style="author-callout-style-b2"><p>49</p></td><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.34</p></td><td style="author-callout-style-b2"><p>191</p></td></tr><tr><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.48</p></td><td style="author-callout-style-b2"><p>123</p></td><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.84</p></td><td style="author-callout-style-b2"><p>422</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.21</p></td><td style="author-callout-style-b2"><p>175</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.19</p></td><td style="author-callout-style-b2"><p>414</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>201</p></td><td style="author-callout-style-b2"><p>0.15</p></td><td style="author-callout-style-b2"><p>444</p></td></tr><tr><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.38</p></td><td style="author-callout-style-b2"><p>189</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.75</p></td><td style="author-callout-style-b2"><p>549</p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>0.19</p></td><td style="author-callout-style-b2"><p>313</p></td><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.14</p></td><td style="author-callout-style-b2"><p>609</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.08</p></td><td style="author-callout-style-b2"><p>771</p></td></tr><tr><td><p>7</p></td><td><p>213</p></td><td><p>0.23</p></td><td><p>497</p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>0.56</p></td><td style="author-callout-style-b2"><p>1010</p></td><td><p>11</p></td><td><p>212</p></td><td><p>0.18</p></td><td><p>323</p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>0.14</p></td><td style="author-callout-style-b2"><p>611</p></td><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.05</p></td><td style="author-callout-style-b2"><p>1171</p></td></tr><tr><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>208</p></td><td style="author-callout-style-b2"><p>0.22</p></td><td style="author-callout-style-b2"><p>578</p></td><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>208</p></td><td style="author-callout-style-b2"><p>0.53</p></td><td style="author-callout-style-b2"><p>1128</p></td><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.18</p></td><td style="author-callout-style-b2"><p>388</p></td><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.11</p></td><td style="author-callout-style-b2"><p>734</p></td><td><p>4</p></td><td><p>205</p></td><td><p>0.04</p></td><td><p>1332</p></td></tr><tr><td><p>19</p></td><td><p>194</p></td><td><p>0.22</p></td><td><p>591</p></td><td><p>14</p></td><td><p>172</p></td><td><p>0.45</p></td><td><p>1577</p></td><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.18</p></td><td style="author-callout-style-b2"><p>400</p></td><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>208</p></td><td style="author-callout-style-b2"><p>0.09</p></td><td style="author-callout-style-b2"><p>860</p></td><td><p>8</p></td><td><p>205</p></td><td><p>0.04</p></td><td><p>1593</p></td></tr><tr><td><p>5</p></td><td><p>208</p></td><td><p>0.22</p></td><td><p>619</p></td><td><p>8</p></td><td><p>205</p></td><td><p>0.44</p></td><td><p>1630</p></td><td><p>5</p></td><td><p>208</p></td><td><p>0.18</p></td><td><p>404</p></td><td><p>12</p></td><td><p>201</p></td><td><p>0.06</p></td><td><p>1072</p></td><td><p>5</p></td><td><p>208</p></td><td><p>0.03</p></td><td><p>1726</p></td></tr><tr><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>0.21</p></td><td style="author-callout-style-b2"><p>670</p></td><td><p>5</p></td><td><p>126</p></td><td><p>0.43</p></td><td><p>1717</p></td><td><p>11</p></td><td><p>174</p></td><td><p>0.16</p></td><td><p>674</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.06</p></td><td style="author-callout-style-b2"><p>1083</p></td><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.03</p></td><td style="author-callout-style-b2"><p>1897</p></td></tr><tr><td><p>11</p></td><td><p>25</p></td><td><p>0.2</p></td><td><p>768</p></td><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.42</p></td><td style="author-callout-style-b2"><p>1761</p></td><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>208</p></td><td style="author-callout-style-b2"><p>0.15</p></td><td style="author-callout-style-b2"><p>753</p></td><td><p>14</p></td><td><p>203</p></td><td><p>0.05</p></td><td><p>1175</p></td><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>208</p></td><td style="author-callout-style-b2"><p>0.03</p></td><td style="author-callout-style-b2"><p>1963</p></td></tr><tr><td><p>8</p></td><td><p>205</p></td><td><p>0.19</p></td><td><p>772</p></td><td><p>17</p></td><td><p>85</p></td><td><p>0.42</p></td><td><p>1851</p></td><td><p>4</p></td><td><p>205</p></td><td><p>0.15</p></td><td><p>793</p></td><td><p>4</p></td><td><p>205</p></td><td><p>0.05</p></td><td><p>1176</p></td><td><p>11</p></td><td><p>202</p></td><td><p>0.03</p></td><td><p>2044</p></td></tr><tr><td><p>8</p></td><td><p>166</p></td><td><p>0.19</p></td><td><p>855</p></td><td><p>5</p></td><td><p>124</p></td><td><p>0.39</p></td><td><p>2105</p></td><td><p>17</p></td><td><p>227</p></td><td><p>0.15</p></td><td><p>802</p></td><td><p>5</p></td><td><p>208</p></td><td><p>0.04</p></td><td><p>1367</p></td><td><p>8</p></td><td><p>198</p></td><td><p>0.02</p></td><td><p>2299</p></td></tr><tr><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.18</p></td><td style="author-callout-style-b2"><p>911</p></td><td><p>5</p></td><td><p>132</p></td><td><p>0.38</p></td><td><p>2131</p></td><td><p>19</p></td><td><p>194</p></td><td><p>0.15</p></td><td><p>960</p></td><td><p>11</p></td><td><p>25</p></td><td><p>0.04</p></td><td><p>1424</p></td><td><p>15</p></td><td><p>202</p></td><td><p>0.02</p></td><td><p>2569</p></td></tr><tr><td><p>17</p></td><td><p>227</p></td><td><p>0.17</p></td><td><p>1003</p></td><td><p>12</p></td><td><p>194</p></td><td><p>0.33</p></td><td><p>2870</p></td><td><p>8</p></td><td><p>214</p></td><td><p>0.15</p></td><td><p>979</p></td><td><p>11</p></td><td><p>174</p></td><td><p>0.04</p></td><td><p>1544</p></td><td><p>4</p></td><td><p>208</p></td><td><p>0.02</p></td><td><p>2846</p></td></tr><tr><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.17</p></td><td style="author-callout-style-b2"><p>1150</p></td><td><p>12</p></td><td><p>75</p></td><td><p>0.32</p></td><td><p>2995</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.15</p></td><td style="author-callout-style-b2"><p>1036</p></td><td><p>15</p></td><td><p>219</p></td><td><p>0.03</p></td><td><p>1574</p></td><td><p>8</p></td><td><p>202</p></td><td><p>0.02</p></td><td><p>3287</p></td></tr><tr><td><p>12</p></td><td><p>201</p></td><td><p>0.16</p></td><td><p>1183</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.32</p></td><td style="author-callout-style-b2"><p>3008</p></td><td><p>12</p></td><td><p>201</p></td><td><p>0.14</p></td><td><p>1158</p></td><td><p>10</p></td><td><p>25</p></td><td><p>0.03</p></td><td><p>1607</p></td><td><p>4</p></td><td><p>206</p></td><td><p>0.01</p></td><td><p>3437</p></td></tr><tr><td rowspan="9"><p>TatA</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.71</p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>1.81</p></td><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.21</p></td><td style="author-callout-style-b2"><p>10</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.72</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.46</p></td><td style="author-callout-style-b2"><p>91</p></td></tr><tr><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.55</p></td><td style="author-callout-style-b2"><p>39</p></td><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>1.12</p></td><td style="author-callout-style-b2"><p>124</p></td><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.19</p></td><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.72</p></td><td style="author-callout-style-b2"><p>13</p></td><td><p>9</p></td><td><p>206</p></td><td><p>0.27</p></td><td><p>229</p></td></tr><tr><td><p>14</p></td><td><p>216</p></td><td><p>0.38</p></td><td><p>122</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>1.06</p></td><td style="author-callout-style-b2"><p>155</p></td><td><p>9</p></td><td><p>206</p></td><td><p>0.14</p></td><td><p>49</p></td><td><p>9</p></td><td><p>206</p></td><td><p>0.40</p></td><td><p>82</p></td><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.27</p></td><td style="author-callout-style-b2"><p>230</p></td></tr><tr><td><p>16</p></td><td><p>82</p></td><td><p>0.37</p></td><td><p>133</p></td><td><p>5</p></td><td><p>136</p></td><td><p>0.91</p></td><td><p>246</p></td><td><p>15</p></td><td><p>163</p></td><td><p>0.14</p></td><td><p>67</p></td><td><p>15</p></td><td><p>163</p></td><td><p>0.35</p></td><td><p>98</p></td><td><p>4</p></td><td><p>206</p></td><td><p>0.18</p></td><td><p>378</p></td></tr><tr><td><p>9</p></td><td><p>206</p></td><td><p>0.34</p></td><td><p>162</p></td><td><p>5</p></td><td><p>210</p></td><td><p>0.86</p></td><td><p>287</p></td><td><p>3</p></td><td><p>205</p></td><td><p>0.14</p></td><td><p>72</p></td><td><p>15</p></td><td><p>202</p></td><td><p>0.29</p></td><td><p>137</p></td><td><p>9</p></td><td><p>202</p></td><td><p>0.11</p></td><td><p>630</p></td></tr><tr><td><p>5</p></td><td><p>162</p></td><td><p>0.29</p></td><td><p>262</p></td><td><p>15</p></td><td><p>163</p></td><td><p>0.86</p></td><td><p>297</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.14</p></td><td style="author-callout-style-b2"><p>74</p></td><td><p>16</p></td><td><p>82</p></td><td><p>0.28</p></td><td><p>146</p></td><td><p>8</p></td><td><p>206</p></td><td><p>0.09</p></td><td><p>830</p></td></tr><tr><td><p>5</p></td><td><p>136</p></td><td><p>0.29</p></td><td><p>270</p></td><td><p>8</p></td><td><p>212</p></td><td><p>0.77</p></td><td><p>408</p></td><td><p>13</p></td><td><p>197</p></td><td><p>0.14</p></td><td><p>79</p></td><td><p>15</p></td><td><p>201</p></td><td><p>0.24</p></td><td><p>184</p></td><td><p>5</p></td><td><p>210</p></td><td><p>0.09</p></td><td><p>835</p></td></tr><tr><td><p>9</p></td><td><p>39</p></td><td><p>0.28</p></td><td><p>290</p></td><td><p>19</p></td><td><p>170</p></td><td><p>0.74</p></td><td><p>470</p></td><td><p>16</p></td><td><p>82</p></td><td><p>0.14</p></td><td><p>86</p></td><td><p>19</p></td><td><p>20</p></td><td><p>0.20</p></td><td><p>271</p></td><td><p>15</p></td><td><p>202</p></td><td><p>0.09</p></td><td><p>866</p></td></tr><tr><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.26</p></td><td style="author-callout-style-b2"><p>339</p></td><td><p>15</p></td><td><p>202</p></td><td><p>0.72</p></td><td><p>530</p></td><td><p>2</p></td><td><p>219</p></td><td><p>0.13</p></td><td><p>121</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.18</p></td><td style="author-callout-style-b2"><p>317</p></td><td><p>18</p></td><td><p>232</p></td><td><p>0.09</p></td><td><p>880</p></td></tr><tr><td rowspan="8"><p>TatB</p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>0.83</p></td><td style="author-callout-style-b2"><p>7</p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>2.19</p></td><td style="author-callout-style-b2"><p>6</p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>0.23</p></td><td style="author-callout-style-b2"><p>9</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.72</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.61</p></td><td style="author-callout-style-b2"><p>41</p></td></tr><tr><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.60</p></td><td style="author-callout-style-b2"><p>30</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>1.45</p></td><td style="author-callout-style-b2"><p>22</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.18</p></td><td style="author-callout-style-b2"><p>14</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.63</p></td><td style="author-callout-style-b2"><p>23</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.47</p></td><td style="author-callout-style-b2"><p>97</p></td></tr><tr><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.56</p></td><td style="author-callout-style-b2"><p>36</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>1.39</p></td><td style="author-callout-style-b2"><p>24</p></td><td><p>18</p></td><td><p>24</p></td><td><p>0.17</p></td><td><p>16</p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>0.61</p></td><td style="author-callout-style-b2"><p>29</p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>0.41</p></td><td style="author-callout-style-b2"><p>125</p></td></tr><tr><td><p>20</p></td><td><p>185</p></td><td><p>0.46</p></td><td><p>65</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>1.13</p></td><td style="author-callout-style-b2"><p>63</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.16</p></td><td style="author-callout-style-b2"><p>17</p></td><td><p>18</p></td><td><p>24</p></td><td><p>0.53</p></td><td><p>47</p></td><td><p>18</p></td><td><p>24</p></td><td><p>0.32</p></td><td><p>194</p></td></tr><tr><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.39</p></td><td style="author-callout-style-b2"><p>110</p></td><td><p>19</p></td><td><p>13</p></td><td><p>1.04</p></td><td><p>89</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.16</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.49</p></td><td style="author-callout-style-b2"><p>55</p></td><td><p>5</p></td><td><p>212</p></td><td><p>0.20</p></td><td><p>350</p></td></tr><tr><td><p>5</p></td><td><p>213</p></td><td><p>0.36</p></td><td><p>127</p></td><td><p>5</p></td><td><p>212</p></td><td><p>0.97</p></td><td><p>117</p></td><td><p>17</p></td><td><p>227</p></td><td><p>0.14</p></td><td><p>71</p></td><td><p>19</p></td><td><p>13</p></td><td><p>0.39</p></td><td><p>79</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.20</p></td><td style="author-callout-style-b2"><p>355</p></td></tr><tr><td><p>7</p></td><td><p>204</p></td><td><p>0.35</p></td><td><p>141</p></td><td><p>5</p></td><td><p>126</p></td><td><p>0.87</p></td><td><p>164</p></td><td><p>14</p></td><td><p>167</p></td><td><p>0.14</p></td><td><p>72</p></td><td><p>14</p></td><td><p>24</p></td><td><p>0.25</p></td><td><p>175</p></td><td><p>4</p></td><td><p>206</p></td><td><p>0.16</p></td><td><p>462</p></td></tr><tr><td><p>5</p></td><td><p>212</p></td><td><p>0.28</p></td><td><p>281</p></td><td><p>18</p></td><td><p>24</p></td><td><p>0.81</p></td><td><p>221</p></td><td><p>5</p></td><td><p>213</p></td><td><p>0.13</p></td><td><p>88</p></td><td><p>19</p></td><td><p>198</p></td><td><p>0.22</p></td><td><p>225</p></td><td><p>5</p></td><td><p>208</p></td><td><p>0.14</p></td><td><p>529</p></td></tr><tr><td rowspan="6"><p>TatA with TatA<sub>basal</sub></p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.75</p></td><td style="author-callout-style-b2"><p>28</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>1.85</p></td><td style="author-callout-style-b2"><p>26</p></td><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.31</p></td><td style="author-callout-style-b2"><p>10</p></td><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.80</p></td><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.47</p></td><td style="author-callout-style-b2"><p>94</p></td></tr><tr><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.70</p></td><td style="author-callout-style-b2"><p>35</p></td><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>1.26</p></td><td style="author-callout-style-b2"><p>62</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.30</p></td><td style="author-callout-style-b2"><p>11</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.77</p></td><td style="author-callout-style-b2"><p>14</p></td><td style="author-callout-style-b2"><p>16</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.43</p></td><td style="author-callout-style-b2"><p>111</p></td></tr><tr><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>201</p></td><td style="author-callout-style-b2"><p>0.60</p></td><td style="author-callout-style-b2"><p>58</p></td><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>1.01</p></td><td style="author-callout-style-b2"><p>130</p></td><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>201</p></td><td style="author-callout-style-b2"><p>0.22</p></td><td style="author-callout-style-b2"><p>40</p></td><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>201</p></td><td style="author-callout-style-b2"><p>0.68</p></td><td style="author-callout-style-b2"><p>36</p></td><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>201</p></td><td style="author-callout-style-b2"><p>0.39</p></td><td style="author-callout-style-b2"><p>136</p></td></tr><tr><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.58</p></td><td style="author-callout-style-b2"><p>64</p></td><td style="author-callout-style-b2"><p>15</p></td><td style="author-callout-style-b2"><p>201</p></td><td style="author-callout-style-b2"><p>1.01</p></td><td style="author-callout-style-b2"><p>131</p></td><td><p>11</p></td><td><p>212</p></td><td><p>0.17</p></td><td><p>126</p></td><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.18</p></td><td style="author-callout-style-b2"><p>309</p></td><td><p>12</p></td><td><p>201</p></td><td><p>0.08</p></td><td><p>644</p></td></tr><tr><td><p>19</p></td><td><p>194</p></td><td><p>0.38</p></td><td><p>141</p></td><td><p>12</p></td><td><p>215</p></td><td><p>0.69</p></td><td><p>378</p></td><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.16</p></td><td style="author-callout-style-b2"><p>143</p></td><td><p>14</p></td><td><p>203</p></td><td><p>0.14</p></td><td><p>419</p></td><td style="author-callout-style-b2"><p>8</p></td><td style="author-callout-style-b2"><p>215</p></td><td style="author-callout-style-b2"><p>0.06</p></td><td style="author-callout-style-b2"><p>832</p></td></tr><tr><td><p>8</p></td><td><p>205</p></td><td><p>0.27</p></td><td><p>309</p></td><td><p>5</p></td><td><p>136</p></td><td><p>0.67</p></td><td><p>426</p></td><td><p>12</p></td><td><p>215</p></td><td><p>0.16</p></td><td><p>159</p></td><td><p>8</p></td><td><p>208</p></td><td><p>0.09</p></td><td><p>605</p></td><td><p>4</p></td><td><p>205</p></td><td><p>0.06</p></td><td><p>852</p></td></tr><tr><td rowspan="6"><p>TatB with TatA<sub>basal</sub></p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>0.48</p></td><td style="author-callout-style-b2"><p>72</p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>1.42</p></td><td style="author-callout-style-b2"><p>44</p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>0.23</p></td><td style="author-callout-style-b2"><p>19</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.65</p></td><td style="author-callout-style-b2"><p>36</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.48</p></td><td style="author-callout-style-b2"><p>94</p></td></tr><tr><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.46</p></td><td style="author-callout-style-b2"><p>79</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.99</p></td><td style="author-callout-style-b2"><p>173</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>198</p></td><td style="author-callout-style-b2"><p>0.19</p></td><td style="author-callout-style-b2"><p>37</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.56</p></td><td style="author-callout-style-b2"><p>55</p></td><td><p>4</p></td><td><p>206</p></td><td><p>0.43</p></td><td><p>119</p></td></tr><tr><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.38</p></td><td style="author-callout-style-b2"><p>123</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.84</p></td><td style="author-callout-style-b2"><p>281</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.19</p></td><td style="author-callout-style-b2"><p>39</p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>0.56</p></td><td style="author-callout-style-b2"><p>56</p></td><td style="author-callout-style-b2"><p>18</p></td><td style="author-callout-style-b2"><p>21</p></td><td style="author-callout-style-b2"><p>0.28</p></td><td style="author-callout-style-b2"><p>214</p></td></tr><tr><td><p>20</p></td><td><p>185</p></td><td><p>0.37</p></td><td><p>134</p></td><td><p>12</p></td><td><p>215</p></td><td><p>0.76</p></td><td><p>383</p></td><td><p>18</p></td><td><p>24</p></td><td><p>0.18</p></td><td><p>44</p></td><td><p>15</p></td><td><p>201</p></td><td><p>0.34</p></td><td><p>122</p></td><td style="author-callout-style-b2"><p>12</p></td><td style="author-callout-style-b2"><p>202</p></td><td style="author-callout-style-b2"><p>0.28</p></td><td style="author-callout-style-b2"><p>218</p></td></tr><tr><td><p>12</p></td><td><p>215</p></td><td><p>0.33</p></td><td><p>180</p></td><td><p>15</p></td><td><p>201</p></td><td><p>0.73</p></td><td><p>427</p></td><td><p>14</p></td><td><p>167</p></td><td><p>0.17</p></td><td><p>69</p></td><td><p>4</p></td><td><p>206</p></td><td><p>0.28</p></td><td><p>168</p></td><td><p>4</p></td><td><p>207</p></td><td><p>0.28</p></td><td><p>225</p></td></tr><tr><td><p>5</p></td><td><p>213</p></td><td><p>0.30</p></td><td><p>252</p></td><td><p>5</p></td><td><p>212</p></td><td><p>0.68</p></td><td><p>514</p></td><td><p>5</p></td><td><p>213</p></td><td><p>0.17</p></td><td><p>88</p></td><td><p>18</p></td><td><p>24</p></td><td><p>0.27</p></td><td><p>175</p></td><td><p>4</p></td><td><p>205</p></td><td><p>0.16</p></td><td><p>415</p></td></tr><tr><td rowspan="5"><p>TatC-TatC</p></td><td style="author-callout-style-b2"><p>64</p></td><td style="author-callout-style-b2"><p>134</p></td><td style="author-callout-style-b2"><p>0.79</p></td><td style="author-callout-style-b2"><p>33</p></td><td><p>27</p></td><td><p>199</p></td><td><p>2.04</p></td><td><p>39</p></td><td style="author-callout-style-b2"><p>64</p></td><td style="author-callout-style-b2"><p>134</p></td><td style="author-callout-style-b2"><p>0.28</p></td><td style="author-callout-style-b2"><p>51</p></td><td style="author-callout-style-b2"><p>64</p></td><td style="author-callout-style-b2"><p>134</p></td><td style="author-callout-style-b2"><p>0.6</p></td><td style="author-callout-style-b2"><p>62</p></td><td style="author-callout-style-b2"><p>64</p></td><td style="author-callout-style-b2"><p>134</p></td><td style="author-callout-style-b2"><p>0.64</p></td><td style="author-callout-style-b2"><p>60</p></td></tr><tr><td style="author-callout-style-b2"><p>71</p></td><td style="author-callout-style-b2"><p>140</p></td><td style="author-callout-style-b2"><p>0.77</p></td><td style="author-callout-style-b2"><p>41</p></td><td style="author-callout-style-b2"><p>65</p></td><td style="author-callout-style-b2"><p>137</p></td><td style="author-callout-style-b2"><p>1.72</p></td><td style="author-callout-style-b2"><p>62</p></td><td style="author-callout-style-b2"><p>71</p></td><td style="author-callout-style-b2"><p>140</p></td><td style="author-callout-style-b2"><p>0.26</p></td><td style="author-callout-style-b2"><p>75</p></td><td style="author-callout-style-b2"><p>71</p></td><td style="author-callout-style-b2"><p>140</p></td><td style="author-callout-style-b2"><p>0.54</p></td><td style="author-callout-style-b2"><p>84</p></td><td><p>27</p></td><td><p>199</p></td><td><p>0.47</p></td><td><p>108</p></td></tr><tr><td style="author-callout-style-b2"><p>65</p></td><td style="author-callout-style-b2"><p>137</p></td><td style="author-callout-style-b2"><p>0.64</p></td><td style="author-callout-style-b2"><p>69</p></td><td style="author-callout-style-b2"><p>71</p></td><td style="author-callout-style-b2"><p>140</p></td><td style="author-callout-style-b2"><p>1.63</p></td><td style="author-callout-style-b2"><p>72</p></td><td style="author-callout-style-b2"><p>65</p></td><td style="author-callout-style-b2"><p>137</p></td><td style="author-callout-style-b2"><p>0.21</p></td><td style="author-callout-style-b2"><p>191</p></td><td><p>27</p></td><td><p>199</p></td><td><p>0.48</p></td><td><p>103</p></td><td style="author-callout-style-b2"><p>65</p></td><td style="author-callout-style-b2"><p>137</p></td><td style="author-callout-style-b2"><p>0.3</p></td><td style="author-callout-style-b2"><p>227</p></td></tr><tr><td><p>28</p></td><td><p>203</p></td><td><p>0.51</p></td><td><p>10864</p></td><td style="author-callout-style-b2"><p>64</p></td><td style="author-callout-style-b2"><p>134</p></td><td style="author-callout-style-b2"><p>1.51</p></td><td style="author-callout-style-b2"><p>93</p></td><td><p>113</p></td><td><p>228</p></td><td><p>0.19</p></td><td><p>319</p></td><td style="author-callout-style-b2"><p>65</p></td><td style="author-callout-style-b2"><p>137</p></td><td style="author-callout-style-b2"><p>0.39</p></td><td style="author-callout-style-b2"><p>150</p></td><td><p>64</p></td><td><p>137</p></td><td><p>0.24</p></td><td><p>283</p></td></tr><tr><td><p>72</p></td><td><p>217</p></td><td><p>0.42</p></td><td><p>155</p></td><td><p>64</p></td><td><p>157</p></td><td><p>1.05</p></td><td><p>284</p></td><td><p>67</p></td><td><p>140</p></td><td><p>0.18</p></td><td><p>345</p></td><td><p>28</p></td><td><p>203</p></td><td><p>0.37</p></td><td><p>162</p></td><td><p>33</p></td><td><p>167</p></td><td><p>0.23</p></td><td><p>289</p></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-20954-v2.xml">elife-20954-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.20954.008</object-id><label>Table 1.</label><caption><p>Characteristics of nanodiscs and contained bilayer for used combinations of different scaffold proteins with various lipids.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.20954.008">http://dx.doi.org/10.7554/eLife.20954.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><p><bold>Scaffold</bold></p> </th><th rowspan="2"><p><bold>Nanodisc diameter [nm]</bold></p> </th><th rowspan="2"><p><bold>Lipid</bold></p> </th><th rowspan="2"><p><bold>Protein:lipid ratio in nanodisc reaction mix</bold></p> </th><th rowspan="2"><p><bold>Attached lipids (LILBID)</bold></p> </th><th rowspan="2"><p><bold>Transition tempature [&#176;C]</bold></p> </th><th rowspan="2"><p><bold>Headgroup charge &#931; charges</bold></p> </th><th colspan="2"><p><bold>Hydrophobic tail</bold></p> </th></tr><tr><th><p><bold>Double bonds</bold></p> </th><th><p><bold>Length</bold></p> </th></tr></thead><tbody><tr><td rowspan="2"><p>MSP1D1&#916;H5</p></td><td rowspan="2"><p>8</p></td><td><p>DMPC</p></td><td><p>1:50</p></td><td><p>15</p></td><td><p>24</p></td><td><p>0</p></td><td><p>0</p></td><td><p>14</p></td></tr><tr><td><p>DMPG</p></td><td><p>1:45</p></td><td><p>6</p></td><td><p>23</p></td><td><p>&#8722;1</p></td><td><p>0</p></td><td><p>14</p></td></tr><tr><td rowspan="2"><p>MSP1</p></td><td rowspan="2"><p>10</p></td><td><p>DMPC</p></td><td><p>1:80</p></td><td><p>17</p></td><td><p>24</p></td><td><p>0</p></td><td><p>0</p></td><td><p>14</p></td></tr><tr><td><p>DMPG</p></td><td><p>1:70</p></td><td><p>9</p></td><td><p>23</p></td><td><p>&#8722;1</p></td><td><p>0</p></td><td><p>14</p></td></tr><tr><td rowspan="14"><p>MSP1E3D1</p></td><td rowspan="14"><p>12</p></td><td><p>DMPC</p></td><td><p>1:115</p></td><td><p>19</p></td><td><p>24</p></td><td><p>0</p></td><td><p>0</p></td><td><p>14</p></td></tr><tr><td><p>DMPG</p></td><td><p>1:110</p></td><td><p>11</p></td><td><p>23</p></td><td><p>&#8722;1</p></td><td><p>0</p></td><td><p>14</p></td></tr><tr><td><p>DPPC</p></td><td><p>1:100</p></td><td><p>18</p></td><td><p>41</p></td><td><p>0</p></td><td><p>0</p></td><td><p>16</p></td></tr><tr><td><p>POPG</p></td><td><p>1:90</p></td><td><p>15</p></td><td><p>&#8722;2</p></td><td><p>&#8722;1</p></td><td><p>0/1</p></td><td><p>16/18</p></td></tr><tr><td><p>DOPC</p></td><td><p>1:80</p></td><td><p>21</p></td><td><p>&#8722;17</p> </td><td><p>0</p></td><td><p>2</p></td><td><p>18</p></td></tr><tr><td><p>DOPG</p></td><td><p>1:80</p></td><td><p>20</p></td><td><p>&#8722;18</p> </td><td><p>&#8722;1</p></td><td><p>2</p></td><td><p>18</p></td></tr><tr><td><p>DOPS</p></td><td><p>1:90</p></td><td><p>20</p></td><td><p>&#8722;11</p> </td><td><p>&#8722;1</p></td><td><p>2</p></td><td><p>18</p></td></tr><tr><td><p>DOPE</p></td><td><p>1:80</p></td><td><p>8</p></td><td><p>&#8722;16</p> </td><td><p>0</p></td><td><p>2</p></td><td><p>18</p></td></tr><tr><td><p><bold>Mixtures</bold></p> </td><td/><td/><td><p><bold>Lipid ratio</bold></p> </td><td/><td/><td/></tr><tr><td><p>DMPG/DMPC</p></td><td><p>1:115</p></td><td><p>11</p></td><td><p>50/50</p></td><td><p>&#8722;1/0</p></td><td><p>0/0</p></td><td><p>14/14</p></td></tr><tr><td><p>DMPG/Cardiolipin</p></td><td><p>1:120</p></td><td><p>12</p></td><td><p>90/10</p></td><td><p>&#8722;1/&#8211;2</p> </td><td><p>0/0</p></td><td><p>14/14</p></td></tr><tr><td><p>DMPG/Inositol</p></td><td><p>1:110</p></td><td><p>15</p></td><td><p>50/50</p></td><td><p>&#8722;1/&#8211;1</p> </td><td><p>0/0;1</p></td><td><p>14/16;18</p></td></tr><tr><td><p>DMPC/Cholesterol</p></td><td><p>1:115</p></td><td><p>23</p></td><td><p>90/10</p></td><td><p>0/0</p></td><td><p>0/ n.d.</p></td><td><p>14/ n.d.</p></td></tr><tr><td><p>E. coli total lipids</p></td><td><p>1:50</p></td><td><p>n.i.</p></td><td><p>n.d.</p></td><td><p>n.d.</p></td><td><p>n.d.</p></td><td><p>n.d.</p></td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>n.d. = not defined.</p></fn><fn id="tblfn2"><p>n.i. = not investigated.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-20954-v2.xml">elife-20954-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.20954.011</object-id><label>Table 2.</label><caption><p>Construct list combining full name, size and expression system with the abbreviation. The theoretical masses of the different constructs are calculated by the Expasy tool protparam according to the primary sequence. Based on that the theoretical molecular weights of labeled proteins are calculated by the addition of masses of the single amino acids subtracted by the number of water molecules corresponding to the number of peptide bonds.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.20954.011">http://dx.doi.org/10.7554/eLife.20954.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><p><bold>Protein</bold></p> </th><th colspan="2" valign="bottom"><p><bold>Molecular weight [kDa]</bold></p> </th><th rowspan="2"><p><bold>Full name</bold></p> </th><th rowspan="2"><p><bold>Reference</bold></p> </th></tr><tr><th valign="bottom"><p><bold>Theoretical</bold></p> </th><th valign="bottom"><p><bold>Experimental</bold></p> </th></tr></thead><tbody><tr><td><p>PR</p></td><td><p>27.09*</p></td><td><p>27.19*</p></td><td><p>Proteorhodopsin (green variant)</p></td><td><p>(<xref ref-type="bibr" rid="bib35">Reckel et al., 2011</xref>)</p></td></tr><tr><td><p>KcsA</p></td><td><p>21.33/22.70<sup>&#8224;</sup></p></td><td><p>21.38/22.77<sup>&#8224;</sup></p></td><td><p>pH-gated potassium channel KcsA</p></td><td><p>-</p></td></tr><tr><td><p>Emre</p></td><td><p>13.00</p></td><td><p>13.09</p></td><td><p>Multidrug transporter EmrE</p></td><td><p>(<xref ref-type="bibr" rid="bib38">Roos et al., 2012</xref>)</p></td></tr><tr><td><p>LspA</p></td><td><p>21.52/22.83<sup>&#8224;</sup></p></td><td><p>21.63/22.84<sup>&#8224;</sup></p></td><td><p>Lipoprotein signal peptidase</p></td><td><p>(<xref ref-type="bibr" rid="bib24">Laguerre et al., 2016</xref>)</p></td></tr><tr><td><p>hHv1-VSD</p></td><td><p>20.80/22.28<sup>&#8224;</sup></p></td><td><p>20.89/22.20<sup>&#8224;</sup></p></td><td><p>Human proton channel - voltage sensing domain</p></td><td><p>(<xref ref-type="bibr" rid="bib25">Li et al., 2015</xref>)</p></td></tr><tr><td><p>Bs-MraY</p></td><td><p>37.88</p></td><td><p>37.93</p></td><td><p>Phospho-N-acetylmuramyl-pentapeptide-transferase <italic>B.subtilis</italic></p> </td><td><p>(<xref ref-type="bibr" rid="bib17">Henrich et al., 2016</xref>)</p></td></tr><tr><td><p>Ec-MraY</p></td><td><p>42.22</p></td><td><p>42.29</p></td><td><p>Phospho-N-acetylmuramyl-pentapeptide-transferase&#160;<italic>E.coli</italic></p> </td><td><p>(<xref ref-type="bibr" rid="bib17">Henrich et al., 2016</xref>)</p></td></tr><tr><td><p>MSP1</p></td><td><p>25.3</p></td><td><p>25.40</p></td><td><p>Membrane scaffold protein</p></td><td><p>(<xref ref-type="bibr" rid="bib10">Denisov et al., 2004</xref>)</p></td></tr><tr><td><p>MSP1E3D1</p></td><td><p>31.96</p></td><td><p>32.08</p></td><td><p>Membrane scaffold protein</p></td><td><p>(<xref ref-type="bibr" rid="bib10">Denisov et al., 2004</xref>)</p></td></tr><tr><td><p>MSP1D1&#916;H5</p></td><td><p>19.49<sup>&#8225;</sup>/21.46</p></td><td><p>19.53<sup>&#8225;</sup>/21.50</p></td><td><p>Membrane scaffold protein</p></td><td><p>(<xref ref-type="bibr" rid="bib14">Hagn et al., 2013</xref>)</p></td></tr></tbody></table><table-wrap-foot><fn id="tblfn3"><p>* Molecular mass according to covalent coupling of all trans-retinal.</p></fn><fn id="tblfn4"><p>&#8224; Molecular mass according to heavy isotope labeling.</p></fn><fn id="tblfn5"><p>&#8225; Molecular mass according to cleavage of the His<sub>6</sub>-tag.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-21350-v2.xml">elife-21350-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.21350.008</object-id><label>Table 2.</label><caption><p>Comparison of results on HR activity and drug response of the BRCA1 variants analyzed in this study obtained from previous and this studies.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.21350.008">http://dx.doi.org/10.7554/eLife.21350.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2">Domain</th><th colspan="6">HR activity (%)</th><th colspan="2">Cisplatin response</th><th colspan="2">Olaparib response</th></tr><tr><th valign="top"><xref ref-type="bibr" rid="bib50">Sy et al., 2009</xref></th><th valign="top"><xref ref-type="bibr" rid="bib38">Ransburgh et al. (2010)</xref></th><th valign="top"><xref ref-type="bibr" rid="bib51">Towler et al. (2013)</xref></th><th valign="top"><xref ref-type="bibr" rid="bib1">Bouwman et al. (2013)</xref></th><th valign="top"><xref ref-type="bibr" rid="bib27">Lu et al., 2015</xref></th><th valign="top">This study</th><th valign="top"><xref ref-type="bibr" rid="bib1">Bouwman et al. (2013)</xref></th><th valign="top">This study</th><th valign="top"><xref ref-type="bibr" rid="bib1">Bouwman et al. (2013)</xref></th><th valign="top">This study</th></tr></thead><tbody><tr><td valign="top">Vector</td><td valign="top"/><td valign="top">N/A</td><td valign="top">~10</td><td valign="top">~9</td><td valign="top">~20</td><td valign="top">~18</td><td valign="top">17.3</td><td valign="top">S</td><td valign="top">S</td><td valign="top">S</td><td valign="top">S</td></tr><tr><td valign="top">WT</td><td valign="top"/><td valign="top">98</td><td valign="top">100</td><td valign="top">100</td><td valign="top">100</td><td valign="top">100</td><td valign="top">100</td><td valign="top">R</td><td valign="top">R</td><td valign="top">R</td><td valign="top">R</td></tr><tr><td valign="top">C61G</td><td valign="top">RING</td><td valign="top">-</td><td valign="top">~17</td><td valign="top">-</td><td valign="top">-</td><td valign="top">23.6</td><td valign="top">21.2</td><td valign="top">S</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">C64R</td><td valign="top">RING</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">22.3</td><td valign="top">-</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">Y179C</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">~95</td><td valign="top">-</td><td valign="top">157.1</td><td valign="top">92.7</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">L246V</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">91.5</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">Q356R</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">95.5</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">F486L</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">160</td><td valign="top">95.8</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">R496H</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">95.9</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">N550H</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">90.8</td><td valign="top">88.4</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">L668F</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">96.8</td><td valign="top">93.6</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">D693N</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">111.5</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">V772A</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">110.4</td><td valign="top">84.5</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">R841W</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">98.2</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">P871L</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td></tr><tr><td valign="top">M1008I</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">99.2</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">E1038G</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td></tr><tr><td valign="top">S1040N</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">98.0</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">S1101N</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">97.3</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">S1140G</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">106.2</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">K1183R</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td></tr><tr><td valign="top">R1347G</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">106.5</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">M1400V</td><td valign="top">CC</td><td valign="top">56</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">74.8</td><td valign="top">R</td><td valign="top">S</td><td valign="top">R</td><td valign="top">S</td></tr><tr><td valign="top">L1407P</td><td valign="top">CC</td><td valign="top">24</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">24</td><td valign="top">S</td><td valign="top">S</td><td valign="top">S</td><td valign="top">S</td></tr><tr><td valign="top">M1411T</td><td valign="top">CC</td><td valign="top">25</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">26</td><td valign="top">R</td><td valign="top">S</td><td valign="top">R</td><td valign="top">S</td></tr><tr><td valign="top">S1512I</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">95.3</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">T1561I</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">133.2</td><td valign="top">102.3</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">S1613G</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td></tr><tr><td valign="top">M1628T</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">107</td><td valign="top">107.6</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">P1637L</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">98.8</td><td valign="top">99.5</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">5055&#9651;G</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">22.5</td><td valign="top">-</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">M1652I</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">106.2</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">S1655F</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">~30</td><td valign="top">-</td><td valign="top">8.4</td><td valign="top">S</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">C1697R</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">8.0</td><td valign="top">-</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">R1699Q</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">~45</td><td valign="top">-</td><td valign="top">16.9</td><td valign="top">S</td><td valign="top">S</td><td valign="top">S</td><td valign="top">S</td></tr><tr><td valign="top">A1708E</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">10.7</td><td valign="top">-</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">S1715R</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">9.3</td><td valign="top">-</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">M1775R</td><td valign="top">BRCT</td><td valign="top">36</td><td valign="top">-</td><td valign="top">~6</td><td valign="top">-</td><td valign="top">-</td><td valign="top">10.2</td><td valign="top">-</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr></tbody></table><table-wrap-foot><fn id="tblfn2"><p>R, resistant; S, sensitive. See <xref ref-type="supplementary-material" rid="SD1-data">Figure 1&#8212;source data 1</xref>, <xref ref-type="supplementary-material" rid="SD2-data">Figure 2&#8212;source data 1</xref> and <xref ref-type="supplementary-material" rid="SD3-data">Figure 3&#8212;source data 1</xref> for details.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-21350-v3.xml">elife-21350-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.21350.008</object-id><label>Table 2.</label><caption><p>Comparison of results on HR activity and drug response of the BRCA1 variants analyzed in this study obtained from previous and this studies.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.21350.008">http://dx.doi.org/10.7554/eLife.21350.008</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2">Domain</th><th colspan="6">HR activity (%)</th><th colspan="2">Cisplatin response</th><th colspan="2">Olaparib response</th></tr><tr><th valign="top"><xref ref-type="bibr" rid="bib50">Sy et al., 2009</xref></th><th valign="top"><xref ref-type="bibr" rid="bib38">Ransburgh et al. (2010)</xref></th><th valign="top"><xref ref-type="bibr" rid="bib51">Towler et al. (2013)</xref></th><th valign="top"><xref ref-type="bibr" rid="bib1">Bouwman et al. (2013)</xref></th><th valign="top"><xref ref-type="bibr" rid="bib27">Lu et al., 2015</xref></th><th valign="top">This study</th><th valign="top"><xref ref-type="bibr" rid="bib1">Bouwman et al. (2013)</xref></th><th valign="top">This study</th><th valign="top"><xref ref-type="bibr" rid="bib1">Bouwman et al. (2013)</xref></th><th valign="top">This study</th></tr></thead><tbody><tr><td valign="top">Vector</td><td valign="top"/><td valign="top">N/A</td><td valign="top">~10</td><td valign="top">~9</td><td valign="top">~20</td><td valign="top">~18</td><td valign="top">17.3</td><td valign="top">S</td><td valign="top">S</td><td valign="top">S</td><td valign="top">S</td></tr><tr><td valign="top">WT</td><td valign="top"/><td valign="top">98</td><td valign="top">100</td><td valign="top">100</td><td valign="top">100</td><td valign="top">100</td><td valign="top">100</td><td valign="top">R</td><td valign="top">R</td><td valign="top">R</td><td valign="top">R</td></tr><tr><td valign="top">C61G</td><td valign="top">RING</td><td valign="top">-</td><td valign="top">~17</td><td valign="top">-</td><td valign="top">-</td><td valign="top">23.6</td><td valign="top">21.2</td><td valign="top">S</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">C64R</td><td valign="top">RING</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">22.3</td><td valign="top">-</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">Y179C</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">~95</td><td valign="top">-</td><td valign="top">157.1</td><td valign="top">92.7</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">L246V</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">91.5</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">Q356R</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">95.5</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">F486L</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">160</td><td valign="top">95.8</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">R496H</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">95.9</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">N550H</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">90.8</td><td valign="top">88.4</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">L668F</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">96.8</td><td valign="top">93.6</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">D693N</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">111.5</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">V772A</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">110.4</td><td valign="top">84.5</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">R841W</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">98.2</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">P871L</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td></tr><tr><td valign="top">M1008I</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">99.2</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">E1038G</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td></tr><tr><td valign="top">S1040N</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">98.0</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">S1101N</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">97.3</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">S1140G</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">106.2</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">K1183R</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td></tr><tr><td valign="top">R1347G</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">106.5</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">M1400V</td><td valign="top">CC</td><td valign="top">56</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">74.8</td><td valign="top">R</td><td valign="top">S</td><td valign="top">R</td><td valign="top">S</td></tr><tr><td valign="top">L1407P</td><td valign="top">CC</td><td valign="top">24</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">24</td><td valign="top">S</td><td valign="top">S</td><td valign="top">S</td><td valign="top">S</td></tr><tr><td valign="top">M1411T</td><td valign="top">CC</td><td valign="top">25</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">26</td><td valign="top">R</td><td valign="top">S</td><td valign="top">R</td><td valign="top">S</td></tr><tr><td valign="top">S1512I</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">95.3</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">T1561I</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">133.2</td><td valign="top">102.3</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">S1613G</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td><td valign="top">-</td><td valign="top">wt</td></tr><tr><td valign="top">M1628T</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">107</td><td valign="top">107.6</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">P1637L</td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">98.8</td><td valign="top">99.5</td><td valign="top">-</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">5055&#9651;G</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">22.5</td><td valign="top">-</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">M1652I</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">106.2</td><td valign="top">R</td><td valign="top">R</td><td valign="top">-</td><td valign="top">R</td></tr><tr><td valign="top">S1655F</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">~30</td><td valign="top">-</td><td valign="top">8.4</td><td valign="top">S</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">C1697R</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">8.0</td><td valign="top">-</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">R1699Q</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">~45</td><td valign="top">-</td><td valign="top">16.9</td><td valign="top">S</td><td valign="top">S</td><td valign="top">S</td><td valign="top">S</td></tr><tr><td valign="top">A1708E</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">10.7</td><td valign="top">-</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">S1715R</td><td valign="top">BRCT</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">9.3</td><td valign="top">-</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr><tr><td valign="top">M1775R</td><td valign="top">BRCT</td><td valign="top">36</td><td valign="top">-</td><td valign="top">~6</td><td valign="top">-</td><td valign="top">-</td><td valign="top">10.2</td><td valign="top">-</td><td valign="top">S</td><td valign="top">-</td><td valign="top">S</td></tr></tbody></table><table-wrap-foot><fn id="tblfn2"><p>R, resistant; S, sensitive. See <xref ref-type="supplementary-material" rid="SD1-data">Figure 1&#8212;source data 1</xref>, <xref ref-type="supplementary-material" rid="SD2-data">Figure 2&#8212;source data 1</xref> and <xref ref-type="supplementary-material" rid="SD3-data">Figure 3&#8212;source data 1</xref> for details.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-21459-v3.xml">elife-21459-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.21459.004</object-id><label>Table 1.</label><caption><p>Relative gene expression level of selected WNT pathway regulators in HAP1 cells. RPKM values from duplicate RNAseq datasets generated as described in Materials and methods from two different passages of WT HAP1 cells are shown. Groups of paralogues and genes with similar&#160;functions are shaded in alternating colors to facilitate comparisons.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.21459.004">http://dx.doi.org/10.7554/eLife.21459.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><bold>Gene</bold></th><th colspan="3" valign="bottom"><bold>RPKM</bold></th></tr><tr><th><bold>Replicate 1</bold></th><th><bold>Replicate 2</bold></th><th><bold>Average</bold></th></tr></thead><tbody><tr><td style="author-callout-style-b1"><italic>LGR4</italic></td><td style="author-callout-style-b1">160.61</td><td style="author-callout-style-b1">174.69</td><td style="author-callout-style-b1">167.65</td></tr><tr><td style="author-callout-style-b1"><italic>LGR5</italic></td><td style="author-callout-style-b1">0.02</td><td style="author-callout-style-b1">0.00</td><td style="author-callout-style-b1">0.01</td></tr><tr><td style="author-callout-style-b1"><italic>LGR6</italic></td><td style="author-callout-style-b1">0.02</td><td style="author-callout-style-b1">0.00</td><td style="author-callout-style-b1">0.01</td></tr><tr><td style="author-callout-style-b3"><italic>ZNRF3</italic></td><td style="author-callout-style-b3">30.90</td><td style="author-callout-style-b3">33.30</td><td style="author-callout-style-b3">32.10</td></tr><tr><td style="author-callout-style-b3"><italic>RNF43</italic></td><td style="author-callout-style-b3">0.12</td><td style="author-callout-style-b3">0.08</td><td style="author-callout-style-b3">0.10</td></tr><tr><td style="author-callout-style-b1"><italic>LRP5</italic></td><td style="author-callout-style-b1">55.92</td><td style="author-callout-style-b1">64.38</td><td style="author-callout-style-b1">60.15</td></tr><tr><td style="author-callout-style-b1"><italic>LRP6</italic></td><td style="author-callout-style-b1">109.51</td><td style="author-callout-style-b1">121.08</td><td style="author-callout-style-b1">115.30</td></tr><tr><td style="author-callout-style-b3"><italic>FZD1</italic></td><td style="author-callout-style-b3">19.57</td><td style="author-callout-style-b3">18.85</td><td style="author-callout-style-b3">19.21</td></tr><tr><td style="author-callout-style-b3"><italic>FZD2</italic></td><td style="author-callout-style-b3">19.56</td><td style="author-callout-style-b3">21.09</td><td style="author-callout-style-b3">20.33</td></tr><tr><td style="author-callout-style-b3"><italic>FZD3</italic></td><td style="author-callout-style-b3">48.02</td><td style="author-callout-style-b3">55.82</td><td style="author-callout-style-b3">51.92</td></tr><tr><td style="author-callout-style-b3"><italic>FZD4</italic></td><td style="author-callout-style-b3">19.60</td><td style="author-callout-style-b3">22.10</td><td style="author-callout-style-b3">20.85</td></tr><tr><td style="author-callout-style-b3"><italic>FZD5</italic></td><td style="author-callout-style-b3">31.85</td><td style="author-callout-style-b3">34.52</td><td style="author-callout-style-b3">33.19</td></tr><tr><td style="author-callout-style-b3"><italic>FZD6</italic></td><td style="author-callout-style-b3">33.53</td><td style="author-callout-style-b3">31.95</td><td style="author-callout-style-b3">32.74</td></tr><tr><td style="author-callout-style-b3"><italic>FZD7</italic></td><td style="author-callout-style-b3">13.89</td><td style="author-callout-style-b3">14.89</td><td style="author-callout-style-b3">14.39</td></tr><tr><td style="author-callout-style-b3"><italic>FZD8</italic></td><td style="author-callout-style-b3">4.02</td><td style="author-callout-style-b3">4.48</td><td style="author-callout-style-b3">4.25</td></tr><tr><td style="author-callout-style-b3"><italic>FZD9</italic></td><td style="author-callout-style-b3">3.66</td><td style="author-callout-style-b3">2.80</td><td style="author-callout-style-b3">3.23</td></tr><tr><td style="author-callout-style-b3"><italic>FZD10</italic></td><td style="author-callout-style-b3">10.40</td><td style="author-callout-style-b3">9.86</td><td style="author-callout-style-b3">10.13</td></tr><tr><td style="author-callout-style-b1"><italic>DVL1</italic></td><td style="author-callout-style-b1">73.91</td><td style="author-callout-style-b1">69.61</td><td style="author-callout-style-b1">71.76</td></tr><tr><td style="author-callout-style-b1"><italic>DVL2</italic></td><td style="author-callout-style-b1">51.74</td><td style="author-callout-style-b1">48.80</td><td style="author-callout-style-b1">50.27</td></tr><tr><td style="author-callout-style-b1"><italic>DVL3</italic></td><td style="author-callout-style-b1">88.84</td><td style="author-callout-style-b1">90.25</td><td style="author-callout-style-b1">89.54</td></tr><tr><td style="author-callout-style-b3"><italic>APC</italic></td><td style="author-callout-style-b3">80.47</td><td style="author-callout-style-b3">85.22</td><td style="author-callout-style-b3">82.84</td></tr><tr><td style="author-callout-style-b3"><italic>APC2</italic></td><td style="author-callout-style-b3">2.94</td><td style="author-callout-style-b3">3.69</td><td style="author-callout-style-b3">3.32</td></tr><tr><td style="author-callout-style-b1"><italic>AXIN1</italic></td><td style="author-callout-style-b1">55.97</td><td style="author-callout-style-b1">54.07</td><td style="author-callout-style-b1">55.02</td></tr><tr><td style="author-callout-style-b1"><italic>AXIN2</italic></td><td style="author-callout-style-b1">10.04</td><td style="author-callout-style-b1">12.51</td><td style="author-callout-style-b1">11.27</td></tr><tr><td style="author-callout-style-b3"><italic>CSNK1A1</italic></td><td style="author-callout-style-b3">111.18</td><td style="author-callout-style-b3">109.57</td><td style="author-callout-style-b3">110.37</td></tr><tr><td style="author-callout-style-b1"><italic>GSK3A</italic></td><td style="author-callout-style-b1">75.97</td><td style="author-callout-style-b1">69.21</td><td style="author-callout-style-b1">72.59</td></tr><tr><td style="author-callout-style-b1"><italic>GSK3B</italic></td><td style="author-callout-style-b1">62.79</td><td style="author-callout-style-b1">69.98</td><td style="author-callout-style-b1">66.39</td></tr><tr><td style="author-callout-style-b3"><italic>TCF7L2</italic></td><td style="author-callout-style-b3">23.89</td><td style="author-callout-style-b3">27.69</td><td style="author-callout-style-b3">25.79</td></tr><tr><td style="author-callout-style-b3"><italic>LEF1</italic></td><td style="author-callout-style-b3">12.34</td><td style="author-callout-style-b3">14.80</td><td style="author-callout-style-b3">13.57</td></tr><tr><td style="author-callout-style-b1"><italic>CTNNB1</italic></td><td style="author-callout-style-b1">324.05</td><td style="author-callout-style-b1">308.53</td><td style="author-callout-style-b1">316.29</td></tr><tr><td style="author-callout-style-b3"><italic>CREBBP</italic></td><td style="author-callout-style-b3">141.92</td><td style="author-callout-style-b3">165.58</td><td style="author-callout-style-b3">153.75</td></tr><tr><td style="author-callout-style-b1"><italic>PIGL</italic></td><td style="author-callout-style-b1">4.07</td><td style="author-callout-style-b1">4.51</td><td style="author-callout-style-b1">4.29</td></tr><tr><td style="author-callout-style-b3"><italic>GPC4</italic></td><td style="author-callout-style-b3">209.39</td><td style="author-callout-style-b3">229.86</td><td style="author-callout-style-b3">219.63</td></tr><tr><td style="author-callout-style-b3"><italic>GPC6</italic></td><td style="author-callout-style-b3">13.88</td><td style="author-callout-style-b3">14.90</td><td style="author-callout-style-b3">14.39</td></tr><tr><td style="author-callout-style-b1"><italic>TFAP4</italic></td><td style="author-callout-style-b1">36.99</td><td style="author-callout-style-b1">41.94</td><td style="author-callout-style-b1">39.46</td></tr><tr><td style="author-callout-style-b3"><italic>SERBP1</italic></td><td style="author-callout-style-b3">721.65</td><td style="author-callout-style-b3">698.99</td><td style="author-callout-style-b3">710.32</td></tr><tr><td style="author-callout-style-b1"><italic>HUWE1</italic></td><td style="author-callout-style-b1">631.58</td><td style="author-callout-style-b1">777.06</td><td style="author-callout-style-b1">704.32</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-21459-v4.xml">elife-21459-v4.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.21459.005</object-id><label>Table 1.</label><caption><p>Relative gene expression level of selected WNT pathway regulators in HAP1 cells. RPKM values from duplicate RNAseq datasets generated as described in Materials and methods from two different passages of WT HAP1 cells are shown. Groups of paralogues and genes with similar&#160;functions are shaded in alternating colors to facilitate comparisons.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.21459.005">http://dx.doi.org/10.7554/eLife.21459.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><bold>Gene</bold></th><th colspan="3" valign="bottom"><bold>RPKM</bold></th></tr><tr><th><bold>Replicate 1</bold></th><th><bold>Replicate 2</bold></th><th><bold>Average</bold></th></tr></thead><tbody><tr><td style="author-callout-style-b1"><italic>LGR4</italic></td><td style="author-callout-style-b1">160.61</td><td style="author-callout-style-b1">174.69</td><td style="author-callout-style-b1">167.65</td></tr><tr><td style="author-callout-style-b1"><italic>LGR5</italic></td><td style="author-callout-style-b1">0.02</td><td style="author-callout-style-b1">0.00</td><td style="author-callout-style-b1">0.01</td></tr><tr><td style="author-callout-style-b1"><italic>LGR6</italic></td><td style="author-callout-style-b1">0.02</td><td style="author-callout-style-b1">0.00</td><td style="author-callout-style-b1">0.01</td></tr><tr><td style="author-callout-style-b3"><italic>ZNRF3</italic></td><td style="author-callout-style-b3">30.90</td><td style="author-callout-style-b3">33.30</td><td style="author-callout-style-b3">32.10</td></tr><tr><td style="author-callout-style-b3"><italic>RNF43</italic></td><td style="author-callout-style-b3">0.12</td><td style="author-callout-style-b3">0.08</td><td style="author-callout-style-b3">0.10</td></tr><tr><td style="author-callout-style-b1"><italic>LRP5</italic></td><td style="author-callout-style-b1">55.92</td><td style="author-callout-style-b1">64.38</td><td style="author-callout-style-b1">60.15</td></tr><tr><td style="author-callout-style-b1"><italic>LRP6</italic></td><td style="author-callout-style-b1">109.51</td><td style="author-callout-style-b1">121.08</td><td style="author-callout-style-b1">115.30</td></tr><tr><td style="author-callout-style-b3"><italic>FZD1</italic></td><td style="author-callout-style-b3">19.57</td><td style="author-callout-style-b3">18.85</td><td style="author-callout-style-b3">19.21</td></tr><tr><td style="author-callout-style-b3"><italic>FZD2</italic></td><td style="author-callout-style-b3">19.56</td><td style="author-callout-style-b3">21.09</td><td style="author-callout-style-b3">20.33</td></tr><tr><td style="author-callout-style-b3"><italic>FZD3</italic></td><td style="author-callout-style-b3">48.02</td><td style="author-callout-style-b3">55.82</td><td style="author-callout-style-b3">51.92</td></tr><tr><td style="author-callout-style-b3"><italic>FZD4</italic></td><td style="author-callout-style-b3">19.60</td><td style="author-callout-style-b3">22.10</td><td style="author-callout-style-b3">20.85</td></tr><tr><td style="author-callout-style-b3"><italic>FZD5</italic></td><td style="author-callout-style-b3">31.85</td><td style="author-callout-style-b3">34.52</td><td style="author-callout-style-b3">33.19</td></tr><tr><td style="author-callout-style-b3"><italic>FZD6</italic></td><td style="author-callout-style-b3">33.53</td><td style="author-callout-style-b3">31.95</td><td style="author-callout-style-b3">32.74</td></tr><tr><td style="author-callout-style-b3"><italic>FZD7</italic></td><td style="author-callout-style-b3">13.89</td><td style="author-callout-style-b3">14.89</td><td style="author-callout-style-b3">14.39</td></tr><tr><td style="author-callout-style-b3"><italic>FZD8</italic></td><td style="author-callout-style-b3">4.02</td><td style="author-callout-style-b3">4.48</td><td style="author-callout-style-b3">4.25</td></tr><tr><td style="author-callout-style-b3"><italic>FZD9</italic></td><td style="author-callout-style-b3">3.66</td><td style="author-callout-style-b3">2.80</td><td style="author-callout-style-b3">3.23</td></tr><tr><td style="author-callout-style-b3"><italic>FZD10</italic></td><td style="author-callout-style-b3">10.40</td><td style="author-callout-style-b3">9.86</td><td style="author-callout-style-b3">10.13</td></tr><tr><td style="author-callout-style-b1"><italic>DVL1</italic></td><td style="author-callout-style-b1">73.91</td><td style="author-callout-style-b1">69.61</td><td style="author-callout-style-b1">71.76</td></tr><tr><td style="author-callout-style-b1"><italic>DVL2</italic></td><td style="author-callout-style-b1">51.74</td><td style="author-callout-style-b1">48.80</td><td style="author-callout-style-b1">50.27</td></tr><tr><td style="author-callout-style-b1"><italic>DVL3</italic></td><td style="author-callout-style-b1">88.84</td><td style="author-callout-style-b1">90.25</td><td style="author-callout-style-b1">89.54</td></tr><tr><td style="author-callout-style-b3"><italic>APC</italic></td><td style="author-callout-style-b3">80.47</td><td style="author-callout-style-b3">85.22</td><td style="author-callout-style-b3">82.84</td></tr><tr><td style="author-callout-style-b3"><italic>APC2</italic></td><td style="author-callout-style-b3">2.94</td><td style="author-callout-style-b3">3.69</td><td style="author-callout-style-b3">3.32</td></tr><tr><td style="author-callout-style-b1"><italic>AXIN1</italic></td><td style="author-callout-style-b1">55.97</td><td style="author-callout-style-b1">54.07</td><td style="author-callout-style-b1">55.02</td></tr><tr><td style="author-callout-style-b1"><italic>AXIN2</italic></td><td style="author-callout-style-b1">10.04</td><td style="author-callout-style-b1">12.51</td><td style="author-callout-style-b1">11.27</td></tr><tr><td style="author-callout-style-b3"><italic>CSNK1A1</italic></td><td style="author-callout-style-b3">111.18</td><td style="author-callout-style-b3">109.57</td><td style="author-callout-style-b3">110.37</td></tr><tr><td style="author-callout-style-b1"><italic>GSK3A</italic></td><td style="author-callout-style-b1">75.97</td><td style="author-callout-style-b1">69.21</td><td style="author-callout-style-b1">72.59</td></tr><tr><td style="author-callout-style-b1"><italic>GSK3B</italic></td><td style="author-callout-style-b1">62.79</td><td style="author-callout-style-b1">69.98</td><td style="author-callout-style-b1">66.39</td></tr><tr><td style="author-callout-style-b3"><italic>TCF7L2</italic></td><td style="author-callout-style-b3">23.89</td><td style="author-callout-style-b3">27.69</td><td style="author-callout-style-b3">25.79</td></tr><tr><td style="author-callout-style-b3"><italic>LEF1</italic></td><td style="author-callout-style-b3">12.34</td><td style="author-callout-style-b3">14.80</td><td style="author-callout-style-b3">13.57</td></tr><tr><td style="author-callout-style-b1"><italic>CTNNB1</italic></td><td style="author-callout-style-b1">324.05</td><td style="author-callout-style-b1">308.53</td><td style="author-callout-style-b1">316.29</td></tr><tr><td style="author-callout-style-b3"><italic>CREBBP</italic></td><td style="author-callout-style-b3">141.92</td><td style="author-callout-style-b3">165.58</td><td style="author-callout-style-b3">153.75</td></tr><tr><td style="author-callout-style-b1"><italic>PIGL</italic></td><td style="author-callout-style-b1">4.07</td><td style="author-callout-style-b1">4.51</td><td style="author-callout-style-b1">4.29</td></tr><tr><td style="author-callout-style-b3"><italic>GPC4</italic></td><td style="author-callout-style-b3">209.39</td><td style="author-callout-style-b3">229.86</td><td style="author-callout-style-b3">219.63</td></tr><tr><td style="author-callout-style-b3"><italic>GPC6</italic></td><td style="author-callout-style-b3">13.88</td><td style="author-callout-style-b3">14.90</td><td style="author-callout-style-b3">14.39</td></tr><tr><td style="author-callout-style-b1"><italic>TFAP4</italic></td><td style="author-callout-style-b1">36.99</td><td style="author-callout-style-b1">41.94</td><td style="author-callout-style-b1">39.46</td></tr><tr><td style="author-callout-style-b3"><italic>SERBP1</italic></td><td style="author-callout-style-b3">721.65</td><td style="author-callout-style-b3">698.99</td><td style="author-callout-style-b3">710.32</td></tr><tr><td style="author-callout-style-b1"><italic>HUWE1</italic></td><td style="author-callout-style-b1">631.58</td><td style="author-callout-style-b1">777.06</td><td style="author-callout-style-b1">704.32</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-21477-v2.xml">elife-21477-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.21477.015</object-id><label>Table 1.</label><caption><p>Overview of crosslinks formed by ISWI<sub>H483B.</sub></p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.21477.015">http://dx.doi.org/10.7554/eLife.21477.015</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">ID</th><th rowspan="2" valign="top">Mass (D a)</th><th rowspan="2" valign="top">Error (ppm)</th><th colspan="2" valign="top">Bpa peptide</th><th colspan="2" valign="top">Target peptide</th></tr><tr><th valign="top">Sequence<sup>*,&#8224;</sup></th><th valign="top">Site</th><th valign="top">Sequence<sup>&#8224;</sup></th><th valign="top">Site</th></tr></thead><tbody><tr><td valign="top">XL1</td><td valign="top">3220.4946</td><td valign="top">&#8722;1.9</td><td valign="top">LDGQTPBEDRNR</td><td valign="top">483</td><td valign="top">QTEIFT<underline>HF</underline>M<sup>ox</sup>TNSAK<sup>&#8225;</sup></td><td valign="top">59&#8211;60</td></tr><tr><td valign="top">XL2</td><td valign="top">3204.5056</td><td valign="top">&#8722;3.7</td><td valign="top">LDGQTPBEDRNR</td><td valign="top">483</td><td valign="top">QTEIFTH<underline>FM</underline>TNSAK<sup>&#8225;</sup></td><td valign="top">60&#8211;61</td></tr><tr><td valign="top">XL3</td><td valign="top">2950.3474</td><td valign="top">&#8722;1.0</td><td valign="top">LDGQTPBEDR</td><td valign="top">483</td><td valign="top">QT<underline>EIFTH</underline>FM<sup>ox</sup>TNSAK<sup>&#8225;</sup></td><td valign="top">55&#8211;59</td></tr><tr><td valign="top">XL4</td><td valign="top">2934.3571</td><td valign="top">&#8722;2.6</td><td valign="top">LDGQTPBEDR</td><td valign="top">483</td><td valign="top">QTEIFT<underline>HFM</underline>TNSAK<sup>&#8225;</sup></td><td valign="top">59&#8211;61</td></tr><tr><td valign="top">XL5</td><td valign="top">2207.0968</td><td valign="top">+0.2</td><td valign="top">LDGQTPBEDRNR</td><td valign="top">483</td><td valign="top">SP<underline>TKPK<sup>&#8225;</sup></underline></td><td valign="top">69&#8211;72</td></tr><tr><td valign="top">XL6</td><td valign="top">1936.9645</td><td valign="top">&#8722;6.1</td><td valign="top">LDGQTPBEDR</td><td valign="top">483</td><td valign="top">SPTK<underline>PK<sup>&#8225;</sup></underline></td><td valign="top">71&#8211;72</td></tr><tr><td valign="top">XL7</td><td valign="top">1736.8594</td><td valign="top">&#8722;6.1</td><td valign="top">LDGQTPBEDR</td><td valign="top">483</td><td valign="top">GR<underline>P</underline>K</td><td valign="top">75</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p><sup>*</sup>B symbolizes Bpa.</p></fn><fn id="tblfn2"><p><sup>&#8224;</sup>Crosslinked amino acids are underlined; <sup>ox</sup>indicates oxidized methionine (+15.9949 Da).</p></fn><fn id="tblfn3"><p><sup>&#8225;</sup>Precise attachment sites not distinguishable from data.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-21477-v2.xml">elife-21477-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.21477.016</object-id><label>Table 2.</label><caption><p>Overview of H4-tail mediated crosslinks.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.21477.016">http://dx.doi.org/10.7554/eLife.21477.016</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">ID</th><th rowspan="2" valign="top">Reliability</th><th rowspan="2" valign="top">H4</th><th rowspan="2" valign="top">Remodeler construct</th><th rowspan="2" valign="top">Mass (Da)</th><th rowspan="2" valign="top">Error (ppm)</th><th colspan="2" valign="top">H4 peptide</th><th colspan="2" valign="top">Remodeler peptide</th></tr><tr><th valign="top">Sequence<sup>*</sup></th><th valign="top">Site</th><th valign="top">Sequence<sup>&#8224;</sup></th><th valign="top">Site</th></tr></thead><tbody><tr><td valign="top">XL11</td><td valign="top">high</td><td valign="top">nucleosomal</td><td valign="top">ISWI<sub>WT</sub></td><td valign="top">2034.8571</td><td valign="top">&#8722;0.2</td><td valign="top">XGR</td><td valign="top">1</td><td valign="top">QIQEFN<underline>M</underline>DNSAK</td><td valign="top">495</td></tr><tr><td valign="top">XL12</td><td valign="top">high</td><td valign="top">nucleosomal</td><td valign="top">SNF2H</td><td valign="top">2251.9753</td><td valign="top">&#8722;0.4</td><td valign="top">GXGK</td><td valign="top">10</td><td valign="top">VLDILEDYC<underline>M</underline>WR</td><td valign="top">520</td></tr><tr><td valign="top">XL13a</td><td valign="top">high</td><td valign="top">peptide</td><td valign="top">ISWI<sub>26-648</sub></td><td valign="top">1648.7601</td><td valign="top">&#8722;0.4</td><td valign="top">BGR</td><td valign="top">1</td><td valign="top">LDGQT<underline>P</underline>HEDR</td><td valign="top">482</td></tr><tr><td valign="top">XL13b</td><td valign="top">high</td><td valign="top">peptide</td><td valign="top">ISWI<sub>26-648</sub></td><td valign="top">1918.9052</td><td valign="top">&#8722;0.9</td><td valign="top">BGR</td><td valign="top">1</td><td valign="top">LDGQT<underline>P</underline>HEDRNR</td><td valign="top">482</td></tr><tr><td valign="top">XL13c</td><td valign="top">high</td><td valign="top">peptide</td><td valign="top">ISWI<sub>26-648</sub></td><td valign="top">3340.5374</td><td valign="top">&#8722;2.8</td><td valign="top">BGR</td><td valign="top">1</td><td valign="top">LDGQT<underline>P</underline>HEDRNRQIQEFNMDNSAK</td><td valign="top">482</td></tr><tr><td valign="top">XL14</td><td valign="top">medium</td><td valign="top">nucleosomal</td><td valign="top">SNF2H</td><td valign="top">2222.9624</td><td valign="top">&#8722;1.5</td><td valign="top">XGR</td><td valign="top">1</td><td valign="top">VLDILEDY<underline>CMWR<sup>&#8225;</sup></underline></td><td valign="top">519&#8211;22</td></tr><tr><td valign="top">XL15</td><td valign="top">medium</td><td valign="top">peptide</td><td valign="top">ISWI<sub>26-648</sub></td><td valign="top">1257.6261</td><td valign="top">+2.3</td><td valign="top">BGR</td><td valign="top">1</td><td valign="top"><underline>M</underline>VIQGGR</td><td valign="top">578</td></tr><tr><td valign="top">XL16</td><td valign="top">medium</td><td valign="top">peptide</td><td valign="top">ISWI<sub>26-648</sub></td><td valign="top">1424.7832</td><td valign="top">&#8722;3.9</td><td valign="top">BGR</td><td valign="top">1</td><td valign="top">IVE<underline>R</underline>AEVK</td><td valign="top">568</td></tr><tr><td valign="top">XL17</td><td valign="top">medium</td><td valign="top">peptide</td><td valign="top">ISWI<sub>26-648</sub></td><td valign="top">1453.7998</td><td valign="top">&#8722;4.6</td><td valign="top">GBGK</td><td valign="top">10</td><td valign="top">IVE<underline>R</underline>AEVK</td><td valign="top">568</td></tr></tbody></table><table-wrap-foot><fn id="tblfn4"><p><sup>*</sup>B symbolizes Bpa; X symbolizes Benzophenone-labeled cysteine.</p></fn><fn id="tblfn5"><p><sup>&#8224;</sup>Crosslinked amino acids are underlined.</p></fn><fn id="tblfn6"><p><sup>&#8225;</sup>Precise attachment sites not distinguishable from data.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-22069-v2.xml">elife-22069-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.22069.005</object-id><label>Table 2.</label><caption><p>Number and origin of blood meals according to the fly species (Fsp), park and climatic season.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22069.005">http://dx.doi.org/10.7554/eLife.22069.005</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3" rowspan="3"/><th colspan="23">Number of identified blood meals by fly species (Fsp)</th></tr><tr><th colspan="11">Moukalaba-Doudou</th><th colspan="12">Lop&#233;</th></tr><tr><th colspan="5">Rainy season</th><th colspan="6">Dry season</th><th colspan="5">Rainy season</th><th colspan="7">Dry season</th></tr></thead><tbody><tr><td>Taxonomic group/Order/Family</td><td>Host species</td><td>N&#176; Identified</td><td>Fsp1</td><td>Fsp2</td><td>Fsp3</td><td>Fsp4</td><td>Fsp5</td><td>Fsp1</td><td>Fsp2</td><td>Fsp3</td><td>Fsp4</td><td>Fsp5</td><td>Fsp8</td><td>Fsp1</td><td>Fsp2</td><td>Fsp3</td><td>Fsp4</td><td>Fsp5</td><td>Fsp1</td><td>Fsp2</td><td>Fsp3</td><td>Fsp4</td><td>Fsp5</td><td>Fsp6</td><td>Fsp7</td></tr><tr><td>Mammals</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<bold>Artiodactyla</bold></td><td/><td><bold>295</bold></td><td/><td>1</td><td>4</td><td>3</td><td>8</td><td>1</td><td>1</td><td>7</td><td>1</td><td>14</td><td/><td/><td>2</td><td>1</td><td/><td>2</td><td>3</td><td>3</td><td>4</td><td>1</td><td>3</td><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Bovidae</td><td><italic>Cephalophus</italic> <break/><italic>silvicultor</italic></td><td>65</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>kobus</italic> <break/><italic>ellipsiprymnus</italic></td><td>4</td><td/><td/><td/><td/><td>3</td><td/><td/><td/><td/><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>Syncerus caffer</italic></td><td>126</td><td>3</td><td/><td>5</td><td/><td>7</td><td>1</td><td>1</td><td>3</td><td>2</td><td>9</td><td/><td>2</td><td>1</td><td>10</td><td>5</td><td>7</td><td>1</td><td>2</td><td>8</td><td>1</td><td>6</td><td/><td>1</td></tr><tr><td/><td><italic>Tragelaphus spekii</italic></td><td>95</td><td/><td/><td/><td/><td/><td/><td/><td>1</td><td/><td>6</td><td/><td>4</td><td>1</td><td>5</td><td>1</td><td>3</td><td>7</td><td>6</td><td>9</td><td>4</td><td>12</td><td>1</td><td/></tr><tr><td>&#8195;&#8195;&#8195;Hippopotamidae</td><td><italic>Hippopotamus amphibius</italic></td><td>2</td><td/><td/><td/><td/><td>1</td><td/><td/><td/><td/><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Suidae</td><td><italic>Potamochoerus</italic> <break/><italic>porcus</italic></td><td>3</td><td/><td/><td/><td/><td>1</td><td/><td/><td/><td/><td>2</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<bold>Carnivora</bold></td><td/><td><bold>1</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Herpestidae</td><td><italic>Herpestinae sp</italic></td><td>1</td><td/><td/><td/><td/><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<bold>Primates</bold></td><td/><td><bold>67</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Hominidae</td><td><italic>Gorilla gorilla</italic></td><td>3</td><td/><td/><td/><td/><td>2</td><td/><td/><td/><td/><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>Homo sapiens</italic></td><td>64</td><td/><td/><td>1</td><td>1</td><td>13</td><td/><td/><td>2</td><td/><td>22</td><td>1</td><td/><td>3</td><td/><td>1</td><td>2</td><td/><td/><td>1</td><td/><td>4</td><td/><td>1</td></tr><tr><td>&#8195;&#8195;<bold>Proboscidae</bold></td><td/><td><bold>10</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Elephantidae</td><td><italic>Loxodonta cyclotis</italic></td><td>10</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td>7</td><td/><td/><td/><td>1</td><td/><td>2</td><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>Reptiles</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<bold>Crocodilia</bold></td><td/><td><bold>23</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td>3</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Crocodylidae</td><td><italic>Crocodylus niloticus</italic></td><td>3</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>Mecistops</italic> <break/><italic>cataphractus</italic></td><td>19</td><td/><td/><td>1</td><td/><td>1</td><td/><td/><td/><td/><td>6</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>Osteolaemus tetraspis</italic></td><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<bold>Squamata</bold></td><td/><td><bold>12</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Pythonidae</td><td><italic>Python sebae</italic></td><td>8</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td>2</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Varanidae</td><td><italic>Varanus sp</italic></td><td>4</td><td/><td/><td/><td/><td/><td/><td/><td>2</td><td/><td>1</td><td/><td/><td/><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<bold>Testudines</bold></td><td/><td><bold>16</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Testunidae</td><td><italic>Kinixys erosa</italic></td><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Pelomedusidae</td><td><italic>Pelusios castaneus</italic></td><td>3</td><td/><td/><td/><td/><td/><td/><td/><td>1</td><td>1</td><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>Pelusios chapini</italic></td><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>Pelusios marani</italic></td><td>11</td><td/><td/><td/><td/><td>3</td><td/><td/><td/><td/><td>8</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>Birds</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<bold>Ciconiformes</bold></td><td/><td><bold>4</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Ciconiidae</td><td><italic>Ciconia sp</italic></td><td>4</td><td/><td/><td/><td/><td>1</td><td/><td/><td/><td/><td>2</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td><bold>8 orders/12 families</bold></td><td><bold>20 species</bold></td><td><bold>428</bold></td><td>3</td><td>1</td><td>11</td><td>4</td><td>41</td><td>2</td><td>2</td><td>16</td><td>4</td><td>89</td><td>1</td><td>6</td><td>7</td><td>18</td><td>7</td><td>16</td><td>11</td><td>11</td><td>22</td><td>6</td><td>25</td><td>1</td><td>2</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>Fsp1 = <italic>Glossina caliginea</italic>; Fsp2 = <italic>G. fusca congolensis</italic>; Fsp3 = <italic>G. fuscipes fuscipes</italic>; Fsp4 = <italic>G. pallicera newsteadi</italic>; Fsp5 = <italic>G. palpalis palpalis</italic>; Fsp6 = <italic>Stomoxys calcitrans</italic>; Fsp7 = <italic>S. niger niger</italic>; Fsp8 = <italic>Chrysops</italic> sp.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-22069-v2.xml">elife-22069-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.22069.006</object-id><label>Table 3.</label><caption><p>Number and origin of blood meals according to the fly species (Fsp), park and climatic season.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22069.006">http://dx.doi.org/10.7554/eLife.22069.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3" rowspan="3"/><th colspan="14">Number of identified blood meals by fly species (Fsp)</th></tr><tr><th colspan="10">La L&#233;k&#233;di</th><th colspan="4">Ivindo</th></tr><tr><th colspan="5">Rainy season</th><th colspan="5">Dry season</th><th colspan="4">Dry season</th></tr></thead><tbody><tr><td>Taxonomic group/Order/Family</td><td>Host species</td><td>N&#176; Identified</td><td>Fsp1</td><td>Fsp3</td><td>Fsp4</td><td>Fsp5</td><td>Fsp6</td><td>Fsp1</td><td>Fsp2</td><td>Fsp3</td><td>Fsp4</td><td>Fsp5</td><td>Fsp1</td><td>Fsp2</td><td>Fsp3</td><td>Fsp5</td></tr><tr><td>Mammals</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<bold>Artiodactyla</bold></td><td/><td><bold>295</bold></td><td/><td>2</td><td>1</td><td>3</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Bovidae</td><td><italic>Cephalophus</italic> <break/><italic>silvicultor</italic></td><td>65</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>kobus</italic> <break/><italic>ellipsiprymnus</italic></td><td>4</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>Syncerus caffer</italic></td><td>126</td><td>2</td><td>1</td><td/><td>1</td><td/><td>6</td><td>10</td><td>8</td><td>8</td><td>13</td><td/><td/><td>1</td><td>1</td></tr><tr><td/><td><italic>Tragelaphus spekii</italic></td><td>95</td><td/><td>1</td><td/><td>3</td><td>1</td><td>2</td><td>8</td><td>5</td><td>5</td><td>9</td><td/><td>1</td><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Hippopotamidae</td><td><italic>Hippopotamus amphibius</italic></td><td>2</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Suidae</td><td><italic>Potamochoerus</italic> <break/><italic>porcus</italic></td><td>3</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<bold>Carnivora</bold></td><td/><td><bold>1</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Herpestidae</td><td><italic>Herpestinae sp</italic></td><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<bold>Primates</bold></td><td/><td><bold>67</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Hominidae</td><td><italic>Gorilla gorilla</italic></td><td>3</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>Homo sapiens</italic></td><td>64</td><td/><td/><td/><td>4</td><td/><td/><td/><td>1</td><td/><td>5</td><td/><td/><td>1</td><td>1</td></tr><tr><td>&#8195;&#8195;<bold>Proboscidae</bold></td><td/><td><bold>10</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Elephantidae</td><td><italic>Loxodonta cyclotis</italic></td><td>10</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>Reptiles</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<bold>Crocodilia</bold></td><td/><td><bold>23</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Crocodylidae</td><td><italic>Crocodylus niloticus</italic></td><td>3</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>Mecistops</italic> <break/><italic>cataphractus</italic></td><td>19</td><td/><td/><td/><td/><td/><td/><td>2</td><td>4</td><td>2</td><td>3</td><td/><td/><td/><td/></tr><tr><td/><td><italic>Osteolaemus</italic> <break/><italic>tetraspis</italic></td><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<bold>Squamata</bold></td><td/><td><bold>12</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Pythonidae</td><td><italic>Python sebae</italic></td><td>8</td><td/><td/><td/><td>2</td><td/><td/><td/><td>2</td><td/><td>2</td><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Varanidae</td><td><italic>Varanus sp</italic></td><td>4</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;<bold>Testudines</bold></td><td/><td><bold>16</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Testunidae</td><td><italic>Kinixys erosa</italic></td><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Pelomedusidae</td><td><italic>Pelusios</italic> <break/><italic>castaneus</italic></td><td>3</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>Pelusios chapini</italic></td><td>1</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>Pelusios marani</italic></td><td>11</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>Birds</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td><bold>&#8195;&#8195;Ciconiformes</bold></td><td/><td><bold>4</bold></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8195;&#8195;Ciconiidae</td><td><italic>Ciconia sp</italic></td><td>4</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td>1</td><td/><td/><td/></tr><tr><td><bold>8 orders/12 families</bold></td><td><bold>20 species</bold></td><td><bold>428</bold></td><td>2</td><td>4</td><td>1</td><td>13</td><td>1</td><td>8</td><td>20</td><td>20</td><td>15</td><td>32</td><td>1</td><td>1</td><td>2</td><td>2</td></tr></tbody></table><table-wrap-foot><fn id="tblfn2"><p>Fsp1 = <italic>Glossina caliginea</italic>; Fsp2 = <italic>G. fusca congolensis</italic>; Fsp3 = <italic>G. fuscipes fuscipes</italic>; Fsp4 = <italic>G. pallicera newsteadi</italic>; Fsp5 = <italic>G. palpalis palpalis</italic>; Fsp6 = <italic>Stomoxys calcitrans</italic>; Fsp7 = <italic>S. niger niger</italic>; Fsp8 = <italic>Chrysops</italic> sp.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-22152-v1.xml">elife-22152-v1.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.22152.019</object-id><label>Table 2.</label><caption><p>Voltage-clamp protocol parameters for the five ion type classes. Times are stated in units of ms, voltages in units of mV. See <xref ref-type="fig" rid="fig3s1">Figure 3&#8212;figure supplement 1</xref> for graphical description. Items <inline-formula><mml:math id="inf40"><mml:msubsup><mml:mi>T</mml:mi><mml:mi>A</mml:mi><mml:mo>*</mml:mo></mml:msubsup></mml:math></inline-formula> and <inline-formula><mml:math id="inf41"><mml:msubsup><mml:mi>T</mml:mi><mml:mi>B</mml:mi><mml:mo>*</mml:mo></mml:msubsup></mml:math></inline-formula> represent the starting and ending times, respectively, of the regions used for analysis (dashed areas in <xref ref-type="fig" rid="fig3">Figure 3</xref> of the main text, as well as <xref ref-type="fig" rid="fig3s1">Figure 3&#8212;figure supplement 1</xref>).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22152.019">http://dx.doi.org/10.7554/eLife.22152.019</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="6">Act</th><th>Ion type</th><th><inline-formula><mml:math id="inf42"><mml:msub><mml:mi>V</mml:mi><mml:mn>0</mml:mn></mml:msub></mml:math></inline-formula></th><th><inline-formula><mml:math id="inf43"><mml:msub><mml:mi>V</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:math></inline-formula></th><th><inline-formula><mml:math id="inf44"><mml:msub><mml:mi>V</mml:mi><mml:mn>2</mml:mn></mml:msub></mml:math></inline-formula></th><th><inline-formula><mml:math id="inf45"><mml:mrow><mml:mi mathvariant="normal">&#916;</mml:mi><mml:mo>&#8290;</mml:mo><mml:mi>V</mml:mi></mml:mrow></mml:math></inline-formula></th><th><inline-formula><mml:math id="inf46"><mml:msub><mml:mi>T</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:math></inline-formula></th><th><inline-formula><mml:math id="inf47"><mml:msub><mml:mi>T</mml:mi><mml:mn>2</mml:mn></mml:msub></mml:math></inline-formula></th><th><inline-formula><mml:math id="inf48"><mml:msub><mml:mi>T</mml:mi><mml:mn>3</mml:mn></mml:msub></mml:math></inline-formula></th><th><inline-formula><mml:math id="inf49"><mml:msubsup><mml:mi>T</mml:mi><mml:mi>A</mml:mi><mml:mo>*</mml:mo></mml:msubsup></mml:math></inline-formula></th><th><inline-formula><mml:math id="inf50"><mml:msubsup><mml:mi>T</mml:mi><mml:mi>B</mml:mi><mml:mo>*</mml:mo></mml:msubsup></mml:math></inline-formula></th><th/><th/><th/><th/></tr></thead><tbody><tr><td>Kv</td><td>&#8722;80</td><td>&#8722;80</td><td>70</td><td>10</td><td>100</td><td>500</td><td>100</td><td>100</td><td>700</td><td/><td/><td/><td/></tr><tr><td>Nav</td><td>&#8722;80</td><td>&#8722;80</td><td>70</td><td>10</td><td>20</td><td>50</td><td>30</td><td>18</td><td>100</td><td/><td/><td/><td/></tr><tr><td>Cav</td><td>&#8722;80</td><td>&#8722;80</td><td>70</td><td>10</td><td>100</td><td>500</td><td>100</td><td>98</td><td>700</td><td/><td/><td/><td/></tr><tr><td>KCa</td><td>&#8722;80</td><td>&#8722;80</td><td>70</td><td>10</td><td>100</td><td>500</td><td>100</td><td>95</td><td>605</td><td/><td/><td/><td/></tr><tr><td>Ih</td><td>&#8722;40</td><td>&#8722;150</td><td>0</td><td>10</td><td>100</td><td>2000</td><td>100</td><td>95</td><td>2105</td><td/><td/><td/><td/></tr><tr><td rowspan="6">Inact</td><td>Ion Type</td><td><inline-formula><mml:math id="inf51"><mml:msub><mml:mi>V</mml:mi><mml:mn>0</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf52"><mml:msub><mml:mi>V</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf53"><mml:msub><mml:mi>V</mml:mi><mml:mn>2</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf54"><mml:msub><mml:mi>V</mml:mi><mml:mn>3</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf55"><mml:mrow><mml:mi mathvariant="normal">&#916;</mml:mi><mml:mo>&#8290;</mml:mo><mml:mi>V</mml:mi></mml:mrow></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf56"><mml:msub><mml:mi>T</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf57"><mml:msub><mml:mi>T</mml:mi><mml:mn>2</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf58"><mml:msub><mml:mi>T</mml:mi><mml:mn>3</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf59"><mml:msub><mml:mi>T</mml:mi><mml:mn>4</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf60"><mml:msubsup><mml:mi>T</mml:mi><mml:mi>A</mml:mi><mml:mo>*</mml:mo></mml:msubsup></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf61"><mml:msubsup><mml:mi>T</mml:mi><mml:mi>B</mml:mi><mml:mo>*</mml:mo></mml:msubsup></mml:math></inline-formula></td><td/><td/></tr><tr><td>Kv</td><td>&#8722;80</td><td>&#8722;40</td><td>70</td><td>30</td><td>10</td><td>100</td><td>1500</td><td>50</td><td>100</td><td>1600</td><td>1700</td><td/><td/></tr><tr><td>Nav</td><td>&#8722;80</td><td>&#8722;40</td><td>70</td><td>30</td><td>10</td><td>100</td><td>1500</td><td>50</td><td>100</td><td>1580</td><td>1750</td><td/><td/></tr><tr><td>Cav</td><td>&#8722;80</td><td>&#8722;40</td><td>70</td><td>30</td><td>10</td><td>100</td><td>1500</td><td>50</td><td>100</td><td>1580</td><td>1750</td><td/><td/></tr><tr><td>KCa</td><td>&#8722;80</td><td>&#8722;40</td><td>70</td><td>30</td><td>10</td><td>100</td><td>1500</td><td>50</td><td>100</td><td>1595</td><td>1700</td><td/><td/></tr><tr><td>Ih</td><td>&#8722;40</td><td>&#8722;150</td><td>&#8722;40</td><td>&#8722;120</td><td>10</td><td>100</td><td>1000</td><td>300</td><td>100</td><td>1095</td><td>1405</td><td/><td/></tr><tr><td rowspan="6">Deact</td><td>Ion Type</td><td><inline-formula><mml:math id="inf62"><mml:msub><mml:mi>V</mml:mi><mml:mn>0</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf63"><mml:msub><mml:mi>V</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf64"><mml:msub><mml:mi>V</mml:mi><mml:mn>2</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf65"><mml:msub><mml:mi>V</mml:mi><mml:mn>3</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf66"><mml:mrow><mml:mi mathvariant="normal">&#916;</mml:mi><mml:mo>&#8290;</mml:mo><mml:mi>V</mml:mi></mml:mrow></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf67"><mml:msub><mml:mi>T</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf68"><mml:msub><mml:mi>T</mml:mi><mml:mn>2</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf69"><mml:msub><mml:mi>T</mml:mi><mml:mn>3</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf70"><mml:msub><mml:mi>T</mml:mi><mml:mn>4</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf71"><mml:msubsup><mml:mi>T</mml:mi><mml:mi>A</mml:mi><mml:mo>*</mml:mo></mml:msubsup></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf72"><mml:msubsup><mml:mi>T</mml:mi><mml:mi>B</mml:mi><mml:mo>*</mml:mo></mml:msubsup></mml:math></inline-formula></td><td/><td/></tr><tr><td>Kv</td><td>&#8722;80</td><td>70</td><td>&#8722;100</td><td>40</td><td>10</td><td>100</td><td>300</td><td>200</td><td>100</td><td>400</td><td>600</td><td/><td/></tr><tr><td>Nav</td><td>&#8722;80</td><td>70</td><td>&#8722;100</td><td>40</td><td>10</td><td>20</td><td>10</td><td>30</td><td>20</td><td>29</td><td>80</td><td/><td/></tr><tr><td>Cav</td><td>&#8722;80</td><td>70</td><td>&#8722;100</td><td>40</td><td>10</td><td>100</td><td>300</td><td>200</td><td>100</td><td>380</td><td>700</td><td/><td/></tr><tr><td>KCa</td><td>&#8722;80</td><td>70</td><td>&#8722;100</td><td>40</td><td>10</td><td>100</td><td>300</td><td>200</td><td>100</td><td>395</td><td>605</td><td/><td/></tr><tr><td>Ih</td><td>&#8722;40</td><td>&#8722;140</td><td>&#8722;110</td><td>0</td><td>10</td><td>100</td><td>1500</td><td>500</td><td>400</td><td>1595</td><td>2105</td><td/><td/></tr><tr><td rowspan="6">Ramp</td><td>Ion Type</td><td><inline-formula><mml:math id="inf73"><mml:msub><mml:mi>V</mml:mi><mml:mn>0</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf74"><mml:msub><mml:mi>V</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf75"><mml:msub><mml:mi>T</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf76"><mml:msub><mml:mi>T</mml:mi><mml:mn>2</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf77"><mml:msub><mml:mi>T</mml:mi><mml:mn>3</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf78"><mml:msub><mml:mi>T</mml:mi><mml:mn>4</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf79"><mml:msub><mml:mi>T</mml:mi><mml:mn>5</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf80"><mml:msub><mml:mi>T</mml:mi><mml:mn>6</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf81"><mml:msub><mml:mi>T</mml:mi><mml:mn>7</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf82"><mml:msub><mml:mi>T</mml:mi><mml:mn>8</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf83"><mml:msub><mml:mi>T</mml:mi><mml:mn>9</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf84"><mml:msubsup><mml:mi>T</mml:mi><mml:mi>A</mml:mi><mml:mo>*</mml:mo></mml:msubsup></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf85"><mml:msubsup><mml:mi>T</mml:mi><mml:mi>B</mml:mi><mml:mo>*</mml:mo></mml:msubsup></mml:math></inline-formula></td></tr><tr><td>Kv</td><td>&#8722;80</td><td>70</td><td>100</td><td>800</td><td>400</td><td>400</td><td>400</td><td>200</td><td>400</td><td>100</td><td>100</td><td>100</td><td>2800</td></tr><tr><td>Nav</td><td>&#8722;80</td><td>70</td><td>100</td><td>800</td><td>400</td><td>400</td><td>400</td><td>200</td><td>400</td><td>100</td><td>100</td><td>98</td><td>2800</td></tr><tr><td>Cav</td><td>&#8722;80</td><td>70</td><td>100</td><td>800</td><td>400</td><td>400</td><td>400</td><td>200</td><td>400</td><td>100</td><td>100</td><td>98</td><td>2800</td></tr><tr><td>KCa</td><td>&#8722;80</td><td>70</td><td>100</td><td>800</td><td>400</td><td>400</td><td>400</td><td>200</td><td>400</td><td>100</td><td>100</td><td>100</td><td>2800</td></tr><tr><td>Ih</td><td>&#8722;80</td><td>70</td><td>100</td><td>800</td><td>400</td><td>400</td><td>400</td><td>200</td><td>400</td><td>100</td><td>100</td><td>100</td><td>2800</td></tr><tr><td rowspan="6">AP</td><td>Ion Type</td><td><inline-formula><mml:math id="inf86"><mml:msub><mml:mi>T</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf87"><mml:msubsup><mml:mi>T</mml:mi><mml:mi>A</mml:mi><mml:mo>*</mml:mo></mml:msubsup></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf88"><mml:msubsup><mml:mi>T</mml:mi><mml:mi>B</mml:mi><mml:mo>*</mml:mo></mml:msubsup></mml:math></inline-formula></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>Kv</td><td>1800</td><td>100</td><td>1800</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>Nav</td><td>1800</td><td>98</td><td>1800</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>Cav</td><td>1800</td><td>98</td><td>1800</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>KCa</td><td>1800</td><td>95</td><td>1655</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>Ih</td><td>1800</td><td>95</td><td>1655</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-22352-v2.xml">elife-22352-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.22352.023</object-id><label>Table 3.</label><caption><p>Minimal spanning trees (MSTs) simulation parameters. 16 STG neuronal structures were simulated using MST analyses from the TREES toolbox (<xref ref-type="bibr" rid="bib20">Cuntz et al., 2010</xref>). Carrier point numbers (column 4) were tuned to generate MSTs with branch point numbers within 20% of the branch point counts in the actual neurons (column 2, gray shaded). MSTs were generated for balancing factors between 0&#160;and&#160;0.6, at increments of 0.1. Columns 5&#8211;18 show branch point numbers and cable lengths for each of these MSTs. The MSTs often had cable lengths within 20% of that measured in the actual neurons (column 3), but in some cases, the MSTs were not a close approximation of the actual cable length.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22352.023">http://dx.doi.org/10.7554/eLife.22352.023</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="bottom"><p><bold>Cell</bold></p> </th><th colspan="2" style="author-callout-style-b8" valign="bottom"><p><bold>Actual neuron</bold></p> </th><th colspan="15" valign="bottom"><p><bold>Minimum spanning trees</bold></p> </th></tr><tr><th rowspan="2" style="author-callout-style-b8" valign="bottom"><p><bold>Branch Points</bold></p> </th><th rowspan="2" style="author-callout-style-b8" valign="bottom"><p><bold>Cable length<break/>(x 10<sup>4</sup> &#181;m)</bold></p> </th><th rowspan="2" valign="bottom"><p><bold>Carrier Points</bold></p> </th><th colspan="7" valign="bottom"><p><bold>Branch point numbers</bold></p> </th><th colspan="7" valign="bottom"><p><bold>Cable length (x 10<sup>4</sup> &#181;m)</bold></p> </th></tr><tr><th valign="bottom"><p>0</p></th><th valign="bottom"><p>0.1</p></th><th valign="bottom"><p>0.2</p></th><th valign="bottom"><p>0.3</p></th><th valign="bottom"><p>0.4</p></th><th valign="bottom"><p>0.5</p></th><th valign="bottom"><p>0.6</p></th><th valign="bottom"><p><bold>0</bold></p> </th><th valign="bottom"><p>0.1</p></th><th valign="bottom"><p>0.2</p></th><th valign="bottom"><p>0.3</p></th><th valign="bottom"><p>0.4</p></th><th valign="bottom"><p>0.5</p></th><th valign="bottom"><p>0.6</p></th></tr></thead><tbody><tr><td valign="bottom"><p><bold>GM1</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>2671</p> </td><td style="author-callout-style-b8" valign="bottom"><p>6.1</p></td><td valign="bottom"><p>8500</p> </td><td valign="bottom"><p>2416</p> </td><td valign="bottom"><p>2477</p> </td><td valign="bottom"><p>2512</p> </td><td valign="bottom"><p>2576</p> </td><td valign="bottom"><p>2727</p> </td><td valign="bottom"><p>2916</p> </td><td valign="bottom"><p>2971</p> </td><td valign="bottom"><p>16.8</p></td><td valign="bottom"><p>17.2</p></td><td valign="bottom"><p>17.7</p></td><td valign="bottom"><p>18.2</p></td><td valign="bottom"><p>19.0</p></td><td valign="bottom"><p>19.9</p></td><td valign="bottom"><p>20.7</p></td></tr><tr><td valign="bottom"><p><bold>GM2</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>7125</p> </td><td style="author-callout-style-b8" valign="bottom"><p>8.9</p></td><td valign="bottom"><p>23000</p> </td><td valign="bottom"><p>6296</p> </td><td valign="bottom"><p>6789</p> </td><td valign="bottom"><p>7030</p> </td><td valign="bottom"><p>7271</p> </td><td valign="bottom"><p>7592</p> </td><td valign="bottom"><p>7773</p> </td><td valign="bottom"><p>7994</p> </td><td valign="bottom"><p>2.5</p></td><td valign="bottom"><p>2.6</p></td><td valign="bottom"><p>2.7</p></td><td valign="bottom"><p>2.8</p></td><td valign="bottom"><p>2.9</p></td><td valign="bottom"><p>3.1</p></td><td valign="bottom"><p>3.3</p></td></tr><tr><td valign="bottom"><p><bold>GM3</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>2675</p> </td><td style="author-callout-style-b8" valign="bottom"><p>4.6</p></td><td valign="bottom"><p>8800</p> </td><td valign="bottom"><p>2384</p> </td><td valign="bottom"><p>2537</p> </td><td valign="bottom"><p>2665</p> </td><td valign="bottom"><p>2749</p> </td><td valign="bottom"><p>2827</p> </td><td valign="bottom"><p>2959</p> </td><td valign="bottom"><p>3026</p> </td><td valign="bottom"><p>5.5</p></td><td valign="bottom"><p>5.7</p></td><td valign="bottom"><p>5.9</p></td><td valign="bottom"><p>6.1</p></td><td valign="bottom"><p>6.4</p></td><td valign="bottom"><p>6.8</p></td><td valign="bottom"><p>7.3</p></td></tr><tr><td valign="bottom"><p><bold>GM4</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>5984</p> </td><td style="author-callout-style-b8" valign="bottom"><p>10.4</p></td><td valign="bottom"><p>19000</p> </td><td valign="bottom"><p>5274</p> </td><td valign="bottom"><p>5574</p> </td><td valign="bottom"><p>5762</p> </td><td valign="bottom"><p>5969</p> </td><td valign="bottom"><p>6187</p> </td><td valign="bottom"><p>6379</p> </td><td valign="bottom"><p>6571</p> </td><td valign="bottom"><p>14.3</p></td><td valign="bottom"><p>14.7</p></td><td valign="bottom"><p>15.2</p></td><td valign="bottom"><p>15.8</p></td><td valign="bottom"><p>16.6</p></td><td valign="bottom"><p>17.5</p></td><td valign="bottom"><p>18.8</p></td></tr><tr><td valign="bottom"><p><bold>LG1</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>2002</p> </td><td style="author-callout-style-b8" valign="bottom"><p>2.0</p></td><td valign="bottom"><p>6500</p> </td><td valign="bottom"><p>1746</p> </td><td valign="bottom"><p>1860</p> </td><td valign="bottom"><p>1976</p> </td><td valign="bottom"><p>2092</p> </td><td valign="bottom"><p>2154</p> </td><td valign="bottom"><p>2209</p> </td><td valign="bottom"><p>2302</p> </td><td valign="bottom"><p>1.7</p></td><td valign="bottom"><p>1.8</p></td><td valign="bottom"><p>1.9</p></td><td valign="bottom"><p>1.9</p></td><td valign="bottom"><p>2.0</p></td><td valign="bottom"><p>2.1</p></td><td valign="bottom"><p>2.3</p></td></tr><tr><td valign="bottom"><p><bold>LG2</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>705</p></td><td style="author-callout-style-b8" valign="bottom"><p>1.0</p></td><td valign="bottom"><p>2300</p> </td><td valign="bottom"><p>641</p></td><td valign="bottom"><p>672</p></td><td valign="bottom"><p>696</p></td><td valign="bottom"><p>710</p></td><td valign="bottom"><p>750</p></td><td valign="bottom"><p>768</p></td><td valign="bottom"><p>815</p></td><td valign="bottom"><p>1.5</p></td><td valign="bottom"><p>1.5</p></td><td valign="bottom"><p>1.6</p></td><td valign="bottom"><p>1.7</p></td><td valign="bottom"><p>1.7</p></td><td valign="bottom"><p>1.8</p></td><td valign="bottom"><p>2.0</p></td></tr><tr><td valign="bottom"><p><bold>LG3</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>2460</p> </td><td style="author-callout-style-b8" valign="bottom"><p>4.3</p></td><td valign="bottom"><p>8000</p> </td><td valign="bottom"><p>2168</p> </td><td valign="bottom"><p>2302</p> </td><td valign="bottom"><p>2389</p> </td><td valign="bottom"><p>2498</p> </td><td valign="bottom"><p>2559</p> </td><td valign="bottom"><p>2612</p> </td><td valign="bottom"><p>2707</p> </td><td valign="bottom"><p>4.3</p></td><td valign="bottom"><p>4.4</p></td><td valign="bottom"><p>4.6</p></td><td valign="bottom"><p>4.8</p></td><td valign="bottom"><p>5.0</p></td><td valign="bottom"><p>5.3</p></td><td valign="bottom"><p>5.7</p></td></tr><tr><td valign="bottom"><p><bold>LG4</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>2311</p> </td><td style="author-callout-style-b8" valign="bottom"><p>4.9</p></td><td valign="bottom"><p>7500</p> </td><td valign="bottom"><p>2050</p> </td><td valign="bottom"><p>2189</p> </td><td valign="bottom"><p>2282</p> </td><td valign="bottom"><p>2346</p> </td><td valign="bottom"><p>2430</p> </td><td valign="bottom"><p>2502</p> </td><td valign="bottom"><p>2603</p> </td><td valign="bottom"><p>2.4</p></td><td valign="bottom"><p>2.4</p></td><td valign="bottom"><p>2.5</p></td><td valign="bottom"><p>2.6</p></td><td valign="bottom"><p>2.8</p></td><td valign="bottom"><p>2.9</p></td><td valign="bottom"><p>3.1</p></td></tr><tr><td valign="bottom"><p><bold>LP1</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>2015</p> </td><td style="author-callout-style-b8" valign="bottom"><p>4.1</p></td><td valign="bottom"><p>6550</p> </td><td valign="bottom"><p>1825</p> </td><td valign="bottom"><p>1957</p> </td><td valign="bottom"><p>1998</p> </td><td valign="bottom"><p>2059</p> </td><td valign="bottom"><p>2126</p> </td><td valign="bottom"><p>2195</p> </td><td valign="bottom"><p>2263</p> </td><td valign="bottom"><p>3.3</p></td><td valign="bottom"><p>3.3</p></td><td valign="bottom"><p>3.5</p></td><td valign="bottom"><p>3.6</p></td><td valign="bottom"><p>3.8</p></td><td valign="bottom"><p>4.0</p></td><td valign="bottom"><p>4.3</p></td></tr><tr><td valign="bottom"><p><bold>LP2</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>704</p></td><td style="author-callout-style-b8" valign="bottom"><p>1.8</p></td><td valign="bottom"><p>2330</p> </td><td valign="bottom"><p>649</p></td><td valign="bottom"><p>666</p></td><td valign="bottom"><p>706</p></td><td valign="bottom"><p>716</p></td><td valign="bottom"><p>748</p></td><td valign="bottom"><p>773</p></td><td valign="bottom"><p>788</p></td><td valign="bottom"><p>2.0</p></td><td valign="bottom"><p>2.1</p></td><td valign="bottom"><p>2.1</p></td><td valign="bottom"><p>2.2</p></td><td valign="bottom"><p>2.3</p></td><td valign="bottom"><p>2.5</p></td><td valign="bottom"><p>2.6</p></td></tr><tr><td valign="bottom"><p><bold>LP3</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>499</p></td><td style="author-callout-style-b8" valign="bottom"><p>1.4</p></td><td valign="bottom"><p>1705</p> </td><td valign="bottom"><p>452</p></td><td valign="bottom"><p>493</p></td><td valign="bottom"><p>509</p></td><td valign="bottom"><p>534</p></td><td valign="bottom"><p>531</p></td><td valign="bottom"><p>560</p></td><td valign="bottom"><p>574</p></td><td valign="bottom"><p>1.6</p></td><td valign="bottom"><p>1.7</p></td><td valign="bottom"><p>1.7</p></td><td valign="bottom"><p>1.8</p></td><td valign="bottom"><p>1.9</p></td><td valign="bottom"><p>2.0</p></td><td valign="bottom"><p>2.1</p></td></tr><tr><td valign="bottom"><p><bold>LP4</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>1708</p> </td><td style="author-callout-style-b8" valign="bottom"><p>2.4</p></td><td valign="bottom"><p>5650</p> </td><td valign="bottom"><p>1574</p> </td><td valign="bottom"><p>1639</p> </td><td valign="bottom"><p>1697</p> </td><td valign="bottom"><p>1743</p> </td><td valign="bottom"><p>1819</p> </td><td valign="bottom"><p>1883</p> </td><td valign="bottom"><p>1921</p> </td><td valign="bottom"><p>3.8</p></td><td valign="bottom"><p>3.9</p></td><td valign="bottom"><p>4.0</p></td><td valign="bottom"><p>4.2</p></td><td valign="bottom"><p>4.4</p></td><td valign="bottom"><p>4.6</p></td><td valign="bottom"><p>5.0</p></td></tr><tr><td valign="bottom"><p><bold>PD1</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>424</p></td><td style="author-callout-style-b8" valign="bottom"><p>1.9</p></td><td valign="bottom"><p>1400</p> </td><td valign="bottom"><p>370</p></td><td valign="bottom"><p>398</p></td><td valign="bottom"><p>422</p></td><td valign="bottom"><p>444</p></td><td valign="bottom"><p>442</p></td><td valign="bottom"><p>464</p></td><td valign="bottom"><p>501</p></td><td valign="bottom"><p>1.6</p></td><td valign="bottom"><p>1.7</p></td><td valign="bottom"><p>1.8</p></td><td valign="bottom"><p>1.8</p></td><td valign="bottom"><p>1.9</p></td><td valign="bottom"><p>2.0</p></td><td valign="bottom"><p>2.2</p></td></tr><tr><td valign="bottom"><p><bold>PD2</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>418</p></td><td style="author-callout-style-b8" valign="bottom"><p>1.5</p></td><td valign="bottom"><p>1400</p> </td><td valign="bottom"><p>383</p></td><td valign="bottom"><p>406</p></td><td valign="bottom"><p>426</p></td><td valign="bottom"><p>425</p></td><td valign="bottom"><p>455</p></td><td valign="bottom"><p>460</p></td><td valign="bottom"><p>477</p></td><td valign="bottom"><p>1.2</p></td><td valign="bottom"><p>1.2</p></td><td valign="bottom"><p>1.3</p></td><td valign="bottom"><p>1.3</p></td><td valign="bottom"><p>1.4</p></td><td valign="bottom"><p>1.5</p></td><td valign="bottom"><p>1.6</p></td></tr><tr><td valign="bottom"><p><bold>PD3</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>1597</p> </td><td style="author-callout-style-b8" valign="bottom"><p>3.0</p></td><td valign="bottom"><p>5000</p> </td><td valign="bottom"><p>1344</p> </td><td valign="bottom"><p>1462</p> </td><td valign="bottom"><p>1519</p> </td><td valign="bottom"><p>1587</p> </td><td valign="bottom"><p>1667</p> </td><td valign="bottom"><p>1724</p> </td><td valign="bottom"><p>1776</p> </td><td valign="bottom"><p>3.3</p></td><td valign="bottom"><p>3.4</p></td><td valign="bottom"><p>3.5</p></td><td valign="bottom"><p>3.7</p></td><td valign="bottom"><p>3.8</p></td><td valign="bottom"><p>4.1</p></td><td valign="bottom"><p>4.3</p></td></tr><tr><td valign="bottom"><p><bold>PD4</bold></p> </td><td style="author-callout-style-b8" valign="bottom"><p>410</p></td><td style="author-callout-style-b8" valign="bottom"><p>1.7</p></td><td valign="bottom"><p>1300</p> </td><td valign="bottom"><p>363</p></td><td valign="bottom"><p>389</p></td><td valign="bottom"><p>397</p></td><td valign="bottom"><p>396</p></td><td valign="bottom"><p>429</p></td><td valign="bottom"><p>442</p></td><td valign="bottom"><p>451</p></td><td valign="bottom"><p>1.2</p></td><td valign="bottom"><p>1.3</p></td><td valign="bottom"><p>1.3</p></td><td valign="bottom"><p>1.4</p></td><td valign="bottom"><p>1.4</p></td><td valign="bottom"><p>1.5</p></td><td valign="bottom"><p>1.6</p></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-22519-v1.xml">elife-22519-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.22519.010</object-id><label>Table 1.</label><caption><p>Molecular evolution analyses of the provirus containing hsaHTenv in its orthologs. (1) Divergence measured in substitutions per site. (2) Integration dates inferred from the divergence of the paired LTRs for human, gorilla and orangutan proviruses. Age is calculated assuming a human neutral substitution rate of 2.2 &#215; 10<sup>&#8722;9</sup> substitutions per site per year. (3) Pairwise dN/dS ratios for the <italic>env</italic> genes calculated using codeml (CodonFreq&#160;=&#160;F3&#160;&#215;&#160;4, Kappa and Omega estimated). (**) Significantly different from dN/dS&#160;=&#160;1 (p&lt;0.01).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22519.010">http://dx.doi.org/10.7554/eLife.22519.010</ext-link></p><p><supplementary-material id="SD2-data"><object-id pub-id-type="doi">10.7554/eLife.22519.011</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Likelihood ratio tests on dN/dS estimates for hsaHTenv and its orthologs. dN/dS ratios (&#969;) were estimated on a pairwise basis using codeml.</title><p>Likelihood ratio tests were performed comparing the log likelihood of the estimated &#969; (L<sub>1</sub>) to the log likelihood when &#969; was fixed to 1 (L<sub>0</sub>, neutral selection). The probability (P) of twice the difference (<italic>D</italic>&#160;=&#160;2(L<sub>1</sub>&#8722;L<sub>0</sub>)) was calculated using a chi-squared distribution (degrees of freedom&#160;=&#160;1). The single nucleotide frame-shift insertion in the gorilla sequence was artificially deleted in order to compare orthologous codons.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22519.011">http://dx.doi.org/10.7554/eLife.22519.011</ext-link></p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-22519-table1-data1-v1.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Locus</th><th rowspan="2">LTR divergence<sup>(1)</sup></th><th rowspan="2">Age (MY)<sup>(2)</sup></th><th colspan="2"><italic>Env</italic> dN/dS ratio<sup>(3)</sup></th></tr><tr><th>Gorilla</th><th>Orangutan</th></tr></thead><tbody><tr><td>Human</td><td>0.0317</td><td>7.20</td><td>1.4860</td><td>0.5184**</td></tr><tr><td>Gorilla</td><td>0.0414</td><td>9.41</td><td>&#8722;</td><td>0.7592</td></tr><tr><td>Orangutan</td><td>0.0855</td><td>19.43</td><td>0.7592</td><td>&#8722;</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-22679-v2.xml">elife-22679-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.22679.021</object-id><label>Table 2.</label><caption><p>Biochemical characterization of PriA, SubHisA2 and SubTrpF homologs.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22679.021">http://dx.doi.org/10.7554/eLife.22679.021</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Enzymes</th><th colspan="2" valign="top">In vivo activity</th><th colspan="6" valign="top">In vitro activity <sup>*</sup></th></tr><tr><th colspan="2" valign="top"><break/></th><th colspan="3" valign="top">ProFAR isomerase (HisA)</th><th colspan="3" valign="top">PRA isomerase (TrpF)</th></tr><tr><th>HisA</th><th>TrpF</th><th valign="top"><italic>K<sub>M</sub></italic> <break/>(&#181;M)</th><th valign="top"><italic>k<sub>cat</sub></italic> <break/>(s<sup>&#8722;1</sup>)</th><th valign="top"><italic>k<sub>cat</sub></italic>/<italic>K<sub>M</sub></italic> <break/>(s<sup>&#8722;1</sup>&#181;M<sup>&#8722;1</sup>)</th><th valign="top"><italic>K<sub>M</sub></italic> <break/>(&#181;M)</th><th valign="top"><italic>k<sub>cat</sub></italic> <break/>(s<sup>&#8722;1</sup>)</th><th valign="top"><italic>k<sub>cat</sub></italic>/<italic>K<sub>M</sub></italic> <break/>(s<sup>&#8722;1</sup>&#181;M<sup>&#8722;1</sup>)</th></tr></thead><tbody><tr><td>PriA_Org3_<italic>B. longum</italic></td><td>+</td><td>+</td><td>2.7&#160;&#177;&#160;0.5</td><td>0.4&#160;&#177;&#160;0.1</td><td>0.1</td><td>6.1&#160;&#177;&#160;0.1</td><td>2.1&#160;&#177;&#160;0.5</td><td>0.3</td></tr><tr><td>PriA_Org1_<italic>B. gallicum</italic></td><td>+</td><td>+</td><td>1.7&#160;&#177;&#160;0.3</td><td>0.3&#160;&#177;&#160;0.1</td><td>0.2</td><td>40&#160;&#177;&#160;9</td><td>3.5&#160;&#177;&#160;0.1</td><td>0.09</td></tr><tr><td>PriA_Org6_<italic>B. adolescentis</italic></td><td>+</td><td>+</td><td>17&#160;&#177;&#160;4.3</td><td>2.3&#160;&#177;&#160;0.01</td><td>0.1</td><td>21&#160;&#177;&#160;5</td><td>0.9&#160;&#177;&#160;0.2</td><td>0.04</td></tr><tr><td>PriA_Org15_<italic>A. urogenitalis</italic></td><td>+</td><td>+</td><td>4.0&#160;&#177;&#160;0.9</td><td>0.2&#160;&#177;&#160;0.03</td><td>0.04</td><td>23&#160;&#177;&#160;6.5</td><td>0.5&#160;&#177;&#160;0.05</td><td>0.02</td></tr><tr><td>PriA_Org22_<italic>A. sp. oral</italic> taxon 171</td><td>+</td><td>+</td><td>3&#160;&#177;&#160;0.3</td><td>0.3&#160;&#177;&#160;0.09</td><td>0.1</td><td>8&#160;&#177;&#160;2</td><td>0.4&#160;&#177;&#160;0.1</td><td>0.04</td></tr><tr><td>PriA_Org21_<italic>A. oris</italic> MG-1</td><td>+</td><td>+</td><td>10&#160;&#177;&#160;2</td><td>0.2&#160;&#177;&#160;0.09</td><td>0.02</td><td>30&#160;&#177;&#160;7</td><td>0.3&#160;&#177;&#160;0.03</td><td>0.01</td></tr><tr><td>SubHisA2_Org34_<italic>A. vaccimaxillae</italic></td><td>+</td><td>+</td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td></tr><tr><td>SubHisA2_Org36_<italic>A. cardiffensis</italic></td><td>+</td><td>+</td><td>56&#160;&#177;&#160;17</td><td>0.14&#160;&#177;&#160;0.05</td><td>0.002</td><td>n.d.</td><td>n.d.</td><td>n.d.</td></tr><tr><td>SubTrpF_Org10_<italic>A. sp. oral</italic> taxon 848</td><td>&#8722;</td><td>+</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td>0.0001</td></tr><tr><td>SubTrpF_Org13_<italic>A. graevenitzii</italic></td><td>&#8722;</td><td>+</td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td></tr><tr><td>SubTrpF_Org39_<italic>A. sp. oral</italic> taxon 180</td><td>&#8722;</td><td>+</td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td></tr><tr><td>SubTrpF_Org41_<italic>A. odontolyticus</italic></td><td>&#8722;</td><td>+</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td>8.5&#160;&#177;&#160;0.9</td><td>0.15&#160;&#177;&#160;0.06</td><td>0.02</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p><sup>*</sup> Each data point comes from at least three independent determinations using freshly purified enzyme. n.d., activity not detected, even using active-site saturation conditions. Empty entries reflect our inability to properly express and/or solubilize these proteins. The detection limits for the PRA and ProFAR isomerase assay used in the present study are 0.0001 &#956;M<sup>&#8722;1</sup>s<sup>&#8722;1</sup> and 0.001 &#956;M<sup>&#8722;1</sup>s<sup>&#8722;1</sup>, respectively (<xref ref-type="bibr" rid="bib28">Noda-Garc&#237;a et al., 2013</xref>; <xref ref-type="bibr" rid="bib42">Verduzco-Castro et al., 2016</xref>; <xref ref-type="bibr" rid="bib27">Noda-Garc&#237;a et al., 2010</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-22767-v2.xml">elife-22767-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.22767.009</object-id><label>Table 1.</label><caption><p>Ikaros represses transcription by interfering with RNAP2 recruitment rather than elongation. Analysis of differential gene expression (adj. p&lt;0.05) 360 min after 4-OHT-induced nuclear translocation of Ikaros in B3 cells, RNAP2 ChIP-seq, and the distribution of RNAP2 over the TSS and the gene body derived from RNAP2 ChIP-seq. RNAP2 occupancy was significantly decreased at only 40.3% (372 of 924) of genes downregulated at adj. p&lt;0.05 but increased to 62% when considering only genes downregulated with adj. p&lt;0.01 and a minimal log2 fold-change of 2. This suggests that the failure to detect decreased RNAP2 occupancy at the majority of downregulated genes may be due to the limited sensitivity of RNAP2 ChIP-seq compared to RNA-seq.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22767.009">http://dx.doi.org/10.7554/eLife.22767.009</ext-link></p><p><supplementary-material id="SD4-data"><object-id pub-id-type="doi">10.7554/eLife.22767.010</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Numerical data used to generate <xref ref-type="table" rid="tbl1">Table 1</xref>.</title><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22767.010">http://dx.doi.org/10.7554/eLife.22767.010</ext-link></p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-22767-table1-data1-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Gene expression</th><th colspan="4" valign="bottom">RNAP2 occupancy by ChIP</th></tr><tr><th valign="bottom">Global</th><th valign="bottom">TSS~gene body</th><th valign="bottom">TSS&gt;gene body</th><th valign="bottom">TSS&lt;gene body</th></tr></thead><tbody><tr><td rowspan="3" valign="top">Unchanged: 8799</td><td valign="top">Unchanged: 8125</td><td valign="top">8019</td><td valign="top">49</td><td valign="top">57</td></tr><tr><td valign="top">Reduced: 405</td><td valign="top">378</td><td valign="top">11</td><td valign="top">16</td></tr><tr><td valign="top">Increased: 269</td><td valign="top">256</td><td valign="top">10</td><td valign="top">3</td></tr><tr><td rowspan="3" valign="top">Downregulated by Ikaros: 924</td><td valign="top">Unchanged: 549</td><td valign="top">520</td><td valign="top">12</td><td valign="top">17</td></tr><tr><td valign="top">Reduced: 372</td><td valign="top">324</td><td valign="top">15</td><td valign="top">33</td></tr><tr><td valign="top">Increased: 3</td><td valign="top">3</td><td valign="top">0</td><td valign="top">0</td></tr><tr><td rowspan="3" valign="top">Upregulated <break/>by Ikaros: 855</td><td valign="top">Unchanged: 634</td><td valign="top">615</td><td valign="top">15</td><td valign="top">4</td></tr><tr><td valign="top">Reduced: 6</td><td valign="top">6</td><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top">Increased: 215</td><td valign="top">200</td><td valign="top">13</td><td valign="top">2</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-22772-v2.xml">elife-22772-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="A2-tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.22772.079</object-id><label>Appendix 2&#8212;table 1.</label><caption><p>Parameter values in the reaction-diffusion system: Appendix 2-<xref ref-type="disp-formula" rid="equ5 equ5">Equations (1&#8211;4)</xref>. (1) In <xref ref-type="fig" rid="fig1">Figure 1C</xref>, <inline-formula><mml:math id="inf454"><mml:msub><mml:mi>&#946;</mml:mi><mml:mi>I</mml:mi></mml:msub></mml:math></inline-formula> ranges from &#8722;11 to 2, and <xref ref-type="fig" rid="fig1">Figure 1B</xref> is obtained under <inline-formula><mml:math id="inf455"><mml:mrow><mml:msub><mml:mi>&#946;</mml:mi><mml:mi>I</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mn>0</mml:mn></mml:mrow></mml:math></inline-formula>; both are noise-free, thus <inline-formula><mml:math id="inf456"><mml:mrow><mml:msub><mml:mi>&#955;</mml:mi><mml:mi>A</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:msub><mml:mi>&#955;</mml:mi><mml:mi>I</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mn>0</mml:mn></mml:mrow></mml:math></inline-formula>. (2) In <xref ref-type="fig" rid="fig42">Appendix 2&#8212;figure 6</xref>, low, medium and high inhibitor L correspond to <inline-formula><mml:math id="inf457"><mml:mrow><mml:msub><mml:mi>&#946;</mml:mi><mml:mi>I</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mrow><mml:mrow><mml:mo>-</mml:mo><mml:mn>6</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>3</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0</mml:mn></mml:mrow></mml:mrow></mml:math></inline-formula>, respectively, and weak, medium and strong noise correspond to <inline-formula><mml:math id="inf458"><mml:mrow><mml:mrow><mml:msub><mml:mi>&#955;</mml:mi><mml:mi>A</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:msub><mml:mi>&#955;</mml:mi><mml:mi>I</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mn>2</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mn>4</mml:mn><mml:mo>,</mml:mo><mml:mn>6</mml:mn></mml:mrow></mml:mrow></mml:math></inline-formula> &#8211; the same for both multiplicative and additive noise. (3) In <xref ref-type="fig" rid="fig44">Appendix 2&#8212;figure 8</xref> and <xref ref-type="fig" rid="fig45">9</xref>, low <inline-formula><mml:math id="inf459"><mml:mi>R</mml:mi></mml:math></inline-formula> and high <inline-formula><mml:math id="inf460"><mml:mi>R</mml:mi></mml:math></inline-formula> correspond to <inline-formula><mml:math id="inf461"><mml:mrow><mml:msubsup><mml:mi>R</mml:mi><mml:mtext>tot</mml:mtext><mml:mi>A</mml:mi></mml:msubsup><mml:mo>=</mml:mo><mml:msubsup><mml:mi>R</mml:mi><mml:mtext>tot</mml:mtext><mml:mi>I</mml:mi></mml:msubsup><mml:mo>=</mml:mo><mml:mi>R</mml:mi><mml:mo>=</mml:mo><mml:mn>6</mml:mn></mml:mrow></mml:math></inline-formula> or <inline-formula><mml:math id="inf462"><mml:mn>60</mml:mn></mml:math></inline-formula>, respectively, while low, medium and high inhibitor L correspond to <inline-formula><mml:math id="inf463"><mml:mrow><mml:msub><mml:mi>&#946;</mml:mi><mml:mi>I</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mrow><mml:mrow><mml:mo>-</mml:mo><mml:mn>6</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>3</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0</mml:mn></mml:mrow></mml:mrow></mml:math></inline-formula>, respectively, which is the same as in <xref ref-type="fig" rid="fig42">Appendix 2&#8212;figure 6</xref>. <xref ref-type="fig" rid="fig44">Appendix 2&#8212;figure 8</xref> and <xref ref-type="fig" rid="fig45">9</xref> again show noise-free results, hence <inline-formula><mml:math id="inf464"><mml:mrow><mml:msub><mml:mi>&#955;</mml:mi><mml:mi>A</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:msub><mml:mi>&#955;</mml:mi><mml:mi>I</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mn>0</mml:mn></mml:mrow></mml:math></inline-formula>. Also see <xref ref-type="table" rid="A2-tbl2">Appendix 2&#8212;table 2</xref> for supplementary parameter values.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22772.079">http://dx.doi.org/10.7554/eLife.22772.079</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="7">Reaction - Diffusion System</th></tr><tr><th colspan="3" rowspan="2">Parameters</th><th rowspan="2">1D</th><th rowspan="2">2D</th><th colspan="2">3D</th></tr><tr><th>Dorsal</th><th>Ventral</th></tr></thead><tbody><tr><td colspan="2" rowspan="2">Preservation of Receptors</td><td><inline-formula><mml:math id="inf465"><mml:msubsup><mml:mi>R</mml:mi><mml:mtext>tot</mml:mtext><mml:mi>A</mml:mi></mml:msubsup></mml:math></inline-formula></td><td rowspan="2">10</td><td rowspan="2">10</td><td rowspan="2">15</td><td rowspan="2">50</td></tr><tr><td><inline-formula><mml:math id="inf466"><mml:msubsup><mml:mi>R</mml:mi><mml:mtext>tot</mml:mtext><mml:mi>I</mml:mi></mml:msubsup></mml:math></inline-formula></td></tr><tr><td colspan="2" rowspan="2">Diffusion</td><td><inline-formula><mml:math id="inf467"><mml:msub><mml:mi>D</mml:mi><mml:mi>A</mml:mi></mml:msub></mml:math></inline-formula></td><td rowspan="2">1</td><td rowspan="2">0.5</td><td colspan="2" rowspan="2">0.5</td></tr><tr><td><inline-formula><mml:math id="inf468"><mml:msub><mml:mi>D</mml:mi><mml:mi>I</mml:mi></mml:msub></mml:math></inline-formula></td></tr><tr><td colspan="2" rowspan="4">Reaction</td><td><inline-formula><mml:math id="inf469"><mml:msubsup><mml:mi>k</mml:mi><mml:mtext>on</mml:mtext><mml:mi>A</mml:mi></mml:msubsup></mml:math></inline-formula></td><td colspan="4">0.3</td></tr><tr><td><inline-formula><mml:math id="inf470"><mml:msubsup><mml:mi>k</mml:mi><mml:mtext>off</mml:mtext><mml:mi>A</mml:mi></mml:msubsup></mml:math></inline-formula></td><td colspan="4">0</td></tr><tr><td><inline-formula><mml:math id="inf471"><mml:msubsup><mml:mi>k</mml:mi><mml:mtext>on</mml:mtext><mml:mi>I</mml:mi></mml:msubsup></mml:math></inline-formula></td><td colspan="4">0.5</td></tr><tr><td><inline-formula><mml:math id="inf472"><mml:msubsup><mml:mi>k</mml:mi><mml:mtext>off</mml:mtext><mml:mi>I</mml:mi></mml:msubsup></mml:math></inline-formula></td><td colspan="4">0</td></tr><tr><td colspan="2" rowspan="2">Degradation</td><td><inline-formula><mml:math id="inf473"><mml:msubsup><mml:mi>k</mml:mi><mml:mtext>deg</mml:mtext><mml:mi>A</mml:mi></mml:msubsup></mml:math></inline-formula></td><td colspan="4">2</td></tr><tr><td><inline-formula><mml:math id="inf474"><mml:msubsup><mml:mi>k</mml:mi><mml:mtext>deg</mml:mtext><mml:mi>I</mml:mi></mml:msubsup></mml:math></inline-formula></td><td>4</td><td>6.5</td><td colspan="2">7</td></tr><tr><td colspan="2" rowspan="2">Concentration Threshold</td><td><inline-formula><mml:math id="inf475"><mml:msub><mml:mi mathvariant="normal">&#916;</mml:mi><mml:mo>+</mml:mo></mml:msub></mml:math></inline-formula></td><td>1.5</td><td>2.25</td><td colspan="2">2.6</td></tr><tr><td><inline-formula><mml:math id="inf476"><mml:msub><mml:mi mathvariant="normal">&#916;</mml:mi><mml:mo>-</mml:mo></mml:msub></mml:math></inline-formula></td><td colspan="4">0</td></tr><tr><td colspan="2" rowspan="2">Noise</td><td><inline-formula><mml:math id="inf477"><mml:msub><mml:mi>&#955;</mml:mi><mml:mi>A</mml:mi></mml:msub></mml:math></inline-formula></td><td rowspan="2">2/4/6</td><td rowspan="2">3.2</td><td colspan="2" rowspan="2">See <xref ref-type="table" rid="A2-tbl2">Appendix 2&#8212;table 2</xref>.</td></tr><tr><td><inline-formula><mml:math id="inf478"><mml:msub><mml:mi>&#955;</mml:mi><mml:mi>I</mml:mi></mml:msub></mml:math></inline-formula></td></tr><tr><td colspan="2" rowspan="2">Others</td><td><inline-formula><mml:math id="inf479"><mml:msub><mml:mi>&#946;</mml:mi><mml:mi>A</mml:mi></mml:msub></mml:math></inline-formula></td><td colspan="4">4</td></tr><tr><td><inline-formula><mml:math id="inf480"><mml:msub><mml:mi>&#946;</mml:mi><mml:mi>I</mml:mi></mml:msub></mml:math></inline-formula></td><td>&#8722;11 <inline-formula><mml:math id="inf481"><mml:mo>&#8594;</mml:mo></mml:math></inline-formula> 2</td><td colspan="3">2.3</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-22772-v2.xml">elife-22772-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="A2-tbl4" position="float"><object-id pub-id-type="doi">10.7554/eLife.22772.082</object-id><label>Appendix 2&#8212;table 4.</label><caption><p>Activity strength of activator/inhibitor ligands (L), antagonists (A), receptors (R) and ligand-bond receptors (LR) in different phases within a full HF growth cycle.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22772.082">http://dx.doi.org/10.7554/eLife.22772.082</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="3"/><th colspan="2">Activator</th><th colspan="2">Inhibitor</th></tr><tr><th>Region I</th><th>Region II</th><th>Region I</th><th>Region II</th></tr><tr><th>(Bulge)</th><th>(sHG, Matrix, DP)</th><th>(Bulge)</th><th>(sHG, Matrix, DP)</th></tr></thead><tbody><tr><td rowspan="4">Late <break/>Competent <break/>Telogen <break/>(late C)</td><td>L</td><td>3</td><td>(3,0,4)</td><td>2</td><td>(2,0,3)</td></tr><tr><td>A</td><td>2</td><td>(2,0,1)</td><td>0</td><td>(0,0,3)</td></tr><tr><td>R</td><td>2</td><td>(2,0,4)</td><td>3</td><td>(3,0,3)</td></tr><tr><td>LR</td><td>4</td><td>(4,0,4)</td><td>1</td><td>(1,0,3)</td></tr><tr><td rowspan="4">Propagating <break/>Anagen <break/>(P)</td><td>L</td><td>2</td><td>(0,4,4)</td><td>2</td><td>(0,3,3)</td></tr><tr><td>A</td><td>2</td><td>(0,2,3)</td><td>0</td><td>(0,2,2)</td></tr><tr><td>R</td><td>2</td><td>(0,3,4)</td><td>3</td><td>(0,3,3)</td></tr><tr><td>LR</td><td>4</td><td>(0,4,4)</td><td>1</td><td>(0,3,3)</td></tr><tr><td rowspan="4">Autonomous <break/>Anagen <break/>(A)</td><td>L</td><td>2</td><td>(0,4,4)</td><td>2</td><td>(0,4,4)</td></tr><tr><td>A</td><td>2</td><td>(0,4,4)</td><td>0</td><td>(0,2,2)</td></tr><tr><td>R</td><td>2</td><td>(0,3,4)</td><td>3</td><td>(0,3,3)</td></tr><tr><td>LR</td><td>2</td><td>(0,4,4)</td><td>3</td><td>(0,3,3)</td></tr><tr><td rowspan="4">Catagen <break/>(Cat)</td><td>L</td><td>2</td><td>(2,0,2)</td><td>2</td><td>(2,0,4)</td></tr><tr><td>A</td><td>2</td><td>(2,0,4)</td><td>0</td><td>(0,0,1)</td></tr><tr><td>R</td><td>2</td><td>(2,0,4)</td><td>3</td><td>(3,0,3)</td></tr><tr><td>LR</td><td>1</td><td>(1,0,1)</td><td>3</td><td>(3,0,3)</td></tr><tr><td rowspan="4">Refractory <break/>Telogen <break/>(R)</td><td>L</td><td>2</td><td>(2,0,1)</td><td>2</td><td>(2,0,4)</td></tr><tr><td>A</td><td>2</td><td>(2,0,4)</td><td>0</td><td>(0,0,1)</td></tr><tr><td>R</td><td>2</td><td>(2,0,4)</td><td>3</td><td>(3,0,3)</td></tr><tr><td>LR</td><td>1</td><td>(1,0,1)</td><td>4</td><td>(4,0,4)</td></tr><tr><td rowspan="4">Early <break/>Competent <break/>Telogen <break/>(early C)</td><td>L</td><td colspan="2" rowspan="4">&#8195; <break/>Similar to refractory telogen <break/>&#8195; <break/>&#8195;</td><td colspan="2" rowspan="4">&#8195; <break/>Similar to late competent <break/>telogen <break/>&#8195;</td></tr><tr><td>A</td></tr><tr><td>R</td></tr><tr><td>LR</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-22835-v2.xml">elife-22835-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.22835.031</object-id><label>Table 1.</label><caption><p>Concentrations of DAPG, pyoluteorin, and expression of <italic>pltL</italic>::<italic>gfp</italic> in pure cultures and co-culture of the &#916;<italic>phlA</italic>&#916;<italic>pltM</italic> mutant and the &#916;<italic>phlA</italic>&#916;<italic>phlD</italic>&#916;<italic>pltA</italic> mutant of <italic>P. protegens</italic> Pf-5.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22835.031">http://dx.doi.org/10.7554/eLife.22835.031</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Product/expression*</th><th rowspan="2" valign="top">Amendment<sup>&#8224;</sup></th><th colspan="3" valign="top">Bacterial cultures<sup>&#8225;</sup></th></tr><tr><th valign="top">&#916;<italic>phlA</italic>&#916;<italic>pltM</italic></th><th valign="top">&#916;<italic>phlA</italic>&#916;<italic>pltM</italic> + &#916;<italic>phlA</italic>&#916;<italic>phlD</italic>&#916;<italic>pltA</italic></th><th valign="top">&#916;<italic>phlA</italic>&#916;<italic>phlD</italic>&#916;<italic>pltA</italic></th></tr></thead><tbody><tr><td valign="top">Pyoluteorin</td><td valign="top">None</td><td valign="top">BD</td><td valign="top">6.2&#160;&#177;&#160;1.8 nmol/g</td><td valign="top">BD</td></tr><tr><td valign="top">DAPG</td><td valign="top">None</td><td valign="top">BD</td><td valign="top">BD</td><td valign="top">BD</td></tr><tr><td rowspan="4" valign="top">Relative GFP</td><td valign="top">None</td><td valign="top">34.6&#160;&#177;&#160;2.3</td><td valign="top">137.4 &#177; 3.4<sup>&#167;</sup>; 214.7&#160;&#177;&#160;11.2<sup>&#182;</sup></td><td valign="top">27.6&#160;&#177;&#160;1.9</td></tr><tr><td valign="top">PG (10 nM)</td><td valign="top">37.6&#160;&#177;&#160;3.1</td><td valign="top">NT</td><td valign="top">133.3&#160;&#177;&#160;6.4</td></tr><tr><td valign="top">PG-Cl (1 &#181;M)</td><td valign="top">261.0&#160;&#177;&#160;13.3</td><td valign="top">NT</td><td valign="top">308.6&#160;&#177;&#160;21.8</td></tr><tr><td valign="top">PG-Cl<sub>2</sub> (10 nM)</td><td valign="top">128.2&#160;&#177;&#160;10.7</td><td valign="top">NT</td><td valign="top">268.4&#160;&#177;&#160;16.2</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>*The mutants were cultured&#160;for 3 d on&#160;a NAGly plate to determine production of secondary metabolites, and&#160;for 20 hr in NBGly broth to determine expression of <italic>pltL::gfp</italic> from the reporter construct pL-gfp (<xref ref-type="fig" rid="fig1">Figure 1E</xref>). The co-culture was prepared by mixing the two mutants at a 1:1 ratio.</p></fn><fn id="tblfn2"><p><bold><sup>&#8224;</sup></bold>PG, PG-Cl or PG-Cl<sub>2</sub> was amended in the mutant cultures to test their influences on the expression of <italic>pltL::gfp</italic>. The concentrations of the tested compounds are shown in parenthesis.</p></fn><fn id="tblfn3"><p><sup>&#8225;</sup>Secondary metabolites were extracted from the agar plate and the concentrations of DAPG and pyoluteorin were determined by HPLC. Expression of <italic>pltL::gfp</italic> was monitored and recorded as relative GFP (fluorescence of GFP divided by OD<sub>600</sub>). <sup>&#167;</sup>The &#916;<italic>phlA</italic>&#916;<italic>pltM</italic> mutant, but not the &#916;<italic>phlA</italic>&#916;<italic>phlD</italic>&#916;<italic>pltA</italic> mutant, contains pL-gfp in the co-culture. <sup>&#182;</sup>The &#916;<italic>phlA</italic>&#916;<italic>phlD</italic>&#916;<italic>pltA</italic> mutant, but not the &#916;<italic>phlA</italic>&#916;<italic>pltM</italic> mutant, contains pL-gfp in the co-culture. Data are presented as mean &#177; standard derivation. Mean values are calculated from three biological replicates. BD = below detection. NT = not tested.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-22900-v2.xml">elife-22900-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.22900.012</object-id><label>Table 1.</label><caption><p>Impact of phospho-mimicking or phospho-blocking mutations on the thermodynamics parameters of XRCC4-XLF protein-protein interaction by micro-calorimetry.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22900.012">http://dx.doi.org/10.7554/eLife.22900.012</ext-link></p><p><supplementary-material id="SD2-data"><object-id pub-id-type="doi">10.7554/eLife.22900.013</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Thermograms of XRCC4-XLF WT-WT, Ala-Ala, and Asp-Asp experiments.</title><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22900.013">http://dx.doi.org/10.7554/eLife.22900.013</ext-link></p></caption><media mime-subtype="plain" mimetype="text" xlink:href="elife-22900-table1-data1-v2.txt"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><break/></th><th rowspan="2">T <break/>(&#176;C)</th><th rowspan="2">2-Me <break/>(mM)</th><th>[XRCC4] <break/>(&#956;M)</th><th>[XLF] <break/>(&#956;M)</th><th rowspan="2">Ratio</th><th rowspan="2"><italic>N</italic></th><th rowspan="2"><italic>K</italic><sub>a</sub> <break/>(M<sup>&#8722;1</sup>)</th><th rowspan="2">&#916;<italic>H</italic><sup>0</sup> <break/>(kcal/mol)</th><th rowspan="2"><italic>K</italic><sub>d</sub> <break/>(&#956;M)</th><th rowspan="2">&#8722;&#916;<italic>G</italic><sup>0</sup> <break/>(kcal/mol)</th><th rowspan="2">&#8722;<italic>T</italic>&#916;<italic>S</italic><sup>0</sup> <break/>(kcal/mol)</th><th rowspan="2">Thermogram shown in</th><th rowspan="2"> <break/><break/>Source data</th><th rowspan="2"><italic>&lt;K</italic><sub>d</sub>&gt; <break/>(&#956;M)</th><th rowspan="2"><italic>&lt;&#916;H</italic><sup>0</sup>&gt; <break/>(kcal/mol)</th><th rowspan="2">&lt;-&#916;<italic>G</italic><sup>0</sup>&gt; <break/>(kcal/mol)</th><th rowspan="2">&lt;-<italic>T</italic>&#916;<italic>S</italic><sup>0</sup>&gt; <break/>(kcal/mol)</th></tr><tr><th>ITC cell</th><th>ITC syringe</th></tr></thead><tbody><tr><td rowspan="8"><bold>XRCC4-WT</bold> <break/><bold>XLF-WT</bold></td><td>10</td><td>10</td><td>17</td><td>168</td><td>10</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td><xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1A</xref></td><td>A</td><td colspan="4"><break/></td></tr><tr><td>25</td><td>10</td><td>17</td><td>168</td><td>10</td><td>0.97</td><td>536000</td><td>&#8722;1.7</td><td>1.8</td><td>&#8722;7.8</td><td>&#8722;6.1</td><td>--</td><td>B</td><td rowspan="3">3.2&#160;&#177;&#160;0.8</td><td rowspan="3">&#8722;2.4&#160;&#177;&#160;0.4</td><td rowspan="3">&#8722;7.5&#160;&#177;&#160;0.2</td><td rowspan="3">&#8722;5.2&#160;&#177;&#160;0.5</td></tr><tr><td>25</td><td>10</td><td>33</td><td>330</td><td>10</td><td>0.91</td><td>306000</td><td>&#8722;2.9</td><td>3.2</td><td>&#8722;7.5</td><td>&#8722;4.6</td><td><xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1B</xref></td><td>C</td></tr><tr><td>25</td><td>10</td><td>25</td><td>330</td><td>13</td><td>0.84</td><td>222000</td><td>&#8722;2.6</td><td>4.5</td><td>&#8722;7.3</td><td>&#8722;4.7</td><td>--</td><td>D</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">23</td><td style="author-callout-style-b8">300</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.85</td><td style="author-callout-style-b8">273000</td><td style="author-callout-style-b8">&#8722;6.0</td><td style="author-callout-style-b8">3.7</td><td style="author-callout-style-b8">&#8722;7.7</td><td style="author-callout-style-b8">&#8722;1.7</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">E</td><td rowspan="4" style="author-callout-style-b8">2.4&#160;&#177;&#160;0.7</td><td rowspan="4" style="author-callout-style-b8">&#8722;7.1&#160;&#177;&#160;1</td><td rowspan="4" style="author-callout-style-b8">&#8722;8.0&#160;&#177;&#160;0.2</td><td rowspan="4" style="author-callout-style-b8">&#8722;1.0&#160;&#177;&#160;0.8</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">23</td><td style="author-callout-style-b8">300</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.89</td><td style="author-callout-style-b8">278000</td><td style="author-callout-style-b8">&#8722;4.9</td><td style="author-callout-style-b8">3.6</td><td style="author-callout-style-b8">&#8722;7.7</td><td style="author-callout-style-b8">&#8722;2.9</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">F</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">12</td><td style="author-callout-style-b8">154</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.92</td><td style="author-callout-style-b8">848000</td><td style="author-callout-style-b8">&#8722;8.6</td><td style="author-callout-style-b8">1.2</td><td style="author-callout-style-b8">&#8722;8.4</td><td style="author-callout-style-b8">0.2</td><td style="author-callout-style-b8"><xref ref-type="fig" rid="fig5">Figure 5A</xref></td><td style="author-callout-style-b8">G</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">12</td><td style="author-callout-style-b8">154</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.84</td><td style="author-callout-style-b8">760000</td><td style="author-callout-style-b8">&#8722;8.8</td><td style="author-callout-style-b8">1.3</td><td style="author-callout-style-b8">&#8722;8.3</td><td style="author-callout-style-b8">0.4</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">H</td></tr><tr><td rowspan="5"><bold>XRCC4-Ala</bold> <break/><bold>XLF-Ala</bold></td><td>25</td><td>10</td><td>25</td><td>330</td><td>13</td><td>1.21</td><td>300000</td><td>&#8722;1.7</td><td>3.3</td><td>&#8722;7.5</td><td>&#8722;5.8</td><td><xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1C</xref></td><td>I</td><td colspan="4"><break/></td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">25</td><td style="author-callout-style-b8">325</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">1.07</td><td style="author-callout-style-b8">637000</td><td style="author-callout-style-b8">&#8722;3.1</td><td style="author-callout-style-b8">1.6</td><td style="author-callout-style-b8">&#8722;8.2</td><td style="author-callout-style-b8">&#8722;5.1</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">J</td><td rowspan="4" style="author-callout-style-b8">2.3&#160;&#177;&#160;0.4</td><td rowspan="4" style="author-callout-style-b8">&#8722;5.8&#160;&#177;&#160;1.4</td><td rowspan="4" style="author-callout-style-b8">&#8722;8.0&#160;&#177;&#160;0.1</td><td rowspan="4" style="author-callout-style-b8">&#8722;2.2&#160;&#177;&#160;1.4</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">25</td><td style="author-callout-style-b8">325</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">1</td><td style="author-callout-style-b8">343000</td><td style="author-callout-style-b8">&#8722;3.7</td><td style="author-callout-style-b8">2.9</td><td style="author-callout-style-b8">&#8722;7.8</td><td style="author-callout-style-b8">&#8722;4.1</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">K</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">19</td><td style="author-callout-style-b8">250</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.92</td><td style="author-callout-style-b8">345000</td><td style="author-callout-style-b8">&#8722;8.4</td><td style="author-callout-style-b8">2.9</td><td style="author-callout-style-b8">&#8722;7.8</td><td style="author-callout-style-b8">0.6</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">L</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">19</td><td style="author-callout-style-b8">250</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.92</td><td style="author-callout-style-b8">582000</td><td style="author-callout-style-b8">&#8722;8.1</td><td style="author-callout-style-b8">1.7</td><td style="author-callout-style-b8">&#8722;8.2</td><td style="author-callout-style-b8">&#8722;0.1</td><td style="author-callout-style-b8"><xref ref-type="fig" rid="fig5">Figure 5B</xref></td><td style="author-callout-style-b8">M</td></tr><tr><td rowspan="6"><bold>XRCC4-Asp</bold> <break/><bold>XLF-Asp</bold></td><td>25</td><td>10</td><td>35</td><td>350</td><td>10</td><td>1.27</td><td>672000</td><td>&#8722;0.9</td><td>1.5</td><td>&#8722;7.9</td><td>&#8722;7.0</td><td>--</td><td>N</td><td colspan="4" rowspan="2"><break/></td></tr><tr><td>25</td><td>10</td><td>25</td><td>330</td><td>13</td><td>0.82</td><td>263000</td><td>&#8722;1.2</td><td>3.8</td><td>&#8722;7.4</td><td>&#8722;6.2</td><td><xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1D</xref></td><td>O</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">32</td><td style="author-callout-style-b8">416</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">1.06</td><td style="author-callout-style-b8">289000</td><td style="author-callout-style-b8">&#8722;2.4</td><td style="author-callout-style-b8">3.4</td><td style="author-callout-style-b8">&#8722;7.8</td><td style="author-callout-style-b8">&#8722;5.4</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">P</td><td rowspan="4" style="author-callout-style-b8">2.4&#160;&#177;&#160;0.4</td><td rowspan="4" style="author-callout-style-b8">&#8722;3.7&#160;&#177;&#160;0.8</td><td rowspan="4" style="author-callout-style-b8">&#8722;8.0&#160;&#177;&#160;0.1</td><td rowspan="4" style="author-callout-style-b8">&#8722;4.3&#160;&#177;&#160;0.7</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">32</td><td style="author-callout-style-b8">416</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.95</td><td style="author-callout-style-b8">425000</td><td style="author-callout-style-b8">&#8722;2.3</td><td style="author-callout-style-b8">2.4</td><td style="author-callout-style-b8">&#8722;8.0</td><td style="author-callout-style-b8">&#8722;5.7</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">Q</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">19</td><td style="author-callout-style-b8">250</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.88</td><td style="author-callout-style-b8">566000</td><td style="author-callout-style-b8">&#8722;4.7</td><td style="author-callout-style-b8">1.8</td><td style="author-callout-style-b8">&#8722;8.2</td><td style="author-callout-style-b8">&#8722;3.5</td><td style="author-callout-style-b8"><xref ref-type="fig" rid="fig5">Figure 5C</xref></td><td style="author-callout-style-b8">R</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">17</td><td style="author-callout-style-b8">220</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.85</td><td style="author-callout-style-b8">538000</td><td style="author-callout-style-b8">&#8722;5.5</td><td style="author-callout-style-b8">1.8</td><td style="author-callout-style-b8">&#8722;8.1</td><td style="author-callout-style-b8">&#8722;2.6</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">S</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>2-Me stands for 2-Mercaptoethanol</p></fn><fn id="tblfn2"><p>Ratio indicates the titrant over analyte concentration ([XLF]/[XRCC4])</p></fn><fn id="tblfn3"><p>Source data specifies the column corresponding to each thermogram in <xref ref-type="supplementary-material" rid="SD2-data">Table 1&#8212;source data 1</xref>.</p></fn><fn id="tblfn4"><p>Last four columns report mean values &#177; Standard Error of the Mean (SEM) from four experimental runs at 37&#176;C (grey cells) and 3 runs at 25&#176;C (white cells)</p></fn><fn id="tblfn5"><p>n.d. denotes no detectable heat transfer (at 10&#176;C)</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-22900-v3.xml">elife-22900-v3.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.22900.012</object-id><label>Table 1.</label><caption><p>Impact of phospho-mimicking or phospho-blocking mutations on the thermodynamics parameters of XRCC4-XLF protein-protein interaction by micro-calorimetry.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22900.012">http://dx.doi.org/10.7554/eLife.22900.012</ext-link></p><p><supplementary-material id="SD2-data"><object-id pub-id-type="doi">10.7554/eLife.22900.013</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Thermograms of XRCC4-XLF WT-WT, Ala-Ala, and Asp-Asp experiments.</title><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.22900.013">http://dx.doi.org/10.7554/eLife.22900.013</ext-link></p></caption><media mime-subtype="plain" mimetype="text" xlink:href="elife-22900-table1-data1-v3.txt"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><break/></th><th rowspan="2">T <break/>(&#176;C)</th><th rowspan="2">2-Me <break/>(mM)</th><th>[XRCC4] <break/>(&#956;M)</th><th>[XLF] <break/>(&#956;M)</th><th rowspan="2">Ratio</th><th rowspan="2"><italic>N</italic></th><th rowspan="2"><italic>K</italic><sub>a</sub> <break/>(M<sup>&#8722;1</sup>)</th><th rowspan="2">&#916;<italic>H</italic><sup>0</sup> <break/>(kcal/mol)</th><th rowspan="2"><italic>K</italic><sub>d</sub> <break/>(&#956;M)</th><th rowspan="2">&#8722;&#916;<italic>G</italic><sup>0</sup> <break/>(kcal/mol)</th><th rowspan="2">&#8722;<italic>T</italic>&#916;<italic>S</italic><sup>0</sup> <break/>(kcal/mol)</th><th rowspan="2">Thermogram shown in</th><th rowspan="2"> <break/><break/>Source data</th><th rowspan="2"><italic>&lt;K</italic><sub>d</sub>&gt; <break/>(&#956;M)</th><th rowspan="2"><italic>&lt;&#916;H</italic><sup>0</sup>&gt; <break/>(kcal/mol)</th><th rowspan="2">&lt;-&#916;<italic>G</italic><sup>0</sup>&gt; <break/>(kcal/mol)</th><th rowspan="2">&lt;-<italic>T</italic>&#916;<italic>S</italic><sup>0</sup>&gt; <break/>(kcal/mol)</th></tr><tr><th>ITC cell</th><th>ITC syringe</th></tr></thead><tbody><tr><td rowspan="8"><bold>XRCC4-WT</bold> <break/><bold>XLF-WT</bold></td><td>10</td><td>10</td><td>17</td><td>168</td><td>10</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td>n.d.</td><td><xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1A</xref></td><td>A</td><td colspan="4"><break/></td></tr><tr><td>25</td><td>10</td><td>17</td><td>168</td><td>10</td><td>0.97</td><td>536000</td><td>&#8722;1.7</td><td>1.8</td><td>&#8722;7.8</td><td>&#8722;6.1</td><td>--</td><td>B</td><td rowspan="3">3.2&#160;&#177;&#160;0.8</td><td rowspan="3">&#8722;2.4&#160;&#177;&#160;0.4</td><td rowspan="3">&#8722;7.5&#160;&#177;&#160;0.2</td><td rowspan="3">&#8722;5.2&#160;&#177;&#160;0.5</td></tr><tr><td>25</td><td>10</td><td>33</td><td>330</td><td>10</td><td>0.91</td><td>306000</td><td>&#8722;2.9</td><td>3.2</td><td>&#8722;7.5</td><td>&#8722;4.6</td><td><xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1B</xref></td><td>C</td></tr><tr><td>25</td><td>10</td><td>25</td><td>330</td><td>13</td><td>0.84</td><td>222000</td><td>&#8722;2.6</td><td>4.5</td><td>&#8722;7.3</td><td>&#8722;4.7</td><td>--</td><td>D</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">23</td><td style="author-callout-style-b8">300</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.85</td><td style="author-callout-style-b8">273000</td><td style="author-callout-style-b8">&#8722;6.0</td><td style="author-callout-style-b8">3.7</td><td style="author-callout-style-b8">&#8722;7.7</td><td style="author-callout-style-b8">&#8722;1.7</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">E</td><td rowspan="4" style="author-callout-style-b8">2.4&#160;&#177;&#160;0.7</td><td rowspan="4" style="author-callout-style-b8">&#8722;7.1&#160;&#177;&#160;1</td><td rowspan="4" style="author-callout-style-b8">&#8722;8.0&#160;&#177;&#160;0.2</td><td rowspan="4" style="author-callout-style-b8">&#8722;1.0&#160;&#177;&#160;0.8</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">23</td><td style="author-callout-style-b8">300</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.89</td><td style="author-callout-style-b8">278000</td><td style="author-callout-style-b8">&#8722;4.9</td><td style="author-callout-style-b8">3.6</td><td style="author-callout-style-b8">&#8722;7.7</td><td style="author-callout-style-b8">&#8722;2.9</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">F</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">12</td><td style="author-callout-style-b8">154</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.92</td><td style="author-callout-style-b8">848000</td><td style="author-callout-style-b8">&#8722;8.6</td><td style="author-callout-style-b8">1.2</td><td style="author-callout-style-b8">&#8722;8.4</td><td style="author-callout-style-b8">0.2</td><td style="author-callout-style-b8"><xref ref-type="fig" rid="fig5">Figure 5A</xref></td><td style="author-callout-style-b8">G</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">12</td><td style="author-callout-style-b8">154</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.84</td><td style="author-callout-style-b8">760000</td><td style="author-callout-style-b8">&#8722;8.8</td><td style="author-callout-style-b8">1.3</td><td style="author-callout-style-b8">&#8722;8.3</td><td style="author-callout-style-b8">0.4</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">H</td></tr><tr><td rowspan="5"><bold>XRCC4-Ala</bold> <break/><bold>XLF-Ala</bold></td><td>25</td><td>10</td><td>25</td><td>330</td><td>13</td><td>1.21</td><td>300000</td><td>&#8722;1.7</td><td>3.3</td><td>&#8722;7.5</td><td>&#8722;5.8</td><td><xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1C</xref></td><td>I</td><td colspan="4"><break/></td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">25</td><td style="author-callout-style-b8">325</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">1.07</td><td style="author-callout-style-b8">637000</td><td style="author-callout-style-b8">&#8722;3.1</td><td style="author-callout-style-b8">1.6</td><td style="author-callout-style-b8">&#8722;8.2</td><td style="author-callout-style-b8">&#8722;5.1</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">J</td><td rowspan="4" style="author-callout-style-b8">2.3&#160;&#177;&#160;0.4</td><td rowspan="4" style="author-callout-style-b8">&#8722;5.8&#160;&#177;&#160;1.4</td><td rowspan="4" style="author-callout-style-b8">&#8722;8.0&#160;&#177;&#160;0.1</td><td rowspan="4" style="author-callout-style-b8">&#8722;2.2&#160;&#177;&#160;1.4</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">25</td><td style="author-callout-style-b8">325</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">1</td><td style="author-callout-style-b8">343000</td><td style="author-callout-style-b8">&#8722;3.7</td><td style="author-callout-style-b8">2.9</td><td style="author-callout-style-b8">&#8722;7.8</td><td style="author-callout-style-b8">&#8722;4.1</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">K</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">19</td><td style="author-callout-style-b8">250</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.92</td><td style="author-callout-style-b8">345000</td><td style="author-callout-style-b8">&#8722;8.4</td><td style="author-callout-style-b8">2.9</td><td style="author-callout-style-b8">&#8722;7.8</td><td style="author-callout-style-b8">0.6</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">L</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">19</td><td style="author-callout-style-b8">250</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.92</td><td style="author-callout-style-b8">582000</td><td style="author-callout-style-b8">&#8722;8.1</td><td style="author-callout-style-b8">1.7</td><td style="author-callout-style-b8">&#8722;8.2</td><td style="author-callout-style-b8">&#8722;0.1</td><td style="author-callout-style-b8"><xref ref-type="fig" rid="fig5">Figure 5B</xref></td><td style="author-callout-style-b8">M</td></tr><tr><td rowspan="6"><bold>XRCC4-Asp</bold> <break/><bold>XLF-Asp</bold></td><td>25</td><td>10</td><td>35</td><td>350</td><td>10</td><td>1.27</td><td>672000</td><td>&#8722;0.9</td><td>1.5</td><td>&#8722;7.9</td><td>&#8722;7.0</td><td>--</td><td>N</td><td colspan="4" rowspan="2"><break/></td></tr><tr><td>25</td><td>10</td><td>25</td><td>330</td><td>13</td><td>0.82</td><td>263000</td><td>&#8722;1.2</td><td>3.8</td><td>&#8722;7.4</td><td>&#8722;6.2</td><td><xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1D</xref></td><td>O</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">32</td><td style="author-callout-style-b8">416</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">1.06</td><td style="author-callout-style-b8">289000</td><td style="author-callout-style-b8">&#8722;2.4</td><td style="author-callout-style-b8">3.4</td><td style="author-callout-style-b8">&#8722;7.8</td><td style="author-callout-style-b8">&#8722;5.4</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">P</td><td rowspan="4" style="author-callout-style-b8">2.4&#160;&#177;&#160;0.4</td><td rowspan="4" style="author-callout-style-b8">&#8722;3.7&#160;&#177;&#160;0.8</td><td rowspan="4" style="author-callout-style-b8">&#8722;8.0&#160;&#177;&#160;0.1</td><td rowspan="4" style="author-callout-style-b8">&#8722;4.3&#160;&#177;&#160;0.7</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">32</td><td style="author-callout-style-b8">416</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.95</td><td style="author-callout-style-b8">425000</td><td style="author-callout-style-b8">&#8722;2.3</td><td style="author-callout-style-b8">2.4</td><td style="author-callout-style-b8">&#8722;8.0</td><td style="author-callout-style-b8">&#8722;5.7</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">Q</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">19</td><td style="author-callout-style-b8">250</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.88</td><td style="author-callout-style-b8">566000</td><td style="author-callout-style-b8">&#8722;4.7</td><td style="author-callout-style-b8">1.8</td><td style="author-callout-style-b8">&#8722;8.2</td><td style="author-callout-style-b8">&#8722;3.5</td><td style="author-callout-style-b8"><xref ref-type="fig" rid="fig5">Figure 5C</xref></td><td style="author-callout-style-b8">R</td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">17</td><td style="author-callout-style-b8">220</td><td style="author-callout-style-b8">13</td><td style="author-callout-style-b8">0.85</td><td style="author-callout-style-b8">538000</td><td style="author-callout-style-b8">&#8722;5.5</td><td style="author-callout-style-b8">1.8</td><td style="author-callout-style-b8">&#8722;8.1</td><td style="author-callout-style-b8">&#8722;2.6</td><td style="author-callout-style-b8">--</td><td style="author-callout-style-b8">S</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>2-Me stands for 2-Mercaptoethanol</p></fn><fn id="tblfn2"><p>Ratio indicates the titrant over analyte concentration ([XLF]/[XRCC4])</p></fn><fn id="tblfn3"><p>Source data specifies the column corresponding to each thermogram in <xref ref-type="supplementary-material" rid="SD2-data">Table 1&#8212;source data 1</xref>.</p></fn><fn id="tblfn4"><p>Last four columns report mean values &#177; Standard Error of the Mean (SEM) from four experimental runs at 37&#176;C (grey cells) and 3 runs at 25&#176;C (white cells)</p></fn><fn id="tblfn5"><p>n.d. denotes no detectable heat transfer (at 10&#176;C)</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23013-v2.xml">elife-23013-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.23013.011</object-id><label>Table 1.</label><caption><p>Estimated rates of replacement of CD4 effector and central memory through influx of new cells in 14 week-old mice. We quote both absolute influx (cells per day as a percentage of the pool size) and percent replaced per week. The latter is slightly less than 7 <inline-formula><mml:math id="inf5"><mml:mo>&#215;</mml:mo></mml:math></inline-formula> the&#160;daily rate of influx because immigrant cells are assumed to be lost at the same rate as existing displaceable cells (see Methods). AIC differences (<inline-formula><mml:math id="inf6"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>AIC) are quoted relative to the best-fitting model for each cell type. These differences reflect the relative support for two models, with exp(<inline-formula><mml:math id="inf7"><mml:mrow><mml:mo>-</mml:mo><mml:mi mathvariant="normal">&#916;</mml:mi></mml:mrow></mml:math></inline-formula>AIC/2) being the relative probability that it is the model with the lower penalised likelihood (larger AIC value) that minimises the information lost in describing the data. For CD4 T<sub>EM</sub> the two models have equal support, but for CD4 T<sub>CM</sub> the resistant memory model is favoured (<inline-formula><mml:math id="inf8"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>AIC&#160;=&#160;8.3, exp(<inline-formula><mml:math id="inf9"><mml:mrow><mml:mo>-</mml:mo><mml:mi mathvariant="normal">&#916;</mml:mi></mml:mrow></mml:math></inline-formula>AIC/2)=0.02). Models considered most plausible are highlighted.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.23013.011">http://dx.doi.org/10.7554/eLife.23013.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th/><th colspan="5">Resistant memory model</th><th colspan="5">Declining recruitment model</th></tr><tr><th>Source</th><th colspan="2">% Input/day</th><th colspan="2">% Replaced/wk</th><th><inline-formula><mml:math id="inf10"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>AIC</th><th colspan="2">% Input/day</th><th colspan="2">% Replaced/wk</th><th><inline-formula><mml:math id="inf11"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>AIC</th></tr></thead><tbody><tr><td>CD4 T<sub>EM</sub></td><td style="author-callout-style-b1">Naive</td><td style="author-callout-style-b1">1.0</td><td style="author-callout-style-b1">(0.7, 1.4)</td><td style="author-callout-style-b1">6.3</td><td style="author-callout-style-b1">(4.4, 8.6)</td><td style="author-callout-style-b1">0.16</td><td style="author-callout-style-b1">1.0</td><td style="author-callout-style-b1">(0.7, 1.6)</td><td style="author-callout-style-b1">6.4</td><td style="author-callout-style-b1">(4.3, 9.7)</td><td style="author-callout-style-b1">0</td></tr><tr><td/><td>CM</td><td>4.1</td><td>(2.6, 7.5)</td><td>23.0</td><td>(16, 35)</td><td>11</td><td>3.6</td><td>(2.1, 6.2)</td><td>21</td><td>(13, 30)</td><td>11</td></tr><tr><td>CD4 T<sub>CM</sub></td><td style="author-callout-style-b1">Naive</td><td style="author-callout-style-b1">1.5</td><td style="author-callout-style-b1">(1.0, 2.8)</td><td style="author-callout-style-b1">9.8</td><td style="author-callout-style-b1">(6.5, 17)</td><td style="author-callout-style-b1">0</td><td>2.3</td><td>(1.3, 4.1)</td><td>13.5</td><td>(8.4, 22)</td><td>8.3</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23013-v2.xml">elife-23013-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.23013.019</object-id><label>Table 2.</label><caption><p>Parameters describing homeostasis of murine CD4 memory subsets, using a two-population model of kinetic heterogeneity and the best estimates of the magnitudes of the influx into each subset from the naive pool. Pool-average lifetimes and interdivision times are defined to be the mean of the corresponding quantities for the fast and slow subpopulations weighted by their size estimates. The best-fitting models were those that assumed no difference in death rates of Ki67<sup>high</sup> and Ki67<sup>low</sup> cells (indicated by rows in which ratio of loss rates&#160;=&#160;1).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.23013.019">http://dx.doi.org/10.7554/eLife.23013.019</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Parameter</th><th>Ratio of loss rates</th><th colspan="2">CD4 effector memory</th><th colspan="2">CD4 central memory</th></tr><tr><th>Ki67<sup>high</sup>:Ki67<sup>low</sup></th><th>Estimate</th><th>95% CI</th><th>Estimate</th><th>95% CI</th></tr></thead><tbody><tr><td>Pool-average lifetime (days)</td><td>1 <break/>10 <break/>0.1</td><td>29 <break/>41 <break/>55</td><td>(28, 30) <break/>(39, 42) <break/>(54, 56)</td><td>21 <break/>27 <break/>44</td><td>(19, 24) <break/>(26, 35) <break/>(39, 51)</td></tr><tr><td>Pool-average interdivision time (days)</td><td>1 <break/>10 <break/>0.1</td><td>88 <break/>63 <break/>84</td><td>(83, 158) <break/>(53, 353) <break/>(80, 108)</td><td>86 <break/>59 <break/>51</td><td>(47, 144) <break/>(35, 74) <break/>(38, 84)</td></tr><tr><td>Ki67 lifetime (days)</td><td>1 <break/>10 <break/>0.1</td><td>3.28 <break/>3.23 <break/>3.28</td><td>(3.14, 3.39) <break/>(3.03, 3.30) <break/>(3.18, 3.41)</td><td>3.59 <break/>3.74 <break/>3.32</td><td>(3.47, 3.70) <break/>(3.57, 3.82) <break/>(3.31, 3.52)</td></tr><tr><td>Efficiency of BrdU uptake (%)</td><td>1 <break/>10 <break/>0.1</td><td>76 <break/>75 <break/>78</td><td>(74, 79) <break/>(73, 78) <break/>(76, 81)</td><td>77 <break/>78 <break/>78</td><td>(76, 79) <break/>(76, 79) <break/>(78, 81)</td></tr><tr><td>Delay before source switches&#160;to BrdU<sup>&#8722;</sup>&#160;post-labelling (days)</td><td>1 <break/>10 <break/>0.1</td><td>2.5 <break/>2.6 <break/>2.1</td><td>(1.7, 3.0) <break/>(1.9, 3.5) <break/>(1.3, 2.3)</td><td>0.085 <break/>0.006 <break/>0.14</td><td>(0.002, 1.16) <break/>(0.003, 1.43) <break/>(0.003, 0.66)</td></tr><tr><td>Source contribution&#160;to peripheral production (fraction)</td><td>1 <break/>10 <break/>0.1</td><td>0.12 <break/>0.11 <break/>0.15</td><td>(0.12, 0.13) <break/>(0.099, 0.11) <break/>(0.15, 0.16)</td><td>0.092 <break/>0.085 <break/>0.12</td><td>(0.088, 0.096) <break/>(0.08, 0.088) <break/>(0.11, 0.12)</td></tr><tr><td>Fraction of source enteringslow subpopulation</td><td>1 <break/>10 <break/>0.1</td><td>1 <break/>1 <break/>1</td><td>(0.98, 1) <break/>(0.95, 1) <break/>(0.98, 1)</td><td>0.69 <break/>1 <break/>0.46</td><td>(0.38, 0.85) <break/>(0.45, 1) <break/>(0.19, 0.62)</td></tr><tr><td>Mean lifetime offast subpopulation (days)</td><td>1 <break/>10 <break/>0.1</td><td>5.7 <break/>17 <break/>18</td><td>(5.5, 6.8) <break/>(15, 20) <break/>(18, 21)</td><td>3.3 <break/>7.1 <break/>6.4</td><td>(3.1, 3.4) <break/>(6.2, 7.5) <break/>(6.4, 6.9)</td></tr><tr><td>Mean lifetime ofslow subpopulation (days)</td><td>1 <break/>10 <break/>0.1</td><td>43 <break/>61 <break/>75</td><td>(43, 48) <break/>(60, 65) <break/>(74, 77)</td><td>38 <break/>48 <break/>76</td><td>(34, 44) <break/>(47, 65) <break/>(68, 90)</td></tr><tr><td>Mean interdivision time offast subpopulation (days)</td><td>1 <break/>10 <break/>0.1</td><td>5.7 <break/>5.7 <break/>6.8</td><td>(5.5, 6.9) <break/>(5.1, 6.7) <break/>(6.8, 7.6)</td><td>3.4 <break/>3.3 <break/>4.2</td><td>(3.2, 3.5) <break/>(3.0, 3.4) <break/>(4.2, 4.4)</td></tr><tr><td>Mean interdivision time ofslow subpopulation (days)</td><td>1 <break/>10 <break/>0.1</td><td>138 <break/>113 <break/>125</td><td>(130, 275) <break/>(90, 750) <break/>(119, 169)</td><td>167 <break/>118 <break/>92</td><td>(89, 280) <break/>(72, 151) <break/>(69, 151)</td></tr><tr><td>Size of fast subpopulation(fraction of total)</td><td>1 <break/>10 <break/>0.1</td><td>0.38 <break/>0.46 <break/>0.35</td><td>(0.37, 0.44) <break/>(0.43, 0.54) <break/>(0.34, 0.38)</td><td>0.49 <break/>0.52 <break/>0.47</td><td>(0.47, 0.51) <break/>(0.49, 0.53) <break/>(0.45, 0.48)</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23013-v3.xml">elife-23013-v3.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.23013.011</object-id><label>Table 1.</label><caption><p>Estimated rates of replacement of CD4 effector and central memory through influx of new cells in 14 week-old mice. We quote both absolute influx (cells per day as a percentage of the pool size) and percent replaced per week. The latter is slightly less than 7 <inline-formula><mml:math id="inf5"><mml:mo>&#215;</mml:mo></mml:math></inline-formula> the&#160;daily rate of influx because immigrant cells are assumed to be lost at the same rate as existing displaceable cells (see Methods). AIC differences (<inline-formula><mml:math id="inf6"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>AIC) are quoted relative to the best-fitting model for each cell type. These differences reflect the relative support for two models, with exp(<inline-formula><mml:math id="inf7"><mml:mrow><mml:mo>-</mml:mo><mml:mi mathvariant="normal">&#916;</mml:mi></mml:mrow></mml:math></inline-formula>AIC/2) being the relative probability that it is the model with the lower penalised likelihood (larger AIC value) that minimises the information lost in describing the data. For CD4 T<sub>EM</sub> the two models have equal support, but for CD4 T<sub>CM</sub> the resistant memory model is favoured (<inline-formula><mml:math id="inf8"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>AIC&#160;=&#160;8.3, exp(<inline-formula><mml:math id="inf9"><mml:mrow><mml:mo>-</mml:mo><mml:mi mathvariant="normal">&#916;</mml:mi></mml:mrow></mml:math></inline-formula>AIC/2)=0.02). Models considered most plausible are highlighted.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.23013.011">http://dx.doi.org/10.7554/eLife.23013.011</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th/><th colspan="5">Resistant memory model</th><th colspan="5">Declining recruitment model</th></tr><tr><th>Source</th><th colspan="2">% Input/day</th><th colspan="2">% Replaced/wk</th><th><inline-formula><mml:math id="inf10"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>AIC</th><th colspan="2">% Input/day</th><th colspan="2">% Replaced/wk</th><th><inline-formula><mml:math id="inf11"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>AIC</th></tr></thead><tbody><tr><td>CD4 T<sub>EM</sub></td><td style="author-callout-style-b1">Naive</td><td style="author-callout-style-b1">1.0</td><td style="author-callout-style-b1">(0.7, 1.4)</td><td style="author-callout-style-b1">6.3</td><td style="author-callout-style-b1">(4.4, 8.6)</td><td style="author-callout-style-b1">0.16</td><td style="author-callout-style-b1">1.0</td><td style="author-callout-style-b1">(0.7, 1.6)</td><td style="author-callout-style-b1">6.4</td><td style="author-callout-style-b1">(4.3, 9.7)</td><td style="author-callout-style-b1">0</td></tr><tr><td/><td>CM</td><td>4.1</td><td>(2.6, 7.5)</td><td>23.0</td><td>(16, 35)</td><td>11</td><td>3.6</td><td>(2.1, 6.2)</td><td>21</td><td>(13, 30)</td><td>11</td></tr><tr><td>CD4 T<sub>CM</sub></td><td style="author-callout-style-b1">Naive</td><td style="author-callout-style-b1">1.5</td><td style="author-callout-style-b1">(1.0, 2.8)</td><td style="author-callout-style-b1">9.8</td><td style="author-callout-style-b1">(6.5, 17)</td><td style="author-callout-style-b1">0</td><td>2.3</td><td>(1.3, 4.1)</td><td>13.5</td><td>(8.4, 22)</td><td>8.3</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23013-v3.xml">elife-23013-v3.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.23013.019</object-id><label>Table 2.</label><caption><p>Parameters describing homeostasis of murine CD4 memory subsets, using a two-population model of kinetic heterogeneity and the best estimates of the magnitudes of the influx into each subset from the naive pool. Pool-average lifetimes and interdivision times are defined to be the mean of the corresponding quantities for the fast and slow subpopulations weighted by their size estimates. The best-fitting models were those that assumed no difference in death rates of Ki67<sup>high</sup> and Ki67<sup>low</sup> cells (indicated by rows in which ratio of loss rates&#160;=&#160;1).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.23013.019">http://dx.doi.org/10.7554/eLife.23013.019</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Parameter</th><th>Ratio of loss rates</th><th colspan="2">CD4 effector memory</th><th colspan="2">CD4 central memory</th></tr><tr><th>Ki67<sup>high</sup>:Ki67<sup>low</sup></th><th>Estimate</th><th>95% CI</th><th>Estimate</th><th>95% CI</th></tr></thead><tbody><tr><td>Pool-average lifetime (days)</td><td>1 <break/>10 <break/>0.1</td><td>29 <break/>41 <break/>55</td><td>(28, 30) <break/>(39, 42) <break/>(54, 56)</td><td>21 <break/>27 <break/>44</td><td>(19, 24) <break/>(26, 35) <break/>(39, 51)</td></tr><tr><td>Pool-average interdivision time (days)</td><td>1 <break/>10 <break/>0.1</td><td>88 <break/>63 <break/>84</td><td>(83, 158) <break/>(53, 353) <break/>(80, 108)</td><td>86 <break/>59 <break/>51</td><td>(47, 144) <break/>(35, 74) <break/>(38, 84)</td></tr><tr><td>Ki67 lifetime (days)</td><td>1 <break/>10 <break/>0.1</td><td>3.28 <break/>3.23 <break/>3.28</td><td>(3.14, 3.39) <break/>(3.03, 3.30) <break/>(3.18, 3.41)</td><td>3.59 <break/>3.74 <break/>3.32</td><td>(3.47, 3.70) <break/>(3.57, 3.82) <break/>(3.31, 3.52)</td></tr><tr><td>Efficiency of BrdU uptake (%)</td><td>1 <break/>10 <break/>0.1</td><td>76 <break/>75 <break/>78</td><td>(74, 79) <break/>(73, 78) <break/>(76, 81)</td><td>77 <break/>78 <break/>78</td><td>(76, 79) <break/>(76, 79) <break/>(78, 81)</td></tr><tr><td>Delay before source switches&#160;to BrdU<sup>&#8722;</sup>&#160;post-labelling (days)</td><td>1 <break/>10 <break/>0.1</td><td>2.5 <break/>2.6 <break/>2.1</td><td>(1.7, 3.0) <break/>(1.9, 3.5) <break/>(1.3, 2.3)</td><td>0.085 <break/>0.006 <break/>0.14</td><td>(0.002, 1.16) <break/>(0.003, 1.43) <break/>(0.003, 0.66)</td></tr><tr><td>Source contribution&#160;to peripheral production (fraction)</td><td>1 <break/>10 <break/>0.1</td><td>0.12 <break/>0.11 <break/>0.15</td><td>(0.12, 0.13) <break/>(0.099, 0.11) <break/>(0.15, 0.16)</td><td>0.092 <break/>0.085 <break/>0.12</td><td>(0.088, 0.096) <break/>(0.08, 0.088) <break/>(0.11, 0.12)</td></tr><tr><td>Fraction of source enteringslow subpopulation</td><td>1 <break/>10 <break/>0.1</td><td>1 <break/>1 <break/>1</td><td>(0.98, 1) <break/>(0.95, 1) <break/>(0.98, 1)</td><td>0.69 <break/>1 <break/>0.46</td><td>(0.38, 0.85) <break/>(0.45, 1) <break/>(0.19, 0.62)</td></tr><tr><td>Mean lifetime offast subpopulation (days)</td><td>1 <break/>10 <break/>0.1</td><td>5.7 <break/>17 <break/>18</td><td>(5.5, 6.8) <break/>(15, 20) <break/>(18, 21)</td><td>3.3 <break/>7.1 <break/>6.4</td><td>(3.1, 3.4) <break/>(6.2, 7.5) <break/>(6.4, 6.9)</td></tr><tr><td>Mean lifetime ofslow subpopulation (days)</td><td>1 <break/>10 <break/>0.1</td><td>43 <break/>61 <break/>75</td><td>(43, 48) <break/>(60, 65) <break/>(74, 77)</td><td>38 <break/>48 <break/>76</td><td>(34, 44) <break/>(47, 65) <break/>(68, 90)</td></tr><tr><td>Mean interdivision time offast subpopulation (days)</td><td>1 <break/>10 <break/>0.1</td><td>5.7 <break/>5.7 <break/>6.8</td><td>(5.5, 6.9) <break/>(5.1, 6.7) <break/>(6.8, 7.6)</td><td>3.4 <break/>3.3 <break/>4.2</td><td>(3.2, 3.5) <break/>(3.0, 3.4) <break/>(4.2, 4.4)</td></tr><tr><td>Mean interdivision time ofslow subpopulation (days)</td><td>1 <break/>10 <break/>0.1</td><td>138 <break/>113 <break/>125</td><td>(130, 275) <break/>(90, 750) <break/>(119, 169)</td><td>167 <break/>118 <break/>92</td><td>(89, 280) <break/>(72, 151) <break/>(69, 151)</td></tr><tr><td>Size of fast subpopulation(fraction of total)</td><td>1 <break/>10 <break/>0.1</td><td>0.38 <break/>0.46 <break/>0.35</td><td>(0.37, 0.44) <break/>(0.43, 0.54) <break/>(0.34, 0.38)</td><td>0.49 <break/>0.52 <break/>0.47</td><td>(0.47, 0.51) <break/>(0.49, 0.53) <break/>(0.45, 0.48)</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23049-v2.xml">elife-23049-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.23049.003</object-id><label>Table 1.</label><caption><p>Selected proteins identified in IgG-sepharose pulldowns on ZZ-TAPBPR</p><p>Affinity chromatography with IgG-sepharose was performed on HeLaM cells expressing a protein-A-tagged TAPBPR molecule (ZZ-TAPBPR) or HeLaM cells transduced with an empty vector (control). Immunoprecipitates were analysed by in gel tryptic digest followed by liquid chromatography-tandem mass spectrometry and data were processed using Scaffold. Identified proteins are shown with their exclusive unique peptide count, percentage coverage, and exclusive unique spectrum count as determined by Scaffold. Rank denotes the position when data are sorted by exclusive unique peptide count with all proteins present in the control removed. Pep: exclusive unique peptide count; Cov: percentage coverage; Count: exclusive unique spectrum count.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.23049.003">http://dx.doi.org/10.7554/eLife.23049.003</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Protein</th><th rowspan="2" valign="top">Gene name</th><th colspan="2" valign="top">Control</th><th colspan="2" valign="top">ZZ-TAPBPR</th><th rowspan="2" valign="top">Rank</th></tr><tr><th valign="top">Pep&#160;(Cov)</th><th valign="top">Count</th><th valign="top">Pep&#160;(Cov)</th><th valign="top">Count</th></tr></thead><tbody><tr><td valign="top">Tapasin-related protein</td><td valign="top"><italic>TAPBPL</italic></td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">8 (16)</td><td valign="top">11</td><td valign="top">2</td></tr><tr><td valign="top">HLA class 1, A-68</td><td valign="top"><italic>HLA-A</italic></td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">14 (35)</td><td valign="top">21</td><td valign="top">1</td></tr><tr><td valign="top">&#946;-2-microglobulin</td><td valign="top"><italic>&#946;2M</italic></td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">1 (8.4)</td><td valign="top">1</td><td valign="top">95</td></tr><tr><td valign="top">UDP-glucose:glycoprotein glucosyltransferase&#160;1</td><td valign="top"><italic>UGGT1</italic></td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">10 (7.3)</td><td valign="top">10</td><td valign="top">3</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23049-v2.xml">elife-23049-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.23049.009</object-id><label>Table 2.</label><caption><p>Selected proteins identified in TAPBPR co-immunoprecipitates</p><p>TAPBPR was immunuoprecipitated using PeTe4 from IFN-&#947;-treated HeLaM-TAPBPR<sup>KO</sup>(HeLaM<sup>KO</sup>) cells reconstituted with either TAPBPR<sup>WT</sup> or TAPBPR<sup>C94A</sup>. Immunoprecipitates were analysed&#160;by in gel tryptic digest followed by liquid chromatography-tandem mass spectrometry and data were processed using Scaffold. Identified proteins are shown with their exclusive unique peptide count, total percentage coverage, and exclusive unique spectrum count as determined by Scaffold. Pep: exclusive unique peptide count; Cov: percentage coverage; Count: exclusive unique spectrum count.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.23049.009">http://dx.doi.org/10.7554/eLife.23049.009</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Protein</th><th rowspan="2" valign="top">Gene name</th><th colspan="2" valign="top">TAPBPR<sup>WT</sup></th><th colspan="2" valign="top">TAPBPR<sup>C94A</sup></th></tr><tr><th valign="top">Pep&#160;(Cov)</th><th valign="top">Count</th><th valign="top">Pep&#160;(Cov)</th><th valign="top">Count</th></tr></thead><tbody><tr><td valign="top">Tapasin-related protein</td><td valign="top"><italic>TAPBPL</italic></td><td valign="top">32 (43)</td><td valign="top">54</td><td valign="top">29 (47)</td><td valign="top">50</td></tr><tr><td valign="top">HLA class 1, A-68</td><td valign="top"><italic>HLA-A</italic></td><td valign="top">50 (64)</td><td valign="top">88</td><td valign="top">41 (59)</td><td valign="top">70</td></tr><tr><td valign="top">&#946;-2-microglobulin</td><td valign="top"><italic>&#946;2M</italic></td><td valign="top">4 (46)</td><td valign="top">7</td><td valign="top">1 (8.4)</td><td valign="top">1</td></tr><tr><td valign="top">UDP-glucose:glycoprotein glucosyltransferase&#160;1</td><td valign="top"><italic>UGGT1</italic></td><td valign="top">19 (11)</td><td valign="top">25</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23481-v2.xml">elife-23481-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.23481.006</object-id><label>Table 1.</label><caption><p>Numbers of IL17 genes by subfamily in echinoderm species.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.23481.006">http://dx.doi.org/10.7554/eLife.23481.006</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th/><th colspan="6">Echinodermata</th></tr><tr><th/><th colspan="5">Echinoidea</th><th rowspan="3">Asteroidea</th></tr><tr><th/><th colspan="4">Euechinoidea</th><th rowspan="2">Cidaroidea</th></tr><tr><th/><th colspan="3">Strongylocentrotidae</th><th>Toxopneustidae</th></tr><tr><th>Subfamily</th><th valign="top"><italic>S. purpuratus</italic></th><th><italic>S. fragilis*</italic> <break/>(5&#8211;7 myr)<sup>&#8224;</sup></th><th><italic>M. franciscanus*</italic><sup>a</sup> <break/>(20 myr)<sup>&#8224;</sup></th><th><italic>L. variegatus</italic> <break/>(50 myr)<sup>&#8224;</sup></th><th><italic>E. tribuloides</italic> <break/>(268 myr)<sup>&#8224;</sup></th><th valign="top"><italic>P. miniata</italic> <break/>(480 myr)<sup>&#8224;</sup></th></tr></thead><tbody><tr><td>1</td><td>11</td><td>8.6</td><td>10.0</td><td>2</td><td>6</td><td>0</td></tr><tr><td>2</td><td>1</td><td>0.5</td><td>0.4</td><td>1</td><td>1</td><td>0</td></tr><tr><td>3</td><td>1</td><td>0.5</td><td>0.9</td><td>2</td><td>0</td><td>0</td></tr><tr><td>4</td><td>1</td><td>1.9</td><td>0.4</td><td>1</td><td>3</td><td>0</td></tr><tr><td>5</td><td>2</td><td>3.3</td><td>3.5</td><td>2</td><td>2</td><td valign="top">0</td></tr><tr><td>6</td><td>3</td><td>4.4</td><td>4.4</td><td>2</td><td>3</td><td valign="top">0</td></tr><tr><td>7</td><td>1</td><td>2.9</td><td>7.8</td><td>1</td><td>2</td><td valign="top">0</td></tr><tr><td>8</td><td>1</td><td>1.9</td><td>6.5</td><td>1</td><td>2</td><td valign="top">0</td></tr><tr><td>9</td><td>7</td><td>13.8</td><td>7.1</td><td>1</td><td>0</td><td valign="top">0</td></tr><tr><td>10</td><td>2</td><td>1.0</td><td>5.2</td><td>1</td><td>0</td><td>0</td></tr><tr><td>Other</td><td>-</td><td>-</td><td>-</td><td>-</td><td>4</td><td>12<sup>&#8225;</sup></td></tr><tr><td>Total</td><td>30</td><td>38.6</td><td>47.0</td><td>15</td><td>22</td><td>12</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>*Estimates are based on the number of best reciprocal blast hits using the SpIL17 sequences against the unassembled genomic trace sequences (<xref ref-type="bibr" rid="bib6">Buckley and Rast, 2012</xref>).</p></fn><fn id="tblfn2"><p><sup>&#8224;</sup>Estimated divergence times shown in million years from <italic>S. purpuratus</italic> (<xref ref-type="bibr" rid="bib79">Thompson et al., 2015</xref>; <xref ref-type="bibr" rid="bib50">Pisani et al., 2012</xref>; <xref ref-type="bibr" rid="bib4">Biermann et al., 2003</xref>; <xref ref-type="bibr" rid="bib67">Smith et al., 2006</xref>).</p></fn><fn id="tblfn3"><p><sup>&#8225;</sup>See <xref ref-type="fig" rid="fig2s1">Figure 2&#8212;figure supplement 1</xref> for the phylogenetic analysis of these genes.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23551-v2.xml">elife-23551-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.23551.003</object-id><label>Table 1.</label><caption><p>Mean offspring numbers, TFT haplotype frequencies, and genotyping outcomes for the two experiments (regulated conditions and fluctuating conditions).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.23551.003">http://dx.doi.org/10.7554/eLife.23551.003</ext-link></p><p><supplementary-material id="SD1-data"><object-id pub-id-type="doi">10.7554/eLife.23551.004</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Raw data for offspring number and TFT frequency for Experiments 1 and 2.</title><p>Offspring number per population for each of 10 generations in Experiment 1 and 2. TFT frequency for each population at generations 1, 5 and 10 for Experiment 1 and at generation 10 for Experiment 2.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.23551.004">http://dx.doi.org/10.7554/eLife.23551.004</ext-link></p></caption><media mime-subtype="excel" mimetype="application" xlink:href="elife-23551-table1-data1-v2.xls"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2" valign="bottom"/><th valign="bottom">TFT treatment (starting frequency)</th><th valign="bottom">0% TFT</th><th valign="bottom">25% TFT</th><th valign="bottom">50% TFT</th><th valign="bottom">75% TFT</th></tr><tr><th valign="bottom">Populations [n]</th><th valign="bottom">21</th><th valign="bottom">21</th><th valign="bottom">21</th><th valign="bottom">21</th></tr></thead><tbody><tr><td rowspan="14">Experiment</td><td rowspan="8">Regulated</td><td valign="bottom">F<sub>1</sub> Frequency</td><td valign="bottom">0.00</td><td valign="bottom">0.26&#160;&#177;&#160;0.03</td><td valign="bottom">0.44&#160;&#177;&#160;0.03</td><td valign="bottom">0.63&#160;&#177;&#160;0.03</td></tr><tr><td valign="bottom">F<sub>5</sub> Frequency</td><td valign="bottom">0.00</td><td valign="bottom">0.15&#160;&#177;&#160;0.03</td><td valign="bottom">0.63&#160;&#177;&#160;0.03</td><td valign="bottom">0.71&#160;&#177;&#160;0.02</td></tr><tr><td valign="bottom">F<sub>10</sub> Frequency</td><td valign="bottom">0.00</td><td valign="bottom">0.17&#160;&#177;&#160;0.03</td><td valign="bottom">0.67&#160;&#177;&#160;0.03</td><td valign="bottom">0.80&#160;&#177;&#160;0.03</td></tr><tr><td valign="bottom">Loss</td><td valign="bottom">-</td><td valign="bottom">4</td><td valign="bottom">0</td><td valign="bottom">0</td></tr><tr><td valign="bottom">Fixation</td><td valign="bottom">-</td><td valign="bottom">0</td><td valign="bottom">0</td><td valign="bottom">3</td></tr><tr><td valign="bottom">Heteroplasmy</td><td valign="bottom">-</td><td valign="bottom">0</td><td valign="bottom">0</td><td valign="bottom">0</td></tr><tr><td valign="bottom">Mean offspring number (F<sub>10</sub>)</td><td valign="bottom">74.15&#160;&#177;&#160;1.00</td><td valign="bottom">72.05&#160;&#177;&#160;0.89</td><td valign="bottom">67.43&#160;&#177;&#160;0.99</td><td valign="bottom">66.38&#160;&#177;&#160;0.70</td></tr><tr><td valign="bottom">Population extinction</td><td valign="bottom">1</td><td valign="bottom">0</td><td valign="bottom">0</td><td valign="bottom">0</td></tr><tr><td rowspan="6">Fluctuating</td><td valign="bottom">F<sub>10</sub>frequency</td><td valign="bottom">0.00</td><td valign="bottom">0.35&#160;&#177;&#160;0.06</td><td valign="bottom">0.59&#160;&#177;&#160;0.06</td><td valign="bottom">0.75&#160;&#177;&#160;0.05</td></tr><tr><td valign="bottom">Loss</td><td valign="bottom">-</td><td valign="bottom">4</td><td valign="bottom">1</td><td valign="bottom">0</td></tr><tr><td valign="bottom">Fixation</td><td valign="bottom">-</td><td valign="bottom">1</td><td valign="bottom">4</td><td valign="bottom">4</td></tr><tr><td valign="bottom">Heteroplasmy</td><td valign="bottom">-</td><td valign="bottom">0</td><td valign="bottom">2</td><td valign="bottom">0</td></tr><tr><td valign="bottom">Mean offspring number (F<sub>10</sub>)</td><td valign="bottom">80.47&#160;&#177;&#160;5.12</td><td valign="bottom">81.19&#160;&#177;&#160;4.59</td><td valign="bottom">78.56&#160;&#177;&#160;5.63</td><td valign="bottom">85.86&#160;&#177;&#160;3.73</td></tr><tr><td valign="bottom">Population extinction</td><td valign="bottom">2</td><td valign="bottom">0</td><td valign="bottom">2</td><td valign="bottom">0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23584-v4.xml">elife-23584-v4.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.23584.009</object-id><label>Table 1.</label><caption><p>Comparison between the response rate for different sensory stimuli applied to the fore&#8211; or hindlimb of a rat standing on four or two limbs. Results of Fisher Exact test, 2 &#215; 2 contingency tables. Values in red are significant. Values in black are not significant. In parenthesis is the rate for full withdrawal for each condition.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.23584.009">http://dx.doi.org/10.7554/eLife.23584.009</ext-link></p><p><supplementary-material id="SD1-data"><object-id pub-id-type="doi">10.7554/eLife.23584.010</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Proportion of withdrawals for different sensory stimuli applied to the fore- or hindlimb of a rat standing on four or two limbs.</title><p>Source data and Column histogram presenting the number of trials the rats responded with a full withdrawal to a stimulus. Each column presents the total number of trials. (Black: full withdrawal; Gray: no withdrawal). l-laser stimulus; s-sharp stimulus; 4-animal on all four limbs; 2-animal on two hind limbs; f -forelimb stimulation; h -hindlimb stimulation.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.23584.010">http://dx.doi.org/10.7554/eLife.23584.010</ext-link></p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-23584-table1-data1-v4.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3" rowspan="3" valign="bottom"/><th colspan="4" valign="top">Laser</th><th colspan="4" valign="top">Sharp</th></tr><tr><th colspan="2" valign="top">Standing on four</th><th colspan="2" valign="top">Standing on two</th><th colspan="2" valign="top">Standing on four</th><th colspan="2" valign="top">Standing on two</th></tr><tr><th valign="top">Forelimb <break/>(%100)</th><th valign="top">Hindlimb <break/>(%82.8)</th><th valign="top">Hindlimb <break/>(%48.7)</th><th valign="top">Hindlimb+NL <break/>(%63.2)</th><th valign="top">Forelimb <break/>(%100)</th><th valign="top">Hindlimb <break/>(%100)</th><th valign="top">Hindlimb <break/>(%66.7)</th><th valign="top">Hindlimb+NL <break/>(%55)</th></tr></thead><tbody><tr><td rowspan="4">Laser</td><td rowspan="2">Standing <break/>on four</td><td>Forelimb <break/>(%100)</td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td></tr><tr><td>Hindlimb <break/>(%82.8)</td><td>1.0</td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td></tr><tr><td rowspan="2">Standing <break/>on two</td><td>Hindlimb <break/>(%48.7)</td><td>&gt;0.05</td><td><named-content content-type="author-callout-style-a3">&lt;0.01</named-content></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td></tr><tr><td>Hindlimb+NL <break/>(%63.2)</td><td>&gt;0.2</td><td>&gt;0.1</td><td>&gt;0.4</td><td><break/></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td></tr><tr><td rowspan="4">Sharp</td><td rowspan="2">Standing <break/>on four</td><td>Forelimb <break/>(%100)</td><td>1.0</td><td>&gt;0.1</td><td><named-content content-type="author-callout-style-a3">&lt;0.001</named-content></td><td><named-content content-type="author-callout-style-a3">&lt;0.05</named-content></td><td><break/></td><td><break/></td><td><break/></td><td><break/></td></tr><tr><td>Hindlimb <break/>(%100)</td><td>1.0</td><td style="color: #D50000;">&lt;0.05</td><td style="color: #D50000;">&lt;0.001</td><td style="color: #D50000;">&lt;0.001</td><td>1.0</td><td><break/></td><td><break/></td><td><break/></td></tr><tr><td rowspan="2">Standing <break/>on two</td><td>Hindlimb <break/>(%66.7)</td><td>&gt;0.3</td><td>&gt;0.1</td><td>&gt;0.1</td><td>&gt;0.7</td><td style="color: #D50000;">&lt;0.05</td><td style="color: #D50000;">&lt;0.001</td><td><break/></td><td><break/></td></tr><tr><td>Hindlimb+NL <break/>(%55)</td><td>&gt;0.1</td><td>&gt;0.5</td><td>&gt;0.7</td><td>&gt;0.7</td><td style="color: #D50000;">&lt;0.001</td><td style="color: #D50000;">&lt;0.001</td><td>&gt;0.4</td><td><break/></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23608-v2.xml">elife-23608-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.23608.009</object-id><label>Table 2.</label><caption><p>Results from voxelwise GBC analysis.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.23608.009">http://dx.doi.org/10.7554/eLife.23608.009</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Label</th><th rowspan="2">Volume</th><th rowspan="2"><italic>t</italic>-value</th><th colspan="3" valign="bottom">Peak activation (LPI)</th></tr><tr><th valign="bottom">x</th><th valign="bottom">y</th><th valign="bottom">z</th></tr></thead><tbody><tr><td valign="bottom">Right Angular Gyrus</td><td valign="bottom">158</td><td valign="bottom">3.45</td><td valign="bottom">48</td><td valign="bottom">&#8722;51</td><td valign="bottom">27</td></tr><tr><td valign="bottom">Right Intraparietal Sulcus</td><td valign="bottom">83</td><td valign="bottom">3.42</td><td valign="bottom">21</td><td valign="bottom">&#8722;60</td><td valign="bottom">66</td></tr><tr><td valign="bottom">Left Intraparietal Sulcus</td><td valign="bottom">81</td><td valign="bottom">3.6</td><td valign="bottom">&#8722;18</td><td valign="bottom">&#8722;63</td><td valign="bottom">66</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23608-v2.xml">elife-23608-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.23608.012</object-id><label>Table 3.</label><caption><p>Results from voxelwise IPS connectivity analysis.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.23608.012">http://dx.doi.org/10.7554/eLife.23608.012</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Label</th><th rowspan="2">Volume</th><th rowspan="2"><italic>t</italic>-value</th><th colspan="3" valign="bottom">Peak activation (LPI)</th></tr><tr><th valign="bottom">x</th><th valign="bottom">y</th><th valign="bottom">z</th></tr></thead><tbody><tr><td valign="bottom">Left Thalamus</td><td valign="bottom">342</td><td valign="bottom">3.92</td><td valign="bottom">-9</td><td valign="bottom">6</td><td valign="bottom">12</td></tr><tr><td valign="bottom">Right Inferior Parietal Lobule</td><td valign="bottom">208</td><td valign="bottom">3.67</td><td valign="bottom">57</td><td valign="bottom">&#8722;57</td><td valign="bottom">39</td></tr><tr><td valign="bottom">Left Superior Medial Gyrus</td><td valign="bottom">184</td><td valign="bottom">3.65</td><td valign="bottom">3</td><td valign="bottom">36</td><td valign="bottom">42</td></tr><tr><td valign="bottom">Left Precuneus</td><td valign="bottom">179</td><td valign="bottom">3.59</td><td valign="bottom">3</td><td valign="bottom">&#8722;69</td><td valign="bottom">48</td></tr><tr><td valign="bottom">Right Middle Frontal Gyrus</td><td valign="bottom">137</td><td valign="bottom">3.64</td><td valign="bottom">33</td><td valign="bottom">15</td><td valign="bottom">60</td></tr><tr><td valign="bottom">Left Angular Gyrus</td><td valign="bottom">113</td><td valign="bottom">3.51</td><td valign="bottom">&#8722;57</td><td valign="bottom">&#8722;54</td><td valign="bottom">30</td></tr><tr><td valign="bottom">Left Middle Frontal Gyrus</td><td valign="bottom">96</td><td valign="bottom">3.69</td><td valign="bottom">&#8722;24</td><td valign="bottom">15</td><td valign="bottom">60</td></tr><tr><td valign="bottom">Left Middle Frontal Gyrus</td><td valign="bottom">90</td><td valign="bottom">3.48</td><td valign="bottom">&#8722;45</td><td valign="bottom">51</td><td valign="bottom">-3</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23708-v2.xml">elife-23708-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.23708.004</object-id><label>Table 2.</label><caption><title><italic>Plasmodium</italic> infection status at enrolment by village.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Village</th><th colspan="5" valign="top"><italic>P. falciparum</italic></th><th colspan="5" valign="top"><italic>P. vivax</italic></th><th colspan="3" valign="top"><italic>P. malariae</italic></th><th colspan="3" valign="top"><italic>P. ovale</italic></th></tr><tr><th>N pos.</th><th>Prevalence by qPCR (CI<sub>95</sub>)</th><th>% mixed<sup>*</sup></th><th>Mean<sup>&#8224;</sup>&#160;density (IQR)</th><th>Mean MOI<sup>&#8225;</sup>&#160;(range)</th><th>N pos.</th><th>Prevalence by qPCR <break/>(CI<sub>95</sub>)</th><th>% mixed<sup>*</sup></th><th>Mean<sup>&#8224;</sup>&#160;density (IQR)</th><th>Mean MOI<sup>&#8225;</sup>&#160;(range)</th><th>N pos.</th><th>Prevalence by qPCR <break/>(CI<sub>95</sub>)</th><th>% mixed<sup>*</sup></th><th>N pos.</th><th>Prevalence by qPCR <break/>(CI<sub>95</sub>)</th><th>% mixed<sup>*</sup></th></tr></thead><tbody><tr><td>Albinama</td><td>18</td><td>18 (11&#8211;27)</td><td>72</td><td>131 (38&#8211;189)</td><td>1.4 (1&#8211;4)</td><td>54</td><td>55 (44&#8211;65)</td><td>24</td><td>3 (1&#8211;17)</td><td>1.8 (1&#8211;7)</td><td>9</td><td>9 (5&#8211;17)</td><td>67</td><td>5</td><td>5 (2&#8211;12)</td><td>100</td></tr><tr><td>Amahup</td><td>14</td><td>12 (7&#8211;19)</td><td>57</td><td>56 (14&#8211;105)</td><td>1.6 (1&#8211;5)</td><td>46</td><td>39 (30&#8211;48)</td><td>24</td><td>3 (1&#8211;29)</td><td>2.2 (1&#8211;7)</td><td>12</td><td>10 (6&#8211;17)</td><td>83</td><td>0</td><td>0</td><td/></tr><tr><td>Balanga</td><td>15</td><td>28 (17&#8211;42)</td><td>67</td><td>79 (30&#8211;848)</td><td>1.7 (1&#8211;5)</td><td>23</td><td>43 (30&#8211;57)</td><td>43</td><td>2 (1&#8211;28)</td><td>2.0 (1&#8211;7)</td><td>9</td><td>17 (8&#8211;30)</td><td>56</td><td>2</td><td>3 (0&#8211;14)</td><td>50</td></tr><tr><td>Balif</td><td>8</td><td>9 (4&#8211;17)</td><td>63</td><td>64 (10&#8211;325)</td><td>2.0 (1&#8211;4)</td><td>35</td><td>38 (30&#8211;48)</td><td>14</td><td>2 (1&#8211;14)</td><td>1.9 (1&#8211;6)</td><td>7</td><td>8 (3&#8211;15)</td><td>57</td><td>0</td><td>0</td><td/></tr><tr><td>Bolumita</td><td>50</td><td>71 (59&#8211;81)</td><td>80</td><td>331 (62&#8211;1988)</td><td>2.2 (1&#8211;8)</td><td>47</td><td>67 (55&#8211;78)</td><td>81</td><td>3 (2&#8211;27)</td><td>2.9 (1&#8211;10)</td><td>28</td><td>40 (29&#8211;52)</td><td>89</td><td>8</td><td>11 (5&#8211;22)</td><td>100</td></tr><tr><td>Numangu</td><td>8</td><td>26 (13&#8211;45)</td><td>75</td><td>192 (30&#8211;848)</td><td>1.1 (1&#8211;2)</td><td>18</td><td>58 (39&#8211;75)</td><td>28</td><td>3 (1&#8211;25)</td><td>1.6 (1&#8211;5)</td><td>4</td><td>13 (4&#8211;31)</td><td>50</td><td>0</td><td>0</td><td/></tr><tr><td><bold>Overall</bold></td><td><bold>113</bold></td><td><bold>24 (20&#8211;28)</bold></td><td><bold>73</bold></td><td><bold>163 (20&#8211;1103)</bold></td><td><bold>1.9 (1&#8211;8)</bold></td><td><bold>223</bold></td><td><bold>48 (43&#8211;52)</bold></td><td><bold>37</bold></td><td><bold>3 (1&#8211;23)</bold></td><td><bold>2.2 (1&#8211;10)</bold></td><td><bold>69</bold></td><td><bold>15 (12&#8211;18)</bold></td><td><bold>75</bold></td><td><bold>15</bold></td><td><bold>3 (2&#8211;5)</bold></td><td><bold>93</bold></td></tr><tr><td><bold><italic>p</italic>-value<sup>&#167;</sup></bold></td><td/><td>&lt;0.001</td><td>0.034</td><td>0.086</td><td>0.047</td><td/><td>&lt;0.001</td><td>&lt;0.001</td><td>0.947</td><td>0.020</td><td/><td>&lt;0.001</td><td>0.086</td><td/><td>&lt;0.001</td><td>0.133</td></tr></tbody></table><table-wrap-foot><fn><p>* % of infections by qPCR that are mixed-species infections.</p><p>&#8224; Geometric mean of species-specific <italic>18S rRNA</italic> copy numbers per &#181;l blood.</p></fn><fn><p>&#8225; MOI, multiplicity of infection: number of <italic>Pf-msp2</italic> and <italic>Pv-msp1F3</italic> alleles per infection.</p><p>&#167; Differences between villages were tested for using Chi<sup>2</sup> and Fisher&#8217;s exact test (prevalence, proportion mixed) or Kruskal-Wallis test (MOI, log<sub>10</sub>-transformed parasite density).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23708-v2.xml">elife-23708-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.23708.008</object-id><label>Table 3.</label><caption><title>Multivariable predictors for time to recurrent blood-stage infection with <italic>Plasmodium</italic> species by qPCR</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Variable</th><th colspan="3"><italic>P. vivax</italic></th><th colspan="3"><italic>P. falciparum</italic></th><th colspan="3"><italic>P. malariae</italic></th><th colspan="3"><italic>P. ovale</italic></th></tr><tr><th>AHR<sup>*</sup></th><th>CI<sub>95</sub></th><th><italic>p</italic>-value</th><th>AHR<sup>*</sup></th><th>CI<sub>95</sub></th><th><italic>p</italic>-value</th><th>AHR<sup>*</sup></th><th>CI<sub>95</sub></th><th><italic>p</italic>-value</th><th>AHR<sup>*</sup></th><th>CI<sub>95</sub></th><th><italic>p</italic>-value</th></tr></thead><tbody><tr><td>PQ treatment</td><td>0.18</td><td>0.13&#8211;0.25</td><td>&lt;0.001</td><td>0.73</td><td>0.52&#8211;1.02</td><td>0.064</td><td>0.51</td><td>0.22&#8211;1.19</td><td>0.121</td><td>0.31</td><td>0.12&#8211;0.75</td><td>0.010</td></tr><tr><td>Age</td><td>0.95</td><td>0.87&#8211;1.04</td><td>0.247</td><td>1.05</td><td>0.94&#8211;1.17</td><td>0.361</td><td>0.98</td><td>0.75&#8211;1.29</td><td>0.905</td><td>0.96</td><td>0.74&#8211;1.26</td><td>0.793</td></tr><tr><td>LLIN use at enrolment</td><td>0.62</td><td>0.39&#8211;0.98</td><td>0.043</td><td>0.84</td><td>0.49&#8211;1.44</td><td>0.531</td><td>1.33</td><td>0.33&#8211;6.09</td><td>0.715</td><td>0.95</td><td>0.26&#8211;3.43</td><td>0.936</td></tr><tr><td>Hb at enrolment (g/dl)</td><td>0.88</td><td>0.80&#8211;0.98</td><td>0.019</td><td>0.90</td><td>0.80&#8211;1.02</td><td>0.099</td><td>0.83</td><td>0.61&#8211;1.12</td><td>0.224</td><td>0.92</td><td>0.66&#8211;1.28</td><td>0.634</td></tr><tr><td><italic>Village</italic></td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;Albinama (ref)</td><td>1</td><td/><td/><td>1</td><td/><td/><td>1</td><td/><td/><td>1</td><td/><td/></tr><tr><td>&#8195;Amahup</td><td>0.45</td><td>0.29&#8211;0.71</td><td>0.001</td><td>0.58</td><td>0.31&#8211;1.11</td><td>0.101</td><td>0.34</td><td>0.07&#8211;1.79</td><td>0.205</td><td>2.83</td><td>0.29&#8211;27.48</td><td>0.370</td></tr><tr><td>&#8195;Balanga</td><td>2.15</td><td>1.40&#8211;3.31</td><td>&lt;0.001</td><td>1.81</td><td>0.99&#8211;3.30</td><td>0.054</td><td>0.92</td><td>0.24&#8211;3.60</td><td>0.910</td><td>7.74</td><td>0.85&#8211;70.45</td><td>0.070</td></tr><tr><td>&#8195;Balif</td><td>1.00</td><td>0.66&#8211;1.54</td><td>0.983</td><td>0.60</td><td>0.30&#8211;1.19</td><td>0.145</td><td>0.24</td><td>0.03&#8211;2.07</td><td>0.193</td><td>4.60</td><td>0.51&#8211;41.41</td><td>0.173</td></tr><tr><td>&#8195;Bolumita</td><td>3.34</td><td>2.09&#8211;5.33</td><td>&lt;0.001</td><td>4.73</td><td>2.69&#8211;8.30</td><td>&lt;0.001</td><td>1.21</td><td>0.34&#8211;4.31</td><td>0.770</td><td>19.43</td><td>2.19&#8211;172.37</td><td>0.008</td></tr><tr><td>&#8195;Numangu</td><td>0.83</td><td>0.44&#8211;1.59</td><td>0.583</td><td>2.29</td><td>1.17&#8211;4.50</td><td>0.015</td><td>0.82</td><td>0.15&#8211;4.53</td><td>0.823</td><td>3.17</td><td>0.19&#8211;52.41</td><td>0.420</td></tr><tr><td colspan="13"><italic>Infection status at enrolment (by qPCR)</italic></td></tr><tr><td>&#8195;Uninfected (ref)</td><td>1</td><td/><td/><td>1</td><td/><td/><td>1</td><td/><td/><td>1</td><td/><td/></tr><tr><td>&#8195;<italic>P. vivax</italic></td><td>1.27</td><td>0.91&#8211;1.78</td><td>0.165</td><td>1.37</td><td>0.86&#8211;2.20</td><td>0.186</td><td>0.92</td><td>0.20&#8211;4.18</td><td>0.913</td><td>2.17</td><td>0.68&#8211;6.97</td><td>0.192</td></tr><tr><td>&#8195;<italic>P. falciparum</italic></td><td>1.36</td><td>0.84&#8211;2.19</td><td>0.205</td><td>1.56</td><td>0.86&#8211;2.82</td><td>0.145</td><td>3.54</td><td>0.85&#8211;14.72</td><td>0.083</td><td>1.25</td><td>0.26&#8211;5.90</td><td>0.779</td></tr><tr><td>&#8195;<italic>P. malariae</italic></td><td>0.83</td><td>0.38&#8211;1.85</td><td>0.655</td><td>0.99</td><td>0.38&#8211;2.56</td><td>0.977</td><td>6.35</td><td>1.31&#8211;30.81</td><td>0.022</td><td>1.58</td><td>0.17&#8211;14.30</td><td>0.676</td></tr><tr><td>&#8195;Mixed <italic>P.f. or P.v</italic>.<sup>&#8224;</sup></td><td>1.74</td><td>1.14&#8211;2.65</td><td>0.010</td><td>2.08</td><td>1.25&#8211;3.48</td><td>0.005</td><td>3.37</td><td>0.88&#8211;12.90</td><td>0.076</td><td>2.03</td><td>0.55&#8211;7.53</td><td>0.287</td></tr></tbody></table><table-wrap-foot><fn><p>* AHRs were modeled using Cox proportional hazard regression.</p><p>&#8224; Mixed infection including <italic>P. falciparum</italic> or <italic>P. vivax</italic> infection in conjunction with one or more other <italic>Plasmodium spp.</italic></p></fn><fn><p>PQ: Primaquine; LLIN: long-lasting insecticide-treated net; Hb: haemoglobin.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23708-v2.xml">elife-23708-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.23708.011</object-id><label>Table 4.</label><caption><title>Multivariable predictors of <italic>Pv-</italic> and <italic>Pf-</italic><sub>mol</sub>FOB per follow-up interval.</title><p>Model predictions from this model were used for mapping <sub>mol</sub>FOB in <xref ref-type="fig" rid="fig3">Figure 3A</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Variable</th><th colspan="6" valign="top"><italic>P. vivax</italic></th><th colspan="3" valign="top"><italic>P. falciparum</italic></th></tr><tr><th colspan="3" valign="top">PQ arm</th><th colspan="3" valign="top">Placebo arm</th><th colspan="3" valign="top">Combined arms</th></tr><tr><th valign="top">IRR<sup>*</sup></th><th valign="top">CI<sub>95</sub></th><th valign="top"><italic>p</italic>-value</th><th valign="top">IRR<sup>*</sup></th><th valign="top">CI<sub>95</sub></th><th valign="top"><italic>p</italic>-value</th><th valign="top">IRR<sup>*</sup></th><th valign="top">CI<sub>95</sub></th><th valign="top"><italic>p</italic>-value</th></tr></thead><tbody><tr><td>PQ treatment</td><td>n.a.<sup>&#8224;</sup></td><td>n.a.</td><td>n.a.</td><td>n.a.</td><td>n.a.</td><td>n.a.</td><td>0.89</td><td>0.65&#8211;1.22</td><td>0.474</td></tr><tr><td>New <italic>P. falc.</italic> infections in interval<sup>&#8225;</sup></td><td>1.32</td><td>0.92&#8211;1.89</td><td>0.134</td><td>1.10</td><td>0.85&#8211;1.42</td><td>0.466</td><td>n.a.</td><td>n.a.</td><td>n.a.</td></tr><tr><td>New <italic>P. vivax</italic> infections in interval<sup>&#8225;</sup></td><td>n.a.</td><td>n.a.</td><td>n.a.</td><td>n.a.</td><td>n.a.</td><td>n.a.</td><td>1.15</td><td>0.97&#8211;1.36</td><td>0.100</td></tr><tr><td>Age</td><td>0.86</td><td>0.74&#8211;1.01</td><td>0.059</td><td>0.95</td><td>0.87&#8211;1.04</td><td>0.305</td><td>1.03</td><td>0.92&#8211;1.14</td><td>0.640</td></tr><tr><td>LLIN use at enrolment</td><td>0.96</td><td>0.51&#8211;1.79</td><td>0.897</td><td>0.62</td><td>0.43&#8211;0.91</td><td>0.013</td><td>1.07</td><td>0.7&#8211;1.62</td><td>0.755</td></tr><tr><td>Hb at enrolment (g/dl)</td><td>0.85</td><td>0.72&#8211;1.01</td><td>0.063</td><td>0.91</td><td>0.85&#8211;0.99</td><td>0.025</td><td>0.85</td><td>0.75&#8211;0.97</td><td>0.013</td></tr><tr><td><italic>Village</italic></td><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;Albinama (ref)</td><td>1</td><td/><td/><td>1</td><td/><td/><td>1</td><td/><td/></tr><tr><td>&#8195;Amahup</td><td>0.02</td><td>0&#8211;0.11</td><td>&lt;0.001</td><td>0.56</td><td>0.34&#8211;0.91</td><td>0.020</td><td>0.52</td><td>0.25&#8211;1.07</td><td>0.074</td></tr><tr><td>&#8195;Balif</td><td>0.85</td><td>0.4&#8211;1.8</td><td>0.664</td><td>1.74</td><td>1.16&#8211;2.61</td><td>0.007</td><td>1.81</td><td>0.98&#8211;3.35</td><td>0.059</td></tr><tr><td>&#8195;Balanga</td><td>0.28</td><td>0.1&#8211;0.82</td><td>0.020</td><td>1.13</td><td>0.73&#8211;1.73</td><td>0.590</td><td>0.75</td><td>0.37&#8211;1.52</td><td>0.423</td></tr><tr><td>&#8195;Bolumita</td><td>1.52</td><td>0.73&#8211;3.17</td><td>0.268</td><td>2.67</td><td>1.83&#8211;3.9</td><td>&lt;0.001</td><td>6.05</td><td>3.32&#8211;11.05</td><td>&lt;0.001</td></tr><tr><td>&#8195;Numangu</td><td>0.5</td><td>0.15&#8211;1.68</td><td>0.264</td><td>0.76</td><td>0.4&#8211;1.43</td><td>0.394</td><td>2.8</td><td>1.39&#8211;5.64</td><td>0.004</td></tr><tr><td><italic>Study Day</italic></td><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;Day 0&#8211;35 (ref)</td><td>1</td><td/><td/><td>1</td><td/><td/><td>1</td><td/><td/></tr><tr><td>&#8195;Day 36&#8211;80</td><td>1.37</td><td>0.54&#8211;3.48</td><td>0.509</td><td>1.99</td><td>1.39&#8211;2.84</td><td>&lt;0.001</td><td>2.42</td><td>1.44&#8211;4.07</td><td>0.001</td></tr><tr><td>&#8195;Day 81&#8211;175</td><td>1.34</td><td>0.57&#8211;3.12</td><td>0.503</td><td>0.89</td><td>0.61&#8211;1.3</td><td>0.538</td><td>1.13</td><td>0.7&#8211;1.84</td><td>0.616</td></tr><tr><td>&#8195;Day&#160;&gt;&#160;175</td><td>0.65</td><td>0.25&#8211;1.69</td><td>0.374</td><td>0.56</td><td>0.38&#8211;0.83</td><td>0.004</td><td>0.87</td><td>0.48&#8211;1.56</td><td>0.643</td></tr></tbody></table><table-wrap-foot><fn><p>*IRRs were modeled per sampling interval using negative binomial generalized estimating equations allowing for repeated visits with log-link and an exchangeable correlation structure.</p><p>&#8224; n.a., not applicable.</p></fn><fn><p>&#8225;<sub>mol</sub>FOB in the follow-up interval (time-varying covariate).</p><p>PQ: Primaquine; LLIN: long-lasting insecticide-treated net; Hb: haemoglobin.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-23708-v2.xml">elife-23708-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table5" position="float"><object-id pub-id-type="doi">10.7554/eLife.23708.013</object-id><label>Table 5.</label><caption><title>Multivariable predictors for time to <italic>P. vivax</italic> and <italic>P. falciparum</italic> clinical episodes.</title><p>Model predictions from this model were used for mapping the relative risk of clinical malaria episodes in <xref ref-type="fig" rid="fig3">Figure 3C and D</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Variable</th><th colspan="3"><italic>P. vivax</italic></th><th colspan="3"><italic>P. falciparum</italic></th></tr><tr><th>AHR<sup>*</sup></th><th>CI<sub>95</sub></th><th><italic>p</italic>-value</th><th>AHR<sup>*</sup></th><th>CI<sub>95</sub></th><th><italic>p</italic>-value</th></tr></thead><tbody><tr><td>PQ treatment</td><td>0.76</td><td>0.34&#8211;1.68</td><td>0.497</td><td>1.79</td><td>1.05&#8211;3.03</td><td>0.031</td></tr><tr><td><italic>P. vivax</italic> <sub>mol</sub>FOB<sup>&#8225;</sup></td><td>1.07</td><td>1.04&#8211;1.09</td><td>&lt;0.001</td><td>n.a.</td><td>n.a.</td><td>n.a.</td></tr><tr><td><italic>P. falciparum</italic> <sub>mol</sub>FOB<sup>&#8225;</sup></td><td>n.a.</td><td>n.a.</td><td>n.a.</td><td>1.15</td><td>1.11&#8211;1.21</td><td>&lt;0.001</td></tr><tr><td>Age</td><td>0.62</td><td>0.46&#8211;0.84</td><td>0.002</td><td>0.98</td><td>0.85&#8211;1.13</td><td>0. 799</td></tr><tr><td>LLIN use&#160;at enrolment</td><td>0.84</td><td>0.24&#8211;2.88</td><td>0.778</td><td>0.44</td><td>0.22&#8211;0.87</td><td>0.018</td></tr><tr><td>Hb at enrolment (g/dl)</td><td>0.95</td><td>0.74&#8211;0.67</td><td>0.668</td><td>0.85</td><td>0.71&#8211;1.01</td><td>0.070</td></tr><tr><td><italic>Village</italic></td><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;Albinama (ref)</td><td>1</td><td/><td/><td>1</td><td/><td/></tr><tr><td>&#8195;Amahup</td><td>0.89</td><td>0.23&#8211;3.46</td><td>0.871</td><td>0.65</td><td>0.20&#8211;2.08</td><td>0.465</td></tr><tr><td>&#8195;Balif</td><td>1.48</td><td>0.45&#8211;4.86</td><td>0.518</td><td>1.26</td><td>0.50&#8211;3.14</td><td>0.626</td></tr><tr><td>&#8195;Balanga</td><td>0.85</td><td>0.21&#8211;3.53</td><td>0.827</td><td>1.39</td><td>0.59&#8211;3.30</td><td>0.455</td></tr><tr><td>&#8195;Bolumita</td><td>0.99</td><td>0.24&#8211;4.03</td><td>0.987</td><td>1.32</td><td>0.58&#8211;3.03</td><td>0.508</td></tr><tr><td>&#8195;Numangu</td><td>1.00</td><td>0.23&#8211;4.31</td><td>0.997</td><td>4.29</td><td>2.06&#8211;8.97</td><td>&lt;0.001</td></tr><tr><td colspan="7"><italic>Infection status at enrolment (by qPCR)</italic></td></tr><tr><td>&#8195;Uninfected (ref)</td><td>1</td><td/><td/><td>1</td><td/><td/></tr><tr><td>&#8195;<italic>P. vivax</italic></td><td>0.77</td><td>0.29&#8211;2.07</td><td>0.608</td><td>1.64</td><td>0.91&#8211;2.95</td><td>0.101</td></tr><tr><td>&#8195;<italic>P. falciparum</italic></td><td>1.74</td><td>0.59&#8211;5.11</td><td>0.316</td><td>0.97</td><td>0.34&#8211;2.77</td><td>0.954</td></tr><tr><td>&#8195;Mixed <italic>P.f. or P.v</italic>.</td><td>1.59</td><td>0.56&#8211;4.50</td><td>0.381</td><td>1.24</td><td>0.57&#8211;2.68</td><td>0.582</td></tr></tbody></table><table-wrap-foot><fn><p>* AHRs were modeled using multiple failure Cox proportional hazard regression.</p><p>&#8224; n.a., not applicable</p></fn><fn><p>&#8225; Average <sub>mol</sub>FOB until the time of failure (time-varying covariate).</p><p>PQ: Primaquine; LLIN: long-lasting insecticide-treated net; Hb: haemoglobin.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24001-v2.xml">elife-24001-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.24001.022</object-id><label>Author response table 1.</label><caption><p>Protein concentration estimation.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.24001.022">http://dx.doi.org/10.7554/eLife.24001.022</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><bold>Protein Name</bold></th><th rowspan="2"><bold>Concentration of protein taken (mg/ml) measured using A<sub>280</sub></bold></th><th colspan="2"><bold>Protein concentration estimation (mg/ml) using</bold></th></tr><tr><th><bold>Bicinchoninic acid Assay</bold></th><th><bold>Bradford Assay</bold></th></tr></thead><tbody><tr><td rowspan="2"><bold><italic>E. coli</italic> AlaRS</bold></td><td><bold>4</bold></td><td><bold>3.82</bold></td><td><bold>5.22</bold></td></tr><tr><td><bold>8</bold></td><td><bold>7.45</bold></td><td><bold>9.02</bold></td></tr><tr><td rowspan="2"><bold><italic>E. coli</italic> DTD</bold></td><td><bold>4</bold></td><td><bold>4.36</bold></td><td><bold>4.83</bold></td></tr><tr><td><bold>8</bold></td><td><bold>8.08</bold></td><td><bold>7.93</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24192-v2.xml">elife-24192-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.24192.007</object-id><label>Table 1.</label><caption><title>Shisa7 is a native interactor of hippocampal AMPARs.</title><p>Shisa7 complexes were immunoprecipitated from the hippocampi of <italic>Shisa7</italic> WT and KO animals (DDM-extracted crude synaptic membranes; n&#160;=&#160;3 IPs per genotype) and subjected to mass spectrometric analysis. The following protein categories are highlighted: Shisa7 (dark blue), AMPAR subunits (light blue), established non-PDZ AMPAR-interacting proteins (<xref ref-type="bibr" rid="bib56">Schwenk et al., 2012</xref>) (dark yellow), PDZ domain-containing interactors (light yellow). Percent coverage, percent of the database protein sequence covered by all matching peptides. Green background highlights values of statistical significance upon performing a Student&#8217;s t-test with permutation-based FDR analysis (S0&#160;=&#160;1, FDR&#160;=&#160;0.01 (dark green) or FDR&#160;=&#160;0.05 (light green)). Note the identification of a peptide specific for exon 4 containing Shisa7 (NLYNTMKPSNLDwNLHYNVNSPK; diamond indicates exon3-exon4 boundary). The full list of identified protein groups is reported in <xref ref-type="supplementary-material" rid="table1sdata1">Table 1&#8212;source data 1</xref>, and the statistical analysis is provided in <xref ref-type="supplementary-material" rid="table1sdata2">Table 1&#8212;source data 2</xref>, which includes a volcano-plot of differentially IP-ed proteins.</p><p><supplementary-material id="table1sdata1"><object-id pub-id-type="doi">10.7554/eLife.24192.008</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Maxquant analysis of hippocampal Shisa7 immunoprecipitation experiments, as detailed in the Materials and methods section.</title><p>The Maxquant &#8216;proteinGroups.txt&#8217; output file was supplemented with &#8216;LFQ intensity _ Average KO&#8217;, &#8216;LFQ intensity _ Average WT&#8217;, and the &#8216;KO/WT&#8217; and &#8216;WT/KO&#8217; ratios thereof.</p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-24192-table1-data1-v2.xlsx"/></supplementary-material></p><p><supplementary-material id="table1sdata2"><object-id pub-id-type="doi">10.7554/eLife.24192.009</object-id><label>Table 1&#8212;source data 2.</label><caption><title>Statistical analysis of hippocampal Shisa7 immunoprecipitation data, as detailed in the Materials and methods section.</title><p>Tab 1:&#160;In summary, the Maxquant &#8216;proteinGroups.txt&#8217; output file was imported into Perseus, and processed in the following manner: (1) Removal of &#8216;Reverse&#8217;, &#8216;Potential contaminant&#8217;, and &#8216;Only identified by site&#8217; protein&#160;groups; (2) Log(2) transformation of all LFQ intensity values; (3) Removal of protein&#160;groups with less than three valid &#8216;Log(2) LFQ intensity&#8217; values in either the WT or KO groups; (4) Imputation of missing values (8.6% of the population) from a normal distribution (width 0.3, down shift 1.8, whole matrix); (5) Performing a Student's t-test followed by permutation-based FDR analysis (S0&#160;=&#160;1, FDR&#160;=&#160;0.01, 2500 permutations). Tab 2: Visualization of the data by means of Histogram and Vulcanoplot is presented in the additional sheets. Tab 3: Distribution of LFQ intensities after replacing missing values from a normal distribution. The imputed value distribution is depicted in red.&#160;</p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-24192-table1-data2-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene name</th><th rowspan="2">Uniprot recommendedprotein name(s)</th><th rowspan="2">Uniprot ID</th><th rowspan="2">PDZ-domains</th><th colspan="6">Number of unique peptides</th><th colspan="6">LFQ intensity</th><th rowspan="2">Average KO <break/>LFQ intensity</th><th rowspan="2">Average WT <break/>LFQ intensity</th><th rowspan="2">Average KO/WT LFQ intensity</th><th rowspan="2">Average WT/KO LFQ intensity</th><th rowspan="2">T-test significant <break/>FDR0.01 = ++ <break/>DR0.05 = +</th><th rowspan="2">T-test q-value</th><th rowspan="2">Percent cover-age</th></tr><tr><th>KO1</th><th>KO2</th><th>KO3</th><th>WT1</th><th>WT2</th><th>WT3</th><th>KO1</th><th>KO2</th><th>KO3</th><th>WT1</th><th>WT2</th><th>WT3</th></tr></thead><tbody><tr><td rowspan="2">Shisa7</td><td>Protein Shisa-7</td><td>Q8C3Q5</td><td>0</td><td>1</td><td>1</td><td>2</td><td>17</td><td>18</td><td>16</td><td>865</td><td>873</td><td>1750</td><td>319820</td><td>305190</td><td>346730</td><td>1163</td><td>323913</td><td>0.4%</td><td>278.58</td><td style="author-callout-style-b2">++</td><td style="author-callout-style-b2">0.0000</td><td>39.2</td></tr><tr><td>Protein Shisa-7:Exon4-specificpeptideNLYNTMKPSNLDNLHYNVNSPK</td><td>-</td><td>-</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>Gria1</td><td>Glutamatereceptor 1</td><td>P23818</td><td>0</td><td>0</td><td>0</td><td>0</td><td>10</td><td>12</td><td>12</td><td>5921</td><td>0</td><td>3217</td><td>17828</td><td>52304</td><td>54212</td><td>4569</td><td>41448</td><td>11.0%</td><td>9.07</td><td style="author-callout-style-b2">++</td><td style="author-callout-style-b2">0.0000</td><td>22.8</td></tr><tr><td>Gria2</td><td>Glutamatereceptor 2</td><td>P23819</td><td>0</td><td>2</td><td>1</td><td>0</td><td>17</td><td>17</td><td>17</td><td>2743</td><td>0</td><td>0</td><td>104860</td><td>101430</td><td>93097</td><td>2743</td><td>99796</td><td>2.7%</td><td>36.39</td><td style="author-callout-style-b2">++</td><td style="author-callout-style-b2">0.0000</td><td>36.4</td></tr><tr><td>Gria3</td><td>Glutamatereceptor 3</td><td>Q9Z2W9</td><td>0</td><td>0</td><td>0</td><td>0</td><td>4</td><td>4</td><td>5</td><td>0</td><td>0</td><td>0</td><td>6529</td><td>7068</td><td>10102</td><td>0</td><td>7900</td><td>0.0%</td><td>NaN</td><td style="author-callout-style-b2">++</td><td style="author-callout-style-b2">0.0000</td><td>17.7</td></tr><tr><td style="author-callout-style-b3">Cacng8</td><td style="author-callout-style-b3">Voltage-dependentcalciumchannelgamma-8subunit;TARPgamma-8</td><td>Q8VHW2</td><td>0</td><td>2</td><td>2</td><td>1</td><td>2</td><td>1</td><td>5</td><td>4023</td><td>0</td><td>0</td><td>0</td><td>0</td><td>3962</td><td>4023</td><td>3962</td><td>101.5%</td><td>0.98</td><td>-</td><td>N/A</td><td>30.5</td></tr><tr><td style="author-callout-style-b3">Olfm1</td><td style="author-callout-style-b3">Noelin</td><td>O88998</td><td>0</td><td>1</td><td>1</td><td>0</td><td>0</td><td>2</td><td>2</td><td>0</td><td>1076</td><td>0</td><td>0</td><td>1004</td><td>779</td><td>1076</td><td>892</td><td>120.6%</td><td>0.83</td><td>-</td><td>N/A</td><td>7.0</td></tr><tr><td style="author-callout-style-b3">Prrt1</td><td style="author-callout-style-b3">Proline-richtransmem-braneprotein 1;SynDIG4</td><td>O35449</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>969</td><td>0</td><td>0</td><td>0</td><td>969</td><td>0.0%</td><td>NaN</td><td>-</td><td>N/A</td><td>7.2</td></tr><tr><td style="author-callout-style-b3">Prrt2</td><td style="author-callout-style-b3">Proline-richtransmem-braneprotein 2</td><td>E9PUL5</td><td>0</td><td>0</td><td>1</td><td>0</td><td>1</td><td>1</td><td>1</td><td>0</td><td>333</td><td>0</td><td>318</td><td>0</td><td>202</td><td>333</td><td>260</td><td>127.9%</td><td>0.78</td><td>-</td><td>N/A</td><td>3.8</td></tr><tr><td style="author-callout-style-b3">Rap2b</td><td style="author-callout-style-b3">Ras-relatedproteinRap-2b</td><td>P61226</td><td>0</td><td>0</td><td>1</td><td>1</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2588</td><td>0</td><td>0</td><td>0</td><td>2588</td><td>0</td><td>NaN</td><td>0.00</td><td>-</td><td>N/A</td><td>12.0</td></tr><tr><td style="author-callout-style-b3">Shisa6</td><td style="author-callout-style-b3">Proteinshisa-6homolog</td><td>Q3UH99</td><td>0</td><td>1</td><td>0</td><td>0</td><td>3</td><td>3</td><td>1</td><td>0</td><td>0</td><td>0</td><td>2666</td><td>3580</td><td>2136</td><td>0</td><td>2794</td><td>0.0%</td><td>NaN</td><td>+</td><td>0.0306</td><td>9.9</td></tr><tr><td style="author-callout-style-b4">Dlg1</td><td style="author-callout-style-b4">Disks largehomolog 1;SAP97</td><td>Q811D0</td><td>3</td><td>0</td><td>1</td><td>1</td><td>3</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2126</td><td>0</td><td>0</td><td>0</td><td>2126</td><td>0.0%</td><td>NaN</td><td>-</td><td>N/A</td><td>4.4</td></tr><tr><td style="author-callout-style-b4">Dlg3</td><td style="author-callout-style-b4">Disks largehomolog 3;SAP102</td><td>P70175</td><td>3</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2</td><td>2</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2441</td><td>0</td><td>0</td><td>2441</td><td>0.0%</td><td>NaN</td><td>-</td><td>N/A</td><td>4.8</td></tr><tr><td style="author-callout-style-b4">Dlg4</td><td style="author-callout-style-b4">Disks largehomolog 4;PSD95</td><td>Q62108</td><td>3</td><td>0</td><td>0</td><td>0</td><td>9</td><td>8</td><td>7</td><td>0</td><td>0</td><td>0</td><td>11835</td><td>13410</td><td>9284</td><td>0</td><td>11510</td><td>0.0%</td><td>NaN</td><td style="author-callout-style-b2">++</td><td style="author-callout-style-b2">0.0000</td><td>21.7</td></tr><tr><td style="author-callout-style-b4">Magi2</td><td style="author-callout-style-b4">Membrane-associatedguanylatekinase,WW and PDZdomain-containingprotein 2</td><td>Q9WVQ1</td><td>6</td><td>0</td><td>0</td><td>0</td><td>1</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1558</td><td>0</td><td>0</td><td>1558</td><td>0.0%</td><td>NaN</td><td>-</td><td>N/A</td><td>1.6</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24192-v4.xml">elife-24192-v4.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.24192.007</object-id><label>Table 1.</label><caption><title>Shisa7 is a native interactor of hippocampal AMPARs.</title><p>Shisa7 complexes were immunoprecipitated from the hippocampi of <italic>Shisa7</italic> WT and KO animals (DDM-extracted crude synaptic membranes; n&#160;=&#160;3 IPs per genotype) and subjected to mass spectrometric analysis. The following protein categories were indicated: Shisa7, AMPAR subunits, established non-PDZ AMPAR-interacting proteins (<xref ref-type="bibr" rid="bib56">Schwenk et al., 2012</xref>), PDZ domain-containing interactors. Percent coverage, percent of the database protein sequence covered by all matching peptides. Values of statistical significance upon performing a Student&#8217;s t-test with permutation-based FDR analysis (S0 = 1, FDR &lt; 0.05 (bold)) are indicated.&#160;Note the identification of a peptide specific for exon 4 containing Shisa7 (NLYNTMKPSNLDwNLHYNVNSPK; diamond indicates exon3-exon4 boundary). The full list of identified protein groups is reported in <xref ref-type="supplementary-material" rid="table1sdata1">Table 1&#8212;source data 1</xref>, and the statistical analysis is provided in <xref ref-type="supplementary-material" rid="table1sdata2">Table 1&#8212;source data 2</xref>, which includes a volcano-plot of differentially IP-ed proteins.</p><p><supplementary-material id="table1sdata1"><object-id pub-id-type="doi">10.7554/eLife.24192.008</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Maxquant analysis of hippocampal Shisa7 immunoprecipitation experiments, as detailed in the Materials and methods section.</title><p>The Maxquant &#8216;proteinGroups.txt&#8217; output file was supplemented with &#8216;LFQ intensity _ Average KO&#8217;, &#8216;LFQ intensity _ Average WT&#8217;, and the &#8216;KO/WT&#8217; and &#8216;WT/KO&#8217; ratios thereof.</p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-24192-table1-data1-v4.xlsx"/></supplementary-material></p><p><supplementary-material id="table1sdata2"><object-id pub-id-type="doi">10.7554/eLife.24192.009</object-id><label>Table 1&#8212;source data 2.</label><caption><title>Statistical analysis of hippocampal Shisa7 immunoprecipitation data, as detailed in the Materials and methods section.</title><p>Tab 1:&#160;In summary, the Maxquant &#8216;proteinGroups.txt&#8217; output file was imported into Perseus, and processed in the following manner: (1) Removal of &#8216;Reverse&#8217;, &#8216;Potential contaminant&#8217;, and &#8216;Only identified by site&#8217; protein&#160;groups; (2) Log(2) transformation of all LFQ intensity values; (3) Removal of protein&#160;groups with less than three valid &#8216;Log(2) LFQ intensity&#8217; values in either the WT or KO groups; (4) Imputation of missing values (8.6% of the population) from a normal distribution (width 0.3, down shift 1.8, whole matrix); (5) Performing a Student's t-test followed by permutation-based FDR analysis (S0&#160;=&#160;1, FDR&#160;=&#160;0.01, 2500 permutations). Tab 2: Visualization of the data by means of Histogram and Vulcanoplot is presented in the additional sheets. Tab 3: Distribution of LFQ intensities after replacing missing values from a normal distribution. The imputed value distribution is depicted in red.&#160;</p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-24192-table1-data2-v4.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene name</th><th rowspan="2">Uniprot recommendedprotein name(s)</th><th rowspan="2">Uniprot ID</th><th rowspan="2">PDZ-domains</th><th colspan="6">Number of unique peptides</th><th colspan="6">LFQ intensity</th><th rowspan="2">Average KO <break/>LFQ intensity</th><th rowspan="2">Average WT <break/>LFQ intensity</th><th rowspan="2">Average KO/WT LFQ intensity</th><th rowspan="2">Average WT/KO LFQ intensity</th><th rowspan="2">T-test significant <break/>FDR0.01 = ++ <break/>DR0.05 = +</th><th rowspan="2">T-test q-value</th><th rowspan="2">Percent cover-age</th></tr><tr><th>KO1</th><th>KO2</th><th>KO3</th><th>WT1</th><th>WT2</th><th>WT3</th><th>KO1</th><th>KO2</th><th>KO3</th><th>WT1</th><th>WT2</th><th>WT3</th></tr></thead><tbody><tr><td rowspan="2"><bold>Shisa7</bold></td><td><bold>Protein Shisa-7</bold></td><td>Q8C3Q5</td><td>0</td><td>1</td><td>1</td><td>2</td><td>17</td><td>18</td><td>16</td><td>865</td><td>873</td><td>1750</td><td>319820</td><td>305190</td><td>346730</td><td>1163</td><td>323913</td><td>0.4%</td><td>278.58</td><td><bold>++</bold></td><td><bold>0.0000</bold></td><td>39.2</td></tr><tr><td><bold>Protein Shisa-7:Exon4-specificpeptideNLYNTMKPSNLDNLHYNVNSPK</bold></td><td>-</td><td>-</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td><bold>Gria1</bold></td><td><bold>Glutamatereceptor 1</bold></td><td>P23818</td><td>0</td><td>0</td><td>0</td><td>0</td><td>10</td><td>12</td><td>12</td><td>5921</td><td>0</td><td>3217</td><td>17828</td><td>52304</td><td>54212</td><td>4569</td><td>41448</td><td>11.0%</td><td>9.07</td><td><bold>++</bold></td><td><bold>0.0000</bold></td><td>22.8</td></tr><tr><td><bold>Gria2</bold></td><td><bold>Glutamatereceptor 2</bold></td><td>P23819</td><td>0</td><td>2</td><td>1</td><td>0</td><td>17</td><td>17</td><td>17</td><td>2743</td><td>0</td><td>0</td><td>104860</td><td>101430</td><td>93097</td><td>2743</td><td>99796</td><td>2.7%</td><td>36.39</td><td><bold>++</bold></td><td><bold>0.0000</bold></td><td>36.4</td></tr><tr><td><bold>Gria3</bold></td><td><bold>Glutamatereceptor 3</bold></td><td>Q9Z2W9</td><td>0</td><td>0</td><td>0</td><td>0</td><td>4</td><td>4</td><td>5</td><td>0</td><td>0</td><td>0</td><td>6529</td><td>7068</td><td>10102</td><td>0</td><td>7900</td><td>0.0%</td><td>NaN</td><td><bold>++</bold></td><td><bold>0.0000</bold></td><td>17.7</td></tr><tr><td><bold>Cacng8</bold></td><td><bold>Voltage-dependentcalciumchannelgamma-8subunit;TARPgamma-8</bold></td><td>Q8VHW2</td><td>0</td><td>2</td><td>2</td><td>1</td><td>2</td><td>1</td><td>5</td><td>4023</td><td>0</td><td>0</td><td>0</td><td>0</td><td>3962</td><td>4023</td><td>3962</td><td>101.5%</td><td>0.98</td><td>-</td><td>N/A</td><td>30.5</td></tr><tr><td><bold>Olfm1</bold></td><td><bold>Noelin</bold></td><td>O88998</td><td>0</td><td>1</td><td>1</td><td>0</td><td>0</td><td>2</td><td>2</td><td>0</td><td>1076</td><td>0</td><td>0</td><td>1004</td><td>779</td><td>1076</td><td>892</td><td>120.6%</td><td>0.83</td><td>-</td><td>N/A</td><td>7.0</td></tr><tr><td><bold>Prrt1</bold></td><td><bold>Proline-richtransmem-braneprotein 1;SynDIG4</bold></td><td>O35449</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>969</td><td>0</td><td>0</td><td>0</td><td>969</td><td>0.0%</td><td>NaN</td><td>-</td><td>N/A</td><td>7.2</td></tr><tr><td><bold>Prrt2</bold></td><td><bold>Proline-richtransmem-braneprotein 2</bold></td><td>E9PUL5</td><td>0</td><td>0</td><td>1</td><td>0</td><td>1</td><td>1</td><td>1</td><td>0</td><td>333</td><td>0</td><td>318</td><td>0</td><td>202</td><td>333</td><td>260</td><td>127.9%</td><td>0.78</td><td>-</td><td>N/A</td><td>3.8</td></tr><tr><td><bold>Rap2b</bold></td><td><bold>Ras-relatedproteinRap-2b</bold></td><td>P61226</td><td>0</td><td>0</td><td>1</td><td>1</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2588</td><td>0</td><td>0</td><td>0</td><td>2588</td><td>0</td><td>NaN</td><td>0.00</td><td>-</td><td>N/A</td><td>12.0</td></tr><tr><td><bold>Shisa6</bold></td><td><bold>Proteinshisa-6homolog</bold></td><td>Q3UH99</td><td>0</td><td>1</td><td>0</td><td>0</td><td>3</td><td>3</td><td>1</td><td>0</td><td>0</td><td>0</td><td>2666</td><td>3580</td><td>2136</td><td>0</td><td>2794</td><td>0.0%</td><td>NaN</td><td><bold>+</bold></td><td><bold>0.0306</bold></td><td>9.9</td></tr><tr><td><bold>Dlg1</bold></td><td><bold>Disks largehomolog 1;SAP97</bold></td><td>Q811D0</td><td>3</td><td>0</td><td>1</td><td>1</td><td>3</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2126</td><td>0</td><td>0</td><td>0</td><td>2126</td><td>0.0%</td><td>NaN</td><td>-</td><td>N/A</td><td>4.4</td></tr><tr><td><bold>Dlg3</bold></td><td><bold>Disks largehomolog 3;SAP102</bold></td><td>P70175</td><td>3</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2</td><td>2</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2441</td><td>0</td><td>0</td><td>2441</td><td>0.0%</td><td>NaN</td><td>-</td><td>N/A</td><td>4.8</td></tr><tr><td><bold>Dlg4</bold></td><td><bold>Disks largehomolog 4;PSD95</bold></td><td>Q62108</td><td>3</td><td>0</td><td>0</td><td>0</td><td>9</td><td>8</td><td>7</td><td>0</td><td>0</td><td>0</td><td>11835</td><td>13410</td><td>9284</td><td>0</td><td>11510</td><td>0.0%</td><td>NaN</td><td><bold>++</bold></td><td><bold>0.0000</bold></td><td>21.7</td></tr><tr><td><bold>Magi2</bold></td><td><bold>Membrane-associatedguanylatekinase,WW and PDZdomain-containingprotein 2</bold></td><td>Q9WVQ1</td><td>6</td><td>0</td><td>0</td><td>0</td><td>1</td><td>1</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1558</td><td>0</td><td>0</td><td>1558</td><td>0.0%</td><td>NaN</td><td>-</td><td>N/A</td><td>1.6</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24414-v2.xml">elife-24414-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.24414.007</object-id><label>Table 2.</label><caption><p>Summary of mutations detected following experimental evolution of <italic>V. fischeri</italic> using Illumina genome resequencing and targeted Sanger sequencing. For culture-evolved populations of <italic>V. fischeri</italic> MJ11, five isolates from each evolved population were combined to generate five metagenomes. For squid-evolved populations of MJ11, EM17, WH1 and H905, individual isolates were sequenced from lineages that ultimately survived 15 host passages. Isolates saved from early evolutionary time-points (host passage 2) are shown along with isolate genomes from the endpoint (host passage 15). Mean read depth and genome coverage for isolates analyzed with WGS are also provided.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.24414.007">http://dx.doi.org/10.7554/eLife.24414.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Ancestor</th><th rowspan="3">Evolved Passage (EP)</th><th rowspan="3">Population</th><th rowspan="3">Isolate<sup>&#8224;</sup></th><th colspan="4">Detected mutations<sup>&#8225;</sup></th><th colspan="4">Illumina sequencing statistics</th></tr><tr><th colspan="2"><italic>binK</italic> (VFMJ11_A0397)</th><th rowspan="2"><italic>tadC1</italic> <break/>(MJ11_0520); <break/>mutation <break/>(reads)</th><th rowspan="2">All other mutations <break/>detectected by WGS <break/>gene (locus); <break/>mutation (reads)</th><th rowspan="2">Reads</th><th rowspan="2">% Mapped to ancestor</th><th colspan="2">Coverage</th></tr><tr><th>allele/mutation</th><th>Method (reads)<sup>&#167;</sup></th><th>ChI</th><th>ChII</th></tr></thead><tbody><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">1</td><td valign="bottom">1</td><td valign="bottom"><italic>binK3</italic>/S311L</td><td valign="bottom">WGS (35)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3753352</td><td valign="bottom">99.5</td><td valign="bottom">135.2</td><td valign="bottom">118</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">1</td><td valign="bottom">3</td><td valign="bottom"><italic>binK3</italic>/S311L</td><td valign="bottom">WGS (32)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3717088</td><td valign="bottom">99.6</td><td valign="bottom">134.2</td><td valign="bottom">113.5</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">1</td><td valign="bottom">4</td><td valign="bottom"><italic>binK3</italic>/S311L</td><td valign="bottom">WGS (17)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">1716144</td><td valign="bottom">99.5</td><td valign="bottom">46.8</td><td valign="bottom">42.5</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">3</td><td valign="bottom">3</td><td valign="bottom"><italic>binK4</italic>/N292T</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">3</td><td valign="bottom">4</td><td valign="bottom"><italic>binK4</italic>/N292T</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">3</td><td valign="bottom">5</td><td valign="bottom"><italic>binK4</italic>/N292T</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">3</td><td valign="bottom">6</td><td valign="bottom"><italic>binK4</italic>/N292T</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">3</td><td valign="bottom">7</td><td valign="bottom"><italic>binK4</italic>/N292T</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">3</td><td valign="bottom">8</td><td valign="bottom"><italic>binK4</italic>/N292T</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">3</td><td valign="bottom">1</td><td valign="bottom"><italic>binK4</italic>/N292T</td><td valign="bottom">WGS (42)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3031149</td><td valign="bottom">98.9</td><td valign="bottom">104.3</td><td valign="bottom">93.5</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">3</td><td valign="bottom">3</td><td valign="bottom"><italic>binK4</italic>/N292T</td><td valign="bottom">WGS (63)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3777714</td><td valign="bottom">99.4</td><td valign="bottom">114.6</td><td valign="bottom">105.2</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">3</td><td valign="bottom">4</td><td valign="bottom"><italic>binK4</italic>/N292T</td><td valign="bottom">WGS (42)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3420212</td><td valign="bottom">99.5</td><td valign="bottom">106.4</td><td valign="bottom">97.1</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">3</td><td valign="bottom">7</td><td valign="bottom"><italic>binK4</italic>/N292T</td><td valign="bottom">WGS (41)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3304891</td><td valign="bottom">99.5</td><td valign="bottom">90.3</td><td valign="bottom">82.5</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">3</td><td valign="bottom">8</td><td valign="bottom"><italic>binK4</italic>/N292T</td><td valign="bottom">WGS (63)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2948743</td><td valign="bottom">99.6</td><td valign="bottom">85.5</td><td valign="bottom">81.2</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">1</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">WGS (62)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2511256</td><td valign="bottom">99</td><td valign="bottom">84</td><td valign="bottom">78</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">3</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">4</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">5</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">6</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">7</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">8</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">9</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">10</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">11</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">12</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">13</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">14</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">15</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">16</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">4</td><td valign="bottom">1</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">WGS (131)</td><td valign="bottom">G198V (85)</td><td valign="bottom">&#8211;</td><td valign="bottom">4126149</td><td valign="bottom">99.4</td><td valign="bottom">117.8</td><td valign="bottom">106.1</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">4</td><td valign="bottom">6</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">WGS (61)</td><td valign="bottom">G198V (55)</td><td valign="bottom">&#8211;</td><td valign="bottom">2266821</td><td valign="bottom">99.2</td><td valign="bottom">60.8</td><td valign="bottom">52.5</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">4</td><td valign="bottom">7</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">WGS (89)</td><td valign="bottom">G198V (93)</td><td valign="bottom">&#8211;</td><td valign="bottom">3074437</td><td valign="bottom">99.6</td><td valign="bottom">92</td><td valign="bottom">83.6</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">4</td><td valign="bottom">8</td><td valign="bottom"><italic>binK1</italic>/R537C</td><td valign="bottom">WGS (47)</td><td valign="bottom">G198V (96)</td><td valign="bottom">&#8211;</td><td valign="bottom">2902977</td><td valign="bottom">99.5</td><td valign="bottom">84</td><td valign="bottom">77.5</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">5</td><td valign="bottom">2</td><td valign="bottom"><italic>binK3</italic>/S311L</td><td valign="bottom">WGS (26)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3771048</td><td valign="bottom">99.6</td><td valign="bottom">132.4</td><td valign="bottom">123.7</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">5</td><td valign="bottom">3</td><td valign="bottom"><italic>binK3</italic>/S311L</td><td valign="bottom">WGS (46)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2595518</td><td valign="bottom">99.6</td><td valign="bottom">84.2</td><td valign="bottom">83.7</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">5</td><td valign="bottom">4</td><td valign="bottom"><italic>binK3</italic>/S311L</td><td valign="bottom">WGS (20)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">1785713</td><td valign="bottom">99.5</td><td valign="bottom">60.6</td><td valign="bottom">57.2</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">5</td><td valign="bottom">5</td><td valign="bottom"><italic>binK3</italic>/S311L</td><td valign="bottom">WGS (62)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3641346</td><td valign="bottom">99.6</td><td valign="bottom">117.4</td><td valign="bottom">113.1</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">5</td><td valign="bottom">6</td><td valign="bottom"><italic>binK3</italic>/S311L</td><td valign="bottom">WGS (81)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">4128751</td><td valign="bottom">99.6</td><td valign="bottom">141.1</td><td valign="bottom">134.8</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">5</td><td valign="bottom">2</td><td valign="bottom"><italic>binK4</italic>/N292T</td><td valign="bottom">WGS (89)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">4430823</td><td valign="bottom">99.1</td><td valign="bottom">152.3</td><td valign="bottom">138.4</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">5</td><td valign="bottom">3</td><td valign="bottom"><italic>binK3</italic>/S311L</td><td valign="bottom">WGS (10)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3248580</td><td valign="bottom">99.3</td><td valign="bottom">88</td><td valign="bottom">81.1</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">5</td><td valign="bottom">4</td><td valign="bottom"><italic>binK3</italic>/S311L</td><td valign="bottom">WGS (59)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3609382</td><td valign="bottom">99.5</td><td valign="bottom">106.8</td><td valign="bottom">97.1</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">5</td><td valign="bottom">5</td><td valign="bottom"><italic>binK3</italic>/S311L</td><td valign="bottom">WGS (28)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2915570</td><td valign="bottom">99.5</td><td valign="bottom">87.4</td><td valign="bottom">82.6</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">6</td><td valign="bottom">1</td><td valign="bottom"><italic>binK2</italic>/K482N</td><td valign="bottom">WGS (104)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">4748569</td><td valign="bottom">99.1</td><td valign="bottom">164.6</td><td valign="bottom">147</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">2</td><td valign="bottom">6</td><td valign="bottom">2</td><td valign="bottom"><italic>binK2</italic>/K482N</td><td valign="bottom">PCR/SS</td><td valign="bottom">n.d.</td><td valign="bottom">n.d.</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">6</td><td valign="bottom">1</td><td valign="bottom"><italic>binK2</italic>/K482N</td><td valign="bottom">WGS (75)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2764910</td><td valign="bottom">99.4</td><td valign="bottom">83.2</td><td valign="bottom">75.5</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">6</td><td valign="bottom">2</td><td valign="bottom"><italic>binK2</italic>/K482N</td><td valign="bottom">WGS (63)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3240968</td><td valign="bottom">99.2</td><td valign="bottom">88</td><td valign="bottom">72.6</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">6</td><td valign="bottom">3</td><td valign="bottom"><italic>binK2</italic>/K482N</td><td valign="bottom">WGS (93)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3814367</td><td valign="bottom">99.5</td><td valign="bottom">108.1</td><td valign="bottom">101.7</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">6</td><td valign="bottom">4</td><td valign="bottom"><italic>binK2</italic>/K482N</td><td valign="bottom">WGS (108)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3714638</td><td valign="bottom">99.5</td><td valign="bottom">121.4</td><td valign="bottom">85.7</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">6</td><td valign="bottom">5</td><td valign="bottom"><italic>binK2</italic>/K482N</td><td valign="bottom">WGS (90)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3006362</td><td valign="bottom">99.4</td><td valign="bottom">85.5</td><td valign="bottom">72</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">Culture1</td><td valign="bottom">mg</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">10319291</td><td valign="bottom">98</td><td valign="bottom">272.8</td><td valign="bottom">237.8</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">Culture3</td><td valign="bottom">mg</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">7496847</td><td valign="bottom">98.2</td><td valign="bottom">196.7</td><td valign="bottom">195</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">Culture4</td><td valign="bottom">mg</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom"><italic>fliA</italic> (VF_1834); G80D (63)</td><td valign="bottom">2894160</td><td valign="bottom">98.3</td><td valign="bottom">76.6</td><td valign="bottom">67.4</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">Culture5</td><td valign="bottom">mg</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom"><italic>fliP</italic> (VF_1842); &#8710;1 @ 476/870nt (110)</td><td valign="bottom">5571439</td><td valign="bottom">97.9</td><td valign="bottom">148.5</td><td valign="bottom">132.1</td></tr><tr><td valign="bottom">MJ11</td><td valign="bottom">15</td><td valign="bottom">Culture2</td><td valign="bottom">mg</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">5411032</td><td valign="bottom">98</td><td valign="bottom">144.2</td><td valign="bottom">129.4</td></tr><tr><td valign="bottom">WH1</td><td valign="bottom">15</td><td valign="bottom">4</td><td valign="bottom">1</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">7273244</td><td valign="bottom">98.6</td><td valign="bottom">257.8</td><td valign="bottom">251.1</td></tr><tr><td valign="bottom">WH1</td><td valign="bottom">15</td><td valign="bottom">4</td><td valign="bottom">2</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2144381</td><td valign="bottom">99.6</td><td valign="bottom">61.4</td><td valign="bottom">65.1</td></tr><tr><td valign="bottom">WH1</td><td valign="bottom">15</td><td valign="bottom">4</td><td valign="bottom">3</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2260232</td><td valign="bottom">99.6</td><td valign="bottom">62.1</td><td valign="bottom">66.6</td></tr><tr><td valign="bottom">WH1</td><td valign="bottom">15</td><td valign="bottom">4</td><td valign="bottom">4</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2341428</td><td valign="bottom">99.7</td><td valign="bottom">61.6</td><td valign="bottom">65</td></tr><tr><td valign="bottom">WH1</td><td valign="bottom">15</td><td valign="bottom">5</td><td valign="bottom">1</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">NADH oxidase (VF_A0027); A402T (62)</td><td valign="bottom">1732106</td><td valign="bottom">99.5</td><td valign="bottom">60.8</td><td valign="bottom">64.7</td></tr><tr><td valign="bottom">WH1</td><td valign="bottom">15</td><td valign="bottom">5</td><td valign="bottom">2</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">NADH oxidase (VF_A0027); A402T (61)</td><td valign="bottom">1737095</td><td valign="bottom">99.4</td><td valign="bottom">61.9</td><td valign="bottom">64.9</td></tr><tr><td valign="bottom">WH1</td><td valign="bottom">15</td><td valign="bottom">5</td><td valign="bottom">3</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">NADH oxidase (VF_A0027); A402T (80)</td><td valign="bottom">2194847</td><td valign="bottom">96</td><td valign="bottom">60.8</td><td valign="bottom">63.4</td></tr><tr><td valign="bottom">WH1</td><td valign="bottom">15</td><td valign="bottom">5</td><td valign="bottom">4</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2191986</td><td valign="bottom">99.8</td><td valign="bottom">61.9</td><td valign="bottom">64.9</td></tr><tr><td valign="bottom">WH1</td><td valign="bottom">15</td><td valign="bottom">6</td><td valign="bottom">1</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">9256547</td><td valign="bottom">99.3</td><td valign="bottom">212.6</td><td valign="bottom">220.3</td></tr><tr><td valign="bottom">WH1</td><td valign="bottom">15</td><td valign="bottom">6</td><td valign="bottom">2</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2131144</td><td valign="bottom">99.6</td><td valign="bottom">62</td><td valign="bottom">64.7</td></tr><tr><td valign="bottom">WH1</td><td valign="bottom">15</td><td valign="bottom">6</td><td valign="bottom">3</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">1908857</td><td valign="bottom">99.5</td><td valign="bottom">62.4</td><td valign="bottom">60.5</td></tr><tr><td valign="bottom">EM17</td><td valign="bottom">15</td><td valign="bottom">6</td><td valign="bottom">2</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2611609</td><td valign="bottom">99.6</td><td valign="bottom">93.3</td><td valign="bottom">89.3</td></tr><tr><td valign="bottom">EM17</td><td valign="bottom">15</td><td valign="bottom">7</td><td valign="bottom">1</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">6690137</td><td valign="bottom">98.6</td><td valign="bottom">225.8</td><td valign="bottom">227.1</td></tr><tr><td valign="bottom">EM17</td><td valign="bottom">15</td><td valign="bottom">7</td><td valign="bottom">4</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2977429</td><td valign="bottom">99.5</td><td valign="bottom">83.4</td><td valign="bottom">82.1</td></tr><tr><td valign="bottom">EM17</td><td valign="bottom">15</td><td valign="bottom">7</td><td valign="bottom">5</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom"><italic>icmF</italic> (VF_0992);S171N, (72)</td><td valign="bottom">2414288</td><td valign="bottom">99.5</td><td valign="bottom">71.6</td><td valign="bottom">71.5</td></tr><tr><td valign="bottom">EM17</td><td valign="bottom">15</td><td valign="bottom">8</td><td valign="bottom">1</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3177981</td><td valign="bottom">99.5</td><td valign="bottom">97.5</td><td valign="bottom">94.6</td></tr><tr><td valign="bottom">EM17</td><td valign="bottom">15</td><td valign="bottom">8</td><td valign="bottom">2</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3138175</td><td valign="bottom">99.5</td><td valign="bottom">92.4</td><td valign="bottom">92.3</td></tr><tr><td valign="bottom">EM17</td><td valign="bottom">15</td><td valign="bottom">8</td><td valign="bottom">3</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2810099</td><td valign="bottom">99.5</td><td valign="bottom">81.2</td><td valign="bottom">80</td></tr><tr><td valign="bottom">EM17</td><td valign="bottom">15</td><td valign="bottom">8</td><td valign="bottom">5</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">5230411</td><td valign="bottom">99.6</td><td valign="bottom">144.9</td><td valign="bottom">143.2</td></tr><tr><td valign="bottom">EM17</td><td valign="bottom">15</td><td valign="bottom">9</td><td valign="bottom">1</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">8022935</td><td valign="bottom">99.4</td><td valign="bottom">184.2</td><td valign="bottom">173.5</td></tr><tr><td valign="bottom">EM17</td><td valign="bottom">15</td><td valign="bottom">9</td><td valign="bottom">2</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3346216</td><td valign="bottom">99.6</td><td valign="bottom">113.7</td><td valign="bottom">106.9</td></tr><tr><td valign="bottom">EM17</td><td valign="bottom">15</td><td valign="bottom">9</td><td valign="bottom">3</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom"><italic>gdh2</italic> (VF_1284); E732D (72)</td><td valign="bottom">3484188</td><td valign="bottom">99.5</td><td valign="bottom">95.7</td><td valign="bottom">93.2</td></tr><tr><td valign="bottom">EM17</td><td valign="bottom">15</td><td valign="bottom">9</td><td valign="bottom">5</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2445758</td><td valign="bottom">99.5</td><td valign="bottom">72.8</td><td valign="bottom">72.6</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">1</td><td valign="bottom">1</td><td valign="bottom">(&#8710;37168 bp/25 genes)</td><td valign="bottom">WGS (230)</td><td valign="bottom">&#8211;</td><td valign="bottom"><italic>IlvY</italic> (VF_2529); M25I (233)</td><td valign="bottom">7645508</td><td valign="bottom">94.2</td><td valign="bottom">250.4</td><td valign="bottom">222.1</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">1</td><td valign="bottom">2</td><td valign="bottom">(&#8710;37168 bp/25 genes)</td><td valign="bottom">WGS (167)</td><td valign="bottom">&#8211;</td><td valign="bottom"><italic>IlvY</italic> (VF_2529); M25I (112)</td><td valign="bottom">3531114</td><td valign="bottom">96.8</td><td valign="bottom">117.5</td><td valign="bottom">104.4</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">1</td><td valign="bottom">3</td><td valign="bottom">(&#8710;37168 bp/25 genes)</td><td valign="bottom">WGS (175)</td><td valign="bottom">&#8211;</td><td valign="bottom"><italic>IlvY</italic> (VF_2529); M25I (97)</td><td valign="bottom">3596689</td><td valign="bottom">97</td><td valign="bottom">122.3</td><td valign="bottom">109.1</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">2</td><td valign="bottom">2</td><td valign="bottom">&#8710;16 bp@ 498/2595</td><td valign="bottom">WGS (75)</td><td valign="bottom">&#8211;</td><td valign="bottom"><italic>purR</italic> (VF_1572); N71T (60)</td><td valign="bottom">2819387</td><td valign="bottom">97.6</td><td valign="bottom">91.4</td><td valign="bottom">79.6</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">2</td><td valign="bottom">4</td><td valign="bottom">&#8710;16 bp@ 498/2595</td><td valign="bottom">WGS (94)</td><td valign="bottom">&#8211;</td><td valign="bottom"><italic>purR</italic> (VF_1572); N71T (52)</td><td valign="bottom">2992978</td><td valign="bottom">96.9</td><td valign="bottom">103.3</td><td valign="bottom">91.4</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">2</td><td valign="bottom">5</td><td valign="bottom">&#8710;16 bp@ 498/2595</td><td valign="bottom">WGS (90)</td><td valign="bottom">&#8211;</td><td valign="bottom"><italic>purR</italic> (VF_1572); N71T (95)</td><td valign="bottom">3844830</td><td valign="bottom">96.3</td><td valign="bottom">123.6</td><td valign="bottom">109</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">2</td><td valign="bottom">3</td><td valign="bottom">1</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom"><italic>tadF2</italic> (VF_A0228); G21D (68)</td><td valign="bottom">3393611</td><td valign="bottom">90.7</td><td valign="bottom">99.5</td><td valign="bottom">92.2</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">3</td><td valign="bottom">1</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom"><italic>tadF2</italic> (VF_A0228); G21D (140)</td><td valign="bottom">7974773</td><td valign="bottom">91.5</td><td valign="bottom">147.9</td><td valign="bottom">143.9</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">3</td><td valign="bottom">2</td><td valign="bottom">T195I</td><td valign="bottom">WGS (65)</td><td valign="bottom">&#8211;</td><td valign="bottom"><italic>tadF2</italic> (VF_A0228); G21D (28)</td><td valign="bottom">1989875</td><td valign="bottom">95.5</td><td valign="bottom">65.4</td><td valign="bottom">58.2</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">3</td><td valign="bottom">3</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom"><italic>tadF2</italic> (VF_A0228); G21D (77)</td><td valign="bottom">3253899</td><td valign="bottom">96.7</td><td valign="bottom">103.8</td><td valign="bottom">94.4</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">3</td><td valign="bottom">4</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom"><italic>tadF2</italic> (VF_A0228); G21D (58)</td><td valign="bottom">3242749</td><td valign="bottom">97.1</td><td valign="bottom">103.3</td><td valign="bottom">94.7</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">3</td><td valign="bottom">5</td><td valign="bottom">&#8211;</td><td valign="bottom"/><td valign="bottom">&#8211;</td><td valign="bottom"><italic>tadF2</italic> (VF_A0228); G21D (25)</td><td valign="bottom">2190771</td><td valign="bottom">95.9</td><td valign="bottom">67.5</td><td valign="bottom">59</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">4</td><td valign="bottom">1</td><td valign="bottom">E43*</td><td valign="bottom">WGS (102)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">6651385</td><td valign="bottom">92</td><td valign="bottom">125.1</td><td valign="bottom">130</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">4</td><td valign="bottom">3</td><td valign="bottom">E43*</td><td valign="bottom">WGS (111)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">4032373</td><td valign="bottom">96.4</td><td valign="bottom">135.9</td><td valign="bottom">120.4</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">4</td><td valign="bottom">4</td><td valign="bottom">E43*</td><td valign="bottom">WGS (187)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">6122168</td><td valign="bottom">95.8</td><td valign="bottom">203.4</td><td valign="bottom">179.4</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">4</td><td valign="bottom">5</td><td valign="bottom">E43*</td><td valign="bottom">WGS (90)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3177817</td><td valign="bottom">96.7</td><td valign="bottom">100.8</td><td valign="bottom">90.6</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">5</td><td valign="bottom">1</td><td valign="bottom">&#8710;1 bp @ 2325/2595nt</td><td valign="bottom">WGS (113)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">7166870</td><td valign="bottom">90.4</td><td valign="bottom">134.5</td><td valign="bottom">130.9</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">5</td><td valign="bottom">2</td><td valign="bottom">&#8710;1 bp @ 2325/2595nt</td><td valign="bottom">WGS (94)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">3703946</td><td valign="bottom">96.7</td><td valign="bottom">118.6</td><td valign="bottom">108.3</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">5</td><td valign="bottom">3</td><td valign="bottom">&#8710;1 bp @ 2325/2595nt</td><td valign="bottom">WGS (66)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">2828102</td><td valign="bottom">97.4</td><td valign="bottom">98.6</td><td valign="bottom">90.4</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">5</td><td valign="bottom">4</td><td valign="bottom">&#8710;1 bp @ 2325/2595nt</td><td valign="bottom">WGS (109)</td><td valign="bottom">&#8211;</td><td valign="bottom">&#8211;</td><td valign="bottom">4721575</td><td valign="bottom">97</td><td valign="bottom">158.9</td><td valign="bottom">143.8</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">2</td><td valign="bottom">6</td><td valign="bottom">1</td><td valign="bottom">T195I</td><td valign="bottom">WGS (105)</td><td valign="bottom">&#8211;</td><td valign="bottom"><italic>tadF2</italic> (VF_A0228); G21D (28)</td><td valign="bottom">2743693</td><td valign="bottom">94</td><td valign="bottom">83.3</td><td valign="bottom">73.6</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">6</td><td valign="bottom">3</td><td valign="bottom">T195I</td><td valign="bottom">WGS (142)</td><td valign="bottom">&#8211;</td><td valign="bottom"><italic>tadF2</italic> (VF_A0228); G21D (49)</td><td valign="bottom">5594771</td><td valign="bottom">97.5</td><td valign="bottom">191.7</td><td valign="bottom">175.3</td></tr><tr><td valign="bottom">H905</td><td valign="bottom">15</td><td valign="bottom">6</td><td valign="bottom">4</td><td valign="bottom">T195I</td><td valign="bottom">WGS (105)</td><td valign="bottom">&#8211;</td><td valign="bottom"><italic>tadF2</italic> (VF_A0228); G21D (37)</td><td valign="bottom">3361206</td><td valign="bottom">96</td><td valign="bottom">115.9</td><td valign="bottom">101.4</td></tr></tbody></table><table-wrap-foot><fn id="tblfn3"><p><sup>&#8224;</sup>Individual characterized strain collection names assigned to isolates were derived from their ancestral lineage (e.g. MJ11), evolved passage (e.g. EP2), the population (e.g. 1), and isolate number (e.g. 1), which in the preceding example would give rise to strain collection name of MJ11EP2-1-1. Isolates in bold served as allelic <italic>binK</italic> representatives for further assays. mg: metagenome sequencing by pooling five isolates from a population.</p></fn><fn id="tblfn4"><p><sup>&#8225;</sup>The&#160;presence of mutations was determined from Illumina short read (100PE) whole genome sequencing (WGS), by allele-specific PCR (PCR), and/or by locus PCR-amplification, followed by Sanger sequencing (SS). &#8216;&#8211;' indicates that no mutations were identified at this locus&#160;by breseq&#160;(<xref ref-type="bibr" rid="bib25">Deatherage and Barrick, 2014</xref>) in this isolate using WGS. &#8216;n.d.&#8217; indicates that the presence of mutations at this locus was not determined.</p></fn><fn id="tblfn5"><p><sup>&#167;</sup>The number of reads supporting the mutation call from WGS data is provided. Mutations were called for sites with minimum coverage of 20 mappable reads. Mutations identified by Sanger sequencing (SS) of PCR-generated amplicons were confirmed from alignments of both forward and reverse reads. Coding genes reference <italic>V. fischeri</italic> ES114 locus tags.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24502-v2.xml">elife-24502-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.24502.005</object-id><label>Table 2.</label><caption><title>Activity of novel indole compounds against DYRK1A and related kinases relative to ID-8.</title></caption><table frame="hsides" rules="groups"><thead><tr valign="top"><th rowspan="2">Compound</th><th rowspan="2">R<sub>1</sub></th><th rowspan="2">R<sub>2</sub></th><th rowspan="2">R<sub>3</sub></th><th align="center" colspan="10">IC<sub>50</sub> (&#956;M)</th></tr><tr valign="top"><th>DYRK1A</th><th>DYRK1B</th><th>DYRK2</th><th>CLK1</th><th>CLK4</th><th>GSK3&#945;</th><th>GSK3&#946;</th><th>LRRK2</th><th>PIM1</th><th>PIM3</th></tr></thead><tbody><tr valign="top"><td>ID-8</td><td>6-OH</td><td>NO<sub>2</sub></td><td>p-OCH<sub>3</sub></td><td>0.104</td><td>0.040</td><td>NC</td><td>1.37</td><td>1.05</td><td>0.428</td><td>0.153</td><td>0.100</td><td>0.376</td><td>0.176</td></tr><tr valign="top"><td>34</td><td>6-OCOCH<sub>3</sub></td><td>NHCOCH<sub>3</sub></td><td>p-OCH3</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td></tr><tr valign="top"><td>28</td><td>6-OCH<sub>3</sub></td><td>NO<sub>2</sub></td><td>p-OCH<sub>3</sub></td><td>0.346</td><td>0.695</td><td>NC</td><td>NC</td><td>2.23</td><td>2.36</td><td>3.44</td><td>0.854</td><td>0.176</td><td>0.259</td></tr><tr valign="top"><td>29</td><td>6-OCH<sub>2</sub>C<sub>6</sub>H<sub>5</sub></td><td>NO<sub>2</sub></td><td>p-OCH<sub>3</sub></td><td>6.52</td><td>3.57</td><td>NC</td><td>NC</td><td>NC</td><td>18.0</td><td>14.3</td><td>8.16</td><td>18.5</td><td>8.28</td></tr><tr valign="top"><td>30</td><td>6-OCOCH<sub>3</sub></td><td>NO<sub>2</sub></td><td>p-OCH<sub>3</sub></td><td>0.680</td><td>0.299</td><td>NC</td><td>NC</td><td>11.95</td><td>22.15</td><td>1.71</td><td>0.891</td><td>4.15</td><td>1.21</td></tr><tr valign="top"><td>45</td><td>6-OH</td><td>NO<sub>2</sub></td><td>p-OH</td><td>0.122</td><td>0.060</td><td>2.30</td><td>0.364</td><td>0.163</td><td>0.375</td><td>0.233</td><td>0.046</td><td>0.243</td><td>0.180</td></tr><tr valign="top"><td>48</td><td>6-OH</td><td>NO<sub>2</sub></td><td>m-OCH<sub>3</sub></td><td>0.594</td><td>0.257</td><td>NC</td><td>13.0</td><td>4.77</td><td>2.32</td><td>0.692</td><td>2.17</td><td>0.654</td><td>0.593</td></tr><tr valign="top"><td>49</td><td>6-OH</td><td>NO<sub>2</sub></td><td>o-OCH<sub>3</sub></td><td>0.829</td><td>0.341</td><td>41.9</td><td>10.7</td><td>8.59</td><td>6.12</td><td>3.82</td><td>1.98</td><td>1.44</td><td>0.892</td></tr><tr valign="top"><td>51</td><td>5-NO<sub>2</sub></td><td>NO<sub>2</sub></td><td>p-OCH<sub>3</sub></td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>7.62</td><td>4.57</td></tr><tr valign="top"><td>38</td><td>H</td><td>NO<sub>2</sub></td><td>p-OCH<sub>3</sub></td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>NC</td><td>20.74</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24669-v2.xml">elife-24669-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl11" position="float"><object-id pub-id-type="doi">10.7554/eLife.24669.036</object-id><label>Table 11.</label><caption><p>Rich medium replicate 3: All mutations detected in rounds 5, 10 and 15 of rich medium replicate 3. See <xref ref-type="table" rid="tbl9">Table 9</xref> caption. Note low coverage on <inline-formula><mml:math id="inf382"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp mutation at 523086 noted in bold.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.24669.036">http://dx.doi.org/10.7554/eLife.24669.036</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="4">Rich medium replicate 3</th></tr><tr><th>Round, (coverage)</th><th>5, (<inline-formula><mml:math id="inf383"><mml:mrow><mml:mn>291</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</th><th>10, (<inline-formula><mml:math id="inf384"><mml:mrow><mml:mn>45</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</th><th>15, (<inline-formula><mml:math id="inf385"><mml:mrow><mml:mn>186</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</th></tr></thead><tbody><tr><th rowspan="2">Mutation (loc., mut., frac., cov.)</th><td>457978, <italic>clpX</italic> E185<inline-formula><mml:math id="inf386"><mml:msup><mml:mi/><mml:mo>*</mml:mo></mml:msup></mml:math></inline-formula>, 100%, 300</td><td>457978, <italic>clpX</italic> E185<inline-formula><mml:math id="inf387"><mml:msup><mml:mi/><mml:mo>*</mml:mo></mml:msup></mml:math></inline-formula>, 100%, 43</td><td>457978, <italic>clpX</italic> E185<inline-formula><mml:math id="inf388"><mml:msup><mml:mi/><mml:mo>*</mml:mo></mml:msup></mml:math></inline-formula>, 100%, 185</td></tr><tr><td>523086, IG&#160;<inline-formula><mml:math id="inf389"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp, 50%, <bold>38</bold></td><td>523086, IG&#160;<inline-formula><mml:math id="inf390"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp, 100%, <bold>8</bold></td><td>523086, IG&#160;<inline-formula><mml:math id="inf391"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp, 100%, <bold>16</bold></td></tr><tr><th/><td>950518, <italic>pflA</italic> T188I, 26.3%, 332</td><td>950518, <italic>pflA</italic> T188I, 53.3%, 53</td><td>950518, <italic>pflA</italic> T188I, 30.6%, 190</td></tr><tr><th/><td/><td>321263, IG&#160;T<inline-formula><mml:math id="inf392"><mml:mo>&#8594;</mml:mo></mml:math></inline-formula>C, 25%, 16</td><td>1968653, <italic>cheR</italic> Q238K, 29.6%, 190</td></tr><tr><th/><td/><td>382794,&#160;<italic>yaiX</italic> <inline-formula><mml:math id="inf393"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>+</mml:mo><mml:mn>9</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula>bp insertion, 64%, NA</td><td>382794,&#160;<italic>yaiX</italic>&#160;<inline-formula><mml:math id="inf394"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>+</mml:mo><mml:mn>9</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula>bp insertion, 25.8%, NA</td></tr><tr><th/><td/><td/><td>4161562,&#160;<italic>fabR</italic>&#160;<inline-formula><mml:math id="inf395"><mml:mrow><mml:mi mathvariant="normal">&#916;</mml:mi><mml:mo>&#8290;</mml:mo><mml:mn>17</mml:mn></mml:mrow></mml:math></inline-formula>bp, 46.2%, 67</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24669-v2.xml">elife-24669-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl12" position="float"><object-id pub-id-type="doi">10.7554/eLife.24669.037</object-id><label>Table 12.</label><caption><p>Rich medium replicate 4: All mutations detected in rounds 5, 10 and 15 of rich medium replicate 4. See <xref ref-type="table" rid="tbl9">Table 9</xref> caption. Note low coverage on <inline-formula><mml:math id="inf396"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp mutation at 523086 noted in bold.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.24669.037">http://dx.doi.org/10.7554/eLife.24669.037</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="4">Rich medium replicate 4</th></tr><tr><th>Round, (coverage)</th><th>5, (<inline-formula><mml:math id="inf397"><mml:mrow><mml:mn>384</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</th><th>10, (<inline-formula><mml:math id="inf398"><mml:mrow><mml:mn>555</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</th><th>15, (<inline-formula><mml:math id="inf399"><mml:mrow><mml:mn>333</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</th></tr></thead><tbody><tr><th rowspan="3">Mutation (loc., mut., frac., cov.)</th><td>457978, <italic>clpX</italic> E185<inline-formula><mml:math id="inf400"><mml:msup><mml:mi/><mml:mo>*</mml:mo></mml:msup></mml:math></inline-formula>, 100%, 370</td><td>457978, <italic>clpX</italic> E185<inline-formula><mml:math id="inf401"><mml:msup><mml:mi/><mml:mo>*</mml:mo></mml:msup></mml:math></inline-formula>, 100%, 559</td><td>457978, <italic>clpX</italic> E185<inline-formula><mml:math id="inf402"><mml:msup><mml:mi/><mml:mo>*</mml:mo></mml:msup></mml:math></inline-formula>, 100%, 339</td></tr><tr><td>523086, IG&#160;<inline-formula><mml:math id="inf403"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp, 50%, <bold>72</bold></td><td>523086, IG&#160;<inline-formula><mml:math id="inf404"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp, 100%, <bold>34/83</bold></td><td>523086, IG&#160;<inline-formula><mml:math id="inf405"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp, 100%, <bold>19/33</bold></td></tr><tr><td>950518, <italic>pflA</italic> T188I, 31.7%, 446</td><td/><td>3619915, <italic>rhsB&#160;</italic>W242G, 24.9%, 20</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24669-v2.xml">elife-24669-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl5" position="float"><object-id pub-id-type="doi">10.7554/eLife.24669.026</object-id><label>Table 5.</label><caption><p>Minimal medium replicate 1: All mutations detected in rounds 5 and 10 of minimal medium replicate 1. The <italic>galS</italic>L22R mutation in rounds 5 and 10 was confirmed by Sanger sequencing. See <xref ref-type="table" rid="tbl9">Table 9</xref> caption.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.24669.026">http://dx.doi.org/10.7554/eLife.24669.026</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3">Minimal medium replicate 1</th></tr></thead><tbody><tr><th>round, (coverage)</th><td>5, (<inline-formula><mml:math id="inf250"><mml:mrow><mml:mn>71</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</td><td>10, (<inline-formula><mml:math id="inf251"><mml:mrow><mml:mn>214</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</td></tr><tr><th rowspan="2">Mutation (loc., mut., frac., cov.)</th><td>1196220, <italic>icd&#160;</italic>H366H,78.4%, 46</td><td>1196220, <italic>icd&#160;</italic>H366H,&#160;100%, 141</td></tr><tr><td>1196232, <italic>icd&#160;</italic>T370T, 71.1%, 34</td><td>1196232, <italic>icd&#160;</italic>T370T, 100%, 101</td></tr><tr><th/><td>1196247, <italic>icd&#160;</italic>L375L, 72.0%, 25</td><td>1196247, <italic>icd&#160;</italic>L375L, 100%, 75</td></tr><tr><th/><td>1196277, <italic>icd&#160;</italic>N385N, 47.1%, 17</td><td>2015871, <italic>fliG&#160;</italic>V331D, 100%, 111</td></tr><tr><th/><td>1196280, <italic>icd&#160;</italic>A386A, 47.1%, 17</td><td>2241604, <italic>galS&#160;</italic>L22R, 100%, 184</td></tr><tr><th/><td>1196283, <italic>icd&#160;</italic>K387K, 47.2%, 17</td><td>2685013, <italic>glyA&#160;</italic>H165H, 100%, 197</td></tr><tr><th/><td>1196292, <italic>icd&#160;</italic>T390T, 46.2%, 13</td><td>3815859, <italic>rph&#160;<inline-formula><mml:math id="inf252"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula></italic>82 bp, 100%, 260</td></tr><tr><th/><td>1196304, <italic>icd&#160;</italic>E394E, 46.2%, 13</td><td/></tr><tr><th/><td>2015871, <italic>fliG&#160;</italic>V331D, 70.0%, 60</td><td/></tr><tr><th/><td>2241604, <italic>galS&#160;</italic>L22R, 100%, 45</td><td/></tr><tr><th/><td>2685013, <italic>glyA&#160;</italic>H165H, 100%, 62</td><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24669-v2.xml">elife-24669-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl6" position="float"><object-id pub-id-type="doi">10.7554/eLife.24669.027</object-id><label>Table 6.</label><caption><p>Minimal medium replicate 2: All mutations detected in rounds 5&#160;and 10 of minimal medium replicate 2. The <italic>galS</italic>L22R mutation in rounds 5 and 10 was confirmed by Sanger sequencing. See <xref ref-type="table" rid="tbl9">Table 9</xref> caption.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.24669.027">http://dx.doi.org/10.7554/eLife.24669.027</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3">Minimal medium replicate 2</th></tr></thead><tbody><tr><th>round, (coverage)</th><td>5, (<inline-formula><mml:math id="inf253"><mml:mrow><mml:mn>67</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</td><td>10, (<inline-formula><mml:math id="inf254"><mml:mrow><mml:mn>64</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</td></tr><tr><th rowspan="2">Mutation (loc., mut., frac., cov.)</th><td>2241604, <italic>galS&#160;</italic>L22R, 100%, 70</td><td>1757419, IG <inline-formula><mml:math id="inf255"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>+</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula>17 bp&#160;insertion, 94.9%, 37</td></tr><tr><td>2685013, <italic>glyA&#160;</italic>H165H, 100%, 65</td><td>2241604, <italic>galS&#160;</italic>L22R, 100%, 47</td></tr><tr><th/><td/><td>2685013, <italic>glyA&#160;</italic>H165H, 100%, 79</td></tr><tr><th/><td/><td>3815828, IG T<inline-formula><mml:math id="inf256"><mml:mo>&#8594;</mml:mo></mml:math></inline-formula>G, 43.5%, 62</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24669-v2.xml">elife-24669-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl7" position="float"><object-id pub-id-type="doi">10.7554/eLife.24669.028</object-id><label>Table 7.</label><caption><p>Minimal medium replicate 3: All mutations detected in rounds 5 and 10 of minimal medium replicate 3. See <xref ref-type="table" rid="tbl9">Table 9</xref> caption.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.24669.028">http://dx.doi.org/10.7554/eLife.24669.028</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3">Minimal medium replicate 3</th></tr></thead><tbody><tr><th>round, (coverage)</th><td>5, (208<inline-formula><mml:math id="inf257"><mml:mo>&#215;</mml:mo></mml:math></inline-formula>)</td><td>10, (229<inline-formula><mml:math id="inf258"><mml:mo>&#215;</mml:mo></mml:math></inline-formula>)</td></tr><tr><th rowspan="2">Mutation (loc., mut., frac., cov.)</th><td>1291079, <italic>rssB&#160;</italic>A280T,&#160;29.7%, 54</td><td>2241595, <italic>galS&#160;<inline-formula><mml:math id="inf259"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula></italic>1bp, 100%, 218</td></tr><tr><td>2241595, <italic>galS&#160;<inline-formula><mml:math id="inf260"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula></italic>1bp, 64.7%, 102</td><td>3277264, <italic>prlF&#160;</italic>+CATTCAA insertion, 93.6%, 109</td></tr><tr><th/><td>3762200, <italic>rhsA&#160;</italic>A6A, 23.5%, 181</td><td>3350529, IG&#160;T<inline-formula><mml:math id="inf261"><mml:mo>&#8594;</mml:mo></mml:math></inline-formula>C, 100%, 117</td></tr><tr><th/><td>3762212, <italic>rhsA&#160;</italic>G10G, 23.1%, 164</td><td>3762200, <italic>rhsA&#160;</italic>A6A, 45.8%, 320</td></tr><tr><th/><td/><td>3762212, <italic>rhsA&#160;</italic>G10G, 42.0%, 292</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24669-v2.xml">elife-24669-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl8" position="float"><object-id pub-id-type="doi">10.7554/eLife.24669.029</object-id><label>Table 8.</label><caption><p>Minimal medium replicate 4: All mutations detected in rounds 5 and 10 of minimal medium replicate 4. See <xref ref-type="table" rid="tbl9">Table 9</xref> caption.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.24669.029">http://dx.doi.org/10.7554/eLife.24669.029</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3">Minimal medium replicate 4</th></tr></thead><tbody><tr><th>round, (coverage)</th><td>5, (256<inline-formula><mml:math id="inf262"><mml:mo>&#215;</mml:mo></mml:math></inline-formula>)</td><td>10, (230<inline-formula><mml:math id="inf263"><mml:mo>&#215;</mml:mo></mml:math></inline-formula>)</td></tr><tr><th rowspan="2">Mutation (loc., mut., frac., cov.)</th><td>2241232, <italic>galS&#160;</italic>R146L,&#160;72.4%, 274</td><td>2020519, <italic>fliM&#160;</italic>E145K, 100%, 205</td></tr><tr><td/><td>2241665, <italic>galS&#160;</italic>I2L, 100%, 304</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24669-v2.xml">elife-24669-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl9" position="float"><object-id pub-id-type="doi">10.7554/eLife.24669.034</object-id><label>Table 9.</label><caption><p>Rich medium replicate 1: All mutations detected above a frequency of 0.2 in rounds 5, 10 and 15 of rich medium selection replicate 1. The first number in each cell denotes the distance in base pairs from <italic>ori</italic> (location). The second entry (mutation) identifies the mutations with &#8216;IG&#8217; denoting an intergenic mutation. The third entry (fraction) is the fraction of the population carrying this mutation (as inferred by breseq in polymorphism mode). The fourth entry (coverage) is the number of reads that aligned to this location. In the round 15 strain, the <italic>clpX</italic> SNP and <inline-formula><mml:math id="inf358"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp deletion at position 523,086 were confirmed by Sanger sequencing.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.24669.034">http://dx.doi.org/10.7554/eLife.24669.034</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="4">Rich medium replicate 1</th></tr><tr><th>Round, (coverage)</th><th>5, (<inline-formula><mml:math id="inf359"><mml:mrow><mml:mn>172</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</th><th>10, (<inline-formula><mml:math id="inf360"><mml:mrow><mml:mn>213</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</th><th>15, (<inline-formula><mml:math id="inf361"><mml:mrow><mml:mn>180</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</th></tr></thead><tbody><tr><th rowspan="2">Mutation (loc., mut., frac., cov.)</th><td>457978, <italic>clpX</italic> E185<inline-formula><mml:math id="inf362"><mml:msup><mml:mi/><mml:mo>*</mml:mo></mml:msup></mml:math></inline-formula>, 75.2%, 179</td><td>457978, <italic>clpX</italic> E185<inline-formula><mml:math id="inf363"><mml:msup><mml:mi/><mml:mo>*</mml:mo></mml:msup></mml:math></inline-formula>, 100%, 199</td><td>457978, <italic>clpX</italic> E185<inline-formula><mml:math id="inf364"><mml:msup><mml:mi/><mml:mo>*</mml:mo></mml:msup></mml:math></inline-formula>, 100%, 164</td></tr><tr><td>523086, IG&#160;<inline-formula><mml:math id="inf365"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp, 100%, 194</td><td>523086, IG&#160;<inline-formula><mml:math id="inf366"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp, 100%, 266</td><td>523086, IG&#160;<inline-formula><mml:math id="inf367"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp, 100%, 168</td></tr><tr><th/><td>950518, <italic>pflA</italic> T188I, 22.2%, 144</td><td>990379,&#160;IG&#160;A<inline-formula><mml:math id="inf368"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo stretchy="false">&#8594;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula>C,&#160;100%, 201</td><td>663115,&#160;<italic>dacA</italic>&#160;<inline-formula><mml:math id="inf369"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp, 100%,&#160;150</td></tr><tr><th/><td>1978458, IG&#160;G<inline-formula><mml:math id="inf370"><mml:mo>&#8594;</mml:mo></mml:math></inline-formula>T, 21.2%, 156</td><td/><td>990379,&#160;IG&#160;A<inline-formula><mml:math id="inf371"><mml:mo>&#8594;</mml:mo></mml:math></inline-formula>C, 100%, 156</td></tr><tr><th/><td>3618863, <italic>nikR</italic> H92H, 20.7%, 189</td><td/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24669-v2.xml">elife-24669-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl10" position="float"><object-id pub-id-type="doi">10.7554/eLife.24669.035</object-id><label>Table 10.</label><caption><p>Rich medium replicate 2: All mutations detected in rounds 5, 10 and 15 of rich medium replicate 2. See <xref ref-type="table" rid="tbl9">Table 9</xref> caption. Note low coverage on <inline-formula><mml:math id="inf372"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp mutation at 523086 noted in bold.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.24669.035">http://dx.doi.org/10.7554/eLife.24669.035</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="4">Rich medium replicate 2</th></tr><tr><th>Round, (coverage)</th><th>5, (<inline-formula><mml:math id="inf373"><mml:mrow><mml:mn>218</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</th><th>10, (<inline-formula><mml:math id="inf374"><mml:mrow><mml:mn>100</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</th><th>15, (<inline-formula><mml:math id="inf375"><mml:mrow><mml:mn>166</mml:mn><mml:mo>&#215;</mml:mo></mml:mrow></mml:math></inline-formula>)</th></tr></thead><tbody><tr><th rowspan="2"><bold>Mutation (loc., mut., frac., cov.)</bold></th><td>457978, <italic>clpX</italic> E185<inline-formula><mml:math id="inf376"><mml:msup><mml:mi/><mml:mo>*</mml:mo></mml:msup></mml:math></inline-formula>, 100%, 220</td><td>457978, <italic>clpX</italic> E185<inline-formula><mml:math id="inf377"><mml:msup><mml:mi/><mml:mo>*</mml:mo></mml:msup></mml:math></inline-formula>, 100%, 109</td><td>457978, <italic>clpX</italic> E185<inline-formula><mml:math id="inf378"><mml:msup><mml:mi/><mml:mo>*</mml:mo></mml:msup></mml:math></inline-formula>, 100%, 184</td></tr><tr><td>950518, <italic>pflA</italic> T188I, 27.2%, 210</td><td>523086, IG&#160;<inline-formula><mml:math id="inf379"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp, 100%, <bold>16</bold></td><td>523086, IG&#160;<inline-formula><mml:math id="inf380"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp, 100%, <bold>24</bold></td></tr><tr><th/><td>523086, IG&#160;<inline-formula><mml:math id="inf381"><mml:mi mathvariant="normal">&#916;</mml:mi></mml:math></inline-formula>1 bp, 100%, <bold>10/18</bold></td><td/><td>667259, <italic>mrdA</italic> R320H, 39.5%, 159</td></tr><tr><th/><td/><td/><td>794472, <italic>modE</italic> L58*, 42.4%, 136</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-24998-v2.xml">elife-24998-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.24998.004</object-id><label>Table 1.</label><caption><p>Summary of kinetic analysis of ARPP-16 phosphorylation by MAST3 and PKA.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.24998.004">http://dx.doi.org/10.7554/eLife.24998.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom"><break/></th><th rowspan="2" valign="top">MAST3</th><th>Vm</th><th>Km</th><th valign="bottom">Catalytic efficiency</th></tr><tr><th><sup>32</sup>P-incorporation/minute</th><th>nM</th><th valign="bottom">Vm/Km</th></tr></thead><tbody><tr><td valign="bottom"><xref ref-type="fig" rid="fig1">Figure 1b</xref></td><td valign="bottom">ARPP-16</td><td valign="bottom">1388&#160;&#177;&#160;27</td><td valign="bottom">90&#160;&#177;&#160;5</td><td valign="bottom">15.4</td></tr><tr><td valign="bottom"><break/></td><td valign="bottom">P-S88-ARPP-16</td><td valign="bottom">866&#160;&#177;&#160;59</td><td valign="bottom">158&#160;&#177;&#160;26</td><td valign="bottom">5.5</td></tr><tr><td valign="bottom"><xref ref-type="fig" rid="fig1s1">Figure 1&#8212;figure supplement 1</xref></td><td valign="bottom">ARPP-16</td><td valign="bottom">1391&#160;&#177;&#160;34</td><td valign="bottom">92&#160;&#177;&#160;6</td><td valign="bottom">15.1</td></tr><tr><td valign="bottom"><break/></td><td valign="bottom">P-S88D-ARPP-16</td><td valign="bottom">465&#160;&#177;&#160;32</td><td valign="bottom">138&#160;&#177;&#160;24</td><td valign="bottom">3.4</td></tr><tr><td valign="bottom"><break/></td><td rowspan="2" valign="top">PKA</td><td>Vm</td><td>Km</td><td valign="bottom">Catalytic efficiency</td></tr><tr><td valign="bottom"><break/></td><td><sup>32</sup>P-incorporation/minute</td><td>nM</td><td valign="bottom">Vm/Km</td></tr><tr><td valign="bottom"><xref ref-type="fig" rid="fig1">Figure 1d</xref></td><td valign="bottom">ARPP-16</td><td valign="bottom">6999&#160;&#177;&#160;127</td><td valign="bottom">1685&#160;&#177;&#160;51</td><td valign="bottom">4.1</td></tr><tr><td valign="bottom"><break/></td><td valign="bottom">P-S46-ARPP-16</td><td valign="bottom">1027&#160;&#177;&#160;39</td><td valign="bottom">870&#160;&#177;&#160;47</td><td valign="bottom">1.2</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-25068-v1.xml">elife-25068-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.25068.003</object-id><label>Table 1.</label><caption><p>Binding affinities of the different YAP<sup>50-151</sup> proteins for wt hTEAD4<sup>217-434</sup>. The affinities were measured at 298&#176;K by Surface Plasmon Resonance in n&#160;&#8805;&#160;3 independent experiments. N-Avitagged hTEAD4<sup>217-434</sup> was immobilized on sensor chips. The K<sub>d</sub> values were obtained from equilibrium data (K<sub>d</sub><sup>eq</sup>). Averages and standard errors (SE) are given. &#916;&#916;G = &#916;G<sub>mutant</sub> &#8211; &#916;G<sub>wt</sub> and SE<sub>&#916;&#916;G</sub> = (SE<sub>&#916;Gmutant</sub><sup>2</sup> + SE<sub>&#916;Gwt</sub><sup>2</sup>)<sup>1/2</sup>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.25068.003">http://dx.doi.org/10.7554/eLife.25068.003</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><bold>Mutation site</bold></th><th><bold>YAP<sup>50-171</sup></bold></th><th><bold>K<sub>d</sub><sup>eq</sup></bold> <break/><bold>(nM)</bold></th><th><bold>&#916;G</bold> <break/><bold>(kcal/mol)</bold></th><th><bold>&#916;&#916;G</bold> <break/><bold>(kcal/mol)</bold></th></tr><tr><th>Wild type</th><th>18&#160;&#177;&#160;0</th><th>&#8722;10.56&#160;&#177;&#160;0.01</th><th><break/></th></tr></thead><tbody><tr><td rowspan="3"><bold>&#945;-helix</bold></td><td>Leu65Ala</td><td>794&#160;&#177;&#160;4</td><td>&#8722;8.31&#160;&#177;&#160;0.01</td><td>2.24&#160;&#177;&#160;0.01</td></tr><tr><td>Leu68Ala</td><td>464&#160;&#177;&#160;4</td><td>&#8722;8.63&#160;&#177;&#160;0.01</td><td>1.92&#160;&#177;&#160;0.01</td></tr><tr><td>Phe69Ala</td><td>6447&#160;&#177;&#160;400</td><td>&#8722;7.08&#160;&#177;&#160;0.04</td><td>3.48&#160;&#177;&#160;0.04</td></tr><tr><td rowspan="6"><bold>&#8486;-loop</bold></td><td>Met86Ala</td><td>2080&#160;&#177;&#160;69</td><td>&#8722;7.74&#160;&#177;&#160;0.02</td><td>2.81&#160;&#177;&#160;0.02</td></tr><tr><td>Arg89Ala</td><td>27423&#160;&#177;&#160;381</td><td>&#8722;6.22&#160;&#177;&#160;0.01</td><td>4.34&#160;&#177;&#160;0.01</td></tr><tr><td>Leu91Ala</td><td>30550&#160;&#177;&#160;2250</td><td>&#8722;6.15&#160;&#177;&#160;0.04</td><td>4.40&#160;&#177;&#160;0.05</td></tr><tr><td>Ser94Ala</td><td>5623&#160;&#177;&#160;341</td><td>&#8722;7.16&#160;&#177;&#160;0.04</td><td>3.40&#160;&#177;&#160;0.04</td></tr><tr><td>Phe95Ala</td><td>26045&#160;&#177;&#160;755</td><td>&#8722;6.25&#160;&#177;&#160;0.02</td><td>4.31&#160;&#177;&#160;0.02</td></tr><tr><td>Phe96Ala</td><td>4755&#160;&#177;&#160;245</td><td>&#8722;7.25&#160;&#177;&#160;0.03</td><td>3.30&#160;&#177;&#160;0.03</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-25068-v1.xml">elife-25068-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.25068.004</object-id><label>Table 2.</label><caption><p>Properties of the different hTEAD4<sup>217-434</sup> proteins. The melting temperatures (T<sub>m</sub>) of the proteins were determined in n&#160;&#8805;&#160;3 independent experiments in a fluorescence-based thermal shift assay. Averages and standard errors (SE) are given. &#916;T<sub>m</sub> = T<sub>m</sub><sup>mutant</sup> &#8211; T<sub>m</sub><sup>wt</sup> and SE<sub>&#916;Tm</sub> = (SE<sub>Tmmutant</sub><sup>2</sup> + SE<sub>Tmwt</sub><sup>2</sup>)<sup>1/2</sup>. For dissociation constant measurements, the different N-Avitagged hTEAD4<sup>217-434</sup> proteins were immobilized on sensor chips and their affinity for wt hYAP<sup>50-171</sup> was measured at 298&#176;K by Surface Plasmon Resonance in n&#160;&#8805;&#160;3 independent experiments. K<sub>d</sub> values were obtained from equilibrium data (K<sub>d</sub><sup>eq</sup>). Averages and standard errors (SE) are given. &#916;&#916;G = &#916;G<sub>mutant</sub> &#8211; &#916;G<sub>wt</sub> and SE<sub>&#916;&#916;G</sub> = (SE<sub>&#916;Gmutant</sub> + SE<sub>&#916;Gwt</sub>)<sup>1/2</sup>. n. d.: not determined.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.25068.004">http://dx.doi.org/10.7554/eLife.25068.004</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><bold>Mutation site</bold></th><th><bold>hTEAD4<sup>217-434</sup></bold></th><th><bold>T<sub>m</sub></bold> <break/><bold>(&#176;C)</bold></th><th><bold>&#916;T<sub>m</sub></bold> <break/><bold>(&#176;C)</bold></th><th><bold>K<sub>d</sub><sup>eq</sup></bold> <break/><bold>(nM)</bold></th><th><bold>&#916;G</bold> <break/><bold>(kcal/mol)</bold></th><th><bold>&#916;&#916;G</bold> <break/><bold>(kcal/mol)</bold></th></tr><tr><th>Wild type</th><th>54.0&#160;&#177;&#160;0.1</th><th><break/></th><th>18&#160;&#177;&#160;0</th><th>&#8722;10.56&#160;&#177;&#160;0.01</th><th><break/></th></tr></thead><tbody><tr><td rowspan="7"><bold>&#945;-helix</bold></td><td>Phe337Ala</td><td>53.1&#160;&#177;&#160;0.1</td><td>&#8722;0.9&#160;&#177;&#160;0.1</td><td>202&#160;&#177;&#160;8</td><td>&#8722;9.12&#160;&#177;&#160;0.02</td><td>1.43&#160;&#177;&#160;0.02</td></tr><tr><td>Tyr369Ala</td><td>49.9&#160;&#177;&#160;0.2</td><td>&#8722;4.2&#160;&#177;&#160;0.3</td><td>n. d.</td><td><break/></td><td><break/></td></tr><tr><td>Phe373Ala</td><td>53.0&#160;&#177;&#160;0.3</td><td>&#8722;1.0&#160;&#177;&#160;0.3</td><td>12&#160;&#177;&#160;0</td><td>&#8722;10.79&#160;&#177;&#160;0.00</td><td>&#8722;0.24&#160;&#177;&#160;0.01</td></tr><tr><td>Lys376Ala</td><td>53.4&#160;&#177;&#160;0.1</td><td>&#8722;0.7&#160;&#177;&#160;0.1</td><td>203&#160;&#177;&#160;9</td><td>&#8722;9.13&#160;&#177;&#160;0.03</td><td>1.43&#160;&#177;&#160;0.03</td></tr><tr><td>Leu377Ala</td><td>49.8&#160;&#177;&#160;0.1</td><td>&#8722;4.3&#160;&#177;&#160;0.2</td><td>n. d.</td><td><break/></td><td><break/></td></tr><tr><td>Leu380Ala</td><td>52.8&#160;&#177;&#160;0.1</td><td>&#8722;1.3&#160;&#177;&#160;0.2</td><td>162&#160;&#177;&#160;7</td><td>&#8722;9.26&#160;&#177;&#160;0.03</td><td>1.30&#160;&#177;&#160;0.03</td></tr><tr><td>Val389Ala</td><td>53.0&#160;&#177;&#160;0.0</td><td>&#8722;1.0&#160;&#177;&#160;0.1</td><td>304&#160;&#177;&#160;5</td><td>&#8722;8.88&#160;&#177;&#160;0.01</td><td>1.67&#160;&#177;&#160;0.01</td></tr><tr><td rowspan="8"><bold>&#8486;-loop</bold></td><td>Glu263Ala</td><td>53.0&#160;&#177;&#160;0.2</td><td>&#8722;1.1&#160;&#177;&#160;0.2</td><td>220&#160;&#177;&#160;9</td><td>&#8722;9.07&#160;&#177;&#160;0.02</td><td>1.48&#160;&#177;&#160;0.03</td></tr><tr><td>Val265Ala</td><td>52.0&#160;&#177;&#160;0.0</td><td>&#8722;2.0&#160;&#177;&#160;0.1</td><td>106&#160;&#177;&#160;3</td><td>&#8722;9.51&#160;&#177;&#160;0.02</td><td>1.05&#160;&#177;&#160;0.02</td></tr><tr><td>Asp272Ala</td><td>54.1&#160;&#177;&#160;0.1</td><td>0.1&#160;&#177;&#160;0.1</td><td>6995&#160;&#177;&#160;317</td><td>&#8722;7.03&#160;&#177;&#160;0.03</td><td>3.53&#160;&#177;&#160;0.03</td></tr><tr><td>Lys273Ala</td><td>48.1&#160;&#177;&#160;0.2</td><td>&#8722;5.9&#160;&#177;&#160;0.3</td><td>n. d.</td><td><break/></td><td><break/></td></tr><tr><td>Leu295Ala</td><td>47.4&#160;&#177;&#160;0.1</td><td>&#8722;6.6&#160;&#177;&#160;0.1</td><td>n. d.</td><td><break/></td><td><break/></td></tr><tr><td>Lys297Ala</td><td>48.4&#160;&#177;&#160;0.2</td><td>&#8722;5.7&#160;&#177;&#160;0.3</td><td>n. d.</td><td><break/></td><td><break/></td></tr><tr><td>Val414Ala</td><td>50.9&#160;&#177;&#160;0.1</td><td>&#8722;3.2&#160;&#177;&#160;0.1</td><td>14&#160;&#177;&#160;1</td><td>&#8722;10.73&#160;&#177;&#160;0.02</td><td>&#8722;0.17&#160;&#177;&#160;0.02</td></tr><tr><td>Tyr429Phe</td><td>52.3&#160;&#177;&#160;0.1</td><td>&#8722;1.7&#160;&#177;&#160;0.1</td><td>71&#160;&#177;&#160;6</td><td>&#8722;9.75&#160;&#177;&#160;0.05</td><td>0.81&#160;&#177;&#160;0.05</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-25068-v1.xml">elife-25068-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl3" position="float"><object-id pub-id-type="doi">10.7554/eLife.25068.007</object-id><label>Table 3.</label><caption><p>Summary of the coupling energy (&#916;&#916;G<sub>int</sub>) measured for each pair of residues. &#916;&#916;G<sub>int</sub> values are in kcal/mol and were calculated according to the description given in the Materials and methods section. Average and standard errors (SE) are shown. The gray cells&#160;indicate mutations of YAP and TEAD in the same binding pocket. n. m. (not measured) indicates that &#916;&#916;G<sub>int</sub> could not be determined experimentally because the K<sub>d</sub> values for the mutant YAP:mutant TEAD interaction in the corresponding double mutant cycles were above our assay limit (&gt;200 &#181;M).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.25068.007">http://dx.doi.org/10.7554/eLife.25068.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3" rowspan="3" valign="top"/><th colspan="9" valign="top"><bold>hYAP<sup>50-171</sup></bold></th></tr><tr><th colspan="3" valign="top"><bold>&#945;-helix</bold></th><th colspan="6" valign="top"><bold>&#8486;-loop</bold></th></tr><tr><th valign="top">Leu65</th><th valign="top">Leu68</th><th valign="top">Phe69</th><th valign="top">Met86</th><th valign="top">Arg89</th><th valign="top">Leu91</th><th valign="top">Ser94</th><th valign="top">Phe95</th><th valign="top">Phe96</th></tr></thead><tbody><tr><td rowspan="7"><bold>hTEAD<sup>217-434</sup></bold></td><td rowspan="4"><bold>&#945;-helix</bold></td><td>Phe337</td><td style="author-callout-style-b8">&#8722;0.34 <break/>&#177;0.04</td><td style="author-callout-style-b8">&#8722;0.65 <break/>&#177;0.04</td><td style="author-callout-style-b8">&#8722;0.96 <break/>&#177;0.04</td><td>0.17 <break/>&#177;0.04</td><td>&#8722;0.44 <break/>&#177;0.03</td><td>n. m.</td><td>&#8722;0.09 <break/>&#177;0.05</td><td>n. m.</td><td>0.02 <break/>&#177;0.06</td></tr><tr><td>Lys376</td><td style="author-callout-style-b8">&#8722;0.65 <break/>&#177;0.04</td><td style="author-callout-style-b8">&#8722;0.41 <break/>&#177;0.05</td><td style="author-callout-style-b8">&#8722;1.28 <break/>&#177;0.06</td><td>0.13 <break/>&#177;0.04</td><td>&#8722;0.62 <break/>&#177;0.03</td><td>&#8722;0.41 <break/>&#177;0.06</td><td>&#8722;0.22 <break/>&#177;0.06</td><td>&#8722;0.30 <break/>&#177;0.04</td><td>&#8722;0.14 <break/>&#177;0.05</td></tr><tr><td>Leu380</td><td style="author-callout-style-b8">&#8722;0.36 <break/>&#177;0.05</td><td style="author-callout-style-b8">&#8722;0.29 <break/>&#177;0.04</td><td style="author-callout-style-b8">&#8722;1.01 <break/>&#177;0.06</td><td>0.07 <break/>&#177;0.03</td><td>&#8722;0.29 <break/>&#177;0.03</td><td>&#8722;0.26 <break/>&#177;0.05</td><td>&#8722;0.08 <break/>&#177;0.05</td><td>&#8722;0.18 <break/>&#177;0.03</td><td>&#8722;0.02 <break/>&#177;0.06</td></tr><tr><td>Val389</td><td style="author-callout-style-b8">&#8722;0.52 <break/>&#177;0.06</td><td style="author-callout-style-b8">&#8722;0.29 <break/>&#177;0.05</td><td style="author-callout-style-b8">&#8722;1.11 <break/>&#177;0.05</td><td>0.16 <break/>&#177;0.03</td><td>n. m.</td><td>n. m.</td><td>0.03 <break/>&#177;0.05</td><td>n. m.</td><td>&#8722;0.06 <break/>&#177;0.04</td></tr><tr><td rowspan="3"><bold>&#8486;-loop</bold></td><td>Glu263</td><td>&#8722;0.05 <break/>&#177;0.05</td><td>&#8722;0.09 <break/>&#177;0.03</td><td>&#8722;0.10 <break/>&#177;0.06</td><td style="author-callout-style-b8">0.07 <break/>&#177;0.05</td><td style="author-callout-style-b8">&#8722;0.59 <break/>&#177;0.06</td><td style="author-callout-style-b8">&#8722;0.62 <break/>&#177;0.05</td><td style="author-callout-style-b8">&#8722;0.96 <break/>&#177;0.05</td><td style="author-callout-style-b8">&#8722;0.51 <break/>&#177;0.05</td><td style="author-callout-style-b8">&#8722;0.04 <break/>&#177;0.07</td></tr><tr><td>Val265</td><td>0.26 <break/>&#177;0.03</td><td>0.15 <break/>&#177;0.03</td><td>0.13 <break/>&#177;0.05</td><td style="author-callout-style-b8">0.09 <break/>&#177;0.03</td><td style="author-callout-style-b8">&#8722;0.58 <break/>&#177;0.02</td><td style="author-callout-style-b8">&#8722;0.28 <break/>&#177;0.06</td><td style="author-callout-style-b8">&#8722;0.06 <break/>&#177;0.06</td><td style="author-callout-style-b8">&#8722;0.61 <break/>&#177;0.04</td><td style="author-callout-style-b8">&#8722;0.22 <break/>&#177;0.06</td></tr><tr><td>Asp272</td><td>n. m.</td><td>&#8722;0.02 <break/>&#177;0.04</td><td>n. m.</td><td style="author-callout-style-b8">&#8722;1.85 <break/>&#177;0.04</td><td style="author-callout-style-b8">&#8722;3.49 <break/>&#177;0.05</td><td style="author-callout-style-b8">&#8722;2.74 <break/>&#177;0.06</td><td style="author-callout-style-b8">&#8722;1.65 <break/>&#177;0.05</td><td style="author-callout-style-b8">&#8722;2.50 <break/>&#177;0.07</td><td style="author-callout-style-b8">&#8722;1.67 <break/>&#177;0.06</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-25069-v1.xml">elife-25069-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.25069.007</object-id><label>Table 1.</label><caption><p>CP behavioral scores ordered by severity as indicated by performance on the CFMT.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.25069.007">http://dx.doi.org/10.7554/eLife.25069.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Participant</th><th rowspan="2"><bold>Sex</bold></th><th rowspan="2">Age</th><th colspan="2">Famous faces questionnaire</th><th colspan="2" rowspan="2">CFMT (total)</th></tr><tr><th>% corr.</th><th>Z- score</th></tr></thead><tbody><tr><td>BL</td><td>F</td><td>18</td><td>20</td><td>-4.88</td><td>28</td><td>-4.15</td></tr><tr><td>BQ</td><td>F</td><td>29</td><td>16</td><td>-4.92</td><td>30</td><td>-3.89</td></tr><tr><td>KG</td><td>F</td><td>49</td><td>75</td><td>-0.69</td><td>33</td><td>-3.5</td></tr><tr><td valign="top">ON (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>48</td><td>60.7</td><td>-1.77</td><td>35</td><td>-3.23</td></tr><tr><td valign="top">MT (<xref ref-type="bibr" rid="bib53">Nishimura et al., 2010</xref>), (<xref ref-type="bibr" rid="bib6">Avidan et al., 2011</xref>), <break/>(<xref ref-type="bibr" rid="bib10">Behrmann and Avidan, 2005</xref>), (<xref ref-type="bibr" rid="bib75">Thomas et al., 2009</xref>), <break/>(<xref ref-type="bibr" rid="bib2">Avidan and Behrmann, 2008</xref>), <break/>(<xref ref-type="bibr" rid="bib40">Humphreys et al., 2007</xref>) , (<xref ref-type="bibr" rid="bib8">Behrmann et al., 2007</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>M</td><td>50</td><td>62.5</td><td>-1.64</td><td>36</td><td>-3.11</td></tr><tr><td valign="top">WA (<xref ref-type="bibr" rid="bib53">Nishimura et al., 2010</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>23</td><td>45.7</td><td>-2.91</td><td>40</td><td>-2.58</td></tr><tr><td valign="top">KE (<xref ref-type="bibr" rid="bib6">Avidan et al., 2011</xref>), (<xref ref-type="bibr" rid="bib75">Thomas et al., 2009</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>67</td><td>42.9</td><td>-3.12</td><td>40</td><td>-2.58</td></tr><tr><td valign="top">TD (<xref ref-type="bibr" rid="bib53">Nishimura et al., 2010</xref>), (<xref ref-type="bibr" rid="bib6">Avidan et al., 2011</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>38</td><td>46.4</td><td>-2.85</td><td>41</td><td>-2.45</td></tr><tr><td valign="top">MN (<xref ref-type="bibr" rid="bib53">Nishimura et al., 2010</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>50</td><td>60.7</td><td>-1.77</td><td>52</td><td>-1</td></tr><tr><td valign="top">BT (<xref ref-type="bibr" rid="bib6">Avidan et al., 2011</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>M</td><td>32</td><td>55.3</td><td>-2.18</td><td>58</td><td>-0.21</td></tr><tr><td>CP&#160;Mean&#160;&#177; s.d</td><td/><td>40.4&#177; 15.03</td><td>48.52 &#177; 18.72</td><td/><td>39.3 &#177; 9.41</td><td/></tr><tr><td>Control Mean &#177; s.d</td><td/><td>39.3 &#177; 13.4</td><td>91.57&#177;6.24</td><td/><td>58.28 &#177; 5.87</td><td/></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>The table shows the age and gender of participants and their performance (raw values and <italic>z</italic>-normalized scores relative to a large control group) on the famous faces questionnaire and CFMT. Note that 7 of the 10 CPs have participated in previous behavioral (3 CPs), and imaging (7 CPs) studies; additional behavioral measures for the CP individuals can be found in these references. Specific details regarding diagnostic and inclusion criteria can be found in the Materials and methods section and in the related studies. The average performance on the famous faces questionnaire and the CFMT of the controls who participated in the present study is also provided (t-test comparing performance across the CP and the controls, for famous faces questionnaire p&lt;0.0005 and CFMT p&lt;0.0005).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-25069-v2.xml">elife-25069-v2.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.25069.007</object-id><label>Table 1.</label><caption><p>CP behavioral scores ordered by severity as indicated by performance on the CFMT.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.25069.007">http://dx.doi.org/10.7554/eLife.25069.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Participant</th><th rowspan="2"><bold>Sex</bold></th><th rowspan="2">Age</th><th colspan="2">Famous faces questionnaire</th><th colspan="2" rowspan="2">CFMT (total)</th></tr><tr><th>% corr.</th><th>Z- score</th></tr></thead><tbody><tr><td>BL</td><td>F</td><td>18</td><td>20</td><td>-4.88</td><td>28</td><td>-4.15</td></tr><tr><td>BQ</td><td>F</td><td>29</td><td>16</td><td>-4.92</td><td>30</td><td>-3.89</td></tr><tr><td>KG</td><td>F</td><td>49</td><td>75</td><td>-0.69</td><td>33</td><td>-3.5</td></tr><tr><td valign="top">ON (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>48</td><td>60.7</td><td>-1.77</td><td>35</td><td>-3.23</td></tr><tr><td valign="top">MT (<xref ref-type="bibr" rid="bib53">Nishimura et al., 2010</xref>), (<xref ref-type="bibr" rid="bib6">Avidan et al., 2011</xref>), <break/>(<xref ref-type="bibr" rid="bib10">Behrmann and Avidan, 2005</xref>), (<xref ref-type="bibr" rid="bib75">Thomas et al., 2009</xref>), <break/>(<xref ref-type="bibr" rid="bib2">Avidan and Behrmann, 2008</xref>), <break/>(<xref ref-type="bibr" rid="bib40">Humphreys et al., 2007</xref>) , (<xref ref-type="bibr" rid="bib8">Behrmann et al., 2007</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>M</td><td>50</td><td>62.5</td><td>-1.64</td><td>36</td><td>-3.11</td></tr><tr><td valign="top">WA (<xref ref-type="bibr" rid="bib53">Nishimura et al., 2010</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>23</td><td>45.7</td><td>-2.91</td><td>40</td><td>-2.58</td></tr><tr><td valign="top">KE (<xref ref-type="bibr" rid="bib6">Avidan et al., 2011</xref>), (<xref ref-type="bibr" rid="bib75">Thomas et al., 2009</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>67</td><td>42.9</td><td>-3.12</td><td>40</td><td>-2.58</td></tr><tr><td valign="top">TD (<xref ref-type="bibr" rid="bib53">Nishimura et al., 2010</xref>), (<xref ref-type="bibr" rid="bib6">Avidan et al., 2011</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>38</td><td>46.4</td><td>-2.85</td><td>41</td><td>-2.45</td></tr><tr><td valign="top">MN (<xref ref-type="bibr" rid="bib53">Nishimura et al., 2010</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>50</td><td>60.7</td><td>-1.77</td><td>52</td><td>-1</td></tr><tr><td valign="top">BT (<xref ref-type="bibr" rid="bib6">Avidan et al., 2011</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>M</td><td>32</td><td>55.3</td><td>-2.18</td><td>58</td><td>-0.21</td></tr><tr><td>CP&#160;Mean&#160;&#177; s.d</td><td/><td>40.4&#177; 15.03</td><td>48.52 &#177; 18.72</td><td/><td>39.3 &#177; 9.41</td><td/></tr><tr><td>Control Mean &#177; s.d</td><td/><td>39.3 &#177; 13.4</td><td>91.57&#177;6.24</td><td/><td>58.28 &#177; 5.87</td><td/></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>The table shows the age and gender of participants and their performance (raw values and <italic>z</italic>-normalized scores relative to a large control group) on the famous faces questionnaire and CFMT. Note that 7 of the 10 CPs have participated in previous behavioral (3 CPs), and imaging (7 CPs) studies; additional behavioral measures for the CP individuals can be found in these references. Specific details regarding diagnostic and inclusion criteria can be found in the Materials and methods section and in the related studies. The average performance on the famous faces questionnaire and the CFMT of the controls who participated in the present study is also provided (t-test comparing performance across the CP and the controls, for famous faces questionnaire p&lt;0.0005 and CFMT p&lt;0.0005).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-25069-v3.xml">elife-25069-v3.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.25069.007</object-id><label>Table 1.</label><caption><p>CP behavioral scores ordered by severity as indicated by performance on the CFMT.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.25069.007">http://dx.doi.org/10.7554/eLife.25069.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Participant</th><th rowspan="2"><bold>Sex</bold></th><th rowspan="2">Age</th><th colspan="2">Famous faces questionnaire</th><th colspan="2" rowspan="2">CFMT (total)</th></tr><tr><th>% corr.</th><th>Z- score</th></tr></thead><tbody><tr><td>BL</td><td>F</td><td>18</td><td>20</td><td>-4.88</td><td>28</td><td>-4.15</td></tr><tr><td>BQ</td><td>F</td><td>29</td><td>16</td><td>-4.92</td><td>30</td><td>-3.89</td></tr><tr><td>KG</td><td>F</td><td>49</td><td>75</td><td>-0.69</td><td>33</td><td>-3.5</td></tr><tr><td valign="top">ON (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>48</td><td>60.7</td><td>-1.77</td><td>35</td><td>-3.23</td></tr><tr><td valign="top">MT (<xref ref-type="bibr" rid="bib53">Nishimura et al., 2010</xref>), (<xref ref-type="bibr" rid="bib6">Avidan et al., 2011</xref>), <break/>(<xref ref-type="bibr" rid="bib10">Behrmann and Avidan, 2005</xref>), (<xref ref-type="bibr" rid="bib75">Thomas et al., 2009</xref>), <break/>(<xref ref-type="bibr" rid="bib2">Avidan and Behrmann, 2008</xref>), <break/>(<xref ref-type="bibr" rid="bib40">Humphreys et al., 2007</xref>) , (<xref ref-type="bibr" rid="bib8">Behrmann et al., 2007</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>M</td><td>50</td><td>62.5</td><td>-1.64</td><td>36</td><td>-3.11</td></tr><tr><td valign="top">WA (<xref ref-type="bibr" rid="bib53">Nishimura et al., 2010</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>23</td><td>45.7</td><td>-2.91</td><td>40</td><td>-2.58</td></tr><tr><td valign="top">KE (<xref ref-type="bibr" rid="bib6">Avidan et al., 2011</xref>), (<xref ref-type="bibr" rid="bib75">Thomas et al., 2009</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>67</td><td>42.9</td><td>-3.12</td><td>40</td><td>-2.58</td></tr><tr><td valign="top">TD (<xref ref-type="bibr" rid="bib53">Nishimura et al., 2010</xref>), (<xref ref-type="bibr" rid="bib6">Avidan et al., 2011</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>38</td><td>46.4</td><td>-2.85</td><td>41</td><td>-2.45</td></tr><tr><td valign="top">MN (<xref ref-type="bibr" rid="bib53">Nishimura et al., 2010</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>F</td><td>50</td><td>60.7</td><td>-1.77</td><td>52</td><td>-1</td></tr><tr><td valign="top">BT (<xref ref-type="bibr" rid="bib6">Avidan et al., 2011</xref>), (<xref ref-type="bibr" rid="bib7">Avidan et al., 2014</xref>)</td><td>M</td><td>32</td><td>55.3</td><td>-2.18</td><td>58</td><td>-0.21</td></tr><tr><td>CP&#160;Mean&#160;&#177; s.d</td><td/><td>40.4&#177; 15.03</td><td>48.52 &#177; 18.72</td><td/><td>39.3 &#177; 9.41</td><td/></tr><tr><td>Control Mean &#177; s.d</td><td/><td>39.3 &#177; 13.4</td><td>91.57&#177;6.24</td><td/><td>58.28 &#177; 5.87</td><td/></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p>The table shows the age and gender of participants and their performance (raw values and <italic>z</italic>-normalized scores relative to a large control group) on the famous faces questionnaire and CFMT. Note that 7 of the 10 CPs have participated in previous behavioral (3 CPs), and imaging (7 CPs) studies; additional behavioral measures for the CP individuals can be found in these references. Specific details regarding diagnostic and inclusion criteria can be found in the Materials and methods section and in the related studies. The average performance on the famous faces questionnaire and the CFMT of the controls who participated in the present study is also provided (t-test comparing performance across the CP and the controls, for famous faces questionnaire p&lt;0.0005 and CFMT p&lt;0.0005).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-25078-v2.xml">elife-25078-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.25078.007</object-id><label>Table 1.</label><caption><p>Statistical properties of all KV analyzed. Table summarizing some of the cilia features collected from the 3D-CiliaMap for individual KV at 3-, 8- and 9&#8211;14- somite stage (SS).</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.25078.007">http://dx.doi.org/10.7554/eLife.25078.007</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">stage</th><th rowspan="2">KV <break/>number</th><th rowspan="2">N cilia</th><th rowspan="2">% <break/>immotile cilia</th><th colspan="2">Ellipsoid axis</th><th rowspan="2">Axis ratio <break/>a / b</th><th rowspan="2">Volume (pl)</th><th colspan="3">Average motile cilium</th><th rowspan="2">Ellipsoid <break/>fit RMS residue (&#181;m)</th><th rowspan="2">&#8486; <break/>(s<sup>-1</sup>)</th></tr><tr><th>a (&#181;m)</th><th>b (&#181;m)</th><th>r</th><th>&#952; (&#176;)</th><th>&#966; (&#176;)</th></tr></thead><tbody><tr><td rowspan="9"> <break/><break/> <break/><break/> <break/><break/> <break/><break/><bold>3-SS</bold></td><td valign="top">1</td><td valign="top">56</td><td valign="top">11%</td><td valign="top">27</td><td valign="top">12</td><td valign="top">2.3</td><td valign="top">35</td><td valign="top">0.8</td><td valign="top">37</td><td valign="top">6</td><td valign="top">2.07</td><td valign="top">0.423</td></tr><tr><td valign="top">2</td><td valign="top">23</td><td valign="top">39%</td><td valign="top">21</td><td valign="top">12</td><td valign="top">1.8</td><td valign="top">21</td><td valign="top">0.8</td><td valign="top">44</td><td valign="top">2</td><td valign="top">1.56</td><td valign="top">0.207</td></tr><tr><td valign="top">3</td><td valign="top">35</td><td valign="top">3%</td><td valign="top">23</td><td valign="top">12</td><td valign="top">1.9</td><td valign="top">27</td><td valign="top">0.8</td><td valign="top">42</td><td valign="top">-11</td><td valign="top">2.38</td><td valign="top">0.457</td></tr><tr><td valign="top">4</td><td valign="top">21</td><td valign="top">33%</td><td valign="top">18</td><td valign="top">11</td><td valign="top">1.6</td><td valign="top">15</td><td valign="top">0.8</td><td valign="top">22</td><td valign="top">12</td><td valign="top">2.14</td><td valign="top">0.350</td></tr><tr><td valign="top">5</td><td valign="top">19</td><td valign="top">100%</td><td valign="top">15</td><td valign="top">9</td><td valign="top">1.7</td><td valign="top">8</td><td valign="top">NA</td><td valign="top">NA</td><td valign="top">NA</td><td valign="top">1.5</td><td valign="top">0.000</td></tr><tr><td valign="top">6</td><td valign="top">24</td><td valign="top">54%</td><td valign="top">15</td><td valign="top">10</td><td valign="top">1.5</td><td valign="top">9</td><td valign="top">0.7</td><td valign="top">28</td><td valign="top">3</td><td valign="top">1.78</td><td valign="top">0.230</td></tr><tr><td valign="top">7</td><td valign="top">23</td><td valign="top">61%</td><td valign="top">14</td><td valign="top">12</td><td valign="top">1.2</td><td valign="top">10</td><td valign="top">0.8</td><td valign="top">18</td><td valign="top">-46</td><td valign="top">1.92</td><td valign="top">0.110</td></tr><tr><td valign="top">8</td><td valign="top">40</td><td valign="top">53%</td><td valign="top">18</td><td valign="top">11</td><td valign="top">1.6</td><td valign="top">16</td><td valign="top">0.7</td><td valign="top">8</td><td valign="top">22</td><td valign="top">2.05</td><td valign="top">0.034</td></tr><tr><td valign="top"><bold>mean &#177; SD</bold></td><td valign="top"><bold>30 &#177; 13</bold></td><td valign="top"><bold>44% &#177; 31%</bold></td><td valign="top"><bold>19 &#177; 5</bold></td><td valign="top"><bold>11 &#177; 1</bold></td><td valign="top"><bold>1.7 &#177; 0.3</bold></td><td valign="top"><bold>17 &#177; 10</bold></td><td valign="top"><bold>0.8 &#177; 0.1</bold></td><td valign="top"><bold>28 &#177; 13</bold></td><td valign="top"><bold>-2 &#177; 22</bold></td><td valign="top"><bold>1.9 &#177; 0.3</bold></td><td valign="top"><bold>0.226 &#177; 0.173</bold></td></tr><tr><td rowspan="7"> <break/><break/> <break/><break/> <break/><break/><bold>8-SS</bold></td><td valign="top">1</td><td valign="top">50</td><td valign="top">4%</td><td valign="top">29</td><td valign="top">18</td><td valign="top">1.6</td><td valign="top">64</td><td valign="top">0.8</td><td valign="top">37</td><td valign="top">-14</td><td valign="top">1.2</td><td valign="top">0.270</td></tr><tr><td valign="top">2</td><td valign="top">49</td><td valign="top">2%</td><td valign="top">22</td><td valign="top">8</td><td valign="top">2.8</td><td valign="top">15</td><td valign="top">0.8</td><td valign="top">35</td><td valign="top">6</td><td valign="top">1.1</td><td valign="top">0.595</td></tr><tr><td valign="top">3</td><td valign="top">43</td><td valign="top">0%</td><td valign="top">25</td><td valign="top">11</td><td valign="top">2.3</td><td valign="top">30</td><td valign="top">0.7</td><td valign="top">46</td><td valign="top">12</td><td valign="top">2.3</td><td valign="top">0.561</td></tr><tr><td valign="top">4</td><td valign="top">50</td><td valign="top">4%</td><td valign="top">27</td><td valign="top">12</td><td valign="top">2.3</td><td valign="top">39</td><td valign="top">0.7</td><td valign="top">41</td><td valign="top">16</td><td valign="top">2.5</td><td valign="top">0.329</td></tr><tr><td valign="top">5</td><td valign="top">71</td><td valign="top">0%</td><td valign="top">26</td><td valign="top">13</td><td valign="top">2.0</td><td valign="top">36</td><td valign="top">0.8</td><td valign="top">35</td><td valign="top">16</td><td valign="top">1.8</td><td valign="top">0.469</td></tr><tr><td valign="top">6</td><td valign="top">47</td><td valign="top">0%</td><td valign="top">30</td><td valign="top">22</td><td valign="top">1.4</td><td valign="top">85</td><td valign="top">0.7</td><td valign="top">21</td><td valign="top">39</td><td valign="top">1.7</td><td valign="top">0.110</td></tr><tr><td valign="top"><bold>mean &#177; SD</bold></td><td valign="top"><bold>52 &#177; 10</bold></td><td valign="top"><bold>2% &#177; 2%</bold></td><td valign="top"><bold>26 &#177; 3</bold></td><td valign="top"><bold>14 &#177; 5</bold></td><td valign="top"><bold>2.0 &#177; 0.5</bold></td><td valign="top"><bold>45 &#177; 25</bold></td><td valign="top"><bold>0.7 &#177; 0.1</bold></td><td valign="top"><bold>36 &#177; 8</bold></td><td valign="top"><bold>13 &#177; 17</bold></td><td valign="top"><bold>1.7 &#177; 0.5</bold></td><td valign="top"><bold>0.389 &#177; 0.186</bold></td></tr><tr><td rowspan="15"> <break/><break/> <break/><break/> <break/><break/> <break/><break/> <break/><break/> <break/><break/> <break/><break/><bold>9-14-SS</bold></td><td valign="top">1</td><td valign="top">89</td><td valign="top">16%</td><td valign="top">37</td><td valign="top">31</td><td valign="top">1.2</td><td valign="top">174</td><td valign="top">0.7</td><td valign="top">49</td><td valign="top">12</td><td valign="top">2.1</td><td valign="top">0.271</td></tr><tr><td valign="top">2</td><td valign="top">78</td><td valign="top">6%</td><td valign="top">27</td><td valign="top">15</td><td valign="top">1.8</td><td valign="top">47</td><td valign="top">0.7</td><td valign="top">46</td><td valign="top">18</td><td valign="top">2.0</td><td valign="top">0.600</td></tr><tr><td valign="top">3</td><td valign="top">80</td><td valign="top">3%</td><td valign="top">33</td><td valign="top">20</td><td valign="top">1.7</td><td valign="top">93</td><td valign="top">0.8</td><td valign="top">54</td><td valign="top">13</td><td valign="top">2.2</td><td valign="top">0.412</td></tr><tr><td valign="top">4</td><td valign="top">50</td><td valign="top">0%</td><td valign="top">33</td><td valign="top">20</td><td valign="top">1.7</td><td valign="top">91</td><td valign="top">0.7</td><td valign="top">47</td><td valign="top">14</td><td valign="top">2.6</td><td valign="top">0.290</td></tr><tr><td valign="top">5</td><td valign="top">78</td><td valign="top">6%</td><td valign="top">31</td><td valign="top">17</td><td valign="top">1.8</td><td valign="top">70</td><td valign="top">0.7</td><td valign="top">36</td><td valign="top">20</td><td valign="top">2.0</td><td valign="top">0.393</td></tr><tr><td valign="top">6</td><td valign="top">94</td><td valign="top">4%</td><td valign="top">36</td><td valign="top">21</td><td valign="top">1.7</td><td valign="top">111</td><td valign="top">0.8</td><td valign="top">52</td><td valign="top">10</td><td valign="top">1.7</td><td valign="top">0.424</td></tr><tr><td valign="top">7</td><td valign="top">62</td><td valign="top">11%</td><td valign="top">34</td><td valign="top">26</td><td valign="top">1.3</td><td valign="top">124</td><td valign="top">0.8</td><td valign="top">46</td><td valign="top">11</td><td valign="top">1.8</td><td valign="top">0.216</td></tr><tr><td valign="top">8</td><td valign="top">39</td><td valign="top">3%</td><td valign="top">24</td><td valign="top">20</td><td valign="top">1.2</td><td valign="top">48</td><td valign="top">0.7</td><td valign="top">54</td><td valign="top">36</td><td valign="top">4.4</td><td valign="top">0.378</td></tr><tr><td valign="top">9</td><td valign="top">35</td><td valign="top">0%</td><td valign="top">31</td><td valign="top">13</td><td valign="top">2.4</td><td valign="top">52</td><td valign="top">0.8</td><td valign="top">50</td><td valign="top">10</td><td valign="top">2.6</td><td valign="top">0.245</td></tr><tr><td valign="top">10</td><td valign="top">49</td><td valign="top">4%</td><td valign="top">32</td><td valign="top">17</td><td valign="top">1.9</td><td valign="top">74</td><td valign="top">0.8</td><td valign="top">53</td><td valign="top">2</td><td valign="top">2.3</td><td valign="top">0.275</td></tr><tr><td valign="top">11</td><td valign="top">50</td><td valign="top">2%</td><td valign="top">30</td><td valign="top">22</td><td valign="top">1.4</td><td valign="top">82</td><td valign="top">0.9</td><td valign="top">64</td><td valign="top">3</td><td valign="top">2.9</td><td valign="top">0.433</td></tr><tr><td valign="top">12</td><td valign="top">55</td><td valign="top">2%</td><td valign="top">30</td><td valign="top">23</td><td valign="top">1.3</td><td valign="top">85</td><td valign="top">0.8</td><td valign="top">61</td><td valign="top">11</td><td valign="top">2.6</td><td valign="top">0.388</td></tr><tr><td valign="top">13</td><td valign="top">49</td><td valign="top">4%</td><td valign="top">33</td><td valign="top">10</td><td valign="top">3.3</td><td valign="top">47</td><td valign="top">0.7</td><td valign="top">40</td><td valign="top">32</td><td valign="top">2.1</td><td valign="top">0.224</td></tr><tr><td valign="top">14</td><td valign="top">79</td><td valign="top">5%</td><td valign="top">27</td><td valign="top">19</td><td valign="top">1.4</td><td valign="top">56</td><td valign="top">0.7</td><td valign="top">51</td><td valign="top">22</td><td valign="top">1.8</td><td valign="top">0.521</td></tr><tr><td valign="top"><bold>mean &#177; SD</bold></td><td valign="top"><bold>63 &#177; 19</bold></td><td valign="top"><bold>5% &#177; 4%</bold></td><td valign="top"><bold>31 &#177; 4</bold></td><td valign="top"><bold>19 &#177; 5</bold></td><td valign="top"><bold>1.7 &#177; 0.6</bold></td><td valign="top"><bold>82 &#177; 36</bold></td><td valign="top"><bold>0.8 &#177; 0.05</bold></td><td valign="top"><bold>50 &#177; 7</bold></td><td valign="top"><bold>15 &#177; 10</bold></td><td valign="top"><bold>2.3 &#177; 0.7</bold></td><td valign="top"><bold>0.362 &#177; 0.114</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-25100-v1.xml">elife-25100-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl1" position="float"><object-id pub-id-type="doi">10.7554/eLife.25100.026</object-id><label>Table 1.</label><caption><p>Differences in amplification cost indicated by the extinction risk of populations with amplifications. Populations with amplifications of higher (strain <italic>B&#916;IS5I</italic>) or lower (strain <italic>E</italic>) expected cost of amplifications have a higher or lower risk of becoming extinct, respectively. n = initial number of replicate populations used for analysis (n = 285 includes replicate evolution experiments to increase statistical power), &#8216;Extinct&#8217; and &#8216;Rescued&#8217; = numbers of extinct and rescued populations with amplifications as indicated by (transiently) increased CFP fluorescence (see Materials and methods), sample odds ratio compared to strain <italic>B</italic>, p-values: 2 &#215; 2 Fisher&#8217;s exact test.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.25100.026">http://dx.doi.org/10.7554/eLife.25100.026</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">n</th><th rowspan="2">Strain</th><th colspan="2">Populations with (transiently) increased CFP fluorescence</th><th rowspan="2">Sample Odds Ratio</th><th rowspan="2"><italic>P-value</italic></th></tr><tr><th>Extinct</th><th>Rescued</th></tr></thead><tbody><tr><td>95</td><td style="author-callout-style-b8"><italic>B&#916;IS5I</italic></td><td style="author-callout-style-b8">12</td><td style="author-callout-style-b8">5</td><td style="author-callout-style-b8">10.1</td><td style="author-callout-style-b8">10<sup>&#8722;4</sup></td></tr><tr><td>95</td><td><italic>B</italic></td><td>18</td><td>76</td><td>1 (ref)</td><td>&#8211;</td></tr><tr><td>95</td><td style="author-callout-style-b8"><italic>E</italic></td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">95</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">10<sup>&#8722;6</sup></td></tr><tr><td colspan="6"/></tr><tr><td align="center">285</td><td><italic>A</italic></td><td>8</td><td>4</td><td>5.8</td><td>10<sup>&#8722;3</sup></td></tr><tr><td>285</td><td style="author-callout-style-b8"><italic>B</italic></td><td style="author-callout-style-b8">58</td><td style="author-callout-style-b8">168</td><td style="author-callout-style-b8">1 (ref)</td><td style="author-callout-style-b8">&#8211;</td></tr><tr><td>285</td><td><italic>C</italic></td><td>0</td><td>1</td><td>0</td><td>n.s.</td></tr><tr><td>285</td><td style="author-callout-style-b8"><italic>D</italic></td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">7</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">n.s.</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-25624-v2.xml">elife-25624-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.25624.026</object-id><label>Table 2.</label><caption><p>Minimum inhibitory concentrations (MICs) of cell wall targeting drugs for different <italic>Mycobacterium tuberculosis</italic> strains.</p><p>Source data file for the calculation of MIC values is available in <xref ref-type="supplementary-material" rid="SD4-data">Table 2&#8211;source data 1</xref>.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.25624.026">http://dx.doi.org/10.7554/eLife.25624.026</ext-link></p><p><supplementary-material id="SD4-data"><object-id pub-id-type="doi">10.7554/eLife.25624.027</object-id><label>Table 2&#8212;source data 1.</label><caption><title>Percentage growth inhibition values for&#160;<italic>Mtb</italic>,&#160;<italic>Mtb&#916;whiB4&#160;</italic>and&#160;<italic>whiB4-OE&#160;</italic>in presence of different drugs for calculation of minimum inhibitory concentration (MIC).</title><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.25624.027">http://dx.doi.org/10.7554/eLife.25624.027</ext-link></p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-25624-table2-data1-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><bold><italic>Drugs</italic></bold></th><th colspan="3" valign="top"><bold><italic>&#181;g/ml</italic></bold></th></tr><tr><th valign="top"><bold><italic>Mtb</italic></bold></th><th valign="top"><bold><italic>Mtb</italic>&#916;<italic>whib4B4</italic></bold></th><th valign="top"><bold><italic>whiB4-OE</italic></bold></th></tr></thead><tbody><tr><td valign="top">&#8195;Amoxicillin</td><td valign="top">80</td><td valign="top">&gt;160</td><td valign="top">40</td></tr><tr><td valign="top">&#8195;Ampicillin</td><td valign="top">500</td><td valign="top">4000</td><td valign="top">250</td></tr><tr><td valign="top">&#8195;Cloxacillin</td><td valign="top">400</td><td valign="top">800</td><td valign="top">200</td></tr><tr><td valign="top">&#8195;Carbenicillin</td><td valign="top">1024</td><td valign="top">4096</td><td valign="top">512</td></tr><tr><td valign="top">&#8195;Meropenem</td><td valign="top">5</td><td valign="top">20</td><td valign="top">2.5</td></tr><tr><td valign="top">&#8195;Penicillin</td><td valign="top">200</td><td valign="top">800</td><td valign="top">100</td></tr><tr><td valign="top">&#8195;Lysozyme</td><td valign="top">50</td><td valign="top">200</td><td valign="top">25</td></tr><tr><td valign="top">&#8195;Vancomycin</td><td valign="top">10</td><td valign="top">80</td><td valign="top">2.5</td></tr><tr><td valign="top">&#8195;Isoniazid</td><td valign="top">0.0625</td><td valign="top">0.0625</td><td valign="top">0.03125</td></tr><tr><td valign="top">&#8195;Rifampicin</td><td valign="top">0.0625</td><td valign="top">0.0625</td><td valign="top">0.0625</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-25923-v2.xml">elife-25923-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.25923.009</object-id><label>Table 2.</label><caption><title>Circumcision levels and commitment for MC, by segment.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Country</th><th rowspan="2">Segment</th><th colspan="2">All men in segment*</th><th colspan="2">Uncircumcised men in segment<sup>&#8224;</sup></th></tr><tr><th>Circumcised <break/>% (n)</th><th>Uncircumcised % (n)</th><th>Not committed <break/>% (n)</th><th>Committed <break/>% (n)</th></tr></thead><tbody><tr><td rowspan="6"><bold>Zimbabwe</bold></td><td><bold>Enthusiasts</bold></td><td>42.6 (211)</td><td>57.4 (284)</td><td>15.5 (44)</td><td>84.5 (240)</td></tr><tr><td><bold>Champions</bold></td><td>76.2 (269)</td><td>23.8 (84)</td><td>8.3 (7)</td><td>91.7 (77)</td></tr><tr><td><bold>Neophytes</bold></td><td>6.1 (17)</td><td>93.9 (260)</td><td>43.1 (112)</td><td>56.9 (148)</td></tr><tr><td><bold>Scared rejecters</bold></td><td>2.6 (6)</td><td>97.4 (228)</td><td>78.1 (178)</td><td>21.9 (50)</td></tr><tr><td><bold>Embarrassed rejecters</bold></td><td>32.7 (112)</td><td>67.3 (230)</td><td>56.1 (129)</td><td>43.9 (101)</td></tr><tr><td><bold>Highly resistant</bold></td><td>0.7 (2)</td><td>99.3 (298)</td><td>92.6 (276)</td><td>7.4 (22)</td></tr><tr><td rowspan="7"><bold>Zambia</bold></td><td><bold>Socially-supported believers</bold></td><td>56.1 (160)</td><td>43.9 (125)</td><td>20.0 (25)</td><td>80.0 (100)</td></tr><tr><td><bold>Self-reliant believers</bold></td><td>71.2 (272)</td><td>28.8 (110)</td><td>14.5 (16)</td><td>85.5 (94)</td></tr><tr><td><bold>Knowledgeable hesitant</bold></td><td>49.8 (119)</td><td>50.2 (120)</td><td>41.7 (50)</td><td>58.3 (70)</td></tr><tr><td><bold>Friends-driven hesitant</bold></td><td>14.1 (38)</td><td>85.9 (231)</td><td>29.0 (67)</td><td>71.0 (164)</td></tr><tr><td><bold>Scared rejecters</bold></td><td>9.7 (22)</td><td>90.3 (204)</td><td>62.7 (128)</td><td>37.3 (76)</td></tr><tr><td><bold>Indifferent resisters</bold></td><td>5.5 (19)</td><td>94.5 (325)</td><td>79.4 (258)</td><td>20.6 (67)</td></tr><tr><td><bold>Traditional believers</bold></td><td>70.6 (180)</td><td>29.4 (75)</td><td>16.0 (12)</td><td>84.0 (63)</td></tr><tr><td colspan="6" valign="top"/></tr></tbody></table><table-wrap-foot><fn><p>*No. of circumcised OR uncircumcised men in segment/no. total men in segment; <italic>Zambia, N&#160;=&#160;2000; Zimbabwe, N&#160;=&#160;2001</italic></p><p><sup>&#8224;</sup>uncircumcised committed OR not-committed men in segment/all uncircumcised men in segment; <italic>Zimbabwe, N&#160;=&#160;1384; Zambia, N&#160;=&#160;1189</italic></p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-26117-v1.xml">elife-26117-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app6table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.26117.072</object-id><label>Appendix 6&#8212;table 1.</label><caption><title>Response latency to dot motion analyses in <italic>Calliphora</italic> R1-R6s (Mean&#160;&#177;&#160;SD).</title><p>Intracellular recordings were performed at 19&#176;C. The tested moving dot (point-object) velocities were: 334.6, 167.3, 111.53, 83.65, 66.92, 55.77, 47.8, 41.83, 37.18 and 33.46 <sup>o</sup>/sec.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Animal</th><th rowspan="2">Flash response time-to-peak (ms)</th><th colspan="2">Peaks corresponding to the receptive field center</th><th colspan="2">Peaks corresponding to the light-point No.22</th></tr><tr><th>Lag-time <inline-formula><mml:math id="inf26"><mml:mi>b</mml:mi></mml:math></inline-formula> (ms)</th><th>Adj. R-Sqr</th><th>Lag-time <inline-formula><mml:math id="inf27"><mml:mrow><mml:mi>b</mml:mi><mml:mo>'</mml:mo></mml:mrow></mml:math></inline-formula> (ms)</th><th>Adj. R-Sqr</th></tr></thead><tbody><tr><td rowspan="2"><italic>Calliphora</italic></td><td rowspan="2">14.85&#160;&#177;&#160;0.78</td><td>14.6&#160;&#177;&#160;0.64</td><td>0.99985&#160;&#177;&#160;0.00012</td><td>13.9&#160;&#177;&#160;3.59</td><td>0.99985&#160;&#177;&#160;0.00017</td></tr><tr><td colspan="2">n&#160;=&#160;7</td><td colspan="2">n&#160;=&#160;5</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-26117-v1.xml">elife-26117-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app6table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.26117.073</object-id><label>Appendix 6&#8212;table 2.</label><caption><title>Response latency to dot motion analyses in <italic>Drosophila</italic> wild-type and hdc<sup>JK910</sup> R1-R6s (Mean&#160;&#177;&#160;SD).</title><p>Intracellular recordings were performed at 19&#176;C. The tested point-object velocities were: 818.4, 409.2, 272.8, 204.6, 163.68, 136.4, 116.91, 102.3, 90.93 and 81.84 <sup>o</sup>/s. Note, these statistics are collected from individual recordings, not from paired data.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Animal</th><th rowspan="2">Flash response time-to-peak (ms)</th><th colspan="2">Front-to-back</th><th colspan="2">Back-to-front</th></tr><tr><th>Lag-time <inline-formula><mml:math id="inf28"><mml:mi>b</mml:mi></mml:math></inline-formula> (ms)</th><th>Adj. R-Sqr</th><th>Lag-time <inline-formula><mml:math id="inf29"><mml:mi>b</mml:mi></mml:math></inline-formula> (ms)</th><th>Adj. R-Sqr</th></tr></thead><tbody><tr><td rowspan="2">Wild-type <italic>Drosophila</italic></td><td rowspan="2">23.81&#160;&#177;&#160;1.41</td><td>21.41&#160;&#177;&#160;4.5</td><td>0.99649&#160;&#177;&#160;0.0065</td><td>22.54&#160;&#177;&#160;4.15</td><td>0.99378&#160;&#177;&#160;0.007</td></tr><tr><td colspan="2">n&#160;=&#160;12</td><td colspan="2">n&#160;=&#160;5</td></tr><tr><td rowspan="2"><italic>hdc<sup>JK910</sup></italic></td><td rowspan="2">24.4&#160;&#177;&#160;1.08</td><td>21.82&#160;&#177;&#160;1.36</td><td>0.9992&#160;&#177;&#160;0.0008</td><td>23.79&#160;&#177;&#160;5.72</td><td>0.9978&#160;&#177;&#160;0.003</td></tr><tr><td colspan="4">n&#160;=&#160;3</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-26117-v2.xml">elife-26117-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app6table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.26117.072</object-id><label>Appendix 6&#8212;table 1.</label><caption><title>Response latency to dot motion analyses in <italic>Calliphora</italic> R1-R6s (Mean&#160;&#177;&#160;SD).</title><p>Intracellular recordings were performed at 19&#176;C. The tested moving dot (point-object) velocities were: 334.6, 167.3, 111.53, 83.65, 66.92, 55.77, 47.8, 41.83, 37.18 and 33.46 <sup>o</sup>/sec.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Animal</th><th rowspan="2">Flash response time-to-peak (ms)</th><th colspan="2">Peaks corresponding to the receptive field center</th><th colspan="2">Peaks corresponding to the light-point No.22</th></tr><tr><th>Lag-time <inline-formula><mml:math id="inf26"><mml:mi>b</mml:mi></mml:math></inline-formula> (ms)</th><th>Adj. R-Sqr</th><th>Lag-time <inline-formula><mml:math id="inf27"><mml:mrow><mml:mi>b</mml:mi><mml:mo>'</mml:mo></mml:mrow></mml:math></inline-formula> (ms)</th><th>Adj. R-Sqr</th></tr></thead><tbody><tr><td rowspan="2"><italic>Calliphora</italic></td><td rowspan="2">14.85&#160;&#177;&#160;0.78</td><td>14.6&#160;&#177;&#160;0.64</td><td>0.99985&#160;&#177;&#160;0.00012</td><td>13.9&#160;&#177;&#160;3.59</td><td>0.99985&#160;&#177;&#160;0.00017</td></tr><tr><td colspan="2">n&#160;=&#160;7</td><td colspan="2">n&#160;=&#160;5</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-26117-v2.xml">elife-26117-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app6table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.26117.073</object-id><label>Appendix 6&#8212;table 2.</label><caption><title>Response latency to dot motion analyses in <italic>Drosophila</italic> wild-type and hdc<sup>JK910</sup> R1-R6s (Mean&#160;&#177;&#160;SD).</title><p>Intracellular recordings were performed at 19&#176;C. The tested point-object velocities were: 818.4, 409.2, 272.8, 204.6, 163.68, 136.4, 116.91, 102.3, 90.93 and 81.84 <sup>o</sup>/s. Note, these statistics are collected from individual recordings, not from paired data.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Animal</th><th rowspan="2">Flash response time-to-peak (ms)</th><th colspan="2">Front-to-back</th><th colspan="2">Back-to-front</th></tr><tr><th>Lag-time <inline-formula><mml:math id="inf28"><mml:mi>b</mml:mi></mml:math></inline-formula> (ms)</th><th>Adj. R-Sqr</th><th>Lag-time <inline-formula><mml:math id="inf29"><mml:mi>b</mml:mi></mml:math></inline-formula> (ms)</th><th>Adj. R-Sqr</th></tr></thead><tbody><tr><td rowspan="2">Wild-type <italic>Drosophila</italic></td><td rowspan="2">23.81&#160;&#177;&#160;1.41</td><td>21.41&#160;&#177;&#160;4.5</td><td>0.99649&#160;&#177;&#160;0.0065</td><td>22.54&#160;&#177;&#160;4.15</td><td>0.99378&#160;&#177;&#160;0.007</td></tr><tr><td colspan="2">n&#160;=&#160;12</td><td colspan="2">n&#160;=&#160;5</td></tr><tr><td rowspan="2"><italic>hdc<sup>JK910</sup></italic></td><td rowspan="2">24.4&#160;&#177;&#160;1.08</td><td>21.82&#160;&#177;&#160;1.36</td><td>0.9992&#160;&#177;&#160;0.0008</td><td>23.79&#160;&#177;&#160;5.72</td><td>0.9978&#160;&#177;&#160;0.003</td></tr><tr><td colspan="4">n&#160;=&#160;3</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-26487-v2.xml">elife-26487-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.26487.020</object-id><label>Table 1.</label><caption><title>Dimeric and trimeric dUTPases induce PICIs from other species encoding SaPIbov1-like Stl repressors<sup>a</sup>.</title><p><supplementary-material id="table1sdata1"><object-id pub-id-type="doi">10.7554/eLife.26487.021</object-id><label>Table 1&#8212;source data 1.</label><caption><title>&#946;-lactamase assay data and statistical analysis for the SaPIbov1 Stl homologues.</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-26487-table1-data1-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"/><th colspan="4" valign="top">Phage Duts</th></tr><tr><th colspan="2" valign="top">Dimeric</th><th colspan="2" valign="top">Trimeric</th></tr><tr><th valign="top"> <bold>&#981;NM1</bold> <break/><italic>S. aureus</italic></th><th><bold>&#981;O11</bold> <break/><italic>S. aureus</italic></th><th><bold>&#981;IPLA6</bold> <break/><italic>S. epidermidis</italic></th><th><bold>&#981;11</bold> <break/><italic>S. aureus</italic></th></tr></thead><tbody><tr><td><bold>PICIs</bold></td><td/><td/><td/><td/></tr><tr><td>SaPIbov1</td><td>++</td><td>++</td><td>++</td><td>+++</td></tr><tr><td>ShoCI794_SEPI</td><td>++</td><td>++</td><td>++</td><td>+++</td></tr><tr><td>ShaCI51-48</td><td>+</td><td>+</td><td>+</td><td>+</td></tr></tbody></table><table-wrap-foot><fn><p><sup>a</sup>Results are from 5 independent &#946;-lactamase/Nitrocefin assay experiments, using the dual plasmid system described in the text. Levels of induction were based on the calculated units/ml with the following ranges: (-): no induction,&#160;&lt;1 Units/ml; (+): low induction, 1&#8211;5 Units/ml; (++): moderate induction, 5&#8211;10 Units/ml; (+++): high induction,&#160;&gt;10 Units/ml. A 2-way ANOVA with Sidak's multiple comparisons test was performed to compare mean differences within rows. The differences observed among the +,++or&#160;+++ samples, compared with the controls or the - samples, were in all cases statistically significant (p&lt;0.05).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-26487-v2.xml">elife-26487-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.26487.022</object-id><label>Table 2.</label><caption><title>Unrelated SSAPs differentially induce PICIs from other species encoding SaPI2-like Stl repressors<sup>a</sup>.</title><p><supplementary-material id="table2sdata1"><object-id pub-id-type="doi">10.7554/eLife.26487.023</object-id><label>Table 2&#8212;source data 1.</label><caption><title>&#946;-lactamase assay data and statistical analysis for the SaPI2 Stl homologues.</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-26487-table2-data1-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th valign="top"/><th valign="top"/><th colspan="3" valign="top">SSAPs</th><th/></tr><tr><th valign="top"><bold>Sak</bold> (80&#945;) <break/><italic>S. aureus</italic></th><th valign="top"><bold>Chimera</bold> (80&#945;) <break/><italic>S. aureus</italic></th><th valign="top"><bold>Sak</bold> (PH15) <break/><italic>S. epidermidis</italic></th><th><bold>Sak4</bold> (52A) <break/><italic>S. aureus</italic></th><th><bold>Erf</bold> (SLT) <break/><italic>S. aureus</italic></th><th><bold>Red&#946;</bold> (N315) <break/><italic>S. aureus</italic></th></tr></thead><tbody><tr><td><bold>PICIs</bold></td><td/><td valign="top"/><td valign="top"/><td/><td/><td/></tr><tr><td>SaPI2</td><td>++</td><td valign="top">-</td><td>++</td><td>+++</td><td>++</td><td>+</td></tr><tr><td>SeCINIHLM095</td><td>++</td><td valign="top">-</td><td>++</td><td>-</td><td>++</td><td>-</td></tr><tr><td>ShaCI137133</td><td>++</td><td valign="top">-</td><td>++</td><td>-</td><td>-</td><td>-</td></tr><tr><td>ScCIUMC-CNS990</td><td>-</td><td valign="top">-</td><td>-</td><td>+++</td><td>-</td><td>-</td></tr></tbody></table><table-wrap-foot><fn><p><sup>a</sup>Results are from 3 independent &#946;-lactamase/Nitrocefin assay experiments, using the dual plasmid system described in the text. Levels of induction were based on the calculated units/ml with the following ranges: (-): no induction,&#160;&lt;1 Units/ml; (+): low induction, 1&#8211;5 Units/ml; (++): moderate induction, 5&#8211;10 Units/ml; (+++): high induction,&gt;10 Units/ml. A 2-way ANOVA with Sidak's multiple comparisons test was performed to compare mean differences within rows. The differences observed among the +,++or&#160;+++ samples, compared with the controls or the - samples, were in all cases statistically significant (p&lt;0.05).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-26526-v2.xml">elife-26526-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.26526.010</object-id><label>Table 1.</label><caption><title>Populations and lifetimes of hydrogen-bonded turns.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><bold>Turn</bold></th><th colspan="2" valign="top"><bold>Population (%)</bold></th><th colspan="2" valign="top"><bold>Lifetime (ns)</bold></th></tr><tr><th valign="top"><bold>SC</bold></th><th valign="top"><bold>MC</bold></th><th valign="top"><bold>SC</bold></th><th valign="top"><bold>MC</bold></th></tr></thead><tbody><tr><td valign="top">VPGV<sup>*</sup></td><td valign="top">18&#160;&#177;&#160;1</td><td valign="top">16&#160;&#177;&#160;2</td><td valign="top">0.18&#160;&#177;&#160;0.02</td><td valign="top">0.19&#160;&#177;&#160;0.01</td></tr><tr><td valign="top">VPGVG</td><td valign="top">5.9&#160;&#177;&#160;0.9</td><td valign="top">4.6&#160;&#177;&#160;0.7</td><td valign="top">0.9&#160;&#177;&#160;0.3</td><td valign="top">0.8&#160;&#177;&#160;0.1</td></tr><tr><td valign="top">PGV</td><td valign="top">1.5&#160;&#177;&#160;0.1</td><td valign="top">2.0&#160;&#177;&#160;0.2</td><td valign="top">0.008&#160;&#177;&#160;0.002</td><td valign="top">0.009&#160;&#177;&#160;0.001</td></tr><tr><td valign="top">PGVG</td><td valign="top">2.3&#160;&#177;&#160;0.2</td><td valign="top">2.9&#160;&#177;&#160;0.3</td><td valign="top">0.30&#160;&#177;&#160;0.05</td><td valign="top">0.39&#160;&#177;&#160;0.03</td></tr><tr><td valign="top">GVGV</td><td valign="top">20&#160;&#177;&#160;3</td><td valign="top">17&#160;&#177;&#160;2</td><td valign="top">0.15&#160;&#177;&#160;0.02</td><td valign="top">0.24&#160;&#177;&#160;0.07</td></tr></tbody></table><table-wrap-foot><fn><p><sup>*</sup>A hydrogen-bonded turn refers to a hydrogen bond between the first and last residue in the sequences shown. For example, the VPGV turn has a hydrogen bond between valine 1 and valine 4.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-26653-v2.xml">elife-26653-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.26653.009</object-id><label>Table 1.</label><caption><title>Regions of interest used in this study and their BOLD activity during the gambling task.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Brain regions</th><th rowspan="2" valign="top">X</th><th rowspan="2" valign="top">Y</th><th rowspan="2" valign="top">Z</th><th colspan="3" valign="top">Brain Activity (T-score)</th></tr><tr><th valign="top">&#8195;Reward</th><th valign="top">Punishment</th><th valign="top">RPE</th></tr></thead><tbody><tr><td valign="top">SN subdivisions</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">vSN</td><td valign="top"><inline-formula><mml:math id="inf4"><mml:mo>&#177;</mml:mo></mml:math></inline-formula>9</td><td valign="top">&#8722;13</td><td valign="top">&#8722;12</td><td valign="top">11.12 **</td><td valign="top">8.63 **</td><td valign="top">1.85</td></tr><tr><td valign="top">medial SNc</td><td valign="top"><inline-formula><mml:math id="inf5"><mml:mo>&#177;</mml:mo></mml:math></inline-formula>8</td><td valign="top">&#8722;16</td><td valign="top">&#8722;12</td><td valign="top">17.45 **</td><td valign="top">12.48 **</td><td valign="top">3.96 **</td></tr><tr><td valign="top">lateral SNc</td><td valign="top"><inline-formula><mml:math id="inf6"><mml:mo>&#177;</mml:mo></mml:math></inline-formula>12</td><td valign="top">&#8722;17</td><td valign="top">-9</td><td valign="top">9.72 **</td><td valign="top">8.87 **</td><td valign="top">0.93</td></tr><tr><td valign="top">Ventral Striatum</td><td valign="top"><inline-formula><mml:math id="inf7"><mml:mo>&#177;</mml:mo></mml:math></inline-formula>12</td><td valign="top">15</td><td valign="top">-6</td><td valign="top">8.91 **</td><td valign="top">- 9.33 **</td><td valign="top">16.25 **</td></tr><tr><td valign="top">vmPFC</td><td valign="top"><inline-formula><mml:math id="inf8"><mml:mo>&#177;</mml:mo></mml:math></inline-formula>6</td><td valign="top">45</td><td valign="top">-9</td><td valign="top">- 26.72 **</td><td valign="top">- 32.37 **</td><td valign="top">8.75 **</td></tr><tr><td valign="top">Anterior insula</td><td valign="top"><inline-formula><mml:math id="inf9"><mml:mo>&#177;</mml:mo></mml:math></inline-formula>32</td><td valign="top">22</td><td valign="top">-6</td><td valign="top">35.54 **</td><td valign="top">36.62 **</td><td valign="top">1.07</td></tr><tr><td valign="top">dACC</td><td valign="top"><inline-formula><mml:math id="inf10"><mml:mo>&#177;</mml:mo></mml:math></inline-formula>4</td><td valign="top">40</td><td valign="top">24</td><td valign="top">8.43 **</td><td valign="top">7.64 **</td><td valign="top">1.37</td></tr></tbody></table><table-wrap-foot><fn><p>Notes: **p-value&lt;0.01; *p-value&lt;0.05 with FDR correction</p><p>SN: substantia nigra; vSN: ventral subregion of SN; SNc: SN pars compacta; vmPFC: ventral medial prefrontal cortex; dACC: dorsal anterior cingulate cortex;&#160;RPE: reward prediction error</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-26973-v2.xml">elife-26973-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.26973.009</object-id><label>Table 1.</label><caption><title>Response strength during episodes of singing.</title><p>Standardized response strength (mean &#177; s.e.m.) for <sc>core</sc> and <sc>shell</sc> neurons in LMAN that showed significant excitation or suppression during song production compared with quiet baseline periods (see Materials and methods).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="2"><sc>CORE</sc></th><th colspan="2"><sc>SHELL</sc></th></tr><tr><th>Fraction</th><th>Response strength</th><th>Fraction</th><th>Response strength</th></tr></thead><tbody><tr><td><bold>Excited</bold></td><td>0.72 (66/92)</td><td>7.06 &#177; 0.71</td><td>0.65 (66/102)</td><td>7.28 &#177; 0.44</td></tr><tr><td><bold>Suppressed</bold></td><td>0.28 (26/92)</td><td>&#8722;7.32 &#177; 1.15</td><td>0.35 (36/102)</td><td>&#8722;5.82 &#177; 0.45</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27038-v2.xml">elife-27038-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.27038.020</object-id><label>Table 1.</label><caption><title>Model Parameters.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th rowspan="2">Parameter</th><th rowspan="2">Unit</th><th rowspan="2">Description</th><th colspan="2">Acceptable range</th><th rowspan="2">Default</th></tr><tr><th>Max.</th><th>Min.</th></tr></thead><tbody><tr><td rowspan="7">BOR</td><td><inline-formula><mml:math id="inf152"><mml:mrow><mml:mi>&#1013;</mml:mi><mml:msub><mml:mi/><mml:mi>B</mml:mi></mml:msub></mml:mrow></mml:math></inline-formula></td><td>-</td><td>Time constant for transporter regulation</td><td>-</td><td>-</td><td>1</td></tr><tr><td><inline-formula><mml:math id="inf153"><mml:mrow><mml:mi>&#945;</mml:mi><mml:msub><mml:mi/><mml:mi>B</mml:mi></mml:msub></mml:mrow></mml:math></inline-formula></td><td>&#956;m&#8201;s<sup>-2</sup></td><td>Production rate of transporter activity</td><td><inline-formula><mml:math id="inf154"><mml:mrow><mml:mn>4.9</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mn>10</mml:mn><mml:mn>2</mml:mn></mml:msup></mml:mrow></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf155"><mml:mrow><mml:mn>3.7</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mn>10</mml:mn><mml:mrow><mml:mo>-</mml:mo><mml:mn>9</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf156"><mml:mrow><mml:mn>2.0</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mn>10</mml:mn><mml:mrow><mml:mo>-</mml:mo><mml:mn>1</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></td></tr><tr><td><inline-formula><mml:math id="inf157"><mml:mrow><mml:mi>&#958;</mml:mi><mml:msub><mml:mi/><mml:mi>B</mml:mi></mml:msub></mml:mrow></mml:math></inline-formula></td><td>s<sup>-1</sup></td><td>Basal degradation rate</td><td/><td><inline-formula><mml:math id="inf158"><mml:mrow><mml:mn>1.6</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mn>10</mml:mn><mml:mrow><mml:mo>-</mml:mo><mml:mn>6</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf159"><mml:mrow><mml:mn>7.6</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mn>10</mml:mn><mml:mrow><mml:mo>-</mml:mo><mml:mn>2</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></td></tr><tr><td><inline-formula><mml:math id="inf160"><mml:msub><mml:mi>k</mml:mi><mml:mi>B</mml:mi></mml:msub></mml:math></inline-formula></td><td>&#956;M</td><td>Boron concentration for half-maximum in Hill&#8217;s function</td><td>1000</td><td>1</td><td>600</td></tr><tr><td><inline-formula><mml:math id="inf161"><mml:msub><mml:mi>d</mml:mi><mml:mi>B</mml:mi></mml:msub></mml:math></inline-formula></td><td>-</td><td>Amplitude of increased degradation rate by boron</td><td>100</td><td>0</td><td>50</td></tr><tr><td><inline-formula><mml:math id="inf162"><mml:msub><mml:mi>n</mml:mi><mml:mi>B</mml:mi></mml:msub></mml:math></inline-formula></td><td>-</td><td>Hill&#8217;s coefficient</td><td>-</td><td>-</td><td>2</td></tr><tr><td><inline-formula><mml:math id="inf163"><mml:msub><mml:mi>c</mml:mi><mml:mi>B</mml:mi></mml:msub></mml:math></inline-formula></td><td>&#956;M</td><td>Boron concentration at which the flux reaches its half-maximum value</td><td><inline-formula><mml:math id="inf164"><mml:mi mathvariant="normal">&#8734;</mml:mi></mml:math></inline-formula></td><td>500</td><td>1000</td></tr><tr><td rowspan="5">NIP</td><td><inline-formula><mml:math id="inf165"><mml:mrow><mml:mi>&#1013;</mml:mi><mml:msub><mml:mi/><mml:mi>N</mml:mi></mml:msub></mml:mrow></mml:math></inline-formula></td><td>-</td><td>Time constant for transporter regulation</td><td>-</td><td>-</td><td>1</td></tr><tr><td><inline-formula><mml:math id="inf166"><mml:mrow><mml:mi>&#945;</mml:mi><mml:msub><mml:mi/><mml:mi>N</mml:mi></mml:msub></mml:mrow></mml:math></inline-formula></td><td>&#956;m&#8201;s<sup>-2</sup></td><td>Production rate of transporter activity</td><td><inline-formula><mml:math id="inf167"><mml:mrow><mml:mn>4.9</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mn>10</mml:mn><mml:mn>2</mml:mn></mml:msup></mml:mrow></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf168"><mml:mrow><mml:mn>3.7</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mn>10</mml:mn><mml:mrow><mml:mo>-</mml:mo><mml:mn>9</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf169"><mml:mrow><mml:mn>2.0</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mn>10</mml:mn><mml:mrow><mml:mo>-</mml:mo><mml:mn>1</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></td></tr><tr><td><inline-formula><mml:math id="inf170"><mml:mrow><mml:mi>&#958;</mml:mi><mml:msub><mml:mi/><mml:mi>N</mml:mi></mml:msub></mml:mrow></mml:math></inline-formula></td><td>s<sup>-1</sup></td><td>Basal degradation rate</td><td/><td><inline-formula><mml:math id="inf171"><mml:mrow><mml:mn>1.6</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mn>10</mml:mn><mml:mrow><mml:mo>-</mml:mo><mml:mn>6</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf172"><mml:mrow><mml:mn>7.6</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mn>10</mml:mn><mml:mrow><mml:mo>-</mml:mo><mml:mn>2</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></td></tr><tr><td><inline-formula><mml:math id="inf173"><mml:msub><mml:mi>k</mml:mi><mml:mi>N</mml:mi></mml:msub></mml:math></inline-formula></td><td>&#956;M</td><td>Boron concentration for half-maximum in Hill&#8217;s function</td><td>1000</td><td>1</td><td>20</td></tr><tr><td><inline-formula><mml:math id="inf174"><mml:msub><mml:mi>n</mml:mi><mml:mi>N</mml:mi></mml:msub></mml:math></inline-formula></td><td>-</td><td>Hill&#8217;s coefficient</td><td>-</td><td>-</td><td>2</td></tr><tr><td rowspan="3">Cell size</td><td><inline-formula><mml:math id="inf175"><mml:mrow><mml:mi>l</mml:mi><mml:mo>&#8290;</mml:mo><mml:mi>c</mml:mi></mml:mrow></mml:math></inline-formula></td><td>&#956;m</td><td>Cell width</td><td>20</td><td>5</td><td>10</td></tr><tr><td><inline-formula><mml:math id="inf176"><mml:mrow><mml:mi>l</mml:mi><mml:mo>&#8290;</mml:mo><mml:mi>w</mml:mi></mml:mrow></mml:math></inline-formula></td><td>&#956;m</td><td>Cell wall width</td><td>2</td><td>0.2</td><td>0.5</td></tr><tr><td><inline-formula><mml:math id="inf177"><mml:mrow><mml:mi>h</mml:mi><mml:mo>&#8290;</mml:mo><mml:mi>c</mml:mi></mml:mrow></mml:math></inline-formula></td><td>&#956;m</td><td>Cell height</td><td>150</td><td>5</td><td>20</td></tr><tr><td rowspan="3">Other</td><td><inline-formula><mml:math id="inf178"><mml:mi>p</mml:mi></mml:math></inline-formula></td><td>&#956;m&#8201;s<sup>-1</sup></td><td>Membrane background permeability of boron</td><td><inline-formula><mml:math id="inf179"><mml:mrow><mml:mn>8</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mn>10</mml:mn><mml:mrow><mml:mo>-</mml:mo><mml:mn>2</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf180"><mml:mrow><mml:mn>2.3</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mn>10</mml:mn><mml:mrow><mml:mo>-</mml:mo><mml:mn>3</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></td><td><inline-formula><mml:math id="inf181"><mml:mrow><mml:mn>3</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mn>10</mml:mn><mml:mrow><mml:mo>-</mml:mo><mml:mn>2</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></td></tr><tr><td><inline-formula><mml:math id="inf182"><mml:mi>a</mml:mi></mml:math></inline-formula></td><td>&#956;m&#8201;s<sup>-1</sup></td><td>Xylem loading rate (in the last cell)</td><td>500</td><td>0</td><td>0.5</td></tr><tr><td><inline-formula><mml:math id="inf183"><mml:msub><mml:mi>c</mml:mi><mml:mn>0</mml:mn></mml:msub></mml:math></inline-formula></td><td>&#956;m</td><td>Boron concentration in medium</td><td>5000</td><td>0</td><td>300</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27167-v2.xml">elife-27167-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.27167.013</object-id><label>Table 2.</label><caption><title>Average fitness increments of populations in home and away environments, in percent.</title><p>Numbers in parentheses denote 95% confidence intervals.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th align="center" colspan="2"><italic>Kre33</italic> -RM</th><th align="center" colspan="2"><italic>Kre33</italic> -BY</th></tr><tr><th align="center">Evolved at OT</th><th align="center">Evolved at HT</th><th align="center">Evolved at OT</th><th align="center">Evolved at HT</th></tr></thead><tbody><tr><td align="center">Fitnessat OT, %</td><td align="center">8.33(8.27, 8.38)</td><td align="center">8.86(8.81, 8.92)</td><td align="center">5.01(4.94, 5.07)</td><td align="center">4.834.76, 4.90</td></tr><tr><td align="center">Fitnessat HT, %</td><td align="center">9.57(9.42, 9.77)</td><td align="center">20.33(20.18, 20.53)</td><td align="center">6.25(6.12, 6.38)</td><td align="center">11.51(11.38, 11.64)</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27441-v1.xml">elife-27441-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.27441.011</object-id><label>Table 1.</label><caption><title>Fractions and molecular properties of diffusional particles of Syt1<sup>C2AB</sup> generated various conditions as measured by fluorescence correlation spectroscopy.</title><p>Averages and standard errors from three independent trials are shown.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Proteins</th><th rowspan="2">Total protein conc. (nM)</th><th rowspan="2">Mg<sup>2+</sup> + polyphosphate</th><th rowspan="2">Readout channel</th><th colspan="3">Component 1</th><th colspan="3">Component 2</th></tr><tr><th>Fraction (%)</th><th>Diffusion time (&#181;s)</th><th>Diffusion coefficient (&#956;m<sup>2</sup>/s)</th><th>Fraction (%)</th><th>Diffusion time (&#181;s)</th><th>Diffusion coefficient (&#956;m<sup>2</sup>/s)</th></tr></thead><tbody><tr><td>Syt1-Alexa488</td><td>~30</td><td>No</td><td>Alexa488</td><td>100</td><td>620&#160;&#177;&#160;35</td><td>70&#160;&#177;&#160;4</td><td>n/a</td><td>n/a</td><td>n/a</td></tr><tr><td>Syt1-Alexa647</td><td>~30</td><td>No</td><td>Alexa647</td><td>100</td><td>616&#160;&#177;&#160;22</td><td>70&#160;&#177;&#160;3</td><td>n/a</td><td>n/a</td><td>n/a</td></tr><tr><td rowspan="2">Syt1-Alexa488&#160;+&#160;Syt1-Alexa647&#160;+&#160;unlabeled Syt1</td><td rowspan="2">~5000</td><td rowspan="2">No</td><td>Alexa488</td><td>100</td><td>583&#160;&#177;&#160;20</td><td>74&#160;&#177;&#160;8</td><td>n/a</td><td>n/a</td><td>n/a</td></tr><tr><td>Alexa647</td><td>100</td><td>577&#160;&#177;&#160;20</td><td>75&#160;&#177;&#160;8</td><td>n/a</td><td>n/a</td><td>n/a</td></tr><tr><td rowspan="2">Syt1-Alexa488&#160;+&#160;Syt1-Alexa647&#160;+&#160;unlabeled Syt1</td><td rowspan="2">~5000</td><td rowspan="2">Yes</td><td>Alexa488</td><td>77&#160;&#177;&#160;2</td><td>527&#160;&#177;&#160;20</td><td>82&#160;&#177;&#160;4</td><td>23&#160;&#177;&#160;2</td><td>4168&#160;&#177;&#160;194</td><td>10&#160;&#177;&#160;1</td></tr><tr><td>Alexa647</td><td>76&#160;&#177;&#160;2</td><td>567&#160;&#177;&#160;26</td><td>77&#160;&#177;&#160;4</td><td>24&#160;&#177;&#160;2</td><td>3862&#160;&#177;&#160;196</td><td>11&#160;&#177;&#160;1</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27451-v2.xml">elife-27451-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.27451.009</object-id><label>Table 2.</label><caption><title>Copy number data.</title><p>Mean average and SD of copy number in pool and foci in each compartment.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"/><th colspan="4" valign="top"><bold>Mig1-GFP</bold></th><th colspan="4" valign="top"><bold>Msn2-GFP</bold></th></tr><tr><th colspan="2" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="2" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th><th colspan="2" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="2" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th></tr><tr><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th></tr></thead><tbody><tr><td valign="top"><bold>Cytoplasmic Pool</bold></td><td valign="top">509</td><td valign="top">274</td><td valign="top">949</td><td valign="top">394</td><td valign="top">1422</td><td valign="top">977</td><td valign="top">2487</td><td valign="top">1360</td></tr><tr><td valign="top"><bold>Nuclear Pool</bold></td><td valign="top">77</td><td valign="top">101</td><td valign="top">140</td><td valign="top">97</td><td valign="top">551</td><td valign="top">608</td><td valign="top">1692</td><td valign="top">1221</td></tr><tr><td valign="top"><bold>Total Pool</bold></td><td valign="top">586</td><td valign="top">336</td><td valign="top">1088</td><td valign="top">392</td><td valign="top">1973</td><td valign="top">1585</td><td valign="top">4179</td><td valign="top">2581</td></tr><tr><td valign="top"><bold>Cytoplasmic Spots</bold></td><td valign="top">57</td><td valign="top">79</td><td valign="top">311</td><td valign="top">212</td><td valign="top">333</td><td valign="top">196</td><td valign="top">776</td><td valign="top">635</td></tr><tr><td valign="top"><bold>Nuclear Spots</bold></td><td valign="top">190</td><td valign="top">99</td><td valign="top">35</td><td valign="top">63</td><td valign="top">81</td><td valign="top">138</td><td valign="top">320</td><td valign="top">269</td></tr><tr><td valign="top"><bold>Total Spots</bold></td><td valign="top">246</td><td valign="top">100</td><td valign="top">345</td><td valign="top">203</td><td valign="top">414</td><td valign="top">334</td><td valign="top">1096</td><td valign="top">904</td></tr><tr><td valign="top"><bold>Total Cytoplasm</bold></td><td valign="top">580</td><td valign="top">276</td><td valign="top">1156</td><td valign="top">399</td><td valign="top">1755</td><td valign="top">1173</td><td valign="top">3263</td><td valign="top">1995</td></tr><tr><td valign="top"><bold>Total Nuclear</bold></td><td valign="top">226</td><td valign="top">155</td><td valign="top">176</td><td valign="top">124</td><td valign="top">632</td><td valign="top">746</td><td valign="top">2012</td><td valign="top">1490</td></tr><tr><td valign="top"><bold>Total Cell</bold></td><td valign="top">806</td><td valign="top">353</td><td valign="top">1331</td><td valign="top">352</td><td valign="top">2387</td><td valign="top">1919</td><td valign="top">5274</td><td valign="top">3485</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27451-v2.xml">elife-27451-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.27451.010</object-id><label>Table 3.</label><caption><title>Foci tracking data.</title><p>Mean average, SD and mean number detected per cell (N) of stoichiometry values (molecules), and microscopic diffusion coefficients <italic>D</italic> in each compartment detected within the depth of field.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"/><th colspan="6" valign="top"><bold>Mig1-GFP</bold></th><th colspan="6" valign="top"><bold>Msn2-GFP</bold></th></tr><tr><th colspan="3" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="3" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th><th colspan="3" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="3" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th></tr><tr><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th></tr></thead><tbody><tr><td valign="top"><bold>Stoichiometry of Nuclear Spots</bold></td><td valign="top">19.0</td><td valign="top">16.2</td><td valign="top">7.2</td><td valign="top">8.5</td><td valign="top">4.8</td><td valign="top">5.8</td><td valign="top">34.5</td><td valign="top">26.6</td><td valign="top">3.5</td><td valign="top">46.5</td><td valign="top">31.6</td><td valign="top">4.7</td></tr><tr><td valign="top"><bold>Diffusion Constant of Nuclear Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">0.8</td><td valign="top">0.8</td><td valign="top">7.2</td><td valign="top">1.3</td><td valign="top">1.5</td><td valign="top">5.8</td><td valign="top">0.7</td><td valign="top">0.9</td><td valign="top">3.5</td><td valign="top">0.9</td><td valign="top">0.9</td><td valign="top">4.7</td></tr><tr><td valign="top"><bold>Stoichiometry of Trans-Nuclear Spots</bold></td><td valign="top">10.6</td><td valign="top">10.2</td><td valign="top">1.0</td><td valign="top">8.7</td><td valign="top">5.3</td><td valign="top">5.1</td><td valign="top">21.8</td><td valign="top">16.7</td><td valign="top">1.9</td><td valign="top">43.9</td><td valign="top">35.0</td><td valign="top">0.9</td></tr><tr><td valign="top"><bold>Diffusion Constant of Trans-Nuclear Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.3</td><td valign="top">1.2</td><td valign="top">1.0</td><td valign="top">1.5</td><td valign="top">1.6</td><td valign="top">5.1</td><td valign="top">1.5</td><td valign="top">1.2</td><td valign="top">1.9</td><td valign="top">1.1</td><td valign="top">1.1</td><td valign="top">0.9</td></tr><tr><td valign="top"><bold>Stoichiometry of Cytoplasmic Spots</bold></td><td valign="top">6.6</td><td valign="top">4.9</td><td valign="top">1.1</td><td valign="top">7.2</td><td valign="top">3.7</td><td valign="top">17.8</td><td valign="top">25.7</td><td valign="top">19.5</td><td valign="top">4.8</td><td valign="top">30.1</td><td valign="top">17.5</td><td valign="top">4.0</td></tr><tr><td valign="top"><bold>Diffusion Constant of Cytoplasmic Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.4</td><td valign="top">1.4</td><td valign="top">1.1</td><td valign="top">1.2</td><td valign="top">1.2</td><td valign="top">17.8</td><td valign="top">1.2</td><td valign="top">1.1</td><td valign="top">4.8</td><td valign="top">1.0</td><td valign="top">1.4</td><td valign="top">4.0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27451-v2.xml">elife-27451-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.27451.015</object-id><label>Table 4.</label><caption><title><italic>snf1&#916;</italic> foci tracking and copy number data.</title><p>Upper panel: Mean average, SD and mean number detected per cell (N) of stoichiometry values (molecules), and microscopic diffusion coefficients <italic>D</italic> in each compartment detected within the depth of field. Lower panel: Mean average and SD of copy number in pool and foci in each compartment.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"/><th colspan="6" valign="top"><bold>Mig1-GFP <italic>snf1&#916;</italic></bold></th></tr><tr><th colspan="3" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="3" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th></tr><tr><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th></tr></thead><tbody><tr><td valign="top"><bold>Stoichiometry of Nuclear Spots</bold></td><td valign="top">17.5</td><td valign="top">10.9</td><td valign="top">13.2</td><td valign="top">23.5</td><td valign="top">15.4</td><td valign="top">10.9</td></tr><tr><td valign="top"><bold>Diffusion Constant of Nuclear Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.1</td><td valign="top">1.1</td><td valign="top">13.2</td><td valign="top">0.7</td><td valign="top">0.8</td><td valign="top">10.9</td></tr><tr><td valign="top"><bold>Stoichiometry of Trans-Nuclear Spots</bold></td><td valign="top">8.9</td><td valign="top">6.0</td><td valign="top">1.2</td><td valign="top">12.7</td><td valign="top">6.1</td><td valign="top">0.5</td></tr><tr><td valign="top"><bold>Diffusion Constant of Trans-Nuclear Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.9</td><td valign="top">2.0</td><td valign="top">1.2</td><td valign="top">1.1</td><td valign="top">1.4</td><td valign="top">0.5</td></tr><tr><td valign="top"><bold>Stoichiometry of Cytoplasmic Spots</bold></td><td valign="top">6.2</td><td valign="top">2.2</td><td valign="top">5.0</td><td valign="top">8.3</td><td valign="top">4.1</td><td valign="top">9.1</td></tr><tr><td valign="top"><bold>Diffusion Constant of Cytoplasmic Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.3</td><td valign="top">1.2</td><td valign="top">5.0</td><td valign="top">1.0</td><td valign="top">1.2</td><td valign="top">9.1</td></tr><tr><td valign="top"><bold>Copy Numbers</bold></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><bold>Cytoplasmic Pool</bold></td><td valign="top">947</td><td valign="top">728</td><td valign="top">30</td><td valign="top">608</td><td valign="top">450</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Nuclear Pool</bold></td><td valign="top">807</td><td valign="top">398</td><td valign="top">30</td><td valign="top">611</td><td valign="top">325</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Pool</bold></td><td valign="top">1754</td><td valign="top">1127</td><td valign="top">30</td><td valign="top">1219</td><td valign="top">775</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Cytoplasmic Spots</bold></td><td valign="top">118</td><td valign="top">169</td><td valign="top">30</td><td valign="top">334</td><td valign="top">374</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Nuclear Spots</bold></td><td valign="top">162</td><td valign="top">69</td><td valign="top">30</td><td valign="top">164</td><td valign="top">71</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Spots</bold></td><td valign="top">280</td><td valign="top">238</td><td valign="top">30</td><td valign="top">498</td><td valign="top">445</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Cytoplasm</bold></td><td valign="top">1065</td><td valign="top">897</td><td valign="top">30</td><td valign="top">941</td><td valign="top">824</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Nuclear</bold></td><td valign="top">969</td><td valign="top">467</td><td valign="top">30</td><td valign="top">775</td><td valign="top">396</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Cell</bold></td><td valign="top">2034</td><td valign="top">1364</td><td valign="top">30</td><td valign="top">1717</td><td valign="top">1220</td><td valign="top">30</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27451-v3.xml">elife-27451-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.27451.009</object-id><label>Table 2.</label><caption><title>Copy number data.</title><p>Mean average and SD of copy number in pool and foci in each compartment.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"/><th colspan="4" valign="top"><bold>Mig1-GFP</bold></th><th colspan="4" valign="top"><bold>Msn2-GFP</bold></th></tr><tr><th colspan="2" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="2" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th><th colspan="2" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="2" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th></tr><tr><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th></tr></thead><tbody><tr><td valign="top"><bold>Cytoplasmic Pool</bold></td><td valign="top">509</td><td valign="top">274</td><td valign="top">949</td><td valign="top">394</td><td valign="top">1422</td><td valign="top">977</td><td valign="top">2487</td><td valign="top">1360</td></tr><tr><td valign="top"><bold>Nuclear Pool</bold></td><td valign="top">77</td><td valign="top">101</td><td valign="top">140</td><td valign="top">97</td><td valign="top">551</td><td valign="top">608</td><td valign="top">1692</td><td valign="top">1221</td></tr><tr><td valign="top"><bold>Total Pool</bold></td><td valign="top">586</td><td valign="top">336</td><td valign="top">1088</td><td valign="top">392</td><td valign="top">1973</td><td valign="top">1585</td><td valign="top">4179</td><td valign="top">2581</td></tr><tr><td valign="top"><bold>Cytoplasmic Spots</bold></td><td valign="top">57</td><td valign="top">79</td><td valign="top">311</td><td valign="top">212</td><td valign="top">333</td><td valign="top">196</td><td valign="top">776</td><td valign="top">635</td></tr><tr><td valign="top"><bold>Nuclear Spots</bold></td><td valign="top">190</td><td valign="top">99</td><td valign="top">35</td><td valign="top">63</td><td valign="top">81</td><td valign="top">138</td><td valign="top">320</td><td valign="top">269</td></tr><tr><td valign="top"><bold>Total Spots</bold></td><td valign="top">246</td><td valign="top">100</td><td valign="top">345</td><td valign="top">203</td><td valign="top">414</td><td valign="top">334</td><td valign="top">1096</td><td valign="top">904</td></tr><tr><td valign="top"><bold>Total Cytoplasm</bold></td><td valign="top">580</td><td valign="top">276</td><td valign="top">1156</td><td valign="top">399</td><td valign="top">1755</td><td valign="top">1173</td><td valign="top">3263</td><td valign="top">1995</td></tr><tr><td valign="top"><bold>Total Nuclear</bold></td><td valign="top">226</td><td valign="top">155</td><td valign="top">176</td><td valign="top">124</td><td valign="top">632</td><td valign="top">746</td><td valign="top">2012</td><td valign="top">1490</td></tr><tr><td valign="top"><bold>Total Cell</bold></td><td valign="top">806</td><td valign="top">353</td><td valign="top">1331</td><td valign="top">352</td><td valign="top">2387</td><td valign="top">1919</td><td valign="top">5274</td><td valign="top">3485</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27451-v3.xml">elife-27451-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.27451.010</object-id><label>Table 3.</label><caption><title>Foci tracking data.</title><p>Mean average, SD and mean number detected per cell (N) of stoichiometry values (molecules), and microscopic diffusion coefficients <italic>D</italic> in each compartment detected within the depth of field.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"/><th colspan="6" valign="top"><bold>Mig1-GFP</bold></th><th colspan="6" valign="top"><bold>Msn2-GFP</bold></th></tr><tr><th colspan="3" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="3" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th><th colspan="3" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="3" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th></tr><tr><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th></tr></thead><tbody><tr><td valign="top"><bold>Stoichiometry of Nuclear Spots</bold></td><td valign="top">19.0</td><td valign="top">16.2</td><td valign="top">7.2</td><td valign="top">8.5</td><td valign="top">4.8</td><td valign="top">5.8</td><td valign="top">34.5</td><td valign="top">26.6</td><td valign="top">3.5</td><td valign="top">46.5</td><td valign="top">31.6</td><td valign="top">4.7</td></tr><tr><td valign="top"><bold>Diffusion Constant of Nuclear Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">0.8</td><td valign="top">0.8</td><td valign="top">7.2</td><td valign="top">1.3</td><td valign="top">1.5</td><td valign="top">5.8</td><td valign="top">0.7</td><td valign="top">0.9</td><td valign="top">3.5</td><td valign="top">0.9</td><td valign="top">0.9</td><td valign="top">4.7</td></tr><tr><td valign="top"><bold>Stoichiometry of Trans-Nuclear Spots</bold></td><td valign="top">10.6</td><td valign="top">10.2</td><td valign="top">1.0</td><td valign="top">8.7</td><td valign="top">5.3</td><td valign="top">5.1</td><td valign="top">21.8</td><td valign="top">16.7</td><td valign="top">1.9</td><td valign="top">43.9</td><td valign="top">35.0</td><td valign="top">0.9</td></tr><tr><td valign="top"><bold>Diffusion Constant of Trans-Nuclear Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.3</td><td valign="top">1.2</td><td valign="top">1.0</td><td valign="top">1.5</td><td valign="top">1.6</td><td valign="top">5.1</td><td valign="top">1.5</td><td valign="top">1.2</td><td valign="top">1.9</td><td valign="top">1.1</td><td valign="top">1.1</td><td valign="top">0.9</td></tr><tr><td valign="top"><bold>Stoichiometry of Cytoplasmic Spots</bold></td><td valign="top">6.6</td><td valign="top">4.9</td><td valign="top">1.1</td><td valign="top">7.2</td><td valign="top">3.7</td><td valign="top">17.8</td><td valign="top">25.7</td><td valign="top">19.5</td><td valign="top">4.8</td><td valign="top">30.1</td><td valign="top">17.5</td><td valign="top">4.0</td></tr><tr><td valign="top"><bold>Diffusion Constant of Cytoplasmic Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.4</td><td valign="top">1.4</td><td valign="top">1.1</td><td valign="top">1.2</td><td valign="top">1.2</td><td valign="top">17.8</td><td valign="top">1.2</td><td valign="top">1.1</td><td valign="top">4.8</td><td valign="top">1.0</td><td valign="top">1.4</td><td valign="top">4.0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27451-v3.xml">elife-27451-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.27451.015</object-id><label>Table 4.</label><caption><title><italic>snf1&#916;</italic> foci tracking and copy number data.</title><p>Upper panel: Mean average, SD and mean number detected per cell (N) of stoichiometry values (molecules), and microscopic diffusion coefficients <italic>D</italic> in each compartment detected within the depth of field. Lower panel: Mean average and SD of copy number in pool and foci in each compartment.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"/><th colspan="6" valign="top"><bold>Mig1-GFP <italic>snf1&#916;</italic></bold></th></tr><tr><th colspan="3" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="3" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th></tr><tr><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th></tr></thead><tbody><tr><td valign="top"><bold>Stoichiometry of Nuclear Spots</bold></td><td valign="top">17.5</td><td valign="top">10.9</td><td valign="top">13.2</td><td valign="top">23.5</td><td valign="top">15.4</td><td valign="top">10.9</td></tr><tr><td valign="top"><bold>Diffusion Constant of Nuclear Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.1</td><td valign="top">1.1</td><td valign="top">13.2</td><td valign="top">0.7</td><td valign="top">0.8</td><td valign="top">10.9</td></tr><tr><td valign="top"><bold>Stoichiometry of Trans-Nuclear Spots</bold></td><td valign="top">8.9</td><td valign="top">6.0</td><td valign="top">1.2</td><td valign="top">12.7</td><td valign="top">6.1</td><td valign="top">0.5</td></tr><tr><td valign="top"><bold>Diffusion Constant of Trans-Nuclear Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.9</td><td valign="top">2.0</td><td valign="top">1.2</td><td valign="top">1.1</td><td valign="top">1.4</td><td valign="top">0.5</td></tr><tr><td valign="top"><bold>Stoichiometry of Cytoplasmic Spots</bold></td><td valign="top">6.2</td><td valign="top">2.2</td><td valign="top">5.0</td><td valign="top">8.3</td><td valign="top">4.1</td><td valign="top">9.1</td></tr><tr><td valign="top"><bold>Diffusion Constant of Cytoplasmic Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.3</td><td valign="top">1.2</td><td valign="top">5.0</td><td valign="top">1.0</td><td valign="top">1.2</td><td valign="top">9.1</td></tr><tr><td valign="top"><bold>Copy Numbers</bold></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><bold>Cytoplasmic Pool</bold></td><td valign="top">947</td><td valign="top">728</td><td valign="top">30</td><td valign="top">608</td><td valign="top">450</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Nuclear Pool</bold></td><td valign="top">807</td><td valign="top">398</td><td valign="top">30</td><td valign="top">611</td><td valign="top">325</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Pool</bold></td><td valign="top">1754</td><td valign="top">1127</td><td valign="top">30</td><td valign="top">1219</td><td valign="top">775</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Cytoplasmic Spots</bold></td><td valign="top">118</td><td valign="top">169</td><td valign="top">30</td><td valign="top">334</td><td valign="top">374</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Nuclear Spots</bold></td><td valign="top">162</td><td valign="top">69</td><td valign="top">30</td><td valign="top">164</td><td valign="top">71</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Spots</bold></td><td valign="top">280</td><td valign="top">238</td><td valign="top">30</td><td valign="top">498</td><td valign="top">445</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Cytoplasm</bold></td><td valign="top">1065</td><td valign="top">897</td><td valign="top">30</td><td valign="top">941</td><td valign="top">824</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Nuclear</bold></td><td valign="top">969</td><td valign="top">467</td><td valign="top">30</td><td valign="top">775</td><td valign="top">396</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Cell</bold></td><td valign="top">2034</td><td valign="top">1364</td><td valign="top">30</td><td valign="top">1717</td><td valign="top">1220</td><td valign="top">30</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27451-v4.xml">elife-27451-v4.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.27451.009</object-id><label>Table 2.</label><caption><title>Copy number data.</title><p>Mean average and SD of copy number in pool and foci in each compartment.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"/><th colspan="4" valign="top"><bold>Mig1-GFP</bold></th><th colspan="4" valign="top"><bold>Msn2-GFP</bold></th></tr><tr><th colspan="2" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="2" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th><th colspan="2" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="2" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th></tr><tr><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th></tr></thead><tbody><tr><td valign="top"><bold>Cytoplasmic Pool</bold></td><td valign="top">509</td><td valign="top">274</td><td valign="top">949</td><td valign="top">394</td><td valign="top">1422</td><td valign="top">977</td><td valign="top">2487</td><td valign="top">1360</td></tr><tr><td valign="top"><bold>Nuclear Pool</bold></td><td valign="top">77</td><td valign="top">101</td><td valign="top">140</td><td valign="top">97</td><td valign="top">551</td><td valign="top">608</td><td valign="top">1692</td><td valign="top">1221</td></tr><tr><td valign="top"><bold>Total Pool</bold></td><td valign="top">586</td><td valign="top">336</td><td valign="top">1088</td><td valign="top">392</td><td valign="top">1973</td><td valign="top">1585</td><td valign="top">4179</td><td valign="top">2581</td></tr><tr><td valign="top"><bold>Cytoplasmic Spots</bold></td><td valign="top">57</td><td valign="top">79</td><td valign="top">311</td><td valign="top">212</td><td valign="top">333</td><td valign="top">196</td><td valign="top">776</td><td valign="top">635</td></tr><tr><td valign="top"><bold>Nuclear Spots</bold></td><td valign="top">190</td><td valign="top">99</td><td valign="top">35</td><td valign="top">63</td><td valign="top">81</td><td valign="top">138</td><td valign="top">320</td><td valign="top">269</td></tr><tr><td valign="top"><bold>Total Spots</bold></td><td valign="top">246</td><td valign="top">100</td><td valign="top">345</td><td valign="top">203</td><td valign="top">414</td><td valign="top">334</td><td valign="top">1096</td><td valign="top">904</td></tr><tr><td valign="top"><bold>Total Cytoplasm</bold></td><td valign="top">580</td><td valign="top">276</td><td valign="top">1156</td><td valign="top">399</td><td valign="top">1755</td><td valign="top">1173</td><td valign="top">3263</td><td valign="top">1995</td></tr><tr><td valign="top"><bold>Total Nuclear</bold></td><td valign="top">226</td><td valign="top">155</td><td valign="top">176</td><td valign="top">124</td><td valign="top">632</td><td valign="top">746</td><td valign="top">2012</td><td valign="top">1490</td></tr><tr><td valign="top"><bold>Total Cell</bold></td><td valign="top">806</td><td valign="top">353</td><td valign="top">1331</td><td valign="top">352</td><td valign="top">2387</td><td valign="top">1919</td><td valign="top">5274</td><td valign="top">3485</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27451-v4.xml">elife-27451-v4.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.27451.010</object-id><label>Table 3.</label><caption><title>Foci tracking data.</title><p>Mean average, SD and mean number detected per cell (N) of stoichiometry values (molecules), and microscopic diffusion coefficients <italic>D</italic> in each compartment detected within the depth of field.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"/><th colspan="6" valign="top"><bold>Mig1-GFP</bold></th><th colspan="6" valign="top"><bold>Msn2-GFP</bold></th></tr><tr><th colspan="3" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="3" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th><th colspan="3" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="3" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th></tr><tr><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th></tr></thead><tbody><tr><td valign="top"><bold>Stoichiometry of Nuclear Spots</bold></td><td valign="top">19.0</td><td valign="top">16.2</td><td valign="top">7.2</td><td valign="top">8.5</td><td valign="top">4.8</td><td valign="top">5.8</td><td valign="top">34.5</td><td valign="top">26.6</td><td valign="top">3.5</td><td valign="top">46.5</td><td valign="top">31.6</td><td valign="top">4.7</td></tr><tr><td valign="top"><bold>Diffusion Constant of Nuclear Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">0.8</td><td valign="top">0.8</td><td valign="top">7.2</td><td valign="top">1.3</td><td valign="top">1.5</td><td valign="top">5.8</td><td valign="top">0.7</td><td valign="top">0.9</td><td valign="top">3.5</td><td valign="top">0.9</td><td valign="top">0.9</td><td valign="top">4.7</td></tr><tr><td valign="top"><bold>Stoichiometry of Trans-Nuclear Spots</bold></td><td valign="top">10.6</td><td valign="top">10.2</td><td valign="top">1.0</td><td valign="top">8.7</td><td valign="top">5.3</td><td valign="top">5.1</td><td valign="top">21.8</td><td valign="top">16.7</td><td valign="top">1.9</td><td valign="top">43.9</td><td valign="top">35.0</td><td valign="top">0.9</td></tr><tr><td valign="top"><bold>Diffusion Constant of Trans-Nuclear Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.3</td><td valign="top">1.2</td><td valign="top">1.0</td><td valign="top">1.5</td><td valign="top">1.6</td><td valign="top">5.1</td><td valign="top">1.5</td><td valign="top">1.2</td><td valign="top">1.9</td><td valign="top">1.1</td><td valign="top">1.1</td><td valign="top">0.9</td></tr><tr><td valign="top"><bold>Stoichiometry of Cytoplasmic Spots</bold></td><td valign="top">6.6</td><td valign="top">4.9</td><td valign="top">1.1</td><td valign="top">7.2</td><td valign="top">3.7</td><td valign="top">17.8</td><td valign="top">25.7</td><td valign="top">19.5</td><td valign="top">4.8</td><td valign="top">30.1</td><td valign="top">17.5</td><td valign="top">4.0</td></tr><tr><td valign="top"><bold>Diffusion Constant of Cytoplasmic Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.4</td><td valign="top">1.4</td><td valign="top">1.1</td><td valign="top">1.2</td><td valign="top">1.2</td><td valign="top">17.8</td><td valign="top">1.2</td><td valign="top">1.1</td><td valign="top">4.8</td><td valign="top">1.0</td><td valign="top">1.4</td><td valign="top">4.0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27451-v4.xml">elife-27451-v4.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.27451.015</object-id><label>Table 4.</label><caption><title><italic>snf1&#916;</italic> foci tracking and copy number data.</title><p>Upper panel: Mean average, SD and mean number detected per cell (N) of stoichiometry values (molecules), and microscopic diffusion coefficients <italic>D</italic> in each compartment detected within the depth of field. Lower panel: Mean average and SD of copy number in pool and foci in each compartment.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"/><th colspan="6" valign="top"><bold>Mig1-GFP <italic>snf1&#916;</italic></bold></th></tr><tr><th colspan="3" valign="top"><bold><italic>Glucose</italic> (+)</bold></th><th colspan="3" valign="top"><bold><italic>Glucose</italic> (&#8722;)</bold></th></tr><tr><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>Mean</bold></th><th valign="top"><bold>SD</bold></th><th valign="top"><bold>N</bold></th></tr></thead><tbody><tr><td valign="top"><bold>Stoichiometry of Nuclear Spots</bold></td><td valign="top">17.5</td><td valign="top">10.9</td><td valign="top">13.2</td><td valign="top">23.5</td><td valign="top">15.4</td><td valign="top">10.9</td></tr><tr><td valign="top"><bold>Diffusion Constant of Nuclear Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.1</td><td valign="top">1.1</td><td valign="top">13.2</td><td valign="top">0.7</td><td valign="top">0.8</td><td valign="top">10.9</td></tr><tr><td valign="top"><bold>Stoichiometry of Trans-Nuclear Spots</bold></td><td valign="top">8.9</td><td valign="top">6.0</td><td valign="top">1.2</td><td valign="top">12.7</td><td valign="top">6.1</td><td valign="top">0.5</td></tr><tr><td valign="top"><bold>Diffusion Constant of Trans-Nuclear Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.9</td><td valign="top">2.0</td><td valign="top">1.2</td><td valign="top">1.1</td><td valign="top">1.4</td><td valign="top">0.5</td></tr><tr><td valign="top"><bold>Stoichiometry of Cytoplasmic Spots</bold></td><td valign="top">6.2</td><td valign="top">2.2</td><td valign="top">5.0</td><td valign="top">8.3</td><td valign="top">4.1</td><td valign="top">9.1</td></tr><tr><td valign="top"><bold>Diffusion Constant of Cytoplasmic Spots (&#181;m<sup>2</sup>/s)</bold></td><td valign="top">1.3</td><td valign="top">1.2</td><td valign="top">5.0</td><td valign="top">1.0</td><td valign="top">1.2</td><td valign="top">9.1</td></tr><tr><td valign="top"><bold>Copy Numbers</bold></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><bold>Cytoplasmic Pool</bold></td><td valign="top">947</td><td valign="top">728</td><td valign="top">30</td><td valign="top">608</td><td valign="top">450</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Nuclear Pool</bold></td><td valign="top">807</td><td valign="top">398</td><td valign="top">30</td><td valign="top">611</td><td valign="top">325</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Pool</bold></td><td valign="top">1754</td><td valign="top">1127</td><td valign="top">30</td><td valign="top">1219</td><td valign="top">775</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Cytoplasmic Spots</bold></td><td valign="top">118</td><td valign="top">169</td><td valign="top">30</td><td valign="top">334</td><td valign="top">374</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Nuclear Spots</bold></td><td valign="top">162</td><td valign="top">69</td><td valign="top">30</td><td valign="top">164</td><td valign="top">71</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Spots</bold></td><td valign="top">280</td><td valign="top">238</td><td valign="top">30</td><td valign="top">498</td><td valign="top">445</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Cytoplasm</bold></td><td valign="top">1065</td><td valign="top">897</td><td valign="top">30</td><td valign="top">941</td><td valign="top">824</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Nuclear</bold></td><td valign="top">969</td><td valign="top">467</td><td valign="top">30</td><td valign="top">775</td><td valign="top">396</td><td valign="top">30</td></tr><tr><td valign="top"><bold>Total Cell</bold></td><td valign="top">2034</td><td valign="top">1364</td><td valign="top">30</td><td valign="top">1717</td><td valign="top">1220</td><td valign="top">30</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27469-v1.xml">elife-27469-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table6" position="float"><object-id pub-id-type="doi">10.7554/eLife.27469.017</object-id><label>Table 6.</label><caption><title>Phenotypic alterations as a function of time spent in quiescence</title></caption><table frame="hsides" rules="groups"><thead><tr><th>Time</th><th rowspan="2"># of colonies w/phenotype</th><th rowspan="2">18&#176;C</th><th rowspan="2">37&#176;C</th><th rowspan="2">KCl 1M</th><th rowspan="2">Ca(NO<sub>3</sub>)2 0.15M</th><th rowspan="2">CaCl<sub>2</sub>0.3M</th><th rowspan="2">TBZ 15 &#956;g/ml</th><th rowspan="2">HU 4 mM</th><th rowspan="2">SDS 0.01%</th></tr><tr><th>in G0</th></tr></thead><tbody><tr><td><bold>1 day</bold></td><td>0/384</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td><bold>1 month</bold></td><td>0/384</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td><bold>2 months</bold></td><td>4/376</td><td>-</td><td>-</td><td>1</td><td>1</td><td>4</td><td>-</td><td>-</td><td>-</td></tr><tr><td><bold>3 months</bold></td><td>6/334</td><td>1</td><td>1</td><td>1</td><td>1</td><td>3</td><td>3</td><td>2</td><td>1</td></tr></tbody></table><table-wrap-foot><fn><p>(<bold>-</bold>) indicates the absence of a selected phenotype (all the cells form colonies) The numbers in the columns indicate the number of colonies exhibiting a sensitivity to a given treatment (some mutants display a sensitivity to multiple treatments).</p><p>Serial dilutions were spotted on rich medium plates containing (or not) the drugs at the indicated concentrations. (TBZ): Thiabendazole; (HU): Hydroxyurea; (SDS): Sodium Dodecyl Sulfate</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27702-v2.xml">elife-27702-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.27702.021</object-id><label>Table 2.</label><caption><title>Proportion of units that have stable neural dynamics and motor representations in the striatum and motor cortex.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Activity metric</th><th colspan="2">Dorsolateral striatum</th><th colspan="2">Motor cortex</th></tr><tr><th>&#8195;<italic>% stable</italic></th><th><italic>n</italic></th><th><italic>% stable</italic></th><th><italic>n</italic></th></tr></thead><tbody><tr><td>&#8195;Firing rate</td><td>90.3</td><td>186</td><td>88.1</td><td>42</td></tr><tr><td>&#8195;ISI histogram</td><td>97.9</td><td>187</td><td>97.6</td><td>42</td></tr><tr><td>&#8195;State tuning</td><td>93.9</td><td>163</td><td>92.7</td><td>41</td></tr><tr><td>&#8195;STA</td><td>93.6</td><td>202</td><td>95.0</td><td>80</td></tr><tr><td>&#8195;PETH</td><td>89.4</td><td>113</td><td>89.5</td><td>57</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27711-v1.xml">elife-27711-v1.xml</a></h2><table-wrap xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.27711.012</object-id><label>Table 2.</label><caption><p>C&#946;-C&#946; and N-C&#946; interresidue distance restraints, measured in the molecular models of the CRF- and <sup>dFX</sup>CRF(12-41)-CRF1R complexes.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.27711.012">http://dx.doi.org/10.7554/eLife.27711.012</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="6" valign="top">CRF1R bound to agonist CRF</th></tr><tr><th rowspan="2" valign="top">No</th><th rowspan="2" valign="top">CRF-CRF1R residue pair</th><th rowspan="2" valign="top">Region of <break/>CRF</th><th rowspan="2" valign="top">Region of CRF1R</th><th colspan="2" valign="top">C&#946;-C&#946; distance [&#197;]</th></tr></thead><tbody><tr><td valign="top">Constraint</td><td valign="top">Measured</td></tr><tr><td valign="top">1</td><td valign="top">H13-F330</td><td valign="top">N-term</td><td valign="top">Helix VI</td><td valign="top">10.0</td><td valign="top">5.1</td></tr><tr><td valign="top">2</td><td valign="top">H13-F344</td><td valign="top">N-term</td><td valign="top">Helix VII</td><td valign="top">10.0</td><td valign="top">9.4</td></tr><tr><td valign="top">3</td><td valign="top">L15-Q173</td><td valign="top">N-term</td><td valign="top">Helix II</td><td valign="top">10.0</td><td valign="top">7.1</td></tr><tr><td valign="top">4</td><td valign="top">E17-I345</td><td valign="top">N-term</td><td valign="top">Helix VII</td><td valign="top">10.0</td><td valign="top">9.2</td></tr><tr><td valign="top">5</td><td valign="top">V18-V120</td><td valign="top">C-term</td><td valign="top">Helix I</td><td valign="top">10.0</td><td valign="top">6.4</td></tr><tr><td valign="top">6</td><td valign="top">A31-E104</td><td valign="top">C-term</td><td valign="top">Hinge</td><td valign="top">10.0</td><td valign="top">8.6</td></tr><tr><td valign="top">7</td><td valign="top">S33-Y73</td><td valign="top">C-term</td><td valign="top">ECD</td><td valign="top">10.0</td><td valign="top">6.7</td></tr><tr><td colspan="6" valign="top"><bold>CRF1R bound to Antagonist <sup>dFX</sup>CRF(12-41)</bold></td></tr><tr><td rowspan="2" valign="top">No</td><td rowspan="2" valign="top">CRF1R-<sup>dFX</sup>CRF(12-41) residue pair</td><td rowspan="2" valign="top">Region of <sup>dFX</sup>CRF(12-41)</td><td rowspan="2" valign="top">Region of CRF1R</td><td colspan="2" valign="top">C&#946;-C&#946; or N-C&#946;* distance [&#197;]</td></tr><tr><td valign="top">Constraint</td><td valign="top">Measured</td></tr><tr><td valign="top">1</td><td valign="top">F12-L329</td><td valign="top">N-term</td><td valign="top">Helix VI</td><td valign="top">5.0*</td><td valign="top">4.4*</td></tr><tr><td valign="top">2</td><td valign="top">F12-F330</td><td valign="top">N-term</td><td valign="top">Helix VI</td><td valign="top">5.0*</td><td valign="top">4.6*</td></tr><tr><td valign="top">3</td><td valign="top">F12-N348</td><td valign="top">N-term</td><td valign="top">Helix VII</td><td valign="top">5.0*</td><td valign="top">4.8*</td></tr><tr><td valign="top">4</td><td valign="top">H13-L329</td><td valign="top">N-term</td><td valign="top">Helix VI</td><td valign="top">10.0</td><td valign="top">6.7</td></tr><tr><td valign="top">5</td><td valign="top">H13-F330</td><td valign="top">N-term</td><td valign="top">Helix VI</td><td valign="top">10.0</td><td valign="top">5.5</td></tr><tr><td valign="top">6</td><td valign="top">H13-F344</td><td valign="top">N-term</td><td valign="top">Helix</td><td valign="top">10.0</td><td valign="top">5.0</td></tr><tr><td valign="top">7</td><td valign="top">L14-N123</td><td valign="top">N-term</td><td valign="top">Helix I</td><td valign="top">10.0</td><td valign="top">9.1</td></tr><tr><td valign="top">8</td><td valign="top">L15-R341</td><td valign="top">N-term</td><td valign="top">Helix VII</td><td valign="top">10.0</td><td valign="top">5.9</td></tr><tr><td valign="top">9</td><td valign="top">L15-I345</td><td valign="top">N-term</td><td valign="top">Helix VII</td><td valign="top">10.0</td><td valign="top">6.8</td></tr><tr><td valign="top">10</td><td valign="top">E17-N123</td><td valign="top">N-term</td><td valign="top">Helix I</td><td valign="top">10.0</td><td valign="top">7.4</td></tr><tr><td valign="top">11</td><td valign="top">E17-Q173</td><td valign="top">N-term</td><td valign="top">Helix II</td><td valign="top">10.0</td><td valign="top">6.3</td></tr><tr><td valign="top">12</td><td valign="top">E17-I345</td><td valign="top">N-term</td><td valign="top">Helix VII</td><td valign="top">10.0</td><td valign="top">6.1</td></tr><tr><td valign="top">13</td><td valign="top">V18-V120</td><td valign="top">C-term</td><td valign="top">Helix I</td><td valign="top">10.0</td><td valign="top">4.7</td></tr><tr><td valign="top">14</td><td valign="top">A31-E104</td><td valign="top">C-term</td><td valign="top">Hinge</td><td valign="top">10.0</td><td valign="top">4.3</td></tr><tr><td valign="top">15</td><td valign="top">S33-Y73</td><td valign="top">C-term</td><td valign="top">ECD</td><td valign="top">10.0</td><td valign="top">6.7</td></tr></tbody></table><table-wrap-foot><fn id="tblfn1"><p><sup>*</sup>Pair-wise crosslinking between N-terminal ClAc in the peptide&#160;and&#160;Cys thiol in CRF1R.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27793-v2.xml">elife-27793-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.27793.005</object-id><label>Table 1.</label><caption><title>Kinetic and affinity data of monomeric Kinesin-1,&#8211;3, and &#8722;5 compared to MKLP2-MD.</title><p>A summary of steady state ATPase activities and MT affinities of monomeric Kin1 (<xref ref-type="bibr" rid="bib3">Atherton et al., 2014</xref>; <xref ref-type="bibr" rid="bib74">Woehlke et al., 1997</xref>; <xref ref-type="bibr" rid="bib33">Hackney, 1988</xref>; <xref ref-type="bibr" rid="bib27">Gilbert et al., 1995</xref>; <xref ref-type="bibr" rid="bib48">Ma and Taylor, 1995</xref>; <xref ref-type="bibr" rid="bib65">Rosenfeld et al., 1996</xref>; <xref ref-type="bibr" rid="bib54">Nitta et al., 2004</xref>; <xref ref-type="bibr" rid="bib26">Gigant et al., 2013</xref>), whose structures were used to compared with MKLP2, as well as Kin3 (<xref ref-type="bibr" rid="bib54">Nitta et al., 2004</xref>). Kin5&#8217;s MT-stimulated ATPase activity has similar values as MKLP2 (<xref ref-type="bibr" rid="bib12">Cochran et al., 2004</xref>; <xref ref-type="bibr" rid="bib11">Cochran et al., 2006</xref>).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Monomeric motor</th><th colspan="2" valign="top">MT-stimulated ATPase</th><th colspan="4" valign="top">MT affinity; K<sub>d</sub> (&#956;M)</th></tr><tr><th>&#8195;K<sub>0.5, MT</sub> (&#956;M)</th><th>k<sub>cat</sub> (S<sup>&#8722;1</sup>)</th><th><italic>ADP</italic></th><th><italic>NN</italic></th><th><italic>ADP.AlFx</italic></th><th><italic>AMPPNP</italic></th></tr></thead><tbody><tr><td valign="top">MKLP2-MD</td><td>1.07&#160;&#177;&#160;0.18</td><td>4.38&#160;&#177;&#160;0.20</td><td>0.363&#160;&#177;&#160;0.057</td><td>0.043&#160;&#177;&#160;0.002</td><td>0.355&#160;&#177;&#160;0.073</td><td>0.043&#160;&#177;&#160;0.035</td></tr><tr><td valign="top">Kin1 (Kif5a/b)</td><td>12.7&#160;&#177;&#160;4.0 (<xref ref-type="bibr" rid="bib3">Atherton et al., 2014</xref>) <break/>26.0&#160;&#177;&#160;5.8 (<xref ref-type="bibr" rid="bib65">Rosenfeld et al., 1996</xref>)</td><td>34.2&#160;&#177;&#160;5.7 (<xref ref-type="bibr" rid="bib3">Atherton et al., 2014</xref>) <break/>43.6&#160;&#177;&#160;7.8 (<xref ref-type="bibr" rid="bib65">Rosenfeld et al., 1996</xref>) <break/>50.3&#160;&#177;&#160;1.6 (26)</td><td>20.8&#160;&#177;&#160;2.4 (<xref ref-type="bibr" rid="bib65">Rosenfeld et al., 1996</xref>) <break/>2.0&#160;&#177;&#160;0.3 (<xref ref-type="bibr" rid="bib26">Gigant et al., 2013</xref>)</td><td>n.a.</td><td>1.4&#160;&#177;&#160;0.2 (<xref ref-type="bibr" rid="bib65">Rosenfeld et al., 1996</xref>) <break/>0.3&#160;&#177;&#160;0.006 (<xref ref-type="bibr" rid="bib26">Gigant et al., 2013</xref>)</td><td>1.1&#160;&#177;&#160;0.1 (<xref ref-type="bibr" rid="bib65">Rosenfeld et al., 1996</xref>) <break/>0.045&#160;&#177;&#160;0.012 (<xref ref-type="bibr" rid="bib26">Gigant et al., 2013</xref>)</td></tr><tr><td valign="top">Kin3 (Kif1a)</td><td>0.0537&#160;&#177;&#160;0.0057 (<xref ref-type="bibr" rid="bib3">Atherton et al., 2014</xref>)</td><td>43.4&#160;&#177;&#160;1.0 (<xref ref-type="bibr" rid="bib3">Atherton et al., 2014</xref>)</td><td>0.0068&#160;&#177;&#160;0.0025 (<xref ref-type="bibr" rid="bib54">Nitta et al., 2004</xref>)</td><td>n.a.</td><td>0.0059&#160;&#177;&#160;0.0015 (<xref ref-type="bibr" rid="bib54">Nitta et al., 2004</xref>)</td><td>0.0042&#160;&#177;&#160;0.0013 (<xref ref-type="bibr" rid="bib54">Nitta et al., 2004</xref>)</td></tr><tr><td valign="top">Kin5 <break/>(Eg5)</td><td>4.5 (<xref ref-type="bibr" rid="bib12">Cochran et al., 2004</xref>) <break/>0.29&#160;&#177;&#160;0.02 (<xref ref-type="bibr" rid="bib11">Cochran et al., 2006</xref>)</td><td>2.9 (<xref ref-type="bibr" rid="bib12">Cochran et al., 2004</xref>) <break/>5.5&#160;&#177;&#160;0.1 (<xref ref-type="bibr" rid="bib11">Cochran et al., 2006</xref>)</td><td>n.a.</td><td>n.a.</td><td>n.a.</td><td>n.a.</td></tr></tbody></table><table-wrap-foot><fn><p>n.a.: not available.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27793-v2.xml">elife-27793-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.27793.011</object-id><label>Table 2.</label><caption><title>MKLP2 Homology modelling and validation.</title><p>Columns 2&#8211;3 show the list of PDB IDs that were used as templates for specific structural regions of each MKLP2 nucleotide state (4LNU (<xref ref-type="bibr" rid="bib7">Cao et al., 2014</xref>), 4HNA (<xref ref-type="bibr" rid="bib26">Gigant et al., 2013</xref>), 4OZQ (<xref ref-type="bibr" rid="bib2">Arora et al., 2014</xref>), 1VFV (<xref ref-type="bibr" rid="bib54">Nitta et al., 2004</xref>), 3GBJ and 4Y05 are structures from the Structural Genomics Consortium). Columns 4&#8211;5 show the overall QMEAN scores calculated before, and after refinement and show that model quality is maintained following refinement. Columns 6&#8211;7 show the global cross-correlation scores before and after refinement.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Nucleotide state</th><th colspan="2">Templates used for homology modelling</th><th colspan="2">QMEAN scores</th><th colspan="2">Global Cross-correlation scores</th></tr><tr><th>&#8195;<italic>MKLP2 Structure</italic></th><th><italic>Template</italic></th><th><italic>Homology Model</italic></th><th><italic>Refined Model</italic></th><th><italic>Homology Model</italic></th><th><italic>Refined Model</italic></th></tr></thead><tbody><tr><td rowspan="4">&#8195;<italic>ADP</italic></td><td>Main template</td><td>3GBJ</td><td rowspan="4">0.636</td><td rowspan="4">0.623</td><td rowspan="4">0.908</td><td rowspan="4">0.931</td></tr><tr><td>&#8195;Helix &#945;4, helix &#945;5, loop 8</td><td>4LNU</td></tr><tr><td>&#8195;Helix &#945;4/loop 12</td><td>4OZQ</td></tr><tr><td>&#8195;Loop 2</td><td>4Y05</td></tr><tr><td rowspan="4">&#8195;<italic>NN</italic></td><td>Main template</td><td>4LNU</td><td rowspan="4">0.669</td><td rowspan="4">0.663</td><td rowspan="4">0.913</td><td rowspan="4">0.930</td></tr><tr><td>&#8195;Helix &#945;4/loop 12</td><td>4OZQ</td></tr><tr><td>&#8195;Loop 2</td><td>4Y05</td></tr><tr><td>&#8195;Loop 5</td><td>3GBJ</td></tr><tr><td rowspan="4">&#8195;<italic>ADP.AlFx</italic></td><td>Main template</td><td>4HNA</td><td rowspan="4">0.671</td><td rowspan="4">0.689</td><td rowspan="4">0.860</td><td rowspan="4">0.890</td></tr><tr><td>&#8195;Helix &#945;4/loop 12</td><td>4OZQ</td></tr><tr><td>&#8195;Loop 2</td><td>4Y05</td></tr><tr><td>&#8195;Loop 5</td><td>3GBJ</td></tr><tr><td rowspan="4">&#8195;<italic>AMPPNP</italic></td><td>Main template</td><td>1VFV</td><td rowspan="4">0.658</td><td rowspan="4">0.679</td><td rowspan="4">0.925</td><td rowspan="4">0.943</td></tr><tr><td>&#8195;Helix &#945;4, helix &#945;5, loop 8</td><td>4LNU</td></tr><tr><td>&#8195;Helix &#945;4/loop 12</td><td>4OZQ</td></tr><tr><td>&#8195;Loop 2</td><td>4Y05</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27793-v3.xml">elife-27793-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.27793.005</object-id><label>Table 1.</label><caption><title>Kinetic and affinity data of monomeric Kinesin-1,&#8211;3, and &#8722;5 compared to MKLP2-MD.</title><p>A summary of steady state ATPase activities and MT affinities of monomeric Kin1 (<xref ref-type="bibr" rid="bib3">Atherton et al., 2014</xref>; <xref ref-type="bibr" rid="bib74">Woehlke et al., 1997</xref>; <xref ref-type="bibr" rid="bib33">Hackney, 1988</xref>; <xref ref-type="bibr" rid="bib27">Gilbert et al., 1995</xref>; <xref ref-type="bibr" rid="bib48">Ma and Taylor, 1995</xref>; <xref ref-type="bibr" rid="bib65">Rosenfeld et al., 1996</xref>; <xref ref-type="bibr" rid="bib54">Nitta et al., 2004</xref>; <xref ref-type="bibr" rid="bib26">Gigant et al., 2013</xref>), whose structures were used to compared with MKLP2, as well as Kin3 (<xref ref-type="bibr" rid="bib54">Nitta et al., 2004</xref>). Kin5&#8217;s MT-stimulated ATPase activity has similar values as MKLP2 (<xref ref-type="bibr" rid="bib12">Cochran et al., 2004</xref>; <xref ref-type="bibr" rid="bib11">Cochran et al., 2006</xref>).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Monomeric motor</th><th colspan="2" valign="top">MT-stimulated ATPase</th><th colspan="4" valign="top">MT affinity; K<sub>d</sub> (&#956;M)</th></tr><tr><th>&#8195;K<sub>0.5, MT</sub> (&#956;M)</th><th>k<sub>cat</sub> (S<sup>&#8722;1</sup>)</th><th><italic>ADP</italic></th><th><italic>NN</italic></th><th><italic>ADP.AlFx</italic></th><th><italic>AMPPNP</italic></th></tr></thead><tbody><tr><td valign="top">MKLP2-MD</td><td>1.07&#160;&#177;&#160;0.18</td><td>4.38&#160;&#177;&#160;0.20</td><td>0.363&#160;&#177;&#160;0.057</td><td>0.043&#160;&#177;&#160;0.002</td><td>0.355&#160;&#177;&#160;0.073</td><td>0.043&#160;&#177;&#160;0.035</td></tr><tr><td valign="top">Kin1 (Kif5a/b)</td><td>12.7&#160;&#177;&#160;4.0 (<xref ref-type="bibr" rid="bib3">Atherton et al., 2014</xref>) <break/>26.0&#160;&#177;&#160;5.8 (<xref ref-type="bibr" rid="bib65">Rosenfeld et al., 1996</xref>)</td><td>34.2&#160;&#177;&#160;5.7 (<xref ref-type="bibr" rid="bib3">Atherton et al., 2014</xref>) <break/>43.6&#160;&#177;&#160;7.8 (<xref ref-type="bibr" rid="bib65">Rosenfeld et al., 1996</xref>) <break/>50.3&#160;&#177;&#160;1.6 (26)</td><td>20.8&#160;&#177;&#160;2.4 (<xref ref-type="bibr" rid="bib65">Rosenfeld et al., 1996</xref>) <break/>2.0&#160;&#177;&#160;0.3 (<xref ref-type="bibr" rid="bib26">Gigant et al., 2013</xref>)</td><td>n.a.</td><td>1.4&#160;&#177;&#160;0.2 (<xref ref-type="bibr" rid="bib65">Rosenfeld et al., 1996</xref>) <break/>0.3&#160;&#177;&#160;0.006 (<xref ref-type="bibr" rid="bib26">Gigant et al., 2013</xref>)</td><td>1.1&#160;&#177;&#160;0.1 (<xref ref-type="bibr" rid="bib65">Rosenfeld et al., 1996</xref>) <break/>0.045&#160;&#177;&#160;0.012 (<xref ref-type="bibr" rid="bib26">Gigant et al., 2013</xref>)</td></tr><tr><td valign="top">Kin3 (Kif1a)</td><td>0.0537&#160;&#177;&#160;0.0057 (<xref ref-type="bibr" rid="bib3">Atherton et al., 2014</xref>)</td><td>43.4&#160;&#177;&#160;1.0 (<xref ref-type="bibr" rid="bib3">Atherton et al., 2014</xref>)</td><td>0.0068&#160;&#177;&#160;0.0025 (<xref ref-type="bibr" rid="bib54">Nitta et al., 2004</xref>)</td><td>n.a.</td><td>0.0059&#160;&#177;&#160;0.0015 (<xref ref-type="bibr" rid="bib54">Nitta et al., 2004</xref>)</td><td>0.0042&#160;&#177;&#160;0.0013 (<xref ref-type="bibr" rid="bib54">Nitta et al., 2004</xref>)</td></tr><tr><td valign="top">Kin5 <break/>(Eg5)</td><td>4.5 (<xref ref-type="bibr" rid="bib12">Cochran et al., 2004</xref>) <break/>0.29&#160;&#177;&#160;0.02 (<xref ref-type="bibr" rid="bib11">Cochran et al., 2006</xref>)</td><td>2.9 (<xref ref-type="bibr" rid="bib12">Cochran et al., 2004</xref>) <break/>5.5&#160;&#177;&#160;0.1 (<xref ref-type="bibr" rid="bib11">Cochran et al., 2006</xref>)</td><td>n.a.</td><td>n.a.</td><td>n.a.</td><td>n.a.</td></tr></tbody></table><table-wrap-foot><fn><p>n.a.: not available.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27793-v3.xml">elife-27793-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.27793.011</object-id><label>Table 2.</label><caption><title>MKLP2 Homology modelling and validation.</title><p>Columns 2&#8211;3 show the list of PDB IDs that were used as templates for specific structural regions of each MKLP2 nucleotide state (4LNU (<xref ref-type="bibr" rid="bib7">Cao et al., 2014</xref>), 4HNA (<xref ref-type="bibr" rid="bib26">Gigant et al., 2013</xref>), 4OZQ (<xref ref-type="bibr" rid="bib2">Arora et al., 2014</xref>), 1VFV (<xref ref-type="bibr" rid="bib54">Nitta et al., 2004</xref>), 3GBJ and 4Y05 are structures from the Structural Genomics Consortium). Columns 4&#8211;5 show the overall QMEAN scores calculated before, and after refinement and show that model quality is maintained following refinement. Columns 6&#8211;7 show the global cross-correlation scores before and after refinement.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Nucleotide state</th><th colspan="2">Templates used for homology modelling</th><th colspan="2">QMEAN scores</th><th colspan="2">Global Cross-correlation scores</th></tr><tr><th>&#8195;<italic>MKLP2 Structure</italic></th><th><italic>Template</italic></th><th><italic>Homology Model</italic></th><th><italic>Refined Model</italic></th><th><italic>Homology Model</italic></th><th><italic>Refined Model</italic></th></tr></thead><tbody><tr><td rowspan="4">&#8195;<italic>ADP</italic></td><td>Main template</td><td>3GBJ</td><td rowspan="4">0.636</td><td rowspan="4">0.623</td><td rowspan="4">0.908</td><td rowspan="4">0.931</td></tr><tr><td>&#8195;Helix &#945;4, helix &#945;5, loop 8</td><td>4LNU</td></tr><tr><td>&#8195;Helix &#945;4/loop 12</td><td>4OZQ</td></tr><tr><td>&#8195;Loop 2</td><td>4Y05</td></tr><tr><td rowspan="4">&#8195;<italic>NN</italic></td><td>Main template</td><td>4LNU</td><td rowspan="4">0.669</td><td rowspan="4">0.663</td><td rowspan="4">0.913</td><td rowspan="4">0.930</td></tr><tr><td>&#8195;Helix &#945;4/loop 12</td><td>4OZQ</td></tr><tr><td>&#8195;Loop 2</td><td>4Y05</td></tr><tr><td>&#8195;Loop 5</td><td>3GBJ</td></tr><tr><td rowspan="4">&#8195;<italic>ADP.AlFx</italic></td><td>Main template</td><td>4HNA</td><td rowspan="4">0.671</td><td rowspan="4">0.689</td><td rowspan="4">0.860</td><td rowspan="4">0.890</td></tr><tr><td>&#8195;Helix &#945;4/loop 12</td><td>4OZQ</td></tr><tr><td>&#8195;Loop 2</td><td>4Y05</td></tr><tr><td>&#8195;Loop 5</td><td>3GBJ</td></tr><tr><td rowspan="4">&#8195;<italic>AMPPNP</italic></td><td>Main template</td><td>1VFV</td><td rowspan="4">0.658</td><td rowspan="4">0.679</td><td rowspan="4">0.925</td><td rowspan="4">0.943</td></tr><tr><td>&#8195;Helix &#945;4, helix &#945;5, loop 8</td><td>4LNU</td></tr><tr><td>&#8195;Helix &#945;4/loop 12</td><td>4OZQ</td></tr><tr><td>&#8195;Loop 2</td><td>4Y05</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-27860-v2.xml">elife-27860-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.27860.009</object-id><label>Table 1.</label><caption><title>AltORF and alternative protein annotations in different organisms.</title></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Genomes</th><th colspan="6" valign="top">Features</th></tr><tr><th rowspan="2" valign="top"/><th colspan="2" valign="top">Transcripts</th><th colspan="2" valign="top">Current annotations</th><th colspan="2" valign="top">Annotations of alternative protein coding sequences</th></tr><tr><th>mRNAs</th><th>Others1*</th><th>CDSs</th><th>Proteins</th><th>AltORFs</th><th>Alternative proteins</th></tr></thead><tbody><tr><td valign="top"><italic>H. sapiens</italic> GRCh38 RefSeq GCF_000001405.26</td><td valign="top">67,765</td><td valign="top">11,755</td><td valign="top">68,264</td><td valign="top">54,498</td><td valign="top">539,134</td><td valign="top">183,191</td></tr><tr><td valign="top"><italic>P. troglodytes</italic> 2.1.4 RefSeq GCF_000001515.6</td><td valign="top">55,034</td><td valign="top">7527</td><td valign="top">55,243</td><td valign="top">41,774</td><td valign="top">416,515</td><td valign="top">161,663</td></tr><tr><td valign="top"><italic>M. musculus</italic> GRCm38p2, RefSeq GCF_000001635.22</td><td valign="top">73,450</td><td valign="top">18,886</td><td valign="top">73,55&#160;1</td><td valign="top">53,573</td><td valign="top">642,203</td><td valign="top">215,472</td></tr><tr><td valign="top"><italic>B. Taurus</italic> UMD3.1.86</td><td valign="top">22,089</td><td valign="top">838</td><td valign="top">22,089</td><td valign="top">21,915</td><td valign="top">79,906</td><td valign="top">73,603</td></tr><tr><td valign="top"><italic>X. tropicalis</italic> Ensembl JGI_4.2</td><td valign="top">28,462</td><td valign="top">4644</td><td valign="top">28,462</td><td valign="top">22,614</td><td valign="top">141,894</td><td valign="top">69,917</td></tr><tr><td valign="top"><italic>D. rerio</italic> <break/>Ensembl ZV10.84</td><td valign="top">44,198</td><td valign="top">8196</td><td valign="top">44,198</td><td valign="top">41,460</td><td valign="top">214,628</td><td valign="top">150,510</td></tr><tr><td valign="top"><italic>D. melanogaster</italic> <break/>RefSeq GCA_000705575.1</td><td valign="top">30,255</td><td valign="top">3474</td><td valign="top">30,715</td><td valign="top">20,995</td><td valign="top">174,771</td><td valign="top">71,705</td></tr><tr><td valign="top"><italic>C. elegans</italic> WBcel235, RefSeq GCF_000002985.6</td><td valign="top">28,653</td><td valign="top">25,256</td><td valign="top">26,458</td><td valign="top">25,750</td><td valign="top">131,830</td><td valign="top">45,603</td></tr><tr><td valign="top"><italic>S. cerevisiae</italic> YJM993_v1, RefSeq GCA_000662435.1</td><td valign="top">5471</td><td valign="top">1463</td><td valign="top">5463</td><td valign="top">5423</td><td valign="top">12,401</td><td valign="top">9492</td></tr></tbody></table><table-wrap-foot><fn><p><sup>*</sup>Other transcripts include miRNAs, rRNAs, ncRNAs, snRNAs, snoRNAs, tRNAs.</p><p><sup>&#8224;</sup>Annotated retained-intron and processed transcripts were classified as mRNAs.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28020-v1.xml">elife-28020-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.28020.004</object-id><label>Table 2.</label><caption><title>Antiviral activity of GRL-121 and &#8722;142 against cHIV<sub>NL4-3</sub><sup>WT</sup> and two HIV-2 strains and their cytotoxicities in vitro.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Drug</th><th colspan="3">Mean IC<sub>50</sub>(nM)&#160;&#177;&#160;SD</th><th rowspan="2">CC<sub>50</sub> (&#956;M)</th><th rowspan="2">Selectivity index*</th></tr><tr><th>cHIV<sub>NL4-3</sub><sup>WT</sup></th><th>HIV-2<sub>ROD</sub></th><th>HIV-2<sub>EHO</sub></th></tr></thead><tbody><tr><td>SQV</td><td>12&#160;&#177;&#160;3</td><td>9.0&#160;&#177;&#160;5.0</td><td>8.8&#160;&#177;&#160;2.1</td><td>33</td><td>2750</td></tr><tr><td>IDV</td><td>18&#160;&#177;&#160;5</td><td>31&#160;&#177;&#160;3</td><td>55&#160;&#177;&#160;25</td><td>75</td><td>4167</td></tr><tr><td>NFV</td><td>23&#160;&#177;&#160;5</td><td>27&#160;&#177;&#160;0.4</td><td>84&#160;&#177;&#160;20</td><td>32</td><td>1391</td></tr><tr><td>RTV</td><td>34&#160;&#177;&#160;10</td><td>136&#160;&#177;&#160;165</td><td>278&#160;&#177;&#160;88</td><td>35</td><td>1.029</td></tr><tr><td>TPV</td><td>330&#160;&#177;&#160;13</td><td>293&#160;&#177;&#160;45</td><td>313&#160;&#177;&#160;48</td><td>34</td><td>103</td></tr><tr><td>APV</td><td>26&#160;&#177;&#160;8</td><td>170&#160;&#177;&#160;82</td><td>305&#160;&#177;&#160;78</td><td>&gt;150</td><td>&gt;4167</td></tr><tr><td>LPV</td><td>13&#160;&#177;&#160;8</td><td>40&#160;&#177;&#160;28</td><td>11&#160;&#177;&#160;2</td><td>33</td><td>2538</td></tr><tr><td>ATV</td><td>4.0&#160;&#177;&#160;2.3</td><td>28&#160;&#177;&#160;6</td><td>10&#160;&#177;&#160;8</td><td>80</td><td>20,000</td></tr><tr><td>DRV</td><td>3.2&#160;&#177;&#160;0.7</td><td>8.5&#160;&#177;&#160;0.7</td><td>6.2&#160;&#177;&#160;0.7</td><td>133</td><td>41,562</td></tr><tr><td>GRL-121</td><td>0.26&#160;&#177;&#160;0.05</td><td>0.020&#160;&#177;&#160;0.014</td><td>0.071&#160;&#177;&#160;0.071</td><td>34</td><td>130,769</td></tr><tr><td>GRL-142</td><td>0.019&#160;&#177;&#160;0.017</td><td>0.00032&#160;&#177;&#160;0.00015</td><td>0.000059&#160;&#177;&#160;0.000025</td><td>47</td><td>2,473,684</td></tr></tbody></table><table-wrap-foot><fn><p>*Each selectivity index denotes a ratio of CC<sub>50</sub> to IC<sub>50</sub> against cHIV<sub>NL4-3</sub><sup>WT</sup>.</p><p>The data shown represent mean values (&#177;1 standard deviation) derived from the results of three independent experiments.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28020-v1.xml">elife-28020-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table5" position="float"><object-id pub-id-type="doi">10.7554/eLife.28020.009</object-id><label>Table 5.</label><caption><title>van der Waals interaction energies between GRL-142 or DRV with the protease dimer and selected active site residues.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th>GRL-142</th><th>DRV</th></tr><tr><th>(kcal/mol)</th><th>(kcal/mol)</th></tr></thead><tbody><tr><td>Protease dimer</td><td>&#8722;67.5</td><td>&#8722;57.8</td></tr><tr><td>Asp29 &amp; Asp29&#8217;</td><td>&#8722;7.4</td><td>&#8722;4.2</td></tr><tr><td>Asp30 &amp; Asp30&#8217;</td><td>&#8722;5.4</td><td>&#8722;2.9</td></tr><tr><td>Val82 &amp; Val82&#8217;</td><td>&#8722;2.2</td><td>&#8722;2.3</td></tr><tr><td>Ile84 &amp; Ile84&#8217;</td><td>&#8722;4.3</td><td>&#8722;4.6</td></tr><tr><td>Ile47 &amp; Ile47&#8217;</td><td>&#8722;6.5</td><td>&#8722;3.6</td></tr><tr><td>Gly48 &amp; Gly48&#8217;</td><td>&#8722;3.3</td><td>&#8722;2.8</td></tr><tr><td>Gly49 &amp; Gly49&#8217;</td><td>&#8722;3.7</td><td>&#8722;3.3</td></tr><tr><td>Ile50 &amp; Ile50&#8217;</td><td>&#8722;8.7</td><td>&#8722;9.2</td></tr><tr><td>Pro81 &amp; Pro81&#8217;</td><td>&#8722;1.6</td><td>&#8722;1.9</td></tr><tr><td>Arg8 &amp; Arg8&#8217;</td><td>&#8722;2.2</td><td>&#8722;2</td></tr></tbody></table><table-wrap-foot><fn><p>Average van der Waals energies were calculated by analyzing the trajectories from a 1.2 ns molecular dynamics simulation using Desmond molecular dynamics system (D.E. Shaw Research, New York, NY 2017).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28257-v2.xml">elife-28257-v2.xml</a></h2><table-wrap xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="tbl2" position="float"><object-id pub-id-type="doi">10.7554/eLife.28257.016</object-id><label>Table 2.</label><caption><p>Specific pairwise overlap between activator C-terminal BioID datasets. Twelve activator BioID datasets were contrasted (6 N-terminal and 6 C-terminal) to determine for each activator which of its C-terminal hits were specifically shared with other datasets. Those shared with N-terminal datasets were removed. The specific pairwise overlap of the remaining hits with each activator is reported (<italic>n</italic> and %). Input <italic>n</italic> = BICD1 (92), BICD2 (87), HOOK1 (39), HOOK3 (37), NIN (74), NINL (53). Only pairwise overlap is represented in this analysis; overlap with multiple activators (e.g. BICD1 overlap with both BICD2 and HOOK1) is not shown.</p><p><bold>DOI:</bold> <ext-link ext-link-type="doi" xlink:href="10.7554/eLife.28257.016">http://dx.doi.org/10.7554/eLife.28257.016</ext-link></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><break/></th><th colspan="2" valign="top"><bold>BICD1</bold></th><th colspan="2" valign="top"><bold>BICD2</bold></th><th colspan="2" valign="top"><bold>HOOK1</bold></th><th colspan="2" valign="top"><bold>HOOK3</bold></th><th colspan="2" valign="top"><bold>NIN</bold></th><th colspan="2" valign="top"><bold>NINL</bold></th></tr><tr><th valign="top"><italic>N</italic></th><th valign="top">%</th><th valign="top"><italic>N</italic></th><th valign="top">%</th><th valign="top"><italic>N</italic></th><th valign="top">%</th><th valign="top"><italic>N</italic></th><th valign="top">%</th><th valign="top"><italic>N</italic></th><th valign="top">%</th><th valign="top"><italic>N</italic></th><th valign="top">%</th></tr></thead><tbody><tr><td valign="top">BICD1</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">44</td><td valign="bottom">50.6</td><td valign="bottom">4</td><td valign="bottom">10.3</td><td valign="bottom">0</td><td valign="bottom">0</td><td valign="bottom">5</td><td valign="bottom">6.8</td><td valign="bottom">5</td><td valign="bottom">9.4</td></tr><tr><td valign="top">BICD2</td><td valign="bottom">44</td><td valign="bottom">47.8</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">4</td><td valign="bottom">10.3</td><td valign="bottom">7</td><td valign="bottom">18.9</td><td valign="bottom">6</td><td valign="bottom">8.1</td><td valign="bottom">0</td><td valign="bottom">0</td></tr><tr><td valign="top">HOOK1</td><td valign="bottom">4</td><td valign="bottom">4.3</td><td valign="bottom">4</td><td valign="bottom">4.6</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">14</td><td valign="bottom">37.8</td><td valign="bottom">3</td><td valign="bottom">4.1</td><td valign="bottom">1</td><td valign="bottom">1.9</td></tr><tr><td valign="top">HOOK3</td><td valign="bottom">0</td><td valign="bottom">0</td><td valign="bottom">7</td><td valign="bottom">8.0</td><td valign="bottom">14</td><td valign="bottom">35.9</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">2</td><td valign="bottom">2.7</td><td valign="bottom">1</td><td valign="bottom">1.9</td></tr><tr><td valign="top">NIN</td><td valign="bottom">5</td><td valign="bottom">5.4</td><td valign="bottom">6</td><td valign="bottom">6.9</td><td valign="bottom">3</td><td valign="bottom">7.7</td><td valign="bottom">2</td><td valign="bottom">5.4</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">27</td><td valign="bottom">50.9</td></tr><tr><td valign="top">NINL</td><td valign="bottom">5</td><td valign="bottom">5.4</td><td valign="bottom">0</td><td valign="bottom">0</td><td valign="bottom">1</td><td valign="bottom">2.6</td><td valign="bottom">1</td><td valign="bottom">2.7</td><td valign="bottom">27</td><td valign="bottom">36.5</td><td valign="bottom"/><td valign="bottom"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28275-v2.xml">elife-28275-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.28275.008</object-id><label>Table 2.</label><caption><title>Number of overlapping Bcd ChIP peaks and Vienna Tile-GAL4 enhancer reporters.</title><p>Note that some Bcd peaks overlap with more than one Vienna Tile, and vice versa. The reporters expressed at stage 4&#8211;6 that overlapped with more than one Bcd peak were excluded from the plot in <xref ref-type="fig" rid="fig2">Figure 2B</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="5">Overlaps with Vienna Tile-GAL4 reporters</th></tr><tr><th rowspan="2"/><th rowspan="2">Total Vienna tiles</th><th colspan="2">Total overlaps</th><th rowspan="2" valign="top">Single overlaps</th></tr><tr><th>Vienna tiles</th><th>Bcd peaks</th></tr></thead><tbody><tr><td>Active <break/>(all stages)</td><td>3604</td><td>238</td><td>193</td><td>127</td></tr><tr><td>Active <break/>(stage 4&#8211;6)</td><td>666</td><td>163</td><td>151</td><td>115</td></tr><tr><td>Patterned (stage 4&#8211;6)</td><td>627*</td><td>159</td><td>147</td><td>112</td></tr><tr><td>Not Active</td><td>4189</td><td>55</td><td>41</td><td>28</td></tr><tr><td>Total</td><td>7793</td><td>293</td><td>234</td><td>155</td></tr></tbody></table><table-wrap-foot><fn><p>*The patterned expression from the 666 Vienna Tiles that drive expression at stage 4&#8211;6 was determined by subtracting the number of tiles scored as &#8216;ubiquitous&#8217; (39) from the total number of tiles.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28275-v2.xml">elife-28275-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.28275.009</object-id><label>Table 3.</label><caption><title>Number of Bcd ChIP-seq peaks in each class overlapping with Vienna Tile-GAL4 reporters.</title><p>Queried Bcd peaks are peaks that are present in the Fly Enhancer database, whose expression can be assessed.</p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="7">Overlaps of peak classes with active enhancers and transcription start sites</th></tr><tr><th rowspan="2">Bcd peak class</th><th colspan="2">Overlaps with Vienna tiles</th><th colspan="3">Overlaps with Vienna tiles active at stg. 4&#8211;6</th><th rowspan="2">Total bcd peaks</th></tr><tr><th>N</th><th>% of Total Bcd Peaks</th><th>N</th><th>% of Queried Bcd Peaks</th><th>% of Total Bcd Peaks</th></tr></thead><tbody><tr><td>Concentration-Sensitive I</td><td>52</td><td>34.2%</td><td>38</td><td>73.1%</td><td>25.0%</td><td>152</td></tr><tr><td>Concentration-Sensitive II</td><td>38</td><td>27.5%</td><td>29</td><td>73.7%</td><td>21.0%</td><td>138</td></tr><tr><td>Concentration-Sensitive III</td><td>109</td><td>18.4%</td><td>57</td><td>52.3%</td><td>9.6%</td><td>593</td></tr><tr><td>Concentration-Insensitive</td><td>34</td><td>23.8%</td><td>26</td><td>76.5%</td><td>18.2%</td><td>143</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28275-v2.xml">elife-28275-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.28275.010</object-id><label>Table 4.</label><caption><title>Number of Bcd ChIP-seq peaks in each class within 500 bp Transcription Start Sites.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Bcd peak class</th><th colspan="2">Overlaps with TSS</th><th rowspan="2">Total bcd peaks</th></tr><tr><th>N</th><th>%</th></tr></thead><tbody><tr><td>Concentration-Sensitive I</td><td>29</td><td>19.1%</td><td>152</td></tr><tr><td>Concentration-Sensitive II</td><td>35</td><td>25.4%</td><td>138</td></tr><tr><td>Concentration-Sensitive III</td><td>291</td><td>49.1%</td><td>593</td></tr><tr><td>Concentration-Insensitive</td><td>74</td><td>51.7%</td><td>143</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28298-v2.xml">elife-28298-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.28298.028</object-id><label>Table 2.</label><caption><title>Effect of early growth (<italic>t<sup>lag</sup></italic>, <italic>&#181;</italic> and <italic>&#963;<sub>b</sub></italic>) and control (<italic>t&#772;<sup>c</sup></italic>, <italic>V<sup>c</sup></italic> and <italic>n<sup>tip</sup></italic>) parameters on the capacity of the model to predict survival.</title><p>Log-likelihood is computed either on bacterial load data (i.e. on the data set we used to fit the model) or on survival data. In this latter case, we used the probability of control predicted by the model as an offset in a binomial general linear model (glm) fitted to survival data. This glm includes an intercept which quantifies the difference between predicted control probability and observed survival. p-values indicate whether keeping a parameter constant for all bacteria significantly alters survival prediction.</p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2" valign="top"/><th colspan="3" valign="top">Complete data set</th><th colspan="3" valign="top">Intermediate survival</th></tr><tr><th valign="top">Bacterial load</th><th colspan="2" valign="top">Survival</th><th valign="top">Bacterial load</th><th colspan="2" valign="top">Survival</th></tr></thead><tbody><tr><td valign="top"/><td valign="top">df</td><td valign="top">logLik</td><td valign="top">logLik</td><td valign="top">p-value</td><td valign="top">logLik</td><td valign="top">logLik</td><td valign="top">p-value</td></tr><tr><td valign="top">Full</td><td valign="top"/><td valign="top">&#8722;1243.119</td><td valign="top">&#8722;17.85860</td><td valign="top"/><td valign="top">&#8722;800.0603</td><td valign="top">&#8722;11.48770</td><td valign="top"/></tr><tr><td valign="top">Control</td><td valign="top">24</td><td valign="top">&#8722;1629.965</td><td valign="top">&#8722;21.68408</td><td valign="top">0.999</td><td valign="top">&#8722;839.5523</td><td valign="top">&#8722;17.18319</td><td valign="top">0.496</td></tr><tr><td><italic>t&#772;<sup>c</sup></italic></td><td valign="top">8</td><td valign="top">&#8722;1660.396</td><td valign="top">&#8722;19.18941</td><td valign="top">0.954</td><td valign="top"><bold>&#8722;839.5791</bold></td><td valign="top"><bold>&#8722;16.82534</bold></td><td valign="top"><bold>0.030</bold></td></tr><tr><td><italic>V<sup>c</sup></italic></td><td valign="top">8</td><td valign="top">&#8722;1643.985</td><td valign="top">&#8722;17.37663</td><td valign="top">1</td><td valign="top">&#8722;826.4296</td><td valign="top">&#8722;14.67760</td><td valign="top">0.173</td></tr><tr><td><italic>n<sup>tip</sup></italic></td><td valign="top">8</td><td valign="top">&#8722;1634.382</td><td valign="top">&#8722;25.23397</td><td valign="top">0.064</td><td valign="top">&#8722;832.8896</td><td valign="top">&#8722;12.85596</td><td valign="top">0.603</td></tr><tr><td valign="top">Growth</td><td valign="top">24</td><td valign="top"><bold>&#8722;1798.630</bold></td><td valign="top"><bold>&#8722;81.13355</bold></td><td valign="top"><bold>6.512e-16</bold></td><td valign="top">&#8722;854.1100</td><td valign="top">&#8722;11.37739</td><td valign="top">1</td></tr><tr><td><italic>t<sup>lag</sup></italic></td><td valign="top">8</td><td valign="top">&#8722;1692.743</td><td valign="top">&#8722;16.49076</td><td valign="top">1</td><td valign="top">&#8722;835.3833</td><td valign="top">&#8722;11.30397</td><td valign="top">1</td></tr><tr><td><italic>&#181;</italic></td><td valign="top">8</td><td valign="top"><bold>&#8722;1705.175</bold></td><td valign="top"><bold>&#8722;61.56015</bold></td><td valign="top"><bold>1.564e-15</bold></td><td valign="top">&#8722;836.3794</td><td valign="top">&#8722;11.75406</td><td valign="top">0.970</td></tr><tr><td><italic>&#963;<sub>b</sub></italic></td><td valign="top">8</td><td valign="top">&#8722;1704.156</td><td valign="top">&#8722;21.95270</td><td valign="top">0.415</td><td valign="top">&#8722;848.2682</td><td valign="top">&#8722;12.28487</td><td valign="top">0.810</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28477-v2.xml">elife-28477-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.28477.007</object-id><label>Table 1.</label><caption><title>Summary of sequence data used in this paper.</title><p>Putative subfamilies within each TcR alpha V family share at least 85% nucleotide identity using nearest-neighbor consensus trees of V segments. Number of TcR alpha nucleotide (NUC), amino acid (AA) sequences or sequence groups within each category. Highlighted columns specifically refer to data used in this study. (See results for detailed descriptions of sequences included within each column.)</p><p><supplementary-material id="table1sdata1"><object-id pub-id-type="doi">10.7554/eLife.28477.008</object-id><label>Table 1&#8211;source data 1.</label><caption><title>Tab-delimited text file containing sequences used in this paper for analysis of somatic hypermutation of TcR alpha chain.</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-28477-table1-data1-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Alpha V segment</th><th rowspan="2">Putative # Sub-families</th><th rowspan="2">All cloned sequences</th><th rowspan="2">Complete CDR3-J junction<sup>*</sup></th><th colspan="2">Unique V Region <sup>&#8224;</sup></th><th colspan="2">Unique V segment<sup>&#8225;</sup></th><th colspan="2">Unique CDR3-J<sup>&#167;</sup></th><th rowspan="2">Groups with identical CDR3-J<sup>#</sup></th><th rowspan="2">CDR3-J groups in Study<sup>**</sup></th><th rowspan="2">Sequences in each dataset<sup>&#8224;&#8224;</sup></th></tr><tr><th>NUC</th><th>AA</th><th>NUC</th><th>AA</th><th>NUC</th><th>AA</th></tr></thead><tbody><tr><td>TRA V1</td><td>1</td><td>40</td><td>40</td><td>35</td><td>34</td><td>24</td><td>16</td><td>34</td><td>32</td><td>7</td><td>4</td><td>2, 3, 3, 5</td></tr><tr><td>TRA V2</td><td>3</td><td>18</td><td>18</td><td>13</td><td>13</td><td>13</td><td>10</td><td>12</td><td>12</td><td>3</td><td>1</td><td>5</td></tr><tr><td>TRA V3</td><td>3</td><td>217</td><td>194</td><td>55</td><td>52</td><td>35</td><td>34</td><td>51</td><td>50</td><td>5</td><td>1</td><td>4</td></tr><tr><td>TRA V4</td><td>3</td><td>60</td><td>28</td><td>22</td><td>22</td><td>21</td><td>21</td><td>21</td><td>21</td><td>6</td><td>2</td><td>2, 2</td></tr><tr><td>TRA V5</td><td>4</td><td>35</td><td>34</td><td>15</td><td>13</td><td>9</td><td>8</td><td>13</td><td>13</td><td>3</td><td>1</td><td>2</td></tr><tr><td>TRA V6</td><td>2</td><td>9</td><td>9</td><td>7</td><td>7</td><td>5</td><td>6</td><td>7</td><td>7</td><td>1</td><td>0</td><td>0</td></tr><tr><td>TRA V7</td><td>5</td><td>96</td><td>60</td><td>49</td><td>48</td><td>39</td><td>38</td><td>48</td><td>48</td><td>9</td><td>2</td><td>2, 2</td></tr><tr><td>TRA V9</td><td>3</td><td>19</td><td>19</td><td>14</td><td>14</td><td>12</td><td>11</td><td>13</td><td>13</td><td>4</td><td>0</td><td>0</td></tr><tr><td>TRA V10</td><td>2</td><td>45</td><td>45</td><td>29</td><td>26</td><td>21</td><td>19</td><td>27</td><td>26</td><td>10</td><td>2</td><td>4, 9</td></tr><tr><td/><td/><td>539</td><td>447</td><td>239</td><td>229</td><td>179</td><td>163</td><td>226</td><td>224</td><td>48</td><td><bold>13</bold></td><td><bold>45</bold></td></tr></tbody></table><table-wrap-foot><fn><p><sup>*</sup>A full list of these sequences can be found in <xref ref-type="supplementary-material" rid="table1sdata1">Table 1&#8212;source data 1</xref>.</p><p><sup>&#8224;</sup><bold>V Region</bold> includes all bases between the 1 st predicted nucleotide of the V segment to the last predicted nucleotide of the J segment (V and J).</p></fn><fn><p><bold>&#8225;V Segment</bold> includes all bases between the 1 st predicted nucleotide of the V segment to the last predicted nucleotide of the V segment (V only).</p><p><sup>&#167;</sup><bold>CDR3-J</bold> includes all bases after the last predicted nucleotide of the V segment to the last predicted nucleotide of the J segment.</p></fn><fn><p>#Number of groups with identical CDR3-J sequences, which we used to determine sequence relatedness (see text for details).</p><p>**Number of groups with identical CDR3-J sequences used in this study. (Those not used contained no mutation with V segments.)</p></fn><fn><p>&#8224;&#8224;Total number of sequences for each alpha V used to assess somatic hypermutation within this study (e.g., for aV1, 4 different clonal groups contained 2, 3, 3, and 5 identical CDR3-J regions, respectively).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28477-v2.xml">elife-28477-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.28477.010</object-id><label>Table 2.</label><caption><title>Frequencies of somatic hypermutation in nurse shark alpha V groups (&#945;V G) containing the same CDR3.</title><p>Mutation frequency is the total number of nucleotide changes to a Geneious-derived consensus sequence divided by the total number of nucleotides. We counted synonymous (S) and nonsynymous (N) mutations separately for each FR and CDR for 11 different CDR3 groups in seven predicted alpha V genes. [FR: framework region; CDR: complementarity-determining region; Seqs: sequences; Nuc: nucleotides; Freq: frequency]</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left" valign="bottom"/><th align="left" valign="bottom"/><th align="left" colspan="6" valign="middle"><bold>FR1</bold></th><th align="left" colspan="6" valign="middle"><bold>FR2</bold></th><th align="left" colspan="6" valign="middle"><bold>FR3</bold></th><th align="center" colspan="3" rowspan="2" valign="middle"><bold>FR Means</bold></th></tr><tr><th align="center" rowspan="2" valign="middle"><bold>&#945;V Group</bold></th><th align="center" rowspan="2" valign="bottom"><bold># Seqs</bold></th><th align="center" rowspan="2" valign="middle"><bold># Codons</bold></th><th align="center" valign="bottom"><bold>Total</bold></th><th align="center" colspan="2" valign="middle"><bold>Mutations</bold></th><th align="center" valign="middle"><bold>Total</bold></th><th align="center" valign="bottom"><bold>Mutation</bold></th><th align="center" rowspan="2" valign="middle"><bold># Codons</bold></th><th align="center" valign="bottom"><bold>Total</bold></th><th align="center" colspan="2" valign="middle"><bold>Mutations</bold></th><th align="center" valign="middle"><bold>Total</bold></th><th align="center" valign="bottom"><bold>Mutation</bold></th><th align="center" rowspan="2" valign="middle"><bold># Codons</bold></th><th align="center" valign="bottom"><bold>Total</bold></th><th align="center" colspan="2" valign="middle"><bold>Mutations</bold></th><th align="center" valign="middle"><bold>Total</bold></th><th align="center" valign="bottom"><bold>Mutation</bold></th></tr><tr><th align="center" valign="bottom"><bold>Nuc</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>Mutation</bold></th><th align="center" valign="bottom"><bold>Freq (%)</bold></th><th align="center" valign="bottom"><bold>Nuc</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>Mutation</bold></th><th align="center" valign="bottom"><bold>Freq (%)</bold></th><th align="center" valign="bottom"><bold>Nuc</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>Mutation</bold></th><th align="center" valign="bottom"><bold>Freq (%)</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>ALL</bold></th></tr></thead><tbody><tr><td align="left" valign="middle">&#945;V1.1</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">225</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">12</td><td align="center" valign="bottom">14</td><td align="center" valign="bottom"><bold>6.222</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">153</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>1.307</bold></td><td align="center" valign="bottom">41</td><td align="center" valign="bottom">369</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom"><bold>2.168</bold></td><td align="center" valign="middle">6</td><td align="center" valign="middle">18</td><td align="center" valign="middle">24</td></tr><tr><td align="left" valign="middle">&#945;V1.2</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">375</td><td align="center" valign="bottom">10</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">18</td><td align="center" valign="bottom"><bold>4.800</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">255</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom"><bold>3.529</bold></td><td align="center" valign="bottom">41</td><td align="center" valign="bottom">615</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">10</td><td align="center" valign="bottom"><bold>1.626</bold></td><td align="center" valign="middle">16</td><td align="center" valign="middle">21</td><td align="center" valign="middle">37</td></tr><tr><td align="left" valign="middle">&#945;V1.4</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">225</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>0.444</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">153</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>2.614</bold></td><td align="center" valign="bottom">41</td><td align="center" valign="bottom">369</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom"><bold>2.439</bold></td><td align="center" valign="middle">6</td><td align="center" valign="middle">8</td><td align="center" valign="middle">14</td></tr><tr><td align="left" valign="middle">&#945;V2.1</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">375</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">255</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>0.392</bold></td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">630</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="middle">1</td><td align="center" valign="middle">0</td><td align="center" valign="middle">1</td></tr><tr><td align="left" valign="middle">&#945;V3.1</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">300</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>1.333</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">204</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">40</td><td align="center" valign="bottom">480</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>0.208</bold></td><td align="center" valign="middle">0</td><td align="center" valign="middle">5</td><td align="center" valign="middle">5</td></tr><tr><td align="left" valign="middle">&#945;V4.1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">150</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>2.000</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">102</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>1.961</bold></td><td align="center" valign="bottom">43</td><td align="center" valign="bottom">258</td><td align="center" valign="bottom">11</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">18</td><td align="center" valign="bottom"><bold>6.977</bold></td><td align="center" valign="middle">12</td><td align="center" valign="middle">11</td><td align="center" valign="middle">23</td></tr><tr><td align="left" valign="middle">&#945;V4.2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">150</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">102</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>1.961</bold></td><td align="center" valign="bottom">43</td><td align="center" valign="bottom">258</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom"><bold>3.488</bold></td><td align="center" valign="middle">3</td><td align="center" valign="middle">8</td><td align="center" valign="middle">11</td></tr><tr><td align="left" valign="middle">&#945;V5.1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">150</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">6</td><td align="center" valign="bottom"><bold>4.000</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">102</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom"><bold>4.902</bold></td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">252</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>1.190</bold></td><td align="center" valign="middle">3</td><td align="center" valign="middle">11</td><td align="center" valign="middle">14</td></tr><tr><td align="left" valign="middle">&#945;V7.2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">150</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">11</td><td align="center" valign="bottom"><bold>7.333</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">102</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>3.922</bold></td><td align="center" valign="bottom">41</td><td align="center" valign="bottom">246</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom"><bold>2.846</bold></td><td align="center" valign="middle">6</td><td align="center" valign="middle">16</td><td align="center" valign="middle">22</td></tr><tr><td align="left" valign="middle">&#945;V10.1</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">300</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">204</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>0.980</bold></td><td align="center" valign="bottom">41</td><td align="center" valign="bottom">492</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>0.813</bold></td><td align="center" valign="middle">5</td><td align="center" valign="middle">1</td><td align="center" valign="middle">6</td></tr><tr><td align="left" valign="middle">&#945;V10.2</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">675</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">11</td><td align="center" valign="bottom">13</td><td align="center" valign="bottom"><bold>1.926</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">459</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">4</td><td align="center" valign="middle">8</td><td align="center" valign="bottom"><bold>1.743</bold></td><td align="center" valign="bottom">40</td><td align="center" valign="bottom">1080</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">11</td><td align="center" valign="bottom">14</td><td align="center" valign="bottom"><bold>1.296</bold></td><td align="center" valign="middle">9</td><td align="center" valign="middle">26</td><td align="center" valign="middle">35</td></tr><tr><td align="left" valign="middle"><bold>Sum</bold></td><td align="center" valign="bottom"><bold>41</bold></td><td align="center" valign="bottom"><bold>275</bold></td><td align="center" valign="bottom"><bold>3075</bold></td><td align="center" valign="bottom"><bold>20</bold></td><td align="center" valign="bottom"><bold>50</bold></td><td align="center" valign="bottom"><bold>70</bold></td><td align="center" valign="bottom"/><td align="center" valign="bottom"><bold>187</bold></td><td align="center" valign="bottom"><bold>2091</bold></td><td align="center" valign="bottom"><bold>12</bold></td><td align="center" valign="bottom"><bold>27</bold></td><td align="center" valign="bottom"><bold>39</bold></td><td align="center" valign="bottom"/><td align="center" valign="bottom"><bold>455</bold></td><td align="center" valign="bottom"><bold>5049</bold></td><td align="center" valign="bottom"><bold>35</bold></td><td align="center" valign="bottom"><bold>48</bold></td><td align="center" valign="bottom"><bold>83</bold></td><td align="center" valign="bottom"/><td align="center" valign="middle"><bold>67</bold></td><td align="center" valign="middle"><bold>125</bold></td><td align="center" valign="middle"><bold>192</bold></td></tr><tr><td align="left" colspan="4" valign="middle"><bold>Mean Mutation Freq (%)</bold></td><td align="center" valign="middle"><bold>0.65</bold></td><td align="center" valign="middle"><bold>1.63</bold></td><td align="center" valign="middle"><bold>2.28</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>0.57</bold></td><td align="center" valign="middle"><bold>1.29</bold></td><td align="center" valign="middle"><bold>1.87</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>0.69</bold></td><td align="center" valign="middle"><bold>0.95</bold></td><td align="center" valign="middle"><bold>1.64</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"><bold>0.66</bold></td><td align="center" valign="middle"><bold>1.22</bold></td><td align="center" valign="middle"><bold>1.88</bold></td></tr><tr><td align="left" colspan="3" valign="middle"><bold>Standard Deviation</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"><bold>2.994</bold></td><td align="center" valign="middle"><bold>4.547</bold></td><td align="center" valign="middle"><bold>6.531</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>1.136</bold></td><td align="center" valign="middle"><bold>2.252</bold></td><td align="center" valign="middle"><bold>2.841</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>3.125</bold></td><td align="center" valign="middle"><bold>3.414</bold></td><td align="center" valign="middle"><bold>5.429</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"><bold>4.742</bold></td><td align="center" valign="middle"><bold>8.201</bold></td><td align="center" valign="middle"><bold>11.861</bold></td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th align="left" valign="bottom"/><th align="left" valign="bottom"/><th align="left" colspan="6" valign="bottom"><bold>CDR1</bold></th><th align="left" colspan="6" valign="middle"><bold>CDR2</bold></th><th align="left" colspan="6" valign="middle"><bold>CDR3</bold></th><th align="center" colspan="3" rowspan="2" valign="middle"><bold>CDR Means</bold></th></tr><tr><th align="center" rowspan="2" valign="middle"><bold>AlphaV Group</bold></th><th align="center" rowspan="2" valign="bottom"><bold># Seqs</bold></th><th align="center" rowspan="2" valign="middle"><bold># Codons</bold></th><th align="center" valign="bottom"><bold>Total</bold></th><th align="center" colspan="2" valign="middle"><bold>Mutations</bold></th><th align="center" valign="middle"><bold>Total</bold></th><th align="center" valign="bottom"><bold>Mutation</bold></th><th align="center" rowspan="2" valign="middle"><bold># Codons</bold></th><th align="center" valign="bottom"><bold>Total</bold></th><th align="center" colspan="2" valign="middle"><bold>Mutations</bold></th><th align="center" valign="middle"><bold>Total</bold></th><th align="center" valign="bottom"><bold>Mutation</bold></th><th align="center" rowspan="2" valign="middle"><bold># Codons</bold></th><th align="center" valign="bottom"><bold>Total</bold></th><th align="center" colspan="2" valign="middle"><bold>Mutations</bold></th><th align="center" valign="middle"><bold>Total</bold></th><th align="center" valign="bottom"><bold>Mutation</bold></th></tr><tr><th align="center" valign="bottom"><bold>Nuc</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>Mutation</bold></th><th align="center" valign="bottom"><bold>Freq (%)</bold></th><th align="center" valign="bottom"><bold>Nuc</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>Mutation</bold></th><th align="center" valign="bottom"><bold>Freq (%)</bold></th><th align="center" valign="bottom"><bold>Nuc</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>Mutation</bold></th><th align="center" valign="bottom"><bold>Freq (%)</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>ALL</bold></th></tr></thead><tbody><tr><td align="left" valign="middle">&#945;V1.1</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">72</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>5.556</bold></td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">45</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>2.222</bold></td><td align="center" valign="bottom">10</td><td align="center" valign="bottom">90</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom"><bold>7.778</bold></td><td align="center" valign="middle">4</td><td align="center" valign="middle">8</td><td align="center" valign="middle">12</td></tr><tr><td align="left" valign="middle">&#945;V1.2</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">120</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom"><bold>7.500</bold></td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">75</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>4.000</bold></td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">120</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom"><bold>4.167</bold></td><td align="center" valign="middle">7</td><td align="center" valign="middle">10</td><td align="center" valign="middle">17</td></tr><tr><td align="left" valign="middle">&#945;V1.4</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">72</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>1.389</bold></td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">45</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">45</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>4.444</bold></td><td align="center" valign="middle">0</td><td align="center" valign="middle">3</td><td align="center" valign="middle">3</td></tr><tr><td align="left" valign="middle">&#945;V2.1</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">105</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>0.952</bold></td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">75</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">45</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="middle">0</td><td align="center" valign="middle">1</td><td align="center" valign="middle">1</td></tr><tr><td align="left" valign="middle">&#945;V3.1</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">72</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>4.167</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">72</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">12</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="middle">0</td><td align="center" valign="middle">3</td><td align="center" valign="middle">3</td></tr><tr><td align="left" valign="middle">&#945;V4.1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>9.524</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">36</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>2.778</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">36</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>5.556</bold></td><td align="center" valign="middle">5</td><td align="center" valign="middle">2</td><td align="center" valign="middle">7</td></tr><tr><td align="left" valign="middle">&#945;V4.2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>4.762</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">36</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">36</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="middle">1</td><td align="center" valign="middle">1</td><td align="center" valign="middle">2</td></tr><tr><td align="left" valign="middle">&#945;V5.1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>7.143</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">36</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom"><bold>19.444</bold></td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="middle">0</td><td align="center" valign="middle">10</td><td align="center" valign="middle">10</td></tr><tr><td align="left" valign="middle">&#945;V7.2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">30</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>4.762</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">36</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>5.556</bold></td><td align="center" valign="middle">1</td><td align="center" valign="middle">3</td><td align="center" valign="middle">4</td></tr><tr><td align="left" valign="middle">&#945;V10.1</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">84</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>3.571</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">72</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">84</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>3.571</bold></td><td align="center" valign="middle">3</td><td align="center" valign="middle">3</td><td align="center" valign="middle">6</td></tr><tr><td align="left" valign="middle">&#945;V10.2</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">189</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom"><bold>4.762</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">162</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>2.469</bold></td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">189</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>1.587</bold></td><td align="center" valign="middle">6</td><td align="center" valign="middle">10</td><td align="center" valign="middle">16</td></tr><tr><td align="left" valign="middle"><bold>Sum</bold></td><td align="center" valign="bottom"><bold>41</bold></td><td align="center" valign="bottom"><bold>77</bold></td><td align="center" valign="bottom"><bold>870</bold></td><td align="center" valign="bottom"><bold>15</bold></td><td align="center" valign="bottom"><bold>24</bold></td><td align="center" valign="bottom"><bold>39</bold></td><td align="center" valign="bottom"/><td align="center" valign="bottom"><bold>63</bold></td><td align="center" valign="bottom"><bold>696</bold></td><td align="center" valign="bottom"><bold>5</bold></td><td align="center" valign="bottom"><bold>13</bold></td><td align="center" valign="bottom"><bold>18</bold></td><td align="center" valign="bottom"/><td align="center" valign="bottom"><bold>66</bold></td><td align="center" valign="bottom"><bold>735</bold></td><td align="center" valign="bottom"><bold>7</bold></td><td align="center" valign="bottom"><bold>17</bold></td><td align="center" valign="bottom"><bold>24</bold></td><td align="center" valign="bottom"/><td align="center" valign="middle"><bold>27</bold></td><td align="center" valign="middle"><bold>54</bold></td><td align="center" valign="middle"><bold>81</bold></td></tr><tr><td align="left" colspan="4" valign="middle"><bold>Mean Mutation Freq (%)</bold></td><td align="center" valign="middle"><bold>1.72</bold></td><td align="center" valign="middle"><bold>2.76</bold></td><td align="center" valign="middle"><bold>4.48</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>0.72</bold></td><td align="center" valign="middle"><bold>1.87</bold></td><td align="center" valign="middle"><bold>2.59</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>0.95</bold></td><td align="center" valign="middle"><bold>2.31</bold></td><td align="center" valign="middle"><bold>3.27</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"><bold>1.17</bold></td><td align="center" valign="middle"><bold>2.35</bold></td><td align="center" valign="middle"><bold>3.52</bold></td></tr><tr><td align="left" colspan="3" valign="middle"><bold>Standard Deviation</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"><bold>1.690</bold></td><td align="center" valign="middle"><bold>2.316</bold></td><td align="center" valign="middle"><bold>2.979</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>0.688</bold></td><td align="center" valign="middle"><bold>2.089</bold></td><td align="center" valign="middle"><bold>2.248</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>1.027</bold></td><td align="center" valign="middle"><bold>2.067</bold></td><td align="center" valign="middle"><bold>2.272</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"><bold>2.659</bold></td><td align="center" valign="middle"><bold>3.754</bold></td><td align="center" valign="middle"><bold>5.626</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28477-v2.xml">elife-28477-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.28477.011</object-id><label>Table 3.</label><caption><title>Number and frequency of DNA mutations that occur in tandem within framework regions (FR) and complementarity determining regions (CDR) in nurse shark alpha V (&#945;V) groups.</title><p>All mutations include both tandem and point mutations within a region. [<italic>Seqs:</italic> sequences]</p></caption><table frame="hsides" rules="groups"><thead><tr><th/><th/><th colspan="6">FR</th><th colspan="6">CDR</th></tr><tr><th/><th/><th colspan="4"># Nucleotides Tandemly Mutated</th><th rowspan="2">All mutation</th><th rowspan="2">Frequency of tandem mutation</th><th colspan="4"># Nucleotides Tandemly Mutated</th><th rowspan="2">All mutation</th><th rowspan="2">Frequency of tandem mutation</th></tr><tr><th>&#945;V Group</th><th># Seqs</th><th>2</th><th>3</th><th>4</th><th>Sum</th><th>2</th><th>3</th><th>4</th><th>Sum</th></tr></thead><tbody><tr><td>&#945;V1.1</td><td valign="bottom">3</td><td>1</td><td>0</td><td>1</td><td>6</td><td>24</td><td>25.0</td><td>3</td><td>0</td><td>0</td><td>6</td><td>12</td><td>50.0</td></tr><tr><td>&#945;V1.2</td><td valign="bottom">5</td><td>4</td><td>1</td><td>0</td><td>11</td><td>37</td><td>29.7</td><td>5</td><td>1</td><td>0</td><td>13</td><td>17</td><td>76.5</td></tr><tr><td>&#945;V1.4</td><td valign="bottom">3</td><td>3</td><td>0</td><td>0</td><td>6</td><td>14</td><td>42.9</td><td>0</td><td>0</td><td>0</td><td>0</td><td>3</td><td>0.0</td></tr><tr><td>&#945;V2</td><td valign="bottom">5</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0.0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0.0</td></tr><tr><td>&#945;V3</td><td valign="bottom">4</td><td>2</td><td>0</td><td>0</td><td>4</td><td>5</td><td>80.0</td><td>1</td><td>0</td><td>0</td><td>2</td><td>3</td><td>66.7</td></tr><tr><td>&#945;V4.1</td><td valign="bottom">2</td><td>4</td><td>1</td><td>0</td><td>11</td><td>23</td><td>47.8</td><td>0</td><td>1</td><td>0</td><td>3</td><td>7</td><td>42.9</td></tr><tr><td>&#945;V4.2</td><td valign="bottom">2</td><td>1</td><td>1</td><td>0</td><td>5</td><td>11</td><td>45.5</td><td>0</td><td>1</td><td>0</td><td>3</td><td>2</td><td>150.0</td></tr><tr><td>&#945;V5</td><td valign="bottom">2</td><td>1</td><td>0</td><td>0</td><td>2</td><td>14</td><td>14.3</td><td>1</td><td>1</td><td>0</td><td>5</td><td>10</td><td>50.0</td></tr><tr><td>&#945;V7.2</td><td valign="bottom">2</td><td>4</td><td>1</td><td>0</td><td>11</td><td>22</td><td>50.0</td><td>1</td><td>0</td><td>0</td><td>2</td><td>4</td><td>50.0</td></tr><tr><td>&#945;V10.1</td><td valign="bottom">4</td><td>3</td><td>0</td><td>0</td><td>6</td><td>6</td><td>100.0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>6</td><td>0.0</td></tr><tr><td>&#945;V10.2</td><td valign="bottom">9</td><td>4</td><td>1</td><td>0</td><td>11</td><td>35</td><td>31.4</td><td>2</td><td>1</td><td>0</td><td>7</td><td>16</td><td>43.8</td></tr><tr><td>Total</td><td valign="bottom">41</td><td>27</td><td>5</td><td>1</td><td>73</td><td>192</td><td>38.0</td><td>13</td><td>5</td><td>0</td><td>41</td><td>81</td><td>50.6</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28477-v2.xml">elife-28477-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table6" position="float"><object-id pub-id-type="doi">10.7554/eLife.28477.017</object-id><label>Table 6.</label><caption><title>Frequencies of somatic hypermutation in nurse shark thymus and peripheral lymphoid tissue (blood and spiral valve).</title><p>Mutations were analyzed only in alpha V groups containing the same third complementarity-determining region (CDR). Mutation frequency was measured as the total number of nucleotide changes to a Geneious-derived consensus sequence divided by the total number of nucleotides in all sequences. Nonsynymous (N) and synonymous (S) mutations (mut) were counted separately for each framework (FR) and CDR for two predicted alpha V genes. [FR1, FR2, FR3, CDR1, CDR2, and CDR3 refer to the first, second, or third FR or CDR region, respectively.]</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Tissue type</th><th rowspan="2">Mut type</th><th colspan="4">FR Mutations (#)</th><th colspan="4">CDR Mutations (#)</th><th colspan="3">FR mutation frequency</th><th colspan="3">CDR mutation frequency</th></tr><tr><th>&#8195;FR1</th><th>FR2</th><th>FR3</th><th>All FR</th><th>CDR1</th><th>CDR2</th><th>CDR3</th><th>All CDR</th><th>FR1</th><th>FR2</th><th>FR3</th><th>CDR1</th><th>CDR2</th><th>CDR3</th></tr></thead><tbody><tr><td rowspan="3">Thymus <break/>&#8195;(6 sequences) <break/></td><td>N</td><td>8</td><td>8</td><td>7</td><td>23</td><td>1</td><td>3</td><td>3</td><td>7</td><td>0.570</td><td>0.871</td><td>0.317</td><td>0.071</td><td>0.327</td><td>0.136</td></tr><tr><td>S</td><td>8</td><td>2</td><td>6</td><td>16</td><td>1</td><td>0</td><td>0</td><td>1</td><td>0.570</td><td>0.218</td><td>0.272</td><td>0.071</td><td>0.000</td><td>0.000</td></tr><tr><td>ALL</td><td>16</td><td>10</td><td>13</td><td>39</td><td>2</td><td>3</td><td>3</td><td>8</td><td>1.140</td><td>1.089</td><td>0.590</td><td>0.529</td><td>1.075</td><td>1.235</td></tr><tr><td colspan="2">Total Nucleotides</td><td>1404</td><td>918</td><td>2205</td><td>4527</td><td>378</td><td>279</td><td>243</td><td>900</td><td/><td/><td/><td/><td/><td/></tr><tr><td rowspan="3">Periphery <break/>&#8195;(2 sequences) <break/></td><td>N</td><td>7</td><td>4</td><td>6</td><td>17</td><td>0</td><td>2</td><td>2</td><td>4</td><td>1.496</td><td>1.307</td><td>0.833</td><td>0.000</td><td>1.852</td><td>1.587</td></tr><tr><td>S</td><td>4</td><td>0</td><td>1</td><td>5</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0.855</td><td>0.000</td><td>0.139</td><td>0.000</td><td>0.000</td><td>0.000</td></tr><tr><td>ALL</td><td>11</td><td>4</td><td>7</td><td>22</td><td>0</td><td>2</td><td>2</td><td>4</td><td>2.350</td><td>1.307</td><td>0.972</td><td>0.000</td><td>1.852</td><td>1.587</td></tr><tr><td colspan="2">Total Nucleotides</td><td>468</td><td>306</td><td>720</td><td>1494</td><td>126</td><td>108</td><td>126</td><td>360</td><td/><td/><td/><td/><td/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28477-v3.xml">elife-28477-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.28477.007</object-id><label>Table 1.</label><caption><title>Summary of sequence data used in this paper.</title><p>Putative subfamilies within each TcR alpha V family share at least 85% nucleotide identity using nearest-neighbor consensus trees of V segments. Number of TcR alpha nucleotide (NUC), amino acid (AA) sequences or sequence groups within each category. Highlighted columns specifically refer to data used in this study. (See results for detailed descriptions of sequences included within each column.)</p><p><supplementary-material id="table1sdata1"><object-id pub-id-type="doi">10.7554/eLife.28477.008</object-id><label>Table 1&#8211;source data 1.</label><caption><title>Tab-delimited text file containing sequences used in this paper for analysis of somatic hypermutation of TcR alpha chain.</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-28477-table1-data1-v3.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Alpha V segment</th><th rowspan="2">Putative # Sub-families</th><th rowspan="2">All cloned sequences</th><th rowspan="2">Complete CDR3-J junction<sup>*</sup></th><th colspan="2">Unique V Region <sup>&#8224;</sup></th><th colspan="2">Unique V segment<sup>&#8225;</sup></th><th colspan="2">Unique CDR3-J<sup>&#167;</sup></th><th rowspan="2">Groups with identical CDR3-J<sup>#</sup></th><th rowspan="2">CDR3-J groups in Study<sup>**</sup></th><th rowspan="2">Sequences in each dataset<sup>&#8224;&#8224;</sup></th></tr><tr><th>NUC</th><th>AA</th><th>NUC</th><th>AA</th><th>NUC</th><th>AA</th></tr></thead><tbody><tr><td>TRA V1</td><td>1</td><td>40</td><td>40</td><td>35</td><td>34</td><td>24</td><td>16</td><td>34</td><td>32</td><td>7</td><td>4</td><td>2, 3, 3, 5</td></tr><tr><td>TRA V2</td><td>3</td><td>18</td><td>18</td><td>13</td><td>13</td><td>13</td><td>10</td><td>12</td><td>12</td><td>3</td><td>1</td><td>5</td></tr><tr><td>TRA V3</td><td>3</td><td>217</td><td>194</td><td>55</td><td>52</td><td>35</td><td>34</td><td>51</td><td>50</td><td>5</td><td>1</td><td>4</td></tr><tr><td>TRA V4</td><td>3</td><td>60</td><td>28</td><td>22</td><td>22</td><td>21</td><td>21</td><td>21</td><td>21</td><td>6</td><td>2</td><td>2, 2</td></tr><tr><td>TRA V5</td><td>4</td><td>35</td><td>34</td><td>15</td><td>13</td><td>9</td><td>8</td><td>13</td><td>13</td><td>3</td><td>1</td><td>2</td></tr><tr><td>TRA V6</td><td>2</td><td>9</td><td>9</td><td>7</td><td>7</td><td>5</td><td>6</td><td>7</td><td>7</td><td>1</td><td>0</td><td>0</td></tr><tr><td>TRA V7</td><td>5</td><td>96</td><td>60</td><td>49</td><td>48</td><td>39</td><td>38</td><td>48</td><td>48</td><td>9</td><td>2</td><td>2, 2</td></tr><tr><td>TRA V9</td><td>3</td><td>19</td><td>19</td><td>14</td><td>14</td><td>12</td><td>11</td><td>13</td><td>13</td><td>4</td><td>0</td><td>0</td></tr><tr><td>TRA V10</td><td>2</td><td>45</td><td>45</td><td>29</td><td>26</td><td>21</td><td>19</td><td>27</td><td>26</td><td>10</td><td>2</td><td>4, 9</td></tr><tr><td/><td/><td>539</td><td>447</td><td>239</td><td>229</td><td>179</td><td>163</td><td>226</td><td>224</td><td>48</td><td><bold>13</bold></td><td><bold>45</bold></td></tr></tbody></table><table-wrap-foot><fn><p><sup>*</sup>A full list of these sequences can be found in <xref ref-type="supplementary-material" rid="table1sdata1">Table 1&#8212;source data 1</xref>.</p><p><sup>&#8224;</sup><bold>V Region</bold> includes all bases between the 1 st predicted nucleotide of the V segment to the last predicted nucleotide of the J segment (V and J).</p></fn><fn><p><bold>&#8225;V Segment</bold> includes all bases between the 1 st predicted nucleotide of the V segment to the last predicted nucleotide of the V segment (V only).</p><p><sup>&#167;</sup><bold>CDR3-J</bold> includes all bases after the last predicted nucleotide of the V segment to the last predicted nucleotide of the J segment.</p></fn><fn><p>#Number of groups with identical CDR3-J sequences, which we used to determine sequence relatedness (see text for details).</p><p>**Number of groups with identical CDR3-J sequences used in this study. (Those not used contained no mutation with V segments.)</p></fn><fn><p>&#8224;&#8224;Total number of sequences for each alpha V used to assess somatic hypermutation within this study (e.g., for aV1, 4 different clonal groups contained 2, 3, 3, and 5 identical CDR3-J regions, respectively).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28477-v3.xml">elife-28477-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.28477.010</object-id><label>Table 2.</label><caption><title>Frequencies of somatic hypermutation in nurse shark alpha V groups (&#945;V G) containing the same CDR3.</title><p>Mutation frequency is the total number of nucleotide changes to a Geneious-derived consensus sequence divided by the total number of nucleotides. We counted synonymous (S) and nonsynymous (N) mutations separately for each FR and CDR for 11 different CDR3 groups in seven predicted alpha V genes. [FR: framework region; CDR: complementarity-determining region; Seqs: sequences; Nuc: nucleotides; Freq: frequency]</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left" valign="bottom"/><th align="left" valign="bottom"/><th align="left" colspan="6" valign="middle"><bold>FR1</bold></th><th align="left" colspan="6" valign="middle"><bold>FR2</bold></th><th align="left" colspan="6" valign="middle"><bold>FR3</bold></th><th align="center" colspan="3" rowspan="2" valign="middle"><bold>FR Means</bold></th></tr><tr><th align="center" rowspan="2" valign="middle"><bold>&#945;V Group</bold></th><th align="center" rowspan="2" valign="bottom"><bold># Seqs</bold></th><th align="center" rowspan="2" valign="middle"><bold># Codons</bold></th><th align="center" valign="bottom"><bold>Total</bold></th><th align="center" colspan="2" valign="middle"><bold>Mutations</bold></th><th align="center" valign="middle"><bold>Total</bold></th><th align="center" valign="bottom"><bold>Mutation</bold></th><th align="center" rowspan="2" valign="middle"><bold># Codons</bold></th><th align="center" valign="bottom"><bold>Total</bold></th><th align="center" colspan="2" valign="middle"><bold>Mutations</bold></th><th align="center" valign="middle"><bold>Total</bold></th><th align="center" valign="bottom"><bold>Mutation</bold></th><th align="center" rowspan="2" valign="middle"><bold># Codons</bold></th><th align="center" valign="bottom"><bold>Total</bold></th><th align="center" colspan="2" valign="middle"><bold>Mutations</bold></th><th align="center" valign="middle"><bold>Total</bold></th><th align="center" valign="bottom"><bold>Mutation</bold></th></tr><tr><th align="center" valign="bottom"><bold>Nuc</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>Mutation</bold></th><th align="center" valign="bottom"><bold>Freq (%)</bold></th><th align="center" valign="bottom"><bold>Nuc</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>Mutation</bold></th><th align="center" valign="bottom"><bold>Freq (%)</bold></th><th align="center" valign="bottom"><bold>Nuc</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>Mutation</bold></th><th align="center" valign="bottom"><bold>Freq (%)</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>ALL</bold></th></tr></thead><tbody><tr><td align="left" valign="middle">&#945;V1.1</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">225</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">12</td><td align="center" valign="bottom">14</td><td align="center" valign="bottom"><bold>6.222</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">153</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>1.307</bold></td><td align="center" valign="bottom">41</td><td align="center" valign="bottom">369</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom"><bold>2.168</bold></td><td align="center" valign="middle">6</td><td align="center" valign="middle">18</td><td align="center" valign="middle">24</td></tr><tr><td align="left" valign="middle">&#945;V1.2</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">375</td><td align="center" valign="bottom">10</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">18</td><td align="center" valign="bottom"><bold>4.800</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">255</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom"><bold>3.529</bold></td><td align="center" valign="bottom">41</td><td align="center" valign="bottom">615</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">10</td><td align="center" valign="bottom"><bold>1.626</bold></td><td align="center" valign="middle">16</td><td align="center" valign="middle">21</td><td align="center" valign="middle">37</td></tr><tr><td align="left" valign="middle">&#945;V1.4</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">225</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>0.444</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">153</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>2.614</bold></td><td align="center" valign="bottom">41</td><td align="center" valign="bottom">369</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom"><bold>2.439</bold></td><td align="center" valign="middle">6</td><td align="center" valign="middle">8</td><td align="center" valign="middle">14</td></tr><tr><td align="left" valign="middle">&#945;V2.1</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">375</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">255</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>0.392</bold></td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">630</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="middle">1</td><td align="center" valign="middle">0</td><td align="center" valign="middle">1</td></tr><tr><td align="left" valign="middle">&#945;V3.1</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">300</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>1.333</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">204</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">40</td><td align="center" valign="bottom">480</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>0.208</bold></td><td align="center" valign="middle">0</td><td align="center" valign="middle">5</td><td align="center" valign="middle">5</td></tr><tr><td align="left" valign="middle">&#945;V4.1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">150</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>2.000</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">102</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>1.961</bold></td><td align="center" valign="bottom">43</td><td align="center" valign="bottom">258</td><td align="center" valign="bottom">11</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">18</td><td align="center" valign="bottom"><bold>6.977</bold></td><td align="center" valign="middle">12</td><td align="center" valign="middle">11</td><td align="center" valign="middle">23</td></tr><tr><td align="left" valign="middle">&#945;V4.2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">150</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">102</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>1.961</bold></td><td align="center" valign="bottom">43</td><td align="center" valign="bottom">258</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom"><bold>3.488</bold></td><td align="center" valign="middle">3</td><td align="center" valign="middle">8</td><td align="center" valign="middle">11</td></tr><tr><td align="left" valign="middle">&#945;V5.1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">150</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">6</td><td align="center" valign="bottom"><bold>4.000</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">102</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom"><bold>4.902</bold></td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">252</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>1.190</bold></td><td align="center" valign="middle">3</td><td align="center" valign="middle">11</td><td align="center" valign="middle">14</td></tr><tr><td align="left" valign="middle">&#945;V7.2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">150</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">11</td><td align="center" valign="bottom"><bold>7.333</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">102</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>3.922</bold></td><td align="center" valign="bottom">41</td><td align="center" valign="bottom">246</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom"><bold>2.846</bold></td><td align="center" valign="middle">6</td><td align="center" valign="middle">16</td><td align="center" valign="middle">22</td></tr><tr><td align="left" valign="middle">&#945;V10.1</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">300</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">204</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>0.980</bold></td><td align="center" valign="bottom">41</td><td align="center" valign="bottom">492</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>0.813</bold></td><td align="center" valign="middle">5</td><td align="center" valign="middle">1</td><td align="center" valign="middle">6</td></tr><tr><td align="left" valign="middle">&#945;V10.2</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom">25</td><td align="center" valign="bottom">675</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">11</td><td align="center" valign="bottom">13</td><td align="center" valign="bottom"><bold>1.926</bold></td><td align="center" valign="bottom">17</td><td align="center" valign="bottom">459</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">4</td><td align="center" valign="middle">8</td><td align="center" valign="bottom"><bold>1.743</bold></td><td align="center" valign="bottom">40</td><td align="center" valign="bottom">1080</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">11</td><td align="center" valign="bottom">14</td><td align="center" valign="bottom"><bold>1.296</bold></td><td align="center" valign="middle">9</td><td align="center" valign="middle">26</td><td align="center" valign="middle">35</td></tr><tr><td align="left" valign="middle"><bold>Sum</bold></td><td align="center" valign="bottom"><bold>41</bold></td><td align="center" valign="bottom"><bold>275</bold></td><td align="center" valign="bottom"><bold>3075</bold></td><td align="center" valign="bottom"><bold>20</bold></td><td align="center" valign="bottom"><bold>50</bold></td><td align="center" valign="bottom"><bold>70</bold></td><td align="center" valign="bottom"/><td align="center" valign="bottom"><bold>187</bold></td><td align="center" valign="bottom"><bold>2091</bold></td><td align="center" valign="bottom"><bold>12</bold></td><td align="center" valign="bottom"><bold>27</bold></td><td align="center" valign="bottom"><bold>39</bold></td><td align="center" valign="bottom"/><td align="center" valign="bottom"><bold>455</bold></td><td align="center" valign="bottom"><bold>5049</bold></td><td align="center" valign="bottom"><bold>35</bold></td><td align="center" valign="bottom"><bold>48</bold></td><td align="center" valign="bottom"><bold>83</bold></td><td align="center" valign="bottom"/><td align="center" valign="middle"><bold>67</bold></td><td align="center" valign="middle"><bold>125</bold></td><td align="center" valign="middle"><bold>192</bold></td></tr><tr><td align="left" colspan="4" valign="middle"><bold>Mean Mutation Freq (%)</bold></td><td align="center" valign="middle"><bold>0.65</bold></td><td align="center" valign="middle"><bold>1.63</bold></td><td align="center" valign="middle"><bold>2.28</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>0.57</bold></td><td align="center" valign="middle"><bold>1.29</bold></td><td align="center" valign="middle"><bold>1.87</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>0.69</bold></td><td align="center" valign="middle"><bold>0.95</bold></td><td align="center" valign="middle"><bold>1.64</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"><bold>0.66</bold></td><td align="center" valign="middle"><bold>1.22</bold></td><td align="center" valign="middle"><bold>1.88</bold></td></tr><tr><td align="left" colspan="3" valign="middle"><bold>Standard Deviation</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"><bold>2.994</bold></td><td align="center" valign="middle"><bold>4.547</bold></td><td align="center" valign="middle"><bold>6.531</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>1.136</bold></td><td align="center" valign="middle"><bold>2.252</bold></td><td align="center" valign="middle"><bold>2.841</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>3.125</bold></td><td align="center" valign="middle"><bold>3.414</bold></td><td align="center" valign="middle"><bold>5.429</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"><bold>4.742</bold></td><td align="center" valign="middle"><bold>8.201</bold></td><td align="center" valign="middle"><bold>11.861</bold></td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th align="left" valign="bottom"/><th align="left" valign="bottom"/><th align="left" colspan="6" valign="bottom"><bold>CDR1</bold></th><th align="left" colspan="6" valign="middle"><bold>CDR2</bold></th><th align="left" colspan="6" valign="middle"><bold>CDR3</bold></th><th align="center" colspan="3" rowspan="2" valign="middle"><bold>CDR Means</bold></th></tr><tr><th align="center" rowspan="2" valign="middle"><bold>AlphaV Group</bold></th><th align="center" rowspan="2" valign="bottom"><bold># Seqs</bold></th><th align="center" rowspan="2" valign="middle"><bold># Codons</bold></th><th align="center" valign="bottom"><bold>Total</bold></th><th align="center" colspan="2" valign="middle"><bold>Mutations</bold></th><th align="center" valign="middle"><bold>Total</bold></th><th align="center" valign="bottom"><bold>Mutation</bold></th><th align="center" rowspan="2" valign="middle"><bold># Codons</bold></th><th align="center" valign="bottom"><bold>Total</bold></th><th align="center" colspan="2" valign="middle"><bold>Mutations</bold></th><th align="center" valign="middle"><bold>Total</bold></th><th align="center" valign="bottom"><bold>Mutation</bold></th><th align="center" rowspan="2" valign="middle"><bold># Codons</bold></th><th align="center" valign="bottom"><bold>Total</bold></th><th align="center" colspan="2" valign="middle"><bold>Mutations</bold></th><th align="center" valign="middle"><bold>Total</bold></th><th align="center" valign="bottom"><bold>Mutation</bold></th></tr><tr><th align="center" valign="bottom"><bold>Nuc</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>Mutation</bold></th><th align="center" valign="bottom"><bold>Freq (%)</bold></th><th align="center" valign="bottom"><bold>Nuc</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>Mutation</bold></th><th align="center" valign="bottom"><bold>Freq (%)</bold></th><th align="center" valign="bottom"><bold>Nuc</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>Mutation</bold></th><th align="center" valign="bottom"><bold>Freq (%)</bold></th><th align="center" valign="middle"><bold>S</bold></th><th align="center" valign="middle"><bold>N</bold></th><th align="center" valign="middle"><bold>ALL</bold></th></tr></thead><tbody><tr><td align="left" valign="middle">&#945;V1.1</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">72</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>5.556</bold></td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">45</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>2.222</bold></td><td align="center" valign="bottom">10</td><td align="center" valign="bottom">90</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom"><bold>7.778</bold></td><td align="center" valign="middle">4</td><td align="center" valign="middle">8</td><td align="center" valign="middle">12</td></tr><tr><td align="left" valign="middle">&#945;V1.2</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">120</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom"><bold>7.500</bold></td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">75</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>4.000</bold></td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">120</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom"><bold>4.167</bold></td><td align="center" valign="middle">7</td><td align="center" valign="middle">10</td><td align="center" valign="middle">17</td></tr><tr><td align="left" valign="middle">&#945;V1.4</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">72</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>1.389</bold></td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">45</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">45</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>4.444</bold></td><td align="center" valign="middle">0</td><td align="center" valign="middle">3</td><td align="center" valign="middle">3</td></tr><tr><td align="left" valign="middle">&#945;V2.1</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">105</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>0.952</bold></td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">75</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">45</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="middle">0</td><td align="center" valign="middle">1</td><td align="center" valign="middle">1</td></tr><tr><td align="left" valign="middle">&#945;V3.1</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">72</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>4.167</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">72</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">12</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="middle">0</td><td align="center" valign="middle">3</td><td align="center" valign="middle">3</td></tr><tr><td align="left" valign="middle">&#945;V4.1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>9.524</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">36</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom"><bold>2.778</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">36</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>5.556</bold></td><td align="center" valign="middle">5</td><td align="center" valign="middle">2</td><td align="center" valign="middle">7</td></tr><tr><td align="left" valign="middle">&#945;V4.2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>4.762</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">36</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">36</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="middle">1</td><td align="center" valign="middle">1</td><td align="center" valign="middle">2</td></tr><tr><td align="left" valign="middle">&#945;V5.1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>7.143</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">36</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom"><bold>19.444</bold></td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="middle">0</td><td align="center" valign="middle">10</td><td align="center" valign="middle">10</td></tr><tr><td align="left" valign="middle">&#945;V7.2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">5</td><td align="center" valign="bottom">30</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">42</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>4.762</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">36</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom"><bold>5.556</bold></td><td align="center" valign="middle">1</td><td align="center" valign="middle">3</td><td align="center" valign="middle">4</td></tr><tr><td align="left" valign="middle">&#945;V10.1</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">84</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>3.571</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">72</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom"><bold>0.000</bold></td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">84</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>3.571</bold></td><td align="center" valign="middle">3</td><td align="center" valign="middle">3</td><td align="center" valign="middle">6</td></tr><tr><td align="left" valign="middle">&#945;V10.2</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">189</td><td align="center" valign="bottom">1</td><td align="center" valign="bottom">8</td><td align="center" valign="bottom">9</td><td align="center" valign="bottom"><bold>4.762</bold></td><td align="center" valign="bottom">6</td><td align="center" valign="bottom">162</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">2</td><td align="center" valign="bottom">4</td><td align="center" valign="bottom"><bold>2.469</bold></td><td align="center" valign="bottom">7</td><td align="center" valign="bottom">189</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom">0</td><td align="center" valign="bottom">3</td><td align="center" valign="bottom"><bold>1.587</bold></td><td align="center" valign="middle">6</td><td align="center" valign="middle">10</td><td align="center" valign="middle">16</td></tr><tr><td align="left" valign="middle"><bold>Sum</bold></td><td align="center" valign="bottom"><bold>41</bold></td><td align="center" valign="bottom"><bold>77</bold></td><td align="center" valign="bottom"><bold>870</bold></td><td align="center" valign="bottom"><bold>15</bold></td><td align="center" valign="bottom"><bold>24</bold></td><td align="center" valign="bottom"><bold>39</bold></td><td align="center" valign="bottom"/><td align="center" valign="bottom"><bold>63</bold></td><td align="center" valign="bottom"><bold>696</bold></td><td align="center" valign="bottom"><bold>5</bold></td><td align="center" valign="bottom"><bold>13</bold></td><td align="center" valign="bottom"><bold>18</bold></td><td align="center" valign="bottom"/><td align="center" valign="bottom"><bold>66</bold></td><td align="center" valign="bottom"><bold>735</bold></td><td align="center" valign="bottom"><bold>7</bold></td><td align="center" valign="bottom"><bold>17</bold></td><td align="center" valign="bottom"><bold>24</bold></td><td align="center" valign="bottom"/><td align="center" valign="middle"><bold>27</bold></td><td align="center" valign="middle"><bold>54</bold></td><td align="center" valign="middle"><bold>81</bold></td></tr><tr><td align="left" colspan="4" valign="middle"><bold>Mean Mutation Freq (%)</bold></td><td align="center" valign="middle"><bold>1.72</bold></td><td align="center" valign="middle"><bold>2.76</bold></td><td align="center" valign="middle"><bold>4.48</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>0.72</bold></td><td align="center" valign="middle"><bold>1.87</bold></td><td align="center" valign="middle"><bold>2.59</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>0.95</bold></td><td align="center" valign="middle"><bold>2.31</bold></td><td align="center" valign="middle"><bold>3.27</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"><bold>1.17</bold></td><td align="center" valign="middle"><bold>2.35</bold></td><td align="center" valign="middle"><bold>3.52</bold></td></tr><tr><td align="left" colspan="3" valign="middle"><bold>Standard Deviation</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"><bold>1.690</bold></td><td align="center" valign="middle"><bold>2.316</bold></td><td align="center" valign="middle"><bold>2.979</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>0.688</bold></td><td align="center" valign="middle"><bold>2.089</bold></td><td align="center" valign="middle"><bold>2.248</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"><bold>1.027</bold></td><td align="center" valign="middle"><bold>2.067</bold></td><td align="center" valign="middle"><bold>2.272</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"><bold>2.659</bold></td><td align="center" valign="middle"><bold>3.754</bold></td><td align="center" valign="middle"><bold>5.626</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28477-v3.xml">elife-28477-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.28477.011</object-id><label>Table 3.</label><caption><title>Number and frequency of DNA mutations that occur in tandem within framework regions (FR) and complementarity determining regions (CDR) in nurse shark alpha V (&#945;V) groups.</title><p>All mutations include both tandem and point mutations within a region. [<italic>Seqs:</italic> sequences]</p></caption><table frame="hsides" rules="groups"><thead><tr><th/><th/><th colspan="6">FR</th><th colspan="6">CDR</th></tr><tr><th/><th/><th colspan="4"># Nucleotides Tandemly Mutated</th><th rowspan="2">All mutation</th><th rowspan="2">Frequency of tandem mutation</th><th colspan="4"># Nucleotides Tandemly Mutated</th><th rowspan="2">All mutation</th><th rowspan="2">Frequency of tandem mutation</th></tr><tr><th>&#945;V Group</th><th># Seqs</th><th>2</th><th>3</th><th>4</th><th>Sum</th><th>2</th><th>3</th><th>4</th><th>Sum</th></tr></thead><tbody><tr><td>&#945;V1.1</td><td valign="bottom">3</td><td>1</td><td>0</td><td>1</td><td>6</td><td>24</td><td>25.0</td><td>3</td><td>0</td><td>0</td><td>6</td><td>12</td><td>50.0</td></tr><tr><td>&#945;V1.2</td><td valign="bottom">5</td><td>4</td><td>1</td><td>0</td><td>11</td><td>37</td><td>29.7</td><td>5</td><td>1</td><td>0</td><td>13</td><td>17</td><td>76.5</td></tr><tr><td>&#945;V1.4</td><td valign="bottom">3</td><td>3</td><td>0</td><td>0</td><td>6</td><td>14</td><td>42.9</td><td>0</td><td>0</td><td>0</td><td>0</td><td>3</td><td>0.0</td></tr><tr><td>&#945;V2</td><td valign="bottom">5</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0.0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0.0</td></tr><tr><td>&#945;V3</td><td valign="bottom">4</td><td>2</td><td>0</td><td>0</td><td>4</td><td>5</td><td>80.0</td><td>1</td><td>0</td><td>0</td><td>2</td><td>3</td><td>66.7</td></tr><tr><td>&#945;V4.1</td><td valign="bottom">2</td><td>4</td><td>1</td><td>0</td><td>11</td><td>23</td><td>47.8</td><td>0</td><td>1</td><td>0</td><td>3</td><td>7</td><td>42.9</td></tr><tr><td>&#945;V4.2</td><td valign="bottom">2</td><td>1</td><td>1</td><td>0</td><td>5</td><td>11</td><td>45.5</td><td>0</td><td>1</td><td>0</td><td>3</td><td>2</td><td>150.0</td></tr><tr><td>&#945;V5</td><td valign="bottom">2</td><td>1</td><td>0</td><td>0</td><td>2</td><td>14</td><td>14.3</td><td>1</td><td>1</td><td>0</td><td>5</td><td>10</td><td>50.0</td></tr><tr><td>&#945;V7.2</td><td valign="bottom">2</td><td>4</td><td>1</td><td>0</td><td>11</td><td>22</td><td>50.0</td><td>1</td><td>0</td><td>0</td><td>2</td><td>4</td><td>50.0</td></tr><tr><td>&#945;V10.1</td><td valign="bottom">4</td><td>3</td><td>0</td><td>0</td><td>6</td><td>6</td><td>100.0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>6</td><td>0.0</td></tr><tr><td>&#945;V10.2</td><td valign="bottom">9</td><td>4</td><td>1</td><td>0</td><td>11</td><td>35</td><td>31.4</td><td>2</td><td>1</td><td>0</td><td>7</td><td>16</td><td>43.8</td></tr><tr><td>Total</td><td valign="bottom">41</td><td>27</td><td>5</td><td>1</td><td>73</td><td>192</td><td>38.0</td><td>13</td><td>5</td><td>0</td><td>41</td><td>81</td><td>50.6</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28477-v3.xml">elife-28477-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table6" position="float"><object-id pub-id-type="doi">10.7554/eLife.28477.017</object-id><label>Table 6.</label><caption><title>Frequencies of somatic hypermutation in nurse shark thymus and peripheral lymphoid tissue (blood and spiral valve).</title><p>Mutations were analyzed only in alpha V groups containing the same third complementarity-determining region (CDR). Mutation frequency was measured as the total number of nucleotide changes to a Geneious-derived consensus sequence divided by the total number of nucleotides in all sequences. Nonsynymous (N) and synonymous (S) mutations (mut) were counted separately for each framework (FR) and CDR for two predicted alpha V genes. [FR1, FR2, FR3, CDR1, CDR2, and CDR3 refer to the first, second, or third FR or CDR region, respectively.]</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Tissue type</th><th rowspan="2">Mut type</th><th colspan="4">FR Mutations (#)</th><th colspan="4">CDR Mutations (#)</th><th colspan="3">FR mutation frequency</th><th colspan="3">CDR mutation frequency</th></tr><tr><th>&#8195;FR1</th><th>FR2</th><th>FR3</th><th>All FR</th><th>CDR1</th><th>CDR2</th><th>CDR3</th><th>All CDR</th><th>FR1</th><th>FR2</th><th>FR3</th><th>CDR1</th><th>CDR2</th><th>CDR3</th></tr></thead><tbody><tr><td rowspan="3">Thymus <break/>&#8195;(6 sequences) <break/></td><td>N</td><td>8</td><td>8</td><td>7</td><td>23</td><td>1</td><td>3</td><td>3</td><td>7</td><td>0.570</td><td>0.871</td><td>0.317</td><td>0.071</td><td>0.327</td><td>0.136</td></tr><tr><td>S</td><td>8</td><td>2</td><td>6</td><td>16</td><td>1</td><td>0</td><td>0</td><td>1</td><td>0.570</td><td>0.218</td><td>0.272</td><td>0.071</td><td>0.000</td><td>0.000</td></tr><tr><td>ALL</td><td>16</td><td>10</td><td>13</td><td>39</td><td>2</td><td>3</td><td>3</td><td>8</td><td>1.140</td><td>1.089</td><td>0.590</td><td>0.529</td><td>1.075</td><td>1.235</td></tr><tr><td colspan="2">Total Nucleotides</td><td>1404</td><td>918</td><td>2205</td><td>4527</td><td>378</td><td>279</td><td>243</td><td>900</td><td/><td/><td/><td/><td/><td/></tr><tr><td rowspan="3">Periphery <break/>&#8195;(2 sequences) <break/></td><td>N</td><td>7</td><td>4</td><td>6</td><td>17</td><td>0</td><td>2</td><td>2</td><td>4</td><td>1.496</td><td>1.307</td><td>0.833</td><td>0.000</td><td>1.852</td><td>1.587</td></tr><tr><td>S</td><td>4</td><td>0</td><td>1</td><td>5</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0.855</td><td>0.000</td><td>0.139</td><td>0.000</td><td>0.000</td><td>0.000</td></tr><tr><td>ALL</td><td>11</td><td>4</td><td>7</td><td>22</td><td>0</td><td>2</td><td>2</td><td>4</td><td>2.350</td><td>1.307</td><td>0.972</td><td>0.000</td><td>1.852</td><td>1.587</td></tr><tr><td colspan="2">Total Nucleotides</td><td>468</td><td>306</td><td>720</td><td>1494</td><td>126</td><td>108</td><td>126</td><td>360</td><td/><td/><td/><td/><td/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28721-v2.xml">elife-28721-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.28721.003</object-id><label>Table 1.</label><caption><title>Transmission chain size distribution and model parameters.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th align="center" colspan="6"><bold>Subtype</bold></th><th rowspan="2">Overall</th></tr><tr><th align="center">B</th><th align="center">C</th><th align="center">01_AE</th><th align="center">02_AG</th><th align="center">A</th><th align="center">Other</th></tr></thead><tbody><tr><td valign="middle">Total number of chains,&#160;<inline-formula><mml:math id="inf24"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mi>n</mml:mi></mml:mrow></mml:mstyle></mml:math></inline-formula>&#160;(%)</td><td align="right">1643 <break/>(53%)</td><td align="right">322 <break/>(10%)</td><td align="right">239 <break/>(7.7%)</td><td align="right">331 <break/>(11%)</td><td align="right">327 <break/>(11%)</td><td align="right">238 <break/>(7.7%)</td><td align="right"><bold>3100</bold> <break/><bold>(100%)</bold></td></tr><tr><td style="author-callout-style-b8">Chain size, <inline-formula><mml:math id="inf25"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mi>n</mml:mi></mml:mrow></mml:mstyle></mml:math></inline-formula> (%)</td><td align="right" style="author-callout-style-b8"/><td align="right" style="author-callout-style-b8"/><td align="right" style="author-callout-style-b8"/><td align="right" style="author-callout-style-b8"/><td align="right" style="author-callout-style-b8"/><td align="right" style="author-callout-style-b8"/><td align="right" style="author-callout-style-b8"/></tr><tr><td valign="middle">&#8195;1</td><td align="right">1437 <break/>(87%)</td><td align="right">280 <break/>(87%)</td><td align="right">206 <break/>(86%)</td><td align="right">272 <break/>(82%)</td><td align="right">269 <break/>(82%)</td><td align="right">195 <break/>(82%)</td><td align="right"><bold>2659</bold> <break/><bold>(86%)</bold></td></tr><tr><td style="author-callout-style-b8" valign="middle">&#8195;2</td><td align="right" style="author-callout-style-b8">158 <break/>(9.6%)</td><td align="right" style="author-callout-style-b8">34 <break/>(11%)</td><td align="right" style="author-callout-style-b8">31 <break/>(13%)</td><td align="right" style="author-callout-style-b8">40 <break/>(12%)</td><td align="right" style="author-callout-style-b8">44 <break/>(13%)</td><td align="right" style="author-callout-style-b8">36 <break/>(15%)</td><td align="right" style="author-callout-style-b8"><bold>343</bold> <break/><bold>(11%)</bold></td></tr><tr><td valign="middle">&#8195;3</td><td align="right">30 <break/>(1.8%)</td><td align="right">7 <break/>(2.2%)</td><td align="right">1 <break/>(0.42%)</td><td align="right">10 <break/>(3.0%)</td><td align="right">10 <break/>(3.1%)</td><td align="right">6 <break/>(2.5%)</td><td align="right"><bold>64</bold> <break/><bold>(2.1%)</bold></td></tr><tr><td style="author-callout-style-b8" valign="middle">&#8195;4</td><td align="right" style="author-callout-style-b8">12 <break/>(0.73%)</td><td align="right" style="author-callout-style-b8">-</td><td align="right" style="author-callout-style-b8">1 <break/>(0.42%)</td><td align="right" style="author-callout-style-b8">6 <break/>(1.8%)</td><td align="right" style="author-callout-style-b8">3 <break/>(0.92%)</td><td align="right" style="author-callout-style-b8">1 <break/>(0.42%)</td><td align="right" style="author-callout-style-b8"><bold>23</bold> <break/><bold>(0.74%)</bold></td></tr><tr><td valign="middle">&#8195;5</td><td align="right">1 <break/>(0.06%)</td><td align="right">1 <break/>(0.31%)</td><td align="right">-</td><td align="right">2 <break/>(0.6%)</td><td align="right">1 <break/>(0.31%)</td><td align="right">-</td><td align="right"><bold>5</bold> <break/><bold>(0.16%)</bold></td></tr><tr><td style="author-callout-style-b8" valign="middle">&#8195;6</td><td align="right" style="author-callout-style-b8">1 <break/>(0.06%)</td><td align="right" style="author-callout-style-b8">-</td><td align="right" style="author-callout-style-b8">-</td><td align="right" style="author-callout-style-b8">1 <break/>(0.3%)</td><td align="right" style="author-callout-style-b8">-</td><td align="right" style="author-callout-style-b8">-</td><td align="right" style="author-callout-style-b8"><bold>2</bold> <break/><bold>(0.06%)</bold></td></tr><tr><td valign="middle">&#8195;7</td><td align="right">1 <break/>(0.06%)</td><td align="right">-</td><td align="right">-</td><td align="right">-</td><td align="right">-</td><td align="right">-</td><td align="right"><bold>1</bold> <break/><bold>(0.03%)</bold></td></tr><tr><td style="author-callout-style-b8" valign="middle">&#8195;8</td><td align="right" style="author-callout-style-b8">2 <break/>(0.12%)</td><td align="right" style="author-callout-style-b8">-</td><td align="right" style="author-callout-style-b8">-</td><td align="right" style="author-callout-style-b8">-</td><td align="right" style="author-callout-style-b8">-</td><td align="right" style="author-callout-style-b8">-</td><td align="right" style="author-callout-style-b8"><bold>2</bold> <break/><bold>(0.06%)</bold></td></tr><tr><td valign="middle">&#8195;9</td><td align="right">1 <break/>(0.06%)</td><td align="right">-</td><td align="right">-</td><td align="right">-</td><td align="right">-</td><td align="right">-</td><td align="right"><bold>1</bold> <break/><bold>(0.03%)</bold></td></tr><tr><td style="author-callout-style-b8" valign="middle">Sampling probability, <inline-formula><mml:math id="inf26"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mi>p</mml:mi></mml:mrow></mml:mstyle></mml:math></inline-formula> (SD)</td><td align="right" style="author-callout-style-b8">0.39</td><td align="right" style="author-callout-style-b8">0.29</td><td align="right" style="author-callout-style-b8">0.34</td><td align="right" style="author-callout-style-b8">0.26</td><td align="right" style="author-callout-style-b8">0.33</td><td align="right" style="author-callout-style-b8">0.29</td><td align="right" style="author-callout-style-b8"><bold>0.35</bold> <break/><bold>(0.05)</bold></td></tr><tr><td>Chain origin, <inline-formula><mml:math id="inf27"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mi>n</mml:mi></mml:mrow></mml:mstyle></mml:math></inline-formula> (%)</td><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/><td align="right"/></tr><tr><td style="author-callout-style-b8" valign="middle">&#8195;Swiss (<inline-formula><mml:math id="inf28"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>&#961;</mml:mi><mml:mrow><mml:mtext>index</mml:mtext></mml:mrow></mml:msub><mml:mo>=</mml:mo><mml:mn>1</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula>)</td><td align="right" style="author-callout-style-b8">948 <break/>(58%)</td><td align="right" style="author-callout-style-b8">36 <break/>(11%)</td><td align="right" style="author-callout-style-b8">36 <break/>(15%)</td><td align="right" style="author-callout-style-b8">36 <break/>(11%)</td><td align="right" style="author-callout-style-b8">47 <break/>(14%)</td><td align="right" style="author-callout-style-b8">30 <break/>(13%)</td><td align="right" style="author-callout-style-b8"><bold>1133</bold> <break/><bold>(37%)</bold></td></tr><tr><td valign="middle">&#8195;non-Swiss (<inline-formula><mml:math id="inf29"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>&#961;</mml:mi><mml:mrow><mml:mtext>index</mml:mtext></mml:mrow></mml:msub><mml:mo>=</mml:mo><mml:mn>0.35</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula>)</td><td align="right">695 <break/>(42%)</td><td align="right">286 <break/>(89%)</td><td align="right">203 <break/>(85%)</td><td align="right">295 <break/>(89%)</td><td align="right">280 <break/>(86%)</td><td align="right">208 <break/>(87%)</td><td align="right"><bold>1967</bold> <break/><bold>(63%)</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28721-v2.xml">elife-28721-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.28721.006</object-id><label>Table 2.</label><caption><title>Patients&#8217; demographic characteristics.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th align="center" rowspan="2"><bold>Patients</bold></th><th align="center"><bold>Transmission chains</bold></th></tr><tr><th align="right">Index case</th></tr></thead><tbody><tr><td>Total number, <inline-formula><mml:math id="inf94"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mi>n</mml:mi></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right">3698</td><td align="right">3100</td></tr><tr><td style="author-callout-style-b8">Age at estimated date of infection [in years], median (IQR)</td><td align="right" style="author-callout-style-b8">29.2 (23.1&#8212;37.8)</td><td align="right" style="author-callout-style-b8">28.8 (22.8&#8212;37.4)</td></tr><tr><td>Estimated date of infection, median (IQR)</td><td align="right">Jun 1996 (Sep 1990&#8212;Nov 2001)</td><td align="right">Nov 1995 (Sep 1989&#8212;May 2001)</td></tr><tr><td style="author-callout-style-b8">Time to diagnosis [in years], median (IQR)</td><td align="right" style="author-callout-style-b8">3.40&#160;(1.66&#8212;5.24)</td><td align="right" style="author-callout-style-b8">3.54&#160;(1.78&#8212;5.43)</td></tr><tr><td>Reported sex with occasional partner [as fraction of FUPs*], median (IQR)</td><td align="right">0.53 (0.09&#8212;0.89)</td><td align="right">0.50 (0.07&#8212;0.88)</td></tr><tr><td style="author-callout-style-b8">&#8195;No available FUP<sup>&#8224;</sup>, <inline-formula><mml:math id="inf95"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mi>n</mml:mi></mml:mrow></mml:mstyle></mml:math></inline-formula> (%)</td><td align="right" style="author-callout-style-b8">250 (6.8%)</td><td align="right" style="author-callout-style-b8">226 (7.3%)</td></tr><tr><td>Earliest CD4 count [per &#956;L]<sup>&#8225;</sup>, median (IQR)</td><td align="right">310 (143&#8212;510)</td><td align="right">300 (134&#8212;507)</td></tr></tbody></table><table-wrap-foot><fn><p>*Follow-up visit (FUP).</p><p><sup>&#8224;</sup>Patients without FUP questionnaire regarding the sexual risk behavior. See Sensitivity analyses.</p></fn><fn><p><sup>&#8225;</sup>One patient did not have any available CD4 cell count. The missing value was imputed with the mean CD4 cell count.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28721-v2.xml">elife-28721-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app1table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.28721.021</object-id><label>Appendix 1&#8212;table 1.</label><caption><title>Overview of all the parameters, their estimates and the <inline-formula><mml:math id="inf309"><mml:mrow><mml:mn mathsize="125%">95</mml:mn><mml:mo lspace="0pt" mathsize="125%" rspace="3.5pt" stretchy="false">%</mml:mo></mml:mrow></mml:math></inline-formula>-confidence intervals fitted in all the models presented in this study.</title></caption><table frame="hsides" rules="groups"><thead><tr><th><bold>Subtypes</bold></th><th><bold>Parameter&#160;</bold> <break/><bold>number</bold></th><th><bold>Parameter&#160;</bold> <break/><bold>name</bold></th><th><bold>Parameter&#160;</bold> <break/><bold>estimate</bold></th><th><bold>Wald-type</bold> <break/><bold><inline-formula><mml:math id="inf310"><mml:mrow><mml:mn>95</mml:mn><mml:mo lspace="0pt" rspace="3.5pt">%</mml:mo></mml:mrow></mml:math></inline-formula>-CI</bold></th><th><bold>Profile likelihood</bold> <break/><bold><inline-formula><mml:math id="inf311"><mml:mrow><mml:mn>95</mml:mn><mml:mo lspace="0pt" rspace="3.5pt">%</mml:mo></mml:mrow></mml:math></inline-formula>-CI</bold></th></tr></thead><tbody><tr><th style="author-callout-style-b8">Overall</th><td style="author-callout-style-b8">1</td><td style="author-callout-style-b8"><inline-formula><mml:math id="inf312"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf313"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.823</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf314"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.876</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.770</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf315"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.878</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.772</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th>B</th><td>2</td><td><inline-formula><mml:math id="inf316"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf317"><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.037</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf318"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.121</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.952</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf319"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.124</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.955</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th>C</th><td>3</td><td><inline-formula><mml:math id="inf320"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf321"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.719</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf322"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.879</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.559</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf323"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.892</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.571</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th>01_AE</th><td>4</td><td><inline-formula><mml:math id="inf324"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf325"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.826</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf326"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.036</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.615</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf327"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.057</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.632</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th>02_AG</th><td>5</td><td><inline-formula><mml:math id="inf328"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf329"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.483</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf330"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.587</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.378</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf331"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.594</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.384</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th>A</th><td>6</td><td><inline-formula><mml:math id="inf332"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf333"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.618</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf334"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.751</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.485</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf335"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.760</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.492</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th>other</th><td>7</td><td><inline-formula><mml:math id="inf336"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf337"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.605</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf338"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.758</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.451</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf339"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.771</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.461</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th rowspan="2" style="author-callout-style-b8" valign="middle">Overall</th><td style="author-callout-style-b8">8</td><td style="author-callout-style-b8"><inline-formula><mml:math id="inf340"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mrow><mml:mn>0</mml:mn><mml:mo>,</mml:mo><mml:mi>&#119903;&#119890;&#119891;</mml:mi></mml:mrow></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf341"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.839</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf342"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.894</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.784</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf343"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.895</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.785</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8">9</td><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf344"><mml:mfrac><mml:mrow><mml:msub><mml:mi>&#119863;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119894;&#119899;&#119891;&#119890;&#119888;&#119905;&#119894;&#119900;&#119899;</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf345"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.112</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf346"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.187</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.037</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf347"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.188</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.037</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th rowspan="2" valign="middle">B</th><td>10</td><td><inline-formula><mml:math id="inf348"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mrow><mml:mn>0</mml:mn><mml:mo>,</mml:mo><mml:mi>&#119903;&#119890;&#119891;</mml:mi></mml:mrow></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf349"><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.070</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf350"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.165</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.975</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf351"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.169</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.979</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>11</td><td><p><inline-formula><mml:math id="inf352"><mml:mfrac><mml:mrow><mml:msub><mml:mi>&#119863;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119894;&#119899;&#119891;&#119890;&#119888;&#119905;&#119894;&#119900;&#119899;</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac></mml:math></inline-formula></p></td><td align="center"><inline-formula><mml:math id="inf353"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.112</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf354"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.234</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0.010</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf355"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.236</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0.008</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th rowspan="2" valign="middle">C</th><td>12</td><td><inline-formula><mml:math id="inf356"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mrow><mml:mn>0</mml:mn><mml:mo>,</mml:mo><mml:mi>&#119903;&#119890;&#119891;</mml:mi></mml:mrow></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf357"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.692</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf358"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.851</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.533</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf359"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.864</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.544</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>13</td><td><p><inline-formula><mml:math id="inf360"><mml:mfrac><mml:mrow><mml:msub><mml:mi>&#119863;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119894;&#119899;&#119891;&#119890;&#119888;&#119905;&#119894;&#119900;&#119899;</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac></mml:math></inline-formula></p></td><td align="center"><inline-formula><mml:math id="inf361"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.209</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf362"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.466</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0.049</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf363"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.473</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0.046</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th rowspan="2" valign="middle">01_AE</th><td>14</td><td><inline-formula><mml:math id="inf364"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mrow><mml:mn>0</mml:mn><mml:mo>,</mml:mo><mml:mi>&#119903;&#119890;&#119891;</mml:mi></mml:mrow></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf365"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.781</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf366"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.991</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.570</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf367"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.013</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.588</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>15</td><td><p><inline-formula><mml:math id="inf368"><mml:mfrac><mml:mrow><mml:msub><mml:mi>&#119863;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119894;&#119899;&#119891;&#119890;&#119888;&#119905;&#119894;&#119900;&#119899;</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac></mml:math></inline-formula></p></td><td align="center"><inline-formula><mml:math id="inf369"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.255</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf370"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.616</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0.106</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf371"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.629</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0.101</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th rowspan="2" valign="middle">02_AG</th><td>16</td><td><inline-formula><mml:math id="inf372"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mrow><mml:mn>0</mml:mn><mml:mo>,</mml:mo><mml:mi>&#119903;&#119890;&#119891;</mml:mi></mml:mrow></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf373"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.434</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf374"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.539</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.329</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf375"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.545</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.333</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>17</td><td><p><inline-formula><mml:math id="inf376"><mml:mfrac><mml:mrow><mml:msub><mml:mi>&#119863;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119894;&#119899;&#119891;&#119890;&#119888;&#119905;&#119894;&#119900;&#119899;</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac></mml:math></inline-formula></p></td><td align="center"><inline-formula><mml:math id="inf377"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.415</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf378"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.609</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.222</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf379"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.615</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.226</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th rowspan="2" valign="middle">A</th><td>18</td><td><inline-formula><mml:math id="inf380"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mrow><mml:mn>0</mml:mn><mml:mo>,</mml:mo><mml:mi>&#119903;&#119890;&#119891;</mml:mi></mml:mrow></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf381"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.725</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf382"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.892</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.558</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf383"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.907</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.571</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>19</td><td><p><inline-formula><mml:math id="inf384"><mml:mfrac><mml:mrow><mml:msub><mml:mi>&#119863;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119894;&#119899;&#119891;&#119890;&#119888;&#119905;&#119894;&#119900;&#119899;</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac></mml:math></inline-formula></p></td><td align="center"><inline-formula><mml:math id="inf385"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.430</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf386"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.660</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.199</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf387"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.672</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.209</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th rowspan="2" valign="middle">other</th><td>20</td><td><inline-formula><mml:math id="inf388"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mrow><mml:mn>0</mml:mn><mml:mo>,</mml:mo><mml:mi>&#119903;&#119890;&#119891;</mml:mi></mml:mrow></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf389"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.600</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf390"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.754</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.446</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf391"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.767</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.456</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>21</td><td><p><inline-formula><mml:math id="inf392"><mml:mfrac><mml:mrow><mml:msub><mml:mi>&#119863;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119894;&#119899;&#119891;&#119890;&#119888;&#119905;&#119894;&#119900;&#119899;</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac></mml:math></inline-formula></p></td><td align="center"><inline-formula><mml:math id="inf393"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.162</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf394"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.397</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0.073</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf395"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.403</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0.072</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th rowspan="3" style="author-callout-style-b8" valign="middle">Overall</th><td style="author-callout-style-b8">22</td><td style="author-callout-style-b8"><inline-formula><mml:math id="inf396"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mrow><mml:mn>0</mml:mn><mml:mo>,</mml:mo><mml:mi>&#119903;&#119890;&#119891;</mml:mi></mml:mrow></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf397"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.710</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf398"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.780</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.640</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf399"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.782</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.641</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8">23</td><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf400"><mml:msup><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:mrow><mml:msub><mml:mi>&#119863;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119894;&#119899;&#119891;&#119890;&#119888;&#119905;&#119894;&#119900;&#119899;</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac><mml:mo>)</mml:mo></mml:mrow><mml:mn>2</mml:mn></mml:msup></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf401"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.313</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf402"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.451</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.176</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf403"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.457</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.182</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8">24</td><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf404"><mml:msup><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:mrow><mml:msub><mml:mi>&#119863;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119894;&#119899;&#119891;&#119890;&#119888;&#119905;&#119894;&#119900;&#119899;</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac><mml:mo>)</mml:mo></mml:mrow><mml:mn>3</mml:mn></mml:msup></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf405"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.184</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf406"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.283</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.086</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf407"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.288</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.091</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th rowspan="11" valign="middle">Overall</th><td>25</td><td><inline-formula><mml:math id="inf408"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mrow><mml:mn>0</mml:mn><mml:mo>,</mml:mo><mml:mi>&#119903;&#119890;&#119891;</mml:mi></mml:mrow></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf409"><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.252</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf410"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.366</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.137</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf411"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.369</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.140</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>26</td><td><inline-formula><mml:math id="inf412"><mml:msub><mml:mi>&#119878;&#119906;&#119887;&#119905;&#119910;&#119901;&#119890;</mml:mi><mml:mi>C</mml:mi></mml:msub></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf413"><mml:mn>0.352</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf414"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.167</mml:mn><mml:mo>,</mml:mo><mml:mn>0.538</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf415"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.158</mml:mn><mml:mo>,</mml:mo><mml:mn>0.531</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>27</td><td><inline-formula><mml:math id="inf416"><mml:msub><mml:mi>&#119878;&#119906;&#119887;&#119905;&#119910;&#119901;&#119890;</mml:mi><mml:mrow><mml:mn mathvariant="italic">01</mml:mn><mml:mo>&#8290;</mml:mo><mml:mi mathvariant="normal">_</mml:mi><mml:mo>&#8290;</mml:mo><mml:mi>&#119860;&#119864;</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf417"><mml:mn>0.274</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf418"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.046</mml:mn><mml:mo>,</mml:mo><mml:mn>0.502</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf419"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.029</mml:mn><mml:mo>,</mml:mo><mml:mn>0.490</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>28</td><td><inline-formula><mml:math id="inf420"><mml:msub><mml:mi>&#119878;&#119906;&#119887;&#119905;&#119910;&#119901;&#119890;</mml:mi><mml:mrow><mml:mn mathvariant="italic">02</mml:mn><mml:mo>&#8290;</mml:mo><mml:mi mathvariant="normal">_</mml:mi><mml:mo>&#8290;</mml:mo><mml:mi>&#119860;&#119866;</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf421"><mml:mn>0.575</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf422"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.428</mml:mn><mml:mo>,</mml:mo><mml:mn>0.721</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf423"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.426</mml:mn><mml:mo>,</mml:mo><mml:mn>0.720</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>29</td><td><inline-formula><mml:math id="inf424"><mml:msub><mml:mi>&#119878;&#119906;&#119887;&#119905;&#119910;&#119901;&#119890;</mml:mi><mml:mi>A</mml:mi></mml:msub></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf425"><mml:mn>0.430</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf426"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.271</mml:mn><mml:mo>,</mml:mo><mml:mn>0.588</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf427"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.266</mml:mn><mml:mo>,</mml:mo><mml:mn>0.584</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>30</td><td><inline-formula><mml:math id="inf428"><mml:msub><mml:mi>&#119878;&#119906;&#119887;&#119905;&#119910;&#119901;&#119890;</mml:mi><mml:mi>&#119900;&#119905;&#8462;&#119890;&#119903;</mml:mi></mml:msub></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf429"><mml:mn>0.426</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf430"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.247</mml:mn><mml:mo>,</mml:mo><mml:mn>0.606</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf431"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.238</mml:mn><mml:mo>,</mml:mo><mml:mn>0.600</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>31</td><td><p><inline-formula><mml:math id="inf432"><mml:mfrac><mml:mrow><mml:msub><mml:mi>&#119863;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119894;&#119899;&#119891;&#119890;&#119888;&#119905;&#119894;&#119900;&#119899;</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac></mml:math></inline-formula></p></td><td align="center"><inline-formula><mml:math id="inf433"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.214</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf434"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.301</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.127</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf435"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.301</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.128</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>32</td><td><inline-formula><mml:math id="inf436"><mml:mfrac><mml:mrow><mml:mi>&#119860;&#119892;&#119890;</mml:mi><mml:mo>-</mml:mo><mml:mn>32</mml:mn></mml:mrow><mml:mn>10</mml:mn></mml:mfrac></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf437"><mml:mn>0.007</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf438"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.045</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0.058</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf439"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.046</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0.057</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>33</td><td><p><inline-formula><mml:math id="inf440"><mml:mfrac><mml:mrow><mml:mi mathvariant="italic">CD4</mml:mi><mml:mo>-</mml:mo><mml:mn>350</mml:mn></mml:mrow><mml:mn>100</mml:mn></mml:mfrac></mml:math></inline-formula></p></td><td align="center"><inline-formula><mml:math id="inf441"><mml:mn>0.000</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf442"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.018</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0.019</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf443"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.019</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mn>0.018</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>34</td><td><inline-formula><mml:math id="inf444"><mml:msub><mml:mi>&#119877;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119903;&#119894;&#119904;&#119896;</mml:mi></mml:msub></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf445"><mml:mn>0.230</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf446"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.095</mml:mn><mml:mo>,</mml:mo><mml:mn>0.364</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf447"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.096</mml:mn><mml:mo>,</mml:mo><mml:mn>0.365</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td>35</td><td><p><inline-formula><mml:math id="inf448"><mml:mfrac><mml:mrow><mml:msub><mml:mi>&#119884;&#119890;&#119886;&#119903;&#119904;</mml:mi><mml:mi>&#119889;&#119894;&#119886;&#119892;&#119899;&#119900;&#119904;&#119894;&#119904;</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:mn>3</mml:mn></mml:mrow><mml:mn>10</mml:mn></mml:mfrac></mml:math></inline-formula></p></td><td align="center"><inline-formula><mml:math id="inf449"><mml:mn>0.351</mml:mn></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf450"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.210</mml:mn><mml:mo>,</mml:mo><mml:mn>0.492</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf451"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.207</mml:mn><mml:mo>,</mml:mo><mml:mn>0.490</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><th rowspan="10" style="author-callout-style-b8" valign="middle">Overall</th><td style="author-callout-style-b8">36</td><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf452"><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:msub><mml:mi>R</mml:mi><mml:mrow><mml:mn>0</mml:mn><mml:mo>,</mml:mo><mml:mi>&#119903;&#119890;&#119891;</mml:mi></mml:mrow></mml:msub><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf453"><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.173</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf454"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.301</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.045</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf455"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.304</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>1.048</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8">37</td><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf456"><mml:mrow><mml:mfrac><mml:mn>1</mml:mn><mml:mn>10</mml:mn></mml:mfrac><mml:mo>&#8290;</mml:mo><mml:mrow><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:msub><mml:mi>&#119884;&#119890;&#119886;&#119903;&#119904;</mml:mi><mml:mi>&#119889;&#119894;&#119886;&#119892;&#119899;&#119900;&#119904;&#119894;&#119904;</mml:mi></mml:msub><mml:mn>3</mml:mn></mml:mfrac><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:mrow></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf457"><mml:mn>1.727</mml:mn></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf458"><mml:mrow><mml:mo>(</mml:mo><mml:mn>1.049</mml:mn><mml:mo>,</mml:mo><mml:mn>2.405</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf459"><mml:mrow><mml:mo>(</mml:mo><mml:mn>1.064</mml:mn><mml:mo>,</mml:mo><mml:mn>2.420</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8">38</td><td style="author-callout-style-b8"><inline-formula><mml:math id="inf460"><mml:msub><mml:mi>&#119878;&#119906;&#119887;&#119905;&#119910;&#119901;&#119890;</mml:mi><mml:mi>C</mml:mi></mml:msub></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf461"><mml:mn>0.322</mml:mn></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf462"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.140</mml:mn><mml:mo>,</mml:mo><mml:mn>0.505</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf463"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.131</mml:mn><mml:mo>,</mml:mo><mml:mn>0.498</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8">39</td><td style="author-callout-style-b8"><inline-formula><mml:math id="inf464"><mml:msub><mml:mi>&#119878;&#119906;&#119887;&#119905;&#119910;&#119901;&#119890;</mml:mi><mml:mrow><mml:mn mathvariant="italic">01</mml:mn><mml:mo>&#8290;</mml:mo><mml:mi mathvariant="normal">_</mml:mi><mml:mo>&#8290;</mml:mo><mml:mi>&#119860;&#119864;</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf465"><mml:mn>0.246</mml:mn></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf466"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.020</mml:mn><mml:mo>,</mml:mo><mml:mn>0.472</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf467"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.004</mml:mn><mml:mo>,</mml:mo><mml:mn>0.460</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8">40</td><td style="author-callout-style-b8"><inline-formula><mml:math id="inf468"><mml:msub><mml:mi>&#119878;&#119906;&#119887;&#119905;&#119910;&#119901;&#119890;</mml:mi><mml:mrow><mml:mn mathvariant="italic">02</mml:mn><mml:mo>&#8290;</mml:mo><mml:mi mathvariant="normal">_</mml:mi><mml:mo>&#8290;</mml:mo><mml:mi>&#119860;&#119866;</mml:mi></mml:mrow></mml:msub></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf469"><mml:mn>0.516</mml:mn></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf470"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.374</mml:mn><mml:mo>,</mml:mo><mml:mn>0.659</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf471"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.372</mml:mn><mml:mo>,</mml:mo><mml:mn>0.658</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8">41</td><td style="author-callout-style-b8"><inline-formula><mml:math id="inf472"><mml:msub><mml:mi>&#119878;&#119906;&#119887;&#119905;&#119910;&#119901;&#119890;</mml:mi><mml:mi>A</mml:mi></mml:msub></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf473"><mml:mn>0.404</mml:mn></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf474"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.246</mml:mn><mml:mo>,</mml:mo><mml:mn>0.562</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf475"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.241</mml:mn><mml:mo>,</mml:mo><mml:mn>0.558</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8">42</td><td style="author-callout-style-b8"><inline-formula><mml:math id="inf476"><mml:msub><mml:mi>&#119878;&#119906;&#119887;&#119905;&#119910;&#119901;&#119890;</mml:mi><mml:mi>&#119900;&#119905;&#8462;&#119890;&#119903;</mml:mi></mml:msub></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf477"><mml:mn>0.401</mml:mn></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf478"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.223</mml:mn><mml:mo>,</mml:mo><mml:mn>0.580</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf479"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.214</mml:mn><mml:mo>,</mml:mo><mml:mn>0.574</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8">43</td><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf480"><mml:msup><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:mrow><mml:msub><mml:mi>&#119863;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119894;&#119899;&#119891;&#119890;&#119888;&#119905;&#119894;&#119900;&#119899;</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac><mml:mo>)</mml:mo></mml:mrow><mml:mn>3</mml:mn></mml:msup></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf481"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.231</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf482"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.337</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.124</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf483"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.345</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.131</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8">44</td><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf484"><mml:msqrt><mml:msub><mml:mi>&#119877;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119903;&#119894;&#119904;&#119896;</mml:mi></mml:msub></mml:msqrt></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf485"><mml:mn>0.230</mml:mn></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf486"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.094</mml:mn><mml:mo>,</mml:mo><mml:mn>0.366</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf487"><mml:mrow><mml:mo>(</mml:mo><mml:mn>0.096</mml:mn><mml:mo>,</mml:mo><mml:mn>0.368</mml:mn><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8">45</td><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf488"><mml:msup><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:mrow><mml:msub><mml:mi>&#119863;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119894;&#119899;&#119891;&#119890;&#119888;&#119905;&#119894;&#119900;&#119899;</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac><mml:mo>)</mml:mo></mml:mrow><mml:mn>4</mml:mn></mml:msup></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf489"><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.129</mml:mn></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf490"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.227</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.031</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf491"><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.235</mml:mn></mml:mrow><mml:mo>,</mml:mo><mml:mrow><mml:mo>-</mml:mo><mml:mn>0.038</mml:mn></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:math></inline-formula></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28721-v2.xml">elife-28721-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app2table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.28721.025</object-id><label>Appendix 2&#8212;table 1.</label><caption><title>Establishment date models obtained with the AIC/BIC forward selection and backward elimination and their respective AIC and BIC values as well as the <inline-formula><mml:math id="inf508"><mml:mi>p</mml:mi></mml:math></inline-formula>-values from the likelihood ratio test compared to the null model without any covariates.</title><p>Terms that were part of the respective final model are marked by <inline-formula><mml:math id="inf509"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th align="center" colspan="2"><bold>AIC</bold></th><th align="center" colspan="2"><bold>BIC</bold></th></tr><tr><th align="center">Forward</th><th align="center">Backward</th><th align="center">Forward</th><th align="center">Backward</th></tr></thead><tbody><tr><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf510"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mfrac><mml:mrow><mml:msub><mml:mrow><mml:mi mathvariant="italic">D</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">e</mml:mi></mml:mrow><mml:mrow><mml:mrow><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">n</mml:mi><mml:mi mathvariant="italic">f</mml:mi><mml:mi mathvariant="italic">e</mml:mi><mml:mi mathvariant="italic">c</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">o</mml:mi><mml:mi mathvariant="italic">n</mml:mi></mml:mrow></mml:mrow></mml:msub><mml:mo>&#8722;</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac></mml:mstyle></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"/><td align="center" style="author-callout-style-b8"/><td align="center" style="author-callout-style-b8"/><td align="center" style="author-callout-style-b8"/></tr><tr><td><p><inline-formula><mml:math id="inf511"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mstyle displaystyle="true" scriptlevel="0"><mml:msup><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:mrow><mml:msub><mml:mrow><mml:mi mathvariant="italic">D</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">e</mml:mi></mml:mrow><mml:mrow><mml:mrow><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">n</mml:mi><mml:mi mathvariant="italic">f</mml:mi><mml:mi mathvariant="italic">e</mml:mi><mml:mi mathvariant="italic">c</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">o</mml:mi><mml:mi mathvariant="italic">n</mml:mi></mml:mrow></mml:mrow></mml:msub><mml:mo>&#8722;</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac><mml:mo>)</mml:mo></mml:mrow><mml:mrow><mml:mn>2</mml:mn></mml:mrow></mml:msup></mml:mstyle></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center"/><td align="center"><inline-formula><mml:math id="inf512"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center"/><td align="center"><inline-formula><mml:math id="inf513"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf514"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mstyle displaystyle="true" scriptlevel="0"><mml:msup><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:mrow><mml:msub><mml:mrow><mml:mi mathvariant="italic">D</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">e</mml:mi></mml:mrow><mml:mrow><mml:mrow><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">n</mml:mi><mml:mi mathvariant="italic">f</mml:mi><mml:mi mathvariant="italic">e</mml:mi><mml:mi mathvariant="italic">c</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">o</mml:mi><mml:mi mathvariant="italic">n</mml:mi></mml:mrow></mml:mrow></mml:msub><mml:mo>&#8722;</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac><mml:mo>)</mml:mo></mml:mrow><mml:mrow><mml:mn>3</mml:mn></mml:mrow></mml:msup></mml:mstyle></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf515"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf516"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf517"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf518"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td></tr><tr><td><p><inline-formula><mml:math id="inf519"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mstyle displaystyle="true" scriptlevel="0"><mml:msup><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:mrow><mml:msub><mml:mrow><mml:mi mathvariant="italic">D</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">e</mml:mi></mml:mrow><mml:mrow><mml:mrow><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">n</mml:mi><mml:mi mathvariant="italic">f</mml:mi><mml:mi mathvariant="italic">e</mml:mi><mml:mi mathvariant="italic">c</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">o</mml:mi><mml:mi mathvariant="italic">n</mml:mi></mml:mrow></mml:mrow></mml:msub><mml:mo>&#8722;</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac><mml:mo>)</mml:mo></mml:mrow><mml:mrow><mml:mn>4</mml:mn></mml:mrow></mml:msup></mml:mstyle></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center"><inline-formula><mml:math id="inf520"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center"/><td align="center"><inline-formula><mml:math id="inf521"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center"/></tr><tr><td style="author-callout-style-b8">AIC</td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf522"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3364.3</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf523"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3364.2</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf524"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3364.3</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf525"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3364.2</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td></tr><tr><td>BIC</td><td align="right"><inline-formula><mml:math id="inf526"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3382.4</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf527"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3382.3</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf528"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3382.4</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf529"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3382.3</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8"><inline-formula><mml:math id="inf530"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mi>p</mml:mi></mml:mrow></mml:mstyle></mml:math></inline-formula>-value from LR test</td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf531"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&lt;</mml:mo><mml:mn>0.0001</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf532"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&lt;</mml:mo><mml:mn>0.0001</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf533"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&lt;</mml:mo><mml:mn>0.0001</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf534"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&lt;</mml:mo><mml:mn>0.0001</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28721-v2.xml">elife-28721-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app2table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.28721.026</object-id><label>Appendix 2&#8212;table 2.</label><caption><title>Multivariate models obtained with the AIC/BIC forward selection and backward elimination algorithms.</title><p>The terms listed in the table are the terms identified from the single determinant model selections and the crosses indicate the terms entering the multivariate models. The null model from the likelihood ratio test refers to the baseline model without any covariates (not even the subtype).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th align="center" colspan="2"><bold>AIC</bold></th><th align="center" colspan="2"><bold>BIC</bold></th></tr><tr><th align="center">Forward</th><th align="center">Backward</th><th align="center">Forward</th><th align="center">Backward</th></tr></thead><tbody><tr><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf535"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mrow><mml:mi mathvariant="italic">S</mml:mi><mml:mi mathvariant="italic">u</mml:mi><mml:mi mathvariant="italic">b</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">y</mml:mi><mml:mi mathvariant="italic">p</mml:mi><mml:mi mathvariant="italic">e</mml:mi></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf536"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf537"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf538"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf539"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td></tr><tr><td><p><inline-formula><mml:math id="inf540"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mstyle displaystyle="true" scriptlevel="0"><mml:msup><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:mrow><mml:msub><mml:mrow><mml:mi mathvariant="italic">D</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">e</mml:mi></mml:mrow><mml:mrow><mml:mrow><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">n</mml:mi><mml:mi mathvariant="italic">f</mml:mi><mml:mi mathvariant="italic">e</mml:mi><mml:mi mathvariant="italic">c</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">o</mml:mi><mml:mi mathvariant="italic">n</mml:mi></mml:mrow></mml:mrow></mml:msub><mml:mo>&#8722;</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac><mml:mo>)</mml:mo></mml:mrow><mml:mrow><mml:mn>2</mml:mn></mml:mrow></mml:msup></mml:mstyle></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center"/><td align="center"/><td align="center"/><td align="center"/></tr><tr><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf541"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mstyle displaystyle="true" scriptlevel="0"><mml:msup><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:mrow><mml:msub><mml:mrow><mml:mi mathvariant="italic">D</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">e</mml:mi></mml:mrow><mml:mrow><mml:mrow><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">n</mml:mi><mml:mi mathvariant="italic">f</mml:mi><mml:mi mathvariant="italic">e</mml:mi><mml:mi mathvariant="italic">c</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">o</mml:mi><mml:mi mathvariant="italic">n</mml:mi></mml:mrow></mml:mrow></mml:msub><mml:mo>&#8722;</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac><mml:mo>)</mml:mo></mml:mrow><mml:mrow><mml:mn>3</mml:mn></mml:mrow></mml:msup></mml:mstyle></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf542"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf543"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf544"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf545"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td></tr><tr><td><p><inline-formula><mml:math id="inf546"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mstyle displaystyle="true" scriptlevel="0"><mml:msup><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:mrow><mml:msub><mml:mrow><mml:mi mathvariant="italic">D</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">e</mml:mi></mml:mrow><mml:mrow><mml:mrow><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">n</mml:mi><mml:mi mathvariant="italic">f</mml:mi><mml:mi mathvariant="italic">e</mml:mi><mml:mi mathvariant="italic">c</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">o</mml:mi><mml:mi mathvariant="italic">n</mml:mi></mml:mrow></mml:mrow></mml:msub><mml:mo>&#8722;</mml:mo><mml:mn>1.1.1996</mml:mn></mml:mrow><mml:mrow><mml:mn>365</mml:mn><mml:mo>&#8901;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:mfrac><mml:mo>)</mml:mo></mml:mrow><mml:mrow><mml:mn>4</mml:mn></mml:mrow></mml:msup></mml:mstyle></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center"><inline-formula><mml:math id="inf547"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf548"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf549"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center"/></tr><tr><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf550"><mml:msub><mml:mi>&#119877;&#119886;&#119905;&#119890;</mml:mi><mml:mi>&#119903;&#119894;&#119904;&#119896;</mml:mi></mml:msub></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf551"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf552"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf553"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf554"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td></tr><tr><td><p><inline-formula><mml:math id="inf555"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msqrt><mml:msub><mml:mrow><mml:mi mathvariant="italic">R</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">t</mml:mi><mml:mi mathvariant="italic">e</mml:mi></mml:mrow><mml:mrow><mml:mrow><mml:mi mathvariant="italic">r</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">s</mml:mi><mml:mi mathvariant="italic">k</mml:mi></mml:mrow></mml:mrow></mml:msub></mml:msqrt></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center"><inline-formula><mml:math id="inf556"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf557"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center"><inline-formula><mml:math id="inf558"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center"/></tr><tr><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf559"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mfrac><mml:mn>1</mml:mn><mml:mn>10</mml:mn></mml:mfrac><mml:mi>log</mml:mi><mml:mo>&#8289;</mml:mo><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:msub><mml:mrow><mml:mi mathvariant="italic">Y</mml:mi><mml:mi mathvariant="italic">e</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">r</mml:mi><mml:mi mathvariant="italic">s</mml:mi></mml:mrow><mml:mrow><mml:mrow><mml:mi mathvariant="italic">d</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">g</mml:mi><mml:mi mathvariant="italic">n</mml:mi><mml:mi mathvariant="italic">o</mml:mi><mml:mi mathvariant="italic">s</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">s</mml:mi></mml:mrow></mml:mrow></mml:msub><mml:mn>3</mml:mn></mml:mfrac><mml:mo>)</mml:mo></mml:mrow></mml:mstyle></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"/><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf560"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"/><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf561"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td></tr><tr><td><p><inline-formula><mml:math id="inf562"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mfrac><mml:mrow><mml:msqrt><mml:msub><mml:mrow><mml:mi mathvariant="italic">Y</mml:mi><mml:mi mathvariant="italic">e</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">r</mml:mi><mml:mi mathvariant="italic">s</mml:mi></mml:mrow><mml:mrow><mml:mrow><mml:mi mathvariant="italic">d</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">g</mml:mi><mml:mi mathvariant="italic">n</mml:mi><mml:mi mathvariant="italic">o</mml:mi><mml:mi mathvariant="italic">s</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">s</mml:mi></mml:mrow></mml:mrow></mml:msub></mml:msqrt><mml:mo>&#8722;</mml:mo><mml:msqrt><mml:mn>3</mml:mn></mml:msqrt></mml:mrow><mml:msqrt><mml:mn>10</mml:mn></mml:msqrt></mml:mfrac></mml:mstyle></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center"><inline-formula><mml:math id="inf563"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center"/><td align="center"><inline-formula><mml:math id="inf564"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center"/></tr><tr><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf565"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mfrac><mml:mrow><mml:msub><mml:mrow><mml:mi mathvariant="italic">Y</mml:mi><mml:mi mathvariant="italic">e</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">r</mml:mi><mml:mi mathvariant="italic">s</mml:mi></mml:mrow><mml:mrow><mml:mrow><mml:mi mathvariant="italic">d</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">g</mml:mi><mml:mi mathvariant="italic">n</mml:mi><mml:mi mathvariant="italic">o</mml:mi><mml:mi mathvariant="italic">s</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">s</mml:mi></mml:mrow></mml:mrow></mml:msub><mml:mo>&#8722;</mml:mo><mml:mrow><mml:mn>3</mml:mn></mml:mrow></mml:mrow><mml:mn>10</mml:mn></mml:mfrac></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf566"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf567"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"><inline-formula><mml:math id="inf568"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center" style="author-callout-style-b8"/></tr><tr><td><p><inline-formula><mml:math id="inf569"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msup><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:mrow><mml:msqrt><mml:msub><mml:mrow><mml:mi mathvariant="italic">Y</mml:mi><mml:mi mathvariant="italic">e</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">r</mml:mi><mml:mi mathvariant="italic">s</mml:mi></mml:mrow><mml:mrow><mml:mrow><mml:mi mathvariant="italic">d</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">a</mml:mi><mml:mi mathvariant="italic">g</mml:mi><mml:mi mathvariant="italic">n</mml:mi><mml:mi mathvariant="italic">o</mml:mi><mml:mi mathvariant="italic">s</mml:mi><mml:mi mathvariant="italic">i</mml:mi><mml:mi mathvariant="italic">s</mml:mi></mml:mrow></mml:mrow></mml:msub></mml:msqrt><mml:mo>&#8722;</mml:mo><mml:msqrt><mml:mn>3</mml:mn></mml:msqrt></mml:mrow><mml:msqrt><mml:mn>10</mml:mn></mml:msqrt></mml:mfrac><mml:mo>)</mml:mo></mml:mrow><mml:mrow><mml:mn>3</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center"/><td align="center"><inline-formula><mml:math id="inf570"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center"/><td align="center"><inline-formula><mml:math id="inf571"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&#215;</mml:mo></mml:mrow></mml:mstyle></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8"><p><inline-formula><mml:math id="inf572"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mfrac><mml:mrow><mml:msqrt><mml:mrow><mml:mi mathvariant="italic">C</mml:mi><mml:mi mathvariant="italic">D</mml:mi><mml:mn mathvariant="italic">4</mml:mn></mml:mrow></mml:msqrt><mml:mo>&#8722;</mml:mo><mml:msqrt><mml:mn>350</mml:mn></mml:msqrt></mml:mrow><mml:mn>10</mml:mn></mml:mfrac></mml:mstyle></mml:mrow></mml:mstyle></mml:math></inline-formula></p></td><td align="center" style="author-callout-style-b8"/><td align="center" style="author-callout-style-b8"/><td align="center" style="author-callout-style-b8"/><td align="center" style="author-callout-style-b8"/></tr><tr><td><inline-formula><mml:math id="inf573"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msup><mml:mrow><mml:mo>(</mml:mo><mml:mfrac><mml:mrow><mml:mi>A</mml:mi><mml:mi>g</mml:mi><mml:mi>e</mml:mi><mml:mo>&#8722;</mml:mo><mml:mn>32</mml:mn></mml:mrow><mml:mn>10</mml:mn></mml:mfrac><mml:mo>)</mml:mo></mml:mrow><mml:mn>2</mml:mn></mml:msup></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="center"/><td align="center"/><td align="center"/><td align="center"/></tr><tr><td style="author-callout-style-b8">AIC</td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf574"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3254</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf575"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3252</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf576"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3254</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf577"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3262</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td></tr><tr><td>BIC</td><td align="right"><inline-formula><mml:math id="inf578"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3314</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf579"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3331</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf580"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3314</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right"><inline-formula><mml:math id="inf581"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mn>3316</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td></tr><tr><td style="author-callout-style-b8"><inline-formula><mml:math id="inf582"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mi>p</mml:mi></mml:mrow></mml:mstyle></mml:math></inline-formula>-value from LR test</td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf583"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&lt;</mml:mo><mml:mn>0.0001</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf584"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&lt;</mml:mo><mml:mn>0.0001</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf585"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&lt;</mml:mo><mml:mn>0.0001</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td><td align="right" style="author-callout-style-b8"><inline-formula><mml:math id="inf586"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mo>&lt;</mml:mo><mml:mn>0.0001</mml:mn></mml:mrow></mml:mstyle></mml:math></inline-formula></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28751-v2.xml">elife-28751-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.28751.009</object-id><label>Table 1.</label><caption><title>Correlation between mGluR5 availability in distinct brain regions and the EEG&#160;&lt;1 Hz activity.</title></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="3">Brain region</th><th colspan="4">Correlation between mGluR5 availability and EEG&#160;&lt;&#160;1 Hz activity</th></tr><tr><th colspan="2">Baseline</th><th colspan="2">Sleep deprivation</th></tr><tr><th>r<sub>P</sub>, p</th><th>r<sub>S</sub>, p</th><th>r<sub>P</sub>, p</th><th>r<sub>S</sub>, p</th></tr></thead><tbody><tr><td rowspan="2">Medial superior <break/>frontal cortex <bold>*</bold></td><td>Left</td><td>0.75,&lt;0.0001</td><td>0.73,&#160;&lt;0.0001</td><td>0.60, 0.0022</td><td>0.63, 0.0011</td></tr><tr><td>Right</td><td>0.79,&lt;0.0001</td><td>0.76,&#160;&lt;0.0001</td><td>0.60, 0.0024</td><td>0.66, 0.0006</td></tr><tr><td rowspan="2">Orbitofrontal <break/>cortex</td><td>Left</td><td>0.60, 0.0027</td><td>0.60, 0.0026</td><td>0.58, 0.0036</td><td>0.57, 0.0046</td></tr><tr><td>Right</td><td>0.68, 0.0003</td><td>0.69, 0.0002</td><td>0.57, 0.0046</td><td>0.56, 0.0051</td></tr><tr><td rowspan="2">Dorsolateral <break/>prefrontal cortex <bold>*</bold></td><td>Left</td><td>0.76,&#160;&lt;0.0001</td><td>0.77,&#160;&lt;0.0001</td><td>0.67, 0.0005</td><td>0.66, 0.0006</td></tr><tr><td>Right</td><td>0.75, 0.0001</td><td>0.76,&#160;&lt;0.0001</td><td>0.67, 0.0004</td><td>0.72, 0.0001</td></tr><tr><td rowspan="2">Anterior cingulate <break/>cortex</td><td>Left</td><td>0.67, 0.0004</td><td>0.70, 0.0002</td><td>0.51, 0.0126</td><td>0.59, 0.0032</td></tr><tr><td>Right</td><td>0.66, 0.0007</td><td>0.64, 0.001</td><td>0.55, 0.0061</td><td>0.61, 0.0021</td></tr><tr><td rowspan="2">Inferior parietal <break/>cortex <bold>*</bold></td><td>Left</td><td>0.80,&#160;&lt;0.0001</td><td>0.79,&#160;&lt;0.0001</td><td>0.77,&#160;&lt;0.0001</td><td>0.76,&#160;&lt;0.0001</td></tr><tr><td>Right</td><td>0.79,&#160;&lt;0.0001</td><td>0.78,&#160;&lt;0.0001</td><td>0.71, 0.0002</td><td>0.75,&#160;&lt;0.0001</td></tr><tr><td rowspan="2">Precuneus <bold>*</bold></td><td>Left</td><td>0.82,&#160;&lt;0.0001</td><td>0.77,&lt;0.0001</td><td>0.78,&#160;&lt;0.0001</td><td>0.80,&#160;&lt;0.0001</td></tr><tr><td>Right</td><td>0.77,&#160;&lt;0.0001</td><td>0.76,&#160;&lt;0.0001</td><td>0.76,&#160;&lt;0.0001</td><td>0.82,&#160;&lt;0.0001</td></tr><tr><td rowspan="2">Insula</td><td>Left</td><td>0.63, 0.0014</td><td>0.63, 0.0012</td><td>0.47, 0.0244</td><td>0.41, 0.0532</td></tr><tr><td>Right</td><td>0.69, 0.0003</td><td>0.67, 0.0004</td><td>0.58, 0.0041</td><td>0.58, 0.004</td></tr><tr><td rowspan="2">Striatum</td><td>Left</td><td>0.62, 0.0016</td><td>0.63, 0.0013</td><td>0.50, 0.0147</td><td>0.49, 0.0176</td></tr><tr><td>Right</td><td>0.68, 0.0004</td><td>0.66, 0.0006</td><td>0.52, 0.0104</td><td>0.58, 0.0037</td></tr><tr><td rowspan="2">Parahippocampal <break/>gyrus</td><td>Left</td><td>0.41, 0.0536</td><td>0.46, 0.0267</td><td>0.33, 0.1296</td><td>0.28, 0.1897</td></tr><tr><td>Right</td><td>0.53, 0.0093</td><td>0.53, 0.01</td><td>0.38, 0.0728</td><td>0.40, 0.0585</td></tr><tr><td rowspan="2">Hippocampus</td><td>Left</td><td>0.52, 0.0101</td><td>0.47, 0.0245</td><td>0.36, 0.0886</td><td>0.37, 0.0853</td></tr><tr><td>Right</td><td>0.53, 0.009</td><td>0.48, 0.0198</td><td>0.43, 0.0413</td><td>0.37, 0.0835</td></tr></tbody></table><table-wrap-foot><fn><p>r<sub>P</sub>&#160;=&#160;Pearson Product Moment Correlation coefficient; r<sub>S</sub>&#160;=&#160;Spearman Rank Correlation coefficient. Those brain regions that showed a significant (p<sub>corr</sub>&#160;&lt;0.00278) correlation between mGluR5 availability and EEG&#160;&lt;1 Hz activity on left and right hemisphere in both baseline and sleep deprivation conditions are highlighted by a star (<bold>*</bold>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28877-v1.xml">elife-28877-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.28877.007</object-id><label>Table 1.</label><caption><title>Active fractions of each chromatography and the amino acid sequences of the purified peptides.</title></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2">Ion-exchange- <break/>HPLC <break/>Elution time (min)</th><th rowspan="2">Reversed-phase- <break/>HPLC <break/>Retention time (min)</th><th rowspan="2">Observed&#160;mass <break/>(MH<sup>+</sup>)</th><th rowspan="2">Theoretical&#160;mass <break/>(MH<sup>+</sup>)</th><th rowspan="2">Sequence</th><th rowspan="2">Precursor gene</th><th rowspan="2">Peptide name*</th></tr><tr><th>pH 6.5</th><th>pH4.8</th></tr></thead><tbody><tr><td rowspan="2">48&#8211;52 (Fr. A)</td><td rowspan="2">46&#8211;48</td><td>13&#8211;13.5</td><td>717.25</td><td>717.42</td><td>AVLPRY-NH<sub>2</sub></td><td><italic>lury-1</italic></td><td>LURY-1-2</td></tr><tr><td>17&#8211;18</td><td>818.34</td><td>818.46</td><td>PALLSRY-NH<sub>2</sub></td><td><italic>lury-1</italic></td><td>LURY-1-1</td></tr><tr><td>60&#8211;62 (Fr. B)</td><td>56&#8211;60</td><td>25.5&#8211;26.5</td><td>792.29</td><td>792.43</td><td>PNFLRF-NH<sub>2</sub></td><td><italic>flp-1</italic></td><td>FLP-1-6</td></tr><tr><td>92&#8211;94 (Fr. C)</td><td>108&#8211;112</td><td>27.5&#8211;28.5</td><td>1255.45</td><td>1255.68</td><td>RNKFEFIRF-NH<sub>2</sub></td><td><italic>flp-12</italic></td><td>FLP-12</td></tr><tr><td rowspan="2">94&#8211;96 (Fr. D)</td><td>86&#8211;88</td><td>15.5&#8211;16.5</td><td>901.27</td><td>901.45</td><td>KSAYMRF-NH<sub>2</sub></td><td><italic>flp-6</italic></td><td>FLP-6-1</td></tr><tr><td>90&#8211;92</td><td>23&#8211;23.5</td><td>1108.33</td><td>1108.55</td><td>SPSAKWMRF-NH<sub>2</sub></td><td><italic>flp-22</italic></td><td>FLP-22</td></tr></tbody></table><table-wrap-foot><fn><p>*Peptide names other than LURY-1 peptides are taken from <xref ref-type="bibr" rid="bib37">Li and Kim, 2008</xref>.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28921-v3.xml">elife-28921-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.28921.015</object-id><label>Table 1.</label><caption><title>Additive null predictions for interactions between mutations.</title><p>Most interactions result in high expression phenotypes because the wildtype in the presence of CI has no expression, meaning that mutations are either neutral or increase wildtype expression. If an effect of a mutation is positive, the additive model states that the effect remains positive (and the same), independent of the genetic background. As such, these predictions are true only for a system that is tightly repressed, where the wildtype has no expression. &#8216;High +&#8217; indicates predictions that result in expression levels above the biologically meaningful limit, which is defined by the unrepressed <italic>P<sub>R</sub></italic> promoter (shown in <xref ref-type="fig" rid="fig2">Figure 2A</xref>). We treat these predictions as high expression phenotypes. We consider three categorical single component effects (&#8216;no expression&#8217;, &#8216;intermediate expression&#8217;, and &#8216;high expression&#8217;), and show the categorical effect predicted by the additive null model for the system. We use categorical effects only to provide an intuition for what the additive model predicts - to obtain actual predictions of system DMEs, we use convolution (as explained in detail in Materials and methods section <italic>Na&#239;ve convolution of component distributions as the null model for additivity between mutations</italic>).</p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top"/><th valign="top"/><th align="center" colspan="3"><bold>Effects of mutations in <italic>trans</italic></bold></th></tr><tr><th valign="top"/><th valign="top"/><th valign="top"><bold>No expression</bold></th><th valign="top"><bold>Intermediate expression</bold></th><th valign="top"><bold>High expression</bold></th></tr></thead><tbody><tr><th rowspan="3" valign="middle"><bold>Effects of mutations in <italic>cis</italic></bold></th><th valign="top"><bold>No expression</bold></th><td valign="top">No</td><td valign="top">Intermediate</td><td valign="top">High</td></tr><tr><th valign="top"><bold>Intermediate expression</bold></th><td valign="top">Intermediate</td><td valign="top">Intermediate <break/>+high</td><td valign="top">High +</td></tr><tr><th valign="top"><bold>High expression</bold></th><td valign="top">High</td><td valign="top">High +</td><td valign="top">High +</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28939-v2.xml">elife-28939-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="inlinetable14" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Treatment</th><th rowspan="2">Stg (hpf)</th><th rowspan="2">n</th><th colspan="3"><italic>heg1</italic></th><th colspan="3"><italic>krit1</italic></th><th colspan="3"><italic>klf2a</italic></th></tr><tr><th>Mean</th><th>SEM</th><th>p</th><th>Mean</th><th>SEM</th><th>p</th><th>Mean</th><th>SEM</th><th>p</th></tr></thead><tbody><tr><td><italic>tnnt2a</italic> MO</td><td valign="top">54</td><td>3</td><td>0.85</td><td>0.008</td><td>0.013</td><td>1.06</td><td>0.009</td><td>0.103</td><td>-</td><td>-</td><td>-</td></tr><tr><td><italic>klf2a/b</italic> MO</td><td valign="top">54</td><td>4</td><td>0.77</td><td>0.034</td><td>0.045</td><td>1.17</td><td>0.029</td><td>0.103</td><td>-</td><td>-</td><td>-</td></tr><tr><td valign="top"><italic>hs:</italic>Klf2a</td><td valign="top">54</td><td valign="top">3</td><td valign="top">2.56</td><td valign="top">0.089</td><td valign="top">0.044</td><td valign="top">1.38</td><td valign="top">0.061</td><td valign="top">0.151</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td></tr><tr><td><italic>hs:</italic>Krit1</td><td valign="top">48</td><td>4</td><td>-</td><td>-</td><td>-</td><td>9.77</td><td>0.063</td><td>0.001</td><td>0.63</td><td>0.059</td><td>0.044</td></tr><tr><td><italic>heg1</italic> mRNA</td><td valign="top">24</td><td>3</td><td>4.01</td><td>0.059</td><td>0.0095</td><td>-</td><td>-</td><td>-</td><td>0.31</td><td>0.107</td><td>0.04</td></tr><tr><td><italic>heg1</italic> mRNA</td><td valign="top">48</td><td>3</td><td>2.47</td><td>0.036</td><td>0.0082</td><td>-</td><td>-</td><td>-</td><td>0.68</td><td>0.023</td><td>0.019</td></tr><tr><td><italic>egfp-krit1</italic> mRNA</td><td valign="top">24</td><td>3</td><td>-</td><td>-</td><td>-</td><td>7.06</td><td>0.125</td><td>0.021</td><td>-</td><td>-</td><td>-</td></tr><tr><td><italic>egfp-krit1</italic> mRNA</td><td valign="top">48</td><td>3</td><td>-</td><td>-</td><td>-</td><td>3.24</td><td>0.053</td><td>0.011</td><td>-</td><td>-</td><td>-</td></tr><tr><td><italic>egfp-krit1</italic> mRNA</td><td valign="top">96</td><td>3</td><td>-</td><td>-</td><td>-</td><td>1.20</td><td>0.161</td><td>0.675</td><td>-</td><td>-</td><td>-</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-28939-v2.xml">elife-28939-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="inlinetable4" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="4">Within each row, compare columns (simple effects within rows)</th></tr><tr><th>Uncorrected Fisher's LSD</th><th>Mean diff.</th><th>Summary</th><th>Individual P value</th></tr></thead><tbody><tr><td rowspan="3">48 hpf</td><td valign="bottom">WT vs. <italic>krit1</italic></td><td valign="bottom">100.0</td><td valign="bottom">****</td><td valign="bottom">&lt;0.0001</td></tr><tr><td valign="bottom">WT vs. WT&#160;+&#160;<italic>egfp-krit1</italic> mRNA</td><td valign="bottom">0.0</td><td valign="bottom">ns</td><td valign="bottom">&gt;0.9999</td></tr><tr><td valign="bottom">WT vs. <italic>krit1&#160;+&#160;egfp-krit1</italic> mRNA</td><td valign="bottom">0.0</td><td valign="bottom">ns</td><td valign="bottom">&gt;0.9999</td></tr><tr><td rowspan="3">96 hpf</td><td valign="bottom">WT vs. <italic>krit1</italic></td><td valign="bottom">100.0</td><td valign="bottom">****</td><td valign="bottom">&lt;0.0001</td></tr><tr><td valign="bottom">WT vs. WT&#160;+&#160;<italic>egfp-krit1</italic> mRNA</td><td valign="bottom">1.590</td><td valign="bottom">ns</td><td valign="bottom">0.7212</td></tr><tr><td valign="bottom">WT vs. <italic>krit1&#160;+&#160;egfp-krit1</italic> mRNA</td><td valign="bottom">76.44</td><td valign="bottom">****</td><td valign="bottom">&lt;0.0001</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29023-v2.xml">elife-29023-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.29023.021</object-id><label>Table 3.</label><caption><title>Retrospective test cases.</title><p>Columns indicate the PDB ID of each Ab-Ag pair; the number of residues for various subsets of the Ag; the number and success of EpiScope designs based on crystal and model Ag structures; a measure of the quality of the closest native-like docking model among ClusPro generated models (f<sub>nat</sub>[<xref ref-type="bibr" rid="bib48">Lensink et al., 2007</xref>]); the quality of the homology models built for Ab and Ags (TM-score [<xref ref-type="bibr" rid="bib89">Zhang and Skolnick, 2004</xref>]); and the number of docking decoys generated by ClusPro.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">PDB code</th><th colspan="3">Number of residues</th><th colspan="2">Crystal structure</th><th colspan="2">Model structure</th><th colspan="2">Fnat</th><th colspan="2">TM-score</th><th colspan="2">Number of docking decoys</th></tr><tr><th>Whole</th><th>Surface</th><th>Epitopes</th><th>Number of designs</th><th>Overlap with epitopes</th><th>Number of designs</th><th>Overlap with epitopes</th><th>Crystal</th><th>Model</th><th>Antibody</th><th>Antigen</th><th>Crystal</th><th>Model</th></tr></thead><tbody><tr><td><bold>1FE8</bold></td><td>196</td><td>124</td><td>27</td><td>4</td><td>Y</td><td>3</td><td>N</td><td>0.1</td><td>0.04</td><td>0.96</td><td>0.84</td><td>30</td><td>24</td></tr><tr><td><bold>1FNS</bold></td><td>196</td><td>120</td><td>12</td><td>5</td><td>Y</td><td>2</td><td>Y</td><td>0.39</td><td>0.09</td><td>0.96</td><td>0.86</td><td>26</td><td>20</td></tr><tr><td><bold>1H0D</bold></td><td>123</td><td>96</td><td>14</td><td>3</td><td>N</td><td>2</td><td>Y</td><td>0</td><td>0.05</td><td>0.98</td><td>0.79</td><td>30</td><td>30</td></tr><tr><td><bold>1LK3</bold></td><td>160</td><td>102</td><td>26</td><td>3</td><td>Y</td><td>3</td><td>Y</td><td>0.73</td><td>0.44</td><td>0.97</td><td>0.74</td><td>23</td><td>29</td></tr><tr><td><bold>1OAZ</bold></td><td>123</td><td>101</td><td>14</td><td>2</td><td>N</td><td>2</td><td>N</td><td>0.1</td><td>0.1</td><td>0.97</td><td>0.77</td><td>30</td><td>29</td></tr><tr><td><bold>1OB1</bold></td><td>99</td><td>74</td><td>13</td><td>3</td><td>Y</td><td>4</td><td>Y</td><td>0.62</td><td>0.61</td><td>0.97</td><td>0.85</td><td>30</td><td>29</td></tr><tr><td><bold>1RJL</bold></td><td>95</td><td>82</td><td>13</td><td>3</td><td>Y</td><td>2</td><td>Y</td><td>0.29</td><td>0.3</td><td>0.96</td><td>0.89</td><td>30</td><td>27</td></tr><tr><td><bold>1V7M</bold></td><td>163</td><td>113</td><td>20</td><td>6</td><td>Y</td><td>3</td><td>Y</td><td>0.45</td><td>0.19</td><td>0.96</td><td>0.78</td><td>27</td><td>24</td></tr><tr><td><bold>1YJD</bold></td><td>140</td><td>86</td><td>14</td><td>2</td><td>Y</td><td>3</td><td>Y</td><td>0.42</td><td>0.13</td><td>0.98</td><td>0.77</td><td>25</td><td>30</td></tr><tr><td><bold>2ARJ</bold></td><td>123</td><td>90</td><td>17</td><td>3</td><td>Y</td><td>3</td><td>Y</td><td>0.63</td><td>0.26</td><td>0.97</td><td>0.75</td><td>24</td><td>30</td></tr><tr><td><bold>2VXQ</bold></td><td>96</td><td>71</td><td>21</td><td>3</td><td>Y</td><td>3</td><td>Y</td><td>0.32</td><td>0.21</td><td>0.89</td><td>0.89</td><td>30</td><td>30</td></tr><tr><td><bold>2VXT</bold></td><td>157</td><td>116</td><td>19</td><td>3</td><td>Y</td><td>3</td><td>Y</td><td>0.83</td><td>0.13</td><td>0.95</td><td>0.93</td><td>13</td><td>19</td></tr><tr><td><bold>2XQB</bold></td><td>114</td><td>87</td><td>18</td><td>2</td><td>Y</td><td>3</td><td>Y</td><td>0.16</td><td>0.25</td><td>0.92</td><td>0.89</td><td>17</td><td>21</td></tr><tr><td><bold>3D9A</bold></td><td>129</td><td>93</td><td>19</td><td>3</td><td>Y</td><td>2</td><td>Y</td><td>0.09</td><td>0.63</td><td>0.93</td><td>0.93</td><td>22</td><td>29</td></tr><tr><td><bold>3HI1</bold></td><td>290</td><td>246</td><td>20</td><td>4</td><td>N</td><td>3</td><td>N</td><td>0.28</td><td>0.02</td><td>0.97</td><td>0.83</td><td>30</td><td>29</td></tr><tr><td><bold>3L5X</bold></td><td>113</td><td>83</td><td>8</td><td>6</td><td>Y</td><td>3</td><td>Y</td><td>0.18</td><td>0.24</td><td>0.98</td><td>0.83</td><td>30</td><td>30</td></tr><tr><td><bold>3MXW</bold></td><td>169</td><td>108</td><td>22</td><td>2</td><td>Y</td><td>2</td><td>Y</td><td>0.52</td><td>0.47</td><td>0.96</td><td>0.92</td><td>20</td><td>23</td></tr><tr><td><bold>3QWO</bold></td><td>57</td><td>48</td><td>10</td><td>3</td><td>Y</td><td>4</td><td>Y</td><td>0.32</td><td>0.53</td><td>0.96</td><td>0.86</td><td>30</td><td>30</td></tr><tr><td><bold>3RKD</bold></td><td>146</td><td>105</td><td>18</td><td>5</td><td>Y</td><td>3</td><td>Y</td><td>0.55</td><td>0.48</td><td>0.97</td><td>0.49</td><td>30</td><td>30</td></tr><tr><td><bold>4DN4</bold></td><td>76</td><td>50</td><td>12</td><td>2</td><td>Y</td><td>2</td><td>Y</td><td>0.49</td><td>0.28</td><td>0.92</td><td>0.88</td><td>20</td><td>27</td></tr><tr><td><bold>4DW2</bold></td><td>222</td><td>175</td><td>20</td><td>4</td><td>Y</td><td>3</td><td>Y</td><td>0.1</td><td>0.35</td><td>0.92</td><td>0.85</td><td>30</td><td>30</td></tr><tr><td><bold>4ETQ</bold></td><td>226</td><td>186</td><td>22</td><td>4</td><td>Y</td><td>3</td><td>Y</td><td>0.45</td><td>0.57</td><td>0.96</td><td>0.94</td><td>30</td><td>30</td></tr><tr><td><bold>4G3Y</bold></td><td>157</td><td>114</td><td>12</td><td>3</td><td>Y</td><td>4</td><td>Y</td><td>0.35</td><td>0.12</td><td>0.94</td><td>0.86</td><td>30</td><td>30</td></tr><tr><td><bold>4G6J</bold></td><td>158</td><td>109</td><td>13</td><td>3</td><td>Y</td><td>3</td><td>Y</td><td>0.57</td><td>0.15</td><td>0.97</td><td>0.85</td><td>30</td><td>30</td></tr><tr><td><bold>4I3S</bold></td><td>190</td><td>163</td><td>23</td><td>4</td><td>Y</td><td>4</td><td>Y</td><td>0.05</td><td>0.09</td><td>0.81</td><td>0.82</td><td>30</td><td>30</td></tr><tr><td><bold>4JZJ</bold></td><td>252</td><td>210</td><td>18</td><td>4</td><td>Y</td><td>6</td><td>Y</td><td>0.31</td><td>0.25</td><td>0.95</td><td>0.33</td><td>30</td><td>30</td></tr><tr><td><bold>4KI5</bold></td><td>183</td><td>108</td><td>7</td><td>1</td><td>Y</td><td>1</td><td>Y</td><td>0.39</td><td>0.29</td><td>0.95</td><td>0.93</td><td>24</td><td>20</td></tr><tr><td><bold>4L5F</bold></td><td>111</td><td>79</td><td>9</td><td>2</td><td>Y</td><td>3</td><td>Y</td><td>0.52</td><td>0.1</td><td>0.97</td><td>0.8</td><td>30</td><td>30</td></tr><tr><td><bold>4LVH</bold></td><td>223</td><td>184</td><td>13</td><td>5</td><td>Y</td><td>6</td><td>N</td><td>0.12</td><td>0.05</td><td>0.93</td><td>0.9</td><td>30</td><td>29</td></tr><tr><td><bold>4M62</bold></td><td>155</td><td>105</td><td>6</td><td>2</td><td>Y</td><td>2</td><td>N</td><td>0.12</td><td>0.11</td><td>0.92</td><td>0.81</td><td>24</td><td>24</td></tr><tr><td><bold>4NP4</bold></td><td>272</td><td>230</td><td>25</td><td>3</td><td>Y</td><td>5</td><td>Y</td><td>0.09</td><td>0.07</td><td>0.96</td><td>0.67</td><td>30</td><td>30</td></tr><tr><td><bold>4RGO</bold></td><td>226</td><td>187</td><td>17</td><td>3</td><td>N</td><td>6</td><td>Y</td><td>0.16</td><td>0.21</td><td>0.97</td><td>0.96</td><td>30</td><td>30</td></tr><tr><td><bold>5D96</bold></td><td>235</td><td>198</td><td>22</td><td>4</td><td>Y</td><td>4</td><td>Y</td><td>0.23</td><td>0.15</td><td>0.95</td><td>0.96</td><td>30</td><td>30</td></tr><tr><td><bold>Average</bold></td><td>162.88</td><td>122.52</td><td>16.48</td><td>3.30</td><td/><td>3.18</td><td/><td>0.33</td><td>0.24</td><td>0.95</td><td>0.82</td><td>27.12</td><td>27.67</td></tr><tr><td><bold>STD</bold></td><td>57.57</td><td>51.53</td><td>5.54</td><td>1.19</td><td/><td>1.21</td><td/><td>0.21</td><td>0.18</td><td>0.03</td><td>0.13</td><td>4.53</td><td>3.55</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29023-v2.xml">elife-29023-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.29023.022</object-id><label>Table 4.</label><caption><title>Ab modeling quality.</title><p>Antibody structures were generally highly accurately predicted both overall (average TM-score: 0.95) and for CDRs (all-backbone-atom, including N, C, C&#945; and O, RMSDs reported). Overall, non-CDR-H3 loops were very well predicted based on the canonical rules, and even for CDR-H3 loops the average RMSDs was&#160;&lt;2 &#197;.</p><p><supplementary-material id="table4sdata1"><object-id pub-id-type="doi">10.7554/eLife.29023.023</object-id><label>Table 4&#8212;source data 1.</label><caption/><media mime-subtype="octet-stream" mimetype="application" xlink:href="elife-29023-table4-data1-v2.csv"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Target</th><th rowspan="2">Species</th><th rowspan="2">CDR-L1</th><th rowspan="2">L2</th><th rowspan="2">L3</th><th rowspan="2">H1</th><th rowspan="2">H2</th><th colspan="3" valign="bottom">CDR-H3</th><th rowspan="2">TM-score</th></tr><tr><th>RMSD</th><th>Sequence</th><th>Length</th></tr></thead><tbody><tr><td valign="bottom"><bold>1FE8</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">0.42</td><td valign="bottom">0.22</td><td valign="bottom">0.74</td><td valign="bottom">1.01</td><td valign="bottom">0.51</td><td valign="bottom">0.63</td><td valign="bottom">AGNYYGMDY</td><td valign="bottom">9</td><td valign="bottom">0.96</td></tr><tr><td valign="bottom"><bold>1FNS</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">0.54</td><td valign="bottom">0.18</td><td valign="bottom">0.93</td><td valign="bottom">0.27</td><td valign="bottom">0.60</td><td valign="bottom">2.10</td><td valign="bottom">VRDPADYGNYDYALDY</td><td valign="bottom">16</td><td valign="bottom">0.96</td></tr><tr><td valign="bottom"><bold>1H0D</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">1.43</td><td valign="bottom">0.57</td><td valign="bottom">0.42</td><td valign="bottom">0.44</td><td valign="bottom">1.11</td><td valign="bottom">0.66</td><td valign="bottom">TRLGDYGYAYTMDY</td><td valign="bottom">14</td><td valign="bottom">0.98</td></tr><tr><td valign="bottom"><bold>1LK3</bold></td><td valign="bottom"><bold>RAT</bold></td><td valign="bottom">0.41</td><td valign="bottom">0.43</td><td valign="bottom">0.52</td><td valign="bottom">0.57</td><td valign="bottom">1.51</td><td valign="bottom">1.00</td><td valign="bottom">TRGVPGNNWFPY</td><td valign="bottom">12</td><td valign="bottom">0.97</td></tr><tr><td valign="bottom"><bold>1OAZ</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">1.15</td><td valign="bottom">0.44</td><td valign="bottom">0.88</td><td valign="bottom">1.30</td><td valign="bottom">0.56</td><td valign="bottom">1.25</td><td valign="bottom">ARMWYYGTYYFDY</td><td valign="bottom">13</td><td valign="bottom">0.97</td></tr><tr><td valign="bottom"><bold>1OB1</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">0.58</td><td valign="bottom">0.31</td><td valign="bottom">0.63</td><td valign="bottom">0.42</td><td valign="bottom">0.63</td><td valign="bottom">1.97</td><td valign="bottom">ARNYYRFDGGMDF</td><td valign="bottom">13</td><td valign="bottom">0.97</td></tr><tr><td valign="bottom"><bold>1RJL</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">1.43</td><td valign="bottom">0.57</td><td valign="bottom">4.96</td><td valign="bottom">0.69</td><td valign="bottom">1.00</td><td valign="bottom">1.16</td><td valign="bottom">ARMRYGDYYAMDN</td><td valign="bottom">13</td><td valign="bottom">0.96</td></tr><tr><td valign="bottom"><bold>1V7M</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">0.70</td><td valign="bottom">0.26</td><td valign="bottom">0.83</td><td valign="bottom">0.65</td><td valign="bottom">1.10</td><td valign="bottom">0.59</td><td valign="bottom">SGWSFLY</td><td valign="bottom">7</td><td valign="bottom">0.96</td></tr><tr><td valign="bottom"><bold>1YJD</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">0.88</td><td valign="bottom">0.51</td><td valign="bottom">1.34</td><td valign="bottom">0.62</td><td valign="bottom">1.19</td><td valign="bottom">1.76</td><td valign="bottom">TRSHYGLDWNFDV</td><td valign="bottom">13</td><td valign="bottom">0.98</td></tr><tr><td valign="bottom"><bold>2ARJ</bold></td><td valign="bottom"><bold>RAT</bold></td><td valign="bottom">0.71</td><td valign="bottom">0.67</td><td valign="bottom">1.12</td><td valign="bottom">0.46</td><td valign="bottom">0.70</td><td valign="bottom">0.65</td><td valign="bottom">TPLIGSWYFDF</td><td valign="bottom">11</td><td valign="bottom">0.97</td></tr><tr><td valign="bottom"><bold>2VXQ</bold></td><td valign="bottom"><bold>HUMAN</bold></td><td valign="bottom">0.35</td><td valign="bottom">0.74</td><td valign="bottom">0.96</td><td valign="bottom">0.90</td><td valign="bottom">1.27</td><td valign="bottom">1.05</td><td valign="bottom">ARLDGYTLDI</td><td valign="bottom">10</td><td valign="bottom">0.89</td></tr><tr><td valign="bottom"><bold>2VXT</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">0.47</td><td valign="bottom">0.37</td><td valign="bottom">1.14</td><td valign="bottom">0.45</td><td valign="bottom">0.53</td><td valign="bottom">0.43</td><td valign="bottom">ARGLRF</td><td valign="bottom">6</td><td valign="bottom">0.95</td></tr><tr><td valign="bottom"><bold>2XQB</bold></td><td valign="bottom"><bold>HUMAN</bold></td><td valign="bottom">1.61</td><td valign="bottom">0.43</td><td valign="bottom">0.98</td><td valign="bottom">1.19</td><td valign="bottom">0.89</td><td valign="bottom">7.21</td><td valign="bottom">ARDPAAWPLQQSLAWFDP</td><td valign="bottom">18</td><td valign="bottom">0.92</td></tr><tr><td valign="bottom"><bold>3D9A</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">0.40</td><td valign="bottom">0.61</td><td valign="bottom">1.18</td><td valign="bottom">0.99</td><td valign="bottom">1.88</td><td valign="bottom">0.51</td><td valign="bottom">ANWDGDY</td><td valign="bottom">7</td><td valign="bottom">0.93</td></tr><tr><td valign="bottom"><bold>3HI1</bold></td><td valign="bottom"><bold>HUMAN</bold></td><td valign="bottom">0.80</td><td valign="bottom">0.86</td><td valign="bottom">0.83</td><td valign="bottom">0.61</td><td valign="bottom">0.44</td><td valign="bottom">1.25</td><td valign="bottom">ARGPVPAVFYGDYRLDP</td><td valign="bottom">17</td><td valign="bottom">0.97</td></tr><tr><td valign="bottom"><bold>3L5X</bold></td><td valign="bottom"><bold>HUMAN</bold></td><td valign="bottom">0.56</td><td valign="bottom">0.61</td><td valign="bottom">0.91</td><td valign="bottom">1.05</td><td valign="bottom">0.90</td><td valign="bottom">1.73</td><td valign="bottom">ARMGSDYDVWFDY</td><td valign="bottom">13</td><td valign="bottom">0.98</td></tr><tr><td valign="bottom"><bold>3MXW</bold></td><td valign="bottom"><bold>HUMAN</bold></td><td valign="bottom">0.58</td><td valign="bottom">0.71</td><td valign="bottom">0.71</td><td valign="bottom">1.09</td><td valign="bottom">0.82</td><td valign="bottom">0.96</td><td valign="bottom">ARDWERGDFFDY</td><td valign="bottom">12</td><td valign="bottom">0.96</td></tr><tr><td valign="bottom"><bold>3QWO</bold></td><td valign="bottom"><bold>HUMANIZED</bold></td><td valign="bottom">0.48</td><td valign="bottom">0.28</td><td valign="bottom">1.09</td><td valign="bottom">0.87</td><td valign="bottom">0.50</td><td valign="bottom">1.13</td><td valign="bottom">ARDMIFNFYFDV</td><td valign="bottom">12</td><td valign="bottom">0.96</td></tr><tr><td valign="bottom"><bold>3RKD</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">0.62</td><td valign="bottom">0.42</td><td valign="bottom">0.52</td><td valign="bottom">1.06</td><td valign="bottom">0.65</td><td valign="bottom">1.45</td><td valign="bottom">ARIKSVITTGDYALDY</td><td valign="bottom">16</td><td valign="bottom">0.97</td></tr><tr><td valign="bottom"><bold>4DN4</bold></td><td valign="bottom"><bold>HUMAN</bold></td><td valign="bottom">2.11</td><td valign="bottom">0.37</td><td valign="bottom">1.58</td><td valign="bottom">1.64</td><td valign="bottom">2.40</td><td valign="bottom">2.36</td><td valign="bottom">ARYDGIYGELDF</td><td valign="bottom">12</td><td valign="bottom">0.92</td></tr><tr><td valign="bottom"><bold>4DW2</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">1.20</td><td valign="bottom">0.43</td><td valign="bottom">4.12</td><td valign="bottom">0.85</td><td valign="bottom">1.14</td><td valign="bottom">3.18</td><td valign="bottom">ERGELTYAMDY</td><td valign="bottom">11</td><td valign="bottom">0.92</td></tr><tr><td valign="bottom"><bold>4ETQ</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">1.07</td><td valign="bottom">0.29</td><td valign="bottom">1.59</td><td valign="bottom">0.35</td><td valign="bottom">0.91</td><td valign="bottom">0.94</td><td valign="bottom">TRSNYRYDYFDV</td><td valign="bottom">12</td><td valign="bottom">0.96</td></tr><tr><td valign="bottom"><bold>4G3Y</bold></td><td valign="bottom"><bold>CHIMERIC</bold></td><td valign="bottom">0.68</td><td valign="bottom">0.71</td><td valign="bottom">0.57</td><td valign="bottom">0.90</td><td valign="bottom">0.98</td><td valign="bottom">1.22</td><td valign="bottom">SRNYYGSTYDY</td><td valign="bottom">11</td><td valign="bottom">0.94</td></tr><tr><td valign="bottom"><bold>4G6J</bold></td><td valign="bottom"><bold>HUMAN</bold></td><td valign="bottom">0.72</td><td valign="bottom">0.44</td><td valign="bottom">0.90</td><td valign="bottom">0.41</td><td valign="bottom">0.35</td><td valign="bottom">1.14</td><td valign="bottom">ARDLRTGPFDY</td><td valign="bottom">11</td><td valign="bottom">0.97</td></tr><tr><td valign="bottom"><bold>4I3S</bold></td><td valign="bottom"><bold>HUMAN</bold></td><td valign="bottom">1.34</td><td valign="bottom">0.46</td><td valign="bottom">0.64</td><td valign="bottom">4.33</td><td valign="bottom">1.08</td><td valign="bottom">3.49</td><td valign="bottom">ARQKFYTGGQGWYFDL</td><td valign="bottom">16</td><td valign="bottom">0.81</td></tr><tr><td valign="bottom"><bold>4JZJ</bold></td><td valign="bottom"><bold>HUMAN</bold></td><td valign="bottom">0.59</td><td valign="bottom">0.54</td><td valign="bottom">0.98</td><td valign="bottom">0.84</td><td valign="bottom">1.04</td><td valign="bottom">2.96</td><td valign="bottom">ARSHLLRASWFAY</td><td valign="bottom">13</td><td valign="bottom">0.95</td></tr><tr><td valign="bottom"><bold>4KI5</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">0.74</td><td valign="bottom">0.52</td><td valign="bottom">0.78</td><td valign="bottom">2.14</td><td valign="bottom">0.44</td><td valign="bottom">1.49</td><td valign="bottom">AREDDGLAS</td><td valign="bottom">9</td><td valign="bottom">0.95</td></tr><tr><td valign="bottom"><bold>4L5F</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">0.76</td><td valign="bottom">0.42</td><td valign="bottom">1.03</td><td valign="bottom">0.49</td><td valign="bottom">0.94</td><td valign="bottom">1.83</td><td valign="bottom">TKRINWALDY</td><td valign="bottom">10</td><td valign="bottom">0.97</td></tr><tr><td valign="bottom"><bold>4LVH</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">1.63</td><td valign="bottom">0.71</td><td valign="bottom">2.82</td><td valign="bottom">1.46</td><td valign="bottom">2.70</td><td valign="bottom">1.91</td><td valign="bottom">ARHGSPGYTLYAWDY</td><td valign="bottom">15</td><td valign="bottom">0.93</td></tr><tr><td valign="bottom"><bold>4M62</bold></td><td valign="bottom"><bold>HUMAN</bold></td><td valign="bottom">2.08</td><td valign="bottom">0.79</td><td valign="bottom">1.40</td><td valign="bottom">2.55</td><td valign="bottom">2.78</td><td valign="bottom">8.26</td><td valign="bottom">AREGTTGSGWLGKPIGAFAY</td><td valign="bottom">20</td><td valign="bottom">0.92</td></tr><tr><td valign="bottom"><bold>4NP4</bold></td><td valign="bottom"><bold>HUMAN</bold></td><td valign="bottom">2.21</td><td valign="bottom">0.87</td><td valign="bottom">2.84</td><td valign="bottom">0.88</td><td valign="bottom">0.55</td><td valign="bottom">1.53</td><td valign="bottom">ARRRNWGNAFDI</td><td valign="bottom">12</td><td valign="bottom">0.96</td></tr><tr><td valign="bottom"><bold>4RGO</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">0.53</td><td valign="bottom">1.01</td><td valign="bottom">0.75</td><td valign="bottom">0.71</td><td valign="bottom">0.31</td><td valign="bottom">2.20</td><td valign="bottom">VRDLYGDYVGRYAY</td><td valign="bottom">14</td><td valign="bottom">0.97</td></tr><tr><td valign="bottom"><bold>5D96</bold></td><td valign="bottom"><bold>MOUSE</bold></td><td valign="bottom">0.74</td><td valign="bottom">0.57</td><td valign="bottom">0.53</td><td valign="bottom">0.62</td><td valign="bottom">0.89</td><td valign="bottom">3.43</td><td valign="bottom">ASDSMDPGSFAY</td><td valign="bottom">12</td><td valign="bottom">0.95</td></tr><tr><td colspan="2" valign="bottom"><bold>Average</bold></td><td valign="bottom">0.92</td><td valign="bottom">0.52</td><td valign="bottom">1.25</td><td valign="bottom">0.99</td><td valign="bottom">1.01</td><td valign="bottom">1.92</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">0.95</td></tr><tr><td colspan="2" valign="bottom"><bold>STD</bold></td><td valign="bottom">0.53</td><td valign="bottom">0.20</td><td valign="bottom">1.01</td><td valign="bottom">0.78</td><td valign="bottom">0.62</td><td valign="bottom">1.71</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">0.03</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29023-v2.xml">elife-29023-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table6" position="float"><object-id pub-id-type="doi">10.7554/eLife.29023.025</object-id><label>Table 6.</label><caption><title>Success rates with epitopes defined according to IEDB or according to contacts in the binding interface.</title><p>Success is indicated as &#8216;T&#8217; and failure as &#8216;F&#8217;. In test cases colored blue, EpiScope failed to find IEDB epitopes but did find binding interface residues.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Target</th><th colspan="2">Crystal structure</th><th colspan="2">Model structure</th><th rowspan="2">Target</th><th colspan="2">Crystal structure</th><th colspan="2">Model structure</th></tr><tr><th>IEDB</th><th>Interface</th><th>IEDB</th><th>Interface</th><th>IEDB</th><th>Interface</th><th>IEDB</th><th>Interface</th></tr></thead><tbody><tr><td>1FE8</td><td>T</td><td>T</td><td>F</td><td>F</td><td>3QWO</td><td>T</td><td>T</td><td>T</td><td>T</td></tr><tr><td>1FNS</td><td>T</td><td>T</td><td>T</td><td>T</td><td>3RKD</td><td>T</td><td>T</td><td>T</td><td>T</td></tr><tr><td>1H0D</td><td>F</td><td>F</td><td>T</td><td>T</td><td>4DN4</td><td>T</td><td>T</td><td>T</td><td>T</td></tr><tr><td>1LK3</td><td>T</td><td>T</td><td>T</td><td>T</td><td>4DW2</td><td>T</td><td>T</td><td>T</td><td>T</td></tr><tr><td>1OAZ</td><td>F</td><td>F</td><td>F</td><td>F</td><td>4ETQ</td><td>T</td><td>T</td><td>T</td><td>T</td></tr><tr><td>1OB1</td><td>T</td><td>T</td><td>T</td><td>T</td><td>4G3Y</td><td>T</td><td>T</td><td>T</td><td>T</td></tr><tr><td>1RJL</td><td>T</td><td>T</td><td>T</td><td>T</td><td>4G6J</td><td>T</td><td>T</td><td>T</td><td>T</td></tr><tr><td>1V7M</td><td>T</td><td>T</td><td>T</td><td>T</td><td>4I3S</td><td>T</td><td>T</td><td>T</td><td>T</td></tr><tr><td>1YJD</td><td>T</td><td>T</td><td>T</td><td>T</td><td>4JZJ</td><td>T</td><td>T</td><td>T</td><td>T</td></tr><tr><td>2ARJ</td><td>T</td><td>T</td><td>T</td><td>T</td><td>4KI5</td><td>T</td><td>T</td><td>T</td><td>T</td></tr><tr><td>2VXQ</td><td>T</td><td>T</td><td>T</td><td>T</td><td>4L5F</td><td>T</td><td>T</td><td>T</td><td>T</td></tr><tr><td>2VXT</td><td>T</td><td>T</td><td>T</td><td>T</td><td>4LVH</td><td>T</td><td>T</td><td style="author-callout-style-b1">F</td><td style="author-callout-style-b1">T</td></tr><tr><td>2XQB</td><td>T</td><td>T</td><td>T</td><td>T</td><td>4M62</td><td>T</td><td>T</td><td style="author-callout-style-b1">F</td><td style="author-callout-style-b1">T</td></tr><tr><td>3D9A</td><td>T</td><td>T</td><td>T</td><td>T</td><td>4NP4</td><td>T</td><td>T</td><td>T</td><td>T</td></tr><tr><td>3HI1</td><td>F</td><td>F</td><td>F</td><td>F</td><td>4RGO</td><td style="author-callout-style-b1">F</td><td style="author-callout-style-b1">T</td><td>T</td><td>T</td></tr><tr><td>3L5X</td><td>T</td><td>T</td><td>T</td><td>T</td><td>5D96</td><td>T</td><td>T</td><td>T</td><td>T</td></tr><tr><td>3MXW</td><td>T</td><td>T</td><td>T</td><td>T</td><td>Total</td><td>29 (88%)</td><td>30 (91%)</td><td>28 (85%)</td><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29023-v2.xml">elife-29023-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table8" position="float"><object-id pub-id-type="doi">10.7554/eLife.29023.034</object-id><label>Table 8.</label><caption><title>Comparison of predictive components of PEASE and EpiScope on retrospective test set of 33 non-redundant Ab-Ag pairs.</title><p>The number of designs needed/considered indicates the number of designs generated by EpiScope to cover all ClusPro docking models. An equivalent number of the top ranked PEASE patch predictions are considered for each Ab. Coloring highlights the cases in which Episcope (green) or PEASE (red) succeeded where the other method failed. Grey coloring indicates cases in which both methods failed.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Target</th><th colspan="3">Crystal structure of ag</th><th colspan="3">Modeled structure of ag</th></tr><tr><th># of Designs Needed/Considered</th><th># of EpiScopeDesigns Overlapping True Epitope</th><th># of PEASE patches Overlapping True Epitope</th><th># of Designs Needed/Considered</th><th># of EpiScopeDesigns Overlapping True Epitope</th><th># of PEASE patches Overlapping True Epitope</th></tr></thead><tbody><tr><td><bold>1FE8</bold></td><td>4</td><td>2</td><td>4</td><td>3</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">0</td></tr><tr><td><bold>1FNS</bold></td><td>5</td><td>2</td><td>5</td><td>2</td><td>1</td><td>2</td></tr><tr><td><bold>1H0D</bold></td><td>3</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">0</td><td>2</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td></tr><tr><td><bold>1LK3</bold></td><td>3</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>3</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td></tr><tr><td><bold>1OAZ</bold></td><td>2</td><td style="author-callout-style-b6">0</td><td style="author-callout-style-b6">2</td><td>2</td><td style="author-callout-style-b6">0</td><td style="author-callout-style-b6">2</td></tr><tr><td><bold>1OB1</bold></td><td>3</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>4</td><td>2</td><td>4</td></tr><tr><td><bold>1RJL</bold></td><td>3</td><td>1</td><td>3</td><td>2</td><td>1</td><td>2</td></tr><tr><td><bold>1</bold> V7M</td><td>6</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>3</td><td>1</td><td>2</td></tr><tr><td><bold>1YJD</bold></td><td>2</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>3</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td></tr><tr><td><bold>2ARJ</bold></td><td>3</td><td>1</td><td>3</td><td>3</td><td>1</td><td>3</td></tr><tr><td><bold>2VXQ</bold></td><td>3</td><td>1</td><td>3</td><td>3</td><td>1</td><td>3</td></tr><tr><td><bold>2VXT</bold></td><td>3</td><td>1</td><td>3</td><td>3</td><td>1</td><td>3</td></tr><tr><td><bold>2XQB</bold></td><td>2</td><td>1</td><td>2</td><td>3</td><td>1</td><td>3</td></tr><tr><td><bold>3D9A</bold></td><td>3</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>2</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td></tr><tr><td><bold>3HI1</bold></td><td>4</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">0</td><td>3</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">0</td></tr><tr><td><bold>3L5X</bold></td><td>6</td><td>1</td><td>6</td><td>3</td><td>2</td><td>3</td></tr><tr><td><bold>3MXW</bold></td><td>2</td><td>1</td><td>2</td><td>2</td><td>1</td><td>2</td></tr><tr><td><bold>3QWO</bold></td><td>3</td><td>2</td><td>3</td><td>4</td><td>3</td><td>4</td></tr><tr><td><bold>3RKD</bold></td><td>5</td><td>2</td><td>3</td><td>3</td><td>1</td><td>2</td></tr><tr><td><bold>4DN4</bold></td><td>2</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>2</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td></tr><tr><td><bold>4DW2</bold></td><td>4</td><td>1</td><td>2</td><td>3</td><td>1</td><td>1</td></tr><tr><td><bold>4ETQ</bold></td><td>4</td><td>1</td><td>1</td><td>3</td><td>1</td><td>1</td></tr><tr><td><bold>4G3Y</bold></td><td>3</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>4</td><td>2</td><td>3</td></tr><tr><td><bold>4G6J</bold></td><td>3</td><td>1</td><td>2</td><td>3</td><td>1</td><td>3</td></tr><tr><td><bold>4I3S</bold></td><td>4</td><td>2</td><td>3</td><td>4</td><td>1</td><td>2</td></tr><tr><td><bold>4JZJ</bold></td><td>4</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>6</td><td style="author-callout-style-b2">4</td><td style="author-callout-style-b2">0</td></tr><tr><td><bold>4KI5</bold></td><td>1</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>1</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td></tr><tr><td><bold>4</bold> L5F</td><td>2</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>3</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td></tr><tr><td><bold>4LVH</bold></td><td>5</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>6</td><td style="author-callout-style-b6">0</td><td style="author-callout-style-b6">2</td></tr><tr><td><bold>4</bold> M62</td><td>2</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>2</td><td style="author-callout-style-b8">0</td><td style="author-callout-style-b8">0</td></tr><tr><td><bold>4</bold> NP4</td><td>3</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>5</td><td style="author-callout-style-b2">3</td><td style="author-callout-style-b2">0</td></tr><tr><td><bold>4RGO</bold></td><td>3</td><td style="author-callout-style-b6">0</td><td style="author-callout-style-b6">2</td><td>6</td><td>1</td><td>3</td></tr><tr><td><bold>5D96</bold></td><td>4</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td><td>4</td><td style="author-callout-style-b2">1</td><td style="author-callout-style-b2">0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29088-v1.xml">elife-29088-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.29088.003</object-id><label>Table 1.</label><caption><title>Neonatal, socio-demographic, cognitive and scanning measures.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th>Very preterm-perinatal <break/>brain injury</th><th>Very preterm-no <break/>diagnosed injury</th><th>Controls</th><th rowspan="2">Test statistic</th><th rowspan="2">Significance</th></tr><tr><th>(n&#160;=&#160;16)</th><th>(n&#160;=&#160;13)</th><th>(n&#160;=&#160;14)</th></tr></thead><tbody><tr><td>Gestational age in weeks Mean (SD)</td><td>28.44 (2.28)</td><td>30.46 (1.76)</td><td/><td>U<sub>27</sub>&#160;=&#160;47.00</td><td>p=0.011</td></tr><tr><td>Birth weight in grams Mean (SD)</td><td>1203.19 (304.95)</td><td>1557.15 (364.98)</td><td/><td>U<sub>27</sub>&#160;=&#160;46.50</td><td>p=0.012</td></tr><tr><td>Age in years Mean (SD)</td><td>30.21 (1.78)</td><td>30.85 (2.09)</td><td>29.81 (3.24)</td><td>F<sub>2,40</sub> = 1.50</td><td>p=0.236</td></tr><tr><td>Sex (female:male)</td><td>03:13</td><td>04:09</td><td>05:09</td><td><italic>X</italic> <sup>2</sup><sub>2</sub> = 1.14</td><td>p=0.564</td></tr><tr><td>High SES (%)*</td><td>68.75</td><td>69.23</td><td>61.53</td><td><italic>X</italic> <sup>2</sup><sub>2</sub> =0.22</td><td>p=0.894</td></tr><tr><td>IQ Mean (SD)</td><td>106.67 (14.52)</td><td>107.73 (10.07)</td><td>110.40 (10.52)</td><td>F<sub>2,33</sub> = 0.28</td><td>p=0.755</td></tr><tr><td>Alcohol consumption (Units/week)</td><td>7.40 (11.30)</td><td>12.50 (11.99)</td><td>5.50 (4.72)</td><td><italic>X</italic> <sup>2</sup><sub>2</sub> =3.172</td><td>p=0.205</td></tr><tr><td>Injected dose (MBq) <break/>Mean (SD)</td><td>146.44 (2.15)</td><td>146.25 (2.52)</td><td>145.73 (2.38)</td><td>F<sub>2,40</sub> = 0.23</td><td>p=0.793</td></tr></tbody></table><table-wrap-foot><fn><p>*SES was collapsed into two groups; the percent of participants belonging to the high SES (level 1&#8211;2) category is presented in the table.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29275-v2.xml">elife-29275-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.29275.021</object-id><label>Table 2.</label><caption><title>Parameters for the biophysical model capturing the release dynamics during forward-masking and repetitive tone burst experiments</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2">Forward masking</th><th colspan="2">All tone bursts &#8211; global fit</th></tr><tr><th>RBE<sup>WT/WT</sup></th><th>RBE<sup>KO/KO</sup></th><th>RBE<sup>WT/WT</sup></th><th>RBE<sup>KO/KO</sup></th></tr></thead><tbody><tr><td><bold>N<sub>slots</sub></bold></td><td>8</td><td>6.5</td><td>15.6</td><td>13.7</td></tr><tr><td><bold>k<sub>refill, stim</sub></bold></td><td>29.5</td><td>18.2</td><td>18.4</td><td>19.9</td></tr><tr><td><bold>k<sub>refill, spont</sub></bold></td><td>13.3</td><td>7.1</td><td>5.59</td><td>2.71</td></tr><tr><td><bold>k<sub>fusion, stim</sub></bold></td><td>32.2</td><td>31</td><td>82.5</td><td>45.1</td></tr><tr><td><bold>k<sub>fusion, spont</sub></bold></td><td>0.6</td><td>0.9</td><td>0.63</td><td>0.54</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29365-v2.xml">elife-29365-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.29365.013</object-id><label>Table 4.</label><caption><title>Subgroup analysis across sex of the effect of N&#160;+&#160;WSH intervention on telomere length (T/S ratio) after 1 and 2 years of intervention and on change in telomere length between Year 1 and Year 2</title><p><supplementary-material id="table4sdata1"><object-id pub-id-type="doi">10.7554/eLife.29365.014</object-id><label>Table 4&#8212;source data 1.</label><caption><title>Source data and code for <xref ref-type="table" rid="table4">Table 4</xref>.</title></caption><media mime-subtype="zip" mimetype="application" xlink:href="elife-29365-table4-data1-v2.zip"/></supplementary-material> </p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Arm</th><th colspan="2" valign="top">Female children</th><th colspan="2" valign="top">Male children</th><th rowspan="2" valign="top">Female children&#160;Unadjusted difference: Intervention vs. Control (95% CI)</th><th rowspan="2" valign="top">Male children Unadjusted difference: Intervention vs. Control (95% CI)</th><th rowspan="2" valign="top">Interaction term sex by treatment (95% CI)</th></tr><tr><th valign="top">N</th><th valign="top">Mean</th><th valign="top">N</th><th valign="top">Mean</th></tr></thead><tbody><tr><td colspan="8" valign="top">After 1 year of intervention (age&#160;~&#160;14 months)</td></tr><tr><td valign="top">&#8195;Control</td><td valign="top">162</td><td valign="top">1.50</td><td valign="top">159</td><td valign="top">1.43</td><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">&#8195;N&#160;+&#160;WSH</td><td valign="top">180</td><td valign="top">1.42</td><td valign="top">161</td><td valign="top">1.37</td><td valign="top">&#8722;0.08 (&#8722;0.13,&#8211;0.03) p=0.001*</td><td valign="top">&#8722;0.06 (&#8722;0.11,&#8211;0.00) p=0.040*</td><td valign="top">0.03 (&#8722;0.04,0.09) p=0.435</td></tr><tr><td colspan="8" valign="top">After 2 years of intervention (age&#160;~&#160;28 months)</td></tr><tr><td valign="top">&#8195;Control</td><td valign="top">167</td><td valign="top">1.50</td><td valign="top">163</td><td valign="top">1.42</td><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">&#8195;N&#160;+&#160;WSH</td><td valign="top">202</td><td valign="top">1.45</td><td valign="top">181</td><td valign="top">1.42</td><td valign="top">&#8722;0.05 (&#8722;0.10,0.01) p=0.082</td><td valign="top">0.00 (&#8722;0.05,0.05) p=0.877</td><td valign="top">0.05 (&#8722;0.01,0.12) p=0.105</td></tr><tr><td colspan="8" valign="top">Change in telomere length between Year 1 and 2</td></tr><tr><td valign="top">&#8195;Control</td><td valign="top">134</td><td valign="top">&#8211;0.01</td><td valign="top">126</td><td valign="top">&#8722;0.01</td><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">&#8195;N&#160;+&#160;WSH</td><td valign="top">160</td><td valign="top">0.03</td><td valign="top">137</td><td valign="top">0.04</td><td valign="top">0.04 (&#8722;0.02,0.09) p=0.227</td><td valign="top">0.05 (0.00,0.09) p=0.048*</td><td valign="top">0.01 (&#8722;0.06,0.08) p=0.747</td></tr></tbody></table><table-wrap-foot><fn><p>*P&lt;0.05</p><p>Confidence intervals were adjusted for clustered observations using robust standard errors.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29365-v3.xml">elife-29365-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><label>Table 4.</label><caption><title>Subgroup analysis across sex of the effect of N&#160;+&#160;WSH intervention on telomere length (T/S ratio) after 1 and 2 years of intervention and on change in telomere length between Year 1 and Year 2</title><p><supplementary-material id="table4sdata1"><label>Table 4&#8212;source data 1.</label><caption><title>Source data and code for <xref ref-type="table" rid="table4">Table 4</xref>.</title></caption><media mime-subtype="zip" mimetype="application" xlink:href="elife-29365-table4-data1-v3.zip"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Arm</th><th colspan="2" valign="top">Female children</th><th colspan="2" valign="top">Male children</th><th rowspan="2" valign="top">Female children&#160;Unadjusted difference: Intervention vs. Control (95% CI)</th><th rowspan="2" valign="top">Male children Unadjusted difference: Intervention vs. Control (95% CI)</th><th rowspan="2" valign="top">Interaction term sex by treatment (95% CI)</th></tr><tr><th valign="top">N</th><th valign="top">Mean</th><th valign="top">N</th><th valign="top">Mean</th></tr></thead><tbody><tr><td colspan="8" valign="top">After 1 year of intervention (age&#160;~&#160;14 months)</td></tr><tr><td valign="top">&#8195;Control</td><td valign="top">162</td><td valign="top">1.50</td><td valign="top">159</td><td valign="top">1.43</td><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">&#8195;N&#160;+&#160;WSH</td><td valign="top">180</td><td valign="top">1.42</td><td valign="top">161</td><td valign="top">1.37</td><td valign="top">&#8722;0.08 (&#8722;0.13,&#8211;0.03) p=0.001*</td><td valign="top">&#8722;0.06 (&#8722;0.11,&#8211;0.00) p=0.040*</td><td valign="top">0.03 (&#8722;0.04,0.09) p=0.435</td></tr><tr><td colspan="8" valign="top">After 2 years of intervention (age&#160;~&#160;28 months)</td></tr><tr><td valign="top">&#8195;Control</td><td valign="top">167</td><td valign="top">1.50</td><td valign="top">163</td><td valign="top">1.42</td><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">&#8195;N&#160;+&#160;WSH</td><td valign="top">202</td><td valign="top">1.45</td><td valign="top">181</td><td valign="top">1.42</td><td valign="top">&#8722;0.05 (&#8722;0.10,0.01) p=0.082</td><td valign="top">0.00 (&#8722;0.05,0.05) p=0.877</td><td valign="top">0.05 (&#8722;0.01,0.12) p=0.105</td></tr><tr><td colspan="8" valign="top">Change in telomere length between Year 1 and 2</td></tr><tr><td valign="top">&#8195;Control</td><td valign="top">134</td><td valign="top">&#8211;0.01</td><td valign="top">126</td><td valign="top">&#8722;0.01</td><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">&#8195;N&#160;+&#160;WSH</td><td valign="top">160</td><td valign="top">0.03</td><td valign="top">137</td><td valign="top">0.04</td><td valign="top">0.04 (&#8722;0.02,0.09) p=0.227</td><td valign="top">0.05 (0.00,0.09) p=0.048*</td><td valign="top">0.01 (&#8722;0.06,0.08) p=0.747</td></tr></tbody></table><table-wrap-foot><fn><p>*P&lt;0.05</p><p>Confidence intervals were adjusted for clustered observations using robust standard errors.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29742-v2.xml">elife-29742-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.29742.012</object-id><label>Table 4.</label><caption><title>Connection probabilities for the random network model.</title></caption><table frame="hsides" rules="groups"><thead><tr><th/><th/><th colspan="4">From</th></tr><tr><th/><th/><th>E</th><th>PV</th><th>SST</th><th>VIP</th></tr></thead><tbody><tr><th rowspan="4">to</th><th>E</th><td>0.02</td><td>1</td><td>1</td><td>0</td></tr><tr><th>PV</th><td>0.01</td><td>1</td><td>0.85</td><td>0</td></tr><tr><th>SST</th><td>0.01</td><td>0</td><td>0</td><td>&#8722;0.55</td></tr><tr><th>VIP</th><td>0.01</td><td>0</td><td>0.5</td><td>0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29742-v3.xml">elife-29742-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.29742.012</object-id><label>Table 4.</label><caption><title>Connection probabilities for the random network model.</title></caption><table frame="hsides" rules="groups"><thead><tr><th/><th/><th colspan="4">From</th></tr><tr><th/><th/><th>E</th><th>PV</th><th>SST</th><th>VIP</th></tr></thead><tbody><tr><th rowspan="4">to</th><th>E</th><td>0.02</td><td>1</td><td>1</td><td>0</td></tr><tr><th>PV</th><td>0.01</td><td>1</td><td>0.85</td><td>0</td></tr><tr><th>SST</th><td>0.01</td><td>0</td><td>0</td><td>&#8722;0.55</td></tr><tr><th>VIP</th><td>0.01</td><td>0</td><td>0.5</td><td>0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29742-v4.xml">elife-29742-v4.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><label>Table 4.</label><caption><title>Connection probabilities for the random network model.</title></caption><table frame="hsides" rules="groups"><thead><tr><th/><th/><th colspan="4">From</th></tr><tr><th/><th/><th>E</th><th>PV</th><th>SST</th><th>VIP</th></tr></thead><tbody><tr><th rowspan="4">to</th><th>E</th><td>0.02</td><td>1</td><td>1</td><td>0</td></tr><tr><th>PV</th><td>0.01</td><td>1</td><td>0.85</td><td>0</td></tr><tr><th>SST</th><td>0.01</td><td>0</td><td>0</td><td>&#8722;0.55</td></tr><tr><th>VIP</th><td>0.01</td><td>0</td><td>0.5</td><td>0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29944-v1.xml">elife-29944-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="app1table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.29944.020</object-id><label>Appendix 1&#8212;table 1.</label><caption><title>Selected models of molecular evolution for partitions of the first five mtDNA genome alignment data sets.</title><p>All lengths are in base pairs. Reduced length excludes the Coding3 and CR partitions. For all RAxML analyses the GTR model was implemented. *The TrN model was selected, but this cannot be implemented in MrBayes and so the HKY model was used. EPA: evolutionary placement algorithm; CR: control region.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Data set</th><th rowspan="2" valign="bottom"/><th colspan="6">Partition</th><th colspan="2">Total length</th></tr><tr><th>Coding1</th><th>Coding2</th><th>Coding3</th><th>rRNAs</th><th>tRNAs</th><th>CR</th><th>All</th><th>Reduced</th></tr></thead><tbody><tr><td rowspan="2">1. White rhino outgroup</td><td>Length</td><td>3803</td><td>3803</td><td>3803</td><td>2579</td><td>1529</td><td>1066</td><td>16583</td><td>11714</td></tr><tr><td>Model</td><td>GTR&#160;+&#160;I&#160;+&#160;G</td><td>HKY&#160;+&#160;I&#160;+&#160;G</td><td>GTR&#160;+&#160;I&#160;+&#160;G</td><td>GTR&#160;+&#160;I&#160;+&#160;G</td><td>HKY&#160;+&#160;I&#160;+&#160;G</td><td>HKY*+I&#160;+&#160;G</td><td/><td/></tr><tr><td rowspan="2">2. Malayan tapir outgroup</td><td>Length</td><td>3803</td><td>3803</td><td>3803</td><td>2585</td><td>1530</td><td>1065</td><td>16589</td><td>11721</td></tr><tr><td>Model</td><td>GTR&#160;+&#160;I&#160;+&#160;G</td><td>HKY&#160;+&#160;I&#160;+&#160;G</td><td>GTR&#160;+&#160;I&#160;+&#160;G</td><td>GTR&#160;+&#160;I&#160;+&#160;G</td><td>HKY&#160;+&#160;I&#160;+&#160;G</td><td>HKY*+G</td><td/><td/></tr><tr><td rowspan="2">3. Dog&#160;+&#160;ceratomorphs outgroups</td><td>Length</td><td>3803</td><td>3803</td><td>3803</td><td>2615</td><td>1540</td><td>N/A</td><td>15564</td><td>11761</td></tr><tr><td>Model</td><td>GTR&#160;+&#160;I&#160;+&#160;G</td><td>HKY&#160;+&#160;I&#160;+&#160;G</td><td>GTR&#160;+&#160;I&#160;+&#160;G</td><td>GTR&#160;+&#160;I&#160;+&#160;G</td><td>HKY&#160;+&#160;I&#160;+&#160;G</td><td>N/A</td><td/><td/></tr><tr><td rowspan="2">4. EPA</td><td>Length</td><td>3803</td><td>3803</td><td>3803</td><td>2601</td><td>1534</td><td>1118</td><td>16662</td><td>11741</td></tr><tr><td>Model</td><td>GTR&#160;+&#160;I&#160;+&#160;G</td><td>TrN&#160;+&#160;I&#160;+&#160;G</td><td>GTR&#160;+&#160;I&#160;+&#160;G</td><td>GTR&#160;+&#160;I&#160;+&#160;G</td><td>HKY&#160;+&#160;I&#160;+&#160;G</td><td>HKY&#160;+&#160;I&#160;+&#160;G</td><td/><td/></tr><tr><td rowspan="2">5. Equids only</td><td>Length</td><td>3802</td><td>3802</td><td>3802</td><td>2571</td><td>1528</td><td>971</td><td>16476</td><td>11703</td></tr><tr><td>Model</td><td>TrN&#160;+&#160;I&#160;+&#160;G</td><td>TrN&#160;+&#160;I&#160;+&#160;G</td><td>GTR&#160;+&#160;G</td><td>TrN&#160;+&#160;I&#160;+&#160;G</td><td>HKY&#160;+&#160;I</td><td>HKY&#160;+&#160;G</td><td valign="bottom"/><td valign="bottom"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29944-v1.xml">elife-29944-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="app2table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.29944.028</object-id><label>Appendix 2&#8212;table 1.</label><caption><title>Topological shape and support values for the best supported trees.</title><p>These results are from the Bayesian and maximum likelihood (ML) analyses of mtDNA data sets 1&#8211;3, including either the all or reduced partition sets, and with <italic>Hippidion</italic> sequences either included or excluded. Topology numbers and node letters refer to those outlined in <xref ref-type="fig" rid="app2fig3">Appendix 2&#8212;figure 3</xref>. Bayesian posterior probability support of&#160;&gt;0.99 and ML bootstrap support of&#160;&gt;95% are in bold for nodes A and B. *support for nodes that are consistent with topology one in <xref ref-type="fig" rid="app2fig3">Appendix 2&#8212;figure 3</xref>. NCs: non-caballines.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Outgroup</th><th rowspan="2">Partitions</th><th rowspan="2"><italic>Hippidion</italic>?</th><th rowspan="2">Tips</th><th rowspan="2">Analysis method</th><th rowspan="2">Topology</th><th colspan="6">Support</th></tr><tr><th>Node A</th><th>Node B</th><th><italic>Hippidion</italic></th><th>NWSL</th><th>NCs</th><th>Caballines</th></tr></thead><tbody><tr><td rowspan="8" valign="bottom">White rhino <break/>&#8195;(Data set 1)</td><td rowspan="4">All</td><td rowspan="2">Excluded</td><td rowspan="2">63</td><td>Bayesian</td><td><bold>1/2/3</bold></td><td><bold>0.996*</bold></td><td>N/A</td><td>N/A</td><td>1.000</td><td>1.000</td><td>1.000</td></tr><tr><td>ML</td><td>1/2/3</td><td>71*</td><td>N/A</td><td>N/A</td><td>100</td><td>99</td><td>100</td></tr><tr><td rowspan="2">Included</td><td rowspan="2">69</td><td>Bayesian</td><td>2</td><td>0.751</td><td><bold>1.000*</bold></td><td>1.000</td><td>1.000</td><td>1.000</td><td>1.000</td></tr><tr><td>ML</td><td>1</td><td>64*</td><td><bold>96*</bold></td><td>100</td><td>100</td><td>100</td><td>100</td></tr><tr><td rowspan="4">Reduced</td><td rowspan="2">Excluded</td><td rowspan="2">63</td><td>Bayesian</td><td><bold>1/2/3</bold></td><td><bold>1.000*</bold></td><td>N/A</td><td>N/A</td><td>1.000</td><td>1.000</td><td>1.000</td></tr><tr><td>ML</td><td><bold>1/2/3</bold></td><td><bold>100*</bold></td><td>N/A</td><td>N/A</td><td>99</td><td>100</td><td>100</td></tr><tr><td rowspan="2">Included</td><td rowspan="2">69</td><td>Bayesian</td><td>2</td><td>0.948</td><td><bold>1.000*</bold></td><td>1.000</td><td>1.000</td><td>1.000</td><td>1.000</td></tr><tr><td>ML</td><td>2</td><td>73</td><td><bold>98*</bold></td><td>100</td><td>99</td><td>100</td><td>100</td></tr><tr><td rowspan="8" valign="bottom">Malayan tapir <break/>&#8195;(Data set 2)</td><td rowspan="4">All</td><td rowspan="2">Excluded</td><td rowspan="2">63</td><td>Bayesian</td><td>5/7</td><td>0.971</td><td>N/A</td><td>N/A</td><td>1.000</td><td>1.000</td><td>1.000</td></tr><tr><td>ML</td><td>5/7</td><td>87</td><td>N/A</td><td>N/A</td><td>100</td><td>99</td><td>99</td></tr><tr><td rowspan="2">Included</td><td rowspan="2">69</td><td>Bayesian</td><td>6</td><td>0.808</td><td>0.867</td><td>1.000</td><td>1.000</td><td>1.000</td><td>1.000</td></tr><tr><td>ML</td><td>6</td><td>55</td><td>63</td><td>100</td><td>100</td><td>100</td><td>100</td></tr><tr><td rowspan="4">Reduced</td><td rowspan="2">Excluded</td><td rowspan="2">63</td><td>Bayesian</td><td>1/2/3</td><td>0.675*</td><td>N/A</td><td>N/A</td><td>1.000</td><td>1.000</td><td>1.000</td></tr><tr><td>ML</td><td>4/6</td><td>28</td><td>N/A</td><td>N/A</td><td>100</td><td>96</td><td>98</td></tr><tr><td rowspan="2">Included</td><td rowspan="2">69</td><td>Bayesian</td><td>3</td><td>0.685</td><td>0.864*</td><td>1.000</td><td>1.000</td><td>1.000</td><td>1.000</td></tr><tr><td>ML</td><td>3</td><td>70</td><td>69</td><td>100</td><td>100</td><td>100</td><td>100</td></tr><tr><td rowspan="8">Dog&#160;+&#160;ceratomorphs <break/>&#8195;(Data set 3)</td><td rowspan="4">All</td><td rowspan="2">Excluded</td><td rowspan="2">71</td><td>Bayesian</td><td>1/2/3</td><td>0.598*</td><td>N/A</td><td>N/A</td><td>1.000</td><td>1.000</td><td>1.000</td></tr><tr><td>ML</td><td>4/6</td><td>59</td><td>N/A</td><td>N/A</td><td>100</td><td>100</td><td>100</td></tr><tr><td rowspan="2">Included</td><td rowspan="2">77</td><td>Bayesian</td><td><bold>1</bold></td><td><bold>1.000*</bold></td><td><bold>1.000*</bold></td><td>1.000</td><td>1.000</td><td>1.000</td><td>1.000</td></tr><tr><td>ML</td><td>1</td><td>94*</td><td><bold>96*</bold></td><td>100</td><td>100</td><td>100</td><td>100</td></tr><tr><td rowspan="4">Reduced</td><td rowspan="2">Excluded</td><td rowspan="2">71</td><td>Bayesian</td><td><bold>1/2/3</bold></td><td><bold>0.999*</bold></td><td>N/A</td><td>N/A</td><td>1.000</td><td>1.000</td><td>1.000</td></tr><tr><td>ML</td><td>1/2/3</td><td><bold>97*</bold></td><td>N/A</td><td>N/A</td><td>100</td><td>100</td><td>100</td></tr><tr><td rowspan="2">Included</td><td rowspan="2">77</td><td>Bayesian</td><td><bold>1</bold></td><td><bold>1.000*</bold></td><td><bold>1.000*</bold></td><td>1.000</td><td>1.000</td><td>1.000</td><td>1.000</td></tr><tr><td>ML</td><td>1</td><td><bold>99*</bold></td><td><bold>100*</bold></td><td>100</td><td>100</td><td>100</td><td>100</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29944-v1.xml">elife-29944-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="app2table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.29944.029</object-id><label>Appendix 2&#8212;table 2.</label><caption><title>The <italic>a posteriori</italic> phylogenetic placement likelihood for eight ceratomorph (rhino and tapir) outgroups.</title><p>These analyses used a ML evolutionary placement algorithm, whilst varying the partition set used (all or reduced), and either including or excluding <italic>Hippidion</italic> sequences. Likelihoods&#160;&gt;0.95 are in bold. Topology numbers refer to those outlined in <xref ref-type="fig" rid="app2fig3">Appendix 2&#8212;figure 3</xref>. Genbank accession numbers are given in parentheses after outgroup names.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Partitions</th><th rowspan="2">Outgroup</th><th><italic>Hippidion</italic>?</th><th colspan="4">Included</th><th colspan="3">Excluded</th></tr><tr><th>Topology</th><th>1</th><th>2</th><th>3</th><th>6</th><th>1/2/3</th><th>4/6</th><th>5/7</th></tr></thead><tbody><tr><td rowspan="8">&#8195;All</td><td colspan="2"><italic>Tapirus terrestris</italic> (AJ428947)</td><td>0.456</td><td>0.317</td><td>0.205</td><td>0.018</td><td>0.549</td><td>0.313</td><td>0.139</td></tr><tr><td colspan="2"><italic>Tapirus indicus</italic> (NC023838)</td><td>0.275</td><td>0.105</td><td>0.225</td><td>0.389</td><td>0.050</td><td>0.908</td><td>0.042</td></tr><tr><td colspan="2"><italic>Coelodonta antiquitatis</italic> (NC012681)</td><td><bold>0.998</bold></td><td/><td/><td/><td>0.248</td><td>0.451</td><td>0.301</td></tr><tr><td colspan="2"><italic>Dicerorhinus sumatrensis</italic> (NC012684)</td><td><bold>0.981</bold></td><td/><td>0.009</td><td/><td>0.155</td><td>0.553</td><td>0.292</td></tr><tr><td colspan="2"><italic>Rhinoceros unicornis</italic> (NC001779)</td><td><bold>0.998</bold></td><td/><td/><td/><td>0.529</td><td>0.334</td><td>0.137</td></tr><tr><td colspan="2"><italic>Rhinoceros sondaicus</italic> (NC012683)</td><td><bold>0.989</bold></td><td>0.006</td><td/><td/><td>0.732</td><td>0.196</td><td>0.072</td></tr><tr><td colspan="2"><italic>Ceratotherium simum</italic> (NC001808)</td><td>0.448</td><td>0.499</td><td>0.053</td><td/><td>0.949</td><td>0.018</td><td>0.033</td></tr><tr><td colspan="2"><italic>Diceros bicornis</italic> (NC012682)</td><td>0.917</td><td>0.065</td><td>0.018</td><td/><td>0.851</td><td>0.073</td><td>0.076</td></tr><tr><td rowspan="8">&#8195;Reduced</td><td colspan="2"><italic>Tapirus terrestris</italic> (AJ428947)</td><td>0.410</td><td>0.391</td><td>0.199</td><td/><td><bold>0.987</bold></td><td/><td>0.012</td></tr><tr><td colspan="2"><italic>Tapirus indicus</italic> (NC023838)</td><td>0.536</td><td>0.298</td><td>0.166</td><td/><td><bold>0.995</bold></td><td/><td/></tr><tr><td colspan="2"><italic>Coelodonta antiquitatis</italic> (NC012681)</td><td>0.411</td><td>0.554</td><td>0.035</td><td/><td><bold>1.000</bold></td><td/><td/></tr><tr><td colspan="2"><italic>Dicerorhinus sumatrensis</italic> (NC012684)</td><td><bold>0.983</bold></td><td>0.015</td><td/><td/><td><bold>1.000</bold></td><td/><td/></tr><tr><td colspan="2"><italic>Rhinoceros unicornis</italic> (NC001779)</td><td><bold>0.998</bold></td><td/><td/><td/><td><bold>1.000</bold></td><td/><td/></tr><tr><td colspan="2"><italic>Rhinoceros sondaicus</italic> (NC012683)</td><td>0.895</td><td>0.102</td><td/><td/><td><bold>1.000</bold></td><td/><td/></tr><tr><td colspan="2"><italic>Ceratotherium simum</italic> (NC001808)</td><td>0.296</td><td>0.704</td><td/><td/><td><bold>1.000</bold></td><td/><td/></tr><tr><td colspan="2"><italic>Diceros bicornis</italic> (NC012682)</td><td><bold>0.996</bold></td><td/><td/><td/><td><bold>1.000</bold></td><td/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-29944-v1.xml">elife-29944-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="app2table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.29944.030</object-id><label>Appendix 2&#8212;table 3.</label><caption><title>The <italic>a posteriori</italic> phylogenetic placement likelihood for 21 published equid mitochondrial sequences.</title><p>These analyses used the ML evolutionary placement algorithm, whilst varying the partition set used (all or reduced), and either including or excluding <italic>Hippidion</italic> sequences. Sample names are given in parentheses after the species or group name. Localities are given for NWSL equids only. Likelihoods&#160;&gt;0.95 are in bold. *<italic>Equus</italic> includes only caballines and non-caballine equids (NCE). **For EQ04 from Alberta, other placement likelihood values for the <italic>Hippidion</italic>&#160;included/excluded partitions were: Within caballines: 0.003/0.002, Sister to caballines: 0.002/0.002, Within NCE: 0.246/0.245, Sister to NCE: 0.004/0.003. No placements were returned for &#8216;within <italic>Hippidion</italic>&#8217;. bp: base pairs.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><italic>Hippidion</italic>?</th><th rowspan="2">Partition</th><th rowspan="2">Published sample</th><th rowspan="2">Sequence length (bp)</th><th rowspan="2">Locality</th><th colspan="6">Placement</th></tr><tr><th>Sister to <italic>E. ovodovi</italic></th><th>Sister to <italic>Hippidion</italic></th><th>Within NWSL</th><th>Sister to NWSL</th><th>Sister to <italic>Equus*</italic></th><th>Other**</th></tr></thead><tbody><tr><td rowspan="10">Included</td><td rowspan="10">All</td><td><italic>E. ovodovi</italic> (ACAD2305)</td><td>688</td><td/><td><bold>1.000</bold></td><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>E. ovodovi</italic> (ACAD2302)</td><td>688</td><td/><td><bold>1.000</bold></td><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>E. ovodovi</italic> (ACAD2303)</td><td>688</td><td/><td><bold>1.000</bold></td><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>H. devillei</italic> (ACAD3615)</td><td>476</td><td/><td>N/A</td><td><bold>1.000</bold></td><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>H. devillei</italic> (ACAD3625)</td><td>543</td><td/><td>N/A</td><td><bold>1.000</bold></td><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>H. devillei</italic> (ACAD3627)</td><td>543</td><td/><td>N/A</td><td><bold>1.000</bold></td><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>H. devillei</italic> (ACAD3628)</td><td>543</td><td/><td>N/A</td><td><bold>0.999</bold></td><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>H. devillei</italic> (ACAD3629)</td><td>476</td><td/><td>N/A</td><td><bold>0.999</bold></td><td/><td/><td/><td valign="bottom"/></tr><tr><td>NWSL equid (JW125)</td><td>720</td><td>Klondike, YT</td><td>N/A</td><td/><td><bold>0.996</bold></td><td/><td/><td valign="bottom"/></tr><tr><td>NWSL equid (JW126)</td><td>720</td><td>Klondike, YT</td><td>N/A</td><td/><td><bold>0.999</bold></td><td/><td/><td valign="bottom"/></tr><tr><td rowspan="13">Included</td><td rowspan="11">All</td><td>NWSL equid (EQ01)</td><td>620</td><td>Dry Cave, NM</td><td>N/A</td><td/><td>0.735</td><td>0.256</td><td/><td valign="bottom"/></tr><tr><td>NWSL equid (EQ03)</td><td>117</td><td>Dry Cave, NM</td><td>N/A</td><td>0.002</td><td><bold>0.974</bold></td><td>0.011</td><td>0.003</td><td valign="bottom"/></tr><tr><td>NWSL equid (EQ04)</td><td>117</td><td>Edmonton, AB</td><td>N/A</td><td>0.004</td><td>0.703</td><td>0.014</td><td>0.007</td><td valign="bottom">0.255</td></tr><tr><td>NWSL equid (EQ09)</td><td>620</td><td>Natural Trap Cave, WY</td><td>N/A</td><td/><td><bold>0.981</bold></td><td>0.014</td><td/><td valign="bottom"/></tr><tr><td>NWSL equid (EQ13)</td><td>620</td><td>Natural Trap Cave, WY</td><td>N/A</td><td/><td><bold>0.992</bold></td><td/><td/><td valign="bottom"/></tr><tr><td>NWSL equid (EQ16)</td><td>464</td><td>Dry Cave, NM</td><td>N/A</td><td valign="bottom"/><td>0.854</td><td>0.138</td><td/><td valign="bottom"/></tr><tr><td>NWSL equid (EQ22)</td><td>620</td><td>Natural Trap Cave, WY</td><td>N/A</td><td/><td><bold>0.999</bold></td><td/><td/><td valign="bottom"/></tr><tr><td>NWSL equid (EQ30)</td><td>393</td><td>San Josecito Cave, MX-NL</td><td>N/A</td><td/><td>0.792</td><td>0.198</td><td/><td valign="bottom"/></tr><tr><td>NWSL equid (EQ41)</td><td>398</td><td>Natural Trap Cave, WY</td><td>N/A</td><td/><td><bold>0.997</bold></td><td/><td/><td valign="bottom"/></tr><tr><td>NWSL equid (JW328)</td><td>mitogenome</td><td>Mineral Hill Cave, NV</td><td>N/A</td><td/><td><bold>1.000</bold></td><td/><td/><td valign="bottom"/></tr><tr><td>NWSL equid (MS272)</td><td>mitogenome</td><td>Klondike, YT</td><td>N/A</td><td/><td/><td><bold>1.000</bold></td><td/><td valign="bottom"/></tr><tr><td rowspan="2">Reduced</td><td>NWSL equid (JW328)</td><td>mitogenome</td><td>Mineral Hill Cave, NV</td><td>N/A</td><td/><td><bold>0.996</bold></td><td/><td/><td valign="bottom"/></tr><tr><td>NWSL equid (MS272)</td><td>mitogenome</td><td>Klondike, YT</td><td>N/A</td><td/><td/><td><bold>1.000</bold></td><td/><td valign="bottom"/></tr><tr><td rowspan="18">Excluded</td><td rowspan="16">All</td><td><italic>E. ovodovi</italic> (ACAD2305)</td><td>688</td><td/><td><bold>1.000</bold></td><td>N/A</td><td/><td/><td>N/A</td><td valign="bottom"/></tr><tr><td><italic>E. ovodovi</italic> (ACAD2302)</td><td>688</td><td/><td><bold>1.000</bold></td><td>N/A</td><td/><td/><td>N/A</td><td valign="bottom"/></tr><tr><td><italic>E. ovodovi</italic> (ACAD2303)</td><td>688</td><td/><td><bold>1.000</bold></td><td>N/A</td><td/><td/><td>N/A</td><td valign="bottom"/></tr><tr><td>NWSL equid (JW125)</td><td>720</td><td>Klondike, YT</td><td>N/A</td><td>N/A</td><td><bold>0.996</bold></td><td/><td>N/A</td><td valign="bottom"/></tr><tr><td>NWSL equid (JW126)</td><td>720</td><td>Klondike, YT</td><td>N/A</td><td>N/A</td><td><bold>0.999</bold></td><td/><td>N/A</td><td valign="bottom"/></tr><tr><td>NWSL equid (EQ01)</td><td>620</td><td>Dry Cave, NM</td><td>N/A</td><td>N/A</td><td>0.731</td><td>0.259</td><td>N/A</td><td valign="bottom"/></tr><tr><td>NWSL equid (EQ03)</td><td>117</td><td>Dry Cave, NM</td><td>N/A</td><td>N/A</td><td><bold>0.980</bold></td><td>0.010</td><td>N/A</td><td valign="bottom"/></tr><tr><td>NWSL equid (EQ04)</td><td>117</td><td>Edmonton, AB</td><td>N/A</td><td>N/A</td><td>0.721</td><td>0.013</td><td>N/A</td><td valign="bottom">0.252</td></tr><tr><td>NWSL equid (EQ09)</td><td>620</td><td>Natural Trap Cave, WY</td><td>N/A</td><td>N/A</td><td><bold>0.987</bold></td><td>0.008</td><td>N/A</td><td/></tr><tr><td>NWSL equid (EQ13)</td><td>620</td><td>Natural Trap Cave, WY</td><td>N/A</td><td>N/A</td><td><bold>0.993</bold></td><td/><td>N/A</td><td/></tr><tr><td>NWSL equid (EQ16)</td><td>464</td><td>Dry Cave, NM</td><td>N/A</td><td>N/A</td><td>0.844</td><td>0.148</td><td>N/A</td><td/></tr><tr><td>NWSL equid (EQ22)</td><td>620</td><td>Natural Trap Cave, WY</td><td>N/A</td><td>N/A</td><td><bold>0.999</bold></td><td/><td>N/A</td><td/></tr><tr><td>NWSL equid (EQ30)</td><td>393</td><td>San Josecito Cave, MX-NL</td><td>N/A</td><td>N/A</td><td>0.788</td><td>0.203</td><td>N/A</td><td/></tr><tr><td>NWSL equid (EQ41)</td><td>398</td><td>Natural Trap Cave, WY</td><td>N/A</td><td>N/A</td><td><bold>0.995</bold></td><td/><td>N/A</td><td/></tr><tr><td>NWSL equid (JW328)</td><td>mitogenome</td><td>Mineral Hill Cave, NV</td><td>N/A</td><td>N/A</td><td><bold>1.000</bold></td><td/><td>N/A</td><td/></tr><tr><td>NWSL equid (MS272)</td><td>mitogenome</td><td>Klondike, YT</td><td>N/A</td><td>N/A</td><td/><td><bold>1.000</bold></td><td>N/A</td><td/></tr><tr><td rowspan="2">Reduced</td><td>NWSL equid (JW328)</td><td>mitogenome</td><td>Mineral Hill Cave, NV</td><td>N/A</td><td>N/A</td><td><bold>0.995</bold></td><td/><td>N/A</td><td valign="bottom"/></tr><tr><td>NWSL equid (MS272)</td><td>mitogenome</td><td>Klondike, YT</td><td>N/A</td><td>N/A</td><td/><td><bold>1.000</bold></td><td>N/A</td><td valign="bottom"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-30454-v1.xml">elife-30454-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="inlinetable1" position="anchor"><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Allele</th><th colspan="6" valign="top">PCR conditions and comments</th><th colspan="3" valign="top">Genotping methods</th></tr><tr><th valign="top">Primer pair <break/>(ID#s)</th><th valign="top">Align <break/>(T<sup>o</sup>C)</th><th valign="top">Extend <break/>(secs)</th><th valign="top">Cycles</th><th valign="top">Size in bp <break/>(WT/M)</th><th valign="top">Notes</th><th valign="top">AGE</th><th valign="top">HMA</th><th valign="top">Sequencing <break/>Primer ID#</th></tr></thead><tbody><tr><td rowspan="2" valign="top"><italic>gipc1<sup>skt1</sup></italic></td><td valign="top">12891&#8211;12889</td><td valign="top">62</td><td valign="top">25</td><td valign="top">34</td><td valign="top">372/368</td><td rowspan="2" valign="top">N-PCR <break/>(if fixed)</td><td valign="top">n/a</td><td valign="top">For hetz</td><td valign="top">12891</td></tr><tr><td valign="top">13655&#8211;13653</td><td valign="top">64</td><td valign="top">19</td><td valign="top">40</td><td valign="top">300/296</td><td valign="top">n/a</td><td valign="top">For hetz</td><td valign="top">13653</td></tr><tr><td valign="top"><italic>gipc1 <sup>skt2</sup></italic></td><td valign="top">12891&#8211;12889</td><td valign="top">57</td><td valign="top">30</td><td valign="top">34</td><td valign="top">372/379</td><td valign="top">n/a</td><td valign="top">n/a</td><td valign="top">For hetz</td><td valign="top">12891</td></tr><tr><td valign="top"><italic>gipc2 <sup>skt3</sup></italic></td><td valign="top">13667&#8211;13422</td><td valign="top">60</td><td valign="top">14</td><td valign="top">35</td><td valign="top">218/216</td><td valign="top">n/a</td><td valign="top">n/a</td><td valign="top">n/a</td><td valign="top">13421</td></tr><tr><td valign="top"><italic>gipc2 <sup>skt4</sup></italic></td><td valign="top">13667&#8211;13422</td><td valign="top">60</td><td valign="top">14</td><td valign="top">35</td><td valign="top">218/172</td><td valign="top">n/a</td><td valign="top">Yes</td><td valign="top">n/a</td><td valign="top">13421</td></tr><tr><td valign="top"><italic>gipc3 <sup>skt5</sup></italic></td><td valign="top">13510&#8211;13512</td><td valign="top">67.7</td><td valign="top">25</td><td valign="top">38</td><td valign="top">377/337</td><td valign="top">n/a</td><td valign="top">Yes</td><td valign="top">n/a</td><td valign="top">13512</td></tr><tr><td rowspan="2" valign="top"><italic>plxnd1<sup>skt6</sup></italic></td><td valign="top">11844&#8211;11845</td><td valign="top">65.9</td><td valign="top">30</td><td valign="top">34</td><td valign="top">442/438</td><td rowspan="2">N-PCR</td><td valign="top">n/a</td><td valign="top">n/a</td><td valign="top">n/a</td></tr><tr><td valign="top">12532&#8211;11845</td><td valign="top">57.7</td><td valign="top">30</td><td valign="top">34</td><td valign="top">262/258</td><td valign="top">n/a</td><td valign="top">n/a</td><td valign="top">12532</td></tr><tr><td rowspan="2" valign="top"><italic>plxnd1<sup>fov01b</sup></italic></td><td valign="top">13498&#8211;10167</td><td valign="top">63</td><td valign="top">16</td><td valign="top">38</td><td valign="top">519/519</td><td rowspan="2">N-PCR</td><td rowspan="2">n/a</td><td rowspan="2">n/a</td><td rowspan="2">13646</td></tr><tr><td valign="top">13646&#8211;13647</td><td valign="top">57.5</td><td valign="top">11</td><td valign="top">38</td><td valign="top">395/395</td></tr><tr><td valign="top"><italic>Tg(fli1a:</italic> <break/><italic>GAL4FF)<sup>ubs4</sup></italic></td><td valign="top">11730&#8211;11731</td><td valign="top">57.5</td><td valign="top">45</td><td valign="top">35</td><td valign="top">402</td><td valign="top">Only Tg presence</td><td valign="top">Yes</td><td valign="top">n/a</td><td valign="top">n/a</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-30483-v1.xml">elife-30483-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.30483.013</object-id><label>Table 2.</label><caption><title>Statistics of PilQ domain fits.</title><p>Homology models of the different PilQ domains were fitted into the respective ring module maps.&#160;Each domain was initially placed within the density maps in multiple orientations (<italic>n</italic>&#160;=&#160;10,000), spanning the entire Euler angular space, and subjected to steepest descent local optimization. The resulting fits were clustered and ranked by their CC-scores with the corresponding density maps fits. The table lists the top-5 fits for each domain in the Euler-angle search, along with their CC-scores and the number of independent optimization runs ending in this cluster. The top-ranked fits were assessed for compatibility of the N-to-C-terminal domain orientations (bottom-to-top). The compatible orientations with the highest CC-value was selected as initial fit. Subsequent refinement using MDFF flexible fitting (<xref ref-type="bibr" rid="bib58">Trabuco et al., 2008</xref>) further improved the fits with minimal structural change.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">PilQ domain fits</th><th rowspan="2" valign="top">Total no. of fits</th><th rowspan="2" valign="top">No. of unique fits</th><th colspan="3" valign="top">Top 5 solutions</th><th rowspan="2" valign="top">Initial chosen <break/>protomer fit. <break/>CC (Cluster #)</th><th rowspan="2" valign="top">Fit after MDFF. <break/>CC <break/>(RMSD, &#197;)</th><th rowspan="2" valign="top">CC for C<sub>13</sub> <break/>oligomeric ring <break/>model</th><th rowspan="2" valign="top">CC after MDFF <break/>run with masked <break/>density map</th></tr><tr><th valign="top">Rank #</th><th valign="top">CC with map</th><th valign="top">Runs</th></tr></thead><tbody><tr><td valign="top">N0</td><td valign="top">10000</td><td valign="top">152</td><td valign="top">Rank 1 <break/>Rank 2 <break/>Rank 3 <break/>Rank 4 <break/>Rank 5</td><td valign="top">0.7028 <break/>0.6715 <break/>0.6696 <break/>0.6681 <break/>0.6670</td><td valign="top">576 <break/>327 <break/>167 <break/>33 <break/>174</td><td valign="top">0.7028 <break/>(Rank&#160;1)</td><td valign="top">0.7863 <break/>(3.210)</td><td valign="top">0.70</td><td valign="top">0.85</td></tr><tr><td valign="top">N1</td><td valign="top">10000</td><td valign="top">115</td><td valign="top">Rank 1 <break/>Rank 2 <break/>Rank 3 <break/>Rank 4 <break/>Rank 5</td><td valign="top">0.6673 <break/>0.6639 <break/>0.6583 <break/>0.6570 <break/>0.6551</td><td valign="top">530 <break/>682 <break/>336 <break/>110 <break/>211</td><td valign="top">0.6673 <break/>(Rank 1)</td><td valign="top">0.7682 <break/>(4.044)</td><td valign="top">0.70</td><td valign="top">0.85</td></tr><tr><td valign="top">N2</td><td valign="top">10000</td><td valign="top">48</td><td valign="top">Rank 1 <break/>Rank 2 <break/>Rank 3 <break/>Rank 4 <break/>Rank 5</td><td valign="top">0.8381 <break/>0.8270 <break/>0.8270 <break/>0.8256 <break/>0.8248</td><td valign="top">784 <break/>815 <break/>364 <break/>475 <break/>953</td><td valign="top">0.8381 <break/>(Rank 1)</td><td valign="top">0.9139 <break/>(1.883)</td><td valign="top">0.84</td><td valign="top">0.91</td></tr><tr><td valign="top">N3</td><td valign="top">10000</td><td valign="top">32</td><td valign="top">Rank 1 <break/>Rank 2 <break/>Rank 3 <break/>Rank 4 <break/>Rank 5</td><td valign="top">0.8279 <break/>0.8253 <break/>0.8249 <break/>0.8237 <break/>0.8222</td><td valign="top">473 <break/>457 <break/>889 <break/>829 <break/>408</td><td valign="top">0.8249 (Rank 3)</td><td valign="top">0.9026 <break/>(3.376)</td><td valign="top">0.84</td><td valign="top">0.91</td></tr><tr><td valign="top">N4</td><td valign="top">10000</td><td valign="top">32</td><td valign="top">Rank 1 <break/>Rank 2 <break/>Rank 3 <break/>Rank 4 <break/>Rank 5</td><td valign="top">0.7982 <break/>0.7931 <break/>0.7793 <break/>0.7521 <break/>0.7479</td><td valign="top">1106 <break/>757 <break/>483 <break/>434 <break/>511</td><td valign="top">0.7982 (Rank 1)</td><td valign="top">0.8539 <break/>(2.166)</td><td valign="top">0.77</td><td valign="top">0.93</td></tr><tr><td valign="top">N5</td><td valign="top">10000</td><td valign="top">65</td><td valign="top">Rank 1 <break/>Rank 2 <break/>Rank 3 <break/>Rank 4 <break/>Rank 5</td><td valign="top">0.7394 <break/>0.7371 <break/>0.7274 <break/>0.7218 <break/>0.7203</td><td valign="top">206 <break/>292 <break/>364 <break/>559 <break/>329</td><td valign="top">0.7203 (Rank 5)</td><td valign="top">0.7562 <break/>(2.378)</td><td valign="top">0.77</td><td valign="top">0.82</td></tr><tr><td valign="top">Secretin</td><td valign="top">9527</td><td valign="top">86</td><td valign="top">Rank 1 <break/>Rank 2 <break/>Rank 3 <break/>Rank 4 <break/>Rank 5</td><td valign="top">0.7634 <break/>0.7063 <break/>0.7013 <break/>0.6958 <break/>0.6899</td><td valign="top">672 <break/>156 <break/>285 <break/>104 <break/>370</td><td valign="top">0.7634 (Rank 1)</td><td valign="top">0.8314 <break/>(1.872)</td><td valign="top">0.77</td><td valign="top">0.82</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-30490-v2.xml">elife-30490-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.30490.007</object-id><label>Table 4.</label><caption><title>Compilation of Whole-Exome-Sequencing data for the genes of the FANC pathway, excluding all the genes except FANCM as potentially harboring a causative variant.</title></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2" valign="middle">Gene name</th><th align="center" rowspan="2" valign="middle">Alias</th><th align="center" colspan="3" valign="middle">Mean depth in targeted exons (1)</th><th align="center" valign="middle">Nb variants in</th><th align="center" valign="middle">Htz variants</th><th align="center" valign="middle">Mean ratio for htz allelic</th><th align="center" colspan="5" valign="middle">Presence of htz variants (5)</th><th align="center" colspan="2" valign="middle">Nbr of rare variants (6)</th></tr><tr><th align="center" valign="middle"><break/>mother</th><th align="center" valign="middle">POI&#160;1</th><th align="center" valign="middle"><break/>POI 2</th><th align="center" valign="middle">family (2)</th><th align="center" valign="middle">in &gt;= 1 sister (3)</th><th align="center" valign="middle">reads in sisters (4)</th><th align="center" valign="middle">upstream</th><th align="center" valign="middle">5'UTR</th><th align="center" valign="middle">deep intronic</th><th align="center" valign="middle">3'UTR</th><th align="center" valign="middle">downstream</th><th align="center" valign="middle">correct segregation</th><th align="center" valign="middle">and pathogenic</th></tr></thead><tbody><tr><td align="center" valign="middle"><bold><italic>FANCA</italic></bold></td><td align="center" valign="middle"/><td align="center" valign="middle">87.9</td><td align="center" valign="middle">94.9</td><td align="center" valign="middle">90.3</td><td align="center" valign="middle">29</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0.92</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCB</italic></bold></td><td align="center" valign="middle"/><td align="center" valign="middle">88.6</td><td align="center" valign="middle">93.8</td><td align="center" valign="middle">89.9</td><td align="center" valign="middle">2</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">1.14</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCC</italic></bold></td><td align="center" valign="middle"/><td align="center" valign="middle">64.4</td><td align="center" valign="middle">71.5</td><td align="center" valign="middle">68.2</td><td align="center" valign="middle">0 in this family</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/></tr><tr><td align="center" valign="middle"><bold><italic>FANCD1</italic></bold></td><td align="center" valign="middle"><bold><italic>BRCA2</italic></bold></td><td align="center" valign="middle">84.6</td><td align="center" valign="middle">90.5</td><td align="center" valign="middle">84.5</td><td align="center" valign="middle">12</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">1.01</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCD2</italic></bold></td><td align="center" valign="middle"/><td align="center" valign="middle">76.1</td><td align="center" valign="middle">82.1</td><td align="center" valign="middle">75.0</td><td align="center" valign="middle">2</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0.825</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCE</italic></bold></td><td align="center" valign="middle"/><td align="center" valign="middle">89.5</td><td align="center" valign="middle">100.2</td><td align="center" valign="middle">99.0</td><td align="center" valign="middle">2</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">1.08</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCF</italic></bold></td><td align="center" valign="middle"/><td align="center" valign="middle">143.0</td><td align="center" valign="middle">150.2</td><td align="center" valign="middle">139.7</td><td align="center" valign="middle">0 in this family</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/></tr><tr><td align="center" valign="middle"><bold><italic>FANCG</italic></bold></td><td align="center" valign="middle"/><td align="center" valign="middle">123.5</td><td align="center" valign="middle">139.8</td><td align="center" valign="middle">130.2</td><td align="center" valign="middle">1</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">1.22</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCI</italic></bold></td><td align="center" valign="middle"/><td align="center" valign="middle">75.7</td><td align="center" valign="middle">80.3</td><td align="center" valign="middle">76.6</td><td align="center" valign="middle">9</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0.89</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCJ</italic></bold></td><td align="center" valign="middle"><bold><italic>BRIP1</italic></bold></td><td align="center" valign="middle">104.2</td><td align="center" valign="middle">113.0</td><td align="center" valign="middle">107.7</td><td align="center" valign="middle">4</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0.76</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCL</italic></bold></td><td align="center" valign="middle"/><td align="center" valign="middle">80.2</td><td align="center" valign="middle">88.5</td><td align="center" valign="middle">80.7</td><td align="center" valign="middle">3</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0.96</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCM</italic></bold></td><td align="center" valign="middle"/><td align="center" valign="middle">86.9</td><td align="center" valign="middle">91.3</td><td align="center" valign="middle">87.8</td><td align="center" valign="middle"><bold>1</bold></td><td align="center" valign="middle">no</td><td align="center" valign="middle">/</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle"><bold>1</bold></td><td align="center" valign="middle"><bold>1</bold></td></tr><tr><td align="center" valign="middle"><bold><italic>FANCN</italic></bold></td><td align="center" valign="middle"><bold><italic>PALB2</italic></bold></td><td align="center" valign="middle">97.0</td><td align="center" valign="middle">106.9</td><td align="center" valign="middle">101.1</td><td align="center" valign="middle">0 in this family</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/></tr><tr><td align="center" valign="middle"><bold><italic>FANCO</italic></bold></td><td align="center" valign="middle"><bold><italic>RAD51C</italic></bold></td><td align="center" valign="middle">92.1</td><td align="center" valign="middle">103.2</td><td align="center" valign="middle">94.9</td><td align="center" valign="middle">1</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0.77</td><td align="center" valign="middle">no</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCP</italic></bold></td><td align="center" valign="middle"><bold><italic>SLX4</italic></bold></td><td align="center" valign="middle">107.9</td><td align="center" valign="middle">116.4</td><td align="center" valign="middle">112.8</td><td align="center" valign="middle">15</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0.88</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCQ</italic></bold></td><td align="center" valign="middle"><bold><italic>ERCC4</italic></bold></td><td align="center" valign="middle">91.5</td><td align="center" valign="middle">98.1</td><td align="center" valign="middle">90.9</td><td align="center" valign="middle">2</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">1.21</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCR</italic></bold></td><td align="center" valign="middle"><bold><italic>RAD51</italic></bold></td><td align="center" valign="middle">75.9</td><td align="center" valign="middle">82.4</td><td align="center" valign="middle">74.5</td><td align="center" valign="middle">0 in this family</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/></tr><tr><td align="center" valign="middle"><bold><italic>FANCS</italic></bold></td><td align="center" valign="middle"><bold><italic>BRCA1</italic></bold></td><td align="center" valign="middle">87.6</td><td align="center" valign="middle">94.3</td><td align="center" valign="middle">86.8</td><td align="center" valign="middle">1</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0.85</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCT</italic></bold></td><td align="center" valign="middle"><bold><italic>UBE2T</italic> </bold></td><td align="center" valign="middle">53.9</td><td align="center" valign="middle">63.2</td><td align="center" valign="middle">52.6</td><td align="center" valign="middle">2</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0.91</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FANCU</italic></bold></td><td align="center" valign="middle"><bold><italic>XRCC2</italic></bold></td><td align="center" valign="middle">104.3</td><td align="center" valign="middle">116.1</td><td align="center" valign="middle">113.1</td><td align="center" valign="middle">0 in this family</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/></tr><tr><td align="center" valign="middle"><bold><italic>FANCV</italic></bold></td><td align="center" valign="middle"><bold><italic>MAD2L2</italic> </bold></td><td align="center" valign="middle">72.6</td><td align="center" valign="middle">85.4</td><td align="center" valign="middle">74.3</td><td align="center" valign="middle">0 in this family</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/></tr><tr><td align="center" valign="middle"><bold><italic>FAAP100</italic></bold></td><td align="center" valign="middle"><bold><italic>C17orf70</italic> </bold></td><td align="center" valign="middle">69.0</td><td align="center" valign="middle">66.8</td><td align="center" valign="middle">69.1</td><td align="center" valign="middle">7</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0.94</td><td align="center" valign="middle">no</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FAAP24</italic></bold></td><td align="center" valign="middle"><bold><italic>C19orf40</italic> </bold></td><td align="center" valign="middle">58.3</td><td align="center" valign="middle">64.6</td><td align="center" valign="middle">59.2</td><td align="center" valign="middle">2</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0.89</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FAAP20</italic></bold></td><td align="center" valign="middle"><bold><italic>C1orf86</italic> </bold></td><td align="center" valign="middle">57.7</td><td align="center" valign="middle">62.0</td><td align="center" valign="middle">59.7</td><td align="center" valign="middle">3</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">1.20</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">no</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FAAP16</italic></bold></td><td align="center" valign="middle"><bold><italic>APITD1</italic></bold></td><td align="center" valign="middle">37.7</td><td align="center" valign="middle">44.2</td><td align="center" valign="middle">40.7</td><td align="center" valign="middle">4</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">1.12</td><td align="center" valign="middle">no</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FAAP10</italic></bold></td><td align="center" valign="middle"><bold><italic>STRA13</italic></bold></td><td align="center" valign="middle">57.3</td><td align="center" valign="middle">62.1</td><td align="center" valign="middle">57.8</td><td align="center" valign="middle">4</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">1.18</td><td align="center" valign="middle">no</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"><bold><italic>FAN1</italic></bold></td><td align="center" valign="middle"/><td align="center" valign="middle">101.8</td><td align="center" valign="middle">112.9</td><td align="center" valign="middle">102.6</td><td align="center" valign="middle">4</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">0.97</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">no</td><td align="center" valign="middle">yes</td><td align="center" valign="middle">no</td><td align="center" valign="middle">0</td><td align="center" valign="middle">0</td></tr><tr><td align="center" valign="middle"/><td align="center" valign="middle"><bold>mean</bold></td><td align="center" valign="middle"><bold>84.0</bold></td><td align="center" valign="middle"><bold>91.7</bold></td><td align="center" valign="middle"><bold>85.9</bold></td><td align="center" valign="middle"><bold>110</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"><bold>0.99</bold></td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/></tr></tbody></table><table-wrap-foot><fn><p>(1) For each gene, the mean depth per exon was averaged over all exons of the gene. The high coverage for all genes excludes the possibility of not detecting a causative variant in other FANC genes.</p><p>(2) Total number of upstream, downstream, 5' and 3' UTRs, intronic, synonymous, splice site, missense, frameshift and stop variants in each gene.</p></fn><fn><p>(3) The presence of heterozygous variants in at least one of the patients excludes the possibility of hemizygosity for all genes (with the exception of the variant in FANCM that is homozygous)</p><p>(4) The ratio between the number of reads for each allele was averaged for all heterozygous variants in the two affected sisters. A ratio close to one for each gene indicates no bias and argues against a possible deletion for the gene.</p></fn><fn><p>(5) The presence of heterozygous variants in the various genic portions argues against the possibility of partial deletions.</p><p>(6) Among the 110 variants detected in the genes included in the FANC pathway, only the non-sense variant found in FANCM is homozygous in both patients, is rare (below 1% in ExAC database) and predicted as pathogenic.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-30637-v1.xml">elife-30637-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.30637.002</object-id><label>Table 1.</label><caption><title>Distribution of infection types and relatedness of nose-colonizing and infecting <italic>S. aureus</italic> among 105 patients revealed by genomic comparison.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top">Infection sites</th><th colspan="3" valign="top">Relation of nose-colonizing to infecting bacteria</th></tr><tr><th rowspan="2" valign="top">Unrelated <break/>(&#8805;1104 variants)</th><th colspan="2" valign="top">Closely related <break/>(&#8804;66 variants)</th></tr><tr><th valign="top">Zero shared genotypes</th><th valign="top">One shared genotype</th></tr></thead><tbody><tr><td valign="top">Bloodstream</td><td valign="top">4</td><td valign="top">43</td><td valign="top">8</td></tr><tr><td valign="top">Soft tissue</td><td valign="top">4</td><td valign="top">23</td><td valign="top">10</td></tr><tr><td valign="top">Bone and joint</td><td valign="top">2</td><td valign="top">8</td><td valign="top">3</td></tr><tr><td valign="top">Total</td><td valign="top">10</td><td valign="top">74</td><td valign="top">21</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-30743-v2.xml">elife-30743-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.30743.006</object-id><label>Table 1.</label><caption><title>Experimental data used to fit the model (training set).</title><p>Animals heterozygous for null mutations in <italic>lin-3</italic> or <italic>lin-12</italic> are described in the model by dividing the signal by two (&#8216;half dose&#8217;). The EGF level for one overexpression line (JU1107) was set according to the measured <italic>lin-3</italic> mRNA level. For other perturbations, the change in signal is included as a fitting parameter, cf. Table S2. * Notch null mutations result in the formation of two anchor cells, thus the effect of suppressing Notch signaling between VPCs it not obvious; however, several lines of evidence suggest that it yields the pattern 33133.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Experiment</th><th colspan="5" valign="top">VPC fates (fate or % 1&#176;, 2&#176;, 3&#176;)</th><th rowspan="2" valign="top">References</th></tr><tr><th valign="top">P4.p</th><th valign="top">P5.p</th><th valign="top">P6.p</th><th valign="top">P7.p</th><th valign="top">P8.p</th></tr></thead><tbody><tr><td valign="top">Wild type</td><td valign="top">3&#176;</td><td valign="top">2&#176;</td><td valign="top">1&#176;</td><td valign="top">2&#176;</td><td valign="top">3&#176;</td><td valign="top"/></tr><tr><td colspan="7" valign="top"><bold>EGF/Notch mutants</bold></td></tr><tr><td valign="top"><italic>let-23</italic> mosaic <break/>(no EGF receptors in P5/7 .p)</td><td colspan="5" valign="top">wild type</td><td valign="top">(<xref ref-type="bibr" rid="bib15">Koga and Ohshima, 1995</xref>; <xref ref-type="bibr" rid="bib20">Simske et al., 1995</xref>)</td></tr><tr><td valign="top">Half dose of EGF (<italic>lin-3</italic>/+)</td><td colspan="5" valign="top">wild type</td><td valign="top">(<xref ref-type="bibr" rid="bib6">Ferguson and Horvitz, 1985</xref>)</td></tr><tr><td valign="top">Half dose of Notch (<italic>lin-12</italic>/+)</td><td colspan="5" valign="top">wild type</td><td valign="top">(<xref ref-type="bibr" rid="bib10">Greenwald et al., 1983</xref>)</td></tr><tr><td valign="top">No Notch signaling *</td><td valign="top">3&#176;</td><td valign="top">3&#176;</td><td valign="top">1&#176;</td><td valign="top">3&#176;</td><td valign="top">3&#176;</td><td valign="top">(<xref ref-type="bibr" rid="bib26">Sundaram and Greenwald, 1993</xref>; <break/><xref ref-type="bibr" rid="bib19">Shaye and Greenwald, 2002</xref>; <break/><xref ref-type="bibr" rid="bib16">Komatsu et al., 2008</xref>)</td></tr><tr><td valign="top">Notch null mutant, 2 ACs * <break/>(described as 2&#160;&#215;&#160;WT EGF)</td><td valign="top">3&#176;</td><td valign="top">1&#176;</td><td valign="top">1&#176;</td><td valign="top">1&#176;</td><td valign="top">3&#176;</td><td valign="top">(<xref ref-type="bibr" rid="bib10">Greenwald et al., 1983</xref>)</td></tr><tr><td valign="top"><italic>lin-15</italic> mutant <break/>(ectopic EGF; fit to ~0.4&#160;&#215;&#160;WT)</td><td colspan="5" valign="top">alternating 1&#176; and 2&#176; fates <break/>(with or without AC)</td><td valign="top">(<xref ref-type="bibr" rid="bib24">Sternberg, 1988</xref>)</td></tr><tr><td colspan="7" valign="top"><bold>EGF overexpression</bold></td></tr><tr><td valign="top">JU1107 <break/>2.75&#160;&#215;&#160;WT EGF (based on measured mRNA level)</td><td valign="top">2, 16, 82</td><td valign="top">22, 78, 0</td><td valign="top">100, 0, 0</td><td valign="top">16, 84, 0</td><td valign="top">3, 13, 84</td><td valign="top">(<xref ref-type="bibr" rid="bib2">Barkoulas et al., 2013</xref>)</td></tr><tr><td valign="top">JU1100 <break/>(mRNA level not determined; level fit)</td><td valign="top">24, 54, 21</td><td valign="top">54, 46, 0</td><td valign="top">96, 4, 0</td><td valign="top">37, 63, 0</td><td valign="top">12, 39, 49</td><td valign="top">(<xref ref-type="bibr" rid="bib11">Hoyos et al., 2011</xref>)</td></tr><tr><td colspan="6" valign="top"><bold>AC ablation at successive developmental stages</bold> <break/>(mapped to evenly spaced time points in the model)</td><td valign="top">(<xref ref-type="bibr" rid="bib17">Milloz et al., 2008</xref>)</td></tr><tr><td valign="top">L2 lethargus (<italic>t</italic>&#160;=&#160;0.2)</td><td valign="top"/><td valign="top">3&#176;</td><td valign="top">3&#176;</td><td valign="top">3&#176;</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">early L3 (<italic>t</italic>&#160;=&#160;0.32)</td><td valign="top"/><td valign="top">3, 24, 74</td><td valign="top">18, 19, 64</td><td valign="top">0, 20, 80</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">DU divided (<italic>t</italic>&#160;=&#160;0.44)</td><td valign="top"/><td valign="top">0, 59, 41</td><td valign="top">31, 37, 32</td><td valign="top">0, 53, 47</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">VU divided (<italic>t</italic>&#160;=&#160;0.56)</td><td valign="top"/><td valign="top">0, 95, 5</td><td valign="top">53, 48, 0</td><td valign="top">8, 86, 6</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">3&#176; divided (<italic>t</italic>&#160;=&#160;0.68)</td><td valign="top"/><td valign="top">3, 98, 0</td><td valign="top">65, 35, 0</td><td valign="top">0, 100, 0</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">two-cell stage (<italic>t</italic>&#160;=&#160;0.8)</td><td valign="top"/><td valign="top">1, 99, 0</td><td valign="top">93, 7, 0</td><td valign="top">1, 99, 0</td><td valign="top"/><td valign="top"/></tr><tr><td colspan="6" valign="top"><bold>Epistasis between EGF/Notch</bold></td><td valign="top">(<xref ref-type="bibr" rid="bib2">Barkoulas et al., 2013</xref>)</td></tr><tr><td valign="top">CB1417 <break/>(<italic>lin-3(e1417)</italic> EGF hypomorph; level fit)</td><td valign="top">0, 0, 100</td><td valign="top">0, 10, 90</td><td valign="top">54, 0, 46</td><td valign="top">0, 10, 90</td><td valign="top">0, 0, 100</td><td valign="top"/></tr><tr><td valign="top">JU2064 <break/>(mild ectopic Notch activity; level fit)</td><td valign="top">0, 1, 99</td><td valign="top">0, 100, 0</td><td valign="top">100, 0, 0</td><td valign="top">0, 100, 0</td><td valign="top">1, 2, 97</td><td valign="top"/></tr><tr><td valign="top">JU2095 <break/>(mild ectopic Notch activity in the <italic>lin-3(e1417)</italic> background)</td><td valign="top">0, 0, 100</td><td valign="top">1, 14, 86</td><td valign="top">72, 1, 28</td><td valign="top">0, 16, 84</td><td valign="top">0, 2, 98</td><td valign="top"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-30743-v2.xml">elife-30743-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.30743.016</object-id><label>Table 4.</label><caption><title>Comparison between model predictions and experimental outcomes from (<xref ref-type="bibr" rid="bib2">Barkoulas et al., 2013</xref>).</title><p>The model prediction (mean&#160;&#177;SD within the parameter ensemble) is displayed below the experimental outcome.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Experiment</th><th colspan="5">VPC fates (% 1&#176;, 2&#176;, 3&#176;)</th></tr><tr><th>P4.p</th><th>P5.p</th><th>P6.p</th><th>P7.p</th><th>P8.p</th></tr></thead><tbody><tr><td colspan="6"><bold>Excess EGF&#160;&#215;&#160;reduced Notch (<xref ref-type="fig" rid="fig3">Figure 3</xref>)</bold> <break/>The Notch level is our fit for Notch RNAi in the JU2039 line&#160;~&#160;0.4&#160;&#215; WT. The EGF level is based on the mRNA level measured in the JU2036 EGF perturbation line (<xref ref-type="bibr" rid="bib2">Barkoulas et al., 2013</xref>).</td></tr><tr><td>JU2039 (WT EGF)</td><td>1, 0, 99</td><td>1, 91, 8</td><td>100, 0, 0</td><td>1, 87, 11</td><td>0, 0, 100</td></tr><tr><td/><td valign="top">0&#160;&#177;&#160;0, 0&#160;&#177;&#160;0, 99&#160;&#177;&#160;0</td><td valign="top">3&#160;&#177;&#160;1, 88&#160;&#177;&#160;1, 8&#160;&#177;&#160;1</td><td valign="top">99&#160;&#177;&#160;0, 0&#160;&#177;&#160;0, 1&#160;&#177;&#160;0</td><td valign="top">3&#160;&#177;&#160;1, 89&#160;&#177;&#160;1, 8&#160;&#177;&#160;1</td><td valign="top">0&#160;&#177;&#160;0, 0&#160;&#177;&#160;0, 99&#160;&#177;&#160;0</td></tr><tr><td>JU2113 (1.25&#160;&#215;&#160;WT EGF)</td><td>7, 10, 83</td><td>32, 60, 8</td><td>100, 0, 0</td><td>4, 79, 17</td><td>0, 2, 98</td></tr><tr><td/><td valign="top">0&#160;&#177;&#160;0, 0&#160;&#177;&#160;0, 99&#160;&#177;&#160;0</td><td valign="top">16&#160;&#177;&#160;8, 76&#160;&#177;&#160;9, 9&#160;&#177;&#160;2</td><td valign="top">99&#160;&#177;&#160;0, 0&#160;&#177;&#160;0, 1&#160;&#177;&#160;0</td><td valign="top">16&#160;&#177;&#160;9, 75&#160;&#177;&#160;10, 9&#160;&#177;&#160;2</td><td valign="top">0&#160;&#177;&#160;0, 0&#160;&#177;&#160;0, 99&#160;&#177;&#160;0</td></tr><tr><td colspan="6"><bold>Excess EGF&#160;&#215;&#160;ectopic Notch activity (<xref ref-type="fig" rid="fig4">Figure 4</xref>)</bold> <break/>Increasing EGF levels in a background with mild ectopic Notch activity. The level of ectopic Notch activity is our fit for the JU2064 line&#160;~&#160;WT&#160;+&#160;0.05. EGF levels are based on measured mRNA levels in the EGF perturbation lines JU2036, JU2035, and JU1107 (<xref ref-type="bibr" rid="bib2">Barkoulas et al., 2013</xref>).</td></tr><tr><td>JU2091 (1.25&#160;&#215;&#160;WT EGF)</td><td>0, 0, 100</td><td>0, 100, 0</td><td>100, 0, 0</td><td>0, 100, 0</td><td>0, 1, 99</td></tr><tr><td/><td valign="top">0&#160;&#177;&#160;0, 1&#160;&#177;&#160;1, 99&#160;&#177;&#160;1</td><td valign="top">1&#160;&#177;&#160;0, 99&#160;&#177;&#160;1, 1&#160;&#177;&#160;0</td><td valign="top">98&#160;&#177;&#160;0, 1&#160;&#177;&#160;0, 1&#160;&#177;&#160;0</td><td valign="top">1&#160;&#177;&#160;0, 99&#160;&#177;&#160;1, 1&#160;&#177;&#160;0</td><td valign="top">0&#160;&#177;&#160;0, 1&#160;&#177;&#160;1, 99&#160;&#177;&#160;1</td></tr><tr><td>JU2089 (1.79&#160;&#215;&#160;WT EGF)</td><td valign="top">0, 5, 96</td><td valign="top">2, 98, 0</td><td valign="top">100, 0, 0</td><td valign="top">2, 99, 0</td><td valign="top">1, 7, 92</td></tr><tr><td/><td valign="top">0&#160;&#177;&#160;0, 6&#160;&#177;&#160;5, 93&#160;&#177;&#160;5</td><td valign="top">2&#160;&#177;&#160;1, 97&#160;&#177;&#160;2, 1&#160;&#177;&#160;1</td><td valign="top">99&#160;&#177;&#160;0, 1&#160;&#177;&#160;0, 1&#160;&#177;&#160;0</td><td valign="top">2&#160;&#177;&#160;1, 97&#160;&#177;&#160;2, 1&#160;&#177;&#160;1</td><td valign="top">1&#160;&#177;&#160;0, 6&#160;&#177;&#160;5, 93&#160;&#177;&#160;6</td></tr><tr><td>JU2092 (2.75&#160;&#215;&#160;WT EGF)</td><td valign="top">3, 24, 74</td><td valign="top">0, 100, 0</td><td valign="top">100, 0, 0</td><td valign="top">0, 100, 0</td><td valign="top">8, 24, 68</td></tr><tr><td/><td valign="top">1&#160;&#177;&#160;1, 38&#160;&#177;&#160;14, 61&#160;&#177;&#160;14</td><td valign="top">9&#160;&#177;&#160;4, 88&#160;&#177;&#160;5, 3&#160;&#177;&#160;1</td><td valign="top">99&#160;&#177;&#160;0, 1&#160;&#177;&#160;0, 1&#160;&#177;&#160;0</td><td valign="top">8&#160;&#177;&#160;4, 88&#160;&#177;&#160;5, 3&#160;&#177;&#160;1</td><td valign="top">1&#160;&#177;&#160;1, 38&#160;&#177;&#160;14, 61&#160;&#177;&#160;14</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-30822-v2.xml">elife-30822-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.30822.008</object-id><label>Table 1.</label><caption><title>Root-mean square deviation (RMSD) values (in &#197;) between subunit pairs within 80&#945;, within SaPI1 and between 80&#945; and SaPI1.</title><p>Values for &#8216;unpruned&#8217; structures include all atoms when determining the best fit. Values for &#8216;pruned&#8217; structures only include the best matching residue pairs. Pruning was done in UCSF Chimera using standard parameters. For 80&#945;, pairs of subunits related across the quasi-twofold axis are highlighted in green. Comparison of subunits for which the E-loops are in the &#8216;UP&#8217; conformation are highlighted in blue.</p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="6" valign="bottom">Root mean square deviation between CP subunits</th></tr><tr><th colspan="2" rowspan="2" valign="bottom"/><th colspan="2" valign="bottom">Pruned</th><th colspan="2" valign="bottom">Unpruned</th></tr><tr><th># res.</th><th>RMSD (&#197;)</th><th># res.</th><th>RMSD (&#197;)</th></tr></thead><tbody><tr><td colspan="6" valign="bottom"><bold>80&#945; procapsid</bold></td></tr><tr><td valign="bottom">&#8195;subA</td><td valign="bottom">subB</td><td valign="bottom">215</td><td valign="bottom">0.986</td><td valign="bottom">284</td><td valign="bottom">2.927</td></tr><tr><td valign="bottom">&#8195;subA</td><td valign="bottom">subC</td><td valign="bottom">250</td><td valign="bottom">0.897</td><td valign="bottom">284</td><td valign="bottom">1.521</td></tr><tr><td valign="bottom">&#8195;subA</td><td valign="bottom">subD</td><td valign="bottom">239</td><td valign="bottom">0.943</td><td valign="bottom">284</td><td valign="bottom">2.902</td></tr><tr><td valign="bottom">&#8195;subA</td><td valign="bottom">subE</td><td valign="bottom">218</td><td valign="bottom">0.989</td><td valign="bottom">284</td><td valign="bottom">2.900</td></tr><tr><td valign="bottom">&#8195;subA</td><td valign="bottom">subF</td><td valign="bottom">252</td><td valign="bottom">0.899</td><td valign="bottom">284</td><td valign="bottom">1.533</td></tr><tr><td valign="bottom">&#8195;subA</td><td valign="bottom">subG</td><td valign="bottom">239</td><td valign="bottom">1.024</td><td valign="bottom">284</td><td valign="bottom">2.878</td></tr><tr><td valign="bottom">&#8195;subB</td><td valign="bottom">subC</td><td valign="bottom">224</td><td valign="bottom">0.881</td><td valign="bottom">284</td><td valign="bottom">3.072</td></tr><tr><td valign="bottom">&#8195;subB</td><td valign="bottom">subD</td><td valign="bottom">229</td><td valign="bottom">0.955</td><td valign="bottom">284</td><td valign="bottom">1.901</td></tr><tr><td valign="bottom">&#8195;subB</td><td valign="bottom">subE</td><td valign="bottom">251</td><td valign="bottom">0.727</td><td valign="bottom">284</td><td valign="bottom">1.506</td></tr><tr><td valign="bottom">&#8195;subB</td><td valign="bottom">subF</td><td valign="bottom">219</td><td valign="bottom">0.907</td><td valign="bottom">284</td><td valign="bottom">3.244</td></tr><tr><td valign="bottom">&#8195;subB</td><td valign="bottom">subG</td><td valign="bottom">232</td><td valign="bottom">0.909</td><td valign="bottom">284</td><td valign="bottom">1.686</td></tr><tr><td valign="bottom">&#8195;subC</td><td valign="bottom">subD</td><td valign="bottom">241</td><td valign="bottom">0.853</td><td valign="bottom">284</td><td valign="bottom">3.002</td></tr><tr><td valign="bottom">&#8195;subC</td><td valign="bottom">subE</td><td valign="bottom">225</td><td valign="bottom">0.923</td><td valign="bottom">284</td><td valign="bottom">3.108</td></tr><tr><td valign="bottom">&#8195;subC</td><td valign="bottom">subF</td><td valign="bottom">271</td><td valign="bottom">0.764</td><td valign="bottom">284</td><td valign="bottom">0.987</td></tr><tr><td valign="bottom">&#8195;subC</td><td valign="bottom">subG</td><td valign="bottom">237</td><td valign="bottom">0.860</td><td valign="bottom">284</td><td valign="bottom">2.974</td></tr><tr><td valign="bottom">&#8195;subD</td><td valign="bottom">subE</td><td valign="bottom">231</td><td valign="bottom">1.001</td><td valign="bottom">284</td><td valign="bottom">1.715</td></tr><tr><td valign="bottom">&#8195;subD</td><td valign="bottom">subF</td><td valign="bottom">246</td><td valign="bottom">0.781</td><td valign="bottom">284</td><td valign="bottom">3.195</td></tr><tr><td valign="bottom">&#8195;subD</td><td valign="bottom">subG</td><td valign="bottom">262</td><td valign="bottom">0.704</td><td valign="bottom">284</td><td valign="bottom">1.302</td></tr><tr><td valign="bottom">&#8195;subE</td><td valign="bottom">subF</td><td valign="bottom">220</td><td valign="bottom">0.979</td><td valign="bottom">284</td><td valign="bottom">3.257</td></tr><tr><td valign="bottom">&#8195;subE</td><td valign="bottom">subG</td><td valign="bottom">236</td><td valign="bottom">0.981</td><td valign="bottom">284</td><td valign="bottom">1.654</td></tr><tr><td valign="bottom">&#8195;subF</td><td valign="bottom">subG</td><td valign="bottom">242</td><td valign="bottom">0.847</td><td valign="bottom">284</td><td valign="bottom">3.104</td></tr><tr><td colspan="6" valign="bottom"><bold>SaPI1 procapsid</bold></td></tr><tr><td valign="bottom">&#8195;subA</td><td valign="bottom">subB</td><td valign="bottom">218</td><td valign="bottom">1.047</td><td valign="bottom">284</td><td valign="bottom">2.656</td></tr><tr><td valign="bottom">&#8195;subA</td><td valign="bottom">subC</td><td valign="bottom">247</td><td valign="bottom">1.092</td><td valign="bottom">284</td><td valign="bottom">1.572</td></tr><tr><td valign="bottom">&#8195;subA</td><td valign="bottom">subD</td><td valign="bottom">223</td><td valign="bottom">1.108</td><td valign="bottom">284</td><td valign="bottom">2.862</td></tr><tr><td valign="bottom">&#8195;subB</td><td valign="bottom">subC</td><td valign="bottom">220</td><td valign="bottom">1.040</td><td valign="bottom">284</td><td valign="bottom">2.865</td></tr><tr><td valign="bottom">&#8195;subB</td><td valign="bottom">subD</td><td valign="bottom">231</td><td valign="bottom">1.040</td><td valign="bottom">284</td><td valign="bottom">1.633</td></tr><tr><td valign="bottom">&#8195;subC</td><td valign="bottom">subD</td><td valign="bottom">242</td><td valign="bottom">0.921</td><td valign="bottom">284</td><td valign="bottom">2.669</td></tr><tr><td colspan="6"><bold>SaPI1 vs. 80&#945;</bold></td></tr><tr><td valign="bottom"><bold>SaPI1</bold></td><td valign="bottom"><bold>80&#945;</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">&#8195;subA</td><td valign="bottom">subA</td><td valign="bottom">259</td><td valign="bottom">0.920</td><td valign="bottom">284</td><td valign="bottom">1.495</td></tr><tr><td valign="bottom">&#8195;subB</td><td valign="bottom">subB</td><td valign="bottom">250</td><td valign="bottom">0.926</td><td valign="bottom">284</td><td valign="bottom">1.443</td></tr><tr><td valign="bottom">&#8195;subB</td><td valign="bottom">subE</td><td valign="bottom">257</td><td valign="bottom">0.823</td><td valign="bottom">284</td><td valign="bottom">1.380</td></tr><tr><td valign="bottom">&#8195;subC</td><td valign="bottom">subC</td><td valign="bottom">268</td><td valign="bottom">0.935</td><td valign="bottom">284</td><td valign="bottom">1.165</td></tr><tr><td valign="bottom">&#8195;subC</td><td valign="bottom">subF</td><td valign="bottom">268</td><td valign="bottom">0.997</td><td valign="bottom">284</td><td valign="bottom">1.190</td></tr><tr><td valign="bottom">&#8195;subD</td><td valign="bottom">subD</td><td valign="bottom">247</td><td valign="bottom">0.948</td><td valign="bottom">284</td><td valign="bottom">1.536</td></tr><tr><td valign="bottom">&#8195;subD</td><td valign="bottom">subG</td><td valign="bottom">262</td><td valign="bottom">0.939</td><td valign="bottom">284</td><td valign="bottom">1.170</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31023-v1.xml">elife-31023-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.31023.006</object-id><label>Table 1.</label><caption><title>Examples of proteins identified by enChIP-MS.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Gene name</th><th colspan="4">Sample</th><th rowspan="3">Category</th></tr><tr><th colspan="2">gD4Z4</th><th colspan="2">gMYOD1</th></tr><tr><th># peptides*</th><th>% coverage<sup>&#8224;</sup></th><th># peptides*</th><th>% coverage<sup>&#8224;</sup></th></tr></thead><tbody><tr><td>CBX3/HP1&#947;</td><td>4.7</td><td>15.7</td><td>0.0</td><td>0.0</td><td rowspan="7">Known D4Z4-associated proteins</td></tr><tr><td>NCL</td><td>47.4</td><td>22.3</td><td>34.0</td><td>14.2</td></tr><tr><td>PDS5B</td><td>2.0</td><td>11.2</td><td>0.0</td><td>0.0</td></tr><tr><td>RAD21</td><td>1.8</td><td>2.9</td><td>0.0</td><td>0.0</td></tr><tr><td>SMC1A</td><td>7.0</td><td>5.8</td><td>2.0</td><td>1.8</td></tr><tr><td>SMC3</td><td>17.0</td><td>6.7</td><td>1.0</td><td>1.0</td></tr><tr><td>SMCHD1</td><td>1.6</td><td>2.4</td><td>0.0</td><td>0.0</td></tr><tr><td>CHD4</td><td>8.3</td><td>3.1</td><td>0.0</td><td>0.0</td><td rowspan="4">NuRD complex components</td></tr><tr><td>HDAC2</td><td>2.5</td><td>5.6</td><td>3.0</td><td>6.8</td></tr><tr><td>MTA2</td><td>1.2</td><td>2.2</td><td>1.0</td><td>1.5</td></tr><tr><td>RBBP4</td><td>4.5</td><td>7.5</td><td>4.0</td><td>6.9</td></tr></tbody></table><table-wrap-foot><fn><p>*Average number of peptides recovered from each sample type, combining like (gD4Z4 or gMYOD1) immunoprecipitations.</p><p><sup>&#8224;</sup>Average percentage of each protein covered by the identified peptides from each sample type, combining like (gD4Z4 or gMYOD1) immunoprecipitations.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31126-v1.xml">elife-31126-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.31126.005</object-id><label>Table 1.</label><caption><title>Summary of demographics for donors with and without dementia.</title><p>P-values for upper seven and lower two metrics are uncorrected significance values from T-tests and hypergeometric tests, respectively (*p&lt;0.05 after Bonferroni correction for multiple comparisons). Demographic summary includes 106 donors used in analysis (Materials and methods).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Category</th><th colspan="2" valign="bottom">Non-demented</th><th colspan="2" valign="bottom">Demented</th><th valign="bottom"/></tr><tr><th valign="bottom"><italic>Mean</italic></th><th valign="bottom"><italic>SD</italic></th><th valign="bottom"><italic>Mean</italic></th><th valign="bottom"><italic>SD</italic></th><th valign="bottom"><italic>P-value</italic></th></tr></thead><tbody><tr><td valign="bottom">Age at death (yrs)</td><td valign="bottom">89</td><td valign="bottom">7</td><td valign="bottom">90</td><td valign="bottom">6</td><td valign="bottom">2.8E-01</td></tr><tr><td valign="bottom">Education (yrs)</td><td valign="bottom">15</td><td valign="bottom">3</td><td valign="bottom">14</td><td valign="bottom">3</td><td valign="bottom">4.9E-02</td></tr><tr><td valign="bottom">Number of TBIs</td><td valign="bottom">0.6</td><td valign="bottom">0.7</td><td valign="bottom">0.7</td><td valign="bottom">0.7</td><td valign="bottom">8.0E-01</td></tr><tr><td valign="bottom">Age at first TBI</td><td valign="bottom">23</td><td valign="bottom">31</td><td valign="bottom">24</td><td valign="bottom">32</td><td valign="bottom">8.2E-01</td></tr><tr><td valign="bottom">Braak stage</td><td valign="bottom">2.8</td><td valign="bottom">1.5</td><td valign="bottom">4.1</td><td valign="bottom">1.7</td><td valign="bottom">8.9E-05*</td></tr><tr><td valign="bottom">NIA Reagan</td><td valign="bottom">1.4</td><td valign="bottom">0.7</td><td valign="bottom">1.9</td><td valign="bottom">0.9</td><td valign="bottom">1.8E-03*</td></tr><tr><td valign="bottom">CERAD score</td><td valign="bottom">1.2</td><td valign="bottom">0.9</td><td valign="bottom">1.8</td><td valign="bottom">1.2</td><td valign="bottom">1.4E-02</td></tr><tr><td valign="bottom"/><td colspan="2" valign="bottom"><italic>Count</italic></td><td colspan="2" valign="bottom"><italic>Count</italic></td><td valign="bottom"><italic>P-value</italic></td></tr><tr><td valign="bottom">Sex</td><td colspan="2" valign="bottom">20 F / 36 M</td><td colspan="2" valign="bottom">23 F / 27 M</td><td valign="bottom">1.0E-01</td></tr><tr><td valign="bottom">&gt;0 APOE &#603;4 alleles</td><td colspan="2" valign="bottom">47 No/7 Yes</td><td colspan="2" valign="bottom">32 No/13 Yes</td><td valign="bottom">2.1E-02</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31225-v1.xml">elife-31225-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.31225.004</object-id><label>Table 1.</label><caption><title>Parameters of analyses of variance for the effects of group (early-born,&#160;late-born)&#160;and diet (fed ad libitum or intermittently fasted) on the rate (&#8216;slope&#8217;) of body mass gain and structural growth (assessed by body length), the maximal level (&#8216;plateau&#8217;) at which BM and BL levelled off and the time (&#8216;time of plateau&#8217;) at which the plateau of BM or BL occurred during the pre-hibernation period.</title><p>p-values shown in bold correspond to statistically significant and interpretable values.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top">Parameter</th><th rowspan="3" valign="top">Term</th><th colspan="6" valign="top">Response variables</th></tr><tr><th colspan="2" valign="top">Slope</th><th colspan="2" valign="top">Plateau</th><th colspan="2" valign="top">Time of plateau</th></tr><tr><th valign="top">F-value</th><th valign="top">p-value</th><th valign="top">F-value</th><th valign="top">p-value</th><th valign="top">F-value</th><th valign="top">p-value</th></tr></thead><tbody><tr><td valign="top">Body mass</td><td valign="top">Group <break/>Diet <break/>Group &#1093; Diet</td><td valign="top">70.93 <break/>1.00 <break/>3.58</td><td valign="top"><bold>&lt;0.001</bold> <break/>0.33 <break/>0.07</td><td valign="top">5.73 <break/>0.05 <break/>0.13</td><td valign="top"><bold>&lt;0.05</bold> <break/>0.83 <break/>0.72</td><td valign="top">368.71 <break/>0.07 <break/>0.44</td><td valign="top"><bold>&lt;0.001</bold> <break/>0.80 <break/>0.52</td></tr><tr><td valign="top">Body length</td><td valign="top">Group <break/>Diet <break/>Group &#1093; Diet</td><td valign="top">103.55 <break/>0.01 <break/>1.65</td><td valign="top"><bold>&lt;0.001</bold> <break/>0.92 <break/>0.21</td><td valign="top">1.46 <break/>0.01 <break/>0.12</td><td valign="top">0.24 <break/>0.97 <break/>0.73</td><td valign="top">190.11 <break/>0.43 <break/>0.29</td><td valign="top"><bold>&lt;0.001</bold> <break/>0.52 <break/>0.60</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31225-v1.xml">elife-31225-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.31225.009</object-id><label>Table 4.</label><caption><title>Parameters of analyses of variance for the effects of group and diet on the time (&#8216;starting time&#8217;) and the rate (&#8216;slope&#8217;) of torpor increase or decrease of activity time.</title><p>Mean torpor duration was also added as a predictor variable in the model for activity time and mean ambient temperature was included as a random effect in all models. p-values shown in bold correspond to statistically significant and interpretable values.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top">Response variable</th><th rowspan="3" valign="top">Term</th><th colspan="4" valign="top">Parameters</th></tr><tr><th colspan="2" valign="top">Starting time</th><th colspan="2" valign="top">Slope</th></tr><tr><th valign="top">&#967;<sup>2</sup></th><th valign="top">p-value</th><th valign="top">&#967; <sup>2</sup></th><th valign="top">p-value</th></tr></thead><tbody><tr><td valign="top">Torpor frequency</td><td valign="top">Group <break/>Diet <break/>Group &#1093; Diet</td><td valign="top">46.47 <break/>8.64 <break/>1.82</td><td valign="top"><bold>&lt;0.001</bold> <break/><bold>&lt;0.01</bold> <break/>0.18</td><td valign="top">3.16 <break/>1.18 <break/>7.02</td><td valign="top">0.08 <break/>0.28 <break/><bold>&lt;0.01</bold></td></tr><tr><td valign="top">Mean torpor duration</td><td valign="top">Group <break/>Diet <break/>Group &#1093; Diet</td><td valign="top">43.49 <break/>8.84 <break/>0.50</td><td valign="top"><bold>&lt;0.001</bold> <break/><bold>&lt;0.01</bold> <break/>0.48</td><td valign="top">38.26 <break/>0.35 <break/>16.64</td><td valign="top"><bold>&lt;0.001</bold> <break/>0.55 <break/><bold>&lt;0.001</bold></td></tr><tr><td valign="top">Total torpor duration</td><td valign="top">Group <break/>Diet <break/>Group &#1093; Diet</td><td valign="top">43.49 <break/>8.84 <break/>0.50</td><td valign="top"><bold>&lt;0.001</bold> <break/><bold>&lt;0.01</bold> <break/>0.48</td><td valign="top">77.86 <break/>0.55 <break/>9.66</td><td valign="top">&lt;0.001 <break/>0.46 <break/><bold>&lt;0.01</bold></td></tr><tr><td valign="top">Activity time</td><td valign="top">Group <break/>Diet <break/>Group &#1093; Diet <break/>Mean torpor duration</td><td valign="top">0.09 <break/>14.06 <break/>6.96 <break/>7.61</td><td valign="top">0.76 <break/>&lt;0.001 <break/><bold>&lt;0.01</bold> <break/><bold>&lt;0.01</bold></td><td valign="top">6.45 <break/>2.64 <break/>0.22 <break/>2.68</td><td valign="top">&lt;0.05 <break/>0.10 <break/>0.64 <break/>0.10</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31469-v2.xml">elife-31469-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.31469.016</object-id><label>Table 2.</label><caption><title>Disassembly rates <italic>k<sub>0</sub></italic> (min<sup>&#8722;1</sup>) and dispersion of F-actin length distributions.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="4" valign="top">UtrCH-stabilized</th><th colspan="2" valign="top">Lat A-treated*</th><th colspan="2" valign="top">SMIFH2-treated</th></tr><tr><th>Control</th><th>Low</th><th>Medium</th><th>High</th><th>Control</th><th>Lat A</th><th valign="top">Control</th><th valign="top">SMIFH2</th></tr></thead><tbody><tr><td valign="top"><italic>k<sub>0</sub></italic></td><td>0.162</td><td>0.126</td><td>0.073</td><td>0.048</td><td valign="top">0.0819</td><td valign="top">0.5106</td><td valign="top">0.1995</td><td valign="top">0.1650</td></tr><tr><td valign="top"><p><inline-formula><mml:math id="inf50"><mml:mi>&#955;</mml:mi><mml:mi>&#8217;</mml:mi></mml:math></inline-formula></p></td><td valign="top">0.677</td><td valign="top">0.889</td><td valign="top">6.946</td><td valign="top">709.4</td><td valign="top">0.0671</td><td valign="top">14.66</td><td valign="top">1.555</td><td valign="top">1.540</td></tr></tbody></table><table-wrap-foot><fn><p>*The solvent, DMSO had an effect on the viscolelastic parameters even in controls (<xref ref-type="supplementary-material" rid="supp3">Supplementary file 3</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31486-v2.xml">elife-31486-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app1table7" position="float"><object-id pub-id-type="doi">10.7554/eLife.31486.040</object-id><label>Appendix 1&#8212;table 7.</label><caption><title>Sequence similarity comparison.</title><p>Frequencies of dipeptides (pairs of neighboring amino residues) were computed for phase-separating proteins and the human proteome, and enrichment was measured by the percentage of human proteins with lower frequency than found in a given sequence. The fifteen dipeptides enriched (&#8805;99%) in the most sequences within the phase separation test sets are shown in the table vs. enrichment values obtained for the phase separation training set and three experimentally verified proteins. Values in the top fifth percentile are shown in bold.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Protein <break/>Name</th><th colspan="15" valign="top">Dipeptide enrichment (Percentage of human proteome with lower frequency)</th></tr><tr><th valign="top">GV</th><th valign="top">VG</th><th valign="top">VP</th><th valign="top">PG</th><th valign="top">FG</th><th valign="top">RG</th><th valign="top">GR</th><th valign="top">GG</th><th valign="top">YG</th><th valign="top">GS</th><th valign="top">SG</th><th valign="top">GA</th><th valign="top">GF</th><th valign="top">GD</th><th valign="top">DS</th></tr></thead><tbody><tr><td colspan="16" valign="top">Training Set Proteins</td></tr><tr><td valign="top">Elastin</td><td valign="top"><bold>100</bold></td><td valign="top"><bold>100</bold></td><td valign="top"><bold>100</bold></td><td valign="top"><bold>100</bold></td><td valign="top"><bold>97</bold></td><td valign="top">31</td><td valign="top">32</td><td valign="top"><bold>99</bold></td><td valign="top"><bold>99</bold></td><td valign="top">20</td><td valign="top">20</td><td valign="top"><bold>100</bold></td><td valign="top">89</td><td valign="top">38</td><td valign="top">30</td></tr><tr><td valign="top">Nsp1</td><td valign="top">30</td><td valign="top">34</td><td valign="top">31</td><td valign="top">26</td><td valign="top"><bold>100</bold></td><td valign="top">31</td><td valign="top">30</td><td valign="top">75</td><td valign="top">52</td><td valign="top">90</td><td valign="top">38</td><td valign="top"><bold>99</bold></td><td valign="top">60</td><td valign="top">66</td><td valign="top">68</td></tr><tr><td valign="top">TIA1</td><td valign="top">73</td><td valign="top">75</td><td valign="top">46</td><td valign="top">26</td><td valign="top">86</td><td valign="top">31</td><td valign="top">86</td><td valign="top">77</td><td valign="top"><bold>99</bold></td><td valign="top">29</td><td valign="top">53</td><td valign="top">26</td><td valign="top">84</td><td valign="top">54</td><td valign="top">30</td></tr><tr><td valign="top">LAF1</td><td valign="top">30</td><td valign="top">78</td><td valign="top">65</td><td valign="top">29</td><td valign="top">67</td><td valign="top"><bold>99</bold></td><td valign="top"><bold>99</bold></td><td valign="top"><bold>100</bold></td><td valign="top">77</td><td valign="top">88</td><td valign="top"><bold>97</bold></td><td valign="top">65</td><td valign="top">78</td><td valign="top"><bold>97</bold></td><td valign="top">32</td></tr><tr><td valign="top">EIF4H</td><td valign="top">30</td><td valign="top">65</td><td valign="top">31</td><td valign="top">52</td><td valign="top"><bold>98</bold></td><td valign="top"><bold>99</bold></td><td valign="top"><bold>95</bold></td><td valign="top"><bold>99</bold></td><td valign="top">52</td><td valign="top"><bold>99</bold></td><td valign="top">42</td><td valign="top">79</td><td valign="top"><bold>99</bold></td><td valign="top"><bold>98</bold></td><td valign="top">89</td></tr><tr><td valign="top">Ddx3x</td><td valign="top">51</td><td valign="top">70</td><td valign="top">34</td><td valign="top">43</td><td valign="top">89</td><td valign="top"><bold>98</bold></td><td valign="top"><bold>97</bold></td><td valign="top"><bold>96</bold></td><td valign="top">93</td><td valign="top">93</td><td valign="top"><bold>95</bold></td><td valign="top">68</td><td valign="top"><bold>96</bold></td><td valign="top">59</td><td valign="top">78</td></tr><tr><td valign="top">hnRNPA1</td><td valign="top">30</td><td valign="top">55</td><td valign="top">31</td><td valign="top">44</td><td valign="top"><bold>100</bold></td><td valign="top"><bold>99</bold></td><td valign="top"><bold>99</bold></td><td valign="top"><bold>100</bold></td><td valign="top"><bold>99</bold></td><td valign="top"><bold>99</bold></td><td valign="top"><bold>98</bold></td><td valign="top">44</td><td valign="top"><bold>99</bold></td><td valign="top">60</td><td valign="top">79</td></tr><tr><td valign="top">DDX4</td><td valign="top">33</td><td valign="top">77</td><td valign="top">48</td><td valign="top">53</td><td valign="top"><bold>98</bold></td><td valign="top"><bold>96</bold></td><td valign="top">91</td><td valign="top">89</td><td valign="top">59</td><td valign="top">87</td><td valign="top"><bold>96</bold></td><td valign="top">29</td><td valign="top"><bold>98</bold></td><td valign="top"><bold>96</bold></td><td valign="top">45</td></tr><tr><td valign="top">FUS</td><td valign="top">30</td><td valign="top">31</td><td valign="top">31</td><td valign="top">83</td><td valign="top">78</td><td valign="top"><bold>100</bold></td><td valign="top"><bold>99</bold></td><td valign="top"><bold>100</bold></td><td valign="top"><bold>100</bold></td><td valign="top"><bold>98</bold></td><td valign="top"><bold>99</bold></td><td valign="top">33</td><td valign="top">93</td><td valign="top">91</td><td valign="top">57</td></tr><tr><td valign="top">EWS</td><td valign="top">52</td><td valign="top">31</td><td valign="top">35</td><td valign="top">97</td><td valign="top">51</td><td valign="top"><bold>100</bold></td><td valign="top"><bold>99</bold></td><td valign="top"><bold>100</bold></td><td valign="top"><bold>100</bold></td><td valign="top">72</td><td valign="top">61</td><td valign="top">30</td><td valign="top"><bold>97</bold></td><td valign="top">91</td><td valign="top">48</td></tr><tr><td valign="top">TAF15</td><td valign="top">36</td><td valign="top">38</td><td valign="top">31</td><td valign="top">30</td><td valign="top">53</td><td valign="top"><bold>100</bold></td><td valign="top"><bold>99</bold></td><td valign="top"><bold>100</bold></td><td valign="top"><bold>100</bold></td><td valign="top">92</td><td valign="top"><bold>99</bold></td><td valign="top">26</td><td valign="top">71</td><td valign="top"><bold>100</bold></td><td valign="top">94</td></tr><tr><td colspan="16" valign="top">Experimentally Verified Proteins</td></tr><tr><td valign="top">FMR1</td><td valign="top">69</td><td valign="top">89</td><td valign="top">93</td><td valign="top">44</td><td valign="top">43</td><td valign="top"><bold>96</bold></td><td valign="top">94</td><td valign="top">83</td><td valign="top">62</td><td valign="top">62</td><td valign="top">34</td><td valign="top">70</td><td valign="top">48</td><td valign="top">41</td><td valign="top">67</td></tr><tr><td valign="top">SCAF pAP</td><td valign="top">75</td><td valign="top">50</td><td valign="top">89</td><td valign="top">91</td><td valign="top">43</td><td valign="top">49</td><td valign="top">73</td><td valign="top">97</td><td valign="top">75</td><td valign="top">92</td><td valign="top"><bold>96</bold></td><td valign="top"><bold>96</bold></td><td valign="top">40</td><td valign="top">36</td><td valign="top">44</td></tr><tr><td valign="top">Engrailed-2</td><td valign="top">30</td><td valign="top">31</td><td valign="top">31</td><td valign="top"><bold>97</bold></td><td valign="top">70</td><td valign="top">78</td><td valign="top">90</td><td valign="top"><bold>100</bold></td><td valign="top">52</td><td valign="top"><bold>99</bold></td><td valign="top">91</td><td valign="top"><bold>99</bold></td><td valign="top">40</td><td valign="top"><bold>95</bold></td><td valign="top"><bold>97</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31511-v2.xml">elife-31511-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.31511.016</object-id><label>Table 1.</label><caption><title><italic>SLC7A8</italic> Humans mutations found in ARHL and controls individuals.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Phenotype</th><th rowspan="2">Age</th><th rowspan="2">Sex</th><th rowspan="2">Chr. 14</th><th rowspan="2">Variant</th><th rowspan="2">Consequence</th><th rowspan="2">Code</th><th colspan="5">Frequency</th></tr><tr><th>Esp6500siv2</th><th>1000 g</th><th>Campion</th><th>ExAC</th><th>Studied cohort</th></tr></thead><tbody><tr><td>ARHL</td><td>75</td><td>Female</td><td>23597290</td><td>14:23597291 C / T</td><td>p.Val460Glu</td><td>V460E</td><td>NA</td><td>NA</td><td>0.0013</td><td>0.00002475</td><td>0.015</td></tr><tr><td>ARHL</td><td>57</td><td>Male</td><td>23598917</td><td>14:23598917 G / A</td><td>p.Thr402Met</td><td>T402M</td><td>NA</td><td>NA</td><td>0.0047</td><td>0.00002471</td><td>0.015</td></tr><tr><td>ARHL</td><td>75</td><td>Male</td><td>23608641</td><td>14:23608641 C / T <break/>(rs142951280)</td><td>p.Val302Ile</td><td>V302I</td><td>0.0005</td><td>NA</td><td>0.0047</td><td>0.0004613</td><td>0.015</td></tr><tr><td>ARHL</td><td>86</td><td>Female</td><td>23598870</td><td>14:23598869 C / T</td><td>p.Arg418His</td><td>R418C</td><td>0.0005</td><td>NA</td><td>0.002</td><td>0.00002477</td><td>0.015</td></tr><tr><td>control</td><td>50</td><td>Male</td><td>23652101</td><td>14:23652101 C / G <break/>(rs141772308)</td><td>p.Arg8Pro</td><td>R8P</td><td>0.0008</td><td>NA</td><td>0.0013</td><td>0.0008156</td><td>0.012</td></tr><tr><td>control</td><td>50</td><td>Male</td><td>23635621</td><td>14:23635621 C / T <break/>(rs139927895)</td><td>p.Ala94Thr</td><td>A94T</td><td>0.0012</td><td>0.002</td><td>0.0013</td><td>0.00202</td><td>0.012</td></tr><tr><td>control</td><td>90</td><td>Female</td><td>23612368</td><td>14:23612368 C / T <break/>(rs149245114)</td><td>p.Arg185Gln</td><td>R185L</td><td>NA</td><td>NA</td><td>0.002</td><td>0.00002471</td><td>0.012</td></tr></tbody></table><table-wrap-foot><fn><p>ARHL (age-related hearing loss). The age (years) of the subject when the Audiogram was performed is indicated. Variant [CHR: position reference/alternate (dbSNP135rsID)]. Consequence [HGUS annotation (protein change)]. Code [short description of the alternate variant]. Frequency of the mutations: Esp6500siv2 (NHLBI Exome Sequencing Project), 1000 g (1000 Genomes Project), Campion (The Allele Frequency Net Database) and ExAC (The Exome Aggregation Consortium).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31511-v3.xml">elife-31511-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.31511.016</object-id><label>Table 1.</label><caption><title><italic>SLC7A8</italic> Humans mutations found in ARHL and controls individuals.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Phenotype</th><th rowspan="2">Age</th><th rowspan="2">Sex</th><th rowspan="2">Chr. 14</th><th rowspan="2">Variant</th><th rowspan="2">Consequence</th><th rowspan="2">Code</th><th colspan="5">Frequency</th></tr><tr><th>Esp6500siv2</th><th>1000 g</th><th>Campion</th><th>ExAC</th><th>Studied cohort</th></tr></thead><tbody><tr><td>ARHL</td><td>75</td><td>Female</td><td>23597290</td><td>14:23597290 A / T</td><td>p.Val460Glu</td><td>V460E</td><td>NA</td><td>NA</td><td>0.0013</td><td>0.00002475</td><td>0.015</td></tr><tr><td>ARHL</td><td>57</td><td>Male</td><td>23598917</td><td>14:23598917 G / A</td><td>p.Thr402Met</td><td>T402M</td><td>NA</td><td>NA</td><td>0.0047</td><td>0.00002471</td><td>0.015</td></tr><tr><td>ARHL</td><td>75</td><td>Male</td><td>23608641</td><td>14:23608641 C / T <break/>(rs142951280)</td><td>p.Val302Ile</td><td>V302I</td><td>0.0005</td><td>NA</td><td>0.0047</td><td>0.0004613</td><td>0.015</td></tr><tr><td>ARHL</td><td>86</td><td>Female</td><td>23598870</td><td>14:23598870 G / A <break/>(rs146946494) <break/></td><td>p.Arg418His</td><td>R418C</td><td>0.0005</td><td>NA</td><td>0.002</td><td>0.00002477</td><td>0.015</td></tr><tr><td>control</td><td>50</td><td>Male</td><td>23652101</td><td>14:23652101 C / G <break/>(rs141772308)</td><td>p.Arg8Pro</td><td>R8P</td><td>0.0008</td><td>NA</td><td>0.0013</td><td>0.0008156</td><td>0.012</td></tr><tr><td>control</td><td>50</td><td>Male</td><td>23635621</td><td>14:23635621 C / T <break/>(rs139927895)</td><td>p.Ala94Thr</td><td>A94T</td><td>0.0012</td><td>0.002</td><td>0.0013</td><td>0.00202</td><td>0.012</td></tr><tr><td>control</td><td>90</td><td>Female</td><td>23612368</td><td>14:23612368 C / A <break/>(rs149245114)</td><td>p.Arg185Gln</td><td>R185L</td><td>NA</td><td>NA</td><td>0.002</td><td>0.00002471</td><td>0.012</td></tr></tbody></table><table-wrap-foot><fn><p>ARHL (age-related hearing loss). The age (years) of the subject when the Audiogram was performed is indicated. Variant [CHR: position reference/alternate (dbSNP135rsID)]. Consequence [HGUS annotation (protein change)]. Code [short description of the alternate variant]. Frequency of the mutations: Esp6500siv2 (NHLBI Exome Sequencing Project), 1000 g (1000 Genomes Project), Campion (The Allele Frequency Net Database) and ExAC (The Exome Aggregation Consortium).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31511-v4.xml">elife-31511-v4.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.31511.016</object-id><label>Table 1.</label><caption><title><italic>SLC7A8</italic> Humans mutations found in ARHL and controls individuals.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Phenotype</th><th rowspan="2">Age</th><th rowspan="2">Sex</th><th rowspan="2">Chr. 14</th><th rowspan="2">Variant</th><th rowspan="2">Consequence</th><th rowspan="2">Code</th><th colspan="5">Frequency</th></tr><tr><th>Esp6500siv2</th><th>1000 g</th><th>Campion</th><th>ExAC</th><th>Studied cohort</th></tr></thead><tbody><tr><td>ARHL</td><td>75</td><td>Female</td><td>23597290</td><td>14:23597290 A / T</td><td>p.Val460Glu</td><td>V460E</td><td>NA</td><td>NA</td><td>0.0013</td><td>0.00002475</td><td>0.015</td></tr><tr><td>ARHL</td><td>57</td><td>Male</td><td>23598917</td><td>14:23598917 G / A</td><td>p.Thr402Met</td><td>T402M</td><td>NA</td><td>NA</td><td>0.0047</td><td>0.00002471</td><td>0.015</td></tr><tr><td>ARHL</td><td>75</td><td>Male</td><td>23608641</td><td>14:23608641 C / T <break/>(rs142951280)</td><td>p.Val302Ile</td><td>V302I</td><td>0.0005</td><td>NA</td><td>0.0047</td><td>0.0004613</td><td>0.015</td></tr><tr><td>ARHL</td><td>86</td><td>Female</td><td>23598870</td><td>14:23598870 G / A <break/>(rs146946494) <break/></td><td>p.Arg418Cys</td><td>R418C</td><td>0.0005</td><td>NA</td><td>0.002</td><td>0.00002477</td><td>0.015</td></tr><tr><td>control</td><td>50</td><td>Male</td><td>23652101</td><td>14:23652101 C / G <break/>(rs141772308)</td><td>p.Arg8Pro</td><td>R8P</td><td>0.0008</td><td>NA</td><td>0.0013</td><td>0.0008156</td><td>0.012</td></tr><tr><td>control</td><td>50</td><td>Male</td><td>23635621</td><td>14:23635621 C / T <break/>(rs139927895)</td><td>p.Ala94Thr</td><td>A94T</td><td>0.0012</td><td>0.002</td><td>0.0013</td><td>0.00202</td><td>0.012</td></tr><tr><td>control</td><td>90</td><td>Female</td><td>23612368</td><td>14:23612368 C / A <break/>(rs149245114)</td><td>p.Arg185Gln</td><td>R185L</td><td>NA</td><td>NA</td><td>0.002</td><td>0.00002471</td><td>0.012</td></tr></tbody></table><table-wrap-foot><fn><p>ARHL (age-related hearing loss). The age (years) of the subject when the Audiogram was performed is indicated. Variant [CHR: position reference/alternate (dbSNP135rsID)]. Consequence [HGUS annotation (protein change)]. Code [short description of the alternate variant]. Frequency of the mutations: Esp6500siv2 (NHLBI Exome Sequencing Project), 1000 g (1000 Genomes Project), Campion (The Allele Frequency Net Database) and ExAC (The Exome Aggregation Consortium).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31543-v2.xml">elife-31543-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.31543.009</object-id><label>Table 1.</label><caption><title>Bursting characteristics of CA1 place cells during exploration and rest.</title><p><supplementary-material id="table1sdata1"><object-id pub-id-type="doi">10.7554/eLife.31543.010</object-id><label>Table 1&#8212;source data 1.</label><caption><title>CA1 bursting analysis - full data set.</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-31543-table1-data1-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom"/><th colspan="8" valign="bottom">Linear track</th><th colspan="8" valign="bottom">Rest</th></tr><tr><th colspan="3" valign="bottom">WT</th><th colspan="3" valign="bottom">Dp16</th><th valign="bottom"/><th valign="bottom"><italic>p-value</italic></th><th colspan="3" valign="bottom">WT</th><th colspan="3" valign="bottom">Dp16</th><th valign="bottom"/><th valign="bottom"><italic>p-value</italic></th></tr></thead><tbody><tr><td valign="bottom">Bursts per min</td><td valign="bottom">7.901</td><td valign="bottom">&#177;</td><td valign="bottom">0.532</td><td valign="bottom">5.777</td><td valign="bottom">&#177;</td><td valign="bottom">0.383</td><td valign="bottom">***</td><td valign="bottom"><italic>0.0007</italic></td><td valign="bottom">7.152</td><td valign="bottom">&#177;</td><td valign="bottom">0.685</td><td valign="bottom">5.741</td><td valign="bottom">&#177;</td><td valign="bottom">0.468</td><td valign="bottom">*</td><td valign="bottom"><italic>0.0315</italic></td></tr><tr><td valign="bottom">Mean Inter-burst Interval (s)</td><td valign="bottom">18.940</td><td valign="bottom">&#177;</td><td valign="bottom">1.692</td><td valign="bottom">25.145</td><td valign="bottom">&#177;</td><td valign="bottom">1.907</td><td valign="bottom">***</td><td valign="bottom"><italic>0.0002</italic></td><td valign="bottom">30.061</td><td valign="bottom">&#177;</td><td valign="bottom">2.141</td><td valign="bottom">56.144</td><td valign="bottom">&#177;</td><td valign="bottom">6.640</td><td valign="bottom">*</td><td valign="bottom"><italic>0.0320</italic></td></tr><tr><td valign="bottom">Mean burst duration (ms)</td><td valign="bottom">7.720</td><td valign="bottom">&#177;</td><td valign="bottom">0.186</td><td valign="bottom">8.435</td><td valign="bottom">&#177;</td><td valign="bottom">0.318</td><td valign="bottom">***</td><td valign="bottom"><italic>2.67E-07</italic></td><td valign="bottom">12.718</td><td valign="bottom">&#177;</td><td valign="bottom">4.251</td><td valign="bottom">8.281</td><td valign="bottom">&#177;</td><td valign="bottom">0.072</td><td valign="bottom"/><td valign="bottom">0.1305</td></tr><tr><td valign="bottom">% of spikes in burst</td><td valign="bottom">33.1%</td><td valign="bottom">&#177;</td><td valign="bottom">0.8%</td><td valign="bottom">24.4%</td><td valign="bottom">&#177;</td><td valign="bottom">0.7%</td><td valign="bottom">***</td><td valign="bottom"><italic>2.17E-13</italic></td><td valign="bottom">29.7%</td><td valign="bottom">&#177;</td><td valign="bottom">0.8%</td><td valign="bottom">22.3%</td><td valign="bottom">&#177;</td><td valign="bottom">0.7%</td><td valign="bottom">***</td><td valign="bottom"><italic>9.58E-12</italic></td></tr><tr><td valign="bottom">Nb. Spikes per burst</td><td valign="bottom">2.264</td><td valign="bottom">&#177;</td><td valign="bottom">0.008</td><td valign="bottom">2.243</td><td valign="bottom">&#177;</td><td valign="bottom">0.009</td><td valign="bottom">*</td><td valign="bottom"><italic>0.0110</italic></td><td valign="bottom">2.374</td><td valign="bottom">&#177;</td><td valign="bottom">0.010</td><td valign="bottom">2.264</td><td valign="bottom">&#177;</td><td valign="bottom">0.009</td><td valign="bottom">***</td><td valign="bottom"><italic>1.59E-14</italic></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31596-v1.xml">elife-31596-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.31596.007</object-id><label>Table 1.</label><caption><title>Assignments of Ile &#948;1 methyl groups of full-length MgtE.</title><p>Chemical shifts for &#948;1 methyl groups in Ile residues in the Mg<sup>2+</sup>-free and bound states are shown. Labels a and b are used to discriminate two signals from a single Ile residue. <sup>1</sup>H chemical shifts were referenced to external sodium 2,2-dimethyl-2-silapentane-5-sulfonate (0 ppm), and <sup>13</sup>C chemical shifts were referenced indirectly. Spectral width (eight ppm) in the <sup>13</sup>C dimension was subtracted from the <sup>13</sup>C chemical shift of I338 owing to spectral aliasing.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Methyl group</th><th colspan="2" valign="top">Mg<sup>2+</sup>-free state (0 mM Mg<sup>2+</sup>)</th><th colspan="2" valign="top">Mg<sup>2+</sup>-bound state (5.0 mM Mg<sup>2+</sup>)</th></tr><tr><th valign="top"><sup>1</sup>H (ppm)</th><th valign="top"><sup>13</sup>C (ppm)</th><th valign="top"><sup>1</sup>H (ppm)</th><th valign="top"><sup>13</sup>C (ppm)</th></tr></thead><tbody><tr><td valign="top">I28</td><td valign="top">0.729</td><td valign="top">14.16</td><td valign="top">0.714</td><td valign="top">14.13</td></tr><tr><td valign="top">I84</td><td valign="top">0.559</td><td valign="top">13.63</td><td valign="top">0.526</td><td valign="top">13.62</td></tr><tr><td valign="top">I168a</td><td valign="top">0.539</td><td valign="top">14.12</td><td valign="top">0.526</td><td valign="top">14.14</td></tr><tr><td valign="top">I168b</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">0.606</td><td valign="top">14.62</td></tr><tr><td valign="top">I171a</td><td valign="top">0.651</td><td valign="top">13.99</td><td valign="top">0.644</td><td valign="top">13.98</td></tr><tr><td valign="top">I171b</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">0.606</td><td valign="top">14.62</td></tr><tr><td valign="top">I190a</td><td valign="top">0.627</td><td valign="top">13.77</td><td valign="top">0.621</td><td valign="top">13.77</td></tr><tr><td valign="top">I190b</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">0.641</td><td valign="top">12.83</td></tr><tr><td valign="top">I201</td><td valign="top">0.606</td><td valign="top">14.62</td><td valign="top">0.599</td><td valign="top">14.33</td></tr><tr><td valign="top">I242</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">0.85</td><td valign="top">14.13</td></tr><tr><td valign="top">I260</td><td valign="top">0.771</td><td valign="top">13.13</td><td valign="top">0.762</td><td valign="top">13.15</td></tr><tr><td valign="top">I291</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8722;0.171</td><td valign="top">11.63</td></tr><tr><td valign="top">I293</td><td valign="top">0.773</td><td valign="top">12.66</td><td valign="top">0.858</td><td valign="top">15.45</td></tr><tr><td valign="top">I302</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">0.872</td><td valign="top">14.68</td></tr><tr><td valign="top">I338</td><td valign="top">0.145</td><td valign="top">8.67</td><td valign="top">0.264</td><td valign="top">10.16</td></tr><tr><td valign="top">I339</td><td valign="top">0.799</td><td valign="top">12.66</td><td valign="top">0.798</td><td valign="top">12.65</td></tr><tr><td valign="top">I397</td><td valign="top">0.771</td><td valign="top">14.59</td><td valign="top">0.728</td><td valign="top">14.92</td></tr><tr><td valign="top">I438</td><td valign="top">0.674</td><td valign="top">13.54</td><td valign="top">0.693</td><td valign="top">13.58</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31596-v2.xml">elife-31596-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.31596.007</object-id><label>Table 1.</label><caption><title>Assignments of Ile &#948;1 methyl groups of full-length MgtE.</title><p>Chemical shifts for &#948;1 methyl groups in Ile residues in the Mg<sup>2+</sup>-free and bound states are shown. Labels a and b are used to discriminate two signals from a single Ile residue. <sup>1</sup>H chemical shifts were referenced to external sodium 2,2-dimethyl-2-silapentane-5-sulfonate (0 ppm), and <sup>13</sup>C chemical shifts were referenced indirectly. Spectral width (eight ppm) in the <sup>13</sup>C dimension was subtracted from the <sup>13</sup>C chemical shift of I338 owing to spectral aliasing.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Methyl group</th><th colspan="2" valign="top">Mg<sup>2+</sup>-free state (0 mM Mg<sup>2+</sup>)</th><th colspan="2" valign="top">Mg<sup>2+</sup>-bound state (5.0 mM Mg<sup>2+</sup>)</th></tr><tr><th valign="top"><sup>1</sup>H (ppm)</th><th valign="top"><sup>13</sup>C (ppm)</th><th valign="top"><sup>1</sup>H (ppm)</th><th valign="top"><sup>13</sup>C (ppm)</th></tr></thead><tbody><tr><td valign="top">I28</td><td valign="top">0.729</td><td valign="top">14.16</td><td valign="top">0.714</td><td valign="top">14.13</td></tr><tr><td valign="top">I84</td><td valign="top">0.559</td><td valign="top">13.63</td><td valign="top">0.526</td><td valign="top">13.62</td></tr><tr><td valign="top">I168a</td><td valign="top">0.539</td><td valign="top">14.12</td><td valign="top">0.526</td><td valign="top">14.14</td></tr><tr><td valign="top">I168b</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">0.606</td><td valign="top">14.62</td></tr><tr><td valign="top">I171a</td><td valign="top">0.651</td><td valign="top">13.99</td><td valign="top">0.644</td><td valign="top">13.98</td></tr><tr><td valign="top">I171b</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">0.606</td><td valign="top">14.62</td></tr><tr><td valign="top">I190a</td><td valign="top">0.627</td><td valign="top">13.77</td><td valign="top">0.621</td><td valign="top">13.77</td></tr><tr><td valign="top">I190b</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">0.641</td><td valign="top">12.83</td></tr><tr><td valign="top">I201</td><td valign="top">0.606</td><td valign="top">14.62</td><td valign="top">0.599</td><td valign="top">14.33</td></tr><tr><td valign="top">I242</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">0.85</td><td valign="top">14.13</td></tr><tr><td valign="top">I260</td><td valign="top">0.771</td><td valign="top">13.13</td><td valign="top">0.762</td><td valign="top">13.15</td></tr><tr><td valign="top">I291</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8722;0.171</td><td valign="top">11.63</td></tr><tr><td valign="top">I293</td><td valign="top">0.773</td><td valign="top">12.66</td><td valign="top">0.858</td><td valign="top">15.45</td></tr><tr><td valign="top">I302</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">0.872</td><td valign="top">14.68</td></tr><tr><td valign="top">I338</td><td valign="top">0.145</td><td valign="top">8.67</td><td valign="top">0.264</td><td valign="top">10.16</td></tr><tr><td valign="top">I339</td><td valign="top">0.799</td><td valign="top">12.66</td><td valign="top">0.798</td><td valign="top">12.65</td></tr><tr><td valign="top">I397</td><td valign="top">0.771</td><td valign="top">14.59</td><td valign="top">0.728</td><td valign="top">14.92</td></tr><tr><td valign="top">I438</td><td valign="top">0.674</td><td valign="top">13.54</td><td valign="top">0.693</td><td valign="top">13.58</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31677-v1.xml">elife-31677-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.31677.014</object-id><label>Table 1.</label><caption><title>Mating procedure results for the six proof-of-principle AGOC sublines from the F3 to the F7 generation.</title><p>Bold entries mark progeny that were used in the subsequent cross. F6-S, F7-O and F7-C are control crosses. No significant differences between the arithmetic means and the theoretical Mendelian ratios were found. See Source Data 1 for raw scores ordered by transgenic sublines.</p><p><supplementary-material id="table1sdata1"><object-id pub-id-type="doi">10.7554/eLife.31677.015</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Raw scores for all mating procedure result tables ordered by transgenic sublines.</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-31677-table1-data1-v1.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gen</th><th rowspan="2">Cross</th><th rowspan="2">Subline</th><th colspan="9">Progeny</th></tr><tr><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf1-v1"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf2-v1"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf3-v1"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf4-v1"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf5-v1"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf6-v1"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf7-v1"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf8-v1"/></th><th>Total</th></tr></thead><tbody><tr><td rowspan="8"><bold>F3</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf9-v1"/></td><td>Theoretical</td><td>-</td><td>50.0%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>50.0%</bold></td><td>-</td></tr><tr><td>AGOC #1</td><td>-</td><td>47.9% (46)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>52.1% (50)</bold></td><td>96</td></tr><tr><td>AGOC #2</td><td>-</td><td>52.4% (44)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>47.6% (40)</bold></td><td>84</td></tr><tr><td>AGOC #3</td><td>-</td><td>55.1% (49)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>44.9% (40)</bold></td><td>89</td></tr><tr><td>AGOC #4</td><td>-</td><td>44.1% (41)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>55.9% (52)</bold></td><td>93</td></tr><tr><td>AGOC #5</td><td>-</td><td>47.6% (39)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>52.4% (43)</bold></td><td>82</td></tr><tr><td>AGOC #6</td><td>-</td><td>48.2% (27)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>51.8% (29)</bold></td><td>56</td></tr><tr><td>Mean</td><td>-</td><td>49.2 &#177; 3.9%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>50.8 &#177; 3.9%</bold></td><td>83.3</td></tr><tr><td rowspan="8"><bold>F4</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf10-v1"/></td><td>Theoretical</td><td>25.0%</td><td>25.0%</td><td><bold>12.5%</bold></td><td><bold>12.5%</bold></td><td>12.5%</td><td>12.5%</td><td>-</td><td>-</td><td>-</td></tr><tr><td>AGOC #1</td><td>25.6% (30)</td><td>34.2% (40)</td><td><bold>4.3% (5)</bold></td><td><bold>10.2% (12)</bold></td><td>6.0% (7)</td><td>19.7% (23)</td><td>-</td><td>-</td><td>117</td></tr><tr><td>AGOC #2</td><td>27.3% (36)</td><td>28.9% (38)</td><td><bold>15.1% (20)</bold></td><td><bold>8.3% (11)</bold></td><td>9.8% (13)</td><td>10.6% (14)</td><td>-</td><td>-</td><td>132</td></tr><tr><td>AGOC #3</td><td>25.8% (33)</td><td>32.8% (42)</td><td><bold>13.3% (17)</bold></td><td><bold>10.1% (13)</bold></td><td>5.5% (7)</td><td>12.5% (16)</td><td>-</td><td>-</td><td>128</td></tr><tr><td>AGOC #4*</td><td>14.0% (14)</td><td>15.0% (15)</td><td><bold>20.0% (20)</bold></td><td><bold>13.0% (13)</bold></td><td>22.0% (22)</td><td>9.0% (9)</td><td>3.0% (3)</td><td>4.0% (4)</td><td>100</td></tr><tr><td>AGOC #5</td><td>17.1% (20)</td><td>40.2% (47)</td><td><bold>16.2% (19)</bold></td><td><bold>8.5% (10)</bold></td><td>10.3% (12)</td><td>7.7% (9)</td><td>-</td><td>-</td><td>117</td></tr><tr><td>AGOC #6</td><td>33.6% (39)</td><td>33.6% (39)</td><td><bold>8.6% (10)</bold></td><td><bold>6.9% (8)</bold></td><td>6.1% (7)</td><td>11.2% (13)</td><td>-</td><td>-</td><td>116</td></tr><tr><td>Mean</td><td>23.9 &#177; 7.2%</td><td>30.8 &#177; 8.5%</td><td><bold>12.9 &#177; 5.6%</bold></td><td><bold>9.5 &#177; 2.1%</bold></td><td>9.9 &#177; 6.3%</td><td>11.8 &#177; 4.2%</td><td>0.5%</td><td>0.7%</td><td>118.3</td></tr><tr><td rowspan="8"><bold>F5</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf11-v1"/></td><td>Theoretical</td><td>25.0%</td><td>-</td><td>25.0%</td><td>25.0%</td><td>-</td><td>-</td><td><bold>25.0%</bold></td><td>-</td><td>-</td></tr><tr><td>AGOC #1</td><td>27.2% (31)</td><td>-</td><td>26.3% (30)</td><td>23.7% (27)</td><td>-</td><td>-</td><td><bold>22.8% (26)</bold></td><td>-</td><td>114</td></tr><tr><td>AGOC #2</td><td>28.9% (26)</td><td>-</td><td>33.3% (30)</td><td>17.8% (16)</td><td>-</td><td>-</td><td><bold>20.0% (18)</bold></td><td>-</td><td>90</td></tr><tr><td>AGOC #3</td><td>24.8% (30)</td><td>-</td><td>27.3% (33)</td><td>24.8% (30)</td><td>-</td><td>-</td><td><bold>23.1% (28)</bold></td><td>-</td><td>121</td></tr><tr><td>AGOC #4</td><td>19.3% (21)</td><td>-</td><td>22.9% (25)</td><td>37.6% (41)</td><td>-</td><td>-</td><td><bold>20.2% (22)</bold></td><td>-</td><td>109</td></tr><tr><td>AGOC #5</td><td>28.2% (31)</td><td>-</td><td>29.1% (32)</td><td>12.7% (14)</td><td>-</td><td>-</td><td><bold>30.0% (33)</bold></td><td>-</td><td>110</td></tr><tr><td>AGOC #6</td><td>26.2% (22)</td><td>-</td><td>29.7% (25)</td><td>16.7% (14)</td><td>-</td><td>-</td><td><bold>27.4% (23)</bold></td><td>-</td><td>84</td></tr><tr><td>Mean</td><td>25.8 &#177; 3.5%</td><td>-</td><td>28.1 &#177; 3.5%</td><td>22.2 &#177; 8.8%</td><td>-</td><td>-</td><td><bold>23.9 &#177; 4.0%</bold></td><td>-</td><td>104.7</td></tr><tr><td rowspan="8"><bold>F6-S</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf12-v1"/></td><td>Theoretical</td><td>-</td><td>-</td><td>50.0%</td><td>50.0%</td><td>-</td><td>-</td><td>-</td><td>-</td><td/></tr><tr><td>AGOC #1</td><td>-</td><td>-</td><td>46.4% (39)</td><td>53.6% (44)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>84</td></tr><tr><td>AGOC #2</td><td>-</td><td>-</td><td>50.0% (49)</td><td>50.0% (49)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>98</td></tr><tr><td>AGOC #3</td><td>-</td><td>-</td><td>54.0% (68)</td><td>46.0% (58)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>126</td></tr><tr><td>AGOC #4</td><td>-</td><td>-</td><td>53.8% (50)</td><td>46.2% (43)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>93</td></tr><tr><td>AGOC #5</td><td>-</td><td>-</td><td>51.3% (59)</td><td>48.7% (56)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>115</td></tr><tr><td>AGOC #6</td><td>-</td><td>-</td><td>57.0% (49)</td><td>43.0% (37)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>86</td></tr><tr><td>Mean</td><td>-</td><td>-</td><td>52.1 &#177; 3.7%</td><td>47.9 &#177; 3.7%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>100.3</td></tr><tr><td rowspan="8"><bold>F6</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf13-v1"/></td><td>Theoretical</td><td>-</td><td>-</td><td><bold>25.0%</bold></td><td><bold>25.0%</bold></td><td>-</td><td>-</td><td>50.0%</td><td>-</td><td>-</td></tr><tr><td>AGOC #1</td><td>-</td><td>-</td><td><bold>20.3% (23)</bold></td><td><bold>24.8% (28)</bold></td><td>-</td><td>-</td><td>54.9% (62)</td><td>-</td><td>113</td></tr><tr><td>AGOC #2</td><td>-</td><td>-</td><td><bold>21.5% (23)</bold></td><td><bold>35.5% (38)</bold></td><td>-</td><td>-</td><td>43.0% (46)</td><td>-</td><td>117</td></tr><tr><td>AGOC #3</td><td>-</td><td>-</td><td><bold>22.9% (27)</bold></td><td><bold>22.9% (27)</bold></td><td>-</td><td>-</td><td>54.2% (64)</td><td>-</td><td>118</td></tr><tr><td>AGOC #4</td><td>-</td><td>-</td><td><bold>22.0% (29)</bold></td><td><bold>22.7% (30)</bold></td><td>-</td><td>-</td><td>55.3% (73)</td><td>-</td><td>132</td></tr><tr><td>AGOC #5</td><td>-</td><td>-</td><td><bold>17.5% (18)</bold></td><td><bold>31.1% (32)</bold></td><td>-</td><td>-</td><td>51.4% (53)</td><td>-</td><td>103</td></tr><tr><td>AGOC #6</td><td>-</td><td>-</td><td><bold>19.8% (22)</bold></td><td><bold>24.3% (27)</bold></td><td>-</td><td>-</td><td>55.9% (62)</td><td>-</td><td>111</td></tr><tr><td>Mean</td><td>-</td><td>-</td><td><bold>20.7 &#177; 1.9%</bold></td><td><bold>26.9 &#177; 5.2%</bold></td><td>-</td><td>-</td><td>52.4 &#177; 4.9%</td><td>-</td><td>115.7</td></tr><tr><td rowspan="8"><bold>F7-O</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf14-v1"/></td><td>Theoretical</td><td>-</td><td>-</td><td>100%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>AGOC #1</td><td>-</td><td>-</td><td>100% (94)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>94</td></tr><tr><td>AGOC #2</td><td>-</td><td>-</td><td>100% (50)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>50</td></tr><tr><td>AGOC #3</td><td>-</td><td>-</td><td>100% (49)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>49</td></tr><tr><td>AGOC #4</td><td>-</td><td>-</td><td>100% (79)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>79</td></tr><tr><td>AGOC #5</td><td>-</td><td>-</td><td>100% (79)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>79</td></tr><tr><td>AGOC #6</td><td>-</td><td>-</td><td>100% (63)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>63</td></tr><tr><td>Mean</td><td>-</td><td>-</td><td>100 &#177; 0%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>69.0</td></tr><tr><td rowspan="8"><bold>F7-C</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf15-v1"/></td><td>Theoretical</td><td>-</td><td>-</td><td>-</td><td>100%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>AGOC #1</td><td>-</td><td>-</td><td>-</td><td>100% (101)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>101</td></tr><tr><td>AGOC #2</td><td>-</td><td>-</td><td>-</td><td>100% (105)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>105</td></tr><tr><td>AGOC #3</td><td>-</td><td>-</td><td>-</td><td>100% (89)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>89</td></tr><tr><td>AGOC #4</td><td>-</td><td>-</td><td>-</td><td>100% (54)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>54</td></tr><tr><td>AGOC #5</td><td>-</td><td>-</td><td>-</td><td>100% (74)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>74</td></tr><tr><td>AGOC #6</td><td>-</td><td>-</td><td>-</td><td>100% (64)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>64</td></tr><tr><td>Mean</td><td>-</td><td>-</td><td>-</td><td>100 &#177; 0%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>81.2</td></tr></tbody></table><table-wrap-foot><fn><p>*In the AGOC #4 subline, incomplete recombination occurred in the F4 (mCe; mO-mC) double hemizygous generation, as we obtained several F5 individuals that still carried&#160;both transformation markers (7.0% in total). We continued the mating procedure with the F5 (mO) and (mC) post-recombination hemizygous progeny.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31677-v2.xml">elife-31677-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.31677.014</object-id><label>Table 1.</label><caption><title>Mating procedure results for the six proof-of-principle AGOC sublines from the F3 to the F7 generation.</title><p>Bold entries mark progeny that were used in the subsequent cross. F6-S, F7-O and F7-C are control crosses. No significant differences between the arithmetic means and the theoretical Mendelian ratios were found. See Source Data 1 for raw scores ordered by transgenic sublines.</p><p><supplementary-material id="table1sdata1"><object-id pub-id-type="doi">10.7554/eLife.31677.015</object-id><label>Table 1&#8212;source data 1.</label><caption><title>Raw scores for all mating procedure result tables ordered by transgenic sublines.</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-31677-table1-data1-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gen</th><th rowspan="2">Cross</th><th rowspan="2">Subline</th><th colspan="9">Progeny</th></tr><tr><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf1-v2"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf2-v2"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf3-v2"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf4-v2"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf5-v2"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf6-v2"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf7-v2"/></th><th><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf8-v2"/></th><th>Total</th></tr></thead><tbody><tr><td rowspan="8"><bold>F3</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf9-v2"/></td><td>Theoretical</td><td>-</td><td>50.0%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>50.0%</bold></td><td>-</td></tr><tr><td>AGOC #1</td><td>-</td><td>47.9% (46)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>52.1% (50)</bold></td><td>96</td></tr><tr><td>AGOC #2</td><td>-</td><td>52.4% (44)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>47.6% (40)</bold></td><td>84</td></tr><tr><td>AGOC #3</td><td>-</td><td>55.1% (49)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>44.9% (40)</bold></td><td>89</td></tr><tr><td>AGOC #4</td><td>-</td><td>44.1% (41)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>55.9% (52)</bold></td><td>93</td></tr><tr><td>AGOC #5</td><td>-</td><td>47.6% (39)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>52.4% (43)</bold></td><td>82</td></tr><tr><td>AGOC #6</td><td>-</td><td>48.2% (27)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>51.8% (29)</bold></td><td>56</td></tr><tr><td>Mean</td><td>-</td><td>49.2 &#177; 3.9%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td><bold>50.8 &#177; 3.9%</bold></td><td>83.3</td></tr><tr><td rowspan="8"><bold>F4</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf10-v2"/></td><td>Theoretical</td><td>25.0%</td><td>25.0%</td><td><bold>12.5%</bold></td><td><bold>12.5%</bold></td><td>12.5%</td><td>12.5%</td><td>-</td><td>-</td><td>-</td></tr><tr><td>AGOC #1</td><td>25.6% (30)</td><td>34.2% (40)</td><td><bold>4.3% (5)</bold></td><td><bold>10.2% (12)</bold></td><td>6.0% (7)</td><td>19.7% (23)</td><td>-</td><td>-</td><td>117</td></tr><tr><td>AGOC #2</td><td>27.3% (36)</td><td>28.9% (38)</td><td><bold>15.1% (20)</bold></td><td><bold>8.3% (11)</bold></td><td>9.8% (13)</td><td>10.6% (14)</td><td>-</td><td>-</td><td>132</td></tr><tr><td>AGOC #3</td><td>25.8% (33)</td><td>32.8% (42)</td><td><bold>13.3% (17)</bold></td><td><bold>10.1% (13)</bold></td><td>5.5% (7)</td><td>12.5% (16)</td><td>-</td><td>-</td><td>128</td></tr><tr><td>AGOC #4*</td><td>14.0% (14)</td><td>15.0% (15)</td><td><bold>20.0% (20)</bold></td><td><bold>13.0% (13)</bold></td><td>22.0% (22)</td><td>9.0% (9)</td><td>3.0% (3)</td><td>4.0% (4)</td><td>100</td></tr><tr><td>AGOC #5</td><td>17.1% (20)</td><td>40.2% (47)</td><td><bold>16.2% (19)</bold></td><td><bold>8.5% (10)</bold></td><td>10.3% (12)</td><td>7.7% (9)</td><td>-</td><td>-</td><td>117</td></tr><tr><td>AGOC #6</td><td>33.6% (39)</td><td>33.6% (39)</td><td><bold>8.6% (10)</bold></td><td><bold>6.9% (8)</bold></td><td>6.1% (7)</td><td>11.2% (13)</td><td>-</td><td>-</td><td>116</td></tr><tr><td>Mean</td><td>23.9 &#177; 7.2%</td><td>30.8 &#177; 8.5%</td><td><bold>12.9 &#177; 5.6%</bold></td><td><bold>9.5 &#177; 2.1%</bold></td><td>9.9 &#177; 6.3%</td><td>11.8 &#177; 4.2%</td><td>0.5%</td><td>0.7%</td><td>118.3</td></tr><tr><td rowspan="8"><bold>F5</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf11-v2"/></td><td>Theoretical</td><td>25.0%</td><td>-</td><td>25.0%</td><td>25.0%</td><td>-</td><td>-</td><td><bold>25.0%</bold></td><td>-</td><td>-</td></tr><tr><td>AGOC #1</td><td>27.2% (31)</td><td>-</td><td>26.3% (30)</td><td>23.7% (27)</td><td>-</td><td>-</td><td><bold>22.8% (26)</bold></td><td>-</td><td>114</td></tr><tr><td>AGOC #2</td><td>28.9% (26)</td><td>-</td><td>33.3% (30)</td><td>17.8% (16)</td><td>-</td><td>-</td><td><bold>20.0% (18)</bold></td><td>-</td><td>90</td></tr><tr><td>AGOC #3</td><td>24.8% (30)</td><td>-</td><td>27.3% (33)</td><td>24.8% (30)</td><td>-</td><td>-</td><td><bold>23.1% (28)</bold></td><td>-</td><td>121</td></tr><tr><td>AGOC #4</td><td>19.3% (21)</td><td>-</td><td>22.9% (25)</td><td>37.6% (41)</td><td>-</td><td>-</td><td><bold>20.2% (22)</bold></td><td>-</td><td>109</td></tr><tr><td>AGOC #5</td><td>28.2% (31)</td><td>-</td><td>29.1% (32)</td><td>12.7% (14)</td><td>-</td><td>-</td><td><bold>30.0% (33)</bold></td><td>-</td><td>110</td></tr><tr><td>AGOC #6</td><td>26.2% (22)</td><td>-</td><td>29.7% (25)</td><td>16.7% (14)</td><td>-</td><td>-</td><td><bold>27.4% (23)</bold></td><td>-</td><td>84</td></tr><tr><td>Mean</td><td>25.8 &#177; 3.5%</td><td>-</td><td>28.1 &#177; 3.5%</td><td>22.2 &#177; 8.8%</td><td>-</td><td>-</td><td><bold>23.9 &#177; 4.0%</bold></td><td>-</td><td>104.7</td></tr><tr><td rowspan="8"><bold>F6-S</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf12-v2"/></td><td>Theoretical</td><td>-</td><td>-</td><td>50.0%</td><td>50.0%</td><td>-</td><td>-</td><td>-</td><td>-</td><td/></tr><tr><td>AGOC #1</td><td>-</td><td>-</td><td>46.4% (39)</td><td>53.6% (44)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>84</td></tr><tr><td>AGOC #2</td><td>-</td><td>-</td><td>50.0% (49)</td><td>50.0% (49)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>98</td></tr><tr><td>AGOC #3</td><td>-</td><td>-</td><td>54.0% (68)</td><td>46.0% (58)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>126</td></tr><tr><td>AGOC #4</td><td>-</td><td>-</td><td>53.8% (50)</td><td>46.2% (43)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>93</td></tr><tr><td>AGOC #5</td><td>-</td><td>-</td><td>51.3% (59)</td><td>48.7% (56)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>115</td></tr><tr><td>AGOC #6</td><td>-</td><td>-</td><td>57.0% (49)</td><td>43.0% (37)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>86</td></tr><tr><td>Mean</td><td>-</td><td>-</td><td>52.1 &#177; 3.7%</td><td>47.9 &#177; 3.7%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>100.3</td></tr><tr><td rowspan="8"><bold>F6</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf13-v2"/></td><td>Theoretical</td><td>-</td><td>-</td><td><bold>25.0%</bold></td><td><bold>25.0%</bold></td><td>-</td><td>-</td><td>50.0%</td><td>-</td><td>-</td></tr><tr><td>AGOC #1</td><td>-</td><td>-</td><td><bold>20.3% (23)</bold></td><td><bold>24.8% (28)</bold></td><td>-</td><td>-</td><td>54.9% (62)</td><td>-</td><td>113</td></tr><tr><td>AGOC #2</td><td>-</td><td>-</td><td><bold>21.5% (23)</bold></td><td><bold>35.5% (38)</bold></td><td>-</td><td>-</td><td>43.0% (46)</td><td>-</td><td>117</td></tr><tr><td>AGOC #3</td><td>-</td><td>-</td><td><bold>22.9% (27)</bold></td><td><bold>22.9% (27)</bold></td><td>-</td><td>-</td><td>54.2% (64)</td><td>-</td><td>118</td></tr><tr><td>AGOC #4</td><td>-</td><td>-</td><td><bold>22.0% (29)</bold></td><td><bold>22.7% (30)</bold></td><td>-</td><td>-</td><td>55.3% (73)</td><td>-</td><td>132</td></tr><tr><td>AGOC #5</td><td>-</td><td>-</td><td><bold>17.5% (18)</bold></td><td><bold>31.1% (32)</bold></td><td>-</td><td>-</td><td>51.4% (53)</td><td>-</td><td>103</td></tr><tr><td>AGOC #6</td><td>-</td><td>-</td><td><bold>19.8% (22)</bold></td><td><bold>24.3% (27)</bold></td><td>-</td><td>-</td><td>55.9% (62)</td><td>-</td><td>111</td></tr><tr><td>Mean</td><td>-</td><td>-</td><td><bold>20.7 &#177; 1.9%</bold></td><td><bold>26.9 &#177; 5.2%</bold></td><td>-</td><td>-</td><td>52.4 &#177; 4.9%</td><td>-</td><td>115.7</td></tr><tr><td rowspan="8"><bold>F7-O</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf14-v2"/></td><td>Theoretical</td><td>-</td><td>-</td><td>100%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>AGOC #1</td><td>-</td><td>-</td><td>100% (94)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>94</td></tr><tr><td>AGOC #2</td><td>-</td><td>-</td><td>100% (50)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>50</td></tr><tr><td>AGOC #3</td><td>-</td><td>-</td><td>100% (49)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>49</td></tr><tr><td>AGOC #4</td><td>-</td><td>-</td><td>100% (79)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>79</td></tr><tr><td>AGOC #5</td><td>-</td><td>-</td><td>100% (79)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>79</td></tr><tr><td>AGOC #6</td><td>-</td><td>-</td><td>100% (63)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>63</td></tr><tr><td>Mean</td><td>-</td><td>-</td><td>100 &#177; 0%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td><td>69.0</td></tr><tr><td rowspan="8"><bold>F7-C</bold></td><td rowspan="8"><inline-graphic mime-subtype="x-tiff" mimetype="image" xlink:href="elife-31677-inf15-v2"/></td><td>Theoretical</td><td>-</td><td>-</td><td>-</td><td>100%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>AGOC #1</td><td>-</td><td>-</td><td>-</td><td>100% (101)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>101</td></tr><tr><td>AGOC #2</td><td>-</td><td>-</td><td>-</td><td>100% (105)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>105</td></tr><tr><td>AGOC #3</td><td>-</td><td>-</td><td>-</td><td>100% (89)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>89</td></tr><tr><td>AGOC #4</td><td>-</td><td>-</td><td>-</td><td>100% (54)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>54</td></tr><tr><td>AGOC #5</td><td>-</td><td>-</td><td>-</td><td>100% (74)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>74</td></tr><tr><td>AGOC #6</td><td>-</td><td>-</td><td>-</td><td>100% (64)</td><td>-</td><td>-</td><td>-</td><td>-</td><td>64</td></tr><tr><td>Mean</td><td>-</td><td>-</td><td>-</td><td>100 &#177; 0%</td><td>-</td><td>-</td><td>-</td><td>-</td><td>81.2</td></tr></tbody></table><table-wrap-foot><fn><p>*In the AGOC #4 subline, incomplete recombination occurred in the F4 (mCe; mO-mC) double hemizygous generation, as we obtained several F5 individuals that still carried&#160;both transformation markers (7.0% in total). We continued the mating procedure with the F5 (mO) and (mC) post-recombination hemizygous progeny.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31724-v2.xml">elife-31724-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.31724.037</object-id><label>Table 1.</label><caption><title>Summary of topo VI functional activities and mutant effects.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Assay</th><th rowspan="2">Wildtype activity</th><th colspan="3" valign="top">Effect of mutation to:</th></tr><tr><th valign="top"><italic>KGRR loop</italic></th><th valign="top"><italic>Stalk/WKxY</italic></th><th valign="top"><italic>H2TH</italic></th></tr></thead><tbody><tr><td valign="top">Supercoil relaxation</td><td valign="top">distributive strand passage activity</td><td valign="top">kills strand passage</td><td valign="top">kills strand passage</td><td valign="top">greatly impairs strand passage</td></tr><tr><td colspan="5" valign="top">ATPase activity</td></tr><tr><td valign="top"><italic>-on linear DNA</italic></td><td valign="top">stimulates above basal activity</td><td valign="top">no activity</td><td valign="top">no activity</td><td valign="top">~wildtype activity</td></tr><tr><td valign="top"><italic>-on supercoiled DNA</italic></td><td valign="top">stimulates more than linear DNA</td><td valign="top">increased activity, futile cycling</td><td valign="top">no activity</td><td valign="top">~wildtype activity, futile cycling</td></tr><tr><td colspan="5" valign="top">DNA binding</td></tr><tr><td valign="top"><italic>-short duplexes</italic></td><td valign="top">affinity increases from 20 bp to 40 bp in length</td><td valign="top">moderately impairs binding for longer duplexes</td><td valign="top">greatly impairs binding</td><td valign="top">slightly impairs binding for longer duplexes</td></tr><tr><td valign="top"><italic>-sheared salmon-sperm DNA</italic></td><td valign="top">similar affinity as for 40&#8211;70 bp duplexes</td><td valign="top">moderately impairs binding</td><td valign="top">N.D.</td><td valign="top">slightly impairs binding</td></tr><tr><td valign="top"><italic>-supercoiled DNA</italic></td><td valign="top">increased affinity compared to linear DNA</td><td valign="top">moderately impairs binding</td><td valign="top">N.D.</td><td valign="top">moderately impairs binding</td></tr><tr><td valign="top"><italic>-stacked junction</italic></td><td valign="top">tighter binding than to duplex</td><td valign="top">greatly impairs binding</td><td valign="top">N.D.</td><td valign="top">~wildtype affinity</td></tr><tr><td colspan="5" valign="top">Top6B dimerization</td></tr><tr><td valign="top"><italic>-on short DNA duplexes</italic></td><td valign="top">DNA promotes closure <break/>AMPPNP promotes further closure</td><td valign="top">loss of substrate promoted closure <break/>AMPPNP promotes some closure</td><td valign="top">N.D.</td><td valign="top">N.D.</td></tr><tr><td valign="top"><italic>-on supercoiled DNA</italic></td><td valign="top">promotes greater closure than linear DNA <break/>AMPPNP promotes further closure</td><td valign="top">supercoiled DNA promotes closure loss of AMPPNP promoted closure</td><td valign="top">N.D.</td><td valign="top">weaker substrate dependent closure than wildtype AMPPNP promotes further closure</td></tr><tr><td valign="top">Short duplex cleavage</td><td valign="top">AMPPNP promotes cleavage on 60 and 70 bp duplexes</td><td valign="top">similar to wildtype</td><td valign="top">greatly impairs cleavage</td><td valign="top">no cleavage</td></tr><tr><td valign="top">Short duplex bending</td><td valign="top">AMPPNP promotes bending</td><td valign="top">similar to wildtype</td><td valign="top">greatly slows bending</td><td valign="top">no bending</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-31724-v3.xml">elife-31724-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.31724.037</object-id><label>Table 1.</label><caption><title>Summary of topo VI functional activities and mutant effects.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Assay</th><th rowspan="2">Wildtype activity</th><th colspan="3" valign="top">Effect of mutation to:</th></tr><tr><th valign="top"><italic>KGRR loop</italic></th><th valign="top"><italic>Stalk/WKxY</italic></th><th valign="top"><italic>H2TH</italic></th></tr></thead><tbody><tr><td valign="top">Supercoil relaxation</td><td valign="top">distributive strand passage activity</td><td valign="top">kills strand passage</td><td valign="top">kills strand passage</td><td valign="top">greatly impairs strand passage</td></tr><tr><td colspan="5" valign="top">ATPase activity</td></tr><tr><td valign="top"><italic>-on linear DNA</italic></td><td valign="top">stimulates above basal activity</td><td valign="top">no activity</td><td valign="top">no activity</td><td valign="top">~wildtype activity</td></tr><tr><td valign="top"><italic>-on supercoiled DNA</italic></td><td valign="top">stimulates more than linear DNA</td><td valign="top">increased activity, futile cycling</td><td valign="top">no activity</td><td valign="top">~wildtype activity, futile cycling</td></tr><tr><td colspan="5" valign="top">DNA binding</td></tr><tr><td valign="top"><italic>-short duplexes</italic></td><td valign="top">affinity increases from 20 bp to 40 bp in length</td><td valign="top">moderately impairs binding for longer duplexes</td><td valign="top">greatly impairs binding</td><td valign="top">slightly impairs binding for longer duplexes</td></tr><tr><td valign="top"><italic>-sheared salmon-sperm DNA</italic></td><td valign="top">similar affinity as for 40&#8211;70 bp duplexes</td><td valign="top">moderately impairs binding</td><td valign="top">N.D.</td><td valign="top">slightly impairs binding</td></tr><tr><td valign="top"><italic>-supercoiled DNA</italic></td><td valign="top">increased affinity compared to linear DNA</td><td valign="top">moderately impairs binding</td><td valign="top">N.D.</td><td valign="top">moderately impairs binding</td></tr><tr><td valign="top"><italic>-stacked junction</italic></td><td valign="top">tighter binding than to duplex</td><td valign="top">greatly impairs binding</td><td valign="top">N.D.</td><td valign="top">~wildtype affinity</td></tr><tr><td colspan="5" valign="top">Top6B dimerization</td></tr><tr><td valign="top"><italic>-on short DNA duplexes</italic></td><td valign="top">DNA promotes closure <break/>AMPPNP promotes further closure</td><td valign="top">loss of substrate promoted closure <break/>AMPPNP promotes some closure</td><td valign="top">N.D.</td><td valign="top">N.D.</td></tr><tr><td valign="top"><italic>-on supercoiled DNA</italic></td><td valign="top">promotes greater closure than linear DNA <break/>AMPPNP promotes further closure</td><td valign="top">supercoiled DNA promotes closure loss of AMPPNP promoted closure</td><td valign="top">N.D.</td><td valign="top">weaker substrate dependent closure than wildtype AMPPNP promotes further closure</td></tr><tr><td valign="top">Short duplex cleavage</td><td valign="top">AMPPNP promotes cleavage on 60 and 70 bp duplexes</td><td valign="top">similar to wildtype</td><td valign="top">greatly impairs cleavage</td><td valign="top">no cleavage</td></tr><tr><td valign="top">Short duplex bending</td><td valign="top">AMPPNP promotes bending</td><td valign="top">similar to wildtype</td><td valign="top">greatly slows bending</td><td valign="top">no bending</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32110-v2.xml">elife-32110-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.32110.022</object-id><label>Table 1.</label><caption><title>Predicted gene function: Mutants with increased lipid accumulation.</title><p>Predicted functions for genes for which mutants were high-confidence candidates for increased lipid accumulation (enrichment scores clustered in LA1, <xref ref-type="fig" rid="fig5">Figure 5</xref>).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom"/><th rowspan="2">Gene ID</th><th rowspan="2">Short <break/>name</th><th rowspan="2">Annotation <break/>from</th><th rowspan="2">Description</th><th colspan="2" valign="bottom">Enrichment</th></tr><tr><th>BD</th><th>FACS</th></tr></thead><tbody><tr><td colspan="3"><bold>G Protein Switches</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">&#8195;*</td><td valign="bottom"><italic>RTO4_15883</italic></td><td valign="bottom"><italic>RAS1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">GTPase</td><td valign="bottom">2.0</td><td valign="bottom">2.3</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_14088</italic></td><td valign="bottom"><italic>RAC1</italic></td><td valign="bottom"><italic>H. sapiens</italic></td><td valign="bottom">GTPase</td><td valign="bottom">2.0</td><td valign="bottom">0.9</td></tr><tr><td valign="bottom">&#8195;*</td><td valign="bottom"><italic>RTO4_16215</italic></td><td valign="bottom"><italic>GNAI1-like</italic></td><td valign="bottom"><italic>H. sapiens</italic></td><td valign="bottom">GTPase</td><td valign="bottom">1.6</td><td valign="bottom">1.0</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_11402</italic></td><td valign="bottom"><italic>gapA</italic></td><td valign="bottom"><italic>A. nidulans</italic></td><td valign="bottom">GTPase-activating protein</td><td valign="bottom">0.6</td><td valign="bottom">1.4</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_13336</italic></td><td valign="bottom"><italic>RIC8A</italic></td><td valign="bottom"><italic>H. sapiens</italic></td><td valign="bottom">Guanine nucleotide exchange factor</td><td valign="bottom">1.3</td><td valign="bottom">1.4</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_16170</italic></td><td valign="bottom"><italic>sif-like</italic></td><td valign="bottom"><italic>D. melanogaster</italic></td><td valign="bottom">Guanine nucleotide exchange factor</td><td valign="bottom">1.5</td><td valign="bottom">0.9</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_16644</italic></td><td valign="bottom"><italic>BMH1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">14-3-3 protein</td><td valign="bottom">1.3</td><td valign="bottom">2.2</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_16068</italic></td><td valign="bottom"><italic>BMH1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">14-3-3 protein</td><td valign="bottom">0.7</td><td valign="bottom">1.2</td></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Kinases and Phosphatases</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_13246</italic></td><td valign="bottom"><italic>CNA1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Phosphatase (Calcineurin catalytic subunit)</td><td valign="bottom">0.8</td><td valign="bottom">1.2</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_11675</italic></td><td valign="bottom"><italic>CNB1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Phosphatase (Calcineurin regulatory subunit)</td><td valign="bottom">1.1</td><td valign="bottom">1.2</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_11667</italic></td><td valign="bottom"><italic>PTC1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Phosphatase</td><td valign="bottom">0.9</td><td valign="bottom">1.2</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_10638</italic></td><td valign="bottom"><italic>CLA4</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Kinase</td><td valign="bottom">3.4</td><td valign="bottom">4.5</td></tr><tr><td valign="bottom">&#8195;*</td><td valign="bottom"><italic>RTO4_16605</italic></td><td valign="bottom"><italic>TPK1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Kinase</td><td valign="bottom">1.1</td><td valign="bottom">0.5</td></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="2"><bold>Gene Expresssion</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_10333</italic></td><td valign="bottom"><italic>SET1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Chromatin modifying</td><td valign="bottom">3.0</td><td valign="bottom">1.1</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_10279</italic></td><td valign="bottom"><italic>BRE2</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Chromatin modifying</td><td valign="bottom">2.5</td><td valign="bottom">1.0</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_12689</italic></td><td valign="bottom"><italic>SPP1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Chromatin modifying</td><td valign="bottom">2.0</td><td valign="bottom">1.3</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_15412</italic></td><td valign="bottom"><italic>RCO1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Chromatin modifying</td><td valign="bottom">3.5</td><td valign="bottom">1.6</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_10209</italic></td><td valign="bottom"><italic>MIT1-like</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Transcripition factor</td><td valign="bottom">1.4</td><td valign="bottom">0.3</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_14550</italic></td><td valign="bottom"><italic>CYC8</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Transcription factor</td><td valign="bottom">3.7</td><td valign="bottom">3.8</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_10274</italic></td><td valign="bottom"><italic>SKN7-like</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Transcription factor</td><td valign="bottom">2.2</td><td valign="bottom">1.5</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_13346</italic></td><td valign="bottom"><italic>CBC2</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">RNA splicing factor</td><td valign="bottom">1.6</td><td valign="bottom">1.2</td></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Protein Modification</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_11272</italic></td><td valign="bottom"><italic>ALG12</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Alpha-1,6-mannosyltransferase</td><td valign="bottom">3.5</td><td valign="bottom">1.7</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_14881</italic></td><td valign="bottom"><italic>CAP10-like</italic></td><td valign="bottom"><italic>C. neoformans</italic></td><td valign="bottom">Xylosyltransferase</td><td valign="bottom">1.5</td><td valign="bottom">2.0</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_16598</italic></td><td valign="bottom"><italic>LARGE1</italic></td><td valign="bottom"><italic>H. sapiens</italic></td><td valign="bottom">N-acetylglucosaminyltransferase-like protein</td><td valign="bottom">1.8</td><td valign="bottom">1.3</td></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Protein Trafficking</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_12145</italic></td><td valign="bottom"><italic>ERP1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">COPII cargo adapter protein (p24 family)</td><td valign="bottom">2.4</td><td valign="bottom">2.7</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_16731</italic></td><td valign="bottom"><italic>ERP2</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">COPII cargo adapter protein (p24 family)</td><td valign="bottom">1.7</td><td valign="bottom">2.0</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_12521</italic></td><td valign="bottom"><italic>EMP24</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">COPII cargo adapter protein (p24 family)</td><td valign="bottom">1.9</td><td valign="bottom">2.4</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_14054</italic></td><td valign="bottom"><italic>BST1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">GPI inositol-deacylase</td><td valign="bottom">1.5</td><td valign="bottom">0.2</td></tr><tr><td valign="bottom">&#8195;*</td><td valign="bottom"><italic>RTO4_15883</italic></td><td valign="bottom"><italic>RAS1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">GTPase</td><td valign="bottom">2.0</td><td valign="bottom">2.3</td></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Other ER/Golgi Proteins</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_10371</italic></td><td valign="bottom"><italic>KDELC1-like</italic></td><td valign="bottom"><italic>H. sapiens</italic></td><td valign="bottom">Endoplasmic reticulum protein EP58</td><td valign="bottom">3.1</td><td valign="bottom">6.0</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_15763</italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">SH3 Domain-containing ER Protein</td><td valign="bottom">1.0</td><td valign="bottom">1.5</td></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Amino Acid Biosynthesis</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_11050</italic></td><td valign="bottom"><italic>MET1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Uroporphyrinogen III transmethylase</td><td valign="bottom">3.8</td><td valign="bottom">2.0</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_8744</italic></td><td valign="bottom"><italic>MET5</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Sulfite reductase</td><td valign="bottom">4.4</td><td valign="bottom">2.1</td></tr><tr><td valign="bottom">&#8195;&#167;</td><td valign="bottom"><italic>RTO4_10374</italic></td><td valign="bottom"><italic>MET10</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Sulfite reductase</td><td valign="bottom">2.5</td><td valign="bottom">1.3</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_8709</italic></td><td valign="bottom"><italic>MET14</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Adenylylsulfate kinase</td><td valign="bottom">4.1</td><td valign="bottom">1.1</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_11741</italic></td><td valign="bottom"><italic>MET16</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Phosphoadenosine phosphosulfate reductase</td><td valign="bottom">1.7</td><td valign="bottom">1.1</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_12031</italic></td><td valign="bottom"><italic>cysB</italic></td><td valign="bottom"><italic>A. nidulans</italic></td><td valign="bottom">Cysteine synthase A</td><td valign="bottom">3.3</td><td valign="bottom">2.1</td></tr><tr><td valign="bottom">&#8195;*</td><td valign="bottom"><italic>RTO4_16196</italic></td><td valign="bottom"><italic>ARG1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Argininosuccinate synthase</td><td valign="bottom">1.3</td><td valign="bottom">1.8</td></tr><tr><td valign="bottom"/><td/><td/><td/><td/><td/><td/></tr><tr><td colspan="2"><bold>Translation</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_12273</italic></td><td valign="bottom"><italic>MRN1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">RNA-binding protein</td><td valign="bottom">2.5</td><td valign="bottom">1.6</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_8595</italic></td><td valign="bottom"><italic>EIF4E2</italic></td><td valign="bottom"><italic>H. sapiens</italic></td><td valign="bottom">Translation initiation factor</td><td valign="bottom">2.0</td><td valign="bottom">0.5</td></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Ubiquitination and Proteolysis</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_11150</italic></td><td valign="bottom"><italic>Mub1-like</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Ubiquitin ligase complex member</td><td valign="bottom">3.8</td><td valign="bottom">2.0</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_15576</italic></td><td valign="bottom"><italic>CDC4</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Ubiquitin ligase complex member</td><td valign="bottom">1.7</td><td valign="bottom">1.8</td></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Triacylglyceride Synthesis</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">&#8195;&#8224;&#160;</td><td valign="bottom"><italic>RTO4_8972</italic></td><td valign="bottom"><italic>NDE1</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">NADH dehydrogenase</td><td valign="bottom">1.6</td><td valign="bottom">1.9</td></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Lipid Droplet Associated</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_14088</italic></td><td valign="bottom"><italic>RAC1</italic></td><td valign="bottom"><italic>H. sapiens</italic></td><td valign="bottom">GTPase</td><td valign="bottom">2.0</td><td valign="bottom">0.9</td></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Mitochondrial Beta-oxidation</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_16284</italic></td><td valign="bottom"><italic>HSD17B10</italic></td><td valign="bottom"><italic>H. sapiens</italic></td><td valign="bottom">3-hydroxyacyl-CoA dehydrogenase</td><td valign="bottom">1.6</td><td valign="bottom">0.5</td></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="2"><bold>Other</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_12175</italic></td><td valign="bottom"><italic>mesA</italic></td><td valign="bottom"><italic>A. nidulans</italic></td><td valign="bottom">Myosin binding protein</td><td valign="bottom">1.3</td><td valign="bottom">1.8</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_8401</italic></td><td valign="bottom"><italic>SHE4</italic></td><td valign="bottom"><italic>S. cerevisiae</italic></td><td valign="bottom">Transmembrane protein involved in cell polarity</td><td valign="bottom">1.0</td><td valign="bottom">1.3</td></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Unknown Function</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_16524</italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">Protein of unknown function</td><td valign="bottom">3.1</td><td valign="bottom">1.9</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_11613</italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">Protein of unknown function</td><td valign="bottom">2.5</td><td valign="bottom">1.7</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_12505</italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">Protein of unknown function</td><td valign="bottom">2.1</td><td valign="bottom">2.1</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_13512</italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">Protein of unknown function</td><td valign="bottom">1.5</td><td valign="bottom">1.9</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_10805</italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">Protein of unknown function</td><td valign="bottom">1.2</td><td valign="bottom">1.8</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_15251</italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">Protein of unknown function</td><td valign="bottom">1.6</td><td valign="bottom">1.3</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_15358</italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">Protein of unknown function</td><td valign="bottom">2.0</td><td valign="bottom">0.5</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_13513</italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">Protein of unknown function</td><td valign="bottom">1.3</td><td valign="bottom">1.2</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_12461</italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">Protein of unknown function</td><td valign="bottom">1.5</td><td valign="bottom">0.8</td></tr><tr><td valign="bottom"/><td valign="bottom"><italic>RTO4_13351</italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">Protein of unknown function</td><td valign="bottom">1.2</td><td valign="bottom">1.0</td></tr></tbody></table><table-wrap-foot><fn><p>Cellular processes grouped as in <xref ref-type="fig" rid="fig6">Figure 6</xref>. BD: Enrichment score from buoyant density separation. FACS: Enrichment score from fluorescence activated cell sorting.</p><p>Protein abundance under nitrogen limitation: * increased; &#8224; increased 10-fold or more; &#8225; decreased; &#167; decreased 10-fold or more (<xref ref-type="bibr" rid="bib212">Zhu et al., 2012</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32110-v2.xml">elife-32110-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.32110.023</object-id><label>Table 2.</label><caption><title>Predicted gene function: Mutants with decreased lipid accumulation.</title><p>Predicted functions for genes for which mutants were high-confidence candidates for decreased lipid accumulation (enrichment scores clustered in LA6 - LA8, <xref ref-type="fig" rid="fig5">Figure 5</xref>).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom"/><th rowspan="2">Gene ID</th><th rowspan="2">Short <break/>name</th><th rowspan="2">Annotation <break/>from</th><th rowspan="2">Description</th><th rowspan="2">Cluster</th><th colspan="2" valign="bottom">Enrichment</th></tr><tr><th>BD</th><th>FACS</th></tr></thead><tbody><tr><td colspan="2"><bold>tRNA thiolation</bold></td><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>RTO4_10764</italic></td><td><italic>NCS2</italic></td><td><italic>S. cerevisiae</italic></td><td>tRNA 2-thiolation protein</td><td>LA7</td><td valign="bottom">0.5</td><td valign="bottom">&#8722;2.3</td></tr><tr><td/><td><italic>RTO4_12817</italic></td><td><italic>NCS6</italic></td><td><italic>S. cerevisiae</italic></td><td>tRNA 2-thiolation protein</td><td>LA7</td><td valign="bottom">0.7</td><td valign="bottom">&#8722;2.6</td></tr><tr><td/><td><italic>RTO4_14918</italic></td><td><italic>ELP2</italic></td><td><italic>S. cerevisiae</italic></td><td>Elongator complex protein</td><td>LA7</td><td valign="bottom">0.7</td><td valign="bottom">&#8722;1.2</td></tr><tr><td/><td><italic>RTO4_14716</italic></td><td><italic>IKI3</italic></td><td><italic>S. cerevisiae</italic></td><td>Elongator complex protein</td><td>LA7</td><td valign="bottom">0.4</td><td valign="bottom">&#8722;1.1</td></tr><tr><td/><td><italic>RTO4_11341</italic></td><td><italic>UBA4</italic></td><td><italic>S. cerevisiae</italic></td><td>Adenylyltransferase and sulfurtransferase</td><td>LA7</td><td valign="bottom">0.6</td><td valign="bottom">&#8722;2.6</td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td colspan="3"><bold>G Protein Switches</bold></td><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8224;</td><td><italic>RTO4_15198</italic></td><td><italic>Rab6</italic></td><td><italic>H. sapiens</italic></td><td>GTPase</td><td>LA6</td><td valign="bottom">&#8722;1.3</td><td valign="bottom">&#8722;1.6</td></tr><tr><td/><td><italic>RTO4_14622</italic></td><td><italic>RGP1</italic></td><td><italic>H. sapiens</italic></td><td>Guanine nucleotide exchange factor</td><td>LA6</td><td valign="bottom">&#8722;1.4</td><td valign="bottom">&#8722;1.5</td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td colspan="3"><bold>Kinases and Phosphatases</bold></td><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>RTO4_10698</italic></td><td><italic>VHS1</italic></td><td><italic>S. cerevisiae</italic></td><td>Kinase</td><td>LA6</td><td valign="bottom">0.8</td><td valign="bottom">&#8722;3.7</td></tr><tr><td/><td><italic>RTO4_16375</italic></td><td><italic>HRK1</italic></td><td><italic>S. cerevisiae</italic></td><td>Kinase</td><td>LA6</td><td valign="bottom">0.4</td><td valign="bottom">&#8722;2.2</td></tr><tr><td>&#8195;*</td><td><italic>RTO4_11453</italic></td><td><italic>GLC7</italic></td><td><italic>S. cerevisiae</italic></td><td>Kinase</td><td>LA8</td><td valign="bottom">&#8722;1.2</td><td valign="bottom">&#8722;0.9</td></tr><tr><td/><td><italic>RTO4_16810</italic></td><td><italic>KIN1</italic></td><td><italic>S. cerevisiae</italic></td><td>Kinase</td><td>LA6</td><td valign="bottom">0.1</td><td valign="bottom">&#8722;1.1</td></tr><tr><td/><td><italic>RTO4_10025</italic></td><td><italic>SAT4</italic></td><td><italic>S. cerevisiae</italic></td><td>Kinase</td><td>LA7</td><td valign="bottom">1.6</td><td valign="bottom">&#8722;3.6</td></tr><tr><td/><td><italic>RTO4_13327</italic></td><td><italic>ATG1</italic></td><td><italic>S. cerevisiae</italic></td><td>Kinase</td><td>LA6</td><td valign="bottom">0.1</td><td valign="bottom">&#8722;2.5</td></tr><tr><td/><td><italic>RTO4_14907</italic></td><td><italic>SCH9</italic></td><td><italic>S. cerevisiae</italic></td><td>Kinase</td><td>LA6</td><td valign="bottom">&#8722;0.6</td><td valign="bottom">&#8722;2.0</td></tr><tr><td/><td><italic>RTO4_14906</italic></td><td><italic>kinase-like</italic></td><td><italic>S. cerevisiae</italic></td><td>Kinase</td><td>LA6</td><td valign="bottom">&#8722;0.3</td><td valign="bottom">&#8722;1.8</td></tr><tr><td/><td><italic>RTO4_13290</italic></td><td><italic>YAK1</italic></td><td><italic>S. cerevisiae</italic></td><td>Kinase</td><td>LA8</td><td valign="bottom">&#8722;1.1</td><td valign="bottom">&#8722;0.9</td></tr><tr><td/><td><italic>RTO4_11732</italic></td><td><italic>PPH3</italic></td><td><italic>S. cerevisiae</italic></td><td>Phosphatase&#160;4 catalytic subunit</td><td>LA6</td><td valign="bottom">0.9</td><td valign="bottom">&#8722;3.6</td></tr><tr><td/><td><italic>RTO4_12586</italic></td><td><italic>PSY2</italic></td><td><italic>S. cerevisiae</italic></td><td>Phosphatase&#160;4 regulatory subunit</td><td>LA6</td><td valign="bottom">0.2</td><td valign="bottom">&#8722;1.2</td></tr><tr><td/><td><italic>RTO4_16463</italic></td><td><italic>PTC7-like</italic></td><td><italic>S. cerevisiae</italic></td><td>Phosphatase</td><td>LA6</td><td valign="bottom">0.1</td><td valign="bottom">&#8722;2.0</td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td colspan="2"><bold>Autophagy</bold></td><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>RTO4_13327</italic></td><td><italic>ATG1</italic></td><td><italic>S. cerevisiae</italic></td><td>Kinase</td><td>LA6</td><td valign="bottom">0.1</td><td valign="bottom">&#8722;2.5</td></tr><tr><td/><td><italic>RTO4_13598</italic></td><td><italic>ATG2</italic></td><td><italic>S. cerevisiae</italic></td><td>Membrane protein</td><td>LA6</td><td valign="bottom">&#8722;0.6</td><td valign="bottom">&#8722;3.4</td></tr><tr><td/><td><italic>RTO4_12968</italic></td><td><italic>ATG3</italic></td><td><italic>S. cerevisiae</italic></td><td>Ubiquitin-like-conjugating enzyme</td><td>LA6</td><td valign="bottom">&#8722;0.8</td><td valign="bottom">&#8722;4.5</td></tr><tr><td/><td><italic>RTO4_13496</italic></td><td><italic>ATG4</italic></td><td><italic>S. cerevisiae</italic></td><td>Cysteine protease</td><td>LA6</td><td valign="bottom">&#8722;0.1</td><td valign="bottom">&#8722;2.3</td></tr><tr><td/><td><italic>RTO4_11901</italic></td><td><italic>ATG7</italic></td><td><italic>S. cerevisiae</italic></td><td>Ubiquitin-like modifier-activating enzyme</td><td>LA6</td><td valign="bottom">&#8722;0.8</td><td valign="bottom">&#8722;4.2</td></tr><tr><td/><td><italic>RTO4_13543</italic></td><td><italic>ATG8</italic></td><td><italic>S. cerevisiae</italic></td><td>Ubiquitin-like protein</td><td>LA6</td><td valign="bottom">&#8722;1.0</td><td valign="bottom">&#8722;4.2</td></tr><tr><td/><td><italic>RTO4_11326</italic></td><td><italic>ATG9</italic></td><td><italic>S. cerevisiae</italic></td><td>Membrane protein</td><td>LA6</td><td valign="bottom">0.0</td><td valign="bottom">&#8722;1.3</td></tr><tr><td/><td><italic>RTO4_9008</italic></td><td><italic>ATG14</italic></td><td><italic>S. cerevisiae</italic></td><td>Autophagy-specific subunit of PtdIns3P-kinase complex</td><td>LA6</td><td valign="bottom">0.0</td><td valign="bottom">&#8722;5.0</td></tr><tr><td/><td><italic>RTO4_16723</italic></td><td><italic>ATG18</italic></td><td><italic>S. cerevisiae</italic></td><td>Phosphoinositide binding protein</td><td>LA6</td><td valign="bottom">&#8722;0.9</td><td valign="bottom">&#8722;5.8</td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td colspan="3"><bold>Ubiquitination and Proteolysis</bold></td><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8224;</td><td><italic>RTO4_16672</italic></td><td><italic>PRB1</italic></td><td><italic>S. cerevisiae</italic></td><td>Vacuolar proteinase</td><td>LA6</td><td valign="bottom">&#8722;0.2</td><td valign="bottom">&#8722;1.7</td></tr><tr><td/><td><italic>RTO4_15345</italic></td><td><italic>SIS1</italic></td><td><italic>S. cerevisiae</italic></td><td>Protein chaperone</td><td>LA6</td><td valign="bottom">&#8722;0.4</td><td valign="bottom">&#8722;1.2</td></tr><tr><td/><td><italic>RTO4_10423</italic></td><td><italic>RMD5</italic></td><td><italic>S. cerevisiae</italic></td><td>GID complex E3 ubiquitin ligase</td><td>LA6</td><td valign="bottom">&#8722;0.4</td><td valign="bottom">&#8722;2.0</td></tr><tr><td/><td><italic>RTO4_11737</italic></td><td><italic>GID8</italic></td><td><italic>H. sapiens</italic></td><td>GID complex member</td><td>LA6</td><td valign="bottom">&#8722;0.1</td><td valign="bottom">&#8722;1.5</td></tr><tr><td/><td><italic>RTO4_9816</italic></td><td><italic>LONRF1</italic></td><td><italic>H. sapiens</italic></td><td>E3 ubiquitin ligase</td><td>LA6</td><td valign="bottom">&#8722;0.5</td><td valign="bottom">&#8722;4.5</td></tr><tr><td/><td><italic>RTO4_15320</italic></td><td><italic>USP48</italic></td><td><italic>H. sapiens</italic></td><td>Ubiquitin carboxyl-terminal hydrolase</td><td>LA6</td><td valign="bottom">0.0</td><td valign="bottom">&#8722;1.2</td></tr><tr><td/><td><italic>RTO4_9600</italic></td><td><italic>COPS3</italic></td><td><italic>H. sapiens</italic></td><td>COP9 signalosome complex subunit</td><td>LA1</td><td valign="bottom">1.4</td><td valign="bottom">0.6</td></tr><tr><td/><td><italic>RTO4_11569</italic></td><td><italic>GPS1</italic></td><td><italic>H. sapiens</italic></td><td>COP9 signalosome complex subunit</td><td>LA6</td><td valign="bottom">0.7</td><td valign="bottom">&#8722;2.1</td></tr><tr><td colspan="3"><bold>Triacylglyceride Synthesis</bold></td><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8224;</td><td><italic>RTO4_12154</italic></td><td><italic>GPD1</italic></td><td><italic>S. cerevisiae</italic></td><td>Glycerol-3-phosphate dehydrogenase</td><td>LA6</td><td valign="bottom">&#8722;1.7</td><td valign="bottom">&#8722;4.0</td></tr><tr><td/><td><italic>RTO4_11043</italic></td><td><italic>BCSL2-like</italic></td><td><italic>H. sapiens</italic></td><td>Seipin</td><td>LA6</td><td valign="bottom">&#8722;0.8</td><td valign="bottom">&#8722;2.9</td></tr><tr><td/><td><italic>RTO4_16460</italic></td><td><italic>DGA1</italic></td><td><italic>H. sapiens</italic></td><td>Diacylglycerol acyltransferase</td><td>LA6</td><td valign="bottom">&#8722;0.7</td><td valign="bottom">&#8722;4.0</td></tr><tr><td/><td><italic>RTO4_14597</italic></td><td><italic>ACS1</italic></td><td><italic>S. cerevisiae</italic></td><td>Acetyl-CoA synthetase</td><td>LA8</td><td valign="bottom">&#8722;1.7</td><td valign="bottom">&#8722;1.0</td></tr><tr><td/><td><italic>RTO4_10182</italic></td><td><italic>YEF1</italic></td><td><italic>S. cerevisiae</italic></td><td>NAD+/NADH&#160;kinase</td><td>LA6</td><td valign="bottom">&#8722;0.1</td><td valign="bottom">&#8722;1.6</td></tr><tr><td>&#8195;&#8225;</td><td><italic>RTO4_11039</italic></td><td><italic>GUT2</italic></td><td><italic>S. cerevisiae</italic></td><td>Glycerol-3-phosphate dehydrogenase</td><td>LA6</td><td valign="bottom">&#8722;0.2</td><td valign="bottom">&#8722;1.1</td></tr><tr><td/><td/><td/><td/><td/><td/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Lipid Droplet Associated</bold></td><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>RTO4_16381</italic></td><td><italic>PLIN1-like</italic></td><td><italic>S. cerevisiae</italic></td><td>Perilipin</td><td>LA6</td><td valign="bottom">&#8722;1.7</td><td valign="bottom">&#8722;4.3</td></tr><tr><td>&#8195;&#8225;</td><td><italic>RTO4_11039</italic></td><td><italic>GUT2</italic></td><td><italic>S. cerevisiae</italic></td><td>Glycerol-3-phosphate dehydrogenase</td><td>LA6</td><td valign="bottom">&#8722;0.2</td><td valign="bottom">&#8722;1.1</td></tr><tr><td/><td><italic>RTO4_15372</italic></td><td><italic>EGH1</italic></td><td><italic>S. cerevisiae</italic></td><td>Steryl-beta-glucosidase</td><td>LA6</td><td valign="bottom">0.7</td><td valign="bottom">&#8722;2.5</td></tr><tr><td/><td><italic>RTO4_13614</italic></td><td><italic>RIP1</italic></td><td><italic>S. cerevisiae</italic></td><td>Mitochondrial complex III iron-sulfur protein</td><td>LA6</td><td valign="bottom">&#8722;0.5</td><td valign="bottom">&#8722;2.8</td></tr><tr><td/><td><italic>RTO4_11043</italic></td><td><italic>BCSL2-like</italic></td><td><italic>H. sapiens</italic></td><td>Seipin</td><td>LA6</td><td valign="bottom">&#8722;0.8</td><td valign="bottom">&#8722;2.9</td></tr><tr><td/><td><italic>RTO4_16460</italic></td><td><italic>DGA1</italic></td><td><italic>H. sapiens</italic></td><td>Diacylglycerol acyltransferase</td><td>LA6</td><td valign="bottom">&#8722;0.7</td><td valign="bottom">&#8722;4.0</td></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Protein Modification</bold></td><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>RTO4_12670</italic></td><td><italic>B3GALT1-like</italic></td><td><italic>H. sapiens</italic></td><td>Beta-1,3-Galactosyltransferase</td><td>LA6</td><td valign="bottom">&#8722;0.9</td><td valign="bottom">&#8722;3.1</td></tr><tr><td/><td/><td/><td/><td/><td/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Protein Trafficking</bold></td><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8224;</td><td><italic>RTO4_15198</italic></td><td><italic>Rab6</italic></td><td><italic>H. sapiens</italic></td><td>GTPase</td><td>LA6</td><td valign="bottom">&#8722;1.3</td><td valign="bottom">&#8722;1.6</td></tr><tr><td/><td/><td/><td/><td/><td/><td valign="bottom"/><td valign="bottom"/></tr><tr><td colspan="3"><bold>Other ER/Golgi Proteins</bold></td><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>RTO4_8838</italic></td><td><italic>DNAJC4</italic></td><td><italic>H. sapiens</italic></td><td>DnaJ family chaperone</td><td>LA6</td><td valign="bottom">&#8722;0.8</td><td valign="bottom">&#8722;1.3</td></tr><tr><td/><td><italic>RTO4_13971</italic></td><td><italic>DNAJC3</italic></td><td><italic>H. sapiens</italic></td><td>DnaJ family chaperone</td><td>LA6</td><td valign="bottom">&#8722;1.1</td><td valign="bottom">&#8722;2.2</td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td colspan="2"><bold>Gene Expression</bold></td><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>RTO4_11333</italic></td><td><italic>KLF18-like</italic></td><td><italic>H. sapiens</italic></td><td>Transcription factor</td><td>LA6</td><td valign="bottom">&#8722;0.2</td><td valign="bottom">&#8722;1.1</td></tr><tr><td/><td><italic>RTO4_15641</italic></td><td><italic>SKN7</italic></td><td><italic>S. cerevisiae</italic></td><td>Transcription factor</td><td>LA6</td><td valign="bottom">0.9</td><td valign="bottom">&#8722;2.9</td></tr><tr><td/><td><italic>RTO4_14676</italic></td><td><italic>LHX5-like</italic></td><td><italic>H. sapiens</italic></td><td>Transcription factor</td><td>LA6</td><td valign="bottom">&#8722;0.2</td><td valign="bottom">&#8722;2.8</td></tr><tr><td/><td><italic>RTO4_11891</italic></td><td><italic>HAP2</italic></td><td><italic>S. cerevisiae</italic></td><td>Transcription factor</td><td>LA6</td><td valign="bottom">&#8722;0.8</td><td valign="bottom">&#8722;2.4</td></tr><tr><td/><td><italic>RTO4_12420</italic></td><td><italic>OPI1-like</italic></td><td><italic>S. cerevisiae</italic></td><td>Transcription factor</td><td>LA6</td><td valign="bottom">0.0</td><td valign="bottom">&#8722;3.7</td></tr><tr><td/><td><italic>RTO4_14100</italic></td><td><italic>HAPX</italic></td><td><italic>C. neoformans</italic></td><td>Transcription factor</td><td>LA8</td><td valign="bottom">&#8722;1.2</td><td valign="bottom">&#8722;1.7</td></tr><tr><td/><td><italic>RTO4_13255</italic></td><td><italic>SGF73</italic></td><td><italic>S. cerevisiae</italic></td><td>SAGA-associated factor</td><td>LA6</td><td valign="bottom">0.4</td><td valign="bottom">&#8722;1.5</td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td colspan="3"><bold>Methylcitrate Cycle</bold></td><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>RTO4_14162</italic></td><td><italic>ICL2</italic></td><td><italic>S. cerevisiae</italic></td><td>2-methylisocitrate lyase</td><td>LA6</td><td valign="bottom">&#8722;0.3</td><td valign="bottom">&#8722;1.8</td></tr><tr><td/><td><italic>RTO4_12642</italic></td><td><italic>PDH1</italic></td><td><italic>S. cerevisiae</italic></td><td>2-methylcitrate dehydratase</td><td>LA6</td><td valign="bottom">&#8722;0.1</td><td valign="bottom">&#8722;1.7</td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td colspan="4"><bold>Electron Transport and Redox Balancing</bold></td><td/><td/><td/><td/></tr><tr><td/><td><italic>RTO4_11165</italic></td><td><italic>CBP4</italic></td><td><italic>S. cerevisiae</italic></td><td>Mitochondrial complex III assembly factor</td><td>LA6</td><td valign="bottom">&#8722;0.4</td><td valign="bottom">&#8722;2.5</td></tr><tr><td/><td><italic>RTO4_13614</italic></td><td><italic>RIP1</italic></td><td><italic>S. cerevisiae</italic></td><td>Mitochondrial complex III iron-sulfur protein</td><td>LA6</td><td valign="bottom">&#8722;0.5</td><td valign="bottom">&#8722;2.8</td></tr><tr><td/><td><italic>RTO4_13902</italic></td><td><italic>AFG1</italic></td><td><italic>S. cerevisiae</italic></td><td>Mitochondrial complex IV assembly factor</td><td>LA6</td><td valign="bottom">&#8722;0.3</td><td valign="bottom">&#8722;1.3</td></tr><tr><td>&#8195;&#8225;</td><td><italic>RTO4_10010</italic></td><td><italic>NDUFS4</italic></td><td><italic>H. sapiens</italic></td><td>Mitochondrial complex I accessory factor</td><td>LA8</td><td valign="bottom">&#8722;1.3</td><td valign="bottom">&#8722;0.1</td></tr><tr><td/><td><italic>RTO4_13925</italic></td><td><italic>NDUFAF3</italic></td><td><italic>H. sapiens</italic></td><td>Mitochondrial complex I assembly factor</td><td>LA8</td><td valign="bottom">&#8722;1.0</td><td valign="bottom">&#8722;1.6</td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td colspan="3"><bold>Amino Acid Biosynthesis</bold></td><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8224;</td><td><italic>RTO4_12302</italic></td><td><italic>CPA2</italic></td><td><italic>S. cerevisiae</italic></td><td>Large subunit of carbamoyl phosphate synthetase</td><td>LA6</td><td valign="bottom">&#8722;0.4</td><td valign="bottom">&#8722;2.4</td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td colspan="3"><bold>Glucose and Energy Metabolism</bold></td><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>RTO4_10423</italic></td><td><italic>RMD5</italic></td><td><italic>S. cerevisiae</italic></td><td>GID complex E3 ubiquitin ligase</td><td>LA6</td><td valign="bottom">&#8722;0.4</td><td valign="bottom">&#8722;2.0</td></tr><tr><td/><td><italic>RTO4_11737</italic></td><td><italic>GID8</italic></td><td><italic>H. sapiens</italic></td><td>GID complex member</td><td>LA6</td><td valign="bottom">&#8722;0.1</td><td valign="bottom">&#8722;1.5</td></tr><tr><td/><td><italic>RTO4_12034</italic></td><td><italic>TPS2</italic></td><td><italic>S. cerevisiae</italic></td><td>Trehalose 6-phosphate synthase</td><td>LA6</td><td valign="bottom">0.0</td><td valign="bottom">&#8722;3.8</td></tr><tr><td>&#8195;*</td><td><italic>RTO4_10264</italic></td><td><italic>GLK1</italic></td><td><italic>S. cerevisiae</italic></td><td>Hexokinase</td><td>LA7</td><td valign="bottom">2.1</td><td valign="bottom">&#8722;2.0</td></tr><tr><td colspan="2"><bold>Transporters</bold></td><td/><td/><td/><td/><td/><td/></tr><tr><td>&#8195;&#8224;</td><td><italic>RTO4_12909</italic></td><td><italic>OAT1</italic></td><td><italic>C. neoformans</italic></td><td>Nucleobase transporter</td><td>LA6</td><td valign="bottom">&#8722;0.2</td><td valign="bottom">&#8722;1.1</td></tr><tr><td/><td><italic>RTO4_11397</italic></td><td><italic>COT1</italic></td><td><italic>S. cerevisiae</italic></td><td>Vacuolar zinc transporter</td><td>LA6</td><td valign="bottom">&#8722;0.2</td><td valign="bottom">&#8722;1.1</td></tr><tr><td/><td><italic>RTO4_11924</italic></td><td><italic>SNF3</italic></td><td><italic>S. cerevisiae</italic></td><td>Plasma membrane low glucose sensor</td><td>LA6</td><td valign="bottom">0.0</td><td valign="bottom">&#8722;2.8</td></tr><tr><td valign="bottom"/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td colspan="2"><bold>Other</bold></td><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>RTO4_12512</italic></td><td><italic>cry</italic></td><td><italic>N. crassa</italic></td><td>Blue-light photoreceptor cryptochrome</td><td>LA7</td><td valign="bottom">0.6</td><td valign="bottom">&#8722;1.6</td></tr><tr><td/><td><italic>RTO4_14974</italic></td><td/><td/><td>Steroidogenesis/phosphatidylcholine transfer domain</td><td>LA6</td><td valign="bottom">&#8722;0.3</td><td valign="bottom">&#8722;1.2</td></tr><tr><td/><td><italic>RTO4_15889</italic></td><td><italic>MAEA</italic></td><td><italic>H. sapiens</italic></td><td>EMP macrophage erythroblast attacher</td><td>LA6</td><td valign="bottom">&#8722;0.1</td><td valign="bottom">&#8722;1.7</td></tr><tr><td/><td><italic>RTO4_16287</italic></td><td><italic>CDD1</italic></td><td><italic>S. cerevisiae</italic></td><td>Cytidine deaminase</td><td>LA6</td><td valign="bottom">0.3</td><td valign="bottom">&#8722;2.3</td></tr><tr><td/><td><italic>RTO4_15247</italic></td><td><italic>WDR26</italic></td><td><italic>H. sapiens</italic></td><td>WD repeat protein</td><td>LA6</td><td valign="bottom">&#8722;0.9</td><td valign="bottom">&#8722;1.3</td></tr><tr><td/><td><italic>RTO4_8764</italic></td><td><italic>MGS1</italic></td><td><italic>S. cerevisiae</italic></td><td>DNA-dependent ATPase and ssDNA annealing protein</td><td>LA6</td><td valign="bottom">0.2</td><td valign="bottom">&#8722;1.2</td></tr><tr><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td colspan="2"><bold>Unknown</bold></td><td/><td/><td/><td/><td/><td/></tr><tr><td/><td><italic>RTO4_10431</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">0.7</td><td valign="bottom">&#8722;1.6</td></tr><tr><td/><td><italic>RTO4_8973</italic></td><td/><td/><td>Protein of unknown function</td><td>LA8</td><td valign="bottom">&#8722;0.2</td><td valign="bottom">&#8722;1.1</td></tr><tr><td/><td><italic>RTO4_13195</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">&#8722;0.2</td><td valign="bottom">&#8722;1.1</td></tr><tr><td/><td><italic>RTO4_10367</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">&#8722;0.1</td><td valign="bottom">&#8722;1.3</td></tr><tr><td/><td><italic>RTO4_10102</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">&#8722;0.3</td><td valign="bottom">&#8722;1.2</td></tr><tr><td/><td><italic>RTO4_14926</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">0.2</td><td valign="bottom">&#8722;1.7</td></tr><tr><td/><td><italic>RTO4_12045</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">0.0</td><td valign="bottom">&#8722;1.5</td></tr><tr><td/><td><italic>RTO4_13600</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">&#8722;0.3</td><td valign="bottom">&#8722;1.3</td></tr><tr><td/><td><italic>RTO4_10976</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">&#8722;0.2</td><td valign="bottom">&#8722;1.5</td></tr><tr><td/><td><italic>RTO4_9970</italic></td><td><italic>LDB17</italic></td><td><italic>S. cerevisiae</italic></td><td>Protein of unknown function</td><td>LA8</td><td valign="bottom">&#8722;1.3</td><td valign="bottom">&#8722;0.5</td></tr><tr><td/><td><italic>RTO4_13435</italic></td><td/><td/><td>Protein of unknown function</td><td>LA7</td><td valign="bottom">0.2</td><td valign="bottom">&#8722;2.0</td></tr><tr><td/><td><italic>RTO4_9692</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">&#8722;0.5</td><td valign="bottom">&#8722;1.4</td></tr><tr><td/><td><italic>RTO4_15521</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">0.2</td><td valign="bottom">&#8722;2.2</td></tr><tr><td/><td><italic>RTO4_8769</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">&#8722;0.5</td><td valign="bottom">&#8722;1.6</td></tr><tr><td/><td><italic>RTO4_8770</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">&#8722;0.5</td><td valign="bottom">&#8722;1.9</td></tr><tr><td/><td><italic>RTO4_11259</italic></td><td/><td/><td>Protein of unknown function</td><td>LA7</td><td valign="bottom">0.7</td><td valign="bottom">&#8722;3.3</td></tr><tr><td/><td><italic>RTO4_9490</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">&#8722;0.6</td><td valign="bottom">&#8722;2.4</td></tr><tr><td/><td><italic>RTO4_15520</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">&#8722;0.5</td><td valign="bottom">&#8722;2.5</td></tr><tr><td/><td><italic>RTO4_8771</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">&#8722;0.6</td><td valign="bottom">&#8722;2.5</td></tr><tr><td/><td><italic>RTO4_13452</italic></td><td/><td/><td>Protein of unknown function</td><td>LA6</td><td valign="bottom">&#8722;1.3</td><td valign="bottom">&#8722;4.0</td></tr><tr><td/><td><italic>RTO4_15211</italic></td><td/><td/><td>Protein of unknown function</td><td>LA8</td><td valign="bottom">&#8722;1.1</td><td valign="bottom">&#8722;1.5</td></tr></tbody></table><table-wrap-foot><fn><p>Cellular processes grouped as in <xref ref-type="fig" rid="fig6">Figure 6</xref>. BD: Enrichment score from buoyant density separation. FACS: Enrichment score from fluorescence activated cell sorting.</p><p>Protein abundance under nitrogen limitation: * increased; &#8224; increased 10-fold or more; &#8225; decreased; &#167; decreased 10-fold or more (<xref ref-type="bibr" rid="bib212">Zhu et al., 2012</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32208-v2.xml">elife-32208-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.32208.004</object-id><label>Table 1.</label><caption><title>Associations with death in 817 subjects with admission retinal exam and retinopathy-positive paediatric cerebral malaria, 137 of whom died and 680 survived.</title><p>Retinal features are presented for the worse eye. Estimates are from unadjusted logistic regression. p&#8804;0.01 is bold.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><break/>Variable name</th><th rowspan="2" valign="top">Units</th><th colspan="3">Died</th><th colspan="3">Survived</th><th colspan="3">Association with death</th></tr><tr><th colspan="2">Numerical characteristics</th><th/><th colspan="2">Numerical characteristics</th><th/><th>OR</th><th>95% CI</th><th>p</th></tr></thead><tbody><tr><td colspan="11">Demographics</td></tr><tr><td valign="top">Age <break/>(median, IQR)</td><td valign="top">months</td><td>35</td><td>23&#8211;59</td><td>136</td><td>39</td><td>27&#8211;58.75</td><td>680</td><td>0.99</td><td>0.00&#8211;1.00</td><td>0.43</td></tr><tr><td valign="top">Weight <break/>(median, IQR)</td><td valign="top">kg</td><td>11</td><td>9&#8211;15</td><td>137</td><td>12</td><td>10&#8211;15</td><td>680</td><td>0.97</td><td>0.93&#8211;1.02</td><td>0.22</td></tr><tr><td valign="top">Height <break/>(median, IQR)</td><td valign="top">cm</td><td>89</td><td>79&#8211;103</td><td>135</td><td>92</td><td>83&#8211;103</td><td>671</td><td>0.99</td><td>0.98&#8211;1.00</td><td>0.15</td></tr><tr><td rowspan="2" valign="top">Sex <break/>(%)</td><td valign="top">boy</td><td>48.9</td><td/><td>66</td><td>50.29</td><td/><td>680</td><td>1.06</td><td>0.73&#8211;1.53</td><td>0.77</td></tr><tr><td valign="top">girl</td><td>51.1</td><td/><td>69</td><td>49.71</td><td/><td/><td/><td/><td/></tr><tr><td colspan="11">Clinical</td></tr><tr><td rowspan="3" valign="top">Coma score <break/>(%)</td><td valign="top">0</td><td>23.3</td><td/><td>32</td><td>9.85</td><td/><td>67</td><td>3.57</td><td>2.13&#8211;5.88</td><td><bold>&lt;0.001</bold></td></tr><tr><td valign="top">1</td><td>41.6</td><td/><td>57</td><td>37.7</td><td/><td>256</td><td>2.13</td><td>1.28&#8211;3.57</td><td><bold>0.003</bold></td></tr><tr><td valign="top">2</td><td>35.0</td><td/><td>48</td><td>52.5</td><td/><td>357</td><td colspan="3">reference</td></tr><tr><td rowspan="2" valign="top">Respiratory distress <break/>(%)</td><td valign="top">Present</td><td>48.9</td><td/><td>67</td><td>39.0</td><td/><td>265</td><td rowspan="2">1.5</td><td rowspan="2">1.04&#8211;2.17</td><td rowspan="2">0.03</td></tr><tr><td valign="top">Absent</td><td>51.1</td><td/><td>70</td><td>61.0</td><td/><td>415</td></tr><tr><td rowspan="2" valign="top">Convulsions at admission (%)</td><td valign="top">Present</td><td>12.4</td><td/><td>17</td><td>14.9</td><td/><td>98</td><td rowspan="2">0.83</td><td rowspan="2">0.45&#8211;1.44</td><td rowspan="2">0.51</td></tr><tr><td valign="top">Absent</td><td>87.6</td><td/><td>120</td><td>85.4</td><td/><td>574</td></tr><tr><td valign="top">Temperature <break/>(median, IQR)</td><td valign="top">degrees C</td><td>38.7</td><td>37.8&#8211;39.5</td><td>137</td><td>38.9</td><td>38&#8211;39.7</td><td>680</td><td>0.89</td><td>0.77&#8211;1.03</td><td>0.12</td></tr><tr><td valign="top">Systolic BP <break/>(median, IQR)</td><td valign="top">mmHg</td><td>100</td><td>90&#8211;110</td><td>127</td><td>100</td><td>90&#8211;110</td><td>652</td><td>0.99</td><td>0.99&#8211;1.01</td><td>0.63</td></tr><tr><td valign="top">Pulse <break/>(median, IQR)</td><td valign="top">beats/min</td><td>156</td><td>136.5&#8211;170.5</td><td>137</td><td>152</td><td>136.75&#8211;169</td><td>678</td><td>1.0</td><td>0.99&#8211;1.01</td><td>0.98</td></tr><tr><td valign="top">Duration of coma <break/>(median, IQR)</td><td valign="top">Hours</td><td>7</td><td>4&#8211;18</td><td>110</td><td>7</td><td>4&#8211;17</td><td>558</td><td>0.99</td><td>0.98&#8211;1.01</td><td>0.29</td></tr><tr><td valign="top">Duration of fever <break/>(median, IQR)</td><td valign="top">Hours</td><td>48</td><td>33.25&#8211;72</td><td>130</td><td>60</td><td>43.25&#8211;72</td><td>652</td><td>0.99</td><td>0.99&#8211;1.00</td><td>0.09</td></tr><tr><td rowspan="2" valign="top">Hypoglycaemia on ward (%)</td><td valign="top">Present</td><td>14.6</td><td/><td>20</td><td>7.81</td><td/><td>53</td><td rowspan="2">2.02</td><td rowspan="2">1.16&#8211;3.5</td><td rowspan="2">0.012</td></tr><tr><td valign="top">Absent</td><td>85.4</td><td/><td>117</td><td>92.1</td><td/><td>626</td></tr><tr><td colspan="11">Laboratory</td></tr><tr><td valign="top">Parasitaemia <break/>(median, IQR)</td><td valign="top">#cells</td><td>79052</td><td>16695&#8211;357000</td><td>134</td><td>68076</td><td>11700&#8211;298000</td><td>649</td><td>1.0</td><td>0.99&#8211;1.00</td><td>0.27</td></tr><tr><td valign="top">White cell count <break/>(median, IQR)</td><td valign="top">#cells</td><td>11300</td><td>6925&#8211;18225</td><td>120</td><td>9200</td><td>6600&#8211;13725</td><td>630</td><td>1.0</td><td>1.00&#8211;1.00</td><td><bold>0.004</bold></td></tr><tr><td valign="top">Haematocrit <break/>(median, IQR)</td><td valign="top">%</td><td>19.5</td><td>15&#8211;24.75</td><td>136</td><td>20</td><td>15.8&#8211;25</td><td>673</td><td>0.99</td><td>0.97&#8211;1.02</td><td>0.69</td></tr><tr><td valign="top">Lactate <break/>(median, IQR)</td><td valign="top">mmol/L</td><td>8.75</td><td>5.38&#8211;12.78</td><td>92</td><td>5.3</td><td>3.2&#8211;9.9</td><td>519</td><td>1.11</td><td>1.06&#8211;1.16</td><td><bold>&lt;0.001</bold></td></tr><tr><td valign="top">HRP2 <break/>(median, IQR)</td><td valign="top">ng/ml</td><td>8838.5</td><td>4435.5&#8211;15102.3</td><td>120</td><td>5765</td><td>2471.5&#8211;10031</td><td>609</td><td>1.0</td><td>1.00&#8211;1.00</td><td><bold>0.004</bold></td></tr><tr><td rowspan="2" valign="top">HIV <break/>(%)</td><td valign="top">Positive</td><td>22.5</td><td/><td>29</td><td>14.9</td><td/><td>88</td><td rowspan="2">1.66</td><td rowspan="2">1.03&#8211;2.66</td><td rowspan="2">0.036</td></tr><tr><td valign="top">Negative</td><td>77.5</td><td/><td>100</td><td>85.1</td><td/><td>503</td></tr><tr><td colspan="11">Ophthalmoscopy</td></tr><tr><td rowspan="5" valign="top">Retinal haemorrhage <break/>(%)</td><td valign="top">&gt;50</td><td>16.0</td><td/><td>22</td><td>4.7</td><td/><td>32</td><td>3.4</td><td>1.78&#8211;6.5</td><td><bold>&lt;0.001</bold></td></tr><tr><td valign="top">21 to 50</td><td>11.0</td><td/><td>15</td><td>6.50</td><td/><td>44</td><td>1.69</td><td>0.85&#8211;3.34</td><td>0.14</td></tr><tr><td valign="top">6 to 20</td><td>13.1</td><td/><td>18</td><td>19.0</td><td/><td>129</td><td>0.69</td><td>0.38&#8211;1.27</td><td>0.23</td></tr><tr><td valign="top">1 to 5</td><td>32.9</td><td/><td>45</td><td>42.9</td><td/><td>291</td><td>0.76</td><td>0.48&#8211;1.23</td><td>0.27</td></tr><tr><td valign="top">None</td><td>27.0</td><td/><td>37</td><td>27.0</td><td/><td>183</td><td colspan="3">reference</td></tr><tr><td rowspan="4" valign="top">Macular whitening <break/>(%)</td><td valign="top">&gt;1</td><td>23.9</td><td/><td>32</td><td>14.8</td><td/><td>100</td><td>2.31</td><td>1.16&#8211;4.59</td><td>0.017</td></tr><tr><td valign="top">1/3 to 1</td><td>28.4</td><td/><td>38</td><td>25.1</td><td/><td>170</td><td>1.61</td><td>0.83&#8211;3.12</td><td>0.16</td></tr><tr><td valign="top">&lt;1/3</td><td>37.3</td><td/><td>50</td><td>45.2</td><td/><td>306</td><td>1.18</td><td>0.63&#8211;2.22</td><td>0.61</td></tr><tr><td valign="top">None</td><td>10.5</td><td/><td>14</td><td>14.9</td><td/><td>101</td><td colspan="3">reference</td></tr><tr><td rowspan="4" valign="top">Foveal whitening <break/>(% of foveal zone)</td><td valign="top">&gt;2/3</td><td>23.3</td><td/><td>31</td><td>11.5</td><td/><td>78</td><td>3.39</td><td>1.83&#8211;6.26</td><td><bold>&lt;0.001</bold></td></tr><tr><td valign="top">1/3 to 2/3</td><td>18.1</td><td/><td>24</td><td>15.2</td><td/><td>103</td><td>1.99</td><td>1.05&#8211;3.74</td><td>0.03</td></tr><tr><td valign="top">&lt;1/3</td><td>42.8</td><td/><td>57</td><td>46.8</td><td/><td>316</td><td>1.54</td><td>0.90&#8211;2.62</td><td>0.11</td></tr><tr><td valign="top">none</td><td>15.8</td><td/><td>21</td><td>26.5</td><td/><td>179</td><td colspan="3">reference</td></tr><tr><td rowspan="4" valign="top">Temporal whitening <break/>(%)</td><td valign="top">3</td><td>10.0</td><td/><td>13</td><td>12.9</td><td/><td>87</td><td>0.83</td><td>0.41&#8211;1.66</td><td>0.60</td></tr><tr><td valign="top">2</td><td>24.6</td><td/><td>32</td><td>18.4</td><td/><td>124</td><td>1.43</td><td>0.83&#8211;2.47</td><td>0.20</td></tr><tr><td valign="top">1</td><td>41.5</td><td/><td>54</td><td>43.1</td><td/><td>290</td><td>1.03</td><td>0.64&#8211;1.67</td><td>0.89</td></tr><tr><td valign="top">none</td><td>23.9</td><td/><td>31</td><td>25.6</td><td/><td>172</td><td colspan="3">reference</td></tr><tr><td rowspan="2" valign="top">Orange vessels, temporal quadrant (%)</td><td valign="top">present</td><td>44.6</td><td/><td>58</td><td>21.7</td><td/><td>145</td><td rowspan="2">2.9</td><td rowspan="2">1.96&#8211;4.3</td><td rowspan="2"><bold>&lt;0.001</bold></td></tr><tr><td>absent</td><td>55.4</td><td/><td>72</td><td>78.3</td><td/><td>523</td></tr><tr><td rowspan="2" valign="top">White vessels, temporal quadrant (%)</td><td>present</td><td>25.4</td><td/><td>33</td><td>24.3</td><td/><td>162</td><td rowspan="2">1.06</td><td rowspan="2">0.69&#8211;1.64</td><td rowspan="2">0.78</td></tr><tr><td>absent</td><td>74.6</td><td/><td>97</td><td>75.8</td><td/><td>506</td></tr><tr><td rowspan="2" valign="top">White capillaries <break/>(%)</td><td valign="top">present</td><td>26.9</td><td/><td>35</td><td>33.1</td><td/><td>221</td><td rowspan="2">0.75</td><td rowspan="2">0.49&#8211;1.13</td><td rowspan="2">0.17</td></tr><tr><td valign="top">absent</td><td>73.1</td><td/><td>95</td><td>66.9</td><td/><td>447</td></tr><tr><td rowspan="2" valign="top">Papilloedema <break/>(%)</td><td valign="top">present</td><td>39.0</td><td/><td>53</td><td>21.8</td><td/><td>148</td><td rowspan="2">2.29</td><td rowspan="2">1.55&#8211;3.38</td><td rowspan="2"><bold>&lt;0.001</bold></td></tr><tr><td valign="top">absent</td><td>61.0</td><td/><td>83</td><td>78.2</td><td/><td>530</td></tr><tr><td rowspan="2" valign="top">Disc hyperaemia <break/>(%)</td><td valign="top">present</td><td>48.7</td><td/><td>54</td><td>35.3</td><td/><td>212</td><td rowspan="2">1.73</td><td rowspan="2">1.15&#8211;2.61</td><td rowspan="2"><bold>0.008</bold></td></tr><tr><td valign="top">absent</td><td>51.4</td><td/><td>57</td><td>64.7</td><td/><td>388</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32208-v2.xml">elife-32208-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.32208.010</object-id><label>Table 4.</label><caption><title>Relationship between severe sequestration (pigmented/late parasitised RBCs sequestered around 360<bold>&#176;&#160;</bold>of the lumen circumference) and orange discoloration visible clinically and on gross pathology in 412 venules (diameter 10&#8211;50 &#956;m) from nine cases</title></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2"/><th colspan="2">Orange discolouration</th></tr><tr><th>+</th><th>-</th></tr></thead><tbody><tr><td rowspan="2">Severe sequestration</td><td>+</td><td>188</td><td>5</td></tr><tr><td>-</td><td>24</td><td>195</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32208-v2.xml">elife-32208-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table5" position="float"><object-id pub-id-type="doi">10.7554/eLife.32208.023</object-id><label>Table 5.</label><caption><title>Frequency of intravascular filling defects (worse eye) on fluorescein angiography manual grading by involvement of retinal vessel in 259 children with MR-positive disease and FA within 24 hr of admission and unadjusted association with death (n&#160;=&#160;35) and coma recovery of consciousness (BCS&#160;&#8805;3; n&#160;=&#160;225)</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><break/>Retinal vessel</th><th rowspan="2">Sequestration</th><th colspan="3">Died*</th><th colspan="3">Survived*</th><th colspan="3">Association with death</th></tr><tr><th>N</th><th>%</th><th>Total</th><th>N</th><th>%</th><th>Total</th><th>OR</th><th>95% CI</th><th>p</th></tr></thead><tbody><tr><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td rowspan="2">large venules</td><td>present</td><td>26</td><td>86.7</td><td rowspan="2">30</td><td>172</td><td>79.3</td><td rowspan="2">217</td><td rowspan="2">1.70</td><td rowspan="2">0.56&#8211;5.12</td><td rowspan="2">0.35</td></tr><tr><td>absent</td><td>4</td><td>13.3</td><td>45</td><td>20.7</td></tr><tr><td rowspan="2">small venules</td><td>present</td><td>29</td><td>96.7</td><td rowspan="2">30</td><td>211</td><td>98.1</td><td rowspan="2">215</td><td rowspan="2">0.88</td><td rowspan="2">0.71&#8211;1.09</td><td rowspan="2">0.23</td></tr><tr><td>absent</td><td>1</td><td>3.33</td><td>4</td><td>1.86</td></tr><tr><td rowspan="2">post-capillary venules</td><td>present</td><td>25</td><td>96.2</td><td rowspan="2">26</td><td>201</td><td>98.5</td><td rowspan="2">204</td><td rowspan="2">0.37</td><td rowspan="2">0.04&#8211;3.70</td><td rowspan="2">0.4</td></tr><tr><td>absent</td><td>1</td><td>3.85</td><td>3</td><td>1.47</td></tr><tr><td rowspan="2">pre-capillary arterioles</td><td>present</td><td>19</td><td>76.0</td><td rowspan="2">25</td><td>109</td><td>56.2</td><td rowspan="2">194</td><td rowspan="2">2.47</td><td rowspan="2">0.94&#8211;6.45</td><td rowspan="2">0.065</td></tr><tr><td>absent</td><td>6</td><td>24.0</td><td>85</td><td>43.8</td></tr><tr><td rowspan="2">small arterioles</td><td>present</td><td>15</td><td>51.7</td><td rowspan="2">29</td><td>93</td><td>42.9</td><td rowspan="2">217</td><td rowspan="2">1.43</td><td rowspan="2">0.66&#8211;3.11</td><td rowspan="2">0.37</td></tr><tr><td>absent</td><td>14</td><td>48.3</td><td>124</td><td>57.1</td></tr><tr><td rowspan="2">large arterioles</td><td>present</td><td>9</td><td>30.0</td><td rowspan="2">30</td><td>29</td><td>13.2</td><td rowspan="2">219</td><td rowspan="2">2.81</td><td rowspan="2">1.17&#8211;6.72</td><td rowspan="2"><bold>0.02</bold></td></tr><tr><td>absent</td><td>21</td><td>70.0</td><td>190</td><td>86.8</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32213-v3.xml">elife-32213-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.32213.030</object-id><label>Table 1.</label><caption><title>Histopathologic analysis relative to <xref ref-type="fig" rid="fig3">Figure 3</xref>.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="5">Grade</th></tr><tr><th>No PIN</th><th>Low grade PIN</th><th>High grade PIN</th><th>Microinvasive <break/>carcinoma</th><th>Invasive carcinoma</th></tr></thead><tbody><tr><td><italic>Pbsn-Cre4;Pten<sup>f/f*</sup></italic></td><td/><td/><td>66%</td><td>33%</td><td/></tr><tr><td><italic>Pbsn-Cre4;Pten<sup>f/f</sup></italic> R&#160;+&#160;P<sup>&#8224;</sup></td><td>33%</td><td>16%</td><td>33%</td><td>16%</td><td/></tr><tr><td><italic>Pbsn-Cre4;Pten<sup>f/f</sup></italic> <break/>+&#160;NAC<sup>&#8225;</sup></td><td/><td/><td/><td>25%</td><td>75%</td></tr></tbody></table><table-wrap-foot><fn><p>*The anterior lobes of prostates from untreated mice were analyzed by histopathology at 8 months (percentage of mice with highest grade is indicated).</p><p>&#8224;The anterior lobes of prostates from mice treated at 4 months with rapamycin and PEITC (R&#160;+&#160;P) were analyzed by histopathology at 8 months (percentage of mice with highest grade is indicated).</p></fn><fn><p>&#8225;The anterior lobes of prostates from mice treated at 4 months with NAC were analyzed by histopathology at 8 months (percentage of mice with highest grade is indicated).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32353-v2.xml">elife-32353-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.32353.004</object-id><label>Table 1.</label><caption><title>Overlap of striosomes outlined using tdTomato and MOR1.</title></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2" valign="bottom"/><th colspan="2" valign="bottom"><bold>MOR1</bold></th></tr><tr><th valign="bottom">Positive</th><th valign="bottom">Negative</th></tr></thead><tbody><tr><td rowspan="2"><bold>tdTomato</bold></td><td valign="bottom">Positive</td><td valign="bottom">14.2%&#177;1.3%</td><td valign="bottom">2.0%&#177;0.3%</td></tr><tr><td valign="bottom">Negative</td><td valign="bottom">3.7%&#177;0.6%</td><td valign="bottom">80.2%&#177;1.9%</td></tr></tbody></table><table-wrap-foot><fn><p>MOR1 test-retest error rate&#160;=&#160;2.4%</p><p>tdTomato test-retest error rate&#160;=&#160;2.3%</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32353-v2.xml">elife-32353-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.32353.008</object-id><label>Table 3.</label><caption><title>Baseline fluorescence and &#916;F/F values for striosomal and matrix neurons.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><break/></th><th align="center" colspan="4">Cell type</th></tr><tr><th>Striosomal</th><th>In striosomal neuropil</th><th>tdTomato labeled</th><th>Matrix</th></tr></thead><tbody><tr><td>Baseline fluorescence</td><td>290.0 (8.5) ***</td><td>274.9 (8.1) ***</td><td>337.2 (27.9)</td><td>364.5 (6.8)</td></tr><tr><td>&#916;F/F baseline mean</td><td>11.3 (0.7)</td><td>11.9 (0.7)</td><td>9.3 (1.9)</td><td>11.9 (0.4)</td></tr><tr><td>&#916;F/F baseline standard deviation</td><td>37.2 (1.3)</td><td>38.2 (1.4)</td><td>33.5 (3.5)</td><td>38.5 (0.7)</td></tr><tr><td>&#916;F/F baseline maximum</td><td>250.6 (9.9)</td><td>259.3 (11.2)</td><td>216.2 (22.8)</td><td>255.2 (6.0)</td></tr></tbody></table><table-wrap-foot><fn><p>***p&lt;0.001.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32506-v1.xml">elife-32506-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.32506.022</object-id><label>Table 1.</label><caption><title>Frequency of cell-intercalation events.</title><p><supplementary-material id="table1sdata1"><object-id pub-id-type="doi">10.7554/eLife.32506.023</object-id><label>Table 1-source data 1</label><caption><title>Source data for <xref ref-type="supplementary-material" rid="table1sdata1">Table 1</xref>.</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-32506-fig1-v1.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr valign="top"><th rowspan="2">Genotype <break/></th><th colspan="3">Frequency of intercalation (%)</th><th rowspan="2">n <break/></th></tr><tr valign="top"><th>Leftward</th><th>Rightward</th><th>Total</th></tr></thead><tbody><tr valign="top"><td>+/+</td><td>4.0</td><td>4.8</td><td>9.2*</td><td>250</td></tr><tr valign="top"><td><italic>Myo31DF</italic></td><td>2.2</td><td>3.3</td><td>6.0*</td><td>182</td></tr></tbody></table><table-wrap-foot><fn><p>Frequency is defined as the number of intercalation events divided by the total number of examined cells, in 30 min.</p><p>When two cells in a column had initial contact and were subsequently separated by another intervening cell, cell intercalation was said to occur, and the direction from which the intervening cell came was determined.</p></fn><fn><p>*includes one event that was not distinguished as leftward or rightward.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32605-v2.xml">elife-32605-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.32605.017</object-id><label>Table 1.</label><caption><title>SVM statistics.&#160;AUC: area under curve; Acc: accuracy; Prc: precision; Rec: recall.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Behavior</th><th colspan="6">Train</th><th colspan="6">Withheld</th><th colspan="6">Test</th></tr><tr><th>AUC</th><th>AUC chance</th><th>Acc</th><th>Acc chance</th><th>Prc</th><th>Rec</th><th>AUC</th><th>AUC chance</th><th>Acc</th><th>Acc chance</th><th>Prc</th><th>Rec</th><th>AUC</th><th>AUC chance</th><th>Acc</th><th>Acc chance</th><th>Prc</th><th>Rec</th></tr></thead><tbody><tr><td>Silent</td><td>1</td><td>0.5</td><td>100%</td><td>9.6%</td><td>100%</td><td>100%</td><td>0.98</td><td>0.5</td><td>95.6%</td><td>9.6%</td><td>75.6%</td><td>97.4%</td><td>0.95</td><td>0.5</td><td>90.3%</td><td>1.9%</td><td>18.4%</td><td>90.3%</td></tr><tr><td>Elongation</td><td>1</td><td>0.5</td><td>100%</td><td>14.2%</td><td>100%</td><td>100%</td><td>0.96</td><td>0.5</td><td>93.4%</td><td>13.6%</td><td>76.4%</td><td>95.9%</td><td>0.91</td><td>0.5</td><td>87.9%</td><td>22.2%</td><td>71.4%</td><td>92.6%</td></tr><tr><td>Tentacle sway</td><td>1</td><td>0.5</td><td>100%</td><td>25.1%</td><td>100%</td><td>100%</td><td>0.95</td><td>0.5</td><td>89.6%</td><td>25.0%</td><td>77.5%</td><td>92.4%</td><td>0.76</td><td>0.5</td><td>71.9%</td><td>30.2%</td><td>47.9%</td><td>76.7%</td></tr><tr><td>Body sway</td><td>1</td><td>0.5</td><td>100%</td><td>10.0%</td><td>100%</td><td>100%</td><td>0.92</td><td>0.5</td><td>92.9%</td><td>9.3%</td><td>65.7%</td><td>97.0%</td><td>0.75</td><td>0.5</td><td>83.4%</td><td>17.7%</td><td>52.8%</td><td>95.4%</td></tr><tr><td>Bending</td><td>1</td><td>0.5</td><td>100%</td><td>5.2%</td><td>100%</td><td>100%</td><td>0.98</td><td>0.5</td><td>97.3%</td><td>6.1%</td><td>74.4%</td><td>98.4%</td><td>0.81</td><td>0.5</td><td>93.9%</td><td>6.1%</td><td>38.9%</td><td>96.5%</td></tr><tr><td>Contraction</td><td>1</td><td>0.5</td><td>100%</td><td>6.6%</td><td>100%</td><td>100%</td><td>0.97</td><td>0.5</td><td>95.7%</td><td>6.9%</td><td>70.4%</td><td>97.7%</td><td>0.92</td><td>0.5</td><td>92.8%</td><td>11.7%</td><td>63.2%</td><td>95.5%</td></tr><tr><td>Feeding</td><td>1</td><td>0.5</td><td>100%</td><td>29.2%</td><td>100%</td><td>100%</td><td>1</td><td>0.5</td><td>98.8%</td><td>29.6%</td><td>98.5%</td><td>99.4%</td><td>0.83</td><td>0.5</td><td>81.0%</td><td>10.2%</td><td>39.6%</td><td>94.1%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32638-v2.xml">elife-32638-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.32638.013</object-id><label>Table 2.</label><caption><title>Pharmacological alterations of NAD<sup>+</sup> and NADPH/NADP&#160;+&#160;in U2OS cells measured by flow cytometry.</title></caption><table frame="hsides" rules="groups"><thead><tr><th/><th colspan="4">Normalized FRET ratio (TMR/FRET)</th></tr><tr><th rowspan="2">Treatment</th><th colspan="2">NAD-Snifit</th><th colspan="2">NADP-Snifit</th></tr><tr><th>Cytosol</th><th>Mitochondria</th><th>Cytosol</th><th>Mitochondria</th></tr></thead><tbody><tr><td>Control</td><td>1.00 (&#177;0.03)</td><td>1.00 (&#177;0.02)</td><td>1.00 (&#177;0.01)</td><td>1.00 (&#177;0.01)</td></tr><tr><td>1 mM NA</td><td>0.91 (&#177;0.01)</td><td>n.d.</td><td>0.92 (&#177;0.01)</td><td>1.00 (&#177;0.01)*</td></tr><tr><td>10 mM NAM</td><td>0.92 (&#177;0.02)</td><td>n.d.</td><td>1.05 (&#177;0.01)</td><td>n.d.</td></tr><tr><td>1 mM NMN</td><td>0.82 (&#177;0.01)</td><td>n.d.</td><td>0.95 (&#177;0.01)</td><td>0.99 (&#177;0.01)*</td></tr><tr><td>1 mM NR</td><td>0.80 (&#177;0.02)</td><td>n.d.</td><td>0.96 (&#177;0.01)</td><td>1.00 (&#177;0.01)*</td></tr><tr><td>100 nM FK866</td><td>1.61 (&#177;0.06)</td><td>1.48 (&#177;0.04)</td><td>1.05 (&#177;0.01)</td><td>0.99 (&#177;0.01)*</td></tr><tr><td>1 mM 6-AN</td><td>n.d.</td><td>n.d.</td><td>0.80 (&#177;0.02)</td><td>n.d.</td></tr><tr><td>1 mM Metformin</td><td>0.89 (&#177;0.04)</td><td>1.09 (&#177;0.03)</td><td>0.90 (&#177;0.01)</td><td>0.95 (&#177;0.01)</td></tr><tr><td>1 mM Phenformin</td><td>0.79 (&#177;0.05)</td><td>1.13 (&#177;0.06)</td><td>0.88 (&#177;0.01)</td><td>0.83 (&#177;0.01)</td></tr><tr><td>10 &#181;M Rotenone</td><td>0.67 (&#177;0.03)</td><td>1.08 (&#177;0.02)</td><td>0.75 (&#177;0.02)</td><td>0.80 (&#177;0.02)</td></tr><tr><td>25 &#181;M Oligomycin A</td><td>1.14 (&#177;0.03)</td><td>1.63 (&#177;0.01)</td><td>1.12 (&#177;0.03)</td><td>1.36 (&#177;0.07)</td></tr></tbody></table><table-wrap-foot><fn><p>Values represent the average of medians (&#177;s.d.) TMR/FRET ratios of three independent measurements normalized to control condition (n&#160;=&#160;3). Control: untreated cells (full growth medium with 25 mM glucose), NA: nicotinic acid, NAM: nicotinamide, NMN: nicotinamide mononucleotide, NR: nicotinamide riboside, FK866: (E)-N-[4-(1-benzoylpiperidin-4-yl)butyl]&#8722;3-(pyridin-3-yl)acrylamide, 6-AN: 6-aminonicotinamide.</p><p>*The effect of the treatment is not statistically significant compared to the control condition (Kruskal-Wallis with Dunn&#8217;s post-hoc multiple comparison test, &#945;&#160;=&#160;0.05). n.d., not determined. All compounds were also tested for interactions with the sensor in vitro (<xref ref-type="fig" rid="app1fig5">Appendix 1&#8212;figures 5</xref>, <xref ref-type="fig" rid="app1fig6">6</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32638-v2.xml">elife-32638-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app1table6" position="float"><object-id pub-id-type="doi">10.7554/eLife.32638.043</object-id><label>Appendix 1&#8212;table 6.</label><caption><title>Determination of FRET efficiency in U2OS cells.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Sensors</th><th rowspan="2">Localization</th><th colspan="3">&lt;&#964;&gt; (ns)&#160;&#177;&#160;SD</th><th rowspan="2">E (%)&#160;&#177;&#160;SD <break/>(Basal)</th><th rowspan="2">E<sub>min</sub> (%)&#160;&#177;&#160;SD <break/>(2 mM SPY)</th></tr><tr><th>Donor&#160;only</th><th>FRET, basal</th><th>FRET, 2 mM SPY</th></tr></thead><tbody><tr><td rowspan="3">NADP-Snifit</td><td>Cytosol</td><td>2.80&#160;&#177;&#160;0.02</td><td>2.12&#160;&#177;&#160;0.06</td><td>2.35&#160;&#177;&#160;0.04</td><td>24.2&#160;&#177;&#160;0.7</td><td>16.0&#160;&#177;&#160;0.3</td></tr><tr><td>&#8195;Nucleus</td><td>2.69&#160;&#177;&#160;0.03</td><td>1.98&#160;&#177;&#160;0.04</td><td>2.28&#160;&#177;&#160;0.04</td><td>26.3&#160;&#177;&#160;0.6</td><td>15.5&#160;&#177;&#160;0.4</td></tr><tr><td>&#8195;Mitochondria</td><td>2.55&#160;&#177;&#160;0.05</td><td>2.08&#160;&#177;&#160;0.03</td><td>2.16&#160;&#177;&#160;0.02</td><td>18.2&#160;&#177;&#160;0.4</td><td>15.2&#160;&#177;&#160;0.3</td></tr><tr><td rowspan="3">NAD-Snifit</td><td>Cytosol</td><td>2.89&#160;&#177;&#160;0.04</td><td>2.19&#160;&#177;&#160;0.02</td><td>2.42&#160;&#177;&#160;0.02</td><td>24.3&#160;&#177;&#160;0.4</td><td>16.3&#160;&#177;&#160;0.3</td></tr><tr><td>&#8195;Nucleus</td><td>2.66&#160;&#177;&#160;0.03</td><td>2.11&#160;&#177;&#160;0.04</td><td>2.49&#160;&#177;&#160;0.06</td><td>20.6&#160;&#177;&#160;0.4</td><td>6.5&#160;&#177;&#160;0.2</td></tr><tr><td>&#8195;Mitochondria</td><td>2.63&#160;&#177;&#160;0.03</td><td>2.02&#160;&#177;&#160;0.02</td><td>2.30&#160;&#177;&#160;0.06</td><td>23.0&#160;&#177;&#160;0.3</td><td>12.2&#160;&#177;&#160;0.3</td></tr></tbody></table><table-wrap-foot><fn><p>The data represent the amplitude-weighted average lifetime &lt;&#964;&gt;&#160;as mean&#160;&#177;&#160;SD (N&#160;=&#160;10) measured in living U2OS cells in full growth medium (DMEM&#160;+10% FBS) at 37&#160;&#176;C. The &#8216;donor-only&#8217; sample was obtained by single labeling of the sensor constructs with CP-TMR-SMX. The FRET samples are labeled with both CP-TMR-SMX and SiR-Halo. The cells labeled with both fluorophores were first measured without treatment to obtain their basal fluorescence lifetime, then the same cells were measured again after the treatment with 2 mM sulfapyridine (SPY) to obtain the fully sensor open state. The correlated FRET efficiencies (E) were calculated for each conditions.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-32740-v2.xml">elife-32740-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.32740.014</object-id><label>Table 2.</label><caption><title>Number of videos for each intensity presented as InputMovie in the EEG and TMS experiment.</title><p>The last line shows the average movie intensities and their standard deviation presented for each condition and experiment.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Perceived intensity</th><th colspan="2">EEG experiment</th><th colspan="2">TMS experiment&#160;per session</th></tr><tr><th>Hand</th><th>Face</th><th>Hand</th><th>Face</th></tr></thead><tbody><tr><td valign="top">2</td><td valign="top">13</td><td valign="top">23</td><td valign="top">4</td><td valign="top">10</td></tr><tr><td valign="top">3</td><td valign="top">2</td><td valign="top">3</td><td valign="top">10</td><td valign="top">4</td></tr><tr><td valign="top">4</td><td valign="top">0</td><td valign="top">4</td><td valign="top">4</td><td valign="top">4</td></tr><tr><td valign="top">5</td><td valign="top">27</td><td valign="top">10</td><td valign="top">10</td><td valign="top">6</td></tr><tr><td valign="top">6</td><td valign="top">1</td><td valign="top">11</td><td valign="top">2</td><td valign="top">6</td></tr><tr><td valign="top">7</td><td valign="top"/><td valign="top">1</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">Average intensity</td><td valign="top">4&#160;&#177;&#160;1.4</td><td valign="top">3.7&#160;&#177;&#160;1.7</td><td valign="top">3.9&#160;&#177;&#160;1.2</td><td valign="top">3.8&#160;&#177;&#160;1.6</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-33105-v1.xml">elife-33105-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.33105.004</object-id><label>Table 1.</label><caption><title>Reagents permuted during optimisation of the single cell RNAseq protocol and stats of each treatment condition after sequencing.</title><p>Different combinations of the protocol were tested by sequencing. Initial trials were performed with 2 &#181;l of lysis buffer, this was increased to 4 &#181;l to augment capture efficiency. Permutations of the protocol that were tested were a terminal anchoring base (A,G,C; V) or not (T), two reverse transcriptase enzymes (Smartscribe (SmSc); Superscript II (SII)) and 25 or 30 cycles of preamplification. Both sexual and asexual cells of <italic>P. berghei</italic> and <italic>P. falciparum</italic> were tested. For each sequenced dataset, we calculated the mean percentages of rRNA, mRNA and other reads across the cells. For some samples we also downsampled the data to 50,000 reads per cell to allow comparison of the number of genes detected. This was done to determine differences in the complexity of each library. For the three larger datasets produced (<italic>P. falciparum</italic> gametocytes, <italic>P. berghei</italic> mixed blood stages, and <italic>P. falciparum</italic> asexual stages), we provide the numbers of pre- and post-filtered cells and median number of genes in those filtered cells.</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left" rowspan="3" valign="middle">Conditions tested</th><th align="left" valign="middle">Protocol</th><th align="center" valign="middle">SSII, V30, 30 cycles</th><th align="center" valign="middle">SSII, T30, 30 cycles</th><th align="center" valign="middle">SmSc, T30, 30 cycles</th><th align="center" valign="middle">SSII, T30, 25 cycles</th><th align="center" valign="middle">SmSc, T30, 25 cycles</th><th align="center" valign="middle">SmSc, T30, 25 cycles</th><th align="center" valign="middle">SmSc, T30, 25 cycles</th><th align="center" valign="middle">SmSc, T30, 25 cycles</th></tr><tr><th align="left" valign="middle">Cells</th><th align="center" valign="middle">Sexual</th><th align="center" valign="middle">Asexual</th><th align="center" valign="middle">Asexual</th><th align="center" valign="middle">Asexual</th><th align="center" valign="middle">Asexual</th><th align="center" valign="middle">Sexual</th><th align="center" valign="middle">Mixed blood</th><th align="center" valign="middle">Asexual</th></tr><tr><th align="left" valign="middle">Species</th><th align="center" valign="middle">Pf</th><th align="center" valign="middle">Pf</th><th align="center" valign="middle">Pf</th><th align="center" valign="middle">Pf</th><th align="center" valign="middle">Pf</th><th align="center" valign="middle">Pf</th><th align="center" valign="middle">Pb</th><th align="center" valign="middle">Pf</th></tr></thead><tbody><tr><td align="left" rowspan="2" valign="middle">Lysis buffer volume</td><td align="left" valign="middle">2 &#181;l</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/></tr><tr><td align="left" valign="middle">4 &#181;l</td><td align="center" valign="middle"/><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td></tr><tr><td align="left" rowspan="2" valign="middle">Oligo Dt (IDT)</td><td align="left" valign="middle">Anchored 30 bp</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/></tr><tr><td align="left" valign="middle">Non-Anchored 30 bp</td><td align="center" valign="middle"/><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td></tr><tr><td align="left" rowspan="2" valign="middle">Reverse transcriptase</td><td align="left" valign="middle">Superscript II (Life Technologies) 10U</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle"/><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/></tr><tr><td align="left" valign="middle">Smartscribe (Clontech) 5U</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle"/><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td></tr><tr><td align="left" rowspan="2" valign="middle">Cycle number</td><td align="left" valign="middle">25</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td></tr><tr><td align="left" valign="middle">30</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/></tr><tr><td align="left" rowspan="2" valign="middle">Sequencing machine</td><td align="left" valign="middle">HiSeq</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td></tr><tr><td align="left" valign="middle">MiSeq</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle">&#10003;</td><td align="center" valign="middle"/><td align="center" valign="middle"/><td align="center" valign="middle"/></tr><tr><td align="left" rowspan="7" valign="middle">Sequencing results summary</td><td align="left" valign="middle">% rRNA</td><td align="center" valign="middle">5.7</td><td align="center" valign="middle">33.5</td><td align="center" valign="middle">36.2</td><td align="center" valign="middle">6.4</td><td align="center" valign="middle">18.4</td><td align="center" valign="middle">17.8</td><td align="center" valign="middle">16.7</td><td align="center" valign="middle">34.8</td></tr><tr><td align="left" valign="middle">% coding genes</td><td align="center" valign="middle">4.4</td><td align="center" valign="middle">11.3</td><td align="center" valign="middle">39.3</td><td align="center" valign="middle">10.5</td><td align="center" valign="middle">33</td><td align="center" valign="middle">51.7</td><td align="center" valign="middle">49</td><td align="center" valign="middle">40.5</td></tr><tr><td align="left" valign="middle">% other</td><td align="center" valign="middle">90</td><td align="center" valign="middle">55.2</td><td align="center" valign="middle">24.4</td><td align="center" valign="middle">83.1</td><td align="center" valign="middle">48.6</td><td align="center" valign="middle">30.5</td><td align="center" valign="middle">34.2</td><td align="center" valign="middle">24.6</td></tr><tr><td align="left" valign="middle">Median genes detected for 50k reads</td><td align="center" valign="middle">25</td><td align="center" valign="middle">84</td><td align="center" valign="middle">145</td><td align="center" valign="middle">174</td><td align="center" valign="middle">181</td><td align="center" valign="middle">502.5</td><td align="center" valign="middle">NA</td><td align="center" valign="middle">NA</td></tr><tr><td align="left" valign="middle">Total cells</td><td align="center" valign="middle">5</td><td align="center" valign="middle">6</td><td align="center" valign="middle">6</td><td align="center" valign="middle">6</td><td align="center" valign="middle">6</td><td align="center" valign="middle">237</td><td align="center" valign="middle">182</td><td align="center" valign="middle">174</td></tr><tr><td align="left" valign="middle">Cells passing filters</td><td align="center" valign="middle">NA</td><td align="center" valign="middle">NA</td><td align="center" valign="middle">NA</td><td align="center" valign="middle">NA</td><td align="center" valign="middle">NA</td><td align="center" valign="middle">191</td><td align="center" valign="middle">144</td><td align="center" valign="middle">161</td></tr><tr><td align="left" valign="middle">Median gene count</td><td align="center" valign="middle">NA</td><td align="center" valign="middle">NA</td><td align="center" valign="middle">NA</td><td align="center" valign="middle">NA</td><td align="center" valign="middle">NA</td><td align="center" valign="middle">2011</td><td align="center" valign="middle">1922.5</td><td align="center" valign="middle">1793</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-33125-v2.xml">elife-33125-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.33125.019</object-id><label>Table 1.</label><caption><title>Summary of simulation results and comparison of methods.</title><p>The table shows the bias (mean error), &#8216;std&#8217; (standard deviation) and &#8216;iqr&#8217; (inter-quartile range: difference between the 75th and 25th percentile) for each method for all 3480 simulations. The left column shows the relative bias/std/iqr for the <italic>D</italic><sub>FREE</sub>-estimate and the right column shows the absolute bias/std/iqr for the <italic>F</italic><sub>BOUND</sub>-estimate.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Analysis method</th><th align="center" colspan="3" valign="top"><italic>D</italic><sub>FREE</sub></th><th align="center" colspan="3" valign="top"><italic>F</italic><sub>BOUND</sub></th></tr><tr><th>bias</th><th>std</th><th>iqr</th><th>bias</th><th>std</th><th>iqr</th></tr></thead><tbody><tr><td valign="top">Spot-On (all)</td><td valign="top">&#8722;4.8%</td><td valign="top">3.3%</td><td valign="top">3.5%</td><td valign="top">&#8722;1.7%</td><td valign="top">1.2%</td><td valign="top">1.8%</td></tr><tr><td valign="top">vbSPT (2-state)</td><td valign="top">0.8%</td><td valign="top">12.5%</td><td valign="top">6.8%</td><td valign="top">5.0%</td><td valign="top">4.6%</td><td valign="top">6.1%</td></tr><tr><td valign="top">MSD<sub>i</sub> (R<sup>2</sup>&#160;&gt;&#160;0.8)</td><td valign="top">8.0%</td><td valign="top">28.5%</td><td valign="top">4.9%</td><td valign="top">&#8722;20.6%</td><td valign="top">26.4%</td><td valign="top">32.1%</td></tr><tr><td valign="top">MSD<sub>i</sub> (all)</td><td valign="top">&#8722;39.6%</td><td valign="top">41.8%</td><td valign="top">19.0%</td><td valign="top">22.0%</td><td valign="top">15.8%</td><td valign="top">17.8%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-33126-v1.xml">elife-33126-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.33126.002</object-id><label>Table 1.</label><caption><title>Relative gene expression level in HAP1 cells of selected genes discussed in this work.</title><p>RPKM (<underline>R</underline>eads <underline>P</underline>er <underline>K</underline>ilobase of transcript per <underline>M</underline>illion mapped reads) values from duplicate RNAseq datasets described previously (NCBI GEO accession number GSE75515, <ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/geo/">https://www.ncbi.nlm.nih.gov/geo/</ext-link>),&#160;obtained from two different passages of WT HAP1 cells (<xref ref-type="bibr" rid="bib13">Dubey et al., 2016</xref>). Groups of paralogues or genes with redundant function are shaded in alternating colors to facilitate comparisons.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene</th><th colspan="3" valign="bottom">RPKM</th></tr><tr><th>Replicate 1</th><th>Replicate 2</th><th>Average</th></tr></thead><tbody><tr><td style="author-callout-style-b1"><italic>LGR4</italic></td><td style="author-callout-style-b1">160.61</td><td style="author-callout-style-b1">174.69</td><td style="author-callout-style-b1">167.65</td></tr><tr><td style="author-callout-style-b1"><italic>LGR5</italic></td><td style="author-callout-style-b1">0.02</td><td style="author-callout-style-b1">0.00</td><td style="author-callout-style-b1">0.01</td></tr><tr><td style="author-callout-style-b1"><italic>LGR6</italic></td><td style="author-callout-style-b1">0.02</td><td style="author-callout-style-b1">0.00</td><td style="author-callout-style-b1">0.01</td></tr><tr><td style="author-callout-style-b3"><italic>ZNRF3</italic></td><td style="author-callout-style-b3">30.9</td><td style="author-callout-style-b3">33.3</td><td style="author-callout-style-b3">32.1</td></tr><tr><td style="author-callout-style-b3"><italic>RNF43</italic></td><td style="author-callout-style-b3">0.12</td><td style="author-callout-style-b3">0.08</td><td style="author-callout-style-b3">0.1</td></tr><tr><td style="author-callout-style-b1" valign="top"><italic>GPC1</italic></td><td style="author-callout-style-b1" valign="top">49.55</td><td style="author-callout-style-b1" valign="top">47.53</td><td style="author-callout-style-b1" valign="top">48.54</td></tr><tr><td style="author-callout-style-b1" valign="top"><italic>GPC2</italic></td><td style="author-callout-style-b1" valign="top">4.17</td><td style="author-callout-style-b1" valign="top">4.79</td><td style="author-callout-style-b1" valign="top">4.48</td></tr><tr><td style="author-callout-style-b1" valign="top"><italic>GPC3</italic></td><td style="author-callout-style-b1" valign="top">170.22</td><td style="author-callout-style-b1" valign="top">144.37</td><td style="author-callout-style-b1" valign="top">157.29</td></tr><tr><td style="author-callout-style-b1"><italic>GPC4</italic></td><td style="author-callout-style-b1">209.39</td><td style="author-callout-style-b1">229.86</td><td style="author-callout-style-b1">219.63</td></tr><tr><td style="author-callout-style-b1" valign="bottom"><italic>GPC5</italic></td><td style="author-callout-style-b1" valign="bottom">0.1</td><td style="author-callout-style-b1" valign="bottom">0.1</td><td style="author-callout-style-b1" valign="bottom">0.1</td></tr><tr><td style="author-callout-style-b1"><italic>GPC6</italic></td><td style="author-callout-style-b1">13.88</td><td style="author-callout-style-b1">14.90</td><td style="author-callout-style-b1">14.39</td></tr><tr><td style="author-callout-style-b3" valign="bottom"><italic>SDC1</italic></td><td style="author-callout-style-b3" valign="bottom">51.37</td><td style="author-callout-style-b3" valign="bottom">47.88</td><td style="author-callout-style-b3" valign="bottom">49.63</td></tr><tr><td style="author-callout-style-b3" valign="bottom"><italic>SDC2</italic></td><td style="author-callout-style-b3" valign="bottom">11.42</td><td style="author-callout-style-b3" valign="bottom">9.2</td><td style="author-callout-style-b3" valign="bottom">10.31</td></tr><tr><td style="author-callout-style-b3" valign="bottom"><italic>SDC3</italic></td><td style="author-callout-style-b3" valign="bottom">43.58</td><td style="author-callout-style-b3" valign="bottom">50.64</td><td style="author-callout-style-b3" valign="bottom">47.11</td></tr><tr><td style="author-callout-style-b3" valign="bottom"><italic>SDC4</italic></td><td style="author-callout-style-b3" valign="bottom">8.16</td><td style="author-callout-style-b3" valign="bottom">8.21</td><td style="author-callout-style-b3" valign="bottom">8.18</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-33126-v2.xml">elife-33126-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.33126.004</object-id><label>Table 1.</label><caption><title>Relative gene expression level in HAP1 cells of selected genes discussed in this work.</title><p>RPKM (<underline>R</underline>eads <underline>P</underline>er <underline>K</underline>ilobase of transcript per <underline>M</underline>illion mapped reads) values from duplicate RNAseq datasets described previously (NCBI GEO accession number GSE75515, <ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/geo/">https://www.ncbi.nlm.nih.gov/geo/</ext-link>),&#160;obtained from two different passages of WT HAP1 cells (<xref ref-type="bibr" rid="bib13">Dubey et al., 2016</xref>). Groups of paralogues or genes with redundant function are shaded in alternating colors to facilitate comparisons.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene</th><th colspan="3" valign="bottom">RPKM</th></tr><tr><th>Replicate 1</th><th>Replicate 2</th><th>Average</th></tr></thead><tbody><tr><td style="author-callout-style-b1"><italic>LGR4</italic></td><td style="author-callout-style-b1">160.61</td><td style="author-callout-style-b1">174.69</td><td style="author-callout-style-b1">167.65</td></tr><tr><td style="author-callout-style-b1"><italic>LGR5</italic></td><td style="author-callout-style-b1">0.02</td><td style="author-callout-style-b1">0.00</td><td style="author-callout-style-b1">0.01</td></tr><tr><td style="author-callout-style-b1"><italic>LGR6</italic></td><td style="author-callout-style-b1">0.02</td><td style="author-callout-style-b1">0.00</td><td style="author-callout-style-b1">0.01</td></tr><tr><td style="author-callout-style-b3"><italic>ZNRF3</italic></td><td style="author-callout-style-b3">30.9</td><td style="author-callout-style-b3">33.3</td><td style="author-callout-style-b3">32.1</td></tr><tr><td style="author-callout-style-b3"><italic>RNF43</italic></td><td style="author-callout-style-b3">0.12</td><td style="author-callout-style-b3">0.08</td><td style="author-callout-style-b3">0.1</td></tr><tr><td style="author-callout-style-b1" valign="top"><italic>GPC1</italic></td><td style="author-callout-style-b1" valign="top">49.55</td><td style="author-callout-style-b1" valign="top">47.53</td><td style="author-callout-style-b1" valign="top">48.54</td></tr><tr><td style="author-callout-style-b1" valign="top"><italic>GPC2</italic></td><td style="author-callout-style-b1" valign="top">4.17</td><td style="author-callout-style-b1" valign="top">4.79</td><td style="author-callout-style-b1" valign="top">4.48</td></tr><tr><td style="author-callout-style-b1" valign="top"><italic>GPC3</italic></td><td style="author-callout-style-b1" valign="top">170.22</td><td style="author-callout-style-b1" valign="top">144.37</td><td style="author-callout-style-b1" valign="top">157.29</td></tr><tr><td style="author-callout-style-b1"><italic>GPC4</italic></td><td style="author-callout-style-b1">209.39</td><td style="author-callout-style-b1">229.86</td><td style="author-callout-style-b1">219.63</td></tr><tr><td style="author-callout-style-b1" valign="bottom"><italic>GPC5</italic></td><td style="author-callout-style-b1" valign="bottom">0.1</td><td style="author-callout-style-b1" valign="bottom">0.1</td><td style="author-callout-style-b1" valign="bottom">0.1</td></tr><tr><td style="author-callout-style-b1"><italic>GPC6</italic></td><td style="author-callout-style-b1">13.88</td><td style="author-callout-style-b1">14.90</td><td style="author-callout-style-b1">14.39</td></tr><tr><td style="author-callout-style-b3" valign="bottom"><italic>SDC1</italic></td><td style="author-callout-style-b3" valign="bottom">51.37</td><td style="author-callout-style-b3" valign="bottom">47.88</td><td style="author-callout-style-b3" valign="bottom">49.63</td></tr><tr><td style="author-callout-style-b3" valign="bottom"><italic>SDC2</italic></td><td style="author-callout-style-b3" valign="bottom">11.42</td><td style="author-callout-style-b3" valign="bottom">9.2</td><td style="author-callout-style-b3" valign="bottom">10.31</td></tr><tr><td style="author-callout-style-b3" valign="bottom"><italic>SDC3</italic></td><td style="author-callout-style-b3" valign="bottom">43.58</td><td style="author-callout-style-b3" valign="bottom">50.64</td><td style="author-callout-style-b3" valign="bottom">47.11</td></tr><tr><td style="author-callout-style-b3" valign="bottom"><italic>SDC4</italic></td><td style="author-callout-style-b3" valign="bottom">8.16</td><td style="author-callout-style-b3" valign="bottom">8.21</td><td style="author-callout-style-b3" valign="bottom">8.18</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-33126-v3.xml">elife-33126-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.33126.004</object-id><label>Table 1.</label><caption><title>Relative gene expression level in HAP1 cells of selected genes discussed in this work.</title><p>RPKM (<underline>R</underline>eads <underline>P</underline>er <underline>K</underline>ilobase of transcript per <underline>M</underline>illion mapped reads) values from duplicate RNAseq datasets described previously (NCBI GEO accession number GSE75515, <ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/geo/">https://www.ncbi.nlm.nih.gov/geo/</ext-link>),&#160;obtained from two different passages of WT HAP1 cells (<xref ref-type="bibr" rid="bib13">Dubey et al., 2016</xref>). Groups of paralogues or genes with redundant function are shaded in alternating colors to facilitate comparisons.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene</th><th colspan="3" valign="bottom">RPKM</th></tr><tr><th>Replicate 1</th><th>Replicate 2</th><th>Average</th></tr></thead><tbody><tr><td style="author-callout-style-b1"><italic>LGR4</italic></td><td style="author-callout-style-b1">160.61</td><td style="author-callout-style-b1">174.69</td><td style="author-callout-style-b1">167.65</td></tr><tr><td style="author-callout-style-b1"><italic>LGR5</italic></td><td style="author-callout-style-b1">0.02</td><td style="author-callout-style-b1">0.00</td><td style="author-callout-style-b1">0.01</td></tr><tr><td style="author-callout-style-b1"><italic>LGR6</italic></td><td style="author-callout-style-b1">0.02</td><td style="author-callout-style-b1">0.00</td><td style="author-callout-style-b1">0.01</td></tr><tr><td style="author-callout-style-b3"><italic>ZNRF3</italic></td><td style="author-callout-style-b3">30.9</td><td style="author-callout-style-b3">33.3</td><td style="author-callout-style-b3">32.1</td></tr><tr><td style="author-callout-style-b3"><italic>RNF43</italic></td><td style="author-callout-style-b3">0.12</td><td style="author-callout-style-b3">0.08</td><td style="author-callout-style-b3">0.1</td></tr><tr><td style="author-callout-style-b1" valign="top"><italic>GPC1</italic></td><td style="author-callout-style-b1" valign="top">49.55</td><td style="author-callout-style-b1" valign="top">47.53</td><td style="author-callout-style-b1" valign="top">48.54</td></tr><tr><td style="author-callout-style-b1" valign="top"><italic>GPC2</italic></td><td style="author-callout-style-b1" valign="top">4.17</td><td style="author-callout-style-b1" valign="top">4.79</td><td style="author-callout-style-b1" valign="top">4.48</td></tr><tr><td style="author-callout-style-b1" valign="top"><italic>GPC3</italic></td><td style="author-callout-style-b1" valign="top">170.22</td><td style="author-callout-style-b1" valign="top">144.37</td><td style="author-callout-style-b1" valign="top">157.29</td></tr><tr><td style="author-callout-style-b1"><italic>GPC4</italic></td><td style="author-callout-style-b1">209.39</td><td style="author-callout-style-b1">229.86</td><td style="author-callout-style-b1">219.63</td></tr><tr><td style="author-callout-style-b1" valign="bottom"><italic>GPC5</italic></td><td style="author-callout-style-b1" valign="bottom">0.1</td><td style="author-callout-style-b1" valign="bottom">0.1</td><td style="author-callout-style-b1" valign="bottom">0.1</td></tr><tr><td style="author-callout-style-b1"><italic>GPC6</italic></td><td style="author-callout-style-b1">13.88</td><td style="author-callout-style-b1">14.90</td><td style="author-callout-style-b1">14.39</td></tr><tr><td style="author-callout-style-b3" valign="bottom"><italic>SDC1</italic></td><td style="author-callout-style-b3" valign="bottom">51.37</td><td style="author-callout-style-b3" valign="bottom">47.88</td><td style="author-callout-style-b3" valign="bottom">49.63</td></tr><tr><td style="author-callout-style-b3" valign="bottom"><italic>SDC2</italic></td><td style="author-callout-style-b3" valign="bottom">11.42</td><td style="author-callout-style-b3" valign="bottom">9.2</td><td style="author-callout-style-b3" valign="bottom">10.31</td></tr><tr><td style="author-callout-style-b3" valign="bottom"><italic>SDC3</italic></td><td style="author-callout-style-b3" valign="bottom">43.58</td><td style="author-callout-style-b3" valign="bottom">50.64</td><td style="author-callout-style-b3" valign="bottom">47.11</td></tr><tr><td style="author-callout-style-b3" valign="bottom"><italic>SDC4</italic></td><td style="author-callout-style-b3" valign="bottom">8.16</td><td style="author-callout-style-b3" valign="bottom">8.21</td><td style="author-callout-style-b3" valign="bottom">8.18</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-33183-v1.xml">elife-33183-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.33183.012</object-id><label>Table&#160;2.</label><caption><title>Transmembrane reversal potentials (<italic>E</italic><sub>rev</sub>) measured from pIMCD and HEK-293 PKD2-GFP cilia.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">External solution</th><th colspan="2" valign="top">pIMCD, whole-cilium</th><th colspan="2" valign="top">HEK-293 PKD2-GFP, whole-cilium</th></tr><tr><th valign="top"><italic>&#916;E</italic><sub>rev</sub> (mean&#160;&#177;&#160;S.D.)</th><th valign="top">P<sub>x</sub>/P<sub>Na</sub> (mean&#160;&#177;&#160;S.D.)</th><th valign="top"><italic>&#916;E</italic><sub>rev</sub> (mean&#160;&#177;&#160;S.D.)</th><th valign="top">P<sub>x</sub>/P<sub>Na</sub> (mean&#160;&#177;&#160;S.D.)</th></tr></thead><tbody><tr><td valign="top">NaCl</td><td valign="top">0</td><td valign="top">1</td><td valign="top">0</td><td valign="top">1</td></tr><tr><td valign="top">NaMES</td><td valign="top">&#8722;2&#160;&#177;&#160;3 mV</td><td valign="top">0.96</td><td valign="top">Not tested</td><td valign="top">-</td></tr><tr><td valign="top">KCl</td><td valign="top">26&#160;&#177;&#160;3 mV</td><td valign="top">2.4&#160;&#177;&#160;0.3</td><td valign="top">23&#160;&#177;&#160;4 mV</td><td valign="top">2.4&#160;&#177;&#160;0.4</td></tr><tr><td valign="top">CaCl<sub>2</sub></td><td valign="top">&#8722;49&#160;&#177;&#160;4 mV</td><td valign="top">0.06&#160;&#177;&#160;0.04</td><td valign="top">&#8722;48&#160;&#177;&#160;3 mV</td><td valign="top">0.09&#160;&#177;&#160;0.04</td></tr><tr><td valign="top">NMDG</td><td valign="top">&#8722;57&#160;&#177;&#160;4 mV</td><td valign="top">0.04&#160;&#177;&#160;0.02</td><td valign="top">&#8722;58&#160;&#177;&#160;4 mV</td><td valign="top">0.04&#160;&#177;&#160;0.02</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-33321-v2.xml">elife-33321-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.33321.004</object-id><label>Table 1.</label><caption><title>Description of the 12 nodes showing larger-than-chance gastric-BOLD phase coupling.</title><p>AAL: Automated Anatomical labeling (<xref ref-type="bibr" rid="bib77">Tzourio-Mazoyer et al., 2002</xref>). MNI: Montreal Neurological Institute.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Node name</th><th rowspan="2">Monte-Carlo p</th><th rowspan="2">Sum (t)</th><th rowspan="2">Cluster <break/>volume <break/>in mm<sup>3</sup></th><th rowspan="2">Label AAL</th><th rowspan="2">mm<sup>3</sup></th><th rowspan="2">% Area in Cluster</th><th rowspan="2">Max t</th><th colspan="3">MNI of max t</th><th rowspan="2">EGG-BOLD shared Variance&#160;&#177;&#160;sem</th></tr><tr><th>X</th><th>Y</th><th>Z</th></tr></thead><tbody><tr><td>Primary Somatosensory Right (SIr)</td><td>0.0049</td><td>216.5</td><td>180</td><td>Postcentral R</td><td>171</td><td>8.8</td><td>4.8</td><td>45</td><td>&#8722;28</td><td>46</td><td>12.5%&#160;&#177;&#160;2.0</td></tr><tr><td rowspan="3">Secondary Somatosensory Right (SIIr)</td><td rowspan="3">0.0131</td><td rowspan="3">146.1</td><td rowspan="3">120</td><td>Rolandic Oper R</td><td>12</td><td>1.8</td><td>4.2</td><td>54</td><td>&#8722;22</td><td>13</td><td rowspan="3">15.7%&#160;&#177;&#160;2.2</td></tr><tr><td>Heschl R</td><td>6</td><td>4.7</td><td>3.9</td><td>51</td><td>&#8722;19</td><td>10</td></tr><tr><td>Temporal Sup R</td><td>99</td><td>6</td><td>5.3</td><td>57</td><td>&#8722;25</td><td>13</td></tr><tr><td rowspan="4">Secondary Somatosensory Left (SIIl)</td><td rowspan="4">0.0094</td><td rowspan="4">167.0</td><td rowspan="4">138</td><td>Postcentral L</td><td>39</td><td>2.0</td><td>4.5</td><td>&#8722;66</td><td>&#8722;22</td><td>22</td><td rowspan="4">16.6%&#160;&#177;&#160;2.2</td></tr><tr><td>SupraMarginal L</td><td>21</td><td>3.3</td><td>3.8</td><td>&#8722;60</td><td>&#8722;25</td><td>16</td></tr><tr><td>Heschl L</td><td>15</td><td>13.1</td><td>3.9</td><td>&#8722;54</td><td>&#8722;16</td><td>7</td></tr><tr><td>Temporal Sup L</td><td>60</td><td>5</td><td>5.2</td><td>&#8722;54</td><td>&#8722;19</td><td>7</td></tr><tr><td rowspan="4">Medial Wall Motor Regions (MWM)</td><td rowspan="4">0.0036</td><td rowspan="4">269.5</td><td rowspan="4">228</td><td>Supp Motor Area L</td><td>30</td><td>2.7</td><td>3.8</td><td>0</td><td>&#8722;13</td><td>49</td><td rowspan="4">15.8%&#160;&#177;&#160;2.1</td></tr><tr><td>Supp Motor Area R</td><td>54</td><td>4.5</td><td>4.8</td><td>9</td><td>-4</td><td>52</td></tr><tr><td>Cingulum Mid L</td><td>66</td><td>6.7</td><td>4.9</td><td>0</td><td>&#8722;22</td><td>43</td></tr><tr><td>Cingulum Mid R</td><td>75</td><td>6.7</td><td>4.7</td><td>6</td><td>&#8722;10</td><td>43</td></tr><tr><td rowspan="5">Posterior Cingulate Sulcus (pCS)</td><td rowspan="5">0.0061</td><td rowspan="5">196.7</td><td rowspan="5">171</td><td>Cingulum Mid L</td><td>39</td><td>3.9</td><td>4.2</td><td>-9</td><td>&#8722;37</td><td>52</td><td rowspan="5">16.9%&#160;&#177;&#160;2.3</td></tr><tr><td>Cingulum Mid R</td><td>24</td><td>2.1</td><td>4.5</td><td>6</td><td>&#8722;37</td><td>49</td></tr><tr><td>Precuneus L</td><td>21</td><td>1.2</td><td>3.9</td><td>0</td><td>&#8722;37</td><td>55</td></tr><tr><td>Precuneus R</td><td>39</td><td>2.3</td><td>3.8</td><td>6</td><td>&#8722;40</td><td>52</td></tr><tr><td>Paracentral Lobule R</td><td>39</td><td>9.1</td><td>4.1</td><td>15</td><td>&#8722;40</td><td>55</td></tr><tr><td rowspan="2">Dorsal Precuneus (dPrec)</td><td rowspan="2">0.0071</td><td rowspan="2">186.6</td><td rowspan="2">156</td><td>Precuneus L</td><td>42</td><td>2.3</td><td>4.9</td><td>-3</td><td>&#8722;67</td><td>61</td><td rowspan="2">12.7%&#160;&#177;&#160;1.9</td></tr><tr><td>Precuneus R</td><td>87</td><td>5.2</td><td>4.4</td><td>3</td><td>&#8722;64</td><td>64</td></tr><tr><td rowspan="2">Dorsal Precuneus Left Anterior (ladPrec)</td><td rowspan="2">0.0178</td><td rowspan="2">125.5</td><td rowspan="2">108</td><td>Precuneus L</td><td>57</td><td>3.2</td><td>4.1</td><td>-6</td><td>&#8722;55</td><td>73</td><td rowspan="2">12.0%&#160;&#177;&#160;2.0</td></tr><tr><td>Paracentral Lobule L</td><td>48</td><td>7.0</td><td>4.3</td><td>-6</td><td>&#8722;34</td><td>76</td></tr><tr><td rowspan="6">Occipital Ventral (vOcc)</td><td rowspan="6">0.0017</td><td rowspan="6">374.2</td><td rowspan="6">321</td><td>Calcarine L</td><td>18</td><td>1.6</td><td>4.1</td><td>0</td><td>&#8722;64</td><td>10</td><td rowspan="6">16.6%&#160;&#177;&#160;2.6</td></tr><tr><td>Lingual L</td><td>66</td><td>6</td><td>4.4</td><td>0</td><td>&#8722;67</td><td>7</td></tr><tr><td>Lingual R</td><td>108</td><td>9</td><td>4.4</td><td>6</td><td>&#8722;70</td><td>-5</td></tr><tr><td>Cerebellum 6 R</td><td>21</td><td>2.3</td><td>4.3</td><td>12</td><td>&#8722;70</td><td>&#8722;14</td></tr><tr><td>Vermis 4 5</td><td>51</td><td>15.0</td><td>4.6</td><td>0</td><td>&#8722;61</td><td>-2</td></tr><tr><td>Vermis 6</td><td>9</td><td>4.8</td><td>3.7</td><td>3</td><td>&#8722;70</td><td>-8</td></tr><tr><td rowspan="3">Occipital Dorsal (dOcc)</td><td rowspan="3">0.0034</td><td rowspan="3">285.1</td><td rowspan="3">243</td><td>Calcarine L</td><td>21</td><td>1.8</td><td>4</td><td>3</td><td>&#8722;76</td><td>16</td><td rowspan="3">16.7%&#160;&#177;&#160;2.7</td></tr><tr><td>Cuneus L</td><td>138</td><td>17.7</td><td>4.5</td><td>-3</td><td>&#8722;85</td><td>31</td></tr><tr><td>Cuneus R</td><td>72</td><td>9.9</td><td>4.9</td><td>6</td><td>&#8722;76</td><td>28</td></tr><tr><td rowspan="3">Extrastriate Body Area Right (EBA)</td><td rowspan="3">0.0103</td><td rowspan="3">163.3</td><td rowspan="3">138</td><td>Occipital Inf R</td><td>45</td><td>8.9</td><td>4.5</td><td>48</td><td>&#8722;76</td><td>&#8722;14</td><td rowspan="3">14.3%&#160;&#177;&#160;2.0</td></tr><tr><td>Temporal Inf R</td><td>27</td><td>1.5</td><td>4.4</td><td>57</td><td>&#8722;70</td><td>&#8722;11</td></tr><tr><td>Cerebellum Crus1 R</td><td>33</td><td>2.4</td><td>4.8</td><td>45</td><td>&#8722;82</td><td>&#8722;26</td></tr><tr><td rowspan="2">Superior Parieto-Occipital Sulcus (sPOS)</td><td rowspan="2">0.0245</td><td rowspan="2">107.1</td><td rowspan="2">87</td><td>Cuneus R</td><td>21</td><td>2.9</td><td>4.6</td><td>21</td><td>&#8722;76</td><td>43</td><td rowspan="2">12.5%&#160;&#177;&#160;1.8</td></tr><tr><td>Occipital Sup R&#160;</td><td>54</td><td>7.5</td><td>5.1</td><td>24</td><td>&#8722;79</td><td>43</td></tr><tr><td rowspan="4">Retrosplenial Cortex (RSC)</td><td rowspan="4">0.0084</td><td rowspan="4">175.6</td><td rowspan="4">147</td><td>Cingulum Post L</td><td>3</td><td>1.3</td><td>3.5</td><td>-6</td><td>&#8722;43</td><td>10</td><td rowspan="4">15.4%&#160;&#177;2.1</td></tr><tr><td>Cingulum Post R</td><td>9</td><td>5.3</td><td>3.7</td><td>6</td><td>&#8722;40</td><td>7</td></tr><tr><td>Lingual R</td><td>18</td><td>1.5</td><td>4.1</td><td>9</td><td>&#8722;37</td><td>-2</td></tr><tr><td>Precuneus L</td><td>36</td><td>2.0</td><td>4.4</td><td>-6</td><td>&#8722;49</td><td>7</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-33420-v2.xml">elife-33420-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.33420.003</object-id><label>Table 1.</label><caption><title>Candidate plasticity proteins (CPPs) include FMRP targets and autism spectrum disorder (ASD) genes.</title><p>CPPs were compared to FMRP targets identified by CLIP (<xref ref-type="bibr" rid="bib21">Darnell et al., 2011</xref>) and genes from the Simons Foundation and Autism Research Initiative (SFARI) database (gene.sfari.org) for potential activity-regulated FMRP targets and ASD genes. The final fold changes in AHA labeling is from the average of two independent experiments (<xref ref-type="supplementary-material" rid="supp2">Supplementary file 2</xref>). The 83 CPPs shown here are AHA-labeled newly synthesized proteins (NSPs) that showed consistent increases or decreases in synthesis in response to visual experience with at least 20% change in one of the experiments.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th rowspan="2">Gene <break/>names</th><th rowspan="2">Human <break/>Uniprot ID</th><th rowspan="2">Final Fold <break/>Change</th><th colspan="2">FMRP targets <break/>and ASD genes</th><th rowspan="2"/><th rowspan="2">Gene <break/>names</th><th rowspan="2">Human <break/>Uniprot ID</th><th rowspan="2">Final Fold <break/>Change</th><th colspan="2">FMRP targets <break/>and ASD genes</th></tr><tr><th>FMRP <break/>targets</th><th>SFARI <break/>database</th><th>FMRP <break/>targets</th><th>SFARI <break/>database</th></tr></thead><tbody><tr><td rowspan="12">Up in two <break/>experiments</td><td>capza1</td><td>P52907</td><td>1.36</td><td/><td/><td rowspan="12">Down in two <break/>experiments</td><td>cct7</td><td>Q99832</td><td>0.40</td><td/><td/></tr><tr><td>capza2</td><td>P47755</td><td>1.33</td><td/><td/><td>clasp1</td><td>Q7Z460</td><td>0.74</td><td>V</td><td/></tr><tr><td>cbx1</td><td>P83916</td><td>1.36</td><td/><td/><td>col18a1</td><td>P39060</td><td>0.71</td><td/><td/></tr><tr><td>cbx3</td><td>Q13185</td><td>1.36</td><td/><td/><td>ctnnb1</td><td>P35222</td><td>0.63</td><td>V</td><td>V</td></tr><tr><td>fus</td><td>P35637</td><td>1.36</td><td/><td/><td>papss1</td><td>O43252</td><td>0.67</td><td/><td/></tr><tr><td>hdlbp</td><td>Q00341</td><td>1.26</td><td>V</td><td/><td>psmc6</td><td>P62333</td><td>0.75</td><td/><td/></tr><tr><td>rps17</td><td>P08708</td><td>1.71</td><td/><td/><td>tln1</td><td>Q9Y490</td><td>0.71</td><td/><td/></tr><tr><td>tuba1b</td><td>P68363</td><td>1.95</td><td/><td/><td>atp2a2</td><td>P16615</td><td>0.66</td><td>V</td><td/></tr><tr><td>lonp1</td><td>P36776</td><td>1.31</td><td/><td/><td>pmp2</td><td>P02689</td><td>0.55</td><td/><td/></tr><tr><td>sfpq</td><td>P23246</td><td>1.54</td><td/><td/><td>acta1</td><td>P68133</td><td>0.73</td><td/><td/></tr><tr><td>ttpal</td><td>Q9BTX7</td><td>1.42</td><td/><td/><td>atp2a1</td><td>O14983</td><td>0.66</td><td/><td/></tr><tr><td>tuba1a</td><td>Q71U36</td><td>1.95</td><td/><td/><td>kif1a</td><td>Q12756</td><td>0.65</td><td>V</td><td/></tr><tr><td rowspan="23">Up in one <break/>experiment</td><td>cox5a</td><td>P20674</td><td>1.30</td><td/><td/><td rowspan="23">Down in one <break/>experiment</td><td>actb</td><td>P60709</td><td>0.78</td><td>V</td><td/></tr><tr><td>eif3a</td><td>Q14152</td><td>1.16</td><td/><td/><td>aplp2</td><td>Q06481</td><td>0.74</td><td/><td/></tr><tr><td>krt75</td><td>O95678</td><td>1.17</td><td/><td/><td>cand1</td><td>Q86VP6</td><td>0.83</td><td>V</td><td/></tr><tr><td>metap2</td><td>P50579</td><td>1.22</td><td/><td/><td>dnm1l</td><td>O00429</td><td>0.78</td><td/><td>V</td></tr><tr><td>naca</td><td>Q13765</td><td>1.38</td><td/><td/><td>hist1h4a</td><td>P62805</td><td>0.61</td><td/><td/></tr><tr><td>nono</td><td>Q15233</td><td>1.15</td><td/><td/><td>hnrnpa1</td><td>P09651</td><td>0.81</td><td/><td/></tr><tr><td>psmd2</td><td>Q13200</td><td>1.40</td><td/><td/><td>hnrnpc</td><td>P07910</td><td>0.76</td><td/><td/></tr><tr><td>rab5a</td><td>P20339</td><td>1.14</td><td/><td/><td>hsp90ab1</td><td>P08238</td><td>0.79</td><td>V</td><td/></tr><tr><td>rab5c</td><td>P51148</td><td>1.14</td><td/><td/><td>hspa5</td><td>P11021</td><td>0.68</td><td/><td/></tr><tr><td>eif4a1</td><td>P60842</td><td>1.18</td><td/><td/><td>mdh2</td><td>P40926</td><td>0.75</td><td/><td/></tr><tr><td>hnrnpab</td><td>Q99729</td><td>1.21</td><td/><td/><td>ncl</td><td>P19338</td><td>0.81</td><td/><td/></tr><tr><td>pbrm1</td><td>Q86U86</td><td>1.36</td><td/><td/><td>pcbp3</td><td>P57721</td><td>0.60</td><td/><td/></tr><tr><td>rab5b</td><td>P61020</td><td>1.20</td><td/><td/><td>snw1</td><td>Q13573</td><td>0.75</td><td/><td/></tr><tr><td>skiv2l2</td><td>P42285</td><td>1.25</td><td/><td/><td>actg1</td><td>P63261</td><td>0.79</td><td/><td/></tr><tr><td>ap2a2</td><td>O94973</td><td>1.15</td><td>V</td><td/><td>hn1</td><td>Q9UK76</td><td>0.82</td><td/><td/></tr><tr><td>iws1</td><td>Q96ST2</td><td>1.25</td><td/><td/><td>kif1b</td><td>O60333</td><td>0.81</td><td>V</td><td/></tr><tr><td>vim</td><td>P08670</td><td>1.19</td><td/><td/><td>pcbp2</td><td>Q15366</td><td>0.60</td><td/><td/></tr><tr><td>nsrp1</td><td>Q9H0G5</td><td>1.36</td><td/><td/><td>smc1a</td><td>Q14683</td><td>0.80</td><td/><td/></tr><tr><td>syp</td><td>P08247</td><td>1.17</td><td/><td/><td>sptan1</td><td>Q13813</td><td>0.84</td><td>V</td><td/></tr><tr><td>cxxc4</td><td>Q9H2H0</td><td>1.47</td><td/><td/><td>stip1</td><td>P31948</td><td>0.74</td><td/><td/></tr><tr><td>dpysl3</td><td>Q14195</td><td>1.19</td><td/><td/><td>trim69</td><td>Q86WT6</td><td>0.71</td><td/><td/></tr><tr><td>fasn</td><td>P49327</td><td>1.23</td><td>V</td><td/><td>arnt2</td><td>Q9HBZ2</td><td>0.78</td><td>V</td><td>V</td></tr><tr><td>kiaa1598</td><td>A0MZ66</td><td>1.16</td><td/><td/><td>cdh11</td><td>P55287</td><td>0.80</td><td/><td>V</td></tr><tr><td rowspan="3">Up in one <break/>experiment</td><td>krt7</td><td>P08729</td><td>1.17</td><td/><td/><td rowspan="10">Down in one <break/>experiment</td><td>hk1</td><td>P19367</td><td>0.80</td><td>V</td><td/></tr><tr><td>lgmn</td><td>Q99538</td><td>1.55</td><td/><td/><td>mcm4</td><td>P33991</td><td>0.76</td><td/><td>V</td></tr><tr><td>sptbn1</td><td>Q01082</td><td>1.18</td><td>V</td><td/><td>smarcd2</td><td>Q92925</td><td>0.84</td><td/><td/></tr><tr><td colspan="6" rowspan="7"/><td>stmn2</td><td>Q93045</td><td>0.70</td><td/><td/></tr><tr><td>acta2</td><td>P62736</td><td>0.80</td><td/><td/></tr><tr><td>actc1</td><td>P68032</td><td>0.80</td><td/><td/></tr><tr><td>kif1c</td><td>O43896</td><td>0.81</td><td/><td/></tr><tr><td>kif5c</td><td>O60282</td><td>0.80</td><td>V</td><td>V</td></tr><tr><td>smarcd1</td><td>Q96GM5</td><td>0.84</td><td/><td/></tr><tr><td>smarcd3</td><td>Q6STE5</td><td>0.84</td><td/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-33465-v2.xml">elife-33465-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.33465.007</object-id><label>Table 1.</label><caption><title>Data for selected proteins from mass spectrometry comparison of GFP-Mto1[9A1-NE]-HTB and GFP-Mto1[9A1-bonsai]-HTB interactomes.</title><p>Peptide counts and label-free quantification (LFQ) values for selected proteins shown in <xref ref-type="fig" rid="fig1">Figure 1F</xref>. Data from two independent biological replicates are shown. Nsp1 and Nup82 are included as likely representative Nup146 interactors, based on homology to budding yeast (<xref ref-type="bibr" rid="bib12">Belgareh et al., 1998</xref>). See also <xref ref-type="supplementary-material" rid="supp3">Supplementary file 3</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Protein</th><th colspan="5" valign="bottom">Replicate 1 (E160307)</th><th colspan="5" valign="bottom">Replicate 2 (E161126)</th></tr><tr><th valign="bottom">Peptides from strain KS7611</th><th valign="bottom">Peptides from strain KS8371</th><th valign="bottom">LFQ intensity from strain KS7611</th><th valign="bottom">LFQ intensity from strain KS8371</th><th valign="bottom">LFQ ratio</th><th valign="bottom">Peptides from strain KS7611</th><th valign="bottom">Peptides from strain KS8371</th><th valign="bottom">LFQ intensity from strain KS7611</th><th valign="bottom">LFQ intensity from strain KS8371</th><th valign="bottom">LFQ ratio</th></tr></thead><tbody><tr><td valign="bottom">&#8195;Alp7</td><td valign="bottom">17</td><td valign="bottom">3</td><td valign="bottom">2.3e8</td><td valign="bottom">7.5e6</td><td valign="bottom">30.5</td><td valign="bottom">14</td><td valign="bottom">5</td><td valign="bottom">1.9e8</td><td valign="bottom">3.7e7</td><td valign="bottom">5.2</td></tr><tr><td valign="bottom">&#8195;Crm1</td><td valign="bottom">20</td><td valign="bottom">5</td><td valign="bottom">2.7e8</td><td valign="bottom">1.7e7</td><td valign="bottom">15.6</td><td valign="bottom">18</td><td valign="bottom">5</td><td valign="bottom">2.7e8</td><td valign="bottom">2.0e7</td><td valign="bottom">13.7</td></tr><tr><td valign="bottom">&#8195;Mto1</td><td valign="bottom">59</td><td valign="bottom">46</td><td valign="bottom">2.8e11</td><td valign="bottom">2.3e11</td><td valign="bottom">1.2</td><td valign="bottom">52</td><td valign="bottom">43</td><td valign="bottom">2.8e11</td><td valign="bottom">2.5e11</td><td valign="bottom">1.1</td></tr><tr><td valign="bottom">&#8195;Mto2</td><td valign="bottom">14</td><td valign="bottom">20</td><td valign="bottom">6.6e9</td><td valign="bottom">5.1e9</td><td valign="bottom">1.3</td><td valign="bottom">16</td><td valign="bottom">22</td><td valign="bottom">6.7e9</td><td valign="bottom">5.8e9</td><td valign="bottom">1.2</td></tr><tr><td valign="bottom">&#8195;Nsp1</td><td valign="bottom">14</td><td valign="bottom">11</td><td valign="bottom">2.5e8</td><td valign="bottom">1.8e8</td><td valign="bottom">1.4</td><td valign="bottom">14</td><td valign="bottom">12</td><td valign="bottom">2.2e8</td><td valign="bottom">1.3e8</td><td valign="bottom">1.7</td></tr><tr><td valign="bottom">&#8195;Nup146</td><td valign="bottom">20</td><td valign="bottom">2</td><td valign="bottom">2.1e8</td><td valign="bottom">7.7e6</td><td valign="bottom">27.3</td><td valign="bottom">20</td><td valign="bottom">1</td><td valign="bottom">2.1e8</td><td valign="bottom">NQ</td><td valign="bottom">NQ</td></tr><tr><td valign="bottom">&#8195;Nup82</td><td valign="bottom">13</td><td valign="bottom">9</td><td valign="bottom">1.5e8</td><td valign="bottom">7.3e7</td><td valign="bottom">2.1</td><td valign="bottom">9</td><td valign="bottom">4</td><td valign="bottom">7.4e7</td><td valign="bottom">2.2e7</td><td valign="bottom">3.4</td></tr><tr><td valign="bottom">&#8195;Plo1</td><td valign="bottom">28</td><td valign="bottom">13</td><td valign="bottom">4.9e8</td><td valign="bottom">1.1e8</td><td valign="bottom">4.6</td><td valign="bottom">20</td><td valign="bottom">9</td><td valign="bottom">3.3e8</td><td valign="bottom">6.3e7</td><td valign="bottom">5.3</td></tr></tbody></table><table-wrap-foot><fn><p>NQ&#160;=&#160;not quantified, because peptide count in the relevant sample was below threshold for quantification.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-33465-v2.xml">elife-33465-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.33465.013</object-id><label>Table 2.</label><caption><title>Data for selected proteins from mass spectrometry comparison of GFP-Mto1[9A1-NE]-HTB interactomes from control cells and after treatment with leptomycin B.</title><p>Peptide counts and label-free quantification (LFQ) values for selected proteins from the two replicate experiments contributing to the graph in <xref ref-type="fig" rid="fig3">Figure 3C</xref>. See also <xref ref-type="supplementary-material" rid="supp4">Supplementary file 4</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Protein</th><th colspan="5" valign="bottom">Replicate 1 (E150924)</th><th colspan="5" valign="bottom">Replicate 2 (E151106)</th></tr><tr><th valign="bottom">Peptides from strain KS7669 -LMB</th><th valign="bottom">Peptides from strain KS7669 <break/>+LMB</th><th valign="bottom">LFQ intensity from strain KS7669 -LMB</th><th valign="bottom">LFQ intensity from strain KS7669&#160;+&#160;LMB</th><th valign="bottom">LFQ ratio</th><th valign="bottom">Peptides from strain KS7669-LMB</th><th valign="bottom">Peptides from strain KS7669 <break/>+LMB</th><th valign="bottom">LFQ intensity from strain KS7669 -LMB</th><th valign="bottom">LFQ intensity from strain KS7669&#160;+&#160;LMB</th><th valign="bottom">LFQ ratio</th></tr></thead><tbody><tr><td valign="bottom">&#8195;Alp7</td><td valign="bottom">20</td><td valign="bottom">13</td><td valign="bottom">2.5e8</td><td valign="bottom">1.4e8</td><td valign="bottom">1.7</td><td valign="bottom">14</td><td valign="bottom">15</td><td valign="bottom">6.3e8</td><td valign="bottom">4.4e8</td><td valign="bottom">1.4</td></tr><tr><td valign="bottom">&#8195;Crm1</td><td valign="bottom">17</td><td valign="bottom">4</td><td valign="bottom">1.1e8</td><td valign="bottom">1.3e7</td><td valign="bottom">9.0</td><td valign="bottom">19</td><td valign="bottom">3</td><td valign="bottom">2.9e8</td><td valign="bottom">6.8e6</td><td valign="bottom">43.3</td></tr><tr><td valign="bottom">&#8195;Mto1</td><td valign="bottom">59</td><td valign="bottom">58</td><td valign="bottom">2.6e11</td><td valign="bottom">2.9e11</td><td valign="bottom">0.9</td><td valign="bottom">58</td><td valign="bottom">59</td><td valign="bottom">6.1e11</td><td valign="bottom">6.7e11</td><td valign="bottom">0.9</td></tr><tr><td valign="bottom">&#8195;Mto2</td><td valign="bottom">7</td><td valign="bottom">7</td><td valign="bottom">2.0e8</td><td valign="bottom">2.2e8</td><td valign="bottom">0.9</td><td valign="bottom">5</td><td valign="bottom">5</td><td valign="bottom">6.5e8</td><td valign="bottom">8.4e8</td><td valign="bottom">0.8</td></tr><tr><td valign="bottom">&#8195;Nsp1</td><td valign="bottom">7</td><td valign="bottom">7</td><td valign="bottom">6.9e7</td><td valign="bottom">6.0e7</td><td valign="bottom">1.2</td><td valign="bottom">11</td><td valign="bottom">11</td><td valign="bottom">3.7e8</td><td valign="bottom">3.7e8</td><td valign="bottom">1.0</td></tr><tr><td valign="bottom">&#8195;Nup146</td><td valign="bottom">14</td><td valign="bottom">8</td><td valign="bottom">7.4e7</td><td valign="bottom">3.3e7</td><td valign="bottom">2.3</td><td valign="bottom">12</td><td valign="bottom">5</td><td valign="bottom">1.9e8</td><td valign="bottom">5.6e7</td><td valign="bottom">3.4</td></tr><tr><td valign="bottom">&#8195;Nup82</td><td valign="bottom">9</td><td valign="bottom">10</td><td valign="bottom">4.5e7</td><td valign="bottom">7.6e7</td><td valign="bottom">0.6</td><td valign="bottom">9</td><td valign="bottom">9</td><td valign="bottom">1.7e8</td><td valign="bottom">2.9e8</td><td valign="bottom">0.6</td></tr><tr><td valign="bottom">&#8195;Plo1</td><td valign="bottom">26</td><td valign="bottom">26</td><td valign="bottom">2.4e8</td><td valign="bottom">2.4e8</td><td valign="bottom">1.0</td><td valign="bottom">25</td><td valign="bottom">27</td><td valign="bottom">8.5e8</td><td valign="bottom">7.7e8</td><td valign="bottom">1.1</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-33465-v2.xml">elife-33465-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.33465.015</object-id><label>Table 3.</label><caption><title>Data for selected proteins from mass spectrometry comparison of GFP-Mto1[9A1-NE]-HTB and GFP-Mto1[&#8710;NES-9A1-NE]-HTB interactomes.</title><p>Peptide counts and label-free quantification (LFQ) values for selected proteins shown in <xref ref-type="fig" rid="fig4">Figure 4D</xref>. Data from two independent biological replicates are shown. See also <xref ref-type="supplementary-material" rid="supp5">Supplementary file 5</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Protein</th><th colspan="5" valign="bottom">Replicate 1 (E160419)</th><th colspan="5" valign="bottom">Replicate 2 ((E161127)</th></tr><tr><th valign="bottom">Peptides from strain KS7611</th><th valign="bottom">Peptides from strain KS8573</th><th valign="bottom">LFQ intensity from strain KS7611</th><th valign="bottom">LFQ intensity from strain KS8573</th><th valign="bottom">LFQ ratio</th><th valign="bottom">Peptides from strain KS7611</th><th valign="bottom">Peptides from strain KS8573</th><th valign="bottom">LFQ intensity from strain KS7611</th><th valign="bottom">LFQ intensity from strain KS8573</th><th valign="bottom">LFQ ratio</th></tr></thead><tbody><tr><td valign="bottom">&#8195;Alp7</td><td valign="bottom">15</td><td valign="bottom">14</td><td valign="bottom">2.2e8</td><td valign="bottom">1.4e8</td><td valign="bottom">1.5</td><td valign="bottom">12</td><td valign="bottom">9</td><td valign="bottom">2.1e8</td><td valign="bottom">1.4e8</td><td valign="bottom">1.5</td></tr><tr><td valign="bottom">&#8195;Crm1</td><td valign="bottom">21</td><td valign="bottom">2</td><td valign="bottom">2.9e8</td><td valign="bottom">3.4e6</td><td valign="bottom">85.2</td><td valign="bottom">20</td><td valign="bottom">2</td><td valign="bottom">3.5e8</td><td valign="bottom">5.5e6</td><td valign="bottom">63.4</td></tr><tr><td valign="bottom">&#8195;Mto1</td><td valign="bottom">47</td><td valign="bottom">47</td><td valign="bottom">3.1e11</td><td valign="bottom">3.2e11</td><td valign="bottom">1.0</td><td valign="bottom">40</td><td valign="bottom">40</td><td valign="bottom">3.1e11</td><td valign="bottom">3.2e11</td><td valign="bottom">1.0</td></tr><tr><td valign="bottom">&#8195;Mto2</td><td valign="bottom">10</td><td valign="bottom">10</td><td valign="bottom">4.8e9</td><td valign="bottom">4.1e9</td><td valign="bottom">1.2</td><td valign="bottom">8</td><td valign="bottom">8</td><td valign="bottom">6.3e9</td><td valign="bottom">5.9e9</td><td valign="bottom">1.1</td></tr><tr><td valign="bottom">&#8195;Nsp1</td><td valign="bottom">10</td><td valign="bottom">9</td><td valign="bottom">2.2e8</td><td valign="bottom">1.2e8</td><td valign="bottom">1.8</td><td valign="bottom">10</td><td valign="bottom">9</td><td valign="bottom">2.4e8</td><td valign="bottom">1.0e8</td><td valign="bottom">2.4</td></tr><tr><td valign="bottom">&#8195;Nup146</td><td valign="bottom">15</td><td valign="bottom">3</td><td valign="bottom">2.0e8</td><td valign="bottom">9.5e6</td><td valign="bottom">20.7</td><td valign="bottom">19</td><td valign="bottom">0</td><td valign="bottom">1.5e8</td><td valign="bottom">NQ</td><td valign="bottom">NQ</td></tr><tr><td valign="bottom">&#8195;Nup82</td><td valign="bottom">10</td><td valign="bottom">7</td><td valign="bottom">8.1e7</td><td valign="bottom">2.2e7</td><td valign="bottom">3.7</td><td valign="bottom">6</td><td valign="bottom">2</td><td valign="bottom">5.1e7</td><td valign="bottom">NQ</td><td valign="bottom">NQ</td></tr><tr><td valign="bottom">&#8195;Plo1</td><td valign="bottom">17</td><td valign="bottom">15</td><td valign="bottom">2.5e8</td><td valign="bottom">1.6e8</td><td valign="bottom">1.5</td><td valign="bottom">17</td><td valign="bottom">10</td><td valign="bottom">2.2e8</td><td valign="bottom">1.8e8</td><td valign="bottom">1.2</td></tr></tbody></table><table-wrap-foot><fn><p>NQ&#160;=&#160;not quantified, because peptide count and/or LFQ intensity in the relevant samples was below threshold for quantification.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-33465-v2.xml">elife-33465-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.33465.023</object-id><label>Table 4.</label><caption><title>Data for selected proteins from mass spectrometry comparison of GFP-Mto1[9A1-NE]-HTB interactomes in wild-type (<italic>nup146+</italic>) and <italic>nup146[&#8710;FG5-12]</italic> cells.</title><p>Peptide counts and label-free quantification (LFQ) values for selected proteins from the two replicate experiments contributing to the graph in <xref ref-type="fig" rid="fig6">Figure 6F</xref>. Complete datasets are in <xref ref-type="supplementary-material" rid="supp6">Supplementary file 6</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Protein</th><th colspan="5" valign="bottom">Replicate 1 (E170214)</th><th colspan="5" valign="bottom">Replicate 2 (E170306)</th></tr><tr><th valign="bottom">Peptides from strain KS9021</th><th valign="bottom">Peptides from strain KS9077</th><th valign="bottom">LFQ intensity from strain KS9021</th><th valign="bottom">LFQ intensity from strain KS9077</th><th valign="bottom">LFQ ratio</th><th valign="bottom">Peptides from strain KS9021</th><th valign="bottom">Peptides from strain KS9077</th><th valign="bottom">LFQ intensity from strain KS9021</th><th valign="bottom">LFQ intensity from strain KS9077</th><th valign="bottom">LFQ ratio</th></tr></thead><tbody><tr><td valign="bottom">&#8195;Alp7</td><td valign="bottom">17</td><td valign="bottom">15</td><td valign="bottom">1.4e8</td><td valign="bottom">1.4e8</td><td valign="bottom">1.0</td><td valign="bottom">16</td><td valign="bottom">16</td><td valign="bottom">1.0e8</td><td valign="bottom">8.2e7</td><td valign="bottom">1.3</td></tr><tr><td valign="bottom">&#8195;Crm1</td><td valign="bottom">31</td><td valign="bottom">17</td><td valign="bottom">3.4e8</td><td valign="bottom">9.2e7</td><td valign="bottom">3.6</td><td valign="bottom">27</td><td valign="bottom">18</td><td valign="bottom">1.7e8</td><td valign="bottom">6.1e7</td><td valign="bottom">2.9</td></tr><tr><td valign="bottom">&#8195;Mto1</td><td valign="bottom">62</td><td valign="bottom">61</td><td valign="bottom">2.7e11</td><td valign="bottom">2.6e11</td><td valign="bottom">1.0</td><td valign="bottom">55</td><td valign="bottom">54</td><td valign="bottom">1.7e11</td><td valign="bottom">1.8e11</td><td valign="bottom">0.9</td></tr><tr><td valign="bottom">&#8195;Mto2</td><td valign="bottom">12</td><td valign="bottom">12</td><td valign="bottom">6.0e9</td><td valign="bottom">5.0e9</td><td valign="bottom">1.2</td><td valign="bottom">13</td><td valign="bottom">13</td><td valign="bottom">3.8e9</td><td valign="bottom">3.1e9</td><td valign="bottom">1.2</td></tr><tr><td valign="bottom">&#8195;Nsp1</td><td valign="bottom">11</td><td valign="bottom">3</td><td valign="bottom">8.3e7</td><td valign="bottom">2.7e7</td><td valign="bottom">3.0</td><td valign="bottom">9</td><td valign="bottom">4</td><td valign="bottom">3.5e7</td><td valign="bottom">1.7e7</td><td valign="bottom">2.1</td></tr><tr><td valign="bottom">&#8195;Nup146</td><td valign="bottom">29</td><td valign="bottom">6</td><td valign="bottom">2.1e8</td><td valign="bottom">1.5e7</td><td valign="bottom">13.9</td><td valign="bottom">31</td><td valign="bottom">10</td><td valign="bottom">1.4e8</td><td valign="bottom">1.5e7</td><td valign="bottom">9.2</td></tr><tr><td valign="bottom">&#8195;Nup82</td><td valign="bottom">13</td><td valign="bottom">10</td><td valign="bottom">1.1e8</td><td valign="bottom">4.0e7</td><td valign="bottom">2.8</td><td valign="bottom">12</td><td valign="bottom">11</td><td valign="bottom">4.9e7</td><td valign="bottom">2.5e7</td><td valign="bottom">2.0</td></tr><tr><td valign="bottom">&#8195;Plo1</td><td valign="bottom">17</td><td valign="bottom">17</td><td valign="bottom">1.5e8</td><td valign="bottom">1.3e8</td><td valign="bottom">1.1</td><td valign="bottom">17</td><td valign="bottom">17</td><td valign="bottom">1.1e8</td><td valign="bottom">8.1e7</td><td valign="bottom">1.3</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-33953-v1.xml">elife-33953-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.33953.036</object-id><label>Table 1.</label><caption><title>Primers and constructs for heterologous expression of Cgr proteins in <italic>R. erythropolis</italic>.</title><p>Restriction sites are bolded.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Construct</th><th>For/</th><th rowspan="2">Sequence</th><th rowspan="2">Restriction sites</th><th rowspan="2">Vector</th><th rowspan="2">Anneal temp (&#176;C)</th><th rowspan="2">Extend <break/>time (s)</th></tr><tr><th>Rev</th></tr></thead><tbody><tr><td rowspan="2"><italic>cgr</italic> operon</td><td>For</td><td>ACTGACCCATGGATGGAATACGGAAAGTGCC</td><td rowspan="2">n/a</td><td rowspan="2">n/a</td><td rowspan="2">71</td><td rowspan="2">75</td></tr><tr><td>Rev</td><td>GTTTTACTGCAGTTACGCCGCCGTCGAA</td></tr><tr><td rowspan="2">Cgr1&#160;+&#160;Cgr2</td><td>For</td><td>TGAC<bold>GAATTC</bold>TAATGGAATACGGAAAGTGCCG</td><td>EcoRI,</td><td rowspan="2">pTipQT2</td><td rowspan="2">70</td><td rowspan="2">90</td></tr><tr><td>Rev</td><td>TTATA<bold>AGATCT</bold>CGCCGCCGTCGAAAG</td><td>BglII</td></tr><tr><td rowspan="2">Cgr1</td><td>For</td><td>TCGAA<bold>CATATG</bold>ATGGCTGAGGAACCTGTGG</td><td>NdeI,</td><td rowspan="2">pTipQT1</td><td rowspan="2">65</td><td rowspan="2">60</td></tr><tr><td>Rev</td><td>ATAA<bold>CTCGAG</bold>TCACGCCGCCGTCGAAA</td><td>XhoI</td></tr><tr><td rowspan="2">Cgr2 (native)</td><td>For</td><td>ACTGAC<bold>CCATGG</bold>GCATGGAATACGGAAAGTGCC</td><td rowspan="2">NcoI, HindIII</td><td rowspan="2">pTipQC2</td><td rowspan="2">65</td><td rowspan="2">60</td></tr><tr><td>Rev</td><td>ATTAG<bold>AAGCTT</bold>TCACTCCCACGGCTCGAG</td></tr><tr><td rowspan="2">Cgr2-CHis<sub>6</sub> (native)</td><td>For</td><td>ACTGAC<bold>CCATGG</bold>GCATGGAATACGGAAAGTGCC</td><td rowspan="2">NcoI, HindIII</td><td rowspan="2">pTipQC1</td><td rowspan="2">65</td><td rowspan="2">60</td></tr><tr><td>Rev</td><td>GTTAG<bold>AAGCTT</bold>CTCCCACGGCTCGAG</td></tr><tr><td rowspan="2">Cgr2(&#8722;48aa)-NHis<sub>6</sub></td><td>For</td><td>TATTA<bold>CCATGG</bold>ATCAGACCGCGCCTGC</td><td rowspan="2">NcoI, HindIII</td><td rowspan="2">pTipQC2</td><td rowspan="2">65</td><td rowspan="2">60</td></tr><tr><td>Rev</td><td>ATACT<bold>AAGCTT</bold>CTCCCACGGCTCGA</td></tr><tr><td rowspan="2">Cgr2(&#8722;48aa)-CHis<sub>6</sub></td><td>For</td><td>TATTA<bold>CCATGG</bold>ATCAGACCGCGCCTGC</td><td rowspan="2">NcoI, HindIII</td><td rowspan="2">pTipQC1</td><td rowspan="2">65</td><td rowspan="2">60</td></tr><tr><td>Rev</td><td>ATACT<bold>AAGCTT</bold>TTACTCCCACGGCTCGA</td></tr><tr><td rowspan="2">Sequencing primers</td><td>For</td><td>CGTGGCACGCGGAAC</td><td rowspan="2">n/a</td><td rowspan="2">All pTip vectors</td><td rowspan="2">n/a</td><td rowspan="2">n/a</td></tr><tr><td>Rev</td><td>GTGCAGGTTTCGCGTG</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-34286-v1.xml">elife-34286-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.34286.016</object-id><label>Table 1.</label><caption><title>Salivary secretion and the transgene enzyme activities from the unilateral parotid gland of transgenic (TG) pigs (Line2) during the grower (92&#8211;96 days old; estimated body weight: 42&#8211;45 kg) and finisher (159&#8211;191 days old; estimated body weight: 100&#8211;115 kg) phases of growth.</title><p><supplementary-material id="table1sdata1"><object-id pub-id-type="doi">10.7554/eLife.34286.017</object-id><label>Table 1&#8211;Source data 1.</label><caption><title>Salivary secretion from the unilateral parotid gland in the growing transgenic (TG) pigs and wild type (WT) pigs during the grower phase (92&#8211;96 days old; estimated body weight of 42&#8211;45 kg).</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-34286-table1-data1-v1.xlsx"/></supplementary-material></p><p><supplementary-material id="table1sdata2"><object-id pub-id-type="doi">10.7554/eLife.34286.018</object-id><label>Table 1&#8211;Source data 2.</label><caption><title>Salivary secretion from the unilateral parotid gland in the transgenic (TG) pigs during the finisher phase (159&#8211;191 days old; estimated body weight of 100&#8211;115 kg).</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-34286-table1-data2-v1.xlsx"/></supplementary-material></p><p><supplementary-material id="table1sdata3"><object-id pub-id-type="doi">10.7554/eLife.34286.019</object-id><label>Table 1&#8211;Source data 3.</label><caption><title>The transgene enzyme activities from the unilateral parotid gland in the transgenic (TG) pigs during the grower (92&#8211;96 days old; estimated body weight of 42&#8211;45 kg) and the finisher phases (159&#8211;191 days old; estimated body weight of 100&#8211;115 kg).</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-34286-table1-data3-v1.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Item</th><th colspan="3">Growth stages</th><th rowspan="2">Pooled <break/>SEM</th><th rowspan="2"><italic>P</italic> values</th></tr><tr><th>Grower</th><th colspan="2">Finisher</th></tr></thead><tbody><tr><td rowspan="2" valign="top">Saliva secreted <sup>*</sup> <break/>Saliva secretion rate (mL/min&#183;pig)</td><td valign="top"/><td colspan="2" valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">8.48</td><td colspan="2" valign="top">16.97<sup>**</sup></td><td valign="top">0.25</td><td valign="top">&lt;0.0001</td></tr><tr><td valign="top">Saliva secretion volume <break/>(mL/kg diet consumed)</td><td valign="top">407.66<sup>**</sup></td><td colspan="2" valign="top">151.70</td><td valign="top">5.33</td><td valign="top">&lt;0.0001</td></tr><tr><td colspan="6" valign="top">Enzymes activity secreted <break/>(U/mL saliva)<sup>&#8224;</sup></td></tr><tr><td valign="top">&#946;-glucanase</td><td colspan="2" valign="top">5.72</td><td valign="top">6.07<sup>**</sup></td><td valign="top">0.05</td><td valign="top">&lt;0.0001</td></tr><tr><td valign="top">Xylanase</td><td colspan="2" valign="top">5.92</td><td valign="top">6.19<sup>**</sup></td><td valign="top">0.05</td><td valign="top">0.0005</td></tr><tr><td valign="top">Phytase</td><td colspan="2" valign="top">7.2</td><td valign="top">6.87</td><td valign="top">0.14</td><td valign="top">0.1033</td></tr><tr><td colspan="6" valign="top">Enzyme activity secreted <break/>(U/kg diet consumed)<sup>&#8224;</sup></td></tr><tr><td valign="top">&#946;-glucanase</td><td colspan="2" valign="top">2,331.84<sup>**</sup></td><td valign="top">920.82</td><td valign="top">30.7</td><td valign="top">&lt;0.0001</td></tr><tr><td valign="top">Xylanase</td><td colspan="2" valign="top">2,413.38<sup>**</sup></td><td valign="top">939.03</td><td valign="top">31.73</td><td valign="top">&lt;0.0001</td></tr><tr><td valign="top">Phytase</td><td colspan="2" valign="top">2,935.19<sup>**</sup></td><td valign="top">1,042.19</td><td valign="top">38.29</td><td valign="top">&lt;0.0001</td></tr></tbody></table><table-wrap-foot><fn><p>*Saliva samples were collected daily from two transgenic growing pigs and two transgenic finishing pigs at 9:00 and 16:00, respectively, over the four-day period. Values are expressed as the mean and pooled SEM (n&#160;=&#160;16 repeated sampling and measurements).</p><p>&#8224;Means and pooled SEM (n&#160;=&#160;16) repeated sampling and measurements). Asterisks indicate significant differences between the grower and the finisher phases within the same row (Unpaired t-test, **p&lt;0.01).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-34801-v2.xml">elife-34801-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.34801.004</object-id><label>Table 3.</label><caption><title>Sample milestones for one of the subcompetencies within Critical Thinking Skills.&#160;Verbs in bold font indicate observable behaviors representing each stage of skill acquisition.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">CRITICAL THINKING SKILLS</th><th colspan="5" valign="top">MILESTONES</th></tr><tr><th valign="bottom">Beginning PhD Student</th><th valign="bottom">Advanced PhD Student</th><th valign="bottom">PhD Graduate</th><th valign="bottom">Early-Career Scientist</th><th valign="bottom">Science Professional</th></tr></thead><tbody><tr><td><bold>B. Design a single</bold> <break/><bold>experiment (answer</bold> <break/><bold>questions, controls, <italic>etc</italic>.)</bold></td><td><bold>Follow</bold> experimental <break/>protocols, <bold>seek</bold> help as <break/>needed, <bold>describe</bold> critical <break/>role of controls</td><td><bold>Plan</bold> experimental protocol; <break/><bold>include</bold> relevant controls; <break/><bold>choose</bold> appropriate methods; <break/><bold>troubleshoot</bold> experimental <break/>problems</td><td><bold>Design and execute</bold> hypothesis- <break/>based experiments <break/>independently; <bold>evaluate</bold> <break/>protocols of others; <break/><bold>predict</bold> range of <break/>experimental outcomes</td><td>Consistently <bold>design and</bold> <break/><bold>execute</bold> experiments with <break/>appropriate controls; <break/><bold>assess</bold> next steps; <bold>critique</bold> <break/>experiments of others</td><td><bold>Teach</bold> experimental <break/>design; <break/><bold>guide</bold> others doing <break/>experiments</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35122-v2.xml">elife-35122-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.35122.008</object-id><label>Table 1.</label><caption><title>List of differentially expressed genes between Hv_Sym and Hv_Apo, which are likely to be involved in symbiotic relationship</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Probename</th><th colspan="3">Fold change</th><th colspan="2" rowspan="2">Human_BestHit</th><th rowspan="2">blast2GO_Description</th></tr><tr><th>Hv_Sym <break/>/Hv_Apo</th><th>Hv_Sym_sexy <break/>/Hv_Apo</th><th>Hv_NC64A <break/>/Hv_Sym</th></tr></thead><tbody><tr><td colspan="7">Localization and Transport</td></tr><tr><td colspan="7">Hv_Sym&#160;&gt;&#160;Hv_Apo</td></tr><tr><td>rc_6788</td><td>9.87</td><td>8.00</td><td>1.01</td><td colspan="2"/><td>helicase conserved c-terminal <break/>domain containing protein</td></tr><tr><td>rc_10246</td><td>8.26</td><td>5.15</td><td>1.82</td><td colspan="2"/><td>protein</td></tr><tr><td>rc_6298</td><td>7.10</td><td>4.73</td><td>0.99</td><td colspan="2">hypothetical protein LOC220081</td><td>protein fam194b</td></tr><tr><td>2268</td><td>6.96</td><td>3.58</td><td>1.26</td><td colspan="2">protein Daple</td><td>viral a-type inclusion protein</td></tr><tr><td>10548</td><td>6.74</td><td>6.89</td><td>0.73</td><td colspan="2">transient receptor potential <break/>cation channel subfamily M <break/>member three isoform d</td><td>transient receptor potential cation <break/>channel subfamily m member 3-like</td></tr><tr><td>rc_1290</td><td>6.44</td><td>7.18</td><td>0.99</td><td colspan="2">tetratricopeptide repeat protein <break/>eight isoform B</td><td>tetratricopeptide repeat protein 8</td></tr><tr><td>18736</td><td>6.04</td><td>6.34</td><td>1.03</td><td colspan="2">BTB/POZ domain-containing <break/>protein KCTD9</td><td>btb poz domain-containing protein <break/>kctd9-like; unnamed protein product</td></tr><tr><td>rc_9270</td><td>5.96</td><td>10.03</td><td>1.37</td><td colspan="2">PREDICTED: hypothetical <break/>protein LOC100131693</td><td>eukaryotic translation initiation factor 4e</td></tr><tr><td>NPNHRC_15697</td><td>3.85</td><td>2.74</td><td>0.62</td><td colspan="2"/><td>major facilitator superfamily domain- <break/>containing protein 1</td></tr><tr><td>290</td><td>3.68</td><td>3.73</td><td>1.32</td><td colspan="2">splicing factor, arginine/ <break/>serine-rich 6</td><td>splicing arginine serine-rich 4</td></tr><tr><td>rc_9596</td><td>3.56</td><td>4.19</td><td>1.62</td><td colspan="2">BTB/POZ domain-containing <break/>protein KCTD10</td><td>btb poz domain-containing adapter for <break/>cul3-mediated degradation protein 3</td></tr><tr><td>rc_6774</td><td>3.34</td><td>3.32</td><td>1.31</td><td colspan="2">solute carrier family 43, <break/>member 2</td><td>large neutral amino acids transporter <break/>small subunit 4</td></tr><tr><td>rc_26218</td><td>3.29</td><td>2.91</td><td>0.41</td><td colspan="2">sodium-dependent phosphate <break/>transport protein 2A isoform 1</td><td>sodium-dependent phosphate transport <break/>protein 2b</td></tr><tr><td>NPNHRC_26094</td><td>3.20</td><td>3.98</td><td>1.31</td><td colspan="2">SPE-39 proteinid="T5"</td><td>spe-39 protein</td></tr><tr><td>9096</td><td>3.10</td><td>2.20</td><td>0.69</td><td colspan="2">otoferlin isoform d</td><td>otoferlin</td></tr><tr><td>rc_21349</td><td>2.89</td><td>4.25</td><td>0.78</td><td colspan="2">5'-AMP-activated protein kinase <break/>catalytic subunit alpha-2</td><td>5 -amp-activated protein kinase catalytic <break/>subunit alpha-2</td></tr><tr><td>npRC_14488</td><td>2.88</td><td>2.65</td><td>0.71</td><td colspan="2">solute carrier family 2, facilitated <break/>glucose transporter member 8</td><td>solute carrier family facilitated glucose <break/>transporter member 8-like</td></tr><tr><td>8863</td><td>2.75</td><td>2.70</td><td>0.81</td><td colspan="2">ATP-binding cassette, sub-family B, <break/>member 10 precursor</td><td>abc transporter b family protein</td></tr><tr><td>rc_11896</td><td>2.49</td><td>2.56</td><td>1.52</td><td colspan="2">ATP-binding cassette, sub-family B, <break/>member 10 precursor</td><td>abc transporter b family member 25-like</td></tr><tr><td>rc_6842</td><td>2.41</td><td>3.35</td><td>1.59</td><td colspan="2">hypothetical protein LOC112752 <break/>isoform 2</td><td>intraflagellar transport protein 43 homolog</td></tr><tr><td>5242</td><td>2.36</td><td>3.35</td><td>1.22</td><td colspan="2">growth arrest-specific protein 8</td><td>growth arrest-specific protein 8</td></tr><tr><td>5815</td><td>2.23</td><td>2.47</td><td>0.78</td><td colspan="2">plasma membrane calcium- <break/>transporting ATPase 4 isoform 4a</td><td>plasma membrane calcium atpase</td></tr><tr><td>8765</td><td>2.22</td><td>3.25</td><td>0.91</td><td colspan="2">growth arrest-specific protein 8</td><td>growth arrest-specific protein 8</td></tr><tr><td>NPNH_14052</td><td>2.19</td><td>2.17</td><td>0.79</td><td colspan="2">V-type proton ATPase 21 kDa <break/>proteolipid subunit isoform 2</td><td>v-type proton atpase 21 kda proteolipid <break/>subunit-like</td></tr><tr><td>rc_2499</td><td>2.18</td><td>2.03</td><td>1.47</td><td colspan="2">endoplasmic reticulum-Golgi <break/>intermediate compartment <break/>protein three isoform a</td><td>endoplasmic reticulum-golgi intermediate <break/>compartment protein 3 isoform 2</td></tr><tr><td>rc_13969</td><td>2.08</td><td>3.09</td><td>0.97</td><td colspan="2"/><td>major facilitator superfamily</td></tr><tr><td colspan="7">(IPR023561) Carbonic anhydrase, alpha-class</td></tr><tr><td>rc_24825</td><td>2.49</td><td>2.38</td><td colspan="2">0.83</td><td>protein tyrosine phosphatase, <break/>receptor type, G precursor</td><td>receptor-type tyrosine-protein phosphatase <break/>gamma</td></tr><tr><td colspan="7">Cell Adhesion and extracelluar matrix</td></tr><tr><td colspan="7">Hv_Sym&#160;&gt;&#160;Hv_Apo</td></tr><tr><td>7915</td><td>4.01</td><td>5.09</td><td>0.94</td><td colspan="2">fibrillin-2 precursor</td><td>fibrillin-1- partial</td></tr><tr><td>npRC_24163</td><td>glutamate3.69</td><td>3.59</td><td>1.32</td><td colspan="2">semaphorin 5A precursor</td><td>rhamnospondin 1</td></tr><tr><td colspan="7">Immunity, apoptosis and recognition</td></tr><tr><td colspan="7">Hv_Sym&#160;&gt;&#160;Hv_Apo</td></tr><tr><td colspan="7">(IPR000157) Toll/interleukin-1 receptor homology (TIR) domain</td></tr><tr><td>5168</td><td>9.28</td><td>4.92</td><td>0.61</td><td colspan="2"/><td>protein; PREDICTED: uncharacterized <break/>protein LOC100893943</td></tr><tr><td>12749</td><td>5.13</td><td>3.35</td><td>1.26</td><td colspan="2"/><td>PREDICTED: uncharacterized protein <break/>LOC100893943 [Strongylocentrotus <break/>purpuratus]</td></tr><tr><td colspan="3">(IPR011029) DEATH-like</td><td/><td colspan="2"/><td/></tr><tr><td>6508</td><td>6.70</td><td>5.10</td><td>0.64</td><td colspan="2"/><td>PREDICTED: hypothetical protein <break/>[Hydra magnipapillata]</td></tr><tr><td>rc_2417</td><td>5.39</td><td>2.70</td><td>1.01</td><td colspan="2"/><td>nod3 partial; PREDICTED: uncharacterized <break/>protein LOC100206003</td></tr><tr><td colspan="7">(IPR002398) Peptidase C14, caspase precursor p45</td></tr><tr><td>NPNH_21275</td><td>2.36</td><td>3.53</td><td>1.18</td><td colspan="2">caspase seven isoform alpha <break/>precursor</td><td>caspase d</td></tr><tr><td colspan="7">(IPR016187) C-type lectin fold</td></tr><tr><td>11411</td><td>2.93</td><td>2.98</td><td>0.75</td><td colspan="2">C-type mannose receptor 2</td><td>PREDICTED: similar to predicted protein, <break/>partial [Hydra magnipapillata]</td></tr><tr><td colspan="7">Hv_Sym&#160;&lt;&#160;Hv_Apo</td></tr><tr><td colspan="7">(IPR000488) Death</td></tr><tr><td>7319</td><td>0.45</td><td>0.31</td><td>1.10</td><td colspan="2">probable ubiquitin carboxyl- <break/>terminal hydrolase CYLD isoform 2</td><td>ubiquitin carboxyl-terminal hydrolase cyld</td></tr><tr><td colspan="7">(IPR001875) Death effector domain</td></tr><tr><td>RC_FV81RT001CSTY</td><td>0.31</td><td>0.39</td><td>0.93</td><td colspan="2">astrocytic phosphoprotein PEA-15</td><td>fadd</td></tr><tr><td colspan="7">Chitinase</td></tr><tr><td colspan="7">Hv_Sym&#160;&gt;&#160;Hv_Apo</td></tr><tr><td colspan="7">(IPR001223) Glycoside hydrolase, family 18, catalytic domain</td></tr><tr><td>rc_4450</td><td>2.78</td><td>3.83</td><td>0.66</td><td colspan="2"/><td>chitinase 2</td></tr><tr><td colspan="7">Hv_Sym&#160;&lt;&#160;Hv_Apo</td></tr><tr><td colspan="7">(IPR000726) Glycoside hydrolase, family 19, catalytic</td></tr><tr><td>FPVQZVL01EAWBY</td><td>0.21</td><td>0.16</td><td>1.78</td><td colspan="2"/><td>endochitinase 1-like</td></tr><tr><td>1028</td><td>0.23</td><td>0.18</td><td>1.47</td><td colspan="2"/><td>endochitinase 1-like</td></tr><tr><td colspan="7">Oxidative Stress Response</td></tr><tr><td colspan="7">Hv_Sym&#160;&gt;&#160;Hv_Apo</td></tr><tr><td>np_1276</td><td>5.99</td><td>7.16</td><td>0.78</td><td colspan="2">glutaredoxin-2, mitochondrial <break/>isoform 2</td><td>cpyc type</td></tr><tr><td>10926</td><td>3.9</td><td>2.3</td><td>0.8</td><td colspan="2">hydroxysteroid dehydrogenase- <break/>like protein 2</td><td>hydroxysteroid dehydrogenase-like <break/>protein 2</td></tr><tr><td>469</td><td>2.97</td><td>3.53</td><td>0.76</td><td colspan="2">cytochrome P450 3A7</td><td>cytochrome p450</td></tr><tr><td>FV81RT001DCTAQ</td><td>2.69</td><td>2.50</td><td>0.75</td><td colspan="2">oxidoreductase NAD-binding <break/>domain-containing protein <break/>one precursor</td><td>oxidoreductase nad-binding domain- <break/>containing protein 1</td></tr><tr><td>696</td><td>2.30</td><td>3.24</td><td>0.69</td><td colspan="2">methionine-R-sulfoxide <break/>reductase B1</td><td>selenoprotein 1; methionine-r-sulfoxide <break/>reductase b1-a-like</td></tr><tr><td>6572</td><td>2.23</td><td>2.15</td><td>1.06</td><td colspan="2">L-xylulose reductase</td><td>l-xylulose reductase</td></tr><tr><td>13298</td><td>2.10</td><td>3.49</td><td>0.64</td><td colspan="2">eosinophil peroxidase <break/>preproprotein</td><td>peroxidase</td></tr><tr><td>npRC_6975</td><td>2.04</td><td>2.77</td><td>1.42</td><td colspan="2">methionine-R-sulfoxide <break/>reductase B1</td><td>selenoprotein 1; methionine-r-sulfoxide <break/>reductase b1-a-like</td></tr><tr><td colspan="7">(IPR024079) Metallopeptidase, catalytic domain</td></tr><tr><td>Hv_array_4952</td><td>4.77</td><td>13.31</td><td>0.72</td><td colspan="2">meprin A subunit beta <break/>precursor</td><td>zinc metalloproteinase nas-4-like</td></tr><tr><td>Hv_array_rc_3992</td><td>2.66</td><td>2.23</td><td>1.27</td><td colspan="2">matrix metalloproteinase <break/>seven preproprotein</td><td>matrix metalloproteinase-24-like</td></tr><tr><td colspan="7">Hv_Sym&#160;&lt;&#160;Hv_Apo</td></tr><tr><td>RC_FWZAEML02HKSC</td><td>0.255</td><td>0.153</td><td>1.444</td><td colspan="2"/><td>ascorbate peroxidase</td></tr><tr><td>np_14962</td><td>0.293</td><td>0.455</td><td>1.390</td><td colspan="2">tryptophan 5-hydroxylase 2</td><td>phenylalanine hydroxylase</td></tr><tr><td>rc_4151</td><td>0.318</td><td>0.463</td><td>1.693</td><td colspan="2">phenylalanine-4-hydroxylase</td><td>phenylalanine hydroxylase</td></tr><tr><td>2835</td><td>0.384</td><td>0.344</td><td>1.787</td><td colspan="2"/><td>u1 small nuclear ribonucleoprotein 70 kda</td></tr><tr><td>rc_11426</td><td>0.413</td><td>0.458</td><td>1.591</td><td colspan="2">short-chain dehydrogenase/ <break/>reductase family 9C member 7</td><td>uncharacterized oxidoreductase -like</td></tr><tr><td>FWZAEML02IC34R</td><td>0.427</td><td>0.448</td><td>1.159</td><td colspan="2">aldehyde dehydrogenase 5A1 <break/>isoform two precursor</td><td>succinate-semialdehyde mitochondrial-like</td></tr><tr><td>FWZAEML02HKSCO</td><td>0.454</td><td>0.307</td><td>0.833</td><td colspan="2"/><td>ascorbate peroxidase</td></tr><tr><td colspan="7">(IPR004045) Glutathione S-transferase, N-terminal</td></tr><tr><td>RC_FWZAEML02GGHN</td><td>0.09</td><td>0.07</td><td>1.81</td><td colspan="2">hematopoietic prostaglandin <break/>D synthase</td><td>glutathione s-transferase family member <break/>(gst-7)</td></tr><tr><td colspan="7">(IPR024079) Metallopeptidase, catalytic domain</td></tr><tr><td>rc_11270</td><td>0.14</td><td>0.20</td><td>1.33</td><td colspan="2">meprin A subunit beta precursor</td><td>protein; zinc metalloproteinase nas-4-like</td></tr><tr><td>rc_RSASM_15059</td><td>0.22</td><td>0.29</td><td>1.42</td><td colspan="2"/><td>---NA---</td></tr><tr><td>2111</td><td>0.37</td><td>0.43</td><td>1.74</td><td colspan="2">meprin A subunit beta precursor</td><td>zinc metalloproteinase nas-4-like</td></tr><tr><td>12451</td><td>0.50</td><td>0.39</td><td>0.78</td><td colspan="2">meprin A subunit alpha precursor</td><td>zinc metalloproteinase nas-13- partial</td></tr><tr><td colspan="7">(IPR013122) Polycystin cation channel, PKD1/PKD2</td></tr><tr><td>28854</td><td>0.37</td><td>0.28</td><td>0.94</td><td colspan="2">polycystin-2</td><td>receptor for egg jelly partial</td></tr><tr><td>15774</td><td>0.40</td><td>0.26</td><td>0.76</td><td colspan="2">polycystic kidney disease protein <break/>1-like two isoform a</td><td>protein</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35122-v2.xml">elife-35122-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.35122.009</object-id><label>Table 2.</label><caption><title>List of genes differentially expressed in Hv_Sym compared to both Hv_Apo and Hv_NC64A (&#8216;A99-specific&#8217;)</title><p>Fold change of expression level determined by microarray analysis and qPCR analysis</p><p><supplementary-material id="table2sdata1"><object-id pub-id-type="doi">10.7554/eLife.35122.010</object-id><label>Table 2&#8212;source data 1.</label><caption><title>Expression level of &#8216;A99-specific&#8217; genes and &#8216;Symbiosis related&#8217; genes examined by microarray and qPCR.</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-35122-table2-data1-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="7">Hv_Sym&#160;&gt;&#160;Hv_Apo, Hv_NC64A</th></tr><tr><th rowspan="2">Probe name (gene ID)</th><th colspan="2">Microarray</th><th colspan="2">qPCR</th><th rowspan="2">Gene annotation</th><th rowspan="2">InterProScan</th></tr><tr><th>Sym/Apo</th><th>Sym/NC64A</th><th>Sym/Apo</th><th>Sym/NC64A</th></tr></thead><tbody><tr><td>rc_13579</td><td>12.8</td><td>4.0</td><td>11.2</td><td>4.0</td><td>(Hydra specific)</td><td/></tr><tr><td>rc_12891</td><td>9.0</td><td>2.9</td><td>14.6</td><td>6.9</td><td>(Hydra viridis specific)</td><td/></tr><tr><td>27417</td><td>4.5</td><td>4.8</td><td>3.0</td><td>3.0</td><td/><td>IPR009786 Spot_14</td></tr><tr><td>rc_26218</td><td>3.3</td><td>2.4</td><td>2.5</td><td>2.3</td><td>sodium-dependent phosphate <break/>transport protein</td><td>PTHR10010 Sodium-dependent <break/>phosphate transport protein 2C</td></tr><tr><td>1046</td><td>3.1</td><td>2.1</td><td>2.2</td><td>1.6</td><td>glutamine synthetase</td><td/></tr><tr><td colspan="7"><bold>Hv_Sym&#160;&lt;&#160;Hv_Apo, Hv_NC64A</bold></td></tr><tr><td rowspan="2"><bold>Probe name (gene ID)</bold></td><td colspan="2"><bold>Microarray</bold></td><td colspan="2"><bold>qPCR</bold></td><td rowspan="2"><bold>Gene Annotation</bold></td><td rowspan="2"><bold>InterProScan</bold></td></tr><tr><td><bold>Apo/Sym</bold></td><td><bold>NC64A/Sym</bold></td><td><bold>Apo/Sym</bold></td><td><bold>NC64A/Sym</bold></td></tr><tr><td>NPNHRC_26859</td><td>83.2</td><td>9.7</td><td>&#8734;</td><td>&#8734;</td><td>(Hydra viridis specific)</td><td/></tr><tr><td>RC_FVQRUGK01AXSJ</td><td>13.7</td><td>2.6</td><td>2.1</td><td>1.5</td><td>acetoacetyl-CoA synthetase</td><td/></tr><tr><td>rc_14793</td><td>7.2</td><td>4.1</td><td>9.4</td><td>4.8</td><td>2-isopropylmalate synthase</td><td>IPR013785 Aldolase_TIM,</td></tr><tr><td>FV81RT002HT2FL</td><td>2.8</td><td>2.0</td><td>3.1</td><td>1.8</td><td>histidine ammonia-lyase</td><td>IPR001106 Aromatic_Lyase <break/>IPR008948 L-Aspartase-like</td></tr><tr><td>NPNHRC_12201</td><td>2.7glutamate</td><td>2.3</td><td>2.6</td><td>2.5</td><td>(Hydra viridis specific)</td><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35122-v2.xml">elife-35122-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.35122.015</object-id><label>Table 4.</label><caption><title>List of the genes differentially expressed between Hv_Sym and Hv_Apo</title><p>Fold change of expression level determined by microarray analysis and qPCR</p><p><supplementary-material id="table4sdata1"><object-id pub-id-type="doi">10.7554/eLife.35122.016</object-id><label>Table 4&#8212;source data 1.</label><caption><title>Expression level of 'Symbiosis related' genes examined by microarray and qPCR.</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-35122-table4-data1-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="5">Hv_Sym&#160;&gt;&#160;Hv_Apo</th></tr><tr><th rowspan="2">Probe name <break/>(gene ID)</th><th>Microarray</th><th>qPCR</th><th rowspan="2">Gene annotation</th><th rowspan="2">InterProScan</th></tr><tr><th>Sym/Apo</th><th>Sym/Apo</th></tr></thead><tbody><tr><td>5168</td><td>9.3</td><td>7.4</td><td/><td>IPR000157 TIR_dom <break/>PTHR23097 Tumor necrosis factor <break/>receptor superfamily member</td></tr><tr><td>6508</td><td>6.7</td><td>2.9</td><td/><td>IPR011029:DEATH-like_dom</td></tr><tr><td>11411</td><td>2.9</td><td>2.0</td><td>C-type mannose receptor 2</td><td>IPR000742 EG-like_dom <break/>IPR001304 C-type_lectin</td></tr><tr><td>26108</td><td>7.2</td><td>7.2</td><td>ephrin type-A receptor six isoform a</td><td/></tr><tr><td>rc_2417</td><td>5.4</td><td>3.5</td><td/><td>IPR000488 Death_domain</td></tr><tr><td>rc_24563</td><td>6.1</td><td>6.7</td><td>Proline-rich transmembrane protein 1</td><td>IPR007593 CD225/Dispanin_fam <break/>PTHR14948 NG5</td></tr><tr><td>rc_9398</td><td>6.2</td><td>5.4</td><td>protein-kinase, interferon-inducible <break/>double stranded RNA dependent inhibitor, <break/>repressor of (P58 repressor)</td><td>PTHR11697 general transcription factor <break/>2-related zinc finger protein</td></tr><tr><td colspan="5"><bold>Hv_Sym&#160;&lt;&#160;Hv_Apo</bold></td></tr><tr><td rowspan="2"><bold>Probe name</bold> <break/><bold>(gene ID)</bold></td><td><bold>Microarray</bold></td><td><bold>qPCR</bold></td><td rowspan="2"><bold>Gene Annotation</bold></td><td rowspan="2"><bold>InterProScan</bold></td></tr><tr><td><bold>Apo/Sym</bold></td><td><bold>Apo/Sym</bold></td></tr><tr><td>rc_10789</td><td>2.5</td><td>3.7</td><td>endoribonuclease Dicer</td><td>IPR000999 RNase_III_dom <break/>PTHR1495 helicase-related</td></tr><tr><td>rc_12826</td><td>3.0</td><td>2.3</td><td>interferon regulatory factor 1</td><td>IPR001346 Interferon_reg_fact_DNA-bd_dom; <break/>IPR011991 WHTH_DNA-bd_dom <break/>PTHR11949 interferon regulatory factor</td></tr><tr><td>rc_8898</td><td>6.1</td><td>4.1</td><td>leucine-rich repeat-containing protein 15 <break/>isoform b</td><td>IPR001611 Leu-rich_rp <break/>PTHR24373 Toll-like receptor 9</td></tr><tr><td>FV81RT001CSTY</td><td>3.2</td><td>2.0</td><td>astrocytic phosphoprotein PEA-15</td><td>IPR001875 DED, IPR011029 DEATH-like_dom</td></tr><tr><td>RSASM_17752</td><td>4.0</td><td>2.1</td><td>CD97 antigen isoform two precursor</td><td>IPR000832 GPCR_2_secretin-like <break/>PTHR12011 vasoactive intestinal polypeptide <break/>receptor 2</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35468-v1.xml">elife-35468-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app1table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.35468.034</object-id><label>Appendix 1&#8212;table 1</label><caption><title><italic>G</italic><sub>min</sub> and power to detect simulated introgression on the X chromosome and autosomes.</title><p>Numbers in parentheses indicate the standard deviation from 100 replicate simulations</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="2" valign="top">400 ybp</th><th colspan="2" valign="top">4000 ybp</th><th colspan="2" valign="top">40,000 ybp</th></tr><tr><th valign="top">A</th><th valign="top">X</th><th valign="top">A</th><th valign="top">X</th><th valign="top">A</th><th valign="top">X</th></tr></thead><tbody><tr><td valign="top">Windows with migration (#)</td><td valign="top">202.12 (18)</td><td valign="top">31.06 (7.2)</td><td valign="bottom">250.65 (16)</td><td valign="bottom">66.48 (10)</td><td valign="bottom">281.15 (20)</td><td valign="bottom">65.22 (8.7)</td></tr><tr><td valign="top">Windows with migration (%)</td><td valign="top">2.4% (0.21)</td><td valign="top">1.7% (0.4)</td><td valign="top">3% (0.19)</td><td valign="top">3.7% (0.56)</td><td valign="top">3.4% (0.24)</td><td valign="top">3.6% (0.48)</td></tr><tr><td valign="top">Significant <italic>G</italic><sub>min</sub> windows (#)</td><td valign="top">179.08 (17)</td><td valign="top">28.5 (6.4)</td><td valign="bottom">111.44 (9.8)</td><td valign="bottom">27.02 (4.7)</td><td valign="bottom">15.87 (4.4)</td><td valign="bottom">2.4 <break/>(1.3)</td></tr><tr><td valign="top">Significant <italic>G</italic><sub>min</sub> windows (%)</td><td valign="top">2.2% (0.2)</td><td valign="top">1.6% (0.35)</td><td valign="top">1.3% (0.12)</td><td valign="top">1.5% (0.26)</td><td valign="top">0.19% (0.052)</td><td valign="top">0.13% (0.07)</td></tr><tr><td valign="top">True positive rate</td><td valign="top">96% (1.5)</td><td valign="top">95% (3.9)</td><td valign="top">94% (2.4)</td><td valign="top">93% (5.1)</td><td valign="top">45% (11)</td><td valign="top">30% (34)</td></tr><tr><td valign="top">False postive rate</td><td valign="top">3.7% (1.5)</td><td valign="top">4.8% (3.9)</td><td valign="top">5.7% (2.4)</td><td valign="top">6.8% (5.1)</td><td valign="top">55% (11)</td><td valign="top">70% (34)</td></tr><tr><td valign="top">Migration Events Detected</td><td valign="top">85% (3.2)</td><td valign="top">88% (7.5)</td><td valign="top">42% (2.9)</td><td valign="top">38% (5.1)</td><td valign="top">2.6% <break/>(1)</td><td valign="top">1.2% (1.4)</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35684-v2.xml">elife-35684-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.35684.008</object-id><label>Table 1.</label><caption><title>Number of orthologs assigned as Z-linked and autosomal in <italic>S. mansoni</italic>, <italic>S. haematobium</italic> and <italic>S. japonicum</italic>,&#160;based on the female:male (or female for <italic>S. haematobium</italic>) coverage patterns.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2" rowspan="2"><break/>Categories</th><th colspan="4"><italic>Schistosoma japonicum</italic></th><th colspan="4"><italic>Schistosoma haematobium</italic></th></tr><tr><th>Z-specific</th><th>Autosomal</th><th>Ambiguous</th><th>Not covered</th><th>Z-specific</th><th>Autosomal</th><th>Ambiguous</th><th>Not covered</th></tr></thead><tbody><tr><td rowspan="4"><italic>Schistosoma mansoni</italic></td><td>Z-specific</td><td colspan="2" valign="top">476 <break/>(<italic>S0</italic>)</td><td>306 <break/>(<italic>S1mans</italic>)</td><td>20</td><td>3</td><td>847</td><td>36</td><td>23</td><td>10</td></tr><tr><td>&#8195;Autosomal</td><td colspan="2">137 <break/>(<italic>S1jap</italic>)</td><td>7062</td><td>91</td><td>13</td><td>216</td><td>7462</td><td>262</td><td>105</td></tr><tr><td>&#8195;Excluded</td><td colspan="2">57</td><td>383</td><td>4</td><td>3</td><td>105</td><td>411</td><td>20</td><td>7</td></tr><tr><td>&#8195;Orthologs total</td><td colspan="5">8555</td><td colspan="4">9504</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35856-v2.xml">elife-35856-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.35856.021</object-id><label>Table 4.</label><caption><title>Analysis of selected predictors for age-5 wheeze within each CAS cluster, with demographic covariates (sex, BMI, parental history of asthma)</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Selected predictors for age-5 wheeze</th><th rowspan="2" valign="top"/></tr><tr><th valign="top">Cas1 (N&#160;=&#160;88) <break/>Or (95% CI)</th><th valign="top">P-value</th><th valign="top">Cas2 (N&#160;=&#160;107) <break/>Or (95% CI)</th><th valign="top">P-value</th><th valign="top">Cas3 (N&#160;=&#160;22) <break/>Or (95% CI)</th><th valign="top">P-value</th><th valign="top">Or (95% CI)</th><th valign="top">P-value</th></tr></thead><tbody><tr><td valign="top">LRI (events per y)</td><td valign="top">1</td><td valign="top">0.97 (0.71&#8211;1.3)</td><td valign="top">0.84</td><td valign="top">1 (0.61&#8211;1.5)</td><td valign="top">0.99</td><td valign="top">0.48 (0.13&#8211;1.1)</td><td valign="top">0.16</td><td valign="top">1 (0.81&#8211;1.2)</td><td valign="top">0.92</td></tr><tr><td valign="top"/><td valign="top">2</td><td valign="top">1.2 (0.88&#8211;1.6)</td><td valign="top">0.26</td><td valign="top">1.5 (0.97&#8211;2.5)</td><td valign="top"><italic>0.069</italic></td><td valign="top">0.99 (0.34&#8211;2.6)</td><td valign="top">0.98</td><td valign="top">1.4 (1.1&#8211;1.7)</td><td valign="top"><bold>5.3E-03</bold></td></tr><tr><td valign="top"/><td valign="top">3</td><td valign="top">2 (1.3&#8211;3.2)</td><td valign="top"><bold>2.3E-03</bold></td><td valign="top">2.6 (1.5&#8211;5.3)</td><td valign="top"><bold>2.7E-03</bold></td><td valign="top">0.98 (0.4&#8211;2.6)</td><td valign="top">0.96</td><td valign="top">2 (1.5&#8211;2.7)</td><td valign="top"><bold>3.8E-06</bold></td></tr><tr><td valign="top"/><td valign="top">4</td><td valign="top">2 (1.4&#8211;3.4)</td><td valign="top"><bold>2.0E-03</bold></td><td valign="top">3.6 (1.8&#8211;8.3)</td><td valign="top"><bold>6.5E-04</bold></td><td valign="top">1.9 (0.57&#8211;8.4)</td><td valign="top">0.32</td><td valign="top">2.5 (1.8&#8211;3.6)</td><td valign="top"><bold>1.5E-07</bold></td></tr><tr><td valign="top">Wheezy LRI (events per y)</td><td valign="top">1</td><td valign="top">1.3 (0.68&#8211;2.4)</td><td valign="top">0.43</td><td valign="top">1.1 (0.35&#8211;3)</td><td valign="top">0.83</td><td valign="top">2.6 (0.62&#8211;58)</td><td valign="top">0.34</td><td valign="top">1.5 (0.98&#8211;2.3)</td><td valign="top"><italic>0.06</italic></td></tr><tr><td valign="top"/><td valign="top">2</td><td valign="top">1.2 (0.8&#8211;2)</td><td valign="top">0.33</td><td valign="top">1.6 (0.89&#8211;2.9)</td><td valign="top">0.12</td><td valign="top">2.4 (0.67&#8211;16)</td><td valign="top">0.24</td><td valign="top">1.6 (1.2&#8211;2.2)</td><td valign="top"><bold>5.6E-03</bold></td></tr><tr><td valign="top"/><td valign="top">3</td><td valign="top">2.8 (1.6&#8211;5.6)</td><td valign="top"><bold>1.3E-03</bold></td><td valign="top">3 (1.4&#8211;8)</td><td valign="top"><bold>0.016</bold></td><td valign="top">1.2 (0.43&#8211;4.6)</td><td valign="top">0.76</td><td valign="top">2.7 (1.8&#8211;4.2)</td><td valign="top"><bold>4.1E-06</bold></td></tr><tr><td valign="top"/><td valign="top">4</td><td valign="top">2.5 (1.5&#8211;5)</td><td valign="top"><bold>4.0E-03</bold></td><td valign="top">6.3 (2.5&#8211;21)</td><td valign="top"><bold>6.8E-04</bold></td><td valign="top">7.1 (1.2&#8211;169)</td><td valign="top">0.1</td><td valign="top">3.9 (2.5&#8211;6.7)</td><td valign="top"><bold>5.4E-08</bold></td></tr><tr><td valign="top">Febrile LRI (events per y)</td><td valign="top">1</td><td valign="top">1.6 (0.77&#8211;3.6)</td><td valign="top">0.21</td><td valign="top">0.84 (0.28&#8211;1.9)</td><td valign="top">0.71</td><td valign="top">7.3 (0.78&#8211;178)</td><td valign="top">0.12</td><td valign="top">1.5 (0.93&#8211;2.4)</td><td valign="top"><italic>0.098</italic></td></tr><tr><td valign="top"/><td valign="top">2</td><td valign="top">1 (0.44&#8211;2.2)</td><td valign="top">1</td><td valign="top">4.8 (1.8&#8211;15)</td><td valign="top"><bold>3.9E-03</bold></td><td valign="top">1.6 (0.48&#8211;10)</td><td valign="top">0.5</td><td valign="top">2.3 (1.4&#8211;3.9)</td><td valign="top"><bold>1.2E-03</bold></td></tr><tr><td valign="top"/><td valign="top">3</td><td valign="top">2 (1&#8211;4.8)</td><td valign="top"><italic>0.08</italic></td><td valign="top">4.3 (1.2&#8211;15)</td><td valign="top"><bold>0.02</bold></td><td valign="top">4.2 (0.55&#8211;519)</td><td valign="top">0.37</td><td valign="top">2.4 (1.4&#8211;4.3)</td><td valign="top"><bold>2.3E-03</bold></td></tr><tr><td valign="top"/><td valign="top">4</td><td valign="top">1.8 (0.97&#8211;4.1)</td><td valign="top"><italic>0.092</italic></td><td valign="top">2.6 (0.88&#8211;8.3)</td><td valign="top"><italic>0.082</italic></td><td valign="top">1.1 (0.11&#8211;18)</td><td valign="top">0.93</td><td valign="top">2.2 (1.3&#8211;4)</td><td valign="top"><bold>5.9E-03</bold></td></tr><tr><td valign="top">Quartile of % healthy NPAs with infection-associated MPGs</td><td valign="top">0&#8211;2</td><td valign="top">1 (0.54&#8211;1.8)</td><td valign="top">0.98</td><td valign="top">1.3 (0.72&#8211;2.4)</td><td valign="top">0.36</td><td valign="top">NA</td><td valign="top">NA</td><td valign="top">1.3 (0.89&#8211;1.8)</td><td valign="top">0.19</td></tr><tr><td valign="top"/><td valign="top">2&#8211;4</td><td valign="top">0.45 (0.19&#8211;0.88)</td><td valign="top"><bold>0.035</bold></td><td valign="top">1 (0.51&#8211;2.1)</td><td valign="top">0.9</td><td valign="top">NA</td><td valign="top">NA</td><td valign="top">0.8 (0.53&#8211;1.2)</td><td valign="top">0.24</td></tr><tr><td valign="top">HDM IgE (kU/L)*</td><td valign="top">6m</td><td valign="top">8 (0.85&#8211;94)</td><td valign="top"><italic>0.074</italic></td><td valign="top">0.93 (0.14&#8211;3.6)</td><td valign="top">0.92</td><td valign="top">3.4 (0.26&#8211;180)</td><td valign="top">0.4</td><td valign="top">2.3 (0.99&#8211;5.8)</td><td valign="top"><italic>0.054</italic></td></tr><tr><td valign="top"/><td valign="top">1</td><td valign="top">1.5 (0.22&#8211;7.8)</td><td valign="top">0.65</td><td valign="top">0.54 (0.039&#8211;2.3)</td><td valign="top">0.51</td><td valign="top">39 (2.5&#8211;22000)</td><td valign="top"><italic>0.082</italic></td><td valign="top">2.7 (1.5&#8211;5)</td><td valign="top"><bold>0.00089</bold></td></tr><tr><td valign="top"/><td valign="top">2</td><td valign="top">0.93 (0.28&#8211;2.5)</td><td valign="top">0.89</td><td valign="top">2 (1.2&#8211;3.7)</td><td valign="top"><bold>0.016</bold></td><td valign="top">1.4 (0.38&#8211;4.8)</td><td valign="top">0.62</td><td valign="top">2 (1.5&#8211;2.8)</td><td valign="top"><bold>2.80E-05</bold></td></tr><tr><td valign="top"/><td valign="top">3</td><td valign="top">1.4 (0.68&#8211;2.9)</td><td valign="top">0.32</td><td valign="top">1.5 (0.9&#8211;2.4)</td><td valign="top">0.12</td><td valign="top">1.5 (0.4&#8211;5.2)</td><td valign="top">0.55</td><td valign="top">1.7 (1.3&#8211;2.2)</td><td valign="top"><bold>1.00E-04</bold></td></tr><tr><td valign="top"/><td valign="top">4</td><td valign="top">1.9 (0.94&#8211;4.1)</td><td valign="top"><italic>0.086</italic></td><td valign="top">1.9 (1.2&#8211;3.1)</td><td valign="top"><bold>0.011</bold></td><td valign="top">1.4 (0.31&#8211;5.5)</td><td valign="top">0.64</td><td valign="top">1.9 (1.5&#8211;2.5)</td><td valign="top"><bold>3.70E-06</bold></td></tr><tr><td valign="top">HDM IgG4 (&#956;g/L)*</td><td valign="top">6m</td><td valign="top">NA (NA-NA)</td><td valign="top">0.55</td><td valign="top">0.053 (NA-6.5e&#160;+&#160;24)</td><td valign="top">0.99</td><td valign="top">28 (1.7e-34-NA)</td><td valign="top">0.99</td><td valign="top">1.4 (0.88&#8211;2.6)</td><td valign="top">0.17</td></tr><tr><td valign="top"/><td valign="top">1</td><td valign="top">NA (NA-NA)</td><td valign="top">0.61</td><td valign="top">1.1 (0.8&#8211;1.5)</td><td valign="top">0.5</td><td valign="top">0.9 (0.58&#8211;1.3)</td><td valign="top">0.6</td><td valign="top">1.2 (1&#8211;1.4)</td><td valign="top"><italic>0.053</italic></td></tr><tr><td valign="top"/><td valign="top">2</td><td valign="top">1.1 (0.71&#8211;1.6)</td><td valign="top">0.67</td><td valign="top">1.1 (0.85&#8211;1.4)</td><td valign="top">0.61</td><td valign="top">0.4 (0.038&#8211;1.2)</td><td valign="top">0.26</td><td valign="top">1.1 (1&#8211;1.3)</td><td valign="top"><italic>0.056</italic></td></tr><tr><td valign="top"/><td valign="top">3</td><td valign="top">1.1 (0.85&#8211;1.5)</td><td valign="top">0.35</td><td valign="top">1.1 (0.77&#8211;2)</td><td valign="top">0.64</td><td valign="top">0.94 (0.19&#8211;2.3)</td><td valign="top">0.9</td><td valign="top">1.1 (0.98&#8211;1.2)</td><td valign="top">0.1</td></tr><tr><td valign="top"/><td valign="top">4</td><td valign="top">1.2 (0.98&#8211;1.5)</td><td valign="top"><italic>0.082</italic></td><td valign="top">0.89 (0.7&#8211;1.1)</td><td valign="top">0.33</td><td valign="top">0.46 (0.031&#8211;5.4)</td><td valign="top">0.53</td><td valign="top">1.1 (1&#8211;1.3)</td><td valign="top"><bold>0.034</bold></td></tr><tr><td valign="top">HDM IgG (mg/L)*</td><td valign="top">1</td><td valign="top">25 (0.32&#8211;1.6E&#160;+&#160;04)</td><td valign="top">0.19</td><td valign="top">3.3 (0.16&#8211;46)</td><td valign="top">0.38</td><td valign="top">5.6E-03 (8.4E-06&#8211;0.57)</td><td valign="top"><italic>0.058</italic></td><td valign="top">2 (0.31&#8211;11)</td><td valign="top">0.44</td></tr><tr><td valign="top"/><td valign="top">2</td><td valign="top">0.8 (0.15&#8211;3.5)</td><td valign="top">0.78</td><td valign="top">0.97 (0.24&#8211;3.7)</td><td valign="top">0.96</td><td valign="top">0.79 (0.031&#8211;18)</td><td valign="top">0.88</td><td valign="top">1.3 (0.6&#8211;2.9)</td><td valign="top">0.48</td></tr><tr><td valign="top"/><td valign="top">3</td><td valign="top">2.3 (0.14&#8211;35)</td><td valign="top">0.54</td><td valign="top">0.48 (0.057&#8211;2.5)</td><td valign="top">0.43</td><td valign="top">3.9 (0.26&#8211;96)</td><td valign="top">0.34</td><td valign="top">2.1 (0.89&#8211;5)</td><td valign="top"><italic>0.089</italic></td></tr></tbody></table><table-wrap-foot><fn><p>BMI&#160;=&#160;body mass index; HDM&#160;=&#160;house dust mite; LRI&#160;=&#160;lower respiratory infection. Association analyses performed via generalised linear models (GLM) with demographic covariates: age-5 wheeze&#160;~predictor&#160;+ sex (male)&#160;+BMI at age 3&#160;+&#160;paternal history of asthma&#160;+maternal history of asthma. Bold text indicates statistical significance (p&lt;0.05); italics indicate near-significance (p&lt;0.10). *Odds ratio (OR) is for every 10-fold increase in IgE, IgG4 or IgG.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35898-v2.xml">elife-35898-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.35898.019</object-id><label>Table 3.</label><caption><title>Identified cross-linked peptides of Commd9 and NN-CH domain of CCDC93.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Predicted mass (Da)</th><th rowspan="2" valign="top">Observed mass (Da)</th><th colspan="1" rowspan="2" valign="top">Error&#160;(ppm)</th><th colspan="1" rowspan="2" valign="top">xQuest score</th></tr><tr><th colspan="2" valign="top">Cross-linked peptides</th></tr></thead><tbody><tr><td valign="top">Sample 1:</td><td colspan="3" valign="top"/><td colspan="2" valign="top">Commd9 - Commd9</td></tr><tr><td valign="top">2751.419</td><td valign="top">2751.407</td><td valign="top">2.3</td><td valign="top">21.10</td><td colspan="2" valign="top">VDIK(133)TSSDSISR-VDIK(133)TSSDSISR</td></tr><tr><td valign="top">2305.209</td><td valign="top">2305.206</td><td valign="top">3.0</td><td valign="top">20.35</td><td colspan="2" valign="top">ASSK(21)DVVR-VDIK(133)TSSDSISR</td></tr><tr><td valign="top">4316.319</td><td valign="top">4316.306</td><td valign="top">3.2</td><td valign="top">18.78</td><td colspan="2" valign="top">DLSSAEAILALFPENFHQNLK(95)NLLTK(100)IILEHVSTWR</td></tr><tr><td valign="top">3266.789</td><td valign="top">3266.782</td><td valign="top">1.8</td><td valign="top">18.67</td><td colspan="2" valign="top">NLLTK(100)IILEHVSTWR-VDIK(133)TSSDSISR</td></tr><tr><td valign="top">2820.589</td><td valign="top">2820.579</td><td valign="top">2.4</td><td valign="top">17.28</td><td colspan="2" valign="top">ASSK(21)DVVR-NLLTK(100)IILEHVSTWR</td></tr><tr><td valign="top">3648.889</td><td valign="top">3648.881</td><td valign="top">1.0</td><td valign="top">15.22</td><td colspan="2" valign="top">LVDLDWRVDIK()TSSDSISR-VDIK(133)TSSDSISR</td></tr><tr><td valign="top">3202.679</td><td valign="top">3202.677</td><td valign="top">2.0</td><td valign="top">14.22</td><td colspan="2" valign="top">ASSK(21)DVVR-LVDLDWRVDIK(133)TSSDSISR</td></tr><tr><td valign="top">2571.299</td><td valign="top">2571.288</td><td valign="top">4.1</td><td valign="top">12.78</td><td colspan="2" valign="top">IQEDPSLCGDK(163)PSISAVTVELSK(175)</td></tr><tr><td valign="top">1859.009</td><td valign="top">1859.006</td><td valign="top">2.7</td><td valign="top">12.40</td><td colspan="2" valign="top">ASSK(21)DVVR-ASSK(21)DVVR</td></tr><tr><td colspan="4" valign="top">Sample 2:</td><td colspan="2" valign="top">Commd9 - CCDC93</td></tr><tr><td valign="top">2998.479</td><td valign="top">2998.476</td><td valign="top">1.5</td><td valign="top">16.86</td><td valign="top">VDIK(133)TSSDSISR</td><td valign="top">AIETK(131)EEMGDYIR</td></tr><tr><td valign="top">3513.859</td><td valign="top">3513.842</td><td valign="top">1.4</td><td valign="top">14.74</td><td valign="top">NLLTK(100)IILEHVSTWR</td><td valign="top">AIETK(131)EEMGDYIR</td></tr><tr><td valign="top">4772.479</td><td valign="top">4772.501</td><td valign="top">5.1</td><td valign="top">7.11</td><td valign="top">MAVPTCLLQMK(152)IQEDPSLCGDKPSISAVTVELSK</td><td valign="top">IK(45)GLSPFDK</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td colspan="2" valign="top">Commd9 - Commd9</td></tr><tr><td valign="top">2820.589</td><td valign="top">2820.583</td><td valign="top">1.1</td><td valign="top">18.60</td><td colspan="2" valign="top">ASSK(21)DVVR-NLLTK(100)IILEHVSTWR</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td colspan="2" valign="top">CCDC93 - CCDC93</td></tr><tr><td valign="top">3569.679</td><td valign="top">3569.675</td><td valign="top">0.3</td><td valign="top">14.92</td><td colspan="2" valign="top">AIETK(131)EEMGDYIR-SYSVSQFQK(148)TYSLPED</td></tr><tr><td colspan="4" valign="top">Sample 3:</td><td colspan="2" valign="top">Commd9 - Commd9</td></tr><tr><td valign="top">2820.589</td><td valign="top">2820.599</td><td valign="top">4.6</td><td valign="top">20.02</td><td colspan="2" valign="top">ASSK(21)DVVR-NLLTK(100)IILEHVSTWR</td></tr><tr><td valign="top">2305.209</td><td valign="top">2305.226</td><td valign="top">5.8</td><td valign="top">19.51</td><td colspan="2" valign="top">ASSK(21)DVVR-VDIK(133)TSSDSISR</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35898-v2.xml">elife-35898-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.35898.020</object-id><label>Table 4.</label><caption><title>Identified cross-linked peptides of zfRetromer.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="1" valign="top">Predicted mass (Da)</th><th rowspan="1" valign="top">Observed mass (Da)</th><th rowspan="1" valign="top">Error&#160;(ppm)</th><th rowspan="1" valign="top">xQuest score</th><th colspan="3" valign="top"/></tr></thead><tbody><tr><td colspan="4" valign="top">Intermolecular cross-linked peptides</td><td valign="top">zfVPS26</td><td valign="top">zfVPS29</td><td valign="top">zfVPS35</td></tr><tr><td valign="top">2929.569</td><td valign="top">2929.550</td><td valign="top">8.0</td><td valign="top">13.20</td><td valign="top">APEK(301)MR</td><td valign="top"/><td valign="top">K(90)VADLYELVQYAGNIIPR</td></tr><tr><td valign="top">1844.019</td><td valign="top">1844.040</td><td valign="top">9.4</td><td valign="top">10.12</td><td valign="top">SKYHLK(188)</td><td valign="top"/><td valign="top">NK(38)LMDALK</td></tr><tr><td valign="top">1342.809</td><td valign="top">1342.805</td><td valign="top">4.9</td><td valign="top">9.99</td><td valign="top">MRK(304)R</td><td valign="top"/><td valign="top">K(127)DILK</td></tr><tr><td valign="top">1756.069</td><td valign="top">1756.070</td><td valign="top">2.8</td><td valign="top">9.83</td><td valign="top"/><td valign="top">FK(23)KLLVPGK</td><td valign="top">WEKK(556)</td></tr><tr><td valign="top">3809.959</td><td valign="top">3809.982</td><td valign="top">7.1</td><td valign="top">8.01</td><td valign="top"/><td valign="top">GDFDENLNYPEQK(73)VVTVGQFK</td><td valign="top">TQCALAASK(659)LLK</td></tr><tr><td colspan="4" valign="top">Intramolecular cross-linked peptides</td><td colspan="3" valign="top">zfVPS26 &#8211; zfVPS26</td></tr><tr><td valign="top">1441.829</td><td valign="top">1441.819</td><td valign="top">3.3</td><td valign="top">28.12</td><td colspan="3" valign="top">VNINVK(57)QTSK(61)R</td></tr><tr><td valign="top">1630.869</td><td valign="top">1630.863</td><td valign="top">4.6</td><td valign="top">26.54</td><td colspan="3" valign="top">TAELK(30)TEEGK(35)LEK</td></tr><tr><td valign="top">1247.689</td><td valign="top">1247.679</td><td valign="top">2.1</td><td valign="top">26.15</td><td colspan="3" valign="top">DVNK(266)K(267)FSVR</td></tr><tr><td valign="top">2571.239</td><td valign="top">2571.229</td><td valign="top">4.3</td><td valign="top">22.06</td><td colspan="3" valign="top">TEEGK(35)LEK(38)HYLFYDGESVSGK</td></tr><tr><td valign="top">3241.639</td><td valign="top">3241.662</td><td valign="top">7.0</td><td valign="top">20.14</td><td colspan="3" valign="top">K(25)TAELKTEEGK(35)LEKHYLFYDGESVSGK</td></tr><tr><td valign="top">3626.859</td><td valign="top">3626.875</td><td valign="top">5.8</td><td valign="top">19.78</td><td colspan="3" valign="top">K(214)EMTGIGPSTTTETETVAK-YFK(288)QQEIVLWR</td></tr><tr><td valign="top">844.489</td><td valign="top">844.487</td><td valign="top">4.4</td><td valign="top">18.66</td><td colspan="3" valign="top">K(25)TAELK(30)</td></tr><tr><td valign="top">1014.549</td><td valign="top">1014.544</td><td valign="top">1.4</td><td valign="top">12.32</td><td colspan="3" valign="top">K(297)APEK(301)MR</td></tr><tr><td colspan="4" valign="top">Intramolecular cross-linked peptides</td><td colspan="3" valign="top">zfVPS29 - zfVPS29</td></tr><tr><td valign="top">1184.749</td><td valign="top">1184.736</td><td valign="top">5.9</td><td valign="top">18.59</td><td colspan="3" valign="top">FK(23)KLLVPGK(30)</td></tr><tr><td colspan="4" valign="top">Intramolecular cross-linked peptides</td><td colspan="3" valign="top">zfVPS35 &#8211; zfVPS35</td></tr><tr><td valign="top">1607.729</td><td valign="top">1607.726</td><td valign="top">3.3</td><td valign="top">22.38</td><td colspan="3" valign="top">ENSSSDDK(552)WEKK(556)</td></tr><tr><td valign="top">844.469</td><td valign="top">844.449</td><td valign="top">6.9</td><td valign="top">19.26</td><td colspan="3" valign="top">EK(208)REK(211)</td></tr><tr><td valign="top">2245.219</td><td valign="top">2245.215</td><td valign="top">5.1</td><td valign="top">18.55</td><td colspan="3" valign="top">LLDEAVQAVK(24)VQSFQMK(31)R</td></tr><tr><td valign="top">1255.699</td><td valign="top">1255.688</td><td valign="top">2.5</td><td valign="top">17.69</td><td colspan="3" valign="top">LLK(662)K(663)PDQCR</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35957-v1.xml">elife-35957-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table10" position="float"><object-id pub-id-type="doi">10.7554/eLife.35957.022</object-id><label>Table 10.</label><caption><title>Analyses of selection on Characiphysi rhodopsin (<italic>rh1</italic>) using PAML random sites models.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Model</th><th rowspan="2" valign="top"><italic>ln</italic>L</th><th colspan="3" valign="top">Parameters1</th><th rowspan="2" valign="top">Null</th><th rowspan="2" valign="top"><italic>P</italic> [df]2</th><th rowspan="2" valign="top">&#916; AIC<sup>&#167;</sup></th></tr><tr><th valign="top"><italic>&#969;</italic><sub>0</sub>/p</th><th valign="top"><italic>&#969;</italic><sub>1</sub>/q</th><th valign="top"><italic>&#969;</italic><sub>2</sub>/<italic>&#969;</italic><sub>p</sub></th></tr></thead><tbody><tr><td valign="top">M0</td><td valign="top">&#8722;10819.13</td><td valign="top">0.06</td><td valign="top">-</td><td valign="top">-</td><td valign="top">N/A</td><td valign="top">-</td><td valign="top">842.6</td></tr><tr><td valign="top">M1a</td><td valign="top">&#8722;10586.68</td><td valign="top">0.03 (91%)</td><td valign="top">1.00 (9%)</td><td valign="top">-</td><td valign="top">M0</td><td valign="top">0.000 [1]</td><td valign="top">379.73</td></tr><tr><td valign="top">M2a</td><td valign="top">&#8722;10586.68</td><td valign="top">0.03 (91%)</td><td valign="top">1.00 (9%)</td><td valign="top">9.07 (0%)</td><td valign="top">M1a</td><td valign="top">1 [(2</td><td valign="top">383.7</td></tr><tr><td valign="top">M3</td><td valign="top">&#8722;10403.20</td><td valign="top">0.00 (60%)</td><td valign="top">0.08 (29%)</td><td valign="top">0.40 (11%)</td><td valign="top">M0</td><td valign="top">0.000 4)</td><td valign="top">18.8</td></tr><tr><td valign="top">M7</td><td valign="top">&#8722;10401.45</td><td valign="top">0.17</td><td valign="top">1.77</td><td valign="top">-</td><td valign="top">N/A</td><td valign="top">-</td><td valign="top">9.27</td></tr><tr><td valign="top">M8a</td><td valign="top">&#8722;10395.82</td><td valign="top">0.18</td><td valign="top">2.27</td><td valign="top">1.00</td><td valign="top">N/A</td><td valign="top"><bold>-</bold></td><td valign="top">0.23</td></tr><tr><td rowspan="2" valign="top">M8</td><td rowspan="2" valign="top">&#8722;10394.82</td><td rowspan="2" valign="top">0.18</td><td rowspan="2" valign="top">2.22</td><td rowspan="2" valign="top">1.50</td><td valign="top">M7</td><td valign="top">0.000 (2]</td><td rowspan="2" valign="top">0*</td></tr><tr><td valign="top">M8a</td><td valign="top">0.136 [1]</td></tr></tbody></table><table-wrap-foot><fn><p><sup>&#8727;&#8224;</sup><italic>&#969;</italic> values of each site class are shown are shown for model M0-M3 (<italic>&#969;</italic><sub>0</sub>&#8211; <italic>&#969;</italic><sub>2</sub>) with the proportion of each site class in parentheses. For M7 and M8, the shape parameters, p and q, which describe the beta distribution are listed instead. In addition, the <italic>&#969;</italic> value for the positively selected site class (<italic>&#969;</italic><sub>p</sub>, with the proportion of sites in parentheses) is shown for M8.</p><p><sup>&#8225;</sup>Significant p-values (&#945;&#160;&#8804;0.05) are bolded. Degrees of freedom are given in square brackets after the p-values.</p></fn><fn><p>Model fits were assessed by Akaike information criterion differences to the best fitting model (asterisk).</p><p>Abbreviations&#8212;<italic>ln</italic>L, ln Likelihood; p, p<italic>-</italic>value; N/A, not applicable.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35957-v1.xml">elife-35957-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.35957.010</object-id><label>Table 3.</label><caption><title>Results of Clade Model C (CmC) analyses of vertebrate <italic>rh1</italic> under various partitions.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Model and <break/>Foreground<sup>&#8224;</sup></th><th rowspan="2" valign="top">&#916;AIC<sup>&#8225;</sup></th><th rowspan="2" valign="top"><italic>ln</italic>L</th><th colspan="3" valign="top">Parameters</th><th rowspan="2" valign="top">Null</th><th rowspan="2" valign="top"><italic>P</italic> [df]</th></tr><tr><th valign="top"><italic>&#969;</italic><sub>0</sub></th><th valign="top"><italic>&#969;</italic><sub>1</sub></th><th valign="top"><italic>&#969;</italic><sub>2</sub>/<italic>&#969;</italic><sub>d</sub></th></tr></thead><tbody><tr><td valign="top">&#8195;M2a_rel</td><td valign="top">225.5</td><td valign="top">&#8722;47185.37</td><td valign="top">0.02 (69%)</td><td valign="top">1 (3%)</td><td valign="top">0.20 (28%)</td><td valign="top">N/A</td><td valign="top">-</td></tr><tr><td valign="top">CmC_<italic>Tetrapod Branch</italic></td><td valign="top">97.44</td><td valign="top">&#8722;47119.33</td><td valign="top">0.20 <break/>(28%)</td><td valign="top">1 <break/>(3%)</td><td valign="top">0.02 (69%) <break/>Tetra Br: 0.00</td><td valign="top">M2a_rel</td><td valign="top">0.000 [1]</td></tr><tr><td valign="top">CmC_<italic>Tetrapod</italic></td><td valign="top">4.92</td><td valign="top">&#8722;47073.06</td><td valign="top">0.02 (67%)</td><td valign="top">1 (3%)</td><td valign="top">0.24 (30%) <break/>Tetra: 0.13</td><td valign="top">M2a_rel</td><td valign="top">0.000 [1]</td></tr><tr><td valign="top">CmC_<italic>Teleost</italic></td><td valign="top">1.88</td><td valign="top">&#8722;47071.54</td><td valign="top">0.02 (67%)</td><td valign="top">1 (3%)</td><td valign="top">0.14 (30%) <break/>Teleost: 0.24</td><td valign="top">M2a_rel</td><td valign="top">0.000 [1]</td></tr><tr><td valign="top">CmC_<italic>Teleost vs Tetrapod</italic></td><td valign="top">0*</td><td valign="top">&#8722;47069.60</td><td valign="top">0.02 (67%)</td><td valign="top">1 (3%)</td><td valign="top">0.17 (30%) <break/>Tetra: 0.13 <break/>Teleost: 0.24</td><td valign="top">M2a_rel</td><td valign="top">0.000 [2]</td></tr></tbody></table><table-wrap-foot><fn><p><sup>&#8224;</sup>The foreground partition is listed after the underscore for the clade models and consists of either: the clade of Teleost fishes (<italic>Teleost</italic>); the clade Tetrapods (<italic>Tetrapod;Tetra</italic>) or branch leading to tetrapods (<italic>Tetrapod branch; Tetra Br</italic>); or the clades of both the teleost fishes and tetrapods as two separate foregrounds (<italic>Teleost vs Tetrapods</italic>). In any partitioning scheme, the entire clade was tested, and all non-foreground data are present in the background partition.</p><p><sup>&#8225;</sup>All &#916;AIC values are calculated from the lowest AIC model. The best fit is shown with an asterisk (*).</p></fn><fn><p><italic>&#969;</italic><sub>d</sub> is the divergent site class, which has a separate value for the foreground and background partitions.</p><p><sup>&#182;</sup>Significant p-values (&#945;&#160;&#8804;0.05) are bolded. Degrees of freedom are given in square brackets after the p-values.</p></fn><fn><p>Abbreviations&#8212;<italic>ln</italic>L, ln Likelihood; p, p<italic>-</italic>value; AIC, Akaike information criterion.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35957-v1.xml">elife-35957-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table5" position="float"><object-id pub-id-type="doi">10.7554/eLife.35957.012</object-id><label>Table 5.</label><caption><title>Results of Clade Model C (CmC) analyses of teleost <italic>rh1</italic> under various partitions.&#160;</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Model and <break/>Foreground<sup>&#8224;</sup></th><th rowspan="2" valign="top">&#916;AIC<sup>&#8225;</sup></th><th rowspan="2" valign="top"><italic>ln</italic>L</th><th colspan="3" valign="top">Parameters</th><th rowspan="2" valign="top">Null</th><th rowspan="2" valign="top"><italic>P</italic> [df]</th></tr><tr><th valign="top"><italic>&#969;</italic><sub>0</sub></th><th valign="top"><italic>&#969;</italic><sub>1</sub></th><th valign="top"><italic>&#969;</italic><sub>2</sub>/<italic>&#969;</italic><sub>d</sub></th></tr></thead><tbody><tr><td valign="top">M2a_rel</td><td valign="top">17.1</td><td valign="top">&#8722;30987.99</td><td valign="top">0.01 (60%)</td><td valign="top">1 (5%)</td><td valign="top">0.19 (35%)</td><td valign="top">N/A</td><td valign="top">-</td></tr><tr><td valign="top">CmC_<italic>Characi branch</italic></td><td valign="top">19.05</td><td valign="top">&#8722;30986.96</td><td valign="top">0.01 (60%)</td><td valign="top">1 (5%)</td><td valign="top">0.19 (35%) <break/>Char Br: 0.20</td><td valign="top">M2a_rel</td><td valign="top">0.794 [1]</td></tr><tr><td valign="top">CmC_<italic>Characi clade</italic></td><td valign="top">0*</td><td valign="top">&#8722;30977.43</td><td valign="top">0.00 (60%)</td><td valign="top">1 (5%)</td><td valign="top">0.20 (20%) <break/>Char Cl: 0.10</td><td valign="top">M2a_rel</td><td valign="top"><bold>0.000 [1]</bold></td></tr></tbody></table><table-wrap-foot><fn><p>The foreground partition is listed after the underscore for the clade models and consists of either: the ancestral branch leading to the Characiphysi (<italic>Characi branch; Char Br</italic>) or the entire Characiphysi clade (<italic>Characi clade; Char Cl</italic>). In any partitioning scheme, the entire clade was tested, and all non-foreground data are present in the background partition.</p><p><sup>&#8225;</sup>All &#916;AIC values are calculated from the lowest AIC model. The best fit is bolded with an asterisk (*).</p></fn><fn><p><sup>&#167;</sup><italic>&#969;</italic><sub>d</sub> is the divergent site class, which has a separate value for the foreground and background partitions.</p><p>Significant p-values (&#945;&#160;&#8804;0.05) are bolded. Degrees of freedom are given in square brackets after the p-values.</p></fn><fn><p>Abbreviations<bold>&#8212;</bold><italic>ln</italic>L, ln Likelihood; p, p<italic>-</italic>value; AIC, Akaike information criterion.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35957-v1.xml">elife-35957-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table7" position="float"><object-id pub-id-type="doi">10.7554/eLife.35957.019</object-id><label>Table 7.</label><caption><title>Analyses of selection on Vertebrate rhodopsin (<italic>rh1</italic>) using PAML random sites models.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Model</th><th rowspan="2" valign="top"><italic>ln</italic>L</th><th colspan="3" valign="top">Parameters1</th><th rowspan="2" valign="top">Null</th><th rowspan="2" valign="top"><italic>P</italic> [df]2</th><th rowspan="2" valign="top">&#916; AIC<sup>&#167;</sup></th></tr><tr><th valign="top"><italic>&#969;</italic><sub>0</sub>/p</th><th valign="top"><italic>&#969;</italic><sub>1</sub>/q</th><th valign="top"><italic>&#969;</italic><sub>2</sub>/<italic>&#969;</italic><sub>p</sub></th></tr></thead><tbody><tr><td valign="top">M0</td><td valign="top">&#8722;49624.89</td><td valign="top">0.08</td><td valign="top">-</td><td valign="top">-</td><td valign="top">N/A</td><td valign="top">-</td><td valign="top">5516.80</td></tr><tr><td valign="top">M1a</td><td valign="top">&#8722;48355.44</td><td valign="top">0.05 (89%)</td><td valign="top">1.00 (11%)</td><td valign="top">-</td><td valign="top">M0</td><td valign="top">0.000 [1]</td><td valign="top">2979.91</td></tr><tr><td valign="top">M2a</td><td valign="top">&#8722;48355.44</td><td valign="top">0.05 (89%)</td><td valign="top">1.00 (3%)</td><td valign="top">1.00 (8%)</td><td valign="top">M1a</td><td valign="top">1 [2]</td><td valign="top">2983.91</td></tr><tr><td valign="top">M3</td><td valign="top">&#8722;47104.84</td><td valign="top">0.01 (58%)</td><td valign="top">0.11 (30%)</td><td valign="top">0.44 (12%)</td><td valign="top">M0</td><td valign="top"><bold>0.000 [4]</bold></td><td valign="top">484.71</td></tr><tr><td valign="top">M7</td><td valign="top">&#8722;46906.24</td><td valign="top">0.24</td><td valign="top">1.19</td><td valign="top">-</td><td valign="top">N/A</td><td valign="top">-</td><td valign="top">81.51</td></tr><tr><td valign="top">M8a</td><td valign="top">&#8722;46864.60</td><td valign="top">0.32</td><td valign="top">3.10</td><td valign="top">1.00</td><td valign="top">N/A</td><td valign="top"><bold>-</bold></td><td valign="top">0.230</td></tr><tr><td rowspan="2" valign="top">M8</td><td rowspan="2" valign="top">&#8722;46863.49</td><td rowspan="2" valign="top">0.32</td><td rowspan="2" valign="top">2.94</td><td rowspan="2" valign="top">1.14</td><td valign="top">M7</td><td valign="top"><bold>0.000 [2]</bold></td><td rowspan="2" valign="top">0*</td></tr><tr><td valign="top">M8a</td><td valign="top">0.135 [1]</td></tr></tbody></table><table-wrap-foot><fn><p><italic>1&#969;</italic> values of each site class are shown are shown for model M0-M3 (<italic>&#969;</italic><sub>0</sub>&#8211; <italic>&#969;</italic><sub>2</sub>) with the proportion of each site class in parentheses. For M7 and M8, the shape parameters, p and q, which describe the beta distribution are listed instead. In addition, the <italic>&#969;</italic> value for the positively selected site class (<italic>&#969;</italic><sub>p</sub>, with the proportion of sites in parentheses) is shown for M8.</p><p>2Significant p-values (&#945;&#160;&#8804;0.05) are bolded. Degrees of freedom are given in square brackets after the p-values.</p></fn><fn><p><sup>3#</sup>Model fits were assessed by Akaike information criterion differences to the best fitting model (asterisk).</p><p>Abbreviations&#8212;<italic>ln</italic>L, ln Likelihood; p, p<italic>-</italic>value; N/A, not applicable.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35957-v1.xml">elife-35957-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table8" position="float"><object-id pub-id-type="doi">10.7554/eLife.35957.020</object-id><label>Table 8.</label><caption><title>Analyses of selection on Teleost rhodopsin (<italic>rh1</italic>) using PAML random sites models.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Model</th><th rowspan="2" valign="top"><italic>ln</italic>L</th><th colspan="3" valign="top">Parameters1<sup>&#8224;</sup> </th><th rowspan="2" valign="top">Null</th><th rowspan="2" valign="top"><italic>P</italic> [df]<sup>&#8224;</sup></th><th rowspan="2" valign="top">&#916; AIC<sup>&#167;</sup></th></tr><tr><th valign="top"><italic>&#969;</italic><sub>0</sub>/p</th><th valign="top"><italic>&#969;</italic><sub>1</sub>/q</th><th valign="top"><italic>&#969;</italic><sub>2</sub>/<italic>&#969;</italic><sub>p</sub></th></tr></thead><tbody><tr><td valign="top">M0</td><td valign="top">&#8722;32949.46</td><td valign="top">0.10</td><td valign="top">-</td><td valign="top">-</td><td valign="top">N/A</td><td valign="top">-</td><td valign="top">4489.79</td></tr><tr><td valign="top">M1a</td><td valign="top">&#8722;31605.10</td><td valign="top">0.05 (86%)</td><td valign="top">1.00 (14%)</td><td valign="top">-</td><td valign="top">M0</td><td valign="top"><bold>0.000 [1]</bold></td><td valign="top">1803.10</td></tr><tr><td valign="top">M2a</td><td valign="top">&#8722;31605.10</td><td valign="top">0.05 (86%)</td><td valign="top">1.00 (10%)</td><td valign="top">1.00 (4%)</td><td valign="top">M1a</td><td valign="top">1 [2]</td><td valign="top">1807.10</td></tr><tr><td valign="top">M3</td><td valign="top">&#8722;30887.40</td><td valign="top">0.01 (58%)</td><td valign="top">0.13 (29%)</td><td valign="top">0.57 (13%)</td><td valign="top">M0</td><td valign="top"><bold>0.000 [4]</bold></td><td valign="top">373.67</td></tr><tr><td valign="top">M8a</td><td valign="top">&#8722;30790.44</td><td valign="top">0.28</td><td valign="top">3.05</td><td valign="top">1.00</td><td valign="top">N/A</td><td valign="top"><bold>-</bold></td><td valign="top">173.76</td></tr><tr><td valign="top">M7</td><td valign="top">&#8722;30767.11</td><td valign="top">0.19</td><td valign="top">0.68</td><td valign="top">-</td><td valign="top">N/A</td><td valign="top">-</td><td valign="top">127.10</td></tr><tr><td rowspan="2" valign="top">M8</td><td rowspan="2" valign="top">&#8722;30702.57</td><td rowspan="2" valign="top">0.25</td><td rowspan="2" valign="top">1.62</td><td rowspan="2" valign="top">1.92</td><td valign="top">M7</td><td valign="top"><bold>0.000 [2]</bold></td><td rowspan="2" valign="top">0*</td></tr><tr><td valign="top">M8a</td><td valign="top"><bold>0.000 [1]</bold></td></tr></tbody></table><table-wrap-foot><fn><p><italic>1&#969;</italic> values of each site class are shown are shown for model M0-M3 (<italic>&#969;</italic><sub>0</sub>&#8211; <italic>&#969;</italic><sub>2</sub>) with the proportion of each site class in parentheses. For M7 and M8, the shape parameters, p and q, which describe the beta distribution are listed instead. In addition, the <italic>&#969;</italic> value for the positively selected site class (<italic>&#969;</italic><sub>p</sub>, with the proportion of sites in parentheses) is shown for M8.</p><p>2Significant p-values (&#945;&#160;&#8804;0.05) are bolded. Degrees of freedom are given in square brackets after the p-values.</p></fn><fn><p><sup>3#&#167;</sup>Model fits were assessed by Akaike information criterion differences to the best fitting model (asterisk).</p><p>Abbreviations&#8212;<bold><italic>l</italic></bold><italic>n</italic>L, ln Likelihood; p, p<italic>-</italic>value; N/A, not applicable.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-35957-v1.xml">elife-35957-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table9" position="float"><object-id pub-id-type="doi">10.7554/eLife.35957.021</object-id><label>Table 9.</label><caption><title>Analyses of selection on Tetrapod rhodopsin (<italic>rh1</italic>) using PAML random sites models.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Model</th><th rowspan="2" valign="top"><italic>ln</italic>L</th><th colspan="3" valign="top">Parameters<sup>&#8224;</sup> </th><th rowspan="2" valign="top">Null</th><th rowspan="2" valign="top"><italic>P</italic> [df]<sup>&#8225;</sup></th><th rowspan="2" valign="top">&#916; AIC<sup>&#167;</sup></th></tr><tr><th valign="top"><italic>&#969;</italic><sub>0</sub>/p</th><th valign="top"><italic>&#969;</italic><sub>1</sub>/q</th><th valign="top"><italic>&#969;</italic><sub>2</sub>/<italic>&#969;</italic><sub>p</sub></th></tr></thead><tbody><tr><td valign="top">M0</td><td valign="top">&#8722;15541.64</td><td valign="top">0.05</td><td valign="top">-</td><td valign="top">-</td><td valign="top">N/A</td><td valign="top">-</td><td valign="top">1154.78</td></tr><tr><td valign="top">M1a</td><td valign="top">&#8722;15345.33</td><td valign="top">0.03 (93%)</td><td valign="top">1.00 (7%)</td><td valign="top">-</td><td valign="top">M0</td><td valign="top">[1]</td><td valign="top">764.17</td></tr><tr><td valign="top">M2a</td><td valign="top">&#8722;15345.33</td><td valign="top">0.03 (93%)</td><td valign="top">1.00 (0%)</td><td valign="top">1.00 (7%)</td><td valign="top">M1a</td><td valign="top">1 [2]</td><td valign="top">768.17</td></tr><tr><td valign="top">M3</td><td valign="top">&#8722;14981.40</td><td valign="top">0.00 (61%)</td><td valign="top">0.06 (28%)</td><td valign="top">0.29 (11%)</td><td valign="top">M0</td><td valign="top"><bold>0.000</bold> [4]</td><td valign="top">42.31</td></tr><tr><td valign="top">M7</td><td valign="top">&#8722;14971.78</td><td valign="top">0.19</td><td valign="top">2.76</td><td valign="top">-</td><td valign="top">N/A</td><td valign="top">-</td><td valign="top">17.10</td></tr><tr><td valign="top">M8</td><td valign="top">&#8722;14961.25</td><td valign="top">0.20</td><td valign="top">3.55</td><td valign="top">1.00</td><td valign="top">M7</td><td valign="top"><bold>0.000</bold> [2]</td><td valign="top">0*</td></tr></tbody></table><table-wrap-foot><fn><p><italic>1&#969;</italic> values of each site class are shown are shown for model M0-M3 (<italic>&#969;</italic><sub>0</sub>&#8211; <italic>&#969;</italic><sub>2</sub>) with the proportion of each site class in parentheses. For M7 and M8, the shape parameters, p and q, which describe the beta distribution are listed instead. In addition, the <italic>&#969;</italic> value for the positively selected site class (<italic>&#969;</italic><sub>p</sub>, with the proportion of sites in parentheses) is shown for M8.</p><p>2Significant p-values (&#945;&#160;&#8804;0.05) are bolded. Degrees of freedom are given in square brackets after the p-values.</p></fn><fn><p><sup>3#</sup>Model fits were assessed by Akaike information criterion differences to the best fitting model (bolded asterisk).</p><p>Abbreviations&#8212;<bold><italic>l</italic></bold><italic>n</italic>L, ln Likelihood; p, p<italic>-</italic>value; N/A, not applicable.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36018-v2.xml">elife-36018-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.36018.012</object-id><label>Table 1.</label><caption><title>Best-fitting parameters of HDDM.</title><p><supplementary-material id="table1sdata1"><object-id pub-id-type="doi">10.7554/eLife.36018.013</object-id><label>Table 1&#8212;source data 1.</label><caption><title>HDDM model fitting parameters for each session.</title><p>The same data are also used in <xref ref-type="fig" rid="fig3">Figures 3</xref>, <xref ref-type="fig" rid="fig4">4</xref>, <xref ref-type="fig" rid="fig5">5</xref>, <xref ref-type="fig" rid="fig5s1">Figure 5&#8212;figure supplement 1</xref>, <xref ref-type="fig" rid="fig6">Figure 6</xref>, <xref ref-type="fig" rid="fig6s1">Figure 6&#8212;figure supplements 1</xref>, <xref ref-type="fig" rid="fig6s2">2</xref>, <xref ref-type="fig" rid="fig6s3">3</xref>, <xref ref-type="fig" rid="fig8">Figure 8</xref> and <xref ref-type="fig" rid="fig8s1">Figure 8&#8212;figure supplement 1</xref>)</p></caption><media mime-subtype="zip" mimetype="application" xlink:href="elife-36018-table1-data1-v2.zip"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="bottom"/><th colspan="4" valign="bottom">Monkey F (26079 trials)</th><th colspan="4" valign="bottom">Monkey C (37161 trials)</th><th colspan="4" valign="bottom">Monkey A (21089 trials)</th></tr><tr><th colspan="2" valign="bottom">LR-Left</th><th colspan="2" valign="bottom">LR-Right</th><th colspan="2" valign="bottom">LR-Left</th><th colspan="2" valign="bottom">LR-Right</th><th colspan="2" valign="bottom">LR-Left</th><th colspan="2" valign="bottom">LR-Right</th></tr><tr><th>Mean</th><th>Std</th><th>Mean</th><th>Std</th><th>Mean</th><th>Std</th><th>Mean</th><th>Std</th><th>Mean</th><th>Std</th><th>Mean</th><th>Std</th></tr></thead><tbody><tr><td valign="bottom"><italic>a</italic></td><td>1.67</td><td>0.16</td><td>1.43</td><td>0.12</td><td>1.77</td><td>0.09</td><td>1.53</td><td>0.13</td><td>1.33</td><td>0.13</td><td>1.36</td><td>0.09</td></tr><tr><td valign="bottom"><italic>k</italic></td><td>10.22</td><td>1.87</td><td>9.91</td><td>2.11</td><td>6.58</td><td>0.51</td><td>5.08</td><td>0.92</td><td>4.04</td><td>0.33</td><td>3.45</td><td>0.46</td></tr><tr><td valign="bottom"><italic>t<sub>1</sub></italic></td><td>0.31</td><td>0.03</td><td>0.29</td><td>0.03</td><td>0.35</td><td>0.04</td><td>0.33</td><td>0.05</td><td>0.29</td><td>0.04</td><td>0.27</td><td>0.04</td></tr><tr><td valign="bottom"><italic>t<sub>0</sub></italic></td><td>0.28</td><td>0.04</td><td>0.31</td><td>0.05</td><td>0.33</td><td>0.04</td><td>0.31</td><td>0.03</td><td>0.21</td><td>0.08</td><td>0.26</td><td>0.04</td></tr><tr><td valign="bottom"><italic>z</italic></td><td>0.60</td><td>0.03</td><td>0.57</td><td>0.04</td><td>0.62</td><td>0.03</td><td>0.40</td><td>0.04</td><td>0.57</td><td>0.06</td><td>0.39</td><td>0.04</td></tr><tr><td valign="bottom"><italic>me</italic></td><td>&#8722;0.06</td><td>0.04</td><td>0.08</td><td>0.05</td><td>&#8722;0.14</td><td>0.04</td><td>0.21</td><td>0.06</td><td>&#8722;0.22</td><td>0.05</td><td>0.27</td><td>0.09</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36018-v2.xml">elife-36018-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.36018.014</object-id><label>Table 2.</label><caption><title>The difference in deviance information criterion (&#916;DIC) between the full model (i.e., the model that includes both <italic>me</italic> and <italic>z</italic>) and either reduced model (<italic>me-</italic>only or <italic>z-</italic>only), for experimental data and data simulated using each reduced model.</title><p>Negative/positive values favor the full/reduced model. Note that the &#916;DIC values for the experimental data were all strongly negative, favoring the full model. In contrast, the &#916;DIC values for the simulated data were all positive, implying that this procedure did not simply prefer the more complex model.</p><p><supplementary-material id="table2sdata1"><object-id pub-id-type="doi">10.7554/eLife.36018.015</object-id><label>Table 2&#8212;source data 1.</label><caption><title>DIC for model fitting to the monkeys&#8217; data and to the simulated data.</title></caption><media mime-subtype="zip" mimetype="application" xlink:href="elife-36018-table2-data1-v2.zip"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="bottom"/><th colspan="4" valign="bottom">Experimental data</th><th colspan="2" valign="bottom">Simu: <italic>me</italic> model</th><th colspan="2" valign="bottom">Simu: <italic>z</italic> model</th></tr><tr><th colspan="2" valign="bottom">&#8710;DIC: full - me</th><th colspan="2" valign="bottom">&#8710;DIC: full - z</th><th colspan="2" valign="bottom">&#8710;DIC: full - me</th><th colspan="2" valign="bottom">&#8710;DIC: full - z</th></tr><tr><th valign="bottom">Mean</th><th valign="bottom">Std</th><th valign="bottom">Mean</th><th valign="bottom">Std</th><th valign="bottom">Mean</th><th valign="bottom">Std</th><th valign="bottom">Mean</th><th valign="bottom">Std</th></tr></thead><tbody><tr><td valign="bottom">Monkey F</td><td valign="bottom">&#8722;124.6</td><td valign="bottom">2.3</td><td valign="bottom">&#8722;2560.4</td><td valign="bottom">5.2</td><td valign="bottom">3.1</td><td valign="bottom">9.8</td><td valign="bottom">0.2</td><td valign="bottom">11.8</td></tr><tr><td valign="bottom">Monkey C</td><td valign="bottom">&#8722;1700.4</td><td valign="bottom">2.1</td><td valign="bottom">&#8722;6937.9</td><td valign="bottom">1.3</td><td valign="bottom">17.5</td><td valign="bottom">11.3</td><td valign="bottom">1.8</td><td valign="bottom">1.3</td></tr><tr><td valign="bottom">Monkey A</td><td valign="bottom">&#8722;793.6</td><td valign="bottom">3.4</td><td valign="bottom">&#8722;2225.7</td><td valign="bottom">4.0</td><td valign="bottom">25.4</td><td valign="bottom">9.0</td><td valign="bottom">1.2</td><td valign="bottom">3.4</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36234-v2.xml">elife-36234-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.36234.021</object-id><label>Table 3.</label><caption><title>Reproduction data and pup weight after in&#160;utero WIN exposure.</title><p>Dam weight gain was calculated from GD 1 to GD 21 for n&#160;=&#160;10 dams per treatment group. Pup weight at different postnatal days (PND) was calculated for n&#160;=&#160;9&#8211;10 male and female pups from different litters. Data represent mean values&#160;&#177;&#160;SEM. Statistical significance was determined using the one-way ANOVA test.</p></caption><table frame="hsides" rules="groups"><thead><tr><th/><th rowspan="2" valign="top"/><th/><th/><th/><th rowspan="3">p-Value <break/>(ANOVA)</th></tr><tr><th/><th>Naive</th><th>SHAM</th><th>WIN</th></tr><tr><th valign="top"/><th valign="top"/><th>mean&#160;&#177;&#160;SEM</th><th>mean&#160;&#177;&#160;SEM</th><th>mean&#160;&#177;&#160;SEM</th></tr></thead><tbody><tr><td/><td rowspan="2">Dam weight gain (%)</td><td/><td/><td/><td/></tr><tr><td/><td>33.8&#160;&#177;&#160;0.93</td><td>34.3&#160;&#177;&#160;1.91</td><td>33.1&#160;&#177;&#160;1.42</td><td>0.745</td></tr><tr><td/><td rowspan="2">Pregnancy length (days)</td><td/><td/><td/><td/></tr><tr><td>Reproduction data</td><td>22.4&#160;&#177;&#160;0.14</td><td>22.6&#160;&#177;&#160;0.29</td><td>22.6&#160;&#177;&#160;0.17</td><td>0.923</td></tr><tr><td valign="top"/><td/><td/><td/><td/><td/></tr><tr><td valign="top"/><td>Litter size</td><td>13&#160;&#177;&#160;0.82</td><td>12.9&#160;&#177;&#160;0.78</td><td>12.7&#160;&#177;&#160;0.67</td><td>0.854</td></tr><tr><td/><td>PND 1</td><td>6.8&#160;&#177;&#160;0.2</td><td>6.7&#160;&#177;&#160;0.1</td><td>6.8&#160;&#177;&#160;0.2</td><td>0.965</td></tr><tr><td/><td>PND 10</td><td>23.5&#160;&#177;&#160;0.89</td><td>24.6&#160;&#177;&#160;0.65</td><td>22.6&#160;&#177;&#160;0.72</td><td>0.207</td></tr><tr><td>Pup weight (grams)</td><td>PND 13</td><td>31.8&#160;&#177;&#160;0.62</td><td>30.3&#160;&#177;&#160;0.82</td><td>30.5&#160;&#177;&#160;0.75</td><td>0.339</td></tr><tr><td valign="top"/><td>PND 25</td><td>64.5&#160;&#177;&#160;2.24</td><td>65.6&#160;&#177;&#160;0.77</td><td>66.7&#160;&#177;&#160;1.55</td><td>0.647</td></tr><tr><td valign="top"/><td>PND 90</td><td>452.7&#160;&#177;&#160;12.28</td><td>484.2&#160;&#177;&#160;12.52</td><td>481.7&#160;&#177;&#160;17.33</td><td>0.261</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36326-v2.xml">elife-36326-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.36326.020</object-id><label>Table 1.</label><caption><title><italic>L<sub>onset</sub></italic>, <italic>L<sub>max</sub></italic>, and <italic>L<sub>end</sub></italic> values calculated from the <italic>f<sub>FL</sub></italic> profiles in <xref ref-type="fig" rid="fig3">Figure 3</xref>.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th valign="top"/><th valign="top">ADR1a</th><th valign="top"/><th valign="top"/><th valign="top">R16</th><th valign="top"/><th valign="top"/><th valign="top">I27</th><th valign="top"/></tr><tr><th valign="top">WT</th><th valign="top">uL23 &#916;loop</th><th valign="top">uL24 &#916;loop</th><th valign="top">WT</th><th valign="top">uL23 &#916;loop</th><th valign="top">uL24 &#916;loop</th><th valign="top">WT</th><th valign="top">uL23 &#916;loop</th><th valign="top">uL24 &#916;loop</th></tr></thead><tbody><tr><td valign="top"><italic>L<sub>onset</sub></italic></td><td valign="top">21</td><td valign="top">17</td><td valign="top">20</td><td valign="top">31</td><td valign="top">31</td><td valign="top">29</td><td valign="top">32</td><td valign="top">33</td><td valign="top">28</td></tr><tr><td valign="top"><italic>L<sub>max</sub></italic></td><td valign="top">25</td><td valign="top">22</td><td valign="top">25</td><td valign="top">35</td><td valign="top">35</td><td valign="top">33</td><td valign="top">35</td><td valign="top">35</td><td valign="top">35</td></tr><tr><td valign="top"><italic>L<sub>end</sub></italic></td><td valign="top">27</td><td valign="top">26</td><td valign="top">27</td><td valign="top">42</td><td valign="top">39</td><td valign="top">40</td><td valign="top">38</td><td valign="top">38</td><td valign="top">41</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36340-v2.xml">elife-36340-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.36340.009</object-id><label>Table 2.</label><caption><title>In vitro inhibition of HIV-1 reverse transcriptase by 25a and RPV.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">RT variants</th><th colspan="3" valign="top">25a</th><th colspan="3" valign="top">RPV</th></tr><tr><th valign="top">IC<sub>50</sub> (nM)</th><th valign="top">Fold R<sup>*</sup></th><th valign="top">Curve slope</th><th valign="top">IC<sub>50</sub> (nM)</th><th valign="top">Fold R</th><th valign="top">Curve slope</th></tr></thead><tbody><tr><td valign="top">WT</td><td valign="top">4.3&#160;&#177;&#160;0.080<sup>&#8224;</sup></td><td valign="top">&#8211;</td><td valign="top">3.8&#160;&#177;&#160;0.70</td><td valign="top">3.5&#160;&#177;&#160;0.052</td><td valign="top">&#8211;</td><td valign="top">3.1&#160;&#177;&#160;0.17</td></tr><tr><td valign="top">K103N/Y181C</td><td valign="top">31&#160;&#177;&#160;0.83</td><td valign="top">7.2</td><td valign="top">1.9&#160;&#177;&#160;0.092</td><td valign="top">51&#160;&#177;&#160;1.5</td><td valign="top">15</td><td valign="top">1.4&#160;&#177;&#160;0.047</td></tr><tr><td valign="top">Y188L</td><td valign="top">3.0&#160;&#177;&#160;0.15</td><td valign="top">0.70</td><td valign="top">1.9&#160;&#177;&#160;0.16</td><td valign="top">7.6&#160;&#177;&#160;0.22</td><td valign="top">2.2</td><td valign="top">1.7&#160;&#177;&#160;0.069</td></tr><tr><td valign="top">V106A/F227L</td><td valign="top">7.3&#160;&#177;&#160;0.23</td><td valign="top">1.7</td><td valign="top">3.1&#160;&#177;&#160;0.17</td><td valign="top">14&#160;&#177;&#160;0.27</td><td valign="top">4.0</td><td valign="top">2.1&#160;&#177;&#160;0.083</td></tr><tr><td valign="top">K101P</td><td valign="top">5.4&#160;&#177;&#160;0.16</td><td valign="top">1.3</td><td valign="top">2.9&#160;&#177;&#160;0.21</td><td valign="top">71&#160;&#177;&#160;3.0</td><td valign="top">20</td><td valign="top">1.2&#160;&#177;&#160;0.056</td></tr><tr><td valign="top">Y181I</td><td valign="top">38&#160;&#177;&#160;1.1</td><td valign="top">8.8</td><td valign="top">2.3&#160;&#177;&#160;0.16</td><td valign="top">315&#160;&#177;&#160;15</td><td valign="top">90</td><td valign="top">1.4&#160;&#177;&#160;0.081</td></tr><tr><td valign="top">K103N/Y181I</td><td valign="top">412&#160;&#177;&#160;13</td><td valign="top">96</td><td valign="top">1.8&#160;&#177;&#160;0.094</td><td valign="top">6317&#160;&#177;&#160;339</td><td valign="top">1805</td><td valign="top">1.4&#160;&#177;&#160;0.10</td></tr><tr><td valign="top">P225H</td><td valign="top">2.5&#160;&#177;&#160;0.059</td><td valign="top">0.58</td><td valign="top">3.5&#160;&#177;&#160;0.15</td><td valign="top">3.7&#160;&#177;&#160;0.070</td><td valign="top">1.1</td><td valign="top">3.4&#160;&#177;&#160;0.34</td></tr><tr><td valign="top">P236L</td><td valign="top">2.6&#160;&#177;&#160;0.13</td><td valign="top">0.60</td><td valign="top">4.3&#160;&#177;&#160;0.43</td><td valign="top">3.7&#160;&#177;&#160;0.085</td><td valign="top">1.1</td><td valign="top">3.6&#160;&#177;&#160;0.47</td></tr></tbody></table><table-wrap-foot><fn><p>* Mean fold change in the IC<sub>50</sub> values of mutant RT versus WT RT.</p><p>&#8224; Data reported as mean&#160;&#177;standard error.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36340-v2.xml">elife-36340-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.36340.013</object-id><label>Table 4.</label><caption><title>Buried area (&#197;<sup>2</sup>) between HIV-1 RT and each NNRTI<sup>*</sup>.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top">Total</th><th colspan="9" valign="top">Individual residue</th></tr><tr><th valign="top">101</th><th valign="top">103</th><th valign="top">106</th><th valign="top">181</th><th valign="top">183</th><th valign="top">188</th><th valign="top">227</th><th valign="top">236</th><th valign="top">138</th></tr></thead><tbody><tr><td valign="top">WT RT/K-5a2</td><td valign="top">584.1</td><td valign="top">46.5</td><td valign="top">81.2</td><td valign="top">91.1</td><td valign="top">63.9</td><td valign="top">0</td><td valign="top">75.4</td><td valign="top">89.3</td><td valign="top">69.6</td><td valign="top">54.6</td></tr><tr><td valign="top">K103N RT/K-5a2</td><td valign="top">579.4</td><td valign="top">44.3</td><td valign="top">73.4</td><td valign="top">85.5</td><td valign="top">59.6</td><td valign="top">0</td><td valign="top">71.2</td><td valign="top">92.9</td><td valign="top">71.2</td><td valign="top">52.6</td></tr><tr><td valign="top">E138K RT/K-5a2</td><td valign="top">587.9</td><td valign="top">44.3</td><td valign="top">83.5</td><td valign="top">89.6</td><td valign="top">63.0</td><td valign="top">0</td><td valign="top">72.9</td><td valign="top">86.4</td><td valign="top">69.9</td><td valign="top">54.3</td></tr><tr><td valign="top">Y188L RT/K-5a2</td><td valign="top">571.3</td><td valign="top">44.3</td><td valign="top">83.0</td><td valign="top">93.9</td><td valign="top">57.9</td><td valign="top">0</td><td valign="top">70.0</td><td valign="top">73.2</td><td valign="top">74.0</td><td valign="top">51.5</td></tr><tr><td valign="top">WT RT/25a</td><td valign="top">620.5</td><td valign="top">51.9</td><td valign="top">88.2</td><td valign="top">91.7</td><td valign="top">68.5</td><td valign="top">19.6</td><td valign="top">84.8</td><td valign="top">96.9</td><td valign="top">68.4</td><td valign="top">54.8</td></tr><tr><td valign="top">K103N RT/25a</td><td valign="top">626.8</td><td valign="top">47.9</td><td valign="top">74.7</td><td valign="top">86.4</td><td valign="top">67.8</td><td valign="top">15.6</td><td valign="top">84.3</td><td valign="top">106</td><td valign="top">72.0</td><td valign="top">53.7</td></tr><tr><td valign="top">E138K RT/25a</td><td valign="top">621.4</td><td valign="top">45.5</td><td valign="top">87.1</td><td valign="top">92.5</td><td valign="top">66.4</td><td valign="top">14.0</td><td valign="top">86.6</td><td valign="top">98.4</td><td valign="top">69.1</td><td valign="top">54.1</td></tr><tr><td valign="top">K103N/Y181C RT/25a</td><td valign="top">624.7</td><td valign="top">51.9</td><td valign="top">76.1</td><td valign="top">86.8</td><td valign="top">56.5</td><td valign="top">33.5</td><td valign="top">83.2</td><td valign="top">106</td><td valign="top">73.3</td><td valign="top">54.1</td></tr><tr><td valign="top">V106A/F227L RT/25a</td><td valign="top">614.9</td><td valign="top">47.6</td><td valign="top">87.1</td><td valign="top">79.2</td><td valign="top">58.1</td><td valign="top">0</td><td valign="top">79.2</td><td valign="top">91.8</td><td valign="top">62.8</td><td valign="top">52.0</td></tr><tr><td valign="top">K101P RT/25a</td><td valign="top">613.6</td><td valign="top">62.6</td><td valign="top">93.9</td><td valign="top">90.7</td><td valign="top">66.6</td><td valign="top">21.1</td><td valign="top">85.6</td><td valign="top">89.9</td><td valign="top">70.5</td><td valign="top">53.5</td></tr><tr><td valign="top">Y181I RT/25a</td><td valign="top">618.5</td><td valign="top">62.9</td><td valign="top">90.9</td><td valign="top">90.8</td><td valign="top">58.9</td><td valign="top">31.5</td><td valign="top">84.6</td><td valign="top">99.8</td><td valign="top">71.7</td><td valign="top">52.4</td></tr></tbody></table><table-wrap-foot><fn><p>* The buried area between HIV-1 RT and each NNRTI was calculated using UCSF ChimeraX.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36340-v2.xml">elife-36340-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table5" position="float"><object-id pub-id-type="doi">10.7554/eLife.36340.014</object-id><label>Table 5.</label><caption><title>Torsion angles and energies of K-5a2 and 25a in different binding poses.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="8" valign="top">Torsion angles (&#176;)</th><th rowspan="2" valign="top">NNRTI energy<sup>*</sup> <break/>(kcal/mol)</th></tr><tr><th valign="top"><bold>&#964;1</bold></th><th valign="top"><bold>&#964;2</bold></th><th valign="top"><bold>&#964;3</bold></th><th valign="top"><bold>&#964;4</bold></th><th valign="top"><bold>&#964;5</bold></th><th valign="top"><bold>&#964;6</bold></th><th valign="top"><bold>&#964;7</bold></th><th valign="top"><bold>&#964;8</bold></th></tr></thead><tbody><tr><td valign="top">&#8195;WT RT/K-5a2</td><td valign="top">14</td><td valign="top">&#8722;17</td><td valign="top">&#8722;84</td><td valign="top">&#8722;71</td><td valign="top">2</td><td valign="top">&#8722;8</td><td valign="top">&#8722;97</td><td valign="top">&#8212;</td><td valign="top">&#8722;161.4</td></tr><tr><td valign="top">&#8195;K103N RT/K-5a2</td><td valign="top">11</td><td valign="top">&#8722;23</td><td valign="top">&#8722;83</td><td valign="top">&#8722;73</td><td valign="top">6</td><td valign="top">&#8722;11</td><td valign="top">&#8722;102</td><td valign="top">&#8212;</td><td valign="top">&#8722;162.4</td></tr><tr><td valign="top">&#8195;E138K RT/K-5a2</td><td valign="top">9</td><td valign="top">&#8722;19</td><td valign="top">&#8722;87</td><td valign="top">&#8722;68</td><td valign="top">2</td><td valign="top">&#8722;9</td><td valign="top">&#8722;97</td><td valign="top">&#8212;</td><td valign="top">&#8722;162.7</td></tr><tr><td valign="top">&#8195;Y188L RT/K-5a2</td><td valign="top">3</td><td valign="top">&#8722;49</td><td valign="top">&#8722;66</td><td valign="top">&#8722;70</td><td valign="top">4</td><td valign="top">&#8722;2</td><td valign="top">&#8722;94</td><td valign="top">&#8212;</td><td valign="top">&#8722;160.9</td></tr><tr><td valign="top">&#8195;WT RT/ETR<sup>&#8224;</sup></td><td valign="top">&#8212;</td><td valign="top">&#8212;</td><td valign="top">&#8212;</td><td valign="top">16</td><td valign="top">&#8722;2</td><td valign="top">&#8722;13</td><td valign="top">&#8722;95</td><td valign="top">&#8212;</td><td valign="top">N/A</td></tr><tr><td valign="top">&#8195;WT RT/25a</td><td valign="top">23</td><td valign="top">&#8722;26</td><td valign="top">&#8722;79</td><td valign="top">&#8722;79</td><td valign="top">1</td><td valign="top">0</td><td valign="top">&#8722;109</td><td valign="top">&#8722;40</td><td valign="top">&#8722;191.1</td></tr><tr><td valign="top">&#8195;K103N RT/25a</td><td valign="top">7</td><td valign="top">&#8722;22</td><td valign="top">&#8722;81</td><td valign="top">&#8722;74</td><td valign="top">3</td><td valign="top">0</td><td valign="top">&#8722;110</td><td valign="top">&#8722;53</td><td valign="top">&#8722;187.7</td></tr><tr><td valign="top">&#8195;E138K RT/25a</td><td valign="top">19</td><td valign="top">&#8722;29</td><td valign="top">&#8722;76</td><td valign="top">&#8722;77</td><td valign="top">0</td><td valign="top">3</td><td valign="top">&#8722;108</td><td valign="top">&#8722;53</td><td valign="top">&#8722;188.5</td></tr><tr><td valign="top">&#8195;K103N/Y181C RT/25a</td><td valign="top">&#8722;4</td><td valign="top">&#8722;26</td><td valign="top">&#8722;76</td><td valign="top">&#8722;77</td><td valign="top">4</td><td valign="top">&#8722;1</td><td valign="top">&#8722;107</td><td valign="top">&#8722;54</td><td valign="top">&#8722;186.2</td></tr><tr><td valign="top">&#8195;V106A/F227L RT/25a</td><td valign="top">14</td><td valign="top">&#8722;24</td><td valign="top">&#8722;72</td><td valign="top">&#8722;82</td><td valign="top">4</td><td valign="top">&#8722;4</td><td valign="top">&#8722;103</td><td valign="top">&#8722;163</td><td valign="top">&#8722;187.3</td></tr><tr><td valign="top">&#8195;K101P RT/25a</td><td valign="top">5</td><td valign="top">&#8722;24</td><td valign="top">&#8722;84</td><td valign="top">&#8722;81</td><td valign="top">5</td><td valign="top">6</td><td valign="top">&#8722;112</td><td valign="top">&#8722;44</td><td valign="top">&#8722;190.1</td></tr><tr><td valign="top">&#8195;Y181I RT/25a</td><td valign="top">&#8722;2</td><td valign="top">&#8722;15</td><td valign="top">&#8722;87</td><td valign="top">&#8722;72</td><td valign="top">6</td><td valign="top">4</td><td valign="top">&#8722;107</td><td valign="top">&#8722;57</td><td valign="top">&#8722;189.2</td></tr><tr><td valign="top">&#8195;WT RT/RPV<sup>&#8225;</sup></td><td valign="top">&#8212;</td><td valign="top">&#8212;</td><td valign="top">&#8212;</td><td valign="top">10</td><td valign="top">&#8722;7</td><td valign="top">&#8722;13</td><td valign="top">&#8722;103</td><td valign="top">&#8722;28</td><td valign="top">N/A</td></tr></tbody></table><table-wrap-foot><fn><p>*&#160;The NNRTI energy refers to the energy of K-5a2 or 25a itself at the specific conformations in different RT complexes. It was calculated using the MacroModel program in the Schr&#246;dinger software suite.</p><p>&#8224;&#160;Torsion angles of ETR were measured using the structure from PDB ID: 3MEC.</p></fn><fn><p>&#8225;&#160;Torsion angles of RPV were measured using the structure from PDB ID: 4G1Q.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36401-v3.xml">elife-36401-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.36401.005</object-id><label>Table 1.</label><caption><title>Summary of all mice.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Experimental group</th><th rowspan="2">Injection site</th><th align="center" colspan="6">Behavioral assays</th></tr><tr><th valign="bottom">Grooming</th><th valign="bottom">Social chamber</th><th valign="bottom">Y-maze</th><th valign="bottom">Elevated Plus Maze</th><th valign="bottom">Gait</th><th valign="bottom">Eyeblink</th></tr></thead><tbody><tr><td rowspan="5"> <break/><break/>Adult</td><td>Lobule VI</td><td>12</td><td>12</td><td>12</td><td>12</td><td>12</td><td>8</td></tr><tr><td>Lobule VII</td><td>10</td><td>10</td><td>7*</td><td>10</td><td>10</td><td>-</td></tr><tr><td>Crus I</td><td>10</td><td>10</td><td>10</td><td>10</td><td>10</td><td>8</td></tr><tr><td>Crus II</td><td>11</td><td>11</td><td>11</td><td>11</td><td>11</td><td>6</td></tr><tr><td>Eyeblink</td><td>&#160;-</td><td>&#160;-</td><td>&#160;-</td><td>&#160;-</td><td>&#160;-</td><td>9</td></tr><tr><td colspan="8">*not counting&#160;three mice excluded from group comparison during habituation phase <break/> <break/><break/></td></tr><tr><td rowspan="4"> <break/> <break/><break/>Developmental</td><td>Lobule VI</td><td>13</td><td>13</td><td>13</td><td>13</td><td>13</td><td>6**</td></tr><tr><td>Lobule VII</td><td>&#160;8</td><td>&#160;8</td><td>&#160;8</td><td>&#160;8</td><td>&#160;8</td><td>&#160;-</td></tr><tr><td>Crus I</td><td>&#160;7</td><td>&#160;7</td><td>&#160;7</td><td>&#160;7</td><td>&#160;7</td><td>6**</td></tr><tr><td>Crus II</td><td>12</td><td>12</td><td>12</td><td>12</td><td>12</td><td>3</td></tr><tr><td colspan="8">**not counting four mice (1 crus I, 3 lobule VI) excluded due to eyeblink-zone spillover expression <break/> <break/><break/></td></tr><tr><td>Controls</td><td>Type</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td rowspan="5"> <break/> <break/> <break/><break/>Adult</td><td>DREADDs (eyeblink zone)+saline injections</td><td>&#160;-</td><td>&#160;-</td><td>&#160;-</td><td>&#160;-</td><td>&#160;-</td><td>&#160;5</td></tr><tr><td valign="bottom">Saline i.p.</td><td>&#160;9</td><td>&#160;9</td><td>&#160;9</td><td>&#160;9</td><td>&#160;9</td><td>&#160;-</td></tr><tr><td>CNO alone</td><td>10</td><td>10</td><td>10</td><td>10</td><td>10</td><td>&#160;9</td></tr><tr><td>No treatment</td><td>&#160;8</td><td>&#160;8</td><td>17</td><td>18</td><td>18</td><td>16</td></tr><tr><td>GCaMP6f injections&#160;+&#160;CNO</td><td>&#160;-</td><td>&#160;-</td><td>&#160;8</td><td>&#160;-</td><td>&#160;-</td><td>&#160;-</td></tr><tr><td rowspan="4">Developmental</td><td rowspan="4">CNO alone</td><td rowspan="4">10</td><td rowspan="4">10</td><td rowspan="4">10</td><td rowspan="4">10</td><td rowspan="4">10</td><td rowspan="4">&#160;-</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36401-v3.xml">elife-36401-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.36401.023</object-id><label>Table 3.</label><caption><title>Cohen&#8217;s d effect size of motor and non-motor metrics.</title><p><supplementary-material id="table3sdata1"><object-id pub-id-type="doi">10.7554/eLife.36401.024</object-id><label>Table 3&#8212;source data 1.</label><caption><title>Extended <xref ref-type="table" rid="table3">Table 3</xref>.</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-36401-table3-data1-v3.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" colspan="10" valign="top">Flexible/Social/Learning&#160;metrics</th></tr><tr><th rowspan="2" valign="bottom"/><th rowspan="2" valign="bottom"/><th align="center" colspan="2" valign="top"><bold>Y-maze&#160;reversal</bold></th><th align="center" colspan="2"><bold>Three-chamber</bold></th><th align="center" colspan="2"><bold>Elevated plus-maze</bold></th><th align="center"><bold>Grooming</bold></th><th align="center"><bold>Eyeblink conditioning</bold></th></tr><tr><th align="center" valign="top">Learning(Finallearning)</th><th align="center">Persistive behavior (Multisession reversal1)</th><th align="center">Social preference</th><th align="center">Novelty-seeking</th><th align="center">Open-arm preference</th><th align="center">Exploratory behavior</th><th align="center">Grooming ratio</th><th>% Conditional responses session 11</th></tr></thead><tbody><tr><td rowspan="2"><bold>Lobule VI</bold></td><td>Developmental</td><td align="center">0.1</td><td align="center" style="author-callout-style-b6"><italic><bold>1.1</bold></italic></td><td align="center">&#8722;0.1</td><td align="center">0.7</td><td align="center">0.3</td><td align="center">0.4</td><td align="center">&#8722;0.6</td><td align="center">&#8722;0.3</td></tr><tr><td>Acute</td><td align="center"><italic><bold>0.5</bold></italic></td><td align="center" style="author-callout-style-b6"><italic><bold>0.9</bold></italic></td><td align="center">0.2</td><td align="center">0.0</td><td align="center">0.3</td><td align="center">0.6</td><td align="center">0.1</td><td align="center">&#8194;0.1</td></tr><tr><td colspan="10" valign="bottom"/></tr><tr><td rowspan="2"><bold>Lobule VII</bold></td><td valign="bottom">Developmental</td><td align="center">0.6</td><td align="center">&#8722;0.3</td><td align="center">&#8722;0.3</td><td align="center" style="author-callout-style-b6"><italic><bold>1.1</bold></italic></td><td align="center" style="author-callout-style-b6"><italic><bold>0.9</bold></italic></td><td align="center" style="author-callout-style-b6"><italic><bold>1.7</bold></italic></td><td align="center" style="author-callout-style-b6"><italic><bold>1.2</bold></italic></td><td align="center">&#8194;nd</td></tr><tr><td valign="bottom">Acute</td><td align="center">&#8722;0.6</td><td align="center">0.3</td><td align="center" style="author-callout-style-b6"><italic><bold>0.9</bold></italic></td><td align="center">&#8722;0.4</td><td align="center" style="author-callout-style-b1"><bold>&#8722;0.9</bold></td><td align="center">0.5</td><td align="center">0.2</td><td align="center">&#8194;nd</td></tr><tr><td colspan="10" valign="bottom"/></tr><tr><td rowspan="2" valign="bottom"><bold>Crus I</bold></td><td valign="bottom">Developmental</td><td align="center">0.4</td><td align="center">0.6</td><td align="center" style="author-callout-style-b6"><bold>1.2</bold></td><td align="center" style="author-callout-style-b1">&#8722;0.8</td><td align="center">0.1</td><td align="center">0.3</td><td align="center">0.0</td><td align="center" style="author-callout-style-b6"><italic><bold>&#8194;1.6</bold></italic></td></tr><tr><td valign="bottom">Acute</td><td align="center">0.0</td><td align="center">0.3</td><td align="center">0.0</td><td align="center">0.3</td><td align="center">&#8722;0.6</td><td align="center">&#8722;0.1</td><td align="center" style="author-callout-style-b6">0.9</td><td align="center" style="author-callout-style-b6"><italic><bold>&#8194;1.0</bold></italic></td></tr><tr><td colspan="10" valign="bottom"/></tr><tr><td rowspan="2" valign="bottom"><bold>Crus II</bold></td><td valign="bottom">Developmental</td><td align="center">0.1</td><td align="center">&#8722;0.5</td><td align="center" style="author-callout-style-b6"><italic><bold>1.2</bold></italic></td><td align="center">0.0</td><td align="center">0.6</td><td align="center">&#8722;0.3</td><td align="center">&#8722;0.5</td><td align="center">&#8722;0.5</td></tr><tr><td valign="bottom">Acute</td><td align="center">&#8722;0.6</td><td align="center">&#8722;0.6</td><td align="center">0.0</td><td align="center">0.2</td><td align="center" style="author-callout-style-b1">&#8722;0.8</td><td align="center">&#8722;0.2</td><td align="center" style="author-callout-style-b6">0.9</td><td align="center">&#8194;0.3</td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th align="center" colspan="6" valign="bottom"><bold>Movement metrics</bold></th></tr><tr><th valign="bottom"/><th valign="bottom"/><th align="center"><bold>Gait</bold></th><th align="center"><bold>Y-maze&#160;reversal</bold></th><th align="center"><bold>Three-chamber</bold></th><th align="center"><bold>EPM</bold></th></tr><tr><th valign="bottom"/><th valign="bottom"/><th align="center">Stance</th><th align="center">Distance</th><th align="center">Distance Baseline</th><th align="center">Distance</th></tr></thead><tbody><tr><td rowspan="2" valign="bottom"><bold>Lobule VI</bold></td><td>Developmental</td><td align="center">0.1</td><td align="center">&#8722;0.1</td><td align="center">0.1</td><td align="center">0.7</td></tr><tr><td>Acute</td><td align="center">&#8722;0.3</td><td align="center"><italic><bold>&#8722;0.7</bold></italic></td><td align="center">&#8722;0.4</td><td align="center">0.7</td></tr><tr><td colspan="6" valign="bottom"/></tr><tr><td rowspan="2" valign="bottom"><bold>Lobule VII</bold></td><td>Developmental</td><td align="center">0.2</td><td align="center">&#8722;0.2</td><td align="center">&#8722;0.3</td><td align="center" style="author-callout-style-b6"><italic><bold>1.2</bold></italic></td></tr><tr><td>Acute</td><td align="center">0.5</td><td align="center">0.1</td><td align="center">&#8722;0.1</td><td align="center">0.3</td></tr><tr><td colspan="6" valign="bottom"/></tr><tr><td rowspan="2" valign="bottom"><bold>Crus I</bold></td><td>Developmental</td><td align="center">&#8722;0.3</td><td align="center">&#8722;0.4</td><td align="center" style="author-callout-style-b1"><italic><bold>&#8722;1.1</bold></italic></td><td align="center">0.7</td></tr><tr><td>Acute</td><td align="center"><bold>0.7</bold></td><td align="center">0.2</td><td align="center">0.2</td><td align="center">0.0</td></tr><tr><td colspan="6" valign="bottom"/></tr><tr><td rowspan="2" valign="bottom"><bold>Crus II</bold></td><td>Developmental</td><td align="center">0.1</td><td align="center">&#8722;0.3</td><td align="center">&#8722;0.2</td><td align="center" style="author-callout-style-b6"><bold><italic>1.2</italic></bold></td></tr><tr><td>Acute</td><td align="center">0.3</td><td align="center">0.2</td><td align="center">&#8722;0.2</td><td align="center">&#8722;0.3</td></tr></tbody></table><table-wrap-foot><fn><p>Effect size, calculated in units (Cohen, 1988) of the two-sample pooled standard deviation, of perturbations on key behavioral parameters. Values in <italic><bold>Bold</bold></italic> indicate statistical significance. Colored fields indicate&#160;large effect sizes (<italic>d</italic>&#8805;0.8); Blue for improvement of function and red for&#160;impairment as defined in Table 2.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36401-v3.xml">elife-36401-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.36401.032</object-id><label>Table 4.</label><caption><title>Relative neocortical expression resulting from long-distance tracing.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom"/><th align="center" colspan="2" valign="bottom">Injection site</th></tr><tr><th align="left" valign="bottom">Lobule VI</th><th valign="bottom">Crus I</th></tr></thead><tbody><tr><td><bold>Target Region</bold></td><td align="left" valign="bottom"><bold>Relative expression</bold></td><td valign="bottom"><bold>Relative expression</bold></td></tr><tr><td valign="bottom">Motor cortex</td><td align="left" valign="bottom">1.00</td><td valign="bottom">1.00</td></tr><tr><td valign="bottom">Somatosensory</td><td align="left" valign="bottom">0.66</td><td valign="bottom">1.23</td></tr><tr><td valign="bottom">Prelimbic</td><td align="left" valign="bottom">0.71</td><td valign="bottom">0.32</td></tr><tr><td valign="bottom">Orbitofrontal</td><td align="left" valign="bottom">0.72</td><td valign="bottom">0.22</td></tr><tr><td valign="bottom">Anterior cingulate</td><td align="left" valign="bottom">0.42</td><td valign="bottom">0.59</td></tr><tr><td valign="bottom">Infralimbic</td><td align="left" valign="bottom">0.28</td><td valign="bottom">0.28</td></tr><tr><td valign="bottom">Visual cortex</td><td align="left" valign="bottom">0.25</td><td valign="bottom">0.27</td></tr><tr><td valign="bottom">Parietal association</td><td align="left" valign="bottom">0.21</td><td valign="bottom">0.02</td></tr><tr><td valign="bottom">Retrosplenial</td><td align="left" valign="bottom">0.11</td><td valign="bottom">0.17</td></tr><tr><td valign="bottom">Agranular insular</td><td align="left" valign="bottom">0.03</td><td valign="bottom">0.00</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36405-v2.xml">elife-36405-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.36405.013</object-id><label>Table 2.</label><caption><title>Sample composition by genotype and period.</title><p><italic>Fgfr2<sup>+/+</sup></italic>: unaffected littermates; <italic>Fgfr2<sup>+/P253R</sup></italic>: Apert syndrome mutant littermates. Developmental periods were defined according to limb staging, as shown in <xref ref-type="fig" rid="fig4">Figure 4</xref>. Hindlimbs and forelimbs were analyzed separately because, in the same embryo, hindlimbs are delayed in development in comparison to forelimbs by about 6&#8211;8 hr. We thus presented in the main text the results from groupings highlighted in color, including those for the hindlimbs from the early and mid groups, as well as those for the forelimbs from the mid and the late groups. This represents a continuous time span of limb development, from E10 to E11.5, divided into four periods of 12 hr each. Results from the complete dataset that considers hindlimbs and forelimbs from each litter separately are available in <xref ref-type="fig" rid="fig5">Figures 5</xref> and <xref ref-type="fig" rid="fig6">6</xref> (<xref ref-type="fig" rid="fig5s2">Figure 5&#8212;figure supplement 2</xref>, <xref ref-type="fig" rid="fig5s3">Figure 5&#8212;figure supplement 3</xref>, <xref ref-type="fig" rid="fig6s1">Figure 6&#8212;figure supplement 1</xref> and <xref ref-type="fig" rid="fig6s2">Figure 6&#8212;figure supplement 2</xref>).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><bold>Genotype</bold></th><th align="center" rowspan="2"><bold>Period</bold></th><th rowspan="2"><bold>N limb</bold></th><th rowspan="2"><bold>N gene</bold></th><th colspan="4" valign="bottom"><bold>LIMB</bold></th></tr><tr><th colspan="2" valign="bottom"><bold>Hindlimb (N&#160;=&#160;34)</bold></th><th colspan="2" valign="bottom"><bold>Forelimb (N&#160;=&#160;46)</bold></th></tr></thead><tbody><tr><td/><td align="center"/><td/><td/><td colspan="2" style="author-callout-style-b8" valign="bottom"><bold>Early (~E10)</bold></td><td valign="bottom"/><td valign="bottom"/></tr><tr><td/><td align="center"/><td/><td/><td style="author-callout-style-b8" valign="bottom"><bold>Limb</bold></td><td style="author-callout-style-b8" valign="bottom"><bold>Gene</bold></td><td valign="bottom"><bold>Limb</bold></td><td valign="bottom"><bold>Gene</bold></td></tr><tr><td valign="bottom"><bold><italic>Fgfr2<sup>+/+</sup></italic></bold></td><td align="center" rowspan="3"><bold>EARLY: E10-E10.5</bold></td><td valign="bottom">30</td><td valign="bottom">24</td><td style="author-callout-style-b8" valign="bottom">13</td><td style="author-callout-style-b8" valign="bottom">9</td><td valign="bottom">17</td><td valign="bottom">15</td></tr><tr><td valign="bottom"><bold><italic>Fgfr2<sup>+/P253R</sup></italic></bold></td><td valign="bottom">16</td><td valign="bottom">15</td><td style="author-callout-style-b8" valign="bottom">11</td><td style="author-callout-style-b8" valign="bottom">9</td><td valign="bottom">5</td><td valign="bottom">6</td></tr><tr><td valign="bottom"><bold>Subtotal</bold></td><td valign="bottom">46</td><td valign="bottom">39</td><td style="author-callout-style-b8" valign="bottom">24</td><td style="author-callout-style-b8" valign="bottom">18</td><td valign="bottom">22</td><td valign="bottom">21</td></tr><tr><td valign="bottom"/><td align="center"/><td valign="bottom"/><td valign="bottom"/><td colspan="2" style="author-callout-style-b7" valign="bottom"><bold>Mid early (~E10.5)</bold></td><td colspan="2" style="author-callout-style-b6" valign="bottom"><bold>Mid late (~E11)</bold></td></tr><tr><td valign="bottom"/><td align="center"/><td valign="bottom"/><td valign="bottom"/><td style="author-callout-style-b7" valign="bottom"><bold>Limb</bold></td><td style="author-callout-style-b7" valign="bottom"><bold>Gene</bold></td><td style="author-callout-style-b6" valign="bottom"><bold>Limb</bold></td><td style="author-callout-style-b6" valign="bottom"><bold>Gene</bold></td></tr><tr><td valign="bottom"><bold><italic>Fgfr2<sup>+/+</sup></italic></bold></td><td align="center" rowspan="3"><bold>MID: E10.5-E11</bold></td><td valign="bottom">15</td><td valign="bottom">10</td><td style="author-callout-style-b7" valign="bottom">7</td><td style="author-callout-style-b7" valign="bottom">3</td><td style="author-callout-style-b6" valign="bottom">8</td><td style="author-callout-style-b6" valign="bottom">7</td></tr><tr><td valign="bottom"><bold><italic>Fgfr2<sup>+/P253R</sup></italic></bold></td><td valign="bottom">16</td><td valign="bottom">9</td><td style="author-callout-style-b7" valign="bottom">8</td><td style="author-callout-style-b7" valign="bottom">3</td><td style="author-callout-style-b6" valign="bottom">8</td><td style="author-callout-style-b6" valign="bottom">6</td></tr><tr><td valign="bottom"><bold>Subtotal</bold></td><td valign="bottom">31</td><td valign="bottom">19</td><td style="author-callout-style-b7" valign="bottom">15</td><td style="author-callout-style-b7" valign="bottom">6</td><td style="author-callout-style-b6" valign="bottom">16</td><td style="author-callout-style-b6" valign="bottom">13</td></tr><tr><td valign="bottom"/><td align="center"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td colspan="2" style="author-callout-style-b5" valign="bottom"><bold>Late (~E11.5)</bold></td></tr><tr><td valign="bottom"/><td align="center"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"><bold>Limb</bold></td><td valign="bottom"><bold>Gene</bold></td><td style="author-callout-style-b5" valign="bottom"><bold>Limb</bold></td><td style="author-callout-style-b5" valign="bottom"><bold>Gene</bold></td></tr><tr><td valign="bottom"><bold><italic>Fgfr2<sup>+/+</sup></italic></bold></td><td align="center" rowspan="3"><bold>LATE: E11-E11.5</bold></td><td valign="bottom">8</td><td valign="bottom">8</td><td valign="bottom">4</td><td valign="bottom">4</td><td style="author-callout-style-b5" valign="bottom">4</td><td style="author-callout-style-b5" valign="bottom">4</td></tr><tr><td valign="bottom"><bold><italic>Fgfr2<sup>+/P253R</sup></italic></bold></td><td valign="bottom">15</td><td valign="bottom">14</td><td valign="bottom">7</td><td valign="bottom">6</td><td style="author-callout-style-b5" valign="bottom">8</td><td style="author-callout-style-b5" valign="bottom">8</td></tr><tr><td valign="bottom"><bold><italic>Subtotal</italic></bold></td><td valign="bottom">23</td><td valign="bottom">22</td><td valign="bottom">11</td><td valign="bottom">10</td><td style="author-callout-style-b5" valign="bottom">12</td><td style="author-callout-style-b5" valign="bottom">12</td></tr><tr><td valign="bottom"><bold>Total</bold></td><td align="center" valign="bottom"/><td valign="bottom">100</td><td valign="bottom">80</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36422-v1.xml">elife-36422-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.36422.008</object-id><label>Table 1.</label><caption><title>Crystallographic and structural features of RNA-ligand complexes.</title></caption><table frame="hsides" rules="groups"><thead><tr><th/><th rowspan="2">RNA- dGMP</th><th rowspan="2">RNA- Gp-AI-pG</th><th colspan="7">Time-resolved structures (RNA-2-AIpG)</th></tr><tr><th/><th>Stage 1 (5 min)</th><th>Stage 2 (15 min)</th><th>Stage 3 (30 min)</th><th>Stage 4 (1 h)</th><th>Stage 5 (1.5 h)</th><th>Stage 6 (2 h)</th><th>Stage 7 (3 h)</th></tr></thead><tbody><tr><td>PDB code</td><td>6C8D</td><td>6C8E</td><td>6C8I</td><td>6C8J</td><td>6C8K</td><td>6C8L</td><td>6C8M</td><td>6C8N</td><td>6C8O</td></tr><tr><td>Ligand binding modes</td><td>Watson-Crick</td><td>Watson-Crick</td><td>mixed</td><td>Watson-Crick</td><td>Watson-Crick</td><td>Watson-Crick</td><td>Watson-Crick</td><td>Watson-Crick</td><td>Watson-Crick</td></tr><tr><td>N7 interaction</td><td>Mg<sup>2+</sup></td><td>N.D.</td><td>H<sub>2</sub>O</td><td>H<sub>2</sub>O</td><td>N.D.</td><td>N.D.</td><td>N.D.</td><td>N.D.</td><td>N.D.</td></tr><tr><td>3&#697;-O-P distance, &#197;</td><td>~6.1</td><td>4.6</td><td>3.7&#8211;4.1</td><td>~5.0</td><td>~4.3</td><td>4.1&#8211;5.0</td><td>1.6&#8211;4.2</td><td>1.6&#8211;4.1</td><td>1.6</td></tr><tr><td>P-P distance between monomers, &#197;</td><td>N.D.</td><td>~5.1</td><td>N.D.</td><td>6.2</td><td>5.2</td><td>5.2&#8211;5.4</td><td>5.5&#8211;7.1</td><td>5.0&#8211;7.3</td><td>~6.2</td></tr><tr><td>N7-N7 distance between primer and&#160;+&#160;1 monomer, &#197;</td><td>4.2</td><td>3.7&#8211;4.0</td><td>3.9&#8211;4.1</td><td>4.0</td><td>3.7&#8211;4.0</td><td>3.7&#8211;4.2</td><td>3.7&#8211;4.3</td><td>3.7&#8211;4.1</td><td>4.1</td></tr></tbody></table><table-wrap-foot><fn><p>N.D.: not detectable</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36426-v2.xml">elife-36426-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.36426.022</object-id><label>Table 1.</label><caption><title>GO term enrichment on chromosome four</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">GO</th><th align="center" rowspan="2" valign="top">Corrected <break/><italic>P</italic>-value</th><th align="center" colspan="2" valign="top">Number on Chr_04</th><th align="center" rowspan="2" valign="top">Percent of <break/>Chr_04 genes</th><th align="left" rowspan="2" valign="top">GO term</th></tr><tr><th align="center" valign="top">Observed</th><th align="center" valign="top">Expected</th></tr></thead><tbody><tr><td valign="top">0043531</td><td valign="top"><p><inline-formula><mml:math id="inf3"><mml:mrow><mml:mn>5.61</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>79</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">140</td><td valign="top">9</td><td valign="top">7.57</td><td valign="top">ADP binding</td></tr><tr><td valign="top">0016705</td><td valign="top"><p><inline-formula><mml:math id="inf4"><mml:mrow><mml:mn>4.40</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>48</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">179</td><td valign="top">39</td><td valign="top">9.68</td><td valign="top">Oxidoreductase activity, actingon paired donors, withincorporation or reduction of <break/>molecular oxygen</td></tr><tr><td valign="top">0004497</td><td valign="top"><p><inline-formula><mml:math id="inf5"><mml:mrow><mml:mn>7.19</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>46</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">158</td><td valign="top">32</td><td valign="top">8.55</td><td valign="top">Monooxygenase activity</td></tr><tr><td valign="top">0005506</td><td valign="top"><p><inline-formula><mml:math id="inf6"><mml:mrow><mml:mn>2.73</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>41</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">181</td><td valign="top">46</td><td valign="top">9.79</td><td valign="top">Iron ion binding</td></tr><tr><td valign="top">0020037</td><td valign="top"><p><inline-formula><mml:math id="inf7"><mml:mrow><mml:mn>2.57</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>37</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">186</td><td valign="top">53</td><td valign="top">10.06</td><td valign="top">Heme binding</td></tr><tr><td valign="top">0010333</td><td valign="top"><p><inline-formula><mml:math id="inf8"><mml:mrow><mml:mn>1.72</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>15</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">39</td><td valign="top">4</td><td valign="top">2.11</td><td valign="top">Terpene synthase activity</td></tr><tr><td valign="top">0016829</td><td valign="top"><p><inline-formula><mml:math id="inf9"><mml:mrow><mml:mn>2.08</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>13</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">39</td><td valign="top">5</td><td valign="top">2.11</td><td valign="top">Lyase activity</td></tr><tr><td valign="top">0055114</td><td valign="top"><p><inline-formula><mml:math id="inf10"><mml:mrow><mml:mn>9.53</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>10</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">247</td><td valign="top">149</td><td valign="top">13.36</td><td valign="top">Oxidation-reduction process</td></tr><tr><td valign="top">0016747</td><td valign="top"><p><inline-formula><mml:math id="inf11"><mml:mrow><mml:mn>6.66</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>5</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">44</td><td valign="top">16</td><td valign="top">2.38</td><td valign="top">Transferase activity,transferring acyl groups other <break/>than amino-acyl groups</td></tr><tr><td valign="top">0000287</td><td valign="top"><p><inline-formula><mml:math id="inf12"><mml:mrow><mml:mn>1.23</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>4</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">42</td><td valign="top">15</td><td valign="top">2.27</td><td valign="top">Magnesium ion binding</td></tr><tr><td valign="top">0008152</td><td valign="top"><p><inline-formula><mml:math id="inf13"><mml:mrow><mml:mn>2.56</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>4</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">137</td><td valign="top">83</td><td valign="top">7.41</td><td valign="top">Metabolic process</td></tr><tr><td valign="top">0006952</td><td valign="top"><p><inline-formula><mml:math id="inf14"><mml:mrow><mml:mn>3.60</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>4</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">32</td><td valign="top">10</td><td valign="top">1.73</td><td valign="top">Defense response</td></tr><tr><td valign="top">0004674</td><td valign="top"><p><inline-formula><mml:math id="inf15"><mml:mrow><mml:mn>4.52</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>4</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">23</td><td valign="top">5</td><td valign="top">1.24</td><td valign="top">Protein serine/threoninekinase activity</td></tr><tr><td valign="top">0016758</td><td valign="top"><p><inline-formula><mml:math id="inf16"><mml:mrow><mml:mn>1.35</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>3</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">44</td><td valign="top">18</td><td valign="top">2.38</td><td valign="top">Transferase activity, transferringhexosyl groups</td></tr><tr><td valign="top">0005622</td><td valign="top"><p><inline-formula><mml:math id="inf17"><mml:mrow><mml:mn>4.14</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>3</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">14</td><td valign="top">42</td><td valign="top">0.76</td><td valign="top">Intracellular</td></tr><tr><td valign="top">0008146</td><td valign="top"><p><inline-formula><mml:math id="inf18"><mml:mrow><mml:mn>2.68</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>2</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">9</td><td valign="top">1</td><td valign="top">0.49</td><td valign="top">Sulfotransferase activity</td></tr><tr><td valign="top">0016760</td><td valign="top"><p><inline-formula><mml:math id="inf19"><mml:mrow><mml:mn>3.72</mml:mn><mml:mo>&#215;</mml:mo><mml:msup><mml:mrow><mml:mn>10</mml:mn></mml:mrow><mml:mrow><mml:mo>&#8722;</mml:mo><mml:mn>2</mml:mn></mml:mrow></mml:msup></mml:mrow></mml:math></inline-formula></p></td><td valign="top">12</td><td valign="top">2</td><td valign="top">0.65</td><td valign="top">Cellulose synthase(UDP-forming) activity</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36426-v2.xml">elife-36426-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.36426.023</object-id><label>Table 2.</label><caption><title>Content of the <italic>A. coerulea</italic> v3.1 reference by chromosome</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><break/></th><th align="center" colspan="7" valign="top">Chromosome</th><th align="center" rowspan="2" valign="top"><break/>Genome</th></tr><tr><th align="center" valign="top">1</th><th align="center" valign="top">2</th><th align="center" valign="top">3</th><th align="center" valign="top">4</th><th align="center" valign="top">5</th><th align="center" valign="top">6</th><th align="center" valign="top">7</th></tr></thead><tbody><tr><td valign="top">Number of genes</td><td align="left" valign="top">5041</td><td valign="top">4390</td><td valign="top">4449</td><td valign="top">3149</td><td valign="top">4786</td><td valign="top">3292</td><td valign="top">4443</td><td valign="top">29550</td></tr><tr><td valign="top">Genes per Mb</td><td align="left" valign="top">112</td><td valign="top">102</td><td valign="top">104</td><td valign="top">69</td><td valign="top">107</td><td valign="top">108</td><td valign="top">102</td><td valign="top">100</td></tr><tr><td valign="top">Mean gene length (bp)</td><td align="left" valign="top">3629</td><td valign="top">3641</td><td valign="top">3689</td><td valign="top">3020</td><td valign="top">3712</td><td valign="top">3620</td><td valign="top">3708</td><td valign="top">3580</td></tr><tr><td valign="top">Percent repetitive</td><td align="left" valign="top">38.9</td><td valign="top">41.1</td><td valign="top">39.1</td><td valign="top">54.2</td><td valign="top">39.4</td><td valign="top">39.3</td><td valign="top">40.6</td><td valign="top">42.0</td></tr><tr><td valign="top">Percent genes withHIGH effect variant</td><td align="left" valign="top">25.3</td><td valign="top">23.8</td><td valign="top">23.6</td><td valign="top">32.3</td><td valign="top">24.1</td><td valign="top">22.1</td><td valign="top">23.6</td><td valign="top">24.7</td></tr><tr><td valign="top">Percent GC</td><td align="left" valign="top">36.8</td><td valign="top">37.0</td><td valign="top">36.9</td><td valign="top">37.0</td><td valign="top">37.1</td><td valign="top">36.8</td><td valign="top">36.8</td><td valign="top">37.0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36426-v2.xml">elife-36426-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.36426.024</object-id><label>Table 3.</label><caption><title>Population genetics parameters for <italic>Semiaquilegia</italic> by chromosome</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"> <break/><break/></th><th align="center" colspan="8" valign="top">Percent pairwise differences</th></tr><tr><th align="center" colspan="7" valign="top">Chromosome</th><th rowspan="2" valign="top">Genome</th></tr><tr><th align="center" valign="top">1</th><th align="center" valign="top">2</th><th align="center" valign="top">3</th><th align="center" valign="top">4</th><th align="center" valign="top">5</th><th align="center" valign="top">6</th><th align="center" valign="top">7</th></tr></thead><tbody><tr><td valign="top">Polymorphism&#160;within&#160;<italic>Semiaquilegia</italic></td><td valign="top">0.079</td><td valign="top">0.085</td><td valign="top">0.081</td><td valign="top">0.162</td><td valign="top">0.076</td><td valign="top">0.078</td><td valign="top">0.071</td><td valign="top">0.082</td></tr><tr><td valign="top">Divergence between <italic>Aquilegia</italic> and <italic>Semiaquilegia</italic></td><td valign="top">2.46</td><td valign="top">2.47</td><td valign="top">2.47</td><td valign="top">2.77</td><td valign="top">2.48</td><td valign="top">2.47</td><td valign="top">2.47</td><td valign="top">2.48</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36435-v1.xml">elife-36435-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.36435.042</object-id><label>Table 1.</label><caption><title>The number of hyper- and hypomethylated DMRs overlapping with ChIP-seq peaks.</title><p>Relation between DMRs and Fam60a binding site for E9.5 embryos. Methyl-seq data were obtained for three <italic>Fam60a</italic><sup>&#8211;/&#8211;</sup> and three WT embryos, and ChIP-seq data were obtained for ChIP-seq1 and ChIP-seq2 experiments. The number of ChIP-seq peaks that overlap with all or the top 500 hyper- and hypomethylated DMRs are shown.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Data set</th><th rowspan="2">Total DMRs</th><th rowspan="2">Direction</th><th rowspan="2">DMRs</th><th colspan="2">Overlap with ChIP-seq peaks</th></tr><tr><th>vs. all DMRs (%)</th><th>vs. top 500 (%)</th></tr></thead><tbody><tr><td rowspan="2">3 embryos (triplicates) (mean Diff&#160;&gt;=&#160;0.05)</td><td rowspan="2">7245</td><td>Hyper</td><td>3049</td><td>388 (12.7)</td><td>83 (16.6)</td></tr><tr><td>Hypo</td><td>4196</td><td>1257 (30.0)</td><td>254 (50.8)</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36607-v2.xml">elife-36607-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.36607.006</object-id><label>Table 1.</label><caption><title>Response types latencies and durations</title></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2"><sc>R</sc><sc>esponse</sc> <sc>type</sc></th><th colspan="2"><sc>P</sc><sc>hase</sc> <sc>1</sc></th><th colspan="2"><sc>P</sc><sc>hase</sc> <sc>2</sc></th><th colspan="2"><sc>P</sc><sc>hase</sc> <sc>3</sc></th></tr><tr><th><sc>Latency</sc></th><th><sc>Duration</sc></th><th><sc>Latency</sc></th><th><sc>Duration</sc></th><th><sc>Latency</sc></th><th><sc>Duration</sc></th></tr></thead><tbody><tr><td rowspan="2">I.</td><td>1.</td><td>37,80&#160;&#177;&#160;3,15 ms</td><td>338.40&#160;&#177;&#160;49.79 ms</td><td>&#8212;</td><td>&#8212;</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td>2.</td><td>20.18&#160;&#177;&#160;3.15 ms</td><td>34.47&#160;&#177;&#160;3.92 ms</td><td>&#8212;</td><td>&#8212;</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td colspan="2">II.</td><td>20.00&#160;&#177;&#160;3.02 ms</td><td>35.00&#160;&#177;&#160;3.86 ms</td><td>97.00&#160;&#177;&#160;6.97 ms</td><td>269.00&#160;&#177;&#160;40.05 ms</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td colspan="2">III.</td><td>23.00&#160;&#177;&#160;4.61 ms</td><td>69.00&#160;&#177;&#160;19.59 ms</td><td>133.70&#160;&#177;&#160;32.76 ms</td><td>176.70&#160;&#177;&#160;3.83 ms</td><td>&#8212;</td><td>&#8212;</td></tr><tr><td colspan="2">IV.</td><td>16.4&#160;&#177;&#160;4.57 ms</td><td>46.4&#160;&#177;&#160;7.90 ms</td><td>101.2&#160;&#177;&#160;18.56 ms</td><td>28.2&#160;&#177;&#160;5.27 ms</td><td>227.6&#160;&#177;&#160;43.53 ms</td><td>216.8&#160;&#177;&#160;69.3 ms</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36607-v2.xml">elife-36607-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.36607.008</object-id><label>Table 2.</label><caption><title>Nociceptive responses in the subthalmic nucleus, superior colliculus and parabrachial nucleus. </title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"/><th colspan="4">Microinjection of muscimol in the superior colliculus</th></tr><tr><th colspan="2">Superior colliculus</th><th colspan="2">Subthalamic nucleus</th></tr><tr><th><italic>Pre muscimol</italic></th><th><italic>Post muscimol</italic></th><th><italic>Pre muscimol</italic></th><th><italic>Post muscimol</italic></th></tr></thead><tbody><tr><td>Latency</td><td>9&#160;&#177;&#160;0.8 ms</td><td>7.60&#160;&#177;&#160;0.68 ms</td><td>25.33&#160;&#177;&#160;5.27 ms</td><td>28.38&#160;&#177;&#160;7.75 ms</td></tr><tr><td>Duration</td><td>19.62&#160;&#177;&#160;5.00 ms</td><td>14&#160;&#177;&#160;3.03 ms</td><td>176.13&#160;&#177;&#160;60.37 ms</td><td>135.74&#160;&#177;&#160;50.19 ms *</td></tr><tr><td>Amplitude</td><td>322.29&#160;&#177;&#160;58.16 Hz</td><td>125.00&#160;&#177;&#160;26.22 Hz *</td><td>15.61&#160;&#177;&#160;3.29 Hz</td><td>14.4&#160;&#177;&#160;2.44 Hz</td></tr><tr><td>Magnitude</td><td>140.20&#160;&#177;&#160;28.01 Hz</td><td>45.01&#160;&#177;&#160;9.16 Hz *</td><td>10.21&#160;&#177;&#160;1.99 Hz</td><td>7.24&#160;&#177;&#160;1.41 Hz</td></tr><tr><td>Baseline FR no footshock</td><td>18.27&#160;&#177;&#160;2.19 Hz</td><td>&#8212;</td><td>4.37&#160;&#177;&#160;0.8 Hz</td><td>-</td></tr><tr><td>Baseline FR footshock</td><td>20.79&#160;&#177;&#160;2.66 Hz</td><td>7.69&#160;&#177;&#160;3.00 Hz *</td><td>6.06&#160;&#177;&#160;1.48 Hz</td><td>6.45&#160;&#177;&#160;1.38 Hz</td></tr></tbody></table><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"/><th colspan="4">Microinjection of muscimol in the parabrachial nucleus</th></tr><tr><th colspan="2">Parabrachial nucleus</th><th colspan="2">Subthalamic nucleus</th></tr><tr><th><italic>Pre muscimol</italic></th><th><italic>Post muscimol</italic></th><th><italic>Pre muscimol</italic></th><th><italic>Post muscimol</italic></th></tr></thead><tbody><tr><td>Latency</td><td>11.55&#160;&#177;&#160;1.35 ms</td><td>19.00&#160;&#177;&#160;2.48 ms *</td><td>24.54&#160;&#177;&#160;3.13 ms</td><td>18.43&#160;&#177;&#160;1.73 ms</td></tr><tr><td>Duration</td><td>26.45.12&#160;&#177;&#160;3.85 ms</td><td>14&#160;&#177;&#160;3.49 ms *</td><td>98.00&#160;&#177;&#160;27.35 ms</td><td>45.75&#160;&#177;&#160;12.04 ms *</td></tr><tr><td>Amplitude</td><td>146.97.42&#160;&#177;&#160;23.66 Hz</td><td>83.26&#160;&#177;&#160;17.02 Hz *</td><td>26.98&#160;&#177;&#160;5.72 Hz</td><td>17.18&#160;&#177;&#160;3.26 Hz *</td></tr><tr><td>Magnitude</td><td>45.16&#160;&#177;&#160;7.45 Hz</td><td>23.70&#160;&#177;&#160;4.57 Hz *</td><td>10.31&#160;&#177;&#160;2 Hz</td><td>6.65&#160;&#177;&#160;1.44 Hz *</td></tr><tr><td>Baseline FR no footshock</td><td>24.25&#160;&#177;&#160;1.97 Hz</td><td>&#8212;</td><td>5.89&#160;&#177;&#160;0.49 Hz</td><td>&#8212;</td></tr><tr><td>Baseline FR footshock</td><td>20.55&#160;&#177;&#160;2.42 Hz</td><td>14.15&#160;&#177;&#160;2.76 Hz</td><td>5.50&#160;&#177;&#160;0.71 Hz</td><td>5.39&#160;&#177;&#160;0.57 Hz</td></tr></tbody></table><table-wrap-foot><fn><p>Mean&#160;<sc>&#177;&#160;</sc><sc>SEM -</sc> * statistically different from pre-muscimol measure.</p><p>Abbreviations: cp, cerebral peduncle; IS, injection site; lPBN, lateral parabrachial nucleus; mPBN, medial parabrachial nucleus; scp, superior cerebral peduncle; STN, subthalamic nucleus.</p></fn><fn><p>Note: the amplitude of the response is the maximum amplitude during the response and the magnitude of the response is the mean number of single multi-unit events between response onset and offset minus the baseline mean.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36745-v1.xml">elife-36745-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.36745.012</object-id><label>Table 1.</label><caption><title>Viability of <italic>scs2/22&#8710;</italic> mutant in combination with <italic>kar9&#8710;</italic>, <italic>dyn1</italic>&#8710;, or <italic>cin8&#8710;</italic> mutant.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Mutant combination</th><th rowspan="2" valign="top">Number of tetrads <break/>analyzed</th><th rowspan="2" valign="top">Number of predicted double <break/>or triple mutants</th><th colspan="2" valign="top">Viability of mutants</th></tr><tr><th valign="top">Viable</th><th valign="top">Microcolony</th></tr></thead><tbody><tr><td valign="top"><italic>scs2/22&#8710;</italic></td><td valign="top">22</td><td valign="top">16</td><td valign="top">16</td><td valign="top">0</td></tr><tr><td valign="top"><italic>scs2/22&#8710; kar9&#8710;</italic></td><td valign="top">13</td><td valign="top">5</td><td valign="top">5</td><td valign="top">0</td></tr><tr><td valign="top"><italic>scs2/22&#8710; dyn1&#8710;</italic></td><td valign="top">22</td><td valign="top">9</td><td valign="top">9</td><td valign="top">0</td></tr><tr><td valign="top"><italic>scs2/22&#8710; cin8&#8710;</italic></td><td valign="top">29</td><td valign="top">17</td><td valign="top">17</td><td valign="top">0</td></tr><tr><td valign="top"><italic>dyn1&#8710; cin8&#8710;</italic></td><td valign="top">10</td><td valign="top">7</td><td valign="top">0</td><td valign="top">7</td></tr></tbody></table><table-wrap-foot><fn><p>For <italic>scs2/22&#8710;</italic> and <italic>scs2/22&#8710; dyn1&#8710;</italic> combinations, YWL4865 was crossed with YWL521. For <italic>scs2/22&#8710; kar9&#8710;</italic> combination, YWL4865 was crossed with YWL4949. For <italic>scs2/22&#8710; cin8&#8710;</italic> and <italic>dyn1&#8710; cin8&#8710;</italic> combinations, YWL3955 was crossed with YWL4866 and YWL504, respectively. The resulting diploid strains were sporulated and tetrads were dissected.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-36979-v2.xml">elife-36979-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.36979.028</object-id><label>Table 1.</label><caption><title>Correspondence of DNAH genes/proteins among zebrafish, human, and <italic>Chlamydomonas</italic>.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="3">Gene/protein names</th></tr><tr><th>Zebrafish</th><th>Human</th><th><italic>Chlamydomonas</italic></th></tr></thead><tbody><tr><td rowspan="8">OAD HCs</td><td/><td/><td>OAD &#945; (DHC13)</td></tr><tr><td>Dnah9</td><td>DNAH9</td><td rowspan="4">OAD &#946; (DHC14)</td></tr><tr><td>Dnah11</td><td>DNAH11</td></tr><tr><td>Dnah9l</td><td/></tr><tr><td/><td>DNAH17</td></tr><tr><td>Dnah5</td><td>DNAH5</td><td rowspan="3">OAD &#947; (DHC15)</td></tr><tr><td>Dnah8 (Dnah5l)</td><td/></tr><tr><td/><td>DNAH8</td></tr><tr><td rowspan="2">IAD f HCs</td><td>Dnah10</td><td>DNAH10</td><td>IAD f&#945; (DHC1)</td></tr><tr><td>Dnah2</td><td>DNAH2</td><td>IAD f&#946; (DHC10)</td></tr><tr><td rowspan="6">IAD HCs</td><td>Dnah3</td><td>DNAH3</td><td rowspan="3">IAD a (DHC6) <break/>IAD b (DHC5) <break/>IAD c (DHC9) <break/>IAD e (DHC8) <break/>minor var. DHC11 <break/>minor var. DHC4</td></tr><tr><td>Dnah7</td><td>DNAH7</td></tr><tr><td>Dnah12</td><td>DNAH12</td></tr><tr><td>Dnah1</td><td>DNAH1</td><td>IAD d (DHC2)</td></tr><tr><td rowspan="2">Dnah7l (Dnah6)</td><td>DNAH6</td><td rowspan="2">IAD g (DHC7) <break/>minor var. DHC3</td></tr><tr><td>DNAH14</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37090-v2.xml">elife-37090-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.37090.005</object-id><label>Table 1.</label><caption><title>Overview of fermentation experiments and model fitting results (RMSE: root mean square error).</title><p>Means and standard deviations are reported across biological replicates.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Strains</th><th rowspan="2" valign="top">Number of&#160;biological replicates</th><th rowspan="2" valign="top">Added energy source and/or co-substrate</th><th colspan="7" valign="top">Consumption&#160;(-)/production&#160;(+) of metabolites in mM</th><th rowspan="2" valign="top">Carbon recovery (%)</th><th rowspan="2" valign="top">O/R balance</th><th rowspan="2" valign="top">Selected for parame-terization 1</th><th rowspan="2" valign="top">RMSE parame-terization 1</th><th rowspan="2" valign="top">Selected for parame-terization 2</th><th rowspan="2" valign="top">RMSE parame-terization 2</th></tr><tr><th valign="top">Fructose</th><th valign="top">Acetate</th><th valign="top">Butyrate</th><th valign="top">Formate</th><th valign="top">Lactate</th><th valign="top">H<sub>2</sub></th><th valign="top">CO<sub>2</sub></th></tr></thead><tbody><tr><td valign="top">Monoculture fermentations</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><italic>R. intestinalis</italic> <break/>L1-82</td><td valign="top">4</td><td valign="top">Fructose/acetate</td><td valign="top">&#8722;48.8&#160;&#177;&#160;0.9</td><td valign="top">&#8722;20.6&#160;&#177;&#160;2.4</td><td valign="top">58.5&#160;&#177;&#160;3.9</td><td valign="top">6.8&#160;&#177;&#160;1.2</td><td valign="top">3.9&#160;&#177;&#160;1.1</td><td valign="top">72.2&#160;&#177;&#160;4.9</td><td valign="top">84.0&#160;&#177;&#160;0.8</td><td valign="top">105&#160;&#177;&#160;14.3</td><td valign="top">0.86&#160;&#177;&#160;0.2</td><td valign="top">Yes</td><td valign="top">0.17&#160;&#177;&#160;0.04</td><td valign="top">No</td><td valign="top">0.83&#160;&#177;&#160;0.13</td></tr><tr><td valign="top"><italic>F. prausnitzii</italic> A2-165</td><td valign="top">3</td><td valign="top">Fructose/acetate</td><td valign="top">&#8722;23.1&#160;&#177;&#160;1.1</td><td valign="top">&#8722;13.5&#160;&#177;&#160;0.8</td><td valign="top">29.8&#160;&#177;&#160;0.2</td><td valign="top">22.7&#160;&#177;&#160;0.9</td><td valign="top">1.8&#160;&#177;&#160;0.2</td><td valign="top">0.2&#160;&#177;&#160;0.2</td><td valign="top">19.3&#160;&#177;&#160;1.6</td><td valign="top">100.9&#160;&#177;&#160;3.4</td><td valign="top">1.05&#160;&#177;&#160;0.09</td><td valign="top">Yes</td><td valign="top">0.15&#160;&#177;&#160;0.08</td><td valign="top">Yes</td><td valign="top">0.15&#160;&#177;&#160;0.08</td></tr><tr><td valign="top"><italic>B. hydrogenotrophica</italic> S5a33</td><td valign="top">3</td><td valign="top">Fructose/formate</td><td valign="top">&#8722;19.0&#160;&#177;&#160;5.6</td><td valign="top">23.0&#160;&#177;&#160;6.6</td><td valign="top">0.0&#160;&#177;&#160;0.0</td><td valign="top">&#8722;36.1&#160;&#177;&#160;1.8</td><td valign="top">6.0&#160;&#177;&#160;1.2</td><td valign="top">31.5&#160;&#177;&#160;7.7</td><td valign="top">26.0&#160;&#177;&#160;5.3</td><td valign="top">60.0&#160;&#177;&#160;2.1</td><td valign="top">0.76&#160;&#177;&#160;0.07</td><td valign="top">Yes</td><td valign="top">1.43&#160;&#177;&#160;0.27</td><td valign="top">No</td><td valign="top">3.53&#160;&#177;&#160;1.04</td></tr><tr><td valign="top">Bi-culture fermentations</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><italic>R. intestinalis</italic> <break/>L1&#8722;82/<italic>F. prausnitzii</italic> A2-165</td><td valign="top">2</td><td valign="top">Fructose/acetate</td><td valign="top">&#8722;47.7&#160;&#177;&#160;2.2</td><td valign="top">&#8722;18.1&#160;&#177;&#160;2.7</td><td valign="top">56.3&#160;&#177;&#160;4.5</td><td valign="top">7.6&#160;&#177;&#160;2.7</td><td valign="top">4.3&#160;&#177;&#160;3.3</td><td valign="top">114&#160;&#177;&#160;16</td><td valign="top">85.7&#160;&#177;&#160;4.1</td><td valign="top">102.8&#160;&#177;&#160;0.3</td><td valign="top">0.79&#160;&#177;&#160;0</td><td valign="top">No</td><td valign="top">0.77&#160;&#177;&#160;0.06</td><td valign="top">No</td><td valign="top">0.68&#160;&#177;&#160;0.03</td></tr><tr><td valign="top"><italic>R. intestinalis</italic> <break/>L1&#8722;82/<italic>B. hydrogenotrophica</italic> S5a33</td><td valign="top">2</td><td valign="top">Fructose/acetate</td><td valign="top">&#8722;46.5&#160;&#177;&#160;1.5</td><td valign="top">&#8722;7.5&#160;&#177;&#160;1.9</td><td valign="top">53.5&#160;&#177;&#160;4.7</td><td valign="top">&#8722;0.5&#160;&#177;&#160;0.8</td><td valign="top">2.2&#160;&#177;&#160;0.3</td><td valign="top">53.0&#160;&#177;&#160;3.0</td><td valign="top">74.9&#160;&#177;&#160;4.1</td><td valign="top">100.2&#160;&#177;&#160;0.6</td><td valign="top">0.94&#160;&#177;&#160;0.12</td><td valign="top">No</td><td valign="top">0.78&#160;&#177;&#160;0.36</td><td valign="top">Yes</td><td valign="top">0.3&#160;&#177;&#160;0.04</td></tr><tr><td valign="top"><italic>R. intestinalis</italic> <break/>L1&#8722;82/<italic>B. hydrogenotrophica</italic> S5a33</td><td valign="top">2</td><td valign="top">Fructose</td><td valign="top">&#8722;48.0&#160;&#177;&#160;0.2</td><td valign="top">44.9&#160;&#177;&#160;5.4</td><td valign="top">27.9&#160;&#177;&#160;3.0</td><td valign="top">&#8722;1.0&#160;&#177;&#160;0.0</td><td valign="top">5.4&#160;&#177;&#160;2.4</td><td valign="top">33.1&#160;&#177;&#160;3.2</td><td valign="top">47.4&#160;&#177;&#160;3.9</td><td valign="top">91.6&#160;&#177;&#160;1.1</td><td valign="top">1.05&#160;&#177;&#160;0.06</td><td valign="top">No</td><td valign="top">0.9&#160;&#177;&#160;0.17</td><td valign="top">No</td><td valign="top">0.32&#160;&#177;&#160;0.04</td></tr><tr><td valign="top"><italic>F. prausnitzii</italic> <break/>A2&#8722;165/<italic>B. hydrogenotrophica</italic> S5a33</td><td valign="top">2</td><td valign="top">Fructose/acetate</td><td valign="top">&#8722;49.3&#160;&#177;&#160;1.0</td><td valign="top">41.4&#160;&#177;&#160;11.7</td><td valign="top">30.7&#160;&#177;&#160;1.8</td><td valign="top">&#8722;1.2&#160;&#177;&#160;0.0</td><td valign="top">3.9&#160;&#177;&#160;1.6</td><td valign="top">56.2&#160;&#177;&#160;37.1</td><td valign="top">54.8&#160;&#177;&#160;33.3</td><td valign="top">91.6&#160;&#177;&#160;9.4</td><td valign="top">0.88&#160;&#177;&#160;0.25</td><td valign="top">No</td><td valign="top">0.6&#160;&#177;&#160;0.14</td><td valign="top">Yes</td><td valign="top">0.26&#160;&#177;&#160;0.13</td></tr><tr><td valign="top"><italic>F. prausnitzii</italic> <break/>A2&#8722;165/<italic>B. hydrogenotrophica</italic> S5a33</td><td valign="top">1</td><td valign="top">Fructose</td><td valign="top">&#8722;47.0</td><td valign="top">62.5</td><td valign="top">25.5</td><td valign="top">&#8722;1.1</td><td valign="top">4.5</td><td valign="top">62.9</td><td valign="top">63.6</td><td valign="top">107.4</td><td valign="top">1.11</td><td valign="top">No</td><td valign="top">0.63</td><td valign="top">No</td><td valign="top">0.46</td></tr><tr><td valign="top">Tri-culture fermentations</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><italic>R. intestinalis</italic> <break/>L1&#8722;82/<italic>F. prausnitzii</italic> A2&#8722;165/<italic>B. hydrogenotrophica</italic> S5a33</td><td valign="top">6</td><td valign="top">Fructose</td><td valign="top">&#8722;48.9&#160;&#177;&#160;1.3</td><td valign="top">38.4&#160;&#177;&#160;16.9</td><td valign="top">32.5&#160;&#177;&#160;5.5</td><td valign="top">&#8722;1.2&#160;&#177;&#160;0.1</td><td valign="top">7.4&#160;&#177;&#160;1.7</td><td valign="top">61.9&#160;&#177;&#160;3.2</td><td valign="top">57.1&#160;&#177;&#160;9.2</td><td valign="top">97.0&#160;&#177;&#160;3.1</td><td valign="top">0.84&#160;&#177;&#160;0.09</td><td valign="top">No</td><td valign="top">0.78&#160;&#177;&#160;0.27</td><td valign="top">No</td><td valign="top">0.58&#160;&#177;&#160;0.11</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37090-v3.xml">elife-37090-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.37090.005</object-id><label>Table 1.</label><caption><title>Overview of fermentation experiments and model fitting results (RMSE: root mean square error).</title><p>Means and standard deviations are reported across biological replicates.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Strains</th><th rowspan="2" valign="top">Number of&#160;biological replicates</th><th rowspan="2" valign="top">Added energy source and/or co-substrate</th><th colspan="7" valign="top">Consumption&#160;(-)/production&#160;(+) of metabolites in mM</th><th rowspan="2" valign="top">Carbon recovery (%)</th><th rowspan="2" valign="top">O/R balance</th><th rowspan="2" valign="top">Selected for parame-terization 1</th><th rowspan="2" valign="top">RMSE parame-terization 1</th><th rowspan="2" valign="top">Selected for parame-terization 2</th><th rowspan="2" valign="top">RMSE parame-terization 2</th></tr><tr><th valign="top">Fructose</th><th valign="top">Acetate</th><th valign="top">Butyrate</th><th valign="top">Formate</th><th valign="top">Lactate</th><th valign="top">H<sub>2</sub></th><th valign="top">CO<sub>2</sub></th></tr></thead><tbody><tr><td valign="top">Monoculture fermentations</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><italic>R. intestinalis</italic> <break/>L1-82</td><td valign="top">4</td><td valign="top">Fructose/acetate</td><td valign="top">&#8722;48.8&#160;&#177;&#160;0.9</td><td valign="top">&#8722;20.6&#160;&#177;&#160;2.4</td><td valign="top">58.5&#160;&#177;&#160;3.9</td><td valign="top">6.8&#160;&#177;&#160;1.2</td><td valign="top">3.9&#160;&#177;&#160;1.1</td><td valign="top">72.2&#160;&#177;&#160;4.9</td><td valign="top">84.0&#160;&#177;&#160;0.8</td><td valign="top">105&#160;&#177;&#160;14.3</td><td valign="top">0.86&#160;&#177;&#160;0.2</td><td valign="top">Yes</td><td valign="top">0.17&#160;&#177;&#160;0.04</td><td valign="top">No</td><td valign="top">0.83&#160;&#177;&#160;0.13</td></tr><tr><td valign="top"><italic>F. prausnitzii</italic> A2-165</td><td valign="top">3</td><td valign="top">Fructose/acetate</td><td valign="top">&#8722;23.1&#160;&#177;&#160;1.1</td><td valign="top">&#8722;13.5&#160;&#177;&#160;0.8</td><td valign="top">29.8&#160;&#177;&#160;0.2</td><td valign="top">22.7&#160;&#177;&#160;0.9</td><td valign="top">1.8&#160;&#177;&#160;0.2</td><td valign="top">0.2&#160;&#177;&#160;0.2</td><td valign="top">19.3&#160;&#177;&#160;1.6</td><td valign="top">100.9&#160;&#177;&#160;3.4</td><td valign="top">1.05&#160;&#177;&#160;0.09</td><td valign="top">Yes</td><td valign="top">0.15&#160;&#177;&#160;0.08</td><td valign="top">Yes</td><td valign="top">0.15&#160;&#177;&#160;0.08</td></tr><tr><td valign="top"><italic>B. hydrogenotrophica</italic> S5a33</td><td valign="top">3</td><td valign="top">Fructose/formate</td><td valign="top">&#8722;19.0&#160;&#177;&#160;5.6</td><td valign="top">23.0&#160;&#177;&#160;6.6</td><td valign="top">0.0&#160;&#177;&#160;0.0</td><td valign="top">&#8722;36.1&#160;&#177;&#160;1.8</td><td valign="top">6.0&#160;&#177;&#160;1.2</td><td valign="top">31.5&#160;&#177;&#160;7.7</td><td valign="top">26.0&#160;&#177;&#160;5.3</td><td valign="top">60.0&#160;&#177;&#160;2.1</td><td valign="top">0.76&#160;&#177;&#160;0.07</td><td valign="top">Yes</td><td valign="top">1.43&#160;&#177;&#160;0.27</td><td valign="top">No</td><td valign="top">3.53&#160;&#177;&#160;1.04</td></tr><tr><td valign="top">Bi-culture fermentations</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><italic>R. intestinalis</italic> <break/>L1&#8722;82/<italic>F. prausnitzii</italic> A2-165</td><td valign="top">2</td><td valign="top">Fructose/acetate</td><td valign="top">&#8722;47.7&#160;&#177;&#160;2.2</td><td valign="top">&#8722;18.1&#160;&#177;&#160;2.7</td><td valign="top">56.3&#160;&#177;&#160;4.5</td><td valign="top">7.6&#160;&#177;&#160;2.7</td><td valign="top">4.3&#160;&#177;&#160;3.3</td><td valign="top">114&#160;&#177;&#160;16</td><td valign="top">85.7&#160;&#177;&#160;4.1</td><td valign="top">102.8&#160;&#177;&#160;0.3</td><td valign="top">0.79&#160;&#177;&#160;0</td><td valign="top">No</td><td valign="top">0.77&#160;&#177;&#160;0.06</td><td valign="top">No</td><td valign="top">0.68&#160;&#177;&#160;0.03</td></tr><tr><td valign="top"><italic>R. intestinalis</italic> <break/>L1&#8722;82/<italic>B. hydrogenotrophica</italic> S5a33</td><td valign="top">2</td><td valign="top">Fructose/acetate</td><td valign="top">&#8722;46.5&#160;&#177;&#160;1.5</td><td valign="top">&#8722;7.5&#160;&#177;&#160;1.9</td><td valign="top">53.5&#160;&#177;&#160;4.7</td><td valign="top">&#8722;0.5&#160;&#177;&#160;0.8</td><td valign="top">2.2&#160;&#177;&#160;0.3</td><td valign="top">53.0&#160;&#177;&#160;3.0</td><td valign="top">74.9&#160;&#177;&#160;4.1</td><td valign="top">100.2&#160;&#177;&#160;0.6</td><td valign="top">0.94&#160;&#177;&#160;0.12</td><td valign="top">No</td><td valign="top">0.78&#160;&#177;&#160;0.36</td><td valign="top">Yes</td><td valign="top">0.3&#160;&#177;&#160;0.04</td></tr><tr><td valign="top"><italic>R. intestinalis</italic> <break/>L1&#8722;82/<italic>B. hydrogenotrophica</italic> S5a33</td><td valign="top">2</td><td valign="top">Fructose</td><td valign="top">&#8722;48.0&#160;&#177;&#160;0.2</td><td valign="top">44.9&#160;&#177;&#160;5.4</td><td valign="top">27.9&#160;&#177;&#160;3.0</td><td valign="top">&#8722;1.0&#160;&#177;&#160;0.0</td><td valign="top">5.4&#160;&#177;&#160;2.4</td><td valign="top">33.1&#160;&#177;&#160;3.2</td><td valign="top">47.4&#160;&#177;&#160;3.9</td><td valign="top">91.6&#160;&#177;&#160;1.1</td><td valign="top">1.05&#160;&#177;&#160;0.06</td><td valign="top">No</td><td valign="top">0.9&#160;&#177;&#160;0.17</td><td valign="top">No</td><td valign="top">0.32&#160;&#177;&#160;0.04</td></tr><tr><td valign="top"><italic>F. prausnitzii</italic> <break/>A2&#8722;165/<italic>B. hydrogenotrophica</italic> S5a33</td><td valign="top">2</td><td valign="top">Fructose/acetate</td><td valign="top">&#8722;49.3&#160;&#177;&#160;1.0</td><td valign="top">41.4&#160;&#177;&#160;11.7</td><td valign="top">30.7&#160;&#177;&#160;1.8</td><td valign="top">&#8722;1.2&#160;&#177;&#160;0.0</td><td valign="top">3.9&#160;&#177;&#160;1.6</td><td valign="top">56.2&#160;&#177;&#160;37.1</td><td valign="top">54.8&#160;&#177;&#160;33.3</td><td valign="top">91.6&#160;&#177;&#160;9.4</td><td valign="top">0.88&#160;&#177;&#160;0.25</td><td valign="top">No</td><td valign="top">0.6&#160;&#177;&#160;0.14</td><td valign="top">Yes</td><td valign="top">0.26&#160;&#177;&#160;0.13</td></tr><tr><td valign="top"><italic>F. prausnitzii</italic> <break/>A2&#8722;165/<italic>B. hydrogenotrophica</italic> S5a33</td><td valign="top">1</td><td valign="top">Fructose</td><td valign="top">&#8722;47.0</td><td valign="top">62.5</td><td valign="top">25.5</td><td valign="top">&#8722;1.1</td><td valign="top">4.5</td><td valign="top">62.9</td><td valign="top">63.6</td><td valign="top">107.4</td><td valign="top">1.11</td><td valign="top">No</td><td valign="top">0.63</td><td valign="top">No</td><td valign="top">0.46</td></tr><tr><td valign="top">Tri-culture fermentations</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><italic>R. intestinalis</italic> <break/>L1&#8722;82/<italic>F. prausnitzii</italic> A2&#8722;165/<italic>B. hydrogenotrophica</italic> S5a33</td><td valign="top">6</td><td valign="top">Fructose</td><td valign="top">&#8722;48.9&#160;&#177;&#160;1.3</td><td valign="top">38.4&#160;&#177;&#160;16.9</td><td valign="top">32.5&#160;&#177;&#160;5.5</td><td valign="top">&#8722;1.2&#160;&#177;&#160;0.1</td><td valign="top">7.4&#160;&#177;&#160;1.7</td><td valign="top">61.9&#160;&#177;&#160;3.2</td><td valign="top">57.1&#160;&#177;&#160;9.2</td><td valign="top">97.0&#160;&#177;&#160;3.1</td><td valign="top">0.84&#160;&#177;&#160;0.09</td><td valign="top">No</td><td valign="top">0.78&#160;&#177;&#160;0.27</td><td valign="top">No</td><td valign="top">0.58&#160;&#177;&#160;0.11</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37125-v1.xml">elife-37125-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.37125.009</object-id><label>Table 1.</label><caption><title>Significant positive clusters of noxious-evoked brain activity, observed across the whole group (n&#160;=&#160;13, cluster forming threshold: z&#160;=&#160;3.1, cluster significance threshold, p&#160;=&#160;0.05).</title><p>This table provides an anatomical description and the location of the peak z-statistic within each active brain region. The group activity reported consisted of 14 distinct clusters, some of which spanned multiple brain regions.</p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2" valign="top">Anatomical description of location of activity</th><th rowspan="2" valign="top">Maximum z-statistic within cluster</th><th colspan="3" valign="top">Coordinates of maximum z-statistic in infant template space</th></tr><tr><th valign="top">X</th><th valign="top">Y</th><th valign="top">Z</th></tr></thead><tbody><tr><td rowspan="2" valign="top">Post-central gyrus</td><td valign="top">Contra</td><td valign="top">4.8</td><td valign="top">7.7</td><td valign="top">&#8722;27.5</td><td valign="top">50.7</td></tr><tr><td valign="top">Ipsi</td><td valign="top">4.7</td><td valign="top">&#8722;23.2</td><td valign="top">&#8722;27.5</td><td valign="top">43.8</td></tr><tr><td rowspan="2" valign="top">Posterior cingulate sulcus</td><td valign="top">Contra</td><td valign="top">4.8</td><td valign="top">6.9</td><td valign="top">&#8722;23.2</td><td valign="top">36.0</td></tr><tr><td valign="top">Ipsi</td><td valign="top">3.8</td><td valign="top">&#8722;6.0</td><td valign="top">&#8722;20.6</td><td valign="top">34.3</td></tr><tr><td rowspan="2" valign="top">Superior parietal lobule</td><td valign="top">Contra</td><td valign="top">4.8</td><td valign="top">12.9</td><td valign="top">&#8722;44.6</td><td valign="top">47.2</td></tr><tr><td valign="top">Ipsi</td><td valign="top">4.7</td><td valign="top">&#8722;14.6</td><td valign="top">&#8722;43.8</td><td valign="top">43.8</td></tr><tr><td rowspan="2" valign="top">Thalamus</td><td valign="top">Contra</td><td valign="top">4.7</td><td valign="top">16.3</td><td valign="top">&#8722;19.7</td><td valign="top">12.0</td></tr><tr><td valign="top">Ipsi</td><td valign="top">4.6</td><td valign="top">&#8722;13.8</td><td valign="top">&#8722;24.0</td><td valign="top">16.3</td></tr><tr><td rowspan="2" valign="top">Supra-marginal gyrus</td><td valign="top">Ipsi</td><td valign="top">4.7</td><td valign="top">&#8722;25.8</td><td valign="top">&#8722;29.2</td><td valign="top">37.8</td></tr><tr><td valign="top">Contra</td><td valign="top">4.6</td><td valign="top">30.1</td><td valign="top">&#8722;24.9</td><td valign="top">28.3</td></tr><tr><td valign="top">Superior frontal sulcus</td><td valign="top">Contra</td><td valign="top">4.7</td><td valign="top">16.3</td><td valign="top">&#8722;4.3</td><td valign="top">44.6</td></tr><tr><td rowspan="2" valign="top">Middle frontal gyrus</td><td valign="top">Ipsi</td><td valign="top">4.7</td><td valign="top">&#8722;27.5</td><td valign="top">0.9</td><td valign="top">30.0</td></tr><tr><td valign="top">Contra</td><td valign="top">4.0</td><td valign="top">24.1</td><td valign="top">5.2</td><td valign="top">33.5</td></tr><tr><td rowspan="2" valign="top">Cuneus</td><td valign="top">Contra</td><td valign="top">4.7</td><td valign="top">7.7</td><td valign="top">&#8722;53.2</td><td valign="top">26.6</td></tr><tr><td valign="top">Ipsi</td><td valign="top">4.2</td><td valign="top">&#8722;5.1</td><td valign="top">&#8722;50.7</td><td valign="top">23.2</td></tr><tr><td rowspan="2" valign="top">Superior parietal lobe&#160;/&#160;Precuneus</td><td valign="top">Contra</td><td valign="top">4.7</td><td valign="top">5.2</td><td valign="top">&#8722;35.2</td><td valign="top">39.5</td></tr><tr><td valign="top">Ipsi</td><td valign="top">4.0</td><td valign="top">&#8722;0.9</td><td valign="top">&#8722;43.8</td><td valign="top">41.2</td></tr><tr><td rowspan="2" valign="top">Pre-central gyrus&#160;/&#160;Central sulcus</td><td valign="top">Contra</td><td valign="top">4.6</td><td valign="top">24.1</td><td valign="top">&#8722;14.6</td><td valign="top">45.5</td></tr><tr><td valign="top">Ipsi</td><td valign="top">3.9</td><td valign="top">&#8722;8.6</td><td valign="top">&#8722;23.2</td><td valign="top">53.2</td></tr><tr><td valign="top">Posterior insula</td><td valign="top">Contra</td><td valign="top">4.6</td><td valign="top">19.0</td><td valign="top">&#8722;19.7</td><td valign="top">24.0</td></tr><tr><td valign="top">Parietal operculum</td><td valign="top">Ipsi</td><td valign="top">4.6</td><td valign="top">&#8722;34.4</td><td valign="top">&#8722;21.4</td><td valign="top">18.0</td></tr><tr><td valign="top">Superior temporal gyrus&#160;/&#160;Posterior operculum</td><td valign="top">Contra</td><td valign="top">4.6</td><td valign="top">31.0</td><td valign="top">&#8722;27.5</td><td valign="top">22.3</td></tr><tr><td valign="top">Occipital gyrus</td><td valign="top">Ipsi</td><td valign="top">4.6</td><td valign="top">&#8722;12.0</td><td valign="top">&#8722;63.5</td><td valign="top">21.4</td></tr><tr><td rowspan="2" valign="top">Anterior cingulate cortex</td><td valign="top">Ipsi</td><td valign="top">4.2</td><td valign="top">&#8722;1.7</td><td valign="top">14.7</td><td valign="top">24.0</td></tr><tr><td valign="top">Contra</td><td valign="top">3.9</td><td valign="top">4.3</td><td valign="top">12.1</td><td valign="top">25.7</td></tr><tr><td valign="top">Superior temporal gyrus</td><td valign="top">Ipsi</td><td valign="top">4.2</td><td valign="top">&#8722;27.5</td><td valign="top">&#8722;28.3</td><td valign="top">12.8</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37161-v2.xml">elife-37161-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.37161.018</object-id><label>Table 2.</label><caption><title>Ligand-binding affinities for WT, Y537S, and D538G mutant ER&#945; LBD.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Ligand/Mutant</th><th align="center" colspan="3">K<sub>d</sub> (nM)<sup>*</sup></th></tr><tr><th>WT</th><th>Y537S</th><th>D538G</th></tr></thead><tbody><tr><td valign="top">E<sub>2</sub><sup>&#8225;</sup></td><td valign="top">0.22&#160;&#177;&#160;0.11</td><td valign="top">1.40&#160;&#177;&#160;0.54</td><td valign="top">1.77&#160;&#177;&#160;0.66</td></tr><tr><td valign="top"/><td align="center" colspan="3">K<sub>i</sub> (nM)<sup>&#8224;</sup></td></tr><tr><td valign="top">4-OHT<sup>&#8225;</sup></td><td valign="top">0.12&#160;&#177;&#160;0.003</td><td valign="top">2.64&#160;&#177;&#160;0.4</td><td valign="top">3.28&#160;&#177;&#160;0.7</td></tr><tr><td valign="top">RAL</td><td valign="top">0.30&#160;&#177;&#160;0.05</td><td valign="top">3.59&#160;&#177;&#160;1.0</td><td valign="top">3.77&#160;&#177;&#160;1.0</td></tr><tr><td valign="top">BZA</td><td valign="top">0.37&#160;&#177;&#160;0.01</td><td valign="top">3.50&#160;&#177;&#160;0.6</td><td valign="top">5.53&#160;&#177;&#160;0.7</td></tr><tr><td valign="top">Fulvestrant<sup>&#8225;</sup></td><td valign="top">0.13&#160;&#177;&#160;0.03</td><td valign="top">3.68&#160;&#177;&#160;0.8</td><td valign="top">5.06&#160;&#177;&#160;1.2</td></tr></tbody></table><table-wrap-foot><fn><p>* Measured directly by Scatchard Analysis using [<sup>3</sup>H]E2.</p><p>&#8224; Calculated using the Cheng-Prusoff equation from the IC<sub>50</sub> values determined in a competitive radiometric binding analysis using [<sup>3</sup>H]E2 as a tracer.</p></fn><fn><p><sup>&#8225;</sup>Indicates previously published data (<xref ref-type="bibr" rid="bib10">Fanning et al., 2016</xref>; <xref ref-type="bibr" rid="bib60">Zhao et al., 2017</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37227-v1.xml">elife-37227-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.37227.006</object-id><label>Table 1.</label><caption><title>Summary demographic details and phantom sensations.</title><p>Data is shown for amputees (AMP), controls (CTR) and congenital one-handers (1H). Congenital one-handers did not feel any phantom limb sensations. All controls have full kinaesthetic sensations. F: female, M: male. Side: side of missing hand; L: left, R: right. Amputation level: 1: shoulder, 2: above elbow; 3: at elbow; 4: below elbow; 5: at wrist. Kin: Phantom limb kinaesthesia (number of independent controllable parts of the hand), Viv: Chronic phantom limb vividness (0: no sensation, 100: intact hand&#8217;s vividness), Pain: Chronic phantom limb pain (0: no pain, 100: worst pain imaginable), AViv/APain: Acute Viv/Pain (on the scanning day), Std: standard deviation, ND: nondominant.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">AMP</th><th rowspan="2">Age <break/>(years)</th><th rowspan="2">Sex</th><th align="center" colspan="4">Amputation</th><th align="center" colspan="5">Phantom sensations</th></tr><tr><th>Side</th><th>Years since</th><th>Age at <break/>(years)</th><th>Level</th><th>Kin <break/>(0-5)</th><th>Viv <break/>(0-100)</th><th>Pain <break/>(0-100)</th><th>AViv <break/>(0-100)</th><th>APain <break/>(0-100)</th></tr></thead><tbody><tr><td>Mean</td><td>50.4</td><td valign="bottom"/><td valign="bottom"/><td>17.6</td><td>32.9</td><td valign="bottom"/><td>3</td><td>58</td><td>46</td><td>65</td><td>21</td></tr><tr><td>St. dev.</td><td>12.1</td><td valign="bottom"/><td valign="bottom"/><td>10.4</td><td>11.8</td><td valign="bottom"/><td>2</td><td>38</td><td>37</td><td>30</td><td>23</td></tr><tr><td>A01</td><td>44</td><td valign="bottom">M</td><td>R</td><td>15</td><td>29</td><td>2</td><td>5</td><td>100</td><td>100</td><td>100</td><td>50</td></tr><tr><td>A02</td><td>53</td><td valign="bottom">M</td><td>L</td><td>32</td><td>21</td><td>2</td><td>5</td><td>50</td><td>100</td><td>60</td><td>70</td></tr><tr><td>A03</td><td>40</td><td valign="bottom">M</td><td>L</td><td>11</td><td>29</td><td>2</td><td>4</td><td>100</td><td>50</td><td>100</td><td>20</td></tr><tr><td>A04</td><td>51</td><td valign="bottom">M</td><td>L</td><td>32</td><td>19</td><td>2</td><td>5</td><td>100</td><td>0</td><td>100</td><td>0</td></tr><tr><td>A05</td><td>27</td><td valign="bottom">F</td><td>R</td><td>7</td><td>20</td><td>4</td><td>2</td><td>50</td><td>40</td><td>60</td><td>0</td></tr><tr><td>A06</td><td>71</td><td valign="bottom">M</td><td>R</td><td>16</td><td>55</td><td>2</td><td>1</td><td>20</td><td>85</td><td>60</td><td>20</td></tr><tr><td>A07</td><td>46</td><td valign="bottom">M</td><td>R</td><td>18</td><td>28</td><td>2</td><td>3</td><td>70</td><td>90</td><td>70</td><td>50</td></tr><tr><td>A08</td><td>56</td><td valign="bottom">M</td><td>L</td><td>26</td><td>30</td><td>4</td><td>5</td><td>6</td><td>40</td><td>10</td><td>0</td></tr><tr><td>A09</td><td>64</td><td valign="bottom">M</td><td>L</td><td>31</td><td>33</td><td>2</td><td>4</td><td>100</td><td>40</td><td>100</td><td>10</td></tr><tr><td>A10</td><td>58</td><td valign="bottom">M</td><td>L</td><td>2</td><td>56</td><td>2</td><td>3</td><td>90</td><td>0</td><td>80</td><td>0</td></tr><tr><td>A11</td><td>28</td><td valign="bottom">M</td><td>L</td><td>8</td><td>20</td><td>5</td><td>4</td><td>40</td><td>40</td><td>20</td><td>0</td></tr><tr><td>A12</td><td>57</td><td valign="bottom">M</td><td>R</td><td>29</td><td>28</td><td>2</td><td>1</td><td>80</td><td>90</td><td>80</td><td>40</td></tr><tr><td>A13</td><td>50</td><td valign="bottom">F</td><td>L</td><td>1</td><td>49</td><td>4</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td>A14</td><td>52</td><td valign="bottom">M</td><td>R</td><td>27</td><td>25</td><td>2</td><td>5</td><td>100</td><td>80</td><td>80</td><td>50</td></tr><tr><td>A15</td><td>68</td><td valign="bottom">M</td><td>R</td><td>26</td><td>42</td><td>4</td><td>1</td><td>16</td><td>0</td><td>80</td><td>0</td></tr><tr><td>A16</td><td>39</td><td valign="bottom">F</td><td>R</td><td>9</td><td>30</td><td>3</td><td>4</td><td>35</td><td>40</td><td>50</td><td>30</td></tr><tr><td>A17</td><td>58</td><td valign="bottom">M</td><td>L</td><td>12</td><td>46</td><td>4</td><td>5</td><td>2</td><td>0</td><td>65</td><td>0</td></tr><tr><td>A18</td><td>46</td><td valign="bottom">F</td><td>L</td><td>14</td><td>32</td><td>4</td><td>3</td><td>80</td><td>30</td><td>50</td><td>30</td></tr><tr><th rowspan="2">CTR</th><th rowspan="2">Age <break/>(years)</th><th rowspan="2">Sex</th><th>ND hand</th><th rowspan="2" valign="bottom"/><th rowspan="2">1H</th><th rowspan="2">Age <break/>(years)</th><th rowspan="2">Sex</th><th colspan="2">Missing hand</th><th colspan="2" rowspan="2" valign="bottom"/></tr><tr><th>Side</th><th>Side</th><th>Level</th></tr><tr><td>Mean</td><td>45.3</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td>Mean</td><td>45.7</td><td valign="bottom"/><td colspan="2" valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td>St. dev.</td><td>14.9</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td>St. dev.</td><td>10.4</td><td valign="bottom"/><td colspan="2" valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td>C01</td><td>29</td><td valign="bottom">M</td><td>R</td><td valign="bottom"/><td>H01</td><td>41</td><td valign="bottom">M</td><td>L</td><td valign="bottom">4</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td>C02</td><td>24</td><td valign="bottom">F</td><td>L</td><td valign="bottom"/><td>H02</td><td>37</td><td valign="bottom">M</td><td>R</td><td valign="bottom">4</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td>C03</td><td>47</td><td valign="bottom">F</td><td>L</td><td valign="bottom"/><td>H03</td><td>31</td><td valign="bottom">F</td><td>L</td><td valign="bottom">4</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td>C04</td><td>39</td><td valign="bottom">M</td><td>L</td><td valign="bottom"/><td>H04</td><td>60</td><td valign="bottom">M</td><td>L</td><td valign="bottom">4</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td>C05</td><td>32</td><td valign="bottom">M</td><td>R</td><td valign="bottom"/><td>H05</td><td>39</td><td valign="bottom">F</td><td>L</td><td valign="bottom">4</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td>C06</td><td>53</td><td valign="bottom">F</td><td>R</td><td valign="bottom"/><td>H06</td><td>54</td><td valign="bottom">F</td><td>L</td><td valign="bottom">4</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td>C07</td><td>38</td><td valign="bottom">F</td><td>R</td><td valign="bottom"/><td>H07</td><td>34</td><td valign="bottom">M</td><td>L</td><td valign="bottom">4</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td>C08</td><td>67</td><td valign="bottom">M</td><td>R</td><td valign="bottom"/><td>H08</td><td>63</td><td valign="bottom">M</td><td>L</td><td valign="bottom">4</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td>C09</td><td>42</td><td valign="bottom">M</td><td>R</td><td valign="bottom"/><td>H09</td><td>44</td><td valign="bottom">F</td><td>R</td><td valign="bottom">4</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td>C10</td><td>41</td><td valign="bottom">M</td><td>R</td><td valign="bottom"/><td>H10</td><td>55</td><td valign="bottom">F</td><td>L</td><td valign="bottom">4</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td>C11</td><td>69</td><td valign="bottom">M</td><td>L</td><td valign="bottom"/><td>H11</td><td>46</td><td valign="bottom">M</td><td>R</td><td valign="bottom">4</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td>C12</td><td>63</td><td valign="bottom">F</td><td>L</td><td valign="bottom"/><td>H12</td><td>37</td><td valign="bottom">M</td><td>R</td><td valign="bottom">4</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td>H13</td><td>53</td><td valign="bottom">F</td><td>L</td><td valign="bottom">4</td><td valign="bottom"/><td valign="bottom"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37231-v2.xml">elife-37231-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.37231.019</object-id><label>Table 1.</label><caption><title>Summary of the deacetylation turnover (V/[E]) from data in <xref ref-type="fig" rid="fig6">Figure 6</xref> and <xref ref-type="fig" rid="fig6s1">Figure 6&#8212;figure supplement 1</xref> (p&lt;0.001 for V/[E] of K9ac nucleosome vs.&#160;K14ac nucleosome substrates in both the monoacetylated and triacetylated assays); (n&#160;=&#160;3 for all measurements except n&#160;=&#160;2 for LHC on H3 and HDAC1 on nucleosomes; kinetic values shown are&#160;&#177;&#160;S.E.M.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2">Substrate</th><th colspan="3" valign="top">V/[E] (min<sup>&#8722;1</sup>)</th></tr><tr><th>K9ac</th><th>K14ac</th><th>K18ac</th></tr></thead><tbody><tr><td rowspan="4">LHC</td><td>Kac Nucleosome</td><td>0.10&#160;&#177;&#160;0.0084</td><td>0.014&#160;&#177;&#160;0.0073</td><td>0.048&#160;&#177;&#160;0.011</td></tr><tr><td>Kac H3</td><td>31&#160;&#177;&#160;3.9</td><td>35&#160;&#177;&#160;8.5</td><td>-</td></tr><tr><td>3Kac Nucleosome</td><td>0.11&#160;&#177;&#160;0.0071</td><td>0.019&#160;&#177;&#160;0.0033</td><td>0.058&#160;&#177;&#160;0.0022</td></tr><tr><td>3Kac H3</td><td>31&#160;&#177;&#160;4.3</td><td>35&#160;&#177;&#160;2.7</td><td>-</td></tr><tr><td rowspan="2">HDAC1</td><td>Kac Nucleosome</td><td>&lt;0.005</td><td>&lt;0.005</td><td>-</td></tr><tr><td>Kac H3</td><td>10&#160;&#177;&#160;0.89</td><td>12&#160;&#177;&#160;0.26</td><td>-</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37294-v2.xml">elife-37294-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.37294.011</object-id><label>Table 2.</label><caption><title>Tumor CNA burden is associated with recurrence and overall survival independent of disease stage in multiple cancer types</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><italic>Cohort</italic></th><th rowspan="2"><italic>Model</italic></th><th colspan="4" valign="top"><italic>Disease free Time</italic></th><th colspan="4" valign="top"><italic>Overall Survival</italic></th></tr><tr><th valign="top">Cases</th><th valign="top">HR</th><th valign="top">95% CI</th><th valign="top">P</th><th valign="top">Cases</th><th valign="top">HR</th><th valign="top">95% CI</th><th colspan="2" valign="top">P</th></tr></thead><tbody><tr><td rowspan="2" valign="top">Prostate cancer TCGA</td><td>Tumor CNA burden, per 5% tumor CNA burden, univariate</td><td>280</td><td>1.27</td><td>1.13, 1.42</td><td>&lt;0.0001</td><td colspan="4" rowspan="4">Insufficient events <break/> <break/><break/></td></tr><tr><td>Tumor CNA burden, per 5% tumor CNA burden, adjusted for Gleason grade and mutation burden</td><td>279</td><td>1.18</td><td>1.03, 1.35</td><td>0.015</td></tr><tr><td/><td>Tumor CNA burden, per 5% tumor CNA burden, adjusted for purity (ABSOLUTE)*</td><td>243</td><td>1.22</td><td>1.07, 1.40</td><td>0.003</td></tr><tr><td/><td>Tumor CNA burden, per 5% tumor CNA burden, adjusted for ploidy</td><td>243</td><td>1.32</td><td>1.11, 1.56</td><td>0.002</td></tr><tr><td rowspan="2" valign="top">Breast cancer TCGA</td><td>Tumor CNA burden, per 5% tumor CNA burden, univariate</td><td>709</td><td>1.07</td><td>1.01, 1.14</td><td>0.028</td><td>794</td><td>1.08</td><td>1.03, 1.13</td><td>0.0005</td></tr><tr><td>Tumor CNA burden, per 5% tumor CNA burden, Multivariable, adjusted for disease stage</td><td>695</td><td>1.07</td><td>1.00, 1.14</td><td>0.049</td><td>777</td><td>1.08</td><td>1.03, 1.13</td><td>0.002</td></tr><tr><td rowspan="2" valign="top">Endometrial Cancer TCGA</td><td>Tumor CNA burden, per 5% tumor CNA burden, univariate</td><td>496</td><td>1.10</td><td>1.06, 1.14</td><td>&lt;0.0001</td><td>536</td><td>1.13</td><td>1.08, 1.17</td><td>&lt;0.0001</td></tr><tr><td>Tumor CNA burden, per 5% tumor CNA burden, multivariable, adjusted for disease stage</td><td>496</td><td>1.08</td><td>1.04, 1.13</td><td>&lt;0.0001</td><td>536</td><td>1.10</td><td>1.05, 1.15</td><td>&lt;0.0001</td></tr><tr><td rowspan="2" valign="top">Renal clear cell cancer TCGA</td><td>Tumor CNA burden, per 5% tumor CNA burden, univariate</td><td>425</td><td>1.05</td><td>1.01, 1.09</td><td>0.028</td><td>525</td><td>1.02</td><td>0.98, 1.06</td><td>NS (0.4)</td></tr><tr><td>Tumor CNA burden, per 5% tumor CNA burden, multivariable, adjusted for disease stage</td><td>423</td><td>1.05</td><td>1.00, 1.11</td><td>0.035</td><td>522</td><td>1.01</td><td>0.97, 1.06</td><td>NS (0.7)</td></tr><tr><td rowspan="2" valign="top">Thyroid cancer TCGA</td><td>Tumor CNA burden, per 5% tumor CNA burden, univariate</td><td>483</td><td>1.17</td><td>1.01, 1.35</td><td>0.033</td><td>497</td><td>1.30</td><td>1.04, 1.63</td><td>0.021</td></tr><tr><td>Tumor CNA burden, per 5% tumor CNA burden, multivariable, adjusted for disease stage</td><td>481</td><td>1.18</td><td>1.00, 1.39</td><td>0.048</td><td colspan="4">Insufficient events</td></tr><tr><td rowspan="2" valign="bottom">Colorectal cancer TCGA</td><td>Tumor CNA burden, per 5% tumor CNA burden, univariate</td><td>512</td><td>1.05</td><td>1.00, 1.11</td><td>0.037</td><td>587</td><td>1.06</td><td>1.01, 1.12</td><td>0.012</td></tr><tr><td>Tumor CNA burden, per 5% tumor CNA burden, multivariable, adjusted for disease stage</td><td>496</td><td>1.03</td><td>0.98, 1.09</td><td>NS (0.3)</td><td>567</td><td>1.03</td><td>0.97, 1.09</td><td>NS (0.3)</td></tr><tr><td valign="bottom"/><td>Tumor CNA burden, per 5% tumor CNA burden, multivariable, adjusted for tumor stage</td><td>511</td><td>1.06</td><td>1.01, 1.12</td><td>0.028</td><td>585</td><td>1.07</td><td>1.02, 1.13</td><td>0.009</td></tr></tbody></table><table-wrap-foot><fn><p>*Result differed with similar adjustment in IMPACT prostate cancer cohort utilizing FACETS, see <xref ref-type="supplementary-material" rid="supp2">Supplementary file 3</xref>.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37294-v2.xml">elife-37294-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.37294.012</object-id><label>Table 3.</label><caption><title>Tumor CNA burden determined by clinically approved sequencing panel is associated with overall survival in primary and metastatic tumors</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"><italic>Model</italic></th><th colspan="12" valign="top"><italic>Overall Survival</italic></th></tr><tr><th colspan="4" valign="top"><italic>Primary tumors</italic></th><th colspan="8" valign="top"><italic>Metastatic tumors</italic></th></tr><tr><th valign="top">HR</th><th colspan="2" valign="top">95%</th><th valign="top">P</th><th colspan="3" valign="top">HR</th><th colspan="3" valign="top">95%</th><th colspan="2" valign="top">P</th></tr></thead><tbody><tr><td colspan="13">Prostate Cancer<sup>*,&#8224;</sup></td></tr><tr><td>Univariate, tumor CNA burden, per 5%</td><td colspan="2">1.17</td><td>1.04, 1.31</td><td>0.007</td><td colspan="2">1.07</td><td colspan="3">1.01, 1.14</td><td colspan="3">0.020</td></tr><tr><td valign="bottom">Multivariable <break/>Tumor CNA burden, per 5% <break/>Mutation burden (per mutation)</td><td colspan="2">1.11 <break/>1.22</td><td>0.98, 1.26 <break/>1.12, 1.33</td><td>0.10 <break/>&lt;0.0001</td><td colspan="2">1.08 <break/>1.05</td><td colspan="3">1.02, 1.15 <break/>1.02, 1.08</td><td colspan="3">0.011 <break/>0.001</td></tr><tr><td valign="bottom">Multivariable <break/>Tumor CNA burden, per 5% <break/><italic>TP53</italic> CN loss or mutation</td><td colspan="2">1.17 <break/>4.12</td><td valign="bottom">1.04, 1.31 <break/>2.02, 8.41</td><td valign="bottom">0.007 <break/>&lt;0.0001</td><td colspan="2" valign="top">1.06 <break/>1.24</td><td colspan="3" valign="top">1.00, 1.13 <break/>0.76, 2.02</td><td colspan="3" valign="top">NS (0.069) <break/>NS (0.4)</td></tr><tr><td valign="bottom">Multivariable <break/>Tumor CNA burden, per 5% <break/> <italic>RB1</italic> CN loss or mutation</td><td colspan="2">1.15 <break/>3.24</td><td>1.02, 1.30 <break/>0.70, 14.98</td><td valign="bottom">0.026 <break/>NS (0.13)</td><td colspan="2" valign="bottom">1.06 <break/>1.68</td><td colspan="3" valign="bottom">0.99, 1.13 <break/>0.94, 2.99</td><td colspan="3" valign="bottom">NS (0.091) <break/>NS (0.080)</td></tr><tr><td valign="bottom">Multivariable <break/>Tumor CNA burden, per 5% <break/><italic>PTEN</italic> CN loss or mutation</td><td colspan="2">1.17 <break/>2.38</td><td>1.04, 1.32 <break/>1.03, 5.51</td><td valign="bottom">0.008 <break/>0.042</td><td colspan="2" valign="top">1.07 <break/>1.15</td><td colspan="3" valign="top">1.01, 1.14 <break/>0.70, 1.89</td><td colspan="3" valign="top">0.023 <break/>NS (0.6)</td></tr><tr><td colspan="13">Pan- Cancer</td></tr><tr><td>Univariate, tumor CNA burden, per 5%<sup>&#8225;&#167;</sup></td><td colspan="2">1.04</td><td>1.02, 1.05</td><td>&lt;0.0001</td><td colspan="2">1.02</td><td colspan="5">1.01, 1.03</td><td>0.005</td></tr><tr><td>Univariate, mutation burden (per five units)<sup>&#8225;&#167;</sup></td><td colspan="2">0.98</td><td>0.97, 1.00</td><td>NS (0.072)</td><td colspan="2">0.99</td><td colspan="5">0.97, 1.01</td><td>NS (0.4)</td></tr><tr><td>Meta-analysis, tumor CNA burden (per 5%)<sup>#</sup></td><td colspan="2">1.04</td><td>1.02, 1.05</td><td>&lt;0.0001**</td><td colspan="2">1.02</td><td colspan="5">1.01, 1.04</td><td>0.005<sup>&#8224;&#8224;</sup></td></tr><tr><td>Meta-analysis, tumor CNA burden (per 5%), excluding outlier cancer types<sup>&#8225;&#8225;</sup></td><td colspan="2">1.05</td><td>1.03, 1.07</td><td>&lt;0.0001<sup>&#167;&#167;</sup></td><td colspan="2" valign="top">1.03</td><td colspan="5">1.01, 1.04</td><td>0.002<sup>##</sup></td></tr></tbody></table><table-wrap-foot><fn><p>*Prostate primary tumors: patient n&#160;=&#160;261 for all models except multivariable model with mutation burden, where n&#160;=&#160;227; event n&#160;=&#160;33; median follow-up time for survivors 40 (IQR 25,81) months.</p><p>&#8224;Prostate metastatic tumors: patient n&#160;=&#160;216 for all models except multivariable model with mutation burden, where n&#160;=&#160;205; event n&#160;=&#160;80; median follow-up time for survivors 59.5 (IQR 32, 129) months.</p></fn><fn><p>&#8225;Pan-cancer primary tumors, univariate models: patient n&#160;=&#160;6610, event n&#160;=&#160;1535, median follow-up time for survivors 24 (IQR 11, 61) months</p><p>&#167;Pan-cancer metastatic tumors, univariate models: patient n&#160;=&#160;4864, event n&#160;=&#160;1467, median follow-up time for survivors 51 (IQR 23, 109) months.</p></fn><fn><p>#Pan-cancer meta-analysis, among ten most prevalent cancer types: primary tumor patient n&#160;=&#160;4863, metastatic tumor patient n&#160;=&#160;3676. Estimates are based on overall fixed effects.</p><p>**p-value corresponds with test of effects size. Chi-square test for heterogeneity p-value=0.003.</p></fn><fn><p>&#8224;&#8224;p-value corresponds with test of effects size. Chi-square test for heterogeneity p-value=0.024.</p><p>&#8225;&#8225;Exclusion of cancer types to reduce heterogeneity: primary tumor patient n&#160;=&#160;3887, metastatic tumor patient n&#160;=&#160;3098. Estimates are based on overall fixed effects.</p></fn><fn><p>&#167;&#167;Excluding pancreatic and colorectal cancer, test of effects size p-value. Chi-square test for heterogeneity p-value=0.3.</p><p>##Excluding pancreatic and prostate cancer, test of effects size p-value. Chi-square test for heterogeneity p-value=0.8.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37294-v3.xml">elife-37294-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.37294.011</object-id><label>Table 2.</label><caption><title>Tumor CNA burden is associated with recurrence and overall survival independent of disease stage in multiple cancer types</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><italic>Cohort</italic></th><th rowspan="2"><italic>Model</italic></th><th colspan="4" valign="top"><italic>Disease free Time</italic></th><th colspan="4" valign="top"><italic>Overall Survival</italic></th></tr><tr><th valign="top">Cases</th><th valign="top">HR</th><th valign="top">95% CI</th><th valign="top">P</th><th valign="top">Cases</th><th valign="top">HR</th><th valign="top">95% CI</th><th colspan="2" valign="top">P</th></tr></thead><tbody><tr><td rowspan="2" valign="top">Prostate cancer TCGA</td><td>Tumor CNA burden, per 5% tumor CNA burden, univariate</td><td>280</td><td>1.27</td><td>1.13, 1.42</td><td>&lt;0.0001</td><td colspan="4" rowspan="4">Insufficient events <break/> <break/><break/></td></tr><tr><td>Tumor CNA burden, per 5% tumor CNA burden, adjusted for Gleason grade and mutation burden</td><td>279</td><td>1.18</td><td>1.03, 1.35</td><td>0.015</td></tr><tr><td/><td>Tumor CNA burden, per 5% tumor CNA burden, adjusted for purity (ABSOLUTE)*</td><td>243</td><td>1.22</td><td>1.07, 1.40</td><td>0.003</td></tr><tr><td/><td>Tumor CNA burden, per 5% tumor CNA burden, adjusted for ploidy</td><td>243</td><td>1.32</td><td>1.11, 1.56</td><td>0.002</td></tr><tr><td rowspan="2" valign="top">Breast cancer TCGA</td><td>Tumor CNA burden, per 5% tumor CNA burden, univariate</td><td>709</td><td>1.07</td><td>1.01, 1.14</td><td>0.028</td><td>794</td><td>1.08</td><td>1.03, 1.13</td><td>0.0005</td></tr><tr><td>Tumor CNA burden, per 5% tumor CNA burden, Multivariable, adjusted for disease stage</td><td>695</td><td>1.07</td><td>1.00, 1.14</td><td>0.049</td><td>777</td><td>1.08</td><td>1.03, 1.13</td><td>0.002</td></tr><tr><td rowspan="2" valign="top">Endometrial Cancer TCGA</td><td>Tumor CNA burden, per 5% tumor CNA burden, univariate</td><td>496</td><td>1.10</td><td>1.06, 1.14</td><td>&lt;0.0001</td><td>536</td><td>1.13</td><td>1.08, 1.17</td><td>&lt;0.0001</td></tr><tr><td>Tumor CNA burden, per 5% tumor CNA burden, multivariable, adjusted for disease stage</td><td>496</td><td>1.08</td><td>1.04, 1.13</td><td>&lt;0.0001</td><td>536</td><td>1.10</td><td>1.05, 1.15</td><td>&lt;0.0001</td></tr><tr><td rowspan="2" valign="top">Renal clear cell cancer TCGA</td><td>Tumor CNA burden, per 5% tumor CNA burden, univariate</td><td>425</td><td>1.05</td><td>1.01, 1.09</td><td>0.028</td><td>525</td><td>1.02</td><td>0.98, 1.06</td><td>NS (0.4)</td></tr><tr><td>Tumor CNA burden, per 5% tumor CNA burden, multivariable, adjusted for disease stage</td><td>423</td><td>1.05</td><td>1.00, 1.11</td><td>0.035</td><td>522</td><td>1.01</td><td>0.97, 1.06</td><td>NS (0.7)</td></tr><tr><td rowspan="2" valign="top">Thyroid cancer TCGA</td><td>Tumor CNA burden, per 5% tumor CNA burden, univariate</td><td>483</td><td>1.17</td><td>1.01, 1.35</td><td>0.033</td><td>497</td><td>1.30</td><td>1.04, 1.63</td><td>0.021</td></tr><tr><td>Tumor CNA burden, per 5% tumor CNA burden, multivariable, adjusted for disease stage</td><td>481</td><td>1.18</td><td>1.00, 1.39</td><td>0.048</td><td colspan="4">Insufficient events</td></tr><tr><td rowspan="2" valign="bottom">Colorectal cancer TCGA</td><td>Tumor CNA burden, per 5% tumor CNA burden, univariate</td><td>512</td><td>1.05</td><td>1.00, 1.11</td><td>0.037</td><td>587</td><td>1.06</td><td>1.01, 1.12</td><td>0.012</td></tr><tr><td>Tumor CNA burden, per 5% tumor CNA burden, multivariable, adjusted for disease stage</td><td>496</td><td>1.03</td><td>0.98, 1.09</td><td>NS (0.3)</td><td>567</td><td>1.03</td><td>0.97, 1.09</td><td>NS (0.3)</td></tr><tr><td valign="bottom"/><td>Tumor CNA burden, per 5% tumor CNA burden, multivariable, adjusted for tumor stage</td><td>511</td><td>1.06</td><td>1.01, 1.12</td><td>0.028</td><td>585</td><td>1.07</td><td>1.02, 1.13</td><td>0.009</td></tr></tbody></table><table-wrap-foot><fn><p>*Result differed with similar adjustment in IMPACT prostate cancer cohort utilizing FACETS, see <xref ref-type="supplementary-material" rid="supp2">Supplementary file 3</xref>.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37294-v3.xml">elife-37294-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.37294.012</object-id><label>Table 3.</label><caption><title>Tumor CNA burden determined by clinically approved sequencing panel is associated with overall survival in primary and metastatic tumors</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"><italic>Model</italic></th><th colspan="12" valign="top"><italic>Overall Survival</italic></th></tr><tr><th colspan="4" valign="top"><italic>Primary tumors</italic></th><th colspan="8" valign="top"><italic>Metastatic tumors</italic></th></tr><tr><th valign="top">HR</th><th colspan="2" valign="top">95%</th><th valign="top">P</th><th colspan="3" valign="top">HR</th><th colspan="3" valign="top">95%</th><th colspan="2" valign="top">P</th></tr></thead><tbody><tr><td colspan="13">Prostate Cancer<sup>*,&#8224;</sup></td></tr><tr><td>Univariate, tumor CNA burden, per 5%</td><td colspan="2">1.17</td><td>1.04, 1.31</td><td>0.007</td><td colspan="2">1.07</td><td colspan="3">1.01, 1.14</td><td colspan="3">0.020</td></tr><tr><td valign="bottom">Multivariable <break/>Tumor CNA burden, per 5% <break/>Mutation burden (per mutation)</td><td colspan="2">1.11 <break/>1.22</td><td>0.98, 1.26 <break/>1.12, 1.33</td><td>0.10 <break/>&lt;0.0001</td><td colspan="2">1.08 <break/>1.05</td><td colspan="3">1.02, 1.15 <break/>1.02, 1.08</td><td colspan="3">0.011 <break/>0.001</td></tr><tr><td valign="bottom">Multivariable <break/>Tumor CNA burden, per 5% <break/><italic>TP53</italic> CN loss or mutation</td><td colspan="2">1.17 <break/>4.12</td><td valign="bottom">1.04, 1.31 <break/>2.02, 8.41</td><td valign="bottom">0.007 <break/>&lt;0.0001</td><td colspan="2" valign="top">1.06 <break/>1.24</td><td colspan="3" valign="top">1.00, 1.13 <break/>0.76, 2.02</td><td colspan="3" valign="top">NS (0.069) <break/>NS (0.4)</td></tr><tr><td valign="bottom">Multivariable <break/>Tumor CNA burden, per 5% <break/> <italic>RB1</italic> CN loss or mutation</td><td colspan="2">1.15 <break/>3.24</td><td>1.02, 1.30 <break/>0.70, 14.98</td><td valign="bottom">0.026 <break/>NS (0.13)</td><td colspan="2" valign="bottom">1.06 <break/>1.68</td><td colspan="3" valign="bottom">0.99, 1.13 <break/>0.94, 2.99</td><td colspan="3" valign="bottom">NS (0.091) <break/>NS (0.080)</td></tr><tr><td valign="bottom">Multivariable <break/>Tumor CNA burden, per 5% <break/><italic>PTEN</italic> CN loss or mutation</td><td colspan="2">1.17 <break/>2.38</td><td>1.04, 1.32 <break/>1.03, 5.51</td><td valign="bottom">0.008 <break/>0.042</td><td colspan="2" valign="top">1.07 <break/>1.15</td><td colspan="3" valign="top">1.01, 1.14 <break/>0.70, 1.89</td><td colspan="3" valign="top">0.023 <break/>NS (0.6)</td></tr><tr><td colspan="13">Pan- Cancer</td></tr><tr><td>Univariate, tumor CNA burden, per 5%<sup>&#8225;&#167;</sup></td><td colspan="2">1.04</td><td>1.02, 1.05</td><td>&lt;0.0001</td><td colspan="2">1.02</td><td colspan="5">1.01, 1.03</td><td>0.005</td></tr><tr><td>Univariate, mutation burden (per five units)<sup>&#8225;&#167;</sup></td><td colspan="2">0.98</td><td>0.97, 1.00</td><td>NS (0.072)</td><td colspan="2">0.99</td><td colspan="5">0.97, 1.01</td><td>NS (0.4)</td></tr><tr><td>Meta-analysis, tumor CNA burden (per 5%)<sup>#</sup></td><td colspan="2">1.04</td><td>1.02, 1.05</td><td>&lt;0.0001**</td><td colspan="2">1.02</td><td colspan="5">1.01, 1.04</td><td>0.005<sup>&#8224;&#8224;</sup></td></tr><tr><td>Meta-analysis, tumor CNA burden (per 5%), excluding outlier cancer types<sup>&#8225;&#8225;</sup></td><td colspan="2">1.05</td><td>1.03, 1.07</td><td>&lt;0.0001<sup>&#167;&#167;</sup></td><td colspan="2" valign="top">1.03</td><td colspan="5">1.01, 1.04</td><td>0.002<sup>##</sup></td></tr></tbody></table><table-wrap-foot><fn><p>*Prostate primary tumors: patient n&#160;=&#160;261 for all models except multivariable model with mutation burden, where n&#160;=&#160;227; event n&#160;=&#160;33; median follow-up time for survivors 40 (IQR 25,81) months.</p><p>&#8224;Prostate metastatic tumors: patient n&#160;=&#160;216 for all models except multivariable model with mutation burden, where n&#160;=&#160;205; event n&#160;=&#160;80; median follow-up time for survivors 59.5 (IQR 32, 129) months.</p></fn><fn><p>&#8225;Pan-cancer primary tumors, univariate models: patient n&#160;=&#160;6610, event n&#160;=&#160;1535, median follow-up time for survivors 24 (IQR 11, 61) months</p><p>&#167;Pan-cancer metastatic tumors, univariate models: patient n&#160;=&#160;4864, event n&#160;=&#160;1467, median follow-up time for survivors 51 (IQR 23, 109) months.</p></fn><fn><p>#Pan-cancer meta-analysis, among ten most prevalent cancer types: primary tumor patient n&#160;=&#160;4863, metastatic tumor patient n&#160;=&#160;3676. Estimates are based on overall fixed effects.</p><p>**p-value corresponds with test of effects size. Chi-square test for heterogeneity p-value=0.003.</p></fn><fn><p>&#8224;&#8224;p-value corresponds with test of effects size. Chi-square test for heterogeneity p-value=0.024.</p><p>&#8225;&#8225;Exclusion of cancer types to reduce heterogeneity: primary tumor patient n&#160;=&#160;3887, metastatic tumor patient n&#160;=&#160;3098. Estimates are based on overall fixed effects.</p></fn><fn><p>&#167;&#167;Excluding pancreatic and colorectal cancer, test of effects size p-value. Chi-square test for heterogeneity p-value=0.3.</p><p>##Excluding pancreatic and prostate cancer, test of effects size p-value. Chi-square test for heterogeneity p-value=0.8.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37373-v2.xml">elife-37373-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.37373.013</object-id><label>Table 1.</label><caption><title>Effects of monovalent cations on the shape of the triphosphate chain of the Mg-ATP complex in water, as inferred from the MD simulation data.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Added cation</th><th colspan="6">Conformation of the triphosphate chain of Mg-ATP<sup>*</sup></th></tr><tr><th colspan="2">&#946;&#947;-coordination</th><th colspan="2">&#946;&#947;-coordination, <break/>&#8216;curled&#8217; phosphate chain</th><th colspan="2">&#945;&#946;&#947;-coordination</th></tr><tr><th>P<sup>A</sup>-P<sup>G</sup> distance, &#197;</th><th>P<sup>B</sup>-O<sup>3B</sup>-P<sup>G</sup> angle</th><th>P<sup>A</sup>-P<sup>G</sup> distance, &#197;</th><th>P<sup>B</sup>-O<sup>3B</sup>-P<sup>G</sup> angle</th><th>P<sup>A</sup>-P<sup>G</sup> distance, &#197;</th><th>P<sup>B</sup>-O<sup>3B</sup>-P<sup>G</sup> angle</th></tr></thead><tbody><tr><td>None</td><td valign="top">5.46&#160;&#177;&#160;0.34</td><td valign="top">122.3&#160;&#177;&#160;3.5</td><td colspan="2" valign="top">N/A</td><td valign="top">4.76&#160;&#177;&#160;0.18</td><td valign="top">124.9&#160;&#177;&#160;3.3</td></tr><tr><td>K<sup>+</sup></td><td valign="top">4.91&#160;&#177;&#160;0.24</td><td valign="top">122.0&#160;&#177;&#160;3.3</td><td colspan="2" valign="top">N/A</td><td valign="top">4.32&#160;&#177;&#160;0.24</td><td valign="top">128.0&#160;&#177;&#160;3.5</td></tr><tr><td>Na<sup>+</sup></td><td valign="top">4.69&#160;&#177;&#160;0.22</td><td valign="top">122.9&#160;&#177;&#160;3.2</td><td valign="top">4.60&#160;&#177;&#160;0.22</td><td valign="top">124.0&#160;&#177;&#160;3.3</td><td valign="top">4.26&#160;&#177;&#160;0.37</td><td valign="top">127.7&#160;&#177;&#160;3.6</td></tr><tr><td>NH<sub>4</sub><sup>+</sup></td><td valign="top">4.85&#160;&#177;&#160;0.22</td><td valign="top">122.3&#160;&#177;&#160;3.3</td><td valign="top">4.56&#160;&#177;&#160;0.21</td><td valign="top">124.6&#160;&#177;&#160;3.3</td><td valign="top">4.22&#160;&#177;&#160;0.16</td><td valign="top">127.8&#160;&#177;&#160;3.</td></tr></tbody></table><table-wrap-foot><fn><p><sup>*</sup>The conformations of the Mg-ATP complex were determined as described in the text. Mean values and standard deviations of P<sup>A</sup>-P<sup>G</sup> distance (in &#197;) and the P<sup>B</sup>-O<sup>3B</sup>-P<sup>G</sup> angle (in degrees) were measured over the respective parts of the simulations. Simulation periods corresponding to &#946;&#947; and &#945;&#946;&#947; conformations were identified by tracking distances between Mg<sup>2+</sup> and non-bridging oxygen atoms of the phosphate chain (<xref ref-type="fig" rid="fig4s1">Figure 4&#8212;figure supplement 1</xref>); simulation periods corresponding to the &#8216;curled&#8217; conformation were identified from P<sup>A</sup>-P<sup>G</sup> distance tracks and visual inspection of the phosphate chain shape (<xref ref-type="fig" rid="fig4">Figure 4</xref>). Data for the &#945;&#946;&#947; coordination of the Mg-ATP complex and conformations with curled phosphate chain were calculated from simulations 1&#8211;4 in <xref ref-type="supplementary-material" rid="supp1">Supplementary file 1C</xref>; characterization of the &#946;&#947;-coordination was based on simulations 5&#8211;8 in <xref ref-type="supplementary-material" rid="supp1">Supplementary file 1C</xref>, see <xref ref-type="supplementary-material" rid="supp1">Supplementary file 1E</xref> for further details.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37373-v2.xml">elife-37373-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.37373.023</object-id><label>Table 3.</label><caption><title>Monovalent cation binding in crystal structures of P-loop NTPases.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Protein</th><th rowspan="2">PDB entry</th><th rowspan="2">Bound NTP analog</th><th colspan="3">Occupation of the AG site</th><th colspan="2">Phosphate chain shape</th></tr><tr><th>Cation</th><th>Distance to the closest O atom of P<sup>A</sup>, &#197;<sup>*</sup></th><th>Distance to the closest O atom of P<sup>G</sup>, &#197;<sup>*,&#8224;</sup></th><th>P<sup>A</sup>-P<sup>G</sup> distance, &#197;<sup>*</sup></th><th>P<sup>B</sup>-O<sup>3B</sup>-P<sup>G</sup> angle, degrees<sup>&#8224;</sup></th></tr></thead><tbody><tr><td colspan="8" valign="top"><bold>TRAFAC class NTPases</bold></td></tr><tr><td rowspan="3" valign="top">GTPase MnmE(TrmE)</td><td valign="top">2gj8</td><td valign="top">GDP AlF<sub>4</sub><sup>-</sup></td><td valign="top">K<sup>+</sup></td><td valign="top">2.8</td><td valign="top">2.6</td><td valign="top">5.4</td><td valign="top">136.3</td></tr><tr><td valign="top">2gja</td><td valign="top">GDP AlF<sub>4</sub><sup>-</sup></td><td valign="top">NH<sub>4</sub><sup>+</sup></td><td valign="top">2.9</td><td valign="top">2.5</td><td valign="top">5.4</td><td valign="top">136.9</td></tr><tr><td valign="top">2gj9</td><td valign="top">GDP AlF<sub>4</sub><sup>-</sup></td><td valign="top">Rb<sup>+</sup></td><td valign="top">2.9</td><td valign="top">2.8</td><td valign="top">5.5</td><td valign="top">131.6</td></tr><tr><td valign="top">GTPase FeoB</td><td valign="top">3ss8</td><td valign="top">GDP AlF<sub>4</sub><sup>-</sup></td><td valign="top">K<sup>+</sup></td><td valign="top">2.8</td><td valign="top">2.6</td><td valign="top">5.4</td><td valign="top">144.9</td></tr><tr><td rowspan="4" valign="top">Dynamin-like proteins</td><td valign="top">2x2e</td><td valign="top">GDP AlF<sub>4</sub><sup>-</sup></td><td valign="top">Na<sup>+</sup></td><td valign="top">4.0</td><td valign="top">2.5</td><td valign="top">5.3</td><td valign="top">131.2</td></tr><tr><td valign="top">2x2f</td><td valign="top">GDP AlF<sub>4</sub><sup>-</sup></td><td valign="top">Na<sup>+</sup></td><td valign="top">4.1</td><td valign="top">2.6</td><td valign="top">5.3</td><td valign="top">133.6</td></tr><tr><td valign="top">3w6p</td><td valign="top">GDP AlF<sub>4</sub><sup>-</sup></td><td valign="top">Na<sup>+</sup></td><td valign="top">4</td><td valign="top">2.4</td><td valign="top">5.5</td><td valign="top">135.3</td></tr><tr><td valign="top">3t34</td><td valign="top">GDP AlF<sub>4</sub><sup>-</sup></td><td valign="top">Na<sup>+</sup></td><td valign="top">3.8</td><td valign="top">2.4</td><td valign="top">5.6</td><td valign="top">149.3</td></tr><tr><td valign="top">GTPase Era</td><td valign="top">3r9w</td><td valign="top">GNP</td><td valign="top">H<sub>2</sub>O<sup>&#8225;</sup></td><td valign="top">3</td><td valign="top">3.4</td><td valign="top">5.1</td><td valign="top">129.2</td></tr><tr><td rowspan="4" valign="top">Eukaryotic translation initiation factor eIF5B</td><td valign="top">4ncn</td><td valign="top">GTP</td><td valign="top">Na<sup>+</sup></td><td valign="top">2.4</td><td valign="top">2.4</td><td valign="top">5.0</td><td valign="top">126.6</td></tr><tr><td valign="top">4tmv</td><td valign="top">GSP</td><td valign="top">Na<sup>+</sup></td><td valign="top">2.4</td><td valign="top">2.8 (S)<sup>&#167;</sup></td><td valign="top">4.9</td><td valign="top">126.3</td></tr><tr><td valign="top">4tmw</td><td valign="top">GTP</td><td valign="top">Na<sup>+</sup></td><td valign="top">2.4</td><td valign="top">2.4</td><td valign="top">4.9</td><td valign="top">125.9</td></tr><tr><td valign="top">4tmz</td><td valign="top">GSP</td><td valign="top">K<sup>+</sup></td><td valign="top">2.7</td><td valign="top">3.3 (S)<sup>&#167;</sup></td><td valign="top">4.9</td><td valign="top">122.1</td></tr><tr><td colspan="8" valign="top"><bold>RecA/F1-like class NTPases</bold></td></tr><tr><td rowspan="4" valign="top">DNA recombinase RadA</td><td valign="top">3ew9</td><td valign="top">ANP</td><td valign="top">K<sup>+</sup></td><td valign="top">6.2</td><td valign="top">3.3</td><td valign="top">5.1</td><td valign="top">124.5</td></tr><tr><td valign="top">2f1h</td><td valign="top">ANP</td><td valign="top">K<sup>+</sup></td><td valign="top">6.6</td><td valign="top">3.5</td><td valign="top">5.3</td><td valign="top">125.3</td></tr><tr><td valign="top">2fpm</td><td valign="top">ANP</td><td valign="top">K<sup>+</sup></td><td valign="top">5.9</td><td valign="top">2.6</td><td valign="top">5.1</td><td valign="top">124.2</td></tr><tr><td valign="top">1xu4</td><td valign="top">ANP</td><td valign="top">K<sup>+</sup></td><td valign="top">6.1</td><td valign="top">2.7</td><td valign="top">5.2</td><td valign="top">125.0</td></tr></tbody></table><table-wrap-foot><fn><p><sup>*</sup>The values were measured directly in the respective protein structures displayed in PyMOL.</p><p><sup>&#8224;</sup>&#160;If the &#947;-phosphate was replaced by an AlF<sub>4</sub><sup>-</sup> complex, the distance was measured to the closest F atom</p></fn><fn><p><sup>&#8225;</sup>&#160;While GTPase Era has been shown to be K<sup>+</sup>-dependent (<xref ref-type="bibr" rid="bib97">Rafay et al., 2012</xref>; <xref ref-type="bibr" rid="bib84">Meier et al., 2000</xref>), the crystallization solution contained no K<sup>+</sup>, only Na<sup>+</sup>, so that the likely cation-binding site is occupied by a water molecule, which forms hydrogen bonds with K<sup>+</sup> ligands.</p><p><sup>&#167;</sup>&#160;Non-hydrolyzable GTP analog GDP-monothiophosphate (GSP) contains a sulfur atom in the place of the O<sup>1G</sup> atom of &#947;-phosphate; this atom in involved in coordination of monovalent cations in respective structures.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37689-v1.xml">elife-37689-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.37689.014</object-id><label>Table 1.</label><caption><title>Values of R<sub>BH3-2A</sub> and R<sub>ABT-263</sub> for all reported interactions.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="bottom">Cell lines <break/>Constructs</th><th colspan="4" valign="bottom"><sup>c</sup>Bcl-XL</th><th colspan="4" valign="bottom"><sup>c</sup>Bcl-2</th></tr><tr><th colspan="2" valign="bottom">R<sub>BH3-2A</sub></th><th colspan="2" valign="bottom">R<sub>ABT-263</sub></th><th colspan="2" valign="bottom">R<sub>BH3-2A</sub></th><th colspan="2" valign="bottom">R<sub>ABT-263</sub></th></tr><tr><th valign="bottom">Value</th><th valign="bottom">95% CI</th><th valign="bottom">Value</th><th valign="bottom">95% CI</th><th valign="bottom">Value</th><th valign="bottom">95% CI</th><th valign="bottom">Value</th><th valign="bottom">95% CI</th></tr></thead><tbody><tr><td valign="bottom"><sup>v</sup>Bad</td><td valign="bottom">2.44</td><td valign="bottom">2.50</td><td valign="bottom">46.36</td><td valign="bottom">8.23</td><td valign="bottom">4.33</td><td valign="bottom">3.48</td><td valign="bottom">18.63</td><td valign="bottom">3.24</td></tr><tr><td valign="bottom"><sup>v</sup>tBid</td><td valign="bottom">33.70</td><td valign="bottom">2.60</td><td valign="bottom">43.95</td><td valign="bottom">6.84</td><td valign="bottom">25.33</td><td valign="bottom">3.98</td><td valign="bottom">30.16</td><td valign="bottom">9.82</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL</td><td valign="bottom">81.43</td><td valign="bottom">4.65</td><td valign="bottom">86.26</td><td valign="bottom">5.10</td><td valign="bottom">66.85</td><td valign="bottom">4.70</td><td valign="bottom">83.06</td><td valign="bottom">8.69</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim</td><td valign="bottom">70.19</td><td valign="bottom">4.03</td><td valign="bottom">97.96</td><td valign="bottom">6.45</td><td valign="bottom">54.91</td><td valign="bottom">3.92</td><td valign="bottom">65.46</td><td valign="bottom">7.40</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-dN</td><td valign="bottom">101.87</td><td valign="bottom">4.37</td><td valign="bottom">90.81</td><td valign="bottom">7.01</td><td valign="bottom">82.57</td><td valign="bottom">4.27</td><td valign="bottom">90.47</td><td valign="bottom">10.25</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-Bad</td><td valign="bottom">26.12</td><td valign="bottom">2.70</td><td valign="bottom">60.55</td><td valign="bottom">4.43</td><td valign="bottom">32.74</td><td valign="bottom">5.80</td><td valign="bottom">65.75</td><td valign="bottom">6.36</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-dCTS</td><td valign="bottom">15.08</td><td valign="bottom">3.17</td><td valign="bottom">69.05</td><td valign="bottom">4.57</td><td valign="bottom">13.60</td><td valign="bottom">2.21</td><td valign="bottom">60.02</td><td valign="bottom">6.73</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-Bad-dCTS</td><td valign="bottom">10.61</td><td valign="bottom">4.26</td><td valign="bottom">40.25</td><td valign="bottom">5.67</td><td valign="bottom">11.95</td><td valign="bottom">4.07</td><td valign="bottom">12.51</td><td valign="bottom">6.60</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim-WAA</td><td valign="bottom">ND</td><td valign="bottom">ND</td><td valign="bottom">78.51</td><td valign="bottom">5.81</td><td valign="bottom">ND</td><td valign="bottom">ND</td><td valign="bottom">50.15</td><td valign="bottom">7.22</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim-I146Y</td><td valign="bottom">46.56</td><td valign="bottom">3.20</td><td valign="bottom">83.24</td><td valign="bottom">5.34</td><td valign="bottom">26.78</td><td valign="bottom">4.21</td><td valign="bottom">53.17</td><td valign="bottom">5.43</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim-Q148R</td><td valign="bottom">74.45</td><td valign="bottom">4.24</td><td valign="bottom">89.82</td><td valign="bottom">6.21</td><td valign="bottom">56.94</td><td valign="bottom">5.46</td><td valign="bottom">53.46</td><td valign="bottom">7.33</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim-I153M</td><td valign="bottom">96.53</td><td valign="bottom">4.60</td><td valign="bottom">92.78</td><td valign="bottom">6.00</td><td valign="bottom">102.30</td><td valign="bottom">8.94</td><td valign="bottom">68.51</td><td valign="bottom">7.51</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim-I146Y-Q148R</td><td valign="bottom">50.46</td><td valign="bottom">3.46</td><td valign="bottom">90.18</td><td valign="bottom">7.59</td><td valign="bottom">37.73</td><td valign="bottom">3.88</td><td valign="bottom">61.77</td><td valign="bottom">7.54</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim-I146Y-I153M</td><td valign="bottom">56.80</td><td valign="bottom">3.27</td><td valign="bottom">88.97</td><td valign="bottom">6.12</td><td valign="bottom">42.59</td><td valign="bottom">4.33</td><td valign="bottom">69.07</td><td valign="bottom">7.73</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-I146Y</td><td valign="bottom">70.99</td><td valign="bottom">7.58</td><td valign="bottom">88.66</td><td valign="bottom">10.85</td><td valign="bottom">54.17</td><td valign="bottom">3.49</td><td valign="bottom">65.32</td><td valign="bottom">8.72</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-I146Y-dCTS</td><td valign="bottom">12.29</td><td valign="bottom">2.89</td><td valign="bottom">38.45</td><td valign="bottom">8.31</td><td valign="bottom">8.57</td><td valign="bottom">4.03</td><td valign="bottom">52.86</td><td valign="bottom">6.02</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185E</td><td valign="bottom">26.68</td><td valign="bottom">3.62</td><td valign="bottom">41.85</td><td valign="bottom">10.24</td><td valign="bottom">16.09</td><td valign="bottom">3.87</td><td valign="bottom">51.02</td><td valign="bottom">12.29</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-I146Y-L185E</td><td valign="bottom">19.44</td><td valign="bottom">4.10</td><td valign="bottom">40.01</td><td valign="bottom">8.07</td><td valign="bottom">5.79</td><td valign="bottom">9.81</td><td valign="bottom">36.85</td><td valign="bottom">8.81</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-CTS2A</td><td valign="bottom">77.99</td><td valign="bottom">4.43</td><td valign="bottom">88.79</td><td valign="bottom">6.24</td><td valign="bottom">60.24</td><td valign="bottom">3.77</td><td valign="bottom">80.69</td><td valign="bottom">11.40</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-I181E</td><td valign="bottom">49.19</td><td valign="bottom">50.23</td><td valign="bottom">63.86</td><td valign="bottom">6.16</td><td valign="bottom">47.20</td><td valign="bottom">2.96</td><td valign="bottom">73.26</td><td valign="bottom">10.45</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185E</td><td valign="bottom">26.68</td><td valign="bottom">3.62</td><td valign="bottom">41.85</td><td valign="bottom">10.24</td><td valign="bottom">16.09</td><td valign="bottom">3.87</td><td valign="bottom">51.02</td><td valign="bottom">12.29</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-I188E</td><td valign="bottom">67.17</td><td valign="bottom">5.14</td><td valign="bottom">50.96</td><td valign="bottom">7.66</td><td valign="bottom">47.47</td><td valign="bottom">3.36</td><td valign="bottom">63.04</td><td valign="bottom">8.04</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-V192E</td><td valign="bottom">52.57</td><td valign="bottom">53.59</td><td valign="bottom">70.14</td><td valign="bottom">7.64</td><td valign="bottom">54.50</td><td valign="bottom">3.56</td><td valign="bottom">64.84</td><td valign="bottom">9.70</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185A</td><td valign="bottom">159.10</td><td valign="bottom">14.88</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185D</td><td valign="bottom">20.31</td><td valign="bottom">5.29</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185E</td><td valign="bottom">26.16</td><td valign="bottom">3.57</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185F</td><td valign="bottom">99.99</td><td valign="bottom">5.15</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185G</td><td valign="bottom">86.82</td><td valign="bottom">7.91</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185H</td><td valign="bottom">49.10</td><td valign="bottom">4.50</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185M</td><td valign="bottom">89.98</td><td valign="bottom">4.42</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185P</td><td valign="bottom">58.95</td><td valign="bottom">3.61</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185R</td><td valign="bottom">48.08</td><td valign="bottom">3.33</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185S</td><td valign="bottom">73.49</td><td valign="bottom">4.67</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185stop</td><td valign="bottom">26.87</td><td valign="bottom">4.94</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">BimEL<sup>v</sup></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">89.41</td><td valign="bottom">5.80</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37689-v2.xml">elife-37689-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.37689.014</object-id><label>Table 1.</label><caption><title>Values of R<sub>BH3-2A</sub> and R<sub>ABT-263</sub> for all reported interactions.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="bottom">Cell lines <break/>Constructs</th><th colspan="4" valign="bottom"><sup>c</sup>Bcl-XL</th><th colspan="4" valign="bottom"><sup>c</sup>Bcl-2</th></tr><tr><th colspan="2" valign="bottom">R<sub>BH3-2A</sub></th><th colspan="2" valign="bottom">R<sub>ABT-263</sub></th><th colspan="2" valign="bottom">R<sub>BH3-2A</sub></th><th colspan="2" valign="bottom">R<sub>ABT-263</sub></th></tr><tr><th valign="bottom">Value</th><th valign="bottom">95% CI</th><th valign="bottom">Value</th><th valign="bottom">95% CI</th><th valign="bottom">Value</th><th valign="bottom">95% CI</th><th valign="bottom">Value</th><th valign="bottom">95% CI</th></tr></thead><tbody><tr><td valign="bottom"><sup>v</sup>Bad</td><td valign="bottom">2.44</td><td valign="bottom">2.50</td><td valign="bottom">46.36</td><td valign="bottom">8.23</td><td valign="bottom">4.33</td><td valign="bottom">3.48</td><td valign="bottom">18.63</td><td valign="bottom">3.24</td></tr><tr><td valign="bottom"><sup>v</sup>tBid</td><td valign="bottom">33.70</td><td valign="bottom">2.60</td><td valign="bottom">43.95</td><td valign="bottom">6.84</td><td valign="bottom">25.33</td><td valign="bottom">3.98</td><td valign="bottom">30.16</td><td valign="bottom">9.82</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL</td><td valign="bottom">81.43</td><td valign="bottom">4.65</td><td valign="bottom">86.26</td><td valign="bottom">5.10</td><td valign="bottom">66.85</td><td valign="bottom">4.70</td><td valign="bottom">83.06</td><td valign="bottom">8.69</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim</td><td valign="bottom">70.19</td><td valign="bottom">4.03</td><td valign="bottom">97.96</td><td valign="bottom">6.45</td><td valign="bottom">54.91</td><td valign="bottom">3.92</td><td valign="bottom">65.46</td><td valign="bottom">7.40</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-dN</td><td valign="bottom">101.87</td><td valign="bottom">4.37</td><td valign="bottom">90.81</td><td valign="bottom">7.01</td><td valign="bottom">82.57</td><td valign="bottom">4.27</td><td valign="bottom">90.47</td><td valign="bottom">10.25</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-Bad</td><td valign="bottom">26.12</td><td valign="bottom">2.70</td><td valign="bottom">60.55</td><td valign="bottom">4.43</td><td valign="bottom">32.74</td><td valign="bottom">5.80</td><td valign="bottom">65.75</td><td valign="bottom">6.36</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-dCTS</td><td valign="bottom">15.08</td><td valign="bottom">3.17</td><td valign="bottom">69.05</td><td valign="bottom">4.57</td><td valign="bottom">13.60</td><td valign="bottom">2.21</td><td valign="bottom">60.02</td><td valign="bottom">6.73</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-Bad-dCTS</td><td valign="bottom">10.61</td><td valign="bottom">4.26</td><td valign="bottom">40.25</td><td valign="bottom">5.67</td><td valign="bottom">11.95</td><td valign="bottom">4.07</td><td valign="bottom">12.51</td><td valign="bottom">6.60</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim-WAA</td><td valign="bottom">ND</td><td valign="bottom">ND</td><td valign="bottom">78.51</td><td valign="bottom">5.81</td><td valign="bottom">ND</td><td valign="bottom">ND</td><td valign="bottom">50.15</td><td valign="bottom">7.22</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim-I146Y</td><td valign="bottom">46.56</td><td valign="bottom">3.20</td><td valign="bottom">83.24</td><td valign="bottom">5.34</td><td valign="bottom">26.78</td><td valign="bottom">4.21</td><td valign="bottom">53.17</td><td valign="bottom">5.43</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim-Q148R</td><td valign="bottom">74.45</td><td valign="bottom">4.24</td><td valign="bottom">89.82</td><td valign="bottom">6.21</td><td valign="bottom">56.94</td><td valign="bottom">5.46</td><td valign="bottom">53.46</td><td valign="bottom">7.33</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim-I153M</td><td valign="bottom">96.53</td><td valign="bottom">4.60</td><td valign="bottom">92.78</td><td valign="bottom">6.00</td><td valign="bottom">102.30</td><td valign="bottom">8.94</td><td valign="bottom">68.51</td><td valign="bottom">7.51</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim-I146Y-Q148R</td><td valign="bottom">50.46</td><td valign="bottom">3.46</td><td valign="bottom">90.18</td><td valign="bottom">7.59</td><td valign="bottom">37.73</td><td valign="bottom">3.88</td><td valign="bottom">61.77</td><td valign="bottom">7.54</td></tr><tr><td valign="bottom"><sup>v</sup>Bad-Bim-I146Y-I153M</td><td valign="bottom">56.80</td><td valign="bottom">3.27</td><td valign="bottom">88.97</td><td valign="bottom">6.12</td><td valign="bottom">42.59</td><td valign="bottom">4.33</td><td valign="bottom">69.07</td><td valign="bottom">7.73</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-I146Y</td><td valign="bottom">70.99</td><td valign="bottom">7.58</td><td valign="bottom">88.66</td><td valign="bottom">10.85</td><td valign="bottom">54.17</td><td valign="bottom">3.49</td><td valign="bottom">65.32</td><td valign="bottom">8.72</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-I146Y-dCTS</td><td valign="bottom">12.29</td><td valign="bottom">2.89</td><td valign="bottom">38.45</td><td valign="bottom">8.31</td><td valign="bottom">8.57</td><td valign="bottom">4.03</td><td valign="bottom">52.86</td><td valign="bottom">6.02</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185E</td><td valign="bottom">26.68</td><td valign="bottom">3.62</td><td valign="bottom">41.85</td><td valign="bottom">10.24</td><td valign="bottom">16.09</td><td valign="bottom">3.87</td><td valign="bottom">51.02</td><td valign="bottom">12.29</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-I146Y-L185E</td><td valign="bottom">19.44</td><td valign="bottom">4.10</td><td valign="bottom">40.01</td><td valign="bottom">8.07</td><td valign="bottom">5.79</td><td valign="bottom">9.81</td><td valign="bottom">36.85</td><td valign="bottom">8.81</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-CTS2A</td><td valign="bottom">77.99</td><td valign="bottom">4.43</td><td valign="bottom">88.79</td><td valign="bottom">6.24</td><td valign="bottom">60.24</td><td valign="bottom">3.77</td><td valign="bottom">80.69</td><td valign="bottom">11.40</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-I181E</td><td valign="bottom">49.19</td><td valign="bottom">50.23</td><td valign="bottom">63.86</td><td valign="bottom">6.16</td><td valign="bottom">47.20</td><td valign="bottom">2.96</td><td valign="bottom">73.26</td><td valign="bottom">10.45</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185E</td><td valign="bottom">26.68</td><td valign="bottom">3.62</td><td valign="bottom">41.85</td><td valign="bottom">10.24</td><td valign="bottom">16.09</td><td valign="bottom">3.87</td><td valign="bottom">51.02</td><td valign="bottom">12.29</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-I188E</td><td valign="bottom">67.17</td><td valign="bottom">5.14</td><td valign="bottom">50.96</td><td valign="bottom">7.66</td><td valign="bottom">47.47</td><td valign="bottom">3.36</td><td valign="bottom">63.04</td><td valign="bottom">8.04</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-V192E</td><td valign="bottom">52.57</td><td valign="bottom">53.59</td><td valign="bottom">70.14</td><td valign="bottom">7.64</td><td valign="bottom">54.50</td><td valign="bottom">3.56</td><td valign="bottom">64.84</td><td valign="bottom">9.70</td></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185A</td><td valign="bottom">159.10</td><td valign="bottom">14.88</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185D</td><td valign="bottom">20.31</td><td valign="bottom">5.29</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185E</td><td valign="bottom">26.16</td><td valign="bottom">3.57</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185F</td><td valign="bottom">99.99</td><td valign="bottom">5.15</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185G</td><td valign="bottom">86.82</td><td valign="bottom">7.91</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185H</td><td valign="bottom">49.10</td><td valign="bottom">4.50</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185M</td><td valign="bottom">89.98</td><td valign="bottom">4.42</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185P</td><td valign="bottom">58.95</td><td valign="bottom">3.61</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185R</td><td valign="bottom">48.08</td><td valign="bottom">3.33</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185S</td><td valign="bottom">73.49</td><td valign="bottom">4.67</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom"><sup>v</sup>BimEL-L185stop</td><td valign="bottom">26.87</td><td valign="bottom">4.94</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">BimEL<sup>v</sup></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">89.41</td><td valign="bottom">5.80</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37813-v1.xml">elife-37813-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.37813.006</object-id><label>Table 1.</label><caption><title>Sub-strains generated from DS9 monomer isolated by SEC or cutoff filter.</title><p>Tau RD-YFP monomer (M<sub>s</sub>) was isolated from DS9 either by SEC or 100kD cutoff filter and inoculated into DS1 to create sub-strains. Multiple clones were isolated and characterized by morphology. Columns indicate the number of clones identified (n) and the percentage this represents of the total (%). A single sub-strain was observed regardless of purification method. Classification of cell morphology was performed using blinded analysis.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><bold>M<sub>s</sub></bold></th><th colspan="2" valign="top"><bold>SEC</bold></th><th colspan="2" valign="top"><bold>100kD filter</bold></th></tr><tr><th valign="top"><bold>N</bold></th><th valign="top"><bold>%</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>%</bold></th></tr></thead><tbody><tr><td valign="top">9.1</td><td valign="top">52</td><td valign="top">100</td><td valign="top">31</td><td valign="top">100</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37813-v1.xml">elife-37813-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.37813.007</object-id><label>Table 2.</label><caption><title>Sub-strains generated from DS10 monomer isolated by SEC or cutoff filter.</title><p>Tau RD-YFP monomer (M<sub>s</sub>) was isolated from DS10 by either by SEC or 100kD cutoff filter and inoculated into DS1 to create sub-strains. Multiple clones were isolated and characterized by morphology. Columns indicate the number of clones identified (n) and the percentage this represents of the total (%). Isolation of M<sub>s</sub> from DS10 by SEC or 100&#160;kD&#160;cutoff&#160;filter each enabled a similar proportion of sub-strains to form. Classification of cell morphology was performed using blinded analysis.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><bold>M<sub>s</sub></bold></th><th colspan="2" valign="top"><bold>SEC</bold></th><th colspan="2" valign="top"><bold>100kD filter</bold></th></tr><tr><th valign="top"><bold>N</bold></th><th valign="top"><bold>%</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>%</bold></th></tr></thead><tbody><tr><td valign="top">10.1</td><td valign="top">19</td><td valign="top">36</td><td valign="top">12</td><td valign="top">43</td></tr><tr><td valign="top">10.2</td><td valign="top">11</td><td valign="top">21</td><td valign="top">3</td><td valign="top">11</td></tr><tr><td valign="top">10.3</td><td valign="top">7</td><td valign="top">13</td><td valign="top">5</td><td valign="top">20</td></tr><tr><td valign="top">10.4</td><td valign="top">5</td><td valign="top">9</td><td valign="top">3</td><td valign="top">11</td></tr><tr><td valign="top">10.5 (sectored)</td><td valign="top">11</td><td valign="top">21</td><td valign="top">4</td><td valign="top">15</td></tr><tr><td valign="top"><bold>Total</bold></td><td valign="top"><bold>53</bold></td><td valign="top"><bold>100</bold></td><td valign="top"><bold>27</bold></td><td valign="top"><bold>100</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37813-v1.xml">elife-37813-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table5" position="float"><object-id pub-id-type="doi">10.7554/eLife.37813.013</object-id><label>Table 5.</label><caption><title>Sub-strains generated from AD monomer isolated by SEC or cutoff filter.</title><p>Tau monomer (M<sub>s</sub>) from AD brain was purified by immunoprecipitation followed by SEC or passage through a 100kD cutoff filter, prior to inoculation into DS1 cells. Columns indicate the number of clones identified (n) and the percentage this represents of the total (%). A single AD sub-strain was identified regardless of purification method. Classification of cell morphology was performed using blinded analysis.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><bold>M<sub>s</sub></bold></th><th colspan="2" valign="top">SEC</th><th colspan="2" valign="top">100kD filter</th></tr><tr><th valign="top"><bold>N</bold></th><th valign="top"><bold>%</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>%</bold></th></tr></thead><tbody><tr><td valign="top">AD(m)</td><td valign="top">47</td><td valign="top">100</td><td valign="top">23</td><td valign="top">100</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-37813-v1.xml">elife-37813-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table6" position="float"><object-id pub-id-type="doi">10.7554/eLife.37813.016</object-id><label>Table 6.</label><caption><title>Sub-strains generated from CBD monomer isolated by SEC or cutoff filter.</title><p>M<sub>s</sub> from CBD brain was purified by immunoprecipitation followed by SEC or passage through a 100kD cutoff filter, prior to inoculation into DS1 cells. CBD sub-strains, CBD1-3(m), were quantified. Isolation of M<sub>s</sub> from CBD brain by SEC or cutoff filter enabled a similar proportion of sub-strains to form. Columns indicate the number of clones identified (n) and the percentage this represents of the total (%). M<sub>s</sub> created similar strain patterns regardless of filtration method. Classification of cell morphology was performed using blinded analysis.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><bold>M<sub>s</sub></bold></th><th colspan="2" valign="top">SEC</th><th colspan="2" valign="top">100kD filter</th></tr><tr><th valign="top">N</th><th valign="top"><bold>%</bold></th><th valign="top"><bold>N</bold></th><th valign="top"><bold>%</bold></th></tr></thead><tbody><tr><td valign="top">CBD1(m)</td><td valign="top">20</td><td valign="top">36</td><td valign="top">4</td><td valign="top">22</td></tr><tr><td valign="top">CBD2(m)</td><td valign="top">18</td><td valign="top">33</td><td valign="top">7</td><td valign="top">39</td></tr><tr><td valign="top">CBD3(m)</td><td valign="top">17</td><td valign="top">31</td><td valign="top">7</td><td valign="top">39</td></tr><tr><td valign="top"><bold>Total</bold></td><td valign="top"><bold>55</bold></td><td valign="top"><bold>100</bold></td><td valign="top"><bold>18</bold></td><td valign="top"><bold>100</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-38131-v2.xml">elife-38131-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.38131.016</object-id><label>Table 1.</label><caption><title>Summary of the features of proteins identified in proteomic analysis of the <italic>Tg</italic>ApiCox25 complex.</title><p>Similarity searches were performed using HMMER (<ext-link ext-link-type="uri" xlink:href="https://www.ebi.ac.uk/Tools/hmmer/">https://www.ebi.ac.uk/Tools/hmmer/</ext-link>). The accession numbers listed were derived from <ext-link ext-link-type="uri" xlink:href="http://EuPathDB.org">http://EuPathDB.org</ext-link> (apicomplexan and chromerid species) or <ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov">www.ncbi.nlm.nih.gov</ext-link> (all others). Abbreviations: <italic>Plasmodium falciparum (Pf), Cryptosporidium parvum (Cp), Vitrella brassicaformis (Vb), Saccharomyces cerevisiae (Sc), Homo sapiens (Hs), Arabidopsis thaliana (At).</italic></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">ToxoDB gene ID (<ext-link ext-link-type="uri" xlink:href="http://toxodb.org">http://toxodb.org</ext-link>)</th><th rowspan="2">Protein <bold>annotation</bold></th><th rowspan="2">Predicted protein mass <break/>(kDa)</th><th rowspan="2">Mitochon-drial proteome <break/>(this study)</th><th rowspan="2">Phenotype score <break/>(<xref ref-type="bibr" rid="bib68">Sidik et al., 2016</xref>)</th><th colspan="6">Similarity search <break/>(E-value)</th></tr><tr><th>&#8195;<italic>Pf</italic></th><th><italic>Cp</italic></th><th><italic>Vb</italic> <break/>(Chromerid)</th><th><italic>Sc</italic> <break/>(Fungi)</th><th><italic>Hs</italic> <break/>(Animal)</th><th><italic>At</italic> <break/>(Plant)</th></tr></thead><tbody><tr><td>TGGT1_264040</td><td>Hypothetical protein <break/>(<bold><italic>Tg</italic>ApiCox25</bold>)</td><td>24.5</td><td>&#10003;</td><td>&#8722;2.54</td><td>Conserved unknown protein <break/>PF3D7_1464000.1 <break/>(2.4e<sup>&#8722;53</sup>)</td><td>x</td><td>Hypothetical Protein <break/>Vbra_12326 .t1 <break/>(5.2e<sup>&#8722;29</sup>)</td><td>x</td><td>x</td><td>x</td></tr><tr><td>TGGT1_265370</td><td>Hypothetical protein <break/>(<bold><italic>Tg</italic>ApiCox16</bold>)</td><td>16.0</td><td>&#10003;</td><td>1.56</td><td>x</td><td>x</td><td>x</td><td>x</td><td>x</td><td>x</td></tr><tr><td>TGGT1_209260</td><td>Putative cytochrome <italic>c</italic> oxidase subunit (<bold><italic>Tg</italic>Cox5b</bold>)</td><td>34.8</td><td>&#10003;</td><td>&#8722;3.07</td><td>Putative COX5B <break/>PF3D7_0927800.1 <break/>(4.3e<sup>&#8722;101</sup>)</td><td>x</td><td>COX5B-2 <break/>Vbra_9355 .t1 <break/>(1.6e<sup>&#8722;92</sup>)</td><td>Cox4p <break/>P04037 <break/>(0.32)</td><td>Cox5B <break/>NP_001853.2 <break/>(0.05)</td><td>COX5b <break/>At1g80230 <break/>(2.1e<sup>&#8722;08</sup>)</td></tr><tr><td>TGGT1_221510</td><td>Hypothetical protein <break/>(<bold><italic>Tg</italic>ApiCox18</bold>)</td><td>17.9</td><td>&#10003;</td><td>&#8722;3.28</td><td>Conserved unknown protein <break/>PF3D7_0523300.1 <break/>(1.5e<sup>&#8722;48</sup>)</td><td>x</td><td>Hypothetical Protein <break/>Vbra_21271 .t1 <break/>(5.2e<sup>&#8722;45</sup>)</td><td>x</td><td>x</td><td>x</td></tr><tr><td>TGGT1_262640</td><td>Cg8 family protein <break/>(<bold><italic>Tg</italic>ApiCox23</bold>)</td><td>23.8</td><td>&#10003;</td><td>&#8722;3.49</td><td>Cg8 protein <break/>PF3D7_0708700.1 <break/>(3.1e<sup>&#8722;64</sup>)</td><td>x</td><td>Hypothetical Protein <break/>Vbra_3012 .t1 <break/>(2.4e<sup>&#8722;53</sup>)</td><td>x</td><td>x</td><td>x</td></tr><tr><td>TGGT1_297810</td><td>Hypothetical protein <break/>(<bold><italic>Tg</italic>ApiCox30</bold>)</td><td>29.6</td><td>&#10003;</td><td>&#8722;3.64</td><td>Conserved unknown protein <break/>PF3D7_0915700.1 <break/>(1.2e<sup>&#8722;46</sup>)</td><td>x</td><td>Hypothetical Protein <break/>Vbra_17445 .t1 <break/>(6.7e<sup>&#8722;33</sup>)</td><td>x</td><td>x</td><td>x</td></tr><tr><td>TGGT1_247770</td><td>Hypothetical protein <break/>(<bold><italic>Tg</italic>ApiCox19</bold>)</td><td>19.2</td><td>&#10003;</td><td>&#8722;2.61</td><td>Conserved unknown protein <break/>PF3D7_1402200.1 <break/>(1.2e<sup>&#8722;34</sup>)</td><td>x</td><td>Hypothetical Protein <break/>Vbra_2065 .t1 <break/>(1.7e<sup>&#8722;27</sup>)</td><td>x</td><td>x</td><td>x</td></tr><tr><td>TGGT1_229920</td><td>Hypothetical protein <break/>(<bold><italic>Tg</italic>ApiCox35</bold>)</td><td>35.0</td><td>&#10003;</td><td>&#8722;3.84</td><td>Conserved unknown protein <break/>PF3D7_0306500.1 <break/>(1.5e<sup>&#8722;90</sup>)</td><td>x</td><td>Hypothetical Protein <break/>Vbra_6819 .t1 <break/>(1.6e<sup>&#8722;73</sup>)</td><td>x</td><td>x</td><td>x</td></tr><tr><td>TGGT1_306670</td><td>Hypothetical protein <break/>(<bold><italic>Tg</italic>ApiCox26</bold>)</td><td>25.8</td><td>&#10003;</td><td>&#8722;3.68</td><td>Conserved unknown protein <break/>PF3D7_1439600.1 <break/>(2.6e<sup>&#8722;43</sup>)</td><td>x</td><td>Hypothetical Protein <break/>Vbra_888 .t1 <break/>(1.2e<sup>&#8722;36</sup>)</td><td>x</td><td>x</td><td>x</td></tr><tr><td>TGGT1_226590</td><td>Putative cytochrome <italic>c</italic> oxidase subunit (<bold><italic>Tg</italic>Cox2a</bold>)</td><td>34.5</td><td>&#10003;</td><td>&#8722;3.80</td><td>Cytochrome c oxidase subunit 2 <break/>PF3D7_1361700.1 <break/>(4.9e<sup>&#8722;58</sup>)</td><td>x</td><td>Cytochrome c oxidase subunit 2 <break/>Vbra_8641 .t1 <break/>(3.6e<sup>&#8722;33</sup>)</td><td>Cox2 <break/>P00410 <break/>(2.6e<sup>&#8722;06</sup>)</td><td>Cox2 <break/>P00403 <break/>(0.0004)</td><td>Cox2 <break/>P93285 <break/>(3.3e<sup>&#8722;06</sup>)</td></tr><tr><td>TGGT1_310470</td><td>Putative cytochrome <italic>c</italic> oxidase subunit (<bold><italic>Tg</italic>Cox2b</bold>)</td><td>21.2</td><td>&#10003;</td><td>&#8722;4.18</td><td>Cytochrome c oxidase subunit 2 <break/>PF3D7_1430900.1 <break/>(7.6e<sup>&#8722;75</sup>)</td><td>x</td><td>Cytochrome c oxidase subunit 2 <break/>Vbra_14923 .t1 <break/>(4.2e<sup>&#8722;7</sup>)</td><td>Cox2 <break/>P00410 <break/>(4.3e<sup>&#8722;31</sup>)</td><td>Cox2 <break/>P00403 <break/>(9.2e<sup>&#8722;29</sup>)</td><td>Cox2 <break/>P93285 <break/>(3.8e<sup>&#8722;37</sup>)</td></tr><tr><td>TGGT1_286530</td><td>Hypothetical protein <break/>(<bold><italic>Tg</italic>ApiCox24</bold>)</td><td>25.4</td><td>&#10003;</td><td>&#8722;2.82</td><td>Conserved unknown protein <break/>PF3D7_1362000.1 <break/>(6.0e<sup>&#8722;45</sup>)</td><td>x</td><td>Hypothetical Protein <break/>Vbra_10089 .t1 <break/>(1.2e<sup>&#8722;11</sup>)</td><td>x</td><td>x</td><td>x</td></tr><tr><td>TGGT1_254030</td><td>Zinc finger CDGSH-type domain-containing protein <break/>(<bold><italic>Tg</italic>ApiCox13</bold>)</td><td>13.2</td><td>&#10003;</td><td>&#8722;4.26</td><td>CDGSH iron-sulfur domain-containing protein <break/>PF3D7_1022900.1 <break/>(7.8e<sup>&#8722;42</sup>)</td><td>x</td><td>CDGSH iron-sulfur domain-containing protein 3 <break/>Vbra_4701 .t1 <break/>(1.2e<sup>&#8722;44</sup>)</td><td>x</td><td>CDGSH iron-sulfur domain-containing protein 3 <break/>(2.7e<sup>&#8722;11</sup>)</td><td>x</td></tr><tr><td>TGGT1_242840</td><td>Membrane protein <break/>(<bold><italic>Tg</italic>ApiCox14</bold>)</td><td>13.9</td><td>&#10003;</td><td>&#8722;3.58</td><td>Conserved unknown protein <break/>PF3D7_1339400.1 <break/>(4.1e<sup>&#8722;16</sup>)</td><td>x</td><td>Hypothetical Protein <break/>Vbra_9996 .t1 <break/>(1.8e<sup>&#8722;9</sup>)</td><td>x</td><td>x</td><td>x</td></tr><tr><td>TGVEG_442760</td><td>Cytochrome C family oxidase subunit III <break/>(<bold><italic>Tg</italic>CoxIII</bold>)</td><td>16.8</td><td>-</td><td>N/A</td><td>Cytochrome c oxidase subunit 3 <break/>mal_mito_1 <break/>(5.2e<sup>&#8722;5</sup>)</td><td>x</td><td>x</td><td>x</td><td>x</td><td>x</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-38190-v2.xml">elife-38190-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.38190.005</object-id><label>Table 1.</label><caption><title>Analyzed animals.</title><p>The table shows the taxonomic classification with class, order and scientific name as reported in <ext-link ext-link-type="uri" xlink:href="http://tolweb.org/tree/">http://tolweb.org/tree/</ext-link>, <ext-link ext-link-type="uri" xlink:href="http://www.arkive.org/">http://www.arkive.org/</ext-link>, <ext-link ext-link-type="uri" xlink:href="https://animaldiversity.org/">https://animaldiversity.org/</ext-link>, and the English name. Animals are sorted in the following order: birds - anseriformes, charadriformes, ciconiformes, columbiformes, galliformes, gruiformes, phoenicopteriformes, struthioniformes - and mammals - artiodactyla, carnivora, dasyuromorphia, didelphimorphia, hyracoidea, perissodactyla, primates, proboscidae, rodentia, scandentia. The locations where the animals were recorded are labeled by capital letters: F - Falconara zoo (Falconara, Italy); N - Iacchelli Farm (Nemi, Italy); R - Zoo of Rome (Rome, Italy); J - Nara City (Nara, Japan); LF &#8211; Fischer laboratory (<xref ref-type="bibr" rid="bib29">Fischer et al., 2002</xref>); LA &#8211; Akay laboratory (<xref ref-type="bibr" rid="bib2">Akay et al., 2014</xref>); LR &#8211; our laboratory (Rome, Italy). Next, we identify each species with a serial number and the first three letters of the corresponding taxonomic order, we report the number of analyzed animals, the typical body weight of adults (data from the literature), the mean speed for the recorded strides, the approximate dimensionless speed (Froude number, Fr), and the total number of analyzed strides for hindlimbs and forelimbs. For humans, we recorded both overground walking in one subject and treadmill walking (at 5 km/hr) in five subjects (data pooled together). For mice, video recordings were used from a previously published study (Movie S1 in <xref ref-type="bibr" rid="bib2">Akay et al., 2014</xref>), while the kinematic data of the six mammals marked by asterisks are derived from <xref ref-type="bibr" rid="bib29">Fischer et al. (2002)</xref>. In the latter case, the number of averaged strides varied: at least 15, but up to 300 step cycles were averaged.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top">Order</th><th rowspan="2" valign="top">Animals</th><th rowspan="2" valign="top">Location</th><th rowspan="2" valign="top">Label</th><th rowspan="2" valign="top">N of <break/>animals</th><th rowspan="2" valign="top">Typical <break/>weight (kg)</th><th rowspan="2" valign="top"><italic>V</italic> <break/>(m/s)</th><th rowspan="2" valign="top"><italic>Fr</italic></th><th colspan="2" valign="top">N of strides</th></tr><tr><th valign="top">HL</th><th valign="top">FL</th></tr></thead><tbody><tr><td rowspan="10" valign="top">Birds</td><td valign="top">Anseriformes</td><td valign="top">Goose&#160;(<italic>Alopochen&#160;aegyptiaca</italic>)</td><td valign="top">F</td><td valign="top">1Ans</td><td valign="top">2</td><td valign="top">1.9</td><td valign="top">0.66</td><td valign="top">0.12</td><td valign="top">9</td><td valign="top">-</td></tr><tr><td valign="top">Charadriiformes</td><td valign="top">Avocet&#160;(<italic>Recurvirostra&#160;avosetta</italic>)</td><td valign="top">F</td><td valign="top">1Cha</td><td valign="top">6</td><td valign="top">0.3</td><td valign="top">0.43</td><td valign="top">0.06</td><td valign="top">8</td><td valign="top">-</td></tr><tr><td valign="top">Ciconiiformes</td><td valign="top">Ibis&#160;(<italic>Threskiornis&#160;aethiopicus</italic>)</td><td valign="top">F</td><td valign="top">1Cic</td><td valign="top">3</td><td valign="top">15</td><td valign="top">0.54</td><td valign="top">0.09</td><td valign="top">5</td><td valign="top">-</td></tr><tr><td valign="top">Columbiformes</td><td valign="top">Pigeon&#160;(<italic>Columba livia</italic>)</td><td valign="top">N</td><td valign="top">1Col</td><td valign="top">3</td><td valign="top">0.4</td><td valign="top">0.72</td><td valign="top">0.32</td><td valign="top">7</td><td valign="top">-</td></tr><tr><td rowspan="2" valign="top">Galliformes</td><td valign="top">Guinea&#160;fowl (<italic>Numida&#160;meleagris</italic>)</td><td valign="top">N</td><td valign="top">1 Gal</td><td valign="top">5</td><td valign="top">1.3</td><td valign="top">0.48</td><td valign="top">0.06</td><td valign="top">11</td><td valign="top">-</td></tr><tr><td valign="top">Peacock&#160;(<italic>Pavo cristatus</italic>)</td><td valign="top">R</td><td valign="top">2 Gal</td><td valign="top">4</td><td valign="top">4.4</td><td valign="top">0.60</td><td valign="top">0.06</td><td valign="top">11</td><td valign="top">-</td></tr><tr><td valign="top">Gruiformes</td><td valign="top">Crane&#160;(<italic>Balearica regulorum</italic>)</td><td valign="top">F</td><td valign="top">1Gru</td><td valign="top">2</td><td valign="top">3.5</td><td valign="top">0.39</td><td valign="top">0.04</td><td valign="top">5</td><td valign="top">-</td></tr><tr><td valign="top">Phoenicopteriformes</td><td valign="top">Flamingo&#160;(<italic>Phoenicopterus&#160;roseus</italic>)</td><td valign="top">F</td><td valign="top">1Pho</td><td valign="top">6</td><td valign="top">2.5</td><td valign="top">0.36</td><td valign="top">0.02</td><td valign="top">11</td><td valign="top">-</td></tr><tr><td rowspan="2" valign="top">Struthioniformes</td><td valign="top">Rhea&#160;(<italic>Rhea americana</italic>)</td><td valign="top">R</td><td valign="top">1Str</td><td valign="top">5</td><td valign="top">25</td><td valign="top">0.56</td><td valign="top">0.04</td><td valign="top">11</td><td valign="top">-</td></tr><tr><td valign="top">Ostrich&#160;(<italic>Struthio camelus</italic>)</td><td valign="top">F</td><td valign="top">2Str</td><td valign="top">6</td><td valign="top">110</td><td valign="top">0.57</td><td valign="top">0.03</td><td valign="top">10</td><td valign="top">-</td></tr><tr><td rowspan="13" valign="top">Mammals <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/><break/></td><td rowspan="13" valign="top">Artiodactyla</td><td valign="top">Addax&#160;(<italic>Addax nasomaculatus</italic>)</td><td valign="top">R</td><td valign="top">1Art</td><td valign="top">1</td><td valign="top">93</td><td valign="top">1.07</td><td valign="top">0.12</td><td valign="top">2</td><td valign="top">2</td></tr><tr><td valign="top">Ox&#160;(<italic>Bos taurus</italic>)</td><td valign="top">F</td><td valign="top">2Art</td><td valign="top">2</td><td valign="top">755</td><td valign="top">0.71</td><td valign="top">0.07</td><td valign="top">6</td><td valign="top">5</td></tr><tr><td valign="top">Camel&#160;(<italic>Camelus&#160;bactrianus</italic>)</td><td valign="top">F</td><td valign="top">3Art</td><td valign="top">2</td><td valign="top">475</td><td valign="top">1.08</td><td valign="top">0.08</td><td valign="top">10</td><td valign="top">7</td></tr><tr><td valign="top">Goat&#160;(<italic>Capra&#160;hircus)</italic></td><td valign="top">F</td><td valign="top">4Art</td><td valign="top">2</td><td valign="top">45</td><td valign="top">1.17</td><td valign="top">0.16</td><td valign="top">3</td><td valign="top">2</td></tr><tr><td valign="top">Waterbuck&#160;(<italic>Kobus&#160;ellipsiprymnus</italic>)</td><td valign="top">F</td><td valign="top">5Art</td><td valign="top">2</td><td valign="top">215</td><td valign="top">0.82</td><td valign="top">0.07</td><td valign="top">8</td><td valign="top">7</td></tr><tr><td valign="top">Deer&#160;(<italic>Dama&#160;dama</italic>)</td><td valign="top">F</td><td valign="top">6Art</td><td valign="top">4</td><td valign="top">56</td><td valign="top">0.95</td><td valign="top">0.12</td><td valign="top">10</td><td valign="top">7</td></tr><tr><td valign="top">Giraffe&#160;(<italic>Giraffa&#160;camelopardalis</italic>)</td><td valign="top">F,R</td><td valign="top">7Art</td><td valign="top">4</td><td valign="top">1560</td><td valign="top">1.21</td><td valign="top">0.06</td><td valign="top">9</td><td valign="top">10</td></tr><tr><td valign="top">Hippopotamus&#160;(<italic>Hippopotamus</italic> <break/><italic>amphibious</italic>)</td><td valign="top">F</td><td valign="top">8Art</td><td valign="top">1</td><td valign="top">2250</td><td valign="top">0.37</td><td valign="top">0.01</td><td valign="top">7</td><td valign="top">1</td></tr><tr><td valign="top">Llama&#160;(<italic>Lama&#160;glama</italic>)</td><td valign="top">F</td><td valign="top">9Art</td><td valign="top">3</td><td valign="top">143</td><td valign="top">0.53</td><td valign="top">0.04</td><td valign="top">5</td><td valign="top">5</td></tr><tr><td valign="top">Lechwe&#160;(<italic>Kobus&#160;megaceros</italic>)</td><td valign="top">R</td><td valign="top">10Art</td><td valign="top">4</td><td valign="top">175</td><td valign="top">0.79</td><td valign="top">0.07</td><td valign="top">7</td><td valign="top">5</td></tr><tr><td valign="top">Nyala&#160;(<italic>Tragelaphus&#160;angasii</italic>)</td><td valign="top">F</td><td valign="top">11Art</td><td valign="top">3</td><td valign="top">91</td><td valign="top">0.57</td><td valign="top">0.03</td><td valign="top">5</td><td valign="top">4</td></tr><tr><td valign="top">Oryx&#160;(<italic>Oryx&#160;dammah</italic>)</td><td valign="top">F</td><td valign="top">12Art</td><td valign="top">2</td><td valign="top">190</td><td valign="top">0.97</td><td valign="top">0.12</td><td valign="top">10</td><td valign="top">10</td></tr><tr><td valign="top">Sika deer&#160;(<italic>Cervus nippon</italic>)</td><td valign="top">J</td><td valign="top">13Art</td><td valign="top">3</td><td valign="top">63</td><td valign="top">0.78</td><td valign="top">0.09</td><td valign="top">6</td><td valign="top">6</td></tr><tr><td rowspan="6" valign="top"/><td rowspan="6" valign="top">Carnivora <break/> <break/> <break/> <break/><break/></td><td valign="top">Dog&#160;(<italic>Canis lupus&#160;familiaris</italic>)</td><td valign="top">R</td><td valign="top">1Car</td><td valign="top">6</td><td valign="top">35</td><td valign="top">1.06</td><td valign="top">0.24</td><td valign="top">35</td><td valign="top">36</td></tr><tr><td valign="top">Cat&#160;(<italic>Felis catus</italic>)</td><td valign="top">R</td><td valign="top">2Car</td><td valign="top">2</td><td valign="top">8</td><td valign="top">1.00</td><td valign="top">0.27</td><td valign="top">6</td><td valign="top">5</td></tr><tr><td valign="top">Cheetah&#160;(<italic>Acinonyx jubatus</italic>)</td><td valign="top">F</td><td valign="top">3Car</td><td valign="top">2</td><td valign="top">50</td><td valign="top">0.87</td><td valign="top">0.10</td><td valign="top">10</td><td valign="top">9</td></tr><tr><td valign="top">Lion&#160;(<italic>Panthera leo</italic>)</td><td valign="top">F</td><td valign="top">4Car</td><td valign="top">1</td><td valign="top">130</td><td valign="top">1.32</td><td valign="top">0.17</td><td valign="top">10</td><td valign="top">10</td></tr><tr><td valign="top">Lynx&#160;(<italic>Lynx lynx</italic>)</td><td valign="top">F</td><td valign="top">5Car</td><td valign="top">1</td><td valign="top">20</td><td valign="top">0.81</td><td valign="top">0.11</td><td valign="top">10</td><td valign="top">10</td></tr><tr><td valign="top">Wolf (<italic>Canis lupus</italic>)</td><td valign="top">F</td><td valign="top">6Car</td><td valign="top">1</td><td valign="top">40</td><td valign="top">1.05</td><td valign="top">0.20</td><td valign="top">6</td><td valign="top">6</td></tr><tr><td rowspan="18" valign="top"> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/> <break/><break/></td><td rowspan="4" valign="top"> <break/> <break/><break/></td><td valign="top">Ocelot&#160;(<italic>Leopardus&#160;pardalis</italic>)</td><td valign="top">F</td><td valign="top">7Car</td><td valign="top">1</td><td valign="top">13</td><td valign="top">0.78</td><td valign="top">0.15</td><td valign="top">9</td><td valign="top">9</td></tr><tr><td valign="top">Cougar&#160;(<italic>Puma concolor</italic>)</td><td valign="top">F</td><td valign="top">8Car</td><td valign="top">2</td><td valign="top">62</td><td valign="top">1.05</td><td valign="top">0.18</td><td valign="top">10</td><td valign="top">10</td></tr><tr><td valign="top">Suricate&#160;(<italic>Suricata suricatta</italic>)</td><td valign="top">F</td><td valign="top">9Car</td><td valign="top">6</td><td valign="top">0.7</td><td valign="top">0.63</td><td valign="top">0.29</td><td valign="top">10</td><td valign="top">8</td></tr><tr><td valign="top">Tiger&#160;(<italic>Panthera tigris)</italic></td><td valign="top">F</td><td valign="top">10Car</td><td valign="top">2</td><td valign="top">257</td><td valign="top">1.22</td><td valign="top">0.13</td><td valign="top">8</td><td valign="top">8</td></tr><tr><td valign="top">Dasyuromorphia</td><td valign="top">Kowari&#160;(<italic>Dasyuroides byrnei*</italic>)</td><td valign="top">LF</td><td valign="top">1Das</td><td valign="top">2</td><td valign="top">0.145</td><td valign="top">*</td><td valign="top">*</td><td valign="top">*</td><td valign="top">*</td></tr><tr><td valign="top">Didelphimorphia</td><td valign="top">Short-tailed&#160;opossum <break/>(<italic>Monodelphis domestica*</italic>)</td><td valign="top">LF</td><td valign="top">1Did</td><td valign="top">2</td><td valign="top">0.092</td><td valign="top">*</td><td valign="top">*</td><td valign="top">*</td><td valign="top">*</td></tr><tr><td valign="top">Hyracoidea</td><td valign="top">Rock&#160;hyrax(<italic>Procavia capensis*</italic>)</td><td valign="top">LF</td><td valign="top">1Hyr</td><td valign="top">2</td><td valign="top">1.2</td><td valign="top">*</td><td valign="top">*</td><td valign="top">*</td><td valign="top">*</td></tr><tr><td rowspan="4" valign="top">Perissodactyla</td><td valign="top">Donkey&#160;(<italic>Equus asinus</italic>)</td><td valign="top">F</td><td valign="top">1Per</td><td valign="top">2</td><td valign="top">203</td><td valign="top">0.81</td><td valign="top">0.07</td><td valign="top">16</td><td valign="top">12</td></tr><tr><td valign="top">Pony&#160;(<italic>Equus</italic>)</td><td valign="top">F</td><td valign="top">2Per</td><td valign="top">1</td><td valign="top">50</td><td valign="top">1.02</td><td valign="top">0.16</td><td valign="top">11</td><td valign="top">11</td></tr><tr><td valign="top">Tapir&#160;(<italic>Tapirus terrestris</italic>)</td><td valign="top">R</td><td valign="top">3Per</td><td valign="top">2</td><td valign="top">200</td><td valign="top">0.99</td><td valign="top">0.12</td><td valign="top">2</td><td valign="top">1</td></tr><tr><td valign="top">Zebra&#160;(<italic>Equus burchellii</italic>)</td><td valign="top">F</td><td valign="top">4Per</td><td valign="top">4</td><td valign="top">310</td><td valign="top">1.18</td><td valign="top">0.12</td><td valign="top">11</td><td valign="top">11</td></tr><tr><td rowspan="6" valign="top">Primates</td><td valign="top">Gibbon&#160;(<italic>Hylobates lar</italic>)</td><td valign="top">F</td><td valign="top">1Pri</td><td valign="top">3</td><td valign="top">5</td><td valign="top">1.03</td><td valign="top">0.23</td><td valign="top">6</td><td valign="top">-</td></tr><tr><td valign="top">Lemur&#160;catta&#160;(<italic>Lemur catta</italic>)</td><td valign="top">F,R</td><td valign="top">2Pri</td><td valign="top">7</td><td valign="top">3.9</td><td valign="top">0.66</td><td valign="top">0.14</td><td valign="top">10</td><td valign="top">3</td></tr><tr><td valign="top">Macaque&#160;(<italic>Macaca fuscata</italic>)</td><td valign="top">R</td><td valign="top">3Pri</td><td valign="top">2</td><td valign="top">9.8</td><td valign="top">0.86</td><td valign="top">0.14</td><td valign="top">2</td><td valign="top">0</td></tr><tr><td valign="top">Mandrill&#160;(<italic>Mandrillus sphinx</italic>)</td><td valign="top">R</td><td valign="top">4Pri</td><td valign="top">3</td><td valign="top">18</td><td valign="top">0.55</td><td valign="top">0.05</td><td valign="top">7</td><td valign="top">4</td></tr><tr><td valign="top">Spider&#160;monkey&#160;(<italic>Ateles fusciceps</italic>)</td><td valign="top">F</td><td valign="top">5Pri</td><td valign="top">3</td><td valign="top">9</td><td valign="top">0.75</td><td valign="top">0.13</td><td valign="top">3</td><td valign="top">2</td></tr><tr><td valign="top">Human&#160;(<italic>Homo sapiens</italic>)</td><td valign="top">LR</td><td valign="top">6Pri</td><td valign="top">6</td><td valign="top">68</td><td valign="top">1.46</td><td valign="top">0.27</td><td valign="top">67</td><td valign="top">-</td></tr><tr><td valign="top">Proboscidea</td><td valign="top">Elephant&#160;(<italic>Elephas maximus</italic>)</td><td valign="top">R</td><td valign="top">1Pro</td><td valign="top">1</td><td valign="top">4050</td><td valign="top">1.24</td><td valign="top">0.07</td><td valign="top">6</td><td valign="top">6</td></tr><tr><td rowspan="7" valign="top"/><td rowspan="6" valign="top">Rodentia</td><td valign="top">Capybara&#160;(<italic>Hydrochoerus&#160;hydrochaeris</italic>)</td><td valign="top">F</td><td valign="top">1Rod</td><td valign="top">4</td><td valign="top">51</td><td valign="top">0.68</td><td valign="top">0.10</td><td valign="top">5</td><td valign="top">3</td></tr><tr><td valign="top">Cavy&#160;(<italic>Galea&#160;musteloides*</italic>)</td><td valign="top">LF</td><td valign="top">2Rod</td><td valign="top">2</td><td valign="top">0.3</td><td valign="top">*</td><td valign="top">*</td><td valign="top">*</td><td valign="top">*</td></tr><tr><td valign="top">Porcupine&#160;(<italic>Hystrix cristata</italic>)</td><td valign="top">F</td><td valign="top">3Rod</td><td valign="top">2</td><td valign="top">20</td><td valign="top">1.17</td><td valign="top">0.50</td><td valign="top">4</td><td valign="top">3</td></tr><tr><td valign="top">Mara&#160;(<italic>Dolichotis patagonum</italic>)</td><td valign="top">F</td><td valign="top">4Rod</td><td valign="top">3</td><td valign="top">8.1</td><td valign="top">0.82</td><td valign="top">0.18</td><td valign="top">9</td><td valign="top">9</td></tr><tr><td valign="top">Rat&#160;(<italic>Rattus norvegicus*</italic>)</td><td valign="top">LF</td><td valign="top">5Rod</td><td valign="top">3</td><td valign="top">0.35</td><td valign="top">*</td><td valign="top">*</td><td valign="top">*</td><td valign="top">*</td></tr><tr><td valign="top">Mouse (<italic>Mus</italic>)</td><td valign="top">LA</td><td valign="top">6Rod</td><td valign="top">1</td><td valign="top">0.03</td><td valign="top">0.52</td><td valign="top">0.34</td><td valign="top">3</td><td valign="top">3</td></tr><tr><td valign="top">Scandentia</td><td valign="top">Common&#160;treeshrew&#160;(<italic>Tupaia glis*</italic>)</td><td valign="top">LF</td><td valign="top">1Sca</td><td valign="top">2</td><td valign="top">0.18</td><td valign="top">*</td><td valign="top">*</td><td valign="top">*</td><td valign="top">*</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-38358-v2.xml">elife-38358-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.38358.024</object-id><label>Table 2.</label><caption><title>Covariation between HLA allele charge and average CDR3 charge of HLA-associated TCRs for HLA positions frequently contacted by CDR3 amino acids in solved TCR:pMHC crystal structures.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">MHC&#160;Class</th><th rowspan="2" valign="top">Position<sup>*</sup></th><th rowspan="2" valign="top">Contact&#160;frequency<sup>&#8224;</sup></th><th colspan="2" valign="top">Full CDR3</th><th colspan="2" valign="top">Non-germline CDR3&#8225;</th><th rowspan="2" valign="top">AAs&#167;</th></tr><tr><th valign="top">R-value</th><th valign="top"><italic>p</italic>-value</th><th valign="top">R-value</th><th valign="top"><italic>p</italic>-value</th></tr></thead><tbody><tr><td valign="top">II-<inline-formula><mml:math id="inf151"><mml:mi>&#946;</mml:mi></mml:math></inline-formula></td><td valign="top">70</td><td valign="top">1.48</td><td valign="top">&#8722;0.47</td><td valign="top">3.3e-04</td><td valign="top">&#8722;0.52</td><td valign="top">6.1e-05</td><td valign="top">DEGQR</td></tr><tr><td valign="top">II-<inline-formula><mml:math id="inf152"><mml:mi>&#945;</mml:mi></mml:math></inline-formula></td><td valign="top">64</td><td valign="top">1.09</td><td valign="top">&#8722;0.15</td><td valign="top">0.33</td><td valign="top">&#8722;0.07</td><td valign="top">0.64</td><td valign="top">ART</td></tr><tr><td valign="top">I</td><td valign="top">152</td><td valign="top">0.47</td><td valign="top">0.00</td><td valign="top">0.99</td><td valign="top">&#8722;0.04</td><td valign="top">0.72</td><td valign="top">AERTVW</td></tr><tr><td valign="top">I</td><td valign="top">151</td><td valign="top">0.46</td><td valign="top">0.08</td><td valign="top">0.50</td><td valign="top">0.06</td><td valign="top">0.59</td><td valign="top">HR</td></tr><tr><td valign="top">I</td><td valign="top">69</td><td valign="top">0.26</td><td valign="top">&#8722;0.13</td><td valign="top">0.28</td><td valign="top">&#8722;0.14</td><td valign="top">0.24</td><td valign="top">ART</td></tr><tr><td valign="top">I</td><td valign="top">76</td><td valign="top">0.21</td><td valign="top">&#8722;0.08</td><td valign="top">0.49</td><td valign="top">&#8722;0.14</td><td valign="top">0.25</td><td valign="top">AEV</td></tr><tr><td valign="top">I</td><td valign="top">70</td><td valign="top">0.12</td><td valign="top">0.02</td><td valign="top">0.86</td><td valign="top">0.08</td><td valign="top">0.50</td><td valign="top">HKNQS</td></tr></tbody></table><table-wrap-foot><fn><p>*Only positions whose charge varies across alleles are included.</p><p>&#8224;Total number of CDR3 residues contacted (using a sidechain heavyatom distance threshold of 4.5 &#197;) divided by number of structures analyzed.</p></fn><fn><p>&#8225;CDR3 charge is calculated over amino acids with at least one&#160;non-germline coding nucleotide.</p><p>&#167;Amino acids present at this HLA position.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-39625-v1.xml">elife-39625-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.39625.016</object-id><label>Table 1.</label><caption><title>Summary of <italic>CrLFY</italic> RNAi transgenic lines and their phenotypic characterization.</title><p>Transgenic lines exhibited gametophytic developmental arrest and/or sporophyte shoot termination at varying stages of development. &#8216;+&#8217; indicates that a particular line was phenotypically normal at the developmental stage indicated, &#8216;&#8722;&#8217; indicates that development had arrested at or prior to this stage. In lines marked &#8216;+/-&#8217; the stage at which developmental defects occurred varied between individuals within the line, and at least some arrested individuals were identified at the stage indicated. The five <italic>ZmUbi<sub>pro</sub>::CrLFY1/2-i1</italic> lines shown were generated from three rounds of transformation, the pairs of lines B16 and B19 and D2 and D4 potentially arising from the same transformation event. The no hairpin control lines NHC-2 (F3) and NHC-3 (F4) may similarly have arisen from a single transformation event. In all other cases, each transgenic line arose from a separate round of transformation and so must represent independent T-DNA insertions.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">RNAi transgene</th><th align="center" rowspan="2" valign="bottom">Line</th><th align="center" rowspan="2" valign="bottom">Transfor-mation replicate</th><th align="center" colspan="4" valign="bottom">Gametophyte phase</th><th align="center" colspan="5" valign="bottom">Sporophyte phase</th></tr><tr><th align="center" valign="bottom">&#8195;Spore germin-ation</th><th align="center" valign="bottom">AC-based growth</th><th align="center" valign="bottom">Notch meristem-based growth</th><th align="center" valign="bottom">% arrested</th><th align="center" valign="bottom">Embryo</th><th align="center" valign="bottom">Shoot apex initiated</th><th align="center" valign="bottom">Simple frond</th><th align="center" valign="bottom">Complex frond</th><th align="center" valign="bottom">% arrested</th></tr></thead><tbody><tr><td valign="top"><italic>ZmUBI<sub>pro</sub>::CrLFY1/2-i1</italic></td><td align="center" valign="top">B16</td><td align="center" valign="top">1</td><td align="center" valign="top">+</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">99.86</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">&lt;5%</td></tr><tr><td valign="top"><italic>ZmUBI<sub>pro</sub>::CrLFY1/2-i1</italic></td><td align="center" valign="top">B19</td><td align="center" valign="top">1</td><td align="center" valign="top">+</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">50.00</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">-</td><td align="center" valign="top">&lt;5%</td></tr><tr><td valign="top"><italic>ZmUBI<sub>pro</sub>::CrLFY1/2-i1</italic></td><td align="center" valign="top">D13</td><td align="center" valign="top">2</td><td align="center" valign="top">+</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">99.80</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">&lt;5%</td></tr><tr><td valign="top"><italic>ZmUBI<sub>pro</sub>::CrLFY1/2-i1</italic></td><td align="center" valign="top">D2</td><td align="center" valign="top">3</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">0.00</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">&lt;5%</td></tr><tr><td valign="top"><italic>ZmUBI<sub>pro</sub>::CrLFY1/2-i1</italic></td><td align="center" valign="top">D4</td><td align="center" valign="top">3</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">0.00</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">&lt;5%</td></tr><tr><td valign="top"><italic>ZmUBI<sub>pro</sub>::CrLFY1/2-i2</italic></td><td align="center" valign="top">F9</td><td align="center" valign="top">4</td><td align="center" valign="top">+</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">0.00</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">&lt;5%</td></tr><tr><td valign="top"><italic>ZmUBI<sub>pro</sub>::CrLFY1/2-i2</italic></td><td align="center" valign="top">F14</td><td align="center" valign="top">5</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">100.00</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">0</td></tr><tr><td valign="top"><italic>ZmUBI<sub>pro</sub>::CrLFY1-i3</italic></td><td align="center" valign="top">E8</td><td align="center" valign="top">6</td><td align="center" valign="top">+</td><td align="center" valign="top">+/-</td><td align="center" valign="top">+/-</td><td align="center" valign="top">100.00</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">0</td></tr><tr><td valign="top"><italic>ZmUBI<sub>pro</sub>::CrLFY1-i3</italic></td><td align="center" valign="top">G13</td><td align="center" valign="top">7</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">0.00</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">&lt;5%</td></tr><tr><td valign="top"><italic>ZmUBI<sub>pro</sub>::CrLFY2-i4</italic></td><td align="center" valign="top">C3</td><td align="center" valign="top">8</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">0.00</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">-</td><td align="center" valign="top">-</td><td align="center" valign="top">&lt;5%</td></tr><tr><td valign="top">NHC-1 (control)</td><td align="center" valign="top">D20</td><td align="center" valign="top">9</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">0.00</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">0</td></tr><tr><td valign="top">NHC-2 (control)</td><td align="center" valign="top">F3</td><td align="center" valign="top">10</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">0.00</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">0</td></tr><tr><td valign="top">NHC-3 (control)</td><td align="center" valign="top">F4</td><td align="center" valign="top">10</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">0.00</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">+</td><td align="center" valign="top">0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-39813-v1.xml">elife-39813-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.39813.003</object-id><label>Table 1.</label><caption><title>Summary statistics of genomes and nuclei analyzed in this study. * Values from (<xref ref-type="bibr" rid="bib45">Ropars et al., 2016</xref>).</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom"/><th align="center" colspan="5" valign="bottom"><italic>Rhizophagus irregularis</italic></th><th rowspan="2" valign="bottom"><italic>Rhizophagus cerebriforme</italic></th><th rowspan="2" valign="bottom"><italic>Rhizophagus diaphanus</italic></th></tr><tr><th valign="bottom">SL1</th><th valign="bottom">A1</th><th valign="bottom">A4</th><th valign="bottom">A5</th><th valign="bottom">C2</th></tr></thead><tbody><tr><td valign="bottom"><bold>A. Genome Assembly Statistics</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">Assembly Coverage</td><td valign="bottom">136x</td><td valign="bottom">68x</td><td valign="bottom">95x</td><td valign="bottom">76x</td><td valign="bottom">96x</td><td valign="bottom">150x</td><td valign="bottom">120x</td></tr><tr><td valign="bottom">Number of Scaffolds</td><td valign="bottom">29,279</td><td valign="bottom">11,301<sup>*</sup></td><td valign="bottom">11,380<sup>*</sup></td><td valign="bottom">14,626<sup>*</sup></td><td valign="bottom">10,857<sup>*</sup></td><td valign="bottom">15,087</td><td valign="bottom">15,496</td></tr><tr><td valign="bottom">Assembly Size (Kb)</td><td valign="bottom">211,501</td><td valign="bottom">125,869<sup>*</sup></td><td valign="bottom">138,327<sup>*</sup></td><td valign="bottom">131,461<sup>*</sup></td><td valign="bottom">122,873<sup>*</sup></td><td valign="bottom">171,896</td><td valign="bottom">170,781</td></tr><tr><td valign="bottom">Assembly SNP/Kb</td><td valign="bottom">0.50</td><td valign="bottom">0.25</td><td valign="bottom">0.74</td><td valign="bottom">0.79</td><td valign="bottom">0.35</td><td valign="bottom">0.41</td><td valign="bottom">0.23</td></tr><tr><td valign="bottom"><bold>B. Single Nucleus Statistics</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">Number of Nuclei Analyzed</td><td valign="bottom">16.0</td><td valign="bottom">12.0</td><td valign="bottom">14.0</td><td valign="bottom">8.0</td><td valign="bottom">9.0</td><td valign="bottom">15.0</td><td valign="bottom">12.0</td></tr><tr><td valign="bottom">Average Assembly Coverage</td><td valign="bottom">14.08%</td><td valign="bottom">57.30%</td><td valign="bottom">46.84%</td><td valign="bottom">58.40%</td><td valign="bottom">61.42%</td><td valign="bottom">22.39%</td><td valign="bottom">11.45%</td></tr><tr><td valign="bottom">Average Position Depth</td><td valign="bottom">11.8</td><td valign="bottom">18.1</td><td valign="bottom">22.1</td><td valign="bottom">22.7</td><td valign="bottom">22.9</td><td valign="bottom">15.4</td><td valign="bottom">12.3</td></tr><tr><td colspan="2" valign="bottom">Number of SNPs Against Reference</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">-- Total SNPs</td><td valign="bottom">346,382</td><td valign="bottom">241,294</td><td valign="bottom">423,166</td><td valign="bottom">291,025</td><td valign="bottom">251,648</td><td valign="bottom">262,030</td><td valign="bottom">101,681</td></tr><tr><td valign="bottom">-- Average SNPs Per Nuclei - basic filtering</td><td valign="bottom">35,473</td><td valign="bottom">51,435</td><td valign="bottom">73,349</td><td valign="bottom">76,055</td><td valign="bottom">58,158</td><td valign="bottom">30,534</td><td valign="bottom">10,554</td></tr><tr><td valign="bottom">-- Average Divergence with reference - basic filtering</td><td valign="bottom">0.12%</td><td valign="bottom">0.07%</td><td valign="bottom">0.12%</td><td valign="bottom">0.10%</td><td valign="bottom">0.08%</td><td valign="bottom">0.08%</td><td valign="bottom">0.06%</td></tr><tr><td valign="bottom">Average - Inter-Nucleus Divergence - basic filtering</td><td valign="bottom">0.38%</td><td valign="bottom">0.13%</td><td valign="bottom">0.24%</td><td valign="bottom">0.16%</td><td valign="bottom">0.14%</td><td valign="bottom">0.24%</td><td valign="bottom">0.21%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-39813-v2.xml">elife-39813-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.39813.003</object-id><label>Table 1.</label><caption><title>Summary statistics of genomes and nuclei analyzed in this study. * Values from (<xref ref-type="bibr" rid="bib45">Ropars et al., 2016</xref>).</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom"/><th align="center" colspan="5" valign="bottom"><italic>Rhizophagus irregularis</italic></th><th rowspan="2" valign="bottom"><italic>Rhizophagus cerebriforme</italic></th><th rowspan="2" valign="bottom"><italic>Rhizophagus diaphanus</italic></th></tr><tr><th valign="bottom">SL1</th><th valign="bottom">A1</th><th valign="bottom">A4</th><th valign="bottom">A5</th><th valign="bottom">C2</th></tr></thead><tbody><tr><td valign="bottom"><bold>A. Genome Assembly Statistics</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">Assembly Coverage</td><td valign="bottom">136x</td><td valign="bottom">68x</td><td valign="bottom">95x</td><td valign="bottom">76x</td><td valign="bottom">96x</td><td valign="bottom">150x</td><td valign="bottom">120x</td></tr><tr><td valign="bottom">Number of Scaffolds</td><td valign="bottom">29,279</td><td valign="bottom">11,301<sup>*</sup></td><td valign="bottom">11,380<sup>*</sup></td><td valign="bottom">14,626<sup>*</sup></td><td valign="bottom">10,857<sup>*</sup></td><td valign="bottom">15,087</td><td valign="bottom">15,496</td></tr><tr><td valign="bottom">Assembly Size (Kb)</td><td valign="bottom">211,501</td><td valign="bottom">125,869<sup>*</sup></td><td valign="bottom">138,327<sup>*</sup></td><td valign="bottom">131,461<sup>*</sup></td><td valign="bottom">122,873<sup>*</sup></td><td valign="bottom">171,896</td><td valign="bottom">170,781</td></tr><tr><td valign="bottom">Assembly SNP/Kb</td><td valign="bottom">0.50</td><td valign="bottom">0.25</td><td valign="bottom">0.74</td><td valign="bottom">0.79</td><td valign="bottom">0.35</td><td valign="bottom">0.41</td><td valign="bottom">0.23</td></tr><tr><td valign="bottom"><bold>B. Single Nucleus Statistics</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">Number of Nuclei Analyzed</td><td valign="bottom">16.0</td><td valign="bottom">12.0</td><td valign="bottom">14.0</td><td valign="bottom">8.0</td><td valign="bottom">9.0</td><td valign="bottom">15.0</td><td valign="bottom">12.0</td></tr><tr><td valign="bottom">Average Assembly Coverage</td><td valign="bottom">14.08%</td><td valign="bottom">57.30%</td><td valign="bottom">46.84%</td><td valign="bottom">58.40%</td><td valign="bottom">61.42%</td><td valign="bottom">22.39%</td><td valign="bottom">11.45%</td></tr><tr><td valign="bottom">Average Position Depth</td><td valign="bottom">11.8</td><td valign="bottom">18.1</td><td valign="bottom">22.1</td><td valign="bottom">22.7</td><td valign="bottom">22.9</td><td valign="bottom">15.4</td><td valign="bottom">12.3</td></tr><tr><td colspan="2" valign="bottom">Number of SNPs Against Reference</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">-- Total SNPs</td><td valign="bottom">346,382</td><td valign="bottom">241,294</td><td valign="bottom">423,166</td><td valign="bottom">291,025</td><td valign="bottom">251,648</td><td valign="bottom">262,030</td><td valign="bottom">101,681</td></tr><tr><td valign="bottom">-- Average SNPs Per Nuclei - basic filtering</td><td valign="bottom">35,473</td><td valign="bottom">51,435</td><td valign="bottom">73,349</td><td valign="bottom">76,055</td><td valign="bottom">58,158</td><td valign="bottom">30,534</td><td valign="bottom">10,554</td></tr><tr><td valign="bottom">-- Average Divergence with reference - basic filtering</td><td valign="bottom">0.12%</td><td valign="bottom">0.07%</td><td valign="bottom">0.12%</td><td valign="bottom">0.10%</td><td valign="bottom">0.08%</td><td valign="bottom">0.08%</td><td valign="bottom">0.06%</td></tr><tr><td valign="bottom">Average - Inter-Nucleus Divergence - basic filtering</td><td valign="bottom">0.38%</td><td valign="bottom">0.13%</td><td valign="bottom">0.24%</td><td valign="bottom">0.16%</td><td valign="bottom">0.14%</td><td valign="bottom">0.24%</td><td valign="bottom">0.21%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-39813-v3.xml">elife-39813-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.39813.003</object-id><label>Table 1.</label><caption><title>Summary statistics of genomes and nuclei analyzed in this study. * Values from (<xref ref-type="bibr" rid="bib45">Ropars et al., 2016</xref>).</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom"/><th align="center" colspan="5" valign="bottom"><italic>Rhizophagus irregularis</italic></th><th rowspan="2" valign="bottom"><italic>Rhizophagus cerebriforme</italic></th><th rowspan="2" valign="bottom"><italic>Rhizophagus diaphanus</italic></th></tr><tr><th valign="bottom">SL1</th><th valign="bottom">A1</th><th valign="bottom">A4</th><th valign="bottom">A5</th><th valign="bottom">C2</th></tr></thead><tbody><tr><td valign="bottom"><bold>A. Genome Assembly Statistics</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">Assembly Coverage</td><td valign="bottom">136x</td><td valign="bottom">68x</td><td valign="bottom">95x</td><td valign="bottom">76x</td><td valign="bottom">96x</td><td valign="bottom">150x</td><td valign="bottom">120x</td></tr><tr><td valign="bottom">Number of Scaffolds</td><td valign="bottom">29,279</td><td valign="bottom">11,301<sup>*</sup></td><td valign="bottom">11,380<sup>*</sup></td><td valign="bottom">14,626<sup>*</sup></td><td valign="bottom">10,857<sup>*</sup></td><td valign="bottom">15,087</td><td valign="bottom">15,496</td></tr><tr><td valign="bottom">Assembly Size (Kb)</td><td valign="bottom">211,501</td><td valign="bottom">125,869<sup>*</sup></td><td valign="bottom">138,327<sup>*</sup></td><td valign="bottom">131,461<sup>*</sup></td><td valign="bottom">122,873<sup>*</sup></td><td valign="bottom">171,896</td><td valign="bottom">170,781</td></tr><tr><td valign="bottom">Assembly SNP/Kb</td><td valign="bottom">0.50</td><td valign="bottom">0.25</td><td valign="bottom">0.74</td><td valign="bottom">0.79</td><td valign="bottom">0.35</td><td valign="bottom">0.41</td><td valign="bottom">0.23</td></tr><tr><td valign="bottom"><bold>B. Single Nucleus Statistics</bold></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">Number of Nuclei Analyzed</td><td valign="bottom">16.0</td><td valign="bottom">12.0</td><td valign="bottom">14.0</td><td valign="bottom">8.0</td><td valign="bottom">9.0</td><td valign="bottom">15.0</td><td valign="bottom">12.0</td></tr><tr><td valign="bottom">Average Assembly Coverage</td><td valign="bottom">14.08%</td><td valign="bottom">57.30%</td><td valign="bottom">46.84%</td><td valign="bottom">58.40%</td><td valign="bottom">61.42%</td><td valign="bottom">22.39%</td><td valign="bottom">11.45%</td></tr><tr><td valign="bottom">Average Position Depth</td><td valign="bottom">11.8</td><td valign="bottom">18.1</td><td valign="bottom">22.1</td><td valign="bottom">22.7</td><td valign="bottom">22.9</td><td valign="bottom">15.4</td><td valign="bottom">12.3</td></tr><tr><td colspan="2" valign="bottom">Number of SNPs Against Reference</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">-- Total SNPs</td><td valign="bottom">346,382</td><td valign="bottom">241,294</td><td valign="bottom">423,166</td><td valign="bottom">291,025</td><td valign="bottom">251,648</td><td valign="bottom">262,030</td><td valign="bottom">101,681</td></tr><tr><td valign="bottom">-- Average SNPs Per Nuclei - basic filtering</td><td valign="bottom">35,473</td><td valign="bottom">51,435</td><td valign="bottom">73,349</td><td valign="bottom">76,055</td><td valign="bottom">58,158</td><td valign="bottom">30,534</td><td valign="bottom">10,554</td></tr><tr><td valign="bottom">-- Average Divergence with reference - basic filtering</td><td valign="bottom">0.12%</td><td valign="bottom">0.07%</td><td valign="bottom">0.12%</td><td valign="bottom">0.10%</td><td valign="bottom">0.08%</td><td valign="bottom">0.08%</td><td valign="bottom">0.06%</td></tr><tr><td valign="bottom">Average - Inter-Nucleus Divergence - basic filtering</td><td valign="bottom">0.38%</td><td valign="bottom">0.13%</td><td valign="bottom">0.24%</td><td valign="bottom">0.16%</td><td valign="bottom">0.14%</td><td valign="bottom">0.24%</td><td valign="bottom">0.21%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-40576-v2.xml">elife-40576-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Kinetics of GreB<sup>Cy3B</sup> interactions with RNAPs.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Experiment</th><th rowspan="2">Species</th><th colspan="3">Time to first binding measurement</th><th rowspan="2">Dwell time distribution fit parameters<sup>&#8224;</sup></th><th rowspan="2"><italic>K</italic><sub>D</sub> (nM)</th></tr><tr><th><italic>k</italic><sub>bkgnd</sub> <break/>(10<sup>7</sup> M<sup>&#8722;1</sup> s<sup>&#8722;1</sup>)</th><th><italic>k</italic><sub>on</sub> <break/>(10<sup>7</sup> M<sup>&#8722;1</sup> s<sup>&#8722;1</sup>)</th><th><italic>A</italic><sub>f</sub><sup>&#8225;</sup></th></tr></thead><tbody><tr><td>FCS (<xref ref-type="fig" rid="fig1">Figure 1</xref>)</td><td>RNAP core</td><td>N.D.</td><td>N.D.</td><td>N.D.</td><td>N.D.</td><td>10&#160;&#177;&#160;1</td></tr><tr><td>FCS (<xref ref-type="fig" rid="fig1">Figure 1</xref>)</td><td>&#963;<sup>70</sup>RNAP</td><td>N.D.</td><td>N.D.</td><td>N.D.</td><td>N.D.</td><td>10&#160;&#177;&#160;1</td></tr><tr><td>CoSMoS <break/>(<xref ref-type="fig" rid="fig1s2">Figure 1&#8212;figure supplement 2</xref>)</td><td>RNAP core</td><td>0.00023&#160;&#177;&#160;0.00009 <break/>(<italic>N&#160;=</italic>&#160;12)</td><td>0.94&#160;&#177;&#160;0.06 <break/>(<italic>N</italic>&#160;=&#160;163)</td><td>0.87&#160;&#177;&#160;0.13</td><td><italic>a</italic>&#160;=&#160;91 &#177; 4% <break/>&#964;<sub>1</sub> = 5.3 &#177; 0.5 <break/>&#964;<sub>2</sub> = 27 &#177; 3 s <break/>(<italic>N</italic>&#160;=&#160;872)</td><td>18 &#177; 2*</td></tr><tr><td>CoSMoS <break/>(<xref ref-type="fig" rid="fig1s2">Figure 1&#8212;figure supplement 2</xref>)</td><td>&#963;<sup>70</sup>RNAP</td><td>0.0017&#160;&#177;&#160;0.0008 <break/>(<italic>N&#160;=</italic>&#160;29)</td><td>1.5&#160;&#177;&#160;0.9 <break/>(<italic>N</italic>&#160;=&#160;152)</td><td>0.43&#160;&#177;&#160;0.11</td><td><italic>a</italic>&#160;=&#160;89 &#177; 7% <break/>&#964;<sub>1</sub> = 5.6 &#177; 0.6 <break/>&#964;<sub>2</sub> = 25 &#177; 7 <break/>(<italic>N</italic>&#160;=&#160;943)</td><td>11 &#177; 1*</td></tr><tr><td>CoSMoS<sup>&#167;</sup></td><td>EC</td><td>N.R.</td><td>2.7&#160;&#177;&#160;1.0</td><td>0.80&#160;&#177;&#160;0.06</td><td>&#964;&#160;=&#160;0.4&#160;&#177;&#160;0.1 s</td><td>93&#160;&#177;&#160;36</td></tr><tr><td>CoSMoS<sup>&#167;</sup></td><td>Static EC&#8722;6</td><td>N.R.</td><td>1.9&#160;&#177;&#160;0.1</td><td>1</td><td>&#964;&#160;=&#160;0.29&#160;&#177;&#160;0.02 s</td><td>184&#160;&#177;&#160;8</td></tr></tbody></table><table-wrap-foot><fn><p>*Calculated as <italic>K</italic><sub>D</sub> <italic>=</italic> (<italic>k</italic><sub>on</sub> &#964;<sub>1</sub>)<sup>&#8722;1</sup>.</p><p><sup>&#8224;</sup>From single- or bi-exponential fit.</p></fn><fn><p><sup>&#8225;</sup>Active fraction of surface-tethered molecules (<xref ref-type="bibr" rid="bib14">Friedman and Gelles, 2015</xref>).</p><p><sup>&#167;</sup>Data from <xref ref-type="bibr" rid="bib51">Tetone et al. (2017)</xref>.</p></fn><fn><p>N.D., not determined; N.R., not reported.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-40576-v2.xml">elife-40576-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><label>Table 3.</label><caption><title>Fit parameters for time to first RNAP binding (<xref ref-type="fig" rid="fig3">Figure 3E</xref> and <xref ref-type="fig" rid="fig3s3">Figure 3&#8212;figure supplement 3</xref>) and RNAP dwell time (<xref ref-type="fig" rid="fig3">Figure 3F</xref>) distributions on <italic>rrnB</italic> P1 DNA<sup>*</sup>.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Experimental conditions</th><th colspan="3">Time to first binding measurement</th><th rowspan="2">Dwell time measurement</th></tr><tr><th><italic>k</italic><sub>bkgnd</sub> <break/>(10<sup>6</sup> M<sup>&#8722;1</sup>s<sup>&#8722;1</sup>)</th><th><italic>k</italic><sub>a</sub> <break/>(10<sup>6</sup> M<sup>&#8722;1</sup>s<sup>&#8722;1</sup>)</th><th><italic>A</italic><sub>f</sub></th></tr></thead><tbody><tr><td>&#963;<sup>70</sup>RNAP<sup>647</sup> <break/>(<xref ref-type="fig" rid="fig3">Figure 3E</xref>)</td><td>0.037&#160;&#177;&#160;0.006 <break/>(<italic>N</italic>&#160;=&#160;34)</td><td>2.2&#160;&#177;&#160;0.6 <break/>(<italic>N</italic>&#160;=&#160;227)</td><td>0.95&#160;&#177;&#160;0.04</td><td><italic>a</italic>&#160;=&#160;91 &#177; 4% <break/>&#964;<sub>1</sub> = 2.6 &#177; 0.4 s <break/>&#964;<sub>2</sub> = 29.7 &#177; 0.2 s <break/>(<italic>N</italic>&#160;=&#160;2,576)</td></tr><tr><td>+10 nM GreB<sup>Cy3B</sup> <break/>(<xref ref-type="fig" rid="fig3">Figure 3E</xref>)</td><td>0.016&#160;&#177;&#160;0.002 <break/>(<italic>N</italic>&#160;=&#160;19)</td><td>2.0&#160;&#177;&#160;0.4 <break/>(<italic>N</italic>&#160;=&#160;269)</td><td>0.93&#160;&#177;&#160;0.06</td><td><italic>a</italic>&#160;=&#160;91 &#177; 6% <break/>&#964;<sub>1</sub> = 3.1 &#177; 0.7 s <break/>&#964;<sub>2</sub> = 42.2 &#177; 0.4 s <break/>(<italic>N&#160;=</italic>&#160;2,408)</td></tr><tr><td>+100 nM DksA <break/>(<xref ref-type="fig" rid="fig3">Figure 3E</xref>)</td><td>0.0093&#160;&#177;&#160;0.0005 <break/>(<italic>N</italic>&#160;=&#160;8)</td><td>1.9&#160;&#177;&#160;0.5 <break/>(<italic>N</italic>&#160;=&#160;187)</td><td>0.89&#160;&#177;&#160;0.05</td><td><italic>a</italic>&#160;=&#160;92 &#177; 7% <break/>&#964;<sub>1</sub> = 2.5 &#177; 0.4 s <break/>&#964;<sub>2</sub> = 29.9 &#177; 0.8 s <break/>(<italic>N&#160;=</italic>&#160;1,381)</td></tr><tr><td>+10 nM GreB<sup>Cy3B</sup>; &#963;<sup>70</sup>RNAP<sup>647</sup>&#183; GreB<sup>Cy3B</sup> subset<sup>&#8224;</sup> <break/>(<xref ref-type="fig" rid="fig3s3">Figure 3&#8212;figure supplement 3</xref>)</td><td>0.016&#160;&#177;&#160;0.002 <break/>(<italic>N</italic>&#160;=&#160;19)</td><td>2.4&#160;&#177;&#160;0.7 <break/>(<italic>N</italic>&#160;=&#160;124)</td><td>0.94&#160;&#177;&#160;0.05</td><td><italic>a</italic>&#160;=&#160;90 &#177; 5% <break/>&#964;<sub>1</sub> = 3.4 &#177; 0.8 s <break/>&#964;<sub>2</sub> = 44.6 &#177; 0.7 s <break/>(<italic>N</italic>&#160;=&#160;1,053)</td></tr><tr><td>+10 nM GreB<sup>Cy3B</sup>; &#963;<sup>70</sup>RNAP<sup>647</sup> subset<sup>&#8224;</sup> <break/>(<xref ref-type="fig" rid="fig3s3">Figure 3&#8212;figure supplement 3</xref>)</td><td>0.016&#160;&#177;&#160;0.002 <break/>(<italic>N</italic>&#160;=&#160;19)</td><td>1.8&#160;&#177;&#160;0.6 <break/>(<italic>N</italic>&#160;=&#160;145)</td><td>0.91&#160;&#177;&#160;0.08</td><td>N.D.</td></tr></tbody></table><table-wrap-foot><fn><p>*<xref ref-type="fig" rid="fig3">Figure 3E and F</xref> each show curves from an individual experimental replicate; fit parameters here represent pooled data from all three replicates.</p><p><sup>&#8224;</sup>Data in 10 nM GreB<sup>Cy3B</sup> experiment were divided into two disjoint subsets of RNAP binding events &#963;<sup>70</sup>RNAP<sup>647</sup> that arrived at DNA 1) without GreB<sup>Cy3B</sup> and 2) as a &#963;<sup>70</sup>RNAP<sup>647</sup>&#8226;GreB<sup>Cy3B</sup> complex (<xref ref-type="fig" rid="fig3s3">Figure 3&#8212;figure supplement 3</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-40580-v1.xml">elife-40580-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app5table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.40580.031</object-id><label>Appendix 5&#8212;table 2.</label><caption><title>Observed allele frequency ranges for different &#945;-thalassaemia forms.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Country</th><th colspan="3" valign="top">Allele frequency range (%)</th></tr><tr><th valign="top">&#945;<sup>0</sup>-thalassaemia</th><th valign="top">&#945;<sup>+</sup>-thalassaemia</th><th valign="top">&#945;<sup>ND</sup>-thalassaemia</th></tr></thead><tbody><tr><td valign="top">Brunei Darussalam</td><td valign="top">No surveys identified</td><td valign="top">No surveys identified</td><td valign="top">No surveys identified</td></tr><tr><td valign="top">Cambodia</td><td valign="top">0.80&#8211;1.10</td><td valign="top">10.30&#8211;26.30</td><td valign="top">2.44&#8211;4.20</td></tr><tr><td valign="top">Indonesia</td><td valign="top">No surveys identified</td><td valign="top">2.91</td><td valign="top">No surveys identified</td></tr><tr><td valign="top">Lao PDR</td><td valign="top">0.00&#8211;6.19</td><td valign="top">4.60&#8211;40.00</td><td valign="top">2.28&#8211;9.00</td></tr><tr><td valign="top">Malaysia</td><td valign="top">0.00&#8211;1.92</td><td valign="top">0.00&#8211;16.80</td><td valign="top">0.00&#8211;16.25</td></tr><tr><td valign="top">Myanmar</td><td valign="top">0.93</td><td valign="top">20.58</td><td valign="top">No surveys identified</td></tr><tr><td valign="top">Philippines</td><td valign="top">No surveys identified</td><td valign="top">No surveys identified</td><td valign="top">No surveys identified</td></tr><tr><td valign="top">Singapore</td><td valign="top">0.86&#8211;0.90</td><td valign="top">1.88&#8211;3.04</td><td valign="top">0.04</td></tr><tr><td valign="top">Thailand</td><td valign="top">0.00&#8211;9.29</td><td valign="top">2.98&#8211;21.43</td><td valign="top">0.00&#8211;7.30</td></tr><tr><td valign="top">Vietnam</td><td valign="top">0.00&#8211;2.66</td><td valign="top">1.59&#8211;14.4</td><td valign="top">2.07&#8211;14.43</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-40630-v1.xml">elife-40630-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.40630.010</object-id><label>Table 2.</label><caption><title>Quantile regression results summary for the roll-snap performance distribution of golden-collared and white-collared manakins, where the display length is evaluated as a predictor of display speed at different quantiles (&#964;) of the distribution.</title><p>A significantly negative slope (p&lt;0.05; values adjusted to control the false discovery rate) suggests that there is a trade-off between speed and endurance at a given quantile. Here we present representative models from every 10th quantile (&#964; interval&#160;=&#160;0.1), but we also characterized the performance distribution at a finer-grained scale (see&#160;<xref ref-type="fig" rid="fig3s1">Figure 3&#8212;figure supplement 1</xref>) at every whole quantile (&#964; interval&#160;=&#160;0.01) between 0.1 and 0.9. We did not extend the analysis below the 10th or above the 90th quantile because quantile regression performs poorly at extreme portions of the distribution for smaller datasets (n&#160;&lt;&#160;1000).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Quantile (&#964;)</th><th colspan="4" valign="top">Golden-collared manakin</th><th colspan="4" valign="top">White-collared manakin</th></tr><tr><th>Slope</th><th>s.e.</th><th>t-value</th><th>p-value</th><th>Slope</th><th>s.e.</th><th>t-value</th><th>p-value</th></tr></thead><tbody><tr><td>0.1</td><td>&#8722;0.13</td><td>0.11</td><td>&#8722;1.18</td><td>.239</td><td>&#8722;0.02</td><td>0.58</td><td>&#8722;0.03</td><td>.976</td></tr><tr><td>0.2</td><td>&#8722;0.20</td><td>0.07</td><td>&#8722;2.95</td><td>.004*</td><td>0.03</td><td>0.51</td><td>0.06</td><td>.976</td></tr><tr><td>0.3</td><td>&#8722;0.31</td><td>0.07</td><td>&#8722;4.13</td><td>.0001*</td><td>&#8722;0.04</td><td>0.31</td><td>&#8722;0.14</td><td>.976</td></tr><tr><td>0.4</td><td>&#8722;0.44</td><td>0.08</td><td>&#8722;5.27</td><td>&lt;0.0001*</td><td>&#8722;0.29</td><td>0.24</td><td>&#8722;1.24</td><td>.305</td></tr><tr><td>0.5</td><td>&#8722;0.53</td><td>0.08</td><td>&#8722;6.51</td><td>&lt;0.0001*</td><td>&#8722;0.20</td><td>0.23</td><td>&#8722;0.85</td><td>.512</td></tr><tr><td>0.6</td><td>&#8722;0.44</td><td>0.07</td><td>&#8722;6.39</td><td>&lt;0.0001*</td><td>&#8722;0.19</td><td>0.15</td><td>&#8722;1.27</td><td>.305</td></tr><tr><td>0.7</td><td>&#8722;0.58</td><td>0.03</td><td>&#8722;21.69</td><td>&lt;0.0001*</td><td>&#8722;0.11</td><td>0.16</td><td>&#8722;0.67</td><td>.606</td></tr><tr><td>0.8</td><td>&#8722;0.60</td><td>0.08</td><td>&#8722;7.91</td><td>&lt;0.0001*</td><td>&#8722;0.31</td><td>0.13</td><td>&#8722;2.33</td><td>.028*</td></tr><tr><td>0.9</td><td>&#8722;0.56</td><td>0.14</td><td>&#8722;4.03</td><td>.0001*</td><td>&#8722;0.41</td><td>0.15</td><td>&#8722;2.76</td><td>.020*</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-40630-v1.xml">elife-40630-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.40630.017</object-id><label>Table 3.</label><caption><title>Statistical summary for linear models assessing how percent relaxation changes over each successive twitch in stimulation trains of different frequencies.</title><p>After evaluating whether each species&#8217; SH exhibited a change in performance with repeated stimulations, we also compared the two slope estimates with an F-test. All p-values reported have been adjusted to control for the false discovery rate.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Stimulation hz</th><th colspan="3">Golden-collared manakin</th><th colspan="3">White-collared manakin</th><th colspan="3">Species comparison</th></tr><tr><th>Slope</th><th>F (1,6)</th><th>p-value</th><th>Slope</th><th>F (1,6)</th><th>p-value</th><th colspan="2">F (1,12)</th><th>p-value</th></tr></thead><tbody><tr><td>50</td><td>0.4002&#160;&#177;&#160;0.3983</td><td>1.01</td><td>.708</td><td>1.154&#160;&#177;&#160;0.6165</td><td>3.50</td><td>.586</td><td colspan="2">1.942</td><td>.189</td></tr><tr><td>55</td><td/><td/><td/><td>3.098&#160;&#177;&#160;1.301</td><td>5.67</td><td>.237</td><td colspan="2"/><td/></tr><tr><td>58</td><td/><td/><td/><td>3.287&#160;&#177;&#160;1.759</td><td>3.49</td><td>.586</td><td colspan="2"/><td/></tr><tr><td>60</td><td>&#8722;1.682&#160;&#177;&#160;0.8066</td><td>4.35</td><td>.246</td><td/><td/><td/><td colspan="2"/><td/></tr><tr><td>65</td><td/><td/><td/><td>2.7&#160;&#177;&#160;1.885</td><td>2.05</td><td>.767</td><td colspan="2"/><td/></tr><tr><td>70</td><td>&#8722;0.5629&#160;&#177;&#160;1.87</td><td>0.09</td><td>.774</td><td>1.705&#160;&#177;&#160;2.215</td><td>0.59</td><td>1.00</td><td colspan="2">1.44</td><td>.254</td></tr><tr><td>80</td><td>&#8722;5.268&#160;&#177;&#160;1.185</td><td>19.8</td><td>.017*</td><td>0.2476&#160;&#177;&#160;1.25</td><td>0.04</td><td>1.00</td><td colspan="2">18.7</td><td>.002**</td></tr><tr><td>90</td><td>&#8722;4.298&#160;&#177;&#160;0.8332</td><td>26.6</td><td>.013*</td><td>1.268&#160;&#177;&#160;1.382</td><td>0.84</td><td>1.00</td><td>33.9</td><td colspan="2">&lt;0.0001***</td></tr><tr><td>100</td><td>&#8722;8.949&#160;&#177;&#160;1.874</td><td>22.8</td><td>.016*</td><td/><td/><td/><td colspan="2"/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-40698-v1.xml">elife-40698-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.40698.005</object-id><label>Table 1.</label><caption><title>Payoff matrix of one-round IADC.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Role</th><th rowspan="3">Participant</th><th rowspan="3">Initial endowment <break/>(MU)</th><th rowspan="3">Individual contribution <break/>(<italic>I</italic><sub>i</sub>)</th><th rowspan="3">Group pool (G)</th><th colspan="3" valign="top">Payment</th></tr><tr><th valign="top">Attackers lose <break/>G<sub>Attacker&#160;</sub> <break/>&#160;&#8805;&#160;G<sub>Defender</sub></th><th colspan="2" valign="top">Attackers win <break/>G<sub>Attacker</sub>&#160;&gt;&#160;G<sub>Defender</sub></th></tr><tr><th valign="top">Leftover</th><th valign="top">Leftover</th><th valign="top">Spoil</th></tr></thead><tbody><tr><td rowspan="3" valign="top">Attack</td><td valign="top">Attacker-<italic>1</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Attacker-1</sub></td><td rowspan="3" valign="top"><p>G<sub>Attacker</sub></p></td><td valign="top"><italic>20</italic> &#8722; <italic>I</italic><sub>Attacker-1</sub></td><td valign="top"><italic>20</italic> &#8722;<inline-formula><mml:math id="inf1"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula> <italic>I</italic><sub>Attacker-1</sub></td><td valign="top">(60&#160;&#8722;&#160;<inline-formula><mml:math id="inf2"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula><italic>G</italic><sub>Defender</sub>)/3</td></tr><tr><td valign="top">Attacker-<italic>2</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Attacker-2</sub></td><td valign="top"><italic>20</italic> &#8722; <italic>I</italic><sub>Attacker-2</sub></td><td valign="top"><italic>20</italic> <inline-formula><mml:math id="inf3"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula>&#8722; <italic>I</italic><sub>Attacker-2</sub></td><td valign="top">(60&#160;&#8722;&#160;<inline-formula><mml:math id="inf4"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula><italic>G</italic><sub>Defender</sub>)/3</td></tr><tr><td valign="top">Attacker-<italic>3</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Attacker-3</sub></td><td valign="top"><italic>20</italic> &#8722;<inline-formula><mml:math id="inf5"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula> <italic>I</italic><sub>Attacker-3</sub></td><td valign="top"><italic>20</italic> <inline-formula><mml:math id="inf6"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula>&#8722; <italic>I</italic><sub>Attacker-3</sub></td><td valign="top">(60<inline-formula><mml:math id="inf7"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula>&#160;&#8722;&#160;<italic>G</italic><sub>Defender</sub>)/3</td></tr><tr><td rowspan="3" valign="top">Defend</td><td valign="top">Defender-<italic>1</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Defender-1</sub></td><td rowspan="3" valign="top"><p>G<sub>Defender</sub></p></td><td valign="top"><italic>20</italic> <inline-formula><mml:math id="inf8"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula>&#8722; <italic>I</italic><sub>Defender-1</sub></td><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top">Defender-<italic>2</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Defender-2</sub></td><td valign="top"><italic>20</italic> <inline-formula><mml:math id="inf9"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula>&#8722; <italic>I</italic><sub>Defender-2</sub></td><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top">Defender-<italic>3</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Defender-3</sub></td><td valign="top"><italic>20</italic> &#8722;<inline-formula><mml:math id="inf10"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula> <italic>I</italic><sub>Defender-3</sub></td><td valign="top">0</td><td valign="top">0</td></tr></tbody></table><table-wrap-foot><fn><p>Table note: For each round, each individual received an initial endowment of 20 MUs (Monetary Units).</p><p>Each individual decided the amount (<italic>I<sub>i</sub>&#160;</italic>, 0&#160;&#8804;&#160;<italic>I<sub>i</sub></italic>&#160;&#8804;&#160;20) to the group&#8217;s pool <italic>G</italic> (0&#160;&#8804; G &#8804; 60, G<italic><sub>Attacker</sub></italic>&#160;=&#160;<italic>I<sub>Attacker-1</sub>+I<sub>Attacker-2</sub>+I<sub>Attacker-3</sub></italic>, G<italic><sub>Defender</sub></italic>&#160;=&#160;<italic>I<sub>Defender-1</sub>+I<sub>Defender-2</sub>+I<sub>Defender-3</sub></italic>). When G<italic><sub>Attacker</sub></italic>&#160;&#8804;G<italic><sub>Defender</sub></italic>, attackers failed and defenders survived and all six individuals kept their remaining endowment (leftovers, 20 &#8211; <italic>I<sub>i</sub></italic>). When <italic>G<sub>Attacker</sub></italic>&#160;&gt;<italic>G<sub>Defender</sub></italic>, defenders failed and left with 0. The attackers won and took away defenders&#8217; remaining MU (spoils from winning, 60 &#8211; <italic>G<sub>Defender</sub></italic>), which were divided equally among attacker group members (each attacker: (60 &#8211; <italic>G<sub>Defender</sub></italic>)/3) and added to their remaining endowments (20 &#8211; <italic>I<sub>Attacker-i</sub></italic>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-40698-v2.xml">elife-40698-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.40698.005</object-id><label>Table 1.</label><caption><title>Payoff matrix of one-round IADC.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Role</th><th rowspan="3">Participant</th><th rowspan="3">Initial endowment <break/>(MU)</th><th rowspan="3">Individual contribution <break/>(<italic>I</italic><sub>i</sub>)</th><th rowspan="3">Group pool (G)</th><th colspan="3" valign="top">Payment</th></tr><tr><th valign="top">Attackers lose <break/>G<sub>Attacker&#160;</sub> <break/>&#160;&#8805;&#160;G<sub>Defender</sub></th><th colspan="2" valign="top">Attackers win <break/>G<sub>Attacker</sub>&#160;&gt;&#160;G<sub>Defender</sub></th></tr><tr><th valign="top">Leftover</th><th valign="top">Leftover</th><th valign="top">Spoil</th></tr></thead><tbody><tr><td rowspan="3" valign="top">Attack</td><td valign="top">Attacker-<italic>1</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Attacker-1</sub></td><td rowspan="3" valign="top"><p>G<sub>Attacker</sub></p></td><td valign="top"><italic>20</italic> &#8722; <italic>I</italic><sub>Attacker-1</sub></td><td valign="top"><italic>20</italic> &#8722;<inline-formula><mml:math id="inf1"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula> <italic>I</italic><sub>Attacker-1</sub></td><td valign="top">(60&#160;&#8722;&#160;<inline-formula><mml:math id="inf2"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula><italic>G</italic><sub>Defender</sub>)/3</td></tr><tr><td valign="top">Attacker-<italic>2</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Attacker-2</sub></td><td valign="top"><italic>20</italic> &#8722; <italic>I</italic><sub>Attacker-2</sub></td><td valign="top"><italic>20</italic> <inline-formula><mml:math id="inf3"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula>&#8722; <italic>I</italic><sub>Attacker-2</sub></td><td valign="top">(60&#160;&#8722;&#160;<inline-formula><mml:math id="inf4"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula><italic>G</italic><sub>Defender</sub>)/3</td></tr><tr><td valign="top">Attacker-<italic>3</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Attacker-3</sub></td><td valign="top"><italic>20</italic> &#8722;<inline-formula><mml:math id="inf5"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula> <italic>I</italic><sub>Attacker-3</sub></td><td valign="top"><italic>20</italic> <inline-formula><mml:math id="inf6"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula>&#8722; <italic>I</italic><sub>Attacker-3</sub></td><td valign="top">(60<inline-formula><mml:math id="inf7"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula>&#160;&#8722;&#160;<italic>G</italic><sub>Defender</sub>)/3</td></tr><tr><td rowspan="3" valign="top">Defend</td><td valign="top">Defender-<italic>1</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Defender-1</sub></td><td rowspan="3" valign="top"><p>G<sub>Defender</sub></p></td><td valign="top"><italic>20</italic> <inline-formula><mml:math id="inf8"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula>&#8722; <italic>I</italic><sub>Defender-1</sub></td><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top">Defender-<italic>2</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Defender-2</sub></td><td valign="top"><italic>20</italic> <inline-formula><mml:math id="inf9"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula>&#8722; <italic>I</italic><sub>Defender-2</sub></td><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top">Defender-<italic>3</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Defender-3</sub></td><td valign="top"><italic>20</italic> &#8722;<inline-formula><mml:math id="inf10"><mml:mo>&#8211;</mml:mo></mml:math></inline-formula> <italic>I</italic><sub>Defender-3</sub></td><td valign="top">0</td><td valign="top">0</td></tr></tbody></table><table-wrap-foot><fn><p>Table note: For each round, each individual received an initial endowment of 20 MUs (Monetary Units).</p><p>Each individual decided the amount (<italic>I<sub>i</sub>&#160;</italic>, 0&#160;&#8804;&#160;<italic>I<sub>i</sub></italic>&#160;&#8804;&#160;20) to the group&#8217;s pool <italic>G</italic> (0&#160;&#8804; G &#8804; 60, G<italic><sub>Attacker</sub></italic>&#160;=&#160;<italic>I<sub>Attacker-1</sub>+I<sub>Attacker-2</sub>+I<sub>Attacker-3</sub></italic>, G<italic><sub>Defender</sub></italic>&#160;=&#160;<italic>I<sub>Defender-1</sub>+I<sub>Defender-2</sub>+I<sub>Defender-3</sub></italic>). When G<italic><sub>Attacker</sub></italic>&#160;&#8804;G<italic><sub>Defender</sub></italic>, attackers failed and defenders survived and all six individuals kept their remaining endowment (leftovers, 20 &#8211; <italic>I<sub>i</sub></italic>). When <italic>G<sub>Attacker</sub></italic>&#160;&gt;<italic>G<sub>Defender</sub></italic>, defenders failed and left with 0. The attackers won and took away defenders&#8217; remaining MU (spoils from winning, 60 &#8211; <italic>G<sub>Defender</sub></italic>), which were divided equally among attacker group members (each attacker: (60 &#8211; <italic>G<sub>Defender</sub></italic>)/3) and added to their remaining endowments (20 &#8211; <italic>I<sub>Attacker-i</sub></italic>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-40698-v3.xml">elife-40698-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.40698.005</object-id><label>Table 1.</label><caption><title>Payoff matrix of one-round IADC.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Role</th><th rowspan="3">Participant</th><th rowspan="3">Initial endowment <break/>(MU)</th><th rowspan="3">Individual contribution <break/>(<italic>I</italic><sub>i</sub>)</th><th rowspan="3">Group pool (G)</th><th colspan="3" valign="top">Payment</th></tr><tr><th valign="top">Attackers lose <break/>G<sub>Attacker&#160;</sub> <break/>&#160;&#8804;&#160;G<sub>Defender</sub></th><th colspan="2" valign="top">Attackers win <break/>G<sub>Attacker</sub>&#160;&gt;&#160;G<sub>Defender</sub></th></tr><tr><th valign="top">Leftover</th><th valign="top">Leftover</th><th valign="top">Spoil</th></tr></thead><tbody><tr><td rowspan="3" valign="top">Attack</td><td valign="top">Attacker-<italic>1</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Attacker-1</sub></td><td rowspan="3" valign="top"><p>G<sub>Attacker</sub></p></td><td valign="top"><italic>20</italic> &#8722; <italic>I</italic><sub>Attacker-1</sub></td><td valign="top"><italic>20</italic> &#8722;&#160;<italic>I</italic><sub>Attacker-1</sub></td><td valign="top">(60&#160;&#8722;&#160;<italic>G</italic><sub>Defender</sub>)/3</td></tr><tr><td valign="top">Attacker-<italic>2</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Attacker-2</sub></td><td valign="top"><italic>20</italic> &#8722; <italic>I</italic><sub>Attacker-2</sub></td><td valign="top"><italic>20&#160;</italic>&#8722; <italic>I</italic><sub>Attacker-2</sub></td><td valign="top">(60&#160;&#8722;&#160;<italic>G</italic><sub>Defender</sub>)/3</td></tr><tr><td valign="top">Attacker-<italic>3</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Attacker-3</sub></td><td valign="top"><italic>20</italic> &#8722; <italic>I</italic><sub>Attacker-3</sub></td><td valign="top"><italic>20&#160;&#8722;&#160;I</italic><sub>Attacker-3</sub></td><td valign="top">(60&#160;&#8722;&#160;<italic>G</italic><sub>Defender</sub>)/3</td></tr><tr><td rowspan="3" valign="top">Defend</td><td valign="top">Defender-<italic>1</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Defender-1</sub></td><td rowspan="3" valign="top"><p>G<sub>Defender</sub></p></td><td valign="top"><italic>20</italic> &#8722; <italic>I</italic><sub>Defender-1</sub></td><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top">Defender-<italic>2</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Defender-2</sub></td><td valign="top"><italic>20</italic> &#8722; <italic>I</italic><sub>Defender-2</sub></td><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top">Defender-<italic>3</italic></td><td valign="top">20</td><td valign="top"><italic>I</italic><sub>Defender-3</sub></td><td valign="top"><italic>20</italic> &#8722; <italic>I</italic><sub>Defender-3</sub></td><td valign="top">0</td><td valign="top">0</td></tr></tbody></table><table-wrap-foot><fn><p>Table note: For each round, each individual received an initial endowment of 20 MUs (Monetary Units).</p><p>Each individual decided the amount (<italic>I<sub>i</sub>&#160;</italic>, 0&#160;&#8804;&#160;<italic>I<sub>i</sub></italic>&#160;&#8804;&#160;20) to the group&#8217;s pool <italic>G</italic> (0&#160;&#8804; G &#8804; 60, G<italic><sub>Attacker</sub></italic>&#160;=&#160;<italic>I<sub>Attacker-1</sub>+I<sub>Attacker-2</sub>+I<sub>Attacker-3</sub></italic>, G<italic><sub>Defender</sub></italic>&#160;=&#160;<italic>I<sub>Defender-1</sub>+I<sub>Defender-2</sub>+I<sub>Defender-3</sub></italic>). When G<italic><sub>Attacker</sub></italic>&#160;&#8804;G<italic><sub>Defender</sub></italic>, attackers failed and defenders survived and all six individuals kept their remaining endowment (leftovers, 20 &#8211; <italic>I<sub>i</sub></italic>). When <italic>G<sub>Attacker</sub></italic>&#160;&gt;<italic>G<sub>Defender</sub></italic>, defenders failed and left with 0. The attackers won and took away defenders&#8217; remaining MU (spoils from winning, 60 &#8211; <italic>G<sub>Defender</sub></italic>), which were divided equally among attacker group members (each attacker: (60 &#8211; <italic>G<sub>Defender</sub></italic>)/3) and added to their remaining endowments (20 &#8211; <italic>I<sub>Attacker-i</sub></italic>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-40815-v2.xml">elife-40815-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.40815.002</object-id><label>Table 1.</label><caption><title>Summary of lncRNA features and mutant phenotypes lncRNA names are shown in the first column.</title><p>lncRNAs were named using the last four digits of their corresponding ENSEMBL Transcript ID or their chromosome number if no transcript ID was available (e.g. lnc-1200 is located on chromosome 12). The second column represents ribosomal occupancy pattern along the length of lncRNAs in comparison to the 5&#8217;UTR, coding and 3&#8217;UTR of typical protein-coding transcripts (<xref ref-type="bibr" rid="bib9">Chew et al., 2013</xref>). The third column shows the transcript ID for the investigated lncRNA or its genomic coordinate in GRCz10. Column Four shows the deletion size. Fifth column represent the percentage decrease in the level of lncRNA in comparison to wild type from three biological replicates (qRT-PCR). The six and seven columns show the presence of embryonic phenotypes, viability and fertility (at least 15 adult pairs per allele) of homozygous mutant fish. Eighth and ninth column show the upstream and downstream neighboring genes in a 200 kb window centered around the lncRNA&#8217;s TSS. The last column provides the selection criteria for each lncRNA.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">lncRNA mutant, <break/>deletion type</th><th rowspan="2">Ribosome <break/>Profiling, <break/>class</th><th rowspan="2" valign="bottom">lncRNA <break/>transcript ID</th><th rowspan="2">Deletion <break/>size</th><th rowspan="2">Percent <break/>reduction</th><th rowspan="2">Embryonic <break/>phenotype</th><th rowspan="2">Viability and fertility</th><th colspan="2" valign="bottom">Neighboring <break/>genes</th><th rowspan="2" valign="bottom">Selection <break/>criteria</th></tr><tr><th valign="bottom">Up 100 Kb</th><th valign="bottom">Down 100 Kb</th></tr></thead><tbody><tr><td><italic>cyrano</italic><sup>a171</sup>, <break/>TSS-del.</td><td>Trailerlike</td><td>ENSDART00000139872</td><td>326 bp</td><td>98%</td><td>No</td><td>Yes</td><td><italic>tmem39b</italic></td><td><italic>oip5</italic></td><td>Syntenic and sequence <break/>conservation, <break/>Reported phenotype</td></tr><tr><td><italic>cyrano</italic><sup>a172</sup>, <break/>gene del.</td><td>Trailerlike</td><td>ENSDART00000139872</td><td>4374 bp</td><td>94%</td><td>No</td><td>Yes</td><td><italic>tmem39b</italic></td><td><italic>oip5</italic></td><td>Syntenic and sequence conservation, <break/>Reported phenotype</td></tr><tr><td><italic>gas5</italic><sup>a173</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>ENSDART00000156268</td><td>296 bp</td><td>100%</td><td>No</td><td>Yes</td><td><italic>osbpl9</italic></td><td><italic>tor3a</italic></td><td>Syntenic conservation, <break/>well studied lncRNA, <break/>host of several snoRNA</td></tr><tr><td><italic>lnc-setd1ba</italic><sup>a174</sup>, <break/>gene del.</td><td>Leaderlike</td><td>ENSDART00000141500</td><td>3137 bp</td><td>100%</td><td>No</td><td>Yes</td><td><italic>setd1ba</italic></td><td><italic>rhoF</italic></td><td>Syntenic and sequence conservation, <break/>Proximity to developmental regulatory genes</td></tr><tr><td><italic>squint</italic><sup>a175</sup>, <break/>gene del.</td><td>Coding</td><td>ENSDART0 <break/>0000079692</td><td>1032 bp</td><td>95%</td><td>No</td><td>Yes</td><td><italic>htr1ab</italic></td><td><italic>eif4ebp1</italic></td><td>Evolutionary conservation, <break/>Reported phenotype, <break/>putative cncRNA</td></tr><tr><td><italic>lnc-phox2bb</italic><sup>a176</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>ENSDART00000158002</td><td>652 bp</td><td>99%</td><td>No</td><td>Yes</td><td><italic>smntl1</italic></td><td><italic>phox2bb</italic></td><td>Syntenic conservation</td></tr><tr><td><italic>lnc-phox2bb</italic><sup>a177</sup>, <break/>gene del.</td><td>Leaderlike</td><td>ENSDART00000158002</td><td>9361 bp</td><td>87%</td><td>Yes</td><td>No</td><td><italic>smntl1</italic></td><td><italic>phox2bb</italic></td><td>Syntenic conservation</td></tr><tr><td><italic>lnc-3852</italic><sup>a178</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>ENSDART00000153852</td><td>447 bp</td><td>100%</td><td>No</td><td>Yes</td><td><italic>lima1a</italic></td><td><italic>hoxc1a</italic></td><td>Maternal expression, <break/>Proximity to developmental <break/>regulatory genes</td></tr><tr><td><italic>lnc-1562</italic><sup>a179</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>ENSDART00000131562</td><td>409 bp</td><td>90%</td><td>No</td><td>Yes</td><td><italic>*</italic></td><td><italic>fgf10a</italic></td><td>Maternal expression, <break/>Proximity to developmental <break/>regulatory genes</td></tr><tr><td><italic>lnc-3982</italic><sup>a180</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>ENSDART00000153982</td><td>352 bp</td><td>97%</td><td>No</td><td>Yes</td><td><italic>*</italic></td><td><italic>bmp2b</italic></td><td>Maternal expression, <break/>Proximity to developmental <break/>regulatory genes</td></tr><tr><td><italic>lnc-6269</italic><sup>a181</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>ENSDART00000156269</td><td>535 bp</td><td>99%</td><td>No</td><td>Yes</td><td><italic>tbx1</italic></td><td><italic>*</italic></td><td>Maternal expression, <break/>Proximity to developmental <break/>regulatory genes</td></tr><tr><td><italic>lnc-2154</italic><sup>a182</sup>, <break/>TSS-del.</td><td>Trailerlike</td><td>ENSDART00000132154</td><td>546 bp</td><td>100%</td><td>No</td><td>Yes</td><td><italic>rpz</italic></td><td><italic>nr2f5</italic></td><td>Maternal expression, <break/>Proximity to developmental <break/>regulatory genes</td></tr><tr><td><italic>lnc-1200</italic><sup>a183</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>Chr12:1708389-1925779:1</td><td>590 bp</td><td>95%</td><td>No</td><td>Yes</td><td><italic>*</italic></td><td><italic>zip11</italic></td><td>Maternal expression, <break/>Longest <break/>selected lncRNA</td></tr><tr><td><italic>lnc-1200</italic><sup>a184</sup>, <break/>gene del.</td><td>Leaderlike</td><td>Chr12:1708389-1925779:1</td><td>203.8 kb</td><td>84%</td><td>No</td><td>Yes</td><td><italic>*</italic></td><td><italic>zip11</italic></td><td>Maternal expression, <break/>Longest selected lncRNA</td></tr><tr><td><italic>lnc-2646</italic><sup>a185</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>ENSDART00 <break/>000152646</td><td>240 bp</td><td>97%</td><td>No</td><td>Yes</td><td><italic>*</italic></td><td><italic>dkk1b</italic></td><td>Proximity to developmental <break/>regulatory genes</td></tr><tr><td><italic>lnc-4468</italic><sup>a186</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>ENSDART00000154468</td><td>306 bp</td><td>100%</td><td>No</td><td>Yes</td><td><italic>fam169ab</italic></td><td><italic>lhx5</italic></td><td>Proximity to developmental <break/>regulatory genes, <break/>Low expression level</td></tr><tr><td><italic>lnc-0600</italic><sup>a187</sup>, <break/>TSS-del.</td><td>Trailerlike</td><td>Chr6:59414652-59443141:1</td><td>244 bp</td><td>95%</td><td>No</td><td>Yes</td><td><italic>*</italic></td><td><italic>gli1</italic></td><td>Proximity to developmental regulatory <break/>genes, <break/>Low expression level</td></tr><tr><td><italic>lnc-0900</italic><sup>a188</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>Chr9:6684669-6691350:1</td><td>377 bp</td><td>83%</td><td>No</td><td>Yes</td><td><italic>pou3f3a</italic></td><td><italic>*</italic></td><td>Syntenic conservation, <break/>Low expression <break/>level</td></tr><tr><td><italic>lnc-8507</italic><sup>a189</sup>, <break/>mTSS-del.</td><td>Leaderlike</td><td>ENSDART00000158507</td><td>323 bp</td><td>81%</td><td>No</td><td>Yes</td><td><italic>npvf</italic></td><td><italic>hoxa1a</italic></td><td>Proximity to Hox genes, <break/>Maternal and Zygotic <break/>promoters</td></tr><tr><td><italic>lnc-8507</italic><sup>a190</sup>, <break/>mzTSS-del.</td><td>Leaderlike</td><td>ENSDART00000158507</td><td>9773 bp</td><td>95%</td><td>No</td><td>Yes</td><td><italic>npvf</italic></td><td><italic>hoxa1a</italic></td><td>Proximity to Hox genes, <break/>Maternal and Zygotic <break/>promoters</td></tr><tr><td><italic>lnc-7620</italic><sup>a191</sup>, <break/>TSS-del.</td><td>Trailerlike</td><td>ENSDART00000137620</td><td>668 bp</td><td>99%</td><td>No</td><td>Yes</td><td><italic>gal3st1b</italic></td><td><italic>srsf9</italic></td><td>Syntenic and sequence conservation, <break/>Implicated in adult fish <break/>and mouse behavior. <break/>Bitetti, A., et al. (2018)</td></tr><tr><td><italic>lnc-1300</italic><sup>a192</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>Chr13:4535992-4538275:1</td><td>367 bp</td><td>92%</td><td>No</td><td>Yes</td><td><italic>c1d</italic></td><td><italic>pla2g12b</italic></td><td>Syntenic and sequence conservation, <break/>High expression <break/>level</td></tr><tr><td><italic>lnc-7118</italic><sup>a193</sup>, <break/>TSS-del.</td><td>Trailerlike</td><td>ENSDART00000157118</td><td>438 bp</td><td>82%</td><td>No</td><td>Yes</td><td><italic>mrps9</italic></td><td><italic>pou3f3b</italic></td><td>Syntenic conservation <break/></td></tr><tr><td><italic>lnc-5888</italic><sup>a194</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>ENSDART00000155888</td><td>606 bp</td><td>96%</td><td>No</td><td>Yes</td><td><italic>glrx5</italic></td><td><italic>zgc:100997</italic></td><td>Syntenic conservation, <break/>scaRNA13 host gene, <break/>shortest selected lncRNA</td></tr><tr><td><italic>lnc-6913</italic><sup>a195</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>ENSDART00000156913</td><td>333 bp</td><td>72%</td><td>No</td><td>Yes</td><td><italic>usp20</italic></td><td><italic>ptges</italic></td><td>Proximity <break/>to developmental regulatory <break/>genes</td></tr><tr><td><italic>lnc-6913</italic><sup>a196</sup>, <break/>gene del.</td><td>Leaderlike</td><td>ENSDART00000156913</td><td>5568 bp</td><td>93%</td><td>No</td><td>Yes</td><td><italic>usp20</italic></td><td><italic>ptges</italic></td><td>Proximity <break/>to developmental regulatory <break/>genes</td></tr><tr><td><italic>lnc-1666</italic><sup>a197</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>ENSDART00000141666</td><td>544 bp</td><td>96%</td><td>No</td><td>Yes</td><td><italic>ptf1a</italic></td><td><italic>*</italic></td><td>Proximity to developmental <break/>regulatory genes, Restricted <break/>late expression</td></tr><tr><td><italic>lnc-6490</italic><sup>a198</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>ENSDART00000146490</td><td>607 bp</td><td>99%</td><td>No</td><td>Yes</td><td><italic>nr2f2</italic></td><td><italic>*</italic></td><td>Syntenic conservation, <break/>Restricted late expression <break/></td></tr><tr><td><italic>lnc-6490</italic><sup>a199</sup>, <break/>gene del.</td><td>Leaderlike</td><td>ENSDART00000146490</td><td>8378 bp</td><td>100%</td><td>No</td><td>Yes</td><td><italic>nr2f2</italic></td><td><italic>*</italic></td><td>Syntenic conservation, <break/>Restricted <break/>late expression</td></tr><tr><td><italic>lnc-0464</italic><sup>a200</sup>, <break/>TSS-del.</td><td>Trailerlike</td><td>ENSDART00000140464</td><td>597 bp</td><td>96%</td><td>No</td><td>Yes</td><td><italic>nr2f1a</italic></td><td><italic>*</italic></td><td>Restricted late expression <break/>pattern</td></tr><tr><td><italic>lnc-4149</italic><sup>a201</sup>, <break/>TSS-del.</td><td>Leaderlike</td><td>ENSDART00000154149</td><td>491 bp</td><td>98%</td><td>No</td><td>Yes</td><td><italic>bhlhe22</italic></td><td><italic>*</italic></td><td>Proximity to developmental <break/>regulatory genes</td></tr><tr><td><italic>lnc-4149</italic><sup>a202</sup>, <break/>gene del.</td><td>Leaderlike</td><td>ENSDART00000154149</td><td>35.11 kb</td><td>100%</td><td>No</td><td>Yes</td><td><italic>bhlhe22</italic></td><td><italic>*</italic></td><td>Proximity to developmental <break/>regulatory genes</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-40946-v1.xml">elife-40946-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.40946.021</object-id><label>Table 4.</label><caption><title>Number of training and test dataset, and performance evaluation of machine learning models (related to <xref ref-type="fig" rid="fig2">Figure 2</xref>).</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Model</th><th colspan="2">Training</th><th colspan="2">Test</th><th rowspan="2">Recall</th><th rowspan="2">Precision</th><th rowspan="2">F score</th></tr><tr><th>Total (n)</th><th>Positive labels (n)</th><th>Total (n)</th><th>Positive labels (n)</th></tr></thead><tbody><tr><td>IHC* 1</td><td>607,954</td><td>5906</td><td>578,851</td><td>5741</td><td>0.961</td><td>0.941</td><td>0.951</td></tr><tr><td>IHC* 2</td><td>37,576</td><td>11,977</td><td>18,104</td><td>5753</td><td>0.977</td><td>0.986</td><td>0.981</td></tr><tr><td>OHC<sup><bold>&#8224;</bold></sup>&#160;1</td><td>1,112,659</td><td>20,576</td><td>1,099,519</td><td>19,959</td><td>0.978</td><td>0.914</td><td>0.945</td></tr><tr><td>OHC<sup><bold>&#8224;</bold></sup>&#160;2</td><td>28,702</td><td>20,576</td><td>27,185</td><td>19,959</td><td>0.959</td><td>0.979</td><td>0.969</td></tr><tr><td>OHC<sup><bold>&#8224;</bold></sup>&#160;3</td><td>20,416</td><td>Row1: 6706 <break/>Row2: 6745 <break/>Row3: 6965</td><td>19,594</td><td>Row1: 6421 <break/>Row2: 6450 <break/>Row3: 6723</td><td>0.993<sup><bold>&#8225;</bold></sup></td><td>0.993<sup><bold>&#8225;</bold></sup></td><td>0.993<sup><bold>&#8225;</bold></sup></td></tr><tr><td>OHC<sup><bold>&#8224;</bold></sup>&#160;4</td><td>4114</td><td>1365</td><td>2990</td><td>905</td><td>0.920</td><td>0.946</td><td>0.933</td></tr></tbody></table><table-wrap-foot><fn><p>*. IHC, inner hair cell.</p><p><bold>&#8224;</bold>. OHC, outer hair cell.</p></fn><fn><p><bold>&#8225;</bold>. Calculated by micro-average of recall and precision (Sokolova M and Lapalme G, 2009)</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-41152-v2.xml">elife-41152-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.41152.006</object-id><label>Table 1.</label><caption><title>Cue-Evoked and Non-Evoked neurons.</title></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2" valign="middle">Brain region</th><th align="center" colspan="5" valign="middle">Cue-Evoked responses</th><th align="center" colspan="2" valign="middle">Non-Evoked</th></tr><tr><th align="center" valign="middle">Phasic</th><th align="center" valign="middle">Inactive</th><th align="center" valign="middle">Active</th><th align="center" valign="middle">Coh-Inac</th><th align="center" valign="middle">Coh-Act</th><th align="center" valign="middle">Non-Mod</th><th align="center" valign="middle">Coh-NonEvo</th></tr></thead><tbody><tr><td align="center" valign="middle">pIC&#160;(n=1348)</td><td align="center" valign="middle">75 (5.6)</td><td align="center" valign="middle">217 (16.1)</td><td align="center" valign="middle">193 (14.3)</td><td align="center" valign="middle">414 (30.7)</td><td align="center" valign="middle">322 (23.9)</td><td align="center" valign="middle">53 (3.9)</td><td align="center" valign="middle">74 (5.5)</td></tr><tr><td align="center" valign="middle">aIC&#160;(n=1169)</td><td align="center" valign="middle">67 (5.7)</td><td align="center" valign="middle">202 (17.3)</td><td align="center" valign="middle">192 (16.4)</td><td align="center" valign="middle">317 (27.2)</td><td align="center" valign="middle"><bold>361 (30.9)*</bold></td><td align="center" valign="middle"><bold>7 (0.6)*</bold></td><td align="center" valign="middle"><bold>23 (2)*</bold></td></tr><tr><td align="center" valign="middle">OFC&#160;(n=1010)</td><td align="center" valign="middle"><bold>27 (2.7)*#</bold></td><td align="center" valign="middle"><bold>386 (38.2)*#</bold></td><td align="center" valign="middle"><bold>265 (26.2)*#</bold></td><td align="center" valign="middle"><bold>169 (16.7)*#</bold></td><td align="center" valign="middle"><bold>90 (8.9)*#</bold></td><td align="center" valign="middle"><bold>62 (6.1)*#</bold></td><td align="center" valign="middle"><bold>11 (1.1)*</bold></td></tr></tbody></table><table-wrap-foot><fn><p>Number of Cue-D responsive neurons (%). Data in bold indicate statistically different against pIC (*) or aIC (#), detected by a chi-squared test. Alpha level set at 0.05.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-41152-v2.xml">elife-41152-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.41152.007</object-id><label>Table 2.</label><caption><title>Cue-Evoked responses: Non-selective and Intensity-selective neurons.</title></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="3" valign="middle">Brain region</th><th align="center" colspan="12" valign="middle">Cue-Evoked responses</th></tr><tr><th align="center" colspan="2" valign="middle">Phasic</th><th align="center" colspan="2" valign="middle">Inactive</th><th align="center" colspan="2" valign="middle">Active</th><th align="center" colspan="2" valign="middle">Coh-Inac</th><th align="center" colspan="2" valign="middle">Coh-Act</th><th align="center" colspan="2" valign="bottom">Total</th></tr><tr><th align="center" valign="middle">Non-Sel</th><th align="center" valign="middle">Int-Sel</th><th align="center" valign="middle">Non-Sel</th><th align="center" valign="middle">Int-Sel</th><th align="center" valign="middle">Non-Sel</th><th align="center" valign="middle">Int-Sel</th><th align="center" valign="middle">Non-Sel</th><th align="center" valign="middle">Int-Sel</th><th align="center" valign="middle">Non-Sel</th><th align="center" valign="middle">Int-Sel</th><th align="center" valign="middle">Non-Sel</th><th align="center" valign="middle">Int-Sel</th></tr></thead><tbody><tr><td align="center" valign="middle">pIC&#160;(n=1348)</td><td align="center" valign="middle">55 (4.1)</td><td align="center" valign="middle">20 (1.5)</td><td align="center" valign="middle">198 (14.7)</td><td align="center" valign="middle">19 (1.4)</td><td align="center" valign="middle">171 (12.7)</td><td align="center" valign="middle">22 (1.6)</td><td align="center" valign="middle">325 (24)</td><td align="center" valign="middle">89 (6.6)</td><td align="center" valign="middle">272 (20.2)</td><td align="center" valign="middle">50 (3.7)</td><td align="center" valign="middle">1021 (75.7)</td><td align="center" valign="middle">200 (14.8)</td></tr><tr><td align="center" valign="middle">aIC&#160;(n=1169)</td><td align="center" valign="middle">55 (4.7)</td><td align="center" valign="middle">12 (1)</td><td align="center" valign="middle">183 (15.7)</td><td align="center" valign="middle">19 (1.6)</td><td align="center" valign="middle">155 (13.3)</td><td align="center" valign="middle"><bold>37 (3.2)*</bold></td><td align="center" valign="middle">254 (21.6)</td><td align="center" valign="middle">63 (5.4)</td><td align="center" valign="middle">283 (24.2)</td><td align="center" valign="middle"><bold>78 (6.7)*</bold></td><td align="center" valign="middle">930 (79.6)</td><td align="center" valign="middle">209 (17.9)</td></tr><tr><td align="center" valign="middle">OFC&#160;(n=1010)</td><td align="center" valign="middle"><bold>21 (2.1)*#</bold></td><td align="center" valign="middle"><bold>6 (0.6)*</bold></td><td align="center" valign="middle"><bold>336 (33.3)*#</bold></td><td align="center" valign="middle"><bold>50 (5)*#</bold></td><td align="center" valign="middle"><bold>204 (20.2)*#</bold></td><td align="center" valign="middle"><bold>61 (6)#</bold></td><td align="center" valign="middle"><bold>124 (12.3)*#</bold></td><td align="center" valign="middle"><bold>45 (4.5)*</bold></td><td align="center" valign="middle"><bold>65 (6.4)*#</bold></td><td align="center" valign="middle"><bold>25 (2.5)#</bold></td><td align="center" valign="middle">750 (74.2)</td><td align="center" valign="middle"><bold>187 (18.5)*</bold></td></tr></tbody></table><table-wrap-foot><fn><p>Number of Non-selective and Intensity-selective neurons (%). Data in bold indicate statistically different against pIC(*) or aIC (#) detected by a chi-squared test. Alpha level set at 0.05.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-41771-v2.xml">elife-41771-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.41771.014</object-id><label>Table 1.</label><caption><title>Properties of the neuronal template complex.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">SNARE <break/>or SM</th><th rowspan="2" valign="top">Mutation <break/>or truncation <break/></th><th rowspan="2" valign="top">Unfolding <break/>energy <break/>(k<sub>B</sub>T)</th><th rowspan="2" valign="top">Equilibrium <break/>force<sup>*</sup> (pN)</th><th rowspan="2" valign="top">Folding <break/>rate <break/>(s<sup>&#8722;1</sup>)</th><th rowspan="2" valign="top">Unfolding <break/>rate (s<sup>&#8722;1</sup>)</th><th valign="top">Partially <break/>closed <break/>syntaxin<sup>&#8224;</sup></th><th colspan="2" valign="top">Template <break/>formation</th><th/><th colspan="2" valign="top">SNAP-25 <break/>binding</th></tr><tr><th valign="top">Prob.</th><th valign="top">Prob.<sup>&#8225;</sup></th><th valign="top">N<sup>&#167;</sup></th><th/><th valign="top">Prob.<sup>&#182;</sup></th><th valign="top">N<sup>**</sup></th></tr></thead><tbody><tr><td valign="top"><bold>WT</bold></td><td valign="top">-</td><td valign="top">5.2 (0.1)</td><td valign="top">5.1 (0.1)</td><td valign="top">132</td><td valign="top">0.7</td><td valign="top">0.4</td><td valign="top">0.5</td><td valign="top">346</td><td/><td valign="top">0.7</td><td valign="top">50</td></tr><tr><td rowspan="13"><bold>Munc18-1</bold></td><td valign="top">L247R</td><td valign="top">1.6 (0.3)</td><td valign="top">2.3 (0.1)</td><td valign="top">-</td><td valign="top">-</td><td valign="top">0.3</td><td valign="top">0.3</td><td valign="top">99</td><td/><td valign="top">0.7</td><td valign="top">6</td></tr><tr><td valign="top">T248G</td><td valign="top">2.9 (0.2)</td><td valign="top">3.1 (0.1)</td><td valign="top">-</td><td valign="top">-</td><td valign="top">0</td><td valign="top">0.3</td><td valign="top">155</td><td/><td valign="top">0.3</td><td valign="top">16</td></tr><tr><td valign="top">L247A/ <break/>T248G</td><td valign="top">&lt;1.5<sup>***</sup></td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">0</td><td valign="top">0</td><td valign="top">241</td><td/><td valign="top">-</td><td valign="top">-</td></tr><tr><td valign="top">S306D<sup>&#182;&#182;</sup></td><td valign="top">5.8 (0.1)</td><td valign="top">5.6 (0.1)</td><td valign="top">184</td><td valign="top">0.6</td><td valign="top">0.4</td><td valign="top">0.9</td><td valign="top">123</td><td/><td valign="top">0.9</td><td valign="top">53</td></tr><tr><td valign="top">L307R</td><td valign="top">4.1 (0.2)</td><td valign="top">4.6 (0.1)</td><td valign="top"/><td valign="top"/><td valign="top">0.07</td><td valign="top">0.43</td><td valign="top">114</td><td/><td valign="top">0.58</td><td valign="top">19</td></tr><tr><td valign="top">S313D<sup>&#182;&#182;</sup></td><td valign="top">6.1 (0.2)</td><td valign="top">5.7 (0.1)</td><td valign="top">568</td><td valign="top">1.5</td><td valign="top">0.4</td><td valign="top">1</td><td valign="top">162</td><td/><td valign="top">0.8</td><td valign="top">70</td></tr><tr><td valign="top">&#916;324&#8211; <break/>339<sup>&#8224;&#8224;,&#8225;&#8225;</sup></td><td valign="top">&lt;1.5<sup>***</sup></td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">0</td><td valign="top">0</td><td valign="top">105</td><td/><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top">D326K<sup>&#182;&#182;</sup></td><td valign="top">6.5 (0.2)</td><td valign="top">5.7 (0.1)</td><td valign="top">420</td><td valign="top">0.6</td><td valign="top">0.03</td><td valign="top">0.9</td><td valign="top">103</td><td/><td valign="top">1</td><td valign="top">27</td></tr><tr><td valign="top">P335A<sup>&#167;&#167;</sup></td><td valign="top">6.0 (0.3)</td><td valign="top">5.9 (0.1)</td><td valign="top">258</td><td valign="top">0.5</td><td valign="top">0.02</td><td valign="top">0.7</td><td valign="top">155</td><td/><td valign="top">0.9</td><td valign="top">11</td></tr><tr><td valign="top">P335L<sup>&#167;&#167;</sup></td><td valign="top">4.3 (0.1)</td><td valign="top">4.8 (0.1)</td><td valign="top">17</td><td valign="top">0.2</td><td valign="top">0.4</td><td valign="top">0.3</td><td valign="top">224</td><td/><td valign="top">0.8</td><td valign="top">36</td></tr><tr><td valign="top">L341P<sup>&#167;&#167;</sup></td><td valign="top">&lt;1.5<sup>***</sup></td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">0.06</td><td valign="top">0.04</td><td valign="top">176</td><td/><td valign="top">0.5</td><td valign="top">4</td></tr><tr><td valign="top">L348R<sup>&#8224;&#8224;,&#8225;&#8225;</sup></td><td valign="top">&lt;1.5<sup>***</sup></td><td valign="top"/><td valign="top">-</td><td valign="top">-</td><td valign="top">0.02</td><td valign="top">0.04</td><td valign="top">222</td><td/><td valign="top">0.7</td><td valign="top">6</td></tr><tr><td valign="top">Y473D<sup>&#8225;&#8225;</sup></td><td valign="top">4.0 (0.1)</td><td valign="top">4.3 (0.2)</td><td valign="top">-</td><td valign="top">-</td><td valign="top">0</td><td valign="top">0.1</td><td valign="top">395</td><td/><td valign="top">0.5</td><td valign="top">24</td></tr><tr><td rowspan="9"><bold>VAMP2</bold></td><td valign="top">L32G/Q33G</td><td valign="top">3.4 (0.2)</td><td valign="top">3.9 (0.1)</td><td valign="top">310</td><td valign="top">10</td><td valign="top">0.4</td><td valign="top">0.6</td><td valign="top">170</td><td/><td valign="top">0.06</td><td valign="top">33</td></tr><tr><td valign="top">V39D</td><td valign="top">3.8 (0.4)</td><td valign="top">3.9 (0.2)</td><td valign="top">90</td><td valign="top">2</td><td valign="top">0.3</td><td valign="top">0.1</td><td valign="top">175</td><td/><td valign="top">0.8</td><td valign="top">13</td></tr><tr><td valign="top">M46A</td><td valign="top">5.2 (0.4)</td><td valign="top">5.1 (0.2)</td><td valign="top">130</td><td valign="top">0.7</td><td valign="top">0.3</td><td valign="top">0.5</td><td valign="top">52</td><td/><td valign="top">0.8</td><td valign="top">13</td></tr><tr><td valign="top">E62T<sup>&#8224;&#8224;</sup></td><td valign="top">4.1 (0.2)</td><td valign="top">4.8 (0.2)</td><td valign="top">107</td><td valign="top">5</td><td valign="top">0.4</td><td valign="top">0.5</td><td valign="top">104</td><td/><td valign="top">0.4</td><td valign="top">23</td></tr><tr><td valign="top">S61D/ <break/>E62T<sup>&#8224;&#8224;</sup></td><td valign="top">3.6 (0.2)</td><td valign="top">4.1 (0.1)</td><td valign="top"/><td valign="top"/><td valign="top">0.4</td><td valign="top">0.7</td><td valign="top">56</td><td/><td valign="top">0.2</td><td valign="top">12</td></tr><tr><td valign="top">Q76A<sup>&#8224;&#8224;</sup></td><td valign="top">4.7 (0.2)</td><td valign="top">4.8 (0.1)</td><td valign="top">166</td><td valign="top">2</td><td valign="top">0.4</td><td valign="top">0.6</td><td valign="top">62</td><td/><td valign="top">0.3</td><td valign="top">12</td></tr><tr><td valign="top">F77A<sup>&#8225;&#8225;</sup></td><td valign="top">1.5 (0.3)</td><td valign="top">2.3</td><td valign="top">-</td><td valign="top">-</td><td valign="top">0.5</td><td valign="top">0.1</td><td valign="top">121</td><td/><td valign="top">0.5</td><td valign="top">6</td></tr><tr><td valign="top">A81G/A82G</td><td valign="top">5.0 (0.3)</td><td valign="top">4.9 (0.2)</td><td valign="top">130</td><td valign="top">0.8</td><td valign="top">0.4</td><td valign="top">0.5</td><td valign="top">149</td><td/><td valign="top">0.4</td><td valign="top">42</td></tr><tr><td valign="top">&#916;85&#8211;94</td><td valign="top">5.1 (0.2)</td><td valign="top">5.0 (0.1)</td><td valign="top">120</td><td valign="top">0.7</td><td valign="top">0.4</td><td valign="top">0.5</td><td valign="top">87</td><td/><td valign="top">0.7</td><td valign="top">29</td></tr><tr><td rowspan="12"><bold>Syntaxin-1</bold></td><td valign="top">&#916;NRD<sup>&#8224;&#8224;,&#8225;&#8225;</sup></td><td valign="top">&lt;1.5<sup>***</sup></td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">0</td><td valign="top">0.08</td><td valign="top">105</td><td/><td valign="top">0.2</td><td valign="top">12</td></tr><tr><td valign="top">&#916;N- <break/>peptide<sup>&#8224;&#8224;,&#8225;&#8225;</sup></td><td valign="top">3.2 (0.2)</td><td valign="top">4.6 (0.1)</td><td valign="top">42</td><td valign="top">2</td><td valign="top">0.03</td><td valign="top">0.5</td><td valign="top">328</td><td/><td valign="top">0.4</td><td valign="top">46</td></tr><tr><td valign="top">&#916;H<sub>abc<sup>&#8225;&#8225;</sup></sub></td><td valign="top">&lt;1.5<sup>***</sup></td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td><td valign="top">0</td><td valign="top">0.06</td><td valign="top">140</td><td/><td valign="top">0.5</td><td valign="top">4</td></tr><tr><td valign="top">L165A/E166A <break/>(LE)<sup>&#182;&#182;</sup></td><td valign="top">6.7 (0.2)</td><td valign="top">6.1 (0.1)</td><td valign="top">406</td><td valign="top">0.5</td><td valign="top">0.07</td><td valign="top">0.7</td><td valign="top">83</td><td/><td valign="top">0.9</td><td valign="top">26</td></tr><tr><td valign="top">LE/E76K</td><td valign="top">6.4 (0.2)</td><td valign="top">6.0 (0.2)</td><td valign="top">123</td><td valign="top">0.2</td><td valign="top">0.07</td><td valign="top">0.9</td><td valign="top">81</td><td/><td valign="top">0.7</td><td valign="top">30</td></tr><tr><td valign="top">I202G/I203G</td><td valign="top">3.0 (0.3)</td><td valign="top">3.8 (0.1)</td><td valign="top">240</td><td valign="top">12</td><td valign="top">0.4</td><td valign="top">0.5</td><td valign="top">177</td><td/><td valign="top">0.4</td><td valign="top">33</td></tr><tr><td valign="top">F216A</td><td valign="top">3.7 (0.1)</td><td valign="top">5.1 (0.1)</td><td valign="top">82</td><td valign="top">2</td><td valign="top">0</td><td valign="top">0.6</td><td valign="top">155</td><td/><td valign="top">0.9</td><td valign="top">32</td></tr><tr><td valign="top">I230G/D231G/ <break/>R232G<sup>&#8224;&#8224;&#8224;</sup></td><td valign="top">3.6 (0.2)</td><td valign="top">4.3 (0.1)</td><td valign="top">-</td><td valign="top">-</td><td valign="top">0</td><td valign="top">0.5</td><td valign="top">111</td><td/><td valign="top">0.4</td><td valign="top">7</td></tr><tr><td valign="top">I233G/E234G/ <break/>Y235G<sup>&#8224;&#8224;&#8224;</sup></td><td valign="top">3.0 (0.2)</td><td valign="top">4.1 (0.1)</td><td valign="top">-</td><td valign="top">-</td><td valign="top">0</td><td valign="top">0.6</td><td valign="top">122</td><td/><td valign="top">0.7</td><td valign="top">30</td></tr><tr><td valign="top">V237G/E238G/ <break/>H239G</td><td valign="top">5.2 (0.2)</td><td valign="top">4.9 (0.1)</td><td valign="top">124</td><td valign="top">0.7</td><td valign="top">0.01</td><td valign="top">0.3</td><td valign="top">182</td><td/><td valign="top">0.4</td><td valign="top">14</td></tr><tr><td valign="top">T251G/K252G</td><td valign="top">5.2 (0.1)</td><td valign="top">4.9 (0.1)</td><td valign="top">126</td><td valign="top">0.7</td><td valign="top">0.5</td><td valign="top">0.8</td><td valign="top">197</td><td/><td valign="top">0.7</td><td valign="top">47</td></tr><tr><td valign="top">&#916;255&#8211;264</td><td valign="top">5.4 (0.2)</td><td valign="top">5.1 (0.1)</td><td valign="top">140</td><td valign="top">0.6</td><td valign="top">0.5</td><td valign="top">0.5</td><td valign="top">134</td><td/><td valign="top">0.7</td><td valign="top">29</td></tr><tr><td valign="top">Syntaxin-1</td><td valign="top">L165A/E166A</td><td rowspan="2" valign="top">6.6 (0.2)</td><td rowspan="2" valign="top">6.2 (0.1)</td><td rowspan="2" valign="top">72</td><td rowspan="2" valign="top">0.1</td><td rowspan="2" valign="top">0.2</td><td rowspan="2" valign="top">0.9</td><td rowspan="2" valign="top">85</td><td rowspan="2"/><td rowspan="2" valign="top">0.2</td><td rowspan="2" valign="top">11</td></tr><tr><td valign="top">Munc18-1</td><td valign="top">D326K<sup>&#182;&#182;</sup></td></tr></tbody></table><table-wrap-foot><fn><p><sup>*</sup> Mean of two average forces for the unfolded and folded states when the two states are equally populated (<xref ref-type="bibr" rid="bib38">Rebane et al., 2016</xref>). The equilibrium force of the template complex generally correlates with its unfolding energy.&#160;The number in parentheses is the standard error of the mean.</p><p><sup>&#8224;&#160;</sup>Detected as the syntaxin- and Munc18-1-dependent transition in the force range of 10&#8211;15 pN.</p></fn><fn><p><sup>&#8225;</sup> Probability per relaxation or pulling measured in the absence of SNAP-25B.</p><p><sup>&#167;</sup> Total number of pulling or relaxation FECs acquired, in which transitions of the template complex or syntaxin are scored, including their average equilibrium forces and extension changes.</p></fn><fn><p><sup>&#182;</sup> Probability of SNAP-25B binding and SNARE assembly per relaxation upon formation of the template complex.</p><p><sup>**</sup> Total number of relaxation FECs containing the template complex transition.</p></fn><fn><p><sup>&#8224;&#8224;</sup> Mutation that reduces membrane fusion in vitro (<xref ref-type="bibr" rid="bib36">Parisotto et al., 2014</xref>; <xref ref-type="bibr" rid="bib42">Shen et al., 2010</xref>; <xref ref-type="bibr" rid="bib41">Shen et al., 2007</xref>).</p><p><sup>&#8225;&#8225;</sup> Mutation that impairs exocytosis or neurotransmitter release in vivo (<xref ref-type="bibr" rid="bib32">Meijer et al., 2018</xref>; <xref ref-type="bibr" rid="bib35">Munch et al., 2016</xref>; <xref ref-type="bibr" rid="bib50">Walter et al., 2010</xref>).</p></fn><fn><p><sup>&#167;&#167;</sup> Mutation associated with epilepsy (<xref ref-type="bibr" rid="bib46">Stamberger et al., 2016</xref>).</p><p><sup>&#182;&#182;</sup> Mutation that <italic>enhances</italic> membrane fusion in vitro or neurotransmitter release in the cell (<xref ref-type="bibr" rid="bib17">Genc et al., 2014</xref>; <xref ref-type="bibr" rid="bib18">Gerber et al., 2008</xref>; <xref ref-type="bibr" rid="bib24">Lai et al., 2017</xref>; <xref ref-type="bibr" rid="bib35">Munch et al., 2016</xref>; <xref ref-type="bibr" rid="bib36">Parisotto et al., 2014</xref>; <xref ref-type="bibr" rid="bib39">Richmond et al., 2001</xref>).</p></fn><fn><p>*** Unfolding energy below the detection limit of our method, estimated to be 1.5 k<sub>B</sub>T, or not available due to no, infrequent, or heterogeneous template complex transition.</p><p><sup>&#8224;&#8224;&#8224;</sup> In the observed template complex transition, the template complex frequently dwelled in the unfolded state for an&#160;unusually long time (<xref ref-type="fig" rid="fig3s1">Figure 3&#8212;figure supplement 1</xref>). Thus, the transition is no longer two-state.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-41841-v2.xml">elife-41841-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.41841.002</object-id><label>Table 1.</label><caption><title>Details for all groups of infant subjects in this study.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Expt.</th><th rowspan="2">Age</th><th rowspan="2">#&#160;of&#160;Animals</th><th rowspan="2">Mean&#160;Weight&#160;(g)</th><th rowspan="2">#&#160;of&#160;Neurons</th><th colspan="5">Neuronal Classification</th><th rowspan="2"/><th rowspan="2">Mean # of Twitches</th><th rowspan="2">Mean # of Wake Movements</th></tr><tr><th>Unresp.</th><th>Resp.</th><th>Wake</th><th>Twitch</th><th>Both</th></tr></thead><tbody><tr><td rowspan="5">M1</td><td>8</td><td>10</td><td>19.0 &#177; 0.40</td><td>160</td><td>84</td><td>76</td><td>0</td><td>64</td><td>12</td><td/><td>302 &#177; 31</td><td>60 &#177; 11</td></tr><tr><td>9</td><td>9</td><td>21.8 &#177; 0.60</td><td>112</td><td>61</td><td>51</td><td>0</td><td>41</td><td>10</td><td/><td>259 &#177; 10</td><td>72 &#177; 10</td></tr><tr><td>10</td><td>8</td><td>24.8 &#177; 0.27</td><td>123</td><td>73</td><td>50</td><td>10</td><td>26</td><td>14</td><td/><td>187 &#177; 26</td><td>54 &#177; 10</td></tr><tr><td>11</td><td>15</td><td>27.4 &#177; 0.34</td><td>197</td><td>113</td><td>84</td><td>23</td><td>18</td><td>43</td><td/><td>157 &#177; 18</td><td>79 &#177; 10</td></tr><tr><td>12</td><td>7</td><td>30.2 &#177; 0.74</td><td>165</td><td>102</td><td>63</td><td>48</td><td>3</td><td>12</td><td/><td>104 &#177; 22</td><td>82 &#177; 14</td></tr><tr><td rowspan="2">ECN</td><td>9</td><td>6</td><td>20.3 &#177; 1.18</td><td>16</td><td>0</td><td>16</td><td>1</td><td>11</td><td>4</td><td/><td>253 &#177; 20</td><td>84 &#177; 15</td></tr><tr><td>12</td><td>7</td><td>28.1 &#177; 1.20</td><td>20</td><td>0</td><td>20</td><td>1</td><td>4</td><td>15</td><td/><td>226 &#177; 27</td><td>70 &#177; 8</td></tr><tr><td>Saline&#160;Pre</td><td rowspan="2" valign="middle">12</td><td rowspan="2" valign="middle">6</td><td rowspan="2" valign="middle">31.1 &#177; 1.27</td><td rowspan="2" valign="middle">107</td><td>73</td><td>38</td><td>30</td><td>2</td><td>6</td><td/><td>177 &#177; 43</td><td>113 &#177; 8</td></tr><tr><td>Saline&#160;Post</td><td>69</td><td>34</td><td>25</td><td>3</td><td>6</td><td/><td>186 &#177; 29</td><td>97 &#177; 14</td></tr><tr><td>Bicuc.&#160;Pre</td><td rowspan="2" valign="middle">12</td><td rowspan="2" valign="middle">6</td><td rowspan="2" valign="middle">30.0 &#177; 1.23</td><td rowspan="2" valign="middle">99</td><td>56</td><td>43</td><td>39</td><td>2</td><td>2</td><td/><td>112 &#177; 19</td><td>80 &#177; 13</td></tr><tr><td>Bicuc.&#160;Post</td><td>34</td><td>65</td><td>19</td><td>7</td><td>39</td><td/><td>131 &#177; 37</td><td>91 &#177; 19</td></tr></tbody></table><table-wrap-foot><fn><p>The number of animals, weight (mean &#177; s.d.), neuronal classification, and number of triggered twitches and wake movements (mean &#177; s.d.) for the M1, ECN, and disinhibition experiments.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-41844-v2.xml">elife-41844-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.41844.006</object-id><label>Table 1.</label><caption><title>Biophysical properties of OSCA and TMEM63 proteins.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene</th><th colspan="3" valign="top">Whole-cell/Poke</th><th colspan="5" valign="top">Cell-attached/Stretch</th></tr><tr><th valign="top">Imax <break/>(pA)</th><th valign="top">Inactivation tau <break/>(ms)</th><th valign="top">Threshold <break/>(&#181;m)</th><th valign="top">Imax <break/>(pA)</th><th valign="top">Activation tau <break/>(ms)</th><th valign="top">Inactivation tau <break/>(ms)</th><th valign="top">P<sub>50</sub> <break/>(-mmHg)</th><th valign="top">Channel conductance (pS)</th></tr></thead><tbody><tr><td>Mock</td><td>4.2&#160;&#177;&#160;0.5 <break/>(10)</td><td>-</td><td>-</td><td>2.32&#160;&#177;&#160;0.3 <break/>(17)</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>Mm <break/>PIEZO1</td><td>3045&#160;&#177;&#160;875 <break/>(5)</td><td>18.7&#160;&#177;&#160;1.7 <break/>(5)</td><td>4.0&#160;&#177;&#160;0.6 <break/>(5)</td><td>122&#160;&#177;&#160;40 <break/>(5)</td><td>9.0&#160;&#177;&#160;1.0 <break/>(5)</td><td>28.8&#160;&#177;&#160;2.0 <break/>(5)</td><td>24.0&#160;&#177;&#160;3.6 <break/>(4)</td><td>27.3&#160;&#177;&#160;0.3 (4)*</td></tr><tr><td>OSCA1.1</td><td>2271&#160;&#177;&#160;918 <break/>(16)</td><td>10.0&#160;&#177;&#160;1.3 <break/>(8)</td><td>8.6&#160;&#177;&#160;0.9 <break/>(9)</td><td>563&#160;&#177;&#160;68 <break/>(11)</td><td>6.7&#160;&#177;&#160;1.0 <break/>(11)</td><td>24.0&#160;&#177;&#160;3.0 <break/>(8)</td><td>58.5&#160;&#177;&#160;2.4 <break/>(6)</td><td>184.4&#160;&#177;&#160;4.4 <break/>(5)</td></tr><tr><td>OSCA1.2</td><td>4039&#160;&#177;&#160;1046 <break/>(12)</td><td>10.4&#160;&#177;&#160;1.7 <break/>(9)</td><td>6.3&#160;&#177;&#160;0.7 <break/>(10)</td><td>576&#160;&#177;&#160;65 <break/>(14)</td><td>5.5&#160;&#177;&#160;0.5 <break/>(14)</td><td>24.6&#160;&#177;&#160;4.7 <break/>(9)</td><td>54.5&#160;&#177;&#160;2.2 (10)</td><td>121.8&#160;&#177;&#160;3.4 <break/>(4)</td></tr><tr><td>OSCA1.8</td><td>7.1&#160;&#177;&#160;0.7 <break/>(11)</td><td>-</td><td>-</td><td>347&#160;&#177;&#160;56 <break/>(7)</td><td>14.0&#160;&#177;&#160;2.4 <break/>(7)</td><td>67.0&#160;&#177;&#160;13.0 <break/>(6)</td><td>79.3&#160;&#177;&#160;9.9 <break/>(6)</td><td>46.6&#160;&#177;&#160;2.8 <break/>(4)</td></tr><tr><td>OSCA2.3</td><td>54.34&#160;&#177;&#160;48.9 <break/>(11)</td><td>-</td><td>-</td><td>31.3&#160;&#177;&#160;11 <break/>(7)</td><td>18.8&#160;&#177;&#160;3.2 <break/>(6)</td><td>110.0&#160;&#177;&#160;70.0 <break/>(6)</td><td>59.7&#160;&#177;&#160;3.8 <break/>(7)</td><td>n.d.</td></tr><tr><td>OSCA3.1</td><td>9.7&#160;&#177;&#160;1.4 <break/>(7)</td><td>-</td><td>-</td><td>306&#160;&#177;&#160;47 <break/>(12)</td><td>5.6&#160;&#177;&#160;1.4 <break/>(8)</td><td>18.5&#160;&#177;&#160;2.5 <break/>(11)</td><td>44.5&#160;&#177;&#160;3.2 <break/>(5)</td><td>24.9&#160;&#177;&#160;3.4 <break/>(4)</td></tr><tr><td>OSCA4.1</td><td>13.3&#160;&#177;&#160;2.0 <break/>(6)</td><td>-</td><td>-</td><td>1.2&#160;&#177;&#160;0.05 <break/>(6)</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>Dm <break/>TMEM63</td><td>9.8&#160;&#177;&#160;2.0 <break/>(11)</td><td>-</td><td>-</td><td>14.2&#160;&#177;&#160;4.5 <break/>(9)</td><td>22.6&#160;&#177;&#160;4.3 <break/>(5)</td><td>154&#160;&#177;&#160;34 <break/>(5)</td><td>90&#160;&#177;&#160;2 <break/>(3)</td><td>n.d.</td></tr><tr><td>Mm <break/>TMEM63A</td><td>2.0&#160;&#177;&#160;1.2 <break/>(13)</td><td>-</td><td>-</td><td>18.75&#160;&#177;&#160;3.0 <break/>(23)</td><td>126&#160;&#177;&#160;19 <break/>(8)</td><td>323&#160;&#177;&#160;30 <break/>(9)</td><td>60&#160;&#177;&#160;5.4 <break/>(6)</td><td>n.d.</td></tr><tr><td>Mm <break/>TMEM63B</td><td>3.8&#160;&#177;&#160;1.8 <break/>(7)</td><td>-</td><td>-</td><td>17.97&#160;&#177;&#160;3.9 <break/>(12)</td><td>245&#160;&#177;&#160;38 <break/>(7)</td><td>323&#160;&#177;&#160;30 <break/>(7)</td><td>66&#160;&#177;&#160;7.8 <break/>(4)</td><td>n.d.</td></tr><tr><td>Mm <break/>TMEM63C</td><td>4.5&#160;&#177;&#160;0.9 <break/>(8)</td><td>-</td><td>-</td><td>4.73&#160;&#177;&#160;0.8 <break/>(7)</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>Hs <break/>TMEM63A</td><td>4.2&#160;&#177;&#160;0.7 <break/>(6)</td><td>-</td><td>-</td><td>12.4&#160;&#177;&#160;4.4 <break/>(9)</td><td>188&#160;&#177;&#160;18 <break/>(5)</td><td>427&#160;&#177;&#160;94 <break/>(2)</td><td>59.7&#160;&#177;&#160;3.2 <break/>(5)</td><td>n.d.</td></tr></tbody></table><table-wrap-foot><fn><p><bold>Note:</bold> n.d.: not determined. Ns are indicated in parenthesis. Imax value is reported for the last indentation or stretch stimulus before losing the cell. Activation is reported as 10&#8211;90% rise of stretch-activated current at saturating stimulus. Inactivation time constant for stretch-activated currents are reported in the range of &#8722;60 to &#8722;80 mmHg stimulus-pressure. All values are mean&#160;&#177;&#160;s.e.m. * values as indicated in <xref ref-type="bibr" rid="bib28">Saotome et al., 2018</xref></p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-41844-v3.xml">elife-41844-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.41844.006</object-id><label>Table 1.</label><caption><title>Biophysical properties of OSCA and TMEM63 proteins.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene</th><th colspan="3" valign="top">Whole-cell/Poke</th><th colspan="5" valign="top">Cell-attached/Stretch</th></tr><tr><th valign="top">Imax <break/>(pA)</th><th valign="top">Inactivation tau <break/>(ms)</th><th valign="top">Threshold <break/>(&#181;m)</th><th valign="top">Imax <break/>(pA)</th><th valign="top">Activation tau <break/>(ms)</th><th valign="top">Inactivation tau <break/>(ms)</th><th valign="top">P<sub>50</sub> <break/>(-mmHg)</th><th valign="top">Channel conductance (pS)</th></tr></thead><tbody><tr><td>Mock</td><td>4.2&#160;&#177;&#160;0.5 <break/>(10)</td><td>-</td><td>-</td><td>2.32&#160;&#177;&#160;0.3 <break/>(17)</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>Mm <break/>PIEZO1</td><td>3045&#160;&#177;&#160;875 <break/>(5)</td><td>18.7&#160;&#177;&#160;1.7 <break/>(5)</td><td>4.0&#160;&#177;&#160;0.6 <break/>(5)</td><td>122&#160;&#177;&#160;40 <break/>(5)</td><td>9.0&#160;&#177;&#160;1.0 <break/>(5)</td><td>28.8&#160;&#177;&#160;2.0 <break/>(5)</td><td>24.0&#160;&#177;&#160;3.6 <break/>(4)</td><td>27.3&#160;&#177;&#160;0.3 (4)*</td></tr><tr><td>OSCA1.1</td><td>2271&#160;&#177;&#160;918 <break/>(16)</td><td>10.0&#160;&#177;&#160;1.3 <break/>(8)</td><td>8.6&#160;&#177;&#160;0.9 <break/>(9)</td><td>563&#160;&#177;&#160;68 <break/>(11)</td><td>6.7&#160;&#177;&#160;1.0 <break/>(11)</td><td>24.0&#160;&#177;&#160;3.0 <break/>(8)</td><td>58.5&#160;&#177;&#160;2.4 <break/>(6)</td><td>184.4&#160;&#177;&#160;4.4 <break/>(5)</td></tr><tr><td>OSCA1.2</td><td>4039&#160;&#177;&#160;1046 <break/>(12)</td><td>10.4&#160;&#177;&#160;1.7 <break/>(9)</td><td>6.3&#160;&#177;&#160;0.7 <break/>(10)</td><td>576&#160;&#177;&#160;65 <break/>(14)</td><td>5.5&#160;&#177;&#160;0.5 <break/>(14)</td><td>24.6&#160;&#177;&#160;4.7 <break/>(9)</td><td>54.5&#160;&#177;&#160;2.2 (10)</td><td>121.8&#160;&#177;&#160;3.4 <break/>(4)</td></tr><tr><td>OSCA1.8</td><td>7.1&#160;&#177;&#160;0.7 <break/>(11)</td><td>-</td><td>-</td><td>347&#160;&#177;&#160;56 <break/>(7)</td><td>14.0&#160;&#177;&#160;2.4 <break/>(7)</td><td>67.0&#160;&#177;&#160;13.0 <break/>(6)</td><td>79.3&#160;&#177;&#160;9.9 <break/>(6)</td><td>46.6&#160;&#177;&#160;2.8 <break/>(4)</td></tr><tr><td>OSCA2.3</td><td>54.34&#160;&#177;&#160;48.9 <break/>(11)</td><td>-</td><td>-</td><td>31.3&#160;&#177;&#160;11 <break/>(7)</td><td>18.8&#160;&#177;&#160;3.2 <break/>(6)</td><td>110.0&#160;&#177;&#160;70.0 <break/>(6)</td><td>59.7&#160;&#177;&#160;3.8 <break/>(7)</td><td>n.d.</td></tr><tr><td>OSCA3.1</td><td>9.7&#160;&#177;&#160;1.4 <break/>(7)</td><td>-</td><td>-</td><td>306&#160;&#177;&#160;47 <break/>(12)</td><td>5.6&#160;&#177;&#160;1.4 <break/>(8)</td><td>18.5&#160;&#177;&#160;2.5 <break/>(11)</td><td>44.5&#160;&#177;&#160;3.2 <break/>(5)</td><td>24.9&#160;&#177;&#160;3.4 <break/>(4)</td></tr><tr><td>OSCA4.1</td><td>13.3&#160;&#177;&#160;2.0 <break/>(6)</td><td>-</td><td>-</td><td>1.2&#160;&#177;&#160;0.05 <break/>(6)</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>Dm <break/>TMEM63</td><td>9.8&#160;&#177;&#160;2.0 <break/>(11)</td><td>-</td><td>-</td><td>14.2&#160;&#177;&#160;4.5 <break/>(9)</td><td>22.6&#160;&#177;&#160;4.3 <break/>(5)</td><td>154&#160;&#177;&#160;34 <break/>(5)</td><td>90&#160;&#177;&#160;2 <break/>(3)</td><td>n.d.</td></tr><tr><td>Mm <break/>TMEM63A</td><td>2.0&#160;&#177;&#160;1.2 <break/>(13)</td><td>-</td><td>-</td><td>18.75&#160;&#177;&#160;3.0 <break/>(23)</td><td>126&#160;&#177;&#160;19 <break/>(8)</td><td>323&#160;&#177;&#160;30 <break/>(9)</td><td>60&#160;&#177;&#160;5.4 <break/>(6)</td><td>n.d.</td></tr><tr><td>Mm <break/>TMEM63B</td><td>3.8&#160;&#177;&#160;1.8 <break/>(7)</td><td>-</td><td>-</td><td>17.97&#160;&#177;&#160;3.9 <break/>(12)</td><td>245&#160;&#177;&#160;38 <break/>(7)</td><td>323&#160;&#177;&#160;30 <break/>(7)</td><td>66&#160;&#177;&#160;7.8 <break/>(4)</td><td>n.d.</td></tr><tr><td>Mm <break/>TMEM63C</td><td>4.5&#160;&#177;&#160;0.9 <break/>(8)</td><td>-</td><td>-</td><td>4.73&#160;&#177;&#160;0.8 <break/>(7)</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>Hs <break/>TMEM63A</td><td>4.2&#160;&#177;&#160;0.7 <break/>(6)</td><td>-</td><td>-</td><td>12.4&#160;&#177;&#160;4.4 <break/>(9)</td><td>188&#160;&#177;&#160;18 <break/>(5)</td><td>427&#160;&#177;&#160;94 <break/>(2)</td><td>59.7&#160;&#177;&#160;3.2 <break/>(5)</td><td>n.d.</td></tr></tbody></table><table-wrap-foot><fn><p><bold>Note:</bold> n.d.: not determined. Ns are indicated in parenthesis. Imax value is reported for the last indentation or stretch stimulus before losing the cell. Activation is reported as 10&#8211;90% rise of stretch-activated current at saturating stimulus. Inactivation time constant for stretch-activated currents are reported in the range of &#8722;60 to &#8722;80 mmHg stimulus-pressure. All values are mean&#160;&#177;&#160;s.e.m. * values as indicated in <xref ref-type="bibr" rid="bib28">Saotome et al., 2018</xref></p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-42014-v2.xml">elife-42014-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.42014.011</object-id><label>Table 1.</label><caption><title>Major loci likely contributing to the selection response.</title><p>These eight loci show significant allele frequency shifts in &#8710;z<sup>2</sup> and are ordered according to their estimated selection coefficients according to <xref ref-type="bibr" rid="bib43">Haldane (1932)</xref>. Shown for each locus are the full hitchhiking spans, peak location and their size covering the core windows, the overlapping TAD and the number of genes found in it. The two top-ranked loci show shifts in parallel in both LS1 and LS2, with the remaining six showing line-specific response (LS1: 1; LS2: 5). Candidate genes found within the TAD with limb, cartilage, or bone developmental knockout phenotype functions are shown, with asterisks (*) marking those with a &#8216;short tibia&#8217; knockout phenotype (see also <xref ref-type="fig" rid="fig2s3">Figure 2&#8212;figure supplement 3</xref> and <xref ref-type="supplementary-material" rid="supp3">Supplementary file 3</xref> for full table).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Rk</th><th rowspan="2" valign="bottom">Chr</th><th rowspan="2" valign="bottom">Span (Mbp)</th><th rowspan="2" valign="bottom">Peak</th><th rowspan="2" valign="bottom">Core (kbp)</th><th rowspan="2" valign="bottom">TAD <break/>(kbp)</th><th rowspan="2" valign="bottom">Genes</th><th colspan="3" valign="bottom">&#8710;q</th><th colspan="2"><break/></th></tr><tr><th valign="bottom">LS1</th><th valign="bottom">LS2</th><th valign="bottom">Ctrl</th><th valign="bottom">Type</th><th valign="bottom">Candidate genes</th></tr></thead><tbody><tr><td align="right" valign="bottom">1</td><td valign="bottom">5</td><td valign="bottom">38.95&#8211;45.13</td><td valign="bottom">41.77</td><td valign="bottom">900</td><td valign="bottom">720</td><td valign="bottom"><italic>3</italic></td><td align="right" valign="bottom">0.69</td><td align="right" valign="bottom">0.86</td><td align="right" valign="bottom">&#8722;0.14</td><td valign="bottom">Parallel</td><td valign="bottom"><italic>Nkx3-2</italic></td></tr><tr><td align="right" valign="top">2</td><td valign="top">10</td><td valign="top">77.47&#8211;87.69</td><td valign="top">81.07</td><td valign="top">5360</td><td valign="top">6520</td><td valign="top"><italic>175</italic></td><td align="right" valign="top">0.79</td><td align="right" valign="top">0.88</td><td align="right" valign="top">&#8722;0.04</td><td valign="top">Parallel</td><td valign="top"><italic>Sbno5, Aes, Adamtsl5*, Chst11*, Cry1, Prdm4*</italic></td></tr><tr><td align="right" valign="bottom">3</td><td valign="bottom">18</td><td valign="bottom">53.63&#8211;63.50</td><td valign="bottom">58.18</td><td valign="bottom">220</td><td valign="bottom">520</td><td valign="bottom"><italic>4</italic></td><td align="right" valign="bottom">0.05</td><td align="right" valign="bottom">0.78</td><td align="right" valign="bottom">&#8722;0.06</td><td valign="bottom">LS2-specific</td><td valign="bottom">-</td></tr><tr><td align="right" valign="top">4</td><td valign="top">13</td><td valign="top">35.59&#8211;55.21</td><td valign="top">48.65</td><td valign="top">70</td><td valign="top">2600</td><td valign="top"><italic>22</italic></td><td align="right" valign="top">0.24</td><td align="right" valign="top">0.80</td><td align="right" valign="top">&#8722;0.03</td><td valign="top">LS2-specific</td><td valign="top"><italic>Id4</italic></td></tr><tr><td align="right" valign="bottom">5</td><td valign="bottom">1</td><td valign="bottom">53.16&#8211;57.13</td><td valign="bottom">55.27</td><td valign="bottom">10</td><td valign="bottom">720</td><td valign="bottom"><italic>4</italic></td><td align="right" valign="bottom">0.65</td><td align="right" valign="bottom">0.01</td><td align="right" valign="bottom">&#8722;0.23</td><td valign="bottom">LS1-specific</td><td valign="bottom">-</td></tr><tr><td align="right" valign="bottom">6</td><td valign="bottom">15</td><td valign="bottom">31.92&#8211;44.43</td><td valign="bottom">41.54</td><td valign="bottom">10</td><td valign="bottom">680</td><td valign="bottom"><italic>3</italic></td><td align="right" valign="bottom">&#8722;0.23</td><td align="right" valign="bottom">0.66</td><td align="right" valign="bottom">0.02</td><td valign="bottom">LS2-specific</td><td valign="bottom"><italic>Rspo2*</italic></td></tr><tr><td align="right" valign="bottom">7</td><td valign="bottom">6</td><td valign="bottom">118.65&#8211;125.25</td><td valign="bottom">120.30</td><td valign="bottom">130</td><td valign="bottom">1360</td><td valign="bottom"><italic>12</italic></td><td align="right" valign="bottom">&#8722;0.03</td><td align="right" valign="bottom">0.79</td><td align="right" valign="bottom">&#8722;0.15</td><td valign="bottom">LS2-specific</td><td valign="bottom"><italic>Wnt5b*</italic></td></tr><tr><td align="right" valign="bottom">&#8195;8</td><td valign="bottom">11</td><td valign="bottom">111.10&#8211;115.06</td><td valign="bottom">113.42</td><td valign="bottom">10</td><td valign="bottom">2120</td><td valign="bottom"><italic>2</italic></td><td align="right" valign="bottom">&#8722;0.14</td><td align="right" valign="bottom">0.66</td><td align="right" valign="bottom">&#8722;0.15</td><td valign="bottom">LS2-specific</td><td valign="bottom"><italic>Sox9*</italic></td></tr></tbody></table><table-wrap-foot><fn><p>Rk,&#160;Rank.</p><p>Chr,&#160;Chromosome.</p></fn><fn><p>Core,&#160;Span of 10 kbp windows above <italic>H<sub>INF, max LD</sub>&#160;p</italic>&#8804;0.05 significance threshold.</p><p>TAD,&#160;Merged span of topologically associating domains (TAD) overlapping the core span. TADs mark segments along a chromosome that share a common regulatory mechanism. Data from <xref ref-type="bibr" rid="bib31">Dixon et al. (2012)</xref>.</p></fn><fn><p>Candidate genes,&#160;Genes within the TAD span showing &#8216;short tibia&#8217;, &#8216;short limbs&#8217;, &#8216;abnormal osteoblast morphology&#8217; or &#8216;abnormal cartilage morphology&#8217; knockout phenotypes are listed, with * marking those with &#8216;short tibia&#8217;.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-42299-v2.xml">elife-42299-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="resptable2" position="float"><object-id pub-id-type="doi">10.7554/eLife.42299.031</object-id><label>Author response table 2.</label><caption><title>Measurement information for the main analysis in Figures 2-7.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><bold>Parameters</bold></th><th rowspan="2"><bold>Depth</bold></th><th colspan="3"><bold>Numbers of samples/mice</bold></th></tr><tr><th><bold>Capillaries</bold></th><th><bold>Arterioles</bold></th><th><bold>Venules</bold></th></tr></thead><tbody><tr><td><bold>Mean-PO<sub>2</sub></bold></td><td>0-600 &#181;m</td><td>978/15</td><td>11/7</td><td>14/7</td></tr><tr><td><bold>RBC Flux</bold></td><td>0-600 &#181;m</td><td>978/15</td><td>N.A.</td><td>N.A.</td></tr><tr><td><bold>Temporal Fluctuation</bold></td><td>0-600 &#181;m</td><td>373/7</td><td>N.A.</td><td>N.A.</td></tr><tr><td><bold>EATs</bold></td><td>0-600 &#181;m</td><td>373/7</td><td>N.A.</td><td>N.A.</td></tr><tr><td><bold>Branching Orders</bold></td><td>0-300 &#181;m</td><td>97/5</td><td>N.A.</td><td>N.A.</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-42627-v1.xml">elife-42627-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.42627.006</object-id><label>Table 2.</label><caption><title>Proportion of samples with gestational age correctly estimated within 1 week, 2 weeks of ultrasound-validated gestational age.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2"/><th colspan="4" valign="top">Heel prick samples</th><th colspan="4">Cord blood samples</th></tr><tr><th valign="top">Overall, <break/>n(%)</th><th valign="top">SGA10, <break/>n(%)</th><th valign="top">SGA3, <break/>n(%)</th><th valign="top">&lt;2500 g, <break/>n(%)</th><th valign="top">Overall, <break/>n(%)</th><th valign="top">SGA10, <break/>n(%)</th><th valign="top">SGA3, <break/>n(%)</th><th valign="top">&lt;2500 g, <break/>n(%)</th></tr></thead><tbody><tr><td><bold>Model 1:</bold> <break/>Baseline Model</td><td>RMSE <break/>n(%) within 1 week <break/>n(%) within 2 weeks</td><td><bold>1.46</bold> <break/><bold>267 (54.8)</bold> <break/><bold>408 (83.8)</bold></td><td>1.76 <break/>103 (44.6) <break/>177 (76.6)</td><td>2.32 <break/>17 (14.4) <break/>64 (54.2)</td><td>2.22 <break/>25 (28.4) <break/>54 (61.4)</td><td><bold>1.51</bold> <break/><bold>549 (53.0)</bold> <break/> <bold>861 (83.1)</bold></td><td>1.82 <break/>180 (42.5) <break/>318 (75.0)</td><td>2.38 <break/>31 (14.4) <break/>111 (51.6)</td><td>2.21 <break/>61 (37.0) <break/>112 (67.9)</td></tr><tr><td><bold>Model 2:</bold> <break/>Analyte Model</td><td>RMSE <break/>n(%) within 1 week <break/>n(%) within 2 weeks</td><td><bold>1.35</bold> <break/> <bold>279 (57.3)</bold> <break/><bold>431 (88.5)</bold></td><td>1.40 <break/>123 (53.4) <break/>204 (88.1)</td><td>1.38 <break/>64 (54.6) <break/>104 (88.1)</td><td>1.47 <break/>38 (43.2) <break/>74 (84.1)</td><td><bold>1.45</bold> <break/> <bold>544 (52.5)</bold> <break/> <bold>874 (84.4)</bold></td><td>1.43 <break/>221 (52.0) <break/>362 (85.4)</td><td>1.48 <break/>113 (52.5) <break/>181 (84.1)</td><td>1.94 <break/>62 (37.6) <break/>116 (70.3)</td></tr><tr><td><bold>Model 3:</bold> <break/>Full Model</td><td>RMSE <break/>n(%) within 1 week <break/>n(%) within 2 weeks</td><td><bold>1.07</bold> <break/> <bold>311 (63.9)</bold> <break/> <bold>459 (94.3)</bold></td><td>1.12 <break/>145 (62.8) <break/>218 (94.3)</td><td>1.30 <break/>63 (53.4) <break/>108 (91.4)</td><td>1.21 <break/>52 (59.1) <break/>83 (94.3)</td><td><bold>1.23</bold> <break/> <bold>615 (59.4)</bold> <break/><bold>937 (90.4)</bold></td><td>1.20 <break/>267 (63.1) <break/>385 (90.7)</td><td>1.40 <break/>116 (54.1) <break/>183 (85.0)</td><td>1.44 <break/>88 (53.3) <break/>139 (84.2)</td></tr></tbody></table><table-wrap-foot><fn><p>Data are presented as the percentage of the number correctly classified within the total of each birthweight category. Counts were based on the average from 10 imputations rounded to the closest integer.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-43400-v2.xml">elife-43400-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.43400.019</object-id><label>Table 2.</label><caption><title>Baseline time spent in sleep-wake states, including theta-dominated waking (TDW;&#160;min), and locomotor activity (LMA; movements) per 12 hr per genotype (mean&#160;&#177;1 SEM), averages of BL1-2.</title><p>Two-way ANOVA (Factor GT and Light/Dark) on those same 12 hr values.&#160;Degrees of freedom for both GT and Light/Dark: df&#160;=&#160;1; error term: df&#160;=&#160;35.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="2" valign="top">WT</th><th colspan="2" valign="top">KO</th><th valign="top">Statistics <break/>(Two-way ANOVA)</th></tr><tr><th valign="top">Light</th><th valign="top">Dark</th><th valign="top">Light</th><th valign="top">Dark</th><th valign="top">Factor GT x Light/Dark, df : 1,35</th></tr></thead><tbody><tr><td valign="top">NREM sleep</td><td valign="top">389&#160;&#177;&#160;4</td><td valign="top">189&#160;&#177;&#160;10</td><td valign="top">376&#160;&#177;&#160;4</td><td valign="top">170&#160;&#177;&#160;13</td><td valign="top">F&#160;=&#160;0.02, p=0.89</td></tr><tr><td valign="top">REM sleep</td><td valign="top">70&#160;&#177;&#160;2</td><td valign="top">19&#160;&#177;&#160;2</td><td valign="top">66&#160;&#177;&#160;2</td><td valign="top">20&#160;&#177;&#160;2</td><td valign="top">F&#160;=&#160;0.83, p=0.37</td></tr><tr><td valign="top">Total waking</td><td valign="top">260&#160;&#177;&#160;4</td><td valign="top">512&#160;&#177;&#160;11</td><td valign="top">277&#160;&#177;&#160;5</td><td valign="top">530&#160;&#177;&#160;14</td><td valign="top">F&#160;=&#160;0.02, p=0.90</td></tr><tr><td valign="top">TDW</td><td valign="top">45&#160;&#177;&#160;3</td><td valign="top">179&#160;&#177;&#160;12</td><td valign="top">55&#160;&#177;&#160;5</td><td valign="top">192&#160;&#177;&#160;15</td><td valign="top">F&#160;=&#160;0.13, p=0.72</td></tr><tr><td valign="top">LMA</td><td valign="top">119&#160;&#177;&#160;16</td><td valign="top">817&#160;&#177;&#160;70</td><td valign="top">181&#160;&#177;&#160;26</td><td valign="top">1370&#160;&#177;&#160;142</td><td valign="top">F&#160;=&#160;7.1, p=0.01</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-43400-v3.xml">elife-43400-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.43400.019</object-id><label>Table 2.</label><caption><title>Baseline time spent in sleep-wake states, including theta-dominated waking (TDW;&#160;min), and locomotor activity (LMA; movements) per 12 hr per genotype (mean&#160;&#177;1 SEM), averages of BL1-2.</title><p>Two-way ANOVA (Factor GT and Light/Dark) on those same 12 hr values.&#160;Degrees of freedom for both GT and Light/Dark: df&#160;=&#160;1; error term: df&#160;=&#160;35.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="2" valign="top">WT</th><th colspan="2" valign="top">KO</th><th valign="top">Statistics <break/>(Two-way ANOVA)</th></tr><tr><th valign="top">Light</th><th valign="top">Dark</th><th valign="top">Light</th><th valign="top">Dark</th><th valign="top">Factor GT x Light/Dark, df : 1,35</th></tr></thead><tbody><tr><td valign="top">NREM sleep</td><td valign="top">389&#160;&#177;&#160;4</td><td valign="top">189&#160;&#177;&#160;10</td><td valign="top">376&#160;&#177;&#160;4</td><td valign="top">170&#160;&#177;&#160;13</td><td valign="top">F&#160;=&#160;0.02, p=0.89</td></tr><tr><td valign="top">REM sleep</td><td valign="top">70&#160;&#177;&#160;2</td><td valign="top">19&#160;&#177;&#160;2</td><td valign="top">66&#160;&#177;&#160;2</td><td valign="top">20&#160;&#177;&#160;2</td><td valign="top">F&#160;=&#160;0.83, p=0.37</td></tr><tr><td valign="top">Total waking</td><td valign="top">260&#160;&#177;&#160;4</td><td valign="top">512&#160;&#177;&#160;11</td><td valign="top">277&#160;&#177;&#160;5</td><td valign="top">530&#160;&#177;&#160;14</td><td valign="top">F&#160;=&#160;0.02, p=0.90</td></tr><tr><td valign="top">TDW</td><td valign="top">45&#160;&#177;&#160;3</td><td valign="top">179&#160;&#177;&#160;12</td><td valign="top">55&#160;&#177;&#160;5</td><td valign="top">192&#160;&#177;&#160;15</td><td valign="top">F&#160;=&#160;0.13, p=0.72</td></tr><tr><td valign="top">LMA</td><td valign="top">119&#160;&#177;&#160;16</td><td valign="top">817&#160;&#177;&#160;70</td><td valign="top">181&#160;&#177;&#160;26</td><td valign="top">1370&#160;&#177;&#160;142</td><td valign="top">F&#160;=&#160;7.1, p=0.01</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-43464-v2.xml">elife-43464-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.43464.006</object-id><label>Table 1.</label><caption><title>Summary of exploratory findings.</title><p>For each discovery sample size, the number of clusters in which gray matter volume is positively or negatively associated with the tested phenotypic or psychological score is reported. The number of splits (out of 100) in which the clusters were detected are noted in parentheses (i.e. % of splits with at least one significant cluster [in the respective direction]).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><bold>Healthy cohort</bold></th><th colspan="2">n_discovery&#160;=&#160;70% n_total</th><th colspan="2">n_discovery&#160;=&#160;50% n_total</th><th colspan="2">n_discovery&#160;=&#160;30% n_total</th></tr><tr><th>&#8195;# positively associated clusters (split%)</th><th># negatively associated clusters (split%)</th><th># positively associated clusters (split%)</th><th># negatively associated clusters (split%)</th><th># positively associated clusters (split%)</th><th># negatively associated clusters (split%)</th></tr></thead><tbody><tr><td>&#8195;Age (years) n-total&#160;=&#160;466</td><td>77 (54%)</td><td>154 (100%)</td><td>5 (4%)</td><td>522 (100%)</td><td>1 (1%)</td><td>1781 (100%)</td></tr><tr><td>&#8195;BMI (kg/m<sup>2</sup>) n-total&#160;=&#160;466</td><td>0</td><td>1741 (100%)</td><td>0</td><td>2276 (100%)</td><td>0</td><td>1937 (96%)</td></tr><tr><td>&#8195;Perceptual IQ (sum of t-scores) n-total&#160;=&#160;466</td><td>499 (83%)</td><td>0</td><td>256 (58%)</td><td>0</td><td>145 (33%)</td><td>0</td></tr><tr><td>&#8195;Word-context (# of consecutively correct) n-total&#160;=&#160;262</td><td>337 (80%)</td><td>0</td><td>159 (47%)</td><td>0</td><td>80 (21%)</td><td>0</td></tr><tr><td>&#8195;CWI (interference) (sec) n-total&#160;=&#160;449</td><td>0</td><td>163 (53%)</td><td>1 (1%)</td><td>122 (39%)</td><td>6 (1%)</td><td>60 (26%)</td></tr><tr><td><bold>Clinical cohort</bold></td><td colspan="2">-</td><td colspan="2">n_discovery&#160;=&#160;50% n_total</td><td colspan="2">-</td></tr><tr><td>&#8195;RAVLT (# total immediate recall)</td><td>-</td><td>-</td><td>309 (84%)</td><td>0</td><td>-</td><td>-</td></tr></tbody></table><table-wrap-foot><fn><p>Abbreviations: BMI: body mass index; IQ: intelligence quotient, CWI: color-word interference task; RAVLT: Rey auditory verbal learning task;</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-43478-v2.xml">elife-43478-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.43478.005</object-id><label>Table 1.</label><caption><title>Summary of inputs to A08a medial and lateral dendritic arbors from the first instar larval EM reconstruction.</title><p>Neurons with the most synapses to A08a medial and lateral arbors shown. Neurons with fewer synapses also show specificity for medial or lateral dendritic arbors.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><bold>A08a inputs (hemisegment)</bold></th><th colspan="2" valign="top"><bold>Pre-synapse number</bold></th><th rowspan="2" valign="top"><bold>A08a arbor targeted</bold></th></tr><tr><th valign="top">&#8195;Total</th><th valign="top">With A08a</th></tr></thead><tbody><tr><td valign="top">&#8195;dbd (A1L)</td><td valign="top">79</td><td valign="top">10</td><td valign="top">Medial only</td></tr><tr><td valign="top">&#8195;dbd (A1R)</td><td valign="top">78</td><td valign="top">13</td><td valign="top">Medial only</td></tr><tr><td valign="top">&#8195;A02d (A1L)</td><td valign="top">66</td><td valign="top">22</td><td valign="top">Medial only</td></tr><tr><td valign="top">&#8195;A02d (A1R)</td><td valign="top">63</td><td valign="top">8</td><td valign="top">Medial only</td></tr><tr><td valign="top">&#8195;A02l (A1L)</td><td valign="top">38</td><td valign="top">12</td><td valign="top">Lateral only</td></tr><tr><td valign="top">&#8195;A02l (A1R)</td><td valign="top">31</td><td valign="top">4</td><td valign="top">Lateral only</td></tr><tr><td valign="top">&#8195;A31x (A1L)</td><td valign="top">19</td><td valign="top">3</td><td valign="top">Lateral only</td></tr><tr><td valign="top">&#8195;A31x (A1R)</td><td valign="top">26</td><td valign="top">9</td><td valign="top">Lateral only</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-43696-v1.xml">elife-43696-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.43696.016</object-id><label>Table 1.</label><caption><title>Connection rates for MCs and NMCs recorded in different transgenic lines; related to <xref ref-type="fig" rid="fig4">Figure 4</xref>.</title><p>Left columns show paired recording data collected using the GIN and X94 lines to respectively target MCs and NMCs in L5. Right columns show the same data and additionally include data collected using the SST-TdT line, with L5 SST cells classified as putative MCs or NMCs based on their intrinsic properties. Columns not displaying p values show the number and fraction of SST-PC pairs in which a monosynaptic connection was detected for a given condition.</p></caption><table frame="hsides" rules="groups"><thead><tr><th/><th colspan="3">GIN&#160;+&#160;X94</th><th colspan="3">GIN&#160;+&#160;X94+classified SST-TdT</th><th rowspan="2">All <break/>SST-TdT</th></tr><tr><th/><th>MCs</th><th>NMCs</th><th>P</th><th>MCs</th><th>NMCs</th><th>P</th></tr></thead><tbody><tr><td>L5SST&#8594;L4PC</td><td>4/47 <break/>9%</td><td>21/34 <break/>62%</td><td>1.6 &#183; 10<sup>&#8722;4</sup></td><td>4/68 <break/>6%</td><td>36/67 <break/>54%</td><td>2.0 &#183; 10<sup>&#8722;5</sup></td><td>15/55 <break/>27%</td></tr><tr><td>L4PC&#8594;L5SST</td><td>0/50 <break/>0%</td><td>13/27 <break/>48%</td><td>5.8 &#183; 10<sup>&#8722;4</sup></td><td>1/95 <break/>1%</td><td>39/72 <break/>54%</td><td>&lt;10<sup>&#8722;5</sup></td><td>27/91 <break/>30%</td></tr><tr><td>L5SST&#8594;L5PC</td><td>19/38 <break/>50%</td><td>1/38 <break/>3%</td><td>2.8 &#183; 10<sup>&#8722;4</sup></td><td>24/46 <break/>52%</td><td>2/65 <break/>3%</td><td>&lt;10<sup>&#8722;5</sup></td><td>19/67 <break/>28%</td></tr><tr><td>L5PC&#8594;L5SST</td><td>2/22 <break/>9%</td><td>0/33 <break/>0%</td><td>0.1431</td><td>4/29 <break/>14%</td><td>1/60 <break/>2%</td><td>0.02</td><td>5/55 <break/>9%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-43696-v2.xml">elife-43696-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.43696.016</object-id><label>Table 1.</label><caption><title>Connection rates for MCs and NMCs recorded in different transgenic lines; related to <xref ref-type="fig" rid="fig4">Figure 4</xref>.</title><p>Left columns show paired recording data collected using the GIN and X94 lines to respectively target MCs and NMCs in L5. Right columns show the same data and additionally include data collected using the SST-TdT line, with L5 SST cells classified as putative MCs or NMCs based on their intrinsic properties. Columns not displaying p values show the number and fraction of SST-PC pairs in which a monosynaptic connection was detected for a given condition.</p></caption><table frame="hsides" rules="groups"><thead><tr><th/><th colspan="3">GIN&#160;+&#160;X94</th><th colspan="3">GIN&#160;+&#160;X94+classified SST-TdT</th><th rowspan="2">All <break/>SST-TdT</th></tr><tr><th/><th>MCs</th><th>NMCs</th><th>P</th><th>MCs</th><th>NMCs</th><th>P</th></tr></thead><tbody><tr><td>L5SST&#8594;L4PC</td><td>4/47 <break/>9%</td><td>21/34 <break/>62%</td><td>1.6 &#183; 10<sup>&#8722;4</sup></td><td>4/68 <break/>6%</td><td>36/67 <break/>54%</td><td>2.0 &#183; 10<sup>&#8722;5</sup></td><td>15/55 <break/>27%</td></tr><tr><td>L4PC&#8594;L5SST</td><td>0/50 <break/>0%</td><td>13/27 <break/>48%</td><td>5.8 &#183; 10<sup>&#8722;4</sup></td><td>1/95 <break/>1%</td><td>39/72 <break/>54%</td><td>&lt;10<sup>&#8722;5</sup></td><td>27/91 <break/>30%</td></tr><tr><td>L5SST&#8594;L5PC</td><td>19/38 <break/>50%</td><td>1/38 <break/>3%</td><td>2.8 &#183; 10<sup>&#8722;4</sup></td><td>24/46 <break/>52%</td><td>2/65 <break/>3%</td><td>&lt;10<sup>&#8722;5</sup></td><td>19/67 <break/>28%</td></tr><tr><td>L5PC&#8594;L5SST</td><td>2/22 <break/>9%</td><td>0/33 <break/>0%</td><td>0.1431</td><td>4/29 <break/>14%</td><td>1/60 <break/>2%</td><td>0.02</td><td>5/55 <break/>9%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-43788-v2.xml">elife-43788-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.43788.020</object-id><label>Table 3.</label><caption><title>Phenotypic analysis of mutant strains</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Genotype</th><th rowspan="2">RNAi</th><th>Wild-type</th><th>Complete tumorous</th><th rowspan="2">N</th></tr><tr><th>Germline, %</th><th>Germline, %</th></tr></thead><tbody><tr><td><italic>glp-1(ar202)</italic></td><td><italic>puf-8</italic></td><td>2</td><td>98</td><td>61</td></tr><tr><td><italic>glp-1(ar202)</italic></td><td>Scramble</td><td>100</td><td>0</td><td>95</td></tr><tr><td><italic>fbf-2(lot14) glp-1(ar202)</italic></td><td><italic>puf-8</italic></td><td>64</td><td>36</td><td>89</td></tr><tr><td><italic>fbf-2(lot14) glp-1(ar202)</italic></td><td>Scramble</td><td>100</td><td>0</td><td>107</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-43788-v3.xml">elife-43788-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.43788.020</object-id><label>Table 3.</label><caption><title>Phenotypic analysis of mutant strains</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Genotype</th><th rowspan="2">RNAi</th><th>Wild-type</th><th>Complete tumorous</th><th rowspan="2">N</th></tr><tr><th>Germline, %</th><th>Germline, %</th></tr></thead><tbody><tr><td><italic>glp-1(ar202)</italic></td><td><italic>puf-8</italic></td><td>2</td><td>98</td><td>61</td></tr><tr><td><italic>glp-1(ar202)</italic></td><td>Scramble</td><td>100</td><td>0</td><td>95</td></tr><tr><td><italic>fbf-2(lot14) glp-1(ar202)</italic></td><td><italic>puf-8</italic></td><td>64</td><td>36</td><td>89</td></tr><tr><td><italic>fbf-2(lot14) glp-1(ar202)</italic></td><td>Scramble</td><td>100</td><td>0</td><td>107</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-43999-v2.xml">elife-43999-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table5" position="float"><object-id pub-id-type="doi">10.7554/eLife.43999.018</object-id><label>Table 5.</label><caption><title>Number of cells (n) and number of mice (N) in each experiment.</title><p>For AAV-injected mice, only animals with bilateral hits are included.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="4" valign="top"><xref ref-type="fig" rid="fig1">Figure 1a, b</xref></th><th colspan="2" valign="top">Control</th><th colspan="2" valign="top">KERKO</th></tr><tr><th colspan="2" valign="top">Intact n&#160;=&#160;12, N&#160;=&#160;7</th><th colspan="2" valign="top">Intact n&#160;=&#160;11, N&#160;=&#160;6</th></tr><tr><th colspan="2" valign="top">OVX n&#160;=&#160;10, N&#160;=&#160;5</th><th colspan="2" valign="top">OVX n&#160;=&#160;11, N&#160;=&#160;4</th></tr><tr><th colspan="2" valign="top">OVX&#160;+&#160;E n&#160;=&#160;10, N&#160;=&#160;6</th><th colspan="2" valign="top">OVX&#160;+&#160;E n&#160;=&#160;9, N&#160;=&#160;5</th></tr></thead><tbody><tr><td rowspan="4" valign="top"><xref ref-type="fig" rid="fig1">Figure 1c&#8211;f</xref>, <break/><xref ref-type="fig" rid="fig1s1">Figure 1&#8212;figure supplement 1a</xref> left, 1b left</td><td colspan="2" valign="top">Control</td><td colspan="2" valign="top">KERKO</td></tr><tr><td colspan="2" valign="top">Intact n&#160;=&#160;11, N&#160;=&#160;4</td><td colspan="2" valign="top">Intact n&#160;=&#160;11, N&#160;=&#160;5</td></tr><tr><td colspan="2" valign="top">OVX n&#160;=&#160;11, N&#160;=&#160;5</td><td colspan="2" valign="top">OVX n&#160;=&#160;9, N&#160;=&#160;4</td></tr><tr><td colspan="2" valign="top">OVX&#160;+&#160;E n&#160;=&#160;11, N&#160;=&#160;7</td><td colspan="2" valign="top">OVX&#160;+&#160;E n&#160;=&#160;12, N&#160;=&#160;5</td></tr><tr><td rowspan="2" valign="top"><xref ref-type="fig" rid="fig2">Figure 2</xref></td><td colspan="2" valign="top">Control</td><td colspan="2" valign="top">KERKO</td></tr><tr><td colspan="2" valign="top">n&#160;=&#160;8, N&#160;=&#160;4</td><td colspan="2" valign="top">n&#160;=&#160;7, N&#160;=&#160;4</td></tr><tr><td rowspan="2" valign="top"><xref ref-type="fig" rid="fig3">Figure 3d,e</xref></td><td valign="top">AVPV-AAV-<italic>lacZ</italic></td><td colspan="2" valign="top">AVPV-AAV-<italic>Esr1</italic>g1</td><td valign="top">AVPV-AAV-<italic>Esr1</italic>g2</td></tr><tr><td valign="top">N&#160;=&#160;3</td><td colspan="2" valign="top">N&#160;=&#160;3</td><td valign="top">N&#160;=&#160;4</td></tr><tr><td rowspan="2" valign="top"><xref ref-type="fig" rid="fig3">Figure 3f</xref></td><td colspan="2" valign="top">AVPV-AAV- <italic>lacZ</italic></td><td colspan="2" valign="top">AVPV-AAV-<italic>Esr1</italic></td></tr><tr><td colspan="2" valign="top">N&#160;=&#160;6</td><td colspan="2" valign="top">N&#160;=&#160;8 (g1 N&#160;=&#160;4, g2 N&#160;=&#160;4)</td></tr><tr><td rowspan="2" valign="top"><xref ref-type="fig" rid="fig3">Figure 3g</xref></td><td colspan="2" valign="top">AVPV-AAV- <italic>lacZ</italic></td><td colspan="2" valign="top">AVPV-AAV-<italic>Esr1</italic></td></tr><tr><td colspan="2" valign="top">N&#160;=&#160;6</td><td colspan="2" valign="top">N&#160;=&#160;9 (g1 N&#160;=&#160;5, g2 N&#160;=&#160;4)</td></tr><tr><td rowspan="4" valign="top"><xref ref-type="fig" rid="fig4">Figure 4d&#8211;j</xref> and <break/><xref ref-type="fig" rid="fig1s1">Figure 1&#8212;figure supplement 1a</xref> middle, 1b middle</td><td colspan="2" valign="top">IF <italic>post hoc</italic></td><td colspan="2" valign="top">PCR <italic>post hoc</italic></td></tr><tr><td colspan="2" valign="top"><italic>Esr1</italic> n&#160;=&#160;15, N&#160;=&#160;5</td><td colspan="2" valign="top"><italic>Esr1</italic> n&#160;=&#160;10, N&#160;=&#160;4</td></tr><tr><td colspan="2" valign="top"><italic>lacZ</italic> n&#160;=&#160;14, N&#160;=&#160;4</td><td colspan="2" valign="top"><italic>lacZ</italic> n&#160;=&#160;9, N&#160;=&#160;3</td></tr><tr><td colspan="2" valign="top">uninfected n&#160;=&#160;8, N&#160;=&#160;4</td><td colspan="2" valign="top">uninfected n&#160;=&#160;4, N&#160;=&#160;2</td></tr><tr><td rowspan="2" valign="top"><xref ref-type="fig" rid="fig5">Figure 5a&#8211;d</xref></td><td valign="top">Arc-AAV-<italic>lacZ</italic></td><td colspan="2" valign="top">Arc-AAV-<italic>Esr1</italic>g1</td><td valign="top">Arc-AAV-<italic>Esr1</italic>g2</td></tr><tr><td valign="top">N&#160;=&#160;6</td><td colspan="2" valign="top">N&#160;=&#160;4</td><td valign="top">N&#160;=&#160;4</td></tr><tr><td rowspan="2" valign="top"><xref ref-type="fig" rid="fig5">Figure 5e&#8211;g</xref></td><td colspan="2" valign="top">Arc-AAV-<italic>lacZ</italic></td><td colspan="2" valign="top">Arc-AAV-<italic>Esr1</italic></td></tr><tr><td colspan="2" valign="top">N&#160;=&#160;6</td><td colspan="2" valign="top">N&#160;=&#160;8 (g1 N&#160;=&#160;4, g2 N&#160;=&#160;4)</td></tr><tr><td rowspan="2" valign="top"><xref ref-type="fig" rid="fig6">Figure 6a&#8211;c</xref></td><td colspan="2" valign="top">Arc-AAV-<italic>lacZ</italic></td><td colspan="2" valign="top">Arc-AAV-<italic>Esr1</italic></td></tr><tr><td colspan="2" valign="top">n&#160;=&#160;11, N&#160;=&#160;5</td><td colspan="2" valign="top">n&#160;=&#160;13, N&#160;=&#160;5</td></tr><tr><td rowspan="2" valign="top"><xref ref-type="fig" rid="fig6">Figure 6d&#8211;f</xref></td><td colspan="2" valign="top">Arc-AAV-<italic>lacZ</italic></td><td colspan="2" valign="top">Arc-AAV- <italic>Esr1</italic></td></tr><tr><td colspan="2" valign="top">n&#160;=&#160;10, N&#160;=&#160;5</td><td colspan="2" valign="top">n&#160;=&#160;12, N&#160;=&#160;5</td></tr><tr><td rowspan="2" valign="top"><xref ref-type="fig" rid="fig1s1">Figure 1&#8212;figure supplement 1a</xref> left, 1b right</td><td colspan="2" valign="top">KERKO</td><td colspan="2" valign="top">AVPV-AAV-<italic>Esr1</italic></td></tr><tr><td colspan="2" valign="top">n&#160;=&#160;12, N&#160;=&#160;5</td><td colspan="2" valign="top">n&#160;=&#160;25, N&#160;=&#160;9</td></tr><tr><td rowspan="4" valign="top"><xref ref-type="fig" rid="fig1s1">Figure 1&#8212;figure supplement 1c</xref> left</td><td colspan="2" valign="top">Control</td><td colspan="2" valign="top">KERKO</td></tr><tr><td colspan="2" valign="top">Intact N&#160;=&#160;6</td><td colspan="2" valign="top">Intact n&#160;=&#160;11, N&#160;=&#160;7</td></tr><tr><td colspan="2" valign="top">OVX N&#160;=&#160;6</td><td colspan="2" valign="top">OVX n&#160;=&#160;11, N&#160;=&#160;6</td></tr><tr><td colspan="2" valign="top">OVX&#160;+&#160;E N&#160;=&#160;5</td><td colspan="2" valign="top">OVX&#160;+&#160;E n&#160;=&#160;9, N&#160;=&#160;7</td></tr><tr><td rowspan="2" valign="top"><xref ref-type="fig" rid="fig1s1">Figure 1&#8212;figure supplement 1c</xref> middle</td><td colspan="2" valign="top">AVPV-AAV-<italic>lacZ</italic></td><td colspan="2" valign="top">AVPV-AAV-<italic>Esr1</italic></td></tr><tr><td colspan="2" valign="top">N&#160;=&#160;7</td><td colspan="2" valign="top">N&#160;=&#160;9</td></tr><tr><td rowspan="2" valign="top"><xref ref-type="fig" rid="fig1s1">Figure 1&#8212;figure supplement 1c</xref> middle</td><td colspan="2" valign="top">Arc-AAV-<italic>lacZ</italic></td><td colspan="2" valign="top">Arc-AAV-<italic>Esr1</italic></td></tr><tr><td colspan="2" valign="top">N&#160;=&#160;5</td><td colspan="2" valign="top">N&#160;=&#160;5</td></tr><tr><td rowspan="2" valign="top"><xref ref-type="fig" rid="fig4s1">Figure 4&#8212;figure supplement 1</xref></td><td colspan="2" valign="top">AVPV-AAV-<italic>lacZ</italic></td><td colspan="2" valign="top">AVPV-AAV-<italic>Esr1</italic></td></tr><tr><td colspan="2" valign="top">n&#160;=&#160;16, N&#160;=&#160;5</td><td colspan="2" valign="top">n&#160;=&#160;23, N&#160;=&#160;5 (g1 N&#160;=&#160;3, g2 N&#160;=&#160;2)</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-44310-v1.xml">elife-44310-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.44310.002</object-id><label>Table 1.</label><caption><title>Clinicopathological characteristics of CM patients from TCGA database.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Characteristics</th><th rowspan="3">Groups</th><th colspan="6">Patients</th></tr><tr><th colspan="2">Total <break/>(<italic>N</italic>&#160;=&#160;461)</th><th colspan="2">Training cohort <break/>(<italic>N</italic>&#160;=&#160;307)</th><th colspan="2">Validation cohort <break/>(<italic>N</italic>&#160;=&#160;154)</th></tr><tr><th>No</th><th>%</th><th>No</th><th>%</th><th>No</th><th>%</th></tr></thead><tbody><tr><td rowspan="2" valign="top">Sex</td><td valign="top">Male</td><td valign="top">286</td><td valign="top">62.04</td><td valign="top">195</td><td valign="top">63.52</td><td valign="top">89</td><td valign="top">57.79</td></tr><tr><td valign="top">Female</td><td valign="top">175</td><td valign="top">37.96</td><td valign="top">112</td><td valign="top">36.48</td><td valign="top">65</td><td valign="top">42.21</td></tr><tr><td rowspan="4" valign="top">Age at diagnosis</td><td valign="top">Median</td><td valign="top">58</td><td valign="top"/><td valign="top">58</td><td valign="top"/><td valign="top">58</td><td valign="top"/></tr><tr><td valign="top">Range</td><td valign="top">15&#8211;90</td><td valign="top"/><td valign="top">15&#8211;90</td><td valign="top"/><td valign="top">19&#8211;90</td><td valign="top"/></tr><tr><td valign="top">&#8804;58</td><td valign="top">234</td><td valign="top">50.76</td><td valign="top">154</td><td valign="top">50.16</td><td valign="top">80</td><td valign="top">51.95</td></tr><tr><td valign="top">&gt;58</td><td valign="top">227</td><td valign="top">49.24</td><td valign="top">153</td><td valign="top">49.84</td><td valign="top">74</td><td valign="top">48.05</td></tr><tr><td rowspan="5" valign="top">Tumor tissue site</td><td valign="top">Primary tumor</td><td valign="top">104</td><td valign="top">22.56</td><td valign="top">76</td><td valign="top">24.76</td><td valign="top">28</td><td valign="top">18.18</td></tr><tr><td valign="top">Regional cutaneous or subcutaneous tissue</td><td valign="top">73</td><td valign="top">15.84</td><td valign="top">51</td><td valign="top">16.61</td><td valign="top">22</td><td valign="top">14.29</td></tr><tr><td valign="top">Regional lymph node metastasis</td><td valign="top">216</td><td valign="top">46.85</td><td valign="top">154</td><td valign="top">50.16</td><td valign="top">62</td><td valign="top">40.26</td></tr><tr><td valign="top">Distant metastasis</td><td valign="top">65</td><td valign="top">14.10</td><td valign="top">23</td><td valign="top">7.49</td><td valign="top">42</td><td valign="top">27.27</td></tr><tr><td valign="top">Unknown</td><td valign="top">3</td><td valign="top">0.65</td><td valign="top">3</td><td valign="top">0.98</td><td valign="top">0</td><td valign="top">0.00</td></tr><tr><td rowspan="6" valign="top">Pathologic stage</td><td valign="top">0</td><td valign="top">6</td><td valign="top">1.30</td><td valign="top">5</td><td valign="top">1.63</td><td valign="top">1</td><td valign="top">0.65</td></tr><tr><td valign="top">I</td><td valign="top">75</td><td valign="top">16.27</td><td valign="top">53</td><td valign="top">17.26</td><td valign="top">22</td><td valign="top">14.29</td></tr><tr><td valign="top">II</td><td valign="top">139</td><td valign="top">30.15</td><td valign="top">92</td><td valign="top">29.97</td><td valign="top">47</td><td valign="top">30.52</td></tr><tr><td valign="top">III</td><td valign="top">171</td><td valign="top">37.09</td><td valign="top">117</td><td valign="top">38.11</td><td valign="top">54</td><td valign="top">35.06</td></tr><tr><td valign="top">IV</td><td valign="top">23</td><td valign="top">4.99</td><td valign="top">13</td><td valign="top">4.23</td><td valign="top">10</td><td valign="top">6.49</td></tr><tr><td valign="top">Unknown</td><td valign="top">47</td><td valign="top">10.20</td><td valign="top">27</td><td valign="top">8.79</td><td valign="top">20</td><td valign="top">12.99</td></tr><tr><td rowspan="4" valign="top">Anatomic site</td><td valign="top">Head and neck</td><td valign="top">36</td><td valign="top">7.81</td><td valign="top">21</td><td valign="top">6.84</td><td valign="top">15</td><td valign="top">9.74</td></tr><tr><td valign="top">Extremity</td><td valign="top">194</td><td valign="top">42.08</td><td valign="top">129</td><td valign="top">42.02</td><td valign="top">65</td><td valign="top">42.21</td></tr><tr><td valign="top">Trunk</td><td valign="top">167</td><td valign="top">36.23</td><td valign="top">117</td><td valign="top">38.11</td><td valign="top">50</td><td valign="top">32.47</td></tr><tr><td valign="top">Others/Unknown</td><td valign="top">64</td><td valign="top">13.88</td><td valign="top">40</td><td valign="top">13.03</td><td valign="top">24</td><td valign="top">15.58</td></tr><tr><td rowspan="4" valign="top">Breslow thickness (mm) <break/><break/></td><td valign="top">&lt;2</td><td valign="top">126</td><td valign="top">27.33</td><td valign="top">85</td><td valign="top">27.69</td><td valign="top">41</td><td valign="top">26.62</td></tr><tr><td valign="top">2&#8211;5</td><td valign="top">124</td><td valign="top">26.90</td><td valign="top">79</td><td valign="top">25.73</td><td valign="top">45</td><td valign="top">29.22</td></tr><tr><td valign="top">&gt;5</td><td valign="top">106</td><td valign="top">22.99</td><td valign="top">78</td><td valign="top">25.41</td><td valign="top">28</td><td valign="top">18.18</td></tr><tr><td valign="top">Unknown</td><td valign="top">105</td><td valign="top">22.78</td><td valign="top">65</td><td valign="top">21.17</td><td valign="top">40</td><td valign="top">25.97</td></tr><tr><td rowspan="3" valign="top">Ulceration <break/></td><td valign="top">Present</td><td valign="top">167</td><td valign="top">36.23</td><td valign="top">120</td><td valign="top">39.09</td><td valign="top">47</td><td valign="top">30.52</td></tr><tr><td valign="top">Absent</td><td valign="top">145</td><td valign="top">31.45</td><td valign="top">100</td><td valign="top">32.57</td><td valign="top">45</td><td valign="top">29.22</td></tr><tr><td valign="top">NA/Unknown</td><td valign="top">149</td><td valign="top">32.32</td><td valign="top">87</td><td valign="top">28.34</td><td valign="top">62</td><td valign="top">40.26</td></tr><tr><td rowspan="3" valign="top">Chemotherapy</td><td valign="top">Yes</td><td valign="top">123</td><td valign="top">26.68</td><td valign="top">70</td><td valign="top">22.80</td><td valign="top">53</td><td valign="top">34.42</td></tr><tr><td valign="top">NO</td><td valign="top">319</td><td valign="top">69.20</td><td valign="top">227</td><td valign="top">73.94</td><td valign="top">92</td><td valign="top">59.74</td></tr><tr><td valign="top">Unknown</td><td valign="top">19</td><td valign="top">4.12</td><td valign="top">10</td><td valign="top">3.26</td><td valign="top">9</td><td valign="top">5.84</td></tr><tr><td rowspan="2" valign="top">Vital Status</td><td valign="top">Alive</td><td valign="top">241</td><td valign="top">52.28</td><td valign="top">167</td><td valign="top">54.40</td><td valign="top">74</td><td valign="top">48.05</td></tr><tr><td valign="top">Dead</td><td valign="top">220</td><td valign="top">47.72</td><td valign="top">140</td><td valign="top">45.60</td><td valign="top">80</td><td valign="top">51.95</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-44558-v2.xml">elife-44558-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.44558.012</object-id><label>Table 1.</label><caption><title>Publically available data for strong candidate F-box phase regulators.</title><p>Circadian expression data is from the Diurnal Project gene expression tool (<xref ref-type="bibr" rid="bib76">Mockler et al., 2007</xref>). Tissue specific expression is from the Arabidopsis eFP browser (<xref ref-type="bibr" rid="bib57">Klepikova et al., 2016</xref>; <xref ref-type="bibr" rid="bib110">Winter et al., 2007</xref>). The closest homolog was determined by WU-BLAST2 using BLASTP and the Araport11 protein sequences database. N/A indicates that data is not available.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Locus ID</th><th rowspan="3">Gene <break/>name</th><th colspan="8">Circadian expression</th><th colspan="2" rowspan="2">Tissue specific expression</th><th colspan="2" rowspan="2">Closest homolog</th><th rowspan="3">Publications (PMID)</th></tr><tr><th colspan="2">LDHC</th><th colspan="2">LL_LDHC</th><th colspan="2">LL12_LDHH</th><th colspan="2">LL23_LDHH</th></tr><tr><th>Phase</th><th>Correlation</th><th>Phase</th><th>Correlation</th><th>Phase</th><th>Correlation</th><th>Phase</th><th>Correlation</th><th>Developmental&#160;map (eFP)</th><th>Klepikova atlas</th><th>Locus</th><th>E-Value</th></tr></thead><tbody><tr><td valign="top"><italic>AT5G44980</italic></td><td valign="top"><italic>ACF1</italic></td><td>16</td><td>0.50</td><td>23</td><td>0.38</td><td>14</td><td>0.69</td><td>4</td><td>0.43</td><td valign="bottom">Global, <break/>especially <break/>senescent leaf</td><td valign="bottom">Global, <break/>stamen</td><td valign="bottom"><italic>AT5G44960</italic></td><td valign="bottom">1.60 &#215; 10<sup>&#8722;146</sup></td><td>N/A</td></tr><tr><td valign="top"><italic>AT5G48980</italic></td><td valign="top"><italic>ACF2</italic></td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td valign="bottom">Silique with <break/>seed, seed</td><td valign="bottom"><italic>AT5G48990</italic></td><td valign="bottom">1.10 &#215; 10<sup>&#8722;124</sup></td><td>N/A</td></tr><tr><td valign="top"><italic>AT5G42350</italic></td><td valign="top"><italic>CFK1</italic></td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td valign="bottom">Global, <break/>especially <break/>pollen</td><td valign="bottom">Global</td><td valign="bottom"><italic>AT5G42360</italic></td><td valign="bottom">3.60 &#215; 10<sup>&#8722;298</sup></td><td><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/pubmed/23475998">23475998</ext-link></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-44558-v2.xml">elife-44558-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.44558.016</object-id><label>Table 2.</label><caption><title>Publically available data for strong candidate F-box period regulators.</title><p>Circadian expression data is from the Diurnal Project gene expression tool (<xref ref-type="bibr" rid="bib76">Mockler et al., 2007</xref>). Tissue specific expression is from the Arabidopsis eFP browser (<xref ref-type="bibr" rid="bib57">Klepikova et al., 2016</xref>; <xref ref-type="bibr" rid="bib110">Winter et al., 2007</xref>). The closest homolog was determined by WU-BLAST2 using BLASTP and the Araport11 protein sequences database. N/A indicates that data is not available.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Locus ID</th><th rowspan="3">Gene <break/>name</th><th colspan="8">Circadian expression</th><th colspan="2" rowspan="2">Tissue specific expression</th><th colspan="2" rowspan="2">Closest homolog</th><th rowspan="3">Publications (PMID)</th></tr><tr><th colspan="2">LDHC</th><th colspan="2">LL_LDHC</th><th colspan="2">LL12_LDHH</th><th colspan="2">LL23_LDHH</th></tr><tr><th>Phase</th><th>Correlation</th><th>Phase</th><th>Correlation</th><th>Phase</th><th>Correlation</th><th>Phase</th><th>Correlation</th><th>Developmental map (eFP)</th><th>Klepikova atlas</th><th>Locus</th><th>E-Value</th></tr></thead><tbody><tr><td><bold><italic>AT1G20800</italic></bold></td><td><italic>ACF3</italic></td><td>18</td><td>0.51</td><td>4</td><td>0.66</td><td>15</td><td>0.65</td><td>16</td><td>0.66</td><td>N/A</td><td>Young flower, <break/>stamen, ovule</td><td><italic>AT1G20803</italic></td><td>1.6 &#215; 10<sup>&#8722;87</sup></td><td>N/A</td></tr><tr><td><bold><italic>AT2G44030</italic></bold></td><td><italic>ACF4</italic></td><td>18</td><td>0.73</td><td>5</td><td>0.54</td><td>6</td><td>0.62</td><td>22</td><td>0.79</td><td>Seed, pollen</td><td>Young <break/>stamen, <break/>flower buds</td><td><italic>AT3G46050</italic></td><td>2.5 &#215; 10<sup>&#8722;59</sup></td><td>N/A</td></tr><tr><td><bold><italic>AT1G50870</italic></bold></td><td><italic>ACF5</italic></td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td/><td><italic>AT1G47790</italic></td><td>1.2 &#215; 10<sup>&#8722;102</sup></td><td>N/A</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-44558-v2.xml">elife-44558-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.44558.020</object-id><label>Table 3.</label><caption><title>Publically available data for strong candidate U-box period regulators.</title><p>Circadian expression data is from the Diurnal Project gene expression tool (<xref ref-type="bibr" rid="bib76">Mockler et al., 2007</xref>). Tissue specific expression is from the Arabidopsis eFP browser (<xref ref-type="bibr" rid="bib57">Klepikova et al., 2016</xref>; <xref ref-type="bibr" rid="bib110">Winter et al., 2007</xref>). The closest homolog was determined by WU-BLAST2 using BLASTP and the Araport11 protein sequences database.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Locus ID</th><th rowspan="3">Gene name</th><th colspan="8">Circadian expression</th><th colspan="2" rowspan="2">Tissue specific expression</th><th colspan="2" rowspan="2">Closest homolog</th><th rowspan="3">Publications (PMID)</th></tr><tr><th colspan="2">LDHC</th><th colspan="2">LL_LDHC</th><th colspan="2">LL12_LDHH</th><th colspan="2">LL23_LDHH</th></tr><tr><th>Phase</th><th>Correlation</th><th>Phase</th><th>Correlation</th><th>Phase</th><th>Correlation</th><th>Phase</th><th>Correlation</th><th>Developmental map (eFP)</th><th>Klepikova atlas</th><th>Locus</th><th>E-Value</th></tr></thead><tbody><tr><td><bold><italic>AT5G18340</italic></bold></td><td><italic>PUB48</italic></td><td>19</td><td>0.68</td><td>17</td><td>0.86</td><td>20</td><td>0.63</td><td>14</td><td>0.86</td><td>Global, especially <break/>young flower <break/>and senescent <break/>leaf</td><td>Young flower <break/>bud, <break/>Stamen, mature <break/>leaf mature <break/>petiole</td><td><italic>AT5G18320</italic></td><td>3.8 &#215; 10<sup>&#8722;127</sup></td><td><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/pubmed/28077082">28077082</ext-link></td></tr><tr><td><bold><italic>AT1G04510</italic></bold></td><td><italic>MAC3A</italic></td><td>20</td><td>0.84</td><td>18</td><td>0.72</td><td>23</td><td>0.48</td><td>14</td><td>0.83</td><td>Global</td><td>Global</td><td><italic>AT2G33340</italic></td><td>1.2 &#215; 10<sup>&#8722;233</sup></td><td><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/pubmed/29437988">29437988</ext-link> <break/><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/pubmed/28947490">28947490</ext-link> <break/><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/pubmed/19629177">19629177</ext-link></td></tr><tr><td><bold><italic>AT2G33340</italic></bold></td><td><italic>MAC3B</italic></td><td>15</td><td>0.49</td><td>8</td><td>0.56</td><td>17</td><td>0.80</td><td>8</td><td>0.73</td><td>Global</td><td>Global</td><td><italic>AT1G04510</italic></td><td>7.2 &#215; 10<sup>&#8722;229</sup></td><td><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/pubmed/29437988">29437988</ext-link> <break/><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/pubmed/28947490">28947490</ext-link> <break/><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/pubmed/19629177">19629177</ext-link></td></tr><tr><td><bold><italic>AT3G49810</italic></bold></td><td><italic>PUB30</italic></td><td>2</td><td>0.87</td><td>1</td><td>0.59</td><td>3</td><td>0.76</td><td>0</td><td>0.91</td><td>Global</td><td>Everywhere <break/>except young <break/>seed and <break/>young silique</td><td><italic>AT5G65920</italic></td><td>1.2 &#215; 10<sup>&#8722;171</sup></td><td><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/pubmed/28865087">28865087</ext-link> <break/><ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/pubmed/25410251">25410251</ext-link></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-44760-v2.xml">elife-44760-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.44760.007</object-id><label>Table 1.</label><caption><title>Interaction distances in cPLA<sub>2</sub>&#945; C2-domain.&#160;</title><p>Interaction distances (&#197;) associated with bound calcium in the lipid-free cPLA<sub>2</sub>&#945; C2-domain structure (2.4&#160;&#197;&#160;resolution; PDB 1RLW) of&#160;<xref ref-type="bibr" rid="bib53">Perisic et al. (1998)</xref>&#160;and with bound calcium and DHPC in the C2-domain&#8211;DHPC crystal complex (2.2&#160;&#197;&#160;resolution) of the present study.</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left" valign="top"><bold>1RLW</bold></th><th colspan="2" valign="top">Ca1</th><th valign="top">Ca4</th></tr></thead><tbody><tr><td align="left" valign="top">&#8195;Asp40</td><td colspan="2" valign="top">2.3/3.4</td><td valign="top">2.3</td></tr><tr><td align="left" valign="top">&#8195;Asp43</td><td colspan="2" valign="top">2.1</td><td valign="top">2.6/2.2</td></tr><tr><td align="left" valign="top">&#8195;Asp93</td><td colspan="2" valign="top">2.7/2.5</td><td valign="top"/></tr><tr><td align="left" valign="top">&#8195;Asn65</td><td colspan="2" valign="top"/><td valign="top">2.1</td></tr><tr><td align="left" valign="top">&#8195;Asn95</td><td colspan="2" valign="top">2.2</td><td valign="top"/></tr><tr><th align="left" rowspan="2" valign="middle"><bold>C2/DHPC</bold></th><th align="center" colspan="4" valign="top">C2&#65279;&#160;</th><th colspan="6" valign="top">DHPC</th></tr><tr><th valign="top">Ca1</th><th colspan="2" valign="top">Ca4</th><th valign="top">Ca<sup>PC</sup></th><th valign="top">N<sup>+</sup>(CH<sub>3</sub>)<sub>3</sub></th><th valign="top">PO<sub>4</sub></th><th valign="top"><italic>sn</italic>-2 C=O</th><th valign="top"><italic>sn</italic>-1 C=O</th><th colspan="2" valign="top"><italic>sn</italic>-1 chain</th></tr><tr><td align="left" valign="top">&#8195;Asp40</td><td valign="top">2.4/3.4</td><td colspan="2" valign="top">2.4</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td colspan="2" valign="top"/></tr><tr><td align="left" valign="top">&#8195;Asp43</td><td valign="top">2.3</td><td colspan="2" valign="top">2.6/2.7</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td colspan="2" valign="top"/></tr><tr><td align="left" valign="top">&#8195;Asp93</td><td valign="top">2.7/2.8</td><td colspan="2" valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td colspan="2" valign="top"/></tr><tr><td align="left" valign="top">&#8195;Asn65</td><td valign="top"/><td colspan="2" valign="top">2.4</td><td valign="top">3.1</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td colspan="2" valign="top"/></tr><tr><td align="left" valign="top">&#8195;Asn95</td><td valign="top">2.3</td><td colspan="2" valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td colspan="2" valign="top"/></tr><tr><td align="left" valign="top">&#8195;Tyr96</td><td valign="top"/><td colspan="2" valign="top"/><td valign="top"/><td valign="top">~4.0</td><td valign="top"/><td valign="top"/><td valign="top"/><td colspan="2" valign="top"/></tr><tr><td align="left" valign="top">&#8195;Ala94</td><td valign="top"/><td colspan="2" valign="top"/><td valign="top"/><td valign="top">3.6</td><td valign="top"/><td valign="top"/><td valign="top"/><td colspan="2" valign="top"/></tr><tr><td align="left" valign="top">&#8195;His62</td><td valign="top"/><td colspan="2" valign="top"/><td valign="top"/><td valign="top">~5.0</td><td valign="top">~8.5</td><td valign="top"/><td valign="top"/><td colspan="2" valign="top"/></tr><tr><td align="left" valign="top">&#8195;Asn64</td><td valign="top"/><td colspan="2" valign="top"/><td valign="top"/><td valign="top">3.6</td><td valign="top"/><td valign="top"/><td valign="top"/><td colspan="2" valign="top"/></tr><tr><td align="left" valign="top">&#8195;Leu39</td><td valign="top"/><td colspan="2" valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td colspan="2" valign="top">5.4</td></tr><tr><td align="left" valign="top">&#8195;Ca1</td><td valign="top"/><td colspan="2" valign="top"/><td valign="top"/><td valign="top">5.5</td><td valign="top">5.7</td><td valign="top"/><td valign="top"/><td colspan="2" valign="top"/></tr><tr><td align="left" valign="top">&#8195;Ca4</td><td valign="top"/><td colspan="2" valign="top"/><td valign="top"/><td valign="top">6.3</td><td valign="top">2.1</td><td valign="top"/><td valign="top"/><td colspan="2" valign="top"/></tr><tr><td align="left" valign="top">&#8195;Ca<sup>PC</sup></td><td valign="top"/><td colspan="2" valign="top"/><td valign="top"/><td valign="top">8.5</td><td valign="top">3.6</td><td valign="top">3.2</td><td valign="top">3.1</td><td colspan="2" valign="top"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-44838-v1.xml">elife-44838-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.44838.010</object-id><label>Table 1.</label><caption><title>Comparison of different models fitted to the animals&#8217; choices.</title><p>Best fitting model indicated in bold.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Model</th><th rowspan="2" valign="top">Description</th><th colspan="2" valign="top">Both animals</th><th colspan="2" valign="top">Animal A</th><th colspan="2" valign="top">Animal B</th></tr><tr><th valign="top">AIC</th><th valign="top">BIC</th><th valign="top">AIC</th><th valign="top">BIC</th><th valign="top">AIC</th><th valign="top">BIC</th></tr></thead><tbody><tr><td valign="top">(1)</td><td valign="top">Value from reward history<sup>1</sup></td><td valign="top">2.2482</td><td valign="top">2.2490</td><td valign="top">1.5077</td><td valign="top">1.5084</td><td valign="top">7.3571</td><td valign="top">7.3636</td></tr><tr><td valign="top">(2)</td><td valign="top">Value from reward history and risk<sup>2</sup></td><td valign="top">2.2477</td><td valign="top">2.2492</td><td valign="top">1.5077</td><td valign="top">1.5092</td><td valign="top">7.3522</td><td valign="top">7.3653</td></tr><tr><td valign="top">(3)</td><td valign="top">Value from choice history<sup>3</sup></td><td valign="top">2.1614</td><td valign="top">2.1622</td><td valign="top">1.4900</td><td valign="top">1.4907</td><td valign="top">6.5043</td><td valign="top">6.5109</td></tr><tr><td valign="top">(4)</td><td valign="top">Value from choice history and risk</td><td valign="top">2.0385</td><td valign="top">2.0400</td><td valign="top">1.4023</td><td valign="top">1.4037</td><td valign="top">7.3528</td><td valign="top">7.3660</td></tr><tr><td valign="top">(5)</td><td valign="top">Value from reward and choice history<sup>4</sup></td><td valign="top">2.0089</td><td valign="top">2.0097</td><td valign="top">1.3914</td><td valign="top">1.3922</td><td valign="top">6.0880</td><td valign="top">6.0945</td></tr><tr><td valign="top"><bold>(6)</bold></td><td valign="top"><bold>Value from reward and choice history and risk</bold></td><td valign="top"><bold>2.0073</bold></td><td valign="top"><bold>2.0088</bold></td><td valign="top"><bold>1.3899</bold></td><td valign="top"><bold>1.3914</bold></td><td valign="top"><bold>6.0747</bold></td><td valign="top"><bold>6.0878</bold></td></tr><tr><td valign="top">(7)</td><td valign="top">Objective reward probabilities<sup>5</sup></td><td valign="top">2.1213</td><td valign="top">2.1220</td><td valign="top">1.4615</td><td valign="top">1.4622</td><td valign="top">6.4972</td><td valign="top">6.5037</td></tr><tr><td valign="top">(8)</td><td valign="top">Objective reward probabilities and objective risk<sup>6</sup></td><td valign="top">2.1210</td><td valign="top">2.1225</td><td valign="top">1.4616</td><td valign="top">1.4631</td><td valign="top">6.4982</td><td valign="top">6.5114</td></tr><tr><td valign="top">(9)</td><td valign="top">Reinforcement learning (RL) model<sup>7</sup></td><td valign="top">2.0763</td><td valign="top">2.0779</td><td valign="top">1.4376</td><td valign="top">1.4391</td><td valign="top">6.2161</td><td valign="top">6.2293</td></tr><tr><td valign="top">(10)</td><td valign="top">RL learning, stack parameter (<xref ref-type="bibr" rid="bib25">Huh et al., 2009</xref>)<sup>8</sup></td><td valign="top">2.0810</td><td valign="top">2.0826</td><td valign="top">1.4374</td><td valign="top">1.4389</td><td valign="top">6.3198</td><td valign="top">6.3330</td></tr><tr><td valign="top">(11)</td><td valign="top">RL, reversal-learning variant<sup>9</sup></td><td valign="top">2.2614</td><td valign="top">2.2630</td><td valign="top">1.5330</td><td valign="top">1.5344</td><td valign="top">7.2808</td><td valign="top">7.2939</td></tr></tbody></table><table-wrap-foot><fn><p>1:Value defined according to <xref ref-type="disp-formula" rid="equ6">Equation 6</xref>; 2: Risk defined according to <xref ref-type="disp-formula" rid="equ8">Equation 8</xref>; 3: Value defined as sum of weighted choice history derived from <xref ref-type="disp-formula" rid="equ5">Equation 5</xref>; 4: Value defined according to <xref ref-type="disp-formula" rid="equ7">Equation 7</xref>; 5: Objective reward probabilities defined according to <xref ref-type="disp-formula" rid="equ1">Equation 1</xref>; 6: Objective reward risk defined according to <xref ref-type="disp-formula" rid="equ2">Equation 2</xref>; 7: Standard Rescorla-Wagner RL model updating value of chosen option based on last outcome; 8: Modified RL model incorporating choice-dependency; 9: Modified RL model updating value of chosen and unchosen option based on last outcome.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-44927-v1.xml">elife-44927-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.44927.013</object-id><label>Table 1.</label><caption><title>Vigilance state parameters recorded from OF (KI/KI) and WT animals.</title><p>The table shows total time spent in individual states in minutes, duration of respective states in seconds and number of episodes (bouts) observed in either 24 hr, or only in light or dark period in OF (KI/KI, n&#160;=&#160;7 mice; WT,&#160;n&#160;=&#160;8 mice). *, p&lt;<italic>0.05</italic>. The p values were determined using two-tailed unpaired student&#8217;s <italic>t</italic>-test. Values are represented as the mean&#160;&#177;&#160;SEM.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th rowspan="2"/><th colspan="2">REM</th><th colspan="2">Cataplexy</th><th colspan="2">NREM</th><th colspan="2">Wake</th></tr><tr><th>Orexin-Flp (KI/KI)</th><th>WT</th><th>Orexin-Flp (KI/KI)</th><th>WT</th><th>Orexin-Flp (KI/KI)</th><th>WT</th><th>Orexin-Flp (KI/KI)</th><th>WT</th></tr></thead><tbody><tr><td rowspan="3">24 hr</td><td>Total time (min)</td><td>96.0 &#177; 5.7</td><td>92.2 &#177; 4.6</td><td>18.7 &#177; 6.3</td><td/><td>584.6 &#177; 19.1</td><td>555.7 &#177; 14.8</td><td>740.7 &#177; 18.3</td><td>792.1 &#177; 19.1</td></tr><tr><td>Duration (sec)</td><td>55.7 &#177; 4.2</td><td>57.3 &#177; 2.4</td><td>61.5 &#177; 12.1</td><td/><td>81.4 &#177; 7.0</td><td>94.8 &#177; 5.8</td><td>102.1 &#177; 8.4*</td><td>189.9 &#177; 16.8</td></tr><tr><td>Bouts</td><td>110.7 &#177; 8.5</td><td>96.8 &#177; 7.5</td><td>17.4 &#177; 5.2</td><td/><td>442.3 &#177; 37.4*</td><td>359.3 &#177; 20.8</td><td>453.0 &#177; 36.4*</td><td>358.6 &#177; 20.8</td></tr><tr><td rowspan="3">Light period</td><td>Total time (min)</td><td>57.3 &#177; 3.8*</td><td>68.4 &#177; 2.9</td><td>2.1 &#177; 0.8</td><td/><td>355.1 &#177; 13.5*</td><td>390.8 &#177; 4.9</td><td>305.5 &#177; 14.6*</td><td>260.8 &#177; 7.2</td></tr><tr><td>Duration (sec)</td><td>48.4 &#177; 3.9*</td><td>58.9 &#177; 1.8</td><td>48.9 &#177; 13.0</td><td/><td>93.4 &#177; 7.5</td><td>94.6 &#177; 5.0</td><td>80.1 &#177; 4.6*</td><td>63.1 &#177; 4.2</td></tr><tr><td>Bouts</td><td>73.4 &#177; 7.6</td><td>70.0 &#177; 4.4</td><td>2.0 &#177; 0.7</td><td/><td>234.6 &#177; 17.3</td><td>251.8 &#177; 14.1</td><td>231.0 &#177; 17.6</td><td>251.1 &#177; 13.9</td></tr><tr><td rowspan="3">Dark period</td><td>Total time (min)</td><td>38.7 &#177; 4.8*</td><td>23.8 &#177; 2.5</td><td>16.6 &#177; 5.6</td><td/><td>229.5 &#177; 12.4*</td><td>164.9 &#177; 10.5</td><td>435.2 &#177; 11.3*</td><td>531.3 &#177; 12.7</td></tr><tr><td>Duration (sec)</td><td>63.0 &#177; 6.2</td><td>55.6 &#177; 4.0</td><td>74.1 &#177; 19.8</td><td/><td>69.3 &#177; 8.2*</td><td>94.9 &#177; 7.6</td><td>124.0 &#177; 12.7*</td><td>316.6 &#177; 30.9</td></tr><tr><td>Bouts</td><td>37.3 &#177; 4.3*</td><td>26.8 &#177; 3.5</td><td>15.4 &#177; 4.7</td><td/><td>207.7 &#177; 21.1*</td><td>107.5 &#177; 10.5</td><td>222.0 &#177; 20.6*</td><td>107.5 &#177; 10.6</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-44934-v2.xml">elife-44934-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.44934.013</object-id><label>Table 1.</label><caption><title>Parameters for the best fits of the <inline-formula><mml:math id="inf300"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:msub><mml:mi>Q</mml:mi><mml:mrow><mml:mrow><mml:mi mathvariant="normal">C</mml:mi></mml:mrow></mml:mrow></mml:msub><mml:mrow><mml:mo>(</mml:mo><mml:mi>V</mml:mi><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:mstyle></mml:math></inline-formula> data using different Ca<sup>2+</sup>-VSD interaction models.</title></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3">One Ca<sup>2+</sup>-site per &#945;-subunit</th><th/><th colspan="6">Two Ca<sup>2+</sup>-sites per &#945;-subunit</th></tr><tr><th/><th rowspan="2">Model I</th><th rowspan="2">Model II</th><th/><th/><th colspan="2">Model I</th><th/><th colspan="2">Model II</th></tr></thead><tbody><tr><td>Parameters</td><td valign="top"/><td>Parameters</td><td valign="top">With cooperativity</td><td valign="top">Without cooperativity</td><td valign="top"/><td valign="top">With cooperativity</td><td valign="top">Without cooperativity</td></tr><tr><td><bold><italic>z<sub>J</sub></italic></bold></td><td>0.61</td><td>0.61</td><td/><td><italic>z<sub>J</sub></italic></td><td>0.61</td><td>0.61</td><td/><td>0.61</td><td>0.61</td></tr><tr><td><bold><italic>J<sub>0</sub></italic></bold></td><td>0.020</td><td>0.018</td><td/><td><italic>J<sub>0</sub></italic></td><td>0.019</td><td>0.019</td><td/><td>0.021</td><td>0.019</td></tr><tr><td rowspan="2"><italic>E</italic></td><td rowspan="2">26.4<sup>*</sup></td><td rowspan="2">2.27<sup>*</sup></td><td/><td><italic>E<sub>S</sub></italic><sub>1</sub></td><td colspan="2">4.57<sup>*</sup></td><td/><td colspan="2">1.46<sup>*</sup></td></tr><tr><td/><td><italic>E<sub>S</sub></italic><sub>2</sub></td><td colspan="2">5.35<sup>*</sup></td><td/><td colspan="2">1.52<sup>*</sup></td></tr><tr><td rowspan="2"><italic>K<sub>D</sub></italic> (&#181;M)</td><td rowspan="2">6.4</td><td rowspan="2">6.1</td><td/><td><italic>K<sub>D</sub></italic><sub>1</sub> (&#181;M)</td><td>3.2</td><td>4.9</td><td/><td>837.7</td><td>5.9</td></tr><tr><td/><td><italic>K<sub>D</sub></italic><sub>2</sub> (&#181;M)</td><td>631.7</td><td>6.9</td><td/><td>6.6</td><td>5.9</td></tr><tr><td/><td/><td/><td/><td><italic>G</italic></td><td>56.1</td><td>1<sup>*</sup></td><td/><td>120.4</td><td>1<sup>*</sup></td></tr><tr><td><italic>AIC</italic></td><td>&#8722;948.4</td><td>&#8722;1150.9</td><td/><td/><td>&#8722;1088.9</td><td>&#8722;1090.3</td><td/><td>&#8722;1162.0</td><td>&#8722;1147.5</td></tr><tr><td><italic>L<sub>i</sub></italic></td><td>4<sup>*</sup>10<sup>&#8722;47</sup></td><td>0.004</td><td/><td/><td>1<sup>*</sup>10<sup>&#8722;16</sup></td><td>3<sup>*</sup>10<sup>&#8722;16</sup></td><td/><td>1</td><td>7<sup>*</sup>10<sup>&#8722;4</sup></td></tr><tr><td><italic>w<sub>i</sub></italic></td><td>4<sup>*</sup>10<sup>&#8722;47</sup></td><td>0.004</td><td/><td/><td>1<sup>*</sup>10<sup>&#8722;16</sup></td><td>3<sup>*</sup>10<sup>&#8722;16</sup></td><td/><td>0.995</td><td>0.001</td></tr></tbody></table><table-wrap-foot><fn><p><sup>*</sup>Fixed parameters in the model fitting. AIC values correspond to Akaike Information Criterion to select the best fit model. &#8466;<italic><sub>i</sub> and w<sub>i</sub></italic> are the relative likelihood and the weight of each model within the set of candidate models.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-45017-v2.xml">elife-45017-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.45017.003</object-id><label>Table 1.</label><caption><title>Summary of the 13 analyzed Cyanidiales genomes.</title><p>The existing genomes of <italic>Galdieria sulphuraria</italic> 074W, <italic>Cyanidioschyzon merolae</italic> 10D, and <italic>Galdieria phlegrea</italic> are marked with &#8216;#'. The remaining 10 genomes are novel. Genome Size (Mb): size of the genome assembly in Megabases. Contigs: number of contigs produced by the genome assembly. The contigs were polished with quiver Contig N50 (kb): Contig N50. %GC Content: GC content of the genome given in percent. Genes: transcriptome size of species. Orthogroups: All Cyanidiales genes were clustered into a total of 9075 OGs. Here we show how many OGs there are per species. HGT Orthogroups: Number of OGs derived from HGT. HGT Genes: Number of HGT gene candidates found in species. %GC Native: GC content of the native transcriptome given in percent. %GC HGT: GC content of the HGT gene candidates given in percent % Multiexon Native: % of multiallelic genes in the native transcriptome. % Multiexon HGT: percent of multiallelic genes in the HGT gene candidates. S/M Native: Ratio of Multiexonic vs Singleexonic genes in native transcriptome. S/M HGT: Ratio of Multiexonic vs Singleexonic genes in HGT candidates. Asterisks (*) denote a significant difference (p&lt;=0.05) between native and HGT gene subsets. EC, PFAM, GO, KEGG: Number of species-specific annotations in EC, PFAM, GO, KEGG.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Strain</th><th colspan="4">Genome features</th><th colspan="2">Gene and OG counts</th><th colspan="2">HGTs</th><th colspan="6">HGT vs native gene subsets</th><th colspan="4">Annotations</th></tr><tr><th>&#8195;Genome Size (Mb)</th><th>Contigs</th><th>Contig N50 (kb)</th><th>%GC Content</th><th>Genes</th><th>Orthogroups</th><th>HGT orthogroups</th><th>HGT genes</th><th>%GC Native</th><th>%GC HGT</th><th>(%) Multiexon Native</th><th>(%) Multiexon HGT</th><th>Exon/Gene Native</th><th>Exon/Gene HGT</th><th>EC</th><th>PFAM</th><th>KEGG</th><th>GO</th></tr></thead><tbody><tr><td valign="bottom"><italic>G. sulphuraria 074W<sup>#</sup></italic></td><td valign="bottom">13.78</td><td valign="bottom">433</td><td valign="bottom">172.3</td><td valign="bottom">36.89</td><td valign="bottom">7174</td><td valign="bottom">5265</td><td valign="bottom">51</td><td valign="bottom">55</td><td valign="bottom">38.99</td><td valign="bottom">39.62*</td><td valign="bottom">73.6</td><td valign="bottom">47.3*</td><td valign="bottom">2.25</td><td valign="bottom">3.2*</td><td valign="bottom">938</td><td valign="bottom">3073</td><td valign="bottom">3241</td><td>6572</td></tr><tr><td valign="bottom"><italic>G. sulphuraria MS1</italic></td><td valign="bottom">14.89</td><td valign="bottom">129</td><td valign="bottom">172.1</td><td valign="bottom">37.62</td><td valign="bottom">7441</td><td valign="bottom">5389</td><td valign="bottom">54</td><td valign="bottom">58</td><td valign="bottom">39.59</td><td valign="bottom">40.79*</td><td valign="bottom">83.4</td><td valign="bottom">62.1*</td><td valign="bottom">2.5</td><td valign="bottom">3.88*</td><td valign="bottom">930</td><td valign="bottom">3077</td><td valign="bottom">3178</td><td>6564</td></tr><tr><td valign="bottom"><italic>G. sulphuraria RT22</italic></td><td valign="bottom">15.62</td><td valign="bottom">118</td><td valign="bottom">172.9</td><td valign="bottom">37.43</td><td valign="bottom">6982</td><td valign="bottom">5186</td><td valign="bottom">51</td><td valign="bottom">54</td><td valign="bottom">39.54</td><td valign="bottom">40.85*</td><td valign="bottom">74.7</td><td valign="bottom">51.9*</td><td valign="bottom">2.63</td><td valign="bottom">3.95*</td><td valign="bottom">941</td><td valign="bottom">3118</td><td valign="bottom">3223</td><td>6504</td></tr><tr><td valign="bottom"><italic>G. sulphuraria SAG21</italic></td><td valign="bottom">14.31</td><td valign="bottom">135</td><td valign="bottom">158.2</td><td valign="bottom">37.92</td><td valign="bottom">5956</td><td valign="bottom">4732</td><td valign="bottom">44</td><td valign="bottom">47</td><td valign="bottom">40.04</td><td valign="bottom">41.47*</td><td valign="bottom">84.8</td><td valign="bottom">83.0</td><td valign="bottom">4.02</td><td valign="bottom">5.03*</td><td valign="bottom">931</td><td valign="bottom">3047</td><td valign="bottom">3143</td><td>6422</td></tr><tr><td valign="bottom"><italic>G. sulphuraria MtSh</italic></td><td valign="bottom">14.95</td><td valign="bottom">101</td><td valign="bottom">186.6</td><td valign="bottom">40.04</td><td valign="bottom">6160</td><td valign="bottom">4746</td><td valign="bottom">46</td><td valign="bottom">47</td><td valign="bottom">41.33</td><td valign="bottom">42.48*</td><td valign="bottom">79.7</td><td valign="bottom">63.8*</td><td valign="bottom">3.15</td><td valign="bottom">4.32*</td><td valign="bottom">939</td><td valign="bottom">3114</td><td valign="bottom">3244</td><td>6450</td></tr><tr><td valign="bottom"><italic>G. sulphuraria Azora</italic></td><td valign="bottom">14.06</td><td valign="bottom">127</td><td valign="bottom">162.3</td><td valign="bottom">40.10</td><td valign="bottom">6305</td><td valign="bottom">4905</td><td valign="bottom">49</td><td valign="bottom">58</td><td valign="bottom">41.34</td><td valign="bottom">42.57*</td><td valign="bottom">84.5</td><td valign="bottom">75.9*</td><td valign="bottom">2.68</td><td valign="bottom">4.03*</td><td valign="bottom">934</td><td valign="bottom">3072</td><td valign="bottom">3181</td><td>6474</td></tr><tr><td valign="bottom"><italic>G. sulphuraria YNP5587.1</italic></td><td valign="bottom">14.42</td><td valign="bottom">115</td><td valign="bottom">170.8</td><td valign="bottom">40.05</td><td valign="bottom">6118</td><td valign="bottom">4846</td><td valign="bottom">46</td><td valign="bottom">46</td><td valign="bottom">41.33</td><td valign="bottom">42.14*</td><td valign="bottom">74.5</td><td valign="bottom">54.3*</td><td valign="bottom">2.61</td><td valign="bottom">3.65*</td><td valign="bottom">938</td><td valign="bottom">3084</td><td valign="bottom">3206</td><td>6516</td></tr><tr><td valign="bottom"><italic>G. sulphuraria 5572</italic></td><td valign="bottom">14.28</td><td valign="bottom">108</td><td valign="bottom">229.7</td><td valign="bottom">37.99</td><td valign="bottom">6472</td><td valign="bottom">5009</td><td valign="bottom">46</td><td valign="bottom">53</td><td valign="bottom">39.68</td><td valign="bottom">40.5*</td><td valign="bottom">78.4</td><td valign="bottom">45.3*</td><td valign="bottom">2.15</td><td valign="bottom">3.53*</td><td valign="bottom">936</td><td valign="bottom">3108</td><td valign="bottom">3252</td><td>6540</td></tr><tr><td valign="bottom"><italic>G. sulphuraria 002</italic></td><td valign="bottom">14.11</td><td valign="bottom">107</td><td valign="bottom">189.3</td><td valign="bottom">39.16</td><td valign="bottom">5912</td><td valign="bottom">4701</td><td valign="bottom">46</td><td valign="bottom">52</td><td valign="bottom">40.76</td><td valign="bottom">41.35*</td><td valign="bottom">97.1</td><td valign="bottom">50.0*</td><td valign="bottom">2.37</td><td valign="bottom">3.73*</td><td valign="bottom">927</td><td valign="bottom">3060</td><td valign="bottom">3184</td><td>6505</td></tr><tr><td valign="bottom"><italic>G. phlegrea DBV009<sup>#</sup></italic></td><td valign="bottom">11.41</td><td valign="bottom">9311</td><td valign="bottom">2.0</td><td valign="bottom">37.86</td><td valign="bottom">7836</td><td valign="bottom">5562</td><td valign="bottom">54</td><td valign="bottom">62</td><td valign="bottom">39.97</td><td valign="bottom">40.58*</td><td valign="bottom">na</td><td valign="bottom">na</td><td valign="bottom">na</td><td valign="bottom">na</td><td valign="bottom">935</td><td valign="bottom">3018</td><td valign="bottom">3125</td><td>6512</td></tr><tr><td valign="bottom"><italic>G. phlegrea Soos</italic></td><td valign="bottom">14.87</td><td valign="bottom">108</td><td valign="bottom">201.1</td><td valign="bottom">37.52</td><td valign="bottom">6125</td><td valign="bottom">4624</td><td valign="bottom">44</td><td valign="bottom">47</td><td valign="bottom">39.57</td><td valign="bottom">40.73*</td><td valign="bottom">77.5</td><td valign="bottom">43.2*</td><td valign="bottom">2.19</td><td valign="bottom">3.33*</td><td valign="bottom">929</td><td valign="bottom">3034</td><td valign="bottom">3197</td><td>6493</td></tr><tr><td valign="bottom"><italic>C. merolae 10D<sup>#</sup></italic></td><td valign="bottom">16.73</td><td valign="bottom">22</td><td valign="bottom">859.1</td><td valign="bottom">54.81</td><td valign="bottom">4803</td><td valign="bottom">3980</td><td valign="bottom">33</td><td valign="bottom">33</td><td valign="bottom">56.57</td><td valign="bottom">56.57</td><td valign="bottom">0.5</td><td valign="bottom">0.0</td><td valign="bottom">1</td><td valign="bottom">1.01</td><td valign="bottom">883</td><td valign="bottom">2811</td><td valign="bottom">2832</td><td>6213</td></tr><tr><td valign="bottom"><italic>C. merolae Soos</italic></td><td valign="bottom">12.33</td><td valign="bottom">35</td><td valign="bottom">567.5</td><td valign="bottom">54.33</td><td valign="bottom">4406</td><td valign="bottom">3574</td><td valign="bottom">34</td><td valign="bottom">34</td><td valign="bottom">54.84</td><td valign="bottom">54.26</td><td valign="bottom">9.4</td><td valign="bottom">2.9</td><td valign="bottom">1.06</td><td valign="bottom">1.1</td><td valign="bottom">886</td><td valign="bottom">2787</td><td valign="bottom">2823</td><td>6188</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-45017-v2.xml">elife-45017-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.45017.105</object-id><label>Table 2.</label><caption><title>Natural habitats of extant prokaryotes harboring the closest orthologs to Cyanidiales HGTs.</title><p>Numbers in brackets represent how many times HGT candidates from Cyanidiales shared monophyly with non-eukaryotic organisms; for&#160;example Proteobacteria were found in 53/96 of the OG monophylies. <bold>Kingdom</bold>: Taxon at kingdom level. <bold>Species</bold>: Scientific species name. <bold>Habitat</bold>: habitat description of the original sampling site. <bold>pH</bold>: pH of the original sampling site. <bold>Temp</bold>: Temperature in Celsius of the sampling site. <bold>Salt</bold>: Ion concentration of the original sampling site. <bold>na</bold>: no information available.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Kingdom</th><th colspan="2" valign="bottom">Phylogeny</th><th colspan="4" valign="bottom">Natural habitat of closest non-eukaryotic ortholog</th></tr><tr><th valign="bottom">Division</th><th valign="bottom">Species</th><th valign="bottom">Habitat description</th><th valign="bottom">pH</th><th valign="bottom">Max. temp</th><th valign="bottom">Salt</th></tr></thead><tbody><tr><td rowspan="22" valign="bottom">Bacteria</td><td rowspan="9" valign="bottom">Proteobacteria (53)</td><td valign="bottom"><italic>Acidithiobacillus thiooxidans (4)</italic></td><td valign="bottom">Mine drainage/Mineral ores</td><td valign="bottom">2.0&#8211;2.5</td><td valign="bottom">30&#176;C</td><td valign="bottom">&#8216;hypersaline&#8217;</td></tr><tr><td valign="bottom"><italic>Carnimonas nigrificans (4)</italic></td><td valign="bottom">Raw cured meat</td><td valign="bottom">3.0</td><td valign="bottom">35&#176;C</td><td valign="bottom">8% NaCl</td></tr><tr><td valign="bottom"><italic>Methylosarcina fibrata (4)</italic></td><td valign="bottom">Landfill</td><td valign="bottom">5.0&#8211;9.0</td><td valign="bottom">37&#176;C</td><td valign="bottom">1% NaCl</td></tr><tr><td valign="bottom"><italic>Sphingomonas phyllosphaerae (3)</italic></td><td valign="bottom">Phyllosphere of Acacia caven</td><td valign="bottom">na</td><td valign="bottom">28&#176;C</td><td valign="bottom">na</td></tr><tr><td valign="bottom"><italic>Gluconacetobacter diazotrophicus (3)</italic></td><td valign="bottom">Symbiont of various plant species</td><td valign="bottom">2.0&#8211;6.0</td><td valign="bottom">na</td><td valign="bottom">&#8216;high salt&#8217;</td></tr><tr><td valign="bottom"><italic>Gluconobacter frateurii (3)</italic></td><td valign="bottom">na</td><td valign="bottom">na</td><td valign="bottom">na</td><td valign="bottom">na</td></tr><tr><td valign="bottom"><italic>Luteibacter yeojuensis (3)</italic></td><td valign="bottom">River</td><td valign="bottom">na</td><td valign="bottom">na</td><td valign="bottom">na</td></tr><tr><td valign="bottom"><italic>Thioalkalivibrio sulfidiphilus (3)</italic></td><td valign="bottom">Soda lake</td><td valign="bottom">8.0&#8211;10.5</td><td valign="bottom">40&#176;C</td><td valign="bottom">15% total salts</td></tr><tr><td valign="bottom"><italic>Thiomonas arsenitoxydans (3)</italic></td><td valign="bottom">Disused mine site</td><td valign="bottom">3.0&#8211;8.0</td><td valign="bottom">30&#176;C</td><td valign="bottom">&#8216;halophilic&#8217;</td></tr><tr><td rowspan="3" valign="bottom">Firmicutes (28)</td><td valign="bottom"><italic>Sulfobacillus thermosulfidooxidans (6)</italic></td><td valign="bottom">Copper mining</td><td valign="bottom">2.0&#8211;2.5</td><td valign="bottom">45&#176;C</td><td valign="bottom">&#8216;salt tolerant&#8217;</td></tr><tr><td valign="bottom"><italic>Alicyclobacillus acidoterrestris (4)</italic></td><td valign="bottom">Soil sample</td><td valign="bottom">2.0&#8211;6.0</td><td valign="bottom">53&#176;C</td><td valign="bottom">5% NaCl</td></tr><tr><td valign="bottom"><italic>Gracilibacillus lacisalsi (3)</italic></td><td valign="bottom">Salt lake</td><td valign="bottom">7.2&#8211;7.6</td><td valign="bottom">50&#176;C</td><td valign="bottom">25% total salts</td></tr><tr><td valign="bottom">Actinobacteria (19)</td><td valign="bottom"><italic>Amycolatopsis halophila (3)</italic></td><td valign="bottom">Salt lake</td><td valign="bottom">6.0&#8211;8.0</td><td valign="bottom">45&#176;C</td><td valign="bottom">15% NaCl</td></tr><tr><td/><td valign="bottom"><italic>Rubrobacter xylanophilus (3)</italic></td><td valign="bottom">Thermal industrial runoff</td><td valign="bottom">6.0&#8211;8.0</td><td valign="bottom">60&#176;C</td><td valign="bottom">6.0% NaCl</td></tr><tr><td valign="bottom">Chloroflexi (12)</td><td valign="bottom"><italic>Caldilinea aerophila (4)</italic></td><td valign="bottom">Thermophilic granular sludge</td><td valign="bottom">6.0&#8211;8.0</td><td valign="bottom">65&#176;C</td><td valign="bottom">3% NaCl</td></tr><tr><td/><td valign="bottom"><italic>Ardenticatena maritima (3)</italic></td><td valign="bottom">Coastal hydrothermal field</td><td valign="bottom">5.5&#8211;8.0</td><td valign="bottom">70&#176;C</td><td valign="bottom">6% NaCl</td></tr><tr><td/><td valign="bottom"><italic>Ktedonobacter racemifer (3)</italic></td><td valign="bottom">Soil sample</td><td valign="bottom">4.8&#8211;6.8</td><td valign="bottom">33&#176;C</td><td valign="bottom">&gt;3% NaCl</td></tr><tr><td valign="bottom">Bacteroidetes Chlorobi (10)</td><td valign="bottom"><italic>Salinibacter ruber (4)</italic></td><td valign="bottom">Saltern crystallizer ponds</td><td valign="bottom">6.5&#8211;8.0</td><td valign="bottom">52&#176;C</td><td valign="bottom">30% total salts</td></tr><tr><td/><td valign="bottom"><italic>Salisaeta longa (3)</italic></td><td valign="bottom">Experimental mesocosm (Salt)</td><td valign="bottom">6.5&#8211;8.5</td><td valign="bottom">46&#176;C</td><td valign="bottom">20% NaCl</td></tr><tr><td valign="bottom">Nitrospirae (7)</td><td valign="bottom"><italic>Leptospirillum ferriphilum (4)</italic></td><td valign="bottom">Arsenopyrite biooxidation tank</td><td valign="bottom">0&#8211;3.0</td><td valign="bottom">40&#176;C</td><td valign="bottom">2% NaCl</td></tr><tr><td valign="bottom">Fibrobacteres (6)</td><td valign="bottom"><italic>Acidobacteriaceae bacterium TAA166 (3)</italic></td><td valign="bottom">na</td><td valign="bottom">na</td><td valign="bottom">na</td><td valign="bottom">na</td></tr><tr><td valign="bottom">Deinococcus (5)</td><td valign="bottom"><italic>Truepera radiovictrix (3)</italic></td><td valign="bottom">Hot spring runoffs</td><td valign="bottom">7.5&#8211;9.5</td><td valign="bottom">na</td><td valign="bottom">6% NaCl</td></tr><tr><td valign="bottom">Archaea</td><td valign="bottom">Euryarchaeota (6)</td><td valign="bottom"><italic>Ferroplasma acidarmanus (3)</italic></td><td valign="bottom">Acid mine drainage</td><td valign="bottom">0&#8211;2.5</td><td valign="bottom">40&#176;C</td><td valign="bottom">&#8216;halophilic&#8217;</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-45204-v2.xml">elife-45204-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.45204.010</object-id><label>Table 2.</label><caption><title>Footprint shape features after <bold>&#8216;</bold>Robbins footprint recording form<bold>&#8217;</bold> (1985, p.97&#8211;102).</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">ID</th><th rowspan="2">Left (L) or right (R) foot</th><th colspan="2">General appearance</th><th rowspan="2">Relative length of toes</th><th rowspan="2">Toes region general appearance length - width</th><th rowspan="2">Toe one position</th><th rowspan="2">Ball region, general appearance length-width</th><th colspan="2">Arch region</th><th rowspan="2">Heel region, general appearance</th><th rowspan="2">Hell posterior margin</th><th rowspan="2"/></tr><tr><th>Length</th><th>Width</th><th>Medial margin</th><th>Lateral margin</th></tr></thead><tbody><tr><td>SM3</td><td>R</td><td>short</td><td>broad</td><td>1, 2, ?</td><td>short - broad</td><td>extended, anteriorly</td><td/><td>straight</td><td>concave</td><td>circular</td><td>convex pronounced</td><td rowspan="4">Morphotype 1</td></tr><tr><td>SM4</td><td>L</td><td/><td/><td>1,2,3,4,5</td><td>short - broad</td><td>extended, anteriorly</td><td>short - moderate</td><td/><td>concave</td><td/><td/></tr><tr><td>SM43</td><td>L</td><td>short</td><td>broad</td><td>1,2, ?</td><td>short - broad</td><td>extended, anteriorly</td><td/><td>straight</td><td>convex</td><td/><td>convex pronounced</td></tr><tr><td>SM17</td><td>R</td><td>short</td><td>broad</td><td>1,2,3,4,5</td><td>short - broad</td><td>extended, anteriorly</td><td/><td>straight/concave</td><td>convex</td><td/><td>convex pronounced</td></tr><tr><td>SM5</td><td>R</td><td>moderate</td><td>moderate</td><td>1,2,3,4,5</td><td>short - broad</td><td>extended oblique medially</td><td>long - moderate</td><td>concave</td><td>concave</td><td/><td>convex slight</td><td rowspan="3">Morphotype 2</td></tr><tr><td>SM42</td><td>R</td><td>moderate</td><td/><td>1,2, ?</td><td/><td>extended oblique medially</td><td/><td>concave</td><td/><td/><td>convex pronounced</td></tr><tr><td>SM26</td><td>L</td><td>moderate</td><td>broad</td><td>1,2,3,4,5</td><td>short - moderate</td><td>extended oblique laterally</td><td/><td>concave</td><td>concave</td><td/><td>convex slight</td></tr><tr><td>CA8</td><td>R</td><td>long</td><td>moderate</td><td>1,2,3,4,5</td><td>moderate - broad</td><td>extended oblique medially</td><td>long - moderate</td><td>concave</td><td>straight</td><td>oblong</td><td>convex pronounced</td><td rowspan="8">Morphotype 3</td></tr><tr><td>CA10</td><td>R</td><td>long</td><td>moderate</td><td>1,2,3,4</td><td>moderate - broad</td><td>extended oblique medially</td><td/><td>straight/concave</td><td>straight</td><td>oblong</td><td>convex pronounced</td></tr><tr><td>SM15</td><td>L</td><td>long</td><td>moderate</td><td>2,3,1,4,5</td><td>moderate - broad</td><td>extended oblique medially</td><td>long - narrow</td><td>concave</td><td>straight</td><td>oblong</td><td>convex pronounced</td></tr><tr><td>SM11</td><td>R</td><td>long</td><td>moderate</td><td>1,2,3,4,5</td><td>short - broad</td><td>extended anteriorly</td><td/><td>straight/concave</td><td>convex</td><td>oblong</td><td>convex pronounced</td></tr><tr><td>SM6</td><td>L</td><td>long</td><td>moderate</td><td>2,1,3,4,5</td><td>short - broad</td><td>flexed slight</td><td>long - narrow</td><td>concave</td><td>convex</td><td>oblong</td><td>convex pronounced</td></tr><tr><td>SM1</td><td>L</td><td>long</td><td>moderate</td><td>2,3,1,4,5</td><td>short - broad</td><td>extended oblique medially</td><td>moderate - narrow</td><td>unknown</td><td>convex</td><td>circular</td><td>convex pronounced</td></tr><tr><td>C33</td><td>L</td><td>long</td><td>broad</td><td>1,2,3,4,5</td><td>long - broad</td><td>extended oblique medially</td><td>long - broad</td><td>concave</td><td>convex</td><td>oblong</td><td>convex pronounced</td></tr><tr><td>C36</td><td>L</td><td>long</td><td>very narrow</td><td>1,2,3,4,5</td><td>moderate - narrow</td><td>extended oblique laterally</td><td>long - narrow</td><td>concave</td><td>straight</td><td>oblong</td><td>convex pronounced</td></tr><tr><td>CA1</td><td>R</td><td>long</td><td/><td>1,2,3,4</td><td>moderate - broad</td><td>extended anteriorly</td><td>moderate - narrow</td><td>straight</td><td/><td>oblong</td><td>convex, pronunced</td><td rowspan="3">Morphotype 4</td></tr><tr><td>CA2</td><td>L</td><td>long</td><td>moderate</td><td>1,2,3,4,5</td><td>moderate - broad</td><td>extended anteriorly</td><td>moderate - narrow</td><td>concave</td><td>straight</td><td>oblong</td><td>convex pronunced</td></tr><tr><td>C61</td><td>L</td><td>long</td><td>moderate</td><td>1,2,3,4,5</td><td>moderate - broad</td><td>extended oblique medially</td><td>moderate - narrow</td><td>straight/concave</td><td>straight/convex</td><td>oblong</td><td>convex pronunced</td></tr><tr><td>C63</td><td>R</td><td>long</td><td>moderate</td><td>1,2,3,4,5</td><td>moderate - broad</td><td>extended anteriorly</td><td>moderate - narrow</td><td>straight/concave</td><td>straight/concave</td><td>oblong</td><td>convex pronunced</td><td/></tr><tr><td>M21</td><td>R</td><td>long</td><td>moderate</td><td>1,2,3,4,5</td><td>moderate - broad</td><td>extended anteriorly</td><td>moderate - narrow</td><td>concave</td><td>straight</td><td>oblong</td><td>convex pronunced</td><td/></tr><tr><td>C9</td><td>R</td><td>long</td><td>moderate</td><td>1</td><td/><td>extended anteriorly</td><td>moderate - narrow</td><td>straight</td><td>convex</td><td>oblong</td><td>convex pronunced</td><td/></tr><tr><td>C44b</td><td>L</td><td>long</td><td>broad</td><td>2,1,3,4,5</td><td>short - broad</td><td>extended oblique medially</td><td>moderate - broad</td><td>concave</td><td>convex</td><td>oblong</td><td>convex, moderate</td><td/></tr><tr><td>C60</td><td>L</td><td>long</td><td>broad</td><td>1,2,3,4,5</td><td>moderate - broad</td><td>extended oblique laterally</td><td>moderate - broad</td><td>straight/concave</td><td>straight/concave</td><td>circular</td><td>convex moderate</td><td rowspan="4">Morphotype 5</td></tr><tr><td>C37</td><td>L</td><td>long</td><td>broad</td><td>1,2,3,4,5</td><td>moderate - broad</td><td>extended anteriorly</td><td>moderate - broad</td><td>straight/concave</td><td>straight/concave</td><td>circular</td><td>convex moderate</td></tr><tr><td>C35b</td><td>R</td><td>long</td><td>broad</td><td>1,2,3,4,5</td><td>moderate - broad</td><td>extended anteriorly</td><td>moderate - broad</td><td/><td>straight/concave</td><td>circular</td><td>convex moderate</td></tr><tr><td>C44</td><td>L</td><td>long</td><td>broad</td><td>1,2,3,4,5</td><td>moderate - broad</td><td>extended anteriorly</td><td>moderate - broad</td><td>concave</td><td>straight/concave</td><td>oblong</td><td>convex pronunced</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-45221-v2.xml">elife-45221-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.45221.011</object-id><label>Table 1.</label><caption><title>Functional properties of various mCST constructs.</title><p>The K<sub>d</sub> for the indicated ligand was determined by either: 1) a full SPA titration as shown in <xref ref-type="fig" rid="fig1">Figure 1A</xref> 2) fitting the two-point SPA data from <xref ref-type="fig" rid="fig5">Figure 5H</xref> to get a crude estimation of the K<sub>d</sub>, or 3) fitting the tryptophan fluorescence quenching data shown in <xref ref-type="fig" rid="fig5">Figure 5G</xref> (K<sub>d</sub>&#8217;s were not fit for the Trp207 mutants which showed no quenching). The K<sub>m</sub> and V<sub>max</sub> were determined from data shown in <xref ref-type="fig" rid="fig1">Figure 1C</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Construct</th><th rowspan="2" valign="bottom">Ligand</th><th colspan="3">K<sub>d</sub> (&#181;M)</th><th rowspan="2" valign="bottom">K<sub>m</sub> (&#181;M)</th><th rowspan="2" valign="bottom">V<sub>max</sub> (nmol/mol mCST/min)</th></tr><tr><th>SPA titration</th><th>SPA 2-point</th><th>Trp-FL</th></tr></thead><tbody><tr><td rowspan="2" valign="middle">mCST</td><td>CMP</td><td>6.3&#160;&#177;&#160;1.2</td><td>5.5&#160;&#177;&#160;1.6</td><td>2.9&#160;&#177;&#160;1.1</td><td/><td/></tr><tr><td>CMP-Sia</td><td>482&#160;&#177;&#160;147</td><td/><td/><td>58.1&#160;&#177;&#160;13.2</td><td>6.5&#160;&#177;&#160;0.6</td></tr><tr><td rowspan="4" valign="middle">mCST&#8710;C</td><td>CMP</td><td>1.8&#160;&#177;&#160;0.4</td><td/><td>11.1&#160;&#177;&#160;2.9</td><td/><td/></tr><tr><td>CMP-Sia</td><td>295&#160;&#177;&#160;100</td><td/><td/><td>62.0&#160;&#177;&#160;14.7</td><td>5.3&#160;&#177;&#160;0.5</td></tr><tr><td>UMP</td><td>1621&#160;&#177;&#160;1119</td><td/><td/><td/><td/></tr><tr><td>GMP</td><td>5376&#160;&#177;&#160;1151</td><td valign="bottom"/><td valign="bottom"/><td/><td/></tr><tr><td>mCST (W145L)</td><td>CMP</td><td/><td>0.8&#160;&#177;&#160;0.3</td><td>3.3&#160;&#177;&#160;1.0</td><td/><td/></tr><tr><td>mCST (W160L)</td><td>CMP</td><td/><td>1.6&#160;&#177;&#160;0.4</td><td>11.9&#160;&#177;&#160;2.4</td><td/><td/></tr><tr><td>mCST (W207L)</td><td>CMP</td><td/><td>1.5&#160;&#177;&#160;0.4</td><td>N/D</td><td/><td/></tr><tr><td>mCST (W207F)</td><td>CMP</td><td/><td>3.8&#160;&#177;&#160;1.5</td><td>N/D</td><td/><td/></tr><tr><td>mCST (W247L)</td><td>CMP</td><td/><td>2.4&#160;&#177;&#160;1.5</td><td>7.4&#160;&#177;&#160;2.6</td><td/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-45221-v3.xml">elife-45221-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.45221.011</object-id><label>Table 1.</label><caption><title>Functional properties of various mCST constructs.</title><p>The K<sub>d</sub> for the indicated ligand was determined by either: 1) a full SPA titration as shown in <xref ref-type="fig" rid="fig1">Figure 1A</xref> 2) fitting the two-point SPA data from <xref ref-type="fig" rid="fig5">Figure 5H</xref> to get a crude estimation of the K<sub>d</sub>, or 3) fitting the tryptophan fluorescence quenching data shown in <xref ref-type="fig" rid="fig5">Figure 5G</xref> (K<sub>d</sub>&#8217;s were not fit for the Trp207 mutants which showed no quenching). The K<sub>m</sub> and V<sub>max</sub> were determined from data shown in <xref ref-type="fig" rid="fig1">Figure 1C</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Construct</th><th rowspan="2" valign="bottom">Ligand</th><th colspan="3">K<sub>d</sub> (&#181;M)</th><th rowspan="2" valign="bottom">K<sub>m</sub> (&#181;M)</th><th rowspan="2" valign="bottom">V<sub>max</sub> (nmol/mol mCST/min)</th></tr><tr><th>SPA titration</th><th>SPA 2-point</th><th>Trp-FL</th></tr></thead><tbody><tr><td rowspan="2" valign="middle">mCST</td><td>CMP</td><td>6.3&#160;&#177;&#160;1.2</td><td>5.5&#160;&#177;&#160;1.6</td><td>2.9&#160;&#177;&#160;1.1</td><td/><td/></tr><tr><td>CMP-Sia</td><td>482&#160;&#177;&#160;147</td><td/><td/><td>58.1&#160;&#177;&#160;13.2</td><td>6.5&#160;&#177;&#160;0.6</td></tr><tr><td rowspan="4" valign="middle">mCST&#8710;C</td><td>CMP</td><td>1.8&#160;&#177;&#160;0.4</td><td/><td>11.1&#160;&#177;&#160;2.9</td><td/><td/></tr><tr><td>CMP-Sia</td><td>295&#160;&#177;&#160;100</td><td/><td/><td>62.0&#160;&#177;&#160;14.7</td><td>5.3&#160;&#177;&#160;0.5</td></tr><tr><td>UMP</td><td>1621&#160;&#177;&#160;1119</td><td/><td/><td/><td/></tr><tr><td>GMP</td><td>5376&#160;&#177;&#160;1151</td><td valign="bottom"/><td valign="bottom"/><td/><td/></tr><tr><td>mCST (W145L)</td><td>CMP</td><td/><td>0.8&#160;&#177;&#160;0.3</td><td>3.3&#160;&#177;&#160;1.0</td><td/><td/></tr><tr><td>mCST (W160L)</td><td>CMP</td><td/><td>1.6&#160;&#177;&#160;0.4</td><td>11.9&#160;&#177;&#160;2.4</td><td/><td/></tr><tr><td>mCST (W207L)</td><td>CMP</td><td/><td>1.5&#160;&#177;&#160;0.4</td><td>N/D</td><td/><td/></tr><tr><td>mCST (W207F)</td><td>CMP</td><td/><td>3.8&#160;&#177;&#160;1.5</td><td>N/D</td><td/><td/></tr><tr><td>mCST (W247L)</td><td>CMP</td><td/><td>2.4&#160;&#177;&#160;1.5</td><td>7.4&#160;&#177;&#160;2.6</td><td/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-45310-v1.xml">elife-45310-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Contra-regulated differential gene expression in etiolated seedlings and young leaves of <italic>ccr2 ziso-155.</italic></title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene id</th><th rowspan="2">GENE</th><th rowspan="2"><italic>PhANG</italic></th><th rowspan="2">Protein encoding description</th><th colspan="2">Etiolated seedlings</th><th colspan="2">Young leaves</th><th rowspan="2"><italic>det1-1</italic></th><th rowspan="2">NF-1</th><th rowspan="2">NF-2</th></tr><tr><th><italic>ccr2</italic></th><th> <italic>ccr2 ziso-155</italic></th><th>ccr2</th><th><italic>ccr2 ziso-155</italic></th></tr></thead><tbody><tr><td>At1g09530</td><td><italic>PIF3</italic></td><td valign="top"/><td>Transcription factor interacts with photoreceptors and negatively regulates signalling</td><td>30</td><td>0.1</td><td>220</td><td>0.1</td><td>&#8595;</td><td>&#8722;5.0</td><td>NS</td></tr><tr><td>At4g10180</td><td><italic>DET1/FUS2</italic></td><td valign="top"/><td>Encodes a nuclear-localized protein repressor of photomorphogenesis</td><td>5.1</td><td>0.1</td><td>5.9</td><td>0.2</td><td>NS</td><td>NS</td><td>NS</td></tr><tr><td>At3g19390</td><td/><td valign="top"/><td>Granulin repeat cysteine protease family protein</td><td>4.4</td><td>NS</td><td>6.8</td><td>NS</td><td>NS</td><td>NS</td><td>NS</td></tr><tr><td>At5g13210</td><td/><td valign="top"/><td>Unknown conserved expressed protein</td><td>3.8</td><td>NS</td><td>0.4</td><td>NS</td><td>&#8593;</td><td>NS</td><td>NS</td></tr><tr><td>At3g45730</td><td/><td valign="top"/><td>Unknown expressed protein</td><td>2.8</td><td>NS</td><td>2.4</td><td>NS</td><td>NS</td><td>NS</td><td>10.6</td></tr><tr><td>At5g43500</td><td><italic>ATARP9</italic></td><td valign="top"/><td>Encodes an expressed protein similar to actin-related proteins</td><td>2.4</td><td>NS</td><td>2.2</td><td>NS</td><td>NS</td><td>NS</td><td>NS</td></tr><tr><td>At5g48240</td><td/><td valign="top"/><td>Unknown expressed protein</td><td>2.1</td><td>NS</td><td>2.2</td><td>NS</td><td>NS</td><td>NS</td><td>NS</td></tr><tr><td>At2g32950</td><td><italic>COP1/FUS3</italic></td><td valign="top"/><td>Repressor of photomorphogenesis and induces skotomorphogenesis</td><td>2.0</td><td>0.0</td><td>8.9</td><td>0.1</td><td>&#8593;</td><td>NS</td><td>NS</td></tr><tr><td>At5g11260</td><td><italic>HY5</italic></td><td valign="top"/><td>Transcription factor negatively regulated by COP1, promotes light responsive gene expression</td><td>0.5</td><td>8.1</td><td>0.3</td><td>8.4</td><td>NS</td><td>NS</td><td>2.8</td></tr><tr><td>At4g02770</td><td><italic>PSAD1</italic></td><td valign="top"/><td>Expressed protein with similarity to photosystem I subunit II</td><td>0.5</td><td>NS</td><td>0.5</td><td>NS</td><td>&#8593;</td><td>&#8722;12.3</td><td>0.15</td></tr><tr><td>At3g17070</td><td/><td valign="top"/><td>Peroxidase family expressed protein</td><td>0.5</td><td>NS</td><td>0.5</td><td>NS</td><td>NS</td><td>NS</td><td>NS</td></tr><tr><td>At2g31751</td><td/><td valign="top"/><td>Potential natural antisense gene, expressed protein</td><td>0.4</td><td>NS</td><td>0.5</td><td>NS</td><td>NS</td><td>NS</td><td>NS</td></tr><tr><td>At4g15560</td><td><italic>DXS/CLA1</italic></td><td valign="top">yes</td><td>1-deoxyxylulose 5-phosphate synthase activity in MEP pathway</td><td>0.3</td><td>4.2</td><td>0.1</td><td>16.2</td><td>NS</td><td>NS</td><td>0.42</td></tr><tr><td>At4g34350</td><td><italic>ISPH/CLB6</italic></td><td valign="top">yes</td><td>4-hydroxy-3-methylbut-2-enyl diphosphate reductase in MEP pathway</td><td>0.3</td><td>9.4</td><td>0.2</td><td>11</td><td>&#8593;</td><td>NS</td><td>NS</td></tr><tr><td>At1g24510</td><td><italic>TCP-1</italic></td><td valign="top"/><td>T-complex expressed protein one epsilon subunit</td><td>0.3</td><td>12.0</td><td>0.1</td><td>7.9</td><td>NS</td><td>NS</td><td>NS</td></tr><tr><td>At3g59010</td><td><italic>PME35</italic></td><td valign="top"/><td>Pectin methylesterase that regulates the cell wall mechanical strength</td><td>0.2</td><td>NS</td><td>0.4</td><td>NS</td><td>&#8595;</td><td>NS</td><td>NS</td></tr><tr><td>At1g29930</td><td><italic>CAB1/LHCB1.3</italic></td><td valign="top">yes</td><td>Subunit of light-harvesting complex II (LHCII), which absorbs light</td><td>0.2</td><td>13</td><td>0.2</td><td>11</td><td>NS</td><td>NS</td><td>NS</td></tr><tr><td>At2g05070</td><td><italic>LHCB2.2</italic></td><td valign="top">yes</td><td>Light-harvesting chlorophyll a/b-binding (LHC) protein that constitute the antenna system</td><td>0.2</td><td>NS</td><td>0.2</td><td>NS</td><td>&#8593;</td><td>&#8722;3.6</td><td>NS</td></tr><tr><td>At5g13630</td><td><italic>GUN5/CHLH</italic></td><td valign="top">yes</td><td>Magnesium chelatase involved in plastid-to-nucleus signalling</td><td>0.2</td><td>17</td><td>0.2</td><td>20</td><td>&#8593;</td><td>&#8722;3.3</td><td>0.33</td></tr><tr><td>At1g67090</td><td><italic>RBCS1a</italic></td><td valign="top">yes</td><td>Member of the Rubisco small subunit (RBCS) multigene family functions in photosynthesis</td><td>0.1</td><td>67</td><td>0.1</td><td>61</td><td>NS</td><td>NS</td><td>NS</td></tr></tbody></table><table-wrap-foot><fn><p>Notes: NS; not significant. Transcriptomic data; <italic>det1-1</italic> (<xref ref-type="bibr" rid="bib92">Schroeder et al., 2002</xref>), norflurazon (NF-1; <xref ref-type="bibr" rid="bib75">Page et al., 2017</xref>), norflurazon (NF-2; <xref ref-type="bibr" rid="bib54">Koussevitzky et al., 2007</xref>), <italic>PhANG</italic>; <italic>Photosynthesis associated nuclear gene</italic>. Numbers refer to fold change relative to WT&#160;=&#160;0 (except for NF-1 where positive and negative numbers indicate up and down-regulation, respectively relative to WT&#160;=&#160;1.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-45474-v2.xml">elife-45474-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.45474.012</object-id><label>Table 2.</label><caption><title>Association between environmental variables and dengue force of infection for 211 municipalities in Colombia</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Variable</th><th colspan="2" valign="bottom">Unadjusted</th><th colspan="2" valign="bottom">Adjusted</th></tr><tr><th valign="bottom">&#8195;Estimate</th><th valign="bottom">95% CI</th><th valign="bottom">Estimate</th><th valign="bottom">95% CI</th></tr></thead><tbody><tr><td valign="bottom">Elevation (per 100 m)</td><td valign="bottom">&#8722;0.005</td><td valign="bottom">&#8722;0.003,&#8211;0.006</td><td valign="bottom">&#8722;0.007</td><td valign="bottom">&#8722;0.008,&#8211;0.006</td></tr><tr><td valign="bottom">Mean temperature (per &#176;C)</td><td valign="bottom">0.006</td><td valign="bottom">0.004, 0.007</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">Mean weekly precipitation</td><td valign="bottom">0.0001</td><td valign="bottom">&#8722;0.000, 0.001</td><td valign="bottom">&#8722;0.0005</td><td valign="bottom">&#8722;0.0008, &#8722;0.0002</td></tr><tr><td valign="bottom">Population density</td><td valign="bottom">&#8722;0.001</td><td valign="bottom">&#8722;0.003, 0.001</td><td valign="bottom">0.007</td><td valign="bottom">0.004, 0.009</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-45474-v2.xml">elife-45474-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.45474.013</object-id><label>Table 3.</label><caption><title>Association between environmental variables and the log incidence of dengue (over the last 5 years) for 211 municipalities in Colombia.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Variable</th><th colspan="2" valign="bottom">Unadjusted</th><th colspan="2" valign="bottom">Adjusted*</th></tr><tr><th valign="bottom">Estimate</th><th valign="bottom">95% CI</th><th valign="bottom">Estimate</th><th valign="bottom">95% CI</th></tr></thead><tbody><tr><td valign="bottom">Elevation (per 100 m)</td><td valign="bottom">&#8722;0.02</td><td valign="bottom">&#8722;0.03, 0</td><td valign="bottom">0.01</td><td valign="bottom">&#8722;0.01, 0.03</td></tr><tr><td valign="bottom">Mean temperature (per &#176;C)</td><td valign="bottom">0.01</td><td valign="bottom">0, 0.05</td><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">Mean weekly precipitation</td><td valign="bottom">0</td><td valign="bottom">0, 0</td><td valign="bottom">&#8722;0.002</td><td valign="bottom">&#8722;0.01, 0</td></tr><tr><td valign="bottom">Population density</td><td valign="bottom">&#8722;0.04</td><td valign="bottom">&#8722;0.07,&#8211;0.01</td><td valign="bottom">&#8722;0.06</td><td valign="bottom">&#8722;0.10, &#8722;0.02</td></tr><tr><td colspan="2" valign="bottom">* R2 of adjusted model: 0.04</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-45644-v2.xml">elife-45644-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.45644.007</object-id><label>Table 2.</label><caption><title>Main protein sequences identified in the double-buttons from Havn&#248;, Hornstaad-H&#246;rnle IA and Pe&#351;tera Ungureasc&#259; and their presence/absence in the analysed set of reference freshwater and marine shells (black dots).</title><p>Numbers indicate total number of peptide sequences identified and the cell colour is proportional to the coverage of the sequence itself. Threshold values for peptide and protein identification: false discovery rate (protein FDR)&#160;=&#160;0.5%, protein score &#8722;10lgP&#160;&#8805;&#160;40, unique peptides&#160;&#8805;&#160;2, de novo sequences scores (ALC%)&#160;&#8805;&#160;50. Asterisks (*) indicate proteins identified only when using less stringent parametres: protein score &#8722;10lgP&#160;&#8805;&#160;20; unique peptides&#160;&#8805;&#160;1. Note that molecular sequence databases for molluscan species are incomplete and biased towards well-studied model organisms. The peptide sequences recovered in our study were identified using sequence homologies with proteins originally described from <italic>Hyriopsis cumingii</italic>, <italic>Crassostrea</italic> sp., <italic>Pinctada</italic> sp., <italic>Mytilus</italic> sp. and several others. As a result of database insufficiency, the bioinformatic search of these &#8216;shellomes&#8217; could not identify the exact taxon of our samples, but provided a strong indication of the fact that the closest taxon to that of the ornaments (and of the freshwater reference shells) is the pearl-producing triangle sail mussel <italic>Hyriopsis cumingii</italic> (Unionoida).</p><p><supplementary-material id="table2sdata1"><object-id pub-id-type="doi">10.7554/eLife.45644.008</object-id><label>Table 2&#8212;source data 1.</label><caption><title>Palaeoshellomics.</title><p>The complete proteomics dataset obtained on reference shells and archaeological ornaments</p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-45644-table2-data1-v2.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" valign="bottom">Proteins present in database from</th><th rowspan="2">Identified proteins</th><th colspan="7">Double-buttons</th><th colspan="4">Freshwater</th><th colspan="2">Marine</th></tr><tr><th>Order</th><th>Genus</th><th>HavA</th><th>HavB</th><th>HavC</th><th>HorA</th><th>HorB</th><th>HorC</th><th>PesB</th><th>U.p</th><th>U.c</th><th>M.m</th><th>P.a</th><th>Mo.M</th><th>O.e</th></tr></thead><tbody><tr><td rowspan="3">Unionoida</td><td rowspan="3"><italic>Hyriopsis</italic></td><td>Hic74 [<italic>Hyriopsis cumingii</italic>]</td><td style="author-callout-style-b7">132</td><td style="author-callout-style-b7">158</td><td style="author-callout-style-b6">260</td><td/><td style="author-callout-style-b1">6</td><td style="author-callout-style-b1">11</td><td style="author-callout-style-b2">21</td><td>&#9679;</td><td>&#9679;</td><td>&#9679;</td><td>&#9679;</td><td valign="bottom"/><td/></tr><tr><td>Hic52 nacreous layer matrix protein [<italic>Hyriopsis cumingii</italic>]</td><td style="author-callout-style-b8">1*</td><td style="author-callout-style-b8">1*</td><td style="author-callout-style-b8">2*</td><td/><td/><td/><td/><td>&#9679;</td><td>&#9679;</td><td>&#9679;</td><td>&#9679;</td><td/><td/></tr><tr><td>Silkmapin (isoforms: nasilin 1 and nasilin 2) [<italic>Hyriopsis cumingii</italic>]</td><td style="author-callout-style-b8">1*</td><td style="author-callout-style-b1">3</td><td style="author-callout-style-b1">3</td><td/><td style="author-callout-style-b2">5</td><td/><td/><td>&#9679;</td><td>&#9679;</td><td>&#9679;*</td><td>&#9679;*</td><td/><td/></tr><tr><td rowspan="5">Ostreida</td><td rowspan="2"><italic>Pinctada</italic></td><td>MSI60-related protein <italic>[Pinctada fucata</italic>]</td><td style="author-callout-style-b1">6</td><td/><td style="author-callout-style-b2">27</td><td/><td/><td style="author-callout-style-b1">12</td><td/><td>&#9679;</td><td>&#9679;</td><td/><td>&#9679;</td><td>&#9679;</td><td/></tr><tr><td>Insoluble matrix protein [<italic>Pinctada fucata</italic>]</td><td style="author-callout-style-b8">4</td><td/><td style="author-callout-style-b2">33</td><td/><td/><td/><td/><td>&#9679;</td><td>&#9679;</td><td/><td>&#9679;</td><td/><td/></tr><tr><td rowspan="3"><italic>Crassostrea</italic></td><td>Glycine-rich cell wall structural protein-like [<italic>Crassostrea virginica/gigas</italic>]</td><td style="author-callout-style-b6">17</td><td style="author-callout-style-b2">11</td><td style="author-callout-style-b7">12</td><td/><td/><td/><td style="author-callout-style-b7">14</td><td/><td>&#9679;</td><td/><td/><td/><td/></tr><tr><td>Glycine-rich protein 23-like [<italic>Crassostrea virginica</italic>]</td><td style="author-callout-style-b7">8</td><td style="author-callout-style-b7">11</td><td style="author-callout-style-b2">6</td><td/><td/><td/><td/><td>&#9679;</td><td/><td>&#9679;</td><td/><td/><td/></tr><tr><td>Antifreeze protein Maxi-like <italic>[Crassostrea virginica</italic>]</td><td/><td style="author-callout-style-b2">4</td><td style="author-callout-style-b2">4</td><td/><td/><td/><td/><td/><td/><td/><td>&#9679;</td><td/><td/></tr><tr><td rowspan="3">Mytilida</td><td><italic>Bathymodiolus</italic></td><td>MSI60-related protein partial [<italic>Bathymodiolus platifrons</italic>]</td><td/><td style="author-callout-style-b2">6</td><td style="author-callout-style-b2">11</td><td/><td/><td/><td/><td>&#9679;</td><td/><td/><td>&#9679;</td><td>&#9679;</td><td/></tr><tr><td rowspan="2"><italic>Mytilus</italic></td><td>Precollagen D [<italic>Mytilus edulis</italic>]</td><td style="author-callout-style-b8">16</td><td style="author-callout-style-b8">26</td><td style="author-callout-style-b8">23</td><td/><td/><td/><td style="author-callout-style-b8">9</td><td>&#9679;</td><td>&#9679;</td><td>&#9679;</td><td/><td/><td/></tr><tr><td>Nongradient byssal precursor [<italic>Mytilus edulis</italic>]</td><td style="author-callout-style-b8">10</td><td style="author-callout-style-b8">10</td><td/><td/><td/><td/><td style="author-callout-style-b8">25</td><td/><td>&#9679;</td><td/><td/><td/><td/></tr><tr><td rowspan="2">&#8195;Other</td><td rowspan="2"><italic>Other</italic></td><td>Predicted: transcription&#160;factor hamlet-like partial [<italic>Octopus bimaculoides</italic>]</td><td style="author-callout-style-b8">5</td><td style="author-callout-style-b8">6</td><td style="author-callout-style-b8">11</td><td/><td/><td/><td/><td>&#9679;</td><td/><td>&#9679;</td><td/><td/><td/></tr><tr><td>Hypothetical protein OCBIM_22008720 mg partial [<italic>Octopus bimaculoides</italic>]</td><td style="author-callout-style-b2">6</td><td style="author-callout-style-b2">11</td><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/><td/></tr><tr><td valign="bottom"/><td/><td/><td/><td/><td/><td/><td colspan="2">Coverage</td><td style="author-callout-style-b6">&#8805;55%</td><td style="author-callout-style-b7">&#8805;35%</td><td style="author-callout-style-b2">&#8805;15%</td><td style="author-callout-style-b1">&#8805;10%</td><td style="author-callout-style-b8">&#8805;1%</td><td colspan="2">Presence &#9679;</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-45644-v2.xml">elife-45644-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="app1table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.45644.036</object-id><label>Appendix 1&#8212;table 3.</label><caption><title>Total hydrolyzable amino acid (THAA) D/L values measured in archaeological double-buttons.</title><p>Average and standard deviation were calculated on two analytical replicates. Values for Havn&#248; include the average and standard deviation for the two subsamples taken from each double-button.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2"><bold>Asx D/L</bold></th><th colspan="2"><bold>Glx D/L</bold></th><th colspan="2"><bold>Ser D/L</bold></th><th colspan="2"><bold>Ala D/L</bold></th></tr><tr><th>AV</th><th>SD</th><th>AV</th><th>SD</th><th>AV</th><th>SD</th><th>AV</th><th>SD</th></tr></thead><tbody><tr><td><bold>HOR-A</bold></td><td>0.430</td><td>0.000</td><td>0.740</td><td>0.003</td><td>0.000</td><td>0.000</td><td>0.880</td><td>0.001</td></tr><tr><td><bold>HOR-B</bold></td><td>0.530</td><td>0.073</td><td>0.150</td><td>0.421</td><td>0.720</td><td>0.515</td><td>0.200</td><td>0.477</td></tr><tr><td><bold>HOR-C</bold></td><td>0.580</td><td>0.033</td><td>0.170</td><td>0.016</td><td>0.760</td><td>0.026</td><td>0.210</td><td>0.010</td></tr><tr><td><bold>HAV-A</bold></td><td>0.313</td><td>0.015</td><td>0.128</td><td>0.032</td><td>0.455</td><td>0.030</td><td>0.165</td><td>0.013</td></tr><tr><td><bold>HAV-B</bold></td><td>0.318</td><td>0.010</td><td>0.135</td><td>0.010</td><td>0.480</td><td>0.024</td><td>0.168</td><td>0.010</td></tr><tr><td><bold>HAV-C</bold></td><td>0.308</td><td>0.010</td><td>0.120</td><td>0.008</td><td>0.475</td><td>0.026</td><td>0.175</td><td>0.006</td></tr><tr><td><bold>PES-B</bold></td><td>0.310</td><td>0.000</td><td>0.080</td><td>0.000</td><td>0.280</td><td>0.000</td><td>0.100</td><td>0.000</td></tr><tr><td rowspan="2"/><td colspan="2"><bold>Val D/L</bold></td><td colspan="2"><bold>Phe D/L</bold></td><td colspan="2"><bold>Ile D/L</bold></td><td/><td/></tr><tr><td>&#8195;AV</td><td>SD</td><td>AV</td><td>SD</td><td>AV</td><td>SD</td><td/><td/></tr><tr><td><bold>HOR-A</bold></td><td>0.740</td><td>0.001</td><td>0.730</td><td>0.004</td><td>0.850</td><td>0.029</td><td/><td/></tr><tr><td><bold>HOR-B</bold></td><td>0.110</td><td>0.449</td><td>0.290</td><td>0.308</td><td>0.140</td><td>0.488</td><td/><td/></tr><tr><td><bold>HOR-C</bold></td><td>0.080</td><td>0.019</td><td>0.320</td><td>0.026</td><td>0.100</td><td>0.039</td><td/><td/></tr><tr><td><bold>HAV-A</bold></td><td>0.073</td><td>0.015</td><td>0.180</td><td>0.008</td><td>0.045</td><td>0.052</td><td/><td/></tr><tr><td><bold>HAV-B</bold></td><td>0.075</td><td>0.013</td><td>0.193</td><td>0.013</td><td>0.065</td><td>0.044</td><td/><td/></tr><tr><td><bold>HAV-C</bold></td><td>0.078</td><td>0.010</td><td>0.170</td><td>0.008</td><td>0.023</td><td>0.045</td><td/><td/></tr><tr><td><bold>PES-B</bold></td><td>0.000</td><td>0.000</td><td>0.140</td><td>0.000</td><td>0.000</td><td>0.000</td><td/><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46080-v2.xml">elife-46080-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.46080.029</object-id><label>Table 1.</label><caption><title>Amnesic patient demographic and neuropsychological data.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Patient #</th><th rowspan="2">Diagnosis</th><th rowspan="2">Gender</th><th rowspan="2">Age</th><th rowspan="2">Edu</th><th colspan="2">WAIS-III</th><th colspan="3">WMS-III</th><th rowspan="2">BNT</th><th rowspan="2">FAS</th><th rowspan="2">L-N sequence</th><th rowspan="2">Years since onset</th></tr><tr><th>VIQ</th><th>WMI</th><th>GM</th><th>VD</th><th>AD</th></tr></thead><tbody><tr><td>P01</td><td>Hypoxic-ischemic</td><td>F</td><td>67</td><td>12</td><td>88</td><td>75</td><td>52</td><td>56</td><td>55</td><td>&#8722;1.3</td><td>&#8722;1.1</td><td>-2</td><td>27.29</td></tr><tr><td>P02</td><td>Status epilepticus&#160;+&#160;left temp. lobectomy</td><td>M</td><td>54</td><td>16</td><td>93</td><td>94</td><td>49</td><td>53</td><td>52</td><td>&#8722;4.6</td><td>&#8722;0.96</td><td>-1</td><td>29.17</td></tr><tr><td>P03</td><td>Hypoxic-ischemic</td><td>M</td><td>61</td><td>14</td><td>106</td><td>115</td><td>59</td><td>72</td><td>52</td><td>0.54</td><td>&#8722;0.78</td><td>1.33</td><td>24.18</td></tr><tr><td>P04</td><td>Hypoxic-ischemic</td><td>M</td><td>65</td><td>17</td><td>131</td><td>126</td><td>86</td><td>78</td><td>86</td><td>1.3</td><td>0.03</td><td>1.33</td><td>15.00</td></tr><tr><td>P05</td><td>Encephalitis</td><td>M</td><td>75</td><td>13</td><td>99</td><td>104</td><td>49</td><td>56</td><td>58</td><td>&#8722;0.11</td><td>&#8722;0.5</td><td>0.33</td><td>5.85</td></tr><tr><td>P06</td><td>Stroke</td><td>M</td><td>53</td><td>20</td><td>111</td><td>99</td><td>60</td><td>65</td><td>58</td><td>1.02</td><td>2.1</td><td>&#8722;0.33</td><td>3.45</td></tr></tbody></table><table-wrap-foot><fn><p>Age in years at first session; Edu, education in years; WAIS-III, Wechsler Adult Intelligence Scale-III (<xref ref-type="bibr" rid="bib101">Wechsler, 1997a</xref>); WMS-III, Wechsler Memory Scale-III (<xref ref-type="bibr" rid="bib102">Wechsler, 1997b</xref>); VIQ, verbal IQ; WMI, working memory index; GM, general memory; VD, visual delayed; AD, auditory delayed; scores are age-adjusted such that a score of 100 is the age-adjusted mean with a standard deviation of 15; BNT, Boston Naming Test; FAS, verbal fluency test; L-N, Letter-Number Sequence. BNT, FAS and L-N scores were z-scored against normative data for each test.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46156-v2.xml">elife-46156-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.46156.003</object-id><label>Table 1.</label><caption><title>Neuropsychological profile of autoimmune LE patients (post-acute phase) and healthy controls</title></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2" valign="top">Domain</th><th rowspan="2" valign="top">Test</th><th rowspan="2" valign="top">Subtest</th><th colspan="2" valign="top">Controls</th><th colspan="2" valign="top">Patients</th><th colspan="3" valign="top">Controls vs patients</th><th colspan="3" valign="top">&#8216;Impaired' range</th></tr><tr><th valign="top">M</th><th valign="top">IQR</th><th valign="top">M</th><th valign="top">IQR</th><th valign="top">Test</th><th valign="top">Statistic</th><th valign="top">p-corr</th><th valign="top">cut-off score</th><th valign="top">Patients (n)</th><th valign="top">Controls (n)</th></tr></thead><tbody><tr><td rowspan="18" valign="top">Episodic Memory</td><td rowspan="3" valign="top">Immediate Verbal Recall</td><td rowspan="2" valign="top">WMS-III</td><td valign="top">Logical Memory I (z)</td><td valign="top">0.33</td><td valign="top">1.92</td><td valign="top">&#8722;1.00</td><td valign="top">1.34</td><td valign="top">t</td><td valign="top">6.78</td><td valign="top">&lt;0.0005</td><td valign="top">&#8804; - 1.67</td><td valign="top">14</td><td valign="top">0</td></tr><tr><td valign="top">&#8195;Word List I (z)</td><td valign="top">0.67</td><td valign="top">1.83</td><td valign="top">&#8722;1.00</td><td valign="top">1.51</td><td valign="top">U</td><td valign="top">178.00</td><td valign="top">&lt;0.0005</td><td valign="top">&#8804; - 1.67</td><td valign="top">11</td><td valign="top">0</td></tr><tr><td valign="top">&#8195;D and P</td><td valign="top">People (z)</td><td valign="top">&#8722;0.33</td><td valign="top">1.59</td><td valign="top">&#8722;1.33</td><td valign="top">1.00</td><td valign="top">U</td><td valign="top">266.00</td><td valign="top">&lt;0.0005</td><td valign="top">&#8804; - 1.67</td><td valign="top">15</td><td valign="top">1</td></tr><tr><td rowspan="2" valign="top">&#8195;Delayed Verbal Recall</td><td rowspan="2" valign="top">WMS-III</td><td valign="top">Logical Memory II (z)</td><td valign="top">0.67</td><td valign="top">1.67</td><td valign="top">&#8722;2.00</td><td valign="top">2.00</td><td valign="top">U</td><td valign="top">106.50</td><td valign="top">&lt;0.0005</td><td valign="top">&#8804; - 1.67</td><td valign="top">23</td><td valign="top">0</td></tr><tr><td valign="top">&#8195;Word List II (z)</td><td valign="top">1.33</td><td valign="top">1.00</td><td valign="top">&#8722;0.67</td><td valign="top">1.83</td><td valign="top">U</td><td valign="top">179.00</td><td valign="top">&lt;0.0005</td><td valign="top">&#8804; - 1.67</td><td valign="top">3</td><td valign="top">0</td></tr><tr><td valign="top">&#8195;Verbal Forgetting</td><td rowspan="2" valign="top">D and P</td><td valign="top">Verbal Forgetting (z)</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">&#8722;0.33</td><td valign="top">1.75</td><td valign="top">U</td><td valign="top">415.50</td><td valign="top">0.0584</td><td valign="top">&#8804; - 1.67</td><td valign="top">8</td><td valign="top">1</td></tr><tr><td rowspan="3" valign="top">&#8195;Verbal Recognition</td><td valign="top">Names (z)</td><td valign="top">0.33</td><td valign="top">2.00</td><td valign="top">&#8722;1.00</td><td valign="top">2.00</td><td valign="top">t</td><td valign="top">5.16</td><td valign="top">&lt;0.0005</td><td valign="top">&#8804; - 1.67</td><td valign="top">13</td><td valign="top">1</td></tr><tr><td valign="top">&#8195;RMT</td><td valign="top">Words (z)</td><td valign="top">1.00</td><td valign="top">1.51</td><td valign="top">0.00</td><td valign="top">2.26</td><td valign="top">U</td><td valign="top">317.50</td><td valign="top">&lt;0.0005</td><td valign="top">&#8804; - 1.67</td><td valign="top">9</td><td valign="top">4</td></tr><tr><td valign="top">&#8195;WMS-III</td><td valign="top">Word List II Recognition (z)</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">&#8722;0.67</td><td valign="top">2.16</td><td valign="top">U</td><td valign="top">266.50</td><td valign="top">&lt;0.0005</td><td valign="top">&#8804; - 1.67</td><td valign="top">9</td><td valign="top">0</td></tr><tr><td rowspan="2" valign="top">&#8195;Immediate Visual Recall</td><td valign="top">D and P</td><td valign="top">Shapes (z)</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">&#8722;0.84</td><td valign="top">2.08</td><td valign="top">Wt</td><td valign="top">5.78</td><td valign="top">&lt;0.0005</td><td valign="top">&#8804; - 1.67</td><td valign="top">13</td><td valign="top">0</td></tr><tr><td rowspan="2" valign="top">&#8195;ROCFT</td><td valign="top">Immediate Recall (z)</td><td valign="top">1.26</td><td valign="top">1.93</td><td valign="top">&#8722;0.93</td><td valign="top">2.58</td><td valign="top">U</td><td valign="top">255.50</td><td valign="top">&lt;0.0005</td><td valign="top">&#8804; - 1.67</td><td valign="top">12</td><td valign="top">1</td></tr><tr><td valign="top">&#8195;Delayed Visual Recall</td><td valign="top">Delayed Recall (z)</td><td valign="top">1.26</td><td valign="top">1.98</td><td valign="top">&#8722;1.37</td><td valign="top">3.55</td><td valign="top">U</td><td valign="top">258.00</td><td valign="top">&lt;0.0005</td><td valign="top">&#8804; - 1.67</td><td valign="top">16</td><td valign="top">1</td></tr><tr><td valign="top">&#8195;Visual Forgetting</td><td rowspan="2" valign="top">D and P</td><td valign="top">Visual Forgetting (z)</td><td valign="top">0.33</td><td valign="top">0.00</td><td valign="top">0.33</td><td valign="top">1.83</td><td valign="top">U</td><td valign="top">433.00</td><td valign="top">0.0098</td><td valign="top">&#8804; - 1.67</td><td valign="top">8</td><td valign="top">1</td></tr><tr><td rowspan="3" valign="top">&#8195;Visual Recognition</td><td valign="top">Doors (z)</td><td valign="top">0.67</td><td valign="top">1.33</td><td valign="top">&#8722;0.67</td><td valign="top">1.75</td><td valign="top">t</td><td valign="top">3.74</td><td valign="top">0.0072</td><td valign="top">&#8804; - 1.67</td><td valign="top">7</td><td valign="top">1</td></tr><tr><td rowspan="2" valign="top">&#8195;RMT</td><td valign="top">Scenes (z)</td><td valign="top">1.00</td><td valign="top">0.99</td><td valign="top">&#8722;0.35</td><td valign="top">2.65</td><td valign="top">U</td><td valign="top">281.00</td><td valign="top">0.0025</td><td valign="top">&#8804; - 1.67</td><td valign="top">9</td><td valign="top">0</td></tr><tr><td valign="top">&#8195;Faces (z)</td><td valign="top">0.00</td><td valign="top">2.33</td><td valign="top">&#8722;0.33</td><td valign="top">1.66</td><td valign="top">t</td><td valign="top">1.29</td><td valign="top">0.6536</td><td valign="top">&#8804; - 1.67</td><td valign="top">8</td><td valign="top">6</td></tr><tr><td rowspan="2" valign="top">&#8195;Autobiographical Memory</td><td rowspan="4" valign="top">AMI</td><td valign="top">Childhood (9)</td><td valign="top">9.00</td><td valign="top">3.00</td><td valign="top">5.00</td><td valign="top">4.00</td><td valign="top">U</td><td valign="top">174.50</td><td valign="top">&lt;0.0005</td><td valign="top">&#8804;3.00 *</td><td valign="top">11</td><td valign="top">1</td></tr><tr><td valign="top">&#8195;Early Adulthood (9)</td><td valign="top">9.00</td><td valign="top">1.50</td><td valign="top">4.00</td><td valign="top">4.00</td><td valign="top">U</td><td valign="top">123.00</td><td valign="top">&lt;0.0005</td><td valign="top">&#8804;3.00 *</td><td valign="top">14</td><td valign="top">0</td></tr><tr><td rowspan="7" valign="top">Intelligence, <break/>Semantic Memory, and Language</td><td rowspan="2" valign="top">Personal Semantic Memory</td><td valign="top">Childhood (21)</td><td valign="top">19.50</td><td valign="top">3.00</td><td valign="top">18.00</td><td valign="top">5.00</td><td valign="top">U</td><td valign="top">267.00</td><td valign="top">0.0835</td><td valign="top">&#8804;11.00 *</td><td valign="top">3</td><td valign="top">0</td></tr><tr><td valign="top">Early Adulthood (21)</td><td valign="top">20.50</td><td valign="top">2.00</td><td valign="top">19.00</td><td valign="top">2.50</td><td valign="top">U</td><td valign="top">263.00</td><td valign="top">0.0687</td><td valign="top">&#8804;14.00 *</td><td valign="top">4</td><td valign="top">1</td></tr><tr><td rowspan="5" valign="top"/><td valign="top">NART</td><td valign="top">p-FSIQ (z)</td><td valign="top">1.44</td><td valign="top">0.85</td><td valign="top">1.04</td><td valign="top">1.05</td><td valign="top">U</td><td valign="top">486.00</td><td valign="top">0.1281</td><td valign="top">&#8804; - 1.67</td><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top">WASI/</td><td valign="top">Vocabulary (z)</td><td valign="top">1.40</td><td valign="top">1.25</td><td valign="top">0.70</td><td valign="top">1.20</td><td valign="top">t</td><td valign="top">3.05</td><td valign="top">0.0584</td><td valign="top">&#8804; - 1.67</td><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top">WASI-II</td><td valign="top">Similarities (z)</td><td valign="top">1.05</td><td valign="top">0.80</td><td valign="top">0.70</td><td valign="top">0.85</td><td valign="top">U</td><td valign="top">378.00</td><td valign="top">0.1024</td><td valign="top">&#8804; - 1.67</td><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top">GNT</td><td valign="top">(z)</td><td valign="top">0.63</td><td valign="top">0.98</td><td valign="top">0.15</td><td valign="top">1.89</td><td valign="top">U</td><td valign="top">423.50</td><td valign="top">0.0683</td><td valign="top">&#8804; - 1.67</td><td valign="top">5</td><td valign="top">0</td></tr><tr><td valign="top">C and CT</td><td valign="top">(z)</td><td valign="top">0.34</td><td valign="top">1.22</td><td valign="top">0.02</td><td valign="top">1.22</td><td valign="top">U</td><td valign="top">496.50</td><td valign="top">0.1484</td><td valign="top">&#8804; - 1.67</td><td valign="top">5</td><td valign="top">0</td></tr><tr><td colspan="2" rowspan="2" valign="top">Executive Function</td><td valign="top">WMS-III</td><td valign="top">Digit Span (z)</td><td valign="top">0.84</td><td valign="top">1.25</td><td valign="top">0.33</td><td valign="top">1.67</td><td valign="top">t</td><td valign="top">2.70</td><td valign="top">0.1024</td><td valign="top">&#8804; - 1.67</td><td valign="top">2</td><td valign="top">1</td></tr><tr><td rowspan="3" valign="top">DKEFS Trails</td><td valign="top">Number-Letter Switching (z)</td><td valign="top">0.67</td><td valign="top">0.67</td><td valign="top">0.33</td><td valign="top">1.00</td><td valign="top">U</td><td valign="top">470.00</td><td valign="top">0.1024</td><td valign="top">&#8804; - 1.67</td><td valign="top">3</td><td valign="top">1</td></tr><tr><td colspan="2" rowspan="6" valign="top">Visuomotor Function</td><td valign="top">Visual Scanning (z)</td><td valign="top">0.67</td><td valign="top">1.50</td><td valign="top">0.00</td><td valign="top">1.34</td><td valign="top">U</td><td valign="top">584.00</td><td valign="top">0.6536</td><td valign="top">&#8804; - 1.67</td><td valign="top">5</td><td valign="top">4</td></tr><tr><td valign="top">Motor Speed (z)</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">0.33</td><td valign="top">1.34</td><td valign="top">U</td><td valign="top">552.00</td><td valign="top">0.4915</td><td valign="top">&#8804; - 1.67</td><td valign="top">7</td><td valign="top">5</td></tr><tr><td valign="top">ROCFT</td><td valign="top">copy rank</td><td valign="top">&gt; 16<sup>th</sup> %ile</td><td valign="top">0.00</td><td valign="top">&gt; 16<sup>th</sup> %ile</td><td valign="top">0.00</td><td valign="top">U</td><td valign="top">619.00</td><td valign="top">0.4915</td><td valign="top">&#8804; 16<sup>th</sup> %ile</td><td valign="top">2</td><td valign="top">1</td></tr><tr><td rowspan="3" valign="top">VOSP</td><td valign="top">Cube Analysis (z)</td><td valign="top">10.00</td><td valign="top">1.00</td><td valign="top">9.00</td><td valign="top">2.00</td><td valign="top">U</td><td valign="top">548.00</td><td valign="top">0.3116</td><td valign="top">&#8804;6.00 **</td><td valign="top">3</td><td valign="top">0</td></tr><tr><td valign="top">Dot Counting (z)</td><td valign="top">10.00</td><td valign="top">0.00</td><td valign="top">10.00</td><td valign="top">0.00</td><td valign="top">U</td><td valign="top">655.50</td><td valign="top">0.6536</td><td valign="top">&#8804;8.00 **</td><td valign="top">1</td><td valign="top">1</td></tr><tr><td valign="top">Position Discrimination (z)</td><td valign="top">20.00</td><td valign="top">0.00</td><td valign="top">20.00</td><td valign="top">1.00</td><td valign="top">U</td><td valign="top">673.00</td><td valign="top">0.6536</td><td valign="top">&#8804;18.00 **</td><td valign="top">4</td><td valign="top">2</td></tr><tr><td colspan="2" rowspan="2" valign="top">Mood</td><td rowspan="2" valign="top">HADS</td><td valign="top">Anxiety (21)</td><td valign="top">4.00</td><td valign="top">4.00</td><td valign="top">5.00</td><td valign="top">5.50</td><td valign="top">U</td><td valign="top">420.00</td><td valign="top">0.0910</td><td valign="top">&#8805;15.00&#160;***</td><td valign="top">3</td><td valign="top">0</td></tr><tr><td valign="top">Depression (21)</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">3.00</td><td valign="top">4.50</td><td valign="top">U</td><td valign="top">298.00</td><td valign="top">0.0006</td><td valign="top">&#8805;15.00&#160;***</td><td valign="top">0</td><td valign="top">0</td></tr></tbody></table><table-wrap-foot><fn><p>AMI: Autobiographical Memory Interview; D and P: Doors and People Test; DKEFS: Delis-Kaplan Executive Function System; GNT: Graded Naming Test; HADS: Hospital Anxiety and Depression Scale; IQR: Inter-Quartile Range; M: median; NART: National Adult Reading Test; p-corr: p values are corrected using the Holm-Bonferroni sequential correction for multiple comparisons (n&#160;=&#160;35); RMT: Warrington Recognition Memory Tests (words, faces) and Warrington Topographical Memory test (scenes); ROCFT: Rey-Osterrieth Complex Figure Test; <italic>t</italic>: Student&#8217;s t-test; <italic>U</italic>: Mann-Whitney U; VOSP: Visual Object and Space Perception Battery; WASI/WASI-II Wechsler Abbreviated Scale of Intelligence; WMS-III: Wechsler Memory Scale III; <italic>Wt</italic>: Welch&#8217;s t-test; *,**,***: no standardized scores available for these subtests; *: highest score of &#8216;definitely abnormal&#8217; range,&#160;that&#160;is scores at or below which none of the healthy controls scored in <xref ref-type="bibr" rid="bib70">Kopelman et al. (1989)</xref>; **: 5% cut-off score; ***: cut-off score for severe range.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46156-v2.xml">elife-46156-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.46156.022</object-id><label>Table 2.</label><caption><title>Clinical details of autoimmune LE patients (acute phase).</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top">Code</th><th rowspan="3" valign="top">Age (years)</th><th rowspan="3" valign="top">Sex</th><th rowspan="3" valign="top">Antibody type</th><th colspan="3" valign="top">Acute T2 scan notes</th></tr><tr><th colspan="2" valign="top">HPC</th><th rowspan="2" valign="top">Other structures</th></tr><tr><th valign="top">R</th><th valign="top">L</th></tr></thead><tbody><tr><td valign="top">1</td><td valign="top">65.75</td><td valign="top">M</td><td valign="top">LGI1</td><td valign="top">Normal T2 signal and volume; facilitated diffusion</td><td valign="top">High T2 signal; swelling; normal diffusion</td><td valign="top">L AMG: high T2 signal</td></tr><tr><td valign="top">2</td><td valign="top">69.98</td><td valign="top">F</td><td valign="top">VGKCC</td><td valign="top">Normal T2 signal; mild atrophy; facilitated diffusion</td><td valign="top">High T2 signal; normal volume; facilitated diffusion</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">3</td><td valign="top">62.23</td><td valign="top">M</td><td valign="top">VGKCC</td><td valign="top">Normal T2 signal and volume; facilitated diffusion</td><td valign="top">High T2 signal; swelling; normal diffusion</td><td valign="top">L AMG, L ERC: high T2 signal</td></tr><tr><td valign="top">4</td><td valign="top">46.41</td><td valign="top">M</td><td valign="top">LGI1</td><td valign="top">High T2 signal; normal volume; normal diffusion</td><td valign="top">Normal T2 signal and volume; facilitated diffusion</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">5</td><td valign="top">56.65</td><td valign="top">M</td><td valign="top">LGI1</td><td colspan="2" valign="top">L/R: high T2 signal</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">6</td><td valign="top">58.18</td><td valign="top">M</td><td valign="top">LGI1</td><td colspan="3" valign="top">No abnormalities</td></tr><tr><td valign="top">7</td><td valign="top">56.13</td><td valign="top">M</td><td valign="top">LGI1</td><td valign="top">Normal volume and signal</td><td valign="top">High T2 signal; swelling</td><td valign="top">L/R AMG: high T2 signal</td></tr><tr><td valign="top">8</td><td valign="top">76.54</td><td valign="top">M</td><td valign="top">LGI1</td><td valign="top">High T2 signal; normal volume and diffusion</td><td valign="top">High T2 signal; normal volume; facilitated diffusion</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">9</td><td valign="top">54.94</td><td valign="top">M</td><td valign="top">LGI1</td><td valign="top">High T2 signal; swelling; normal diffusion</td><td valign="top">High T2 signal; swelling; normal diffusion</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">10</td><td valign="top">44.81</td><td valign="top">M</td><td valign="top">LGI1</td><td colspan="2" valign="top">L/R: high T2 signal; swelling</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">11</td><td valign="top">45.77</td><td valign="top">M</td><td valign="top">LGI1</td><td valign="top">High T2 signal; normal volume</td><td valign="top">High T2 signal; normal volume</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">12</td><td valign="top">46.06</td><td valign="top">M</td><td valign="top">LGI1/Caspr2</td><td valign="top">High T2 signal; atrophy</td><td valign="top">Normal T2 signal and volume</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">13</td><td valign="top">35.75</td><td valign="top">M</td><td valign="top">LGI1/Caspr2</td><td colspan="2" valign="top">L/R: normal T2 signal; mild atrophy; normal diffusion</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">14</td><td valign="top">72.08</td><td valign="top">M</td><td valign="top">LGI1</td><td valign="top">High T2 signal; mild atrophy; facilitated diffusion</td><td valign="top">Normal T2 signal; atrophy; facilitated diffusion</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">15</td><td valign="top">52.28</td><td valign="top">M</td><td valign="top">LGI1</td><td valign="top">High T2 signal; normal volume; facilitated diffusion</td><td valign="top">Normal T2 signal and volume; facilitated diffusion</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">16</td><td valign="top">52.48</td><td valign="top">M</td><td valign="top">LGI1/Caspr2</td><td valign="top">High T2 signal; swelling; facilitated diffusion</td><td valign="top">Normal T2 signal and volume; facilitated diffusion</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">17</td><td valign="top">51.62</td><td valign="top">M</td><td valign="top">VGKCC</td><td valign="top">High T2 signal; swelling</td><td valign="top">Normal T2 signal and volume</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">18</td><td valign="top">75.18</td><td valign="top">M</td><td valign="top">LGI1</td><td colspan="2" valign="top">L/R: high T2 signal; swelling; normal diffusion</td><td valign="top">L/R AMG: high T2 signal; swelling</td></tr><tr><td valign="top">19</td><td valign="top">78.73</td><td valign="top">M</td><td valign="top">LG1/Caspr2</td><td valign="top">High T2 signal; mild atrophy; normal diffusion</td><td valign="top">High T2 signal; normal volume; normal diffusion</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">20</td><td valign="top">53.75</td><td valign="top">F</td><td valign="top">LGI1</td><td colspan="2" valign="top">L/R: high T2 signal; normal volume and diffusion</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">21</td><td valign="top">73.68</td><td valign="top">F</td><td valign="top">VGKCC</td><td colspan="2" valign="top">L/R: high T2 signal; swelling; facilitated diffusion</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">22</td><td valign="top">63.59</td><td valign="top">M</td><td valign="top">LGI1</td><td colspan="2" valign="top">L/R: high T2 signal; normal volume and diffusion</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">23</td><td valign="top">60.35</td><td valign="top">M</td><td valign="top">VGKCC</td><td colspan="3" valign="top">No abnormalities</td></tr><tr><td valign="top">24</td><td valign="top">54.30</td><td valign="top">M</td><td valign="top">VGKCC</td><td colspan="2" valign="top">L/R: high T2 signal; atrophy</td><td valign="top">L/R AMG: high T2 signal; atrophy</td></tr><tr><td valign="top">25</td><td valign="top">52.70</td><td valign="top">M</td><td valign="top">seronegative</td><td colspan="2" valign="top">L/R: high T2 signal</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">26</td><td valign="top">47.43</td><td valign="top">F</td><td valign="top">seronegative</td><td colspan="3" valign="top">No abnormalities</td></tr><tr><td valign="top">27</td><td valign="top">58.60</td><td valign="top">M</td><td valign="top">seronegative</td><td colspan="2" valign="top">L/R: high T2 signal</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">28</td><td valign="top">25.42</td><td valign="top">M</td><td valign="top">Anti-Ma2</td><td colspan="2" valign="top">L/R: high T2 signal and swelling</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">29</td><td valign="top">45.77</td><td valign="top">F</td><td valign="top">seronegative</td><td colspan="2" valign="top">L/R: high T2 signal</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">30</td><td valign="top">16.64</td><td valign="top">F</td><td valign="top">GAD</td><td colspan="3" valign="top">No abnormalities</td></tr><tr><td valign="top">31</td><td valign="top">71.35</td><td valign="top">M</td><td valign="top">seronegative</td><td colspan="2" valign="top">L/R: high T2 signal; atrophy</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">32</td><td valign="top">60.44</td><td valign="top">M</td><td valign="top">VGKCC</td><td colspan="2" valign="top">L/R: atrophy</td><td valign="top">PHC atrophy</td></tr><tr><td valign="top">33</td><td valign="top">53.48</td><td valign="top">M</td><td valign="top">seronegative</td><td colspan="2" valign="top">L/R: atrophy</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">34</td><td valign="top">64.87</td><td valign="top">F</td><td valign="top">seronegative</td><td colspan="2" valign="top">L/R: atrophy</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">35</td><td valign="top">47.32</td><td valign="top">F</td><td valign="top">seronegative</td><td colspan="2" valign="top">L/R: high T2 signal</td><td valign="top">R AMG: high T2 signal; swelling</td></tr><tr><td valign="top">36</td><td valign="top">61.88</td><td valign="top">F</td><td valign="top">seronegative</td><td colspan="2" valign="top">L/R: high T2 signal; atrophy</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">37</td><td valign="top">71.90</td><td valign="top">F</td><td valign="top">seronegative</td><td colspan="2" valign="top">L/R: high T2 signal (especially R)</td><td valign="top">No abnormalities</td></tr><tr><td valign="top">38</td><td valign="top">34.49</td><td valign="top">F</td><td valign="top">GAD</td><td colspan="2" valign="top">L/R: high T2 signal</td><td valign="top">No abnormalities</td></tr></tbody></table><table-wrap-foot><fn><p>Age: age at symptom onset (years); AMG: Amygdala; Caspr2: anti-contactin-associated protein-like 2; ERC: entorhinal cortex; F&#160;=&#160;female; GAD: anti-glutamic acid decarboxylase autoantibody; HPC: hippocampus; L: left hemisphere; LGI1: anti-leucine-rich glioma-inactivated1; M&#160;=&#160;male; PHC: parahippocampal cortex; R: right hemisphere; VGKCC: anti-voltage-gated potassium channel complex. The clinical details of patients 1&#8211;24 have also been presented in <xref ref-type="bibr" rid="bib80">Loane et al. (2019)</xref>.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46156-v2.xml">elife-46156-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.46156.004</object-id><label>Table 3.</label><caption><title>Volumetry of MTL and subcortical structures in autoimmune LE patients.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Structure</th><th colspan="2" valign="top">Controls</th><th colspan="2" valign="top">Patients</th><th rowspan="2" valign="top">Mean % reduction</th><th rowspan="2" valign="top"><italic>F</italic></th><th rowspan="2" valign="top">Partial <italic>&#951;</italic><sup>2</sup></th><th rowspan="2" valign="top">p-corr</th></tr><tr><th valign="top">Mean (mm<sup>3</sup>)</th><th valign="top">SD (mm<sup>3</sup>)</th><th valign="top">Mean (mm<sup>3</sup>)</th><th valign="top">SD (mm<sup>3</sup>)</th></tr></thead><tbody><tr><td valign="top">R HPC</td><td valign="top">3648.99</td><td valign="top">459.59</td><td valign="top">2733.87</td><td valign="top">751.09</td><td valign="top">&#8722;25.08</td><td valign="top">63.38</td><td valign="top">0.390</td><td valign="top">&lt;0.0005</td></tr><tr><td valign="top">L HPC</td><td valign="top">3439.48</td><td valign="top">431.91</td><td valign="top">2671.18</td><td valign="top">710.65</td><td valign="top">&#8722;22.34</td><td valign="top">46.02</td><td valign="top">0.317</td><td valign="top">&lt;0.0005</td></tr><tr><td valign="top">R ERC</td><td valign="top">1602.69</td><td valign="top">324.07</td><td valign="top">1254.97</td><td valign="top">404.70</td><td valign="top">&#8722;21.70</td><td valign="top">10.76</td><td valign="top">0.119</td><td valign="top">0.0308</td></tr><tr><td valign="top">L ERC</td><td valign="top">1508.83</td><td valign="top">326.07</td><td valign="top">1200.18</td><td valign="top">432.07</td><td valign="top">&#8722;20.46</td><td valign="top">9.48</td><td valign="top">0.106</td><td valign="top">0.0534</td></tr><tr><td valign="top">R Thalamus</td><td valign="top">7407.51</td><td valign="top">762.15</td><td valign="top">7072.79</td><td valign="top">845.09</td><td valign="top">&#8722;4.52</td><td valign="top">8.23</td><td valign="top">0.077</td><td valign="top">0.0900</td></tr><tr><td valign="top">L Thalamus</td><td valign="top">7633.09</td><td valign="top">788.11</td><td valign="top">7194.76</td><td valign="top">797.73</td><td valign="top">&#8722;5.74</td><td valign="top">15.41</td><td valign="top">0.135</td><td valign="top">0.0034</td></tr><tr><td valign="top">R PRC</td><td valign="top">1791.42</td><td valign="top">378.97</td><td valign="top">1561.61</td><td valign="top">403.86</td><td valign="top">&#8722;12.83</td><td valign="top">7.11</td><td valign="top">0.082</td><td valign="top">0.1575</td></tr><tr><td valign="top">L PRC</td><td valign="top">1812.50</td><td valign="top">523.31</td><td valign="top">1601.39</td><td valign="top">478.78</td><td valign="top">&#8722;11.65</td><td valign="top">2.88</td><td valign="top">0.035</td><td valign="top">&gt;0.9999</td></tr><tr><td valign="top">R PHC</td><td valign="top">1900.73</td><td valign="top">423.85</td><td valign="top">1665.68</td><td valign="top">331.28</td><td valign="top">&#8722;12.37</td><td valign="top">6.36</td><td valign="top">0.074</td><td valign="top">0.2240</td></tr><tr><td valign="top">L PHC</td><td valign="top">2016.52</td><td valign="top">435.92</td><td valign="top">1851.53</td><td valign="top">445.52</td><td valign="top">&#8722;8.18</td><td valign="top">1.53</td><td valign="top">0.019</td><td valign="top">&gt;0.9999</td></tr><tr><td valign="top">R AMG</td><td valign="top">1395.67</td><td valign="top">267.38</td><td valign="top">1313.76</td><td valign="top">406.51</td><td valign="top">&#8722;5.87</td><td valign="top">5.15</td><td valign="top">0.060</td><td valign="top">0.3899</td></tr><tr><td valign="top">L AMG</td><td valign="top">1321.96</td><td valign="top">217.40</td><td valign="top">1268.47</td><td valign="top">384.22</td><td valign="top">&#8722;4.05</td><td valign="top">4.43</td><td valign="top">0.053</td><td valign="top">0.5320</td></tr><tr><td valign="top">R Nacc</td><td valign="top">339.42</td><td valign="top">107.77</td><td valign="top">318.53</td><td valign="top">109.69</td><td valign="top">&#8722;6.16</td><td valign="top">0.25</td><td valign="top">0.003</td><td valign="top">&gt;0.9999</td></tr><tr><td valign="top">L Nacc</td><td valign="top">433.79</td><td valign="top">127.41</td><td valign="top">381.89</td><td valign="top">153.66</td><td valign="top">&#8722;11.96</td><td valign="top">3.70</td><td valign="top">0.036</td><td valign="top">0.7410</td></tr><tr><td valign="top">R TPC</td><td valign="top">4558.60</td><td valign="top">846.26</td><td valign="top">4520.16</td><td valign="top">1026.08</td><td valign="top">&#8722;0.84</td><td valign="top">0.12</td><td valign="top">0.002</td><td valign="top">&gt;0.9999</td></tr><tr><td valign="top">L TPC</td><td valign="top">4331.40</td><td valign="top">742.35</td><td valign="top">4512.95</td><td valign="top">795.76</td><td valign="top">4.19</td><td valign="top">1.90</td><td valign="top">0.023</td><td valign="top">&gt;0.9999</td></tr><tr><td valign="top">R Putamen</td><td valign="top">4332.54</td><td valign="top">548.04</td><td valign="top">4157.00</td><td valign="top">630.21</td><td valign="top">&#8722;4.05</td><td valign="top">0.85</td><td valign="top">0.008</td><td valign="top">&gt;0.9999</td></tr><tr><td valign="top">L Putamen</td><td valign="top">4382.39</td><td valign="top">705.40</td><td valign="top">4151.89</td><td valign="top">662.93</td><td valign="top">&#8722;5.26</td><td valign="top">1.67</td><td valign="top">0.017</td><td valign="top">&gt;0.9999</td></tr><tr><td valign="top">R Caudate</td><td valign="top">3403.96</td><td valign="top">440.42</td><td valign="top">3369.16</td><td valign="top">453.63</td><td valign="top">&#8722;1.02</td><td valign="top">0.06</td><td valign="top">0.001</td><td valign="top">&gt;0.9999</td></tr><tr><td valign="top">L Caudate</td><td valign="top">3211.40</td><td valign="top">438.40</td><td valign="top">3134.26</td><td valign="top">497.42</td><td valign="top">&#8722;2.40</td><td valign="top">0.12</td><td valign="top">0.001</td><td valign="top">&gt;0.9999</td></tr><tr><td valign="top">R Pallidum</td><td valign="top">1705.61</td><td valign="top">260.63</td><td valign="top">1630.00</td><td valign="top">251.08</td><td valign="top">&#8722;4.43</td><td valign="top">0.90</td><td valign="top">0.009</td><td valign="top">&gt;0.9999</td></tr><tr><td valign="top">L Pallidum</td><td valign="top">1720.40</td><td valign="top">297.86</td><td valign="top">1661.13</td><td valign="top">308.18</td><td valign="top">&#8722;3.45</td><td valign="top">0.41</td><td valign="top">0.004</td><td valign="top">&gt;0.9999</td></tr><tr><td valign="top">brainstem</td><td valign="top">22119.06</td><td valign="top">2258.25</td><td valign="top">21931.42</td><td valign="top">2647.93</td><td valign="top">&#8722;0.85</td><td valign="top">0.23</td><td valign="top">0.002</td><td valign="top">&gt;0.9999</td></tr></tbody></table><table-wrap-foot><fn><p>Volumetry of manually and automatically delineated MTL and other subcortical structures of all patients (n&#160;=&#160;38). Volumes for each structure are compared between patients and controls, using age, sex, TIV, and scan source (MAP, OPTIMA) as between-subjects covariates in a series of univariate ANCOVAs; AMG: amygdala; ANCOVA: analysis of covariance; ; ERC: entorhinal cortex; HPC: hippocampus; L: left hemisphere; MAP: Memory and Amnesia Project; MTL: medial temporal lobe; Nacc: nucleus accumbens; OPTIMA: Oxford Project To Investigate Memory and Aging; p-corr: p values are adjusted with the Holm-Bonferroni sequential correction method for multiple comparisons (n&#160;=&#160;23); PHC: parahippocampal cortex; PRC: perirhinal cortex; R: right&#160;hemisphere; SD: standard deviation; TIV: total intracranial volume; TPC: temporopolar cortex.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46156-v2.xml">elife-46156-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.46156.008</object-id><label>Table 4.</label><caption><title>GM volume reduction in autoimmune LE patients (whole-brain VBM).</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">kE</th><th colspan="5" valign="top">Peak</th><th colspan="3" valign="top">Center of mass</th><th rowspan="2" valign="top">Structure</th></tr><tr><th valign="top">p-FWE</th><th valign="top">T</th><th valign="top">X</th><th valign="top">Y</th><th valign="top">Z</th><th valign="top">X</th><th valign="top">Y</th><th valign="top">Z</th></tr></thead><tbody><tr><td>2574</td><td>&lt;0.0005</td><td>7.53</td><td>28</td><td>&#8722;17</td><td>&#8722;20</td><td valign="bottom">28</td><td>&#8722;16</td><td>&#8722;18</td><td valign="bottom">R HPC</td></tr><tr><td>910</td><td>&lt;0.0005</td><td>6.81</td><td>&#8722;29</td><td>&#8722;12</td><td>&#8722;18</td><td valign="bottom">&#8722;27</td><td>&#8722;15</td><td>&#8722;19</td><td valign="bottom">L HPC</td></tr><tr><td>113</td><td>0.002</td><td>6.18</td><td>19</td><td>&#8722;28</td><td>5</td><td valign="bottom">18</td><td>&#8722;26</td><td>6</td><td valign="bottom">R lateral thalamus</td></tr><tr><td>414</td><td>0.006</td><td>5.85</td><td>-1</td><td>&#8722;16</td><td>0</td><td valign="bottom">3</td><td>&#8722;12</td><td>7</td><td valign="bottom">anterior/mediodorsal thalamus</td></tr></tbody></table><table-wrap-foot><fn><p>Contrast: controls&#160;&gt;&#160;patients; covariates: age, sex, scan source (MAP, OPTIMA), and TIV. Clusters are FWE-corrected at peak-voxel level (p&lt;0.05) over an individual voxel threshold of p&lt;0.001 (unc.); voxel size: 1 mm<sup>3</sup> isotropic; spatial smoothing kernel: 4 mm FWHM; FWHM: Full-width at half-maximum; HPC: hippocampus; kE: cluster size (number of voxels; minimum cluster size: 50 voxels); L: Left hemisphere; mm: millimeter; MAP: Memory and Amnesia Project; OPTIMA: Oxford Project To Investigate Memory and Aging; R: Right hemisphere;&#160;TIV: total intracranial volume; x, y, z: coordinates in mm.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46202-v2.xml">elife-46202-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><object-id pub-id-type="doi">10.7554/eLife.46202.026</object-id><label>Table 4.</label><caption><title>Age-specific mortality rates and etiologic fractions of microbiologic etiologies and other syndromes of pneumonia and diarrhea in 2015 among Indian children aged 1&#8211;59 months.</title><p><supplementary-material id="table4sdata1"><object-id pub-id-type="doi">10.7554/eLife.46202.027</object-id><label>Table 4&#8212;source data 1.</label><caption><title>Regional meta-analysis of etiologic fractions for respiratory syncytial virus.</title><p>Studies reporting laboratory testing data were identified through a literature search of Ovid MEDLINE, Scopus, and Google Scholar. All studies described hospitalized children or children seeking care in emergency departments and reported data from 2005 onwards. Studies were meta-analyzed by administrative region using Stata&#8217;s metaprop package and visualized in RStudio. We weighted each study using the denominator of total number of laboratory tests in the respective study.</p></caption><media mime-subtype="pdf" mimetype="application" xlink:href="elife-46202-table4-data1-v2.pdf"/></supplementary-material></p><p><supplementary-material id="table4sdata2"><object-id pub-id-type="doi">10.7554/eLife.46202.028</object-id><label>Table 4&#8212;source data 2.</label><caption><title>Regional meta-analysis of etiologic fractions for influenza.</title><p>Studies reporting laboratory testing data were identified through a literature search of Ovid MEDLINE, Scopus, and Google Scholar. All studies described hospitalized children or children seeking care in emergency departments and reported data from 2010 onwards. All studies included describe pandemic and seasonal influenza A and influenza B. Studies were meta-analyzed by administrative region using Stata&#8217;s metaprop package and visualized in RStudio. We weighted each study using the denominator of total number of laboratory tests in the respective study.</p></caption><media mime-subtype="pdf" mimetype="application" xlink:href="elife-46202-table4-data2-v2.pdf"/></supplementary-material></p><p><supplementary-material id="table4sdata3"><object-id pub-id-type="doi">10.7554/eLife.46202.029</object-id><label>Table 4&#8212;source data 3.</label><caption><title>Regional meta-analysis of etiologic fractions for rotavirus.</title><p>Studies reporting laboratory testing data were identified through a literature search of Ovid MEDLINE, Scopus, and Google Scholar. All studies described hospitalized children or children seeking care in emergency departments and reported data from 2010 onwards. Studies were meta-analyzed by administrative region using Stata&#8217;s metaprop package and visualized in RStudio. We weighted each study using the denominator of total number of laboratory tests in the respective study.</p></caption><media mime-subtype="pdf" mimetype="application" xlink:href="elife-46202-table4-data3-v2.pdf"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Region</th><th align="center" rowspan="2">Pneumonia mortality rate (per 1000 live births)<sup>1</sup></th><th align="center" colspan="4">Etiologic fraction (%), (uncertainty range)<sup>2</sup></th><th align="center" rowspan="2">Diarrhea mortality rate (per 1000 live births)<sup>1</sup></th><th align="center" colspan="2">Etiologic fraction (%), (uncertainty range)<sup>2</sup></th></tr><tr><th align="center">SPn<sup>3</sup></th><th align="center">RSV<sup>4</sup></th><th align="center">Influenza<sup>4</sup></th><th align="center">Hib<sup>3</sup></th><th align="center">Rotavirus<sup>4</sup></th><th align="center">Diarrhea with fever and bloody stool<sup>3</sup></th></tr></thead><tbody><tr><td>Northeast</td><td align="center">9.50</td><td align="center">40 (28&#8211;41)</td><td align="center">11 (8&#8211;15)</td><td align="center">6 (&lt;1&#8211;17)<sup>4</sup></td><td align="center">7 (5&#8211;9)</td><td align="center">6.96</td><td align="center">41 (33&#8211;49)</td><td align="center" colspan="2">12</td></tr><tr><td>Central</td><td align="center">5.36</td><td align="center">38 (27&#8211;40)</td><td align="center">22 (15&#8211;30)</td><td align="center">16 (11&#8211;21)</td><td align="center">9 (6&#8211;12)</td><td align="center">3.80</td><td align="center">22 (16&#8211;27)</td><td align="center" colspan="2">9</td></tr><tr><td>East</td><td align="center">4.40</td><td align="center">41 (29&#8211;43)</td><td align="center">12 (5&#8211;20)</td><td align="center">6 (&lt;1&#8211;17)</td><td align="center">4 (3&#8211;6)</td><td align="center">4.38</td><td align="center">34 (19&#8211;52)</td><td align="center" colspan="2">5</td></tr><tr><td>North</td><td align="center">3.01</td><td align="center">42 (30&#8211;44)</td><td align="center">25 (20&#8211;30)</td><td align="center">9 (3&#8211;18)</td><td align="center">3 (2&#8211;4)</td><td align="center">2.57</td><td align="center">46 (37&#8211;56)</td><td align="center" colspan="2">8</td></tr><tr><td>West</td><td align="center">1.77</td><td align="center">40 (29&#8211;43)</td><td align="center">18 (16&#8211;21)</td><td align="center">9 (6&#8211;14)</td><td align="center">6 (4&#8211;8)</td><td align="center">1.16</td><td align="center">33 (25&#8211;41)</td><td align="center" colspan="2">7</td></tr><tr><td>South</td><td align="center">1.63</td><td align="center">41 (29&#8211;43)</td><td align="center">35 (23&#8211;47)</td><td align="center">13 (10&#8211;17)</td><td align="center">5 (3&#8211;6)</td><td align="center">0.82</td><td align="center">35 (30&#8211;41)</td><td align="center" colspan="2">4</td></tr><tr><td>India</td><td align="center">4.17</td><td align="center">38 (27&#8211;39)</td><td align="center">19 (13&#8211;27)</td><td align="center">12 (7&#8211;19)</td><td align="center">7 (5&#8211;9)</td><td align="center">3.19</td><td align="center">30 (21&#8211;40)</td><td align="center" colspan="2">8</td></tr><tr><td colspan="2">EF Ratio (Highest: Lowest)</td><td align="center">1.11</td><td align="center">3.18</td><td align="center">2.67</td><td align="center">3.00</td><td align="center"/><td align="center">2.09</td><td align="center" colspan="2">3.00</td></tr><tr><td colspan="2">Pearson (vs. pneumonia)</td><td align="center">&#8722;0.36</td><td align="center">&#8722;0.66</td><td align="center">0.20</td><td align="center">0.46</td><td align="left">Pearson (vs. diarrhea)</td><td align="center">0.12</td><td align="center" colspan="2">0.73</td></tr></tbody></table><table-wrap-foot><fn><p><sup>1</sup>1&#8211;59-month mortality rates were derived using estimates from <xref ref-type="bibr" rid="bib14">Fadel et al. (2017)</xref>. <sup>2</sup>Etiologic fractions between pathogens are not additive and do not have a sum of 100. <sup>3</sup>Etiologic fractions of <italic>Streptococcus pneumoniae</italic> (SPn), <italic>Haemophilus influenzae</italic> type b (Hib), and diarrhea with fever and bloody were calculated by dividing the estimated number of deaths by pathogen and administrative region by the total number of pneumonia or diarrhea deaths by administrative region. <sup>4</sup>Etiologic fractions for respiratory syncytial virus (RSV), influenza, and rotavirus were determined with a meta-analysis of laboratory testing data by region. <sup>5</sup>Given that no studies were identified describing influenza positivity in the Northeast region, the etiologic fraction for the East region (closest geographically) is substituted instead.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46205-v1.xml">elife-46205-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app1table25" position="float"><object-id pub-id-type="doi">10.7554/eLife.46205.094</object-id><label>Appendix&#160;1&#8212;table 25.</label><caption><title>Quebit fluorometer test results on Dinosaur Provincial Park samples.</title><p>Sample ID&#8217;s as in <xref ref-type="table" rid="app1table2">Appendix 1&#8212;table 2</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Bag <break/>number</th><th rowspan="2">Type</th><th rowspan="2">Replicate</th><th rowspan="2">Sample <break/>ID</th><th rowspan="2">Kit</th><th colspan="2" valign="middle">Ng of DNA / &#956;L</th><th rowspan="2" valign="middle">Concentration</th><th colspan="2">Concentrated ng <break/>of DNA / &#956;L</th></tr><tr><th>First read</th><th>Second read</th><th>First read</th><th>Second read</th></tr></thead><tbody><tr><td>NA</td><td>Blank</td><td>NA</td><td>Blank</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>10</td><td>Topsoil</td><td>1</td><td>10</td><td>Power <break/>Viral</td><td>0.151</td><td>0.133</td><td/><td/><td/></tr><tr><td>13</td><td>Mudstone</td><td>1</td><td>13</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>16</td><td>Scrappings</td><td>1</td><td>16S</td><td>Power <break/>Viral</td><td>0.172</td><td>0.164</td><td/><td/><td/></tr><tr><td>16</td><td>Bone</td><td>1</td><td>16B1</td><td>Power <break/>Viral</td><td>0.424</td><td>0.404</td><td/><td/><td/></tr><tr><td>16</td><td>Bone</td><td>2</td><td>16B2</td><td>Power <break/>Viral</td><td>0.592</td><td>0.55</td><td/><td/><td/></tr><tr><td>NA</td><td>Blank</td><td>NA</td><td>Blank</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>1</td><td>Float</td><td>1</td><td>1F1</td><td>Power <break/>Viral</td><td>0.0926</td><td>0.0908</td><td/><td/><td/></tr><tr><td>1</td><td>Scrappings</td><td>1</td><td>1S1</td><td>Power <break/>Viral</td><td>0.128</td><td>0.127</td><td/><td/><td/></tr><tr><td>1</td><td>Scrappings</td><td>2</td><td>1S2</td><td>Power <break/>Viral</td><td>0.0238</td><td>0.0236</td><td/><td/><td/></tr><tr><td>1</td><td>Bone</td><td>1</td><td>1B1</td><td>Power <break/>Viral</td><td>0.0382</td><td>0.0376</td><td/><td/><td/></tr><tr><td>1</td><td>Bone</td><td>2</td><td>1B2</td><td>Power <break/>Viral</td><td>0.0544</td><td>0.0546</td><td/><td/><td/></tr><tr><td>1</td><td>Mudstone</td><td>1</td><td>1M1</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>1</td><td>Mudstone</td><td>2</td><td>1M2</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>NA</td><td>Blank</td><td>NA</td><td>Blank</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td>x 25</td><td>Below <break/>detection</td><td>Below <break/>detection</td></tr><tr><td>1</td><td>Bone</td><td>3</td><td>1B5g</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.798</td><td>0.788</td><td>x 25</td><td>11.1</td><td>10.5</td></tr><tr><td>1</td><td>Mudstone</td><td>3</td><td>1M10g</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.0334</td><td>0.0322</td><td>x 25</td><td>0.626</td><td>0.612</td></tr><tr><td>1</td><td>Mudstone</td><td>4</td><td>1M2</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.0624</td><td>0.0596</td><td>x 25</td><td>0.586</td><td>0.586</td></tr><tr><td>8</td><td>Mudstone</td><td>1</td><td>8M1</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.096</td><td>0.0924</td><td>x 25</td><td>1.64</td><td>1.58</td></tr><tr><td>8</td><td>Mudstone</td><td>2</td><td>8M2</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.144</td><td>0.141</td><td>x 25</td><td>1.6</td><td>1.55</td></tr><tr><td>11</td><td>Mudstone</td><td>1</td><td>11M1</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td>x 25</td><td>0.0306</td><td>0.0292</td></tr><tr><td>11</td><td>Mudstone</td><td>2</td><td>11M2</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td>x 25</td><td>0.0218</td><td>0.021</td></tr><tr><td>13</td><td>Mudstone</td><td>2</td><td>13M1</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td>x 25</td><td>0.123</td><td>0.119</td></tr><tr><td>13</td><td>Mudstone</td><td>3</td><td>13M2</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.0166</td><td>0.016</td><td>x 25</td><td>0.141</td><td>0.138</td></tr><tr><td>2</td><td>Scrappings</td><td>1</td><td>2S1</td><td>Power <break/>Viral</td><td>0.167</td><td>0.168</td><td/><td/><td/></tr><tr><td>2</td><td>Scrappings</td><td>2</td><td>2S2</td><td>Power <break/>Viral</td><td>0.113</td><td>0.109</td><td/><td/><td/></tr><tr><td>2</td><td>Bone</td><td>1</td><td>2B1</td><td>Power <break/>Viral</td><td>0.134</td><td>0.131</td><td/><td/><td/></tr><tr><td>2</td><td>Bone</td><td>2</td><td>2B2</td><td>Power <break/>Viral</td><td>0.118</td><td>0.116</td><td/><td/><td/></tr><tr><td>6</td><td>Scrappings</td><td>1</td><td>6S1</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>6</td><td>Scrappings</td><td>2</td><td>6S2</td><td>Power <break/>Viral</td><td>0.0114</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>6</td><td>Bone</td><td>1</td><td>6B1</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>6</td><td>Bone</td><td>2</td><td>6B2</td><td>Power <break/>Viral</td><td>0.0102</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>1</td><td>Float</td><td>2</td><td>1F2</td><td>Power <break/>Viral</td><td>0.029</td><td>0.0278</td><td/><td/><td/></tr><tr><td>10</td><td>Topsoil</td><td>2</td><td>10T2</td><td>Power <break/>Viral</td><td>1.04</td><td>1.02</td><td/><td/><td/></tr><tr><td>6</td><td>Scrappings</td><td>3</td><td>6S3</td><td>Power <break/>Viral</td><td/><td/><td>x 2</td><td>0.208</td><td>0.206</td></tr><tr><td>6</td><td>Bone</td><td>3</td><td>6B3</td><td>Power <break/>Viral</td><td/><td/><td>x 2</td><td>0.0422</td><td>0.0424</td></tr><tr><td>1</td><td>Bone <break/>(EDTA <break/>demineralised)</td><td>1</td><td>1BEDTA</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.148</td><td>0.145</td><td>x 20</td><td>3.52</td><td>3.44</td></tr><tr><td>6</td><td>Bone <break/>(EDTA <break/>demineralised)</td><td>1</td><td>6BEDTA</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.0144</td><td>0.013</td><td>x 20</td><td>0.324</td><td>0.318</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46205-v2.xml">elife-46205-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app1table25" position="float"><label>Appendix 1&#8212;table 25.</label><caption><title>Quebit fluorometer test results on Dinosaur Provincial Park samples.</title><p>Sample ID&#8217;s as in <xref ref-type="table" rid="app1table2">Appendix 1&#8212;table 2</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Bag <break/>number</th><th rowspan="2">Type</th><th rowspan="2">Replicate</th><th rowspan="2">Sample <break/>ID</th><th rowspan="2">Kit</th><th colspan="2" valign="middle">Ng of DNA / &#956;L</th><th rowspan="2" valign="middle">Concentration</th><th colspan="2">Concentrated ng <break/>of DNA / &#956;L</th></tr><tr><th>First read</th><th>Second read</th><th>First read</th><th>Second read</th></tr></thead><tbody><tr><td>NA</td><td>Blank</td><td>NA</td><td>Blank</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>10</td><td>Topsoil</td><td>1</td><td>10</td><td>Power <break/>Viral</td><td>0.151</td><td>0.133</td><td/><td/><td/></tr><tr><td>13</td><td>Mudstone</td><td>1</td><td>13</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>16</td><td>Scrappings</td><td>1</td><td>16S</td><td>Power <break/>Viral</td><td>0.172</td><td>0.164</td><td/><td/><td/></tr><tr><td>16</td><td>Bone</td><td>1</td><td>16B1</td><td>Power <break/>Viral</td><td>0.424</td><td>0.404</td><td/><td/><td/></tr><tr><td>16</td><td>Bone</td><td>2</td><td>16B2</td><td>Power <break/>Viral</td><td>0.592</td><td>0.55</td><td/><td/><td/></tr><tr><td>NA</td><td>Blank</td><td>NA</td><td>Blank</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>1</td><td>'Float' bone in matrix</td><td>1</td><td>1F1</td><td>Power <break/>Viral</td><td>0.0926</td><td>0.0908</td><td/><td/><td/></tr><tr><td>1</td><td>Scrappings</td><td>1</td><td>1S1</td><td>Power <break/>Viral</td><td>0.128</td><td>0.127</td><td/><td/><td/></tr><tr><td>1</td><td>Scrappings</td><td>2</td><td>1S2</td><td>Power <break/>Viral</td><td>0.0238</td><td>0.0236</td><td/><td/><td/></tr><tr><td>1</td><td>Bone</td><td>1</td><td>1B1</td><td>Power <break/>Viral</td><td>0.0382</td><td>0.0376</td><td/><td/><td/></tr><tr><td>1</td><td>Bone</td><td>2</td><td>1B2</td><td>Power <break/>Viral</td><td>0.0544</td><td>0.0546</td><td/><td/><td/></tr><tr><td>1</td><td>Mudstone</td><td>1</td><td>1M1</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>1</td><td>Mudstone</td><td>2</td><td>1M2</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>NA</td><td>Blank</td><td>NA</td><td>Blank</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td>x 25</td><td>Below <break/>detection</td><td>Below <break/>detection</td></tr><tr><td>1</td><td>Bone</td><td>3</td><td>1B5g</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.798</td><td>0.788</td><td>x 25</td><td>11.1</td><td>10.5</td></tr><tr><td>1</td><td>Mudstone</td><td>3</td><td>1M10g1</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.0334</td><td>0.0322</td><td>x 25</td><td>0.626</td><td>0.612</td></tr><tr><td>1</td><td>Mudstone</td><td>4</td><td>1M10g2</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.0624</td><td>0.0596</td><td>x 25</td><td>0.586</td><td>0.586</td></tr><tr><td>8</td><td>Mudstone</td><td>1</td><td>8M1</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.096</td><td>0.0924</td><td>x 25</td><td>1.64</td><td>1.58</td></tr><tr><td>8</td><td>Mudstone</td><td>2</td><td>8M2</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.144</td><td>0.141</td><td>x 25</td><td>1.6</td><td>1.55</td></tr><tr><td>11</td><td>Mudstone</td><td>1</td><td>11M1</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td>x 25</td><td>0.0306</td><td>0.0292</td></tr><tr><td>11</td><td>Mudstone</td><td>2</td><td>11M2</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td>x 25</td><td>0.0218</td><td>0.021</td></tr><tr><td>13</td><td>Mudstone</td><td>2</td><td>13M1</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td>x 25</td><td>0.123</td><td>0.119</td></tr><tr><td>13</td><td>Mudstone</td><td>3</td><td>13M2</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.0166</td><td>0.016</td><td>x 25</td><td>0.141</td><td>0.138</td></tr><tr><td>2</td><td>Scrappings</td><td>1</td><td>2S1</td><td>Power <break/>Viral</td><td>0.167</td><td>0.168</td><td/><td/><td/></tr><tr><td>2</td><td>Scrappings</td><td>2</td><td>2S2</td><td>Power <break/>Viral</td><td>0.113</td><td>0.109</td><td/><td/><td/></tr><tr><td>2</td><td>Bone</td><td>1</td><td>2B1</td><td>Power <break/>Viral</td><td>0.134</td><td>0.131</td><td/><td/><td/></tr><tr><td>2</td><td>Bone</td><td>2</td><td>2B2</td><td>Power <break/>Viral</td><td>0.118</td><td>0.116</td><td/><td/><td/></tr><tr><td>6</td><td>Scrappings</td><td>1</td><td>6S1</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>6</td><td>Scrappings</td><td>2</td><td>6S2</td><td>Power <break/>Viral</td><td>0.0114</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>6</td><td>Bone</td><td>1</td><td>6B1</td><td>Power <break/>Viral</td><td>Below <break/>detection</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>6</td><td>Bone</td><td>2</td><td>6B2</td><td>Power <break/>Viral</td><td>0.0102</td><td>Below <break/>detection</td><td/><td/><td/></tr><tr><td>1</td><td>Float</td><td>2</td><td>1F2</td><td>Power <break/>Viral</td><td>0.029</td><td>0.0278</td><td/><td/><td/></tr><tr><td>10</td><td>Topsoil</td><td>2</td><td>10T2</td><td>Power <break/>Viral</td><td>1.04</td><td>1.02</td><td/><td/><td/></tr><tr><td>6</td><td>Scrappings</td><td>3</td><td>6S3</td><td>Power <break/>Viral</td><td/><td/><td>x 2</td><td>0.208</td><td>0.206</td></tr><tr><td>6</td><td>Bone</td><td>3</td><td>6B3</td><td>Power <break/>Viral</td><td/><td/><td>x 2</td><td>0.0422</td><td>0.0424</td></tr><tr><td>1</td><td>Bone <break/>(EDTA <break/>demineralised)</td><td>1</td><td>1BEDTA</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.148</td><td>0.145</td><td>x 20</td><td>3.52</td><td>3.44</td></tr><tr><td>6</td><td>Bone <break/>(EDTA <break/>demineralised)</td><td>1</td><td>6BEDTA</td><td>Dneasy <break/>PowerMax <break/>Soil</td><td>0.0144</td><td>0.013</td><td>x 20</td><td>0.324</td><td>0.318</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46380-v2.xml">elife-46380-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.46380.004</object-id><label>Table 1.</label><caption><title>List of PKACa substrates.</title><p>Proteins are listed according to the log of fold changes of AS-PKAC&#945; to WT- PKAC&#945;. Two independent experiments have been done to prepare the PKAC&#945; substrates samples for mass spectrometric analysis.</p><p><supplementary-material id="table1sdata1"><object-id pub-id-type="doi">10.7554/eLife.46380.005</object-id><label>Table 1&#8212;&#65279;source data 1.</label><caption><title>The full list of proteins identified in both experiments is provided.</title></caption><media mime-subtype="docx" mimetype="application" xlink:href="elife-46380-fig1-v2.docx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Uniprot</th><th rowspan="2" valign="bottom">Protein.names</th><th rowspan="2" valign="bottom">Gene.names</th><th rowspan="2" valign="bottom">Peptides</th><th colspan="2" valign="bottom">Log ratio AS/WT</th></tr><tr><th valign="bottom">experiment1</th><th valign="bottom">experiment2</th></tr></thead><tbody><tr><td valign="bottom">Q6AI12</td><td valign="bottom">Ankyrin repeat domain-containing protein 40</td><td valign="bottom">ANKRD40</td><td valign="bottom">9</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">Q6P6C2</td><td valign="bottom">RNA demethylase ALKBH5</td><td valign="bottom">ALKBH5</td><td valign="bottom">6</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">Q9NRY4</td><td valign="bottom">Rho GTPase-activating protein 35</td><td valign="bottom">ARHGAP35</td><td valign="bottom">9</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">E7EVC7</td><td valign="bottom">Autophagy-related protein 16&#8211;1</td><td valign="bottom">ATG16L1</td><td valign="bottom">8</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">J3KPC8</td><td valign="bottom">Serine/threonine-protein kinase SIK3</td><td valign="bottom">SIK3;KIAA0999</td><td valign="bottom">5</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">A1 &#215; 283</td><td valign="bottom">SH3 and PX domain-containing protein 2B</td><td valign="bottom">SH3PXD2B</td><td valign="bottom">4</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">Q8IWZ8</td><td valign="bottom">SURP and G-patch domain-containing protein 1</td><td valign="bottom">SUGP1</td><td valign="bottom">5</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">Q9UJX5</td><td valign="bottom">Anaphase-promoting complex subunit 4</td><td valign="bottom">ANAPC4</td><td valign="bottom">5</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">O43719</td><td valign="bottom">HIV Tat-specific factor 1</td><td valign="bottom">HTATSF1</td><td valign="bottom">4</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">O95644-5</td><td valign="bottom">Nuclear factor of activated T-cells, cytoplasmic 1</td><td valign="bottom">NFATC1</td><td valign="bottom">5</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">G8JLI6</td><td valign="bottom">Prolyl 3-hydroxylase 3</td><td valign="bottom">LEPREL2</td><td valign="bottom">3</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">F8W781</td><td valign="bottom">Zinc finger CCCH domain-containing protein 13</td><td valign="bottom">ZC3H13</td><td valign="bottom">3</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">Q9BZL4</td><td valign="bottom">Protein phosphatase 1 regulatory subunit 12C</td><td valign="bottom">PPP1R12C</td><td valign="bottom">21</td><td valign="bottom">6,04440274</td><td valign="bottom">7,30701515</td></tr><tr><td valign="bottom">O14974</td><td valign="bottom">Protein phosphatase 1 regulatory subunit 12A</td><td valign="bottom">PPP1R12A</td><td valign="bottom">26</td><td valign="bottom">5,72796034</td><td valign="bottom">7,12654716</td></tr><tr><td valign="bottom">Q00537</td><td valign="bottom">Cyclin-dependent kinase 17</td><td valign="bottom">CDK17</td><td valign="bottom">31</td><td valign="bottom">6,37867381</td><td valign="bottom">6,39216838</td></tr><tr><td valign="bottom">Q9Y4G8</td><td valign="bottom">Rap guanine nucleotide exchange factor 2</td><td valign="bottom">RAPGEF2</td><td valign="bottom">21</td><td valign="bottom">10</td><td valign="bottom">6,17455504</td></tr><tr><td valign="bottom">Q9BYB0</td><td valign="bottom">SH3 and multiple ankyrin repeat domains protein 3</td><td valign="bottom">SHANK3</td><td valign="bottom">32</td><td valign="bottom">5,26591421</td><td valign="bottom">5,6389181</td></tr><tr><td valign="bottom">J3KSW8</td><td valign="bottom">Myosin phosphatase Rho-interacting protein</td><td valign="bottom">MPRIP</td><td valign="bottom">18</td><td valign="bottom">4,61398477</td><td valign="bottom">5,61155414</td></tr><tr><td valign="bottom">P31323</td><td valign="bottom">cAMP-dependent protein kinase type II-beta regulatory subunit</td><td valign="bottom">PRKAR2B</td><td valign="bottom">19</td><td valign="bottom">7,05077105</td><td valign="bottom">5,33509437</td></tr><tr><td valign="bottom">P13861</td><td valign="bottom">cAMP-dependent protein kinase type II-alpha regulatory subunit</td><td valign="bottom">PRKAR2A</td><td valign="bottom">24</td><td valign="bottom">5,42841998</td><td valign="bottom">5,04010629</td></tr><tr><td valign="bottom">Q14980-2</td><td valign="bottom">Nuclear mitotic apparatus protein 1</td><td valign="bottom">NUMA1</td><td valign="bottom">61</td><td valign="bottom">3,45625969</td><td valign="bottom">4,47466712</td></tr><tr><td valign="bottom">O15056</td><td valign="bottom">Synaptojanin-2</td><td valign="bottom">SYNJ2</td><td valign="bottom">13</td><td valign="bottom">4,64022655</td><td valign="bottom">4,46069701</td></tr><tr><td valign="bottom">J3KNX9</td><td valign="bottom">Unconventional myosin-XVIIIa</td><td valign="bottom">MYO18A</td><td valign="bottom">10</td><td valign="bottom">10</td><td valign="bottom">4,43208178</td></tr><tr><td valign="bottom">Q86UU1-2</td><td valign="bottom">Pleckstrin homology-like domain family B member 1</td><td valign="bottom">PHLDB1</td><td valign="bottom">19</td><td valign="bottom">5,4105243</td><td valign="bottom">4,10782285</td></tr><tr><td valign="bottom">P28715</td><td valign="bottom">DNA repair protein complementing XP-G cells</td><td valign="bottom">ERCC5;BIVM-ERCC5</td><td valign="bottom">8</td><td valign="bottom">3,3571826</td><td valign="bottom">4,09305592</td></tr><tr><td valign="bottom">P12270</td><td valign="bottom">Nucleoprotein TPR</td><td valign="bottom">TPR</td><td valign="bottom">104</td><td valign="bottom">3,3333472</td><td valign="bottom">4,04477536</td></tr><tr><td valign="bottom">Q15111</td><td valign="bottom">Inactive phospholipase C-like protein 1;Phosphoinositide phospholipase C</td><td valign="bottom">PLCL1</td><td valign="bottom">10</td><td valign="bottom">10</td><td valign="bottom">3,32188704</td></tr><tr><td valign="bottom">Q9HD67</td><td valign="bottom">Unconventional myosin-X</td><td valign="bottom">MYO10</td><td valign="bottom">39</td><td valign="bottom">4,13973415</td><td valign="bottom">3,21827463</td></tr><tr><td valign="bottom">Q14185</td><td valign="bottom">Dedicator of cytokinesis protein 1</td><td valign="bottom">DOCK1</td><td valign="bottom">34</td><td valign="bottom">4,50413426</td><td valign="bottom">3,18515106</td></tr><tr><td valign="bottom">O75116</td><td valign="bottom">Rho-associated protein kinase 2</td><td valign="bottom">ROCK2</td><td valign="bottom">29</td><td valign="bottom">3,27701864</td><td valign="bottom">3,08277835</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46380-v3.xml">elife-46380-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>List of PKACa substrates.</title><p>Proteins are listed according to the log of fold changes of AS-PKAC&#945; to WT- PKAC&#945;. Two independent experiments have been done to prepare the PKAC&#945; substrates samples for mass spectrometric analysis.</p><p><supplementary-material id="table1sdata1"><label>Table 1&#8212;source data 1.</label><caption><title>The full list of proteins identified in both experiments is provided.</title></caption><media mime-subtype="docx" mimetype="application" xlink:href="elife-46380-table1-data1-v3.docx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Uniprot</th><th rowspan="2" valign="bottom">Protein.names</th><th rowspan="2" valign="bottom">Gene.names</th><th rowspan="2" valign="bottom">Peptides</th><th colspan="2" valign="bottom">Log ratio AS/WT</th></tr><tr><th valign="bottom">experiment1</th><th valign="bottom">experiment2</th></tr></thead><tbody><tr><td valign="bottom">Q6AI12</td><td valign="bottom">Ankyrin repeat domain-containing protein 40</td><td valign="bottom">ANKRD40</td><td valign="bottom">9</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">Q6P6C2</td><td valign="bottom">RNA demethylase ALKBH5</td><td valign="bottom">ALKBH5</td><td valign="bottom">6</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">Q9NRY4</td><td valign="bottom">Rho GTPase-activating protein 35</td><td valign="bottom">ARHGAP35</td><td valign="bottom">9</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">E7EVC7</td><td valign="bottom">Autophagy-related protein 16&#8211;1</td><td valign="bottom">ATG16L1</td><td valign="bottom">8</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">J3KPC8</td><td valign="bottom">Serine/threonine-protein kinase SIK3</td><td valign="bottom">SIK3;KIAA0999</td><td valign="bottom">5</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">A1 &#215; 283</td><td valign="bottom">SH3 and PX domain-containing protein 2B</td><td valign="bottom">SH3PXD2B</td><td valign="bottom">4</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">Q8IWZ8</td><td valign="bottom">SURP and G-patch domain-containing protein 1</td><td valign="bottom">SUGP1</td><td valign="bottom">5</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">Q9UJX5</td><td valign="bottom">Anaphase-promoting complex subunit 4</td><td valign="bottom">ANAPC4</td><td valign="bottom">5</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">O43719</td><td valign="bottom">HIV Tat-specific factor 1</td><td valign="bottom">HTATSF1</td><td valign="bottom">4</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">O95644-5</td><td valign="bottom">Nuclear factor of activated T-cells, cytoplasmic 1</td><td valign="bottom">NFATC1</td><td valign="bottom">5</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">G8JLI6</td><td valign="bottom">Prolyl 3-hydroxylase 3</td><td valign="bottom">LEPREL2</td><td valign="bottom">3</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">F8W781</td><td valign="bottom">Zinc finger CCCH domain-containing protein 13</td><td valign="bottom">ZC3H13</td><td valign="bottom">3</td><td valign="bottom">10</td><td valign="bottom">10</td></tr><tr><td valign="bottom">Q9BZL4</td><td valign="bottom">Protein phosphatase 1 regulatory subunit 12C</td><td valign="bottom">PPP1R12C</td><td valign="bottom">21</td><td valign="bottom">6,04440274</td><td valign="bottom">7,30701515</td></tr><tr><td valign="bottom">O14974</td><td valign="bottom">Protein phosphatase 1 regulatory subunit 12A</td><td valign="bottom">PPP1R12A</td><td valign="bottom">26</td><td valign="bottom">5,72796034</td><td valign="bottom">7,12654716</td></tr><tr><td valign="bottom">Q00537</td><td valign="bottom">Cyclin-dependent kinase 17</td><td valign="bottom">CDK17</td><td valign="bottom">31</td><td valign="bottom">6,37867381</td><td valign="bottom">6,39216838</td></tr><tr><td valign="bottom">Q9Y4G8</td><td valign="bottom">Rap guanine nucleotide exchange factor 2</td><td valign="bottom">RAPGEF2</td><td valign="bottom">21</td><td valign="bottom">10</td><td valign="bottom">6,17455504</td></tr><tr><td valign="bottom">Q9BYB0</td><td valign="bottom">SH3 and multiple ankyrin repeat domains protein 3</td><td valign="bottom">SHANK3</td><td valign="bottom">32</td><td valign="bottom">5,26591421</td><td valign="bottom">5,6389181</td></tr><tr><td valign="bottom">J3KSW8</td><td valign="bottom">Myosin phosphatase Rho-interacting protein</td><td valign="bottom">MPRIP</td><td valign="bottom">18</td><td valign="bottom">4,61398477</td><td valign="bottom">5,61155414</td></tr><tr><td valign="bottom">P31323</td><td valign="bottom">cAMP-dependent protein kinase type II-beta regulatory subunit</td><td valign="bottom">PRKAR2B</td><td valign="bottom">19</td><td valign="bottom">7,05077105</td><td valign="bottom">5,33509437</td></tr><tr><td valign="bottom">P13861</td><td valign="bottom">cAMP-dependent protein kinase type II-alpha regulatory subunit</td><td valign="bottom">PRKAR2A</td><td valign="bottom">24</td><td valign="bottom">5,42841998</td><td valign="bottom">5,04010629</td></tr><tr><td valign="bottom">Q14980-2</td><td valign="bottom">Nuclear mitotic apparatus protein 1</td><td valign="bottom">NUMA1</td><td valign="bottom">61</td><td valign="bottom">3,45625969</td><td valign="bottom">4,47466712</td></tr><tr><td valign="bottom">O15056</td><td valign="bottom">Synaptojanin-2</td><td valign="bottom">SYNJ2</td><td valign="bottom">13</td><td valign="bottom">4,64022655</td><td valign="bottom">4,46069701</td></tr><tr><td valign="bottom">J3KNX9</td><td valign="bottom">Unconventional myosin-XVIIIa</td><td valign="bottom">MYO18A</td><td valign="bottom">10</td><td valign="bottom">10</td><td valign="bottom">4,43208178</td></tr><tr><td valign="bottom">Q86UU1-2</td><td valign="bottom">Pleckstrin homology-like domain family B member 1</td><td valign="bottom">PHLDB1</td><td valign="bottom">19</td><td valign="bottom">5,4105243</td><td valign="bottom">4,10782285</td></tr><tr><td valign="bottom">P28715</td><td valign="bottom">DNA repair protein complementing XP-G cells</td><td valign="bottom">ERCC5;BIVM-ERCC5</td><td valign="bottom">8</td><td valign="bottom">3,3571826</td><td valign="bottom">4,09305592</td></tr><tr><td valign="bottom">P12270</td><td valign="bottom">Nucleoprotein TPR</td><td valign="bottom">TPR</td><td valign="bottom">104</td><td valign="bottom">3,3333472</td><td valign="bottom">4,04477536</td></tr><tr><td valign="bottom">Q15111</td><td valign="bottom">Inactive phospholipase C-like protein 1;Phosphoinositide phospholipase C</td><td valign="bottom">PLCL1</td><td valign="bottom">10</td><td valign="bottom">10</td><td valign="bottom">3,32188704</td></tr><tr><td valign="bottom">Q9HD67</td><td valign="bottom">Unconventional myosin-X</td><td valign="bottom">MYO10</td><td valign="bottom">39</td><td valign="bottom">4,13973415</td><td valign="bottom">3,21827463</td></tr><tr><td valign="bottom">Q14185</td><td valign="bottom">Dedicator of cytokinesis protein 1</td><td valign="bottom">DOCK1</td><td valign="bottom">34</td><td valign="bottom">4,50413426</td><td valign="bottom">3,18515106</td></tr><tr><td valign="bottom">O75116</td><td valign="bottom">Rho-associated protein kinase 2</td><td valign="bottom">ROCK2</td><td valign="bottom">29</td><td valign="bottom">3,27701864</td><td valign="bottom">3,08277835</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46402-v3.xml">elife-46402-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Six pairs of colonisations for whom transmission was reconstructed with a posterior score of at least 0.95 and, for those reconstructed transmissions, a posterior median of at least five tips were in the recipient subgraphs.</title><p>Each row gives the posterior median and the limits of the 95% highest posterior density (HPD) interval for the number of reconstructed transmissions between the subjects, in either direction. The posterior median number of tips in the recipient subgraphs, and the nucleotide diversity amongst those tips, are also given. In five cases the inferred direction of transmission is clear, but for C183 and C194 it is not.</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2" valign="bottom">Colonisation A</th><th align="center" rowspan="2" valign="bottom">Colonisation B</th><th align="center" colspan="3" valign="bottom">Number of A to B transitions&#160;95 % HPD</th><th align="center" rowspan="2" valign="bottom">Tips in descendant B subgraphs (median)</th><th align="center" rowspan="2" valign="bottom">Nucleotide diversity transmitted to B (median)</th><th align="center" colspan="3" valign="bottom">Number of B to A transitions&#160;95% HPD</th><th align="center" rowspan="2" valign="bottom">Tips in descendant A subgraphs (median)</th><th align="center" rowspan="2" valign="bottom">Nucleotide diversity transmitted to A (median)</th><th align="center" rowspan="2" valign="bottom">Direction</th></tr><tr><th align="center" valign="bottom">Median</th><th align="center" valign="bottom">Lower</th><th align="center" valign="bottom">Upper</th><th align="center" valign="bottom">Median</th><th align="left" valign="bottom">Lower</th><th align="left" valign="bottom">Upper</th></tr></thead><tbody><tr><td align="left" valign="bottom">C012</td><td align="left" valign="bottom">C137</td><td align="right" valign="bottom">3</td><td align="right" valign="bottom">2</td><td align="right" valign="bottom">3</td><td align="right" valign="bottom">79</td><td align="right" valign="bottom">1.46E-06</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="left" valign="bottom">NA</td><td align="left" valign="bottom">A to B</td></tr><tr><td align="left" valign="bottom">C012</td><td align="left" valign="bottom">C159b</td><td align="right" valign="bottom">34</td><td align="right" valign="bottom">25</td><td align="right" valign="bottom">38</td><td align="right" valign="bottom">39</td><td align="right" valign="bottom">2.20E-06</td><td align="right" valign="bottom">1</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">6</td><td align="right" valign="bottom">1</td><td align="right" valign="bottom">1.40E-06</td><td align="left" valign="bottom">A to B</td></tr><tr><td align="left" valign="bottom">C126</td><td align="left" valign="bottom">C234</td><td align="right" valign="bottom">3</td><td align="right" valign="bottom">3</td><td align="right" valign="bottom">3</td><td align="right" valign="bottom">10</td><td align="right" valign="bottom">1.92E-06</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="left" valign="bottom">NA</td><td align="left" valign="bottom">A to B</td></tr><tr><td align="left" valign="bottom">C126</td><td align="left" valign="bottom">C271a</td><td align="right" valign="bottom">1</td><td align="right" valign="bottom">1</td><td align="right" valign="bottom">1</td><td align="right" valign="bottom">21</td><td align="right" valign="bottom">3.11E-07</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="left" valign="bottom">NA</td><td align="left" valign="bottom">A to B</td></tr><tr><td align="left" valign="bottom">C126</td><td align="left" valign="bottom">C327a</td><td align="right" valign="bottom">5</td><td align="right" valign="bottom">4</td><td align="right" valign="bottom">5</td><td align="right" valign="bottom">9</td><td align="right" valign="bottom">2.68E-06</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="left" valign="bottom">NA</td><td align="left" valign="bottom">A to B</td></tr><tr><td align="left" valign="bottom">C183</td><td align="left" valign="bottom">C194</td><td align="right" valign="bottom">15</td><td align="right" valign="bottom">1</td><td align="right" valign="bottom">23</td><td align="right" valign="bottom">31</td><td align="right" valign="bottom">3.41E-07</td><td align="right" valign="bottom">5</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">19</td><td align="right" valign="bottom">12</td><td align="right" valign="bottom">7.08E-07</td><td align="left" valign="bottom">Unclear</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46402-v3.xml">elife-46402-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Results of the investigation of body site/phylogeny associations.</title><p>Each row corresponds to a single BATS analysis on a posterior set of phylogenies consisting just of the sequences from that subject. The <italic>p</italic>-values are given for the association index (AI), parsimony score (PS) and largest monophyletic clade (MC) size for all body sites present in the dataset. Values with an asterisk (*) correspond to statistics whose values are identical under both hypotheses due to singleton sequences from some sites.</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="3" valign="middle">Subject</th><th align="center" rowspan="3" valign="middle">Number of Sequences</th><th align="center" colspan="7" valign="bottom">p-value</th></tr><tr><th align="center" rowspan="2" valign="middle">AI</th><th align="center" rowspan="2" valign="middle">PS</th><th align="center" colspan="5" valign="bottom">MC</th></tr><tr><th align="left" valign="bottom">Axilla</th><th align="left" valign="bottom">Nose</th><th align="left" valign="bottom">Throat</th><th align="left" valign="bottom">Trachea</th><th align="left" valign="bottom">Wound</th></tr></thead><tbody><tr><td align="left" valign="bottom">T009</td><td align="right" valign="bottom">31</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.026</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.031</bold></td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T012</td><td align="right" valign="bottom">223</td><td align="right" style="author-callout-style-b1" valign="bottom">0.24</td><td align="right" style="author-callout-style-b2" valign="bottom">0.18</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">0.14</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T065</td><td align="right" valign="bottom">11</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>&lt;0.001</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.29</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T071</td><td align="right" valign="bottom">13</td><td align="right" style="author-callout-style-b1" valign="bottom">0.16</td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.16</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T092</td><td align="right" valign="bottom">12</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.035</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T095</td><td align="right" valign="bottom">11</td><td align="right" style="author-callout-style-b1" valign="bottom">0.099</td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T099</td><td align="right" valign="bottom">19</td><td align="right" style="author-callout-style-b1" valign="bottom">1</td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.16</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1*</td></tr><tr><td align="left" valign="bottom">T126</td><td align="right" valign="bottom">239</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>&lt;0.001</bold></td><td align="right" style="author-callout-style-b2" valign="bottom"><bold>&lt;0.001</bold></td><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.004</bold></td><td align="right" style="author-callout-style-b4" valign="bottom">0.1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.004</bold></td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td></tr><tr><td align="left" valign="bottom">T137</td><td align="right" valign="bottom">79</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.004</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.12</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T159</td><td align="right" valign="bottom">46</td><td align="right" style="author-callout-style-b1" valign="bottom">0.7</td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T183</td><td align="right" valign="bottom">48</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.04</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom">0.22</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T188</td><td align="right" valign="bottom">19</td><td align="right" style="author-callout-style-b1" valign="bottom">1</td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T194</td><td align="right" valign="bottom">32</td><td align="right" style="author-callout-style-b1" valign="bottom">0.13</td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">0.37</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td></tr><tr><td align="left" valign="bottom">T197</td><td align="right" valign="bottom">35</td><td align="right" style="author-callout-style-b1" valign="bottom">0.74</td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td></tr><tr><td align="left" valign="bottom">T232</td><td align="right" valign="bottom">11</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>&lt;0.001</bold></td><td align="right" style="author-callout-style-b2" valign="bottom"><bold>&lt;0.001</bold></td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.001</bold></td><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.001</bold></td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T234</td><td align="right" valign="bottom">10</td><td align="right" style="author-callout-style-b1" valign="bottom">1</td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T249</td><td align="right" valign="bottom">14</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.002</bold></td><td align="right" style="author-callout-style-b2" valign="bottom"><bold>&lt;0.001</bold></td><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.017</bold></td><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.016</bold></td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T271</td><td align="right" valign="bottom">23</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.043</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom">0.49</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td></tr><tr><td align="left" valign="bottom">T327</td><td align="right" valign="bottom">10</td><td align="right" style="author-callout-style-b1" valign="bottom">1</td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.51</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T330</td><td align="right" valign="bottom">8</td><td align="right" style="author-callout-style-b1" valign="bottom">1</td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.26</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T358</td><td align="right" valign="bottom">22</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.004</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.06</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46402-v4.xml">elife-46402-v4.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Six pairs of colonisations for whom transmission was reconstructed with a posterior score of at least 0.95 and, for those reconstructed transmissions, a posterior median of at least five tips were in the recipient subgraphs.</title><p>Each row gives the posterior median and the limits of the 95% highest posterior density (HPD) interval for the number of reconstructed transmissions between the subjects, in either direction. The posterior median number of tips in the recipient subgraphs, and the nucleotide diversity amongst those tips, are also given. In five cases the inferred direction of transmission is clear, but for C183 and C194 it is not.</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="2" valign="bottom">Colonisation A</th><th align="center" rowspan="2" valign="bottom">Colonisation B</th><th align="center" colspan="3" valign="bottom">Number of A to B transitions&#160;95 % HPD</th><th align="center" rowspan="2" valign="bottom">Tips in descendant B subgraphs (median)</th><th align="center" rowspan="2" valign="bottom">Nucleotide diversity transmitted to B (median)</th><th align="center" colspan="3" valign="bottom">Number of B to A transitions&#160;95% HPD</th><th align="center" rowspan="2" valign="bottom">Tips in descendant A subgraphs (median)</th><th align="center" rowspan="2" valign="bottom">Nucleotide diversity transmitted to A (median)</th><th align="center" rowspan="2" valign="bottom">Direction</th></tr><tr><th align="center" valign="bottom">Median</th><th align="center" valign="bottom">Lower</th><th align="center" valign="bottom">Upper</th><th align="center" valign="bottom">Median</th><th align="left" valign="bottom">Lower</th><th align="left" valign="bottom">Upper</th></tr></thead><tbody><tr><td align="left" valign="bottom">C012</td><td align="left" valign="bottom">C137</td><td align="right" valign="bottom">3</td><td align="right" valign="bottom">2</td><td align="right" valign="bottom">3</td><td align="right" valign="bottom">79</td><td align="right" valign="bottom">1.46E-06</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="left" valign="bottom">NA</td><td align="left" valign="bottom">A to B</td></tr><tr><td align="left" valign="bottom">C012</td><td align="left" valign="bottom">C159b</td><td align="right" valign="bottom">34</td><td align="right" valign="bottom">25</td><td align="right" valign="bottom">38</td><td align="right" valign="bottom">39</td><td align="right" valign="bottom">2.20E-06</td><td align="right" valign="bottom">1</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">6</td><td align="right" valign="bottom">1</td><td align="right" valign="bottom">1.40E-06</td><td align="left" valign="bottom">A to B</td></tr><tr><td align="left" valign="bottom">C126</td><td align="left" valign="bottom">C234</td><td align="right" valign="bottom">3</td><td align="right" valign="bottom">3</td><td align="right" valign="bottom">3</td><td align="right" valign="bottom">10</td><td align="right" valign="bottom">1.92E-06</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="left" valign="bottom">NA</td><td align="left" valign="bottom">A to B</td></tr><tr><td align="left" valign="bottom">C126</td><td align="left" valign="bottom">C271a</td><td align="right" valign="bottom">1</td><td align="right" valign="bottom">1</td><td align="right" valign="bottom">1</td><td align="right" valign="bottom">21</td><td align="right" valign="bottom">3.11E-07</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="left" valign="bottom">NA</td><td align="left" valign="bottom">A to B</td></tr><tr><td align="left" valign="bottom">C126</td><td align="left" valign="bottom">C327a</td><td align="right" valign="bottom">5</td><td align="right" valign="bottom">4</td><td align="right" valign="bottom">5</td><td align="right" valign="bottom">9</td><td align="right" valign="bottom">2.68E-06</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">0</td><td align="left" valign="bottom">NA</td><td align="left" valign="bottom">A to B</td></tr><tr><td align="left" valign="bottom">C183</td><td align="left" valign="bottom">C194</td><td align="right" valign="bottom">15</td><td align="right" valign="bottom">1</td><td align="right" valign="bottom">23</td><td align="right" valign="bottom">31</td><td align="right" valign="bottom">3.41E-07</td><td align="right" valign="bottom">5</td><td align="right" valign="bottom">0</td><td align="right" valign="bottom">19</td><td align="right" valign="bottom">12</td><td align="right" valign="bottom">7.08E-07</td><td align="left" valign="bottom">Unclear</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46402-v4.xml">elife-46402-v4.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Results of the investigation of body site/phylogeny associations.</title><p>Each row corresponds to a single BATS analysis on a posterior set of phylogenies consisting just of the sequences from that subject. The <italic>p</italic>-values are given for the association index (AI), parsimony score (PS) and largest monophyletic clade (MC) size for all body sites present in the dataset. Values with an asterisk (*) correspond to statistics whose values are identical under both hypotheses due to singleton sequences from some sites.</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" rowspan="3" valign="middle">Subject</th><th align="center" rowspan="3" valign="middle">Number of Sequences</th><th align="center" colspan="7" valign="bottom">p-value</th></tr><tr><th align="center" rowspan="2" valign="middle">AI</th><th align="center" rowspan="2" valign="middle">PS</th><th align="center" colspan="5" valign="bottom">MC</th></tr><tr><th align="left" valign="bottom">Axilla</th><th align="left" valign="bottom">Nose</th><th align="left" valign="bottom">Throat</th><th align="left" valign="bottom">Trachea</th><th align="left" valign="bottom">Wound</th></tr></thead><tbody><tr><td align="left" valign="bottom">T009</td><td align="right" valign="bottom">31</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.026</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.031</bold></td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T012</td><td align="right" valign="bottom">223</td><td align="right" style="author-callout-style-b1" valign="bottom">0.24</td><td align="right" style="author-callout-style-b2" valign="bottom">0.18</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">0.14</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T065</td><td align="right" valign="bottom">11</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>&lt;0.001</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.29</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T071</td><td align="right" valign="bottom">13</td><td align="right" style="author-callout-style-b1" valign="bottom">0.16</td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.16</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T092</td><td align="right" valign="bottom">12</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.035</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T095</td><td align="right" valign="bottom">11</td><td align="right" style="author-callout-style-b1" valign="bottom">0.099</td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T099</td><td align="right" valign="bottom">19</td><td align="right" style="author-callout-style-b1" valign="bottom">1</td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.16</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1*</td></tr><tr><td align="left" valign="bottom">T126</td><td align="right" valign="bottom">239</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>&lt;0.001</bold></td><td align="right" style="author-callout-style-b2" valign="bottom"><bold>&lt;0.001</bold></td><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.004</bold></td><td align="right" style="author-callout-style-b4" valign="bottom">0.1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.004</bold></td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td></tr><tr><td align="left" valign="bottom">T137</td><td align="right" valign="bottom">79</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.004</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.12</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T159</td><td align="right" valign="bottom">46</td><td align="right" style="author-callout-style-b1" valign="bottom">0.7</td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T183</td><td align="right" valign="bottom">48</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.04</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom">0.22</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T188</td><td align="right" valign="bottom">19</td><td align="right" style="author-callout-style-b1" valign="bottom">1</td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T194</td><td align="right" valign="bottom">32</td><td align="right" style="author-callout-style-b1" valign="bottom">0.13</td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">0.37</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td></tr><tr><td align="left" valign="bottom">T197</td><td align="right" valign="bottom">35</td><td align="right" style="author-callout-style-b1" valign="bottom">0.74</td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td></tr><tr><td align="left" valign="bottom">T232</td><td align="right" valign="bottom">11</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>&lt;0.001</bold></td><td align="right" style="author-callout-style-b2" valign="bottom"><bold>&lt;0.001</bold></td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.001</bold></td><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.001</bold></td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T234</td><td align="right" valign="bottom">10</td><td align="right" style="author-callout-style-b1" valign="bottom">1</td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T249</td><td align="right" valign="bottom">14</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.002</bold></td><td align="right" style="author-callout-style-b2" valign="bottom"><bold>&lt;0.001</bold></td><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.017</bold></td><td align="right" style="author-callout-style-b4" valign="bottom"><bold>0.016</bold></td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T271</td><td align="right" valign="bottom">23</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.043</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom">0.49</td><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td></tr><tr><td align="left" valign="bottom">T327</td><td align="right" valign="bottom">10</td><td align="right" style="author-callout-style-b1" valign="bottom">1</td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.51</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T330</td><td align="right" valign="bottom">8</td><td align="right" style="author-callout-style-b1" valign="bottom">1</td><td align="right" style="author-callout-style-b2" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.26</td><td align="right" style="author-callout-style-b4" valign="bottom">1*</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b3" valign="bottom"/></tr><tr><td align="left" valign="bottom">T358</td><td align="right" valign="bottom">22</td><td align="right" style="author-callout-style-b1" valign="bottom"><bold>0.004</bold></td><td align="right" style="author-callout-style-b2" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">0.06</td><td align="right" style="author-callout-style-b3" valign="bottom"/><td align="right" style="author-callout-style-b4" valign="bottom">1</td><td align="right" style="author-callout-style-b3" valign="bottom"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46490-v1.xml">elife-46490-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.46490.007</object-id><label>Table 2.</label><caption><title>Summary of oxidations observed on protein substrates produced by IVTT and reacted with the indicated PHD enzyme(s).</title><p>Mass spectrometry was performed on IVTT-derived substrates (column 1) reacted in the presence or absence of the indicated PHD isoform (column 2). Substrates were immunopurified by FLAG affinity and digested with the specified protease(s) (column 3) to yield peptides encompassing the putative target prolyl site, which are indicated in bold (column 4); note, during sample processing peptides containing cysteine residues were derivatized with iodoacetamide to give carbamidomethylated cysteine (+57.02). Reference to primary data for non-HIF substrates (i.e., MSMS assignment and quantitation) is provided in column five and summarised in columns 6&#8211;10. Assigned oxidations are listed in column 6; no oxidation detected (ND); oxidation detected but not localised to a specific residue (NL). Note, prolyl hydroxylation was not detected. Quantitative data for control and PHD-reacted IVTTs is given for these oxidations (columns 7&#8211;8, respectively). Low abundance peptide ions of compatible mass for oxidation, which were below the threshold for MSMS determination but present in LC-MS data (within &#8722;5 min retention time window), were also quantified (columns 9&#8211;10). The abundance of each assigned or putative oxidation is expressed as a percentage of the non-hydroxylated peptide. Summary results for the products of control reactions that were conducted in parallel on HIF-1&#945; to verify PHD enzyme activity are shown in columns 11&#8211;14; values were obtained by similar methods (see <xref ref-type="supplementary-material" rid="supp2">Supplementary file 2</xref> for figures depicting these primary data). Each row relates a specific &#8216;test&#8217; reaction to its batch control HIF-1&#945; reaction; note that high levels of activity on HIF-1&#945; substrates were observed upon addition of exogenous PHD enzyme in all controls.</p><p><supplementary-material id="table2sdata1"><object-id pub-id-type="doi">10.7554/eLife.46490.008</object-id><label>Table 2&#8212;source data 1.</label><caption><title>Peptide standards employed in IVTT hydroxylation assays.</title><p>The table lists synthetic peptide sequences corresponding to unoxidised and hydroxylated variants (column 3) of protease-digested peptides assigned and quantified in IVTT hydroxylation assays. Equimolar injections of the indicated peptide variants were used for comparison of detection efficiency (column 4) and chromatographic elution time (column 5) by LC-MSMS analysis. References to the primary XIC data are indicated in column 6.</p></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-46490-table2-data1-v1.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3" rowspan="2"/><th align="center" colspan="7">Non-HIF substrate</th><th align="center" colspan="4">HIF-1&#945; control</th></tr><tr><th align="center" rowspan="2">Quantified peptide</th><th rowspan="2">Figure</th><th rowspan="2">Oxidation</th><th colspan="2">Assigned ox (%)</th><th colspan="2">Unassigned ions (%)</th><th colspan="2">HIF-1&#945; P402ox (%)</th><th colspan="2">HIF-1&#945; P564ox (%)</th></tr><tr><th>Substrate</th><th>PHD Isoform</th><th>Protease</th><th>Control</th><th>Enzyme</th><th>Control</th><th>Enzyme</th><th>Control</th><th>Enzyme</th><th>Control</th><th>Enzyme</th></tr></thead><tbody><tr><td rowspan="2">ACACB</td><td rowspan="2">PHD3</td><td rowspan="2">LysC</td><td>RI<bold>P</bold>VQAVWAGWGHASENPKLPELLC(+57.02)K</td><td>3, s1</td><td>ND</td><td valign="bottom">-</td><td valign="bottom">-</td><td valign="bottom">0.4%</td><td valign="bottom">1.7%</td><td rowspan="2">58%</td><td rowspan="2">52%</td><td rowspan="2">53%</td><td rowspan="2">72%</td></tr><tr><td>DVDEGLEAAERIGF<bold>P</bold>LMIK</td><td>3, s2</td><td>M452</td><td valign="bottom">76%</td><td valign="bottom">77%</td><td valign="bottom">2.3%</td><td valign="bottom">0.9%</td></tr><tr><td rowspan="2">ACTB</td><td rowspan="2">PHD3</td><td rowspan="2">Trypsin</td><td>DLYANTVLSGGTTMY<bold>P</bold>GIADR</td><td>3, s3</td><td>M305</td><td valign="bottom">71%</td><td valign="bottom">48%</td><td valign="bottom">ND</td><td valign="bottom">ND</td><td rowspan="2">6.3%</td><td rowspan="2">9.9%</td><td rowspan="2">29%</td><td rowspan="2">91%</td></tr><tr><td>EITALA<bold>P</bold>STMK</td><td>3, s4</td><td>M325</td><td valign="bottom">54%</td><td valign="bottom">30%</td><td valign="bottom">0.3%</td><td valign="bottom">0.1%</td></tr><tr><td>ADRB2</td><td>PHD3</td><td>Trypsin</td><td>LLC(+57.02)EDL<bold>P</bold>GTEDFVGHQGTV<bold>P</bold>SDNIDSQGR</td><td>3, s5</td><td>D380</td><td valign="bottom">2.8%</td><td valign="bottom">1.5%</td><td valign="bottom">ND</td><td valign="bottom">&lt;0.1%</td><td>6.3%</td><td>9.9%</td><td>29%</td><td>91%</td></tr><tr><td rowspan="3">AKT1</td><td rowspan="3">PHD2</td><td rowspan="3">Trypsin</td><td>SGS<bold>P</bold>SDNSGAEEMEVSLAK</td><td>3, s6</td><td>M134</td><td valign="bottom">35%</td><td valign="bottom">39%</td><td valign="bottom">ND</td><td valign="bottom">ND</td><td rowspan="3">18%</td><td rowspan="3">94%</td><td rowspan="3">30%</td><td rowspan="3">96%</td></tr><tr><td>TFC(+57.02)GT<bold>P</bold>EYLA<bold>P</bold>EVLEDNDYGR</td><td>3, s7</td><td>ox: NL</td><td valign="bottom">0.6%</td><td valign="bottom">0.8%</td><td valign="bottom">ND</td><td valign="bottom">ND</td></tr><tr><td>LS<bold>P</bold>PFKPQVTSETDTR</td><td>3, s8</td><td>ND</td><td valign="bottom">-</td><td valign="bottom">-</td><td valign="bottom">ND</td><td valign="bottom">ND</td></tr><tr><td rowspan="2">ATF4</td><td rowspan="2">PHD3</td><td rowspan="2">Elastase</td><td>GHLPESLTKPDQVA<bold>P</bold>FTFLQ<bold>P</bold>L<bold>P</bold>LS<bold>P</bold>G</td><td>3, s9</td><td>ND</td><td valign="bottom">-</td><td valign="bottom">-</td><td valign="bottom">ND</td><td valign="bottom">ND</td><td rowspan="2">6.3%</td><td rowspan="2">9.9%</td><td rowspan="2">29%</td><td rowspan="2">91%</td></tr><tr><td>ST<bold>P</bold>DHSFSLELGSEVDITEGDRKPDYT</td><td>3, s10</td><td>ND</td><td valign="bottom">-</td><td valign="bottom">-</td><td valign="bottom">ND</td><td valign="bottom">ND</td></tr><tr><td>CENPN</td><td>PHD2</td><td>Trypsin</td><td>SLA<bold>P</bold>AGIADAPLSPLLTC(+57.02)IPNKR</td><td>3, s11</td><td>ND</td><td valign="bottom">-</td><td valign="bottom">-</td><td valign="bottom">ND</td><td valign="bottom">ND</td><td>ND</td><td>ND</td><td>22%</td><td>94%</td></tr><tr><td>CEP192</td><td>PHD1</td><td>Trypsin</td><td>WHLSSLA<bold>P</bold>PYVK</td><td>3, s12</td><td>ND</td><td valign="bottom">-</td><td valign="bottom">-</td><td valign="bottom">&lt;0.1%</td><td valign="bottom">ND</td><td>51%</td><td>95%</td><td>60%</td><td>82%</td></tr><tr><td>EEF2K</td><td>PHD2</td><td>Trypsin</td><td>HMPD<bold>P</bold>WAEFHLEDIATER</td><td>3, s13</td><td>M95</td><td valign="bottom">48%</td><td valign="bottom">38%</td><td valign="bottom">ND</td><td valign="bottom">0.2%</td><td>0.5%</td><td>88%</td><td>36%</td><td>93%</td></tr><tr><td rowspan="2">EPOR</td><td rowspan="2">PHD3</td><td rowspan="2">LysC&#160;+&#160;GluC</td><td rowspan="2">YTILDPSSQLLRPWTLC(+57.02)<bold>P</bold>ELPPTP<bold>P</bold>HLK</td><td rowspan="2">3, s14</td><td>ox: NL</td><td valign="bottom">1.4%</td><td valign="bottom">1.4%</td><td rowspan="2">ND</td><td rowspan="2">ND</td><td rowspan="2">9.5%</td><td rowspan="2">15%</td><td rowspan="2">23%</td><td rowspan="2">94%</td></tr><tr><td>diox: W439</td><td valign="bottom">2.0%</td><td valign="bottom">1.8%</td></tr><tr><td>FLNA</td><td>PHD2</td><td>Trypsin</td><td>FNEEHI<bold>P</bold>DSPFVV<bold>P</bold>VASPSGDAR</td><td>3, s15</td><td>ND</td><td valign="bottom">-</td><td valign="bottom">-</td><td valign="bottom">0.1%</td><td valign="bottom">0.3%</td><td>ND</td><td>ND</td><td>22%</td><td>94%</td></tr><tr><td>FOXO3</td><td>PHD1</td><td>Trypsin</td><td>GSGLGS<bold>P</bold>TSSFNSTVFG<bold>P</bold>SSLNSLR</td><td>3, s16</td><td>ND</td><td valign="bottom">-</td><td valign="bottom">-</td><td valign="bottom">2.2%</td><td valign="bottom">0.9%</td><td>14%</td><td>97%</td><td>25%</td><td>86%</td></tr><tr><td>IKBKB</td><td>PHD1</td><td>Trypsin</td><td>ELDQGSLC(+57.02)TSFVGTLQYLA<bold>P</bold>ELLEQQK</td><td>3, s17</td><td>ND</td><td valign="bottom">-</td><td valign="bottom">-</td><td valign="bottom">14%</td><td valign="bottom">14%</td><td>14%</td><td>97%</td><td>25%</td><td>86%</td></tr><tr><td>MAPK6</td><td>PHD3</td><td>Trypsin</td><td>YMDLK<bold>P</bold>LGC(+57.02)GGNGLVFSAVDNDC(+57.02)DKR</td><td>3</td><td>M21</td><td valign="bottom">19%</td><td valign="bottom">12%</td><td valign="bottom">0.3%</td><td valign="bottom">0.3%</td><td>9.5%</td><td>15%</td><td>23%</td><td>94%</td></tr><tr><td>NDRG3</td><td>PHD2</td><td>Trypsin</td><td>MADC(+57.02)GGL<bold>P</bold>QVVQPGK</td><td>3, s18</td><td>M287</td><td>48%</td><td>21%</td><td>1.8%</td><td>0.9%</td><td>17%</td><td>92%</td><td>24%</td><td>93%</td></tr><tr><td rowspan="2">PDE4D</td><td rowspan="2">PHD2</td><td rowspan="2">Trypsin</td><td>LMHSSSLTNSSI<bold>P</bold>R</td><td>3, s19</td><td>M371</td><td>61%</td><td>36%</td><td>ND</td><td>ND</td><td rowspan="2">0.5%</td><td rowspan="2">88%</td><td rowspan="2">36%</td><td rowspan="2">93%</td></tr><tr><td>IAELSGNR<bold>P</bold>LTVIMHTIFQER</td><td>3, s20</td><td>M424</td><td>51%</td><td>45%</td><td>ND</td><td>ND</td></tr><tr><td>PKM</td><td>PHD3</td><td>Trypsin</td><td>LA<bold>P</bold>ITSD<bold>P</bold>TEATAVGAVEASFK</td><td>3, s21</td><td>ND</td><td>-</td><td>-</td><td>&lt;0.1%</td><td>0.1%</td><td>6.3%</td><td>9.9%</td><td>29%</td><td>91%</td></tr><tr><td>PPP2R2A</td><td>PHD2</td><td>Trypsin</td><td>IWDLNMENR<bold>P</bold>VETYQVHEYLR</td><td>3, s22</td><td>M315</td><td>14%</td><td>20%</td><td>ND</td><td>ND</td><td>27%</td><td>86%</td><td>36%</td><td>91%</td></tr><tr><td rowspan="6">SPRY2</td><td>PHD1</td><td rowspan="6">Trypsin</td><td rowspan="2">AQSGNGSQPLLQT<bold>P</bold>R</td><td>-</td><td>-</td><td>-</td><td>-</td><td>ND</td><td>ND</td><td>-</td><td>-</td><td>-</td><td>-</td></tr><tr><td>PHD3</td><td>3, s23</td><td>ND</td><td>-</td><td>-</td><td>ND</td><td>ND</td><td>ND</td><td>ND</td><td>52%</td><td>92%</td></tr><tr><td>PHD1</td><td rowspan="2">LLGSSFSSG<bold>P</bold>VADGIIR</td><td>3, s24</td><td>ND</td><td>-</td><td>-</td><td>0.9%</td><td>ND</td><td>9.5%</td><td>92%</td><td>23%</td><td>83%</td></tr><tr><td>PHD3</td><td>3, s25</td><td>ND</td><td>-</td><td>-</td><td>ND</td><td>ND</td><td>ND</td><td>ND</td><td>52%</td><td>92%</td></tr><tr><td>PHD1</td><td rowspan="2">SELK<bold>P</bold>GELKPLSK</td><td>3, s26</td><td>ND</td><td>-</td><td>-</td><td>ND</td><td>ND</td><td>9.5%</td><td>92%</td><td>23%</td><td>83%</td></tr><tr><td>PHD3</td><td>3, s27</td><td>ND</td><td>-</td><td>-</td><td>ND</td><td>ND</td><td>ND</td><td>ND</td><td>52%</td><td>92%</td></tr><tr><td>TELO2</td><td>PHD3</td><td>Trypsin</td><td>AVLIC(+57.02)LAQLGE<bold>P</bold>ELR</td><td>3, s28</td><td>ND</td><td>-</td><td>-</td><td>8.9%</td><td>6.5%</td><td>ND</td><td>ND</td><td>52%</td><td>92%</td></tr><tr><td rowspan="4">THRA</td><td rowspan="2">PHD2</td><td rowspan="4">Trypsin</td><td rowspan="4">SLQQRPE<bold>P</bold>T<bold>P</bold>EEWDLIHIATEAHR</td><td rowspan="2">3, s29</td><td>ox: NL</td><td>0.9%</td><td>0.7%</td><td rowspan="2">1.8%</td><td rowspan="2">1.3%</td><td rowspan="2">15%</td><td rowspan="2">94%</td><td rowspan="2">26%</td><td rowspan="2">96%</td></tr><tr><td>diox:&#160;W165</td><td>5.4%</td><td>5.1%</td></tr><tr><td rowspan="2">PHD3</td><td rowspan="2">3, s30</td><td>ox: NL</td><td>0.9%</td><td>0.8%</td><td rowspan="2">1.8%</td><td rowspan="2">0.8%</td><td rowspan="2">15%</td><td rowspan="2">16%</td><td rowspan="2">26%</td><td rowspan="2">86%</td></tr><tr><td>diox:&#160;W165</td><td>5.4%</td><td>4.8%</td></tr><tr><td rowspan="2">TP53</td><td>PHD1</td><td>Trypsin</td><td>TC(+57.02)<bold>P</bold>VQLWVDSTPPPGTR</td><td>3, s31</td><td>ND</td><td>-</td><td>-</td><td>ND</td><td>ND</td><td>9.5%</td><td>92%</td><td>23%</td><td>83%</td></tr><tr><td>PHD3</td><td>LysC</td><td>E<bold>P</bold>GGSRAHSSHLK</td><td>3, s32</td><td>ND</td><td>-</td><td>-</td><td>ND</td><td>0.4%</td><td>21%</td><td>24%</td><td>30%</td><td>78%</td></tr><tr><td rowspan="2">TRPA1</td><td rowspan="2">PHD2</td><td rowspan="2">Trypsin</td><td rowspan="2">NLR<bold>P</bold>EFMQMQQIK</td><td rowspan="2">3, s33</td><td>M397</td><td>31%</td><td>30%</td><td rowspan="2">ND</td><td rowspan="2">ND</td><td rowspan="2">0.5%</td><td rowspan="2">88%</td><td rowspan="2">36%</td><td rowspan="2">93%</td></tr><tr><td>M399</td><td>33%</td><td>34%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-46750-v1.xml">elife-46750-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.46750.012</object-id><label>Table 1.</label><caption><title>Comparison of movement speed (measured in movement cycles per 10 s) during the first 10 s of the first and last trials for different effectors, as well as movement speed within a 30 s trial (first vs. last 10 s).</title><p>The results show that movement speed at the beginning of each trial is stable, whereas movement speed decreased significantly within a trial. All values mean &#177;&#160;sem.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="2" valign="top">Speed during first 10s</th><th colspan="2" valign="top">Speed within 30s</th><th rowspan="2">N Subjects</th><th rowspan="2">N Trials</th></tr><tr valign="top"><th>First Trial</th><th>Last Trial</th><th>First 10s</th><th>Last 10s</th></tr></thead><tbody><tr valign="top"><td>2-finger tapping</td><td>39.48&#177;1.61</td><td>37.00&#177;1.24</td><td>37.89&#177;1.12</td><td>31.82&#177;1.04</td><td>25</td><td>16</td></tr><tr valign="top"><td>Foot tapping</td><td>50.00&#177;4.16</td><td>53.67&#177;4.20</td><td>55.167&#177;3.25</td><td>47.233&#177;2.89</td><td>12</td><td>20</td></tr><tr valign="top"><td>Eye Movement</td><td>14.33&#177;0.83</td><td>16.83&#177;0.95</td><td>15.817&#177;0.77</td><td>12.933&#177;0.67</td><td>12</td><td>20</td></tr></tbody></table><table-wrap-foot><fn><p>mean &#177;&#160;standard error of the mean, all values Movement Cycles per 10s.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47035-v1.xml">elife-47035-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.47035.020</object-id><label>Table 1.</label><caption><title>Variance explained by different models (coefficient of determination).</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"><bold>Data</bold></th><th colspan="3" valign="top"><bold>Model</bold></th></tr><tr><th valign="top">&#8195;Mean</th><th valign="top">SOC</th><th valign="top">OV</th></tr></thead><tbody><tr><td valign="top">Broadband</td><td valign="top">41%</td><td valign="top">80%</td><td valign="top">24%</td></tr><tr><td valign="top">Gamma</td><td valign="top">26%</td><td valign="top">62%</td><td valign="top">75%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47138-v1.xml">elife-47138-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.47138.011</object-id><label>Table 1.</label><caption><title>Adjusted p values and direction of bias for gene expression biases of selected gene groups in germ cells.</title><p>Spermatogenesis progresses downward from GSC/Early spermatogonia and ends in late spermatids. Upwards arrows indicate that the top group of genes is biased upwards compared to the bottom group, and downwards arrows indicate that it is biased downward according to a directional Hochberg test. For example, ribosomal proteins are more expressed in late spermatogonia than all other genes, with an adjusted p value of 1.24E-75. Note that while segregating de novo genes are expressed differently from testis-specific genes in GSC, early spermatogonia and early spermatids, fixed de novo genes do not significantly deviate from expression patterns of testis-specific genes in any cell type.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Versus:</th><th colspan="2">Ribosomal protein genes</th><th colspan="2">Segregating de novo genes</th><th colspan="2">Fixed de novo genes</th><th colspan="2">DNA repair genes</th></tr><tr><th>All other genes</th><th>Testis-specific genes</th><th>All other genes</th><th>Testis-specific genes</th><th>All other genes</th><th>Testis-specific genes</th><th>All other genes</th><th>Testis-specific genes</th></tr></thead><tbody><tr><td>GSC, early spermatogonia</td><td>&#8593; 1.13E-82</td><td>&#8593; 1.44E-84</td><td>&#8595; 1.46E-21</td><td>&#8593; 9.35E-04</td><td>&#8595; 2.92E-22</td><td>ns</td><td>&#8593; 4.69E-26</td><td>&#8593; 8.14E-62</td></tr><tr><td>Late spermatogonia</td><td>&#8593; 1.24E-75</td><td>&#8593; 1.62E-74</td><td>&#8595; 8.22E-19</td><td>ns</td><td>&#8595; 5.89E-18</td><td>ns</td><td>&#8593; 2.30E-20</td><td>&#8593; 5.80E-44</td></tr><tr><td>Early spermatocytes</td><td>&#8595; 2.53E-76</td><td>&#8595; 1.08E-71</td><td>&#8593; 4.08E-15</td><td>ns</td><td>&#8593; 5.31E-13</td><td>ns</td><td>&#8595; 6.50E-23</td><td>&#8595; 1.75E-38</td></tr><tr><td>Late spermatocytes</td><td>&#8595; 2.51E-57</td><td>&#8595; 1.90E-58</td><td>&#8593; 1.09E-10</td><td>ns</td><td>&#8593; 2.17E-15</td><td>ns</td><td>&#8595; 3.96E-09</td><td>&#8595; 4.58E-29</td></tr><tr><td>Early spermatids</td><td>&#8595; 8.94E-03</td><td>&#8595; 1.57E-08</td><td>ns</td><td>&#8595; 1.70E-02</td><td>ns</td><td>ns</td><td>&#8593; 5.89E-08</td><td>ns</td></tr><tr><td>Late spermatids</td><td>&#8595; 7.40E-10</td><td>&#8595; 1.70E-02</td><td>ns</td><td>ns</td><td>ns</td><td>ns</td><td>ns</td><td>ns</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47198-v1.xml">elife-47198-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.47198.015</object-id><label>Table 1.</label><caption><title>Summary of heterozygous <italic>AMH</italic> or <italic>AMHR2</italic> mutations identified in patients with congenital hypogonadotropic hypogonadism.</title><p>cDNA and protein changes are based on reference cDNA sequence NM_000479.4 (<italic>AMH</italic>) and NM_020547.3 (<italic>AMHR2</italic>). Functional validation of the mutants has been performed in vitro evaluating AMH secretion in COS-7 cells, cell motility in GN11 cells, and measuring GnRH secretion in GT1-7 cells for nCHH-associated mutants. CHH, congenital hypogonadotropic hypogonadism; nCHH, normosmic CHH; KS, Kallmann syndrome; Sex: F: female; M: male; Inheritance: F: familial, S: sporadic; Puberty: A: absent puberty, P: partial puberty. MAF, minor allele frequency; &#8595;, decreased; NS, not significant; NA, not applicable.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Gene</th><th rowspan="2">Family</th><th rowspan="2">Subject</th><th rowspan="2">Diagnosis</th><th rowspan="2">Sex</th><th rowspan="2">Inheritance</th><th rowspan="2">Puberty</th><th rowspan="2">Associated phenotypes</th><th rowspan="2">dbSNP number</th><th rowspan="2">Nucleotide change</th><th rowspan="2">Amino acid change</th><th rowspan="2">MAF (%) gnomAD MaxPop</th><th colspan="3">In vitro studies</th></tr><tr><th>Released AMH (COS-7 cells)</th><th>Cell motility (GN11 cells)</th><th>GnRH secretion (GT1-7 cells)</th></tr></thead><tbody><tr><td rowspan="3"><italic>AMH</italic></td><td>1</td><td>II-1</td><td>nCHH</td><td>M</td><td>F</td><td>P</td><td>High-arched palate Deviated nasal septumHyperlaxity</td><td>rs200226465</td><td>c.295A&#160;&gt;&#160;T</td><td>p.Thr99Ser</td><td>0.044</td><td>&#8595;&#8595;</td><td>&#8595;&#8595;</td><td>&#8595;&#8595;</td></tr><tr><td>&#8195;2</td><td>II-1</td><td>KS</td><td>M</td><td>S</td><td>A</td><td>Cryptochidism</td><td>rs370532523</td><td>c.451C&#160;&gt;&#160;T</td><td>p.Pro151Ser</td><td>0.011</td><td>&#8595;&#8595;</td><td>&#8595;&#8595;</td><td/></tr><tr><td>&#8195;3</td><td>II-2</td><td>KS</td><td>F</td><td>F</td><td>P</td><td>Osteoporosis Scoliosis</td><td>rs752574731</td><td>c.714C&#160;&gt;&#160;A</td><td>p.Asp238Glu</td><td>0.006</td><td>&#8595;&#8595;</td><td>&#8595;&#8595;</td><td/></tr><tr><td><italic>AMHR2</italic></td><td>4</td><td>II-1</td><td>nCHH</td><td>F</td><td>S</td><td>A</td><td>Osteoporosis</td><td>rs764761319</td><td>c.1330_1356del</td><td>p.Gly445_Leu453del</td><td>0.093</td><td/><td>&#8595;&#8595;</td><td>&#8595;&#8595;</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47262-v2.xml">elife-47262-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.47262.011</object-id><label>Table 1.</label><caption><title>Shared transcriptomic response.</title><p>Number of genes that are influenced by sex (S), diet (D), and their interaction (D&#160;&#215;&#160;S). From this method, we were able to cluster genes into three main categories. Categories highlighted in orange encompass genes that show an additive effect to diet (D), whereas clusters in blue show interactive effects (D&#160;&#215;&#160;S). Green rows show both additive and interactive effects (D+D&#160;&#215;&#160;S).</p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3">Significance (FDR&#160;&lt;&#160;0.05)</th><th rowspan="2">n. genes</th></tr><tr><th>S</th><th>D</th><th>D&#160;&#215;&#160;S</th></tr></thead><tbody><tr><td>&#8195;-</td><td>-</td><td>-</td><td>545</td></tr><tr><td>&#8195;-</td><td>-</td><td>Y</td><td>3</td></tr><tr><td>&#8195;-</td><td>Y</td><td>-</td><td>18</td></tr><tr><td>&#8195;-</td><td>Y</td><td>Y</td><td>4</td></tr><tr><td>&#8195;Y</td><td>-</td><td>-</td><td>7537</td></tr><tr><td>&#8195;Y</td><td>-</td><td>Y</td><td>48</td></tr><tr><td>&#8195;Y</td><td>Y</td><td>-</td><td>621</td></tr><tr><td>&#8195;Y</td><td>Y</td><td>Y</td><td>112</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47433-v1.xml">elife-47433-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.47433.003</object-id><label>Table 1.</label><caption><title>Early adverse conditions and the frequencies with which they occur in maternal and offspring generations of our dataset.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Adverse Condition<sup>*</sup></th><th rowspan="2" valign="top">Criterion</th><th colspan="2" valign="top">Frequency</th></tr><tr><th valign="top">&#8195;<italic>Maternal Generation</italic></th><th valign="top"><italic>Offspring Generation</italic></th></tr></thead><tbody><tr><td valign="top">Drought</td><td valign="top">During the first year of life, the focal individual experienced less than 200 mm of rainfall (i.e., drought conditions; <xref ref-type="bibr" rid="bib15">Beehner et al., 2006</xref>).</td><td valign="top">0.09</td><td valign="top">0.15</td></tr><tr><td valign="top">High Social Density</td><td valign="top">The individual was born into a group with a high social density (&gt;35 adults), indicating high levels of within-group competition.</td><td valign="top">0.06</td><td valign="top">0.32</td></tr><tr><td valign="top">Maternal Loss</td><td valign="top">The mother of the focal individual died within four years of the individual&#8217;s birth.</td><td valign="top">0.21</td><td valign="top">0.25</td></tr><tr><td valign="top">Low Maternal Rank<bold><sup>&#8224;</sup></bold></td><td valign="top">The focal individual was born to a mother with a low social rank (mother&#8217;s rank fell in the bottom quartile of the group&#8217;s dominance hierarchy, rank&#160;&lt;&#160;0.25).</td><td valign="top">0.17</td><td valign="top">0.23</td></tr><tr><td valign="top">Close-In-Age Younger Sibling<sup>&#8225;</sup></td><td valign="top">The focal individual had a younger sibling born to its mother within 18 months of the focal&#8217;s birth.</td><td valign="top">0.20</td><td valign="top">--</td></tr></tbody></table><table-wrap-foot><fn><p><sup>*</sup>These criteria were used in a previous analysis in our population (<xref ref-type="bibr" rid="bib81">Tung et al., 2016</xref>), with the exception of maternal rank, which is evaluated here as a proportional measure rather than an ordinal one as in the previous analysis.</p><p><sup>&#8224;</sup>Proportional rank is the proportion of other adult females in a group that an individual&#8217;s mother outranks. The reduced frequency with which low maternal rank appears in the maternal generation is a likely a result of offspring of low-ranking mothers surviving less well (<xref ref-type="bibr" rid="bib75">Silk et al., 2003</xref>), and therefore not surviving to appear as mothers in our dataset.</p></fn><fn><p><sup>&#8225;</sup>We excluded the birth of a close-in-age younger sibling for the offspring generation to avoid including a potential reverse-causal factor in our model: the closest-in-age siblings in our dataset occur as a result of the focal offspring&#8217;s death, because female baboons (who are not seasonal reproducers) accelerate their next conception after the death of a dependent offspring.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47498-v2.xml">elife-47498-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><object-id pub-id-type="doi">10.7554/eLife.47498.011</object-id><label>Table 2.</label><caption><title>Information of patients included in this study.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Patient ID</th><th rowspan="3">Gender</th><th rowspan="3">Age</th><th rowspan="3">Disease duration (years)</th><th rowspan="3">Braak stages (AD)</th><th rowspan="3">Postmortem delays <break/>[hours]</th><th/></tr><tr><th/></tr><tr><th/></tr></thead><tbody><tr><td>&#8195;Cont. 1</td><td>Male</td><td>86</td><td>-</td><td>II/A</td><td>06:45</td><td/></tr><tr><td>&#8195;Cont. 2</td><td>Male</td><td>61</td><td>-</td><td>I/0</td><td>03:03</td><td/></tr><tr><td>&#8195;Cont. 3</td><td>Male</td><td>74</td><td>-</td><td>II/A</td><td>11:00</td><td/></tr><tr><td>&#8195;Cont. 4</td><td>Male</td><td>86</td><td>-</td><td>II/A</td><td>06:45</td><td/></tr><tr><td>&#8195;Cont. 5</td><td>Female</td><td>73</td><td>-</td><td>I/0</td><td>04:03</td><td/></tr><tr><td>&#8195;Cont. 6</td><td>Male</td><td>69</td><td>-</td><td>II/A</td><td>05:03</td><td/></tr><tr><td>&#8195;Cont. 7</td><td>Male</td><td>68</td><td>-</td><td>I/0</td><td>05:03</td><td/></tr><tr><td>&#8195;Cont. 8</td><td>Female</td><td>64</td><td>-</td><td>I/0</td><td>09:00</td><td/></tr><tr><td>&#8195;Cont. 9</td><td>Male</td><td>67</td><td>-</td><td>I/0</td><td>05:03</td><td/></tr><tr><td>&#8195;spAD1</td><td>Female</td><td>72</td><td>&gt;4</td><td>V/C</td><td>09:30</td><td/></tr><tr><td>&#8195;spAD2</td><td>Female</td><td>75</td><td>&gt;4</td><td>V/C</td><td>04:15</td><td/></tr><tr><td>&#8195;spAD3</td><td>Male</td><td>78</td><td>&gt;4</td><td>V/C</td><td>09:30</td><td/></tr><tr><td>&#8195;spAD4</td><td>Male</td><td>83</td><td>&lt;4</td><td>V/C</td><td>08:20</td><td/></tr><tr><td>&#8195;spAD5</td><td>Female</td><td>56</td><td>&gt;4</td><td>V/C</td><td>07:00</td><td/></tr><tr><td>&#8195;spAD6</td><td>Male</td><td>83</td><td>&gt;4</td><td>III/0</td><td>07:25</td><td/></tr><tr><td>&#8195;spAD7</td><td>Female</td><td>90</td><td>&gt;4</td><td>IV/A</td><td>09:55</td><td/></tr><tr><td>&#8195;spAD8</td><td>Female</td><td>93</td><td>&gt;4</td><td>V/C</td><td>03:00</td><td/></tr><tr><td>&#8195;rpAD1</td><td>Male</td><td>78</td><td>&lt;4</td><td>V/C</td><td>03:30</td><td/></tr><tr><td>&#8195;rpAD2</td><td>Female</td><td>79</td><td>&lt;4</td><td>V</td><td>05:30</td><td/></tr><tr><td>&#8195;rpAD3</td><td>Female</td><td>81</td><td>&lt;4</td><td>III/B</td><td>06:00</td><td/></tr><tr><td>&#8195;rpAD4</td><td>Male</td><td>83</td><td>&lt;4</td><td>VI/C</td><td>05:30</td><td/></tr><tr><td>&#8195;rpAD5</td><td>Male</td><td>83</td><td>&lt;4</td><td>V/C</td><td>08:20</td><td/></tr><tr><td>&#8195;rpAD6</td><td>Male</td><td>70</td><td>&lt;4</td><td>VI/C</td><td>11:30</td><td/></tr><tr><td>&#8195;rpAD7</td><td>Male</td><td>76</td><td>&lt;4</td><td>VI/C</td><td>06:30</td><td/></tr><tr><td>&#8195;rpAD8</td><td>Female</td><td>77</td><td>&lt;4</td><td>IV/A</td><td>12:00</td><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47509-v1.xml">elife-47509-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="app6table1" position="float"><label>Appendix 6&#8212;table 1.</label><caption><title>Marginal likelihoods of six partitioning schemes and two tree priors for the 25 dated mitogenomes.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Partitioning scheme<sup>a</sup></th><th colspan="2">Marginal likelihood<sup>b</sup></th></tr><tr><th>Constant size</th><th>Exponential growth</th></tr></thead><tbody><tr><td>Unpartitioned</td><td>&#8722;24,151.6</td><td>&#8722;24,143.6</td></tr><tr><td>two subsets: (CR rRNA tRNA) (PC1 PC2 PC3)</td><td>&#8722;24,222.3</td><td>&#8722;24,212.4</td></tr><tr><td>three subsets: (CR) (rRNA tRNA) (PC1 PC2 PC3)</td><td>&#8722;24,162.4</td><td>&#8722;24,150.1</td></tr><tr><td>four subsets: (CR) (rRNA tRNA) (PC1 PC2) (PC3)</td><td>&#8722;23,659.7</td><td>&#8722;23,647.5</td></tr><tr><td>five subsets: (CR) (rRNA tRNA) (PC1) (PC2) (PC3)</td><td>&#8722;23,248.7</td><td>&#8722;23,235.9</td></tr><tr><td>six subsets: (CR) (rRNA) (tRNA) (PC1) (PC2) (PC3)</td><td>&#8722;23,229.1</td><td>&#8722;23,217.5</td></tr></tbody></table><table-wrap-foot><fn><p><sup>a</sup>Components of the mitogenome are the ribosomal RNA genes (rRNA), transfer RNA genes (tRNA), three codon positions of the protein-coding genes (PC1, PC2, and PC3), and the control region (CR). <sup>b</sup>Marginal likelihoods were estimated by stepping-stone sampling with 25 path steps, each with a chain length of 2,000,000 steps.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47534-v2.xml">elife-47534-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.47534.005</object-id><label>Table 1.</label><caption><title>(p)ppGpp binding and inhibition of HPRT homologs.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Organism</th><th colspan="2" valign="top">Relative activity<sup>d</sup></th><th rowspan="2" valign="top">K<sub>d (pppGpp)</sub> (&#956;M)</th><th rowspan="2" valign="top">IC<sub>50 (pppGpp)</sub> <break/>(&#956;M)</th></tr><tr><th valign="top">25 &#956;M ppGpp</th><th valign="top">25 &#956;M pppGpp</th></tr></thead><tbody><tr><td valign="top"><bold>Actinobacteria</bold></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><italic>Streptomyces coelicolor</italic></td><td valign="top">&#8722;0.01&#160;&#177;&#160;0.01</td><td valign="top">0.12&#160;&#177;&#160;0.01</td><td valign="top">1.77&#160;&#177;&#160;0.31</td><td valign="top">3.8&#160;&#177;&#160;0.2</td></tr><tr><td valign="top"><italic>Collinsella aerofaciens</italic></td><td valign="top">0.31&#160;&#177;&#160;0.04</td><td valign="top">0.42&#160;&#177;&#160;0.05</td><td valign="top">1.46&#160;&#177;&#160;0.56</td><td valign="top"/></tr><tr><td valign="top"><italic>Cellulomonas gilvus</italic></td><td valign="top">NA<sup>b</sup></td><td valign="top">NA<sup>b</sup></td><td valign="top">0.6&#160;&#177;&#160;0.11</td><td valign="top"/></tr><tr><td valign="top"><italic>Mycobacterium tuberculosis</italic></td><td valign="top">NA<sup>a</sup></td><td valign="top">NA<sup>a</sup></td><td valign="top">NA<sup>c</sup></td><td valign="top"/></tr><tr><td valign="top"><bold>Bacteroidetes</bold></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><italic>Bacteroides intestinalis</italic></td><td valign="top">0.49&#160;&#177;&#160;0.06</td><td valign="top">0.05&#160;&#177;&#160;0.01</td><td valign="top">0.11&#160;&#177;&#160;0.03</td><td valign="top"/></tr><tr><td valign="top"><italic>Bacteroides caccae</italic></td><td valign="top">0.34&#160;&#177;&#160;0.03</td><td valign="top">0.01&#160;&#177;&#160;0.02</td><td valign="top">0.11&#160;&#177;&#160;0.02</td><td valign="top"/></tr><tr><td valign="top"><italic>Bacteroides ovatus</italic></td><td valign="top">0.43&#160;&#177;&#160;0.04</td><td valign="top">&#8722;0.03&#160;&#177;&#160;0.02</td><td valign="top">0.15&#160;&#177;&#160;0.04</td><td valign="top"/></tr><tr><td valign="top"><italic>Bacteroides uniformis</italic></td><td valign="top">0.4&#160;&#177;&#160;0.01</td><td valign="top">0.01&#160;&#177;&#160;0.01</td><td valign="top">0.1&#160;&#177;&#160;0.04</td><td valign="top"/></tr><tr><td valign="top"><italic>Bacteroides thetaiotaomicron</italic></td><td valign="top">0.41&#160;&#177;&#160;0</td><td valign="top">0&#160;&#177;&#160;0.01</td><td valign="top">0.11&#160;&#177;&#160;0.04</td><td valign="top"/></tr><tr><td valign="top"><italic>Bacteroides finegoldii</italic></td><td valign="top">0.46&#160;&#177;&#160;0.03</td><td valign="top">&#8722;0.01&#160;&#177;&#160;0.01</td><td valign="top">0.15&#160;&#177;&#160;0.06</td><td valign="top"/></tr><tr><td valign="top"><bold>Firmicutes</bold></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><italic>Eubacterium ventriosum</italic></td><td valign="top">&#8722;0.03&#160;&#177;&#160;0.03</td><td valign="top">&#8722;0.07&#160;&#177;&#160;0.03</td><td valign="top">0.66&#160;&#177;&#160;0.1</td><td valign="top"/></tr><tr><td valign="top"><italic>Ruminococcus lactaris</italic></td><td valign="top">&#8722;0.01&#160;&#177;&#160;0</td><td valign="top">0&#160;&#177;&#160;0</td><td valign="top">0.11&#160;&#177;&#160;0.03</td><td valign="top"/></tr><tr><td valign="top"><italic>Ruminococcus torques</italic></td><td valign="top">NA<sup>b</sup></td><td valign="top">NA<sup>b</sup></td><td valign="top">0.22&#160;&#177;&#160;0.07</td><td valign="top"/></tr><tr><td valign="top"><italic>Clostridium leptum</italic></td><td valign="top">&#8722;0.01&#160;&#177;&#160;0.01</td><td valign="top">0&#160;&#177;&#160;0</td><td valign="top">3.29&#160;&#177;&#160;0.94</td><td valign="top"/></tr><tr><td valign="top"><italic>Eubacterium eligens</italic></td><td valign="top">&#8722;0.03&#160;&#177;&#160;0.03</td><td valign="top">0.02&#160;&#177;&#160;0.01</td><td valign="top">0.79&#160;&#177;&#160;0.18</td><td valign="top"/></tr><tr><td valign="top"><italic>Blautia hansenii</italic></td><td valign="top">0.07&#160;&#177;&#160;0</td><td valign="top">0.14&#160;&#177;&#160;0.01</td><td valign="top">0.83&#160;&#177;&#160;0.14</td><td valign="top"/></tr><tr><td valign="top"><italic>Eubacterium hallii</italic></td><td valign="top">0.17&#160;&#177;&#160;0.05</td><td valign="top">&#8722;0.06&#160;&#177;&#160;0.05</td><td valign="top">0.07&#160;&#177;&#160;0.02</td><td valign="top"/></tr><tr><td valign="top"><italic>Clostridium spiroforme</italic></td><td valign="top">0.07&#160;&#177;&#160;0.01</td><td valign="top">0.17&#160;&#177;&#160;0</td><td valign="top">9.54&#160;&#177;&#160;1.17</td><td valign="top"/></tr><tr><td valign="top"><italic>Listeria monocytogenes</italic></td><td valign="top">0.13&#160;&#177;&#160;0</td><td valign="top">0.32&#160;&#177;&#160;0.02</td><td valign="top">0.32&#160;&#177;&#160;0.06</td><td valign="top"/></tr><tr><td valign="top"><italic>Bacillus subtilis</italic></td><td valign="top">0.08&#160;&#177;&#160;0.01</td><td valign="top">0.41&#160;&#177;&#160;0.01</td><td valign="top">0.75&#160;&#177;&#160;0.07</td><td valign="top">10.1&#160;&#177;&#160;1.3</td></tr><tr><td valign="top"><italic>Staphylococcus aureus</italic></td><td valign="top">0.31&#160;&#177;&#160;0.01</td><td valign="top">0.48&#160;&#177;&#160;0.01</td><td valign="top">2.83&#160;&#177;&#160;0.38</td><td valign="top">10.6&#160;&#177;&#160;0.3</td></tr><tr><td valign="top"><italic>Streptococcus mutans</italic></td><td valign="top">NA<sup>b</sup></td><td valign="top">NA<sup>b</sup></td><td valign="top">5.12&#160;&#177;&#160;1.45</td><td valign="top"/></tr><tr><td valign="top"><italic>Bacillus anthracis</italic> 1</td><td valign="top">0.25&#160;&#177;&#160;0.01</td><td valign="top">0.5&#160;&#177;&#160;0.01</td><td valign="top">1.54&#160;&#177;&#160;0.18</td><td valign="top">29.5&#160;&#177;&#160;1.2</td></tr><tr><td valign="top"><italic>Enterococcus faecalis</italic></td><td valign="top">0.31&#160;&#177;&#160;0.01</td><td valign="top">0.49&#160;&#177;&#160;0.02</td><td valign="top">1.12&#160;&#177;&#160;0.11</td><td valign="top"/></tr><tr><td valign="top"><italic>Holdemania filiformis</italic></td><td valign="top">0.66&#160;&#177;&#160;0.01</td><td valign="top">0.43&#160;&#177;&#160;0</td><td valign="top">0.17&#160;&#177;&#160;0.11</td><td valign="top"/></tr><tr><td valign="top"><italic>Bacillus anthracis</italic> 2</td><td valign="top">0.81&#160;&#177;&#160;0.04</td><td valign="top">0.7&#160;&#177;&#160;0.05</td><td valign="top">6.18&#160;&#177;&#160;1.77</td><td valign="top"/></tr><tr><td valign="top"><bold>Deinococcus-Thermus</bold></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><italic>Thermus thermophilus</italic></td><td valign="top">0.52&#160;&#177;&#160;0.07</td><td valign="top">&#8722;0.01&#160;&#177;&#160;0.01</td><td valign="top">0.04&#160;&#177;&#160;0.01</td><td valign="top"/></tr><tr><td valign="top"><bold>Proteobacteria</bold></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><italic>Caulobacter crescentus</italic></td><td valign="top">0.33&#160;&#177;&#160;0.05</td><td valign="top">0.47&#160;&#177;&#160;0.07</td><td valign="top">0.24&#160;&#177;&#160;0.09</td><td valign="top"/></tr><tr><td valign="top"><italic>Escherichia coli</italic></td><td valign="top">0.33&#160;&#177;&#160;0.01</td><td valign="top">0.48&#160;&#177;&#160;0.02</td><td valign="top">3.38&#160;&#177;&#160;1.47</td><td valign="top">26.2&#160;&#177;&#160;2.0</td></tr><tr><td valign="top"><italic>Klebsiella pneumoniae</italic></td><td valign="top">0.47&#160;&#177;&#160;0.03</td><td valign="top">0.49&#160;&#177;&#160;0.01</td><td valign="top">3.38&#160;&#177;&#160;0.89</td><td valign="top"/></tr><tr><td valign="top"><italic>Sinorhizobium meliloti</italic></td><td valign="top">NA<sup>b</sup></td><td valign="top">NA<sup>b</sup></td><td valign="top">3.08&#160;&#177;&#160;0.92</td><td valign="top"/></tr><tr><td valign="top"><italic>Yersinia enterocolitica</italic></td><td valign="top">0.7&#160;&#177;&#160;0.01</td><td valign="top">0.7&#160;&#177;&#160;0.03</td><td valign="top">4.44&#160;&#177;&#160;1.25</td><td valign="top"/></tr><tr><td valign="top"><italic>Photobacterium phosphoreum</italic></td><td valign="top">0.97&#160;&#177;&#160;0.01</td><td valign="top">0.96&#160;&#177;&#160;0.03</td><td valign="top">NA<sup>c</sup></td><td valign="top">&gt;100</td></tr><tr><td valign="top"><italic>Pseudomonas syringae</italic></td><td valign="top">0.6&#160;&#177;&#160;0.01</td><td valign="top">0.6&#160;&#177;&#160;0.01</td><td valign="top">5.85&#160;&#177;&#160;1.76</td><td valign="top"/></tr><tr><td valign="top"><italic>Pseudomonas aeruginosa</italic></td><td valign="top">0.84&#160;&#177;&#160;0.01</td><td valign="top">0.83&#160;&#177;&#160;0.02</td><td valign="top">5.28&#160;&#177;&#160;1.52</td><td valign="top"/></tr><tr><td valign="top"><italic>Legionella pneumophila</italic></td><td valign="top">0.78&#160;&#177;&#160;0.01</td><td valign="top">0.69&#160;&#177;&#160;0.03</td><td valign="top">1.72&#160;&#177;&#160;0.38</td><td valign="top">&gt;100</td></tr><tr><td valign="top"><italic>Coxiella burnetii</italic></td><td valign="top">0.79&#160;&#177;&#160;0.14</td><td valign="top">0.72&#160;&#177;&#160;0.16</td><td valign="top">NA<sup>c</sup></td><td valign="top">&gt;100</td></tr><tr><td valign="top"><italic>Neisseria meningitidis</italic></td><td valign="top">0.9&#160;&#177;&#160;0.02</td><td valign="top">0.8&#160;&#177;&#160;0.03</td><td valign="top">1.64&#160;&#177;&#160;0.55</td><td valign="top"/></tr><tr><td valign="top"><bold>Eukarya</bold></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><italic>Saccharomyces cerevisiae</italic></td><td valign="top">0.64&#160;&#177;&#160;0.02</td><td valign="top">0.96&#160;&#177;&#160;0.01</td><td valign="top">8.08&#160;&#177;&#160;8.58</td><td valign="top"/></tr><tr><td valign="top"><italic>Caenorhabditis elegans</italic></td><td valign="top">NA<sup>b</sup></td><td valign="top">NA<sup>b</sup></td><td valign="top">12.77&#160;&#177;&#160;7.68</td><td valign="top">29.6&#160;&#177;&#160;2.8</td></tr><tr><td valign="top"><italic>Homo sapiens</italic></td><td valign="top">NA</td><td valign="top">NA</td><td valign="top">NA</td><td valign="top">66.6&#160;&#177;&#160;5.4</td></tr></tbody></table><table-wrap-foot><fn><p><sup>&#177;&#160;</sup>standard error of the mean.</p><p>NA&#160;=&#160;no data obtained.</p></fn><fn><p><sup>a</sup>Protein not purified.</p><p><sup>b</sup>Protein did not have activity.</p></fn><fn><p><sup>c</sup>Interaction with pppGpp too weak to estimate K<sub>d.</sub></p><p><sup>d</sup>Relative to activity without ppGpp or pppGpp.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47550-v1.xml">elife-47550-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.47550.013</object-id><label>Table 1.</label><caption><title>Comparative table summarizing characteristics of aesthetascs and olfactory neuropils in several malacostracan species.</title></caption><table frame="hsides" rules="groups"><thead><tr><th/><th colspan="2">Aesthetascs</th><th colspan="3">Olfactory neuropils (ON)</th><th rowspan="2">References</th></tr><tr><th>Species (body length)</th><th>Total number</th><th>Length (&#181;m)</th><th>Neuropil total volume (x10^6 &#181;m3)</th><th>Mean glomerular volume (x10^3 &#181;m3)</th><th>Glomerular number</th></tr></thead><tbody><tr><td><bold>Leptostraca</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>Nebalia herbstii</italic> (1.4 cm)</td><td>-</td><td>-</td><td>0.1</td><td>2</td><td>60</td><td valign="bottom"><xref ref-type="bibr" rid="bib66">Kenning et al., 2013</xref></td></tr><tr><td><bold>Stomatopoda</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>Neogonodactylus oerstedii</italic> (4 cm)</td><td>80</td><td>400</td><td>-</td><td>110</td><td>70</td><td valign="bottom"><xref ref-type="bibr" rid="bib27">Derby et al., 2003</xref></td></tr><tr><td><bold>Isopoda</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td valign="top"><italic>Saduria entomon</italic> (8 cm)</td><td valign="top">40&#8211;60</td><td valign="top">240</td><td valign="top">3</td><td valign="top">34</td><td valign="top">80</td><td valign="top"><xref ref-type="bibr" rid="bib67">Kenning and Harzsch, 2013</xref>; <xref ref-type="bibr" rid="bib106">Pynn&#246;nen, 1985</xref></td></tr><tr><td><bold>Dendrobranchiata</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>Penaeus vannamei</italic> (7 cm)</td><td>280</td><td>-</td><td>-</td><td>-</td><td>&lt;100</td><td valign="bottom"><xref ref-type="bibr" rid="bib169">Wittfoth and Harzsch, 2018</xref>; <xref ref-type="bibr" rid="bib179">Zeng et al., 2002</xref></td></tr><tr><td><bold>Caridea</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>Palaemon elegans</italic> (7 cm)</td><td>280</td><td>230</td><td>120</td><td>225</td><td>530</td><td valign="bottom"><xref ref-type="bibr" rid="bib177">Zbinden et al., 2017</xref>; this study*</td></tr><tr><td><bold><italic>Rimicaris exoculata</italic> (6 cm)</bold></td><td><bold>206</bold></td><td><bold>170</bold></td><td><bold>56</bold></td><td><bold>155</bold></td><td><bold>370</bold></td><td valign="bottom"><xref ref-type="bibr" rid="bib177">Zbinden et al., 2017</xref>; this study</td></tr><tr><td><bold>Achelata</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td valign="top"><italic>Panulirus argus</italic> (20&#8211;60 cm)</td><td valign="top">3000</td><td valign="top">1000</td><td valign="top">154</td><td valign="top">118</td><td valign="top">1332</td><td valign="top"><xref ref-type="bibr" rid="bib3">Beltz et al., 2003</xref>; <xref ref-type="bibr" rid="bib47">Gr&#252;nert and Ache, 1988</xref></td></tr><tr><td><bold>Homarida</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td valign="top"><italic>Homarus americanus</italic> (20&#8211;60 cm)</td><td valign="top">2000</td><td valign="top">600</td><td valign="top">141</td><td valign="top">592</td><td valign="top">249</td><td valign="top"><xref ref-type="bibr" rid="bib3">Beltz et al., 2003</xref>; <xref ref-type="bibr" rid="bib48">Guenther and Atema, 1998</xref></td></tr><tr><td><bold>Astacida</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>Procambarus clarkii</italic> (9 cm)</td><td>133</td><td>-</td><td>10</td><td>20</td><td>503</td><td valign="bottom"><xref ref-type="bibr" rid="bib3">Beltz et al., 2003</xref></td></tr><tr><td><bold>Anomura</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>Birgus latro</italic> (20 cm)</td><td>1700</td><td>-</td><td>375</td><td>280</td><td>1338</td><td valign="bottom"><xref ref-type="bibr" rid="bib71">Krieger et al., 2010</xref></td></tr><tr><td><italic>Coenobita clypeatus</italic> (6 cm)</td><td>519</td><td>-</td><td>120</td><td>154</td><td>799</td><td valign="bottom"><xref ref-type="bibr" rid="bib3">Beltz et al., 2003</xref></td></tr><tr><td><italic>Pagurus bernhardus</italic> (3 cm)</td><td>673</td><td>-</td><td>-</td><td>171</td><td>536</td><td valign="bottom"><xref ref-type="bibr" rid="bib156">Tuchina et al., 2015</xref></td></tr><tr><td><bold>Brachyura</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td valign="top"><italic>Carcinus maenas</italic> (9 cm)</td><td valign="top">200</td><td valign="top">750</td><td valign="top">-</td><td valign="top">247</td><td valign="top">-</td><td valign="top"><xref ref-type="bibr" rid="bib40">Fontaine et al., 1982</xref>;&#160;<xref ref-type="bibr" rid="bib49">Hallberg and Skog, 2011</xref></td></tr></tbody></table><table-wrap-foot><fn><p>Estimates of the animal&#8217;s body lengths are given for comparison. Carapace width is given for <italic>B. latro</italic> and <italic>C. maenas</italic>, and total length is given for all other species.</p><p>* The palaemonid shrimp <italic>Palaemon elegans</italic> was investigated in the present study for comparison, as a species closely-related to <italic>R. exoculata</italic> among the Caridea family.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47550-v2.xml">elife-47550-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.47550.013</object-id><label>Table 1.</label><caption><title>Comparative table summarizing characteristics of aesthetascs and olfactory neuropils in several malacostracan species.</title></caption><table frame="hsides" rules="groups"><thead><tr><th/><th colspan="2">Aesthetascs</th><th colspan="3">Olfactory neuropils (ON)</th><th rowspan="2">References</th></tr><tr><th>Species (body length)</th><th>Total number</th><th>Length (&#181;m)</th><th>Neuropil total volume (x10^6 &#181;m3)</th><th>Mean glomerular volume (x10^3 &#181;m3)</th><th>Glomerular number</th></tr></thead><tbody><tr><td><bold>Leptostraca</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>Nebalia herbstii</italic> (1.4 cm)</td><td>-</td><td>-</td><td>0.1</td><td>2</td><td>60</td><td valign="bottom"><xref ref-type="bibr" rid="bib66">Kenning et al., 2013</xref></td></tr><tr><td><bold>Stomatopoda</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>Neogonodactylus oerstedii</italic> (4 cm)</td><td>80</td><td>400</td><td>-</td><td>110</td><td>70</td><td valign="bottom"><xref ref-type="bibr" rid="bib27">Derby et al., 2003</xref></td></tr><tr><td><bold>Isopoda</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td valign="top"><italic>Saduria entomon</italic> (8 cm)</td><td valign="top">40&#8211;60</td><td valign="top">240</td><td valign="top">3</td><td valign="top">34</td><td valign="top">80</td><td valign="top"><xref ref-type="bibr" rid="bib67">Kenning and Harzsch, 2013</xref>; <xref ref-type="bibr" rid="bib106">Pynn&#246;nen, 1985</xref></td></tr><tr><td><bold>Dendrobranchiata</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>Penaeus vannamei</italic> (7 cm)</td><td>280</td><td>-</td><td>-</td><td>-</td><td>&lt;100</td><td valign="bottom"><xref ref-type="bibr" rid="bib169">Wittfoth and Harzsch, 2018</xref>; <xref ref-type="bibr" rid="bib179">Zeng et al., 2002</xref></td></tr><tr><td><bold>Caridea</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>Palaemon elegans</italic> (7 cm)</td><td>280</td><td>230</td><td>120</td><td>225</td><td>530</td><td valign="bottom"><xref ref-type="bibr" rid="bib177">Zbinden et al., 2017</xref>; this study*</td></tr><tr><td><bold><italic>Rimicaris exoculata</italic> (6 cm)</bold></td><td><bold>206</bold></td><td><bold>170</bold></td><td><bold>56</bold></td><td><bold>155</bold></td><td><bold>370</bold></td><td valign="bottom"><xref ref-type="bibr" rid="bib177">Zbinden et al., 2017</xref>; this study</td></tr><tr><td><bold>Achelata</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td valign="top"><italic>Panulirus argus</italic> (20&#8211;60 cm)</td><td valign="top">3000</td><td valign="top">1000</td><td valign="top">154</td><td valign="top">118</td><td valign="top">1332</td><td valign="top"><xref ref-type="bibr" rid="bib3">Beltz et al., 2003</xref>; <xref ref-type="bibr" rid="bib47">Gr&#252;nert and Ache, 1988</xref></td></tr><tr><td><bold>Homarida</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td valign="top"><italic>Homarus americanus</italic> (20&#8211;60 cm)</td><td valign="top">2000</td><td valign="top">600</td><td valign="top">141</td><td valign="top">592</td><td valign="top">249</td><td valign="top"><xref ref-type="bibr" rid="bib3">Beltz et al., 2003</xref>; <xref ref-type="bibr" rid="bib48">Guenther and Atema, 1998</xref></td></tr><tr><td><bold>Astacida</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>Procambarus clarkii</italic> (9 cm)</td><td>133</td><td>-</td><td>10</td><td>20</td><td>503</td><td valign="bottom"><xref ref-type="bibr" rid="bib3">Beltz et al., 2003</xref></td></tr><tr><td><bold>Anomura</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td><italic>Birgus latro</italic> (20 cm)</td><td>1700</td><td>-</td><td>375</td><td>280</td><td>1338</td><td valign="bottom"><xref ref-type="bibr" rid="bib71">Krieger et al., 2010</xref></td></tr><tr><td><italic>Coenobita clypeatus</italic> (6 cm)</td><td>519</td><td>-</td><td>120</td><td>154</td><td>799</td><td valign="bottom"><xref ref-type="bibr" rid="bib3">Beltz et al., 2003</xref></td></tr><tr><td><italic>Pagurus bernhardus</italic> (3 cm)</td><td>673</td><td>-</td><td>-</td><td>171</td><td>536</td><td valign="bottom"><xref ref-type="bibr" rid="bib156">Tuchina et al., 2015</xref></td></tr><tr><td><bold>Brachyura</bold></td><td/><td/><td/><td/><td/><td valign="bottom"/></tr><tr><td valign="top"><italic>Carcinus maenas</italic> (9 cm)</td><td valign="top">200</td><td valign="top">750</td><td valign="top">-</td><td valign="top">247</td><td valign="top">-</td><td valign="top"><xref ref-type="bibr" rid="bib40">Fontaine et al., 1982</xref>;&#160;<xref ref-type="bibr" rid="bib49">Hallberg and Skog, 2011</xref></td></tr></tbody></table><table-wrap-foot><fn><p>Estimates of the animal&#8217;s body lengths are given for comparison. Carapace width is given for <italic>B. latro</italic> and <italic>C. maenas</italic>, and total length is given for all other species.</p><p>* The palaemonid shrimp <italic>Palaemon elegans</italic> was investigated in the present study for comparison, as a species closely-related to <italic>R. exoculata</italic> among the Caridea family.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47602-v2.xml">elife-47602-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table5" position="float"><object-id pub-id-type="doi">10.7554/eLife.47602.010</object-id><label>Table 5.</label><caption><title>Posterior rate ratio estimates and 95% Bayesian credible interval (BCI) for model 4 of mosquito biting rates.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Covariate</th><th align="center" colspan="3" valign="top">95% BCI Rate Ratio</th></tr><tr><th valign="top">Mean</th><th valign="top">2.5%</th><th valign="top">97.5%</th></tr></thead><tbody><tr><td valign="top">Population density</td><td valign="top">0.963</td><td valign="top">0.916</td><td valign="top">1.004</td></tr><tr><td valign="top">EVI</td><td valign="top">3.185</td><td valign="top">1.185</td><td valign="top">8.532</td></tr><tr><td valign="top">Distance to forest (100 m)</td><td valign="top">0.926</td><td valign="top">0.871</td><td valign="top">0.976</td></tr><tr><td valign="top">Spatial range (km)</td><td valign="top">3.120</td><td valign="top">0.514</td><td valign="top">6.926</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47654-v2.xml">elife-47654-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Diffusion rates and percent occupancy in RAS isoforms, and their HVRs.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Dataset <break/>HeLa</th><th colspan="3">Diffusion coefficient (&#181;m<sup>2</sup>/s)</th><th colspan="3">Occupancy (%)</th></tr><tr><th>D1</th><th>D2</th><th>D3</th><th>F1</th><th>F2</th><th>F3</th></tr></thead><tbody><tr><td>KRAS4b</td><td>0.95&#160;&#177;&#160;0.03</td><td>0.24&#160;&#177;&#160;0.03</td><td>0.06&#160;&#177;&#160;0.01</td><td>64&#160;&#177;&#160;2</td><td>22&#160;&#177;&#160;2</td><td>14&#160;&#177;&#160;3</td></tr><tr><td>KRAS4a</td><td>0.82&#160;&#177;&#160;0.04</td><td>0.26&#160;&#177;&#160;0.01</td><td>0.05&#160;&#177;&#160;0.01</td><td>67&#160;&#177;&#160;2</td><td>26&#160;&#177;&#160;3</td><td>7&#160;&#177;&#160;2</td></tr><tr><td>NRAS</td><td>0.84&#160;&#177;&#160;0.04</td><td>0.23&#160;&#177;&#160;0.08</td><td/><td>81&#160;&#177;&#160;2</td><td>19&#160;&#177;&#160;1</td><td/></tr><tr><td>HRAS</td><td>0.81&#160;&#177;&#160;0.02</td><td>0.1&#160;&#177;&#160;0.03</td><td/><td>81&#160;&#177;&#160;1</td><td>19&#160;&#177;&#160;1</td><td/></tr><tr><td>KRAS4b HVR</td><td>0.97&#160;&#177;&#160;0.03</td><td>0.1&#160;&#177;&#160;0.05</td><td/><td>87&#160;&#177;&#160;3</td><td>13&#160;&#177;&#160;3</td><td/></tr><tr><td>HRAS HVR</td><td>0.82&#160;&#177;&#160;0.06</td><td>0.1&#160;&#177;&#160;0.03</td><td/><td>87&#160;&#177;&#160;2</td><td>13&#160;&#177;&#160;2</td><td/></tr><tr><td valign="bottom">NRAS HVR</td><td>0.95&#160;&#177;&#160;0.08</td><td>0.15&#160;&#177;&#160;0.05</td><td/><td>86&#160;&#177;&#160;1</td><td>14&#160;&#177;&#160;1</td><td/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47654-v2.xml">elife-47654-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Diffusion rates and percent occupancy of KRAS4b in cancer cell lines</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Dataset <break/>KRAS4b</th><th colspan="3">Diffusion coefficient (&#181;m<sup>2</sup>/s)</th><th colspan="3">Occupancy (%)</th></tr><tr><th>D1</th><th>D2</th><th>D3</th><th>F1</th><th>F2</th><th>F3</th></tr></thead><tbody><tr><td>HeLa <break/>(KRAS4b WT)</td><td>0.95&#160;&#177;&#160;0.03</td><td>0.24&#160;&#177;&#160;0.03</td><td>0.06&#160;&#177;&#160;0.01</td><td>64&#160;&#177;&#160;2</td><td>22&#160;&#177;&#160;2</td><td>14&#160;&#177;&#160;3</td></tr><tr><td>MEF <break/>(KRAS4b WT)</td><td>0.73&#160;&#177;&#160;0.12</td><td>0.25&#160;&#177;&#160;0.09</td><td>0.05&#160;&#177;&#160;0.01</td><td>42&#160;&#177;&#160;9</td><td>30&#160;&#177;&#160;8</td><td>28&#160;&#177;&#160;2</td></tr><tr><td>PANC-1 <break/>(KRAS4b G12D)</td><td>0.84&#160;&#177;&#160;0.06</td><td>0.22&#160;&#177;&#160;0.01</td><td>0.04&#160;&#177;&#160;0.01</td><td>43&#160;&#177;&#160;6</td><td>40&#160;&#177;&#160;4</td><td>17&#160;&#177;&#160;2</td></tr><tr><td>SU.86.86 <break/>(KRAS4b G12D)</td><td>0.8&#160;&#177;&#160;0.16</td><td>0.18&#160;&#177;&#160;0.09</td><td>0.02&#160;&#177;&#160;0.01</td><td>66&#160;&#177;&#160;14</td><td>28&#160;&#177;&#160;9</td><td>6&#160;&#177;&#160;5</td></tr><tr><td valign="bottom">hTERT-HPNE <break/>(KRAS4b G12D)</td><td>0.92&#160;&#177;&#160;0.06</td><td>0.25&#160;&#177;&#160;0.02</td><td>0.06&#160;&#177;&#160;0.01</td><td>55&#160;&#177;&#160;3</td><td>29&#160;&#177;&#160;1</td><td>16&#160;&#177;&#160;3</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47654-v2.xml">elife-47654-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><label>Table 3.</label><caption><title>Diffusion rates and percent occupancy in full length KRAS4b with increasing concentrations of doxycycline in a dox-inducible HeLa cell pool</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Dataset <break/>HeLa</th><th colspan="3">Diffusion coefficient (&#181;m<sup>2</sup>/s)</th><th colspan="3">Occupancy (%)</th></tr><tr><th>D1</th><th>D2</th><th>D3</th><th>F1</th><th>F2</th><th>F3</th></tr></thead><tbody><tr><td>DOX 1 ng/mL</td><td>0.96&#160;&#177;&#160;0.04</td><td>0.33&#160;&#177;&#160;0.04</td><td>0.09&#160;&#177;&#160;0.01</td><td>71&#160;&#177;&#160;1</td><td>20&#160;&#177;&#160;2</td><td>9&#160;&#177;&#160;1</td></tr><tr><td>DOX 2 ng/mL</td><td>0.93&#160;&#177;&#160;0.02</td><td>0.26&#160;&#177;&#160;0.01</td><td>0.07&#160;&#177;&#160;0.002</td><td>68&#160;&#177;&#160;2</td><td>22&#160;&#177;&#160;2</td><td>10&#160;&#177;&#160;1</td></tr><tr><td>DOX 5 ng/mL</td><td>0.95&#160;&#177;&#160;0.04</td><td>0.27&#160;&#177;&#160;0.01</td><td>0.07&#160;&#177;&#160;0.01</td><td>66&#160;&#177;&#160;3</td><td>24&#160;&#177;&#160;1</td><td>10&#160;&#177;&#160;2</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47654-v2.xml">elife-47654-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><label>Table 4.</label><caption><title>Diffusion rates and percent occupancy in full length KRAS4b wildtype and with G-domain mutations ESR, GNK, and HEK</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Dataset <break/>HeLa</th><th colspan="3">Diffusion coefficient (&#181;m<sup>2</sup>/s)</th><th colspan="3">Occupancy (%)</th></tr><tr><th>D1</th><th>D2</th><th>D3</th><th>F1</th><th>F2</th><th>F3</th></tr></thead><tbody><tr><td>KRAS4b</td><td>0.84&#160;&#177;&#160;0.02</td><td>0.22&#160;&#177;&#160;0.01</td><td>0.05&#160;&#177;&#160;0.003</td><td>57&#160;&#177;&#160;1</td><td>27&#160;&#177;&#160;1</td><td>14&#160;&#177;&#160;1</td></tr><tr><td>KRAS4b ESR (D126E/T127S/K128R)</td><td>0.91&#160;&#177;&#160;0.04</td><td>0.25&#160;&#177;&#160;0.02</td><td>0.06&#160;&#177;&#160;0.01</td><td>63&#160;&#177;&#160;4</td><td>25&#160;&#177;&#160;2</td><td>12&#160;&#177;&#160;2</td></tr><tr><td>KRAS4b GNK (E91G/H94N/H95K)</td><td>0.85&#160;&#177;&#160;0.01</td><td>0.22&#160;&#177;&#160;0.01</td><td>0.05&#160;&#177;&#160;0.01</td><td>66&#160;&#177;&#160;1</td><td>22&#160;&#177;&#160;2</td><td>12&#160;&#177;&#160;1</td></tr><tr><td>KRAS4b HEK (Q131H/D132E/R135K)</td><td>0.90&#160;&#177;&#160;0.06</td><td>0.24&#160;&#177;&#160;0.01</td><td>0.06&#160;&#177;&#160;0.003</td><td>58&#160;&#177;&#160;5</td><td>28&#160;&#177;&#160;1</td><td>14&#160;&#177;&#160;4</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47654-v2.xml">elife-47654-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table5" position="float"><label>Table 5.</label><caption><title>Diffusion rates and percent occupancy in full length KRAS4b, the HVR, and HVR mutants in HeLa cells</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Dataset <break/>HeLa</th><th colspan="3">Diffusion coefficient (&#181;m<sup>2</sup>/s)</th><th colspan="3">Occupancy (%)</th></tr><tr><th>D1</th><th>D2</th><th>D3</th><th>F1</th><th>F2</th><th>F3</th></tr></thead><tbody><tr><td>KRAS4b</td><td>0.95&#160;&#177;&#160;0.06</td><td>0.24&#160;&#177;&#160;0.03</td><td>0.06&#160;&#177;&#160;0.01</td><td>66&#160;&#177;&#160;2</td><td>23&#160;&#177;&#160;2</td><td>14&#160;&#177;&#160;3</td></tr><tr><td>4bHVR</td><td>0.98&#160;&#177;&#160;0.03</td><td>0.17&#160;&#177;&#160;0.03</td><td/><td>87&#160;&#177;&#160;5</td><td>13&#160;&#177;&#160;5</td><td/></tr><tr><td>4bHVR-3A</td><td>1.08&#160;&#177;&#160;0.06</td><td>0.13&#160;&#177;&#160;0.04</td><td/><td>87&#160;&#177;&#160;6</td><td>13&#160;&#177;&#160;5</td><td/></tr><tr><td>4bHVR-5A</td><td>1.07&#160;&#177;&#160;0.04</td><td>0.15&#160;&#177;&#160;0.02</td><td/><td>82&#160;&#177;&#160;8</td><td>18&#160;&#177;&#160;6</td><td/></tr><tr><td>4bHVR-5Ea</td><td>1.21&#160;&#177;&#160;0.05</td><td>0.15&#160;&#177;&#160;0.05</td><td/><td>87&#160;&#177;&#160;3</td><td>13&#160;&#177;&#160;4</td><td/></tr><tr><td valign="bottom">4bHVR-5Eb</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td><td>N/A</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47654-v2.xml">elife-47654-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table6" position="float"><label>Table 6.</label><caption><title>Diffusion rates and percent occupancy in full length KRAS4b, KRAS4b Q61R, KRAS4b Y40C, and KRAS4b Y40C-Q61R in HeLa cells</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Dataset <break/>HeLa</th><th colspan="3">Diffusion coefficient (&#181;m<sup>2</sup>/s)</th><th colspan="3">Occupancy (%)</th></tr><tr><th>D1</th><th>D2</th><th>D3</th><th>F1</th><th>F2</th><th>F3</th></tr></thead><tbody><tr><td>KRAS4b Q61R</td><td>0.90&#160;&#177;&#160;0.07</td><td>0.25&#160;&#177;&#160;0.02</td><td>0.06&#160;&#177;&#160;0.01</td><td>51&#160;&#177;&#160;5</td><td>33&#160;&#177;&#160;3</td><td>16&#160;&#177;&#160;1</td></tr><tr><td>KRAS4b</td><td>0.88&#160;&#177;&#160;0.06</td><td>0.25&#160;&#177;&#160;0.02</td><td>0.08&#160;&#177;&#160;0.01</td><td>56&#160;&#177;&#160;2</td><td>28&#160;&#177;&#160;8</td><td>16&#160;&#177;&#160;1</td></tr><tr><td>KRAS4b Y40C</td><td>0.94&#160;&#177;&#160;0.07</td><td>0.27&#160;&#177;&#160;0.02</td><td>0.07&#160;&#177;&#160;0.01</td><td>66&#160;&#177;&#160;6</td><td>23&#160;&#177;&#160;5</td><td>12&#160;&#177;&#160;2</td></tr><tr><td>KRAS4b Y40C-Q61R</td><td>0.92&#160;&#177;&#160;0.05</td><td>0.31&#160;&#177;&#160;0.04</td><td>0.08&#160;&#177;&#160;0.02</td><td>62&#160;&#177;&#160;6</td><td>26&#160;&#177;&#160;4</td><td>11&#160;&#177;&#160;2</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47654-v2.xml">elife-47654-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table7" position="float"><label>Table 7.</label><caption><title>Diffusion rates and percent occupancy in Ras isoforms, and MEF serum starved cell recovery with serum complete media</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Dataset <break/>KRAS4b</th><th colspan="3">Diffusion coefficient (&#181;m<sup>2</sup>/s)</th><th colspan="3">Occupancy (%)</th></tr><tr><th>D1</th><th>D2</th><th>D3</th><th>F1</th><th>F2</th><th>F3</th></tr></thead><tbody><tr><td>MEF</td><td>0.73&#160;&#177;&#160;0.12</td><td>0.25&#160;&#177;&#160;0.09</td><td>0.05&#160;&#177;&#160;0.01</td><td>42&#160;&#177;&#160;3</td><td>30&#160;&#177;&#160;3</td><td>28&#160;&#177;&#160;4</td></tr><tr><td>MEF (srm 0&#8217;)</td><td>0.72&#160;&#177;&#160;0.02</td><td>0.22&#160;&#177;&#160;0.02</td><td>0.07&#160;&#177;&#160;0.01</td><td>50&#160;&#177;&#160;3</td><td>27&#160;&#177;&#160;0.8</td><td>23&#160;&#177;&#160;3</td></tr><tr><td>MEF (srm 15&#8217;)</td><td>0.77&#160;&#177;&#160;0.04</td><td>0.22&#160;&#177;&#160;0.02</td><td>0.06&#160;&#177;&#160;0.01</td><td>47&#160;&#177;&#160;3</td><td>28&#160;&#177;&#160;3</td><td>25&#160;&#177;&#160;2</td></tr><tr><td>MEF (srm 60&#8217;)</td><td>0.85&#160;&#177;&#160;0.03</td><td>0.21&#160;&#177;&#160;0.02</td><td>0.06&#160;&#177;&#160;0.01</td><td>44&#160;&#177;&#160;4</td><td>29&#160;&#177;&#160;3</td><td>27&#160;&#177;&#160;2</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47701-v1.xml">elife-47701-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.47701.015</object-id><label>Table 3.</label><caption><title>Mean valences calculated in the K<sup>+</sup>-binding site of H<sup>+</sup>,K<sup>+</sup>-ATPase, assuming protonation for the acidic residues as evaluated by MD simulations.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="8" valign="top">Tyr799Trp</th><th valign="top">WT</th></tr><tr><th colspan="3" valign="top">2-proton states</th><th colspan="3" valign="top">3-proton states</th><th colspan="2" valign="top">4-proton states</th><th valign="top"/></tr></thead><tbody><tr><td valign="top">E343</td><td valign="top">H<sup>+</sup></td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">H<sup>+</sup></td><td valign="top">H<sup>+</sup></td><td valign="top">&#8211;</td><td valign="top">H<sup>+</sup></td><td valign="top">H<sup>+</sup></td><td valign="top">H<sup>+</sup></td></tr><tr><td valign="top">E795</td><td valign="top">&#8211;</td><td valign="top">H<sup>+</sup></td><td valign="top">&#8211;</td><td valign="top">H<sup>+</sup></td><td valign="top">&#8211;</td><td valign="top">H<sup>+</sup></td><td valign="top">H<sup>+</sup></td><td valign="top">&#8211;</td><td valign="top">H<sup>+</sup></td></tr><tr><td valign="top">E820</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">H<sup>+</sup></td><td valign="top">&#8211;</td><td valign="top">H<sup>+</sup></td><td valign="top">H<sup>+</sup></td><td valign="top">&#8211;</td><td valign="top">H<sup>+</sup></td><td valign="top">&#8211;</td></tr><tr><td valign="top">D824</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">H<sup>+</sup></td><td valign="top">H<sup>+</sup></td><td valign="top">&#8211;</td></tr><tr><td valign="top">E936</td><td valign="top">H<sup>+</sup></td><td valign="top">H<sup>+</sup></td><td valign="top">H<sup>+</sup></td><td valign="top">H<sup>+</sup></td><td valign="top">H<sup>+</sup></td><td valign="top">H<sup>+</sup></td><td valign="top">H<sup>+</sup></td><td valign="top">&#8211;</td><td valign="top">H<sup>+</sup></td></tr><tr><td valign="top">D942</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">&#8211;</td><td valign="top">H<sup>+</sup></td><td valign="top">&#8211;</td></tr><tr><td valign="top"/><td colspan="9" valign="top">Valence<sup>&#8224;</sup> (Mean&#160;&#177;&#160;SEM)</td></tr><tr><td valign="top">V338 O</td><td valign="top">0.077&#160;&#177;&#160;0.001</td><td valign="top">0.239&#160;&#177;&#160;0.004</td><td valign="top">0.282&#160;&#177;&#160;0.002</td><td valign="top">0.042&#160;&#177;&#160;0.002</td><td valign="top">0.116&#160;&#177;&#160;0.004</td><td valign="top">0.268&#160;&#177;&#160;0.001</td><td valign="top">0.111&#160;&#177;&#160;0.003</td><td valign="top">0.217&#160;&#177;&#160;0.001</td><td valign="top">0.074&#160;&#177;&#160;0.003</td></tr><tr><td valign="top">A339 O</td><td valign="top">0.132&#160;&#177;&#160;0.001</td><td valign="top">0.098&#160;&#177;&#160;0.002</td><td valign="top">0.005&#160;&#177;&#160;0.000</td><td valign="top">0.171&#160;&#177;&#160;0.004</td><td valign="top">0.235&#160;&#177;&#160;0.002</td><td valign="top">0.128&#160;&#177;&#160;0.001</td><td valign="top">0.203&#160;&#177;&#160;0.003</td><td valign="top">0.248&#160;&#177;&#160;0.001</td><td valign="top">0.190&#160;&#177;&#160;0.003</td></tr><tr><td valign="top">V341 O</td><td valign="top">0.242&#160;&#177;&#160;0.001</td><td valign="top">0.278&#160;&#177;&#160;0.005</td><td valign="top">0.224&#160;&#177;&#160;0.001</td><td valign="top">0.239&#160;&#177;&#160;0.003</td><td valign="top">0.269&#160;&#177;&#160;0.001</td><td valign="top">0.283&#160;&#177;&#160;0.002</td><td valign="top">0.295&#160;&#177;&#160;0.003</td><td valign="top">0.29&#160;&#177;&#160;0.001</td><td valign="top">0.283&#160;&#177;&#160;0.003</td></tr><tr><td valign="top">E343 O&#949;1</td><td valign="top">0.21&#160;&#177;&#160;0.001</td><td valign="top">0.169&#160;&#177;&#160;0.004</td><td valign="top">0.257&#160;&#177;&#160;0.002</td><td valign="top">0.046&#160;&#177;&#160;0.002</td><td valign="top">&#8211;</td><td valign="top">0.26&#160;&#177;&#160;0.002</td><td valign="top">0.068&#160;&#177;&#160;0.003</td><td valign="top">&#8211;</td><td valign="top">0.091&#160;&#177;&#160;0.001</td></tr><tr><td valign="top">E343 O&#949;2</td><td valign="top">0.005&#160;&#177;&#160;0.000</td><td valign="top">0.152&#160;&#177;&#160;0.003</td><td valign="top">0.284&#160;&#177;&#160;0.002</td><td valign="top">0.002&#160;&#177;&#160;0.000</td><td valign="top">&#8211;</td><td valign="top">0.206&#160;&#177;&#160;0.002</td><td valign="top">0.002&#160;&#177;&#160;0.000</td><td valign="top">&#8211;</td><td valign="top">0.003&#160;&#177;&#160;0.001</td></tr><tr><td valign="top">E795 O&#949;1</td><td valign="top">0.278&#160;&#177;&#160;0.003</td><td valign="top">0.121&#160;&#177;&#160;0.002</td><td valign="top">0.291&#160;&#177;&#160;0.002</td><td valign="top">0.267&#160;&#177;&#160;0.003</td><td valign="top">0.315&#160;&#177;&#160;0.002</td><td valign="top">0.107&#160;&#177;&#160;0.001</td><td valign="top">0.277&#160;&#177;&#160;0.003</td><td valign="top">0.239&#160;&#177;&#160;0.003</td><td valign="top">0.236&#160;&#177;&#160;0.003</td></tr><tr><td valign="top">E795 O&#949;2</td><td valign="top">0.134&#160;&#177;&#160;0.003</td><td valign="top"><sup>&#8211;&#8225;</sup></td><td valign="top">0.272&#160;&#177;&#160;0.002</td><td valign="top">&#8211;</td><td valign="top">0.053&#160;&#177;&#160;0.002</td><td valign="top">0.001&#160;&#177;&#160;0.000</td><td valign="top">&#8211;</td><td valign="top">0.239&#160;&#177;&#160;0.003</td><td valign="top">0.002&#160;&#177;&#160;0.000</td></tr><tr><td valign="top">E820 O&#949;1</td><td valign="top">0.200&#160;&#177;&#160;0.002</td><td valign="top">0.017&#160;&#177;&#160;0.001</td><td valign="top">0.001&#160;&#177;&#160;0.000</td><td valign="top">0.015&#160;&#177;&#160;0.000</td><td valign="top">0.061&#160;&#177;&#160;0.001</td><td valign="top">0.001&#160;&#177;&#160;0.000</td><td valign="top">0.138&#160;&#177;&#160;0.005</td><td valign="top">0.155&#160;&#177;&#160;0.001</td><td valign="top">0.018&#160;&#177;&#160;0.000</td></tr><tr><td valign="top">E820 O&#949;2</td><td valign="top">0.109&#160;&#177;&#160;0.002</td><td valign="top">0.195&#160;&#177;&#160;0.004</td><td valign="top">0.010&#160;&#177;&#160;0.000</td><td valign="top">0.291&#160;&#177;&#160;0.006</td><td valign="top">0.089&#160;&#177;&#160;0.001</td><td valign="top">0.028&#160;&#177;&#160;0.000</td><td valign="top">0.118&#160;&#177;&#160;0.005</td><td valign="top">0.066&#160;&#177;&#160;0.001</td><td valign="top">0.273&#160;&#177;&#160;0.003</td></tr><tr><td valign="top">total</td><td valign="top">1.387</td><td valign="top">1.269</td><td valign="top">1.626</td><td valign="top">1.073<sup>&#182;</sup></td><td valign="top">1.138</td><td valign="top">1.282</td><td valign="top">1.212</td><td valign="top">1.454</td><td valign="top">1.171</td></tr></tbody></table><table-wrap-foot><fn><p><sup>&#8224;</sup>Ideal value for K<sup>+</sup> is 1.00. <sup>&#8225;</sup>Only oxygen atoms within 4&#197; of bound K<sup>+</sup> were included for the valence calculation. <sup>&#182;</sup>The most likely protonation state of E343p/E795p/E936p shows a total valence closest to the ideal value amongst all simulations evaluated.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47804-v1.xml">elife-47804-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.47804.011</object-id><label>Table 1.</label><caption><title>Rate constants resulting from global fit of experimental ESI-MS data using the model shown in <xref ref-type="fig" rid="fig5">Figure 5</xref>.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Reaction step</th><th colspan="2">Rate constant (min<sup>&#8722;1</sup>)<sup>*,&#8224;</sup> </th><th rowspan="2">Reaction step</th></tr><tr><th valign="top">Anaerobic (-O<sub>2</sub>)</th><th valign="top">Aerobic (+O<sub>2</sub>)</th></tr></thead><tbody><tr><td valign="top"><italic>k<sub>1</sub></italic></td><td valign="top">0.300&#160;&#177;&#160;0.010</td><td valign="top">0.320&#160;&#177;&#160;0.020</td><td valign="top">[4Fe-4S] &#8594; [3Fe-4S] + Fe</td></tr><tr><td valign="top"><italic>k<sub>-1</sub></italic></td><td valign="top">4.67&#160;&#177;&#160;0.33&#160;&#215;&#160;10<sup>3</sup></td><td valign="top">4.67&#160;&#177;&#160;0.43&#160;&#215;&#160;10<sup>3</sup></td><td valign="top">[3Fe-4S] + Fe &#8594; [4Fe-4S]</td></tr><tr><td valign="top"><italic>k<sub>2</sub></italic></td><td valign="top">0.090&#160;&#177;&#160;0.002</td><td valign="top">0.230&#160;&#177;&#160;0.010</td><td valign="top">[3Fe-4S] &#8594; [3Fe-3S]</td></tr><tr><td valign="top"><italic>k<sub>3</sub></italic></td><td valign="top">0.500&#160;&#177;&#160;0.010</td><td valign="top">1.200&#160;&#177;&#160;0.050</td><td valign="top">[3Fe-3S] &#8594; [2Fe-2S]</td></tr><tr><td valign="top"><italic>k<sub>4</sub></italic></td><td valign="top">0.070&#160;&#177;&#160;0.001</td><td valign="top">0.200&#160;&#177;&#160;0.005</td><td valign="top">[2Fe-2S] &#8594; apo</td></tr><tr><td valign="top"><italic>k<sub>5</sub></italic></td><td valign="top">0.008&#160;&#177;&#160;0.002</td><td valign="top">0.007&#160;&#177;&#160;0.002</td><td valign="top">[4Fe-4S] &#8594; [4Fe-3S]</td></tr><tr><td valign="top"><italic>k<sub>6</sub></italic></td><td valign="top">0.087&#160;&#177;&#160;0.003</td><td valign="top">0.087&#160;&#177;&#160;0.008</td><td valign="top">[4Fe-3S] &#8594; [3Fe-3S]</td></tr><tr><td valign="top"><italic>k<sub>7</sub></italic></td><td valign="top">0.083&#160;&#177;&#160;0.001</td><td valign="top">0.150&#160;&#177;&#160;0.008</td><td valign="top">[4Fe-3S] &#8594; [3Fe-2S]</td></tr><tr><td valign="top"><italic>k<sub>8</sub></italic></td><td valign="top">0.030&#160;&#177;&#160;0.001</td><td valign="top">0.026&#160;&#177;&#160;0.020</td><td valign="top">[3Fe-3S] &#8594; [3Fe-2S]</td></tr><tr><td valign="top"><italic>k<sub>9</sub></italic></td><td valign="top">0.044&#160;&#177;&#160;0.004</td><td valign="top">0.140&#160;&#177;&#160;0.002</td><td valign="top">[3Fe-2S] &#8594; [2Fe-2S]</td></tr><tr><td valign="top"><italic>k<sub>10</sub></italic></td><td valign="top">0.160&#160;&#177;&#160;0.004</td><td valign="top">0.300&#160;&#177;&#160;0.010</td><td valign="top">[3Fe-2S] &#8594; apo</td></tr></tbody></table><table-wrap-foot><fn><p><sup>*</sup>With the exception of <italic>k</italic><sub>-1</sub>, which is a second order rate constant with units of M<sup>&#8722;1</sup> min<sup>&#8722;1</sup>.</p><p><sup>&#8224;</sup>Standard errors are indicated.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47859-v3.xml">elife-47859-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="resptable1" position="float"><label>Author response table 1.</label><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="2">Mean Telomere Length</th></tr><tr><th valign="top">MspI-<italic>digested (Figure 1D)</italic></th><th valign="top">HinfI-<italic>digested (Figure 5D)</italic></th></tr></thead><tbody><tr><td valign="top">pR</td><td valign="top">Left marker &#8211; 7.2 Kb</td><td valign="top">Left marker &#8211; 7.00 Kb</td></tr><tr><td valign="top">cR7</td><td valign="top">Left marker &#8211; 6.4 Kb</td><td valign="top">Left marker &#8211; 6.9 Kb</td></tr><tr><td valign="top">cR35</td><td valign="top">Right marker &#8211; 6.6 Kb</td><td valign="top">Left marker &#8211; 6.7 Kb</td></tr><tr><td valign="top">WT1</td><td valign="top">Right marker &#8211; 10.0 Kb</td><td valign="top">Right marker &#8211; 10.4 Kb</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-47864-v2.xml">elife-47864-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.47864.025</object-id><label>Table 1.</label><caption><title>FAMA complex candidates from <xref ref-type="fig" rid="fig5">Figure 5</xref>.</title></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="3" valign="top">Enriched at time point</th><th rowspan="2">AGI</th><th rowspan="2">Gene name</th><th rowspan="2">Functional annotation</th><th rowspan="2">Subcellular localization</th></tr><tr><th valign="top">0 h</th><th valign="top">0.5 h</th><th valign="top">3 h</th></tr></thead><tbody><tr><td valign="top"/><td valign="top">Y</td><td valign="top">Y</td><td>AT3G26744</td><td valign="top">ICE1, SCRM</td><td valign="top">bHLH transcription factor</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top">Y</td><td valign="top">Y</td><td>AT2G46510</td><td valign="top">AIB, JAM1</td><td valign="top">bHLH transcription factor</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT4G16430</td><td valign="top">JAM3</td><td valign="top">bHLH transcription factor</td><td valign="top">N, C</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT5G08130</td><td valign="top">BIM1</td><td valign="top">bHLH transcription factor</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT1G75080</td><td valign="top">BZR1</td><td valign="top">Transcription factor</td><td valign="top">N, C</td></tr><tr><td valign="top"/><td valign="top">Y</td><td valign="top">Y</td><td>AT5G11060</td><td valign="top">KNAT4</td><td valign="top">Homeobox transcription factor</td><td valign="top">N, C</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT2G41900</td><td valign="top">OXS2, TCF7</td><td valign="top">Zinc finger transcription factor</td><td valign="top">(N), C</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT1G79000</td><td valign="top">HAC1, PCAT2</td><td valign="top">Transcriptional co-activator (histone acetyltransferase)</td><td valign="top">N, C</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT4G04920</td><td valign="top">MED16, SFR6</td><td valign="top">Transcriptional co-activator (mediator complex)</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT1G43850</td><td valign="top">SEU</td><td valign="top">Transcriptional co-repressor adapter</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT4G32551</td><td valign="top">LUG, RON2</td><td valign="top">Transcriptional co-repressor</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT2G32700</td><td valign="top">LUH, MUM1</td><td valign="top">Transcriptional co-repressor</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT3G15880</td><td valign="top">TPR4, WSIP2</td><td valign="top">Transcriptional co-repressor</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT5G27030</td><td valign="top">TPR3</td><td valign="top">Transcriptional co-repressor</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT5G02500</td><td valign="top">HSP70-1</td><td valign="top">HSP70 chaperone</td><td valign="top">N, C</td></tr><tr><td valign="top"/><td valign="top">Y</td><td valign="top">Y</td><td>AT5G02490</td><td valign="top">HSP70-2</td><td valign="top">HSP70 chaperone</td><td valign="top">N, C</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT3G09440</td><td valign="top">HSP70-3</td><td valign="top">HSP70 chaperone</td><td valign="top">N, C</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT3G12580</td><td valign="top">HSP70-4</td><td valign="top">HSP70 chaperone</td><td valign="top">N, C</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT5G22060</td><td valign="top">J2</td><td valign="top">HSP70 co-chaperone</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT3G44110</td><td valign="top">J3</td><td valign="top">HSP70 co-chaperone</td><td valign="top">N, C, MA</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT1G62740</td><td valign="top">HOP2</td><td valign="top">HSP90/70 co-chaperone</td><td valign="top">(N), C</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT3G25230</td><td valign="top">FKBP62, ROF1</td><td valign="top">HSP90/70 co-chaperone</td><td valign="top">(N), C</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT4G22670</td><td valign="top">HIP1, TPR11</td><td valign="top">HSP90/70 co-chaperone</td><td valign="top">N, C</td></tr><tr><td valign="top">&#8195;Y</td><td valign="top">Y</td><td valign="top">Y</td><td>AT4G02450</td><td valign="top">P23-1</td><td valign="top">HSP90 co-chaperone</td><td valign="top">N, C</td></tr><tr><td valign="top">&#8195;Y</td><td valign="top">Y</td><td valign="top">Y</td><td>AT5G56460</td><td valign="top"/><td valign="top">Putative protein kinase</td><td valign="top">PM</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT5G35410</td><td valign="top">SOS2, CIPK24</td><td valign="top">Protein kinase</td><td valign="top">N, C, PM</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT3G54170</td><td valign="top">FIP37</td><td valign="top">m6A methyltransferase complex component</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT1G02140</td><td valign="top">HAP1, MAGO</td><td valign="top">Exon-junction complex component</td><td valign="top">N, C</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT5G41880</td><td valign="top">POLA3, POLA4</td><td valign="top">Putative DNA polymerase alpha subunit</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT3G22380</td><td valign="top">TIC</td><td valign="top">Nuclear clock regulation factor</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT2G41100</td><td valign="top">TCH3, CAL12</td><td valign="top">Calcium-binding protein</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT1G72390</td><td valign="top">PHL</td><td valign="top">Nuclear receptor/co-activator</td><td valign="top">N, C</td></tr><tr><td valign="top"/><td valign="top">Y</td><td valign="top">Y</td><td>AT1G20110</td><td valign="top">FREE1, FYVE1</td><td valign="top">ESCRT-I complex component</td><td valign="top">C, ES, N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT1G18660</td><td valign="top">IAP1</td><td valign="top">C3HC4-type RING-finger domain protein</td><td valign="top">MA, N</td></tr><tr><td valign="top">&#8195;Y</td><td valign="top">Y</td><td valign="top">Y</td><td>AT1G12200</td><td valign="top">FMO</td><td valign="top">Putative flavin monooxygenase</td><td valign="top">N/A</td></tr><tr><td valign="top"/><td valign="top">Y</td><td valign="top">Y</td><td>AT3G53260</td><td valign="top">PAL2</td><td valign="top">Phenylalanine ammonia-lyase</td><td valign="top">N, C, EX</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT3G23840</td><td valign="top">CER26-LIKE</td><td valign="top">acyl&#8208;CoA&#8208;dependent acyltransferase</td><td valign="top">N/A</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT5G13710</td><td valign="top">CPH, SMT1</td><td valign="top">C&#8208;24 sterol methyl transferase</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT1G63180</td><td valign="top">UGE3</td><td valign="top">UDP-Glucose 4-Epimerase</td><td valign="top">C*</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT5G17990</td><td valign="top">PAT1, TRP1</td><td valign="top">Phosphoribosylanthranilate transferase</td><td valign="top">CP*</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT1G15980</td><td valign="top">NDH48, NDF1</td><td valign="top">Chloroplast NAD(P)H dehydrogenase complex subunit</td><td valign="top">CP</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT4G30720</td><td valign="top">PDE327</td><td valign="top">Putative oxidoreductase/electron carrier</td><td valign="top">CP</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT1G50570</td><td valign="top"/><td valign="top">Undescribed protein</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT1G30070</td><td valign="top"/><td valign="top">Undescribed protein</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT5G15680</td><td valign="top"/><td valign="top">Undescribed protein</td><td valign="top">N/A</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT5G53330</td><td valign="top"/><td valign="top">Undescribed protein</td><td valign="top">N</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">Y</td><td>AT4G25290</td><td valign="top"/><td valign="top">Undescribed protein</td><td valign="top">N/A</td></tr></tbody></table><table-wrap-foot><fn><p>Column labels: Enriched at time point: time points at which a protein was significantly enriched are marked with Y. AGI: Arabidopsis gene identifier. Subcellular localization: as described for fluorescent protein fusions in literature unless marked with * (localization inferred from functional annotation): N, nucleus; (N), nucleus under heat or other stress; C, cytosol; EX, extracellular; PM, plasma membrane; ES, endosomes; MA, membrane (associated); CP, chloroplast; N/A, localization unknown (no experimental evidence found and localization cannot be clearly inferred from function).</p><p>For further information on the candidate proteins and selected references see <xref ref-type="supplementary-material" rid="supp2">Supplementary file 2</xref> &#8211; Table 4.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-48119-v2.xml">elife-48119-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.48119.007</object-id><label>Table 1.</label><caption><title>Summary of major <italic>nurf-1</italic> transcripts identified in <italic>C. elegans.</italic></title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><italic>Name</italic></th><th colspan="2" valign="top"><italic>Evidence</italic></th><th colspan="2" valign="top"><italic>Size</italic></th><th rowspan="2"><italic>Conserved<sup>c</sup></italic></th><th rowspan="2"><italic>Predicted biological role in</italic> C. elegans<italic><sup>d</sup></italic></th><th rowspan="2"><italic>Other names</italic></th></tr><tr><th valign="top">Transcript<sup>a</sup></th><th valign="top">Protein<sup>b</sup></th><th valign="top">aa</th><th valign="top">kD</th></tr></thead><tbody><tr><td valign="top"><italic>nurf-1.a</italic></td><td valign="top">N</td><td valign="top">-</td><td valign="top">2197</td><td valign="top">252</td><td valign="top">M,D</td><td valign="top">None</td><td valign="top">Full-length</td></tr><tr><td valign="top"> <italic>nurf-1.b</italic></td><td valign="top">C,N,I</td><td valign="top">W</td><td valign="top">1621</td><td valign="top">186</td><td valign="top">D</td><td valign="top">Reproduction, vulval development</td><td valign="top">N-terminal or NURF-1.A</td></tr><tr><td valign="top"><italic>nurf-1.d</italic></td><td valign="top">C,N,I</td><td valign="top">W</td><td valign="top">816</td><td valign="top">92</td><td valign="top">-</td><td valign="top">Size, dauer, reproduction, axon guidance</td><td valign="top">C-terminal or NURF-1.C</td></tr><tr><td valign="top"><italic>nurf-1.f</italic></td><td valign="top">C,N,I</td><td valign="top">W</td><td valign="top">581</td><td valign="top">58</td><td valign="top">-</td><td valign="top">None</td><td valign="top">NURF-1.E</td></tr><tr><td valign="top"><italic>nurf-1.q</italic></td><td valign="top">N,I</td><td valign="top">-</td><td valign="top">243</td><td valign="top">36</td><td valign="top">-</td><td valign="top">None</td><td valign="top">-</td></tr></tbody></table><table-wrap-foot><fn><p><sup>a</sup> C indicates full-length cDNA have been isolated for this transcript, N indicates evidence from direct sequencing of RNA or cDNA using Oxford Nanopore reads support this transcript, and I indicates evidence from Illumina short read RNA-seq supports this transcript</p><p><sup>b</sup> W indicates evidence for the protein isoform was obtained using western blot</p></fn><fn><p><sup>c</sup> M or D indicates an analogous isoform is described in mammals (mice or humans) or <italic>Drosophila</italic>, respectively</p><p><sup>d</sup>Predictions from <xref ref-type="bibr" rid="bib2">Andersen et al. (2006)</xref>, <xref ref-type="bibr" rid="bib54">Large et al. (2016)</xref>, or <xref ref-type="bibr" rid="bib62">Mariani et al. (2016)</xref></p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-48460-v1.xml">elife-48460-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><label>Table 3.</label><caption><title>Detection of IgG by MIA against ZIKV in the paired samples from participants aged under and over 16 years recruited during October-November 2015 and June 2017 in the Central division in Fiji (n&#160;=&#160;189).</title><p>Age groups are defined using age of participants when recruited to the study in 2013.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">2015</th><th align="center" colspan="6">2017</th></tr><tr><th colspan="2">&#8804;16 years</th><th colspan="2">&gt;16 years</th><th colspan="2">Total participants</th></tr><tr><th>ZIKV+</th><th>ZIKV-</th><th>ZIKV+</th><th>ZIKV-</th><th>ZIKV+</th><th>ZIKV-</th></tr></thead><tbody><tr><td>&#8804;16 years</td><td/><td/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td>ZIKV+</td><td>10</td><td>5</td><td valign="top">&#8210;</td><td valign="top">&#8210;</td><td valign="top">&#8210;</td><td valign="top">&#8210;</td></tr><tr><td>ZIKV-</td><td>4</td><td>48</td><td valign="top">&#8210;</td><td valign="top">&#8210;</td><td valign="top">&#8210;</td><td valign="top">&#8210;</td></tr><tr><td>&gt;16 years</td><td/><td/><td/><td/><td/><td/></tr><tr><td>ZIKV+</td><td valign="top">&#8210;</td><td valign="top">&#8210;</td><td>7</td><td>23</td><td valign="top">&#8210;</td><td valign="top">&#8210;</td></tr><tr><td>ZIKV-</td><td valign="top">&#8210;</td><td valign="top">&#8210;</td><td>2</td><td>90</td><td valign="top">&#8210;</td><td valign="top">&#8210;</td></tr><tr><td>Total Participants</td><td/><td/><td/><td/><td/><td/></tr><tr><td>ZIKV+</td><td>&#8210;</td><td>&#8210;</td><td>&#8210;</td><td>&#8210;</td><td>17</td><td>28</td></tr><tr><td>ZIKV-</td><td>&#8210;</td><td>&#8210;</td><td>&#8210;</td><td>&#8210;</td><td>6</td><td>138</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-48968-v2.xml">elife-48968-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.48968.024</object-id><label>Table 3.</label><caption><title>SEQRS enrichment for specific sequence elements.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Protein</th><th rowspan="2">RNA&#160;sequence</th><th rowspan="2">Base +4</th><th rowspan="2">Terminal AU position</th><th rowspan="2">Count</th><th>Ratio compact/</th></tr><tr><th>extended</th></tr></thead><tbody><tr><td/><td>87654 321 repeatCUGUGA AUG (8mer)CUGUGCCAUA (9mer)</td><td/><td/><td/><td/></tr><tr><td>FBF-2</td><td>CTGTA..AT</td><td>A</td><td>+8U</td><td>119374</td><td>0.21</td></tr><tr><td>FBF-2</td><td>CTGTA. AT</td><td>A</td><td>+7U</td><td>24819</td><td/></tr><tr><td>FBF-2</td><td>CTGTG..AT</td><td>G</td><td>+8U</td><td>1970</td><td>2.8</td></tr><tr><td>FBF-2</td><td>CTGTG. AT</td><td>G</td><td>+7U</td><td>5506</td><td/></tr><tr><td>Complex</td><td>CTGTA..AT</td><td>A</td><td>+8U</td><td>170</td><td>0.7</td></tr><tr><td>Complex</td><td>CTGTA. AT</td><td>A</td><td>+7U</td><td>118</td><td/></tr><tr><td>Complex</td><td>CTGTG..AT</td><td>G</td><td>+8U</td><td>113</td><td>1.1</td></tr><tr><td>Complex</td><td>CTGTG. AT</td><td>G</td><td>+7U</td><td>126</td><td/></tr><tr><td>CLIP</td><td>CTGTA..AT</td><td>A</td><td>+8U</td><td>266</td><td valign="bottom">0.44</td></tr><tr><td>CLIP</td><td>CTGTA. AT</td><td>A</td><td>+7U</td><td>117</td><td valign="bottom"/></tr><tr><td>CLIP</td><td>CTGTG..AT</td><td>G</td><td>+8U</td><td>92</td><td valign="bottom">1.1</td></tr><tr><td>CLIP</td><td>CTGTG. AT</td><td>G</td><td>+7U</td><td>102</td><td valign="bottom"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-48968-v3.xml">elife-48968-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><object-id pub-id-type="doi">10.7554/eLife.48968.024</object-id><label>Table 3.</label><caption><title>SEQRS enrichment for specific sequence elements.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Protein</th><th rowspan="2"><monospace>Pattern</monospace></th><th rowspan="2">Base +4</th><th rowspan="2">Terminal AU position</th><th rowspan="2">Count</th><th>Ratio compact/</th></tr><tr><th>extended</th></tr></thead><tbody><tr><td/><td><monospace>&#160;87654&#160;321&#160;repeat</monospace><break/>
<monospace>CUGUGA&#160;AUG&#160;(8mer)</monospace><break/>
<monospace>CUGUGCCAUA&#160;(9mer)</monospace></td><td/><td/><td/><td/></tr><tr><td>FBF-2</td><td><monospace>CTGTA..AT</monospace></td><td>A</td><td>+8U</td><td>119374</td><td>0.21</td></tr><tr><td>FBF-2</td><td><monospace>CTGTA.&#160;AT</monospace></td><td>A</td><td>+7U</td><td>24819</td><td/></tr><tr><td>FBF-2</td><td><monospace>CTGTG..AT</monospace></td><td>G</td><td>+8U</td><td>1970</td><td>2.8</td></tr><tr><td>FBF-2</td><td><monospace>CTGTG.&#160;AT</monospace></td><td>G</td><td>+7U</td><td>5506</td><td/></tr><tr><td>Complex</td><td><monospace>CTGTA..AT</monospace></td><td>A</td><td>+8U</td><td>170</td><td>0.7</td></tr><tr><td>Complex</td><td><monospace>CTGTA.&#160;AT</monospace></td><td>A</td><td>+7U</td><td>118</td><td/></tr><tr><td>Complex</td><td><monospace>CTGTG..AT</monospace></td><td>G</td><td>+8U</td><td>113</td><td>1.1</td></tr><tr><td>Complex</td><td><monospace>CTGTG.&#160;AT</monospace></td><td>G</td><td>+7U</td><td>126</td><td/></tr><tr><td>CLIP</td><td><monospace>CTGTA..AT</monospace></td><td>A</td><td>+8U</td><td>266</td><td valign="bottom">0.44</td></tr><tr><td>CLIP</td><td><monospace>CTGTA.&#160;AT</monospace></td><td>A</td><td>+7U</td><td>117</td><td valign="bottom"/></tr><tr><td>CLIP</td><td><monospace>CTGTG..AT</monospace></td><td>G</td><td>+8U</td><td>92</td><td valign="bottom">1.1</td></tr><tr><td>CLIP</td><td><monospace>CTGTG.&#160;AT</monospace></td><td>G</td><td>+7U</td><td>102</td><td valign="bottom"/></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-48971-v1.xml">elife-48971-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.48971.006</object-id><label>Table 1.</label><caption><title>Major brain activity at the source level.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Brain region</th><th colspan="3">Peak MNI coordinate</th><th rowspan="2"><italic>T</italic> value</th></tr><tr><th align="right"><italic>X</italic></th><th align="right"><italic>Y</italic></th><th align="right"><italic>Z</italic></th></tr></thead><tbody><tr><th colspan="5" valign="top"><bold>Imagery condition at 0.8 Hz</bold></th></tr><tr><td>R postcentral gyrus</td><td align="right">56</td><td align="right">&#8722;22</td><td align="right">29</td><td align="right">4.0</td></tr><tr><td>L postcentral gyrus</td><td align="right">&#8722;51</td><td align="right">-7</td><td align="right">23</td><td align="right">4.9</td></tr><tr><td>L precentral gyrus</td><td align="right">&#8722;50</td><td align="right">-6</td><td align="right">21</td><td align="right">4.7</td></tr><tr><td>L inferior frontal gyrus</td><td align="right">&#8722;46</td><td align="right">3</td><td align="right">28</td><td align="right">3.2</td></tr><tr><td>L inferior occipital gyrus</td><td align="right">&#8722;40</td><td align="right">&#8722;75</td><td align="right">-6</td><td align="right">5.3</td></tr><tr><td>R supramarginal gyrus</td><td align="right">60</td><td align="right">&#8722;31</td><td align="right">24</td><td align="right">6.0</td></tr><tr><th colspan="5" valign="top">Imagery condition at 4 <bold>Hz</bold></th></tr><tr><td>R superior temporal gyrus</td><td align="right">61</td><td align="right">&#8722;12</td><td align="right">8</td><td align="right">10.9</td></tr><tr><td>L superior temporal gyrus</td><td align="right">&#8722;51</td><td align="right">-5</td><td align="right">5</td><td align="right">5.7</td></tr><tr><td>R middle temporal gyrus</td><td align="right">67</td><td align="right">&#8722;19</td><td align="right">-3</td><td align="right">8.5</td></tr><tr><td>L Heschl&#8217;s gyrus</td><td align="right">&#8722;50</td><td align="right">-7</td><td align="right">5</td><td align="right">5.3</td></tr><tr><td>R Heschl&#8217;s gyrus</td><td align="right">58</td><td align="right">&#8722;10</td><td align="right">8</td><td align="right">10.3</td></tr><tr><td>R postcentral gyrus</td><td align="right">59</td><td align="right">&#8722;16</td><td align="right">17</td><td align="right">8.2</td></tr><tr><td>L postcentral gyrus</td><td align="right">&#8722;57</td><td align="right">&#8722;19</td><td align="right">17</td><td align="right">6.4</td></tr><tr><td>R precentral gyrus</td><td align="right">55</td><td align="right">0</td><td align="right">19</td><td align="right">7.3</td></tr><tr><td>L precentral gyrus</td><td align="right">&#8722;50</td><td align="right">-6</td><td align="right">21</td><td align="right">4.7</td></tr><tr><td>R insula</td><td align="right">46</td><td align="right">&#8722;10</td><td align="right">8</td><td align="right">6.9</td></tr><tr><td>L insula</td><td align="right">&#8722;46</td><td align="right">-9</td><td align="right">2</td><td align="right">3.8</td></tr><tr><td>R middle frontal gyrus</td><td align="right">6</td><td align="right">50</td><td align="right">&#8722;11</td><td align="right">4.0</td></tr><tr><td>L supramarginal gyrus</td><td align="right">&#8722;58</td><td align="right">&#8722;21</td><td align="right">17</td><td align="right">6.2</td></tr><tr><th colspan="5" valign="top">Baseline condition at 4 <bold>Hz</bold></th></tr><tr><td>R superior temporal gyrus</td><td align="right">56</td><td align="right">&#8722;16</td><td align="right">1</td><td align="right">11.0</td></tr><tr><td>L superior temporal gyrus</td><td align="right">&#8722;50</td><td align="right">&#8722;18</td><td align="right">12</td><td align="right">7.3</td></tr><tr><td>R middle temporal gyrus</td><td align="right">55</td><td align="right">&#8722;27</td><td align="right">0</td><td align="right">7.3</td></tr><tr><td>L Heschl&#8217;s gyrus</td><td align="right">&#8722;42</td><td align="right">&#8722;22</td><td align="right">12</td><td align="right">7.0</td></tr><tr><td>R Heschl&#8217;s gyrus</td><td align="right">54</td><td align="right">&#8722;13</td><td align="right">8</td><td align="right">10.8</td></tr><tr><td>R postcentral gyrus</td><td align="right">61</td><td align="right">&#8722;15</td><td align="right">15</td><td align="right">8.8</td></tr><tr><td>L postcentral gyrus</td><td align="right">&#8722;53</td><td align="right">&#8722;11</td><td align="right">18</td><td align="right">8.7</td></tr><tr><td>R precentral gyrus</td><td align="right">45</td><td align="right">-8</td><td align="right">29</td><td align="right">6.3</td></tr><tr><td>L precentral gyrus</td><td align="right">&#8722;54</td><td align="right">-4</td><td align="right">21</td><td align="right">5.9</td></tr><tr><td>R insula</td><td align="right">48</td><td align="right">&#8722;10</td><td align="right">4</td><td align="right">6.9</td></tr><tr><td>L insula</td><td align="right">&#8722;38</td><td align="right">&#8722;24</td><td align="right">22</td><td align="right">6.8</td></tr><tr><td>L supramarginal gyrus</td><td align="right">&#8722;43</td><td align="right">&#8722;27</td><td align="right">23</td><td align="right">8.3</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-49223-v2.xml">elife-49223-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><label>Table 4.</label><caption><title>Summary of unbiased-association MD simulations.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"/><th colspan="4">Association Trajectories* (%)</th></tr><tr><th colspan="2">10 mM NaCl <break/>(low salt)</th><th colspan="2">100 mM NaCl <break/>(high salt)</th></tr><tr valign="top"><th>UBC13</th><th>dUBC13</th><th>UBC13</th><th>dUBC13</th></tr></thead><tbody><tr valign="top"><td>Native-like <break/>association</td><td>53 (8)</td><td>20 (3)</td><td>27 (4)</td><td>33 (5)</td></tr><tr valign="top"><td>Non-native <break/>association</td><td>47 (7)</td><td>60 (9)</td><td>40 (6)</td><td>47 (7)</td></tr><tr valign="top"><td>No association</td><td>0 (0)</td><td>20 (3)</td><td>33 (5)</td><td>20 (3)</td></tr><tr valign="top"><td rowspan="2"/><th colspan="4">Mean percentage time (i.e., stability) of native-like association (%)**</th></tr><tr valign="top"><td>32 (9)</td><td>14 (9)</td><td>11 (7)</td><td>11 (5)</td></tr></tbody></table><table-wrap-foot><fn><p>*The number of trajectories used is indicated in brackets.</p><p>**Standard error of mean is shown in brackets.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-49305-v2.xml">elife-49305-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><label>Table 3.</label><caption><title>Flux boundary constraints of Im-/export reactions</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Input (Reaction ID)</th><th colspan="2">Flux [&#956;mol/(m<sup>2</sup>s)]</th></tr><tr><th>Lower bound</th><th>Upper bound</th></tr></thead><tbody><tr><td>Photons (Im_hnu)</td><td>0</td><td>inf</td></tr><tr><td>C0<sub>2</sub> (Im_CO2)</td><td>0</td><td>20</td></tr><tr><td>NO<sub>3</sub><sup>-</sup> (Im_NO3)</td><td>0</td><td>inf</td></tr><tr><td>NH<sub>4</sub><sup>+</sup> (Im_NH4)</td><td>0</td><td>0</td></tr><tr><td>SO<sub>4</sub><sup>2-</sup> (Im_SO4)</td><td>0</td><td>inf</td></tr><tr><td>H<sub>2</sub>S (Im_H2S)</td><td>0</td><td>inf</td></tr><tr><td>Pi</td><td>0</td><td>inf</td></tr><tr><td>H<sub>2</sub>O (Im_H2O)</td><td>-inf</td><td>inf</td></tr><tr><td>O<sub>2</sub> (Im_O2)</td><td>-inf</td><td>inf</td></tr><tr><td>Amino Acids (Ex_AA)</td><td>0</td><td>inf</td></tr><tr><td>Surcose (Ex_Suc)</td><td>0</td><td>inf</td></tr><tr><td>Starch (Ex_starch)</td><td>0</td><td>inf</td></tr><tr><td>Other export reactions</td><td>0</td><td>0</td></tr></tbody></table><table-wrap-foot><fn><p>-inf/inf is approximated by &#8722;10<sup>6</sup> / 10<sup>6</sup></p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-49816-v2.xml">elife-49816-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="app1table1" position="float"><label>Appendix 1&#8212;table 1.</label><caption><title>Differences in the variance of community-level traits between treatments.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Community-level trait</th><th colspan="2">Variance</th><th rowspan="2">Ratio of variances</th><th rowspan="2">Bonett's test <italic>p</italic>-value</th></tr><tr><th>&#8195;Unfertilized</th><th>Fertilized</th></tr></thead><tbody><tr><td>P:C ratio</td><td>2.09 &#215; 10<sup>&#8722;7</sup></td><td>2.06 &#215; 10<sup>&#8722;8</sup></td><td>0.13</td><td>0.058</td></tr><tr><td>Mean genome size</td><td>105,315.4</td><td>10,984.8</td><td>0.1</td><td>0.174</td></tr><tr><td>Log number of rRNA genes</td><td>0.0056</td><td>0.0422</td><td>7.55</td><td>&lt;0.001</td></tr><tr><td>Log number of tRNA genes</td><td>0.0148</td><td>0.1396</td><td>9.46</td><td>0.013</td></tr><tr><td>GC content</td><td>0.0001</td><td>0.0002</td><td>1.9</td><td>0.56</td></tr><tr><td>Mean ENC</td><td>3.0554</td><td>0.3034</td><td>0.1</td><td>0.169</td></tr><tr><td>Median ENC</td><td>2.5861</td><td>0.413</td><td>0.16</td><td>0.275</td></tr><tr><td>Mean ENC'</td><td>2.5494</td><td>0.0824</td><td>0.03</td><td>0.056</td></tr><tr><td>Median ENC'</td><td>1.832</td><td>0.0671</td><td>0.04</td><td>0.069</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-49941-v1.xml">elife-49941-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Summary of within- and between-subject correlations of the cluster-based ROIs.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Correlation between</th><th rowspan="2">Cluster-based ROI</th><th rowspan="2">Hemisphere</th><th colspan="2">Rmcorr</th><th colspan="2">65 dph</th><th colspan="2">200 dph</th></tr><tr><th>R</th><th><italic>P</italic></th><th>Spearmans&#8217; &#961;</th><th><italic>P</italic></th><th>Spearmans&#8217; &#961;</th><th><italic>P</italic></th></tr></thead><tbody><tr><td rowspan="5">% similarity and FA</td><td rowspan="2">tFA</td><td>Left</td><td>0.1290</td><td>0.4200</td><td><bold>0.7846</bold></td><td><bold>0.0009</bold></td><td><bold>0.7714</bold></td><td><bold>0.0012</bold></td></tr><tr><td>Right</td><td>0.0215</td><td>0.8940</td><td><bold>0.7099</bold></td><td><bold>0.0045</bold></td><td><bold>0.7143</bold></td><td><bold>0.0041</bold></td></tr><tr><td rowspan="2">NCM</td><td>Left</td><td>0.2560</td><td>0.1060</td><td><bold>0.6747</bold></td><td><bold>0.0081</bold></td><td><bold>0.6835</bold></td><td><bold>0.0070</bold></td></tr><tr><td>Right</td><td><bold>0.3880</bold></td><td><bold>0.0121</bold></td><td><bold>0.6396</bold></td><td><bold>0.0138</bold></td><td><bold>0.5692</bold></td><td><bold>0.0336</bold></td></tr><tr><td>VP</td><td/><td><bold>0.4960</bold></td><td><bold>0.0010</bold></td><td><bold>0.8154</bold></td><td><bold>0.0004</bold></td><td><bold>0.7890</bold></td><td><bold>0.0008</bold></td></tr><tr><td rowspan="3">% similarity and log mwj</td><td>VP</td><td/><td><bold>&#8722;0.4290</bold></td><td><bold>0.0057</bold></td><td><bold>&#8722;0.7978</bold></td><td><bold>0.0006</bold></td><td>&#8722;0.4418</td><td>0.1138</td></tr><tr><td rowspan="2">CM</td><td>Left</td><td><bold>&#8722;0.3910</bold></td><td><bold>0.0126</bold></td><td>&#8722;0.5297</td><td>0.0514</td><td>&#8722;0.3055</td><td>0.2882</td></tr><tr><td>Right</td><td><bold>&#8722;0.4160</bold></td><td><bold>0.0075</bold></td><td>&#8722;0.4769</td><td>0.0846</td><td>&#8722;0.1912</td><td>0.5126</td></tr></tbody></table><table-wrap-foot><fn><p>&#8216;log mwj&#8217; refers to the log-transformed, modulated and warped jacobian determinants; FA stands for Fractional Anisotropy, one of the DTI metrics. &#8216;r&#8217; is the repeated-measures correlation coefficient of the within-subject correlation analyses. Spearmans&#8217; &#961; informs on potential correlations between the MRI parameters and song similarity at a specific time point between birds. Tests that survive Benjamini-Hochberg FDR correction for multiple comparisons are highlighted in bold (<xref ref-type="supplementary-material" rid="supp6">Supplementary file 6</xref>, <xref ref-type="supplementary-material" rid="supp7">7</xref>) Abbreviations: dph: days post hatching.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-49941-v1.xml">elife-49941-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="resptable1" position="float"><label>Author response table 1.</label><caption><title>Summary of the voxel-based multiple regressions using SwE (% similarity and FA).</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Correlation between</th><th rowspan="2">Cluster</th><th rowspan="2">Hemisphere</th><th colspan="2">Cluster</th><th colspan="2">Peak</th></tr><tr><th>k<sub>E</sub></th><th><italic>p<sub>FWE</sub></italic></th><th><italic>T</italic></th><th><italic>p<sub>uncorr</sub></italic></th></tr></thead><tbody><tr><td rowspan="5">% similarity and FA</td><td rowspan="2">tFA</td><td>Left</td><td>22</td><td>0.675</td><td>3.63</td><td>&lt;0.001</td></tr><tr><td>Right</td><td>7</td><td>0.559</td><td>3.79</td><td>&lt;0.001</td></tr><tr><td>NCM</td><td>Left</td><td>11</td><td>0.635</td><td>3.74</td><td>&lt;0.001</td></tr><tr><td>CMM</td><td>Right</td><td>8</td><td>0.587</td><td>4.10</td><td>&lt;0.001</td></tr><tr><td>VP</td><td/><td>4</td><td>0.933</td><td>3.16</td><td>0.001</td></tr><tr><td rowspan="3">% similarity and log mwj</td><td>VP</td><td/><td>852</td><td>0.201</td><td>4.09</td><td>0.000</td></tr><tr><td rowspan="2">CM</td><td>Left</td><td>6265</td><td>0.925</td><td>2.49</td><td>0.000</td></tr><tr><td>Right</td><td>9372</td><td>0.024*</td><td>5.17</td><td>0.000</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-49941-v1.xml">elife-49941-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="resptable2" position="float"><label>Author response table 2.</label><caption><title>Summary of the voxel-based multiple regressions using SwE (% similarity and FA or log mwj).</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Correlation between</th><th rowspan="2">Cluster</th><th rowspan="2">Hemisphere</th><th colspan="2" valign="top">Between- subject correlation</th><th colspan="2" valign="top">Within-subject correlation</th></tr><tr><th valign="top">Spearman&#8217;s &#961;</th><th valign="top">p value</th><th valign="top">rmcorr r</th><th valign="top">rmcorr p</th></tr></thead><tbody><tr><td rowspan="5"><bold>% similarity and FA</bold></td><td rowspan="2"><bold>tFA</bold></td><td>Left</td><td>0.728</td><td>&lt;0.0001</td><td>0.175</td><td>0.274</td></tr><tr><td>Right</td><td>0.656</td><td>&lt;0.0001</td><td>0.326</td><td>0.0378</td></tr><tr><td><bold>NCM</bold></td><td>Left</td><td>0.635</td><td>&lt;0.0001</td><td>0.184</td><td>0.248</td></tr><tr><td><bold>CMM</bold></td><td>Right</td><td>0.618</td><td>&lt;0.0001</td><td><bold>0.454</bold></td><td><bold>0.0029*</bold></td></tr><tr><td><bold>VP</bold></td><td/><td>0.592</td><td>&lt;0.0001</td><td>-0.041</td><td>0.799</td></tr><tr><td rowspan="3"><bold>% similarity and log mwj</bold></td><td><bold>VP</bold></td><td/><td>-0.583</td><td>&lt;0.0001</td><td><bold>-0.399</bold></td><td><bold>0.0094*</bold></td></tr><tr><td rowspan="2"><bold>CM</bold></td><td>Left</td><td>-0.346</td><td>0.0103</td><td><bold>-0.393</bold></td><td><bold>0.0111*</bold></td></tr><tr><td>Right</td><td>-0.327</td><td>0.0158</td><td><bold>-0.397</bold></td><td><bold>0.0101*</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-49941-v1.xml">elife-49941-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="resptable4" position="float"><label>Author response table 4.</label><caption><title>Summary of the ROI-based between- and within-subject correlation analysis (% similarity and FA).</title></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2" valign="top">ROI</th><th rowspan="2" valign="top">Hemisphere</th><th colspan="2" valign="top">Between- subject correlation</th><th colspan="2" valign="top">Within-subject correlation</th></tr><tr><th valign="top">Spearman&#8217;s &#961;</th><th valign="top">p value</th><th valign="top">rmcorr r</th><th valign="top">rmcorr p</th></tr></thead><tbody><tr><td rowspan="8" valign="top">Song control system</td><td rowspan="2" valign="top">Area X</td><td valign="top">Left</td><td valign="bottom">0.0286</td><td valign="bottom">0.9228</td><td valign="bottom">0.267</td><td valign="bottom">0.0911</td></tr><tr><td valign="top">Right</td><td valign="bottom">-0.1824</td><td valign="bottom">0.5325</td><td valign="bottom">0.162</td><td valign="bottom">0.313</td></tr><tr><td rowspan="2" valign="top">HVC</td><td valign="top">Left</td><td valign="bottom">-0.4989</td><td valign="bottom">0.0694</td><td valign="bottom">-0.271</td><td valign="bottom">0.0871</td></tr><tr><td valign="top">Right</td><td valign="bottom">-0.0725</td><td valign="bottom">0.8054</td><td valign="bottom">0.0806</td><td valign="bottom">0.616</td></tr><tr><td rowspan="2" valign="top">LMAN</td><td valign="top">Left</td><td valign="bottom">0.0989</td><td valign="bottom">0.7366</td><td valign="bottom">-0.135</td><td valign="bottom">0.4</td></tr><tr><td valign="top">Right</td><td valign="bottom">-0.0242</td><td valign="bottom">0.9346</td><td valign="bottom">-0.0114</td><td valign="bottom">0.944</td></tr><tr><td rowspan="2" valign="top">RA</td><td valign="top">Left</td><td valign="bottom">-0.3231</td><td valign="bottom">0.2599</td><td valign="bottom">0.029</td><td valign="bottom">0.857</td></tr><tr><td valign="top">Right</td><td valign="bottom">0.0462</td><td valign="bottom">0.8755</td><td valign="bottom">0.138</td><td valign="bottom">0.391</td></tr><tr><td rowspan="6" valign="top">Auditory system</td><td rowspan="2" valign="top">Field L</td><td valign="top">Left</td><td valign="bottom">-0.0593</td><td valign="bottom">0.8403</td><td valign="bottom">0.108</td><td valign="bottom">0.501</td></tr><tr><td valign="top">Right</td><td valign="bottom">-0.2923</td><td valign="bottom">0.3105</td><td valign="bottom">0.0996</td><td valign="bottom">0.536</td></tr><tr><td rowspan="2" valign="top">NCM</td><td valign="top">Left</td><td valign="bottom"><bold>0.6967</bold></td><td valign="bottom"><bold>0.0056</bold></td><td valign="bottom">0.109</td><td valign="bottom">0.499</td></tr><tr><td valign="top">Right</td><td valign="bottom"><bold>0.5473</bold></td><td valign="bottom"><bold>0.0428</bold></td><td valign="bottom">0.185</td><td valign="bottom">0.247</td></tr><tr><td rowspan="2" valign="top">CM</td><td valign="top">Left</td><td valign="bottom">0.4813</td><td valign="bottom">0.0814</td><td valign="bottom">-0.123</td><td valign="bottom">0.443</td></tr><tr><td valign="top">Right</td><td valign="bottom"><bold>0.622</bold></td><td valign="bottom"><bold>0.0176</bold></td><td valign="bottom">0.283</td><td valign="bottom">0.0732</td></tr><tr><td rowspan="8" valign="top">Other</td><td rowspan="2" valign="top">VP</td><td valign="top">Left</td><td valign="bottom"><bold>0.5956</bold></td><td valign="bottom"><bold>0.0246</bold></td><td valign="bottom">0.136</td><td valign="bottom">0.397</td></tr><tr><td valign="top">Right</td><td valign="bottom"><bold>0.6088</bold></td><td valign="bottom"><bold>0.0209</bold></td><td valign="bottom">0.236</td><td valign="bottom">0.138</td></tr><tr><td rowspan="2" valign="top">tFA</td><td valign="top">Left</td><td valign="bottom"><bold>0.6396</bold></td><td valign="bottom"><bold>0.0138</bold></td><td valign="bottom">0.259</td><td valign="bottom">0.102</td></tr><tr><td valign="top">Right</td><td valign="bottom">0.3451</td><td valign="bottom">0.2269</td><td valign="bottom"><bold>0.343</bold></td><td valign="bottom"><bold>0.0283</bold></td></tr><tr><td rowspan="2" valign="top">Area X surr.</td><td valign="top">Left</td><td valign="bottom">-0.1341</td><td valign="bottom">0.6477</td><td valign="bottom">0.193</td><td valign="bottom">0.225</td></tr><tr><td valign="top">Right</td><td valign="bottom">-0.0637</td><td valign="bottom">0.8286</td><td valign="bottom">0.293</td><td valign="bottom">0.0633</td></tr><tr><td rowspan="2" valign="top">RA surr.</td><td valign="top">Left</td><td valign="bottom">-0.156</td><td valign="bottom">0.5942</td><td valign="bottom">-0.191</td><td valign="bottom">0.232</td></tr><tr><td valign="top">Right</td><td valign="bottom">0.0198</td><td valign="bottom">0.9465</td><td valign="bottom">0.242</td><td valign="bottom">0.128</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-49979-v1.xml">elife-49979-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table7" position="float"><label>Table 7.</label><caption><title>Proportions of cancer deaths in the United States in the 1970s attributed to various different factors, as defined by <xref ref-type="bibr" rid="bib26">Doll and Peto (1981)</xref>.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Factor or class of factors</th><th colspan="2" valign="top">Percent of all cancer deaths</th></tr><tr><th valign="top">Best estimate</th><th valign="top">Range of acceptable estimates</th></tr></thead><tbody><tr><td valign="top">Tobacco</td><td valign="top">30</td><td valign="top">25&#8211;40</td></tr><tr><td valign="top">Alcohol</td><td valign="top">3</td><td valign="top">2&#8211;4</td></tr><tr><td valign="top">Food additives</td><td valign="top">&lt;1</td><td valign="top">&#8722;5&#8211;2</td></tr><tr><td valign="top">Reproductive and sexual behaviour</td><td valign="top">7</td><td valign="top">1&#8211;13</td></tr><tr><td valign="top">Occupation</td><td valign="top">4</td><td valign="top">2&#8211;8</td></tr><tr><td valign="top">Pollution</td><td valign="top">2</td><td valign="top">&lt;1&#8211;5</td></tr><tr><td valign="top">Industrial products</td><td valign="top">&lt;1</td><td valign="top">&lt;1&#8211;2</td></tr><tr><td valign="top">Medicines and medical procedures</td><td valign="top">1</td><td valign="top">0.5&#8211;3</td></tr><tr><td valign="top">Geophysical factors (mostly Ultraviolet light)</td><td valign="top">3</td><td valign="top">2&#8211;4</td></tr><tr><td valign="top">Infection</td><td valign="top">10?</td><td valign="top">1-?</td></tr><tr><td valign="top">Diet</td><td valign="top">35</td><td valign="top">10&#8211;70</td></tr><tr><td valign="top">Unknown</td><td valign="top">?</td><td valign="top">?</td></tr></tbody></table><table-wrap-foot><fn><p>Adapted from <italic>The causes of cancer: Quantitative estimates of avoidable risks of cancer in the United States today</italic>, <xref ref-type="bibr" rid="bib26">Doll and Peto (1981)</xref>, <italic>Journal of the National Cancer Institute, 66</italic>(6), p. 1192&#8211;1308.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-49979-v2.xml">elife-49979-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table7" position="float"><label>Table 7.</label><caption><title>Proportions of cancer deaths in the United States in the 1970s attributed to various different factors, as defined by <xref ref-type="bibr" rid="bib26">Doll and Peto (1981)</xref>.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Factor or class of factors</th><th colspan="2" valign="top">Percent of all cancer deaths</th></tr><tr><th valign="top">Best estimate</th><th valign="top">Range of acceptable estimates</th></tr></thead><tbody><tr><td valign="top">Tobacco</td><td valign="top">30</td><td valign="top">25&#8211;40</td></tr><tr><td valign="top">Alcohol</td><td valign="top">3</td><td valign="top">2&#8211;4</td></tr><tr><td valign="top">Food additives</td><td valign="top">&lt;1</td><td valign="top">&#8722;5&#8211;2</td></tr><tr><td valign="top">Reproductive and sexual behaviour</td><td valign="top">7</td><td valign="top">1&#8211;13</td></tr><tr><td valign="top">Occupation</td><td valign="top">4</td><td valign="top">2&#8211;8</td></tr><tr><td valign="top">Pollution</td><td valign="top">2</td><td valign="top">&lt;1&#8211;5</td></tr><tr><td valign="top">Industrial products</td><td valign="top">&lt;1</td><td valign="top">&lt;1&#8211;2</td></tr><tr><td valign="top">Medicines and medical procedures</td><td valign="top">1</td><td valign="top">0.5&#8211;3</td></tr><tr><td valign="top">Geophysical factors (mostly Ultraviolet light)</td><td valign="top">3</td><td valign="top">2&#8211;4</td></tr><tr><td valign="top">Infection</td><td valign="top">10?</td><td valign="top">1-?</td></tr><tr><td valign="top">Diet</td><td valign="top">35</td><td valign="top">10&#8211;70</td></tr><tr><td valign="top">Unknown</td><td valign="top">?</td><td valign="top">?</td></tr></tbody></table><table-wrap-foot><fn><p>Adapted from <italic>The causes of cancer: Quantitative estimates of avoidable risks of cancer in the United States today</italic>, <xref ref-type="bibr" rid="bib26">Doll and Peto (1981)</xref>, <italic>Journal of the National Cancer Institute, 66</italic>(6), p. 1192&#8211;1308.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-50240-v1.xml">elife-50240-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Results of PERMANOVA analysis investigating the effect of the interaction between disease signatures and age-group, after adjusting for the effects of country (within the continent cohorts) and the independent effects of disease and age-group.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Adonis</th><th colspan="3" valign="bottom">T2D</th><th colspan="3" valign="bottom">IBD</th><th colspan="3" valign="bottom">CRC</th><th colspan="3" valign="bottom">Polyps</th><th colspan="3" valign="bottom">Cirrhosis</th></tr><tr><th valign="bottom">F.Model</th><th valign="bottom">R-Squared</th><th valign="bottom">P-Value</th><th valign="bottom">F.Model</th><th valign="bottom">R-Squared</th><th valign="bottom">P-Value</th><th valign="bottom">F.Model</th><th valign="bottom">R-Squared</th><th valign="bottom">P-Value</th><th valign="bottom">F.Model</th><th valign="bottom">R-Squared</th><th valign="bottom">P-Value</th><th valign="bottom">F.Model</th><th valign="bottom">R-Squared</th><th valign="bottom">P-Value</th></tr></thead><tbody><tr><td valign="bottom">Country</td><td valign="bottom">16.69</td><td valign="bottom">0.029</td><td valign="bottom">0.001</td><td valign="bottom">11.34</td><td valign="bottom">0.017</td><td valign="bottom">0.001</td><td valign="bottom">7.63</td><td valign="bottom">0.027</td><td valign="bottom">0.001</td><td valign="bottom">4.65</td><td valign="bottom">0.022</td><td valign="bottom">0.001</td><td valign="bottom">NA</td><td valign="bottom">NA</td><td valign="bottom">NA</td></tr><tr><td valign="bottom">Disease</td><td valign="bottom">4.81</td><td valign="bottom">0.008</td><td valign="bottom">0.001</td><td valign="bottom">15.79</td><td valign="bottom">0.023</td><td valign="bottom">0.001</td><td valign="bottom">4.70</td><td valign="bottom">0.008</td><td valign="bottom">0.001</td><td valign="bottom">1.25</td><td valign="bottom">0.006</td><td valign="bottom">0.028</td><td valign="bottom">12.107</td><td valign="bottom">0.029</td><td valign="bottom">0.001</td></tr><tr><td valign="bottom">Age-Group</td><td valign="bottom">1.38</td><td valign="bottom">0.005</td><td valign="bottom">0.001</td><td valign="bottom">2.51</td><td valign="bottom">0.007</td><td valign="bottom">0.001</td><td valign="bottom">3.80</td><td valign="bottom">0.007</td><td valign="bottom">0.001</td><td valign="bottom">1.00</td><td valign="bottom">0.004</td><td valign="bottom">0.408</td><td valign="bottom">1.239</td><td valign="bottom">0.006</td><td valign="bottom">0.016</td></tr><tr><td valign="bottom">Disease:Age-Group</td><td valign="bottom">1.12</td><td valign="bottom">0.004</td><td valign="bottom">0.08</td><td valign="bottom">2.73</td><td valign="bottom">0.008</td><td valign="bottom">0.001</td><td valign="bottom">3.57</td><td valign="bottom">0.006</td><td valign="bottom">0.001</td><td valign="bottom">1.38</td><td valign="bottom">0.006</td><td valign="bottom">0.011</td><td valign="bottom">1.035</td><td valign="bottom">0.005</td><td valign="bottom">0.290</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-50243-v2.xml">elife-50243-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Baseline characteristics of outbreaks that were responded to, according to the response promptness (time to the first complete case-area targeted intervention).</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th rowspan="2">All outbreaks</th><th rowspan="2">Outbreaks responded to with&#160;&#8805;&#160;1 complete CATI</th><th colspan="4">Class of response promptness <break/>(time to the first complete CATI)</th><th colspan="2">Comparison between classes of promptness</th></tr><tr><th>&gt;7 days</th><th>3 to 7 days</th><th>2 days</th><th>&#8804;1 day</th><th>Hazard ratio <break/>(95% CI)<sup>&#8224;</sup></th><th>p-value<sup>&#8224;</sup></th></tr></thead><tbody><tr><td valign="top">No. of outbreaks</td><td valign="top">452</td><td valign="top">238 <break/>(53%)</td><td valign="top">48 <break/>(20%)</td><td valign="top">40 <break/>(17%)</td><td valign="top">43 <break/>(18%)</td><td valign="top">107 <break/>(45%)</td><td valign="top"/><td valign="top"/></tr><tr><td valign="top">Semester sinceJanuary 2015</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top">1.10e7 <break/>(1.64e6 to 7.40e7)</td><td valign="top">&lt;0.0001<sup>*</sup></td></tr><tr><td valign="top">Population density, median (IQR; inhab./km<sup>2</sup>)</td><td valign="top">3.5 <break/>(6.5)</td><td valign="top">3.6 <break/>(11.5)</td><td valign="top">4.3 <break/>(10.6)</td><td valign="top">2.8 <break/>(4.6)</td><td valign="top">3.7 <break/>(8.9)</td><td valign="top">3.8 <break/>(12.6)</td><td valign="top">1.01 <break/>(1 to 1.02)</td><td valign="top">0.0039<sup>*</sup></td></tr><tr><td valign="top">Travel time to thenearest town, median (IQR;minutes)</td><td valign="top">26.7 <break/>(33.2)</td><td valign="top">24.9 <break/>(31.8)</td><td valign="top">30 (3 <break/>4.3)</td><td valign="top">27.1 <break/>(42.1)</td><td valign="top">24.8 <break/>(28.4)</td><td valign="top">22 <break/>(32)</td><td valign="top">1 <break/>(0.99 to 1)</td><td valign="top">0.274</td></tr><tr><td valign="top">Accumulated incidence between 2010 and 2014, median (IQR; per 1000&#160;inhabitants)</td><td valign="top">103.8 <break/>(77.5)</td><td valign="top">103.8 <break/>(77.5)</td><td valign="top">103.8 <break/>(131.4)</td><td valign="top">103.8 <break/>(49.1)</td><td valign="top">103.8 <break/>(56.6)</td><td valign="top">103.8 <break/>(77.5)</td><td valign="top">0.4 <break/>(0.09 to 1.83)</td><td valign="top">0.237</td></tr><tr><td valign="top">Coverage of OCVcampaigns between 2012 and 2014, median (IQR;%) [mean, SD]</td><td valign="top">0% (86) <break/>[25%, 40]</td><td valign="top">0% (0) <break/>[21%, 38]</td><td valign="top">0% (86) <break/>[30%, 42]</td><td valign="top">0% (0) <break/>[18%, 36]</td><td valign="top">0% (0) <break/>[15%, 33]</td><td valign="top">0% (0) <break/>[21%, 38]</td><td valign="top">0.61 <break/>(0.38 to 0.98)</td><td valign="top">0.0393<sup>*</sup></td></tr><tr><td valign="top">Previous cases inthe same localityduring the study, median (IQR; no.per year)</td><td valign="top">4.3 <break/>(10.1)</td><td valign="top">5.2 <break/>(10.5)</td><td valign="top">7.2 <break/>(10.1)</td><td valign="top">5 <break/>(11.5)</td><td valign="top">6.9 <break/>(11.1)</td><td valign="top">5 <break/>(8.5)</td><td valign="top">0.99 <break/>(0.97&#8211;1.02)</td><td valign="top">0.6540</td></tr><tr><td valign="top">Previous completeCATIs in the samelocality during thestudy, median(IQR; no. per year)</td><td valign="top">0.2 <break/>(1.9)</td><td valign="top">0.9 <break/>(2.7)</td><td valign="top">0.7 <break/>(2.2)</td><td valign="top">0.5 <break/>(2.3)</td><td valign="top">1.2 <break/>(2.9)</td><td valign="top">1.4 <break/>(2.7)</td><td valign="top">0.98 <break/>(0.91&#8211;1.06)</td><td valign="top">0.6500</td></tr><tr><td valign="top">Daily rainfall during outbreak,median (IQR; mm)</td><td valign="top">6.6 <break/>(13.3)</td><td valign="top">7.7 <break/>(13.3)</td><td valign="top">12 <break/>(6)</td><td valign="top">6.9 <break/>(10.8)</td><td valign="top">10 <break/>(13.7)</td><td valign="top">3.6 <break/>(14.4)</td><td valign="top">0.99 <break/>(0.96 to 1.03)</td><td valign="top">0.638</td></tr><tr><td valign="top">No. of cases duringthe first 3 daysofoutbreak, median (IQR) [mean, SD]</td><td valign="top">2 (1) <break/>[2.5, 1.5]</td><td valign="top">2 (1) <break/>[2.7, 1.9]</td><td valign="top">2 (1) <break/>[2.5, 1.0]</td><td valign="top">2.5 (1) <break/>[3.4, 2.1]</td><td valign="top">2 (0.5) <break/>[2.8, 2.4]</td><td valign="top">2 (1) <break/>[2.5, 1.9]</td><td valign="top">1.04 <break/>(0.93 to 1.16)</td><td valign="top">0.488</td></tr><tr><td valign="top">No. of positiveculture during thefirst 3 daysof outbreak, median(IQR) [mean, SD]</td><td valign="top">0 (0) <break/>[0.2, 0.6]</td><td valign="top">0 (0) <break/>[0.3, 0.7]</td><td valign="top">0 (0) <break/>[0.1, 0.3]</td><td valign="top">0 (0) <break/>[0.2, 0.5]</td><td valign="top">0 (1) <break/>[0.5, 0.9]</td><td valign="top">0 (1) <break/>[0.4, 0.7]</td><td valign="top">2.03 <break/>(1.3 to 3.17)</td><td valign="top">0.0018<sup>*</sup></td></tr></tbody></table><table-wrap-foot><fn><p>CATI, case-area targeted intervention; IQR, interquartile range; SD, standard deviation.</p><p><sup>&#8224;</sup>Univariate comparisons between classes of response promptness using Cox models for Andersen-Gill counting process (AG-CP), with time to the first complete CATI modelled as a recurrent time-to-event outcome.</p></fn><fn><p><sup>*</sup>Significant p-value.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-50541-v2.xml">elife-50541-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Annotation of PHE1 ChIP-seq peaks within genomic features of interest.</title><p>Annotation for each individual replicate, as well as for common peaks, is presented. For target gene analysis, only common peaks located 1.5 kb upstream to 0.5 kb downstream of the TSS were considered (3<sup>rd</sup> row).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Sample</th><th rowspan="2">Total no. of peaks</th><th rowspan="2">No. of peaks in &#8722;1.5 kb to +0.5 kb window around TSS</th><th rowspan="2">Average distance to nearest TSS (bp)</th><th colspan="3">Associated genomic feature (% of peaks)</th><th rowspan="2">No.&#160;of targeted genes</th></tr><tr><th>Promoter</th><th>Gene body</th><th>Intergenic</th></tr></thead><tbody><tr><td>Replicate 1 peaks</td><td>2818</td><td>2182</td><td>445</td><td>88.6</td><td>4.7</td><td>6.6</td><td>1985</td></tr><tr><td>Replicate 2 peaks</td><td>4521</td><td>3508</td><td>600</td><td>86.6</td><td>3.5</td><td>9.9</td><td>2971</td></tr><tr><td>Common peaks&#160;(PHE1&#160;binding&#160;sites)</td><td rowspan="2">2494</td><td rowspan="2">1995</td><td rowspan="2">430</td><td rowspan="2">89.6</td><td rowspan="2">4.6</td><td rowspan="2">5.8</td><td rowspan="2">1694</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-50830-v2.xml">elife-50830-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><label>Table 3.</label><caption><title>Association between the integrity of white matter tracts and the PET networks.</title><p>The values presented in the table correspond to Spearman&#8217;s Rho followed by (p values) for the correlations between the fractional anisotropy or mean diffusivity of white matter tracts and PET networks SUVRs, while controlling for age, sex and presence of cognitive impairment. Values in bold correspond to significant group differences after adjusting for multiple comparisons with false discovery rate corrections (FDR) (q&#160;&lt;&#160;0.05). Correlations were carried out in all amyloid<bold>-&#946;</bold> positive individuals: 28 amyloid-&#946; positive cognitively normal subjects (CN A&#946;+), 18 patients with mild cognitive impairment (MCI A&#946;+) and 26 patients with Alzheimer&#8217;s disease (AD A&#946;+). Three AD patients and 1 MCI patient were excluded due to being outliers in white matter measures. ATR, anterior thalamic radiation; CST, cortico-spinal tract; CCT, cingulate cingulum tract; HCT, hippocampal cingulum tract; FMAJ, forceps major; FMIN, forceps minor; IFO, inferior fronto-occipital fasciculus; ILF, inferior longitudinal fasciculus; SLF, superior longitudinal fasciculus; SLFt, temporal part of the superior longitudinal fasciculus; UNC, uncinate fasciculus.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="11" valign="top">Fractional anisotropy</th></tr><tr><th valign="top">ATR</th><th valign="top">CST</th><th valign="top">CCT</th><th valign="top">HCT</th><th valign="top">FMAJ</th><th valign="top">FMIN</th><th valign="top">IFO</th><th valign="top">ILF</th><th valign="top">SLF</th><th valign="top">SLFt</th><th valign="top">UNC</th></tr></thead><tbody><tr><td valign="top">amyloid-&#946; IC 2</td><td valign="top">&#8722;0.016 (0.904)</td><td valign="top">0.081 (0.527)</td><td valign="top">0.014 <break/>(0.915)</td><td valign="top">0.031 (0.810)</td><td valign="top">0.011 (0.930)</td><td valign="top">0.079 (0.540)</td><td valign="top">&#8722;0.093 (0.467)</td><td valign="top">&#8722;0.073 (0.569)</td><td valign="top">&#8722;0.111 (0.386)</td><td valign="top">&#8722;0.123 (0.336)</td><td valign="top">&#8722;0.047 (0.717)</td></tr><tr><td valign="top">amyloid-&#946; IC 3</td><td valign="top">&#8722;0.162 (0.205)</td><td valign="top">0.133 (0.298)</td><td valign="top">0.027 (0.832)</td><td valign="top">&#8722;0.052 <break/>(0.685)</td><td valign="top">&#8722;0.094 (0.462)</td><td valign="top">&#8722;0.091 (0.478)</td><td valign="top">&#8722;0.180 (0.159)</td><td valign="top">&#8722;0.105 (0.411)</td><td valign="top">&#8722;0.158 (0.217)</td><td valign="top">&#8722;0.211 (0.097)</td><td valign="top">&#8722;0.073 (0.572)</td></tr><tr><td valign="top">amyloid-&#946; IC 4</td><td valign="top">&#8722;0.121 (0.347)</td><td valign="top">0.026 (0.841)</td><td valign="top">&#8722;0.043 (0.740)</td><td valign="top">&#8722;0.107 (0.404)</td><td valign="top">&#8722;0.112 (0.384)</td><td valign="top">&#8722;0.110 (0.393)</td><td valign="top">&#8722;0.166 (0.195)</td><td valign="top">&#8722;0.076 (0.552)</td><td valign="top">&#8722;0.130 (0.310)</td><td valign="top">&#8722;0.187 (0.142)</td><td valign="top">&#8722;0.042 (0.743)</td></tr><tr><td valign="top">amyloid-&#946; IC 5</td><td valign="top">&#8722;0.025 (0.848)</td><td valign="top">0.089 (0.488)</td><td valign="top">0.006 (0.964)</td><td valign="top">&#8722;0.027 (0.836)</td><td valign="top">&#8722;0.073 (0.568)</td><td valign="top">0.018 (0.890)</td><td valign="top">&#8722;0.085 (0.507)</td><td valign="top">&#8722;0.002 (0.989)</td><td valign="top">&#8722;0.036 (0.781)</td><td valign="top">&#8722;0.125 (0.331)</td><td valign="top">0.014 (0.911)</td></tr><tr><td valign="top">amyloid-&#946; IC 6</td><td valign="top">&#8722;0.177 (0.166)</td><td valign="top">0.085 (0.509)</td><td valign="top">0.033 (0.799)</td><td valign="top">&#8722;0.009 (0.947)</td><td valign="top">&#8722;0.123 (0.338)</td><td valign="top">&#8722;0.072 (0.575)</td><td valign="top">&#8722;0.164 (0.199)</td><td valign="top">&#8722;0.080 (0.533)</td><td valign="top">&#8722;0.123 (0.337)</td><td valign="top">&#8722;0.194 (0.128)</td><td valign="top">&#8722;0.061 (0.636)</td></tr><tr><td valign="top">amyloid-&#946; IC 7</td><td valign="top">0.007 (0.957)</td><td valign="top">0.004 (0.978)</td><td valign="top">&#8722;0.071 (0.582)</td><td valign="top">&#8722;0.046 (0.719)</td><td valign="top">&#8722;0.062 (0.630)</td><td valign="top">0.022 (0.863)</td><td valign="top">&#8722;0.072 (0.576)</td><td valign="top">&#8722;0.031 (0.813)</td><td valign="top">&#8722;0.018 (0.890)</td><td valign="top">&#8722;0.064 (0.617)</td><td valign="top">&#8722;0.051 (0.692)</td></tr><tr><td valign="top">amyloid-&#946; IC 8</td><td valign="top">&#8722;0.017 (0.896)</td><td valign="top">0.065 (0.611)</td><td valign="top">0.024 (0.852)</td><td valign="top">0.045 (0.725)</td><td valign="top">&#8722;0.003 (0.979)</td><td valign="top">0.052 (0.687)</td><td valign="top">&#8722;0.037 (0.774)</td><td valign="top">0.024 (0.851)</td><td valign="top">&#8722;0.003 (0.981)</td><td valign="top">&#8722;0.073 (0.572)</td><td valign="top">0.029 (0.824)</td></tr><tr><td valign="top">amyloid-&#946; IC 10</td><td valign="top">&#8722;0.170 (0.183)</td><td valign="top">0.082 (0.525)</td><td valign="top">0.038 (0.765)</td><td valign="top">&#8722;0.062 (0.627)</td><td valign="top">&#8722;0.152 (0.234)</td><td valign="top">&#8722;0.072 (0.576)</td><td valign="top">&#8722;0.175 (0.170)</td><td valign="top">&#8722;0.082 (0.524)</td><td valign="top">&#8722;0.140 (0.275)</td><td valign="top">&#8722;0.175 (0.170)</td><td valign="top">&#8722;0.039 (0.760)</td></tr><tr><td valign="top">amyloid-&#946; IC 11</td><td valign="top">0.040 (0.757)</td><td valign="top">0.064 (0.618)</td><td valign="top">&#8722;0.040 (0.760)</td><td valign="top">&#8722;0.045 (0.724)</td><td valign="top">&#8722;0.020 (0.874)</td><td valign="top">0.096 (0.456)</td><td valign="top">&#8722;0.083 (0.517)</td><td valign="top">&#8722;0.087 (0.497)</td><td valign="top">&#8722;0.070 (0.585)</td><td valign="top">&#8722;0.093 (0.467)</td><td valign="top">&#8722;0.046 (0.718)</td></tr><tr><td valign="top">tau IC 2</td><td valign="top">&#8722;0.214 (0.093)</td><td valign="top">&#8722;0.123 (0.336)</td><td valign="top">&#8722;0.291 (0.021)</td><td valign="top"><bold>&#8722;0.364 (0.003)</bold></td><td valign="top">&#8722;0.283 (0.025)</td><td valign="top">&#8722;0.170 (0.184)</td><td valign="top">&#8722;0.307 (0.014)</td><td valign="top"><bold>&#8722;0.429 (&lt;0.001)</bold></td><td valign="top">&#8722;0.339 (0.007)</td><td valign="top">&#8722;0.259 (0.041)</td><td valign="top">&#8722;0.280 (0.026)</td></tr><tr><td valign="top">tau IC 3</td><td valign="top">&#8722;0.207 (0.104)</td><td valign="top">&#8722;0.092 (0.476)</td><td valign="top">&#8722;0.286 (0.023)</td><td valign="top"><bold>&#8722;0.398 (0.001)</bold></td><td valign="top">&#8722;0.251 (0.047)</td><td valign="top">&#8722;0.234 (0.065)</td><td valign="top">&#8722;0.301 (0.017)</td><td valign="top"><bold>&#8722;0.362 (0.004)</bold></td><td valign="top">&#8722;0.257 (0.042)</td><td valign="top">&#8722;0.283 (0.025)</td><td valign="top">&#8722;0.292 (0.020)</td></tr><tr><td valign="top">tau IC 4</td><td valign="top">&#8722;0.222 (0.081)</td><td valign="top">&#8722;0.305 (0.015)</td><td valign="top">&#8722;0.309 (0.014)</td><td valign="top">&#8722;0.348 (0.005)</td><td valign="top">&#8722;0.292 (0.020)</td><td valign="top">&#8722;0.165 (0.198)</td><td valign="top">&#8722;0.339 (0.007)</td><td valign="top"><bold>&#8722;0.459 (&lt;0.001)</bold></td><td valign="top"><bold>&#8722;0.378 (0.002)</bold></td><td valign="top">&#8722;0.297 (0.018)</td><td valign="top">&#8722;0.303 (0.016)</td></tr><tr><td valign="top">tau IC 5</td><td valign="top">&#8722;0.266 (0.035)</td><td valign="top">&#8722;0.245 (0.053)</td><td valign="top">&#8722;0.370 (0.003)</td><td valign="top"><bold>&#8722;0.361 (0.004)</bold></td><td valign="top">&#8722;0.271 (0.032)</td><td valign="top">&#8722;0.288 (0.022)</td><td valign="top"><bold>&#8722;0.395 (0.001)</bold></td><td valign="top"><bold>&#8722;0.430 (&lt;0.001)</bold></td><td valign="top"><bold>&#8722;0.386 (0.002)</bold></td><td valign="top">&#8722;0.286 (0.023)</td><td valign="top"><bold>&#8722;0.398 (0.001)</bold></td></tr><tr><td valign="top">tau IC 6</td><td valign="top">&#8722;0.224 (0.077)</td><td valign="top">&#8722;0.176 (0.167)</td><td valign="top">&#8722;0.239 (0.059)</td><td valign="top">&#8722;0.309 (0.014)</td><td valign="top">&#8722;0.281 (0.026)</td><td valign="top">&#8722;0.150 (0.241)</td><td valign="top">&#8722;0.292 (0.020)</td><td valign="top"><bold>&#8722;0.393 (0.001)</bold></td><td valign="top">&#8722;0.330 (0.008)</td><td valign="top">&#8722;0.214 (0.093)</td><td valign="top">&#8722;0.225 (0.077)</td></tr><tr><td valign="top">tau IC 7</td><td valign="top">&#8722;0.226 (0.075)</td><td valign="top">&#8722;0.194 (0.127)</td><td valign="top">&#8722;0.352 (0.005)</td><td valign="top"><bold>&#8722;0.424 (&lt;0.001)</bold></td><td valign="top">&#8722;0.337 (0.007)</td><td valign="top">&#8722;0.232 (0.068)</td><td valign="top"><bold>&#8722;0.375 (0.003)</bold></td><td valign="top"><bold>&#8722;0.448 (&lt;0.001)</bold></td><td valign="top"><bold>&#8722;0.370 (0.003)</bold></td><td valign="top">&#8722;0.272 (0.031)</td><td valign="top">&#8722;0.324 (0.010)</td></tr><tr><td valign="top">tau IC 8</td><td valign="top">&#8722;0.064 (0.619)</td><td valign="top">&#8722;0.094 (0.464)</td><td valign="top">&#8722;0.245 (0.053)</td><td valign="top">&#8722;0.323 (0.010)</td><td valign="top">&#8722;0.234 (0.065)</td><td valign="top">&#8722;0.049 (0.702)</td><td valign="top">&#8722;0.222 (0.080)</td><td valign="top">&#8722;0.319 (0.011)</td><td valign="top">&#8722;0.215 (0.091)</td><td valign="top">&#8722;0.171 (0.180)</td><td valign="top">&#8722;0.179 (0.160)</td></tr><tr><td valign="top">tau IC 10</td><td valign="top">&#8722;0.258 (0.041)</td><td valign="top">&#8722;0.121 (0.344)</td><td valign="top">&#8722;0.212 (0.096)</td><td valign="top">&#8722;0.304 (0.016)</td><td valign="top">&#8722;0.324 (0.010)</td><td valign="top">&#8722;0.136 (0.287)</td><td valign="top">&#8722;0.311 (0.013)</td><td valign="top"><bold>&#8722;0.427 (&lt;0.001)</bold></td><td valign="top">&#8722;0.350 (0.005)</td><td valign="top">&#8722;0.250 (0.048)</td><td valign="top">&#8722;0.223 (0.081)</td></tr><tr><td valign="top">tau IC 11</td><td valign="top">&#8722;0.166 (0.192)</td><td valign="top">&#8722;0.188 (0.140)</td><td valign="top">&#8722;0.300 (0.017)</td><td valign="top">&#8722;0.306 (0.015)</td><td valign="top">&#8722;0.208 (0.102)</td><td valign="top">&#8722;0.145 (0.257)</td><td valign="top">&#8722;0.310 (0.013)</td><td valign="top"><bold>&#8722;0.392 (0.002)</bold></td><td valign="top">&#8722;0.347 (0.005)</td><td valign="top">&#8722;0.271 (0.032)</td><td valign="top">&#8722;0.299 (0.017)</td></tr><tr><th rowspan="2" valign="top"/><th colspan="11" valign="top">Mean Diffusivity</th></tr><tr><th valign="top"><bold>ATR</bold></th><th valign="top"><bold>CST</bold></th><th valign="top"><bold>CCT</bold></th><th valign="top">HCT</th><th valign="top">FMAJ</th><th valign="top">FMIN</th><th valign="top">IFO</th><th valign="top">ILF</th><th valign="top">SLF</th><th valign="top">SLFt</th><th valign="top">UNC</th></tr><tr><td valign="top">amyloid-&#946; IC 2</td><td valign="top">0.031 (0.808)</td><td valign="top">&#8722;0.156 (0.223)</td><td valign="top">&#8722;0.061 (0.636)</td><td valign="top">0.011 (0.931)</td><td valign="top">&#8722;0.132 (0.302)</td><td valign="top">0.048 (0.712)</td><td valign="top">&#8722;0.072 (0.573)</td><td valign="top">0.011 (0.930)</td><td valign="top">&#8722;0.042 (0.742)</td><td valign="top">0.087 (0.500)</td><td valign="top">0.078 (0.543)</td></tr><tr><td valign="top">amyloid-&#946; IC 3</td><td valign="top">0.053 (0.679)</td><td valign="top">&#8722;0.362 (0.004)</td><td valign="top">&#8722;0.154 (0.227)</td><td valign="top">0.020 (0.879)</td><td valign="top">&#8722;0.216 (0.089)</td><td valign="top">&#8722;0.051 (0.691)</td><td valign="top">&#8722;0.108 (0.398)</td><td valign="top">&#8722;0.028 (0.829)</td><td valign="top">&#8722;0.168 (0.188)</td><td valign="top">0.082 (0.525)</td><td valign="top">0.065 (0.613)</td></tr><tr><td valign="top">amyloid-&#946; IC 4</td><td valign="top">0.067 (0.601)</td><td valign="top">&#8722;0.143 (0.263)</td><td valign="top">&#8722;0.029 (0.824)</td><td valign="top">0.092 (0.476)</td><td valign="top">0.043 (0.739)</td><td valign="top">&#8722;0.055 (0.669)</td><td valign="top">0.039 (0.763)</td><td valign="top">0.088 (0.495)</td><td valign="top">0.011 (0.935)</td><td valign="top">0.101 (0.433)</td><td valign="top">0.034 (0.793)</td></tr><tr><td valign="top">amyloid-&#946; IC 5</td><td valign="top">0.029 (0.820)</td><td valign="top">&#8722;0.120 (0.348)</td><td valign="top">0.001 (0.995)</td><td valign="top">0.014 (0.915)</td><td valign="top">0.003 (0.983)</td><td valign="top">&#8722;0.029 (0.823)</td><td valign="top">&#8722;0.018 (0.886)</td><td valign="top">0.024 (0.850)</td><td valign="top">&#8722;0.057 (0.656)</td><td valign="top">0.059 (0.645)</td><td valign="top">0.029 (0.823)</td></tr><tr><td valign="top">amyloid-&#946; IC 6</td><td valign="top">0.128 (0.318)</td><td valign="top">&#8722;0.278 (0.028)</td><td valign="top">&#8722;0.085 (0.508)</td><td valign="top">0.088 (0.493)</td><td valign="top">&#8722;0.125 (0.331)</td><td valign="top">0.021 (0.869)</td><td valign="top">&#8722;0.016 (0.899)</td><td valign="top">0.050 (0.698)</td><td valign="top">&#8722;0.083 (0.519)</td><td valign="top">0.148 (0.249)</td><td valign="top">0.122 (0.342)</td></tr><tr><td valign="top">amyloid-&#946; IC 7</td><td valign="top">0.081 (0.530)</td><td valign="top">&#8722;0.088 (0.493)</td><td valign="top">0.085 (0.507)</td><td valign="top">0.096 (0.456)</td><td valign="top">&#8722;0.057 (0.655)</td><td valign="top">0.108 (0.401)</td><td valign="top">0.066 (0.605)</td><td valign="top">0.141 (0.272)</td><td valign="top">0.106 (0.409)</td><td valign="top">0.108 (0.398)</td><td valign="top">0.118 (0.358)</td></tr><tr><td valign="top">amyloid-&#946; IC 8</td><td valign="top">0.011 (0.932)</td><td valign="top">&#8722;0.115 (0.368)</td><td valign="top">&#8722;0.046 (0.718)</td><td valign="top">&#8722;0.005 (0.972)</td><td valign="top">0.037 (0.771)</td><td valign="top">&#8722;0.009 (0.943)</td><td valign="top">&#8722;0.038 (0.767)</td><td valign="top">0.025 (0.847)</td><td valign="top">&#8722;0.085 (0.507)</td><td valign="top">0.009 (0.944)</td><td valign="top">0.015 (0.907)</td></tr><tr><td valign="top">amyloid-&#946; IC 10</td><td valign="top">0.088 (0.494)</td><td valign="top">&#8722;0.310 (0.013)</td><td valign="top">&#8722;0.112 (0.381)</td><td valign="top">0.057 (0.657)</td><td valign="top">&#8722;0.211 (0.097)</td><td valign="top">&#8722;0.056 (0.661)</td><td valign="top">&#8722;0.068 (0.595)</td><td valign="top">&#8722;0.025 (0.844)</td><td valign="top">&#8722;0.142 (0.266)</td><td valign="top">0.107 (0.403)</td><td valign="top">0.063 (0.625)</td></tr><tr><td valign="top">amyloid-&#946; IC 11</td><td valign="top">0.049 (0.703)</td><td valign="top">&#8722;0.096 (0.453)</td><td valign="top">0.034 (0.791)</td><td valign="top">0.023 (0.858)</td><td valign="top">&#8722;0.238 (0.060)</td><td valign="top">0.125 (0.329)</td><td valign="top">&#8722;0.053 (0.679)</td><td valign="top">0.037 (0.771)</td><td valign="top">0.068 (0.598)</td><td valign="top">0.102 (0.428)</td><td valign="top">0.134 (0.296)</td></tr><tr><td valign="top">tau IC 2</td><td valign="top">0.266 (0.035)</td><td valign="top">0.039 (0.760)</td><td valign="top">0.324 (0.010)</td><td valign="top">0.353 (0.005)</td><td valign="top">&#8722;0.011 (0.930)</td><td valign="top">0.244 (0.054)</td><td valign="top">0.271 (0.032)</td><td valign="top">0.307 (0.015)</td><td valign="top">0.183 (0.152)</td><td valign="top">0.432 <break/>(&lt;0.001)</td><td valign="top">0.303 (0.016)</td></tr><tr><td valign="top">tau IC 3</td><td valign="top">0.219 (0.085)</td><td valign="top">0.016 (0.900)</td><td valign="top">0.289 (0.022)</td><td valign="top">0.354 (0.004)</td><td valign="top">0.041 (0.748)</td><td valign="top">0.080 (0.532)</td><td valign="top">0.250 (0.048)</td><td valign="top">0.331 (0.008)</td><td valign="top">0.171 (0.181)</td><td valign="top">0.351 (0.005)</td><td valign="top">0.183 (0.150)</td></tr><tr><td valign="top">tau IC 4</td><td valign="top">0.172 (0.177)</td><td valign="top">0.054 (0.673)</td><td valign="top">0.280 (0.026)</td><td valign="top">0.222 (0.081)</td><td valign="top">0.016 (0.899)</td><td valign="top">0.164 (0.201)</td><td valign="top">0.236 (0.062)</td><td valign="top">0.243 (0.055)</td><td valign="top">0.093 (0.471)</td><td valign="top">0.303 (0.016)</td><td valign="top">0.259 (0.040)</td></tr><tr><td valign="top">tau IC 5</td><td valign="top">0.228 (0.072)</td><td valign="top">0.195 (0.126)</td><td valign="top"> <bold>0.387 (0.002)</bold></td><td valign="top">0.260 (0.039)</td><td valign="top">0.072 (0.576)</td><td valign="top">0.186 (0.145)</td><td valign="top">0.362 (0.004)</td><td valign="top">0.348 (0.005)</td><td valign="top">0.262 (0.038)</td><td valign="top"><bold>0.436 (&lt;0.001)</bold></td><td valign="top">0.337 (0.007)</td></tr><tr><td valign="top">tau IC 6</td><td valign="top">0.226 (0.075)</td><td valign="top">&#8722;0.006 (0.960)</td><td valign="top">0.261 (0.039)</td><td valign="top">0.265 (0.036)</td><td valign="top">&#8722;0.074 (0.565)</td><td valign="top">0.210 (0.099)</td><td valign="top">0.242 (0.056)</td><td valign="top">0.245 (0.054)</td><td valign="top">0.116 (0.365)</td><td valign="top"><bold>0.382 (0.002)</bold></td><td valign="top">0.298 (0.018)</td></tr><tr><td valign="top">tau IC 7</td><td valign="top">0.261 (0.039)</td><td valign="top">0.083 (0.519)</td><td valign="top"> <bold>0.418 (&lt;0.001)</bold></td><td valign="top"><bold>0.371 (0.003)</bold></td><td valign="top">0.086 (0.501)</td><td valign="top">0.243 (0.055)</td><td valign="top"> <bold>0.371 (0.003)</bold></td><td valign="top"> <bold>0.403 (0.001)</bold></td><td valign="top">0.288 (0.022)</td><td valign="top"><bold>0.463 (&lt;0.001)</bold></td><td valign="top">0.319 (0.011)</td></tr><tr><td valign="top">tau IC8</td><td valign="top">0.107 (0.405)</td><td valign="top">0.084 (0.514)</td><td valign="top">0.311 (0.013)</td><td valign="top">0.253 (0.046)</td><td valign="top">0.025 (0.848)</td><td valign="top">0.148 (0.248)</td><td valign="top">0.202 (0.112)</td><td valign="top">0.232 (0.067)</td><td valign="top">0.158 (0.216)</td><td valign="top">0.281 (0.026)</td><td valign="top">0.174 (0.172)</td></tr><tr><td valign="top">tau IC 10</td><td valign="top">0.252 (0.046)</td><td valign="top">&#8722;0.054 (0.672)</td><td valign="top">0.302 (0.016)</td><td valign="top">0.327 (0.009)</td><td valign="top">&#8722;0.052 (0.685)</td><td valign="top">0.212 (0.096)</td><td valign="top">0.225 (0.076)</td><td valign="top">0.239 (0.059)</td><td valign="top">0.074 (0.563)</td><td valign="top"><bold>0.388 (0.002)</bold></td><td valign="top">0.285 (0.024)</td></tr><tr><td valign="top">tau IC 11</td><td valign="top">0.193 (0.129)</td><td valign="top">0.056 (0.663)</td><td valign="top">0.314 (0.012)</td><td valign="top">0.258 (0.042)</td><td valign="top">&#8722;0.075 (0.557)</td><td valign="top">0.259 (0.040)</td><td valign="top">0.226 (0.075)</td><td valign="top">0.299 (0.017)</td><td valign="top">0.199 (0.118)</td><td valign="top"><bold>0.431 (&lt;0.001)</bold></td><td valign="top">0.350 (0.005)</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-50832-v2.xml">elife-50832-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><object-id pub-id-type="doi">10.7554/eLife.50832.006</object-id><label>Table 1.</label><caption><title>Rate of NPC2-mediated membrane interaction greatly increases in the presence of LBPA.</title><p>The effect of surface residue mutations on the ability of NPC2 to induce vesicle-vesicle interactions was assessed by measuring absorbance at 350 nm (light scattering) of 200 &#181;M LUVs in the presence of 1 &#181;M WT or mutant NPC2 protein, as described under Materials&#160;and&#160;methods. Rates of vesicle-vesicle interactions, indicated by increases in A350nm over time, were determined by a three-parameter hyperbolic fit of the data using Sigma Plot software, and are representative of at least three individual experiments. Mutants with substantially attenuated rates of membrane aggregation are indicated in italics.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2">100% EPC</th><th colspan="3">25%LBA/EPC SUV</th></tr><tr><th>Absolute rate (s<sup>1-</sup>)</th><th>Relative rate</th><th>Absolute rate (s<sup>&#8722;1</sup>)</th><th>Relative to WT with LBPA</th><th>Relative to WT with EPC</th></tr></thead><tbody><tr><td>WT</td><td>0.0112&#160;&#177;&#160;0.0006</td><td>1.00&#160;&#177;&#160;0.05</td><td>0.1674&#160;&#177;&#160;0.0122</td><td>1.00&#160;&#177;&#160;0.07</td><td>14.95&#160;&#177;&#160;1.09</td></tr><tr><td>H31A</td><td>0.0018&#160;&#177;&#160;0.0005</td><td><italic>0.16</italic>&#160;&#177;&#160;<italic>0.05</italic></td><td>0.1262&#160;&#177;&#160;0.0079</td><td>0.75&#160;&#177;&#160;0.05</td><td>11.27&#160;&#177;&#160;0.71</td></tr><tr><td>D113A</td><td>&lt;0.0001</td><td><italic>&lt;0.01</italic></td><td>0.1573&#160;&#177;&#160;0.0112</td><td>0.94&#160;&#177;&#160;0.07</td><td>14.04&#160;&#177;&#160;1.00</td></tr><tr><td>Q29A</td><td>0.0029&#160;&#177;&#160;0.0008</td><td><italic>0.26</italic>&#160;&#177;&#160;<italic>0.07</italic></td><td>0.1337&#160;&#177;&#160;0.0039</td><td>0.80&#160;&#177;&#160;0.02</td><td>11.93&#160;&#177;&#160;0.35</td></tr><tr><td>E108A</td><td>0.0032&#160;&#177;&#160;0.0004</td><td><italic>0.29</italic>&#160;&#177;&#160;<italic>0.03</italic></td><td>0.1437&#160;&#177;&#160;0.0094</td><td>0.86&#160;&#177;&#160;0.06</td><td>12.83&#160;&#177;&#160;0.84</td></tr><tr><td>D72A</td><td>0.0032&#160;&#177;&#160;0.0003</td><td><italic>0.29</italic>&#160;&#177;&#160;<italic>0.03</italic></td><td>0.1456&#160;&#177;&#160;0.0077</td><td>0.87&#160;&#177;&#160;0.05</td><td>13.00&#160;&#177;&#160;0.69</td></tr><tr><td>H56A</td><td>0.0107&#160;&#177;&#160;0.0008</td><td>0.95&#160;&#177;&#160;0.07</td><td>0.0261&#160;&#177;&#160;0.0055</td><td><italic>0.16</italic>&#160;&#177;&#160;<italic>0.09</italic></td><td><italic>2.33</italic>&#160;&#177;&#160;<italic>0.49</italic></td></tr><tr><td>G57D</td><td>0.0103&#160;&#177;&#160;0.0016</td><td>0.92&#160;&#177;&#160;0.14</td><td>0.0202&#160;&#177;&#160;0.0066</td><td><italic>0.13</italic>&#160;&#177;&#160;<italic>0.12</italic></td><td><italic>1.81</italic>&#160;&#177;&#160;<italic>0.59</italic></td></tr><tr><td>I58A</td><td>0.0094&#160;&#177;&#160;0.0011</td><td>0.840&#160;&#177;&#160;0.10</td><td>0.0240&#160;&#177;&#160;0.0037</td><td><italic>0.15</italic>&#160;&#177;&#160;<italic>0.06</italic></td><td><italic>2.14</italic>&#160;&#177;&#160;<italic>0.33</italic></td></tr><tr><td>G61A</td><td>0.0075&#160;&#177;&#160;0.0006</td><td><italic>0.67</italic>&#160;&#177;&#160;<italic>0.06</italic></td><td>0.0406&#160;&#177;&#160;0.0012</td><td><italic>0.25</italic>&#160;&#177;&#160;<italic>0.03</italic></td><td><italic>3.63</italic>&#160;&#177;&#160;<italic>0.11</italic></td></tr><tr><td>I62D</td><td>&lt;0.0001</td><td><italic>&lt;0.01</italic></td><td>0.0167&#160;&#177;&#160;0.0017</td><td><italic>0.10</italic>&#160;&#177;&#160;<italic>0.03</italic></td><td><italic>1.50</italic>&#160;&#177;&#160;<italic>0.15</italic></td></tr><tr><td>V64A</td><td>&lt;0.0001</td><td><italic>&lt;0.01</italic></td><td>0.0193&#160;&#177;&#160;0.0040</td><td><italic>0.12</italic>&#160;&#177;&#160;<italic>0.06</italic></td><td><italic>1.72</italic>&#160;&#177;&#160;<italic>0.36</italic></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-50988-v3.xml">elife-50988-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Measurements of spindle dynamics in male meiosis.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Distance</th><th rowspan="2">Spindle parameter</th><th colspan="2">Meiosis I</th><th colspan="2">Meiosis II</th></tr><tr><th>&#8195;Mean</th><th>SD</th><th>Mean</th><th>SD</th></tr></thead><tbody><tr><td rowspan="4"><bold>P-P<sup>1</sup></bold></td><td>Initial spindle length (metaphase)</td><td>4.1 &#181;m</td><td>&#177;0.3 &#181;m</td><td>4.2 &#181;m</td><td>&#177;0.4 &#181;m</td></tr><tr><td>&#8195;Final spindle length (end of anaphase)</td><td>8.0 &#181;m</td><td>&#177;0.6 &#181;m</td><td>8.8 &#181;m</td><td>&#177;0.8 &#181;m</td></tr><tr><td>&#8195;Initial rate (1 st minute)</td><td>1.29 &#956;m/min</td><td>&#177;0.36 &#956;m/min</td><td>1.11 &#956;m/min</td><td>&#177;0.42 &#956;m/min</td></tr><tr><td>&#8195;Duration of elongation</td><td>3&#8211;4 min</td><td/><td>8&#8211;9 min</td><td/></tr><tr><td rowspan="4"><bold>A-A<sup>2</sup></bold></td><td>Initial spindle length (metaphase)</td><td>0.9 &#181;m</td><td>&#177;0.2 &#181;m</td><td>0.6 &#181;m</td><td>&#177;0.2 &#181;m</td></tr><tr><td>&#8195;Final spindle length (end of anaphase)</td><td>6.5 &#181;m</td><td>&#177;0.4 &#181;m</td><td>8.0 &#181;m</td><td>&#177;0.8 &#181;m</td></tr><tr><td>&#8195;Initial rate (1 st minute)</td><td>2.07 &#181;m/min</td><td>&#177;0.37 &#956;m/min</td><td>2.16 &#181;m/min</td><td>&#177;0.32 &#956;m/min</td></tr><tr><td>&#8195;Duration of elongation</td><td>4&#8211;5 min</td><td/><td>8&#8211;9 min</td><td/></tr><tr><td rowspan="3"><bold>P-A<sup>3</sup></bold></td><td>Initial spindle length (metaphase)</td><td>1.6 &#181;m</td><td>&#177;0.3 &#181;m</td><td>2.0 &#181;m</td><td>&#177;0.4 &#181;m</td></tr><tr><td>&#8195;Final spindle length (end of anaphase)</td><td>0.8 &#181;m</td><td>&#177;0.3 &#181;m</td><td>0.4 &#181;m</td><td>&#177;0.2 &#181;m</td></tr><tr><td>&#8195;Initial rate (1 st minute)</td><td>&#8722;0.39 &#181;m/min</td><td>&#177;0.27 &#956;m/min</td><td>&#8722;0.64 &#181;m/min</td><td>&#177;0.33 &#956;m/min</td></tr></tbody></table><table-wrap-foot><fn><p>Distances: <sup>1</sup>P-P, pole-to-pole distance; <sup>2</sup>A-A, autosome-to-autosome distance; <sup>3</sup>P-A, pole-to-autosome distance. Initial spindle length is given at metaphase, final distance refers to the end of anaphase when spindle elongation plateaus. Values are given as mean values (&#177;&#160;standard deviation, SD). The numbers of analyzed spindles are: n&#160;=&#160;31 for meiosis I; n&#160;=&#160;50 for meiosis II.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-50988-v4.xml">elife-50988-v4.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Measurements of spindle dynamics in male meiosis.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Distance</th><th rowspan="2">Spindle parameter</th><th colspan="2">Meiosis I</th><th colspan="2">Meiosis II</th></tr><tr><th>&#8195;Mean</th><th>SD</th><th>Mean</th><th>SD</th></tr></thead><tbody><tr><td rowspan="4"><bold>P-P<sup>1</sup></bold></td><td>Initial spindle length (metaphase)</td><td>4.1 &#181;m</td><td>&#177;0.3 &#181;m</td><td>4.2 &#181;m</td><td>&#177;0.4 &#181;m</td></tr><tr><td>&#8195;Final spindle length (end of anaphase)</td><td>8.0 &#181;m</td><td>&#177;0.6 &#181;m</td><td>8.8 &#181;m</td><td>&#177;0.8 &#181;m</td></tr><tr><td>&#8195;Initial rate (1 st minute)</td><td>1.29 &#956;m/min</td><td>&#177;0.36 &#956;m/min</td><td>1.11 &#956;m/min</td><td>&#177;0.42 &#956;m/min</td></tr><tr><td>&#8195;Duration of elongation</td><td>3&#8211;4 min</td><td/><td>8&#8211;9 min</td><td/></tr><tr><td rowspan="4"><bold>A-A<sup>2</sup></bold></td><td>Initial spindle length (metaphase)</td><td>0.9 &#181;m</td><td>&#177;0.2 &#181;m</td><td>0.6 &#181;m</td><td>&#177;0.2 &#181;m</td></tr><tr><td>&#8195;Final spindle length (end of anaphase)</td><td>6.5 &#181;m</td><td>&#177;0.4 &#181;m</td><td>8.0 &#181;m</td><td>&#177;0.8 &#181;m</td></tr><tr><td>&#8195;Initial rate (1 st minute)</td><td>2.07 &#181;m/min</td><td>&#177;0.37 &#956;m/min</td><td>2.16 &#181;m/min</td><td>&#177;0.32 &#956;m/min</td></tr><tr><td>&#8195;Duration of elongation</td><td>4&#8211;5 min</td><td/><td>8&#8211;9 min</td><td/></tr><tr><td rowspan="3"><bold>P-A<sup>3</sup></bold></td><td>Initial spindle length (metaphase)</td><td>1.6 &#181;m</td><td>&#177;0.3 &#181;m</td><td>2.0 &#181;m</td><td>&#177;0.4 &#181;m</td></tr><tr><td>&#8195;Final spindle length (end of anaphase)</td><td>0.8 &#181;m</td><td>&#177;0.3 &#181;m</td><td>0.4 &#181;m</td><td>&#177;0.2 &#181;m</td></tr><tr><td>&#8195;Initial rate (1 st minute)</td><td>&#8722;0.39 &#181;m/min</td><td>&#177;0.27 &#956;m/min</td><td>&#8722;0.64 &#181;m/min</td><td>&#177;0.33 &#956;m/min</td></tr></tbody></table><table-wrap-foot><fn><p>Distances: <sup>1</sup>P-P, pole-to-pole distance; <sup>2</sup>A-A, autosome-to-autosome distance; <sup>3</sup>P-A, pole-to-autosome distance. Initial spindle length is given at metaphase, final distance refers to the end of anaphase when spindle elongation plateaus. Values are given as mean values (&#177;&#160;standard deviation, SD). The numbers of analyzed spindles are: n&#160;=&#160;31 for meiosis I; n&#160;=&#160;50 for meiosis II.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-51243-v1.xml">elife-51243-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Summary of genetic variation across 151 isolates of the <italic>L. donovani</italic> complex for previously described loci involved in resistance or treatment failure of antimonial drugs and Miltefosine.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">locus/complex</th><th colspan="3" valign="top">gene id</th><th rowspan="2" valign="top">gene name</th><th rowspan="2" valign="top">function prediction</th><th rowspan="2" valign="top">involved in resistance (R)/treatment failure (TF) to drug:</th><th rowspan="2" valign="top">reference</th><th rowspan="2" valign="top">evidence from reference</th><th rowspan="2" valign="top">gene copy number (gene CN)</th></tr><tr><th valign="top"><italic>L. infantum</italic>, JPCM5, v41</th><th valign="top"><italic>L. infantum</italic>, JPCM5, v38</th><th valign="top"><italic>L. donovani</italic> ortholog, BPK282A1, v41</th></tr></thead><tbody><tr><td rowspan="5" valign="top">H-locus</td><td valign="top">LINF_230007700</td><td valign="top">LinJ.23.0280</td><td valign="top">LdBPK_230280</td><td valign="top">terbinafine resistance gene (HTBF), (YIP1)</td><td valign="top"/><td valign="top">Antimonials (R)</td><td valign="top"><xref ref-type="bibr" rid="bib22">Callahan and Beverley, 1991</xref>; <break/><xref ref-type="bibr" rid="bib33">Dias et al., 2007</xref></td><td valign="top">The <italic>Leishmania</italic> H region is frequently amplified in drug-resistant lines and is associated with metal resistance (genes YIP1, MRPA, PTR1).</td><td rowspan="5" valign="top">Genes have an increased CN in 30% (CN +1 to +44), and reduced CN in 9% (CN &#8722;1). <break/>37% of all samples have an insertion including at least three genes (always YIP1, MRPA and argininosuccinate synthase). These amplifications are in groups Ldon1 (42/45), Ldon3 (13/19) and Ldon5 (1/8). The insertion boundaries in isolates from groups Ldon1 and Ldon3 are shared (<xref ref-type="fig" rid="fig9s1">Figure 9&#8212;figure supplement 1A</xref>).</td></tr><tr><td valign="top">LINF_230007800</td><td valign="top">LinJ.23.0290</td><td valign="top">LdBPK_230290</td><td valign="top">P-glycoprotein A (MRPA); pentamidine resistance protein 1</td><td valign="top">ATP-binding cassette (ABC) transporter, ABC-thiol transporter</td><td valign="top">Antimonials (R)</td><td valign="top"><xref ref-type="bibr" rid="bib22">Callahan and Beverley, 1991</xref>; <break/><xref ref-type="bibr" rid="bib33">Dias et al., 2007</xref>; <xref ref-type="bibr" rid="bib65">Leprohon et al., 2009</xref></td><td valign="top">Increased expression of MRPA is often due to the amplification of its gene in antimony-resistant strains.</td></tr><tr><td valign="top">LINF_230007900</td><td valign="top">LinJ.23.0300</td><td valign="top">LdBPK_230300</td><td valign="top"/><td valign="top">argininosuccinate synthase - putative</td><td valign="top">Antimonials</td><td valign="top"><xref ref-type="bibr" rid="bib50">Grondin et al., 1993</xref>; <break/><xref ref-type="bibr" rid="bib65">Leprohon et al., 2009</xref></td><td valign="top"/></tr><tr><td rowspan="2" valign="top">LINF_230008000</td><td rowspan="2" valign="top">LinJ.23.0310</td><td rowspan="2" valign="top">LdBPK_230310</td><td rowspan="2" valign="top">Pteridine reductase 1 (PTR1)</td><td rowspan="2" valign="top"/><td valign="top">Antimonials (R)</td><td valign="top"><xref ref-type="bibr" rid="bib22">Callahan and Beverley, 1991</xref>; <break/><xref ref-type="bibr" rid="bib33">Dias et al., 2007</xref></td><td valign="top">see above, evidence only for H-locus in general</td></tr><tr><td valign="top">Antifolate (R)</td><td valign="top"><xref ref-type="bibr" rid="bib123">Vickers and Beverley, 2011</xref></td><td valign="top"><italic>Leishmania</italic> salvage folate from their hosts. Thereby folates are reduced by a DHFR (dihydrofolate reductase)-TS (thymidylate synthase) and PTR1. PTR1 can act as a metabolic bypass of DHFR inhibition, reducing the effectiveness of existing antifolate drugs.</td></tr><tr><td valign="top">Mitogen-activated protein kinase, MAPK1</td><td valign="top">LINF_360076200</td><td valign="top">LinJ.36.6760</td><td valign="top">LdBPK_366760</td><td valign="top">LMPK, mitogen-activated protein kinase</td><td valign="top">protein phosphorylation</td><td valign="top">Antimonials (R)</td><td valign="top"><xref ref-type="bibr" rid="bib105">Singh et al., 2010</xref>; <xref ref-type="bibr" rid="bib13">Ashutosh et al., 2012</xref></td><td valign="top">Conflicting evidence between up- and down-regulation of Mitogen-Activated Protein Kinase one between different studies.</td><td valign="top">45% of all isolates showed an increased CN, with all isolates of Ldon1 andLdon3 being affected and smaller fractions in other <italic>L. donovani</italic> groups (<xref ref-type="fig" rid="fig9s2">Figure 9&#8212;figure supplement 2A</xref>).</td></tr><tr><td valign="top">Aqua-glyceroporin, AQP1</td><td valign="top">LINF_310005100</td><td valign="top">LinJ.31.0030</td><td valign="top">LdBPK_310030</td><td valign="top">Aquaglyceroporin 1, AQP1</td><td valign="top">drug transmembrane transport</td><td valign="top">Antimonials (R)</td><td valign="top"><xref ref-type="bibr" rid="bib48">Gourbal et al., 2004</xref>; <break/><xref ref-type="bibr" rid="bib121">Uzcategui et al., 2008</xref>; <break/><xref ref-type="bibr" rid="bib74">Monte-Neto et al., 2015</xref>; <break/><xref ref-type="bibr" rid="bib10">Andrade et al., 2016</xref>; <break/><xref ref-type="bibr" rid="bib55">Imamura et al., 2016</xref></td><td valign="top">A frequently resistant <italic>L. donovani</italic> population has a two base-pair insertion in AQP1 preventing antimonial transport. <break/>Increased resistance with decrease in gene CN or expression, while increase leads to higher drug sensitivity.</td><td valign="top">Gene CN deletions and insertions of small effect sizes (CN &#8722;2 to &#8722;1 and +1 to +3) are present in 6% and 35% of isolates but never leading to loss of the locus.</td></tr><tr><td rowspan="3" valign="top">Miltefosine transporter and associated genes</td><td valign="top">LINF_130020800</td><td valign="top">LinJ.13.1590</td><td valign="top">LdBPK_131590</td><td valign="top">Miltefosine transporter, LdMT</td><td valign="top">phospholipid transport</td><td valign="top">Miltefosine (R)</td><td valign="top"><xref ref-type="bibr" rid="bib81">P&#233;rez-Victoria et al., 2006</xref>; <break/><xref ref-type="bibr" rid="bib103">Shaw et al., 2016</xref></td><td valign="top">Gene deletion or different changes in two different strains evolved in promastigote culture for Miltefosine resistance. strain Sb-S: locus deletion and A691P; strain Sb-R: E197D</td><td valign="top">15 isolates: +1 gene CNV (CUK, Ldon1, Ldon2, Ldon3, Ldon5)</td></tr><tr><td valign="top">LINF_130020900</td><td valign="top">LinJ.13.1600</td><td valign="top">LdBPK_131600</td><td valign="top">hypothetical protein</td><td valign="top">unknown function</td><td valign="top">Miltefosine (R)</td><td valign="top"><xref ref-type="bibr" rid="bib103">Shaw et al., 2016</xref></td><td valign="top">Deleted along with the Miltefosine transporter gene in a single line evolved for Miltefosine resistance in promastigote culture.</td><td valign="top">three isolates: +1 gene CNV (Ldon1, Linf1)</td></tr><tr><td valign="top">LINF_320015500</td><td valign="top">LinJ.32.1040</td><td valign="top">LdBPK_321040</td><td valign="top">Ros3, LdRos3</td><td valign="top">Vps23 core domain containing protein - putative</td><td valign="top">Miltefosine (R)</td><td valign="top"><xref ref-type="bibr" rid="bib81">P&#233;rez-Victoria et al., 2006</xref></td><td valign="top">Putative subunit of LdMT; LdMT and LdRos3 seem to form part of the same translocation machinery that determines flippase activity and Miltefosine sensitivity in <italic>Leishmania</italic>.</td><td valign="top">one isolate: +1&#160;gene CNV (Ldon1)</td></tr><tr><td rowspan="4" valign="top">Miltefosine sensitivity locus, MSL</td><td valign="top">LINF_310031200</td><td valign="top">LinJ.31.2370</td><td valign="top">LdBPK_312380</td><td valign="top"/><td valign="top">3'-nucleotidase/nuclease - putative</td><td valign="top">Miltefosine (TF)</td><td valign="top"><xref ref-type="bibr" rid="bib24">Carnielli et al., 2018</xref></td><td rowspan="4" valign="top">MSL: a deletion of this locus was associated with Miltefosine treatment failure in Brazil. While the frequency of the MSL was still relatively high in the North-East it was almost absent in the South-East of Brazil, and it was absent in <italic>L.infantum</italic>/<italic>L.donovani</italic> in the Old World.</td><td rowspan="4" valign="top">Genes have a reduced CN in 55% (CN &#8722;1 to &#8722;8) and increased in 4% (CN +1). <break/>Four isolates, show a complete loss of the MSL at identical boundaries: WC, Cha001, HN167 and HN336 (2/4 isolates from Brazil, 2/2 isolates from Honduras). Two isolates show a reduction of all four genes at this locus but with various deletion boundaries: IMT373cl1 (Portugal), CH35 (Cyprus) (<xref ref-type="fig" rid="fig9">Figure 9B</xref>).</td></tr><tr><td valign="top">LINF_310031300</td><td valign="top">LinJ.31.2380</td><td valign="top">LdBPK_312380</td><td valign="top"/><td valign="top">3'-nucleotidase/nuclease - putative</td><td valign="top">Miltefosine (TF)</td><td valign="top"><xref ref-type="bibr" rid="bib24">Carnielli et al., 2018</xref></td></tr><tr><td valign="top">LINF_310031400</td><td valign="top">LinJ.31.2390</td><td valign="top">LdBPK_312390</td><td valign="top"/><td valign="top">helicase-like protein</td><td valign="top">Miltefosine (TF)</td><td valign="top"><xref ref-type="bibr" rid="bib24">Carnielli et al., 2018</xref></td></tr><tr><td valign="top">LINF_310031500</td><td valign="top">LinJ.31.2400</td><td valign="top">LdBPK_312320, LdBPK_312400</td><td valign="top"/><td valign="top">3&#8211;2-trans-enoyl-CoA isomerase - mitochondrial precursor - putative</td><td valign="top">Miltefosine (TF)</td><td valign="top"><xref ref-type="bibr" rid="bib24">Carnielli et al., 2018</xref></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-51243-v2.xml">elife-51243-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Summary of genetic variation across 151 isolates of the <italic>L. donovani</italic> complex for previously described loci involved in resistance or treatment failure of antimonial drugs and Miltefosine.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">locus/complex</th><th colspan="3" valign="top">gene id</th><th rowspan="2" valign="top">gene name</th><th rowspan="2" valign="top">function prediction</th><th rowspan="2" valign="top">involved in resistance (R)/treatment failure (TF) to drug:</th><th rowspan="2" valign="top">reference</th><th rowspan="2" valign="top">evidence from reference</th><th rowspan="2" valign="top">gene copy number (gene CN)</th></tr><tr><th valign="top"><italic>L. infantum</italic>, JPCM5, v41</th><th valign="top"><italic>L. infantum</italic>, JPCM5, v38</th><th valign="top"><italic>L. donovani</italic> ortholog, BPK282A1, v41</th></tr></thead><tbody><tr><td rowspan="5" valign="top">H-locus</td><td valign="top">LINF_230007700</td><td valign="top">LinJ.23.0280</td><td valign="top">LdBPK_230280</td><td valign="top">terbinafine resistance gene (HTBF), (YIP1)</td><td valign="top"/><td valign="top">Antimonials (R)</td><td valign="top"><xref ref-type="bibr" rid="bib22">Callahan and Beverley, 1991</xref>; <break/><xref ref-type="bibr" rid="bib33">Dias et al., 2007</xref></td><td valign="top">The <italic>Leishmania</italic> H region is frequently amplified in drug-resistant lines and is associated with metal resistance (genes YIP1, MRPA, PTR1).</td><td rowspan="5" valign="top">Genes have an increased CN in 30% (CN +1 to +44), and reduced CN in 9% (CN &#8722;1). <break/>37% of all samples have an insertion including at least three genes (always YIP1, MRPA and argininosuccinate synthase). These amplifications are in groups Ldon1 (42/45), Ldon3 (13/19) and Ldon5 (1/8). The insertion boundaries in isolates from groups Ldon1 and Ldon3 are shared (<xref ref-type="fig" rid="fig9s1">Figure 9&#8212;figure supplement 1A</xref>).</td></tr><tr><td valign="top">LINF_230007800</td><td valign="top">LinJ.23.0290</td><td valign="top">LdBPK_230290</td><td valign="top">P-glycoprotein A (MRPA); pentamidine resistance protein 1</td><td valign="top">ATP-binding cassette (ABC) transporter, ABC-thiol transporter</td><td valign="top">Antimonials (R)</td><td valign="top"><xref ref-type="bibr" rid="bib22">Callahan and Beverley, 1991</xref>; <break/><xref ref-type="bibr" rid="bib33">Dias et al., 2007</xref>; <xref ref-type="bibr" rid="bib65">Leprohon et al., 2009</xref></td><td valign="top">Increased expression of MRPA is often due to the amplification of its gene in antimony-resistant strains.</td></tr><tr><td valign="top">LINF_230007900</td><td valign="top">LinJ.23.0300</td><td valign="top">LdBPK_230300</td><td valign="top"/><td valign="top">argininosuccinate synthase - putative</td><td valign="top">Antimonials</td><td valign="top"><xref ref-type="bibr" rid="bib50">Grondin et al., 1993</xref>; <break/><xref ref-type="bibr" rid="bib65">Leprohon et al., 2009</xref></td><td valign="top"/></tr><tr><td rowspan="2" valign="top">LINF_230008000</td><td rowspan="2" valign="top">LinJ.23.0310</td><td rowspan="2" valign="top">LdBPK_230310</td><td rowspan="2" valign="top">Pteridine reductase 1 (PTR1)</td><td rowspan="2" valign="top"/><td valign="top">Antimonials (R)</td><td valign="top"><xref ref-type="bibr" rid="bib22">Callahan and Beverley, 1991</xref>; <break/><xref ref-type="bibr" rid="bib33">Dias et al., 2007</xref></td><td valign="top">see above, evidence only for H-locus in general</td></tr><tr><td valign="top">Antifolate (R)</td><td valign="top"><xref ref-type="bibr" rid="bib123">Vickers and Beverley, 2011</xref></td><td valign="top"><italic>Leishmania</italic> salvage folate from their hosts. Thereby folates are reduced by a DHFR (dihydrofolate reductase)-TS (thymidylate synthase) and PTR1. PTR1 can act as a metabolic bypass of DHFR inhibition, reducing the effectiveness of existing antifolate drugs.</td></tr><tr><td valign="top">Mitogen-activated protein kinase, MAPK1</td><td valign="top">LINF_360076200</td><td valign="top">LinJ.36.6760</td><td valign="top">LdBPK_366760</td><td valign="top">LMPK, mitogen-activated protein kinase</td><td valign="top">protein phosphorylation</td><td valign="top">Antimonials (R)</td><td valign="top"><xref ref-type="bibr" rid="bib105">Singh et al., 2010</xref>; <xref ref-type="bibr" rid="bib13">Ashutosh et al., 2012</xref></td><td valign="top">Conflicting evidence between up- and down-regulation of Mitogen-Activated Protein Kinase one between different studies.</td><td valign="top">45% of all isolates showed an increased CN, with all isolates of Ldon1 andLdon3 being affected and smaller fractions in other <italic>L. donovani</italic> groups (<xref ref-type="fig" rid="fig9s2">Figure 9&#8212;figure supplement 2A</xref>).</td></tr><tr><td valign="top">Aqua-glyceroporin, AQP1</td><td valign="top">LINF_310005100</td><td valign="top">LinJ.31.0030</td><td valign="top">LdBPK_310030</td><td valign="top">Aquaglyceroporin 1, AQP1</td><td valign="top">drug transmembrane transport</td><td valign="top">Antimonials (R)</td><td valign="top"><xref ref-type="bibr" rid="bib48">Gourbal et al., 2004</xref>; <break/><xref ref-type="bibr" rid="bib121">Uzcategui et al., 2008</xref>; <break/><xref ref-type="bibr" rid="bib74">Monte-Neto et al., 2015</xref>; <break/><xref ref-type="bibr" rid="bib10">Andrade et al., 2016</xref>; <break/><xref ref-type="bibr" rid="bib55">Imamura et al., 2016</xref></td><td valign="top">A frequently resistant <italic>L. donovani</italic> population has a two base-pair insertion in AQP1 preventing antimonial transport. <break/>Increased resistance with decrease in gene CN or expression, while increase leads to higher drug sensitivity.</td><td valign="top">Gene CN deletions and insertions of small effect sizes (CN &#8722;2 to &#8722;1 and +1 to +3) are present in 6% and 35% of isolates but never leading to loss of the locus.</td></tr><tr><td rowspan="3" valign="top">Miltefosine transporter and associated genes</td><td valign="top">LINF_130020800</td><td valign="top">LinJ.13.1590</td><td valign="top">LdBPK_131590</td><td valign="top">Miltefosine transporter, LdMT</td><td valign="top">phospholipid transport</td><td valign="top">Miltefosine (R)</td><td valign="top"><xref ref-type="bibr" rid="bib81">P&#233;rez-Victoria et al., 2006</xref>; <break/><xref ref-type="bibr" rid="bib103">Shaw et al., 2016</xref></td><td valign="top">Gene deletion or different changes in two different strains evolved in promastigote culture for Miltefosine resistance. strain Sb-S: locus deletion and A691P; strain Sb-R: E197D</td><td valign="top">15 isolates: +1 gene CNV (CUK, Ldon1, Ldon2, Ldon3, Ldon5)</td></tr><tr><td valign="top">LINF_130020900</td><td valign="top">LinJ.13.1600</td><td valign="top">LdBPK_131600</td><td valign="top">hypothetical protein</td><td valign="top">unknown function</td><td valign="top">Miltefosine (R)</td><td valign="top"><xref ref-type="bibr" rid="bib103">Shaw et al., 2016</xref></td><td valign="top">Deleted along with the Miltefosine transporter gene in a single line evolved for Miltefosine resistance in promastigote culture.</td><td valign="top">three isolates: +1 gene CNV (Ldon1, Linf1)</td></tr><tr><td valign="top">LINF_320015500</td><td valign="top">LinJ.32.1040</td><td valign="top">LdBPK_321040</td><td valign="top">Ros3, LdRos3</td><td valign="top">Vps23 core domain containing protein - putative</td><td valign="top">Miltefosine (R)</td><td valign="top"><xref ref-type="bibr" rid="bib81">P&#233;rez-Victoria et al., 2006</xref></td><td valign="top">Putative subunit of LdMT; LdMT and LdRos3 seem to form part of the same translocation machinery that determines flippase activity and Miltefosine sensitivity in <italic>Leishmania</italic>.</td><td valign="top">one isolate: +1&#160;gene CNV (Ldon1)</td></tr><tr><td rowspan="4" valign="top">Miltefosine sensitivity locus, MSL</td><td valign="top">LINF_310031200</td><td valign="top">LinJ.31.2370</td><td valign="top">LdBPK_312380</td><td valign="top"/><td valign="top">3'-nucleotidase/nuclease - putative</td><td valign="top">Miltefosine (TF)</td><td valign="top"><xref ref-type="bibr" rid="bib24">Carnielli et al., 2018</xref></td><td rowspan="4" valign="top">MSL: a deletion of this locus was associated with Miltefosine treatment failure in Brazil. While the frequency of the MSL was still relatively high in the North-East it was almost absent in the South-East of Brazil, and it was absent in <italic>L.infantum</italic>/<italic>L.donovani</italic> in the Old World.</td><td rowspan="4" valign="top">Genes have a reduced CN in 55% (CN &#8722;1 to &#8722;8) and increased in 4% (CN +1). <break/>Four isolates, show a complete loss of the MSL at identical boundaries: WC, Cha001, HN167 and HN336 (2/4 isolates from Brazil, 2/2 isolates from Honduras). Two isolates show a reduction of all four genes at this locus but with various deletion boundaries: IMT373cl1 (Portugal), CH35 (Cyprus) (<xref ref-type="fig" rid="fig9">Figure 9B</xref>).</td></tr><tr><td valign="top">LINF_310031300</td><td valign="top">LinJ.31.2380</td><td valign="top">LdBPK_312380</td><td valign="top"/><td valign="top">3'-nucleotidase/nuclease - putative</td><td valign="top">Miltefosine (TF)</td><td valign="top"><xref ref-type="bibr" rid="bib24">Carnielli et al., 2018</xref></td></tr><tr><td valign="top">LINF_310031400</td><td valign="top">LinJ.31.2390</td><td valign="top">LdBPK_312390</td><td valign="top"/><td valign="top">helicase-like protein</td><td valign="top">Miltefosine (TF)</td><td valign="top"><xref ref-type="bibr" rid="bib24">Carnielli et al., 2018</xref></td></tr><tr><td valign="top">LINF_310031500</td><td valign="top">LinJ.31.2400</td><td valign="top">LdBPK_312320, LdBPK_312400</td><td valign="top"/><td valign="top">3&#8211;2-trans-enoyl-CoA isomerase - mitochondrial precursor - putative</td><td valign="top">Miltefosine (TF)</td><td valign="top"><xref ref-type="bibr" rid="bib24">Carnielli et al., 2018</xref></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-51407-v2.xml">elife-51407-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Antibiotic susceptibility profiles of CRO<sup>RS</sup> strains with RNAP mutations.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Strain</th><th colspan="8" valign="top">MIC (&#956;g/ml)</th></tr><tr><th valign="top">CRO</th><th valign="top">CFX</th><th valign="top">PEN</th><th valign="top">EPM</th><th valign="top">AZI</th><th valign="top">TET</th><th valign="top">CIP</th><th valign="top">RIF</th></tr></thead><tbody><tr><td valign="top">GCGS0457</td><td valign="top">0.012</td><td valign="top">0.016</td><td valign="top">2</td><td valign="top">0.016</td><td valign="top">0.25</td><td valign="top">1</td><td valign="top">&#8804;0.015</td><td valign="top">0.125</td></tr><tr><td valign="top">GCGS1095</td><td valign="top">0.19</td><td valign="top">1</td><td valign="top">2</td><td valign="top">0.032</td><td valign="top">0.5</td><td valign="top">1</td><td valign="top">&#8804;0.015</td><td valign="top">&#8804;0.0625</td></tr><tr><td valign="top">GCGS1014</td><td valign="top">0.125</td><td valign="top">0.5</td><td valign="top">1.5</td><td valign="top">0.032</td><td valign="top">0.25</td><td valign="top">1</td><td valign="top">&#8804;0.015</td><td valign="top">&#8804;0.0625</td></tr><tr><td valign="top">GCGS1013</td><td valign="top">0.19</td><td valign="top">0.5</td><td valign="top">1.5</td><td valign="top">0.023</td><td valign="top">0.5</td><td valign="top">1</td><td valign="top">&#8804;0.015</td><td valign="top">&#8804;0.0625</td></tr><tr><td valign="top">GCGS0457 RpoB<sup>R201H</sup></td><td valign="top">0.19</td><td valign="top">&gt;0.5</td><td valign="top">2</td><td valign="top">0.047</td><td valign="top">0.25</td><td valign="top">1</td><td valign="top">&#8804;0.015</td><td valign="top">&#8804;0.0625</td></tr><tr><td valign="top">GCGS0457 RpoD<sup>E98K</sup></td><td valign="top">0.125</td><td valign="top">0.5</td><td valign="top">1.5</td><td valign="top">0.023</td><td valign="top">0.25</td><td valign="top">1</td><td valign="top">&#8804;0.015</td><td valign="top">&#8804;0.0625</td></tr><tr><td valign="top">GCGS0457 RpoD<sup>&#916;92-95</sup></td><td valign="top">0.19</td><td valign="top">0.5</td><td valign="top">2</td><td valign="top">0.023</td><td valign="top">0.25</td><td valign="top">1</td><td valign="top">&#8804;0.015</td><td valign="top">&#8804;0.0625</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-51492-v1.xml">elife-51492-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Mass measurements from native MS analysis of PCAT1 samples.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Sample</th><th/><th colspan="5"><italic>In UDM</italic></th><th colspan="5"><italic>In C<sub>8</sub>E<sub>4</sub></italic></th></tr><tr><th>Expected mass (Da)</th><th colspan="3">Measured Mass&#160;&#177;&#160;SD<sup>a</sup> (Da)</th><th>Mass error (%)<sup>b</sup></th><th/><th colspan="3">Measured Mass&#160;&#177;&#160;SD<sup>a</sup> (Da)</th><th>Mass error (%)<sup>b</sup></th><th/></tr></thead><tbody><tr><td>wt PCAT1 dimer</td><td>162,176</td><td>162,175</td><td>&#177;</td><td>1</td><td>0.0008</td><td>-</td><td>162,171</td><td>&#177;</td><td>1</td><td>0.003</td><td>-</td></tr><tr><td>CtA</td><td valign="bottom">10,207.9</td><td valign="bottom">10,206.5</td><td valign="bottom">&#177;</td><td valign="bottom">0.5</td><td valign="bottom">0.014</td><td/><td valign="bottom">10,206.5</td><td valign="bottom">&#177;</td><td valign="bottom">0.3</td><td>0.014</td><td/></tr><tr><td>-</td><td valign="bottom">-</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom">-</td><td>% Abun- dance<sup>c</sup></td><td valign="bottom">-</td><td valign="bottom"/><td valign="bottom"/><td>-</td><td>% Abun- dance<sup>c</sup></td></tr><tr><td>C21A-PCAT1 dimer + 1 CtA</td><td>172,320</td><td>172,337</td><td>&#177;</td><td>4</td><td>0.010</td><td>25&#160;&#177;&#160;2</td><td>172,341</td><td>&#177;</td><td>3</td><td>0.012</td><td>16&#160;&#177;&#160;2</td></tr><tr><td>C21A-PCAT1 dimer + 2 CtA</td><td>182,528</td><td>182,549</td><td>&#177;</td><td>3</td><td>0.012</td><td>75&#160;&#177;&#160;2</td><td>182,558</td><td>&#177;</td><td>6</td><td>0.017</td><td>84&#160;&#177;&#160;2</td></tr></tbody></table><table-wrap-foot><fn><p><sup>a</sup> From the average and S.D. of the mass values calculated from each charge state peak within a charge-state distribution (n&#160;&#8805;&#160;4 charge states).</p><p><sup>b</sup> Mass accuracy measured by the relative difference between measured and expected masses divided by the expected mass.</p></fn><fn><p><sup>c</sup> Based on relative peak intensities for the peak series corresponding to the 182 kDa and 172 kDa complexes within the same MS spectrum and corrected for gas-phase dissociation (see <xref ref-type="fig" rid="fig1s2">Figure 1&#8212;figure supplement 2</xref> for more details) (n&#160;&#8805;&#160;3 conditions).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-51507-v1.xml">elife-51507-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><label>Table 3.</label><caption><title>Characteristics of baseline (first available) measurements in 845 individuals with information on at least one BA.</title></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top"/><th rowspan="2" valign="top">Baseline measurements in all individuals</th><th colspan="6" valign="top">Baseline measurements in individuals with corresponding BAs</th></tr><tr><th valign="top"/><th valign="top">Telomere length <break/>(T/S ratio)</th><th valign="top">DNAmAge <break/>(year)</th><th valign="top">Physiological age <break/>(year)</th><th valign="top">Cognitive function</th><th valign="top">FAI</th><th valign="top">FI</th></tr></thead><tbody><tr><td valign="top">Number of participants</td><td valign="top">845</td><td valign="top">636</td><td valign="top">387</td><td valign="top">802</td><td valign="top">829</td><td valign="top">739</td><td valign="top">756</td></tr><tr><td valign="top">Women (%)</td><td valign="top">59.5</td><td valign="top">58.5</td><td valign="top">59.9</td><td valign="top">58.9</td><td valign="top">59.6</td><td valign="top">59</td><td valign="top">59.4</td></tr><tr><td valign="top">Above primary education (%)</td><td valign="top">42.2</td><td valign="top">45.4</td><td valign="top">45.5</td><td valign="top">41.8</td><td valign="top">42.6</td><td valign="top">43.7</td><td valign="top">43.3</td></tr><tr><td valign="top">BMI (kg/m<sup>2</sup>)</td><td valign="top">25.7 (3.9)</td><td valign="top">26.3 (4.1)</td><td valign="top">26.3 (4.3)</td><td valign="top">25.6 (3.9)</td><td valign="top">25.7 (3.9)</td><td valign="top">25.8 (4)</td><td valign="top">25.8 (4.1)</td></tr><tr><td valign="top">Current and ex-smokers (%)</td><td valign="top">25.0</td><td valign="top">22.2</td><td valign="top">21.7</td><td valign="top">24.1</td><td valign="top">25.5</td><td valign="top">24.1</td><td valign="top">23.7</td></tr><tr><td valign="top">Age (year)</td><td valign="top">63.6 (8.6)</td><td valign="top">68.8 (9.6)</td><td valign="top">69 (9.6)</td><td valign="top">64.5 (8.9)</td><td valign="top">63.7 (8.3)</td><td valign="top">65.3 (9.2)</td><td valign="top">65.5 (9.4)</td></tr><tr><td valign="top">BA</td><td valign="top"/><td valign="top">0.73 (0.17)</td><td valign="top">60.4 (11.0) <break/>65.2 (10.1) <break/>63.8 (13.61) <break/>69.4 (8.5)</td><td valign="top">64.7 (10.3)</td><td valign="top">51.5 (10.4)</td><td valign="top">48.3 (11.4)</td><td valign="top">0.10 (0.08)</td></tr><tr><td valign="top">Number of measurements</td><td valign="top"/><td valign="top">2.5 (1.3)</td><td valign="top">2.7 (1.3)</td><td valign="top">4.0 (2.2)</td><td valign="top">3.7 (2.1)</td><td valign="top">4.0 (2.4)</td><td valign="top">4.2 (2.5)</td></tr></tbody></table><table-wrap-foot><fn><p>Values are means (standard deviations; SDs) unless stated otherwise.</p><p>Values of &#8216;BA&#8217; in the &#8216;DNAmAge&#8217; column refer to four different types of DNAmAge: Horvath, Hannum, PhenoAge, and GrimAge, respectively.</p></fn><fn><p>BA, biological age; IPT, In-person testing; DNAmAge, DNA methylation age estimator; FAI, functional aging index; FI, frailty index; BMI, body mass index.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-51850-v1.xml">elife-51850-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Expression of the predicted drug target genes across <italic>B. malayi</italic> life stages.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top">Target gene</th><th colspan="10" valign="top">Life stage expression (FPKM)</th></tr><tr><th valign="top">L3</th><th valign="top">L3D6</th><th valign="top">L3D9</th><th valign="top">L4</th><th valign="top">F30</th><th valign="top">F42</th><th valign="top">F120</th><th valign="top">M30</th><th valign="top">M42</th><th valign="top">M120</th></tr></thead><tbody><tr><td valign="top">Fosmidomycin</td><td valign="top">Wbm0179</td><td valign="top">12</td><td valign="top">17</td><td valign="top">18</td><td valign="top">11</td><td valign="top">27</td><td valign="top">11</td><td valign="top">16</td><td valign="top">37</td><td valign="top">31</td><td valign="top">3</td></tr><tr><td rowspan="2" valign="top">MDL-29951</td><td valign="top">Bm13850</td><td valign="top">45</td><td valign="top">21</td><td valign="top">22</td><td valign="top">30</td><td valign="top">22</td><td valign="top">24</td><td valign="top">6</td><td valign="top">31</td><td valign="top">17</td><td valign="top">14</td></tr><tr><td valign="top">Wbm0158</td><td valign="top">68</td><td valign="top">66</td><td valign="top">69</td><td valign="top">49</td><td valign="top">27</td><td valign="top">40</td><td valign="top">80</td><td valign="top">29</td><td valign="top">40</td><td valign="top">95</td></tr><tr><td rowspan="2" valign="top">Tenofovir</td><td valign="top">Bm9070</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">0</td><td valign="top">1</td><td valign="top">1</td><td valign="top">1</td><td valign="top">4</td><td valign="top">26</td></tr><tr><td valign="top">Wbm0321</td><td valign="top">34</td><td valign="top">13</td><td valign="top">14</td><td valign="top">4</td><td valign="top">5</td><td valign="top">3</td><td valign="top">15</td><td valign="top">0</td><td valign="top">18</td><td valign="top">6</td></tr><tr><td rowspan="2" valign="top">Tenofovir-associated</td><td valign="top">Bm3965</td><td valign="top">18</td><td valign="top">19</td><td valign="top">20</td><td valign="top">11</td><td valign="top">57</td><td valign="top">33</td><td valign="top">14</td><td valign="top">15</td><td valign="top">19</td><td valign="top">17</td></tr><tr><td valign="top">Bm14014</td><td valign="top">419</td><td valign="top">101</td><td valign="top">119</td><td valign="top">105</td><td valign="top">76</td><td valign="top">93</td><td valign="top">21</td><td valign="top">75</td><td valign="top">49</td><td valign="top">32</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-52189-v2.xml">elife-52189-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><label>Table 4.</label><caption><title>Clustering results of the OBS conformations sampled in the D<sub>2</sub>R/eticlopride and D<sub>2</sub>R/risperidone simulations.</title><p>The compositions in each cluster are shown as percentages of the frames randomly extracted for each complex (see Materials&#160;and&#160;methods), when sorted by either receptor/ligand complex or EL2 conformation.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="4">Cluster ID</th><th colspan="8" valign="bottom">Percentage (%)</th></tr><tr><th colspan="4" valign="bottom">Complex</th><th colspan="4" valign="bottom">EL2 conformation</th></tr><tr><th colspan="2" valign="bottom">D<sub>2</sub>R/eticlopride</th><th colspan="2" valign="bottom">D<sub>2</sub>R/risperidone</th><th colspan="2" valign="bottom">Extended</th><th colspan="2" valign="bottom">Helical</th></tr><tr><th valign="bottom">&#8195;Mean</th><th valign="bottom">Sd</th><th valign="bottom">Mean</th><th valign="bottom">Sd</th><th valign="bottom">Mean</th><th valign="bottom">Sd</th><th valign="bottom">Mean</th><th valign="bottom">Sd</th></tr></thead><tbody><tr><td valign="bottom"><bold>1</bold></td><td valign="bottom">38.4</td><td valign="bottom">0.7</td><td valign="bottom">0.0</td><td valign="bottom">0.0</td><td valign="bottom">4.9</td><td valign="bottom">0.4</td><td valign="bottom">33.5</td><td valign="bottom">0.5</td></tr><tr><td valign="bottom"><bold>2</bold></td><td valign="bottom">61.6</td><td valign="bottom">0.7</td><td valign="bottom">0.0</td><td valign="bottom">0.0</td><td valign="bottom">45.1</td><td valign="bottom">0.4</td><td valign="bottom">16.5</td><td valign="bottom">0.6</td></tr><tr><td valign="bottom"><bold>3</bold></td><td valign="bottom">0.0</td><td valign="bottom">0.0</td><td valign="bottom">43.7</td><td valign="bottom">1.0</td><td valign="bottom">2.5</td><td valign="bottom">0.4</td><td valign="bottom">41.3</td><td valign="bottom">0.8</td></tr><tr><td valign="bottom"><bold>4</bold></td><td valign="bottom">0.0</td><td valign="bottom">0.0</td><td valign="bottom">56.3</td><td valign="bottom">1.0</td><td valign="bottom">47.5</td><td valign="bottom">0.4</td><td valign="bottom">8.7</td><td valign="bottom">0.8</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-52384-v2.xml">elife-52384-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><label>Table 3.</label><caption><title>Antibody sequence characteristics from patient 013 unstimulated and stimulated PBMCs related to clonal families of single plasmablasts with reactivity to DENV.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Clonal family <break/>(antibodies)</th><th rowspan="2">Germline</th><th rowspan="2">Isotype</th><th align="left" colspan="2">Reads for all clonotypes</th></tr><tr><th align="center">Unstimulated</th><th align="center">Stimulated</th></tr></thead><tbody><tr><td valign="top">10</td><td valign="top">IGHV3-30-3</td><td valign="top">Total</td><td valign="top">143</td><td valign="top">216292</td></tr><tr><td valign="top">(C1, A7, I11, L9, P2, G5)</td><td valign="top"/><td valign="top">IgM</td><td valign="top">8</td><td valign="top">427</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgG</td><td valign="top">112</td><td valign="top">199151</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgA</td><td valign="top">23</td><td valign="top">16714</td></tr><tr><td valign="top">5</td><td valign="top">IGHV1-69</td><td valign="top">total</td><td valign="top">422</td><td valign="top">141308</td></tr><tr><td valign="top">(C4, J2, N2)</td><td valign="top"/><td valign="top">IgM</td><td valign="top">9</td><td valign="top">91</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgG</td><td valign="top">413</td><td valign="top">140117</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgA</td><td valign="top">0</td><td valign="top">1100</td></tr><tr><td valign="top">13</td><td valign="top">IGHV4-34</td><td valign="top">total</td><td valign="top">32</td><td valign="top">82814</td></tr><tr><td valign="top">(N8, F4)</td><td valign="top"/><td valign="top">IgM</td><td valign="top">0</td><td valign="top">195</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgG</td><td valign="top">32</td><td valign="top">76605</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgA</td><td valign="top">0</td><td valign="top">6014</td></tr><tr><td valign="top">7</td><td valign="top">IGHV1-69</td><td valign="top">total</td><td valign="top">579</td><td valign="top">43179</td></tr><tr><td valign="top">(J8, J9)</td><td valign="top"/><td valign="top">IgM</td><td valign="top">10</td><td valign="top">98</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgG</td><td valign="top">568</td><td valign="top">42315</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgA</td><td valign="top">1</td><td valign="top">764</td></tr><tr><td valign="top">9</td><td valign="top">IGHV1-69</td><td valign="top">total</td><td valign="top">673</td><td valign="top">35793</td></tr><tr><td valign="top">(K11, L3, M4, M11, O4)</td><td valign="top"/><td valign="top">IgM</td><td valign="bottom">12</td><td valign="bottom">103</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgG</td><td valign="bottom">656</td><td valign="bottom">34444</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgA</td><td valign="bottom">5</td><td valign="bottom">1246</td></tr><tr><td valign="top">8</td><td valign="top">IGHV1-69</td><td valign="top">total</td><td valign="top">10</td><td valign="top">7349</td></tr><tr><td valign="top">(J3, 403_P4)</td><td valign="top"/><td valign="top">IgM</td><td valign="top">0</td><td valign="top">16</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgG</td><td valign="top">10</td><td valign="top">7001</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgA</td><td valign="top">0</td><td valign="top">332</td></tr><tr><td valign="top">3</td><td valign="top">IGHV1-18</td><td valign="top">total</td><td valign="top">10</td><td valign="top">2993</td></tr><tr><td valign="top">(405_P4, I13)</td><td valign="top"/><td valign="top">IgM</td><td valign="bottom">1</td><td valign="bottom">6</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgG</td><td valign="bottom">9</td><td valign="bottom">2899</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgA</td><td valign="bottom">0</td><td valign="bottom">88</td></tr><tr><td valign="top">1</td><td valign="top">IGHV4-39</td><td valign="top">total</td><td valign="top">4</td><td valign="top">1087</td></tr><tr><td valign="top">(B10, M1, D8)</td><td valign="top"/><td valign="top">IgM</td><td valign="top">1</td><td valign="top">11</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgG</td><td valign="top">3</td><td valign="top">1049</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgA</td><td valign="top">0</td><td valign="top">27</td></tr><tr><td valign="top">2</td><td valign="top">IGHV1-69</td><td valign="top">total</td><td valign="top">0</td><td valign="top">1087</td></tr><tr><td valign="top">(H3, M6)</td><td valign="top"/><td valign="top">IgM</td><td valign="top">0</td><td valign="top">7</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgG</td><td valign="top">0</td><td valign="top">1066</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgA</td><td valign="top">0</td><td valign="top">14</td></tr><tr><td valign="top">11</td><td valign="top">IGHV3-30-3</td><td valign="top">total</td><td valign="top">0</td><td valign="top">19</td></tr><tr><td valign="top">(E9, I8)</td><td valign="top"/><td valign="top">IgM</td><td valign="top">0</td><td valign="top">0</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgG</td><td valign="top">0</td><td valign="top">19</td></tr><tr><td valign="top"/><td valign="top"/><td valign="top">IgA</td><td valign="top">0</td><td valign="top">0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-52426-v2.xml">elife-52426-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><label>Table 4.</label><caption><title>Frequency of different types of link engagement per day.</title><p>The average number of times per day that readers of WPM and W pages engaged with external links using one of the event types captured from March 22<sup>nd</sup> to April 22<sup>nd</sup>, 2019.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Event type</th><th align="center" colspan="3" valign="bottom">WPM</th><th align="center" colspan="3" valign="bottom">W</th></tr><tr><th valign="bottom">&#8195;Total (%)</th><th valign="bottom">Desktop (%)</th><th valign="bottom">Mobile (%)</th><th valign="bottom">Total (%)</th><th valign="bottom">Desktop (%)</th><th valign="bottom">Mobile (%)</th></tr></thead><tbody><tr><td valign="bottom">Hover over link</td><td valign="bottom">48,748.9 (46.9)</td><td valign="bottom">45,814.8 (60.3)</td><td valign="bottom">2,934.1 (10.5)</td><td valign="bottom">1,122,704.0 (32.7)</td><td valign="bottom">1,057,982.0 (47.2)</td><td valign="bottom">64,722.0 (5.4)</td></tr><tr><td valign="bottom">Footnote click</td><td valign="bottom">27,739.4 (26.7)</td><td valign="bottom">10,948.8 (14.4)</td><td valign="bottom">16,790.6 (60.3)</td><td valign="bottom">722,131.0 (21.0)</td><td valign="bottom">235,245.0 (10.5)</td><td valign="bottom">486,886.7 (40.7)</td></tr><tr><td valign="bottom">External click</td><td valign="bottom">25,811.9 (24.9)</td><td valign="bottom">17,792.3 (23.4)</td><td valign="bottom">8,019.7 (28.8)</td><td valign="bottom">1,557,125.0 (45.3)</td><td valign="bottom">915,445.1 (40.9)</td><td valign="bottom">641,676.4 (53.6)</td></tr><tr><td valign="bottom">Up click</td><td valign="bottom">1,539.5 (1.5)</td><td valign="bottom">1,422.8 (1.9)</td><td valign="bottom">116.4 (0.4)</td><td valign="bottom">34,738.0 (1.0)</td><td valign="bottom">31,230.1 (1.4)</td><td valign="bottom">3,508.1 (0.3)</td></tr><tr><td valign="bottom">All events</td><td valign="bottom">103,839.7 (100)</td><td valign="bottom">75,978.7 (100)</td><td valign="bottom">27,860.8 (100)</td><td valign="bottom">3,436,698.0 (100)</td><td valign="bottom">2,239,902.2 (100)</td><td valign="bottom">1,196,793.2 (100)</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-52426-v2.xml">elife-52426-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table5" position="float"><label>Table 5.</label><caption><title>Number of pageviews per engagement event.</title><p>The frequency of each event per day was divided by the average number of daily pageviews for WPM and W pages from March 22<sup>nd</sup> to April 22<sup>nd</sup>, 2019. The lower the number of pageviews per event the greater the event frequency. Difference between each pair of WPM and W distributions is statistically significant as derived from Fisher&#8217;s exact test (p&lt;0.001, two-tailed).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Pageviews/event</th><th align="center" colspan="3" valign="bottom">WPM</th><th align="center" colspan="3" valign="bottom">W</th></tr><tr><th valign="bottom">&#8195;Total</th><th valign="bottom">Desktop</th><th valign="bottom">Mobile</th><th valign="bottom">Total</th><th valign="bottom">Desktop</th><th valign="bottom">Mobile</th></tr></thead><tbody><tr><td valign="bottom">Hover over link</td><td valign="bottom">120.5</td><td valign="bottom">42.7</td><td valign="bottom">1,335.2</td><td valign="bottom">203.5</td><td valign="bottom">92.6</td><td valign="bottom">2,016.1</td></tr><tr><td valign="bottom">Footnote click</td><td valign="bottom">211.8</td><td valign="bottom">178.8</td><td valign="bottom">233.3</td><td valign="bottom">316.3</td><td valign="bottom">416.4</td><td valign="bottom">268.0</td></tr><tr><td valign="bottom">External click</td><td valign="bottom">227.6</td><td valign="bottom">110.0</td><td valign="bottom">488.5</td><td valign="bottom">146.7</td><td valign="bottom">107.0</td><td valign="bottom">203.4</td></tr><tr><td valign="bottom">Up click</td><td valign="bottom">3,816.6</td><td valign="bottom">1,376.1</td><td valign="bottom">33,655.0</td><td valign="bottom">6,576.2</td><td valign="bottom">3,136.6</td><td valign="bottom">37,196.5</td></tr><tr><td valign="bottom">All events</td><td valign="bottom">56.6</td><td valign="bottom">25.8</td><td valign="bottom">140.6</td><td valign="bottom">66.5</td><td valign="bottom">43.7</td><td valign="bottom">109.0</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-52465-v2.xml">elife-52465-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="app4table1" position="float"><label>Appendix 4&#8212;table 1.</label><caption><title>Overview of SCR response quantification specifications (i.e., min. amplitude, scoring approach) and procedural details during fear acquisition training (i.e., number of CS and US presentations) as well as number (mean and range) and percentage of SCR non-responses towards the different stimuli (US, CS+, CS&#8211;, CS).</title><p>TTP: trough-to-peak; CS+E: CS+ extinguished; CS+U: CS+ unextinguished, CS: for both the CS+ and CS&#8211;.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Reference</th><th rowspan="2" valign="top"><italic>N</italic></th><th rowspan="2" valign="top">Minimum amplitude cutoff (in &#181;S) for valid SCRs</th><th rowspan="2" valign="top">Scoring details</th><th colspan="2" valign="top">Number of&#8230;</th><th colspan="8" valign="top">&#8216;Non-responses&#8217; towards&#8230;</th></tr><tr><th valign="bottom">US</th><th valign="bottom">CS (CS+/CS&#8211;)</th><th valign="bottom">US (<italic>M</italic>&#160;&#177;&#160;<italic>SD</italic>, range)</th><th valign="bottom">US (%)</th><th valign="bottom">CS+ (<italic>M</italic>&#160;&#177;&#160;<italic>SD</italic>, range)</th><th valign="bottom">CS+ (%)</th><th valign="bottom">CS&#8211; (<italic>M</italic>&#160;&#177;&#160;<italic>SD</italic>, range)</th><th valign="bottom">CS&#8211; (%)</th><th valign="bottom">CS (<italic>M</italic>&#160;&#177;&#160;<italic>SD</italic>, range)</th><th valign="bottom">CS (%)</th></tr></thead><tbody><tr><td valign="top"><xref ref-type="bibr" rid="bib33">Jentsch et al., 2020</xref></td><td valign="top">41</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (max peak), latency 0.5&#8211;4&#160;s/1&#8211;80.5 s (US/CS)</td><td valign="top">10</td><td valign="top">16/16</td><td valign="top">1.12&#160;&#177;&#160;1.66 (0&#8211;10)</td><td valign="top">11.22</td><td valign="top">2.22&#160;&#177;&#160;3.31 (0&#8211;16)</td><td valign="top">13.87</td><td valign="top">4.49&#160;&#177;&#160;3.92 (0&#8211;16)</td><td valign="top">28.05</td><td valign="top">6.71&#160;&#177;&#160;6.68 (0&#8211;32)</td><td valign="top">20.96</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib29">Hermann et al., 2016</xref></td><td valign="top">45</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (max peak), latency 0.5&#8211;6&#160;s/1&#8211;60.5 s (US/CS)</td><td valign="top">10 (5 for CS+E, 5 for CS+U)</td><td valign="top">8 CS+E/8 CS+U/16 CS&#8211;</td><td valign="top">0.24&#160;&#177;&#160;0.88 (0&#8211;5)</td><td valign="top">2.44</td><td valign="top">2.64&#160;&#177;&#160;3.49 (0&#8211;13); <break/>CS+E: 1.47&#160;&#177;&#160;2.19 (0&#8211;8); CS+U: 1.18&#160;&#177;&#160;1.80 (0&#8211;7)</td><td valign="top">16.53 <break/>CS+E: 18.33; <break/>CS+U: 14.72</td><td valign="top">8.07&#160;&#177;&#160;4.14 (0&#8211;16)</td><td valign="top">50.42</td><td valign="top">10.71&#160;&#177;&#160;6.65 (0&#8211;26)</td><td valign="top">33.47</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib47">Merz et al., 2018a</xref></td><td valign="top">39</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (max peak), latency 0.5&#8211;6&#160;s/1&#8211;60.5 s (US/CS)</td><td valign="top">10 (5 for CS+E, 5 for CS+U)</td><td valign="top">8 CS+E/8 CS+U/8 CS&#8211;</td><td valign="top">2.08&#160;&#177;&#160;1.98 (0&#8211;8)</td><td valign="top">20.77</td><td valign="top">3.36&#160;&#177;&#160;4.55 (0&#8211;16); <break/>CS+E: 1.59&#160;&#177;&#160;2.35 (0&#8211;8); <break/>CS+U: 1.77&#160;&#177;&#160;2.32 (0&#8211;8)</td><td valign="top">21.00; <break/>CS+E: 19.87; <break/>CS+U: 22.12</td><td valign="top">2.41&#160;&#177;&#160;2.27 (0&#8211;8)</td><td valign="top">30.13</td><td valign="top">5.77&#160;&#177;&#160;6.49 (0&#8211;24)</td><td valign="top">24.04</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib46">Merz et al., 2014</xref></td><td valign="top">40</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (max peak), latency 0.5&#8211;6&#160;s/1&#8211;60.5 s (US/CS)</td><td valign="top">10 (5 for CS+E, 5 for CS+U)</td><td valign="top">8 CS+E/8 CS+U/16 CS&#8211;</td><td valign="top">0.13&#160;&#177;&#160;0.33 (0&#8211;1)</td><td valign="top">1.25</td><td valign="top">1.08&#160;&#177;&#160;2.04 (0&#8211;11); <break/>CS+E: 0.58&#160;&#177;&#160;1.08 (0&#8211;5); <break/>CS+U: 0.50&#160;&#177;&#160;1.11 (0&#8211;6)</td><td valign="top">6.72; <break/>CS+E: 7.19 <break/>CS+U: 6.25</td><td valign="top">3.13&#160;&#177;&#160;2.96 (0&#8211;11)</td><td valign="top">19.53</td><td valign="top">4.20&#160;&#177;&#160;4.39 (0&#8211;21)</td><td valign="top">13.13</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib22">Hamacher-Dang et al., 2015</xref></td><td valign="top">39</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (max peak), latency 0.5&#8211;6&#160;s/1&#8211;60.5 s (US/CS)</td><td valign="top">10 (5 for CS+E, 5 for CS+U)</td><td valign="top">8 CS+E/8 CS+U/16 CS&#8211;</td><td valign="top">0.23&#160;&#177;&#160;0.48 (0&#8211;2)</td><td valign="top">2.31</td><td valign="top">2.33&#160;&#177;&#160;3.77 (0&#8211;12); <break/>CS+E: 1.31&#160;&#177;&#160;2.21 (0&#8211;8); <break/>CS+U: 1.03&#160;&#177;&#160;1.81 (0&#8211;7)</td><td valign="top">14.58; <break/>CS+E: 16.35; <break/>CS+U: 12.82</td><td valign="top">3.77&#160;&#177;&#160;4.20 (0&#8211;14)</td><td valign="top">23.56</td><td valign="top">6.10&#160;&#177;&#160;7.71 (0&#8211;26)</td><td valign="top">19.07</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib45">Mertens et al., 2019</xref></td><td valign="top">59</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (max peak), latency 1&#8211;8 s (baseline 0&#8211;2 s)</td><td valign="top">10</td><td valign="top">10/5</td><td valign="top">0.78&#160;&#177;&#160;1.69 (0&#8211;6)</td><td valign="top">7.8</td><td valign="top">4.75&#160;&#177;&#160;2.97 (0&#8211;10)</td><td valign="top">47.5</td><td valign="top">2.93&#160;&#177;&#160;1.66 (0&#8211;5)</td><td valign="top">58.6</td><td valign="top">7.68&#160;&#177;&#160;4.30 (0&#8211;15)</td><td valign="top">51.2</td></tr><tr><td valign="top">Klingel-h&#246;fer-Jens et al., unpublished</td><td valign="top">119</td><td valign="top">&#8805;0.01</td><td valign="top">TTP (first peak), latency 0.9&#8211;2.5&#160;s/3.5 s (US/CS)</td><td valign="top">14</td><td valign="top">14/14</td><td valign="top">1.40&#160;&#177;&#160;2.47 (0&#8211;14)</td><td valign="top">10.0</td><td valign="top">5.30&#160;&#177;&#160;4.42 (0&#8211;14)</td><td valign="top">37.9</td><td valign="top">8.20&#160;&#177;&#160;3.99 (0&#8211;14)</td><td valign="top">58.6</td><td valign="top">6.75&#160;&#177;&#160;4.44 (0&#8211;14)</td><td valign="top">48.2</td></tr><tr><td valign="top">Gerlicher et al. unpublished</td><td valign="top">52</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (first peak) latency 0.9&#8211;4 s</td><td valign="top">6</td><td valign="top">6/6</td><td valign="top">0.73&#160;&#177;&#160;1.39 (0&#8211;6)</td><td valign="top">12.18</td><td valign="top">2.73&#160;&#177;&#160;2.06 (0&#8211;6)</td><td valign="top">45.5</td><td valign="top">3.54&#160;&#177;&#160;1.82 (0&#8211;6)</td><td valign="top">59.0</td><td valign="top">6.27&#160;&#177;&#160;3.54 (0&#8211;12)</td><td valign="top">52.24</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib16">Gerlicher et al., 2018</xref></td><td valign="top">39</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (first peak) latency 0.9&#8211;4 s</td><td valign="top">5</td><td valign="top">10/10</td><td valign="top">0.33&#160;&#177;&#160;0.93 (0&#8211;5)</td><td valign="top">6.67</td><td valign="top">1.05&#160;&#177;&#160;2.21 (0&#8211;10)</td><td valign="top">10.51</td><td valign="top">2.36&#160;&#177;&#160;2.49 (0&#8211;10)</td><td valign="top">23.59</td><td valign="top">3.41&#160;&#177;&#160;4.48 (0&#8211;20)</td><td valign="top">17.05</td></tr><tr><td valign="top">Andreatta et al. unpublished</td><td valign="top">76</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (first peak) latency 0.8&#8211;4 s</td><td valign="top">16 (8 in analysis due to startle probes)</td><td valign="top">16/16 <break/>(8/8 in analysis due to startle probes)</td><td valign="top">1.34&#160;&#177;&#160;1.69 (0&#8211;8)</td><td valign="top">16.78</td><td valign="top">4.17&#160;&#177;&#160;2.30 (0&#8211;8)</td><td valign="top">52.14</td><td valign="top">5.00&#160;&#177;&#160;1.98 (0&#8211;8)</td><td valign="top">62.50</td><td valign="top">9.17&#160;&#177;&#160;3.77 (0&#8211;16)</td><td valign="top">57.32</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib78">Wendt et al., 2020</xref></td><td valign="top">112</td><td valign="top">&#8805;0.04</td><td valign="top">TTP (first peak), latency 0.9&#8211;4 s</td><td valign="top">9</td><td valign="top">12/12</td><td valign="top">0.46&#160;&#177;&#160;1.15 (0&#8211;7)</td><td valign="top">5.06</td><td valign="top">5.88&#160;&#177;&#160;3.63 (0&#8211;12)</td><td valign="top">48.96</td><td valign="top">7.06&#160;&#177;&#160;3.19 (0&#8211;12)</td><td valign="top">58.85</td><td valign="top">12.94&#160;&#177;&#160;6.39 (0&#8211;24)</td><td valign="top">53.91</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib77">Wendt et al., 2015</xref></td><td valign="top">108</td><td valign="top">&#8805;0.04</td><td valign="top">TTP (first peak), latency 0.9&#8211;4 s</td><td valign="top">12</td><td valign="top">12/12</td><td valign="top">0.27&#160;&#177;&#160;0.99 (0&#8211;8)</td><td valign="top">2.24</td><td valign="top">6.44&#160;&#177;&#160;3.81 (0&#8211;12)</td><td valign="top">53.63</td><td valign="top">8.53&#160;&#177;&#160;2.65 (0&#8211;12)</td><td valign="top">71.06</td><td valign="top">14.96&#160;&#177;&#160;6.04 (0&#8211;24)</td><td valign="top">62.35</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib10">Drexler et al., 2015</xref></td><td valign="top">46</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (max peak), latency 1&#8211;4.5 s</td><td valign="top">18</td><td valign="top">13 CS1+/13 CS2+/13 CS&#8211;</td><td valign="top">2.8&#160;&#177;&#160;4.18 (0&#8211;16)</td><td valign="top">15.57</td><td valign="top">9.67&#160;&#177;&#160;7.64 (0&#8211;26); <break/>CS1+: 4.87&#160;&#177;&#160;4.07 (0&#8211;13); <break/>CS2+: 4.80&#160;&#177;&#160;3.78 (0&#8211;13)</td><td valign="top">37.20; <break/>CS1+:37.45; <break/>CS2+: 36.95</td><td valign="top">5.26&#160;&#177;&#160;3.95 (0&#8211;13)</td><td valign="top">40.46</td><td valign="top">14.93&#160;&#177;&#160;11.37 (0&#8211;39)</td><td valign="top">38.29</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib42">Meir Drexler et al., 2016</xref></td><td valign="top">73</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (max peak), latency 1&#8211;4.5 s</td><td valign="top">18</td><td valign="top">13 CS1+/13 CS2+/13 CS&#8211;</td><td valign="top">3.37&#160;&#177;&#160;4.72 (0&#8211;18)</td><td valign="top">18.72</td><td valign="top">11.51&#160;&#177;&#160;7.96 (0&#8211;25); <break/>CS1+: 5.78&#160;&#177;&#160;3.97 (0&#8211;13); <break/>CS2+: 5.73&#160;&#177;&#160;4.22 (0&#8211;13)</td><td valign="top">44.25; <break/>CS1+:44.67; <break/>CS2+:44.04</td><td valign="top">6.29&#160;&#177;&#160;3.94 (0&#8211;13)</td><td valign="top">48.36</td><td valign="top">17.79&#160;&#177;&#160;11.67 (0&#8211;37)</td><td valign="top">45.62</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib44">Meir Drexler and Wolf, 2017</xref></td><td valign="top">72</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (max peak), latency 1&#8211;4.5 s</td><td valign="top">18</td><td valign="top">13 CS1+/13 CS2+/13 CS&#8211;</td><td valign="top">1.92&#160;&#177;&#160;2.96 (0&#8211;11)</td><td valign="top">10.64</td><td valign="top">9.65&#160;&#177;&#160;7.21 (0&#8211;25); <break/>CS1+: 4.78&#160;&#177;&#160;3.72 (0&#8211;12); <break/>CS2+: 4.88&#160;&#177;&#160;3.85 (0&#8211;13)</td><td valign="top">37.12; <break/>CS1+:&#160;36.75; <break/>CS2+:&#160;37.50</td><td valign="top">5.42&#160;&#177;&#160;3.54 (0&#8211;12)</td><td valign="top">41.66</td><td valign="top">15.07&#160;&#177;&#160;10.40 (0&#8211;36)</td><td valign="top">38.63</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib11">Drexler et al., 2018</xref></td><td valign="top">40</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (max peak), latency 1&#8211;4.5 s</td><td valign="top">10 (5 for CS+E, 5 for CS+U)</td><td valign="top">8 CS+E/8 CS+U/16 CS&#8211;</td><td valign="top">0.32&#160;&#177;&#160;0.69 (0&#8211;3)</td><td valign="top">3.25</td><td valign="top">4.17&#160;&#177;&#160;4.45 (0&#8211;16); <break/>CS+E: 2.02&#160;&#177;&#160;2.47 (0&#8211;8); <break/>CS+U: 2.15&#160;&#177;&#160;2.38 (0&#8211;8)</td><td valign="top">26.09; <break/>CS+E:&#160;25.31; <break/>CS+U: 26.87</td><td valign="top">6.07&#160;&#177;&#160;4.37 (0&#8211;16)</td><td valign="top">37.96</td><td valign="top">10.25&#160;&#177;&#160;8.24 (1-27)</td><td valign="top">32.03</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib43">Meir Drexler et al., 2019</xref></td><td valign="top">75</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (max peak), latency 0.5&#8211;6&#160;s/1&#8211;80.5 s (US/CS)</td><td valign="top">6</td><td valign="top">10/10</td><td valign="top">0.89&#160;&#177;&#160;01.57 (0&#8211;6)</td><td valign="top">14.88</td><td valign="top">4.07&#160;&#177;&#160;3.40 (0&#8211;10)</td><td valign="top">40.66</td><td valign="top">4.68&#160;&#177;&#160;3.23 (0&#8211;10)</td><td valign="top">46.8</td><td valign="top">8.75&#160;&#177;&#160;6.41 (0&#8211;20)</td><td valign="top">43.73</td></tr><tr><td valign="top">Chalkia et al., unpublished</td><td valign="top">238</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (first peak), latency 0.5&#8211;4.5 s</td><td valign="top">6</td><td valign="top">16/10 (10/10 in analysis, only unrein-forced trials)</td><td valign="top">0 (0&#8211;6)</td><td valign="top">0</td><td valign="top">0.03&#160;&#177;&#160;0.19 (0&#8211;10)</td><td valign="top">0.29</td><td valign="top">0.05&#160;&#177;&#160;0.29 (0&#8211;10)</td><td valign="top">0.50</td><td valign="top">0.08&#160;&#177;&#160;0.42 (0&#8211;20)</td><td valign="top">0.40</td></tr><tr><td valign="top">Hollandt et al., unpublished</td><td valign="top">30</td><td valign="top">&gt;0.04</td><td valign="top">TTP (first peak), latency 0.9&#8211;4 s</td><td valign="top">6</td><td valign="top">10/10</td><td valign="top">0</td><td valign="top">0</td><td valign="top">2.97&#160;&#177;&#160;2.81 (0&#8211;10)</td><td valign="top">29.67</td><td valign="top">7.23&#160;&#177;&#160;2.61 (0&#8211;10)</td><td valign="top">72.33</td><td valign="top">10.20&#160;&#177;&#160;4.72</td><td valign="top">51.0</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib67">Sjouwerman et al., 2018</xref></td><td valign="top">326</td><td valign="top">&#8805;0.02</td><td valign="top">TTP (first peak), latency 0.9&#8211;4.5 s</td><td valign="top">9</td><td valign="top">9/9</td><td valign="top">1.38&#160;&#177;&#160;1.73 (0&#8211;9)</td><td valign="top">15.37</td><td valign="top">3.11&#160;&#177;&#160;2.69 (0&#8211;9)</td><td valign="top">34.59</td><td valign="top">3.77&#160;&#177;&#160;2.68 (0&#8211;9)</td><td valign="top">41.92</td><td valign="top">6.87&#160;&#177;&#160;5.01 (0&#8211;18)</td><td valign="top">38.26</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-52465-v2.xml">elife-52465-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="app4table2" position="float"><label>Appendix 4&#8212;table 2.</label><caption><title>Number and percentage of individuals in a sample showing SCR non-responses to a certain number of US presentations during fear acquisition training (exemplarily for one to eight USs<sup>#</sup>), as well as mean number of and percentage of CS responses (CS refers to the CS+ and CS&#8211; combined) in these individuals.</title><p><sup>#</sup>Here only up to eight USs are included as eight is half of the maximum number of US presentations in the samples included here.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Reference</th><th colspan="9" valign="top">a) <italic>n</italic> (%) of individuals with 0, 1, 2, 3, 4, 5, 6, 7, and 8 SCRs towards the US. b) <italic>M</italic> (%)&#160;of valid CS responses for these individuals.</th></tr><tr><th valign="bottom">0 US</th><th valign="bottom">1 US</th><th valign="bottom">2 US</th><th valign="bottom">3 US</th><th valign="bottom">4 US</th><th valign="bottom">5 US</th><th valign="bottom">6 US</th><th valign="top">7 US</th><th valign="top">8 US</th></tr></thead><tbody><tr><td valign="top"><xref ref-type="bibr" rid="bib33">Jentsch et al., 2020</xref></td><td valign="top">a) 1 (2.4%) b) 0 (0%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 2 (4.9%) b) 27.5 (85.9%)</td><td valign="top">a) 7 (17.1%) b) 25.4 (79.5%)</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib29">Hermann et al., 2016</xref></td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 1 (2%) b) 12 (37.5%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 1 (2%) b) 14 (43.7%)</td><td valign="top">a) 0 (0%) b) NA</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib47">Merz et al., 2018a</xref></td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 1 (2.6%) b) 23.0 (95.8%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 2 (5.1%) b) 20.0 (83.3%)</td><td valign="top">a) 3 (7.7%) b) 23.0 (95.8%)</td><td valign="top">a) 1 (2.6%) b) 21.0 (87.5%)</td><td valign="top">a) 5 (12.8%) b) 21.4 (89.1%)</td><td valign="top">a) 9 (23.1%) b) 21.6 (85.6%)</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib46">Merz et al., 2014</xref></td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib22">Hamacher-Dang et al., 2015</xref></td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 1 (3%) b) 24 (75.0%)</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib45">Mertens et al., 2019</xref></td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 4 (6.78%) b) 1.75 (11.67%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 2 (3.39%) b) 3.5 (23.33%)</td><td valign="top">a) 2 (3.39%) b) 9 (60%)</td><td valign="top">a) 0 (0%) b) NA</td></tr><tr><td valign="top">Klingelh&#246;fer-Jens et al., unpublished</td><td valign="top">a) 2 (1.68%) b) 0 (0%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 1 (0.84%) b) 10 (35.7%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 1 (0.84%) b) 1 (3.57%)</td><td valign="top">a) 2 (1.68%) b) 2 (7.14%)</td><td valign="top">a) 1 (0.84%) b) 0 (0%)</td></tr><tr><td valign="top">Gerlicher et al., unpublished</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 3 (5.77%) b) 4 (33.33%)</td><td valign="top">a) 5 (9.62%) b) 4.8 (40%)</td><td valign="top">a) 7 (13.46%) b) 6.7 (55.91%)</td><td valign="top">a) 35 (67.31%) b) 6.15 (51.25%)</td><td valign="top">NA</td><td valign="top">NA</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib16">Gerlicher et al., 2018</xref></td><td valign="top">a) 1 (2.56%) b) 0 (0%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 2 (5.13%) b) 19.5 (97.5%)</td><td valign="top">a) 4 (10.26%) b) 17.5 (87.50%)</td><td valign="top">a) 32 (82.05%) b) 16.81 (84.05%)</td><td valign="top">NA</td><td valign="top">NA</td><td valign="top">NA</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib78">Wendt et al., 2020</xref></td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 1 (0.9%) b)18 (75%)</td><td valign="top">a) 1 (0.9%) b) 24 (100%)</td><td valign="top">a) 1 (0.9%) b) 0 (0%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 2 (1.8%) b) 12 (50%)</td><td valign="top">a) 8 (7.1%) b) 13.13 (54.69%)</td><td valign="top">a) 11 (9.9%) b) 11.09 (46.21%)</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib77">Wendt et al., 2015</xref></td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 1 (0.9%) b) 18 (75%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 1 (0.9%) b) 17 (70.83%)</td><td valign="top">a) 0 (0%) b) NA</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib10">Drexler et al., 2015</xref></td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 2 (4.3%) b) 0.5 (1.28%)</td><td valign="top">a) 1 (2.2%) b) 0.0 (0.0%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 1 (2.2%) b) 7.0 (17.94%)</td><td valign="top">a) 0 (0%) b) NA</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib42">Meir Drexler et al., 2016</xref></td><td valign="top">a) 1 (1.4%) b) 29.00 (74.35%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 2 (2.7%) b) 2.0 (5.12%)</td><td valign="top">a) 1 (1.4%) b) 9.0 (23.07%)</td><td valign="top">a) 1 (1.4%) b) 2.0 (5.12%)</td><td valign="top">a) 1 (1.4%) b) 3.0 (7.69%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 4 (5.5%) b) 5.0 (12.82%)</td><td valign="top">a) 2 (2.7%) b) 6.50 (16.66%)</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib44">Meir Drexler and Wolf, 2017</xref></td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 1 (1.4%) b) 5.0 (12.82%)</td><td valign="top">a) 1 (1.4%) b) 5.0 (12.82%)</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib11">Drexler et al., 2018</xref></td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 1 (2.5%) b) 8 (25%)</td><td valign="top">a) 2 (5.0%) b) 12.5 (39.06%)</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib43">Meir Drexler et al., 2019</xref></td><td valign="top">a) 3 (4.0%) b) 0.33 (1.66%)</td><td valign="top">a) 1 (1.3%) b) 1 (5.0%)</td><td valign="top">a) 4 (5.3%) b) 4.25 (21.25%)</td><td valign="top">a) 2 (2.7%) b) 3.0 (15.0%)</td><td valign="top">a) 3 (4.0%) b) 1.33 (6.66%)</td><td valign="top">a) 19 (25.3%) b) 12.63 (63.15%)</td><td valign="top">a) 43 (57.3%) b) 13.21 (66.04%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td></tr><tr><td valign="top">Chalkia et al., unpublished</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 238 (100%) b) 19.92 (99.6%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td></tr><tr><td valign="top">Hollandt et al.,&#160;unpublished</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">NA</td><td valign="top">NA</td></tr><tr><td valign="top"><xref ref-type="bibr" rid="bib67">Sjouwerman et al., 2018</xref></td><td valign="top">a) 4 (1.23%) b) 0.5 (2.78%)</td><td valign="top">a) 2 (0.61%) b) 2.5 (13.89%)</td><td valign="top">a) 4 (1.23%) b) 4.13 (22.92%)</td><td valign="top">a) 2 (0.61%) b) 7.25 (40.28%)</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td><td valign="top">a) 0 (0%) b) NA</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-52466-v2.xml">elife-52466-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><label>Table 4.</label><caption><title>Full distribution of CA1 excitatory neurons for the NMDAR mutant study.</title><p>The numbers in parentheses are cells from the wildtype.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Test conditions</th><th rowspan="2">Delay responsiveness</th><th colspan="2">Neurons</th></tr><tr><th>cKO</th><th>Control</th></tr></thead><tbody><tr><td rowspan="3">Extension</td><td>Delay-active</td><td>28</td><td>25</td></tr><tr><td>Delay-suppressed</td><td>56</td><td>20</td></tr><tr><td>Other</td><td>22</td><td>19</td></tr><tr><td rowspan="3">Reward loss and gain</td><td>Delay-active</td><td>8</td><td>33&#160;(30)</td></tr><tr><td>Delay-suppressed</td><td>6</td><td>0</td></tr><tr><td>Other</td><td>3</td><td>2</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-52482-v2.xml">elife-52482-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>MurNAc-alk incorporation into PG</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Muropeptide <break/>(non-reduced)</th><th rowspan="2" valign="top">Theoretical neutral mass</th><th colspan="3" valign="top">MurNAc-alk labeled <italic>H. pylori</italic></th><th colspan="3" valign="top">Control <italic>H. pylori</italic></th></tr><tr><th valign="top">Observed ion (charge)</th><th valign="top">Rt<sup>*</sup> <break/>(min)</th><th valign="top">Calculated neutral <break/>mass</th><th valign="top">Observed ion (charge)</th><th valign="top">Rt<sup>*</sup> <break/>(min)</th><th valign="top">Calculated neutral <break/>mass</th></tr></thead><tbody><tr><td valign="top">Di</td><td valign="top">696.270</td><td valign="top">697.289 (1+)</td><td valign="top">20.3</td><td valign="top">696.282</td><td valign="top">697.290 (1+)</td><td valign="top">20.4</td><td valign="top">696.283</td></tr><tr><td valign="top"><italic>Alk</italic>-Di</td><td valign="top">734.286</td><td valign="top">735.307 (1+)</td><td valign="top">30.5</td><td valign="top">734.300</td><td valign="top">-<sup>&#8224;</sup></td><td valign="top">-</td><td valign="top">-</td></tr><tr><td valign="top">Tri</td><td valign="top">868.355</td><td valign="top">869.375 (1+)</td><td valign="top">15.8</td><td valign="top">868.368</td><td valign="top">869.374 (1+)</td><td valign="top">15.8</td><td valign="top">868.367</td></tr><tr><td valign="top"><italic>Alk</italic>-Tri</td><td valign="top">906.371</td><td valign="top">907.392 (1+)</td><td valign="top">25.8</td><td valign="top">906.385</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td></tr><tr><td valign="top">Tetra</td><td valign="top">939.392</td><td valign="top">940.411 (1+)</td><td valign="top">20.4</td><td valign="top">939.404</td><td valign="top">940.412 (1+)</td><td valign="top">20.4</td><td valign="top">939.405</td></tr><tr><td valign="top"><italic>Alk</italic>-Tetra</td><td valign="top">977.408</td><td valign="top">978.428 (1+)</td><td valign="top">30.4</td><td valign="top">977.421</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td></tr><tr><td valign="top">Penta</td><td valign="top">1010.429</td><td valign="top">1011.449 (1+)</td><td valign="top">22.9</td><td valign="top">1010.442</td><td valign="top">1011.449 (1+)</td><td valign="top">22.8</td><td valign="top">1010.442</td></tr><tr><td valign="top"><italic>Alk</italic>-Penta</td><td valign="top">1048.445</td><td valign="top">1049.464 (1+)</td><td valign="top">32.9</td><td valign="top">1048.457</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td></tr><tr><td valign="top">TetraTri</td><td valign="top">1789.736</td><td valign="top">895.889 (2+)</td><td valign="top">33.4</td><td valign="top">1789.762</td><td valign="top">895.888 (2+)</td><td valign="top">33.3</td><td valign="top">1789.761</td></tr><tr><td valign="top"><italic>Alk</italic>-TetraTri</td><td valign="top">1827.752</td><td valign="top">914.898 (2+)</td><td valign="top">39.2</td><td valign="top">1827.781</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td></tr><tr><td valign="top">TetraTetra</td><td valign="top">1860.774</td><td valign="top">931.407 (2+)</td><td valign="top">35.0</td><td valign="top">1860.799</td><td valign="top">931.407 (2+)</td><td valign="top">34.9</td><td valign="top">1860.799</td></tr><tr><td valign="top"><italic>Alk</italic>-TetraTetra</td><td valign="top">1898.789</td><td valign="top">950.416 (2+)</td><td valign="top">39.7</td><td valign="top">1898.817</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td></tr><tr><td valign="top">TetraPenta</td><td valign="top">1931.811</td><td valign="top">966.926 (2+)</td><td valign="top">35.8</td><td valign="top">1931.837</td><td valign="top">966.925 (2+)</td><td valign="top">35.7</td><td valign="top">1931.835</td></tr><tr><td valign="top"><italic>Alk</italic>-TetraPenta</td><td valign="top">1969.826</td><td valign="top">985.934 (2+)</td><td valign="top">39.9</td><td valign="top">1969.853</td><td valign="top">-</td><td valign="top">-</td><td valign="top">-</td></tr></tbody></table><table-wrap-foot><fn><p><sup>*</sup> Rt, retention time.</p><p><sup>&#8224;</sup>-, not detected. Muropeptides detected (confirming incorporation) via LC-MS analysis of MurNAc-alk labeled versus control PG digests. The control cells displayed no evidence of any MurNAc-alk&#160;incorporation.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-53582-v1.xml">elife-53582-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>FA composition of type 2&#969;/&#945; WdiEs in meibum lipids and the most abundant molecular species predicted.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">WdiE type</th><th rowspan="2" valign="top">Total carbon chain length and degree of unsaturation</th><th colspan="2" valign="top">FA composition (%)</th><th colspan="2" valign="top">The most abundant molecular species predicted</th></tr><tr><th valign="top">C16:1 FA</th><th valign="top">C18:1 FA</th><th valign="top">&#969;/&#945;-OH diol</th><th valign="top">FAs</th></tr></thead><tbody><tr><td valign="top">2&#969;</td><td valign="top">C62:2</td><td valign="top">95</td><td valign="top">5</td><td valign="top">C30:0</td><td valign="top">C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#969;</td><td>C64:2</td><td>78</td><td>22</td><td>C32:0</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#969;</td><td>C66:2</td><td>55</td><td>45</td><td>C32:0</td><td>C16:1&#8211;C18:1</td></tr><tr><td valign="top">2&#969;</td><td>C68:2</td><td>14</td><td>86</td><td>C32:0</td><td>C18:1&#8211;C18:1</td></tr><tr><td valign="top">2&#969;</td><td>C70:2</td><td>25</td><td>75</td><td>C34:0</td><td>C18:1&#8211;C18:1</td></tr><tr><td valign="top">2&#969;</td><td>C72:2</td><td>93</td><td>7</td><td>C36:0</td><td>C16:1&#8211;C20:1</td></tr><tr><td valign="top">2&#969;</td><td>C62:3</td><td>99</td><td>1</td><td>C30:1</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#969;</td><td>C64:3</td><td>96</td><td>4</td><td>C32:1</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#969;</td><td>C66:3</td><td>78</td><td>22</td><td>C34:1</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#969;</td><td>C68:3</td><td>44</td><td>56</td><td>C34:1</td><td>C16:1&#8211;C18:1</td></tr><tr><td valign="top">2&#969;</td><td>C70:3</td><td>7</td><td>93</td><td>C34:1</td><td>C18:1&#8211;C18:1</td></tr><tr><td valign="top">2&#969;</td><td>C72:3</td><td>14</td><td>86</td><td>C36:1</td><td>C18:1&#8211;C18:1</td></tr><tr><td valign="top">2&#945;</td><td>C52:2</td><td>98</td><td>2</td><td>C20:0</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#945;</td><td>C54:2</td><td>68</td><td>32</td><td>C22:0</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#945;</td><td>C56:2</td><td>91</td><td>9</td><td>C24:0</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#945;</td><td>C58:2</td><td>90</td><td>10</td><td>C26:0</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#945;</td><td>C60:2</td><td>70</td><td>30</td><td>C28:0</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#945;</td><td>C62:2</td><td>69</td><td>31</td><td>C30:0</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#945;</td><td>C54:3</td><td>90</td><td>10</td><td>C22:1</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#945;</td><td>C56:3</td><td>73</td><td>27</td><td>C24:1</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#945;</td><td>C58:3</td><td>98</td><td>2</td><td>C26:1</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#945;</td><td>C60:3</td><td>93</td><td>7</td><td>C28:1</td><td>C16:1&#8211;C16:1</td></tr><tr><td valign="top">2&#945;</td><td>C62:3</td><td>83</td><td>17</td><td>C30:1</td><td>C16:1&#8211;C16:1</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-53608-v1.xml">elife-53608-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Summary of skeletal abnormalities in <italic>Lin28a</italic> mutant mice.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><break/></th><th align="left" rowspan="2">Anterior arch of the atlas<sup>*</sup></th><th align="center" colspan="2">Ribs&#8224;</th><th align="center" colspan="2">Sternum attachment&#8225;</th><th align="center" colspan="3">Lumbar<sup>&#167;</sup></th></tr><tr><th>13</th><th>12</th><th>7</th><th>61</th><th>6</th><th>5</th><th>L6/S1*</th></tr></thead><tbody><tr><td>Wt(n = 16)</td><td>0</td><td>16 (100%)</td><td>0</td><td>16 (100%)</td><td>0</td><td>7 (43.8%)</td><td>8 (50%)</td><td>1 (6.2%)</td></tr><tr><td>Lin28a<sup>+/-</sup>(n = 19)</td><td>4 (21.1%)</td><td>19 (100%)</td><td>0</td><td>19 (100%)</td><td>0</td><td>0</td><td>18 (94.7%)</td><td>1 (5.3%)</td></tr><tr><td>Lin28a<sup>-/-</sup> <break/>(n&#160;=&#160;14)</td><td rowspan="2">9 (64.3%)</td><td rowspan="2">0</td><td rowspan="2">14 (100%)</td><td rowspan="2">0</td><td rowspan="2">14 (100%)</td><td rowspan="2">0</td><td rowspan="2">9 (64.3%)</td><td rowspan="2">5 (35.7%)</td></tr></tbody></table><table-wrap-foot><fn><p>The percentages of each phenotype are shown in parenthesis.</p><p>* The anterior arch of the atlas was formed from C2 or via fusion.</p></fn><fn><p>&#8224; Total number of pairs of ribs.</p><p>&#8225; Total number of pairs of true ribs that were attached to the sternum.</p></fn><fn><p>&#167; Total number of lumbar vertebrae. L6/S1* indicates an abnormal sacral vertebra that had morphological features of a lumbar vertebra on only one side.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-53908-v2.xml">elife-53908-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Pairwise&#160;posthoc&#160;comparisons.&#160;</title><p>Pairwise mean differences (mean&#160;&#177;&#160;95% ci) between intact and reinnervated treatment cohorts (left), and between obstacle stride categories compared to level stride means, within treatment cohorts (intact/reinnervated).&#160;Bolding indicates statistical significance using FDR corrected threshold (p&#160;&lt;=&#160;0.0263). See <xref ref-type="supplementary-material" rid="table2sdata1">Table 2&#8212;source data 1</xref>&#160;for p-values.</p><p><supplementary-material id="table2sdata1"><label>Table 2&#8212;source data 1.</label><caption><title>P-values for posthoc pairwise mean differences between intact and reinnervated treatment cohorts (left column) and between obstacle stride categories compared to the level terrain means, within treatment cohorts (intact/reinnervated).</title></caption><media mime-subtype="octet-stream" mimetype="application" xlink:href="elife-53908-table2-data1-v2.csv"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Variable</th><th rowspan="2">Treatment cohort</th><th colspan="4" valign="bottom">Intact</th><th colspan="4" valign="bottom">Reinnervated</th></tr><tr><th valign="bottom">S &#8722;1</th><th valign="bottom">S 0</th><th valign="bottom">Str +1</th><th valign="bottom">S +2</th><th valign="bottom">S &#8722;1</th><th valign="bottom">S 0</th><th valign="bottom">Str +1</th><th valign="bottom">S +2</th></tr></thead><tbody><tr><td valign="bottom">W<sub>net</sub></td><td valign="bottom">&#8722;0.47&#160;&#177;&#160;0.59</td><td valign="bottom"><bold>&#8722;0.91&#160;&#177;&#160;0.60</bold></td><td valign="bottom"><bold>3.60&#160;&#177;&#160;0.57</bold></td><td valign="bottom">0.00&#160;&#177;&#160;0.59</td><td valign="bottom"><bold>&#8722;0.86&#160;&#177;&#160;0.34</bold></td><td valign="bottom">0.34&#160;&#177;&#160;0.55</td><td valign="bottom"><bold>3.88&#160;&#177;&#160;0.60</bold></td><td valign="bottom">0.49&#160;&#177;&#160;0.70</td><td valign="bottom">&#8722;0.19&#160;&#177;&#160;0.46</td></tr><tr><td valign="bottom">F<sub>pk</sub></td><td valign="bottom">&#8722;0.02&#160;&#177;&#160;0.06</td><td valign="bottom">&#8722;0.04&#160;&#177;&#160;0.07</td><td valign="bottom"><bold>0.17&#160;&#177;&#160;0.06</bold></td><td valign="bottom">0.04&#160;&#177;&#160;0.07</td><td valign="bottom">&#8722;0.04&#160;&#177;&#160;0.04</td><td valign="bottom"><bold>0.09&#160;&#177;&#160;0.06</bold></td><td valign="bottom"><bold>0.62&#160;&#177;&#160;0.07</bold></td><td valign="bottom"><bold>0.10&#160;&#177;&#160;0.08</bold></td><td valign="bottom">0.05&#160;&#177;&#160;0.05</td></tr><tr><td valign="bottom">L<sub>pkF</sub></td><td valign="bottom">&#8722;0.02&#160;&#177;&#160;0.04</td><td valign="bottom"><bold>&#8722;0.02&#160;&#177;&#160;0.01</bold></td><td valign="bottom"><bold>0.10&#160;&#177;&#160;0.01</bold></td><td valign="bottom">&#8722;0.01&#160;&#177;&#160;0.01</td><td valign="bottom"><bold>&#8722;0.02&#160;&#177;&#160;0.01</bold></td><td valign="bottom"><bold>&#8722;0.05&#160;&#177;&#160;0.01</bold></td><td valign="bottom"><bold>0.14&#160;&#177;&#160;0.01</bold></td><td valign="bottom"><bold>&#8722;0.06&#160;&#177;&#160;0.02</bold></td><td valign="bottom"><bold>&#8722;0.04&#160;&#177;&#160;0.01</bold></td></tr><tr><td valign="bottom">V<sub>pkF</sub></td><td valign="bottom"><bold>&#8722;1.89&#160;&#177;&#160;1.10</bold></td><td valign="bottom">0.07&#160;&#177;&#160;0.45</td><td valign="bottom"><bold>&#8722;1.26&#160;&#177;&#160;0.43</bold></td><td valign="bottom">0.00&#160;&#177;&#160;0.45</td><td valign="bottom">0.11&#160;&#177;&#160;0.26</td><td valign="bottom"><bold>0.60&#160;&#177;&#160;0.42</bold></td><td valign="bottom">&#8722;0.39&#160;&#177;&#160;0.46</td><td valign="bottom"><bold>0.63&#160;&#177;&#160;0.53</bold></td><td valign="bottom"><bold>0.81&#160;&#177;&#160;0.35</bold></td></tr><tr><td valign="bottom">T<sub>force</sub></td><td valign="bottom">0.02&#160;&#177;&#160;0.07</td><td valign="bottom">&#8722;0.01&#160;&#177;&#160;0.02</td><td valign="bottom"><bold>0.11&#160;&#177;&#160;0.02</bold></td><td valign="bottom">&#8722;0.01&#160;&#177;&#160;0.02</td><td valign="bottom">&#8722;0.01&#160;&#177;&#160;0.01</td><td valign="bottom"><bold>&#8722;0.07&#160;&#177;&#160;0.02</bold></td><td valign="bottom"><bold>0.07&#160;&#177;&#160;0.02</bold></td><td valign="bottom"><bold>&#8722;0.06&#160;&#177;&#160;0.02</bold></td><td valign="bottom"><bold>&#8722;0.06&#160;&#177;&#160;0.02</bold></td></tr><tr><td valign="bottom">T<sub>stride</sub></td><td valign="bottom">0.00&#160;&#177;&#160;0.02</td><td valign="bottom">&#8722;0.01&#160;&#177;&#160;0.02</td><td valign="bottom"><bold>0.03&#160;&#177;&#160;0.02</bold></td><td valign="bottom"><bold>&#8722;0.02&#160;&#177;&#160;0.02</bold></td><td valign="bottom">0.01&#160;&#177;&#160;0.01</td><td valign="bottom"><bold>&#8722;0.07&#160;&#177;&#160;0.02</bold></td><td valign="bottom"><bold>0.04&#160;&#177;&#160;0.02</bold></td><td valign="bottom"><bold>&#8722;0.04&#160;&#177;&#160;0.03</bold></td><td valign="bottom"><bold>&#8722;0.03&#160;&#177;&#160;0.02</bold></td></tr><tr><td valign="bottom">E<sub>tot</sub></td><td valign="bottom">0.01&#160;&#177;&#160;0.22</td><td valign="bottom">0.12&#160;&#177;&#160;0.18</td><td valign="bottom"><bold>0.76&#160;&#177;&#160;0.18</bold></td><td valign="bottom">0.11&#160;&#177;&#160;0.18</td><td valign="bottom"><bold>0.12&#160;&#177;&#160;0.11</bold></td><td valign="bottom">0.16&#160;&#177;&#160;0.17</td><td valign="bottom"><bold>0.74&#160;&#177;&#160;0.19</bold></td><td valign="bottom"><bold>0.39&#160;&#177;&#160;0.22</bold></td><td valign="bottom">0.09&#160;&#177;&#160;0.14</td></tr><tr><td valign="bottom">E<sub>freq</sub></td><td valign="bottom">55.70&#160;&#177;&#160;55.12</td><td valign="bottom">3.05&#160;&#177;&#160;9.44</td><td valign="bottom"><bold>&#8722;15.05&#160;&#177;&#160;9.04</bold></td><td valign="bottom">&#8722;1.46&#160;&#177;&#160;9.34</td><td valign="bottom">&#8722;0.46&#160;&#177;&#160;5.43</td><td valign="bottom">&#8722;2.86&#160;&#177;&#160;8.75</td><td valign="bottom"><bold>&#8722;22.06&#160;&#177;&#160;9.52</bold></td><td valign="bottom">&#8722;6.81&#160;&#177;&#160;11.04</td><td valign="bottom"><bold>&#8722;7.68&#160;&#177;&#160;7.28</bold></td></tr><tr><td valign="bottom">E<sub>phase</sub></td><td valign="bottom"><bold>&#8722;0.06&#160;&#177;&#160;0.05</bold></td><td valign="bottom">&#8722;0.01&#160;&#177;&#160;0.02</td><td valign="bottom">0.00&#160;&#177;&#160;0.02</td><td valign="bottom">0.00&#160;&#177;&#160;0.02</td><td valign="bottom">&#8722;0.01&#160;&#177;&#160;0.01</td><td valign="bottom">0.01&#160;&#177;&#160;0.02</td><td valign="bottom"><bold>0.04&#160;&#177;&#160;0.02</bold></td><td valign="bottom">&#8722;0.01&#160;&#177;&#160;0.02</td><td valign="bottom">0.00&#160;&#177;&#160;0.01</td></tr><tr><td valign="bottom">E<sub>dur</sub></td><td valign="bottom"><bold>0.08&#160;&#177;&#160;0.05</bold></td><td valign="bottom">0.00&#160;&#177;&#160;0.03</td><td valign="bottom"><bold>0.04&#160;&#177;&#160;0.02</bold></td><td valign="bottom">&#8722;0.01&#160;&#177;&#160;0.02</td><td valign="bottom">0.00&#160;&#177;&#160;0.01</td><td valign="bottom"><bold>&#8722;0.04&#160;&#177;&#160;0.02</bold></td><td valign="bottom">&#8722;0.01&#160;&#177;&#160;0.03</td><td valign="bottom">0.00&#160;&#177;&#160;0.03</td><td valign="bottom"><bold>&#8722;0.03&#160;&#177;&#160;0.02</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-53944-v2.xml">elife-53944-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Coordinates of centromeres and their GC content in <italic>M. sympodialis.</italic></title><p>Coordinates and length of Mtw1-enriched regions in comparison with those of the core centromeres in <italic>M. sympodialis.</italic></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3">Chromosome number</th><th align="center" colspan="4">Core centromere</th><th colspan="3">Full-length centromere</th></tr><tr><th colspan="2">Coordinates</th><th rowspan="2">Length (bp)</th><th rowspan="2">%GC</th><th colspan="2">Coordinates</th><th rowspan="2">Length (bp)</th></tr><tr><th>Start</th><th>End</th><th>Start</th><th>End</th></tr></thead><tbody><tr><td>1</td><td>786,541</td><td>787,061</td><td>520</td><td>16.4</td><td>784,833</td><td>788,599</td><td>3767</td></tr><tr><td>2</td><td>355,760</td><td>355,841</td><td>81</td><td>20</td><td>354,218</td><td>357,486</td><td>3269</td></tr><tr><td>3</td><td>237,534</td><td>238,686</td><td>1152</td><td>15.6</td><td>235,615</td><td>239,940</td><td>4326</td></tr><tr><td>4</td><td>418,202</td><td>418,728</td><td>526</td><td>15.2</td><td>415,985</td><td>420,656</td><td>4672</td></tr><tr><td>5</td><td>125,056</td><td>125,220</td><td>164</td><td>18</td><td>123,219</td><td>127,284</td><td>4066</td></tr><tr><td>6</td><td>101,950</td><td>102,502</td><td>552</td><td>14.4</td><td>100,342</td><td>105,251</td><td>4910</td></tr><tr><td>7</td><td>431,542</td><td>431,987</td><td>445</td><td>13.2</td><td>430,028</td><td>433,194</td><td>3167</td></tr><tr><td>8</td><td>24,694</td><td>25,564</td><td>870</td><td>18.4</td><td>22,334</td><td>27,476</td><td>5143</td></tr></tbody></table><table-wrap-foot><fn><p>Genome average GC content (in %): 58.5.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-53944-v2.xml">elife-53944-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Coordinates, length, and GC content (in %) of the centromeres predicted in <italic>M. furfur</italic>, <italic>M. globosa</italic>, <italic>M. slooffiae</italic>, <italic>and M. restricta</italic>.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top">Chr./scaffold</th><th rowspan="2" valign="top"><italic>CEN</italic></th><th colspan="4">Core centromere</th><th rowspan="2" valign="top">% GC <break/>genome</th></tr><tr><th valign="top">Start</th><th valign="top">End</th><th valign="top">Length (bp)</th><th valign="top">% GC</th></tr></thead><tbody><tr><td rowspan="7" valign="top"><italic>M. furfur</italic> <break/>CBS14141</td><td valign="top">Chr1</td><td valign="top"><italic>CEN1</italic></td><td valign="top">2,850,135</td><td valign="top">2,850,402</td><td valign="top">268</td><td valign="top">15.7</td><td rowspan="7" valign="top">64.9</td></tr><tr><td valign="top">Chr2</td><td valign="top"><italic>CEN2</italic></td><td valign="top">68,763</td><td valign="top">68,931</td><td valign="top">168</td><td valign="top">15.4</td></tr><tr><td valign="top">Chr3</td><td valign="top"><italic>CEN3</italic></td><td valign="top">717,557</td><td valign="top">718,084</td><td valign="top">528</td><td valign="top">22.9</td></tr><tr><td valign="top">Chr4</td><td valign="top"><italic>CEN4</italic></td><td valign="top">155,897</td><td valign="top">156,301</td><td valign="top">405</td><td valign="top">18.3</td></tr><tr><td valign="top">Chr5</td><td valign="top"><italic>CEN5</italic></td><td valign="top">342,885</td><td valign="top">343,372</td><td valign="top">488</td><td valign="top">21.5</td></tr><tr><td valign="top">Chr6</td><td valign="top"><italic>CEN6</italic></td><td valign="top">86,112</td><td valign="top">86,832</td><td valign="top">721</td><td valign="top">27</td></tr><tr><td valign="top">Chr7</td><td valign="top"><italic>CEN7</italic></td><td valign="top">56,894</td><td valign="top">57,339</td><td valign="top">445</td><td valign="top">20.9</td></tr><tr><td rowspan="9" valign="top"><italic>M. globosa</italic> <break/>CBS7966</td><td valign="top">Chr1</td><td valign="top"><italic>CEN1</italic></td><td valign="top">981,894</td><td valign="top">982,242</td><td valign="top">349</td><td valign="top">17.7</td><td rowspan="9" valign="top">52.05</td></tr><tr><td valign="top">Chr2</td><td valign="top"><italic>CEN2</italic></td><td valign="top">362,480</td><td valign="top">362,807</td><td valign="top">327</td><td valign="top">25.9</td></tr><tr><td valign="top">Chr3</td><td valign="top"><italic>CEN3</italic></td><td valign="top">219,647</td><td valign="top">220,121</td><td valign="top">474</td><td valign="top">27.2</td></tr><tr><td valign="top">Chr4</td><td valign="top"><italic>CEN4</italic></td><td valign="top">152,635</td><td valign="top">152,994</td><td valign="top">359</td><td valign="top">18.3</td></tr><tr><td valign="top">Chr5</td><td valign="top"><italic>CEN5</italic></td><td valign="top">215,437</td><td valign="top">215,595</td><td valign="top">158</td><td valign="top">17</td></tr><tr><td valign="top">Chr6</td><td valign="top"><italic>CEN6</italic></td><td valign="top">464,007</td><td valign="top">464,114</td><td valign="top">107</td><td valign="top">32.4</td></tr><tr><td valign="top">Chr7</td><td valign="top"><italic>CEN7</italic></td><td valign="top">736,701</td><td valign="top">737,015</td><td valign="top">314</td><td valign="top">18.1</td></tr><tr><td valign="top">Chr8</td><td valign="top"><italic>CEN8</italic></td><td valign="top">59,472</td><td valign="top">59,817</td><td valign="top">345</td><td valign="top">19.7</td></tr><tr><td valign="top">Chr9</td><td valign="top"><italic>CEN9</italic></td><td valign="top">114,080</td><td valign="top">114,535</td><td valign="top">455</td><td valign="top">23.5</td></tr><tr><td rowspan="9" valign="top"><italic>M. slooffiae</italic> <break/>CBS7956</td><td valign="top">Chr1</td><td valign="top"><italic>CEN1</italic></td><td valign="top">138,919</td><td valign="top">139,465</td><td valign="top">547</td><td valign="top">26</td><td rowspan="9" valign="top">66.31</td></tr><tr><td valign="top">Chr2</td><td valign="top"><italic>CEN2</italic></td><td valign="top">132,717</td><td valign="top">133,193</td><td valign="top">477</td><td valign="top">23.1</td></tr><tr><td valign="top">Chr3</td><td valign="top"><italic>CEN3</italic></td><td valign="top">367,665</td><td valign="top">368,177</td><td valign="top">513</td><td valign="top">23.8</td></tr><tr><td valign="top">Chr4</td><td valign="top"><italic>CEN4</italic></td><td valign="top">130,942</td><td valign="top">131,501</td><td valign="top">560</td><td valign="top">27</td></tr><tr><td valign="top">Chr5</td><td valign="top"><italic>CEN5</italic></td><td valign="top">183,442</td><td valign="top">183,981</td><td valign="top">540</td><td valign="top">28.5</td></tr><tr><td valign="top">Chr6</td><td valign="top"><italic>CEN6</italic></td><td valign="top">411,984</td><td valign="top">412,552</td><td valign="top">569</td><td valign="top">27.4</td></tr><tr><td valign="top">Chr7</td><td valign="top"><italic>CEN7</italic></td><td valign="top">54,307</td><td valign="top">54,889</td><td valign="top">583</td><td valign="top">30</td></tr><tr><td valign="top">Chr8</td><td valign="top"><italic>CEN8</italic></td><td valign="top">497,637</td><td valign="top">498,149</td><td valign="top">513</td><td valign="top">24</td></tr><tr><td valign="top">Chr9</td><td valign="top"><italic>CEN9</italic></td><td valign="top">55,948</td><td valign="top">56,479</td><td valign="top">532</td><td valign="top">26.7</td></tr><tr><td rowspan="9" valign="top"><italic>M. restricta</italic> <break/>CBS877</td><td valign="top">Chr1</td><td valign="top"><italic>CEN1</italic></td><td valign="top">347,813</td><td valign="top">348,406</td><td valign="top">594</td><td valign="top">29.7</td><td rowspan="9" valign="top">55.73</td></tr><tr><td valign="top">Chr2</td><td valign="top"><italic>CEN2</italic></td><td valign="top">87,190</td><td valign="top">87,806</td><td valign="top">617</td><td valign="top">33.3</td></tr><tr><td valign="top">Chr3</td><td valign="top"><italic>CEN3</italic></td><td valign="top">1,101,494</td><td valign="top">1,102,083</td><td valign="top">590</td><td valign="top">33.9</td></tr><tr><td valign="top">Chr4</td><td valign="top"><italic>CEN4</italic></td><td valign="top">754,356</td><td valign="top">754,989</td><td valign="top">634</td><td valign="top">34.2</td></tr><tr><td valign="top">Chr5</td><td valign="top"><italic>CEN6</italic></td><td valign="top">621,177</td><td valign="top">621,863</td><td valign="top">687</td><td valign="top">31.7</td></tr><tr><td valign="top">Chr6</td><td valign="top"><italic>CEN7</italic></td><td valign="top">390,657</td><td valign="top">391,286</td><td valign="top">630</td><td valign="top">35.1</td></tr><tr><td valign="top">Chr7</td><td valign="top"><italic>CEN8</italic></td><td valign="top">362,842</td><td valign="top">363,381</td><td valign="top">540</td><td valign="top">32</td></tr><tr><td valign="top">Chr8</td><td valign="top"><italic>CEN9</italic></td><td valign="top">117,021</td><td valign="top">117,603</td><td valign="top">583</td><td valign="top">32.8</td></tr><tr><td valign="top">Chr9</td><td valign="top"><italic>CENR</italic></td><td valign="top">70,306</td><td valign="top">70,913</td><td valign="top">608</td><td valign="top">36.3</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-53944-v2.xml">elife-53944-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><label>Table 4.</label><caption><title>Coordinates, length, and GC content (in %) of the centromeres predicted in <italic>M. dermatis</italic>, <italic>M. nana</italic>, <italic>M. vespertilionis</italic>, and <italic>M. japonica.</italic></title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top">Chr./scaffold</th><th rowspan="2" valign="top"><italic>CEN</italic></th><th align="center" colspan="4">Core centromere</th><th rowspan="2" valign="top">% GC <break/>genome</th></tr><tr><th valign="top">Start</th><th valign="top">End</th><th valign="top">Length (bp)</th><th valign="top">% GC</th></tr></thead><tbody><tr><td rowspan="8" valign="top"><italic>M. dermatis</italic> <break/>JCM11348</td><td valign="top">BCKX01000001.1 (Scf1)</td><td valign="top"><italic>CEN1</italic></td><td valign="top">711,456</td><td valign="top">711,978</td><td valign="top">523</td><td valign="top">22.8</td><td rowspan="8" valign="top">59.05</td></tr><tr><td valign="top">BCKX01000002.1 (Scf2)</td><td valign="top"><italic>CEN2</italic></td><td valign="top">1,014,281</td><td valign="top">1,014,977</td><td valign="top">697</td><td valign="top">31.7</td></tr><tr><td valign="top">BCKX01000003.1 (Scf3)</td><td valign="top"><italic>CEN3</italic></td><td valign="top">232,065</td><td valign="top">232,795</td><td valign="top">731</td><td valign="top">29.3</td></tr><tr><td valign="top">BCKX01000004.1 (Scf4)</td><td valign="top"><italic>CEN4</italic></td><td valign="top">409,839</td><td valign="top">410,631</td><td valign="top">793</td><td valign="top">29.5</td></tr><tr><td valign="top">BCKX01000005.1 (Scf5)</td><td valign="top"><italic>CEN5</italic></td><td valign="top">94,520</td><td valign="top">95,018</td><td valign="top">499</td><td valign="top">18.2</td></tr><tr><td valign="top">BCKX01000006.1 (Scf6)</td><td valign="top"><italic>CEN6</italic></td><td valign="top">473,487</td><td valign="top">474,334</td><td valign="top">848</td><td valign="top">30.4</td></tr><tr><td valign="top">BCKX01000007.1 (Scf7)</td><td valign="top"><italic>CEN7</italic></td><td valign="top">76,361</td><td valign="top">76,975</td><td valign="top">615</td><td valign="top">26</td></tr><tr><td valign="top">BCKX01000008.1 (Scf8)</td><td valign="top"><italic>CEN8</italic></td><td valign="top">17,893</td><td valign="top">18,540</td><td valign="top">648</td><td valign="top">26.4</td></tr><tr><td rowspan="8" valign="top"><italic>M. nana</italic> <break/>JCM12085</td><td valign="top">BCLA01000001.1 (Scf1)</td><td valign="top"><italic>CEN1</italic></td><td valign="top">715,036</td><td valign="top">715,592</td><td valign="top">557</td><td valign="top">27.8</td><td rowspan="8" valign="top">57.95</td></tr><tr><td valign="top">BCLA01000002.1 (Scf2)</td><td valign="top"><italic>CEN2</italic></td><td valign="top">349,428</td><td valign="top">350,120</td><td valign="top">693</td><td valign="top">33</td></tr><tr><td valign="top">BCLA01000003.1 (Scf3)</td><td valign="top"><italic>CEN3</italic></td><td valign="top">220,773</td><td valign="top">221,345</td><td valign="top">573</td><td valign="top">27.9</td></tr><tr><td valign="top">BCLA01000004.1 (Scf4)</td><td valign="top"><italic>CEN4</italic></td><td valign="top">410,594</td><td valign="top">411,387</td><td valign="top">794</td><td valign="top">33.2</td></tr><tr><td valign="top">BCLA01000005.1 (Scf5)</td><td valign="top"><italic>CEN5</italic></td><td valign="top">524,594</td><td valign="top">525,105</td><td valign="top">512</td><td valign="top">24.8</td></tr><tr><td valign="top">BCLA01000006.1 (Scf6)</td><td valign="top"><italic>CEN6</italic></td><td valign="top">133,647</td><td valign="top">134,324</td><td valign="top">678</td><td valign="top">33.6</td></tr><tr><td valign="top">BCLA01000007.1 (Scf7)</td><td valign="top"><italic>CEN7</italic></td><td valign="top">408,363</td><td valign="top">409,067</td><td valign="top">705</td><td valign="top">34.2</td></tr><tr><td valign="top">BCLA01000008.1 (Scf8)</td><td valign="top"><italic>CEN8</italic></td><td valign="top">398,756</td><td valign="top">399,423</td><td valign="top">668</td><td valign="top">32.5</td></tr><tr><td rowspan="9" valign="top"><italic>M. vespertilionis</italic> <break/>CBS15041</td><td valign="top">KZ454987.1 (Scf1)</td><td valign="top"><italic>CEN1</italic></td><td valign="top">410,820</td><td valign="top">411,340</td><td valign="top">521</td><td valign="top">15.7</td><td rowspan="9" valign="top">56.6</td></tr><tr><td valign="top">KZ454988.1 (Scf2)</td><td valign="top"><italic>CEN2</italic></td><td valign="top">1,275,509</td><td valign="top">1,276,238</td><td valign="top">730</td><td valign="top">25.8</td></tr><tr><td valign="top">KZ454989.1 (Scf3)</td><td valign="top"><italic>CEN3</italic></td><td valign="top">322,361</td><td valign="top">323,277</td><td valign="top">917</td><td valign="top">38.2</td></tr><tr><td valign="top">KZ454990.1 (Scf4)</td><td valign="top"><italic>CEN4</italic></td><td valign="top">583,450</td><td valign="top">584,319</td><td valign="top">870</td><td valign="top">28.9</td></tr><tr><td valign="top">KZ454991.1 (Scf5)</td><td valign="top"><italic>CEN5</italic></td><td valign="top">802,843</td><td valign="top">804,042</td><td valign="top">1200</td><td valign="top">28.8</td></tr><tr><td valign="top">KZ454992.1 (Scf6)</td><td valign="top"><italic>CEN6</italic></td><td valign="top">739,896</td><td valign="top">740,558</td><td valign="top">663</td><td valign="top">22.3</td></tr><tr><td valign="top">KZ454993.1 (Scf7)</td><td valign="top"><italic>CEN7</italic></td><td valign="top">268,699</td><td valign="top">269,626</td><td valign="top">928</td><td valign="top">28.8</td></tr><tr><td valign="top">KZ454994.1 (Scf8)</td><td valign="top"><italic>CEN8</italic></td><td valign="top">10,985</td><td valign="top">11,865</td><td valign="top">881</td><td valign="top">28</td></tr><tr><td valign="top">KZ454995.1 (Scf9)</td><td valign="top"><italic>CEN9</italic></td><td valign="top">19,047</td><td valign="top">19,724</td><td valign="top">678</td><td valign="top">29.1</td></tr><tr><td rowspan="9" valign="top"><italic>M. japonica</italic> <break/>JCM11963</td><td valign="top">BCKY01000001.1 (Scf1)</td><td valign="top"><italic>CEN1</italic></td><td valign="top">1,068,050</td><td valign="top">1,068,614</td><td valign="top">564</td><td valign="top">25.1</td><td rowspan="9" valign="top">62.35</td></tr><tr><td valign="top">BCKY01000002.1 (Scf2)</td><td valign="top"><italic>CEN2</italic></td><td valign="top">139,423</td><td valign="top">139,920</td><td valign="top">497</td><td valign="top">20.3</td></tr><tr><td valign="top">BCKY01000003.1 (Scf3)</td><td valign="top"><italic>CEN3</italic></td><td valign="top">350,068</td><td valign="top">350,603</td><td valign="top">535</td><td valign="top">24.3</td></tr><tr><td valign="top">BCKY01000004.1 (Scf4)</td><td valign="top"><italic>CEN4</italic></td><td valign="top">380,877</td><td valign="top">381,439</td><td valign="top">562</td><td valign="top">24.2</td></tr><tr><td valign="top">BCKY01000005.1 (Scf5)</td><td valign="top"><italic>CEN5</italic></td><td valign="top">507,632</td><td valign="top">508,230</td><td valign="top">598</td><td valign="top">24.5</td></tr><tr><td valign="top">BCKY01000006.1 (Scf6)</td><td valign="top"><italic>CEN6</italic></td><td valign="top">240,968</td><td valign="top">250,550</td><td valign="top">582</td><td valign="top">23.7</td></tr><tr><td valign="top">BCKY01000007.1 (Scf7)</td><td valign="top"><italic>CEN7</italic></td><td valign="top">286,711</td><td valign="top">287,234</td><td valign="top">523</td><td valign="top">24.9</td></tr><tr><td valign="top">BCKY01000008.1 (Scf8)</td><td valign="top"><italic>CEN8</italic></td><td valign="top">87,314</td><td valign="top">87,873</td><td valign="top">559</td><td valign="top">23.9</td></tr><tr><td valign="top">BCKY01000010.1 (Scf10)</td><td valign="top"><italic>CEN9</italic></td><td valign="top">230,906</td><td valign="top">231,456</td><td valign="top">530</td><td valign="top">24.1</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-53999-v2.xml">elife-53999-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Muscle contents of proteins and glycogen, and glucose in blood.</title><p>mc &#8211; microsomal fraction; wm &#8211; whole muscle lysate. Rows 1&#8211;17 list statistical parameters for 13 MHN and 12 MHS subjects. Units for glycogen are g/g&#160;of protein&#160;in&#160;extract, mM for FBS and arbitrary for all other variables (Materials&#160;and&#160;methods). Row 18 lists parameters of fasting blood sugar in 78 MHN and 80 MHS patients of the recent cohort. In row 19 FBS parameters of 6 &#8216;metabolically normal and 6 &#8216;metabolically challenged&#8217; patients (<italic>MN</italic> and <italic>MC</italic>, defined in Discussion) are listed in columns 3&#8211;5 and 6&#8211;8, respectively. <sup>&#167;</sup> Significant at the 0.05 level by 2-tailed <italic>t</italic> test or *Mann-Whitney <italic>u</italic> test. Rows listing significant increases are colored red, reductions are in blue. The inclusion of two variables with p&gt;0.05 is justified based on their correlations listed in <xref ref-type="table" rid="table2">Table 2</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"/><th valign="top">1</th><th valign="top">2</th><th>3</th><th>4</th><th>5</th><th valign="top">6</th><th valign="top">7</th><th valign="top">8</th><th valign="top">9</th></tr><tr><th colspan="2" valign="top"/><th colspan="3">MHN</th><th colspan="3" valign="top">MHS</th><th valign="top"/></tr><tr><th valign="top">Species</th><th valign="top">Fraction</th><th valign="top">Mean</th><th valign="top">Median</th><th valign="top">S.E.M.</th><th valign="top">Mean</th><th valign="top">Median</th><th valign="top">S.E.M.</th><th><italic>P</italic></th></tr></thead><tbody><tr><td style="author-callout-style-b6" valign="top">1</td><td rowspan="2" style="author-callout-style-b6">GP</td><td style="author-callout-style-b6">mc</td><td style="author-callout-style-b6">6.53</td><td style="author-callout-style-b6">5.28</td><td style="author-callout-style-b6">0.813</td><td style="author-callout-style-b6">12.6</td><td style="author-callout-style-b6">11.6</td><td style="author-callout-style-b6">1.440</td><td style="author-callout-style-b6">0.001*</td></tr><tr><td style="author-callout-style-b6" valign="top">2</td><td style="author-callout-style-b6">wm</td><td style="author-callout-style-b6">3.55</td><td style="author-callout-style-b6">3.33</td><td style="author-callout-style-b6">0.198</td><td style="author-callout-style-b6">4.10</td><td style="author-callout-style-b6">4.28</td><td style="author-callout-style-b6">0.208</td><td style="author-callout-style-b6">0.07</td></tr><tr><td style="author-callout-style-b6" valign="top">3</td><td rowspan="2" style="author-callout-style-b6">GP<italic>a</italic></td><td style="author-callout-style-b6">mc</td><td style="author-callout-style-b6">3.81</td><td style="author-callout-style-b6">3.09</td><td style="author-callout-style-b6">0.522</td><td style="author-callout-style-b6">6.50</td><td style="author-callout-style-b6">4.76</td><td style="author-callout-style-b6">1.22</td><td style="author-callout-style-b6">0.05<sup>&#167;</sup></td></tr><tr><td style="author-callout-style-b6" valign="top">4</td><td style="author-callout-style-b6">wm</td><td style="author-callout-style-b6">1.32</td><td style="author-callout-style-b6">1.26</td><td style="author-callout-style-b6">0.090</td><td style="author-callout-style-b6">2.22</td><td style="author-callout-style-b6">2.08</td><td style="author-callout-style-b6">0.198</td><td style="author-callout-style-b6">&lt;0.001*</td></tr><tr><td style="author-callout-style-b6" valign="top">5</td><td rowspan="2" style="author-callout-style-b6">GP<italic>a</italic>/GP</td><td>mc</td><td>0.63</td><td>0.55</td><td>0.082</td><td>0.51</td><td>0.44</td><td>0.074</td><td>0.28</td></tr><tr><td style="author-callout-style-b6" valign="top">6</td><td style="author-callout-style-b6">wm</td><td style="author-callout-style-b6">0.39</td><td style="author-callout-style-b6">0.36</td><td style="author-callout-style-b6">0.061</td><td style="author-callout-style-b6">0.54</td><td style="author-callout-style-b6">0.51</td><td style="author-callout-style-b6">0.065</td><td style="author-callout-style-b6">0.005*</td></tr><tr><td style="author-callout-style-b6" valign="top">7</td><td rowspan="2" style="author-callout-style-b6">PhK</td><td style="author-callout-style-b6">mc</td><td style="author-callout-style-b6">0.85</td><td style="author-callout-style-b6">0.82</td><td style="author-callout-style-b6">0.070</td><td style="author-callout-style-b6">1.57</td><td style="author-callout-style-b6">1.41</td><td style="author-callout-style-b6">0.180</td><td style="author-callout-style-b6">&lt;0.001<sup>&#167;</sup>*</td></tr><tr><td style="author-callout-style-b6" valign="top">8</td><td>wm</td><td>0.81</td><td>0.82</td><td>0.101</td><td>1.06</td><td>0.81</td><td>0.169</td><td>0.57*</td></tr><tr><td style="author-callout-style-b6" valign="top">9</td><td style="author-callout-style-b6">GDE</td><td style="author-callout-style-b6">mc</td><td style="author-callout-style-b6">3.63</td><td style="author-callout-style-b6">3.44</td><td style="author-callout-style-b6">0.402</td><td style="author-callout-style-b6">7.05</td><td style="author-callout-style-b6">5.81</td><td style="author-callout-style-b6">0.902</td><td style="author-callout-style-b6">0.002*</td></tr><tr><td valign="top">10</td><td>GS</td><td>mc</td><td>6.95</td><td>6.10</td><td>1.29</td><td>5.27</td><td>5.12</td><td>0.666</td><td>0.26</td></tr><tr><td style="author-callout-style-b6" valign="top">11</td><td style="author-callout-style-b6">GS<italic>a</italic>/GS</td><td style="author-callout-style-b6">mc</td><td style="author-callout-style-b6">1.03</td><td style="author-callout-style-b6">0.76</td><td style="author-callout-style-b6">0.157</td><td style="author-callout-style-b6">1.39</td><td style="author-callout-style-b6">1.37</td><td style="author-callout-style-b6">0.150</td><td style="author-callout-style-b6">0.11</td></tr><tr><td style="author-callout-style-b1" valign="top">12</td><td style="author-callout-style-b1">GLUT4</td><td style="author-callout-style-b1">mc</td><td style="author-callout-style-b1">1.18</td><td style="author-callout-style-b1">1.23</td><td style="author-callout-style-b1">0.189</td><td style="author-callout-style-b1">0.62</td><td style="author-callout-style-b1">0.54</td><td style="author-callout-style-b1">0.105</td><td style="author-callout-style-b1">0.01<sup>&#167;</sup></td></tr><tr><td style="author-callout-style-b1" valign="top">13</td><td style="author-callout-style-b1">Glycogen</td><td style="author-callout-style-b1">mc</td><td style="author-callout-style-b1">1.17</td><td style="author-callout-style-b1">1.15</td><td style="author-callout-style-b1">0.116</td><td style="author-callout-style-b1">0.75</td><td style="author-callout-style-b1">0.75</td><td style="author-callout-style-b1">0.066</td><td style="author-callout-style-b1">0.01<sup>*</sup></td></tr><tr><td valign="top">14</td><td>Casq1</td><td>mc</td><td>1.71</td><td>1.42</td><td>0.249</td><td>1.64</td><td>1.52</td><td>0.261</td><td>0.84</td></tr><tr><td valign="top">15</td><td>FKBP12</td><td>mc</td><td>1.95</td><td>1.88</td><td>0.227</td><td>1.93</td><td>1.84</td><td>0.223</td><td>0.95</td></tr><tr><td valign="top">16</td><td>SERCA1a</td><td>mc</td><td>18.5</td><td>18.9</td><td>1.365</td><td>17.4</td><td>19.2</td><td>1.728</td><td>0.61</td></tr><tr><td valign="top">17</td><td>SERCA1b</td><td>mc</td><td>7.71</td><td>8.34</td><td>0.839</td><td>7.18</td><td>7.23</td><td>1.019</td><td>0.69</td></tr><tr><td valign="top">18</td><td colspan="2">FBS, recent cohort</td><td>5.10</td><td>5.10</td><td>0.150</td><td>5.44</td><td>5.35</td><td>0.207</td><td>0.19</td></tr><tr><td style="author-callout-style-b6" valign="top">19</td><td colspan="2" style="author-callout-style-b6">FBS, <italic>MN</italic> or <italic>MC</italic></td><td style="author-callout-style-b6">5.05</td><td style="author-callout-style-b6">5.05</td><td style="author-callout-style-b6">0.256</td><td style="author-callout-style-b6">5.90</td><td style="author-callout-style-b6">5.85</td><td style="author-callout-style-b6">0.241</td><td style="author-callout-style-b6">0.03<sup>&#167;</sup>*</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-54076-v1.xml">elife-54076-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Snakebite deaths in the Million Death Study, age-standardized and age-specific mortality rates and risks in India from 2001-2014.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="middle">Year</th><th rowspan="2" valign="middle">Study deaths from snakebite/all causes</th><th colspan="4" valign="top">Standardized death rate /100,000 (all ages) and age-specific rates /100,000<sup>*</sup></th><th valign="top">Snakebite mortality risk<sup>&#8224;</sup></th><th valign="top">Estimated national deaths (000)<sup>&#8225;</sup></th></tr><tr><th valign="top">All ages</th><th valign="top">0-14</th><th valign="top">15-29</th><th valign="top">30-69</th><th valign="top"/><th valign="top"/></tr></thead><tbody><tr><td valign="top">2001</td><td valign="top">199 /41826</td><td valign="top">5.3</td><td valign="top">5.4</td><td valign="top">3.6</td><td valign="top">5.9</td><td valign="top">0.40%</td><td valign="top">55.0</td></tr><tr><td valign="top">2002</td><td valign="top">183 /41740</td><td valign="top">5.2</td><td valign="top">5.2</td><td valign="top">3.5</td><td valign="top">5.8</td><td valign="top">0.39%</td><td valign="top">55.3</td></tr><tr><td valign="top">2003</td><td valign="top">179 /38798</td><td valign="top">5.1</td><td valign="top">5.0</td><td valign="top">3.4</td><td valign="top">5.8</td><td valign="top">0.38%</td><td valign="top">55.8</td></tr><tr><td valign="top">2004</td><td valign="top">190 /37380</td><td valign="top">5.0</td><td valign="top">4.6</td><td valign="top">3.5</td><td valign="top">5.7</td><td valign="top">0.38%</td><td valign="top">55.6</td></tr><tr><td valign="top">2005</td><td valign="top">244 /46755</td><td valign="top">4.9</td><td valign="top">4.8</td><td valign="top">3.4</td><td valign="top">6.4</td><td valign="top">0.40%</td><td valign="top">60.8</td></tr><tr><td valign="top">2006</td><td valign="top">214 /47471</td><td valign="top">5.3</td><td valign="top">4.7</td><td valign="top">3.2</td><td valign="top">6.7</td><td valign="top">0.40%</td><td valign="top">62.7</td></tr><tr><td valign="top">2007</td><td valign="top">225 /48536</td><td valign="top">5.3</td><td valign="top">4.5</td><td valign="top">3.0</td><td valign="top">6.4</td><td valign="top">0.39%</td><td valign="top">61.0</td></tr><tr><td valign="top">2008</td><td valign="top">215 /47673</td><td valign="top">5.1</td><td valign="top">4.2</td><td valign="top">2.8</td><td valign="top">5.9</td><td valign="top">0.36%</td><td valign="top">57.4</td></tr><tr><td valign="top">2009</td><td valign="top">183 /47873</td><td valign="top">4.7</td><td valign="top">3.9</td><td valign="top">2.6</td><td valign="top">5.3</td><td valign="top">0.33%</td><td valign="top">53.8</td></tr><tr><td valign="top">2010</td><td valign="top">200 /45719</td><td valign="top">4.3</td><td valign="top">3.9</td><td valign="top">2.6</td><td valign="top">5.0</td><td valign="top">0.32%</td><td valign="top">52.4</td></tr><tr><td valign="top">2011</td><td valign="top">185 /46099</td><td valign="top">4.2</td><td valign="top">4.0</td><td valign="top">2.7</td><td valign="top">5.1</td><td valign="top">0.33%</td><td valign="top">54.9</td></tr><tr><td valign="top">2012</td><td valign="top">227 /46635</td><td valign="top">4.3</td><td valign="top">4.3</td><td valign="top">2.8</td><td valign="top">5.4</td><td valign="top">0.36%</td><td valign="top">59.2</td></tr><tr><td valign="top">2013</td><td valign="top">214 /45331</td><td valign="top">4.6</td><td valign="top">4.4</td><td valign="top">3.0</td><td valign="top">5.8</td><td valign="top">0.38%</td><td valign="top">62.3</td></tr><tr><td valign="top">2014</td><td valign="top">175 /29647</td><td valign="top">4.7</td><td valign="top">4.2</td><td valign="top">3.0</td><td valign="top">5.9</td><td valign="top">0.37%</td><td valign="top">61.2</td></tr><tr><td valign="top">2001-2014</td><td valign="top">2833 /611483</td><td valign="top">4.8</td><td valign="top">4.5</td><td valign="top">3.1</td><td valign="top">5.8</td><td valign="top">0.37%</td><td valign="top">807.5</td></tr><tr><td colspan="2" valign="top">Plausible range (Lower, Upper)<sup>&#167;</sup></td><td valign="top">(4.4, 5.0)</td><td valign="top">(4.1, 4.7)</td><td valign="top">(2.8, 3.2)</td><td valign="top">(5.3, 6.0)</td><td valign="top">(0.34%, 0.38%)</td><td valign="top">(738.2, 833.4)</td></tr></tbody></table><table-wrap-foot><fn><p><sup>*</sup> Death rates were standardized to the Indian population in census year 2001 to take into account minor changes in the age distribution over&#160;time.</p><p><sup>&#8224;</sup> The probability of dying due to snakebite before reaching age 70 years in the hypothetical absence of other competing causes of death. This was calculated by summing the 5-yearly standardized death rates from ages 0&#160;to&#160;69 years.</p></fn><fn><p><sup>&#8225;</sup> Total death estimates at all ages were calculated by applying the MDS sample weighted proportion of deaths from snakebites, using weighted 3-yearly moving average, to the United Nations Population Division death totals.</p><p><sup>&#167;</sup> Plausible ranges: The inherent variation in these estimates is not from the underlying demographic estimates but in the determination of primary causes of death. Therefore, we used plausible ranges based on independent cause assignment by two physicians and subsequent agreement on ICD-10 codes (X20 or X29). The lower bound was based on immediate agreement of both physicians and upper bound based on either of two physicians coding snakebite deaths.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-54076-v1.xml">elife-54076-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><label>Table 3.</label><caption><title>Snakebite death rates by state in India for 2001-2014.</title></caption><table frame="hsides" rules="groups"><thead><tr><th valign="bottom"/><th colspan="2" valign="bottom"/><th rowspan="2" valign="bottom">Study deaths in MDS</th><th colspan="4" valign="bottom">Annual average standardized death rate /100,000</th><th rowspan="2" valign="bottom">Estimated deaths for 2001-14 (000)</th></tr><tr><th valign="bottom"/><th colspan="2">State</th><th valign="bottom">2001-2004</th><th valign="bottom">2005-2009</th><th valign="bottom">2010-2014</th><th valign="bottom">Trend</th></tr></thead><tbody><tr><td colspan="3" valign="top">Higher burden states</td><td valign="top">1726</td><td valign="top">5.9</td><td valign="top">6.1</td><td valign="top">6.2</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf1-v1.tif"/></td><td valign="top">557.4</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Andhra Pradesh</td><td valign="top">271</td><td valign="top">8.5</td><td valign="top">7.3</td><td valign="top">5.6</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf2-v1.tif"/></td><td valign="top">82.9</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Bihar</td><td valign="top">321</td><td valign="top">5.6</td><td valign="top">7.6</td><td valign="top">8.9</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf3-v1.tif"/></td><td valign="top">101.9</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Odisha</td><td valign="top">191</td><td valign="top">7.5</td><td valign="top">7.2</td><td valign="top">5.9</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf4-v1.tif"/></td><td valign="top">40.3</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Madhya Pradesh</td><td valign="top">195</td><td valign="top">6.7</td><td valign="top">7.7</td><td valign="top">6.0</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf5-v1.tif"/></td><td valign="top">67.8</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Uttar Pradesh</td><td valign="top">322</td><td valign="top">5.2</td><td valign="top">5.9</td><td valign="top">6.0</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf6-v1.tif"/></td><td valign="top">153.6</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Rajasthan</td><td valign="top">192</td><td valign="top">4.9</td><td valign="top">6.7</td><td valign="top">5.0</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf7-v1.tif"/></td><td valign="top">52.1</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Gujarat</td><td valign="top">176</td><td valign="top">4.1</td><td valign="top">4.8</td><td valign="top">5.1</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf8-v1.tif"/></td><td valign="top">38.8</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Jharkhand</td><td valign="top">58</td><td valign="top">4.9</td><td valign="top">2.0</td><td valign="top">7.1</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf9-v1.tif"/></td><td valign="top">20.1</td></tr><tr><td colspan="3" valign="top">Lower burden states</td><td valign="top">1107</td><td valign="top">3.7</td><td valign="top">3.1</td><td valign="top">2.1</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf10-v1.tif"/></td><td valign="top">249.9</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Chhattisgarh</td><td valign="top">42</td><td valign="top">6.0</td><td valign="top">6.5</td><td valign="top">2.5</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf11-v1.tif"/></td><td valign="top">16.8</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Jammu &amp; Kashmir</td><td valign="top">64</td><td valign="top">5.3</td><td valign="top">7.0</td><td valign="top">0.9</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf12-v1.tif"/></td><td valign="top">7.0</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Tamil Nadu</td><td valign="top">176</td><td valign="top">6.1</td><td valign="top">3.4</td><td valign="top">3.0</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf13-v1.tif"/></td><td valign="top">42.1</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Karnataka</td><td valign="top">137</td><td valign="top">5.6</td><td valign="top">3.3</td><td valign="top">2.9</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf14-v1.tif"/></td><td valign="top">33.0</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Maharashtra</td><td valign="top">147</td><td valign="top">4.2</td><td valign="top">3.7</td><td valign="top">2.6</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf15-v1.tif"/></td><td valign="top">56.0</td></tr><tr><td colspan="2" valign="top"/><td valign="top">West Bengal</td><td valign="top">188</td><td valign="top">4.1</td><td valign="top">3.3</td><td valign="top">2.9</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf16-v1.tif"/></td><td valign="top">42.7</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Punjab</td><td valign="top">67</td><td valign="top">2.9</td><td valign="top">3.1</td><td valign="top">4.0</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf17-v1.tif"/></td><td valign="top">14.5</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Haryana</td><td valign="top">45</td><td valign="top">2.9</td><td valign="top">3.3</td><td valign="top">1.8</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf18-v1.tif"/></td><td valign="top">9.5</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Assam</td><td valign="top">27</td><td valign="top">2.8</td><td valign="top">0.7</td><td valign="top">2.1</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf19-v1.tif"/></td><td valign="top">7.3</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Northeastern states</td><td valign="top">37</td><td valign="top">2.3</td><td valign="top">0.9</td><td valign="top">0.7</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf20-v1.tif"/></td><td valign="top">2.4</td></tr><tr><td colspan="2" valign="top"/><td valign="top">Kerala</td><td valign="top">43</td><td valign="top">1.8</td><td valign="top">1.3</td><td valign="top">0.5</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf21-v1.tif"/></td><td valign="top">6.5</td></tr><tr><td colspan="2" valign="top"/><td valign="top">All other states</td><td valign="top">134</td><td valign="top">4.3</td><td valign="top">3.9</td><td valign="top">3.2</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf22-v1.tif"/></td><td valign="top">12.2</td></tr><tr><td colspan="3" valign="top">All India</td><td valign="top">2833</td><td valign="top">5.1</td><td valign="top">4.9</td><td valign="top">4.5</td><td valign="top"><inline-graphic mime-subtype="tiff" mimetype="image" xlink:href="elife-54076-inf23-v1.tif"/></td><td valign="top">807.5</td></tr></tbody></table><table-wrap-foot><fn><p>States are in descending order of annual average death rates for the study period of 2001-2014. We included only the states with populations over 10 million. Andhra Pradesh included Telangana. The Northeastern states include Arunachal Pradesh&#8218; Manipur&#8218; Meghalaya&#8218; Mizoram&#8218; Nagaland&#8218; Sikkim and Tripura.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-54076-v1.xml">elife-54076-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app1table2" position="float"><label>Appendix 1&#8212;table 2.</label><caption><title>Expected snakebite prevalence in 2015.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">In- hospital case-fatality rate/100 bites</th><th rowspan="2" valign="top">Out-of-hospital to in-hospital hypothetical ratio <break/>K = I(h')/I(h)</th><th rowspan="2" valign="top">Hypothetical % who sought hospital treatment 1/(k+1)</th><th colspan="3" valign="top">Expected no. of snakebites in 000</th><th rowspan="2" valign="top">No. of envenomations in 000</th><th rowspan="2" valign="top">No. of dry bites in 000</th></tr><tr><th valign="top">In-hospital</th><th valign="top">Out-of- hospital</th><th valign="top">Total</th></tr></thead><tbody><tr><td valign="bottom">(1)</td><td valign="bottom">(2)</td><td valign="bottom">(3)</td><td valign="bottom">(4)</td><td valign="bottom">(5)</td><td valign="bottom">(6)</td><td valign="bottom">(7)</td><td valign="bottom">(8)</td></tr><tr><td valign="bottom">3.2</td><td valign="bottom">1.5</td><td valign="bottom">40%</td><td valign="bottom">442.2</td><td valign="bottom">663.4</td><td valign="bottom">1,105.6</td><td valign="bottom">773.9</td><td valign="bottom">331.7</td></tr><tr><td valign="bottom">3.2</td><td valign="bottom">2.0</td><td valign="bottom">33%</td><td valign="top">442.2</td><td valign="bottom">884.5</td><td valign="bottom">1,326.7</td><td valign="bottom">928.7</td><td valign="bottom">398.0</td></tr><tr><td valign="bottom">3.2</td><td valign="bottom">3.0</td><td valign="bottom">25%</td><td valign="top">442.2</td><td valign="bottom">1326.8</td><td valign="bottom">1,769.0</td><td valign="bottom">1238.3</td><td valign="bottom">530.7</td></tr></tbody></table><table-wrap-foot><fn><p>Notes:</p><p>1. We calculated in-hospital case-fatality rates (CFR) (Column 1) from a regression analysis of 66 relevant studies in the systematic literature review (<xref ref-type="table" rid="app2table1">Appendix 2&#8212;table 1</xref>). We excluded the Government&#8217;s annual health statistics reporting from public hospitals as case-fatality rates calculated from these data were implausibly low and inconsistent. We used an ordinary least square regression to calculate the combined CFR, treating the number of snakebite deaths as the outcome variable and snakebite prevalence as the explanatory variable while excluding outliers. The in-hospital snakebite case-fatality rate (per 100 bites) is:</p></fn><fn><p><disp-formula id="equ1"><mml:math id="m1"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mi>C</mml:mi><mml:mi>F</mml:mi><mml:mi>R</mml:mi><mml:mrow><mml:mo>(</mml:mo><mml:mi>h</mml:mi><mml:mo>)</mml:mo></mml:mrow><mml:mo>=</mml:mo><mml:mfrac><mml:mrow><mml:mrow><mml:mi mathvariant="normal">D</mml:mi></mml:mrow><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mi mathvariant="normal">h</mml:mi></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:mrow><mml:mrow><mml:mrow><mml:mi mathvariant="normal">I</mml:mi></mml:mrow><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mi mathvariant="normal">h</mml:mi></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:mfrac><mml:mo>&#8727;</mml:mo><mml:mn>100</mml:mn><mml:mspace width="1em"/><mml:mspace width="1em"/><mml:mspace width="1em"/><mml:mspace width="1em"/><mml:mspace width="1em"/><mml:mo stretchy="false">(</mml:mo><mml:mn>1</mml:mn><mml:mo stretchy="false">)</mml:mo></mml:mrow></mml:mstyle></mml:math></disp-formula></p><p>where D(h) represents the number of in-hospital snakebite deaths and I(h) represents the in-hospital snakebite prevalence. The CFR was 3.2% and 95% CI were (2.5, 3.8) (<xref ref-type="table" rid="app2table1">Appendix 2&#8212;table 1</xref>).</p></fn><fn><p>2. The MDS study estimates 62,300 snakebite deaths in 2015, of which 22.7% or 14,200 died in hospitals. Inverting formula 1 with the CFR of 3.2 to solve for I(h) yields 442,200 in-hospital snakebite prevalence in 2015 (Column 4).</p><p>3. To estimate out-of-hospital prevalence of snakebites (Column 5), we used a hypothetical relationship between in-hospital and out-of-hospital prevalence. If the out-of-hospital to in-hospital prevalence proportion is &#8216;K&#8217;, then we can express the out-of-hospital snakebite prevalence I(h&#8217;) as:</p></fn><fn><p><disp-formula id="equ2"><mml:math id="m2"><mml:mstyle displaystyle="true" scriptlevel="0"><mml:mrow><mml:mi>I</mml:mi><mml:mrow><mml:mo>(</mml:mo><mml:msup><mml:mi>h</mml:mi><mml:mrow><mml:mo>&#8242;</mml:mo></mml:mrow></mml:msup><mml:mo>)</mml:mo></mml:mrow><mml:mo>=</mml:mo><mml:mi>K</mml:mi><mml:mo>&#8727;</mml:mo><mml:mi>I</mml:mi><mml:mrow><mml:mo>(</mml:mo><mml:mi>h</mml:mi><mml:mo>)</mml:mo></mml:mrow><mml:mspace width="1em"/><mml:mspace width="1em"/><mml:mspace width="1em"/><mml:mspace width="1em"/><mml:mspace width="1em"/><mml:mo stretchy="false">(</mml:mo><mml:mn>2</mml:mn><mml:mo stretchy="false">)</mml:mo></mml:mrow></mml:mstyle></mml:math></disp-formula></p><p>K is an unknown parameter but can also be expressed by 1/(k+1) to represent the proportion of prevalent snakebite cases that would have sought in-hospital treatment. Given the estimated I(h), we determined I(h&#8217;) by varying the K values. We applied 1.5, 2.0, and 3.0 as plausible K values (Column 2), corresponding to 40%, 33.3% and 25% of cases who sought treatment (Column 3).</p></fn><fn><p>4. The sum of I(h) and I(h&#8217;) or Columns 4 and 5 is the national snakebite prevalence (Column 6).</p><p>5. Among 44 studies, an average of 70% of patients received antivenom after a diagnosis of systematic envenomation (<xref ref-type="table" rid="app2table1">Appendix 2&#8212;table 1</xref>). We applied this percentage to obtain the expected number of envenomations in India (Column 7). The remainder were &#8220;dry bites&#8221; without envenomation (Column 8).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-54076-v1.xml">elife-54076-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app2table2" position="float"><label>Appendix 2&#8212;table 2.</label><caption><title>Systematic review summary of state coverage by years of 78 studies.</title></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top"/><th valign="top"/><th rowspan="2" valign="top">No. of studies</th><th rowspan="2" valign="top">Total no. of snakebites</th><th rowspan="2" valign="top">Total no. of snakebite deaths</th><th colspan="4" valign="top">Sources of studies</th><th colspan="18" valign="top">Publication year/number of studies</th></tr><tr><th valign="top"/><th valign="top">State</th><th valign="top">Autopsy</th><th valign="top">Community survey</th><th valign="top">EMS</th><th valign="top">Hospital</th><th valign="top">1999</th><th valign="top">2002</th><th valign="top">2004</th><th valign="top">2005</th><th valign="top">2006</th><th valign="top">2007</th><th valign="top">2008</th><th valign="top">2009</th><th valign="top">2010</th><th valign="top">2011</th><th valign="top">2012</th><th valign="top">2013</th><th valign="top">2014</th><th valign="top">2015</th><th valign="top">2016</th><th valign="top">2017</th><th valign="top">2018</th><th valign="top">2019</th></tr></thead><tbody><tr><td valign="top">1</td><td valign="top">Andhra Pradesh</td><td valign="top">4</td><td valign="top">6283</td><td valign="top">1457</td><td valign="top">1</td><td valign="top">1</td><td valign="top">1</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">1</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">2</td><td valign="top">Bihar</td><td valign="top">3</td><td valign="top">1171</td><td valign="top">29</td><td valign="top"/><td valign="top">1</td><td valign="top"/><td valign="top">2</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top">1</td><td valign="top"/><td valign="top"/><td valign="top">2</td></tr><tr><td valign="top">3</td><td valign="top">Chandigarh</td><td valign="top">4</td><td valign="top">297</td><td valign="top">6</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">4</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">1</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">4</td><td valign="top">Chhattisgarh</td><td valign="top">1</td><td valign="top">2084</td><td valign="top">52</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">5</td><td valign="top">Dadra and Nagar Haveli</td><td valign="top">1</td><td valign="top">384</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">6</td><td valign="top">Daman and Diu</td><td valign="top">1</td><td valign="top">12</td><td valign="top"/><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">7</td><td valign="top">Goa</td><td valign="top">1</td><td valign="top">244</td><td valign="top"/><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">8</td><td valign="top">Gujarat</td><td valign="top">1</td><td valign="top">3628</td><td valign="top">91</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">9</td><td valign="top">Haryana</td><td valign="top">1</td><td valign="top">17</td><td valign="top"/><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">10</td><td valign="top">Himachal Pradesh</td><td valign="top">4</td><td valign="top">1442</td><td valign="top">28</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">3</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">2</td><td valign="top">1</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">11</td><td valign="top">Jammu and Kashmir</td><td valign="top">1</td><td valign="top">10</td><td valign="top"/><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">12</td><td valign="top">Jharkhand</td><td valign="top">1</td><td valign="top">356</td><td valign="top">19</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">13</td><td valign="top">Karnataka</td><td valign="top">20</td><td valign="top">5281</td><td valign="top">139</td><td valign="top">1</td><td valign="top"/><td valign="top">1</td><td valign="top">18</td><td valign="top"/><td valign="top"/><td valign="top">1</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top">2</td><td valign="top"/><td valign="top">1</td><td valign="top">4</td><td valign="top">4</td><td valign="top">1</td><td valign="top">3</td><td valign="top">1</td><td valign="top">1</td><td valign="top"/><td valign="top">1</td></tr><tr><td valign="top">14</td><td valign="top">Kerala</td><td valign="top">5</td><td valign="top">3169</td><td valign="top">131</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">5</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">3</td><td valign="top">.</td><td valign="top">1</td><td valign="top">1</td><td valign="top">.</td></tr><tr><td valign="top">15</td><td valign="top">Maharashtra</td><td valign="top">18</td><td valign="top">4884</td><td valign="top">432</td><td valign="top">2</td><td valign="top"/><td valign="top"/><td valign="top">16</td><td valign="top">1</td><td valign="top">1</td><td valign="top"/><td valign="top">1</td><td valign="top">1</td><td valign="top"/><td valign="top">1</td><td valign="top"/><td valign="top"/><td valign="top">1</td><td valign="top"/><td valign="top">2</td><td valign="top">3</td><td valign="top">2</td><td valign="top">1</td><td valign="top">1</td><td valign="top">2</td><td valign="top">1</td></tr><tr><td valign="top">16</td><td valign="top">Meghalaya</td><td valign="top">1</td><td valign="top">13</td><td valign="top"/><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">17</td><td valign="top">New Delhi</td><td valign="top">1</td><td valign="top">62</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">18</td><td valign="top">Odisha</td><td valign="top">2</td><td valign="top">101</td><td valign="top">33</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">2</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td></tr><tr><td valign="top">19</td><td valign="top">Pondicherry</td><td valign="top">1</td><td valign="top">50</td><td valign="top">9</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">20</td><td valign="top">Tamil Nadu</td><td valign="top">7</td><td valign="top">51,198</td><td valign="top">371</td><td valign="top"/><td valign="top">2</td><td valign="top">1</td><td valign="top">4</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top">3</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top">1</td><td valign="top">1</td><td valign="top">1</td><td valign="top"/><td valign="top">1</td><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">21</td><td valign="top">Telangana</td><td valign="top">1</td><td valign="top">3956</td><td valign="top">92</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">22</td><td valign="top">Uttar Pradesh</td><td valign="top">4</td><td valign="top">249</td><td valign="top">90</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">3</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">1</td><td valign="top">.</td><td valign="top">1</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">23</td><td valign="top">Uttarakhand</td><td valign="top">1</td><td valign="top">329</td><td valign="top">4</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td><td valign="top">1</td><td valign="top">.</td><td valign="top">.</td><td valign="top">.</td></tr><tr><td valign="top">24</td><td valign="top">West Bengal</td><td valign="top">6</td><td valign="top">2370</td><td valign="top">345</td><td valign="top">2</td><td valign="top"/><td valign="top"/><td valign="top">4</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top">1</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top">1</td><td valign="top"/><td valign="top">1</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top">1</td><td valign="top">1</td><td valign="top">1</td></tr><tr><td/><td>Total</td><td>90</td><td>87,590</td><td>3329</td><td>7</td><td>4</td><td>12</td><td>67</td><td>1</td><td>1</td><td>1</td><td>4</td><td>2</td><td>1</td><td>4</td><td>3</td><td>1</td><td>4</td><td>8</td><td>9</td><td>9</td><td>13</td><td>15</td><td>4</td><td>4</td><td>6</td></tr></tbody></table><table-wrap-foot><fn><p>Note:</p><p>The 78 studies we reviewed were published over 20 year&#8217;s period and data are corresponding to 1999&#8211;2019 from 24 different states. Except in a few studies, data of individual studies were confined to a single state. Therefore, possibility of overlapping of study data within states would be minimal and we verified the areas when several studies were conducted in the same year within the same state. One study from 1999 was included because no other studies were found around the year of 2000.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-54076-v1.xml">elife-54076-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app3table1" position="float"><label>Appendix 3&#8212;table 1.</label><caption><title>Parameter estimates for covariates in the geostatistical Bayesian Poisson model.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="3" valign="top">Relative risk of snakebite mortality</th></tr><tr><th valign="top">0.5 quantile</th><th valign="top">0.025 quantile</th><th valign="top">0.975 quantile</th></tr></thead><tbody><tr><td valign="top">(Intercept)</td><td valign="top">1.011</td><td valign="top">0.456</td><td valign="top">1.705</td></tr><tr><td valign="top">Urban vs rural</td><td valign="top">0.280</td><td valign="top">0.229</td><td valign="top">0.338</td></tr><tr><td valign="top">Female illiteracy in rural areas</td><td valign="top">1.427</td><td valign="top">1.276</td><td valign="top">1.596</td></tr><tr><td valign="top">Altitude (below 400 m)</td><td valign="top">1.905</td><td valign="top">0.812</td><td valign="top">4.392</td></tr><tr><td valign="top">Altitude (above 400 m)</td><td valign="top">0.508</td><td valign="top">0.305</td><td valign="top">0.824</td></tr><tr><td valign="top">Temperature (below 20&#176;C)</td><td valign="top">4.774</td><td valign="top">1.658</td><td valign="top">15.764</td></tr><tr><td valign="top">Temperature (above 20&#176;C)</td><td valign="top">1.170</td><td valign="top">0.502</td><td valign="top">2.641</td></tr><tr><td valign="top">range/1000 (&#981;/1000)</td><td valign="top">442.001</td><td valign="top">244.373</td><td valign="top">847.835</td></tr><tr><td valign="top">sd of spatial random effect (&#963;)</td><td valign="top">0.587</td><td valign="top">0.442</td><td valign="top">0.796</td></tr><tr><td valign="top">sd of random walk two for year</td><td valign="top">0.003</td><td valign="top">0.001</td><td valign="top">0.007</td></tr><tr><td valign="top">sd of sampling unit effect (&#964;)</td><td valign="top">0.556</td><td valign="top">0.466</td><td valign="top">0.655</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-54136-v2.xml">elife-54136-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Summary of interactions detected in coadaptation screen.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th rowspan="2">Total number</th><th colspan="2">IG1</th><th colspan="2">IG2-90</th></tr><tr><th>Number</th><th>Fraction</th><th>Number</th><th>Fraction</th></tr></thead><tbody><tr><td>SNP-SNP pair</td><td>2.3&#160;&#215;&#160;10<sup>10</sup></td><td>289186</td><td>0.00%</td><td>22751</td><td>0.00%</td></tr><tr><td>SNP-Accessory gene pair</td><td>2.2&#160;&#215;&#160;10<sup>9</sup></td><td>113973</td><td>0.01%</td><td>1188</td><td>0.00%</td></tr><tr><td>Accessory gene-gene pair</td><td>2.1&#160;&#215;&#160;10<sup>8</sup></td><td>13487</td><td>0.01%</td><td>12264</td><td>0.01%</td></tr><tr><td>SNP</td><td>151957</td><td>1540</td><td>1.01%</td><td>333</td><td>0.22%</td></tr><tr><td>Synonymous (Syn)</td><td>117541</td><td>1084</td><td>0.92%</td><td>226</td><td>0.19%</td></tr><tr><td>Nonsynonymous (NonSyn)</td><td>23673</td><td>379</td><td>1.60%</td><td>107</td><td>0.45%</td></tr><tr><td>NonSyn/Syn</td><td>0.2</td><td>0.35</td><td/><td>0.47</td><td/></tr><tr><td>Core gene</td><td>3936</td><td>82</td><td>2.25%</td><td>18</td><td>0.56%</td></tr><tr><td>Accessory gene</td><td>14486</td><td>338</td><td>2.33%</td><td>1122</td><td>7.75%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-54662-v1.xml">elife-54662-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Data and results concerning the side of circumnavigation for each panel in all test conditions of both experiments (<xref ref-type="supplementary-material" rid="sdata5">Source data 5</xref>).</title></caption><table frame="hsides" rules="groups"><thead><tr><th align="center" colspan="10" valign="top">Experiment 1</th></tr></thead><tbody><tr><th colspan="2"/><th align="center" colspan="2">Left panel</th><th align="center" colspan="2">Right panel</th><th rowspan="2" valign="middle">BF</th><th rowspan="2" valign="middle">X<sup>2</sup></th><th rowspan="2" valign="middle">P</th><th rowspan="2" valign="middle">Phi</th></tr><tr><th>Test</th><th><italic>Side</italic></th><th><italic>Left</italic></th><th><italic>Right</italic></th><th><italic>Left</italic></th><th><italic>Right</italic></th></tr><tr><th rowspan="2">2 <italic>vs.</italic> 2</th><th valign="top">Count</th><td valign="top">51</td><td valign="top">35</td><td valign="top">18</td><td valign="top">16</td><td rowspan="2">0.301</td><td rowspan="2">0.403</td><td rowspan="2">0.525</td><td rowspan="2">0.058</td></tr><tr><th valign="top">%</th><td valign="top">59.302</td><td valign="top">40.698</td><td valign="top">52.941</td><td valign="top">47.059</td></tr><tr><th rowspan="2">5 <italic>v</italic>s. 5</th><th valign="top">Count</th><td valign="top">41</td><td valign="top">22</td><td valign="top">25</td><td valign="top">32</td><td rowspan="2">3.314</td><td rowspan="2">5.444</td><td rowspan="2">0.020</td><td rowspan="2">0.213</td></tr><tr><th valign="top">%</th><td valign="top">65.079</td><td valign="top">34.921</td><td valign="top">43.860</td><td valign="top">56.140</td></tr><tr><th rowspan="2">8 <italic>vs.</italic> 8</th><th valign="top">Count</th><td valign="top">21</td><td valign="top">13</td><td valign="top">51</td><td valign="top">35</td><td rowspan="2">0.249</td><td rowspan="2">0.062</td><td rowspan="2">0.804</td><td rowspan="2">0.023</td></tr><tr><th valign="top">%</th><td valign="top">61.765</td><td valign="top">38.235</td><td valign="top">59.302</td><td valign="top">40.698</td></tr><tr><th align="center" colspan="10" valign="top">Experiment 2</th></tr><tr><th colspan="2"/><th align="center" colspan="2">Left Panel</th><th align="center" colspan="2">Right Panel</th><th rowspan="2" valign="middle">BF</th><th rowspan="2" valign="middle">X<sup>2</sup></th><th rowspan="2" valign="middle">p</th><th rowspan="2" valign="middle">Phi</th></tr><tr><th>Test</th><th><italic>Side</italic></th><th><italic>Left</italic></th><th><italic>Right</italic></th><th><italic>Left</italic></th><th><italic>Right</italic></th></tr><tr><th rowspan="2">2 <italic>vs.</italic> 2</th><th valign="top">Count</th><td valign="top">62</td><td valign="top">22</td><td valign="top">20</td><td valign="top">16</td><td rowspan="2">1.512</td><td rowspan="2">3.880</td><td rowspan="2">0.049</td><td rowspan="2">0.180</td></tr><tr><th valign="top"><bold>%</bold></th><td valign="top">73.810</td><td valign="top">26.190</td><td valign="top">55.556</td><td valign="top">44.444</td></tr><tr><th rowspan="2"><bold>5</bold> <italic>v</italic>s. <bold>5</bold></th><th valign="top"><bold>Count</bold></th><td valign="top">36</td><td valign="top">25</td><td valign="top">34</td><td valign="top">25</td><td rowspan="2">0.224</td><td rowspan="2">0.024</td><td rowspan="2">0.877</td><td rowspan="2">0.014</td></tr><tr><th valign="top"><bold>%</bold></th><td valign="top">58.333</td><td valign="top">41.667</td><td valign="top">57.626</td><td valign="top">42.373</td></tr><tr><th rowspan="2"><bold>8</bold> <italic>vs.</italic> <bold>8</bold></th><th valign="top"><bold>Count</bold></th><td valign="top">24</td><td valign="top">16</td><td valign="top">23</td><td valign="top">57</td><td rowspan="2">49.104</td><td rowspan="2">10.930</td><td rowspan="2">&lt;0.001</td><td rowspan="2">0.302</td></tr><tr><th valign="top"><bold>%</bold></th><td valign="top">60</td><td valign="top">40</td><td valign="top">28.750</td><td valign="top">71.250</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-54693-v1.xml">elife-54693-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><label>Table 3.</label><caption><title>Characteristics of discs injected with each treatment.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th>Donor ID</th><th>9</th><th>11</th><th>13</th><th>14</th></tr><tr><th>Age</th><th>66</th><th>69</th><th>40</th><th>53</th></tr></thead><tbody><tr><td valign="top"/><td>Sex</td><td>F</td><td>M</td><td>F</td><td>F</td></tr><tr><td rowspan="3" valign="top">CTRL</td><td>Disc Level</td><td>T12/L1</td><td>L1/2</td><td>L1/2</td><td>L2/3</td></tr><tr><td>&#8195;Grade</td><td>2</td><td>2</td><td>2</td><td>2</td></tr><tr><td>&#8195;Disc Height (cm)</td><td>0.84</td><td>0.92</td><td>0.92</td><td>0.88</td></tr><tr><td rowspan="3" valign="top">O-Vanillin</td><td>Disc Level</td><td>L1/2</td><td>L2/3</td><td>L3/4</td><td>L4/5</td></tr><tr><td>&#8195;Grade</td><td>2</td><td>2</td><td>2</td><td>2</td></tr><tr><td>&#8195;Disc Height (cm)</td><td>0.92</td><td>0.89</td><td>0.99</td><td>0.88</td></tr><tr><td rowspan="3" valign="top">RG-7112</td><td>Disc Level</td><td>T11/T12</td><td>L3/4</td><td>L2/3</td><td>L3/4</td></tr><tr><td>&#8195;Grade</td><td>2</td><td>2</td><td>2</td><td>2</td></tr><tr><td>&#8195;Disc Height (cm)</td><td>0.82</td><td>0.9</td><td>0.93</td><td>0.99</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-54841-v2.xml">elife-54841-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Segregation of Disrupted <italic>Ubiad1</italic> Alleles in Mice.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Genotype of breeding pairs</th><th colspan="3"><italic>Ubiad1</italic> genotype of offspring</th></tr><tr><th>+/+</th><th>+/-</th><th><italic>-/-</italic></th></tr></thead><tbody><tr><td>Disrupted Allele A <break/><italic>Ubiad1<sup>+/-</sup>X Ubiad1<sup>+/-</sup></italic></td><td>83</td><td>201</td><td>0</td></tr><tr><td>Ubiad1<sup>+/-</sup>:&#160;:HmgcrKi<sup>Ki/Ki</sup> <break/><italic>X</italic> <break/>Ubiad1<sup>+/-</sup>:&#160;:HmgcrKi<sup>Ki/Ki</sup></td><td>137</td><td>280</td><td>114</td></tr><tr><td>Disrupted allele B <break/><italic>Ubiad1<sup>+/-</sup>X Ubiad1<sup>+/-</sup></italic></td><td>77</td><td>183</td><td>0</td></tr><tr><td>Ubiad1<sup>+/-</sup>:&#160;:HmgcrKi<sup>Ki/Ki</sup> <break/><italic>X</italic> <break/>Ubiad1<sup>+/-</sup>:&#160;:HmgcrKi<sup>Ki/Ki</sup></td><td>23</td><td>36</td><td>20</td></tr></tbody></table><table-wrap-foot><fn><p>Genotype was determined by PCR analysis of genomic DNA prepared from tails of mice.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-54846-v3.xml">elife-54846-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app4table1" position="float"><label>Appendix 4&#8212;table 1.</label><caption><title>Model fits for various choices of string alignment.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Alignment</th><th colspan="4" valign="top">Letter model correlation</th></tr><tr><th valign="top">&#8195;six vs five&#160;letter&#160;strings</th><th valign="top">six vs four-letter&#160;strings</th><th valign="top">five vs three-letter&#160;strings</th><th valign="top">four vs three-letter&#160;strings</th></tr></thead><tbody><tr><td valign="top">Left: ABCDEF vs EFGHxx</td><td valign="top">0.54</td><td valign="top">0.66</td><td valign="top">0.58</td><td valign="top">0.57</td></tr><tr><td valign="top">Right: ABCDEF vs xxEFGH</td><td valign="top">0.51</td><td valign="top">0.66</td><td valign="top">0.57</td><td valign="top">0.58</td></tr><tr><td valign="top">Centre: ABCDEF vs xEFGHx</td><td valign="top">-</td><td valign="top"><bold>0.68</bold></td><td valign="top">0.58</td><td valign="top">-</td></tr><tr><td valign="top">Edge: ABCDEF vs EFxxGH</td><td valign="top"><bold>0.55</bold></td><td valign="top">0.63</td><td valign="top"><bold>0.60</bold></td><td valign="top"><bold>0.59</bold></td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-55038-v2.xml">elife-55038-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Retention times (RT), MRM-transition stages monitored (precursor ion and product ions) and collision energies of analyzed compounds.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Compounds</th><th colspan="2" valign="top">MRM transition</th><th rowspan="2" valign="top">RT (min)</th><th rowspan="2" valign="top">Collision energy (V)</th></tr><tr><th valign="top">Precursor ion</th><th valign="top">Product ion</th></tr></thead><tbody><tr><td valign="top">Bodipy-LacCer</td><td valign="top">926.5</td><td valign="top">562.4</td><td valign="top">1.48</td><td valign="top">30</td></tr><tr><td valign="top">LacCer(d18:1/16:0)</td><td valign="top">862.6</td><td valign="top">520.5</td><td valign="top">2.84</td><td valign="top">20</td></tr><tr><td valign="top">LacCer(d18:1/14:0)</td><td valign="top">834.6</td><td valign="top">264.3</td><td valign="top">2.8</td><td valign="top">40</td></tr><tr><td valign="top">SM(d18:1/16:0)</td><td valign="top">703.6</td><td valign="top">184.1</td><td valign="top">2.9</td><td valign="top">30</td></tr><tr><td valign="top">Bodipy-SM C<sub>12</sub></td><td valign="top">865.6</td><td valign="top">184.1</td><td valign="top">2.12</td><td valign="top">30</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-55207-v2.xml">elife-55207-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Summary of patient samples and associated sequencing experiments.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Patient ID</th><th rowspan="2">Sample type</th><th rowspan="2">Collection site</th><th colspan="2">Analysis performed</th></tr><tr><th>Bulk WGS</th><th>DigiPico sequencing</th></tr></thead><tbody><tr><td>#11152</td><td>Normal</td><td>Blood</td><td>x</td><td/></tr><tr><td>#11152</td><td>Pre-chemo</td><td>Omentum</td><td>x</td><td>x</td></tr><tr><td>#11152</td><td>Recurrence</td><td>Pelvic tumor</td><td>x</td><td>x</td></tr><tr><td>#11152</td><td>Recurrence</td><td>Lymph node</td><td/><td>x</td></tr><tr><td>#11513</td><td>Normal</td><td>Blood</td><td>x</td><td>x</td></tr><tr><td>#11513</td><td>Pre-chemo</td><td>Ascites</td><td>x</td><td>x</td></tr><tr><td>OP1036</td><td>Normal</td><td>Blood</td><td>x</td><td/></tr><tr><td>OP1036</td><td>Pre-chemo</td><td>Paracolic gutter</td><td>x</td><td>x</td></tr><tr><td>#11502</td><td>Normal</td><td>Blood</td><td>x</td><td/></tr><tr><td>#11502</td><td>Pre-chemo</td><td>Peritoneum</td><td>x</td><td>x</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-55207-v3.xml">elife-55207-v3.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Summary of patient samples and associated sequencing experiments.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Patient ID</th><th rowspan="2">Sample type</th><th rowspan="2">Collection site</th><th colspan="2">Analysis performed</th></tr><tr><th>Bulk WGS</th><th>DigiPico sequencing</th></tr></thead><tbody><tr><td>#11152</td><td>Normal</td><td>Blood</td><td>x</td><td/></tr><tr><td>#11152</td><td>Pre-chemo</td><td>Omentum</td><td>x</td><td>x</td></tr><tr><td>#11152</td><td>Recurrence</td><td>Pelvic tumor</td><td>x</td><td>x</td></tr><tr><td>#11152</td><td>Recurrence</td><td>Lymph node</td><td/><td>x</td></tr><tr><td>#11513</td><td>Normal</td><td>Blood</td><td>x</td><td>x</td></tr><tr><td>#11513</td><td>Pre-chemo</td><td>Ascites</td><td>x</td><td>x</td></tr><tr><td>OP1036</td><td>Normal</td><td>Blood</td><td>x</td><td/></tr><tr><td>OP1036</td><td>Pre-chemo</td><td>Paracolic gutter</td><td>x</td><td>x</td></tr><tr><td>#11502</td><td>Normal</td><td>Blood</td><td>x</td><td/></tr><tr><td>#11502</td><td>Pre-chemo</td><td>Peritoneum</td><td>x</td><td>x</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-55571-v2.xml">elife-55571-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>C-linker scrambling mutations and measured V<sub>0.5</sub> in the full-length at both 0 [Ca<sup>2+</sup>] and 100 &#956;M [Ca<sup>2+</sup>] and Core-MT BK channels at 0 [Ca<sup>2+</sup>].</title><p>The Core-MT constructs are based on the TMD, C-linker of mSlo1, and an 11-residue tail from K<sub>V</sub> 1.4 of the mouse Shaker family (<xref ref-type="bibr" rid="bib5">Budelli et al., 2013</xref>; <xref ref-type="bibr" rid="bib58">Zhang et al., 2017</xref>). The location of the nearest Tyr to the S6 C-terminal is underlined. K0 (Y330G) was designed to remove the Tyr sidechain in the K0 background.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top">Mutation</th><th rowspan="2" valign="top">Sequence</th><th colspan="3" valign="top">V<sub>0.5</sub> (mV)</th></tr><tr><th colspan="2" valign="top">Full-Length</th><th valign="top">Core-MT</th></tr></thead><tbody><tr><td valign="top"/><td valign="top"/><td valign="top">0 [Ca<sup>2+</sup>]</td><td valign="top">100 [Ca<sup>2+</sup>]</td><td valign="top">0 [Ca<sup>2+</sup>]</td></tr><tr><td valign="top">WT</td><td valign="top">EIIEL IGNRK K<underline>Y</underline>GGS YSAVS GRK</td><td valign="top">183.4</td><td valign="top">0.2</td><td valign="top">235.0</td></tr><tr><td valign="top">K0</td><td valign="top">EIIEL IGNR<underline>Y</underline> GKGSK YSRAV SKG</td><td valign="top">89.6</td><td valign="top">&#8722;66.7</td><td valign="top">192.6</td></tr><tr><td valign="top">K0(Y330G)</td><td valign="top">EIIEL IGNR<underline>G</underline> GKGSK YSRAV SKG</td><td valign="top">169.8</td><td valign="top">47.5</td><td valign="top">NM</td></tr><tr><td valign="top">K1</td><td valign="top">EIIEL RIGNK <underline>Y</underline>GGSY KSAVR KSG</td><td valign="top">136.2</td><td valign="top">&#8722;4.2</td><td valign="top">NM</td></tr><tr><td valign="top">K2</td><td valign="top">EIIEL IGRKN <underline>Y</underline>KGGS YSARV SGK</td><td valign="top">195.5</td><td valign="top">59.0</td><td valign="top">263.5</td></tr><tr><td valign="top">K3</td><td valign="top">EIIEL IGN<underline>Y</underline>G GRSYS KAKVS RKG</td><td valign="top">NC</td><td valign="top">NC</td><td valign="top">NM</td></tr><tr><td valign="top">K4</td><td valign="top">EIIEL IGRN<underline>Y</underline> GGSYS AKKVR SKG</td><td valign="top">94.6</td><td valign="top">&#8722;50.4</td><td valign="top">NM</td></tr><tr><td valign="top">K5</td><td valign="top">EIIER LIGKK RN<underline>Y</underline>KG GSYSA VSG</td><td valign="top">NC</td><td valign="top">NC</td><td valign="top">NM</td></tr><tr><td valign="top">K6</td><td valign="top">EIIEL RKKIR KGN<underline>Y</underline>G GSYSA VSG</td><td valign="top">NC</td><td valign="top">NC</td><td valign="top">NM</td></tr><tr><td valign="top">K7</td><td valign="top">EIIEL IGN<underline>Y</underline>G GSYSA VRKSK GRK</td><td valign="top">48.7</td><td valign="top">&#8722;63.7</td><td valign="top">167.5</td></tr><tr><td valign="top"/><td colspan="4" valign="top">NC: no current, channel could not be expressed; NM: Mutation Not Made</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56211-v1.xml">elife-56211-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Dream characteristics.</title><p><supplementary-material id="table2sdata1"><label>Table 2&#8212;source data 1.</label><caption><title>This file contains the individual participant data for every dream-related measure that is summarised in <xref ref-type="table" rid="table2">Table 2</xref>.</title></caption><media mime-subtype="xlsx" mimetype="application" xlink:href="elife-56211-table2-data1-v1.xlsx"/></supplementary-material></p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="bottom"/><th>HPC</th><th>CTL</th><th valign="bottom"/><th valign="bottom"/><th valign="bottom"/><th rowspan="2" valign="middle">HPC1</th><th rowspan="2" valign="middle">HPC2</th><th rowspan="2" valign="middle">HPC3</th><th rowspan="2" valign="middle">HPC4</th></tr><tr><th/><th>M (SD)</th><th>M (SD)</th><th>U</th><th>ES</th><th>P-Value</th></tr></thead><tbody><tr><td><italic>General analyses<sup>a</sup></italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td>Number of awakenings</td><td>6.75 (2.75)</td><td>6.90 (1.97)</td><td>18.5</td><td>0.11</td><td>0.829</td><td valign="bottom">10.00</td><td valign="bottom">4.00</td><td valign="bottom">5.00</td><td valign="bottom">8.00</td></tr><tr><td>Proportion of awakenings during NREM</td><td>0.73 (0.09)</td><td>0.67 (0.13)</td><td>14.0</td><td>0.47</td><td>0.383</td><td valign="bottom">0.80</td><td valign="bottom">0.75</td><td valign="bottom">0.60</td><td valign="bottom">0.75</td></tr><tr><td>Proportion of awakenings during REM</td><td>0.23 (0.17)</td><td>0.33 (0.13)</td><td>13.0</td><td>0.55</td><td>0.309</td><td valign="bottom">0.00<sup>c</sup></td><td valign="bottom">0.25</td><td valign="bottom">0.40</td><td valign="bottom">0.25</td></tr><tr><td>Number of probes per awakening</td><td>3.82 (1.75)</td><td>4.22 (1.13)</td><td>17.0</td><td>0.23</td><td>0.671</td><td valign="bottom">5.80</td><td valign="bottom">3.50</td><td valign="bottom">1.60</td><td valign="bottom">4.38</td></tr><tr><td>Dream frequency</td><td>0.23 (0.17)</td><td>0.67 (0.28)</td><td>4.5</td><td>1.45</td><td><bold>0.028</bold></td><td valign="bottom">0.40</td><td valign="bottom">0.25</td><td valign="bottom">0.00</td><td valign="bottom">0.25</td></tr><tr><td>Proportion of dreams during NREM</td><td>0.38 (0.48)</td><td>0.52 (0.20)</td><td>14.5</td><td>0.43</td><td>0.415</td><td valign="bottom">1.00</td><td valign="bottom">0.00</td><td valign="bottom">0.00</td><td valign="bottom">0.50</td></tr><tr><td>Proportion of dreams during REM</td><td>0.38 (0.48)</td><td>0.48 (0.20)</td><td>16.0</td><td>0.31</td><td>0.555</td><td valign="bottom">0.00</td><td valign="bottom">1.00</td><td valign="bottom">0.00</td><td valign="bottom">0.50</td></tr><tr><td>Proportion of no dreams</td><td>0.65 (0.31)</td><td>0.21 (0.24)</td><td>4.0</td><td>1.52</td><td><bold>0.022</bold></td><td valign="bottom">0.60</td><td valign="bottom">0.75</td><td valign="bottom">1.00</td><td valign="bottom">0.25</td></tr><tr><td>Proportion of blank dreams</td><td>0.13 (0.25)</td><td>0.12 (0.15)</td><td>15.5</td><td>0.35</td><td>0.496</td><td valign="bottom">0.00</td><td valign="bottom">0.00</td><td valign="bottom">0.00</td><td valign="bottom">0.50</td></tr><tr><td><italic>Overall qualitative attributes<sup>b</sup></italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td>Number of informative words</td><td>43.17 (16.06)</td><td>95.55 (55.20)</td><td>7.0</td><td>0.81</td><td>0.176</td><td valign="bottom">55.50</td><td valign="bottom">25.00</td><td valign="bottom">.</td><td valign="bottom">49.00</td></tr><tr><td>Complexity</td><td>2.67 (0.58)</td><td>3.32 (0.70)</td><td>5.5</td><td>1.00</td><td>0.098</td><td valign="bottom">3.00</td><td valign="bottom">2.00</td><td valign="bottom">.</td><td valign="bottom">3.00</td></tr><tr><td>Vividness</td><td>3.10 (0.79)</td><td>3.88 (1.08)</td><td>7.5</td><td>0.75</td><td>0.203</td><td valign="bottom">2.80</td><td valign="bottom">4.00</td><td valign="bottom">.</td><td valign="bottom">2.50</td></tr><tr><td>Bizarreness</td><td>1.58 (1.01)</td><td>2.27 (0.95)</td><td>8.0</td><td>0.70</td><td>0.232</td><td valign="bottom">2.75</td><td valign="bottom">1.00</td><td valign="bottom">.</td><td valign="bottom">1.00</td></tr><tr><td>Emotional valence</td><td>2.75 (0.25)</td><td>2.81 (0.22)</td><td>13.0</td><td>0.19</td><td>0.720</td><td valign="bottom">2.75</td><td valign="bottom">3.00</td><td valign="bottom">.</td><td valign="bottom">2.50</td></tr><tr><td>Proportion of self-references</td><td>0.84 (0.29)</td><td>0.90 (0.19)</td><td>15.0</td><td>0.00</td><td>1.000</td><td valign="bottom">1.00</td><td valign="bottom">1.00</td><td valign="bottom">.</td><td valign="bottom">0.50</td></tr><tr><td><italic>Content characterization<sup>b</sup></italic></td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td>Internal (episodic) details</td><td>4.08 (1.47)</td><td>9.13 (3.56)</td><td>2.0</td><td>1.54</td><td><bold>0.028</bold></td><td valign="bottom">5.75</td><td valign="bottom">3.00</td><td valign="bottom">.</td><td valign="bottom">3.50</td></tr><tr><td>External (semantic/other) details</td><td>0.17 (0.29)</td><td>1.07 (1.60)</td><td>8.5</td><td>0.64</td><td>0.258</td><td valign="bottom">0.00</td><td valign="bottom">0.00</td><td valign="bottom">.</td><td valign="bottom">0.50</td></tr><tr><td>Content score</td><td>5.75 (0.90)</td><td>13.10 (6.49)</td><td>1.0</td><td>1.74</td><td><bold>0.018</bold></td><td valign="bottom">6.75</td><td valign="bottom">5.00</td><td valign="bottom">.</td><td valign="bottom">5.50</td></tr><tr><td/><td/><td/><td/><td/><td/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr></tbody></table><table-wrap-foot><fn><p>M&#160;=&#160;mean; SD&#160;=&#160;standard deviation; ES&#160;=&#160;effect size; HPC&#160;=&#160;hippocampal-damaged patients; CTL&#160;=&#160;control participants; NREM&#160;=&#160;non-rapid eye movement sleep; REM&#160;=&#160;rapid eye movement sleep; HPC1&#8722;4&#160;=&#160;each individual hippocampal-damaged patient. P-values relate to between-group non-parametric Mann-Whitney U tests with significant differences depicted in bold. <sup>a</sup>All patients included; means are per awakening. <sup>b</sup>HCP3, who had no dream reports at all, was excluded; means are per dream report. <sup>c</sup>For HCP1, during 20% of his awakenings towards the end of the night, the EEG cap stopped functioning and so designation to NREM or REM sleep was not possible. Hence, it could be that this zero score for REM awakenings is an underestimate, given that REM is more common in the latter part of the night. Note that his dream reports from these awakenings were still included in the dream qualitative attributes and content analyses. See <xref ref-type="supplementary-material" rid="table2sdata1">Table 2&#8212;source data 1</xref> file for the data underpinning this table.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56236-v1.xml">elife-56236-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>H1N1 swine gamma virus characterization before and after infection and transmission in ferrets.</title></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="2" rowspan="2" valign="top">Virus characterization</th><th colspan="2" valign="top">Pair 1 viruses</th><th colspan="2" valign="top">Pair 2 viruses</th></tr><tr><th>G15</th><th>P4</th><th>P19</th><th>P24</th></tr></thead><tbody><tr><td rowspan="2">HA activation pH<sup>*</sup></td><td valign="top">Wild-type viruses</td><td>5.8</td><td>5.5</td><td>5.9</td><td>5.6</td></tr><tr><td valign="top">Recombinant (6+2)<sup>&#8224;</sup></td><td>5.8</td><td>5.5</td><td>6.0</td><td>5.6</td></tr><tr><td>HA inactivation pH<sup>&#8225;</sup></td><td valign="top">Wild-type viruses</td><td>5.8</td><td>5.6</td><td>5.9</td><td>5.7</td></tr><tr><td>Genome variations</td><td valign="top"/><td>HA1-N210 <break/>PB2-L648 <break/>PA-P271</td><td>HA1-S210 <break/>PB2-M648 <break/>PA-T271</td><td>HA2-T117</td><td>HA2-N117</td></tr><tr><td rowspan="2">Replication in vitro</td><td valign="top">ST cells</td><td colspan="2">Similar</td><td colspan="2">Similar</td></tr><tr><td valign="top">MDCK cells</td><td colspan="2">Similar</td><td colspan="2">Similar</td></tr><tr><td rowspan="2">Receptor binding specificity<sup>&#167;</sup></td><td valign="top">&#945;2, 6</td><td>Lower</td><td>Higher</td><td colspan="2">Similar</td></tr><tr><td valign="top">&#945;2, 3</td><td>Higher</td><td>Lower</td><td colspan="2">Similar</td></tr><tr><td rowspan="6">Donor ferrets</td><td valign="top">Viruses isolated</td><td>3/3</td><td>3/3</td><td>3/3</td><td>3/3</td></tr><tr><td valign="top">Seroconversion</td><td>3/3</td><td>3/3</td><td>3/3</td><td>3/3</td></tr><tr><td valign="top">Day of peak titer</td><td>3</td><td>2.3 (&#177;1.2)</td><td>4.3 (&#177;1.2)</td><td>1.7 (&#177;1.2)</td></tr><tr><td valign="top"><italic>P</italic> (<italic>t</italic>-test), peak titer day</td><td colspan="2">0.21</td><td colspan="2">0.047</td></tr><tr><td valign="top">Peak titers (log<sub>10</sub> TCID<sub>50</sub>)</td><td>6.4 (&#177;0.54)</td><td>6.7 (&#177;0.45)</td><td>6.6 (&#177;0.19)</td><td>6.9 (&#177;0.89)</td></tr><tr><td valign="top">HA activation pH range</td><td>5.6&#8211;5.9</td><td>5.5&#8211;5.6</td><td>5.6&#8211;6.0</td><td>5.5-.6</td></tr><tr><td valign="top"/><td>Major mutants</td><td>HA1-N210S</td><td>None</td><td>HA2-T117N</td><td>None</td></tr><tr><td rowspan="3">Contact ferrets</td><td valign="top">Viruses isolated</td><td>3/3</td><td>3/3</td><td>3/3</td><td>3/3</td></tr><tr><td valign="top">Seroconversion</td><td>3/3</td><td>3/3</td><td>3/3</td><td>3/3</td></tr><tr><td valign="top">HA activation pH range</td><td>5.6&#8211;5.9</td><td>5.5&#8211;5.6</td><td>5.6&#8211;5.9</td><td>5.6&#8211;5.7</td></tr><tr><td valign="top"/><td>Major mutants</td><td>HA1-N210S</td><td>None</td><td>HA2-T117N</td><td>None</td></tr><tr><td rowspan="4">Airborne ferrets</td><td valign="top">Viruses detected</td><td>2/3</td><td>1/3</td><td>1/3</td><td>0/3</td></tr><tr><td valign="top">Seroconversion</td><td>3/3</td><td>1/3</td><td>1/3</td><td>0/3</td></tr><tr><td>HA activation pH<sup>&#182;</sup></td><td>5.6</td><td>5.5</td><td>5.6</td><td>NA<sup>**</sup></td></tr><tr><td>Major variants</td><td>HA1-N210S</td><td>None</td><td>HA2-T117N</td><td>NA</td></tr></tbody></table><table-wrap-foot><fn><p><sup>*</sup>HA activation pH measured by syncytia assay.</p><p><sup>&#8224;</sup>Recombinant 6+2 viruses contained HA and NA genes from swine gamma isolates and the six internal genes from A/TN/1-560/09 (H1N1).</p></fn><fn><p><sup>&#8225;</sup>HA inactivation pH measured by acid-induced inactivation pH with TCID<sub>50</sub> readout.</p><p><sup>&#167;</sup>Receptor binding specificity measured by solid-phase receptor binding assay.</p></fn><fn><p><sup>&#182;</sup>HA activation pH of airborne-transmitted virus on first day of isolation.</p><p><sup>**</sup>NA, Not applicable.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56367-v1.xml">elife-56367-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Summary by dataset of the prevalence and distribution of the genetic markers of resistance and resistance phenotypes tested.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Variant</th><th/><th colspan="3">Genetic</th><th colspan="2">Phenotypic</th></tr><tr><th/><th>RplD G70D</th><th>23S rRNA C2611T (2&#8211;4 alleles)</th><th><italic>penA</italic> XXXIV</th><th>CRO-RS (&#8805;0.12 &#956;g/mL)</th><th>CFX-R (&gt;0.25 &#956;g/mL)</th></tr></thead><tbody><tr><td>Drug</td><td/><td>AZM (<xref ref-type="bibr" rid="bib31">Grad et al., 2016</xref>)</td><td>AZM (<xref ref-type="bibr" rid="bib49">Lk et al., 2002</xref>)</td><td>ESCs (<xref ref-type="bibr" rid="bib30">Grad et al., 2014</xref>)</td><td>N/A</td><td>N/A</td></tr><tr><td rowspan="5">Prevalence of variant in dataset</td><td valign="top">1</td><td>10.04%<sup>*</sup></td><td>0.11%</td><td>5.25%</td><td>1.47%</td><td>0.11%</td></tr><tr><td valign="top">2</td><td>1.14%</td><td>1.24%</td><td>1.69%</td><td>0%</td><td>0%</td></tr><tr><td valign="top">3</td><td>2.47%</td><td>0.95%</td><td>15.68%<sup>*</sup></td><td>1.04%</td><td>0.76%</td></tr><tr><td valign="top">4</td><td>11.07%<sup>*</sup></td><td>1.23%</td><td>0.41%</td><td>6.56%</td><td>8.20%</td></tr><tr><td valign="top">5</td><td>0.75%</td><td>0.50%</td><td>2.26%</td><td>0.25%</td><td>0%</td></tr><tr><td rowspan="5">Phylogenetic D statistic for variant in dataset</td><td valign="top">1</td><td>&#8722;0.18</td><td>17.50</td><td>&#8722;0.29</td><td>N/A</td><td>N/A</td></tr><tr><td valign="top">2</td><td>&#8722;0.10</td><td>0.46</td><td>&#8722;0.24</td><td>N/A</td><td>N/A</td></tr><tr><td valign="top">3</td><td>0.05</td><td>0.30</td><td>&#8722;0.20</td><td>N/A</td><td>N/A</td></tr><tr><td valign="top">4</td><td>&#8722;0.16</td><td>1.83</td><td>1.81</td><td>N/A</td><td>N/A</td></tr><tr><td valign="top">5</td><td>0.83</td><td>1.12</td><td>&#8722;0.15</td><td>N/A</td><td>N/A</td></tr></tbody></table><table-wrap-foot><fn><p>*Given the&#160;&gt;10% prevalence of RplD G70D in datasets 1 and 4 and <italic>penA</italic> XXXIV in dataset 3, these variants were excluded from sampling simulations.</p><p>AZM, azithromycin; ESC, extended-spectrum cephalosporin; CRO-RS, ceftriaxone reduced susceptibility; CFX-R, cefixime resistance.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56367-v1.xml">elife-56367-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><label>Table 3.</label><caption><title>Summary of the potential diagnostic escape variants assessed.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Variant</th><th rowspan="2">Diagnostic assay</th><th rowspan="2">Documented association with diagnostic failure</th><th colspan="5">Prevalence in dataset</th></tr><tr><th>1</th><th>2</th><th>3</th><th>4</th><th>5</th></tr></thead><tbody><tr><td>16S rRNA C1209A (four alleles)</td><td>Aptima GC Combo</td><td>Yes (<xref ref-type="bibr" rid="bib32">Guglielmino et al., 2019</xref>)</td><td>0.11%</td><td>0.09%</td><td>0%</td><td>0%</td><td>0%</td></tr><tr><td><italic>N. meningitidis</italic>-like <italic>porA</italic></td><td>In-house (<xref ref-type="bibr" rid="bib71">Whiley et al., 2004</xref>; <xref ref-type="bibr" rid="bib72">Whiley et al., 2005</xref>)</td><td>Yes (<xref ref-type="bibr" rid="bib73">Whiley et al., 2011</xref>; <xref ref-type="bibr" rid="bib28">Golparian et al., 2012</xref>)</td><td>0.11%</td><td>0.05%</td><td>0%</td><td>0%</td><td>0%</td></tr><tr><td><italic>cppB</italic> deletion</td><td>In-house (<xref ref-type="bibr" rid="bib18">Diemert et al., 2002</xref>; <xref ref-type="bibr" rid="bib68">Van Dyck et al., 2001</xref>)</td><td>Yes (<xref ref-type="bibr" rid="bib11">Bruisten et al., 2004</xref>)</td><td>1.12%</td><td>0.05%</td><td>0.47%</td><td>0%</td><td>7.29%</td></tr><tr><td>DR-9A G168A</td><td>Roche COBAS 4800 CT/NG</td><td>No</td><td>0%</td><td>0.09%</td><td>0%</td><td>0%</td><td>0%</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56418-v2.xml">elife-56418-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Nucleotide levels in dark- or light-adapted murine retinas, determined by HPLC.</title><p>Retinas were obtained from 16 dark-adapted mice that were either kept in the dark for 4 hr, or exposed to 4 hr bright light (1600 lux). HPLC determinations, with numbers indicating Mean&#160;&#177;&#160;S.E.M, with n&#160;=&#160;4 biological replicates.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2">Pmol/mg protein</th></tr><tr><th>DARK</th><th>4 hr BL</th></tr></thead><tbody><tr><td>GMP</td><td>557.21&#160;&#177;&#160;88.5</td><td>916.31&#160;&#177;&#160;81.1</td></tr><tr><td>GDP</td><td>762.57&#160;&#177;&#160;47.8</td><td>1174.11&#160;&#177;&#160;109</td></tr><tr><td>GTP</td><td>2461.53&#160;&#177;&#160;178.1</td><td>2737.29&#160;&#177;&#160;310.6</td></tr><tr><td>AMP</td><td>731.84&#160;&#177;&#160;125.4</td><td>1246.16&#160;&#177;&#160;69.8</td></tr><tr><td>ATP</td><td>1579.69&#160;&#177;&#160;110.5</td><td>2759.18&#160;&#177;&#160;226</td></tr><tr><td>GTP/ATP</td><td>1.56</td><td>0.99</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56816-v1.xml">elife-56816-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Persistent firing probability evoked with a 100 pA, 150 pA, and 200 pA training stimulus with statistical differences between YN and AN.</title><p>Related to <xref ref-type="fig" rid="fig1">Figure 1B</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Group</th><th valign="top"><italic>n</italic></th><th valign="top">100 pA (%)</th><th valign="top"><italic>p</italic>-value</th><th valign="top"><italic>n</italic></th><th valign="top">150 pA (%)</th><th valign="top"><italic>p</italic>-value</th><th valign="top"><italic>n</italic></th><th valign="top">200 pA (%)</th><th valign="top">p-Value</th></tr></thead><tbody><tr><td valign="top">YN</td><td valign="top">25</td><td valign="top">97.3&#160;&#177;&#160;2.7</td><td valign="top"/><td valign="top">18</td><td valign="top">96.3&#160;&#177;&#160;2.5</td><td valign="top"/><td valign="top">16</td><td valign="top">85.4&#160;&#177;&#160;6.1</td><td valign="top"/></tr><tr><td valign="top">AN</td><td valign="top">18</td><td valign="top">72.2&#160;&#177;&#160;8.6</td><td valign="top">vs. YN, <break/>0.0016**</td><td valign="top">10</td><td valign="top">56.7&#160;&#177;&#160;14.1</td><td valign="top">vs. YN, <break/>0.0029**</td><td valign="top">10</td><td valign="top">66.7&#160;&#177;&#160;13.2</td><td valign="top">vs. YN, <break/>0.2458</td></tr><tr><th rowspan="2" valign="top">Group</th><th colspan="3" valign="top">100 pA</th><th colspan="3" valign="top">150 pA</th><th colspan="3" valign="top">200 pA</th></tr><tr><th valign="top">Q1 (25%)</th><th valign="top">Median</th><th valign="top">Q3 (75%)</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th></tr><tr><td valign="top">YN</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">1.00</td></tr><tr><td valign="top">AN</td><td valign="top">0.33</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.00</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">0.25</td><td valign="top">0.83</td><td valign="top">1.00</td></tr></tbody></table><table-wrap-foot><fn><p><italic>n</italic>, number of cells in group; <italic>p</italic>-value, Mann-Whitney.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56816-v1.xml">elife-56816-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Persistent firing probability evoked with a 20 Hz current pulses at various stimulus lengths and membrane holding potentials with statistical differences between YN and AN.</title><p>Related to <xref ref-type="fig" rid="fig1">Figure 1D, F, H</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Group</th><th valign="top"><italic>n</italic></th><th valign="top">20 Hz, 2 s, <break/>2 mV below (%)</th><th valign="top">p-Value</th><th valign="top"><italic>n</italic></th><th valign="top">20 Hz, 250 ms, 2 mV below (%)</th><th valign="top">p-Value</th><th valign="top"><italic>n</italic></th><th valign="top">20 Hz, 250 ms, 5 mV below (%)</th><th valign="top">p-Value</th></tr></thead><tbody><tr><td valign="top">YN</td><td valign="top">14</td><td valign="top">97.6&#160;&#177;&#160;2.4</td><td valign="top"/><td valign="top">22</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top"/><td valign="top">21</td><td valign="top">39.7&#160;&#177;&#160;10.2</td><td valign="top"/></tr><tr><td valign="top">AN</td><td valign="top">10</td><td valign="top">66.7&#160;&#177;&#160;11.1</td><td valign="top">vs. YN, <break/>0.0052 **</td><td valign="top">12</td><td valign="top">66.7&#160;&#177;&#160;11.6</td><td valign="top">vs. Young, <break/>0.0007 **</td><td valign="top">14</td><td valign="top">16.7&#160;&#177;&#160;9.7</td><td valign="top">vs. YN, <break/>0.1304</td></tr><tr><th rowspan="2" valign="top">Group</th><th colspan="3" valign="top">20 Hz, 2s, 2 mV below</th><th colspan="3" valign="top">20 Hz, 250 ms, 2 mV below</th><th colspan="3" valign="top">20 Hz, 250 ms, 5 mV below</th></tr><tr><th valign="top">Q1 (25%)</th><th valign="top">Median</th><th valign="top">Q3 (75%)</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th></tr><tr><td valign="top">YN</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.00</td><td valign="top">0.00</td><td valign="top">1.00</td></tr><tr><td valign="top">AN</td><td valign="top">0.33</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">0.33</td><td valign="top">0.83</td><td valign="top">1.00</td><td valign="top">0.00</td><td valign="top">0.00</td><td valign="top">0.08</td></tr></tbody></table><table-wrap-foot><fn><p><italic>n</italic>, number of cells in group; <italic>p</italic>-value, Mann-Whitney.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56816-v1.xml">elife-56816-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><label>Table 4.</label><caption><title>Persistent firing probability evoked with a 20 Hz current pulses at various stimulus lengths and membrane holding potentials with statistical differences between YC, YP, AU, and AI.</title><p>Related to <xref ref-type="fig" rid="fig2">Figure 2C, D, E</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Group</th><th valign="top"><italic>n</italic></th><th valign="top">20 Hz, 2 s, <break/>2 mV below (%)</th><th valign="top">p-Value</th><th valign="top"><italic>n</italic></th><th valign="top">20 Hz, 250 ms, 2 mV below (%)</th><th valign="top">p-Value</th><th valign="top"><italic>n</italic></th><th valign="top">20 Hz, 250 ms, 5 mV below (%)</th><th valign="top">p-Value</th></tr></thead><tbody><tr><td valign="top">YC</td><td valign="top">8</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YP, <break/>&gt;0.9999</td><td valign="top">12</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YP, <break/>&gt;0.9999</td><td valign="top">11</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YP, <break/>0.0063**</td></tr><tr><td valign="top">YP</td><td valign="top">8</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top"/><td valign="top">11</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top"/><td valign="top">13</td><td valign="top">41.0&#160;&#177;&#160;10.8</td><td valign="top"/></tr><tr><td rowspan="2" valign="top">AU</td><td rowspan="2" valign="top">7</td><td rowspan="2" valign="top">95.2&#160;&#177;&#160;4.8</td><td valign="top">vs. YC, <break/>&gt;0.9999</td><td rowspan="2" valign="top">19</td><td rowspan="2" valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YC, <break/>&gt;0.9999</td><td rowspan="2" valign="top">15</td><td rowspan="2" valign="top">73.34&#160;&#177;&#160;9.3</td><td valign="top">vs. YC, <break/>0.5802</td></tr><tr><td valign="top">vs. YP, <break/>&gt;0.9999</td><td valign="top">vs. YP, <break/>&gt;0.9999</td><td valign="top">vs. YP, <break/>0.4308</td></tr><tr><td rowspan="3" valign="top">AI</td><td rowspan="3" valign="top">11</td><td rowspan="3" valign="top">72.7&#160;&#177;&#160;8.8</td><td valign="top">vs. AU, <break/>0.2046</td><td rowspan="3" valign="top">18</td><td rowspan="3" valign="top">85.19&#160;&#177;&#160;6.2</td><td valign="top">vs. AU, <break/>0.0049**</td><td rowspan="3" valign="top">16</td><td rowspan="3" valign="top">12.5&#160;&#177;&#160;6.0</td><td valign="top">vs. AU, <break/>0.0008**</td></tr><tr><td valign="top">vs. YC, <break/>0.0236*</td><td valign="top">vs. YC, <break/>0.0188*</td><td valign="top">vs. YC, <break/>&lt;0.0001***</td></tr><tr><td valign="top">vs. YP, <break/>0.0236*</td><td valign="top">vs. YP, <break/>0.0241*</td><td valign="top">vs. YP, <break/>0.3874</td></tr><tr><th rowspan="2" valign="top">Group</th><th colspan="3" valign="top">20 Hz, 2s, 2 mV below</th><th colspan="3" valign="top">20 Hz, 250 ms, 2 mV below</th><th colspan="3" valign="top">20 Hz, 250 ms, 5 mV below</th></tr><tr><th valign="top">Q1 (25%)</th><th valign="top">Median</th><th valign="top">Q3 (75%)</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th></tr><tr><td valign="top">YC</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td></tr><tr><td valign="top">YP</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.00</td><td valign="top">0.33</td><td valign="top">0.83</td></tr><tr><td valign="top">AU</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">1.00</td></tr><tr><td valign="top">AI</td><td valign="top">0.33</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.00</td><td valign="top">0.00</td><td valign="top">0.25</td></tr></tbody></table><table-wrap-foot><fn><p><italic>n</italic>, number of cells in group; <italic>p</italic>-value, Dunn&#8217;s multiple comparisons test.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56816-v1.xml">elife-56816-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table5" position="float"><label>Table 5.</label><caption><title>Persistent firing probability evoked with a 100 pA, 150 pA, and 200 pA training stimulus with statistical differences between YC, YP, AU, and AI.</title><p>Related to <xref ref-type="fig" rid="fig2s1">Figure 2&#8212;figure supplement 1</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Group</th><th valign="top"><italic>n</italic></th><th valign="top">100 pA (%)</th><th valign="top">p<italic>-</italic>Value</th><th valign="top"><italic>n</italic></th><th valign="top">150 pA (%)</th><th valign="top">p-Value</th><th valign="top"><italic>n</italic></th><th valign="top">200 pA (%)</th><th valign="top">p-Value</th></tr></thead><tbody><tr><td valign="top">YC</td><td valign="top">14</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YP,&gt;0.9999</td><td valign="top">10</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YP,&gt;0.9999</td><td valign="top">8</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YP,&gt;0.9999</td></tr><tr><td valign="top">YP</td><td valign="top">8</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top"/><td valign="top">4</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top"/><td valign="top">4</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top"/></tr><tr><td rowspan="2" valign="top">AU</td><td rowspan="2" valign="top">17</td><td rowspan="2" valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YC,&gt;0.9999</td><td rowspan="2" valign="top">11</td><td rowspan="2" valign="top">81.8&#160;&#177;&#160;10.4</td><td valign="top">vs. YC, <break/>&gt;0.9999</td><td rowspan="2" valign="top">7</td><td rowspan="2" valign="top">95.2&#160;&#177;&#160;4.8</td><td valign="top">vs. YC,&gt;0.9999</td></tr><tr><td valign="top">vs. YP, <break/>&gt;0.9999</td><td valign="top">vs. YP, <break/>&gt;0.9999</td><td valign="top">vs. YP, <break/>&gt;0.9999</td></tr><tr><td rowspan="3" valign="top">AI</td><td rowspan="3" valign="top">16</td><td rowspan="3" valign="top">68.8&#160;&#177;&#160;10.3</td><td valign="top">vs. AU, 0.0012**</td><td rowspan="3" valign="top">14</td><td rowspan="3" valign="top">57.1&#160;&#177;&#160;8.9</td><td valign="top">vs. AU, <break/>0.1846</td><td rowspan="3" valign="top">13</td><td rowspan="3" valign="top">61.5&#160;&#177;&#160;11.8</td><td valign="top">vs. AU, 0.2366</td></tr><tr><td valign="top">vs. YC, <break/>0.0024**</td><td valign="top">vs. YC, <break/>0.0028**</td><td valign="top">vs. YC, 0.0356*</td></tr><tr><td valign="top">vs. YP, <break/>0.0164*</td><td valign="top">vs. YP, <break/>0.0639~</td><td valign="top">vs. YP, 0.1836</td></tr><tr><th rowspan="2" valign="top">Group</th><th colspan="3" valign="top">100 pA</th><th colspan="3" valign="top">150 pA</th><th colspan="3" valign="top">200 pA</th></tr><tr><th valign="top">Q1 (25%)</th><th valign="top">Median</th><th valign="top">Q3 (75%)</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th></tr><tr><td valign="top">YC</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td></tr><tr><td valign="top">YP</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td></tr><tr><td valign="top">AU</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td></tr><tr><td valign="top">AI</td><td valign="top">0.33</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.33</td><td valign="top">0.50</td><td valign="top">1.00</td><td valign="top">0.17</td><td valign="top">0.67</td><td valign="top">1.00</td></tr></tbody></table><table-wrap-foot><fn><p><italic>n,</italic> number of cells in group; <italic>p-</italic>value, Dunn&#8217;s multiple comparisons test.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56816-v2.xml">elife-56816-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Persistent firing probability evoked with a 100 pA, 150 pA, and 200 pA training stimulus with statistical differences between YN and AN.</title><p>Related to <xref ref-type="fig" rid="fig1">Figure 1B</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Group</th><th valign="top"><italic>n</italic></th><th valign="top">100 pA (%)</th><th valign="top"><italic>p</italic>-value</th><th valign="top"><italic>n</italic></th><th valign="top">150 pA (%)</th><th valign="top"><italic>p</italic>-value</th><th valign="top"><italic>n</italic></th><th valign="top">200 pA (%)</th><th valign="top">p-Value</th></tr></thead><tbody><tr><td valign="top">YN</td><td valign="top">25</td><td valign="top">97.3&#160;&#177;&#160;2.7</td><td valign="top"/><td valign="top">18</td><td valign="top">96.3&#160;&#177;&#160;2.5</td><td valign="top"/><td valign="top">16</td><td valign="top">85.4&#160;&#177;&#160;6.1</td><td valign="top"/></tr><tr><td valign="top">AN</td><td valign="top">18</td><td valign="top">72.2&#160;&#177;&#160;8.6</td><td valign="top">vs. YN, <break/>0.0016**</td><td valign="top">10</td><td valign="top">56.7&#160;&#177;&#160;14.1</td><td valign="top">vs. YN, <break/>0.0029**</td><td valign="top">10</td><td valign="top">66.7&#160;&#177;&#160;13.2</td><td valign="top">vs. YN, <break/>0.2458</td></tr><tr><th rowspan="2" valign="top">Group</th><th colspan="3" valign="top">100 pA</th><th colspan="3" valign="top">150 pA</th><th colspan="3" valign="top">200 pA</th></tr><tr><th valign="top">Q1 (25%)</th><th valign="top">Median</th><th valign="top">Q3 (75%)</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th></tr><tr><td valign="top">YN</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">1.00</td></tr><tr><td valign="top">AN</td><td valign="top">0.33</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.00</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">0.25</td><td valign="top">0.83</td><td valign="top">1.00</td></tr></tbody></table><table-wrap-foot><fn><p><italic>n</italic>, number of cells in group; <italic>p</italic>-value, Mann-Whitney.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56816-v2.xml">elife-56816-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Persistent firing probability evoked with a 20 Hz current pulses at various stimulus lengths and membrane holding potentials with statistical differences between YN and AN.</title><p>Related to <xref ref-type="fig" rid="fig1">Figure 1D, F, H</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Group</th><th valign="top"><italic>n</italic></th><th valign="top">20 Hz, 2 s, <break/>2 mV below (%)</th><th valign="top">p-Value</th><th valign="top"><italic>n</italic></th><th valign="top">20 Hz, 250 ms, 2 mV below (%)</th><th valign="top">p-Value</th><th valign="top"><italic>n</italic></th><th valign="top">20 Hz, 250 ms, 5 mV below (%)</th><th valign="top">p-Value</th></tr></thead><tbody><tr><td valign="top">YN</td><td valign="top">14</td><td valign="top">97.6&#160;&#177;&#160;2.4</td><td valign="top"/><td valign="top">22</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top"/><td valign="top">21</td><td valign="top">39.7&#160;&#177;&#160;10.2</td><td valign="top"/></tr><tr><td valign="top">AN</td><td valign="top">10</td><td valign="top">66.7&#160;&#177;&#160;11.1</td><td valign="top">vs. YN, <break/>0.0052**</td><td valign="top">12</td><td valign="top">66.7&#160;&#177;&#160;11.6</td><td valign="top">vs. YN, <break/>0.0007**</td><td valign="top">14</td><td valign="top">16.7&#160;&#177;&#160;9.7</td><td valign="top">vs. YN, <break/>0.1304</td></tr><tr><th rowspan="2" valign="top">Group</th><th colspan="3" valign="top">20 Hz, 2s, 2 mV below</th><th colspan="3" valign="top">20 Hz, 250 ms, 2 mV below</th><th colspan="3" valign="top">20 Hz, 250 ms, 5 mV below</th></tr><tr><th valign="top">Q1 (25%)</th><th valign="top">Median</th><th valign="top">Q3 (75%)</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th></tr><tr><td valign="top">YN</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.00</td><td valign="top">0.00</td><td valign="top">1.00</td></tr><tr><td valign="top">AN</td><td valign="top">0.33</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">0.33</td><td valign="top">0.83</td><td valign="top">1.00</td><td valign="top">0.00</td><td valign="top">0.00</td><td valign="top">0.08</td></tr></tbody></table><table-wrap-foot><fn><p><italic>n</italic>, number of cells in group; <italic>p</italic>-value, Mann-Whitney.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56816-v2.xml">elife-56816-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table17" position="float"><label>Table 17.</label><caption><title>Passive membrane properties of LEC III pyramidal neurons, separated into YN, AN, YC, YP, AU, and AI.</title></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Group</th><th valign="top">RMP (mV)</th><th valign="top">R<sub>input</sub> (M&#937;)</th><th colspan="3"/></tr></thead><tbody><tr><td valign="top">YN</td><td valign="top">&#8722;72.28&#160;&#177;&#160;1.12</td><td valign="top">54.49&#160;&#177;&#160;1.39</td><td colspan="3"/></tr><tr><td valign="top">AN</td><td valign="top">&#8722;70.73&#160;&#177;&#160;0.89</td><td valign="top">60.39&#160;&#177;&#160;2.91</td><td colspan="3"/></tr><tr><th valign="top">Group</th><th valign="top">First AP Threshold (mV)</th><th valign="top">Last AP Threshold (mV)</th><th colspan="3" valign="top">p-Value</th></tr><tr><td rowspan="3" valign="top">YN</td><td rowspan="3" valign="top">-41.54 &#177; 0.33</td><td rowspan="3" valign="top">-41.63 &#177; 0.33</td><td colspan="3" valign="top">AP Threshold, <break/>F<sub>1, 379</sub> = 0.6896, p = 0.4068</td></tr><tr><td colspan="3" valign="top">Group, <break/>F<sub>1, 379</sub> = 8.744, p = 0.003**</td></tr><tr><td colspan="3" valign="top">Group x AP Threshold, <break/>F<sub>1, 379</sub> = 2.902, p = 0.0893~</td></tr><tr><td rowspan="2" valign="top">AN</td><td rowspan="2" valign="top">-43.40 &#177; 0.48</td><td rowspan="2" valign="top">-43.15 &#177; 0.53</td><td colspan="3" valign="top">YN vs. AN <break/>First AP Threshold, p = 0.0028**</td></tr><tr><td colspan="3" valign="top">Last AP Threshold, p = 0.0178*</td></tr><tr><th valign="top">Group</th><th valign="top">First AP Half-Width (ms)</th><th valign="top">Last AP Half-Width (ms)</th><th colspan="3" valign="top">p-Value</th></tr><tr><td rowspan="3" valign="top">YN</td><td rowspan="3" valign="top">1.97 &#177; 0.04</td><td rowspan="3" valign="top">2.00 &#177; 0.03</td><td colspan="3" valign="top">AP Half-Width <break/>F<sub>1, 379</sub> = 8.761, p = 0.0033**</td></tr><tr><td colspan="3" valign="top">Group, <break/>F<sub>1, 379</sub> = 2.667, p = 0.1033</td></tr><tr><td colspan="3" valign="top">Group x AP Half-Width <break/>F<sub>1, 379</sub> = 3.120, p = 0.0781~</td></tr><tr><td rowspan="2" valign="top">AN</td><td rowspan="2" valign="top">1.83 &#177; 0.04</td><td rowspan="2" valign="top">1.95 &#177; 0.04</td><td colspan="3" valign="top">YN vs AN <break/>First AP Half-Width, p = 0.0523~</td></tr><tr><td colspan="3" valign="top">Last AP Half-Width, p = 0.7060</td></tr><tr><th valign="top">Group</th><th valign="top">First AP Amplitude (mV)</th><th valign="top">Last AP Amplitude (mV)</th><th colspan="3" valign="top">p-Value</th></tr><tr><td rowspan="3" valign="top">YN</td><td rowspan="3" valign="top">83.61 &#177; 0.44</td><td rowspan="3" valign="top">80.45 &#177; 0.43</td><td colspan="3" valign="top">Two-Way ANOVA, AP Amplitude <break/>F<sub>1, 379</sub> = 521.7, p &lt; 0.0001***</td></tr><tr><td colspan="3" valign="top">Group, <break/>F<sub>1, 379</sub> = 0.001639, p = 0.9677</td></tr><tr><td colspan="3" valign="top">Group x AP Amplitude <break/>F<sub>1, 379</sub> = 0.3927, p = 0.5313</td></tr><tr><td rowspan="2" valign="top">AN</td><td rowspan="2" valign="top">83.49 &#177; 0.64</td><td rowspan="2" valign="top">80.50 &#177; 0.73</td><td colspan="3" valign="top">YN vs AN <break/>First AP Amplitude, p = 0.9862</td></tr><tr><td colspan="3" valign="top">Last AP Amplitude, p = 0.9970</td></tr><tr><th valign="top">Group</th><th valign="top">First AP dV/dt max (v/s)</th><th valign="top">Last AP dV/dt max (v/s)</th><th colspan="3" valign="top">p-Value</th></tr><tr><td rowspan="3" valign="top">YN</td><td rowspan="3" valign="top">199.3 &#177; 3.0</td><td rowspan="3" valign="top">175.4 &#177; 2.9</td><td colspan="3" valign="top">dV/dt Max <break/>F<sub>1, 379</sub> = 385.7, p &lt; 0.0001***</td></tr><tr><td colspan="3" valign="top">Group, <break/>F<sub>1, 379</sub> = 0.4711, p = 0.4929</td></tr><tr><td colspan="3" valign="top">Group x dV/dt Max <break/>F<sub>1, 379</sub> = 0.1451, p = 0.7034</td></tr><tr><td rowspan="2" valign="top">AN</td><td rowspan="2" valign="top">202.3 &#177; 4.2</td><td rowspan="2" valign="top">179.3 &#177; 4.6</td><td colspan="3" valign="top">YN vs AN <break/>First dV/dt Max, p = 0.8083</td></tr><tr><td colspan="3" valign="middle">Last dV/dt Max, p =&#160;0.6975<break/></td></tr><tr><th valign="top">Group</th><th valign="top">RMP (mV)</th><th valign="top">R<sub>input</sub> (M&#937;)</th><th colspan="3" rowspan="5" valign="top"> <break/> <break/> <break/><break/></th></tr><tr><td valign="top">YC</td><td valign="top">-75.35 &#177; 0.71</td><td valign="top">55.74 &#177; 2.53</td></tr><tr><td valign="top">YP</td><td valign="top">-74.37 &#177; 0.86</td><td valign="top">57.66 &#177; 3.70</td></tr><tr><td valign="top">AU</td><td valign="top">-72.71 &#177; 0.76</td><td valign="top">59.87 &#177; 2.27</td></tr><tr><td valign="top">AI</td><td valign="top">-70.24 &#177; 1.40</td><td valign="top">59.06 &#177; 2.95</td></tr><tr><th valign="top">Group</th><th valign="top">First AP Threshold (mV)</th><th valign="top">Last AP Threshold (mV)</th><th colspan="3" valign="top">p-Value</th></tr><tr><td valign="top">YC</td><td valign="top">-37.33 &#177; 0.40</td><td valign="top">-37.39 &#177; 0.47</td><td colspan="3" valign="top">AP Threshold, <break/>F<sub>1, 643</sub> = 15.71, p &lt; 0.0001***</td></tr><tr><td rowspan="2" valign="top">YP</td><td rowspan="2" valign="top">-40.70 &#177; 0.36</td><td rowspan="2" valign="top">-41.46 &#177; 0.35</td><td colspan="3" valign="top">Group, <break/>F<sub>3, 643</sub> = 20.74, p &lt; 0.0001***</td></tr><tr><td colspan="3" valign="top">Group x AP Threshold, <break/>F<sub>3, 643</sub> = 1.826, p = 0.1412</td></tr><tr><td rowspan="4" valign="top">AU</td><td rowspan="4" valign="top">-40.80 &#177; 0.35</td><td rowspan="4" valign="top">-41.10 &#177; 0.34</td><td valign="top">Comparison</td><td valign="top">First AP</td><td valign="top">Last AP</td></tr><tr><td valign="top">YC vs YP</td><td valign="top">p&#160;&lt;&#160;0.0001***</td><td valign="top">p&#160;&lt;&#160;0.0001***</td></tr><tr><td valign="top">YC vs AU</td><td valign="top">p&#160;&lt;&#160;0.0001***</td><td valign="top">p&#160;&lt;&#160;0.0001***</td></tr><tr><td valign="top">YC vs AI</td><td valign="top">p&#160;&lt;&#160;0.0001***</td><td valign="top">p&#160;&lt;&#160;0.0001***</td></tr><tr><td rowspan="3" valign="top">AI</td><td rowspan="3" valign="top">-40.69 &#177; 0.39</td><td rowspan="3" valign="top">-41.17 &#177; 0.32</td><td valign="top">YP vs AU</td><td valign="top">p&#160;=&#160;0.9974</td><td valign="top">p&#160;=&#160;0.9062</td></tr><tr><td valign="top">YP vs AI</td><td valign="top">p &gt; 0.9999</td><td valign="top">p = 0.9488</td></tr><tr><td valign="top">AU vs AI</td><td valign="top">p = 0.9947</td><td valign="top">p = 0.9990</td></tr><tr><th valign="top">Group</th><th valign="top">First AP Half-Width (ms)</th><th valign="top">Last AP Half-Width (ms)</th><th colspan="3" valign="top">p-Value</th></tr><tr><td valign="top">YC</td><td valign="top">2.23 &#177; 0.06</td><td valign="top">2.32 &#177; 0.09</td><td colspan="3" valign="top">AP Half-Width, <break/>F<sub>1, 643</sub> = 4.928e-005, p = 0.9944</td></tr><tr><td rowspan="2" valign="top">YP</td><td rowspan="2" valign="top">1.96 &#177; 0.05</td><td rowspan="2" valign="top">1.94 &#177; 0.04</td><td colspan="3" valign="top">Group, <break/>F<sub>3, 643</sub> = 28.46, p &lt; 0.0001***</td></tr><tr><td colspan="3" valign="top">Group x Half-Width <break/>F<sub>3, 643</sub> = 1.650, p = 0.1767</td></tr><tr><td rowspan="5" valign="top">AU</td><td rowspan="5" valign="top">1.82 &#177; 0.04</td><td rowspan="5" valign="top">1.80 &#177; 0.04</td><td valign="top">Comparison</td><td valign="top">First AP</td><td valign="top">Last AP</td></tr><tr><td valign="top">YC vs YP</td><td valign="top">p = 0.0007**</td><td valign="top">p &lt; 0.0001***</td></tr><tr><td valign="top">YC vs AU</td><td valign="top">p &lt; 0.0001***</td><td valign="top">p &lt; 0.0001***</td></tr><tr><td valign="top">YC vs AI</td><td valign="top">p &lt; 0.0001***</td><td valign="top">p &lt; 0.0001***</td></tr><tr><td valign="top">YP vs AU</td><td valign="top">p = 0.1239</td><td valign="top">p = 0.1198</td></tr><tr><td rowspan="2" valign="top">AI</td><td rowspan="2" valign="top">1.82 &#177; 0.04</td><td rowspan="2" valign="top">1.77 &#177; 0.03</td><td valign="top">YP vs AI</td><td valign="top">p = 0.1361</td><td valign="top">p = 0.0494*</td></tr><tr><td valign="top">AU vs AI</td><td valign="top">p &gt; 0.9999</td><td valign="top">p = 0.9682</td></tr><tr><th valign="top">Group</th><th valign="top">First AP Amplitude (mV)</th><th valign="top">Last AP Amplitude (mV)</th><th colspan="3" valign="top">p-Value</th></tr><tr><td valign="top">YC</td><td valign="top">80.02 &#177; 1.11</td><td valign="top">76.10 &#177; 1.07</td><td colspan="3" valign="top">AP Amplitude, <break/>F<sub>1, 643</sub> = 286.5, p &lt; 0.0001</td></tr><tr><td rowspan="2" valign="top">YP</td><td rowspan="2" valign="top">81.80 &#177; 0.88</td><td rowspan="2" valign="top">78.92 &#177; 0.82</td><td colspan="3" valign="top">Group, <break/>F<sub>3, 643</sub> = 3.996, p = 0.0078</td></tr><tr><td colspan="3" valign="top">Group x AP Amplitude, <break/>F<sub>3, 643</sub> = 5.083, p = 0.0017</td></tr><tr><td rowspan="4" valign="top">AU</td><td rowspan="4" valign="top">83.10 &#177; 0.66</td><td rowspan="4" valign="top">80.27 &#177; 0.52</td><td valign="top">Comparisons</td><td valign="top">First AP</td><td valign="top">Last AP</td></tr><tr><td valign="top">YC vs YP</td><td valign="top">p = 0.5087</td><td valign="top">p = 0.1274</td></tr><tr><td valign="top">YC vs AU</td><td valign="top">p = 0.0404*</td><td valign="top">p = 0.0019**</td></tr><tr><td valign="top">YC vs AI</td><td valign="top">p = 0.9990</td><td valign="top">p = 0.2916</td></tr><tr><td rowspan="3" valign="top">AI</td><td rowspan="3" valign="top">80.18 &#177; 0.83</td><td rowspan="3" valign="top">78.19 &#177; 0.75</td><td valign="top">YP vs AU</td><td valign="top">p = 0.6715</td><td valign="top">p = 0.6391</td></tr><tr><td valign="top">YP vs AI</td><td valign="top">p = 0.5082</td><td valign="top">p = 0.9233</td></tr><tr><td valign="top">AU vs AI</td><td valign="top">p = 0.0237*</td><td valign="top">p = 0.1762</td></tr><tr><th valign="top">Group</th><th valign="top">First AP dV/dt max (v/s)</th><th valign="top">Last AP dV/dt max (v/s)</th><th colspan="3" valign="top">p-Value</th></tr><tr><td valign="top">YC</td><td valign="top">178.0 &#177; 5.1</td><td valign="top">151.1 &#177; 4.6</td><td colspan="3" valign="top">AP dV/dt max, <break/>F<sub>1, 643</sub> = 15.71, p &lt; 0.0001</td></tr><tr><td rowspan="2" valign="top">YP</td><td rowspan="2" valign="top">183.5 &#177; 5.3</td><td rowspan="2" valign="top">163.8 &#177; 4.6</td><td colspan="3" valign="top">Group, <break/>F<sub>3, 643</sub> = 20.74, p &lt; 0.0001</td></tr><tr><td colspan="3" valign="top">Group x AP dV/dt max, <break/>F<sub>3, 643</sub> = 1.826, p = 0.1412</td></tr><tr><td rowspan="2" valign="top">AU</td><td rowspan="2" valign="top">191.0 &#177; 3.7</td><td rowspan="2" valign="top">172.4 &#177; 2.9</td><td valign="top">Comparison</td><td valign="top">First AP</td><td valign="top">Last AP</td></tr><tr><td valign="top">YC vs YP</td><td valign="top">p = 0.8306</td><td valign="top">p = 0.2052</td></tr><tr><td rowspan="5" valign="top">AI</td><td rowspan="5" valign="top">176.7 &#177; 4.2</td><td rowspan="5" valign="top">162.0 &#177; 3.2</td><td valign="top">YC vs AU</td><td valign="top">p = 0.1219</td><td valign="top">p = 0.0017**</td></tr><tr><td valign="top">YC vs AI</td><td valign="top">p = 0.9966</td><td valign="top">p = 0.2653</td></tr><tr><td valign="top">YP vs AU</td><td valign="top">p = 0.5757</td><td valign="top">p = 0.4517</td></tr><tr><td valign="top">YP vs AI</td><td valign="top">p = 0.6563</td><td valign="top">p = 0.9888</td></tr><tr><td valign="top">AU vs AI</td><td valign="top">p = 0.0313*</td><td valign="top">p = 0.1836</td></tr></tbody></table><table-wrap-foot><fn><p>All statistical analyses were first performed with a Two-Way ANOVA. <italic>Post hoc</italic>&#160;multiple comparisons tests performed on behaviorally na&#239;ve Young and Aged data were done with Sidak&#8217;s tests. Multiple comparisons performed on YC, YP, AU, and AI data were done with Tukey&#8217;s tests.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56816-v2.xml">elife-56816-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table4" position="float"><label>Table 4.</label><caption><title>Persistent firing probability evoked with a 20 Hz current pulses at various stimulus lengths and membrane holding potentials with statistical differences between YC, YP, AU, and AI.</title><p>Related to <xref ref-type="fig" rid="fig2">Figure 2C, D, E</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Group</th><th valign="top"><italic>n</italic></th><th valign="top">20 Hz, 2 s, <break/>2 mV below (%)</th><th valign="top">p-Value</th><th valign="top"><italic>n</italic></th><th valign="top">20 Hz, 250 ms, 2 mV below (%)</th><th valign="top">p-Value</th><th valign="top"><italic>n</italic></th><th valign="top">20 Hz, 250 ms, 5 mV below (%)</th><th valign="top">p-Value</th></tr></thead><tbody><tr><td valign="top">YC</td><td valign="top">8</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YP, <break/>&gt;0.9999</td><td valign="top">12</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YP, <break/>&gt;0.9999</td><td valign="top">11</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YP, <break/>0.0063**</td></tr><tr><td valign="top">YP</td><td valign="top">8</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top"/><td valign="top">11</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top"/><td valign="top">13</td><td valign="top">41.0&#160;&#177;&#160;10.8</td><td valign="top"/></tr><tr><td rowspan="2" valign="top">AU</td><td rowspan="2" valign="top">7</td><td rowspan="2" valign="top">95.2&#160;&#177;&#160;4.8</td><td valign="top">vs. YC, <break/>&gt;0.9999</td><td rowspan="2" valign="top">19</td><td rowspan="2" valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YC, <break/>&gt;0.9999</td><td rowspan="2" valign="top">15</td><td rowspan="2" valign="top">73.34&#160;&#177;&#160;9.3</td><td valign="top">vs. YC, <break/>0.5802</td></tr><tr><td valign="top">vs. YP, <break/>&gt;0.9999</td><td valign="top">vs. YP, <break/>&gt;0.9999</td><td valign="top">vs. YP, <break/>0.4308</td></tr><tr><td rowspan="3" valign="top">AI</td><td rowspan="3" valign="top">11</td><td rowspan="3" valign="top">72.7&#160;&#177;&#160;8.8</td><td valign="top">vs. AU, <break/>0.2046</td><td rowspan="3" valign="top">18</td><td rowspan="3" valign="top">85.19&#160;&#177;&#160;6.2</td><td valign="top">vs. AU, <break/>0.0049**</td><td rowspan="3" valign="top">16</td><td rowspan="3" valign="top">12.5&#160;&#177;&#160;6.0</td><td valign="top">vs. AU, <break/>0.0008**</td></tr><tr><td valign="top">vs. YC, <break/>0.0236*</td><td valign="top">vs. YC, <break/>0.0188*</td><td valign="top">vs. YC, <break/>&lt;0.0001***</td></tr><tr><td valign="top">vs. YP, <break/>0.0236*</td><td valign="top">vs. YP, <break/>0.0241*</td><td valign="top">vs. YP, <break/>0.3874</td></tr><tr><th rowspan="2" valign="top">Group</th><th colspan="3" valign="top">20 Hz, 2s, 2 mV below</th><th colspan="3" valign="top">20 Hz, 250 ms, 2 mV below</th><th colspan="3" valign="top">20 Hz, 250 ms, 5 mV below</th></tr><tr><th valign="top">Q1 (25%)</th><th valign="top">Median</th><th valign="top">Q3 (75%)</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th></tr><tr><td valign="top">YC</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td></tr><tr><td valign="top">YP</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.00</td><td valign="top">0.33</td><td valign="top">0.83</td></tr><tr><td valign="top">AU</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">1.00</td></tr><tr><td valign="top">AI</td><td valign="top">0.33</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.00</td><td valign="top">0.00</td><td valign="top">0.25</td></tr></tbody></table><table-wrap-foot><fn><p><italic>n</italic>, number of cells in group; <italic>p</italic>-value, Dunn&#8217;s multiple comparisons test.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56816-v2.xml">elife-56816-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table5" position="float"><label>Table 5.</label><caption><title>Persistent firing probability evoked with a 100 pA, 150 pA, and 200 pA training stimulus with statistical differences between YC, YP, AU, and AI.</title><p>Related to&#160;<xref ref-type="fig" rid="fig2s1">Figure 2&#8212;figure supplement 1</xref>.</p></caption><table frame="hsides" rules="groups"><thead><tr><th valign="top">Group</th><th valign="top"><italic>n</italic></th><th valign="top">100 pA (%)</th><th valign="top">p<italic>-</italic>Value</th><th valign="top"><italic>n</italic></th><th valign="top">150 pA (%)</th><th valign="top">p-Value</th><th valign="top"><italic>n</italic></th><th valign="top">200 pA (%)</th><th valign="top">p-Value</th></tr></thead><tbody><tr><td valign="top">YC</td><td valign="top">14</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YP,&#160;&gt;0.9999</td><td valign="top">10</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YP,&#160;&gt;0.9999</td><td valign="top">8</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YP,&#160;&gt;0.9999</td></tr><tr><td valign="top">YP</td><td valign="top">8</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top"/><td valign="top">4</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top"/><td valign="top">4</td><td valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top"/></tr><tr><td rowspan="2" valign="top">AU</td><td rowspan="2" valign="top">17</td><td rowspan="2" valign="top">100.0&#160;&#177;&#160;0.0</td><td valign="top">vs. YC,&#160;&gt;0.9999</td><td rowspan="2" valign="top">11</td><td rowspan="2" valign="top">81.8&#160;&#177;&#160;10.4</td><td valign="top">vs. YC,&#160;&gt;0.9999</td><td rowspan="2" valign="top">7</td><td rowspan="2" valign="top">95.2&#160;&#177;&#160;4.8</td><td valign="top">vs. YC,&#160;&gt;0.9999</td></tr><tr><td valign="top">vs. YP,&#160;&gt;0.9999</td><td valign="top">vs. YP,&#160;&gt;0.9999</td><td valign="top">vs. YP,&#160;&gt;0.9999</td></tr><tr><td rowspan="3" valign="top">AI</td><td rowspan="3" valign="top">16</td><td rowspan="3" valign="top">68.8&#160;&#177;&#160;10.3</td><td valign="top">vs.&#160;AU, 0.0012**</td><td rowspan="3" valign="top">14</td><td rowspan="3" valign="top">57.1&#160;&#177;&#160;8.9</td><td valign="top">vs. AU,&#160;0.1846</td><td rowspan="3" valign="top">13</td><td rowspan="3" valign="top">61.5&#160;&#177;&#160;11.8</td><td valign="top">vs.&#160;AU, 0.2366</td></tr><tr><td valign="top">vs. YC,&#160;0.0024**</td><td valign="top">vs. YC,&#160;0.0028**</td><td valign="top">vs.&#160;YC, 0.0356*</td></tr><tr><td valign="top">vs. YP,&#160;0.0164*</td><td valign="top">vs. YP,&#160;0.0639~</td><td valign="top">vs.&#160;YP, 0.1836</td></tr><tr><th rowspan="2" valign="top">Group</th><th colspan="3" valign="top">100 pA</th><th colspan="3" valign="top">150 pA</th><th colspan="3" valign="top">200 pA</th></tr><tr><th valign="top">Q1 (25%)</th><th valign="top">Median</th><th valign="top">Q3 (75%)</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th><th valign="top">Q1</th><th valign="top">Median</th><th valign="top">Q3</th></tr><tr><td valign="top">YC</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td></tr><tr><td valign="top">YP</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td></tr><tr><td valign="top">AU</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.67</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">1.00</td></tr><tr><td valign="top">AI</td><td valign="top">0.33</td><td valign="top">1.00</td><td valign="top">1.00</td><td valign="top">0.33</td><td valign="top">0.50</td><td valign="top">1.00</td><td valign="top">0.17</td><td valign="top">0.67</td><td valign="top">1.00</td></tr></tbody></table><table-wrap-foot><fn><p><italic>n,</italic> number of cells in group; <italic>p-</italic>value, Dunn&#8217;s multiple comparisons test.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56860-v1.xml">elife-56860-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Characteristics of the study cohorts.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="2" valign="top">Matched twin cohort: twins exposed vs. unexposed to loss of a co-twin</th><th colspan="2" valign="top">Twin-sibling cohort: twins exposed to loss of a co-twin vs. their full siblings</th></tr><tr><th valign="top">Exposed twins</th><th valign="top">Matched unexposed twins</th><th valign="top">Exposed Twins&#8217;</th><th valign="top">Exposed full siblings</th></tr></thead><tbody><tr><td valign="top">Number of individuals</td><td valign="top">4528</td><td valign="top">22640</td><td valign="top">2761</td><td valign="top">4939</td></tr><tr><td valign="top">Age at index date, median (IQR), year</td><td valign="top">59(47-67)</td><td valign="top">59(47-67)</td><td valign="top">57(45-65)</td><td valign="top">57(46-65)</td></tr><tr><td valign="top">Follow-up time, median (IQR), year</td><td valign="top">7.4 (3.1&#8211;13.7)</td><td valign="top">7.7 (3.2&#8211;14.2)</td><td valign="top">8.0 (3.3&#8211;14.5)</td><td valign="top">8.9 (3.8&#8211;15.4)</td></tr><tr><td valign="top">% of male</td><td valign="top">50.8%</td><td valign="top">50.8%</td><td valign="top">51.7%</td><td valign="top">49.7%</td></tr><tr><td valign="top">Education level, n (%)</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">&lt;9 years</td><td valign="top">1171 (25.9)</td><td valign="top">5041 (22.3)</td><td valign="top">700 (25.4)</td><td valign="top">1187 (24.0)</td></tr><tr><td valign="top">9&#8211;12 years</td><td valign="top">2439 (53.9)</td><td valign="top">12151 (53.7)</td><td valign="top">1520 (55.1)</td><td valign="top">2775 (56.2)</td></tr><tr><td valign="top">&gt;12 years</td><td valign="top">844 (18.6)</td><td valign="top">5185 (22.9)</td><td valign="top">505 (18.3)</td><td valign="top">939 (19.0)</td></tr><tr><td valign="top">Unknown</td><td valign="top">74 (1.63)</td><td valign="top">263 (1.16)</td><td valign="top">36 (1.30)</td><td valign="top">38 (0.77)</td></tr><tr><td valign="top">Yearly family income level, n (%)</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">Lowest 20%</td><td valign="top">504 (11.1)</td><td valign="top">2275 (10.1)</td><td valign="top">313 (11.3)</td><td valign="top">561 (11.4)</td></tr><tr><td valign="top">Middle</td><td valign="top">2487 (54.9)</td><td valign="top">12092 (53.4)</td><td valign="top">1499 (54.3)</td><td valign="top">2653 (53.7)</td></tr><tr><td valign="top">Top 20%</td><td valign="top">1050 (23.2)</td><td valign="top">5850 (25.8)</td><td valign="top">626 (22.7)</td><td valign="top">1142 (23.1)</td></tr><tr><td valign="top">Unknown</td><td valign="top">487 (10.8)</td><td valign="top">2423 (10.7)</td><td valign="top">323 (11.7)</td><td valign="top">583 (11.8)</td></tr><tr><td valign="top">Marital status, n (%)</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">Single</td><td valign="top">1270 (28.1)</td><td valign="top">6073 (26.8)</td><td valign="top">808 (29.3)</td><td valign="top">1213 (24.6)</td></tr><tr><td valign="top">Married/cohabiting</td><td valign="top">2624 (58.0)</td><td valign="top">13739 (60.7)</td><td valign="top">1564 (56.7)</td><td valign="top">3056 (61.9)</td></tr><tr><td valign="top">Divorced/widowed</td><td valign="top">634 (14.0)</td><td valign="top">2828 (12.5)</td><td valign="top">389 (14.1)</td><td valign="top">670 (13.6)</td></tr><tr><td>History of severe somatic diseases<sup>*</sup>, n (%)</td><td colspan="2"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">Yes</td><td valign="top">893 (19.7)</td><td valign="top">2664 (11.8)</td><td valign="top">514 (18.6)</td><td valign="top">924 (18.7)</td></tr><tr><td valign="top">No</td><td valign="top">3635 (80.3)</td><td valign="top">19976 (88.2)</td><td valign="top">2247 (81.4)</td><td valign="top">4015 (81.3)</td></tr><tr><td valign="top">Family history of psychiatric disorders/suicide, n (%)</td><td colspan="2" valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">Yes</td><td valign="top">2257 (49.9)</td><td valign="top">8440 (37.3)</td><td valign="top">1378 (49.9)</td><td valign="top">2505 (50.7)</td></tr><tr><td valign="top">No</td><td valign="top">2271 (50.2)</td><td valign="top">14200 (62.7)</td><td valign="top">1383 (50.1)</td><td valign="top">2434 (49.3)</td></tr><tr><td valign="top">Cause of the co-twin&#8217;s death, n (%)</td><td colspan="2" valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top"><italic>Unnatural death</italic></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">Yes</td><td valign="top">1020 (22.5)</td><td valign="top">-</td><td valign="top">666 (24.1)</td><td valign="top">1213 (24.6)</td></tr><tr><td valign="top">No</td><td valign="top">3508 (77.5)</td><td valign="top">-</td><td valign="top">2095 (75.9)</td><td valign="top">3726 (75.4)</td></tr><tr><td valign="top">Zygosity of twins, n (%)</td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">Monozygotic twins</td><td valign="top">746 (16.5)</td><td valign="top">2367 (10.5)</td><td valign="top">423 (15.4)</td><td valign="top">-</td></tr><tr><td valign="top">Dizygotic twins</td><td valign="top">3016 (66.6)</td><td valign="top">15722 (69.4)</td><td valign="top">1851 (67.0)</td><td valign="top">-</td></tr><tr><td valign="top">Unknown</td><td valign="top">766 (16.9)</td><td valign="top">4551 (20.1)</td><td valign="top">487 (17.6)</td><td valign="top">-</td></tr></tbody></table><table-wrap-foot><fn><p>* Involved somatic diseases included myocardial infarction, congestive heart failure, cerebrovascular disease, chronic pulmonary disease, connective tissue disease, diabetes, renal diseases, liver diseases, ulcer diseases, and HIV infection/AIDS.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56860-v1.xml">elife-56860-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table2" position="float"><label>Table 2.</label><caption><title>Hazard ratios (HRs) with 95% confidence intervals (CIs) for any psychiatric disorder among the surviving twins after co-twin loss (overall and by follow-up time), compared to matched unexposed twins or their full siblings, derived from different Cox models.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Model information</th><th colspan="2" valign="top">Matched twin cohort: twins exposed vs. unexposed to loss of a co-twin</th><th colspan="2" valign="top">Twin-sibling cohort: twins exposed to loss of a co-twin vs. their full siblings</th></tr><tr><th>Number of cases (Crude incidence rate, per 1000 person years), exposed/unexposed twins</th><th>Hr (95% CI)<sup>*</sup></th><th>Number of cases (Crude incidence rate, per 1000 person years), exposed twins/exposed siblings</th><th>Hr (95% CI)<sup>*</sup></th></tr></thead><tbody><tr><td>Model 1 <break/>Controlled/adjusted for sex, birth year</td><td rowspan="4">526 (12.29)/1521 (6.90)</td><td>1.80 (1.63&#8211;2.00)</td><td rowspan="4">328 (11.92)/415 (7.90)</td><td>1.54 (1.31&#8211;1.81)</td></tr><tr><td>Model 2 <break/>above + socioeconomic status (education level, marital status, family income)</td><td>1.79 (1.62&#8211;1.99)</td><td>1.55 (1.32&#8211;1.83)</td></tr><tr><td>Model 3 <break/>above + history of severe somatic disease</td><td>1.71 (1.55&#8211;1.90)</td><td>1.55 (1.31&#8211;1.82)</td></tr><tr><td>Model 4 <break/>above + family history of psychiatric disorder</td><td>1.65 (1.48&#8211;1.83)</td><td>-</td></tr><tr><td valign="top"><italic>By follow-up period</italic><sup>&#8224;</sup></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">Within 1 month</td><td valign="top">15 (43.47)/10 (5.79)</td><td valign="top">7.16 (3.07&#8211;16.70)</td><td valign="top">10 (47.55)/3 (7.96)</td><td valign="top">7.21 (1.12&#8211;46.5)</td></tr><tr><td valign="top">2&#8211;11 months</td><td valign="top">45 (11.38)/116 (5.85)</td><td valign="top">1.69 (1.18&#8211;2.42)</td><td>29 (12.05)/30 (6.92)</td><td>1.46 (0.83&#8211;2.57)</td></tr><tr><td valign="top">2&#8211;9 years</td><td valign="top">307 (12.24)/864 (6.75)</td><td valign="top">1.61 (1.40&#8211;1.85)</td><td valign="top">180 (11.39)/217 (7.30)</td><td valign="top">1.58 (1.26&#8211;1.97)</td></tr><tr><td valign="top">10 years and onwards</td><td valign="top">159 (11.88)/531 (7.51)</td><td valign="top">1.59 (1.31&#8211;1.92)</td><td valign="top">109 (12.00)/165 (9.15)</td><td valign="top">1.36 (1.03&#8211;1.81)</td></tr></tbody></table><table-wrap-foot><fn><p>* Cox regression models were stratified by matching identifiers (birth year and sex, in matched twin cohort) or family identifiers (in twin-sibling cohort), and adjusted for covariates mentioned in the &#8216;model information&#8217; column. Time since the index date was used as underlying time scale.</p><p>&#8224;&#160;HRs were derived from fully adjusted Cox regression models, that&#160;is model four for matched twin cohort and model three for twin-sibling cohort.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-56860-v1.xml">elife-56860-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table3" position="float"><label>Table 3.</label><caption><title>Hazard ratios (HRs) with 95% confidence intervals (CIs) for any psychiatric disorder among the surviving twins after co-twin loss, compared to matched unexposed twins (matched twin cohort) or their full siblings (twin-sibling cohort), by zygosity of the twin pairs</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th colspan="2" valign="top">Twins who lost a monozygotic twin vs. matched unexposed monozygotic twins or their full siblings</th><th colspan="2" valign="top">Twins who lost a dizygotic twin vs. matched unexposed dizygotic twins or their full siblings</th></tr><tr><th>Number of cases (Crude incidence rate, per 1000 person years), exposed/unexposed</th><th>Hr (95% CI)<sup>*</sup></th><th>Number of cases (Crude incidence rate, per 1000 person years), exposed/unexposed</th><th>Hr (95% CI)<sup>*</sup></th></tr></thead><tbody><tr><td valign="top">Matched twin cohort</td><td valign="top">92 (14.81)/153 (7.26)</td><td valign="top">1.86 (1.40&#8211;2.47)</td><td valign="top">303 (10.89)/1024 (7.08)</td><td valign="top">1.33 (1.15&#8211;1.54)</td></tr><tr><td valign="top"><italic>By follow-up period</italic></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td valign="top">Within 1 month</td><td>6 (105.9)/2 (11.12)</td><td valign="top">9.47 (1.88&#8211;47.8)</td><td valign="top">8 (34.81)/5 (4.17)</td><td valign="top">4.20 (1.15&#8211;15.3)</td></tr><tr><td valign="top">2&#8211;11 months</td><td valign="top">12 (18.7)/14 (6.84)</td><td valign="top">2.54 (1.01&#8211;6.37)</td><td valign="top">22 (8.37)/72 (5.23)</td><td valign="top">1.23 (0.71&#8211;2.13)</td></tr><tr><td valign="top">2&#8211;9 years</td><td valign="top">46 (12.40)/93 (7.42)</td><td valign="top">1.40 (0.95&#8211;2.07)</td><td valign="top">190 (11.41)/603 (6.98)</td><td valign="top">1.36 (1.13&#8211;1.63)</td></tr><tr><td valign="top">10 years and onwards</td><td valign="top">28 (15.56)/44 (6.99)</td><td valign="top">2.50 (1.45&#8211;4.34)</td><td valign="top">83 (10.00)/344 (7.97)</td><td valign="top">1.22 (0.93&#8211;1.60)</td></tr><tr><td valign="top"><italic>By cause of the co-twin&#8217;s death</italic></td><td/><td/><td/><td/></tr><tr><td>Unnatural death</td><td valign="top">28 (21.32)/26 (5.58)</td><td valign="top">4.29 (2.18&#8211;8.45)</td><td valign="top">81 (9.79)/254 (5.97)</td><td valign="top">1.44 (1.09&#8211;1.91)</td></tr><tr><td>Natural death</td><td valign="top">64 (13.07)/127 (7.74)</td><td valign="top">1.46 (1.05&#8211;2.03)</td><td valign="top">222 (11.36)/770 (7.54)</td><td valign="top">1.30 (1.10&#8211;1.54)</td></tr><tr><td><italic>By gender difference</italic></td><td valign="top"/><td valign="top"/><td valign="top"/><td valign="top"/></tr><tr><td>Twins with same gender</td><td valign="top">92 (14.81)/153 (7.26)</td><td valign="top">1.86 (1.40&#8211;2.47)</td><td valign="top">118 (10.51)/240 (6.93)</td><td valign="top">1.43 (1.13&#8211;1.82)</td></tr><tr><td>Twins with different gender</td><td valign="top">-</td><td valign="top">-</td><td valign="top">185 (11.15)/784 (7.13)</td><td valign="top">1.33 (1.10&#8211;1.62)</td></tr><tr><td valign="top">Twin-sibling cohort</td><td valign="top">53 (14.37)/57 (7.24)</td><td valign="top">2.45 (1.56&#8211;3.85)</td><td valign="top">193 (10.73)/279 (8.42)</td><td valign="top">1.29 (1.05&#8211;1.59)</td></tr></tbody></table><table-wrap-foot><fn><p>* Cox regression models were stratified by matching identifiers (birth year and sex, in matched twin cohort) or family identifiers (in twin-sibling cohort), and adjusted for education level, family income, marital status, history of severe somatic diseases, and family history of psychiatric disorder. Time since the index date was used as underlying time scale.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-57264-v2.xml">elife-57264-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Equilibration times (t<sub>equil</sub>) for different affinities and association rate constants.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"><italic>K</italic><sub>D</sub></th><th rowspan="2"><italic>k</italic><sub>on</sub>, M<sup>&#8722;1</sup> s<sup>&#8722;1</sup></th><th colspan="2">t<sub>equil</sub>*</th></tr><tr><th>sec</th><th>hr</th></tr></thead><tbody><tr><td rowspan="3" valign="top">1 &#181;M</td><td valign="top">10<sup>8</sup></td><td valign="bottom">0.04</td><td valign="top"/></tr><tr><td valign="top">10<sup>6</sup></td><td valign="bottom">4</td><td valign="top"/></tr><tr><td valign="top">10<sup>3</sup></td><td valign="bottom"/><td valign="top">1</td></tr><tr><td rowspan="3" valign="top">1 nM</td><td valign="top">10<sup>8</sup></td><td valign="bottom">40</td><td valign="top"/></tr><tr><td valign="top">10<sup>6</sup></td><td valign="bottom"/><td valign="top">1</td></tr><tr><td valign="top">10<sup>3</sup></td><td valign="bottom"/><td valign="top">1000</td></tr><tr><td rowspan="3" valign="top">1 pM</td><td valign="top">10<sup>8</sup></td><td valign="bottom"/><td valign="top">10</td></tr><tr><td valign="top">10<sup>6</sup></td><td valign="bottom"/><td valign="top">1000</td></tr><tr><td valign="top">10<sup>3</sup></td><td valign="bottom"/><td valign="top">1,000,000</td></tr></tbody></table><table-wrap-foot><fn><p>*t<sub>equil</sub> was calculated as five half-lives: t<sub>equil</sub> = 5t<sub>1/2</sub>&#160;=&#160;5&#160;&#215;&#160;0.693/<italic>k</italic><sub>equil</sub>, where <italic>k</italic><sub>equil</sub>&#160;=&#160;<italic>k</italic><sub>off</sub>&#160;=&#160;<italic>K</italic><sub>D&#160;</sub>&#215;&#160;<italic>k</italic><sub>on</sub> (<xref ref-type="disp-formula" rid="equ2">Equation 2</xref> and <xref ref-type="fig" rid="fig3">Figure 3</xref>).</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-57613-v2.xml">elife-57613-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Quantification of PlantSeg performance on the 3D Digital Tissue Atlas, using PlantSeg .</title><p>The Adapted Rand error (ARand) assesses the overall segmentation quality whereas <italic>VOI</italic><sub><italic>merge</italic></sub> and <italic>VOI</italic><sub><italic>split</italic></sub> assess erroneous merge and splitting events. The petal images were not included in our analysis as they are very similar to the leaf and the ground truth is fragmented, making it difficult to evaluate the results from the pipeline in a reproducible way. Segmented images are computed using GASP partitioning with default parameters (left table) and fine-tuned parameters described in Appendix 7: Empirical Example of parameter tuning (right table).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Dataset</th><th colspan="3">PlantSeg (default parameters)</th><th colspan="3">PlantSeg (tuned parameters)</th></tr><tr><th>ARand</th><th><italic>VOI</italic><sub><italic>split</italic></sub></th><th><italic>VOI</italic><sub><italic>merge</italic></sub></th><th>ARand</th><th><italic>VOI</italic><sub><italic>split</italic></sub></th><th><italic>VOI</italic><sub><italic>merge</italic></sub></th></tr></thead><tbody><tr><td>Anther</td><td>0.328</td><td>0.778</td><td>0.688</td><td>0.167</td><td>0.787</td><td>0.399</td></tr><tr><td>Filament</td><td>0.576</td><td>1.001</td><td>1.378</td><td>0.171</td><td>0.687</td><td>0.487</td></tr><tr><td>Leaf</td><td>0.075</td><td>0.353</td><td>0.322</td><td>0.080</td><td>0.308</td><td>0.220</td></tr><tr><td>Pedicel</td><td>0.400</td><td>0.787</td><td>0.869</td><td>0.314</td><td>0.845</td><td>0.604</td></tr><tr><td>Root</td><td>0.248</td><td>0.634</td><td>0.882</td><td>0.101</td><td>0.356</td><td>0.412</td></tr><tr><td>Sepal</td><td>0.527</td><td>0.746</td><td>1.032</td><td>0.257</td><td>0.690</td><td>0.966</td></tr><tr><td>Valve</td><td>0.572</td><td>0.821</td><td>1.315</td><td>0.300</td><td>0.494</td><td>0.875</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-57613-v2.xml">elife-57613-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app7table3" position="float"><label>Appendix 7&#8212;table 3.</label><caption><title>Comparison between 2D vs 3D super pixels.</title><p>From out experiments, segmentation quality is almost always improved by the usage of 3D super pixels. On the other side, the user should be aware that this improvement comes at the cost of a large slow-down of the pipeline (roughly &#215; 4.5 on our system Intel Xenon E5-2660, RAM 252 Gb).</p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="9">ds3 confocal + rescaling</th></tr><tr><th colspan="9">Over/under factor 0.7</th></tr><tr><th rowspan="2">Dataset</th><th colspan="4">Super Pixels 2D (Default)</th><th colspan="4">Super Pixels 3D</th></tr><tr><th>ARand</th><th><italic>VOI</italic><sub><italic>split</italic></sub></th><th><italic>VOI</italic><sub><italic>merge</italic></sub></th><th>time (s)</th><th>ARand</th><th><italic>VOI</italic><sub><italic>split</italic></sub></th><th><italic>VOI</italic><sub><italic>merge</italic></sub></th><th>time (s)</th></tr></thead><tbody><tr><td>Anther</td><td>0.215</td><td>1.130</td><td>0.517</td><td>600</td><td>0.167</td><td>0.787</td><td>0.399</td><td>2310</td></tr><tr><td>Filament</td><td>0.159</td><td>0.899</td><td>0.350</td><td>120</td><td>0.171</td><td>0.687</td><td>0.487</td><td>520</td></tr><tr><td>Leaf</td><td>0.117</td><td>0.502</td><td>0.247</td><td>800</td><td>0.080</td><td>0.308</td><td>0.220</td><td>3650</td></tr><tr><td>Pedicel</td><td>0.339</td><td>1.148</td><td>0.894</td><td>450</td><td>0.314</td><td>0.845</td><td>0.604</td><td>2120</td></tr><tr><td>Root</td><td>0.113</td><td>0.672</td><td>0.485</td><td>210</td><td>0.101</td><td>0.356</td><td>0.412</td><td>920</td></tr><tr><td>Sepal</td><td>0.346</td><td>0.926</td><td>1.211</td><td>770</td><td>0.257</td><td>0.690</td><td>0.966</td><td>3420</td></tr><tr><td>Valve</td><td>0.444</td><td>0.828</td><td>1.138</td><td>530</td><td>0.300</td><td>0.494</td><td>0.875</td><td>2560</td></tr><tr><td>Average</td><td>0.248</td><td>0.872</td><td>0.691</td><td>500</td><td>0.199</td><td>0.595</td><td>0.566</td><td>2210</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-57613-v2.xml">elife-57613-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app7table1" position="float"><label>Appendix 7&#8212;table 1.</label><caption><title>Comparison between the <italic>generic confocal</italic> CNN model (default in PlantSeg), the closest confocal model in terms of xy plant voxels resolution <italic>ds3 confocal</italic> and the combination of <italic>ds3 confocal</italic> and rescaling (in order to mach the training data resolution a rescaling factor of <inline-formula><mml:math id="inf45"><mml:mrow><mml:mo stretchy="false">(</mml:mo><mml:mn>3</mml:mn><mml:mo>,</mml:mo><mml:mn>1</mml:mn><mml:mo>,</mml:mo><mml:mn>1</mml:mn><mml:mo stretchy="false">)</mml:mo></mml:mrow></mml:math></inline-formula> zxy has been used).</title><p>The later combination showed the best overall results. To be noted that <italic>ds3 confocal</italic> was trained on almost isotropic data, while the 3D Digital Tissue Atlas is not isotropic. Therefore poor performances without rescaling are expected. Segmentation obtained with GASP and default parameters</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2">Dataset</th><th colspan="3">Generic confocal (Default)</th><th colspan="3">ds3 confocal</th><th colspan="3">ds3 confocal + rescaling</th></tr><tr><th>ARand</th><th><italic>VOI</italic><sub><italic>split</italic></sub></th><th><italic>VOI</italic><sub><italic>merge</italic></sub></th><th>ARand</th><th><italic>VOI</italic><sub><italic>split</italic></sub></th><th><italic>VOI</italic><sub><italic>merge</italic></sub></th><th>ARand</th><th><italic>VOI</italic><sub><italic>split</italic></sub></th><th><italic>VOI</italic><sub><italic>merge</italic></sub></th></tr></thead><tbody><tr><td>Anther</td><td>0.328</td><td>0.778</td><td>0.688</td><td>0.344</td><td>1.407</td><td>0.735</td><td>0.265</td><td>0.748</td><td>0.650</td></tr><tr><td>Filament</td><td>0.576</td><td>1.001</td><td>1.378</td><td>0.563</td><td>1.559</td><td>1.244</td><td>0.232</td><td>0.608</td><td>0.601</td></tr><tr><td>Leaf</td><td>0.075</td><td>0.353</td><td>0.322</td><td>0.118</td><td>0.718</td><td>0.384</td><td>0.149</td><td>0.361</td><td>0.342</td></tr><tr><td>Pedicel</td><td>0.400</td><td>0.787</td><td>0.869</td><td>0.395</td><td>1.447</td><td>1.082</td><td>0.402</td><td>0.807</td><td>1.161</td></tr><tr><td>Root</td><td>0.248</td><td>0.634</td><td>0.882</td><td>0.219</td><td>1.193</td><td>0.761</td><td>0.123</td><td>0.442</td><td>0.592</td></tr><tr><td>Sepal</td><td>0.527</td><td>0.746</td><td>1.032</td><td>0.503</td><td>1.293</td><td>1.281</td><td>0.713</td><td>0.652</td><td>1.615</td></tr><tr><td>Valve</td><td>0.572</td><td>0.821</td><td>1.315</td><td>0.617</td><td>1.404</td><td>1.548</td><td>0.586</td><td>0.578</td><td>1.443</td></tr><tr><td>Average</td><td>0.389</td><td>0.731</td><td>0.927</td><td>0.394</td><td>1.289</td><td>1.005</td><td>0.353</td><td>0.600</td><td>0.915</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-57613-v2.xml">elife-57613-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="app7table2" position="float"><label>Appendix 7&#8212;table 2.</label><caption><title>Comparison between the results obtained with three different <italic>over/under segmentation factor</italic> <inline-formula><mml:math id="inf46"><mml:mrow><mml:mo stretchy="false">(</mml:mo><mml:mn>0.5</mml:mn><mml:mo>,</mml:mo><mml:mn>0.6</mml:mn><mml:mo>,</mml:mo><mml:mn>0.7</mml:mn><mml:mo stretchy="false">)</mml:mo></mml:mrow></mml:math></inline-formula>.</title><p>The effect of tuning this parameter is mostly reflected in the VOIs scores. In this case the best result have been obtained by steering the segmentation towards the over segmentation.</p></caption><table frame="hsides" rules="groups"><thead><tr><th colspan="10">ds3 confocal + rescaling</th></tr><tr><th rowspan="2">Dataset</th><th colspan="3">Over/under factor 0.5</th><th colspan="3">Over/under factor 0.6 (Default)</th><th colspan="3">Over/under factor 0.7</th></tr><tr><th>ARand</th><th><italic>VOI</italic><sub><italic>split</italic></sub></th><th><italic>VOI</italic><sub><italic>merge</italic></sub></th><th>ARand</th><th><italic>VOI</italic><sub><italic>split</italic></sub></th><th><italic>VOI</italic><sub><italic>merge</italic></sub></th><th>ARand</th><th><italic>VOI</italic><sub><italic>split</italic></sub></th><th><italic>VOI</italic><sub><italic>merge</italic></sub></th></tr></thead><tbody><tr><td>Anther</td><td>0.548</td><td>0.540</td><td>1.131</td><td>0.265</td><td>0.748</td><td>0.650</td><td>0.215</td><td>1.130</td><td>0.517</td></tr><tr><td>Filament</td><td>0.740</td><td>0.417</td><td>1.843</td><td>0.232</td><td>0.608</td><td>0.601</td><td>0.159</td><td>0.899</td><td>0.350</td></tr><tr><td>Leaf</td><td>0.326</td><td>0.281</td><td>0.825</td><td>0.149</td><td>0.361</td><td>0.342</td><td>0.117</td><td>0.502</td><td>0.247</td></tr><tr><td>Pedicel</td><td>0.624</td><td>0.585</td><td>2.126</td><td>0.402</td><td>0.807</td><td>1.161</td><td>0.339</td><td>1.148</td><td>0.894</td></tr><tr><td>Root</td><td>0.244</td><td>0.334</td><td>0.972</td><td>0.123</td><td>0.442</td><td>0.592</td><td>0.113</td><td>0.672</td><td>0.485</td></tr><tr><td>Sepal</td><td>0.904</td><td>0.494</td><td>2.528</td><td>0.713</td><td>0.652</td><td>1.615</td><td>0.346</td><td>0.926</td><td>1.211</td></tr><tr><td>Valve</td><td>0.831</td><td>0.432</td><td>2.207</td><td>0.586</td><td>0.578</td><td>1.443</td><td>0.444</td><td>0.828</td><td>1.138</td></tr><tr><td>Average</td><td>0.602</td><td>0.441</td><td>1.662</td><td>0.353</td><td>0.600</td><td>0.915</td><td>0.248</td><td>0.872</td><td>0.691</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-57670-v1.xml">elife-57670-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Number of recording experiments in each animal.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th rowspan="2" valign="top">Recording site</th><th colspan="5" valign="top">Number of recording sessions</th></tr><tr><th>Dexmedetomidine</th><th>Dexmedetomidine and antagonist</th><th>Arousability testing</th><th>No task control</th><th>Blind-folding control</th></tr></thead><tbody><tr><td>Monkey 1</td><td>S1, S2, PMv</td><td>8</td><td>2</td><td>0</td><td>2</td><td>0</td></tr><tr><td>Monkey 2</td><td>S1, PMv</td><td>9</td><td>3</td><td>2</td><td>2</td><td>2</td></tr></tbody></table><table-wrap-foot><fn><p>Note: The recording sessions were performed with a minimum interval of 2 days in each animal.</p></fn></table-wrap-foot></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-57814-v1.xml">elife-57814-v1.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>mRNAs of conserved ATP synthetase components are localized to mitochondria in respiratory and CHX conditions.</title><p>ATP complex gene conservation was obtained from published annotation&#160;(<xref ref-type="bibr" rid="bib42">R&#252;hle and Leister, 2015</xref>) and we obtained data of CHX-dependent mitochondrial localization from proximity ribosomal profiling data&#160;(<xref ref-type="bibr" rid="bib56">Williams et al., 2014</xref>). The CHX-dependent localization data were consistent with our study. Respiratory-specific localization was determined in this study.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"/><th rowspan="2">Complex</th><th rowspan="2">Subunit</th><th rowspan="2">Gene</th><th colspan="2">Mitochondrial localization</th><th rowspan="2">Respiratory specific localization</th></tr><tr><th>-CHX</th><th>+CHX</th></tr></thead><tbody><tr><td rowspan="6">Conserved from bacteria to eukaryote</td><td rowspan="5">F1</td><td>&#945;</td><td><italic>ATP1</italic></td><td>no</td><td>yes</td><td>yes</td></tr><tr><td>&#946;</td><td><italic>ATP2</italic></td><td>no</td><td>yes</td><td>yes</td></tr><tr><td>&#947;</td><td><italic>ATP3</italic></td><td>no</td><td>yes</td><td>yes</td></tr><tr><td>OSCP/&#948;</td><td><italic>ATP5</italic></td><td>no</td><td>yes</td><td>NA</td></tr><tr><td>&#949;</td><td><italic>ATP16</italic></td><td>no</td><td>no</td><td>NA</td></tr><tr><td>F0</td><td>b</td><td><italic>ATP4</italic></td><td>no</td><td>yes</td><td>NA</td></tr><tr><td>non-conserved</td><td colspan="3">all the others</td><td>no</td><td>no</td><td>NA&#160;(<italic>ATP7</italic>: no)</td></tr><tr><td><xref ref-type="bibr" rid="bib42">R&#252;hle and Leister, 2015</xref></td><td/><td/><td/><td colspan="2"><xref ref-type="bibr" rid="bib56">Williams et al., 2014</xref></td><td>This study</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-58728-v2.xml">elife-58728-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table5" position="float"><label>Table 5.</label><caption><title>PPE protocols (&#8216;Scenarios&#8217;) and their relation to ward category.</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th colspan="4">PPE &#8216;Scenarios&#8217;</th></tr><tr><th valign="top">Scenario 0</th><th valign="top">Scenario 1</th><th valign="top">Scenario 2</th><th valign="top">Scenario 3</th></tr></thead><tbody><tr><td>Area description</td><td valign="top">All clinical areas without any known or suspected COVID-19 cases</td><td valign="top">Designated ward, triage and assessment-based care with suspected or confirmed COVID-19 patients</td><td valign="top">Cohorted areas where aerosol-generating procedures are carried out frequently with suspected or confirmed COVID-19 patients</td><td valign="top">Operating theatres where procedures are performed with suspected or confirmed COVID-19 patients</td></tr><tr><td valign="top">PPE description</td><td valign="top">Fluid resistant face mask at all times, apron and non-sterile gloves for patient contact (within two metres)</td><td valign="top">Surgical scrubs, fluid resistant face mask, theatre cap, eye protection, apron and non-sterile gloves</td><td valign="top">Water repellent gown, FFP3 mask, eye protection, theatre cap, surgical gloves, with an apron and non-sterile gloves in addition for patient contact (within two metres)</td><td valign="top">Water repellent gown, FFP3 mask, eye protection, theatre cap and surgical gloves</td></tr><tr><td valign="top">Ward categories</td><td valign="top">Green wards, <break/>for&#160;example designated areas of emergency department and medical admissions unit. Medical, surgical and haematology wards/outpatient clinics.</td><td valign="top">Amber + red wards, <break/>for&#160;example designated areas of emergency department and medical admissions unit. Designated CoVID-19 confirmed wards.</td><td valign="top">Amber + red wards, <break/>for&#160;example intensive care unit, respiratory units with non-invasive ventilation facilities.</td><td valign="top">All operating theatres, including facilities for bronchoscopy and endoscopy.</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-58807-v2.xml">elife-58807-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Proportion of women authors on 2019 papers and COVID-19 papers by specialty.</title><p>Number of observations, <italic>N</italic>, and proportion of women by author list position for journals grouped by their specialty. The grouped columns show results by journal specialty for COVID papers published in 2020 (four rightmost columns) in contrast to papers from the same journals in 2019. Only papers with a US-based first and/or last author and clear gender for first and last author are included.</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" valign="top"><italic>Journal specialty</italic></th><th align="center" colspan="4" valign="bottom">2019 papers</th><th align="center" colspan="4" valign="bottom">COVID-19 papers</th></tr><tr><th rowspan="2" valign="top"><italic>N</italic></th><th align="center" colspan="3" valign="bottom">Proportion of women</th><th rowspan="2" valign="top"><italic>N</italic></th><th align="center" colspan="3" valign="bottom">Proportion of women</th></tr><tr><th valign="top"><italic>First author</italic></th><th valign="top"><italic>Full group</italic></th><th valign="top"><italic>Last author</italic></th><th valign="top"><italic>First author</italic></th><th valign="top"><italic>Full group</italic></th><th valign="top"><italic>Last author</italic></th></tr></thead><tbody><tr><td valign="top">Dermatology</td><td valign="top">1811</td><td valign="top">0.44</td><td valign="top">0.42</td><td valign="top">0.37</td><td valign="top">72</td><td valign="top">0.46</td><td valign="top">0.41</td><td valign="top">0.31</td></tr><tr><td valign="top">Emergency medicine</td><td valign="top">1283</td><td valign="top">0.32</td><td valign="top">0.30</td><td valign="top">0.22</td><td valign="top">54</td><td valign="top">0.31</td><td valign="top">0.25</td><td valign="top">0.13</td></tr><tr><td valign="top">High impact general medicine</td><td valign="top">7142</td><td valign="top">0.41</td><td valign="top">0.42</td><td valign="top">0.39</td><td valign="top">194</td><td valign="top">0.31</td><td valign="top">0.37</td><td valign="top">0.35</td></tr><tr><td valign="top">Infectious diseases</td><td valign="top">1404</td><td valign="top">0.45</td><td valign="top">0.42</td><td valign="top">0.34</td><td valign="top">44</td><td valign="top">0.20</td><td valign="top">0.32</td><td valign="top">0.34</td></tr><tr><td valign="top">Internal medicine</td><td valign="top">19,980</td><td valign="top">0.36</td><td valign="top">0.33</td><td valign="top">0.25</td><td valign="top">484</td><td valign="top">0.33</td><td valign="top">0.32</td><td valign="top">0.24</td></tr><tr><td valign="top">Other basic sciences</td><td valign="top">6975</td><td valign="top">0.42</td><td valign="top">0.38</td><td valign="top">0.29</td><td valign="top">135</td><td valign="top">0.33</td><td valign="top">0.34</td><td valign="top">0.28</td></tr><tr><td valign="top">Other clinical sciences</td><td valign="top">21,869</td><td valign="top">0.40</td><td valign="top">0.37</td><td valign="top">0.31</td><td valign="top">429</td><td valign="top">0.38</td><td valign="top">0.38</td><td valign="top">0.35</td></tr><tr><td valign="top">Otolaryngology</td><td valign="top">1063</td><td valign="top">0.32</td><td valign="top">0.29</td><td valign="top">0.21</td><td valign="top">106</td><td valign="top">0.28</td><td valign="top">0.29</td><td valign="top">0.24</td></tr><tr><td valign="top">Pathology</td><td valign="top">869</td><td valign="top">0.46</td><td valign="top">0.43</td><td valign="top">0.32</td><td valign="top">66</td><td valign="top">0.27</td><td valign="top">0.37</td><td valign="top">0.30</td></tr><tr><td valign="top">Public health</td><td valign="top">11,015</td><td valign="top">0.47</td><td valign="top">0.41</td><td valign="top">0.35</td><td valign="top">99</td><td valign="top">0.33</td><td valign="top">0.41</td><td valign="top">0.37</td></tr><tr><td valign="top">Radiology</td><td valign="top">2262</td><td valign="top">0.37</td><td valign="top">0.33</td><td valign="top">0.27</td><td valign="top">60</td><td valign="top">0.25</td><td valign="top">0.28</td><td valign="top">0.17</td></tr><tr><td valign="top">Surgery</td><td valign="top">9700</td><td valign="top">0.21</td><td valign="top">0.20</td><td valign="top">0.13</td><td valign="top">186</td><td valign="top">0.26</td><td valign="top">0.22</td><td valign="top">0.16</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-59038-v2.xml">elife-59038-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>IC<sub>50</sub> values of ENaC for three different blockers (amiloride, phenamil mesylate and benzamil).</title><p>IC<sub>50</sub> values (mean&#160;&#177;&#160;S.E.M) determined from dose-response curves for three different blockers (amiloride, phenamil mesylate and benzamil) at different holding voltages (-60 mV, -40 mV, -20 mV, 0 mV).</p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="top"/><th valign="top"/><th valign="top">IC50 values (nM)</th><th valign="top"/></tr><tr><th valign="top">Amiloride</th><th valign="top">Phenamil</th><th valign="top">Benzamil</th></tr></thead><tbody><tr><td valign="top">0 mV <break/>-20 mV <break/>-40 mV <break/>-60 mV</td><td valign="top">97.14&#160;&#177;&#160;21.62 <break/>80.05<italic>&#160;&#177;</italic>&#160;8.78 <break/>80.25&#160;<italic>&#177;</italic>&#160;11.37 <break/>86.34&#160;<italic>&#177;</italic>&#160;27.04</td><td valign="top">51.37&#160;&#177;&#160;10.42 <break/>49.97&#160;&#177;&#160;11.18 <break/>43.37&#160;&#177;&#160;11.86 <break/>51.01&#160;&#177;&#160;14.12</td><td valign="top">36.74&#160;&#177;&#160;13.25 <break/>29.41<italic>&#160;&#177;</italic>&#160;6.47 <break/>27.72<italic>&#160;&#177;</italic>&#160;6.65 <break/>32.90&#160;&#177;&#160;12.66</td></tr></tbody></table></table-wrap><br><h2><a href="https://github.com/elifesciences/elife-article-xml/tree/master/articles/elife-59153-v2.xml">elife-59153-v2.xml</a></h2><table-wrap xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:xlink="http://www.w3.org/1999/xlink" id="table1" position="float"><label>Table 1.</label><caption><title>Prenatal and study baseline (6&#8211;9 weeks postpartum)&#160;characteristics of women with gestational diabetes mellitus in the&#160;SWIFT&#160;cohort (n&#160;=&#160;350).</title></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2" valign="bottom">Prenatal characteristics</th><th valign="bottom">Case <break/>Diabetes at follow up</th><th valign="bottom">Control <break/>No Diabetes at follow up</th><th rowspan="2" valign="bottom">p-value</th></tr><tr><th valign="bottom">(N&#160;=&#160;171)</th><th valign="bottom">(N&#160;=&#160;179)</th></tr></thead><tbody><tr><td valign="bottom">Age (years), Mean (SD)</td><td valign="bottom">33.3 (5.2)</td><td valign="bottom">33.0 (4.5)</td><td valign="bottom">0.63</td></tr><tr><td valign="bottom">Race/ethnicity, n (%)</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">0.72</td></tr><tr><td valign="bottom">&#8195;White</td><td valign="bottom">31 (18.1)</td><td valign="bottom">27 (15.1)</td><td valign="bottom"/></tr><tr><td valign="bottom">&#8195;Asian</td><td valign="bottom">51 (29.8)</td><td valign="bottom">55 (30.7)</td><td valign="bottom"/></tr><tr><td valign="bottom">&#8195;Black</td><td valign="bottom">21 (12.3)</td><td valign="bottom">16 (8.9)</td><td valign="bottom"/></tr><tr><td valign="bottom">&#8195;Hispanic</td><td valign="bottom">66 (38.6)</td><td valign="bottom">79 (44.1)</td><td valign="bottom"/></tr><tr><td valign="bottom">&#8195;Other</td><td valign="bottom">2 (1.2)</td><td valign="bottom">2 (1.1)</td><td valign="bottom"/></tr><tr><td valign="bottom">Parity, n (%)</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">0.80</td></tr><tr><td valign="bottom">&#8195;Primiparous (one birth)</td><td valign="bottom">56 (32.7)</td><td valign="bottom">54 (30.2)</td><td valign="bottom"/></tr><tr><td valign="bottom">&#8195;Biparous (two births)</td><td valign="bottom">62 (36.3)</td><td valign="bottom">64 (35.8)</td><td valign="bottom"/></tr><tr><td valign="bottom">&#8195;Multiparous (&gt;2 births)</td><td valign="bottom">53 (31.0)</td><td valign="bottom">61 (34.1)</td><td valign="bottom"/></tr><tr><td valign="bottom">GDM treatment, n (%)</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom">&lt;0.001</td></tr><tr><td valign="bottom">&#8195;Diet only</td><td valign="bottom">74 (43.3)</td><td valign="bottom">128 (71.5)</td><td valign="bottom"/></tr><tr><td valign="bottom">&#8195;Oral medications</td><td valign="bottom">79 (46.2)</td><td valign="bottom">47 (26.3)</td><td valign="bottom"/></tr><tr><td valign="bottom">&#8195;Insulin</td><td valign="bottom">18 (10.5)</td><td valign="bottom">4 (2.3)</td><td valign="bottom"/></tr><tr><td valign="bottom">Pre-pregnancy BMI (kg/m<sup>2</sup>), Mean (SD)</td><td valign="bottom">33.6 (8.2)</td><td valign="bottom">32.3 (6.9)</td><td valign="bottom">0.10</td></tr><tr><td valign="bottom">Sum of Prenatal 3 hr 100 g OGTT glucose z-scores, Mean (SD)</td><td valign="bottom">1.4 (3.1)</td><td valign="bottom">&#8722;0.2 (2.5)</td><td valign="bottom">&lt;0.001</td></tr><tr><td valign="bottom">Family history of diabetes, n (%)</td><td valign="bottom">101 (59.1)</td><td valign="bottom">89 (52.0)</td><td valign="bottom">0.08</td></tr><tr><td valign="bottom">Baseline characteristics at 6&#8211;9 weeks Postpartum</td><td valign="bottom"/><td valign="bottom"/><td valign="bottom"/></tr><tr><td valign="bottom">BMI (kg/m<sup>2</sup>), Mean (SD)</td><td valign="bottom">33.5 (7.4)</td><td valign="bottom">32.4 (6.3)</td><td valign="bottom">0.18</td></tr><tr><td valign="bottom">Fasting plasma glucose (FPG), mg/dl, Mean (SD)</td><td valign="bottom">101.5 (10.4)</td><td valign="bottom">94.3 (7.7)</td><td valign="bottom">&lt;0.001</td></tr><tr><td valign="bottom">2 hr Post-load plasma glucose (75 g OGTT), mg/dl, Mean (SD)</td><td valign="bottom">131.0 (29.5)</td><td valign="bottom">109.8 (27.4)</td><td valign="bottom">&lt;0.001</td></tr><tr><td valign="bottom">Fasting insulin, &#956;U/ml, Median (IQR)</td><td valign="bottom">26.5 (20.7)</td><td valign="bottom">22.1 (17.4)</td><td valign="bottom">0.001</td></tr><tr><td valign="bottom">2 hr insulin, &#956;U/ml, Median (IQR)</td><td valign="bottom">111.5 (85.7)</td><td valign="bottom">83.3 (73.6)</td><td valign="bottom">&lt;0.001</td></tr><tr><td valign="bottom">Fasting plasma Triglycerides, mg/dl, Median (IQR)</td><td valign="bottom">119.0 (103.0)</td><td valign="bottom">94.0 (72.0)</td><td valign="bottom">0.003</td></tr><tr><td valign="bottom">Fasting plasma HDL-C, mg/dl, Mean (SD)</td><td valign="bottom">49.0 (16.0)</td><td valign="bottom">52.0 (19.0)</td><td valign="bottom">0.017</td></tr><tr><td valign="bottom">Fasting plasma Total Cholesterol, Mean (SD)</td><td valign="bottom">199.4 (34.5)</td><td valign="bottom">203.5 (35.5)</td><td valign="bottom">0.27</td></tr><tr><td valign="bottom">Fasting plasma LDL-C, Mean (SD)</td><td valign="bottom">121.0 (31.1)</td><td valign="bottom">126.4 (31.2)</td><td valign="bottom">0.10</td></tr><tr><td valign="bottom">HOMA-IR, Median (IQR)</td><td valign="bottom">6.8 (5.6)</td><td valign="bottom">5.0 (4.3)</td><td valign="bottom">&lt;0.001</td></tr><tr><td valign="bottom">HOMA-B, Median (IQR)</td><td valign="bottom">268.1 (192.1)</td><td valign="bottom">256.0 (176.2)</td><td valign="bottom">0.61</td></tr><tr><td valign="bottom">Hypertension, n (%)</td><td valign="bottom">14 (8.2)</td><td valign="bottom">5 (2.8)</td><td valign="bottom">0.04</td></tr><tr><td valign="bottom">Smoker, n (%)</td><td valign="bottom">5 (2.9)</td><td valign="bottom">4 (2.2)</td><td valign="bottom">0.68</td></tr><tr><td valign="bottom">Dietary glycemic index, Mean (SD)</td><td valign="bottom">242.5 (106.7)</td><td valign="bottom">246.5 (112.5)</td><td valign="bottom">0.73</td></tr><tr><td valign="bottom">Dietary Intake, Percentage of Kcal as animal fat, Mean SD</td><td valign="bottom">27.0 (7.7)</td><td valign="bottom">25.6 (8.6)</td><td valign="bottom">0.10</td></tr><tr><td valign="bottom">Physical activity score, met-hrs per week, Mean (SD)</td><td valign="bottom">50.7 (23.4)</td><td valign="bottom">47.4 (20.6)</td><td valign="bottom">0.16</td></tr></tbody></table><table-wrap-foot><fn><p>Variables obtained from the SWIFT Study that administered the research 2 hr 75 g OGTTs and other assessments at in-person research visits (baseline). Participants did not have diabetes at study baseline and underwent annual 2 hr 75 g OGTTs at baseline and annually for two years, and thereafter evaluated for diabetes onset from electronic medical records. P-values are for incident diabetes case versus no diabetes controls at follow-up.</p></fn></table-wrap-foot></table-wrap>
            </body>
        </html>
    